{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "LSTM_classification_student.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/kevaniy/mon-nouveau-blog/blob/master/LSTM_classification_title.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "o8JC4LYa8jjE"
      },
      "source": [
        "# Classification de texte en deep learning (LSTM et convolution) \n",
        "\n",
        "## But de la tâche \n",
        "\n",
        "A partir d'un dataset d'articles PUBMED, l'objectif est de construire un modèle utilisant les titres et abstracts des articles pour prédire la\n",
        "catégorie SIGAPS associée à l'article . \n",
        "\n",
        "Après une phase de préprocessing du texte, nous entrainerons un modèle à base de convolutions, puis un modèle à base de réseau de neurones récurrents (LSTM) "
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "vAervZV0djbe"
      },
      "source": [
        "## Cloner le repo https://github.com/aneuraz/intro-keras.git"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rjN4D9A9EGB3",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "e9b286b1-80e0-4d9d-ecff-d428c1f6377c"
      },
      "source": [
        "!git clone https://github.com/aneuraz/intro-keras.git"
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Cloning into 'intro-keras'...\n",
            "remote: Enumerating objects: 18, done.\u001b[K\n",
            "remote: Counting objects: 100% (18/18), done.\u001b[K\n",
            "remote: Compressing objects: 100% (16/16), done.\u001b[K\n",
            "remote: Total 18 (delta 5), reused 0 (delta 0), pack-reused 0\u001b[K\n",
            "Unpacking objects: 100% (18/18), done.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "pFIznPvCc6o_"
      },
      "source": [
        "## Import des libraries"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1TrZgMq4fReH"
      },
      "source": [
        "%tensorflow_version 2.x\n",
        "import json \n",
        "import tensorflow as tf\n",
        "import numpy as np"
      ],
      "execution_count": 16,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "oq5LGnyzduPx"
      },
      "source": [
        "## Chargement des données\n",
        "\n",
        "Toutes les données chargées se situent dans le répertoire `/content/`.\n",
        "Les données sont dans un fichier JSON."
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tYQ5UeLqfnM5"
      },
      "source": [
        "with open('/content/intro-keras/ai_pub_samp.json','r') as f:\n",
        "  data = json.load(f)"
      ],
      "execution_count": 17,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "GdA7ubLh5e0_",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "aa32259c-d9ae-4a57-b3bb-dc56e4089b77"
      },
      "source": [
        "len(data)"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "10000"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 18
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "m_PoXhiLfyAD",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "778e984d-8473-4fba-978c-0af6dada0e47"
      },
      "source": [
        "data\r\n"
      ],
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[{'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '1873-3557',\n",
              "  'IF_2013': '2.129',\n",
              "  'IF_2014': '2.353',\n",
              "  'IF_2015': '2.653',\n",
              "  'IF_2016': '2.536',\n",
              "  'IF_2017': '2.88',\n",
              "  'ISSN': '1386-1425',\n",
              "  'ISSN_online': '1873-3557',\n",
              "  'ISSN_print': '1386-1425',\n",
              "  'IsoAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'JrId': 20555,\n",
              "  'MedAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'NLMid': '9602533',\n",
              "  'Titre': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'abstract': 'In this research, ZnO nanoparticle loaded on activated carbon (ZnO-NPs-AC) was synthesized simply by a low cost and nontoxic procedure. The characterization and identification have been completed by different techniques such as SEM and XRD analysis. A three layer artificial neural network (ANN) model is applicable for accurate prediction of dye removal percentage from aqueous solution by ZnO-NRs-AC following conduction of 270 experimental data. The network was trained using the obtained experimental data at optimum pH with different ZnO-NRs-AC amount (0.005-0.015 g) and 5-40 mg/L of sunset yellow dye over contact time of 0.5-30 min. The ANN model was applied for prediction of the removal percentage of present systems with Levenberg-Marquardt algorithm (LMA), a linear transfer function (purelin) at output layer and a tangent sigmoid transfer function (tansig) in the hidden layer with 6 neurons. The minimum mean squared error (MSE) of 0.0008 and coefficient of determination (R(2)) of 0.998 were found for prediction and modeling of SY removal. The influence of parameters including adsorbent amount, initial dye concentration, pH and contact time on sunset yellow (SY) removal percentage were investigated and optimal experimental conditions were ascertained. Optimal conditions were set as follows: pH, 2.0; 10 min contact time; an adsorbent dose of 0.015 g. Equilibrium data fitted truly with the Langmuir model with maximum adsorption capacity of 142.85 mg/g for 0.005 g adsorbent. The adsorption of sunset yellow followed the pseudo-second-order rate equation.',\n",
              "  'authors': ['Maghsoudi, M',\n",
              "   'Ghaedi, M',\n",
              "   'Zinali, A',\n",
              "   'Ghaedi, A M',\n",
              "   'Habibi, M H'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy',\n",
              "  'keywords': ['Adsorption',\n",
              "   'Algorithms',\n",
              "   'Azo Compounds',\n",
              "   'Charcoal',\n",
              "   'Coloring Agents',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Kinetics',\n",
              "   'Microscopy, Electron, Scanning',\n",
              "   'Nanotubes',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Time Factors',\n",
              "   'Water Pollutants, Chemical',\n",
              "   'X-Ray Diffraction',\n",
              "   'Zinc Oxide'],\n",
              "  'pmid': '24995412',\n",
              "  'title': 'Artificial neural network (ANN) method for modeling of sunset yellow dye adsorption using zinc oxide nanorods loaded on activated carbon: Kinetic and isotherm study.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.925',\n",
              "  'IF_2014': '0.986',\n",
              "  'IF_2015': '1.015',\n",
              "  'IF_2016': '1.048',\n",
              "  'IF_2017': '1.008',\n",
              "  'ISSN': '1790-5427',\n",
              "  'ISSN_print': '1790-5427',\n",
              "  'IsoAbbr': 'Hell J Nucl Med',\n",
              "  'JrId': 32588,\n",
              "  'MedAbbr': 'Hell J Nucl Med',\n",
              "  'NLMid': '101257471',\n",
              "  'Titre': 'Hell J Nucl Med',\n",
              "  'abstract': 'Different techniques have been introduced for full automatic processing of myocardial perfusion imaging with single-photon emission tomography. We tried to evaluate the accuracy of one of these commercial automatic processing methods. The study was performed in 300 patients during 18 months. Two post-stress acquisitions in supine and prone positions and one acquisition at rest , were performed for every patient. All images were processed both automatically and interactively and the results were compared to each other. The automatic method matched the interactive method in 95.22% for left ventricle isolation, in 99.54% for excluding extra-cardiac activity and in 98.22% for reorientation of a single projected data. The automatic method was also successful in complete processing of 81.33% sets of stress (supine)-rest and 79.77% sets of stress (prone)-rest images as compared to the interactive method. In conclusion, the fully automated processing method matches the interactive method in complete processing of myocardial perfusion imaging with single-photon emission tomography more than 79.77% and is of equal accuracy to the interactive method in supine and prone-positioned acquisitions.',\n",
              "  'authors': ['Malek, Hadi',\n",
              "   'Bitarafan-Rajabi, Ahmad',\n",
              "   'Rastgoo, Fereidoon',\n",
              "   'Yaghoobi, Nahid',\n",
              "   'Firoozabadi, Hasan'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Hellenic journal of nuclear medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Artery Disease',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, Emission-Computed, Single-Photon',\n",
              "   'User-Computer Interface',\n",
              "   'Ventricular Dysfunction, Left'],\n",
              "  'pmid': '19330174',\n",
              "  'title': 'Correlation of automatic processing method in myocardial perfusion SPET with the interactive method.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1423-0380',\n",
              "  'IF_2013': '2.84',\n",
              "  'IF_2014': '3.611',\n",
              "  'IF_2015': '2.926',\n",
              "  'IF_2016': '3.65',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '1010-4283',\n",
              "  'ISSN_online': '1423-0380',\n",
              "  'ISSN_print': '1010-4283',\n",
              "  'IsoAbbr': 'Tumour Biol.',\n",
              "  'JrId': 7341,\n",
              "  'MedAbbr': 'Tumour Biol',\n",
              "  'NLMid': '8409922',\n",
              "  'Titre': 'Tumour Biol.',\n",
              "  'abstract': 'The aim of this study was to investigate the differences in the gene expression profiles of radiation-sensitive (RS) and radiation-resistant (RR) sinonasal squamous cell carcinoma (SNSCC) and to identify prognostic markers for the radiation reaction of SNSCC. We first examined the differentially expressed genes (DEGs) in RS and RR SNSCC tissues by analyzing clinical samples with GeneChip Human Transcriptome Array 2.0 (HTA 2.0).To understand the functional significance of the molecular changes, we examined the DEGs with Gene Ontology (GO) and pathway analyses to identify the core genes. The expression of several core genes (CCND2, COL5A2, GADD45B, and THBS2) was confirmed with reverse transcription quantitative PCR (RT-qPCR) in a larger series of tissues. We identified 208 DEGs, of which 76 were upregulated and 132 downregulated in the RS tissues relative to the RR tissues. The DEGs were mainly involved in the regulation of cell proliferation, the NF-kappaB signaling pathway, the cell adhesion molecule signaling pathway, and the extracellular matrix-receptor interaction signaling pathway. RT-qPCR confirmed that the CCND2, COL5A2, GADD45B, and THBS2 genes were significantly differentially expressed in the RS and RR tissues, consistent with the GeneChip data. These results extend our understanding of the molecular mechanisms underlying the sensitivity of SNSCC to radiation. The DEGs are involved in the differential response to radiation therapy and the dysregulated core genes identified in this study can be used to predict radiation sensitivity in SNSCC.',\n",
              "  'authors': ['Yan, Li',\n",
              "   'Zhan, Cheng',\n",
              "   'Wang, Shengzi',\n",
              "   'Wang, Shuyi',\n",
              "   'Guo, Luo'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Carcinoma, Squamous Cell',\n",
              "   'Cell Proliferation',\n",
              "   'Female',\n",
              "   'Gene Ontology',\n",
              "   'Head and Neck Neoplasms',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'NF-kappa B',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Paranasal Sinus Neoplasms',\n",
              "   'Real-Time Polymerase Chain Reaction',\n",
              "   'Squamous Cell Carcinoma of Head and Neck',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27164935',\n",
              "  'title': 'Genetic analysis of radiation-specific biomarkers in sinonasal squamous cell carcinomas.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1559-1182',\n",
              "  'IF_2013': '5.286',\n",
              "  'IF_2014': '5.137',\n",
              "  'IF_2015': '5.397',\n",
              "  'IF_2016': '6.19',\n",
              "  'IF_2017': '5.076',\n",
              "  'ISSN': '0893-7648',\n",
              "  'ISSN_online': '1559-1182',\n",
              "  'ISSN_print': '0893-7648',\n",
              "  'IsoAbbr': 'Mol. Neurobiol.',\n",
              "  'JrId': 1466,\n",
              "  'MedAbbr': 'Mol Neurobiol',\n",
              "  'NLMid': '8900963',\n",
              "  'Titre': 'Mol. Neurobiol.',\n",
              "  'abstract': 'Although the transcription factor serum response factor (SRF) has been suggested to play a role in activity-dependent gene expression and mediate plasticity-associated structural changes in the hippocampus, no unequivocal evidence has been provided for its role in brain pathology, such as epilepsy. A genome-wide program of activity-induced genes that are regulated by SRF also remains unknown. In the present study, we show that the inducible and conditional deletion of SRF in the adult mouse hippocampus increases the epileptic phenotype in the kainic acid model of epilepsy, reflected by more severe and frequent seizures. Moreover, we observe a robust decrease in activity-induced gene transcription in SRF knockout mice. We characterize the genetic program controlled by SRF in neurons and using functional annotation, we find that SRF target genes are associated with synaptic plasticity and epilepsy. Several of these SRF targets function as regulators of inhibitory or excitatory balance and the structural plasticity of neurons. Interestingly, mutations in those SRF targets have found to be associated with such human neuropsychiatric disorders, as autism and intellectual disability. We also identify novel direct SRF targets in hippocampus: Npas4, Gadd45g, and Zfp36. Altogether, our data indicate that proteins that are highly upregulated by neuronal stimulation, identified in the present study as SRF targets, may function as endogenous protectors against overactivation. Thus, the lack of these effector proteins in SRF knockout animals may lead to uncontrolled excitation and eventually epilepsy.',\n",
              "  'authors': ['Kuzniewska, Bozena',\n",
              "   'Nader, Karolina',\n",
              "   'Dabrowski, Michal',\n",
              "   'Kaczmarek, Leszek',\n",
              "   'Kalita, Katarzyna'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Molecular neurobiology',\n",
              "  'keywords': ['Animals',\n",
              "   'Basic Helix-Loop-Helix Transcription Factors',\n",
              "   'Epilepsy',\n",
              "   'Gene Deletion',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Hippocampus',\n",
              "   'Intracellular Signaling Peptides and Proteins',\n",
              "   'Kainic Acid',\n",
              "   'Mice',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Mice, Knockout',\n",
              "   'Nerve Tissue Proteins',\n",
              "   'Neuronal Plasticity',\n",
              "   'Neurons',\n",
              "   'Serum Response Factor',\n",
              "   'Tristetraprolin'],\n",
              "  'pmid': '25636686',\n",
              "  'title': 'Adult Deletion of SRF Increases Epileptogenesis and Decreases Activity-Induced Gene Expression.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Radiomics is one such \"big data\" approach that applies advanced image refining/data characterization algorithms to generate imaging features that can quantitatively classify tumor phenotypes in a non-invasive manner. We hypothesize that certain textural features of oropharyngeal cancer (OPC) primary tumors will have statistically significant correlations to patient outcomes such as local control. Patients from an IRB-approved database dispositioned to (chemo)radiotherapy for locally advanced OPC were included in this retrospective series. Pretreatment contrast CT scans were extracted and radiomics-based analysis of gross tumor volume of the primary disease (GTVp) were performed using imaging biomarker explorer (IBEX) software that runs in Matlab platform. Data set was randomly divided into a training dataset and test and tuning holdback dataset. Machine learning methods were applied to yield a radiomic signature consisting of features with minimal overlap and maximum prognostic significance. The radiomic signature was adapted to discriminate patients, in concordance with other key clinical prognosticators. 465 patients were available for analysis. A signature composed of 2 radiomic features from pre-therapy imaging was derived, based on the Intensity Direct and Neighbor Intensity Difference methods. Analysis of resultant groupings showed robust discrimination of recurrence probability and Kaplan-Meier-estimated local control rate (LCR) differences between \"favorable\" and \"unfavorable\" clusters were noted.',\n",
              "  'authors': [''],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Oropharyngeal Neoplasms',\n",
              "   'Prognosis',\n",
              "   'Retrospective Studies',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '29367653',\n",
              "  'title': 'Investigation of radiomic signatures for local recurrence using primary tumor texture analysis in oropharyngeal head and neck cancer patients.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2054-5703',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '2.243',\n",
              "  'IF_2017': '2.504',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2054-5703',\n",
              "  'IsoAbbr': 'R Soc Open Sci',\n",
              "  'JrId': 43286,\n",
              "  'MedAbbr': 'R Soc Open Sci',\n",
              "  'NLMid': '101647528',\n",
              "  'Titre': 'R Soc Open Sci',\n",
              "  'abstract': \"Students are characterized according to their own distinct learning styles. Discovering students' learning style is significant in the educational system in order to provide adaptivity. Past researches have proposed various approaches to detect the students' learning styles. Among all, the Bayesian network has emerged as a widely used method to automatically detect students' learning styles. On the other hand, tree augmented naive Bayesian network has the ability to improve the naive Bayesian network in terms of better classification accuracy. In this paper, we evaluate the performance of the tree augmented naive Bayesian in automatically detecting students' learning style in the online learning environment. The experimental results are promising as the tree augmented naive Bayes network is shown to achieve higher detection accuracy when compared to the Bayesian network.\",\n",
              "  'authors': ['Li, Ling Xiao', 'Abdul Rahman, Siti Soraya'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Royal Society open science',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30109052',\n",
              "  'title': \"Students' learning style detection using tree augmented naive Bayes.\",\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Similar to intelligent multicellular neural networks controlling human brains, even single cells, surprisingly, are able to make intelligent decisions to classify several external stimuli or to associate them. This happens because of the fact that gene regulatory networks can perform as perceptrons, simple intelligent schemes known from studies on Artificial Intelligence. We study the role of genetic noise in intelligent decision making at the genetic level and show that noise can play a constructive role helping cells to make a proper decision. We show this using the example of a simple genetic classifier able to classify two external stimuli. ',\n",
              "  'authors': ['Bates, Russell',\n",
              "   'Blyuss, Oleg',\n",
              "   'Alsaedi, Ahmed',\n",
              "   'Zaikin, Alexey'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Signal-To-Noise Ratio'],\n",
              "  'pmid': '25965584',\n",
              "  'title': 'Effect of noise in intelligent cellular decision making.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1545-4126',\n",
              "  'IF_2013': '22.66',\n",
              "  'IF_2014': '19.32',\n",
              "  'IF_2015': '14.265',\n",
              "  'IF_2016': '15.63',\n",
              "  'IF_2017': '14.675',\n",
              "  'ISSN': '0147-006X',\n",
              "  'ISSN_online': '1545-4126',\n",
              "  'ISSN_print': '0147-006X',\n",
              "  'IsoAbbr': 'Annu. Rev. Neurosci.',\n",
              "  'JrId': 635,\n",
              "  'MedAbbr': 'Annu Rev Neurosci',\n",
              "  'NLMid': '7804039',\n",
              "  'Titre': 'Annu. Rev. Neurosci.',\n",
              "  'abstract': 'The ability of the human brain to learn is exceptional. Yet, learning is typically quite specific to the exact task used during training, a limiting factor for practical applications such as rehabilitation, workforce training, or education. The possibility of identifying training regimens that have a broad enough impact to transfer to a variety of tasks is thus highly appealing. This work reviews how complex training environments such as action video game play may actually foster brain plasticity and learning. This enhanced learning capacity, termed learning to learn, is considered in light of its computational requirements and putative neural mechanisms.',\n",
              "  'authors': ['Bavelier, Daphne',\n",
              "   'Green, C Shawn',\n",
              "   'Pouget, Alexandre',\n",
              "   'Schrater, Paul'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Annual review of neuroscience',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain',\n",
              "   'Human Development',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Psychomotor Performance',\n",
              "   'Transfer (Psychology)',\n",
              "   'Video Games'],\n",
              "  'pmid': '22715883',\n",
              "  'title': 'Brain plasticity through the life span: learning to learn and action video games.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '2038-2529',\n",
              "  'IF_2013': '1.09',\n",
              "  'IF_2014': '1.269',\n",
              "  'IF_2015': '1.071',\n",
              "  'IF_2016': '1.233',\n",
              "  'IF_2017': '1.304',\n",
              "  'ISSN': '0300-8916',\n",
              "  'ISSN_online': '2038-2529',\n",
              "  'ISSN_print': '0300-8916',\n",
              "  'IsoAbbr': 'Tumori',\n",
              "  'JrId': 7863,\n",
              "  'MedAbbr': 'Tumori',\n",
              "  'NLMid': '0111356',\n",
              "  'Titre': 'Tumori',\n",
              "  'abstract': 'Radiobiological and technical considerations have traditionally limited the role of radiation therapy in the context of primary and secondary cardiac malignancies. Stereotactic body radiotherapy (SBRT) is a promising modality for the delivery of focused high-dose radiation with ablative potential to complex targets such as small, deep-seated, moving lesions, allowing also for re-irradiation.|Between January 2013 and October 2013, 3 patients underwent SBRT for cardiac lesions: 2 patients had recurrent, previously irradiated cardiac angiosarcomas (PCA) and 1 patient had a cardiac metastasis from melanoma. They were treated with fiducial-guided robotic radiotherapy with CyberKnife. As for dose prescription, 24 Gy in 3 fractions (80% isodose) and 30 Gy in 5 fractions (80% isodose) were administered to the recurrent PCAs and cardiac metastasis, respectively.|At 2 months after SBRT, cardiac MRI showed a partial response in the patients treated for recurrent PCA while the cardiac metastasis remained stable. In all cases, absence of local progression was subsequently confirmed by contrast-enhanced cardiac MRI after 6 months, without any evidence of treatment-related side effects.|Fiducial-guided SBRT proved to be feasible and effective in preventing local disease progression in selected patients with cardiac malignancies.',\n",
              "  'authors': ['Bonomo, Pierluigi',\n",
              "   'Cipressi, Samantha',\n",
              "   'Desideri, Isacco',\n",
              "   'Masi, Laura',\n",
              "   'Doro, Raffaela',\n",
              "   'Iermano, Carmine',\n",
              "   'Greto, Daniela',\n",
              "   'Simontacchi, Gabriele',\n",
              "   'Mangoni, Monica',\n",
              "   'Paiar, Fabiola',\n",
              "   'Meattini, Icro',\n",
              "   'Scoccianti, Silvia',\n",
              "   'Santoro, Gennaro',\n",
              "   'Valente, Serafina',\n",
              "   'Gensini, Gian Franco',\n",
              "   'Livi, Lorenzo'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Tumori',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Anthracyclines',\n",
              "   'Antineoplastic Combined Chemotherapy Protocols',\n",
              "   'Disease-Free Survival',\n",
              "   'Dose Fractionation, Radiation',\n",
              "   'Female',\n",
              "   'Fluorodeoxyglucose F18',\n",
              "   'Heart Neoplasms',\n",
              "   'Hemangiosarcoma',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Radiopharmaceuticals',\n",
              "   'Radiosurgery',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Reoperation',\n",
              "   'Retreatment',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '25908042',\n",
              "  'title': 'Stereotactic body radiotherapy with CyberKnife for cardiac malignancies.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In this paper, an online self-organizing scheme for Parsimonious and Accurate Fuzzy Neural Networks (PAFNN), and a novel structure learning algorithm incorporating a pruning strategy into novel growth criteria are presented. The proposed growing procedure without pruning not only simplifies the online learning process but also facilitates the formation of a more parsimonious fuzzy neural network. By virtue of optimal parameter identification, high performance and accuracy can be obtained. The learning phase of the PAFNN involves two stages, namely structure learning and parameter learning. In structure learning, the PAFNN starts with no hidden neurons and parsimoniously generates new hidden units according to the proposed growth criteria as learning proceeds. In parameter learning, parameters in premises and consequents of fuzzy rules, regardless of whether they are newly created or already in existence, are updated by the extended Kalman filter (EKF) method and the linear least squares (LLS) algorithm, respectively. This parameter adjustment paradigm enables optimization of parameters in each learning epoch so that high performance can be achieved. The effectiveness and superiority of the PAFNN paradigm are demonstrated by comparing the proposed method with state-of-the-art methods. Simulation results on various benchmark problems in the areas of function approximation, nonlinear dynamic system identification and chaotic time-series prediction demonstrate that the proposed PAFNN algorithm can achieve more parsimonious network structure, higher approximation accuracy and better generalization simultaneously.',\n",
              "  'authors': ['Wang, Ning', 'Er, Meng Joo', 'Meng, Xian-Yao', 'Li, Xiang'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Fuzzy Logic',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Online Systems'],\n",
              "  'pmid': '20945518',\n",
              "  'title': 'An online self-organizing scheme for Parsimonious and Accurate Fuzzy Neural Networks.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1432-1084',\n",
              "  'IF_2013': '4.338',\n",
              "  'IF_2014': '4.014',\n",
              "  'IF_2015': '3.64',\n",
              "  'IF_2016': '3.967',\n",
              "  'IF_2017': '4.027',\n",
              "  'ISSN': '0938-7994',\n",
              "  'ISSN_online': '1432-1084',\n",
              "  'ISSN_print': '0938-7994',\n",
              "  'IsoAbbr': 'Eur Radiol',\n",
              "  'JrId': 20371,\n",
              "  'MedAbbr': 'Eur Radiol',\n",
              "  'NLMid': '9114774',\n",
              "  'Titre': 'Eur Radiol',\n",
              "  'abstract': 'The purpose of this study was to compare sensitivity for detection of pulmonary nodules in MDCT scans and reading time of radiologists when using CAD as the second reader (SR) respectively concurrent reader (CR). Four radiologists analyzed 50 chest MDCT scans chosen from clinical routine two times and marked all detected pulmonary nodules: first with CAD as CR (display of CAD results immediately in the reading session) and later (median 14 weeks) with CAD as SR (display of CAD markers after completion of first reading without CAD). A Siemens LungCAD prototype was used. Sensitivities for detection of nodules and reading times were recorded. Sensitivity of reading with CAD as SR was significantly higher than reading without CAD (p < 0.001) and CAD as CR (p < 0.001). For nodule size of 1.75 mm or above no significant sensitivity difference between CAD as CR and reading without CAD was observed; e.g., for nodules above 4 mm sensitivity was 68% without CAD, 68% with CAD as CR (p = 0.45) and 75% with CAD as SR (p < 0.001). Reading time was significantly shorter for CR (274 s) compared to reading without CAD (294 s; p = 0.04) and SR (337 s; p < 0.001). In our study CAD could either speed up reading of chest CT cases for pulmonary nodules without relevant loss of sensitivity when used as CR, or it increased sensitivity at the cost of longer reading times when used as SR.',\n",
              "  'authors': ['Beyer, F',\n",
              "   'Zierott, L',\n",
              "   'Fallenberg, E M',\n",
              "   'Juergens, K U',\n",
              "   'Stoeckel, J',\n",
              "   'Heindel, W',\n",
              "   'Wormanns, D'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European radiology',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'False Positive Reactions',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Lung Neoplasms',\n",
              "   'Observer Variation',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Solitary Pulmonary Nodule',\n",
              "   'Task Performance and Analysis',\n",
              "   'Time Factors',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '17929026',\n",
              "  'title': 'Comparison of sensitivity and reading time for the use of computer-aided detection (CAD) of pulmonary nodules at MDCT as concurrent or second reader.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': \"This paper describes a method for predicting a supersecondary structural motif, beta-hairpins, in a protein sequence. The method was trained and tested on a set of 5102 hairpins and 5131 non-hairpins, obtained from a non-redundant dataset of 2880 proteins using the DSSP and PROMOTIF programs. Two machine-learning techniques, an artificial neural network (ANN) and a support vector machine (SVM), were used to predict beta-hairpins. An accuracy of 65.5% was achieved using ANN when an amino acid sequence was used as the input. The accuracy improved from 65.5 to 69.1% when evolutionary information (PSI-BLAST profile), observed secondary structure and surface accessibility were used as the inputs. The accuracy of the method further improved from 69.1 to 79.2% when the SVM was used for classification instead of the ANN. The performances of the methods developed were assessed in a test case, where predicted secondary structure and surface accessibility were used instead of the observed structure. The highest accuracy achieved by the SVM based method in the test case was 77.9%. A maximum accuracy of 71.1% with Matthew's correlation coefficient of 0.41 in the test case was obtained on a dataset previously used by X. Cruz, E. G. Hutchinson, A. Shephard and J. M. Thornton (2002) Proc. Natl Acad. Sci. USA, 99, 11157-11162. The performance of the method was also evaluated on proteins used in the '6th community-wide experiment on the critical assessment of techniques for protein structure prediction (CASP6)'. Based on the algorithm described, a web server, BhairPred (http://www.imtech.res.in/raghava/bhairpred/), has been developed, which can be used to predict beta-hairpins in a protein using the SVM approach.\",\n",
              "  'authors': ['Kumar, Manish',\n",
              "   'Bhasin, Manoj',\n",
              "   'Natt, Navjot K',\n",
              "   'Raghava, G P S'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Internet',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Structure, Secondary',\n",
              "   'Sequence Alignment',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Software'],\n",
              "  'pmid': '15988830',\n",
              "  'title': 'BhairPred: prediction of beta-hairpins in a protein from multiple alignment information using ANN and SVM techniques.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.169',\n",
              "  'IF_2014': '0.233',\n",
              "  'IF_2015': '0.277',\n",
              "  'IF_2016': '0.253',\n",
              "  'IF_2017': '0.3',\n",
              "  'ISSN': '0370-8179',\n",
              "  'ISSN_online': '2406-0895',\n",
              "  'ISSN_print': '0370-8179',\n",
              "  'IsoAbbr': 'Srp Arh Celok Lek',\n",
              "  'JrId': 7584,\n",
              "  'MedAbbr': 'Srp Arh Celok Lek',\n",
              "  'NLMid': '0027440',\n",
              "  'Titre': 'Srp Arh Celok Lek',\n",
              "  'abstract': 'During the last two decades, many important results have been achieved in the field of artificial intelligence. One of the most important results is development and application of programs known as expert systems. Medicine is the one of the fields inwhich expert systems are most frequently developed and applied. This paper describes features of these programs, state of the art in the field of expert systems, and the possibilities of their application in medicine.',\n",
              "  'authors': ['Devedzić, V', 'Bozović, Z'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Srpski arhiv za celokupno lekarstvo',\n",
              "  'keywords': ['Artificial Intelligence', 'Expert Systems'],\n",
              "  'pmid': '18193779',\n",
              "  'title': '[Modern expert systems and their use in medicine].',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1365-229X',\n",
              "  'IF_2013': '1.663',\n",
              "  'IF_2014': '1.759',\n",
              "  'IF_2015': '2.151',\n",
              "  'IF_2016': '2.141',\n",
              "  'IF_2017': '2.282',\n",
              "  'ISSN': '0009-9260',\n",
              "  'ISSN_online': '1365-229X',\n",
              "  'ISSN_print': '0009-9260',\n",
              "  'IsoAbbr': 'Clin Radiol',\n",
              "  'JrId': 3069,\n",
              "  'MedAbbr': 'Clin Radiol',\n",
              "  'NLMid': '1306016',\n",
              "  'Titre': 'Clin Radiol',\n",
              "  'abstract': 'To evaluate the role of magnetic resonance imaging (MRI) for diagnosis and therapeutic planning in patients with abnormal placentation (AP).|Overall, 168 consecutive patients with suspected placenta previa and AP were referred for MRI before caesarean section (CS). The ability of MRI to properly detect and assess abnormal placentation was correlated with findings at CS, which were considered the reference standard diagnostic tool. For each patient, MRI was used to determine whether the AP was suitable for complete/incomplete delivery, hysterectomy, or conservative treatment. Treatment planning with MRI was prospectively compared with the actual treatment that had been carried out in each patient decided at CS.|Placenta previa was detected at MRI in 63 patients and AP in 105 patients; 16 patients had false-positive MRI findings, and three had false-negative findings. The sensitivity, specificity, positive predictive value (PPV), and negative predictive value (NPV) of MRI compared to findings at CS were 88.9% (149 of 168), 96.7% (89 of 92), 78.9% (60 of 76), 84.8% (89 of 105), and 95.2% (60 of 63), respectively. Treatment planning could be correctly made on the basis of MRI with accuracy, sensitivity, specificity, PPV, and NPV of 97%, 100%, 92.6%, 95.2%, and 100%, respectively.|MRI offers high diagnostic accuracy in the detection of AP, and it may be helpful in the detailed planning of treatment.',\n",
              "  'authors': ['Wang, Y-L',\n",
              "   'Duan, X-H',\n",
              "   'Han, X-W',\n",
              "   'Zhao, X-L',\n",
              "   'Chen, Z-M',\n",
              "   'Chu, Q-J'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Clinical radiology',\n",
              "  'keywords': ['Adult',\n",
              "   'Clinical Decision-Making',\n",
              "   'Computer Simulation',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Biological',\n",
              "   'Models, Statistical',\n",
              "   'Multimodal Imaging',\n",
              "   'Patient Care Planning',\n",
              "   'Patient Selection',\n",
              "   'Placenta Previa',\n",
              "   'Pregnancy',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Subtraction Technique',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '27776737',\n",
              "  'title': 'Abnormal placentation: the role of MRI in diagnosis and therapeutic planning.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Without doubt general video and sound, as found in large multimedia archives, carry emotional information. Thus, audio and video retrieval by certain emotional categories or dimensions could play a central role for tomorrow\\'s intelligent systems, enabling search for movies with a particular mood, computer aided scene and sound design in order to elicit certain emotions in the audience, etc. Yet, the lion\\'s share of research in affective computing is exclusively focusing on signals conveyed by humans, such as affective speech. Uniting the fields of multimedia retrieval and affective computing is believed to lend to a multiplicity of interesting retrieval applications, and at the same time to benefit affective computing research, by moving its methodology \"out of the lab\" to real-world, diverse data. In this contribution, we address the problem of finding \"disturbing\" scenes in movies, a scenario that is highly relevant for computer-aided parental guidance. We apply large-scale segmental feature extraction combined with audio-visual classification to the particular task of detecting violence. Our system performs fully data-driven analysis including automatic segmentation. We evaluate the system in terms of mean average precision (MAP) on the official data set of the MediaEval 2012 evaluation campaign\\'s Affect Task, which consists of 18 original Hollywood movies, achieving up to .398 MAP on unseen test data in full realism. An in-depth analysis of the worth of individual features with respect to the target class and the system errors is carried out and reveals the importance of peak-related audio feature extraction and low-level histogram-based video analysis. ',\n",
              "  'authors': ['Eyben, Florian',\n",
              "   'Weninger, Felix',\n",
              "   'Lehment, Nicolas',\n",
              "   'Schuller, Björn',\n",
              "   'Rigoll, Gerhard'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Emotions',\n",
              "   'Humans',\n",
              "   'Motion Pictures',\n",
              "   'Multimedia',\n",
              "   'Violence'],\n",
              "  'pmid': '24391704',\n",
              "  'title': 'Affective video retrieval: violence detection in Hollywood movies by large-scale segmental feature extraction.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['UM'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.275',\n",
              "  'IF_2014': '1.163',\n",
              "  'IF_2015': '0.962',\n",
              "  'IF_2016': '1.167',\n",
              "  'IF_2017': '0.827',\n",
              "  'ISSN': '0304-4920',\n",
              "  'ISSN_online': '2666-0059',\n",
              "  'ISSN_print': '0304-4920',\n",
              "  'IsoAbbr': 'Chin J Physiol',\n",
              "  'JrId': 2926,\n",
              "  'MedAbbr': 'Chin J Physiol',\n",
              "  'NLMid': '7804502',\n",
              "  'Titre': 'Chin J Physiol',\n",
              "  'abstract': \"A scalable hardware/software hybrid module--called Ubidule--endowed with bio-inspired ontogenetic and epigenetic features is configured to run a neural networks simulation with developmental and evolvable capabilities. We simulated the activity of hierarchically organized spiking neural networks characterized by an initial developmental phase featuring cell death followed by spike timing dependent synaptic plasticity in presence of background noise. An upstream 'sensory' network received a spatiotemporally organized external input and downstream networks were activated only via the upstream network. Precise firing sequences, formed by recurrent patterns of spikes intervals above chance levels, were observed in all recording conditions, thus suggesting the build-up of a connectivity able to sustain temporal information processing. The activity of a Ubinet--a network of Ubidules--is analyzed by means of virtual electrodes that recorded neural signals similar to EEG. The analysis of these signals was compared with a small set of human recordings and revealed common patterns of shift in quadratic phase coupling. The results suggest some interpretations of changes and plasticity of functional interactions between cortical areas driven by external stimuli and by learning/cognitive\",\n",
              "  'authors': ['Perrig, Stephen',\n",
              "   'Iglesias, Javier',\n",
              "   'Shaposhnyk, Vladislav',\n",
              "   'Chibirova, Olga',\n",
              "   'Dutoit, Pierre',\n",
              "   'Cabessa, Jérémie',\n",
              "   'Espa-Cervena, Katerina',\n",
              "   'Pelletier, Laurent',\n",
              "   'Berger, François',\n",
              "   'Villa, Alessandro E P'],\n",
              "  'categories': ['PHYSIOLOGY'],\n",
              "  'journal': 'The Chinese journal of physiology',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Electroencephalography',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Sleep Initiation and Maintenance Disorders',\n",
              "   'Synapses',\n",
              "   'Synaptic Transmission',\n",
              "   'Time Factors'],\n",
              "  'pmid': '21793350',\n",
              "  'title': 'Functional interactions in hierarchically organized neural networks studied with spatiotemporal firing patterns and phase-coupling frequencies.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'This study was carried out for rapid and noninvasive determination of the class of sorghum species by using the manifold dimensionality reduction (MDR) method and the nonlinear regression method of least squares support vector machines (LS-SVM) combing with the mid-infrared spectroscopy (MIRS) techniques. The methods of Durbin and Run test of augmented partial residual plot (APaRP) were performed to diagnose the nonlinearity of the raw spectral data. The nonlinear MDR methods of isometric feature mapping (ISOMAP), local linear embedding, laplacian eigenmaps and local tangent space alignment, as well as the linear MDR methods of principle component analysis and metric multidimensional scaling were employed to extract the feature variables. The extracted characteristic variables were utilized as the input of LS-SVM and established the relationship between the spectra and the target attributes. The mean average precision (MAP) scores and prediction accuracy were respectively used to evaluate the performance of models. The prediction results showed that the ISOMAP-LS-SVM model obtained the best classification performance, where the MAP scores and prediction accuracy were 0.947 and 92.86%, respectively. It can be concluded that the ISOMAP-LS-SVM model combined with the MIRS technique has the potential of classifying the species of sorghum in a reasonable accuracy. ',\n",
              "  'authors': ['Chen, Y M', 'Lin, P', 'He, J Q', 'He, Y', 'Li, X L'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Least-Squares Analysis',\n",
              "   'Seeds',\n",
              "   'Sorghum',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '26817580',\n",
              "  'title': 'Combination of the Manifold Dimensionality Reduction Methods with Least Squares Support vector machines for Classifying the Species of Sorghum Seeds.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2167-8359',\n",
              "  'IF_2013': '2.112',\n",
              "  'IF_2014': '2.112',\n",
              "  'IF_2015': '2.183',\n",
              "  'IF_2016': '2.177',\n",
              "  'IF_2017': '2.118',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2167-8359',\n",
              "  'IsoAbbr': 'PeerJ',\n",
              "  'JrId': 41057,\n",
              "  'MedAbbr': 'PeerJ',\n",
              "  'NLMid': '101603425',\n",
              "  'Titre': 'PeerJ',\n",
              "  'abstract': 'Recent years have seen a growing awareness of the role the immune system plays in successful cancer treatment, especially in novel therapies like immunotherapy. The characterization of the immunological composition of tumors and their micro-environment is thus becoming a necessity. In this paper we introduce a deep learning-based immune cell detection and quantification method, which is based on supervised learning, i.e., the input data for training comprises labeled images. Our approach objectively deals with staining variation and staining artifacts in immunohistochemically stained lung cancer tissue and is as precise as humans. This is evidenced by the low cell count difference to humans of 0.033 cells on average. This method, which is based on convolutional neural networks, has the potential to provide a new quantitative basis for research on immunotherapy.',\n",
              "  'authors': ['Aprupe, Lilija',\n",
              "   'Litjens, Geert',\n",
              "   'Brinker, Titus J',\n",
              "   'van der Laak, Jeroen',\n",
              "   'Grabe, Niels'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PeerJ',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30993030',\n",
              "  'title': 'Robust and accurate quantification of biomarkers of immune cells in lung cancer micro-environment using deep convolutional neural networks.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1097-0150',\n",
              "  'IF_2013': '1.077',\n",
              "  'IF_2014': '0.69',\n",
              "  'IF_2015': '0.606',\n",
              "  'IF_2016': '1.667',\n",
              "  'IF_2017': '1.2',\n",
              "  'ISSN': '1092-9088',\n",
              "  'ISSN_online': '1097-0150',\n",
              "  'ISSN_print': '1092-9088',\n",
              "  'IsoAbbr': 'Comput. Aided Surg.',\n",
              "  'JrId': 20599,\n",
              "  'MedAbbr': 'Comput Aided Surg',\n",
              "  'NLMid': '9708375',\n",
              "  'Titre': 'Comput. Aided Surg.',\n",
              "  'abstract': 'This paper reports our progress in developing techniques for \"parsing\" raw motion data from a simple surgical task into a labeled sequence of surgical gestures. The ability to automatically detect and segment surgical motion can be useful in evaluating surgical skill, providing surgical training feedback, or documenting essential aspects of a procedure. If processed online, the information can be used to provide context-specific information or motion enhancements to the surgeon. However, in every case, the key step is to relate recorded motion data to a model of the procedure being performed. Robotic surgical systems such as the da Vinci system from Intuitive Surgical provide a rich source of motion and video data from surgical procedures. The application programming interface (API) of the da Vinci outputs 192 kinematics values at 10 Hz. Through a series of feature-processing steps, tailored to this task, the highly redundant features are projected to a compact and discriminative space. The resulting classifier is simple and effective.Cross-validation experiments show that the proposed approach can achieve accuracies higher than 90% when segmenting gestures in a 4-throw suturing task, for both expert and intermediate surgeons. These preliminary results suggest that gesture-specific features can be extracted to provide highly accurate surgical skill evaluation.',\n",
              "  'authors': ['Lin, Henry C',\n",
              "   'Shafran, Izhak',\n",
              "   'Yuh, David',\n",
              "   'Hager, Gregory D'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer aided surgery : official journal of the International Society for Computer Aided Surgery',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Clinical Competence',\n",
              "   'Humans',\n",
              "   'Man-Machine Systems',\n",
              "   'Models, Theoretical',\n",
              "   'Motion',\n",
              "   'Psychomotor Performance',\n",
              "   'Robotics',\n",
              "   'Software',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Time Factors',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '17127647',\n",
              "  'title': 'Towards automatic skill evaluation: detection and segmentation of robot-assisted surgical motions.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1536-3708',\n",
              "  'IF_2013': '1.458',\n",
              "  'IF_2014': '1.494',\n",
              "  'IF_2015': '1.535',\n",
              "  'IF_2016': '1.596',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0148-7043',\n",
              "  'ISSN_online': '1536-3708',\n",
              "  'ISSN_print': '0148-7043',\n",
              "  'IsoAbbr': 'Ann Plast Surg',\n",
              "  'JrId': 628,\n",
              "  'MedAbbr': 'Ann Plast Surg',\n",
              "  'NLMid': '7805336',\n",
              "  'Titre': 'Ann Plast Surg',\n",
              "  'abstract': 'Endoscopic technique has been used in the management of comminuted malar fractures. However, the reported dissection plane is close to the frontal branch of the facial nerve, and paralysis of the frontal muscle is sometimes noted postoperatively. From January 1998 to November 2001, 42 patients underwent endoscopic-assisted zygomatic bone repair at Kaohsiung Medical University Hospital and Kaohsiung Municipal Hsiao Kang Hospital. The zygomatic arch was approached via a dissection plane beneath the deep temporal fascia, and the plate was fixed on the upper border of the arch. The advantages of this method are 1) one temporal incision is sufficient for dealing with the zygomatic arch fracture; 2) the learning curve for endoscopic technique is shortened; 3) there is less risk of injury to the frontal branch of the facial nerve; and 4) the periosteum at the anterior and inferior border of the zygomatic arch is preserved. The deep method is a safe alternative for endoscopic-assisted comminuted malar fracture repair.',\n",
              "  'authors': ['Lee, Su-Shin',\n",
              "   'Lin, Sin-Daw',\n",
              "   'Chiu, Yu-Te',\n",
              "   'Tsai, Chih-Cheng',\n",
              "   'Lin, Tsai-Ming',\n",
              "   'Chang, Kao-Ping'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Annals of plastic surgery',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Dissection',\n",
              "   'Endoscopy',\n",
              "   'Female',\n",
              "   'Fractures, Comminuted',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Reconstructive Surgical Procedures',\n",
              "   'Zygoma'],\n",
              "  'pmid': '12439010',\n",
              "  'title': 'Deep dissection plane for endoscopic-assisted comminuted malar fracture repair.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1552-4833',\n",
              "  'IF_2013': '2.048',\n",
              "  'IF_2014': '2.159',\n",
              "  'IF_2015': '2.082',\n",
              "  'IF_2016': '2.259',\n",
              "  'IF_2017': '2.264',\n",
              "  'ISSN': '1552-4825',\n",
              "  'ISSN_online': '1552-4833',\n",
              "  'ISSN_print': '1552-4825',\n",
              "  'IsoAbbr': 'Am. J. Med. Genet. A',\n",
              "  'JrId': 32200,\n",
              "  'MedAbbr': 'Am J Med Genet A',\n",
              "  'NLMid': '101235741',\n",
              "  'Titre': 'Am. J. Med. Genet. A',\n",
              "  'abstract': \"Craniofacial dysmorphism recognition is the first step in diagnosing most genetic syndromes. However, the number of genetic syndromes is enormous, and the specific facial features are difficult to memorize. For clinical practice, recent advances in artificial intelligence can be of use. One such tool, Face2Gene (FDNA, Inc., Boston, MA), is an innovative free group of applications, that helps clinicians recognize possible genetic syndromes from patients' facial two-dimensional photos. The initial data set used to train this technology consisted primarily of Caucasian patients. Because ethnic differences affect patients' facial features, the recognition probability in Asian patients may be limited. Our aim was to test the technology's recognition probability on Thai children with Down Syndrome (DS) as compared to Thai children without DS (non-DS). Two separate control groups of Thai non-DS children, either unaffected or having other syndromes, were included. Frontal photographs were obtained from all the participants. All 30 children with DS were recognized as DS in the top 10 syndrome-matches (100% sensitivity), and 27 were in the first ranking of suggested syndromes. Eighteen non-DS were recognized as DS (87.2% specificity) with an accuracy of 89%. We present a scientific basis for this novel tool, useful in the clinic where patients are of a different ethnicity unfamiliar to the evaluator. However, Face2Gene cannot be considered a replacement for clinicians' knowledge of phenotypes. Further studies on other genetic syndromes/ethnicities being identified by software algorithms are needed.\",\n",
              "  'authors': ['Vorravanpreecha, Nattariya',\n",
              "   'Lertboonnum, Thanayoot',\n",
              "   'Rodjanadit, Rungrote',\n",
              "   'Sriplienchan, Pak',\n",
              "   'Rojnueangnit, Kitiwan'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'American journal of medical genetics. Part A',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Case-Control Studies',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Craniofacial Abnormalities',\n",
              "   'Data Anonymization',\n",
              "   'Down Syndrome',\n",
              "   'Facies',\n",
              "   'Female',\n",
              "   'Genetic Testing',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'ROC Curve',\n",
              "   'Thailand'],\n",
              "  'pmid': '30070762',\n",
              "  'title': 'Studying Down syndrome recognition probabilities in Thai children with de-identified computer-aided facial analysis.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1089-8638',\n",
              "  'IF_2013': '3.959',\n",
              "  'IF_2014': '4.333',\n",
              "  'IF_2015': '4.517',\n",
              "  'IF_2016': '4.632',\n",
              "  'IF_2017': '4.894',\n",
              "  'ISSN': '0022-2836',\n",
              "  'ISSN_online': '1089-8638',\n",
              "  'ISSN_print': '0022-2836',\n",
              "  'IsoAbbr': 'J. Mol. Biol.',\n",
              "  'JrId': 4967,\n",
              "  'MedAbbr': 'J Mol Biol',\n",
              "  'NLMid': '2985088R',\n",
              "  'Titre': 'J. Mol. Biol.',\n",
              "  'abstract': \"We describe a fully automated algorithm for finding functional sites on protein structures. Our method finds surface patches of unusual physicochemical properties on protein structures, and estimates the patches' probability of overlapping functional sites. Other methods for predicting the locations of specific types of functional sites exist, but in previous analyses, it has been difficult to compare methods when they are applied to different types of sites. Thus, we introduce a new statistical framework that enables rigorous comparisons of the usefulness of different physicochemical properties for predicting virtually any kind of functional site. The program's statistical models were trained for 11 individual properties (electrostatics, concavity, hydrophobicity, etc.) and for 15 neural network combination properties, all optimized and tested on 15 diverse protein functions. To simulate what to expect if the program were run on proteins of unknown function, as might arise from structural genomics, we tested it on 618 proteins of diverse mixed functions. In the higher-scoring top half of all predictions, a functional residue could typically be found within the first 1.7 residues chosen at random. The program may or may not use partial information about the protein's function type as an input, depending on which statistical model the user chooses to employ. If function type is used as an additional constraint, prediction accuracy usually increases, and is particularly good for enzymes, DNA-interacting sites, and oligomeric interfaces. The program can be accessed online (at http://hotpatch.mbi.ucla.edu).\",\n",
              "  'authors': ['Pettit, Frank K',\n",
              "   'Bare, Emiko',\n",
              "   'Tsai, Albert',\n",
              "   'Bowie, James U'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of molecular biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Motifs',\n",
              "   'Animals',\n",
              "   'Binding Sites',\n",
              "   'Biochemistry',\n",
              "   'Caspase 7',\n",
              "   'Computational Biology',\n",
              "   'Humans',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Conformation',\n",
              "   'Protein Structure, Tertiary',\n",
              "   'Proteins',\n",
              "   'Software'],\n",
              "  'pmid': '17451744',\n",
              "  'title': 'HotPatch: a statistical approach to finding biologically relevant features on protein surfaces.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'The aim of this present study is firstly to compare significant predictors of mortality for hepatocellular carcinoma (HCC) patients undergoing resection between artificial neural network (ANN) and logistic regression (LR) models and secondly to evaluate the predictive accuracy of ANN and LR in different survival year estimation models. We constructed a prognostic model for 434 patients with 21 potential input variables by Cox regression model. Model performance was measured by numbers of significant predictors and predictive accuracy. The results indicated that ANN had double to triple numbers of significant predictors at 1-, 3-, and 5-year survival models as compared with LR models. Scores of accuracy, sensitivity, specificity, and area under the receiver operating characteristic curve (AUROC) of 1-, 3-, and 5-year survival estimation models using ANN were superior to those of LR in all the training sets and most of the validation sets. The study demonstrated that ANN not only had a great number of predictors of mortality variables but also provided accurate prediction, as compared with conventional methods. It is suggested that physicians consider using data mining methods as supplemental tools for clinical decision-making and prognostic evaluation.',\n",
              "  'authors': ['Chiu, Herng-Chia',\n",
              "   'Ho, Te-Wei',\n",
              "   'Lee, King-Teh',\n",
              "   'Chen, Hong-Yaw',\n",
              "   'Ho, Wen-Hsien'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Carcinoma, Hepatocellular',\n",
              "   'Female',\n",
              "   'Hepatectomy',\n",
              "   'Humans',\n",
              "   'Liver Neoplasms',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prevalence',\n",
              "   'Prognosis',\n",
              "   'Proportional Hazards Models',\n",
              "   'Reproducibility of Results',\n",
              "   'Risk Assessment',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Survival Analysis',\n",
              "   'Survival Rate',\n",
              "   'Taiwan',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '23737707',\n",
              "  'title': 'Mortality predicted accuracy for hepatocellular carcinoma patients with hepatic resection using artificial neural network.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.905',\n",
              "  'IF_2014': '0.982',\n",
              "  'IF_2015': '1.066',\n",
              "  'IF_2016': '1.141',\n",
              "  'IF_2017': '1.352',\n",
              "  'ISSN': '1195-9479',\n",
              "  'ISSN_print': '1195-9479',\n",
              "  'IsoAbbr': 'Can J Urol',\n",
              "  'JrId': 21917,\n",
              "  'MedAbbr': 'Can J Urol',\n",
              "  'NLMid': '9515842',\n",
              "  'Titre': 'Can J Urol',\n",
              "  'abstract': \"Partial nephrectomy is an effective surgical treatment for small renal masses. We compare a single surgeon's experience with consecutive laparoscopic and robotic partial nephrectomy to assess potential perioperative outcomes. A review of the literature is provided.|A retrospective review was performed comparing 15 consecutive patients undergoing laparoscopic partial nephrectomy to the subsequent consecutive 13 patients undergoing robotic assisted partial nephrectomy for small renal tumors. All patients had normal contralateral kidney appearance on cross sectional imaging. A similar transperitoneal technique was employed for both cohorts. A 4-arm technique was used for the robotic cases using the da Vinci (Intuitive Surgical, Sunnyvale, USA) surgical system. Patient demographics, tumor characteristics, intraoperative, and postoperative data including tumor size, warm ischemia time, and estimated blood loss (EBL) were compared using Student t-test, Wilcoxon rank-sum, or Chi square test as appropriate.|All cases were completed laparoscopically or with robotic assistance without conversion to open surgery. Demographic data were not statistically different between the two groups. Warm ischemia time (WIT) was shorter in the robotic group: 29.7 minutes versus 39.9 minutes for the laparoscopic group (p < 0.0001). Operative time was longer in the robotic group: 253 versus 352 minutes (p < 0.0001). Mean hospital stay and postoperative complication rates were not statistically different. Two (13%) of patients in the laparoscopic group required conversion of partial nephrectomy to radical nephrectomy while none did in the robotic group. Final pathology revealed negative margins in all cases.|Robotic partial nephrectomy resulted in decreased WIT as compared to the conventional laparoscopic approach. Total operating time was increased in the robotic group.\",\n",
              "  'authors': ['DeLong, Jessica M', 'Shapiro, Oleg', 'Moinzadeh, Alireza'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Canadian journal of urology',\n",
              "  'keywords': ['Female',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics'],\n",
              "  'pmid': '20566016',\n",
              "  'title': \"Comparison of laparoscopic versus robotic assisted partial nephrectomy: one surgeon's initial experience.\",\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.95',\n",
              "  'IF_2014': '0.682',\n",
              "  'IF_2015': '0.581',\n",
              "  'IF_2016': '0.649',\n",
              "  'IF_2017': '0.804',\n",
              "  'ISSN': '1011-601X',\n",
              "  'ISSN_print': '1011-601X',\n",
              "  'IsoAbbr': 'Pak J Pharm Sci',\n",
              "  'JrId': 32507,\n",
              "  'MedAbbr': 'Pak J Pharm Sci',\n",
              "  'NLMid': '9426356',\n",
              "  'Titre': 'Pak J Pharm Sci',\n",
              "  'abstract': 'Microbial fuel cells have been obtaining more and more attention with the associated abilities of continuous electrical power supply and wastewater treatment. Because of its complicated reaction mechanism and its inherent characteristics of time varying, uncertainty, strong coupling and nonlinearity, there are complex control challenges in microbial fuel cells. In this paper, an adaptive fuzzy control scheme is proposed for the microbial fuel cell system to achieve constant voltage output under different loads. A main fuzzy controller is used to track the set value, and an auxiliary fuzzy controller is applied to adjust the factors of the main controller. Simulation results show that the output voltage can track the given value well. The proposed adaptive fuzzy controller can give better steady-state behavior and faster response, and it improves the running performance of the microbial fuel cell. ',\n",
              "  'authors': ['Fan, Liping', 'Li, Chong', 'Boshnakov, Kosta'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Pakistan journal of pharmaceutical sciences',\n",
              "  'keywords': ['Bioelectric Energy Sources',\n",
              "   'Computer Simulation',\n",
              "   'Fuzzy Logic'],\n",
              "  'pmid': '24816708',\n",
              "  'title': 'Performance improvement of a microbial fuel cell based on adaptive fuzzy control.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'This paper focuses on the application of rapid prototyping techniques using additive manufacturing in combination with parametric design to create low-cost, yet accurate and reliable instruments. The methodology followed makes it possible to make instruments with a degree of customization until now available only to a narrow audience, helping democratize science. The proposal discusses a holistic design-for-manufacturing approach that comprises advanced modeling techniques, open-source design strategies, and an optimization algorithm using free parametric software for both professional and educational purposes. The design and fabrication of an instrument for scattering measurement is used as a case of study to present the previous concepts.',\n",
              "  'authors': ['Nadal-Serrano, Jose M',\n",
              "   'Nadal-Serrano, Adolfo',\n",
              "   'Lopez-Vallejo, Marisa'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computer-Aided Design',\n",
              "   'Costs and Cost Analysis',\n",
              "   'Equipment Design',\n",
              "   'Manufactured Materials',\n",
              "   'Optics and Photonics',\n",
              "   'Software'],\n",
              "  'pmid': '29112987',\n",
              "  'title': 'Democratizing science with the aid of parametric design and additive manufacturing: Design and fabrication of a versatile and low-cost optical instrument for scattering measurement.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1097-4199',\n",
              "  'IF_2013': '15.982',\n",
              "  'IF_2014': '15.054',\n",
              "  'IF_2015': '13.974',\n",
              "  'IF_2016': '14.024',\n",
              "  'IF_2017': '14.318',\n",
              "  'ISSN': '0896-6273',\n",
              "  'ISSN_online': '1097-4199',\n",
              "  'ISSN_print': '0896-6273',\n",
              "  'IsoAbbr': 'Neuron',\n",
              "  'JrId': 1600,\n",
              "  'MedAbbr': 'Neuron',\n",
              "  'NLMid': '8809320',\n",
              "  'Titre': 'Neuron',\n",
              "  'abstract': 'Neural circuits display complex activity patterns both spontaneously and when responding to a stimulus or generating a motor output. How are these two forms of activity related? We develop a procedure called FORCE learning for modifying synaptic strengths either external to or within a model neural network to change chaotic spontaneous activity into a wide variety of desired activity patterns. FORCE learning works even though the networks we train are spontaneously chaotic and we leave feedback loops intact and unclamped during learning. Using this approach, we construct networks that produce a wide variety of complex output patterns, input-output transformations that require memory, multiple outputs that can be switched by control inputs, and motor patterns matching human motion capture data. Our results reproduce data on premovement activity in motor and premotor cortex, and suggest that synaptic plasticity may be a more rapid and powerful modulator of network activity than generally appreciated.',\n",
              "  'authors': ['Sussillo, David', 'Abbott, L F'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Neuron',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Feedback, Physiological',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Nonlinear Dynamics'],\n",
              "  'pmid': '19709635',\n",
              "  'title': 'Generating coherent patterns of activity from chaotic neural networks.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1558-318X',\n",
              "  'IF_2013': '1.347',\n",
              "  'IF_2014': '1.204',\n",
              "  'IF_2015': '1.835',\n",
              "  'IF_2016': '2.22',\n",
              "  'IF_2017': '2.149',\n",
              "  'ISSN': '0094-0143',\n",
              "  'ISSN_online': '1558-318X',\n",
              "  'ISSN_print': '0094-0143',\n",
              "  'IsoAbbr': 'Urol. Clin. North Am.',\n",
              "  'JrId': 7899,\n",
              "  'MedAbbr': 'Urol Clin North Am',\n",
              "  'NLMid': '0423221',\n",
              "  'Titre': 'Urol. Clin. North Am.',\n",
              "  'abstract': 'Vesicoureteral reflux (VUR) is a common problem in childhood, affecting approximately 1% to 2% of the pediatric population. Mild cases of VUR are likely to resolve spontaneously, but high-grade VUR may require surgical correction. Pediatric urologists are familiar with open antireflux operations, which can be accomplished with minimal operative morbidity. Minimally invasive endoscopic and laparoscopic techniques that now exist may serve to reduce morbidity further. This article reviews the endoscopic materials, techniques, and outcomes in the treatment of VUR in addition to the techniques and outcomes of laparoscopic and robotic ureteroneocystotomy.',\n",
              "  'authors': ['Hayn, Matthew H',\n",
              "   'Smaldone, Marc C',\n",
              "   'Ost, Michael C',\n",
              "   'Docimo, Steven G'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Urologic clinics of North America',\n",
              "  'keywords': ['Biocompatible Materials',\n",
              "   'Child',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Robotics',\n",
              "   'Ureteroscopy',\n",
              "   'Vesico-Ureteral Reflux'],\n",
              "  'pmid': '18761201',\n",
              "  'title': 'Minimally invasive treatment of vesicoureteral reflux.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The overall survival rate is good for lymph-node-negative breast cancer patients, but they still suffer from serious over- and some undertreatments. Prognostic and predictive gene signatures for node-negative breast cancer have a high number of genes related to proliferation. The prognostic value of gene sets from commercial gene-expression assays were compared with proliferation markers.|Illumina WG6 mRNA microarray analysis was used to examine 94 fresh-frozen tumour samples from node-negative breast cancer patients. The patients were divided into low- and high-risk groups for distant metastasis based on the MammaPrint-related genes, and into low-, intermediate- and high-risk groups based on the recurrence score algorithm with genes included in Oncotype DX. These data were then compared to proliferation status, as measured by the mitotic activity index, the expressions of phosphohistone H3 (PPH3), and Ki67.|Kaplan-Meier survival analysis for distant-metastasis-free survival revealed that patients with weak and strong PPH3 expressions had 14-year survival rates of 87% (n=45), and 65% (n=49, p=0.014), respectively. Analysis of the MammaPrint classification resulted in 14-year survival rates of 80% (n=45) and 71% (n=49, p=0.287) for patients with low and high risks of recurrence, respectively. The Oncotype DX categorization yielded 14-year survival rates of 83% (n=18), 79% (n=42) and 68% (n=34) for those in the low-, intermediate- and high-risk groups, respectively (p=0.52). Supervised hierarchical cluster analysis for distant-metastasis-free survival in the subgroup of patients with strong PPH3 expression revealed that the genes involved in Notch signalling and cell adhesion were expressed at higher levels in those patients with distant metastasis.|This pilot study indicates that proliferation has greater prognostic value than the expressions of either MammaPrint- or Oncotype-DX-related genes. Furthermore, in the subgroup of patients with high proliferation, Notch signalling pathway genes appear to be expressed at higher levels in patients who develop distant metastasis.',\n",
              "  'authors': ['Jonsdottir, Kristin',\n",
              "   'Assmus, Jörg',\n",
              "   'Slewa, Aida',\n",
              "   'Gudlaugsson, Einar',\n",
              "   'Skaland, Ivar',\n",
              "   'Baak, Jan P A',\n",
              "   'Janssen, Emiel A M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Breast Neoplasms',\n",
              "   'Cell Proliferation',\n",
              "   'Cluster Analysis',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Neoplasm',\n",
              "   'Histones',\n",
              "   'Humans',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'Lymph Nodes',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Metastasis',\n",
              "   'Phosphoproteins',\n",
              "   'Prognosis'],\n",
              "  'pmid': '24599057',\n",
              "  'title': 'Prognostic value of gene signatures and proliferation in lymph-node-negative breast cancer.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1873-5894',\n",
              "  'IF_2013': '2.022',\n",
              "  'IF_2014': '2.09',\n",
              "  'IF_2015': '1.98',\n",
              "  'IF_2016': '2.225',\n",
              "  'IF_2017': '2.564',\n",
              "  'ISSN': '0730-725X',\n",
              "  'ISSN_online': '1873-5894',\n",
              "  'ISSN_print': '0730-725X',\n",
              "  'IsoAbbr': 'Magn Reson Imaging',\n",
              "  'JrId': 5670,\n",
              "  'MedAbbr': 'Magn Reson Imaging',\n",
              "  'NLMid': '8214883',\n",
              "  'Titre': 'Magn Reson Imaging',\n",
              "  'abstract': 'The usefulness of neural networks for the classification of signal-time curves from dynamic MR mammography was recently demonstrated by our group. The multi-layer perceptron under study consists of 28 input, 4 hidden, and 3 output nodes, and was trained to classify signal-time curves into three tissue classes: \"carcinoma,\" \"benign lesion,\" and \"parenchyma.\" Extending this approach, it was the aim of the present study to evaluate the performance of the developed network in the segmentation of dynamic MR mammographic images in comparison to a pixel-by-pixel two-compartment pharmacokinetic analysis. The population investigated in this pilot study comprised 15 women with suspicious lesions in the breast, which were confirmed histologically after the MR examination. The neural network classified the same areas as malignant as those which were marked as being highly suspicious by the pharmacokinetic mapping approach but with the advantage that no a priori knowledge on tissue microcirculation was needed, that computation proved to be much faster, and that it yielded a unique classification into just three tissue classes.',\n",
              "  'authors': ['Lucht, Robert', 'Delorme, Stefan', 'Brix, Gunnar'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance imaging',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Neoplasms',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pilot Projects'],\n",
              "  'pmid': '12034335',\n",
              "  'title': 'Neural network-based segmentation of dynamic MR mammographic images.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'Recent reports have begun to elucidate mechanisms by which learning and experience produce white matter changes in the brain. We previously reported changes in white matter surrounding the anterior cingulate cortex in humans after 2-4 weeks of meditation training. We further found that low-frequency optogenetic stimulation of the anterior cingulate in mice increased time spent in the light in a light/dark box paradigm, suggesting decreased anxiety similar to what is observed following meditation training. Here, we investigated the impact of this stimulation at the cellular level. We found that laser stimulation in the range of 1-8 Hz results in changes to subcortical white matter projection fibers in the corpus callosum. Specifically, stimulation resulted in increased oligodendrocyte proliferation, accompanied by a decrease in the g-ratio within the corpus callosum underlying the anterior cingulate cortex. These results suggest that low-frequency stimulation can result in activity-dependent remodeling of myelin, giving rise to enhanced connectivity and altered behavior.',\n",
              "  'authors': ['Piscopo, Denise M',\n",
              "   'Weible, Aldis P',\n",
              "   'Rothbart, Mary K',\n",
              "   'Posner, Michael I',\n",
              "   'Niell, Cristopher M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Animals',\n",
              "   'Anxiety',\n",
              "   'Corpus Callosum',\n",
              "   'Deep Brain Stimulation',\n",
              "   'Mice',\n",
              "   'Optogenetics',\n",
              "   'White Matter'],\n",
              "  'pmid': '29915074',\n",
              "  'title': 'Changes in white matter in mice resulting from low-frequency brain stimulation.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '2168-1813',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '1.247',\n",
              "  'IF_2015': '1.346',\n",
              "  'IF_2016': '1.513',\n",
              "  'IF_2017': '1.684',\n",
              "  'ISSN': '2168-1805',\n",
              "  'ISSN_online': '2168-1813',\n",
              "  'ISSN_print': '2168-1805',\n",
              "  'IsoAbbr': 'Scand J Urol',\n",
              "  'JrId': 40768,\n",
              "  'MedAbbr': 'Scand J Urol',\n",
              "  'NLMid': '101587186',\n",
              "  'Titre': 'Scand J Urol',\n",
              "  'abstract': \"Primary localized bladder amyloidosis is a rare pathology that mimics urothelial cancer. Systemic disease and lymphoproliferative disorders should be excluded. Transurethral resection is the mainstay of treatment, although in cases of extensive bladder involvement or massive haematuria radical cystectomy has been reported. To the authors' knowledge, this is the first robot-assisted prostate-sparing simple cystectomy with intracorporeal neobladder and preservation of the seminal vesicles and vas deferens reported in the literature, in a patient with primary localized amyloidosis of the bladder.\",\n",
              "  'authors': ['Hosseini, Abolfazl',\n",
              "   'Ploumidis, Achilles',\n",
              "   'Adding, Christofer',\n",
              "   'Wiklund, N Peter'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Scandinavian journal of urology',\n",
              "  'keywords': ['Amyloidosis',\n",
              "   'Cystectomy',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Organ Sparing Treatments',\n",
              "   'Prostate',\n",
              "   'Robotics',\n",
              "   'Seminal Vesicles',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Bladder Diseases',\n",
              "   'Urinary Diversion',\n",
              "   'Vas Deferens'],\n",
              "  'pmid': '22746326',\n",
              "  'title': 'Radical surgery for treatment of primary localized bladder amyloidosis: could prostate-sparing robot-assisted cystectomy with intracorporeal urinary diversion be an option?',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1614-7499',\n",
              "  'IF_2013': '2.757',\n",
              "  'IF_2014': '2.828',\n",
              "  'IF_2015': '2.76',\n",
              "  'IF_2016': '2.741',\n",
              "  'IF_2017': '2.8',\n",
              "  'ISSN': '0944-1344',\n",
              "  'ISSN_online': '1614-7499',\n",
              "  'ISSN_print': '0944-1344',\n",
              "  'IsoAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'JrId': 22074,\n",
              "  'MedAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'NLMid': '9441769',\n",
              "  'Titre': 'Environ Sci Pollut Res Int',\n",
              "  'abstract': 'The use of pesticides has been increasing in agriculture, leading to a public health problem. The aim of this study was to evaluate ototoxic effects in farmers who were exposed to cigarette smoke and/or pesticides and to identify possible classification patterns in the exposure groups. The sample included 127 participants of both sexes aged between 18 and 39, who were divided into the following four groups: control group (CG), smoking group (SG), pesticide group (PG), and smoking + pesticide group (SPG). Meatoscopy, pure tone audiometry, logoaudiometry, high-frequency thresholds, and immittance testing were performed. Data were evaluated by artificial neural network (ANN), K-nearest neighbors (K-NN), and support vector machine (SVM). There was symmetry between the right and left ears, an increase in the incidence of hearing loss at high frequency and of downward sloping audiometric curve configuration, and alteration of stapedial reflex in the three exposed groups. The machine-learning classifiers achieved good classification performance (control and exposed). The best classification results occur in high type (I and II) datasets (about 90% accuracy) in k-NN test. It is concluded that both xenobiotic substances have ototoxic potential; however, their combined use does not present additive or potentiating effects recognizable by the algorithms.',\n",
              "  'authors': ['Tomiazzi, Jamile Silveira',\n",
              "   'Pereira, Danillo Roberto',\n",
              "   'Judai, Meire Aparecida',\n",
              "   'Antunes, Patrícia Alexandra',\n",
              "   'Favareto, Ana Paula Alves'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental science and pollution research international',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Air Pollutants, Occupational',\n",
              "   'Algorithms',\n",
              "   'Brazil',\n",
              "   'Farmers',\n",
              "   'Female',\n",
              "   'Hearing Loss',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Occupational Exposure',\n",
              "   'Pesticides',\n",
              "   'Smoking',\n",
              "   'Support Vector Machine',\n",
              "   'Tobacco',\n",
              "   'Tobacco Smoke Pollution',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30623325',\n",
              "  'title': 'Performance of machine-learning algorithms to pattern recognition and classification of hearing impairment in Brazilian farmers exposed to pesticide and/or cigarette smoke.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'In contrast to protein-coding genes, long-noncoding RNAs (lncRNAs) are much less well understood, despite increasing evidence indicating a wide range of their biological functions, and possible roles in various cancers. Based on public RNA-seq datasets of four solid cancer types, we here utilize Weighted Correlation Network Analysis (WGCNA) to propose a strategy for exploring the functions of lncRNAs altered in more than two cancer types, which we call onco-lncRNAs. Results indicate that cancer-expressed lncRNAs show high tissue specificity and are weakly expressed, more so than protein-coding genes. Most of the 236 onco-lncRNAs we identified have not been reported to have associations with cancers before. Our analysis exploits co-expression network to reveal that onco-lncRNAs likely play key roles in the multistep development of human cancers, covering a wide range of functions in genome stability maintenance, signaling, cell adhesion and motility, morphogenesis, cell cycle, immune and inflammatory response. These observations contribute to a more comprehensive understanding of cancer-associated lncRNAs, while demonstrating a novel and efficient strategy for subsequent functional studies of lncRNAs.',\n",
              "  'authors': ['Li, Suqing',\n",
              "   'Li, Bin',\n",
              "   'Zheng, Yuanting',\n",
              "   'Li, Menglong',\n",
              "   'Shi, Leming',\n",
              "   'Pu, Xuemei'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Neoplasms',\n",
              "   'Prognosis',\n",
              "   'RNA, Long Noncoding',\n",
              "   'Signal Transduction',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '28389669',\n",
              "  'title': 'Exploring functions of long noncoding RNAs across multiple cancers through co-expression network.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1094-4087',\n",
              "  'IF_2013': '3.525',\n",
              "  'IF_2014': '3.488',\n",
              "  'IF_2015': '3.148',\n",
              "  'IF_2016': '3.307',\n",
              "  'IF_2017': '3.356',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1094-4087',\n",
              "  'IsoAbbr': 'Opt Express',\n",
              "  'JrId': 29989,\n",
              "  'MedAbbr': 'Opt Express',\n",
              "  'NLMid': '101137103',\n",
              "  'Titre': 'Opt Express',\n",
              "  'abstract': 'We demonstrate an imaging technique that allows identification and classification of objects hidden behind scattering media and is invariant to changes in calibration parameters within a training range. Traditional techniques to image through scattering solve an inverse problem and are limited by the need to tune a forward model with multiple calibration parameters (like camera field of view, illumination position etc.). Instead of tuning a forward model and directly inverting the optical scattering, we use a data driven approach and leverage convolutional neural networks (CNN) to learn a model that is invariant to calibration parameters variations within the training range and nearly invariant beyond that. This effectively allows robust imaging through scattering conditions that is not sensitive to calibration. The CNN is trained with a large synthetic dataset generated with a Monte Carlo (MC) model that contains random realizations of major calibration parameters. The method is evaluated with a time-resolved camera and multiple experimental results are provided including pose estimation of a mannequin hidden behind a paper sheet with 23 correct classifications out of 30 tests in three poses (76.6% accuracy on real-world measurements). This approach paves the way towards real-time practical non line of sight (NLOS) imaging applications.',\n",
              "  'authors': ['Satat, Guy',\n",
              "   'Tancik, Matthew',\n",
              "   'Gupta, Otkrist',\n",
              "   'Heshmat, Barmak',\n",
              "   'Raskar, Ramesh'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics express',\n",
              "  'keywords': [''],\n",
              "  'pmid': '28789238',\n",
              "  'title': 'Object classification through scattering media with deep learning on time resolved measurement.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Emotion recognition plays an important role in human-computer interaction. Previously and currently, many studies focused on speech emotion recognition using several classifiers and feature extraction methods. The majority of such studies, however, address the problem of speech emotion recognition considering emotions solely from the perspective of a single language. In contrast, the current study extends monolingual speech emotion recognition to also cover the case of emotions expressed in several languages that are simultaneously recognized by a complete system. To address this issue, a method, which provides an effective and powerful solution to bilingual speech emotion recognition, is proposed and evaluated. The proposed method is based on a two-pass classification scheme consisting of spoken language identification and speech emotion recognition. In the first pass, the language spoken is identified; in the second pass, emotion recognition is conducted using the emotion models of the language identified. Based on deep learning and the i-vector paradigm, bilingual emotion recognition experiments have been conducted using the state-of-the-art English IEMOCAP (four emotions) and German FAU Aibo (five emotions) corpora. Two classifiers along with i-vector features were used and compared, namely, fully connected deep neural networks (DNN) and convolutional neural networks (CNN). In the case of DNN, 64.0% and 61.14% unweighted average recalls (UARs) were obtained using the IEMOCAP and FAU Aibo corpora, respectively. When using CNN, 62.0% and 59.8% UARs were achieved in the case of the IEMOCAP and FAU Aibo corpora, respectively. These results are very promising, and superior to those obtained in similar studies on multilingual or even monolingual speech emotion recognition. Furthermore, an additional baseline approach for bilingual speech emotion recognition was implemented and evaluated. In the baseline approach, six common emotions were considered, and bilingual emotion models were created, trained on data from the two languages. In this case, 51.2% and 51.5% UARs for six emotions were obtained using DNN and CNN, respectively. The results using the baseline method were reasonable and promising, showing the effectiveness of using i-vectors and deep learning in bilingual speech emotion recognition. On the other hand, the proposed two-pass method based on language identification showed significantly superior performance. Furthermore, the current study was extended to also deal with multilingual speech emotion recognition using corpora collected under similar conditions. Specifically, the English IEMOCAP, the German Emo-DB, and a Japanese corpus were used to recognize four emotions based on the proposed two-pass method. The results obtained were very promising, and the differences in UAR were not statistically significant compared to the monolingual classifiers.',\n",
              "  'authors': ['Heracleous, Panikos', 'Yoneyama, Akio'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31415592',\n",
              "  'title': 'A comprehensive study on bilingual and multilingual speech emotion recognition using a two-pass classification scheme.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1476-7120',\n",
              "  'IF_2013': '1.283',\n",
              "  'IF_2014': '1.337',\n",
              "  'IF_2015': '1.463',\n",
              "  'IF_2016': '1.598',\n",
              "  'IF_2017': '1.652',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1476-7120',\n",
              "  'IsoAbbr': 'Cardiovasc Ultrasound',\n",
              "  'JrId': 30108,\n",
              "  'MedAbbr': 'Cardiovasc Ultrasound',\n",
              "  'NLMid': '101159952',\n",
              "  'Titre': 'Cardiovasc Ultrasound',\n",
              "  'abstract': 'Left ventricular size and function are important prognostic factors in heart disease. Their measurement is the most frequent reason for sending patients to the echo lab. These measurements have important implications for therapy but are sensitive to the skill of the operator. Earlier automated echo-based methods have not become widely used. The aim of our study was to evaluate an automatic echocardiographic method (with manual correction if needed) for determining left ventricular ejection fraction (LVEF) based on an active appearance model of the left ventricle (syngo AutoEF, Siemens Medical Solutions). Comparisons were made with manual planimetry (manual Simpson), visual assessment and automatically determined LVEF from quantitative myocardial gated single photon emission computed tomography (SPECT).|60 consecutive patients referred for myocardial perfusion imaging (MPI) were included in the study. Two-dimensional echocardiography was performed within one hour of MPI at rest. Image quality did not constitute an exclusion criterion. Analysis was performed by five experienced observers and by two novices.|LVEF (%), end-diastolic and end-systolic volume/BSA (ml/m2) were for uncorrected AutoEF 54 +/- 10, 51 +/- 16, 24 +/- 13, for corrected AutoEF 53 +/- 10, 53 +/- 18, 26 +/- 14, for manual Simpson 51 +/- 11, 56 +/- 20, 28 +/- 15, and for MPI 52 +/- 12, 67 +/- 26, 35 +/- 23. The required time for analysis was significantly different for all four echocardiographic methods and was for uncorrected AutoEF 79 +/- 5 s, for corrected AutoEF 159 +/- 46 s, for manual Simpson 177 +/- 66 s, and for visual assessment 33 +/- 14 s. Compared with the expert manual Simpson, limits of agreement for novice corrected AutoEF was lower than for novice manual Simpson (0.8 +/- 10.5 vs. -3.2 +/- 11.4 LVEF percentage points). Calculated for experts and with LVEF (%) categorized into < 30, 30-44, 45-54 and > or = 55, kappa measure of agreement was moderate (0.44-0.53) for all method comparisons (uncorrected AutoEF not evaluated).|Corrected AutoEF reduces the variation in measurements compared with manual planimetry, without increasing the time required. The method seems especially suited for unexperienced readers.',\n",
              "  'authors': ['Maret, Eva',\n",
              "   'Brudin, Lars',\n",
              "   'Lindstrom, Lena',\n",
              "   'Nylander, Eva',\n",
              "   'Ohlsson, Jan L',\n",
              "   'Engvall, Jan E'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Cardiovascular ultrasound',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Artery Disease',\n",
              "   'Echocardiography',\n",
              "   'Endocardium',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Stroke Volume',\n",
              "   'Ventricular Dysfunction, Left'],\n",
              "  'pmid': '19014461',\n",
              "  'title': 'Computer-assisted determination of left ventricular endocardial borders reduces variability in the echocardiographic assessment of ejection fraction.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Biological Ecosystem Networks (BENs) are webs of biological species (nodes) establishing trophic relationships (links). Experimental confirmation of all possible links is difficult and generates a huge volume of information. Consequently, computational prediction becomes an important goal. Artificial Neural Networks (ANNs) are Machine Learning (ML) algorithms that may be used to predict BENs, using as input Shannon entropy information measures (Shk) of known ecosystems to train them. However, it is difficult to select a priori which ANN topology will have a higher accuracy. Interestingly, Auto Machine Learning (AutoML) methods focus on the automatic selection of the more efficient ML algorithms for specific problems. In this work, a preliminary study of a new approach to AutoML selection of ANNs is proposed for the prediction of BENs. We call it the Net-Net AutoML approach, because it uses for the first time Shk values of both networks involving BENs (networks to be predicted) and ANN topologies (networks to be tested). Twelve types of classifiers have been tested for the Net-Net model including linear, Bayesian, trees-based methods, multilayer perceptrons and deep neuronal networks. The best Net-Net AutoML model for 338,050 outputs of 10 ANN topologies for links of 69 BENs was obtained with a deep fully connected neuronal network, characterized by a test accuracy of 0.866 and a test AUROC of 0.935. This work paves the way for the application of Net-Net AutoML to other systems or ML algorithms.',\n",
              "  'authors': ['Barreiro, Enrique',\n",
              "   'Munteanu, Cristian R',\n",
              "   'Cruz-Monteagudo, Maykel',\n",
              "   'Pazos, Alejandro',\n",
              "   'González-Díaz, Humbert'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30120369',\n",
              "  'title': 'Net-Net Auto Machine Learning (AutoML) Prediction of Complex Ecosystems.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1555-8584',\n",
              "  'IF_2013': '5.377',\n",
              "  'IF_2014': '4.974',\n",
              "  'IF_2015': '4.076',\n",
              "  'IF_2016': '3.9',\n",
              "  'IF_2017': '5.216',\n",
              "  'ISSN': '1547-6286',\n",
              "  'ISSN_online': '1555-8584',\n",
              "  'ISSN_print': '1547-6286',\n",
              "  'IsoAbbr': 'RNA Biol',\n",
              "  'JrId': 32563,\n",
              "  'MedAbbr': 'RNA Biol',\n",
              "  'NLMid': '101235328',\n",
              "  'Titre': 'RNA Biol',\n",
              "  'abstract': \"Despite more than 50 years of effort, the origin of the genetic code remains enigmatic. Among different theories, the stereochemical hypothesis suggests that the code evolved as a consequence of direct interactions between amino acids and appropriate bases. If indeed true, such physicochemical foundation of the mRNA/protein relationship could also potentially lead to novel principles of protein-mRNA interactions in general. Inspired by this promise, we have recently explored the connection between the physicochemical properties of mRNAs and their cognate proteins at the proteome level. Using experimentally and computationally derived measures of solubility of amino acids in aqueous solutions of pyrimidine analogs together with knowledge-based interaction preferences of amino acids for different nucleobases, we have revealed a statistically significant matching between the composition of mRNA coding sequences and the base-binding preferences of their cognate protein sequences. Our findings provide strong support for the stereochemical hypothesis of genetic code's origin and suggest the possibility of direct complementary interactions between mRNAs and cognate proteins even in present-day cells. \",\n",
              "  'authors': ['Polyansky, Anton A', 'Hlevnjak, Mario', 'Zagrovic, Bojan'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'RNA biology',\n",
              "  'keywords': ['Amino Acids',\n",
              "   'Chemical Phenomena',\n",
              "   'Evolution, Molecular',\n",
              "   'Genetic Code',\n",
              "   'Knowledge Bases',\n",
              "   'Models, Molecular',\n",
              "   'Proteins',\n",
              "   'Proteome',\n",
              "   'Pyrimidines',\n",
              "   'RNA, Messenger'],\n",
              "  'pmid': '23945356',\n",
              "  'title': 'Proteome-wide analysis reveals clues of complementary interactions between mRNAs and their cognate proteins as the physicochemical foundation of the genetic code.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'De novo molecular design and in silico prediction of polypharmacological profiles are emerging research topics that will profoundly affect the future of drug discovery and chemical biology. The goal is to identify the macromolecular targets of new chemical agents. Although several computational tools for predicting such targets are publicly available, none of these methods was explicitly designed to predict target engagement by de novo-designed molecules. Here we present the development and practical application of a unique technique, self-organizing map-based prediction of drug equivalence relationships (SPiDER), that merges the concepts of self-organizing maps, consensus scoring, and statistical analysis to successfully identify targets for both known drugs and computer-generated molecular scaffolds. We discovered a potential off-target liability of fenofibrate-related compounds, and in a comprehensive prospective application, we identified a multitarget-modulating profile of de novo designed molecules. These results demonstrate that SPiDER may be used to identify innovative compounds in chemical biology and in the early stages of drug discovery, and help investigate the potential side effects of drugs and their repurposing options. ',\n",
              "  'authors': ['Reker, Daniel',\n",
              "   'Rodrigues, Tiago',\n",
              "   'Schneider, Petra',\n",
              "   'Schneider, Gisbert'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Chemical Engineering',\n",
              "   'Drug Discovery',\n",
              "   'Drug Repositioning',\n",
              "   'Macromolecular Substances',\n",
              "   'Polypharmacology',\n",
              "   'Software'],\n",
              "  'pmid': '24591595',\n",
              "  'title': 'Identifying the macromolecular targets of de novo-designed chemical entities through self-organizing map consensus.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '2047-7538',\n",
              "  'IF_2013': '8.476',\n",
              "  'IF_2014': '14.603',\n",
              "  'IF_2015': '13.6',\n",
              "  'IF_2016': '14.098',\n",
              "  'IF_2017': '13.625',\n",
              "  'ISSN': '2095-5545',\n",
              "  'ISSN_online': '2047-7538',\n",
              "  'ISSN_print': '2095-5545',\n",
              "  'IsoAbbr': 'Light Sci Appl',\n",
              "  'JrId': 41525,\n",
              "  'MedAbbr': 'Light Sci Appl',\n",
              "  'NLMid': '101610753',\n",
              "  'Titre': 'Light Sci Appl',\n",
              "  'abstract': 'The smooth and tailorable spectral response of Bragg mirrors has driven their pervasive use in optical systems requiring customizable spectral control of beam propagation. However, the simple nature of Bragg mirror reflection prevents their application to the control of important polarization states such as circular polarization. While helical and gyroid-based nanostructures exhibiting circular dichroism have been developed extensively to address this limitation, they are often restricted by the spectral inconsistency of their optical response. Here we present the fabrication and characterization of quadruple-gyroid 4srs nanostructures exhibiting bio-inspired Bragg-mirror-like circular dichroism: a smooth and uniform band of circular dichroism reminiscent of the spectrum of a simple multilayer Bragg-mirror. Furthermore, we demonstrate that the circular dichroism produced by 4srs nanostructures are robust to changes in incident angle and beam collimation, providing a new platform to create and engineer circular dichroism for functional circular polarization manipulation.',\n",
              "  'authors': ['Cumming, Benjamin P',\n",
              "   'Schröder-Turk, Gerd E',\n",
              "   'Debbarma, Sukanta',\n",
              "   'Gu, Min'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Light, science & applications',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30167193',\n",
              "  'title': 'Bragg-mirror-like circular dichroism in bio-inspired quadruple-gyroid 4srs nanostructures.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1439-3999',\n",
              "  'IF_2013': '0.665',\n",
              "  'IF_2014': '0.459',\n",
              "  'IF_2015': '0.689',\n",
              "  'IF_2016': '0.651',\n",
              "  'IF_2017': '0.882',\n",
              "  'ISSN': '0023-2165',\n",
              "  'ISSN_online': '1439-3999',\n",
              "  'ISSN_print': '0023-2165',\n",
              "  'IsoAbbr': 'Klin Monbl Augenheilkd',\n",
              "  'JrId': 5436,\n",
              "  'MedAbbr': 'Klin Monbl Augenheilkd',\n",
              "  'NLMid': '0014133',\n",
              "  'Titre': 'Klin Monbl Augenheilkd',\n",
              "  'abstract': 'The number of medical screenings is increasing. This fact is specially pronounced in the field of occupational health. Due to restricted financial resources and limitations in time there is a need for rationalisation in medical screenings.|Possible uses of personal computers (PC) enabling organisational and financial benefits in medical screenings are suggested. By means of a test for colour-blindness it is shown how unfavourable technical conditions may be by-passed.|Rationalisation in medical screenings can be achieved when using the PC as a multi-functional screening-device as well as an expert system, measuring apparatus and as a communication-device. Due to a lack of literature it is difficult to discuss possible effects of technical differences between PC-based screenings of visual functions and screenings where \"traditional\". Reliability, sensitivity and validity is insufficiently investigated for many of the commercial available screening-devices based on a PC.',\n",
              "  'authors': ['Menozzi, M'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Klinische Monatsblatter fur Augenheilkunde',\n",
              "  'keywords': ['Color Perception Tests',\n",
              "   'Data Collection',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Microcomputers',\n",
              "   'Software',\n",
              "   'Vision Screening',\n",
              "   'Workplace'],\n",
              "  'pmid': '7609400',\n",
              "  'title': '[Personal computer used in visual function screening].',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1558-349X',\n",
              "  'IF_2013': '2.283',\n",
              "  'IF_2014': '2.836',\n",
              "  'IF_2015': '2.929',\n",
              "  'IF_2016': '2.993',\n",
              "  'IF_2017': '3.383',\n",
              "  'ISSN': '1546-1440',\n",
              "  'ISSN_online': '1558-349X',\n",
              "  'ISSN_print': '1546-1440',\n",
              "  'IsoAbbr': 'J Am Coll Radiol',\n",
              "  'JrId': 33174,\n",
              "  'MedAbbr': 'J Am Coll Radiol',\n",
              "  'NLMid': '101190326',\n",
              "  'Titre': 'J Am Coll Radiol',\n",
              "  'abstract': 'Enterprise imaging has channeled various technological innovations to the field of clinical radiology, ranging from advanced imaging equipment and postacquisition iterative reconstruction tools to image analysis and computer-aided detection tools. More recently, the advancements in the field of quantitative image analysis coupled with machine learning-based data analytics, classification, and integration have ushered us into the era of radiomics, which has tremendous potential in clinical decision support as well as drug discovery. There are important issues to consider to incorporate radiomics as a clinically applicable system and a commercially viable solution. In this two-part series, we offer insights into the development of the translational pipeline for radiomics from methodology to clinical implementation (Part 1) and from that to enterprise development (Part 2).',\n",
              "  'authors': ['Shaikh, Faiq',\n",
              "   'Franc, Benjamin',\n",
              "   'Allen, Erastus',\n",
              "   'Sala, Evis',\n",
              "   'Awan, Omer',\n",
              "   'Hendrata, Kenneth',\n",
              "   'Halabi, Safwan',\n",
              "   'Mohiuddin, Sohaib',\n",
              "   'Malik, Sana',\n",
              "   'Hadley, Dexter',\n",
              "   'Shrestha, Rasu'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of the American College of Radiology : JACR',\n",
              "  'keywords': ['Diagnostic Imaging',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Precision Medicine',\n",
              "   'Radiology',\n",
              "   'Translational Medical Research'],\n",
              "  'pmid': '29366600',\n",
              "  'title': 'Translational Radiomics: Defining the Strategy Pipeline and Considerations for Application-Part 1: From Methodology to Clinical Implementation.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1573-6776',\n",
              "  'IF_2013': '1.736',\n",
              "  'IF_2014': '1.591',\n",
              "  'IF_2015': '1.639',\n",
              "  'IF_2016': '1.73',\n",
              "  'IF_2017': '1.846',\n",
              "  'ISSN': '0141-5492',\n",
              "  'ISSN_online': '1573-6776',\n",
              "  'ISSN_print': '0141-5492',\n",
              "  'IsoAbbr': 'Biotechnol. Lett.',\n",
              "  'JrId': 20267,\n",
              "  'MedAbbr': 'Biotechnol Lett',\n",
              "  'NLMid': '8008051',\n",
              "  'Titre': 'Biotechnol. Lett.',\n",
              "  'abstract': 'Artificial neural networks and genetic algorithms are used to model and optimize a fermentation medium for the production of the enzyme hydantoinase by Agrobacterium radiobacter. Experimental data reported in the literature were used to build two neural network models. The concentrations of four medium components served as inputs to the neural network models, and hydantoinase or cell concentration served as a single output of each model. Genetic algorithms were used to optimize the input space of the neural network models to find the optimum settings for maximum enzyme and cell production. Using this procedure, two artificial intelligence techniques have been effectively integrated to create a powerful tool for process modeling and optimization.',\n",
              "  'authors': ['Nagata, Yuko', 'Chu, Khim Hoong'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Biotechnology letters',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amidohydrolases',\n",
              "   'Bioreactors',\n",
              "   'Cell Culture Techniques',\n",
              "   'Cell Division',\n",
              "   'Computer Simulation',\n",
              "   'Culture Media',\n",
              "   'Feedback',\n",
              "   'Fermentation',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Engineering',\n",
              "   'Quality Control',\n",
              "   'Rhizobium'],\n",
              "  'pmid': '14677708',\n",
              "  'title': 'Optimization of a fermentation medium using neural networks and genetic algorithms.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1520-6882',\n",
              "  'IF_2013': '5.825',\n",
              "  'IF_2014': '5.636',\n",
              "  'IF_2015': '5.886',\n",
              "  'IF_2016': '6.32',\n",
              "  'IF_2017': '6.042',\n",
              "  'ISSN': '0003-2700',\n",
              "  'ISSN_online': '1520-6882',\n",
              "  'ISSN_print': '0003-2700',\n",
              "  'IsoAbbr': 'Anal. Chem.',\n",
              "  'JrId': 519,\n",
              "  'MedAbbr': 'Anal Chem',\n",
              "  'NLMid': '0370536',\n",
              "  'Titre': 'Anal. Chem.',\n",
              "  'abstract': 'Fuzzy c-means (FCM) clustering is an unsupervised method derived from fuzzy logic that is suitable for solving multiclass and ambiguous clustering problems. In this study, FCM clustering is applied to cluster metabolomics data. FCM is performed directly on the data matrix to generate a membership matrix which represents the degree of association the samples have with each cluster. The method is parametrized with the number of clusters (C) and the fuzziness coefficient (m), which denotes the degree of fuzziness in the algorithm. Both have been optimized by combining FCM with partial least-squares (PLS) using the membership matrix as the Y matrix in the PLS model. The quality parameters R(2)Y and Q(2) of the PLS model have been used to monitor and optimize C and m. Data of metabolic profiles from three gene types of Escherichia coli were used to demonstrate the method above. Different multivariable analysis methods have been compared. Principal component analysis failed to model the metabolite data, while partial least-squares discriminant analysis yielded results with overfitting. On the basis of the optimized parameters, the FCM was able to reveal main phenotype changes and individual characters of three gene types of E. coli. Coupled with PLS, FCM provides a powerful research tool for metabolomics with improved visualization, accurate classification, and outlier estimation.',\n",
              "  'authors': ['Li, Xiang',\n",
              "   'Lu, Xin',\n",
              "   'Tian, Jing',\n",
              "   'Gao, Peng',\n",
              "   'Kong, Hongwei',\n",
              "   'Xu, Guowang'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytical chemistry',\n",
              "  'keywords': ['Cluster Analysis',\n",
              "   'Escherichia coli',\n",
              "   'Fuzzy Logic',\n",
              "   'Least-Squares Analysis',\n",
              "   'Metabolomics',\n",
              "   'Mutation'],\n",
              "  'pmid': '19408956',\n",
              "  'title': 'Application of fuzzy c-means clustering in data analysis of metabolomics.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Although there have been reports of the successful diagnosis of skin disorders using deep learning, unrealistically large clinical image datasets are required for artificial intelligence (AI) training. We created datasets of standardized nail images using a region-based convolutional neural network (R-CNN) trained to distinguish the nail from the background. We used R-CNN to generate training datasets of 49,567 images, which we then used to fine-tune the ResNet-152 and VGG-19 models. The validation datasets comprised 100 and 194 images from Inje University (B1 and B2 datasets, respectively), 125 images from Hallym University (C dataset), and 939 images from Seoul National University (D dataset). The AI (ensemble model; ResNet-152 + VGG-19 + feedforward neural networks) results showed test sensitivity/specificity/ area under the curve values of (96.0 / 94.7 / 0.98), (82.7 / 96.7 / 0.95), (92.3 / 79.3 / 0.93), (87.7 / 69.3 / 0.82) for the B1, B2, C, and D datasets. With a combination of the B1 and C datasets, the AI Youden index was significantly (p = 0.01) higher than that of 42 dermatologists doing the same assessment manually. For B1+C and B2+ D dataset combinations, almost none of the dermatologists performed as well as the AI. By training with a dataset comprising 49,567 images, we achieved a diagnostic accuracy for onychomycosis using deep learning that was superior to that of most of the dermatologists who participated in this study.',\n",
              "  'authors': ['Han, Seung Seog',\n",
              "   'Park, Gyeong Hun',\n",
              "   'Lim, Woohyung',\n",
              "   'Kim, Myoung Shin',\n",
              "   'Na, Jung Im',\n",
              "   'Park, Ilwoo',\n",
              "   'Chang, Sung Eun'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Algorithms',\n",
              "   'Area Under Curve',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Dermatologists',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Foot Dermatoses',\n",
              "   'Hand Dermatoses',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Onychomycosis',\n",
              "   'Young Adult'],\n",
              "  'pmid': '29352285',\n",
              "  'title': 'Deep neural networks show an equivalent and often superior performance to dermatologists in onychomycosis diagnosis: Automatic construction of onychomycosis datasets by region-based convolutional deep neural network.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1827-1618',\n",
              "  'IF_2013': '0.481',\n",
              "  'IF_2014': '0.53',\n",
              "  'IF_2015': '0.752',\n",
              "  'IF_2016': '0.695',\n",
              "  'IF_2017': '0.571',\n",
              "  'ISSN': '0026-4725',\n",
              "  'ISSN_online': '1827-1618',\n",
              "  'ISSN_print': '0026-4725',\n",
              "  'IsoAbbr': 'Minerva Cardioangiol',\n",
              "  'JrId': 5827,\n",
              "  'MedAbbr': 'Minerva Cardioangiol',\n",
              "  'NLMid': '0400725',\n",
              "  'Titre': 'Minerva Cardioangiol',\n",
              "  'abstract': 'Catheter ablation of cardiac arrhythmia has evolved to an important treatment modality for patients with various cardiac rhythm abnormalities. Over the past three decades, the understanding of arrhythmia mechanisms and technology for catheter based diagnostic and ablation procedures have evolved rapidly. New technologies and therapies for treatment of cardiac arrhythmia include ablation catheters designed to destroy targeted tissue with improved precision and safety, robotic systems to guide accurate catheter movements, electrical mapping systems, and improved imaging to complement such systems. This review focuses on improved imaging modalities used in the modern electrophysiology laboratory.',\n",
              "  'authors': ['Nazarian, S'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Minerva cardioangiologica',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Arrhythmias, Cardiac',\n",
              "   'Atrial Fibrillation',\n",
              "   'Body Surface Potential Mapping',\n",
              "   'Catheter Ablation',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '21135812',\n",
              "  'title': 'New technologies and therapies for cardiac arrhythmias.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Robotic prostatectomy (RP) has been reported to be technically challenging in patients with a history of prior complex lower abdominal or pelvic surgery, morbid obesity, large prostate, prior pelvic irradiation, neoadjuvant hormonal therapy, or prior prostate surgery. In this paper, we report an experience of RP in a prostate cancer patient with abdominoperineal resection, adjuvant chemotherapy, and pelvic irradiation for rectal cancer.',\n",
              "  'authors': ['Ham, Won Sik',\n",
              "   'Kim, Sang Woon',\n",
              "   'Kim, Won Tae',\n",
              "   'Park, Sung Yul',\n",
              "   'Choi, Young Deuk'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Colostomy',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Invasiveness',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Rectal Neoplasms',\n",
              "   'Robotics'],\n",
              "  'pmid': '19061364',\n",
              "  'title': 'Robotic prostatectomy in patient with an abdominoperineal resection.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': \"The International Mouse Phenotyping Consortium (IMPC) web portal (http://www.mousephenotype.org) provides the biomedical community with a unified point of access to mutant mice and rich collection of related emerging and existing mouse phenotype data. IMPC mouse clinics worldwide follow rigorous highly structured and standardized protocols for the experimentation, collection and dissemination of data. Dedicated 'data wranglers' work with each phenotyping center to collate data and perform quality control of data. An automated statistical analysis pipeline has been developed to identify knockout strains with a significant change in the phenotype parameters. Annotation with biomedical ontologies allows biologists and clinicians to easily find mouse strains with phenotypic traits relevant to their research. Data integration with other resources will provide insights into mammalian gene function and human disease. As phenotype data become available for every gene in the mouse, the IMPC web portal will become an invaluable tool for researchers studying the genetic contributions of genes to human diseases. \",\n",
              "  'authors': ['Koscielny, Gautier',\n",
              "   'Yaikhom, Gagarine',\n",
              "   'Iyer, Vivek',\n",
              "   'Meehan, Terrence F',\n",
              "   'Morgan, Hugh',\n",
              "   'Atienza-Herrero, Julian',\n",
              "   'Blake, Andrew',\n",
              "   'Chen, Chao-Kung',\n",
              "   'Easty, Richard',\n",
              "   'Di Fenza, Armida',\n",
              "   'Fiegel, Tanja',\n",
              "   'Grifiths, Mark',\n",
              "   'Horne, Alan',\n",
              "   'Karp, Natasha A',\n",
              "   'Kurbatova, Natalja',\n",
              "   'Mason, Jeremy C',\n",
              "   'Matthews, Peter',\n",
              "   'Oakley, Darren J',\n",
              "   'Qazi, Asfand',\n",
              "   'Regnart, Jack',\n",
              "   'Retha, Ahmad',\n",
              "   'Santos, Luis A',\n",
              "   'Sneddon, Duncan J',\n",
              "   'Warren, Jonathan',\n",
              "   'Westerberg, Henrik',\n",
              "   'Wilson, Robert J',\n",
              "   'Melvin, David G',\n",
              "   'Smedley, Damian',\n",
              "   'Brown, Steve D M',\n",
              "   'Flicek, Paul',\n",
              "   'Skarnes, William C',\n",
              "   'Mallon, Ann-Marie',\n",
              "   'Parkinson, Helen'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Ontologies',\n",
              "   'Databases, Genetic',\n",
              "   'Internet',\n",
              "   'Mice',\n",
              "   'Mice, Knockout',\n",
              "   'Phenotype'],\n",
              "  'pmid': '24194600',\n",
              "  'title': 'The International Mouse Phenotyping Consortium Web Portal, a unified point of access for knockout mice and related phenotyping data.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'To study the evolution of the yeast protein interaction network, we first classified yeast proteins by their evolutionary histories into isotemporal categories, then analyzed the interaction tendencies within and between the categories, and finally reconstructed the main growth path. We found that two proteins tend to interact with each other if they are in the same or similar categories, but tended to avoid each other otherwise, and that network evolution mirrors the universal tree of life. These observations suggest synergistic selection during network evolution and provide insights into the hierarchical modularity of cellular networks.',\n",
              "  'authors': ['Qin, Hong', 'Lu, Henry H S', 'Wu, Wei B', 'Li, Wen-Hsiung'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Biological Evolution',\n",
              "   'Databases, Genetic',\n",
              "   'Fungi',\n",
              "   'Genome, Fungal',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Phylogeny',\n",
              "   'Saccharomyces cerevisiae',\n",
              "   'Saccharomyces cerevisiae Proteins'],\n",
              "  'pmid': '14557537',\n",
              "  'title': 'Evolution of the yeast protein interaction network.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1089-8638',\n",
              "  'IF_2013': '3.959',\n",
              "  'IF_2014': '4.333',\n",
              "  'IF_2015': '4.517',\n",
              "  'IF_2016': '4.632',\n",
              "  'IF_2017': '4.894',\n",
              "  'ISSN': '0022-2836',\n",
              "  'ISSN_online': '1089-8638',\n",
              "  'ISSN_print': '0022-2836',\n",
              "  'IsoAbbr': 'J. Mol. Biol.',\n",
              "  'JrId': 4967,\n",
              "  'MedAbbr': 'J Mol Biol',\n",
              "  'NLMid': '2985088R',\n",
              "  'Titre': 'J. Mol. Biol.',\n",
              "  'abstract': 'The proposal that coupled folding to binding is not an obligatory mechanism for intrinsically disordered (ID) proteins was put forward 10 years ago. The notion of fuzziness implies that conformational heterogeneity can be maintained upon interactions of ID proteins, which has a functional impact either on regulated assembly or activity of the corresponding complexes. Here I review how the concept has evolved in the past decade, via increasing experimental data providing insights into the mechanisms, pathways and regulatory modes. The effects of structural diversity and transient contacts on protein assemblies have been collected and systematically analyzed (Fuzzy Complexes Database, http://protdyn-database.org). Fuzziness has also been exploited as a framework to decipher molecular organization of higher-order protein structures. Quantification of conformational heterogeneity opens exciting future perspectives for drug discovery from small molecule-ID protein interactions to supramolecular assemblies.',\n",
              "  'authors': ['Fuxreiter, Monika'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of molecular biology',\n",
              "  'keywords': ['Biophysical Phenomena',\n",
              "   'Databases, Protein',\n",
              "   'Fuzzy Logic',\n",
              "   'Gene Expression Regulation',\n",
              "   'Humans',\n",
              "   'Intrinsically Disordered Proteins',\n",
              "   'Models, Molecular',\n",
              "   'Protein Binding',\n",
              "   'Protein Folding'],\n",
              "  'pmid': '29477337',\n",
              "  'title': 'Fuzziness in Protein Interactions-A Historical Perspective.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'Intracorporeal knot tying during urologic reconstructive surgery is one of the most technically challenging skills of laparoscopic surgery. We describe our experience using the Lapra-Ty clip to substitute for knot tying.|Our technique for minimizing knot tying entails the use of the Lapra-Ty clip during closure of the collecting system and renal parenchyma for laparoscopic partial nephrectomy, the vesicourethral anastomosis during robotic laparoscopic radical prostatectomy, and the collecting system during laparoscopic pyeloplasty. From October 2002 to July 2005 at our institution, 75 patients underwent laparoscopic partial nephrectomy, 300 underwent robotic laparoscopic radical prostatectomy, and 14 underwent laparoscopic pyeloplasty. We reviewed the charts retrospectively for intraoperative and postoperative parameters related to the use of these clips.|In the laparoscopic partial nephrectomy group, the mean tumor size, warm ischemia time, and estimated blood loss was 2.53 cm, 30.1 minutes, and 189 mL, respectively. Two postoperative urine leaks (2.7%) developed, and 3 patients experienced postoperative bleeding (4%). In the robotic laparoscopic radical prostatectomy group, the mean operative time was 295 minutes and the mean estimated blood loss was 303.6 mL. Only 3 patients had a urine leak (1%), and 4 patients had bladder neck contracture (1.3%). With regard to the laparoscopic pyeloplasty group, the mean operative time and estimated blood loss was 224 minutes and 36 mL, respectively. No intraoperative complications or urinary leaks occurred.|Using the Lapra-Ty clip, we have safely and efficiently supplemented knot tying in patients undergoing reconstructive laparoscopic surgery.',\n",
              "  'authors': ['Shalhav, Arieh L',\n",
              "   'Orvieto, Marcelo A',\n",
              "   'Chien, Gary W',\n",
              "   'Mikhail, Albert A',\n",
              "   'Zagaja, Gregory P',\n",
              "   'Zorn, Kevin C'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Prostatectomy',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Suture Techniques'],\n",
              "  'pmid': '16979736',\n",
              "  'title': 'Minimizing knot tying during reconstructive laparoscopic urology.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'The hyperspectral leaf reflectance in winter wheat was measured under 4 phosphorus levels at different growth stages, i.e. revival stage, jointing stage, tassel stage and grouting stage. And their first derivative of spectra were calculated and denoised by the threshold denoising method based on wavelet transform. After studying characteristics of the two kinds of spectra resulting from different phosphorus contents levels as well as correlations between leaf phosphorus contents and spectral values, sensitive wavebands and four kinds of absorption areas were extracted. Then the four kinds of absorption areas and their corresponding leaf phosphorus content were normalized and input to RBFNN. Results show that: (1) Sensitive wavebands for monitoring leaf phosphorus contents in original leaf spectra are 426-435 and 669-680 nm. (2) Sensitive wavebands in first derivative of spectra are 481-493 and 685-696 nm. (3) Trained RBFNN can learn and seize the linearity/non-linearity mapping between samples and output targets.',\n",
              "  'authors': ['Liu, Wei',\n",
              "   'Chang, Qing-Rui',\n",
              "   'Guo, Man',\n",
              "   'Xing, Dong-Xing',\n",
              "   'Yuan, Yong-Sheng'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Neural Networks (Computer)',\n",
              "   'Phosphorus',\n",
              "   'Plant Leaves',\n",
              "   'Spectrum Analysis',\n",
              "   'Triticum'],\n",
              "  'pmid': '21714267',\n",
              "  'title': '[Diagnosis of phosphorus nutrition in winter wheat based on first derivative spectra and radial basis function neural network].',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'By introducing the conflicting effects of dynamic changes in blood flow, volume, and blood oxygenation, Balloon model provides a biomechanical compelling interpretation of the BOLD signal. In order to obtain optimal estimates for both the states and parameters involved in this model, a joint filtering (estimate) method has been widely used. However, it is flawed in several aspects (i) Correlation or interaction between the states and parameters is incorporated despite its nonexistence in biophysical reality. (ii) A joint representation for states and parameters necessarily means the large dimension of state space and will in turn lead to huge numerical cost in implementation. Given this knowledge, a dual filtering approach is proposed and demonstrated in this paper as a highly competent alternative, which can not only provide more reliable estimates, but also in a more efficient way. The two approaches in our discussion will be based on unscented Kalman filter, which has become the algorithm of choice in numerous nonlinear estimation and machine learning applications.',\n",
              "  'authors': ['Liu, Cong', 'Hu, Zhenghui'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Biophysics',\n",
              "   'Blood Flow Velocity',\n",
              "   'Brain',\n",
              "   'Cerebrovascular Circulation',\n",
              "   'Computational Biology',\n",
              "   'Computer Simulation',\n",
              "   'Hemodynamics',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Biological',\n",
              "   'Models, Statistical',\n",
              "   'Oxygen',\n",
              "   'Software'],\n",
              "  'pmid': '22997541',\n",
              "  'title': 'Reliable and efficient approach of BOLD signal with dual Kalman filtering.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Conventional vascular interventional surgery (VIS) is manually performed under fluoroscopic guidance, requiring lead protection for the surgeons. In order to reduce the X-ray irradiation, the robot has been introduced to VIS in this study. The purpose of this article is to provide a comprehensive summary of the vascular interventional robot (VIR).|A retrospective review of the currently published literature on VIR is discussed. The overview details the key technologies of VIR.|Surgeons operate the robot far away from the patient, so protecting themselves from X-ray irradiation. The robot system carries out the VIS precisely and provides three-dimensional visualization of vasculature that can be used for preoperative planning and intra-operative navigation.|The use of the vascular interventional robot appears safe and feasible for vascular interventional surgery. VIR can offer radiation safety to surgeon and enhance surgical accuracy.',\n",
              "  'authors': ['Da, Liu', 'Zhang, Dapeng', 'Wang, Tianmiao'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Fluoroscopy',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Radiation Injuries',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Vascular Surgical Procedures'],\n",
              "  'pmid': '18803340',\n",
              "  'title': 'Overview of the vascular interventional robot.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'We describe an automated multidimensional approach for the analysis of flow cytometry data based on pattern classification. Flow cytometry is a widely used technique both for research and clinical purposes where it has become essential for the diagnosis and follow up of a wide spectrum of diseases, such as HIV-infection and neoplastic disorders. Flow cytometry data sets are composed of quite a large number of observations that can be viewed as elements of a n-dimensional space. The aim of the analysis of such data files is typically to classify groups of cellular events as specific populations with biological meaning. Despite significant improvements in data acquisition capabilities of flow cytometers, data analysis is still based on bi-dimensional strategies which were defined a long time ago. These are strongly dependent on the expertise of an expert operator, this approach being relatively subjective and potentially leading to unreliable results. Automated analysis of flow cytometry data is an essential step to improve reproducibility of the results. The proposed automated analysis was implemented on peripherial blood lymphocyte subsets from 307 samples stained and prepared in an identical way and it was capable of identifying all cell subsets present in each sample studied that could also be detected in the same data files by an expert operator. A highly significant correlation was found between the results obtained by an expert operator using a conventional manual method of analysis and those obtained using the implemented automated approach.',\n",
              "  'authors': ['Pedreira, Carlos Eduardo',\n",
              "   'Costa, Elaine S',\n",
              "   'Arroyo, M Elena',\n",
              "   'Almeida, Julia',\n",
              "   'Orfao, Alberto'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Flow Cytometry',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '18334408',\n",
              "  'title': 'A multidimensional classification approach for the automated analysis of flow cytometry data.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'A new fluorescence method by artificial neural network for the simultaneous determination of tryptophan and tyrosine was developed. The determination was carried out in the KH2PO4-K2HPO4 buffer solution (pH = 7.15) and at the EX (excitation) wavelength of 224 nm. In the range of 290-400 nm, the fluorescence intensities at fourteen wavelengths were taken as characteristic parameters of the artificial neural network, and the samples were arranged by the method of equality design. The mean recoveries of tryptophan and tyrosine were 100.9% and 101.6% respectively. The RSDs of the results were 4.18% and 4.17%. The method has been applied to the determination of tryptophan and tyrosine in compound amino acid injection, and the relative errors were 4.0% and 2.6%, respectively. The results were better than those of other networks in training speed and accuracy. In conclusion, the new network spectrofluorimetry is a good choice for multicomponent resolving analysis.',\n",
              "  'authors': ['Wu, Gen-hua', 'He, Chi-yang', 'Chen, Rong'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Amino Acids',\n",
              "   'Injections',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Spectrometry, Fluorescence',\n",
              "   'Tryptophan',\n",
              "   'Tyrosine'],\n",
              "  'pmid': '12961882',\n",
              "  'title': '[Simultaneous determination of tryptophan and tyrosine in injection by artificial neural network and fluorescence spectrum method].',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1097-9867',\n",
              "  'IF_2013': '1.335',\n",
              "  'IF_2014': '1.202',\n",
              "  'IF_2015': '1.566',\n",
              "  'IF_2016': '1.86',\n",
              "  'IF_2017': '1.945',\n",
              "  'ISSN': '1083-7450',\n",
              "  'ISSN_online': '1097-9867',\n",
              "  'ISSN_print': '1083-7450',\n",
              "  'IsoAbbr': 'Pharm Dev Technol',\n",
              "  'JrId': 20935,\n",
              "  'MedAbbr': 'Pharm Dev Technol',\n",
              "  'NLMid': '9610932',\n",
              "  'Titre': 'Pharm Dev Technol',\n",
              "  'abstract': \"The purpose of this study was to evaluate four commercially available artificial neural network (ANN) software programs: NeuroShell2 v3.0, BrainMaker v3.7, CAD/Chem v5.0, and NeuralWorks Professional II/Plus for prediction of in vitro dissolution-time profiles of controlled-release tablets containing a model sympathomimetic drug. Seven independent formulation variables and three other tablet variables (moisture content of granules, granule particle size, and tablet hardness), for 22 tablet formulations, were used as the ANN model input. In vitro dissolution time-profiles at 10 different sampling times were used as the output. The models' optimum architectures were determined for each ANN software by varying the number of hidden layers and number of nodes in hidden layer(s). The ANN developed from the four software programs were validated by predicting the in vitro dissolution time-profiles of each of the 19 formulations, which were excluded from the training process. Although the same data set was used, the optimum ANN architectures generated from the four software programs were different. Using the four optimum ANN models, the plots of predicted vs. observed percentage of drug dissolved gave slopes ranging from 0.95 to 1.01 and r2 values ranging from 0.95 to 0.99 for all 190 dissolution data points for the 19 training formulations. The difference factors (f1) and similarity factors (f2) between the ANN predicted and the observed in vitro dissolution profiles were also used to compare the predictions for the four software programs. It was concluded that the four programs provided reasonable predictions of in vitro dissolution profiles for the data set employed in this study, with NeuralShell2 showing the best overall prediction.\",\n",
              "  'authors': ['Chen, Yixin',\n",
              "   'Jiao, Tianjie',\n",
              "   'McCall, Troy W',\n",
              "   'Baichwal, Anand R',\n",
              "   'Meyer, Marvin C'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Pharmaceutical development and technology',\n",
              "  'keywords': ['Delayed-Action Preparations',\n",
              "   'Excipients',\n",
              "   'Hardness',\n",
              "   'Kinetics',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Particle Size',\n",
              "   'Predictive Value of Tests',\n",
              "   'Solubility',\n",
              "   'Sympathomimetics'],\n",
              "  'pmid': '12229268',\n",
              "  'title': 'Comparison of four artificial neural network software programs used to predict the in vitro dissolution of controlled-release tablets.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '2038-2529',\n",
              "  'IF_2013': '1.09',\n",
              "  'IF_2014': '1.269',\n",
              "  'IF_2015': '1.071',\n",
              "  'IF_2016': '1.233',\n",
              "  'IF_2017': '1.304',\n",
              "  'ISSN': '0300-8916',\n",
              "  'ISSN_online': '2038-2529',\n",
              "  'ISSN_print': '0300-8916',\n",
              "  'IsoAbbr': 'Tumori',\n",
              "  'JrId': 7863,\n",
              "  'MedAbbr': 'Tumori',\n",
              "  'NLMid': '0111356',\n",
              "  'Titre': 'Tumori',\n",
              "  'abstract': 'Noninvasive diagnostic methods such as dermoscopy, sonography, palpation or combined approaches have been developed in an attempt to preoperatively estimate melanoma thickness. However, the clinical presentation is often complex and the evaluation subjective. Multispectral image analysis of melanomas allows selection of features related to the content and distribution of absorbers, mainly melanin and hemoglobin, present within the lesion. Hence, it is reasonable to assume that the same features might be useful to predict melanoma thickness.|A multispectral image system was used to analyze in vivo 1939 pigmented skin lesions. The lesion selection was based on clinical and/or dermoscopic features that supported a suspicion for melanoma. All the lesions were then subjected to surgery for the histopathological diagnosis, and 250 were melanomas. From the multispectral images of the melanomas, we selected 12 features, seven of which were used to train and test an artificial neural network on 155 and 95 melanomas, respectively.|Sensitivity (i.e., melanoma > or = 0.75 mm thick correctly classified) and specificity (i.e., melanoma < 0.75 mm thick correctly classified) evaluated from the receiving operating characteristic curves ranged from 76 to 90% and from 91 to 74%, respectively.|Our approach provides results similar to those obtained with other methods and has the advantage that it is not related to the expertise of the clinician. In addition, the physical interpretation of the selected features suggests a possible role of spectrophotometry as an objective method to study the natural history of the early phases of the disease.',\n",
              "  'authors': ['Marchesini, Renato',\n",
              "   'Bono, Aldo',\n",
              "   'Tomatis, Stefano',\n",
              "   'Bartoli, Cesare',\n",
              "   'Colombo, Ambrogio',\n",
              "   'Lualdi, Manuela',\n",
              "   'Carrara, Mauro'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Tumori',\n",
              "  'keywords': ['Adult',\n",
              "   'Diagnostic Imaging',\n",
              "   'Humans',\n",
              "   'Melanoma',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'ROC Curve',\n",
              "   'Retrospective Studies',\n",
              "   'Skin Neoplasms',\n",
              "   'Spectrophotometry'],\n",
              "  'pmid': '17557564',\n",
              "  'title': 'In vivo evaluation of melanoma thickness by multispectral imaging and an artificial neural network. A retrospective study on 250 cases of cutaneous melanoma.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"As a structure-based image compression technology, fractal image compression (FIC) has been applied not only in image coding but also in many important image processing algorithms. However, two main bottlenecks restrained the develop and application of FIC for a long time. First, the encoding phase of FIC is time-consuming. Second, the quality of the reconstructed images for some images which have low structure-similarity is usually unacceptable. Based on the absolute value of Pearson's correlation coefficient (APCC), we had proposed an accelerating method to significantly speed up the encoding of FIC. In this paper, we make use of the sparse searching strategy to greatly improve the quality of the reconstructed images in FIC. We call it the sparse fractal image compression (SFIC). Furthermore, we combine both the APCC-based accelerating method and the sparse searching strategy to propose the fast sparse fractal image compression (FSFIC), which can effectively improve the two main bottlenecks of FIC. The experimental results show that the proposed algorithm greatly improves both the efficiency and effectiveness of FIC.\",\n",
              "  'authors': ['Wang, Jianji',\n",
              "   'Chen, Pei',\n",
              "   'Xi, Bao',\n",
              "   'Liu, Jianyi',\n",
              "   'Zhang, Yi',\n",
              "   'Yu, Shujian'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Data Compression',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '28886137',\n",
              "  'title': 'Fast sparse fractal image compression.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-5681',\n",
              "  'IF_2013': '0.655',\n",
              "  'IF_2014': '0.495',\n",
              "  'IF_2015': '0.528',\n",
              "  'IF_2016': '0.624',\n",
              "  'IF_2017': '0.652',\n",
              "  'ISSN': '1748-5673',\n",
              "  'ISSN_online': '1748-5681',\n",
              "  'ISSN_print': '1748-5673',\n",
              "  'IsoAbbr': 'Int J Data Min Bioinform',\n",
              "  'JrId': 34214,\n",
              "  'MedAbbr': 'Int J Data Min Bioinform',\n",
              "  'NLMid': '101279469',\n",
              "  'Titre': 'Int J Data Min Bioinform',\n",
              "  'abstract': 'To determine the structure of a protein by X-ray crystallography, the protein needs to be purified and crystallized first. However, some proteins cannot be crystallized. This makes the average cost of protein structure determination much higher. Thus it is desired to predict the crystallizability of a protein by a computational method before starting the wet-lab procedure. Features from the primary structure of a target protein are collected first. With a proper set of features, protein crystallizability can be predicted with a high accuracy. In this research, 74 features from previous researches are re-examined by two filter-mode feature selection methods. The selected features are then used for crystallization prediction by three versions of AdaBoost. The Support Vector Machines (SVMs) are also tested for comparison. The best prediction accuracy of AdaBoost reaches 93 percent and 48 important features are identified from the collected 74 features.',\n",
              "  'authors': ['Hsieh, Cheng-Wei', 'Hsu, Hui-Huang', 'Pai, Tun-Wen'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'International journal of data mining and bioinformatics',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Crystallization',\n",
              "   'Crystallography, X-Ray',\n",
              "   'Databases, Protein',\n",
              "   'Proteins',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '23777177',\n",
              "  'title': 'Protein crystallization prediction with AdaBoost.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Stochastic learning for a kind of feature detectors is studied. The technique is applied to multilayer neural networks intended for T-C discrimination. After evaluating the generalization error function, the training and generalization curves are found by Monte Carlo simulations. Different phase transitions take place and the overall behaviour agrees with previous discussions concerning designs of the same type.',\n",
              "  'authors': ['Romeo, A'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Monte Carlo Method',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Thermodynamics'],\n",
              "  'pmid': '8049788',\n",
              "  'title': 'Thermodynamic transitions in networks for letter distinction.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Co-infection with tuberculosis (TB) is the leading cause of death in HIV-infected individuals. However, diagnosis of TB, especially in the presence of an HIV co-infection, can be limiting due to the high inaccuracy associated with the use of conventional diagnostic methods. Here we report a gene signature that can identify a tuberculosis infection in patients co-infected with HIV as well as in the absence of HIV.|We analyzed global gene expression data from peripheral blood mononuclear cell (PBMC) samples of patients that were either mono-infected with HIV or co-infected with HIV/TB and used support vector machines to identify a gene signature that can distinguish between the two classes. We then validated our results using publically available gene expression data from patients mono-infected with TB.|Our analysis successfully identified a 251-gene signature that accurately distinguishes patients co-infected with HIV/TB from those infected with HIV only, with an overall accuracy of 81.4% (sensitivity = 76.2%, specificity = 86.4%). Furthermore, we show that our 251-gene signature can also accurately distinguish patients with active TB in the absence of an HIV infection from both patients with a latent TB infection and healthy controls (88.9-94.7% accuracy; 69.2-90% sensitivity and 90.3-100% specificity). We also demonstrate that the expression levels of the 251-gene signature diminish as a correlate of the length of TB treatment.|A 251-gene signature is described to (a) detect TB in the presence or absence of an HIV co-infection, and (b) assess response to treatment following anti-TB therapy.',\n",
              "  'authors': ['Dawany, Noor',\n",
              "   'Showe, Louise C',\n",
              "   'Kossenkov, Andrew V',\n",
              "   'Chang, Celia',\n",
              "   'Ive, Prudence',\n",
              "   'Conradie, Francesca',\n",
              "   'Stevens, Wendy',\n",
              "   'Sanne, Ian',\n",
              "   'Azzoni, Livio',\n",
              "   'Montaner, Luis J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Coinfection',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'HIV Infections',\n",
              "   'Humans',\n",
              "   'Leukocytes, Mononuclear',\n",
              "   'Male',\n",
              "   'Mycobacterium tuberculosis',\n",
              "   'Sensitivity and Specificity',\n",
              "   'South Africa',\n",
              "   'Support Vector Machine',\n",
              "   'Transcriptome',\n",
              "   'Tuberculosis'],\n",
              "  'pmid': '24587128',\n",
              "  'title': 'Identification of a 251 gene expression signature that can accurately detect M. tuberculosis in patients with and without HIV co-infection.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1791-7530',\n",
              "  'IF_2013': '1.872',\n",
              "  'IF_2014': '1.826',\n",
              "  'IF_2015': '1.895',\n",
              "  'IF_2016': '1.937',\n",
              "  'IF_2017': '1.865',\n",
              "  'ISSN': '0250-7005',\n",
              "  'ISSN_online': '1791-7530',\n",
              "  'ISSN_print': '0250-7005',\n",
              "  'IsoAbbr': 'Anticancer Res.',\n",
              "  'JrId': 592,\n",
              "  'MedAbbr': 'Anticancer Res',\n",
              "  'NLMid': '8102988',\n",
              "  'Titre': 'Anticancer Res.',\n",
              "  'abstract': 'The potential advantages of robot-assisted thoracoscopic esophagectomy (RATE) have yet to be verified. This study focused on the degree of lymph node dissection around the left recurrent laryngeal nerve (RLN).|A total of 21 consecutive patients who received RATE (RATE group) and 38 who received conventional thoracoscopic esophagectomy (TE) (TE group), both with extended lymphadenectomy, for thoracic squamous cell esophageal carcinoma, were enrolled in this study. We compared the numbers of lymph nodes dissected from the mediastinum, especially around the left RLN.|The median number of lymph nodes dissected from around the left RLN was 6 in the RATE group and 4 in the TE group, and the ratio of the dissected lymph nodes around the left RLN to all lymph nodes was significantly higher in the RATE group (p=0.048).|Our findings indicate that a robot-assisted surgical system enables for more extensive dissection of lymph nodes around the left RLN.',\n",
              "  'authors': ['Motoyama, Satoru',\n",
              "   'Sato, Yusuke',\n",
              "   'Wakita, Akiyuki',\n",
              "   'Kawakita, Yuta',\n",
              "   'Nagaki, Yushi',\n",
              "   'Imai, Kazuhiro',\n",
              "   'Minamiya, Yoshihiro'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Anticancer research',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Esophageal Neoplasms',\n",
              "   'Esophageal Squamous Cell Carcinoma',\n",
              "   'Esophagectomy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Recurrent Laryngeal Nerve',\n",
              "   'Robotics',\n",
              "   'Thoracic Surgery, Video-Assisted'],\n",
              "  'pmid': '30842166',\n",
              "  'title': 'Extensive Lymph Node Dissection Around the Left Laryngeal Nerve Achieved With Robot-assisted Thoracoscopic Esophagectomy.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Pertussis is highly contagious; thus, prompt identification of cases is essential to control outbreaks. Clinicians experienced with the disease can easily identify classic cases, where patients have bursts of rapid coughing followed by gasps, and a characteristic whooping sound. However, many clinicians have never seen a case, and thus may miss initial cases during an outbreak. The purpose of this project was to use voice-recognition software to distinguish pertussis coughs from croup and other coughs.|We collected a series of recordings representing pertussis, croup and miscellaneous coughing by children. We manually categorized coughs as either pertussis or non-pertussis, and extracted features for each category. We used Mel-frequency cepstral coefficients (MFCC), a sampling rate of 16 KHz, a frame Duration of 25 msec, and a frame rate of 10 msec. The coughs were filtered. Each cough was divided into 3 sections of proportion 3-4-3. The average of the 13 MFCCs for each section was computed and made into a 39-element feature vector used for the classification. We used the following machine learning algorithms: Neural Networks, K-Nearest Neighbor (KNN), and a 200 tree Random Forest (RF). Data were reserved for cross-validation of the KNN and RF. The Neural Network was trained 100 times, and the averaged results are presented.|After categorization, we had 16 examples of non-pertussis coughs and 31 examples of pertussis coughs. Over 90% of all pertussis coughs were properly classified as pertussis. The error rates were: Type I errors of 7%, 12%, and 25% and Type II errors of 8%, 0%, and 0%, using the Neural Network, Random Forest, and KNN, respectively.|Our results suggest that we can build a robust classifier to assist clinicians and the public to help identify pertussis cases in children presenting with typical symptoms.',\n",
              "  'authors': ['Parker, Danny',\n",
              "   'Picone, Joseph',\n",
              "   'Harati, Amir',\n",
              "   'Lu, Shuang',\n",
              "   'Jenkyns, Marion H',\n",
              "   'Polgreen, Philip M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Acoustics',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Child',\n",
              "   'Cough',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnosis, Differential',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Speech Recognition Software',\n",
              "   'Whooping Cough'],\n",
              "  'pmid': '24391730',\n",
              "  'title': 'Detecting paroxysmal coughing from pertussis cases using voice recognition technology.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['AY'],\n",
              "  'ESSN': '1098-2353',\n",
              "  'IF_2013': '1.159',\n",
              "  'IF_2014': '1.332',\n",
              "  'IF_2015': '1.316',\n",
              "  'IF_2016': '1.824',\n",
              "  'IF_2017': '1.908',\n",
              "  'ISSN': '0897-3806',\n",
              "  'ISSN_online': '1098-2353',\n",
              "  'ISSN_print': '0897-3806',\n",
              "  'IsoAbbr': 'Clin Anat',\n",
              "  'JrId': 20140,\n",
              "  'MedAbbr': 'Clin Anat',\n",
              "  'NLMid': '8809128',\n",
              "  'Titre': 'Clin Anat',\n",
              "  'abstract': 'Three approaches to study have been described in phenomenographic educational research: deep, strategic, and surface. Deep approaches to learning have been correlated with meaningful learning and academic success, whereas surface approaches produce an externalization of learning and poor outcomes. Students adopting a strategic approach adopt either a deep or surface approach in response to perceived examination demands. Despite being well known in Europe and Australia, this research paradigm has been applied sporadically in the United States. In this study, the approaches to study of a group of first year American medical students were collected using the Approaches and Study Skills Inventory for Students instrument at the beginning and end of their first year to find how consistent these approaches remained over time. At both times, the majority of participants adopted deep approaches, followed by strategic and then surface approaches. The percentage of participants using a surface approach grew during the first year but never exceeded 10%. The mean anatomy grades of students adopting each approach were then compared to find how each approach correlated with success in the course. Mean grades of students using a strategic approach were significantly higher than average at both times. Students who maintained a strategic approach throughout the first year had significantly higher mean grades than average while students who changed to a surface approach had significantly worse mean anatomy grades. Problem-based students had significantly higher scores on several deep submeasures than lecture-based peers and female students demonstrated greater fear of failure than male peers at both times.',\n",
              "  'authors': ['Ward, Peter J'],\n",
              "  'categories': ['ANATOMY & MORPHOLOGY'],\n",
              "  'journal': 'Clinical anatomy (New York, N.Y.)',\n",
              "  'keywords': ['Achievement',\n",
              "   'Anatomy',\n",
              "   'Attitude of Health Personnel',\n",
              "   'Education, Medical, Undergraduate',\n",
              "   'Educational Measurement',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Male',\n",
              "   'Students, Medical',\n",
              "   'Test Taking Skills',\n",
              "   'United States'],\n",
              "  'pmid': '20941709',\n",
              "  'title': \"First year medical students' approaches to study and their outcomes in a gross anatomy course.\",\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'Deep neural networks have achieved state-of-the-art accuracy at classifying molecules with respect to whether they bind to specific protein targets. A key breakthrough would occur if these models could reveal the fragment pharmacophores that are causally involved in binding. Extracting chemical details of binding from the networks could enable scientific discoveries about the mechanisms of drug actions. However, doing so requires shining light into the black box that is the trained neural network model, a task that has proved difficult across many domains. Here we show how the binding mechanism learned by deep neural network models can be interrogated, using a recently described attribution method. We first work with carefully constructed synthetic datasets, in which the molecular features responsible for \"binding\" are fully known. We find that networks that achieve perfect accuracy on held-out test datasets still learn spurious correlations, and we are able to exploit this nonrobustness to construct adversarial examples that fool the model. This makes these models unreliable for accurately revealing information about the mechanisms of protein-ligand binding. In light of our findings, we prescribe a test that checks whether a hypothesized mechanism can be learned. If the test fails, it indicates that the model must be simplified or regularized and/or that the training dataset requires augmentation.',\n",
              "  'authors': ['McCloskey, Kevin',\n",
              "   'Taly, Ankur',\n",
              "   'Monti, Federico',\n",
              "   'Brenner, Michael P',\n",
              "   'Colwell, Lucy J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31127041',\n",
              "  'title': 'Using attribution to decode binding mechanism in neural network models for chemistry.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'We propose a new recognition task in the area of computational paralinguistics: automatic recognition of eating conditions in speech, i. e., whether people are eating while speaking, and what they are eating. To this end, we introduce the audio-visual iHEARu-EAT database featuring 1.6 k utterances of 30 subjects (mean age: 26.1 years, standard deviation: 2.66 years, gender balanced, German speakers), six types of food (Apple, Nectarine, Banana, Haribo Smurfs, Biscuit, and Crisps), and read as well as spontaneous speech, which is made publicly available for research purposes. We start with demonstrating that for automatic speech recognition (ASR), it pays off to know whether speakers are eating or not. We also propose automatic classification both by brute-forcing of low-level acoustic features as well as higher-level features related to intelligibility, obtained from an Automatic Speech Recogniser. Prediction of the eating condition was performed with a Support Vector Machine (SVM) classifier employed in a leave-one-speaker-out evaluation framework. Results show that the binary prediction of eating condition (i. e., eating or not eating) can be easily solved independently of the speaking condition; the obtained average recalls are all above 90%. Low-level acoustic features provide the best performance on spontaneous speech, which reaches up to 62.3% average recall for multi-way classification of the eating condition, i. e., discriminating the six types of food, as well as not eating. The early fusion of features related to intelligibility with the brute-forced acoustic feature set improves the performance on read speech, reaching a 66.4% average recall for the multi-way classification task. Analysing features and classifier errors leads to a suitable ordinal scale for eating conditions, on which automatic regression can be performed with up to 56.2% determination coefficient.',\n",
              "  'authors': ['Hantke, Simone',\n",
              "   'Weninger, Felix',\n",
              "   'Kurle, Richard',\n",
              "   'Ringeval, Fabien',\n",
              "   'Batliner, Anton',\n",
              "   'Mousa, Amr El-Desoky',\n",
              "   'Schuller, Björn'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Audiovisual Aids',\n",
              "   'Automation',\n",
              "   'Databases as Topic',\n",
              "   'Eating',\n",
              "   'Female',\n",
              "   'Food',\n",
              "   'Hearing',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Regression Analysis',\n",
              "   'Self Report',\n",
              "   'Speech',\n",
              "   'Speech Recognition Software',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '27176486',\n",
              "  'title': 'I Hear You Eat and Speak: Automatic Recognition of Eating Condition and Food Type, Use-Cases, and Impact on ASR Performance.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1476-4687',\n",
              "  'IF_2013': '42.351',\n",
              "  'IF_2014': '41.456',\n",
              "  'IF_2015': '38.138',\n",
              "  'IF_2016': '40.137',\n",
              "  'IF_2017': '41.577',\n",
              "  'ISSN': '0028-0836',\n",
              "  'ISSN_online': '1476-4687',\n",
              "  'ISSN_print': '0028-0836',\n",
              "  'IsoAbbr': 'Nature',\n",
              "  'JrId': 6011,\n",
              "  'MedAbbr': 'Nature',\n",
              "  'NLMid': '0410462',\n",
              "  'Titre': 'Nature',\n",
              "  'abstract': \"How does the mammalian retina detect motion? This classic problem in visual neuroscience has remained unsolved for 50 years. In search of clues, here we reconstruct Off-type starburst amacrine cells (SACs) and bipolar cells (BCs) in serial electron microscopic images with help from EyeWire, an online community of 'citizen neuroscientists'. On the basis of quantitative analyses of contact area and branch depth in the retina, we find evidence that one BC type prefers to wire with a SAC dendrite near the SAC soma, whereas another BC type prefers to wire far from the soma. The near type is known to lag the far type in time of visual response. A mathematical model shows how such 'space-time wiring specificity' could endow SAC dendrites with receptive fields that are oriented in space-time and therefore respond selectively to stimuli that move in the outward direction from the soma. \",\n",
              "  'authors': ['Kim, Jinseop S',\n",
              "   'Greene, Matthew J',\n",
              "   'Zlateski, Aleksandar',\n",
              "   'Lee, Kisuk',\n",
              "   'Richardson, Mark',\n",
              "   'Turaga, Srinivas C',\n",
              "   'Purcaro, Michael',\n",
              "   'Balkam, Matthew',\n",
              "   'Robinson, Amy',\n",
              "   'Behabadi, Bardia F',\n",
              "   'Campos, Michael',\n",
              "   'Denk, Winfried',\n",
              "   'Seung, H Sebastian'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature',\n",
              "  'keywords': ['Amacrine Cells',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain Mapping',\n",
              "   'Crowdsourcing',\n",
              "   'Dendrites',\n",
              "   'Mice',\n",
              "   'Models, Neurological',\n",
              "   'Motion',\n",
              "   'Neural Pathways',\n",
              "   'Presynaptic Terminals',\n",
              "   'Retina',\n",
              "   'Retinal Bipolar Cells',\n",
              "   'Spatio-Temporal Analysis'],\n",
              "  'pmid': '24805243',\n",
              "  'title': 'Space-time wiring specificity supports direction selectivity in the retina.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1876-4673',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0910-5050',\n",
              "  'ISSN_online': '1876-4673',\n",
              "  'ISSN_print': '0910-5050',\n",
              "  'IsoAbbr': 'Jpn. J. Cancer Res.',\n",
              "  'JrId': 4485,\n",
              "  'MedAbbr': 'Jpn J Cancer Res',\n",
              "  'NLMid': '8509412',\n",
              "  'Titre': 'Jpn. J. Cancer Res.',\n",
              "  'abstract': 'We studied whether a computer-assisted system using a combination of data collection by image analysis and analysis by neural networks can differentiate benign and malignant breast lesions. Forty-six intraductal lesions of the breast were studied by pathologists and by the computer-assisted system. Histological evaluation was performed independently by three pathologists, and the lesions were classified into pathologically malignant (n = 12), undetermined (n = 13), and benign (n = 21). Computerized nuclear image analysis was performed using the CAS200 (Cell Analysis Systems, Elmhurst, IL) system to obtain data on nuclear morphometric and textural features. A neural network was constructed using the morphometric and texture data obtained from teaching cases of malignant and benign lesions. Then data for unknown cases were classified by the constructed neural network into neural network-malignant (n = 11), -undetermined (n = 5), and -benign (n = 30). The agreement rate between the diagnosis by pathologists and judgment by the computer-assisted system was 75%, excluding pathologically undetermined lesions. There were four false-negative but no false-positive results. False-negative cases had nuclei that were quite different from those of the teaching cases. The agreement rate obtained using either morphometric data or texture data only was lower than that using a combination of both. Selection of appropriate teaching data and incorporation of both morphometric and textural parameters seemed important for obtaining more accurate results. The present data suggest that development of a computer-assisted histopathological diagnosis system for practical use may be possible.',\n",
              "  'authors': ['Fukushima, N',\n",
              "   'Shinbata, H',\n",
              "   'Hasebe, T',\n",
              "   'Yokose, T',\n",
              "   'Sato, A',\n",
              "   'Mukai, K'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Japanese journal of cancer research : Gann',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Neoplasms',\n",
              "   'Carcinoma, Ductal, Breast',\n",
              "   'Cell Nucleus',\n",
              "   'Chromatin',\n",
              "   'False Negative Reactions',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Software'],\n",
              "  'pmid': '9140118',\n",
              "  'title': 'Application of image analysis and neural networks to the pathology diagnosis of intraductal proliferative lesions of the breast.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': \"To develop a neural network based registration quality evaluator (RQE) that can identify unsuccessful 3D/3D image registrations for the head-and-neck patient setup in radiotherapy.|A two-layer feed-forward neural network was used as a RQE to classify 3D/3D rigid registration solutions as successful or unsuccessful based on the features of the similarity surface near the point-of-solution. The supervised training and test data sets were generated by rigidly registering daily cone-beam CTs to the treatment planning fan-beam CTs of six patients with head-and-neck tumors. Two different similarity metrics (mutual information and mean-squared intensity difference) and two different types of image content (entire image versus bony landmarks) were used. The best solution for each registration pair was selected from 50 optimizing attempts that differed only by the initial transformation parameters. The distance from each individual solution to the best solution in the normalized parametrical space was compared to a user-defined error threshold to determine whether that solution was successful or not. The supervised training was then used to train the RQE. The performance of the RQE was evaluated using the test data set that consisted of registration results that were not used in training.|The RQE constructed using the mutual information had very good performance when tested using the test data sets, yielding the sensitivity, the specificity, the positive predictive value, and the negative predictive value in the ranges of 0.960-1.000, 0.993-1.000, 0.983-1.000, and 0.909-1.000, respectively. Adding a RQE into a conventional 3D/3D image registration system incurs only about 10%-20% increase of the overall processing time.|The authors' patient study has demonstrated very good performance of the proposed RQE when used with the mutual information in identifying unsuccessful 3D/3D registrations for daily patient setup. The classifier had very good generality and required only to be trained once for each implementation. When the RQE is incorporated with an automated 3D/3D image registration system, it can improve the robustness of the system.\",\n",
              "  'authors': ['Wu, Jian', 'Murphy, Martin J'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Automation',\n",
              "   'Computer Simulation',\n",
              "   'Diagnostic Imaging',\n",
              "   'Head and Neck Neoplasms',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Scattering, Radiation',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '21158287',\n",
              "  'title': 'A neural network based 3D/3D image registration quality evaluator for the head-and-neck patient setup in the absence of a ground truth.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YO'],\n",
              "  'ESSN': '1099-1263',\n",
              "  'IF_2013': '3.174',\n",
              "  'IF_2014': '2.982',\n",
              "  'IF_2015': '2.722',\n",
              "  'IF_2016': '3.159',\n",
              "  'IF_2017': '2.909',\n",
              "  'ISSN': '0260-437X',\n",
              "  'ISSN_online': '1099-1263',\n",
              "  'ISSN_print': '0260-437X',\n",
              "  'IsoAbbr': 'J Appl Toxicol',\n",
              "  'JrId': 4452,\n",
              "  'MedAbbr': 'J Appl Toxicol',\n",
              "  'NLMid': '8109495',\n",
              "  'Titre': 'J Appl Toxicol',\n",
              "  'abstract': 'Daphnia magna has been used extensively to evaluate organism- and population-level responses to pollutants in acute toxicity and reproductive toxicity tests. We have previously reported that exposure to juvenile hormone (JH) agonists results in a reduction of reproductive function and production of male offspring in a cyclic parthenogenesis, D. magna. Recent advances in molecular techniques have provided tools to understand better the responses to pollutants in aquatic organisms, including D. magna. DNA microarray was used to evaluate gene expression profiles of neonatal daphnids exposed to JH agonists: methoprene (125, 250 and 500 ppb), fenoxycarb (0.5, 1 and 2 ppb) and epofenonane (50, 100 and 200 ppb). Exposure to these JH analogs resulted in chemical-specific patterns of gene expression. The heat map analyses based on hierarchical clustering revealed a similar pattern between treatments with a high dose of methoprene and with epofenonane. In contrast, treatment with low to middle doses of methoprene resulted in similar profiles to fenoxycarb treatments. Hemoglobin and JH epoxide hydrolase genes were clustered as JH-responsive genes. These data suggest that fenoxycarb has high activity as a JH agonist, methoprene shows high toxicity and epofenonane works through a different mechanism compared with other JH analogs, agreeing with data of previously reported toxicity tests. In conclusion, D. magna DNA microarray is useful for the classification of JH analogs and identification of JH-responsive genes.',\n",
              "  'authors': ['Toyota, Kenji',\n",
              "   'Kato, Yasuhiko',\n",
              "   'Miyakawa, Hitoshi',\n",
              "   'Yatsu, Ryohei',\n",
              "   'Mizutani, Takeshi',\n",
              "   'Ogino, Yukiko',\n",
              "   'Miyagawa, Shinichi',\n",
              "   'Watanabe, Hajime',\n",
              "   'Nishide, Hiroyo',\n",
              "   'Uchiyama, Ikuo',\n",
              "   'Tatarazako, Norihisa',\n",
              "   'Iguchi, Taisen'],\n",
              "  'categories': ['TOXICOLOGY'],\n",
              "  'journal': 'Journal of applied toxicology : JAT',\n",
              "  'keywords': ['Animals',\n",
              "   'Animals, Newborn',\n",
              "   'Daphnia',\n",
              "   'Dose-Response Relationship, Drug',\n",
              "   'Down-Regulation',\n",
              "   'Female',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Ontology',\n",
              "   'Juvenile Hormones',\n",
              "   'Male',\n",
              "   'Methoprene',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Phenylcarbamates',\n",
              "   'Reproduction',\n",
              "   'Terpenes',\n",
              "   'Toxicity Tests, Acute',\n",
              "   'Toxicity Tests, Chronic',\n",
              "   'Transcriptome',\n",
              "   'Up-Regulation'],\n",
              "  'pmid': '24038158',\n",
              "  'title': 'Molecular impact of juvenile hormone agonists on neonatal Daphnia magna.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1471-2970',\n",
              "  'IF_2013': '6.314',\n",
              "  'IF_2014': '7.055',\n",
              "  'IF_2015': '5.847',\n",
              "  'IF_2016': '5.846',\n",
              "  'IF_2017': '5.666',\n",
              "  'ISSN': '0962-8436',\n",
              "  'ISSN_online': '1471-2970',\n",
              "  'ISSN_print': '0962-8436',\n",
              "  'IsoAbbr': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'JrId': 8277,\n",
              "  'MedAbbr': 'Philos Trans R Soc Lond B Biol Sci',\n",
              "  'NLMid': '7503623',\n",
              "  'Titre': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'abstract': 'To form an accurate internal representation of visual space, the brain must accurately account for movements of the eyes, head or body. Updating of internal representations in response to these movements is especially important when remembering spatial information, such as the location of an object, since the brain must rely on non-visual extra-retinal signals to compensate for self-generated movements. We investigated the computations underlying spatial updating by constructing a recurrent neural network model to store and update a spatial location based on a gaze shift signal, and to do so flexibly based on a contextual cue. We observed a striking similarity between the patterns of behaviour produced by the model and monkeys trained to perform the same task, as well as between the hidden units of the model and neurons in the lateral intraparietal area (LIP). In this report, we describe the similarities between the model and single unit physiology to illustrate the usefulness of neural networks as a tool for understanding specific computations performed by the brain.',\n",
              "  'authors': ['White, Robert L', 'Snyder, Lawrence H'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Philosophical transactions of the Royal Society of London. Series B, Biological sciences',\n",
              "  'keywords': ['Brain',\n",
              "   'Computer Simulation',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Retina'],\n",
              "  'pmid': '17255021',\n",
              "  'title': 'Spatial constancy and the brain: insights from neural networks.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1520-8532',\n",
              "  'IF_2013': '1.448',\n",
              "  'IF_2014': '1.558',\n",
              "  'IF_2015': '1.457',\n",
              "  'IF_2016': '1.621',\n",
              "  'IF_2017': '1.566',\n",
              "  'ISSN': '1084-7529',\n",
              "  'ISSN_online': '1520-8532',\n",
              "  'ISSN_print': '1084-7529',\n",
              "  'IsoAbbr': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'JrId': 20813,\n",
              "  'MedAbbr': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'NLMid': '9800943',\n",
              "  'Titre': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'abstract': 'Little is known about how color signals and cone- and rod-based luminance signals contribute to perceived contrast in the mesopic range. In this study the perceived contrast of colored, mesopic stimuli was matched with that of spatially equivalent achromatic stimuli. The objective was to develop a metric for perceived contrast in the mesopic range in terms of an equivalent achromatic luminance contrast, referred to here as effective contrast. Stimulus photopic luminance contrast, scotopic luminance contrast, and chromatic difference from the background all contributed to effective contrast over the mid-mesopic range, but their contributions were not independent and varied markedly with background luminance. Surprisingly, color made a significant contribution to effective contrast from 10 to approximately 0.003 cd m(-2). A model describing this relationship is introduced (R2 = 0.89) and compared with predictions of mesopic luminance contrast obtained from a number of models proposed as systems of mesopic photometry.',\n",
              "  'authors': ['Walkey, Helen C',\n",
              "   'Barbur, John L',\n",
              "   'Harlow, J Alister',\n",
              "   'Hurden, Antony',\n",
              "   'Moorhead, Ian R',\n",
              "   'Taylor, Julie A F'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Journal of the Optical Society of America. A, Optics, image science, and vision',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Cluster Analysis',\n",
              "   'Color Perception',\n",
              "   'Colorimetry',\n",
              "   'Computer Simulation',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Male',\n",
              "   'Models, Biological',\n",
              "   'Models, Statistical',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reference Standards',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Task Performance and Analysis'],\n",
              "  'pmid': '15669611',\n",
              "  'title': 'Effective contrast of colored stimuli in the mesopic range: a metric for perceived contrast based on achromatic luminance contrast.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1573-2509',\n",
              "  'IF_2013': '4.426',\n",
              "  'IF_2014': '3.923',\n",
              "  'IF_2015': '4.453',\n",
              "  'IF_2016': '3.986',\n",
              "  'IF_2017': '3.958',\n",
              "  'ISSN': '0920-9964',\n",
              "  'ISSN_online': '1573-2509',\n",
              "  'ISSN_print': '0920-9964',\n",
              "  'IsoAbbr': 'Schizophr. Res.',\n",
              "  'JrId': 1862,\n",
              "  'MedAbbr': 'Schizophr Res',\n",
              "  'NLMid': '8804207',\n",
              "  'Titre': 'Schizophr. Res.',\n",
              "  'abstract': 'Why some individuals, when presented with unstructured sensory inputs, develop altered perceptions not based in reality, is not well understood. Machine learning approaches can potentially help us understand how the brain normally interprets sensory inputs. Artificial neural networks (ANN) progressively extract higher and higher-level features of sensory input and identify the nature of an object based on a priori information. However, some ANNs which use algorithms such as the \"deep-dreaming\" developed by Google, allow the network to over-emphasize some objects it \"thinks\" it recognizes in those areas, and iteratively enhance such outputs leading to representations that appear farther and farther from \"reality\". We suggest that such \"deep dreaming\" ANNs may model aberrant salience, a mechanism suggested for pathogenesis of psychosis. Such models can generate testable predictions for psychosis.',\n",
              "  'authors': ['Keshavan, Matcheri S', 'Sudarshan, Mukund'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Schizophrenia research',\n",
              "  'keywords': ['Brain',\n",
              "   'Creativity',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Psychotic Disorders',\n",
              "   'Schizophrenia'],\n",
              "  'pmid': '28130003',\n",
              "  'title': 'Deep dreaming, aberrant salience and psychosis: Connecting the dots by artificial neural networks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'A brain-computer interface (BCI) is a communication system that allows to control a computer or any other device thanks to the brain activity. The BCI described in this paper is based on the P300 speller BCI paradigm introduced by Farwell and Donchin . An unsupervised algorithm is proposed to enhance P300 evoked potentials by estimating spatial filters; the raw EEG signals are then projected into the estimated signal subspace. Data recorded on three subjects were used to evaluate the proposed method. The results, which are presented using a Bayesian linear discriminant analysis classifier , show that the proposed method is efficient and accurate.',\n",
              "  'authors': ['Rivet, Bertrand',\n",
              "   'Souloumiac, Antoine',\n",
              "   'Attina, Virginie',\n",
              "   'Gibert, Guillaume'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Electroencephalography',\n",
              "   'Evoked Potentials',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Man-Machine Systems',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '19174332',\n",
              "  'title': 'xDAWN algorithm to enhance evoked potentials: application to brain-computer interface.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1520-6882',\n",
              "  'IF_2013': '5.825',\n",
              "  'IF_2014': '5.636',\n",
              "  'IF_2015': '5.886',\n",
              "  'IF_2016': '6.32',\n",
              "  'IF_2017': '6.042',\n",
              "  'ISSN': '0003-2700',\n",
              "  'ISSN_online': '1520-6882',\n",
              "  'ISSN_print': '0003-2700',\n",
              "  'IsoAbbr': 'Anal. Chem.',\n",
              "  'JrId': 519,\n",
              "  'MedAbbr': 'Anal Chem',\n",
              "  'NLMid': '0370536',\n",
              "  'Titre': 'Anal. Chem.',\n",
              "  'abstract': 'A novel, smart, chemical taste sensor that realistically mimics the behavior of the human gustatory system is described. The taste sensor consists of an array of electrochemical sensors that represent the gustatory receptors on the human tongue, and a two-phase optimized radial basis function network (RBFN) to represent the human brain, which comprehensively analyzes the gustatory stimulation and judges the overall taste. In the illustrated model, eight electrodes were fabricated to determine the eight major taste-causing substances, Na+, K+, Cl-, H+, sucrose, glucose, glutamate, and caffeine. The detected signals were fed to a two-phase RBFN optimized by the implementation of a basis optimization algorithm and weight decay term for appropriate data processing. The first phase of the two-phase RBFN quantifies the amount of taste-causing substances in food samples from the responses of the electrodes. These results are then fed to the second phase, which correlates the amount of substances with the overall taste. The final output is scored on a scale of 1-5 for each of the five basic tastes sensed by the human gustatory system, which are saltiness, sourness, sweetness, bitterness, and umami. The constructed network estimated the intensity of the basic tastes of 30 drink varieties with an average relative error of 7.0% compared to the human scores. The network could also estimate the variance in the human sensory perception. Moreover, the sensor successfully predicted the interactions of tastes such as suppression of bitterness by sweetness and enhancement of umami by saltiness, which are illusions sensed by the human gustatory system. With these abilities, the novel taste sensor can be considered as a quantitative yet humanlike sensor with a great potential for practical applications.',\n",
              "  'authors': ['Ishihara, Saeko',\n",
              "   'Ikeda, Atsushi',\n",
              "   'Citterio, Daniel',\n",
              "   'Maruyama, Kenichi',\n",
              "   'Hagiwara, Masafumi',\n",
              "   'Suzuki, Koji'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytical chemistry',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biosensing Techniques',\n",
              "   'Caffeine',\n",
              "   'Chlorides',\n",
              "   'Electrodes',\n",
              "   'Glucose',\n",
              "   'Glutamic Acid',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Potassium',\n",
              "   'Sodium',\n",
              "   'Sucrose',\n",
              "   'Taste'],\n",
              "  'pmid': '16351137',\n",
              "  'title': 'Smart chemical taste sensor for determination and prediction of taste qualities based on a two-phase optimized radial basis function network.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['TC'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0044-3220',\n",
              "  'ISSN_print': '0044-3220',\n",
              "  'IsoAbbr': 'Z Orthop Ihre Grenzgeb',\n",
              "  'JrId': 8162,\n",
              "  'MedAbbr': 'Z Orthop Ihre Grenzgeb',\n",
              "  'NLMid': '1256465',\n",
              "  'Titre': 'Z Orthop Ihre Grenzgeb',\n",
              "  'abstract': 'Distraction osteogenesis for the correction of deformities with an external fixator is well established. The hexapod principle of robotic technique was invented for the Ilisarov apparatus (e. g. Taylor-Spatial-Frame/TSF). Treatment with conventional frames needs a patient-customised frame mounting. This demanding procedure is markedly reduced using this technology. The aim of this study was to analyse the value of the hexapod principle in external fixation.|The potential of a frame to correct deformities is limited by its work space. The geometry of a conventional frame is different from the geometry of a hexapod frame, which is the reason for their different work spaces. The work space of the hexapod frame is compared to the work space of a conventional frame. Important parameters for this analyses are minimal and maximal frame heights and the potential of correction.|The minimal frame height of hexapod fixators is higher compared to conventional Ilisarov fixators. The standard hexapod frame (TSF 155 mm ring diameter) can correct 23 degrees of angulation, 36 mm of shortening, 71 mm of translation and 43 degrees of rotation without changing the telescope rods. The standard conventional frame (160 mm ring diameter) can correct 90 degrees of angulation, 100 mm of shortening, 25 mm of translation and 12.5 degrees of rotation without remounting of the frame.|The different work spaces of the different frames result in consequences for their clinical application. The hexapod frame has more power to correct translation and rotational deformities than a conventional frame. Correction of extensive angulation and shortening deformities almost always needs an exchange of telescopic rods. Conventional frames are usually able to correct these deformities with the primary mounting. Because of its increased minimal frame height, the indication for hexapod constructs in child orthopaedics can be limited.',\n",
              "  'authors': ['Rödl, R', 'Leidinger, B', 'Böhm, A', 'Winkelmann, W'],\n",
              "  'categories': ['ORTHOPEDICS'],\n",
              "  'journal': 'Zeitschrift fur Orthopadie und ihre Grenzgebiete',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Bone Malalignment',\n",
              "   'Equipment Design',\n",
              "   'External Fixators',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Ilizarov Technique',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Joint Dislocations',\n",
              "   'Joint Instability',\n",
              "   'Knee Injuries',\n",
              "   'Knee Joint',\n",
              "   'Microcomputers',\n",
              "   'Postoperative Care',\n",
              "   'Postoperative Complications',\n",
              "   'Robotics',\n",
              "   'Software',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Tibia',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '12605337',\n",
              "  'title': '[Correction of deformities with conventional and hexapod frames--comparison of methods].',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': \"We model dependencies between m multivariate continuous-valued information sources by a combination of (i) a generalized canonical correlations analysis (gCCA) to reduce dimensionality while preserving dependencies in m - 1 of them, and (ii) summarizing dependencies with the remaining one by associative clustering. This new combination of methods avoids multiway associative clustering which would require a multiway contingency table and hence suffer from curse of dimensionality of the table. The method is applied to summarizing properties of yeast stress by searching for dependencies (commonalities) between expression of genes of baker's yeast Saccharomyces cerevisiae in various stressful treatments, and summarizing stress regulation by finally adding data about transcription factor binding sites.\",\n",
              "  'authors': ['Nikkilä, Janne',\n",
              "   'Roos, Christophe',\n",
              "   'Savia, Eerika',\n",
              "   'Kaski, Samuel'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Animals',\n",
              "   'Cluster Analysis',\n",
              "   'Gene Expression',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Fungal',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oxidative Stress',\n",
              "   'Yeasts'],\n",
              "  'pmid': '16187400',\n",
              "  'title': 'Exploratory modeling of yeast stress response and its regulation with gCCA and associative clustering.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'The sophistication, complexity and intelligence of biological systems is a continuous source of inspiration for mankind. Mimicking the natural intelligence to devise tiny systems that are capable of self-regulated, autonomous action to, for example, distinguish different targets, remains among the grand challenges in biomimetic micro-robotics. Herein, we demonstrate an autonomous soft device, a light-driven flytrap, that uses optical feedback to trigger photomechanical actuation. The design is based on light-responsive liquid-crystal elastomer, fabricated onto the tip of an optical fibre, which acts as a power source and serves as a contactless probe that senses the environment. Mimicking natural flytraps, this artificial flytrap is capable of autonomous closure and object recognition. It enables self-regulated actuation within the fibre-sized architecture, thus opening up avenues towards soft, autonomous small-scale devices.',\n",
              "  'authors': ['Wani, Owies M', 'Zeng, Hao', 'Priimagi, Arri'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': [''],\n",
              "  'pmid': '28534872',\n",
              "  'title': 'A light-driven artificial flytrap.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-2199',\n",
              "  'IF_2013': '8.305',\n",
              "  'IF_2014': '8.665',\n",
              "  'IF_2015': '8.285',\n",
              "  'IF_2016': '6.559',\n",
              "  'IF_2017': '6.308',\n",
              "  'ISSN': '1047-3211',\n",
              "  'ISSN_online': '1460-2199',\n",
              "  'ISSN_print': '1047-3211',\n",
              "  'IsoAbbr': 'Cereb. Cortex',\n",
              "  'JrId': 2171,\n",
              "  'MedAbbr': 'Cereb Cortex',\n",
              "  'NLMid': '9110718',\n",
              "  'Titre': 'Cereb. Cortex',\n",
              "  'abstract': 'Neuronal mechanisms underlying beta/gamma oscillations (20-80 Hz) are not completely understood. Here, we show that in vivo beta/gamma oscillations in the cat visual cortex sometimes exhibit remarkably stable frequency even when inputs fluctuate dramatically. Enhanced frequency stability is associated with stronger oscillations measured in individual units and larger power in the local field potential. Simulations of neuronal circuitry demonstrate that membrane properties of inhibitory interneurons strongly determine the characteristics of emergent oscillations. Exploration of networks containing either integrator or resonator inhibitory interneurons revealed that: (i) Resonance, as opposed to integration, promotes robust oscillations with large power and stable frequency via a mechanism called RING (Resonance INduced Gamma); resonance favors synchronization by reducing phase delays between interneurons and imposes bounds on oscillation cycle duration; (ii) Stability of frequency and robustness of the oscillation also depend on the relative timing of excitatory and inhibitory volleys within the oscillation cycle; (iii) RING can reproduce characteristics of both Pyramidal INterneuron Gamma (PING) and INterneuron Gamma (ING), transcending such classifications; (iv) In RING, robust gamma oscillations are promoted by slow but are impaired by fast inputs. Results suggest that interneuronal membrane resonance can be an important ingredient for generation of robust gamma oscillations having stable frequency. ',\n",
              "  'authors': ['Moca, Vasile V',\n",
              "   'Nikolic, Danko',\n",
              "   'Singer, Wolf',\n",
              "   'Mureşan, Raul C'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Cerebral cortex (New York, N.Y. : 1991)',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Beta Rhythm',\n",
              "   'Cats',\n",
              "   'Computer Simulation',\n",
              "   'Cortical Synchronization',\n",
              "   'Electroencephalography',\n",
              "   'Electrophysiological Phenomena',\n",
              "   'Interneurons',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Photic Stimulation',\n",
              "   'Pyramidal Cells',\n",
              "   'Reproducibility of Results',\n",
              "   'Synapses',\n",
              "   'Visual Cortex'],\n",
              "  'pmid': '23042733',\n",
              "  'title': 'Membrane resonance enables stable and robust gamma oscillations.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Artificial intelligence capabilities have, recently, greatly improved. In the past few years, one of the deep learning algorithms, the recurrent neural network (RNN), has shown an outstanding ability in sequence labeling and prediction tasks for sequential data. We built a reliable visual field prediction algorithm using RNN and evaluated its performance in comparison with the conventional pointwise ordinary linear regression (OLR) method. A total of 1,408 eyes were used as a training dataset and another dataset, comprising 281 eyes, was used as a test dataset. Five consecutive visual field tests were provided to the constructed RNN as input and a 6th visual field test was compared with the output of the RNN. The performance of the RNN was compared with that of OLR by predicting the 6th visual field in the test dataset. The overall prediction performance of RNN was significantly better than OLR. The pointwise prediction error of the RNN was significantly smaller than that of the OLR in most areas known to be vulnerable to glaucomatous damage. The RNN was also more robust and reliable regarding worsening in the visual field examination. In clinical practice, the RNN model can therefore assist in decision-making for further treatment of glaucoma.',\n",
              "  'authors': ['Park, Keunheung', 'Kim, Jinmi', 'Lee, Jiwoong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31182763',\n",
              "  'title': 'Visual Field Prediction using Recurrent Neural Network.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': 'A focus-swing liquid-crystal (LC) microlens with two patterned electrodes and filled in nematic liquid crystal is proposed. In order to lower the level of the applied voltage signal and effectively increase the focus-swing range, the bottom electrode is designed as a circular patterned structure. The top electrode is composed of four stripe-patterned subelectrodes, which are powered, respectively to generate expecting potential and drive the focus swing in the focal plane of the microlens. The common optical properties of the LC microlens and the swing behavior of the formed focus in the focal plane are demonstrated experimentally.',\n",
              "  'authors': ['Kang, Shengwu',\n",
              "   'Zhang, Xinyu',\n",
              "   'Xie, Changsheng',\n",
              "   'Zhang, Tianxu'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '23338183',\n",
              "  'title': 'Liquid-crystal microlens with focus swing and low driving voltage.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1094-4087',\n",
              "  'IF_2013': '3.525',\n",
              "  'IF_2014': '3.488',\n",
              "  'IF_2015': '3.148',\n",
              "  'IF_2016': '3.307',\n",
              "  'IF_2017': '3.356',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1094-4087',\n",
              "  'IsoAbbr': 'Opt Express',\n",
              "  'JrId': 29989,\n",
              "  'MedAbbr': 'Opt Express',\n",
              "  'NLMid': '101137103',\n",
              "  'Titre': 'Opt Express',\n",
              "  'abstract': 'Fourier ptychographic microscopy (FPM) is a newly developed microscopic technique for large field of view, high-resolution and quantitative phase imaging by combining the techniques from ptychographic imaging, aperture synthesizing and phase retrieval. In FPM, an LED array is utilized to illuminate the specimen from different angles and the corresponding intensity images are synthesized to reconstruct a high-resolution complex field. As a flexible and low-cost approach to achieve high-resolution, wide-field and quantitative phase imaging, FPM is of enormous potential in biomedical applications such as hematology and pathology. Conventionally, the FPM reconstruction problem is solved by using a phase retrieval method, termed Alternate Projection. By iteratively updating the Fourier spectrum with low-resolution-intensity images, the result converges to a high-resolution complex field. Here we propose a new FPM reconstruction framework with deep learning methods and design a multiscale, deep residual neural network for FPM reconstruction. We employ the widely used open-source deep learning library PyTorch to train and test our model and carefully choose the hyperparameters of our model. To train and analyze our model, we build a large-scale simulation dataset with an FPM imaging model and an actual dataset captured with an FPM system. The simulation dataset and actual dataset are separated as training datasets and test datasets, respectively. Our model is trained with the simulation training dataset and fine tuned with the fine-tune dataset, which contains actual training data. Both our model and the conventional method are tested on the simulation test dataset and the actual test dataset to evaluate the performances. We also show the reconstruction result of another neural network-based method for comparison. The experiments demonstrate that our model achieves better reconstruction results and consumes much less time than conventional methods. The results also point out that our model is more robust under system aberrations such as noise and blurring (fewer intensity images) compared with conventional methods.',\n",
              "  'authors': ['Zhang, Jizhou',\n",
              "   'Xu, Tingfa',\n",
              "   'Shen, Ziyi',\n",
              "   'Qiao, Yifan',\n",
              "   'Zhang, Yizhou'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics express',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31052676',\n",
              "  'title': 'Fourier ptychographic microscopy reconstruction with multiscale deep residual network.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'The da Vinci robot laparoscopic incisional hernia repair with intracorporeal suturing may offer an alternative to transabdominal sutures and tackers.|From 2003 to 2005, 11 patients (median age, 71 years; median body mass index [BMI], 28) with small and medium-sized incisional hernias (median fascial defect, 19.6 cm2) were treated with the da Vinci robot system using intracorporeal mesh fixation with interrupted sutures. This pilot study aimed to assess the feasibility and report the morbidity with special reference to postoperative pain and long-term recurrence.|The median operative time was 180 min. There was no conversion to open or standard laparoscopy and no postoperative mortality. The overall morbidity rate was 27%. One patient underwent reoperation on postoperative day 3 for peritonitis secondary to small bowel injury. The median visual analog pain score on postoperative day 1 was 3. Seven patients (63%) needed parenteral paracetamol until postoperative day 2. The median hospital stay was 3 days. During a median follow-up period of 25 months, no patient experienced recurrent hernia. One patient had a trocar-site herniation at 6 months. No patient experienced chronic suture site pain or discomfort.|This is the first report of robot-assisted laparoscopic incisional hernia with exclusive intracorporeal suturing for mesh fixation in humans. The findings show that this technique is feasible and may not be associated with chronic postoperative pain. Further evaluation is needed to assess the benefit to the patient, but this investigation may be the basis for a future, prospective, randomized study.',\n",
              "  'authors': ['Tayar, C', 'Karoui, M', 'Cherqui, D', 'Fagniez, P L'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Hernia, Ventral',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pilot Projects',\n",
              "   'Robotics',\n",
              "   'Surgical Mesh',\n",
              "   'Suture Techniques'],\n",
              "  'pmid': '17353984',\n",
              "  'title': 'Robot-assisted laparoscopic mesh repair of incisional hernias with exclusive intracorporeal suturing: a pilot study.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.58',\n",
              "  'IF_2014': '0.488',\n",
              "  'IF_2015': '0.918',\n",
              "  'IF_2016': '0.454',\n",
              "  'IF_2017': '0.398',\n",
              "  'ISSN': '0884-6812',\n",
              "  'IsoAbbr': 'Anal. Quant. Cytol. Histol.',\n",
              "  'JrId': 1348,\n",
              "  'MedAbbr': 'Anal Quant Cytol Histol',\n",
              "  'NLMid': '8506819',\n",
              "  'Titre': 'Anal. Quant. Cytol. Histol.',\n",
              "  'abstract': 'Two aspects of expert systems for use in diagnostic histopathology and cytopathology are examined: knowledge representation and the structure and operation of rule-based systems. Knowledge may be represented, e.g., in semantic networks, frames, multiple contexts and model-based structures; the choice of structure should be matched to the type of information to create an efficient and logically adequate expert system. In a rule-based system, knowledge is represented as \"rules,\" often in the form of \"IF (condition)-THEN (conclusion)\" rules. The anatomy of such rules and their operation is explored via the use of examples. Uncertainty in rules is briefly addressed, and their processing by the symbolic reasoning of the \"inference engine\" of the expert system is described, including both \"forward-chaining\" (\"data-driven\") operations and \"backward-chaining\" (\"goal-driven\") operations.',\n",
              "  'authors': ['Bartels, P H', 'Hiessl, H'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Analytical and quantitative cytology and histology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Expert Systems',\n",
              "   'Histology',\n",
              "   'Pathology'],\n",
              "  'pmid': '2663006',\n",
              "  'title': 'Expert systems in histopathology. II. Knowledge representation and rule-based systems.',\n",
              "  'year': '1989'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1873-7560',\n",
              "  'IF_2013': '12.48',\n",
              "  'IF_2014': '13.938',\n",
              "  'IF_2015': '14.976',\n",
              "  'IF_2016': '16.265',\n",
              "  'IF_2017': '17.581',\n",
              "  'ISSN': '0302-2838',\n",
              "  'ISSN_online': '1873-7560',\n",
              "  'ISSN_print': '0302-2838',\n",
              "  'IsoAbbr': 'Eur. Urol.',\n",
              "  'JrId': 3630,\n",
              "  'MedAbbr': 'Eur Urol',\n",
              "  'NLMid': '7512719',\n",
              "  'Titre': 'Eur. Urol.',\n",
              "  'abstract': 'An artificial neural network analysis (ANNA) was developed to predict the biochemical recurrence more effectively than regression models based on the combined use of pelvic coil magnetic resonance imaging (pMRI), prostate specific antigen (PSA) and biopsy Gleason score in patients with clinically organ-confined prostate cancer after radical prostatectomy (RP).|Two-hundred-and-ten patients undergoing retropubic RP with pelvic lymphadenectomy were evaluated. Predictive study variables included clinical TNM classification, preoperative serum PSA, biopsy Gleason score, transrectal ultrasound (TRUS) findings, and pMRI findings. The predicted result was a biochemical failure (PSA >or=0.1 ng/ml). Using a five-way cross-validation method, the predicted ability of ANNA for a validation set of 200 randomly selected patients was compared with those of Cox regression analysis and \"Kattan nomogram\" by area under the receiver operating characteristic curve (AUC) analysis.|Seventy-three patients (35%) failed at median follow-up of 61 (mean: 60, range: 2-94) months. Using similar input variables, the AUC of ANNA (0.765, 95% Confidence Interval [CI]: 0.704-0.825) was comparable (p > 0.05) to those for Cox regression (0.738, 95%CI: 0.691-0.819) and Kattan nomogram (0.728, 95%CI: 0.644-0.819). Contrarily, adding the pMRI findings, the ANNA is significantly (p < 0.05) superior to any other predictive model (0.897, 95%CI: 0.841-0.977). The Gleason score represented the most influential predictor (relative weight: 2.4) of PSA recurrence, followed by pMRI (2.2), and PSA (2.0).|ANNA is superior to regression models to predict accurately biochemical recurrence. The relative importance of pMRI and the utility of ANNA to predict the PSA failure in patients referred for RP must be confirmed in further trials.',\n",
              "  'authors': ['Poulakis, Vassilis',\n",
              "   'Witzsch, Ulrich',\n",
              "   'de Vries, Rachelle',\n",
              "   'Emmerlich, Volker',\n",
              "   'Meves, Michael',\n",
              "   'Altmannsberger, Hans-Michael',\n",
              "   'Becht, Eduard'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'European urology',\n",
              "  'keywords': ['Biomarkers, Tumor',\n",
              "   'Biopsy',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Humans',\n",
              "   'Lymph Node Excision',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Prognosis',\n",
              "   'Proportional Hazards Models',\n",
              "   'Prostate',\n",
              "   'Prostate-Specific Antigen',\n",
              "   'Prostatic Neoplasms'],\n",
              "  'pmid': '15474265',\n",
              "  'title': 'Preoperative neural network using combined magnetic resonance imaging variables, prostate specific antigen, and Gleason score to predict prostate cancer recurrence after radical prostatectomy.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1708-0428',\n",
              "  'IF_2013': '3.739',\n",
              "  'IF_2014': '3.747',\n",
              "  'IF_2015': '3.346',\n",
              "  'IF_2016': '3.947',\n",
              "  'IF_2017': '3.895',\n",
              "  'ISSN': '0960-8923',\n",
              "  'ISSN_online': '1708-0428',\n",
              "  'ISSN_print': '0960-8923',\n",
              "  'IsoAbbr': 'Obes Surg',\n",
              "  'JrId': 20877,\n",
              "  'MedAbbr': 'Obes Surg',\n",
              "  'NLMid': '9106714',\n",
              "  'Titre': 'Obes Surg',\n",
              "  'abstract': \"Fellowships in advanced laparoscopy with emphasis in laparoscopic gastric bypass (LGBP) are available for obtaining experience in performing LGBP. The following is the first report in the literature prospectively documenting a single surgeon's experience with LGBP outcomes following completion of an advanced laparoscopic surgical fellowship.|Outcomes measured prospectively included length of stay, length of operation, complications, reduction in obesity-related co-morbidities, and percentage excess weight loss. Outcomes were analyzed by quartile to see if there was difference over time. Complications were also compared to outcomes in the literature.|175 patients (147 female, 28 male) underwent LGBP. The mean BMI was 49.2. Mean operative time was 123 minutes, and mean length of stay was 2.2 days. The percentage excess weight loss at 1 year was 73% (n = 79). One patient developed an internal hernia (0.6%) and 1 patient developed an anastomotic leak (0.6%). Post operative transfusion rate was 4.6%. There were no deep venous thromboses or pulmonary emboli detected. There were no conversions to open, and there was no mortality. Upon quartile analysis, there was no difference in complication rates. Complication rates were comparable to published outcomes in the literature.|Fellowships in advanced laparoscopy with emphasis on LGBP provide the optimal training environment for acquisition of skills necessary to safely and effectively perform LGBP. With fellowship training, complication rates were comparable to published outcomes in the literature without a period of higher complications (the learning curve).\",\n",
              "  'authors': ['Kothari, Shanu N',\n",
              "   'Boyd, William C',\n",
              "   'Larson, Christopher A',\n",
              "   'Gustafson, Heather L',\n",
              "   'Lambert, Pamela J',\n",
              "   'Mathiason, Michelle A'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Obesity surgery',\n",
              "  'keywords': ['Adult',\n",
              "   'Anastomosis, Roux-en-Y',\n",
              "   'Bariatrics',\n",
              "   'Fellowships and Scholarships',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Gastric Bypass',\n",
              "   'General Surgery',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Obesity',\n",
              "   'Postoperative Complications',\n",
              "   'Prospective Studies',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome',\n",
              "   'Weight Loss'],\n",
              "  'pmid': '15832432',\n",
              "  'title': 'Training of a minimally invasive bariatric surgeon: are laparoscopic fellowships the answer?',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1872-6240',\n",
              "  'IF_2013': '2.828',\n",
              "  'IF_2014': '2.843',\n",
              "  'IF_2015': '2.561',\n",
              "  'IF_2016': '2.746',\n",
              "  'IF_2017': '3.125',\n",
              "  'ISSN': '0006-8993',\n",
              "  'ISSN_online': '1872-6240',\n",
              "  'ISSN_print': '0006-8993',\n",
              "  'IsoAbbr': 'Brain Res.',\n",
              "  'JrId': 1920,\n",
              "  'MedAbbr': 'Brain Res',\n",
              "  'NLMid': '0045503',\n",
              "  'Titre': 'Brain Res.',\n",
              "  'abstract': 'Layer 1 of the dorsal cortex of the inferior colliculus (DCIC) is distinguished from other layers by its cytoarchitecture and fiber connections. However, the information of the sound types represented in layer 1 of the DCIC remains unclear because placing electrodes on such thin structures is challenging. In this study, we utilized in vivo calcium imaging to assess auditory-evoked activities in multiple cells in layer 1 of DCIC and to characterize sound stimuli producing strong activity. Most cells examined showed strong responses to broad-band noise and low-frequency tone bursts of high sound intensity. In some cases, we successfully obtained frequency response areas, which are receptive fields to tone frequencies and intensities, and ~30% of these showed V-shape tunings. This is the first systematic study to record auditory responses of cells in layer 1 of DCIC. These results indicate that cells in this area are selective to tones with low frequency, implying the importance of such auditory information in the neural circuitry of layer 1 of DCIC. ',\n",
              "  'authors': ['Ito, Tetsufumi',\n",
              "   'Hirose, Junichi',\n",
              "   'Murase, Kazuyuki',\n",
              "   'Ikeda, Hiroshi'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Brain research',\n",
              "  'keywords': ['Acoustic Stimulation',\n",
              "   'Animals',\n",
              "   'Calcium',\n",
              "   'Cell Count',\n",
              "   'Electrodes, Implanted',\n",
              "   'Evoked Potentials, Auditory',\n",
              "   'Female',\n",
              "   'Inferior Colliculi',\n",
              "   'Mice',\n",
              "   'Mice, Inbred BALB C',\n",
              "   'Microscopy, Confocal'],\n",
              "  'pmid': '25278189',\n",
              "  'title': 'Determining auditory-evoked activities from multiple cells in layer 1 of the dorsal cortex of the inferior colliculus of mice by in vivo calcium imaging.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1097-0290',\n",
              "  'IF_2013': '4.164',\n",
              "  'IF_2014': '4.126',\n",
              "  'IF_2015': '4.243',\n",
              "  'IF_2016': '4.481',\n",
              "  'IF_2017': '3.952',\n",
              "  'ISSN': '0006-3592',\n",
              "  'ISSN_online': '1097-0290',\n",
              "  'ISSN_print': '0006-3592',\n",
              "  'IsoAbbr': 'Biotechnol. Bioeng.',\n",
              "  'JrId': 1200,\n",
              "  'MedAbbr': 'Biotechnol Bioeng',\n",
              "  'NLMid': '7502021',\n",
              "  'Titre': 'Biotechnol. Bioeng.',\n",
              "  'abstract': 'A fully automated anthrax smoke detector (ASD) has been developed and tested. The ASD is intended to serve as a cost effective front-end monitor for anthrax surveillance systems. The principle of operation is based on measuring airborne endospore concentrations, where a sharp concentration increase signals an anthrax attack. The ASD features an air sampler, a thermal lysis unit, a syringe pump, a time-gated spectrometer, and endospore detection chemistry comprised of dipicolinic acid (DPA)-triggered terbium ion (Tb(3+)) luminescence. Anthrax attacks were simulated using aerosolized Bacillus atrophaeus spores in fumed silica, and corresponding Tb-DPA intensities were monitored as a function of time and correlated to the number of airborne endospores collected. A concentration dependence of 10(2)-10(6) spores/mg of fumed silica yielded a dynamic range of 4 orders of magnitude and a limit of detection of 16 spores/L when 250 L of air were sampled. Simulated attacks were detected in less than 15 min.',\n",
              "  'authors': ['Yung, Pun To',\n",
              "   'Lester, Elizabeth D',\n",
              "   'Bearman, Greg',\n",
              "   'Ponce, Adrian'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Biotechnology and bioengineering',\n",
              "  'keywords': ['Air Pollutants',\n",
              "   'Bacillus anthracis',\n",
              "   'Colony Count, Microbial',\n",
              "   'Environmental Monitoring',\n",
              "   'Equipment Design',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Spores, Bacterial'],\n",
              "  'pmid': '17514759',\n",
              "  'title': 'An automated front-end monitor for anthrax surveillance systems based on the rapid detection of airborne endospores.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1477-9145',\n",
              "  'IF_2013': '3.002',\n",
              "  'IF_2014': '2.897',\n",
              "  'IF_2015': '2.914',\n",
              "  'IF_2016': '3.32',\n",
              "  'IF_2017': '3.179',\n",
              "  'ISSN': '0022-0949',\n",
              "  'ISSN_online': '1477-9145',\n",
              "  'ISSN_print': '0022-0949',\n",
              "  'IsoAbbr': 'J. Exp. Biol.',\n",
              "  'JrId': 4720,\n",
              "  'MedAbbr': 'J Exp Biol',\n",
              "  'NLMid': '0243705',\n",
              "  'Titre': 'J. Exp. Biol.',\n",
              "  'abstract': \"For early vertebrates, a long-standing hypothesis is that vertebrae evolved as a locomotor adaptation, stiffening the body axis and enhancing swimming performance. While supported by biomechanical data, this hypothesis has not been tested using an evolutionary approach. We did so by extending biomimetic evolutionary analysis (BEA), which builds physical simulations of extinct systems, to include use of autonomous robots as proxies of early vertebrates competing in a forage navigation task. Modeled after free-swimming larvae of sea squirts (Chordata, Urochordata), three robotic tadpoles (;Tadros'), each with a propulsive tail bearing a biomimetic notochord of variable spring stiffness, k (N m(-1)), searched for, oriented to, and orbited in two dimensions around a light source. Within each of ten generations, we selected for increased swimming speed, U (m s(-1)) and decreased time to the light source, t (s), average distance from the source, R (m) and wobble maneuvering, W (rad s(-2)). In software simulation, we coded two quantitative trait loci (QTL) that determine k: bending modulus, E (Nm(-2)) and length, L (m). Both QTL were mutated during replication, independently assorted during meiosis and, as haploid gametes, entered into the gene pool in proportion to parental fitness. After random mating created three new diploid genotypes, we fabricated three new offspring tails. In the presence of both selection and chance events (mutation, genetic drift), the phenotypic means of this small population evolved. The classic hypothesis was supported in that k was positively correlated (r(2)=0.40) with navigational prowess, NP, the dimensionless ratio of U to the product of R, t and W. However, the plausible adaptive scenario, even in this simplified system, is more complex, since the remaining variance in NP was correlated with the residuals of R and U taken with respect to k, suggesting that changes in k alone are insufficient to explain the evolution of NP.\",\n",
              "  'authors': ['Long, J H',\n",
              "   'Koob, T J',\n",
              "   'Irving, K',\n",
              "   'Combie, K',\n",
              "   'Engel, V',\n",
              "   'Livingston, N',\n",
              "   'Lammert, A',\n",
              "   'Schumacher, J'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'The Journal of experimental biology',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Robotics',\n",
              "   'Swimming',\n",
              "   'Vertebrates'],\n",
              "  'pmid': '17114406',\n",
              "  'title': 'Biomimetic evolutionary analysis: testing the adaptive value of vertebrate tail stiffness in autonomous swimming robots.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1365-3040',\n",
              "  'IF_2013': '5.906',\n",
              "  'IF_2014': '6.96',\n",
              "  'IF_2015': '6.169',\n",
              "  'IF_2016': '6.173',\n",
              "  'IF_2017': '5.415',\n",
              "  'ISSN': '0140-7791',\n",
              "  'ISSN_online': '1365-3040',\n",
              "  'ISSN_print': '0140-7791',\n",
              "  'IsoAbbr': 'Plant Cell Environ.',\n",
              "  'JrId': 20442,\n",
              "  'MedAbbr': 'Plant Cell Environ',\n",
              "  'NLMid': '9309004',\n",
              "  'Titre': 'Plant Cell Environ.',\n",
              "  'abstract': 'Plants have evolved a series of tolerance mechanisms to saline stress, which perturbs physiological processes throughout the plant. To identify genetic mechanisms associated with salinity tolerance, we performed linkage analysis and genome-wide association study (GWAS) on maintenance of root growth of Arabidopsis thaliana in hydroponic culture with weak and severe NaCl toxicity. The top 200 single-nucleotide polymorphisms (SNPs) determined by GWAS could cumulatively explain approximately 70% of the variation observed at each stress level. The most significant SNPs were linked to the genes of ATP-binding cassette B10 and vacuolar proton ATPase A2. Several known salinity tolerance genes such as potassium channel KAT1 and calcium sensor SOS3 were also linked to SNPs in the top 200. In parallel, we constructed a gene co-expression network to independently verify that particular groups of genes work together to a common purpose. We identify molecular mechanisms to confer salt tolerance from both predictable and novel physiological sources and validate the utility of combined genetic and network analysis. Additionally, our study indicates that the genetic architecture of salt tolerance is responsive to the severity of stress. These gene datasets are a significant information resource for a following exploration of gene function. ',\n",
              "  'authors': ['Kobayashi, Yuriko',\n",
              "   'Sadhukhan, Ayan',\n",
              "   'Tazib, Tanveer',\n",
              "   'Nakano, Yuki',\n",
              "   'Kusunoki, Kazutaka',\n",
              "   'Kamara, Mohamed',\n",
              "   'Chaffai, Radhouane',\n",
              "   'Iuchi, Satoshi',\n",
              "   'Sahoo, Lingaraj',\n",
              "   'Kobayashi, Masatomo',\n",
              "   'Hoekenga, Owen A',\n",
              "   'Koyama, Hiroyuki'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Plant, cell & environment',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Genes, Plant',\n",
              "   'Genetic Linkage',\n",
              "   'Genetic Loci',\n",
              "   'Genetic Variation',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Inbreeding',\n",
              "   'Plant Roots',\n",
              "   'Plants, Genetically Modified',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Quantitative Trait Loci',\n",
              "   'Recombination, Genetic',\n",
              "   'Sodium Chloride',\n",
              "   'Stress, Physiological',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '26667381',\n",
              "  'title': 'Joint genetic and network analyses identify loci associated with root growth under NaCl stress in Arabidopsis thaliana.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'The cWINNOWER algorithm detects fuzzy motifs in DNA sequences rich in protein-binding signals. A signal is defined as any short nucleotide pattern having up to d mutations differing from a motif of length l. The algorithm finds such motifs if a clique consisting of a sufficiently large number of mutated copies of the motif (i.e., the signals) is present in the DNA sequence. The cWINNOWER algorithm substantially improves the sensitivity of the winnower method of Pevzner and Sze by imposing a consensus constraint, enabling it to detect much weaker signals. We studied the minimum detectable clique size qc as a function of sequence length N for random sequences. We found that qc increases linearly with N for a fast version of the algorithm based on counting three-member sub-cliques. Imposing consensus constraints reduces qc by a factor of three in this case, which makes the algorithm dramatically more sensitive. Our most sensitive algorithm, which counts four-member sub-cliques, needs a minimum of only 13 signals to detect motifs in a sequence of length N = 12,000 for (l, d) = (15, 4).',\n",
              "  'authors': ['Liang, S', 'Samanta, M P', 'Biegel, B A'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Motifs',\n",
              "   'Base Sequence',\n",
              "   'Binding Sites',\n",
              "   'DNA',\n",
              "   'Fuzzy Logic',\n",
              "   'Gene Expression Profiling',\n",
              "   'Molecular Sequence Data',\n",
              "   'Protein Binding',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sequence Alignment',\n",
              "   'Sequence Analysis, DNA',\n",
              "   'Sequence Homology, Nucleic Acid',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '15272432',\n",
              "  'title': 'cWINNOWER algorithm for finding fuzzy dna motifs.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'To assess the construct validity of the da Vinci Skills Simulator (Intuitive Surgical, Sunnyvale, CA). Ideally, a well-designed simulator should demonstrate construct validity, which is defined in this study as the correlation between robotic surgical experience and performance on the simulator.|Thirty-nine surgeons (18 [46%] group I [0-20 robotic cases]; 8 [21%] group II [21-150 robotic cases]; and 13 [33%] group III [>150 robotic cases]) were enrolled from September 2010 to December 2010. Participants completed 24 virtual-reality exercises on the da Vinci Skills Simulator. Data on 12 performance metrics were collected by the software. Overall means for score and time across exercises were analyzed.|Overall scores (64.7%/79.1%/87.4%) and time scores (39.1%/58.6%/87.3%) were significantly different among surgeons in groups I-III (P <.001) and demonstrated significant linear relationships (P <.001) for all 24 exercises. Comparisons between the 3 groups using a univariate general linear model (GLM) was used to compare groups I and II and II and III. Groups I and II differed using overall score for 15 exercises and time score for 11 exercises. Groups II and III differed using overall score for 6 exercises and time score for 15 exercises. Mean overall score for 1 exercise displayed significance between both groups I and II and II and III; while using time score, 5 exercises displayed significance between surgeons in groups I and II and II and III.|Initial construct validity analysis revealed that both overall scores and time scores showed a significant linear relationship when comparing the surgeons in groups I, II, and III. Overall score seems to be a stronger indicator for differences between surgeons in groups I and II. Time score seems to be a stronger indicator for differences between surgeons in groups II and III.',\n",
              "  'authors': ['Finnegan, Kyle T',\n",
              "   'Meraney, Anoop M',\n",
              "   'Staff, Ilene',\n",
              "   'Shichman, Steven J'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Computer Simulation',\n",
              "   'Laparoscopy',\n",
              "   'Prospective Studies',\n",
              "   'Robotics',\n",
              "   'Time Factors'],\n",
              "  'pmid': '22704177',\n",
              "  'title': 'da Vinci Skills Simulator construct validation study: correlation of prior robotic experience with overall score and time score simulator performance.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'In the last two decades, Computer Aided Detection (CAD) systems were developed to help radiologists analyse screening mammograms, however benefits of current CAD technologies appear to be contradictory, therefore they should be improved to be ultimately considered useful. Since 2012, deep convolutional neural networks (CNN) have been a tremendous success in image recognition, reaching human performance. These methods have greatly surpassed the traditional approaches, which are similar to currently used CAD solutions. Deep CNN-s have the potential to revolutionize medical image analysis. We propose a CAD system based on one of the most successful object detection frameworks, Faster R-CNN. The system detects and classifies malignant or benign lesions on a mammogram without any human intervention. The proposed method sets the state of the art classification performance on the public INbreast database, AUC = 0.95. The approach described here has achieved 2nd place in the Digital Mammography DREAM Challenge with AUC = 0.85. When used as a detector, the system reaches high sensitivity with very few false positive marks per image on the INbreast dataset. Source code, the trained model and an OsiriX plugin are published online at https://github.com/riblidezso/frcnn_cad .',\n",
              "  'authors': ['Ribli, Dezső',\n",
              "   'Horváth, Anna',\n",
              "   'Unger, Zsuzsa',\n",
              "   'Pollner, Péter',\n",
              "   'Csabai, István'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '29545529',\n",
              "  'title': 'Detecting and classifying lesions in mammograms with Deep Learning.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'Luminosity and brightness of the organs viewed on the screen are important for orientation during teleoperating. They can impair the precision of our orientation ability.|The way in which organs scatter light during laparoscopic operation was detected and analyzed by means of reflection spectrophotometry with the Photoshop program package (ADOBE). The arithmetical skill was measured during different illumination conditions of the screen created randomly, using values of the liver, gallbladder, and omental fat.|Separate human tissues and organs could be divided on the basis of their luminosity into three basic groups: the luminous tissues such as fat, the stomach, and the bowel; medium-luminous organs such as diaphragm and gallbladder; and dark, mostly parenchymatous organs such as the liver and the spleen. The most luminous tissue illuminated the screen intensely. In experiments the hyperillumination impaired arithmetical skills of 15 volunteers. The errors appeared constantly and the time needed to count the generated points was significantly longer when 40% of intensively illuminated view was exceeded.|A situation may occur wherein the imaging system lacks the capacity to transmit needed optical information. Intense brightness of the screen may suppress surgical skills during teleoperating. Critical limit was achieved in our experiment with 40% of the view illuminated by omentum fat. This limit can be important for robotic surgical systems such as automated robotic camera control.',\n",
              "  'authors': ['Danis, J'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adipose Tissue',\n",
              "   'Cholecystectomy, Laparoscopic',\n",
              "   'Contrast Media',\n",
              "   'Evaluation Studies as Topic',\n",
              "   'Gallbladder',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Light',\n",
              "   'Liver',\n",
              "   'Robotics',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Spectrophotometry',\n",
              "   'Task Performance and Analysis',\n",
              "   'Video Recording'],\n",
              "  'pmid': '9685532',\n",
              "  'title': 'The effect of contrast variation on task performance in video evaluation.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.563',\n",
              "  'IF_2014': '0.589',\n",
              "  'IF_2015': '0.605',\n",
              "  'IF_2016': '0.506',\n",
              "  'IF_2017': '0.439',\n",
              "  'ISSN': '0236-5383',\n",
              "  'ISSN_print': '0236-5383',\n",
              "  'IsoAbbr': 'Acta. Biol. Hung.',\n",
              "  'JrId': 30,\n",
              "  'MedAbbr': 'Acta Biol Hung',\n",
              "  'NLMid': '8404358',\n",
              "  'Titre': 'Acta. Biol. Hung.',\n",
              "  'abstract': 'Composite categories of formal neurons or units (FN, FU) and their nets (FNN) are displayed. Long state cycles based on previously described methods in FNN-s are presented too. These results make easier any orientation among the immense number of possible cases of such nets. The taxonomical entities (classes, categories) give also cues for the synthesis of long and complicated cycles. The networks belong to the most complicated kind of net synthesis, thus realisations of other cases are expected to be easier. The observed wiring versus dynamics relationships give an insight into the real network control mechanisms and points to additional mechanisms for real nets. The work is a founded on previous results of the author.',\n",
              "  'authors': ['Lábos, E'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Acta biologica Hungarica',\n",
              "  'keywords': ['Biological Clocks',\n",
              "   'Models, Neurological',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons'],\n",
              "  'pmid': '9123997',\n",
              "  'title': 'Long cycles and special categories of formal neuronal units and networks.',\n",
              "  'year': '1996'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1553-4669',\n",
              "  'IF_2013': '1.575',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.39',\n",
              "  'IF_2016': '3.061',\n",
              "  'IF_2017': '3.061',\n",
              "  'ISSN': '1553-4650',\n",
              "  'ISSN_online': '1553-4669',\n",
              "  'ISSN_print': '1553-4650',\n",
              "  'IsoAbbr': 'J Minim Invasive Gynecol',\n",
              "  'JrId': 32424,\n",
              "  'MedAbbr': 'J Minim Invasive Gynecol',\n",
              "  'NLMid': '101235322',\n",
              "  'Titre': 'J Minim Invasive Gynecol',\n",
              "  'abstract': 'To compare the surgical outcome of elderly and younger patients undergoing laparoscopic or robotic surgical staging of endometrial cancer.|Retrospective analysis (Canadian Task Force classification II-2).|University-affiliated hospital.|One hundred twenty-nine patients comprised the study group. Sixty patients were aged 65 years or older (elderly group), and 69 patients were younger than 65 years (younger group).|Abdominal, laparoscopic, or robotic hysterectomy.|Among the 109 patients who underwent laparoscopic or robotic staging, there were no differences in estimated blood loss, lymph node count, surgical time, complications, rate of blood transfusion, conversion to laparotomy, and mean postoperative stay between elderly and younger patients.|Minimally invasive surgical staging for endometrial cancer is both feasible and safe in the elderly population and offers similar outcomes as in younger patients.',\n",
              "  'authors': ['Frey, Melissa K',\n",
              "   'Ihnow, Stephanie B',\n",
              "   'Worley, Michael J',\n",
              "   'Heyman, Katherine P',\n",
              "   'Kessler, Robin',\n",
              "   'Slomovitz, Brian M',\n",
              "   'Holcomb, Kevin M'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Journal of minimally invasive gynecology',\n",
              "  'keywords': ['Adult',\n",
              "   'Age Factors',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Carcinoma, Endometrioid',\n",
              "   'Endometrial Neoplasms',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Staging',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '21354065',\n",
              "  'title': 'Minimally invasive staging of endometrial cancer is feasible and safe in elderly women.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1938-3797',\n",
              "  'IF_2013': '0.786',\n",
              "  'IF_2014': '0.913',\n",
              "  'IF_2015': '0.814',\n",
              "  'IF_2016': '1.256',\n",
              "  'IF_2017': '1.298',\n",
              "  'ISSN': '1086-8089',\n",
              "  'ISSN_online': '1938-3797',\n",
              "  'ISSN_print': '1086-8089',\n",
              "  'IsoAbbr': 'JSLS',\n",
              "  'JrId': 21218,\n",
              "  'MedAbbr': 'JSLS',\n",
              "  'NLMid': '100884618',\n",
              "  'Titre': 'JSLS',\n",
              "  'abstract': 'The degradation in robotic skills that occurs during periods of robotic surgical inactivity in newly trained surgeons was measured. The role of animate training in robotic skill was also assessed.|Robotically naive resident and attending surgeons underwent training with the da Vinci robot on needle passage (DN), rocking ring transfer peg board (RPB), and running suture pod tasks (SP). Errors were established to convert actual time to adjusted time. Participants were deemed \"proficient\" once their adjusted times were within 80% of those set by experienced surgeons through repeated trials. Participants did not use the robot except for repeating the tasks once at 4, 8, and 12 weeks (tests). Participants then underwent animate training and completed a final test within 7 days.|Twenty-five attending and 29 resident surgeons enrolled; 3 withdrew. There were significant increases in time to complete each of the tasks, and in errors, by 4 weeks (Adjusted times: DN: 122.9 +/- 2.2 to 204.2 +/- 11.7, t = 6.9, P < .001; RPB: 262.4 +/- 2.5 to 364.7 +/- 8.0, t = 12.4, P < .001; SP: 91.4 +/- 1.4 to 169.9 +/- 6.8, t = 11.3, P < .001). Times decreased following animate training, but not to levels observed after proficiency training for the RPB and SP modules.|Robotic surgical skills degrade significantly within 4 weeks of inactivity in newly trained surgeons. Animate training may provide different skills than those acquired in the dry lab.',\n",
              "  'authors': ['Jenison, Eric L',\n",
              "   'Gil, Karen M',\n",
              "   'Lendvay, Thomas S',\n",
              "   'Guy, Michael S'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'JSLS : Journal of the Society of Laparoendoscopic Surgeons',\n",
              "  'keywords': ['Adult',\n",
              "   'Clinical Competence',\n",
              "   'Female',\n",
              "   'General Surgery',\n",
              "   'Gynecology',\n",
              "   'Humans',\n",
              "   'Learning Curve',\n",
              "   'Male',\n",
              "   'Obstetrics',\n",
              "   'Robotics',\n",
              "   'Suture Techniques',\n",
              "   'Task Performance and Analysis',\n",
              "   'Time Factors'],\n",
              "  'pmid': '23477169',\n",
              "  'title': 'Robotic surgical skills: acquisition, maintenance, and degradation.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'A new efficient hybrid intelligent approach based on fuzzy wavelet neural network (FWNN) was proposed for effectively modeling and simulating biodegradation process of Dimethyl phthalate (DMP) in an anaerobic/anoxic/oxic (AAO) wastewater treatment process. With the self learning and memory abilities of neural networks (NN), handling uncertainty capacity of fuzzy logic (FL), analyzing local details superiority of wavelet transform (WT) and global search of genetic algorithm (GA), the proposed hybrid intelligent model can extract the dynamic behavior and complex interrelationships from various water quality variables. For finding the optimal values for parameters of the proposed FWNN, a hybrid learning algorithm integrating an improved genetic optimization and gradient descent algorithm is employed. The results show, compared with NN model (optimized by GA) and kinetic model, the proposed FWNN model have the quicker convergence speed, the higher prediction performance, and smaller RMSE (0.080), MSE (0.0064), MAPE (1.8158) and higher R2 (0.9851) values. which illustrates FWNN model simulates effluent DMP more accurately than the mechanism model.',\n",
              "  'authors': ['Huang, Mingzhi', 'Zhang, Tao', 'Ruan, Jujun', 'Chen, Xiaohong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Aerobiosis',\n",
              "   'Algorithms',\n",
              "   'Anaerobiosis',\n",
              "   'Biodegradation, Environmental',\n",
              "   'Fuzzy Logic',\n",
              "   'Kinetics',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Phthalic Acids'],\n",
              "  'pmid': '28120889',\n",
              "  'title': 'A New Efficient Hybrid Intelligent Model for Biodegradation Process of DMP with Fuzzy Wavelet Neural Networks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'The technology of tongue near-infrared reflectance spectra was used for human serum total protein (TP) content of noninvasive testing for the first time. Reflectance spectrum on the tongue tips of 58 volunteers was collected, and the biochemical values of serum total protein were recorded at the same time. The samples were separated into two parts: training set and prediction set. Two prediction models were established using PCA combined with BP neural network and PLS. Using PCA-BP model to predict the prediction set, the average relative error is 7.35%, RMSEP was 6.3771 g x L(-1), and the correlation coefficient was 0.9021. Using PLS model to predict the prediction set, the average relative error is 4.77%, RMSEP was 0.1304 g x L(-1), and the correlation coefficient was 0.9718. It was approved that reflectance spectra of tongue can be used to predict TP accurately and noninvasively.',\n",
              "  'authors': ['Lin, Ling',\n",
              "   'Li, Zhe',\n",
              "   'Li, Xiao-xia',\n",
              "   'Li, Yong-cheng',\n",
              "   'Li, Gang',\n",
              "   'Zhang, Bao-ju',\n",
              "   'Song, Wei'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Blood Proteins',\n",
              "   'Humans',\n",
              "   'Medicine, Chinese Traditional',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Tongue'],\n",
              "  'pmid': '23156762',\n",
              "  'title': '[Noninvasive measurement of serum total protein content by near-infrared reflection spectra with tongue inspection].',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1553-4669',\n",
              "  'IF_2013': '1.575',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.39',\n",
              "  'IF_2016': '3.061',\n",
              "  'IF_2017': '3.061',\n",
              "  'ISSN': '1553-4650',\n",
              "  'ISSN_online': '1553-4669',\n",
              "  'ISSN_print': '1553-4650',\n",
              "  'IsoAbbr': 'J Minim Invasive Gynecol',\n",
              "  'JrId': 32424,\n",
              "  'MedAbbr': 'J Minim Invasive Gynecol',\n",
              "  'NLMid': '101235322',\n",
              "  'Titre': 'J Minim Invasive Gynecol',\n",
              "  'abstract': 'The objective of this study was to estimate the tensile strength of surgical knots made using the da Vinci robot. Four different types of flat square knots (strand-to-strand 4 throw, strand-to-strand 6 throw, loop-to-strand 4 throw, and loop-to-strand 6 throw) were made using the da Vinci-S system by 4 different surgeons, all experienced with the system. For the knots, we used braided polyglactin 910 (Vicryl 2-0). Hand-tied, flat, square, 4-throw strand-to-strand knots were used as reference. The tensile strength was measured for all knots using the Instron 5566 system calibrated to an accuracy of + or - .5% at 4 to 10 newtons (N) and + or - .4% at greater than 10 N. Compared with reference knots, only 1 of 4 surgeons could make knots as equally strong with the robot. For all surgeons, strand-to-strand knots had a significantly higher tensile strength than loop-to-strand knots when made with the robot. Adding 2 throws to the knot did not increase the knots strength in the robot. It is possible to make equally strong surgical knots with the da Vinci robot as by hand; however, despite previous experience with the robot, only 1 of 4 surgeons managed to do so. Adding 2 throws to R4SS and R4LS knots did not increase the tensile strength significantly for any of the 4 surgeons. It is important to train and tie knots using the da Vinci system with the same care as by hand and to be aware of possible differences in knot-tying technique with the robot and manually. With the robot, strand-to-strand knots were stronger than loop-to-strand knots, and should be preferred.',\n",
              "  'authors': ['Reynisson, Pétur',\n",
              "   'Shokri, Ebi',\n",
              "   'Bendahl, Pär-Ola',\n",
              "   'Persson, Jan'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Journal of minimally invasive gynecology',\n",
              "  'keywords': ['Materials Testing',\n",
              "   'Polyglactin 910',\n",
              "   'Robotics',\n",
              "   'Statistics, Nonparametric',\n",
              "   'Suture Techniques',\n",
              "   'Sutures',\n",
              "   'Tensile Strength'],\n",
              "  'pmid': '20227925',\n",
              "  'title': 'Tensile strength of surgical knots performed with the da Vinci surgical robot.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['EE'],\n",
              "  'ESSN': '1420-3049',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '2.416',\n",
              "  'IF_2015': '2.465',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '3.098',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1420-3049',\n",
              "  'IsoAbbr': 'Molecules',\n",
              "  'JrId': 22275,\n",
              "  'MedAbbr': 'Molecules',\n",
              "  'NLMid': '100964009',\n",
              "  'Titre': 'Molecules',\n",
              "  'abstract': 'Key-based substructural fingerprints are an important element of computer-aided drug design techniques. The usefulness of the fingerprints in filtering compound databases is invaluable, as they allow for the quick rejection of molecules with a low probability of being active. However, this method is flawed, as it does not consider the connections between substructures. After changing the connections between particular chemical moieties, the fingerprint representation of the compound remains the same, which leads to difficulties in distinguishing between active and inactive compounds. In this study, we present a new method of compound representation-substructural connectivity fingerprints (SCFP), providing information not only about the presence of particular substructures in the molecule but also additional data on substructure connections. Such representation was analyzed by the recently developed methodology-extreme entropy machines (EEM). The SCFP can be a valuable addition to virtual screening tools, as it represents compound structure with greater detail and more specificity, allowing for more accurate classification.',\n",
              "  'authors': ['Rataj, Krzysztof',\n",
              "   'Czarnecki, Wojciech',\n",
              "   'Podlewska, Sabina',\n",
              "   'Pocha, Agnieszka',\n",
              "   'Bojarski, Andrzej J'],\n",
              "  'categories': ['CHEMISTRY, ORGANIC'],\n",
              "  'journal': 'Molecules (Basel, Switzerland)',\n",
              "  'keywords': ['Chemistry, Pharmaceutical',\n",
              "   'Computer-Aided Design',\n",
              "   'Databases, Factual',\n",
              "   'Databases, Pharmaceutical',\n",
              "   'Drug Design',\n",
              "   'Drug Evaluation, Preclinical',\n",
              "   'Entropy',\n",
              "   'Machine Learning',\n",
              "   'Molecular Structure',\n",
              "   'Small Molecule Libraries',\n",
              "   'Structure-Activity Relationship'],\n",
              "  'pmid': '29789513',\n",
              "  'title': 'Substructural Connectivity Fingerprint and Extreme Entropy Machines-A New Method of Compound Representation and Analysis.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1536-0210',\n",
              "  'IF_2013': '4.453',\n",
              "  'IF_2014': '4.437',\n",
              "  'IF_2015': '4.887',\n",
              "  'IF_2016': '5.195',\n",
              "  'IF_2017': '6.224',\n",
              "  'ISSN': '0020-9996',\n",
              "  'ISSN_online': '1536-0210',\n",
              "  'ISSN_print': '0020-9996',\n",
              "  'IsoAbbr': 'Invest Radiol',\n",
              "  'JrId': 4373,\n",
              "  'MedAbbr': 'Invest Radiol',\n",
              "  'NLMid': '0045377',\n",
              "  'Titre': 'Invest Radiol',\n",
              "  'abstract': 'Accurate measurement is crucial for the assessment of tumor dimensions to allow accurate evaluation of tumor response. Thus, the purpose of our study was to assess the accuracy of semi-automated RECIST and volumetric measurements of liver lesions in a liver phantom with different CT acquisition parameters.|A phantom of the upper abdomen with 14 hepatic lesions of different sizes (diameter: 12.0-40.0 mm), densities (45/180 HU at 120 kV), or alignment (vertical/transverse) was scanned with a 16-slice multidetector row computed tomography using varying tube currents (40/60/80/100/120/165mAs eff), reconstruction kernels (Siemens B20/30/40/50/70s), or slice thicknesses (1/2/3/4/5 mm). Longest axial diameter and volume of the 14 lesions were quantified using a semi-automated software tool (SyngoOncology, Siemens Medical Solutions, Forchheim, Germany) and compared with the known real longest axial diameter and volume values of the lesions. Absolute percentage errors (APE) were calculated. Degree of agreement in longest axial diameter and volume between software and real measurements was represented graphically in Bland-Altman plots and by corresponding concordance correlation coefficient.|At standard soft tissue reconstruction kernel (Siemens B30s) and slice thickness (3 mm) mean absolute percentage error APE (concordance correlation coefficients) ranged between 6.93 and 14.27 (0.96 and 0.99) for longest axial diameter and between 4.98 and 10.85 (0.99 and 1.00) for volume. At varying reconstruction kernels, APE values (concordance correlation coefficients) ranged between 7.92 and 8.31 (0.98 and 0.99) for longest axial diameter and between 4.95 and 6.93 (1.00) for volume. Applying different slice sections APE values (concordance correlation coefficients) differed from 6.54 to 11.82 (0.97 and 0.99) for longest axial diameter and from 6.93 to 9.17 (1.00) for volume.|Software quantification of longest axial diameter and volume of hepatic lesions in a phantom demonstrated a high correlation and accuracy under varying multidetector row computed tomography parameter.',\n",
              "  'authors': ['Keil, Sebastian',\n",
              "   'Plumhans, Cedric',\n",
              "   'Behrendt, Florian F',\n",
              "   'Stanzel, Sven',\n",
              "   'Suehling, Michael',\n",
              "   'Mühlenbruch, Georg',\n",
              "   'Mahnken, Andreas H',\n",
              "   'Günther, Rolf W',\n",
              "   'Das, Marco'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Investigative radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Liver Neoplasms',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Phantoms, Imaging',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '19104439',\n",
              "  'title': 'Semi-automated quantification of hepatic lesions in a phantom.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1938-1352',\n",
              "  'IF_2013': '1.688',\n",
              "  'IF_2014': '1.43',\n",
              "  'IF_2015': '1.043',\n",
              "  'IF_2016': '1.277',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0748-7711',\n",
              "  'ISSN_online': '1938-1352',\n",
              "  'ISSN_print': '0748-7711',\n",
              "  'IsoAbbr': 'J Rehabil Res Dev',\n",
              "  'JrId': 5189,\n",
              "  'MedAbbr': 'J Rehabil Res Dev',\n",
              "  'NLMid': '8410047',\n",
              "  'Titre': 'J Rehabil Res Dev',\n",
              "  'abstract': 'This study determined the feasibility and performance of center of mass (COM) acceleration feedback control of a neuroprosthesis utilizing functional neuromuscular stimulation (FNS) to restore standing balance to a single subject paralyzed by a motor and sensory complete, thoracic-level spinal cord injury. An artificial neural network (ANN) was created to map gain-modulated changes in total body COM acceleration estimated from body-mounted sensors to optimal changes in stimulation required to maintain standing. Feedback gains were systematically tuned to minimize the upper-limb (UL) loads applied by the subject to an instrumented support device during internally generated postural perturbations produced by volitional reaching and object manipulation. Total body COM acceleration was accurately estimated (>90% variance explained) from 2 three-dimensional (3-D) accelerometers mounted on the pelvis and torso. Compared with constant muscle stimulation employed clinically, COM acceleration feedback control of stimulation improved standing performance by reducing the UL loading required to resist internal postural disturbances by 27%. This case study suggests that COM acceleration feedback could potentially be advantageous in a standing neuroprosthesis since it can be implemented with only a few feedback parameters and requires minimal instrumentation for comprehensive 3-D control of dynamic standing function.',\n",
              "  'authors': ['Nataraj, Raviraj', 'Audu, Musa L', 'Triolo, Ronald J'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'Journal of rehabilitation research and development',\n",
              "  'keywords': ['Acceleration',\n",
              "   'Electric Stimulation Therapy',\n",
              "   'Electromyography',\n",
              "   'Feasibility Studies',\n",
              "   'Feedback, Physiological',\n",
              "   'Female',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Logistic Models',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Postural Balance',\n",
              "   'Posture',\n",
              "   'Psychomotor Performance',\n",
              "   'Spinal Cord Injuries',\n",
              "   'Telemetry'],\n",
              "  'pmid': '23299260',\n",
              "  'title': 'Center of mass acceleration feedback control of functional neuromuscular stimulation for standing in presence of internal postural perturbations.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'Early detection of carotid atherosclerosis on the vessel wall (VW) magnetic resonance imaging (MRI) (VW-MRI) images can prevent the progression of cardiovascular disease. However, the manual inspection process of the VW-MRI images is cumbersome and has low reproducibility. Therefore in this paper, by using the convolutional neural networks (CNNs), we develop a deep morphology aided diagnosis (DeepMAD) network for automated segmentation of the VW of carotid artery and for automated diagnosis of the carotid atherosclerosis with the black-blood (BB) VW-MRI (i.e., the T1-weighted MRI) in a slice-by-slice manner.|The proposed DeepMAD network consists of a segmentation subnetwork and a diagnosis subnetwork for performing the segmentation and diagnosis tasks on the BB-VW-MRI images, where the manual labeled lumen area, the manual labeled outer wall area and the manual labeled lesion Types based on the modified American Heart Association (AHA) criteria are used as the ground-truth. Specifically, a deep U-shape CNN with a weighted fusion layer is designed as the segmentation subnetwork, where the lumen area and the outer wall area can be simultaneously segmented under the supervision of the triple Dice loss to provide the vessel wall map as morphological information. Then, the image stream from the BB-VWMRI image and the morphology stream from the obtained vessel wall map are extracted from two deep CNNs and combined to obtain the diagnosis results of atherosclerosis in the diagnosis subnetwork. In addition, the triple input set is formed by three carotid regions of interest (ROIs) from three consecutive slices of the MRI sequence and input to the DeepMAD network, where the first and last slices used as additional adjacent slices to provide 2.5D spatial information along the carotid artery centerline for the intermediate slice, which is the target slice for segmentation and diagnosis in the study.|Compared to other existing methods, the DeepMAD network can achieve promising segmentation performances (0.9594 Dice for the lumen and 0.9657 Dice for the outer wall) and better diagnosis Accuracy of the carotid atherosclerosis (0.9503 AUC and 0.8916 Accuracy) in the test dataset (including invisible subjects) from same source as the training dataset. In addition, the trained DeepMAD model can be successfully transferred to another test dataset for segmentation and diagnosis tasks with remarkable performance (0.9475 Dice for the lumen and 0.9542 Dice for the outer wall, 0. 9227 AUC and 0.8679 Accuracy for diagnosis).|Even without the intervention of reviewers required for previous works, the proposed DeepMAD network automatically segments the lumen and the outer wall together and diagnoses the carotid atherosclerosis with high performances. The DeepMAD network can be used in clinical trials to help radiologists get rid of tedious reading tasks, such as screening review to separate the normal carotid from the atherosclerotic arteries and outlining the vessel wall contours.',\n",
              "  'authors': ['Wu, Jiayi',\n",
              "   'Xin, Jingmin',\n",
              "   'Yang, Xiaofeng',\n",
              "   'Sun, Jie',\n",
              "   'Xu, Dongxiang',\n",
              "   'Zheng, Nanning',\n",
              "   'Yuan, Chun'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31356693',\n",
              "  'title': 'Deep morphology aided diagnosis network for segmentation of carotid artery vessel wall and diagnosis of carotid atherosclerosis on black-blood vessel wall MRI.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1600-0412',\n",
              "  'IF_2013': '1.985',\n",
              "  'IF_2014': '2.426',\n",
              "  'IF_2015': '2.191',\n",
              "  'IF_2016': '2.48',\n",
              "  'IF_2017': '2.649',\n",
              "  'ISSN': '0001-6349',\n",
              "  'ISSN_online': '1600-0412',\n",
              "  'ISSN_print': '0001-6349',\n",
              "  'IsoAbbr': 'Acta Obstet Gynecol Scand',\n",
              "  'JrId': 139,\n",
              "  'MedAbbr': 'Acta Obstet Gynecol Scand',\n",
              "  'NLMid': '0370343',\n",
              "  'Titre': 'Acta Obstet Gynecol Scand',\n",
              "  'abstract': 'The objective was to assess the impact of robot-assisted radical hysterectomy (RRH) on surgical and oncologic outcome and costs compared with open radical hysterectomy (ORH) at a tertiary referral center in Sweden.|In this retrospective analysis all patients treated with radical hysterectomy and pelvic lymphadenectomy for early stage uterine cervical cancer during 2006-2015 were included (n = 304). The patients were divided into two groups, ORH (n = 155) and RRH (n = 149). Patient characteristics, FIGO stage, histology, adjuvant therapy, operation time, length of stay (LOS), lymph node yield, recurrence rate and survival were retrieved from medical records. Complications were graded according to the Clavien-Dindo classification. In addition, costs related to the surgical treatments were calculated.|Blood loss, LOS and intraoperative complications were significantly lower as well as lymph node yield after RRH. No differences in postoperative complications or costs were observed between the two groups. Recurrence of disease was detected in 13.4 and 10.3% after RRH and ORH, respectively. Regression analysis demonstrated that histology, tumor size, positive lymph nodes and type of operation (RRH) were significantly associated with recurrence.|The introduction of RRH was accompanied by similar postoperative complication rates and costs but lower LOS compared with ORH. An initial learning curve may account for the higher recurrence rate observed after RRH. These data reinforce the need for structured training and monitoring of outcomes when novel treatment modalities are introduced.',\n",
              "  'authors': ['Wallin, Emelie',\n",
              "   'Flöter Rådestad, Angelique',\n",
              "   'Falconer, Henrik'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Acta obstetricia et gynecologica Scandinavica',\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Neoplasm Staging',\n",
              "   'Postoperative Complications',\n",
              "   'Referral and Consultation',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Survival Rate',\n",
              "   'Sweden',\n",
              "   'Treatment Outcome',\n",
              "   'Uterine Cervical Neoplasms'],\n",
              "  'pmid': '28186616',\n",
              "  'title': 'Introduction of robot-assisted radical hysterectomy for early stage cervical cancer: impact on complications, costs and oncologic outcome.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Matrix Metalloproteases (MMPs) are an important family of proteases that play crucial roles in key cellular and disease processes. Therefore, MMPs constitute important targets for drug design, development and delivery. Advanced proteomic technologies have identified type-specific target substrates; however, the complete repertoire of MMP substrates remains uncharacterized. Indeed, computational prediction of substrate-cleavage sites associated with MMPs is a challenging problem. This holds especially true when considering MMPs with few experimentally verified cleavage sites, such as for MMP-2, -3, -7, and -8. To fill this gap, we propose a new knowledge-transfer computational framework which effectively utilizes the hidden shared knowledge from some MMP types to enhance predictions of other, distinct target substrate-cleavage sites. Our computational framework uses support vector machines combined with transfer machine learning and feature selection. To demonstrate the value of the model, we extracted a variety of substrate sequence-derived features and compared the performance of our method using both 5-fold cross-validation and independent tests. The results show that our transfer-learning-based method provides a robust performance, which is at least comparable to traditional feature-selection methods for prediction of MMP-2, -3, -7, -8, -9 and -12 substrate-cleavage sites on independent tests. The results also demonstrate that our proposed computational framework provides a useful alternative for the characterization of sequence-level determinants of MMP-substrate specificity.',\n",
              "  'authors': ['Wang, Yanan',\n",
              "   'Song, Jiangning',\n",
              "   'Marquez-Lago, Tatiana T',\n",
              "   'Leier, André',\n",
              "   'Li, Chen',\n",
              "   'Lithgow, Trevor',\n",
              "   'Webb, Geoffrey I',\n",
              "   'Shen, Hong-Bin'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Sequence',\n",
              "   'Computational Biology',\n",
              "   'Databases, Factual',\n",
              "   'Isoenzymes',\n",
              "   'Matrix Metalloproteinases',\n",
              "   'Reproducibility of Results',\n",
              "   'Substrate Specificity',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '28720874',\n",
              "  'title': 'Knowledge-transfer learning for prediction of matrix metalloprotease substrate-cleavage sites.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'Ultrasound examination (US) does a key role in the diagnosis and management of the patients with clinically suspected appendicitis which is the most common abdominal surgical emergency. Among the various sonographic findings of appendicitis, outer diameter of the appendix is most important. Therefore, clear delineation of the appendix on US images is essential. In this paper, we propose a new intelligent method to extract appendix automatically from abdominal sonographic images as a basic building block of developing such an intelligent tool for medical practitioners. Knowing that the appendix is located at the lower organ area below the bottom fascia line, we conduct a series of image processing techniques to find the fascia line correctly. And then we apply fuzzy ART learning algorithm to the organ area in order to extract appendix accurately. The experiment verifies that the proposed method is highly accurate (successful in 38 out of 40 cases) in extracting appendix. ',\n",
              "  'authors': ['Kim, Kwang Baek',\n",
              "   'Park, Hyun Jun',\n",
              "   'Song, Doo Heon',\n",
              "   'Han, Sang-suk'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Appendicitis',\n",
              "   'Appendix',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Ultrasonography',\n",
              "   'Unsupervised Machine Learning'],\n",
              "  'pmid': '26089963',\n",
              "  'title': 'Developing an Intelligent Automatic Appendix Extraction Method from Ultrasonography Based on Fuzzy ART and Image Processing.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'The design of novel proteins has many applications but remains an attritional process with success in isolated cases. Meanwhile, deep learning technologies have exploded in popularity in recent years and are increasingly applicable to biology due to the rise in available data. We attempt to link protein design and deep learning by using variational autoencoders to generate protein sequences conditioned on desired properties. Potential copper and calcium binding sites are added to non-metal binding proteins without human intervention and compared to a hidden Markov model. In another use case, a grammar of protein structures is developed and used to produce sequences for a novel protein topology. One candidate structure is found to be stable by molecular dynamics simulation. The ability of our model to confine the vast search space of protein sequences and to scale easily has the potential to assist in a variety of protein design tasks.',\n",
              "  'authors': ['Greener, Joe G', 'Moffat, Lewis', 'Jones, David T'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Binding Sites',\n",
              "   'Computational Biology',\n",
              "   'Deep Learning',\n",
              "   'Humans',\n",
              "   'Metalloproteins',\n",
              "   'Molecular Dynamics Simulation',\n",
              "   'Protein Folding'],\n",
              "  'pmid': '30385875',\n",
              "  'title': 'Design of metalloproteins and novel protein folds using variational autoencoders.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': \"To characterize changes in indices of urinary function in prostatectomy patients with presurgical voiding symptoms.|A retrospective analysis of our prostate cancer database identified robot-assisted radical prostatectomy patients between April 2007 and December 2011 who completed pre- and postsurgical (24 months) Expanded Prostate Cancer Index Composite-26 surveys. Gleason score, margins, D'Amico risk, prostate-specific antigen, radiotherapy, and nerve-sparing status were tabulated. Survey questions addressed urinary irritation/obstruction, incontinence, and overall bother. Responses were averaged to calculate a urinary sum (US) score. Patients were stratified according to the severity of their baseline urinary bother (UB), and changes in urinary indices determined at 24 months.|A total of 737 patients were included. Postsurgical improvement in urinary obstruction, bother, and sum score was related to baseline UB (P <.001). Men with severe baseline bother had the greatest improvement in US (+9.3), whereas those with asymptomatic baseline UB experienced a decline in US (-2.8). All patients experienced a decline in urinary incontinence of 6.3-8.3 that was independent of baseline bother (P = .507). Patients with severe UB experienced positive outcomes, whereas those at asymptomatic baseline experienced negative US outcomes. Negative urinary incontinence outcomes were unrelated to baseline UB. Age, radiotherapy, and nerve-sparing status were not associated with improved UB (P = .029). However, baseline UB was significantly associated with improvement in postsurgical UB (P = .001).|Baseline UB is a predictor of postsurgical improvement in urinary function. These data are helpful when counseling a subset of robot-assisted laparoscopic radical prostatectomy patients with severe preoperative urinary symptoms.\",\n",
              "  'authors': ['Murphy, Gregory',\n",
              "   'Haddock, Peter',\n",
              "   'Doak, Hoyt',\n",
              "   'Jackson, Max',\n",
              "   'Dorin, Ryan',\n",
              "   'Meraney, Anoop',\n",
              "   'Kesler, Stuart',\n",
              "   'Staff, Ilene',\n",
              "   'Wagner, Joseph R'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Humans',\n",
              "   'Incidence',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Quality of Life',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'United States',\n",
              "   'Urinary Incontinence',\n",
              "   'Urination'],\n",
              "  'pmid': '26166672',\n",
              "  'title': 'Urinary Bother as a Predictor of Postsurgical Changes in Urinary Function After Robotic Radical Prostatectomy.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1097-0290',\n",
              "  'IF_2013': '4.164',\n",
              "  'IF_2014': '4.126',\n",
              "  'IF_2015': '4.243',\n",
              "  'IF_2016': '4.481',\n",
              "  'IF_2017': '3.952',\n",
              "  'ISSN': '0006-3592',\n",
              "  'ISSN_online': '1097-0290',\n",
              "  'ISSN_print': '0006-3592',\n",
              "  'IsoAbbr': 'Biotechnol. Bioeng.',\n",
              "  'JrId': 1200,\n",
              "  'MedAbbr': 'Biotechnol Bioeng',\n",
              "  'NLMid': '7502021',\n",
              "  'Titre': 'Biotechnol. Bioeng.',\n",
              "  'abstract': 'The concepts of artificial intelligence have been applied for the development of a software system for metabolic pathway synthesis (MPS). An easily expandable data base system for storing enzyme and substance descriptions is used by a search algorithm for the identification of possible ways to interconvert carbon-carrying metabolites. A versatile screening capability permits the user to identify all pathways which contain or exclude any combination of enzymes, substrates, and/or products. Information provided by MPS can be used to predict on a qualitative basis the effects of adding or deleting enzymatic activities to or from the cellular environment, to classify pathways with respect to cellular objectives, and to extract information about metabolic regulation. MPS can be used subsequently to aid the identification of appropriate genotypes or genetic modifications that will redirect metabolism towards amplified production of desirable bioproducts. Two examples illustrating the capabilities of MPS are presented. In the first example, which considers the conversion of glucose 6-phosphate to pyruvate, MPS synthesized the classical catabolic pathways (EMP, pentose phosphate and ED) along with possible variations. A route for the biosynthesis of L-alanine that does not incorporate the enzyme alanine aminotransferase was revealed by MPS during synthesis of alternative pathways which produce L-alanine from pyruvate.',\n",
              "  'authors': ['Seressiotis, A', 'Bailey, J E'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Biotechnology and bioengineering',\n",
              "  'keywords': [''],\n",
              "  'pmid': '18584649',\n",
              "  'title': 'MPS: An artificially intelligent software system for the analysis and synthesis of metabolic pathways.',\n",
              "  'year': '1988'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5305',\n",
              "  'IF_2013': '1.735',\n",
              "  'IF_2014': '1.068',\n",
              "  'IF_2015': '1.069',\n",
              "  'IF_2016': '0.964',\n",
              "  'IF_2017': '1.039',\n",
              "  'ISSN': '0929-8665',\n",
              "  'ISSN_online': '1875-5305',\n",
              "  'ISSN_print': '0929-8665',\n",
              "  'IsoAbbr': 'Protein Pept. Lett.',\n",
              "  'JrId': 20398,\n",
              "  'MedAbbr': 'Protein Pept Lett',\n",
              "  'NLMid': '9441434',\n",
              "  'Titre': 'Protein Pept. Lett.',\n",
              "  'abstract': 'In this paper, we adopted the bounded support vector machine to locate the key long-range interaction sites by the use of predicted local lowest B-factors. As a result, the key long-range interaction residues can be located based on information of local lowest B-factor sites.',\n",
              "  'authors': ['Chen, Peng',\n",
              "   'Han, Kyungsook',\n",
              "   'Li, Xueling',\n",
              "   'Huang, De-Shuang'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Protein and peptide letters',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'Models, Molecular',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Conformation',\n",
              "   'Protein Folding',\n",
              "   'Protein Interaction Domains and Motifs',\n",
              "   'Protein Interaction Mapping',\n",
              "   'Proteins',\n",
              "   'Temperature'],\n",
              "  'pmid': '18537737',\n",
              "  'title': 'Predicting key long-range interaction sites by B-factors.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1973-9095',\n",
              "  'IF_2013': '1.946',\n",
              "  'IF_2014': '1.903',\n",
              "  'IF_2015': '2.063',\n",
              "  'IF_2016': '1.827',\n",
              "  'IF_2017': '2.208',\n",
              "  'ISSN': '1973-9087',\n",
              "  'ISSN_online': '1973-9095',\n",
              "  'ISSN_print': '1973-9087',\n",
              "  'IsoAbbr': 'Eur J Phys Rehabil Med',\n",
              "  'JrId': 34251,\n",
              "  'MedAbbr': 'Eur J Phys Rehabil Med',\n",
              "  'NLMid': '101465662',\n",
              "  'Titre': 'Eur J Phys Rehabil Med',\n",
              "  'abstract': 'In this opinion piece, I will revisit the basis for our belief that robotic manipulation is a valid approach to promote speedier and better recovery following a stroke and discuss some of the clinical evidence that led to the September 2010 guidelines for stroke care of the American Heart Association as well as the December 2010 guidelines of the Veterans Administration endorsing the use of robotic technology for the upper extremity (UE) but not for the lower extremity (LE) post-stroke rehabilitation effort.',\n",
              "  'authors': ['Krebs, H I'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'European journal of physical and rehabilitation medicine',\n",
              "  'keywords': ['Disability Evaluation',\n",
              "   'Humans',\n",
              "   'Physical and Rehabilitation Medicine',\n",
              "   'Recovery of Function',\n",
              "   'Robotics',\n",
              "   'Stroke',\n",
              "   'Stroke Rehabilitation',\n",
              "   'Upper Extremity'],\n",
              "  'pmid': '22614892',\n",
              "  'title': 'Robotic technology and physical medicine and rehabilitation.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1827-1758',\n",
              "  'IF_2013': '0.7',\n",
              "  'IF_2014': '0.965',\n",
              "  'IF_2015': '0.536',\n",
              "  'IF_2016': '0.984',\n",
              "  'IF_2017': '1.449',\n",
              "  'ISSN': '0393-2249',\n",
              "  'ISSN_online': '1827-1758',\n",
              "  'ISSN_print': '0393-2249',\n",
              "  'IsoAbbr': 'Minerva Urol Nefrol',\n",
              "  'JrId': 7387,\n",
              "  'MedAbbr': 'Minerva Urol Nefrol',\n",
              "  'NLMid': '8503649',\n",
              "  'Titre': 'Minerva Urol Nefrol',\n",
              "  'abstract': 'Robotic surgery is being increasingly used for prostatectomy. Its use in renal surgery has been limited by a perception that it does not offer any extra benefit. This article explores the use of robot assistance in renal surgery. Extensive literature search undertaken using Medline and PubMed. The following keywords were used during the search: robot, robotic, kidney, renal, nephrectomy, partial nephrectomy, pyeloplasty. Relevant articles were extracted and data from these articles presented in the review. Comments on relevant aspects of either technique or results are provided. Apart from initial feasibility studies, little has been published on robot assisted laparoscopic nephrectomy (RALN) for diseased kidneys. Most of the cases of RALN has been carried out in live donors for renal transplantation in a few enthusiastic centres. Experience with robot assisted laparoscopic pyeloplasty (RALP) is more mature. Robot assisted laparoscopic partial nephrectomy (RALPN) is continuing to evolve. Robotic NOTES is currently experimental. Robot assistance seems to have a role in reconstructive renal procedures such as pyeloplasty and partial nephrectomy, due mainly to the precise suturing ability.',\n",
              "  'authors': ['Sairam, K',\n",
              "   'Elhage, O',\n",
              "   'Murphy, D',\n",
              "   'Challacombe, B',\n",
              "   'Hegarty, N',\n",
              "   'Dasgupta, P'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Minerva urologica e nefrologica = The Italian journal of urology and nephrology',\n",
              "  'keywords': ['Endoscopy',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Kidney Pelvis',\n",
              "   'Kidney Transplantation',\n",
              "   'Laparoscopes',\n",
              "   'Laparoscopy',\n",
              "   'Nephrectomy',\n",
              "   'Robotics',\n",
              "   'Ureter'],\n",
              "  'pmid': '18787513',\n",
              "  'title': 'Robotic renal surgery.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1546-1726',\n",
              "  'IF_2013': '14.976',\n",
              "  'IF_2014': '16.095',\n",
              "  'IF_2015': '16.724',\n",
              "  'IF_2016': '17.839',\n",
              "  'IF_2017': '19.912',\n",
              "  'ISSN': '1097-6256',\n",
              "  'ISSN_online': '1546-1726',\n",
              "  'ISSN_print': '1097-6256',\n",
              "  'IsoAbbr': 'Nat. Neurosci.',\n",
              "  'JrId': 21092,\n",
              "  'MedAbbr': 'Nat Neurosci',\n",
              "  'NLMid': '9809671',\n",
              "  'Titre': 'Nat. Neurosci.',\n",
              "  'abstract': 'Quantifying behavior is crucial for many applications in neuroscience. Videography provides easy methods for the observation and recording of animal behavior in diverse settings, yet extracting particular aspects of a behavior for further analysis can be highly time consuming. In motor control studies, humans or other animals are often marked with reflective markers to assist with computer-based tracking, but markers are intrusive, and the number and location of the markers must be determined a priori. Here we present an efficient method for markerless pose estimation based on transfer learning with deep neural networks that achieves excellent results with minimal training data. We demonstrate the versatility of this framework by tracking various body parts in multiple species across a broad collection of behaviors. Remarkably, even when only a small number of frames are labeled (~200), the algorithm achieves excellent tracking performance on test frames that is comparable to human accuracy.',\n",
              "  'authors': ['Mathis, Alexander',\n",
              "   'Mamidanna, Pranav',\n",
              "   'Cury, Kevin M',\n",
              "   'Abe, Taiga',\n",
              "   'Murthy, Venkatesh N',\n",
              "   'Mathis, Mackenzie Weygandt',\n",
              "   'Bethge, Matthias'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Nature neuroscience',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Behavior',\n",
              "   'Behavior, Animal',\n",
              "   'Deep Learning',\n",
              "   'Drosophila melanogaster',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Mice',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Odorants',\n",
              "   'Posture',\n",
              "   'Psychomotor Performance',\n",
              "   'Transfer (Psychology)',\n",
              "   'Video Recording'],\n",
              "  'pmid': '30127430',\n",
              "  'title': 'DeepLabCut: markerless pose estimation of user-defined body parts with deep learning.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Modeling of signal transduction pathways plays a major role in understanding cells' function and predicting cellular response. Mathematical formalisms based on a logic formalism are relatively simple but can describe how signals propagate from one protein to the next and have led to the construction of models that simulate the cells response to environmental or other perturbations. Constrained fuzzy logic was recently introduced to train models to cell specific data to result in quantitative pathway models of the specific cellular behavior. There are two major issues in this pathway optimization: i) excessive CPU time requirements and ii) loosely constrained optimization problem due to lack of data with respect to large signaling pathways. Herein, we address both issues: the former by reformulating the pathway optimization as a regular nonlinear optimization problem; and the latter by enhanced algorithms to pre/post-process the signaling network to remove parts that cannot be identified given the experimental conditions. As a case study, we tackle the construction of cell type specific pathways in normal and transformed hepatocytes using medium and large-scale functional phosphoproteomic datasets. The proposed Non Linear Programming (NLP) formulation allows for fast optimization of signaling topologies by combining the versatile nature of logic modeling with state of the art optimization algorithms.\",\n",
              "  'authors': ['Mitsos, Alexander',\n",
              "   'Melas, Ioannis N',\n",
              "   'Morris, Melody K',\n",
              "   'Saez-Rodriguez, Julio',\n",
              "   'Lauffenburger, Douglas A',\n",
              "   'Alexopoulos, Leonidas G'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Fuzzy Logic',\n",
              "   'Hepatocytes',\n",
              "   'Humans',\n",
              "   'Information Theory',\n",
              "   'Models, Biological',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Phosphoproteins',\n",
              "   'Primary Cell Culture',\n",
              "   'Protein Interaction Mapping',\n",
              "   'Proteome',\n",
              "   'Signal Transduction'],\n",
              "  'pmid': '23226239',\n",
              "  'title': 'Non Linear Programming (NLP) formulation for quantitative modeling of protein signal transduction pathways.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1532-8430',\n",
              "  'IF_2013': '1.363',\n",
              "  'IF_2014': '1.361',\n",
              "  'IF_2015': '1.29',\n",
              "  'IF_2016': '1.514',\n",
              "  'IF_2017': '1.421',\n",
              "  'ISSN': '0022-0736',\n",
              "  'ISSN_online': '1532-8430',\n",
              "  'ISSN_print': '0022-0736',\n",
              "  'IsoAbbr': 'J Electrocardiol',\n",
              "  'JrId': 4710,\n",
              "  'MedAbbr': 'J Electrocardiol',\n",
              "  'NLMid': '0153605',\n",
              "  'Titre': 'J Electrocardiol',\n",
              "  'abstract': 'Recent studies have shown that younger women are more likely to die during and after hospitalization for acute myocardial infarction (MI) than older women and men of all ages. This may be partly due to incorrect diagnosis or late detection of acute MI in younger women. At high specificity levels (>98%), the sensitivity of the initial ECG to detect acute MI may be as low as 30% when using traditional criteria by both physicians and computerized interpretation programs. This study examines if women of different age groups have a similar ECG presentation to men during acute inferior MI and if the diagnostic accuracies of the initial ECG are comparable. We analyzed chest pain ECGs from Mayo Clinic and Medical College of Wisconsin, which included 1,339 patients with acute inferior MI and 1,169 age-matched controls with noncardiac chest pain. We subdivided all groups by age (below and above 60 years) and compared ECG parameters (ST elevation, ST depression, QRS duration, R-wave amplitude, Q-wave duration and amplitude, QT interval) between genders. For inferior MI patients under age 60, women had lower ST elevations at the J point in lead II than men (57 +/- 91 microV vs. 86 +/- 117 microV, P < .02). This trend was reversed for patients over age 60 (lead a VF: 102 +/- 126 microV vs. 84+/-117 microV, P < .04; Lead III: 130+/-146 microV vs. 103+/-131 microV, P < .007). A neural network method was used to identify the most significant group of ECG parameters for detecting acute MI. An adaptive fuzzy logic method was developed for adapting to the threshold differences among the different gender and age groups. The new algorithm improved the sensitivity of acute inferior MI detection by more than 25% relative to old algorithm, while maintaining the high specificity around 98% for noncardiac chest pain patients.',\n",
              "  'authors': ['Xue, J',\n",
              "   'Taha, B',\n",
              "   'Reddy, S',\n",
              "   'Wright, R S',\n",
              "   'Aufderheide, T'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Journal of electrocardiology',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Aged',\n",
              "   'Algorithms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electrocardiography',\n",
              "   'Female',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Myocardial Infarction',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sex Factors'],\n",
              "  'pmid': '11781961',\n",
              "  'title': 'A new method to incorporate age and gender into the criteria for the detection of acute inferior myocardial infarction.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'With the development of large-scale knowledge bases (KBs), knowledge-based question answering (KBQA) has become an important research topic in recent years. The key task in KBQA is relation detection, which is the process of finding a compatible answer type for a natural language question and generating its corresponding structured query over a KB. However, existing systems often rely on shallow probabilistic methods, which are less expressive than deep semantic representation methods. In addition, since KBs are still far from complete, it is necessary to develop a new strategy that leverages unstructured resources outside of KBs. In this work, we propose a novel Question Answering method with Relation Detection and Textual Evidence (QARDTE). First, to address the semantic gap problem in relation detection, we use bidirectional long-short term memory networks with different levels of abstraction to better capture sentence structures. Our model achieves improved results with robustness against a wide diversity of expressions and questions with multiple relations. Moreover, to help compensate for the incompleteness of KBs, we utilize external unstructured text to extract additional supporting evidence and combine this evidence with relation information during the answer re-ranking process. In experiments on two well-known benchmarks, our system achieves F1 values of 0.558 (+2.8%) and 0.663 (+5.7%), which are state-of-the-art results that show significant improvement over existing KBQA systems.',\n",
              "  'authors': ['Zheng, Hai-Tao',\n",
              "   'Fu, Zuo-You',\n",
              "   'Chen, Jin-Yuan',\n",
              "   'Sangaiah, Arun Kumar',\n",
              "   'Jiang, Yong',\n",
              "   'Zhao, Cong-Zhi'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Humans',\n",
              "   'Knowledge Bases',\n",
              "   'Natural Language Processing',\n",
              "   'Quality Improvement',\n",
              "   'Semantics'],\n",
              "  'pmid': '30281661',\n",
              "  'title': 'Novel knowledge-based system with relation detection and textual evidence for question answering research.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Detection of dysmorphic cells in peripheral blood (PB) smears is essential in diagnostic screening of hematological diseases. Myelodysplastic syndromes (MDS) are hematopoietic neoplasms characterized by dysplastic and ineffective hematopoiesis, which diagnosis is mainly based on morphological findings of PB and bone marrow. We developed an automated diagnostic support system of MDS by combining an automated blood cell image-recognition system using a deep learning system (DLS) powered by convolutional neural networks (CNNs) with a decision-making system using extreme gradient boosting (XGBoost). The DLS of blood cell image-recognition has been trained using datasets consisting of 695,030 blood cell images taken from 3,261 PB smears including hematopoietic malignancies. The DLS simultaneously classified 17 blood cell types and 97 morphological features of such cells with >93.5% sensitivity and >96.0% specificity. The automated MDS diagnostic system successfully differentiated MDS from aplastic anemia (AA) with high accuracy; 96.2% of sensitivity and 100% of specificity (AUC 0.990). This is the first CNN-based automated initial diagnostic system for MDS using PB smears, which is applicable to develop new automated diagnostic systems for various hematological disorders.',\n",
              "  'authors': ['Kimura, Konobu',\n",
              "   'Tabe, Yoko',\n",
              "   'Ai, Tomohiko',\n",
              "   'Takehara, Ikki',\n",
              "   'Fukuda, Hiroshi',\n",
              "   'Takahashi, Hiromizu',\n",
              "   'Naito, Toshio',\n",
              "   'Komatsu, Norio',\n",
              "   'Uchihashi, Kinya',\n",
              "   'Ohsaka, Akimichi'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31527646',\n",
              "  'title': 'A novel automated image analysis system using deep convolutional neural networks can assist to differentiate MDS and AA.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': \"The daVinci Robot system has been widely lauded for its improved ergonomic characteristics when compared with the pure laparoscopic technique. Our goal in this study was to assess for the existence of guidelines to maximize the ergonomic benefits of the daVinci system. We also compared the surgeon's console with the recommendations of similar workstations.|A literature review of seated ergonomics was performed to identify recommendations for work areas similar to the robotic system, where prolong seating is necessary. An upper body biomechanics and ergonomic expert was consulted to evaluate the daVinci system and aid in the formation of ergonomic positioning guidelines. Link-length proportions were used to evaluate size constraints of potential robot operators.|No published guidelines exist for proper positioning using the daVinci surgeon console. There are, however, several Occupational Safety and Health Administration workstation guidelines as well as microscope ergonomic guidelines. The use of link-length proportions showed that the surgeon console allows a comfortable posture for individuals with height between 64 and 73 inches. Review of the microscope ergonomics literature indicates that a neutral vertical seating position has been associated with decreased strain and musculoskeletal disorders.|The body mechanics of the daVinci robot system best mimics that of microscopy. Future surgeon console modifications could emulate those reported in the microscope ergonomic literature, where a neutral vertical position has been recommended. This may help avoid potential musculoskeletal disorders similar to those previously seen with microscopy usage. Guidelines are suggested to optimize the surgeon's console position.\",\n",
              "  'authors': ['Lux, Matthew M',\n",
              "   'Marshall, Matthew',\n",
              "   'Erturk, Erdal',\n",
              "   'Joseph, Jean V'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Ergonomics',\n",
              "   'Female',\n",
              "   'Guidelines as Topic',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Robotics'],\n",
              "  'pmid': '20073561',\n",
              "  'title': 'Ergonomic evaluation and guidelines for use of the daVinci Robot system.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RX'],\n",
              "  'ESSN': '2213-1582',\n",
              "  'IF_2013': '2.526',\n",
              "  'IF_2014': '2.526',\n",
              "  'IF_2015': '3.857',\n",
              "  'IF_2016': '4.348',\n",
              "  'IF_2017': '3.869',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2213-1582',\n",
              "  'IsoAbbr': 'Neuroimage Clin',\n",
              "  'JrId': 41143,\n",
              "  'MedAbbr': 'Neuroimage Clin',\n",
              "  'NLMid': '101597070',\n",
              "  'Titre': 'Neuroimage Clin',\n",
              "  'abstract': 'To inform an understanding of brain status in HIV infection, quantitative imaging measurements were derived at structural, microstructural and macromolecular levels in three different periods of early infection and then analyzed simultaneously at each stage using data mining. Support vector machine recursive feature elimination was then used for simultaneous analysis of subject characteristics, clinical and behavioral variables, and immunologic measures in plasma and CSF to rank features associated with the most discriminating brain alterations in each period. The results indicate alterations beginning in initial infection and in all periods studied. The severity of immunosuppression in the initial virus host interaction was the most highly ranked determinant of earliest brain alterations. These results shed light on the initial brain changes induced by a neurotropic virus and their subsequent evolution. The pattern of ongoing alterations occurring during and beyond the period in which virus is suppressed in the systemic circulation supports the brain as a viral reservoir that may preclude eradication in the host. Data mining capabilities that can address high dimensionality and simultaneous analysis of disparate information sources have considerable utility for identifying mechanisms underlying onset of neurological injury and for informing new therapeutic targets. ',\n",
              "  'authors': ['Cao, Bokai',\n",
              "   'Kong, Xiangnan',\n",
              "   'Kettering, Casey',\n",
              "   'Yu, Philip',\n",
              "   'Ragin, Ann'],\n",
              "  'categories': ['NEUROIMAGING'],\n",
              "  'journal': 'NeuroImage. Clinical',\n",
              "  'keywords': ['Adult',\n",
              "   'Brain',\n",
              "   'Cytokines',\n",
              "   'Data Mining',\n",
              "   'Disease Progression',\n",
              "   'Female',\n",
              "   'HIV Infections',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Severity of Illness Index',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '26413474',\n",
              "  'title': 'Determinants of HIV-induced brain changes in three different periods of the early clinical course: A data mining analysis.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1873-7560',\n",
              "  'IF_2013': '12.48',\n",
              "  'IF_2014': '13.938',\n",
              "  'IF_2015': '14.976',\n",
              "  'IF_2016': '16.265',\n",
              "  'IF_2017': '17.581',\n",
              "  'ISSN': '0302-2838',\n",
              "  'ISSN_online': '1873-7560',\n",
              "  'ISSN_print': '0302-2838',\n",
              "  'IsoAbbr': 'Eur. Urol.',\n",
              "  'JrId': 3630,\n",
              "  'MedAbbr': 'Eur Urol',\n",
              "  'NLMid': '7512719',\n",
              "  'Titre': 'Eur. Urol.',\n",
              "  'abstract': \"Robot-assisted laparoscopic prostatectomy (RALP) has been disseminated widely, changing the knowledge of surgical anatomy of the prostate. The aim of our study is to demonstrate the feasibility of a new, purely intrafascial approach. The Bocciardi approach for RALP passes through the Douglas space, following a completely intrafascial plane without any dissection of the anterior compartment, which contains neurovascular bundles, Aphrodite's veil, endopelvic fascia, the Santorini plexus, pubourethral ligaments, and all of the structures thought to play a role in maintenance of continence and potency. In this case series, we present our first five patients undergoing the Bocciardi approach for RALP. We report the results of our technique in three patients following two unsuccessful attempts. No perioperative major complication was recorded. Pathologic stage was pT2c in two patients and pT2a in one patient, with no positive surgical margin. The day after removing the catheter, two of the three patients reported use of a single, small safety pad, and one patient was discharged without any pad. One patient reported an erection the day after removing the catheter. The anatomic rationale for better results compared with traditional RALP is strong, but well-designed studies are needed to evaluate the advantages of our technique.\",\n",
              "  'authors': ['Galfano, Antonio',\n",
              "   'Ascione, Assunta',\n",
              "   'Grimaldi, Salvatore',\n",
              "   'Petralia, Giovanni',\n",
              "   'Strada, Elena',\n",
              "   'Bocciardi, Aldo Massimo'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'European urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Feasibility Studies',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics'],\n",
              "  'pmid': '20566236',\n",
              "  'title': 'A new anatomic approach for robot-assisted laparoscopic prostatectomy: a feasibility study for completely intrafascial surgery.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Brain computer interfaces (BCIs) have been developed and implemented in many areas as a new communication channel between the human brain and external devices. Despite their rapid growth and broad popularity, the inaccurate performance and cost of user-training are yet the main issues that prevent their application out of the research and clinical environment. We previously introduced a BCI system for the control of a very humanlike android that could raise a sense of embodiment and agency in the operators only by imagining a movement (motor imagery) and watching the robot perform it. Also using the same setup, we further discovered that the positive bias of subjects' performance both increased their sensation of embodiment and improved their motor imagery skills in a short period. In this work, we studied the shared mechanism between the experience of embodiment and motor imagery. We compared the trend of motor imagery learning when two groups of subjects BCI-operated different looking robots, a very humanlike android's hands and a pair of metallic gripper. Although our experiments did not show a significant change of learning between the two groups immediately during one session, the android group revealed better motor imagery skills in the follow up session when both groups repeated the task using the non-humanlike gripper. This result shows that motor imagery skills learnt during the BCI-operation of humanlike hands are more robust to time and visual feedback changes. We discuss the role of embodiment and mirror neuron system in such outcome and propose the application of androids for efficient BCI training. \",\n",
              "  'authors': ['Alimardani, Maryam', 'Nishio, Shuichi', 'Ishiguro, Hiroshi'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Algorithms',\n",
              "   'Brain',\n",
              "   'Brain-Computer Interfaces',\n",
              "   'Feedback, Sensory',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Imagination',\n",
              "   'Learning',\n",
              "   'Male',\n",
              "   'Motor Skills',\n",
              "   'Movement',\n",
              "   'Robotics'],\n",
              "  'pmid': '27598310',\n",
              "  'title': 'The Importance of Visual Feedback Design in BCIs; from Embodiment to Motor Imagery Learning.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': \"Suspected fractures are among the most common reasons for patients to visit emergency departments (EDs), and X-ray imaging is the primary diagnostic tool used by clinicians to assess patients for fractures. Missing a fracture in a radiograph often has severe consequences for patients, resulting in delayed treatment and poor recovery of function. Nevertheless, radiographs in emergency settings are often read out of necessity by emergency medicine clinicians who lack subspecialized expertise in orthopedics, and misdiagnosed fractures account for upward of four of every five reported diagnostic errors in certain EDs. In this work, we developed a deep neural network to detect and localize fractures in radiographs. We trained it to accurately emulate the expertise of 18 senior subspecialized orthopedic surgeons by having them annotate 135,409 radiographs. We then ran a controlled experiment with emergency medicine clinicians to evaluate their ability to detect fractures in wrist radiographs with and without the assistance of the deep learning model. The average clinician's sensitivity was 80.8% (95% CI, 76.7-84.1%) unaided and 91.5% (95% CI, 89.3-92.9%) aided, and specificity was 87.5% (95 CI, 85.3-89.5%) unaided and 93.9% (95% CI, 92.9-94.9%) aided. The average clinician experienced a relative reduction in misinterpretation rate of 47.0% (95% CI, 37.4-53.9%). The significant improvements in diagnostic accuracy that we observed in this study show that deep learning methods are a mechanism by which senior medical specialists can deliver their expertise to generalists on the front lines of medicine, thereby providing substantial improvements to patient care.\",\n",
              "  'authors': ['Lindsey, Robert',\n",
              "   'Daluiski, Aaron',\n",
              "   'Chopra, Sumit',\n",
              "   'Lachapelle, Alexander',\n",
              "   'Mozer, Michael',\n",
              "   'Sicular, Serge',\n",
              "   'Hanel, Douglas',\n",
              "   'Gardner, Michael',\n",
              "   'Gupta, Anurag',\n",
              "   'Hotchkiss, Robert',\n",
              "   'Potter, Hollis'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Deep Learning',\n",
              "   'Diagnostic Errors',\n",
              "   'Emergency Medicine',\n",
              "   'Fractures, Bone',\n",
              "   'Hospital Rapid Response Team',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Radiography',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Wrist'],\n",
              "  'pmid': '30348771',\n",
              "  'title': 'Deep neural network improves fracture detection by clinicians.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1940-087X',\n",
              "  'IF_2013': '1.325',\n",
              "  'IF_2014': '1.325',\n",
              "  'IF_2015': '1.113',\n",
              "  'IF_2016': '1.232',\n",
              "  'IF_2017': '1.184',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1940-087X',\n",
              "  'IsoAbbr': 'J Vis Exp',\n",
              "  'JrId': 34584,\n",
              "  'MedAbbr': 'J Vis Exp',\n",
              "  'NLMid': '101313252',\n",
              "  'Titre': 'J Vis Exp',\n",
              "  'abstract': 'Skilled reaching tasks are commonly used in studies of motor skill learning and motor function under healthy and pathological conditions, but can be time-intensive and ambiguous to quantify beyond simple success rates. Here, we describe the training procedure for reach-and-pull tasks with ETH Pattus, a robotic platform for automated forelimb reaching training that records pulling and hand rotation movements in rats. Kinematic quantification of the performed pulling attempts reveals the presence of distinct temporal profiles of movement parameters such as pulling velocity, spatial variability of the pulling trajectory, deviation from midline, as well as pulling success. We show how minor adjustments in the training paradigm result in alterations in these parameters, revealing their relation to task difficulty, general motor function or skilled task execution. Combined with electrophysiological, pharmacological and optogenetic techniques, this paradigm can be used to explore the mechanisms underlying motor learning and memory formation, as well as loss and recovery of function (e.g. after stroke).',\n",
              "  'authors': ['Leemburg, Susan',\n",
              "   'Iijima, Maiko',\n",
              "   'Lambercy, Olivier',\n",
              "   'Nallet-Khosrofian, Lauriane',\n",
              "   'Gassert, Roger',\n",
              "   'Luft, Andreas'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Journal of visualized experiments : JoVE',\n",
              "  'keywords': ['Animals',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Forelimb',\n",
              "   'Learning',\n",
              "   'Male',\n",
              "   'Motor Cortex',\n",
              "   'Motor Skills',\n",
              "   'Movement',\n",
              "   'Psychomotor Performance',\n",
              "   'Rats',\n",
              "   'Rats, Long-Evans',\n",
              "   'Robotic Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Stroke'],\n",
              "  'pmid': '28287570',\n",
              "  'title': 'Investigating Motor Skill Learning Processes with a Robotic Manipulandum.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Lysine acetylation is a reversible, dynamic protein modification regulated by lysine acetyltransferases and deacetylases. Recent advances in high-throughput proteomics have greatly contributed to the success of global analysis of lysine acetylation. A large number of proteins of diverse biological functions have been shown to be acetylated in several reports in human cells, E.coli, and dicot plants. However, the extent of lysine acetylation in non-histone proteins remains largely unknown in monocots, particularly in the cereal crops. Here we report the mass spectrometric examination of lysine acetylation in rice (Oryza sativa). We identified 60 lysine acetylated sites on 44 proteins of diverse biological functions. Immunoblot studies further validated the presence of a large number of acetylated non-histone proteins. Examination of the amino acid composition revealed substantial amino acid bias around the acetylation sites and the amino acid preference is conserved among different organisms. Gene ontology analysis demonstrates that lysine acetylation occurs in diverse cytoplasmic, chloroplast and mitochondrial proteins in addition to the histone modifications. Our results suggest that lysine acetylation might constitute a regulatory mechanism for many proteins, including both histones and non-histone proteins of diverse biological functions. ',\n",
              "  'authors': ['Nallamilli, Babi Ramesh Reddy',\n",
              "   'Edelmann, Mariola J',\n",
              "   'Zhong, Xiaoxian',\n",
              "   'Tan, Feng',\n",
              "   'Mujahid, Hana',\n",
              "   'Zhang, Jian',\n",
              "   'Nanduri, Bindu',\n",
              "   'Peng, Zhaohua'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Acetylation',\n",
              "   'Amino Acid Sequence',\n",
              "   'Biological Phenomena',\n",
              "   'Blotting, Western',\n",
              "   'Chromatography, Affinity',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Lysine',\n",
              "   'Molecular Sequence Data',\n",
              "   'Oryza',\n",
              "   'Plant Proteins',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'Proteome',\n",
              "   'Proteomics',\n",
              "   'Tandem Mass Spectrometry'],\n",
              "  'pmid': '24586658',\n",
              "  'title': 'Global analysis of lysine acetylation suggests the involvement of protein acetylation in diverse biological processes in rice (Oryza sativa).',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': \"Piwi-interacting RNAs (piRNAs) were recently discovered as endogenous small noncoding RNAs. Some recent research suggests that piRNAs may play an important role in cancer. So the precise identification of human piRNAs is a significant work. In this paper, we introduce a series of new features with 80 dimension called short sequence motifs (SSM). A hybrid feature vector with 1444 dimension can be formed by combining 1364 features of [Formula: see text]-mer strings and 80 features of SSM features. We optimize the 1444 dimension features using the feature score criterion (FSC) and list them in descending order according to the scores. The first 462 are selected as the input feature vector in the classifier. Moreover, eight of 80 SSM features appear in the top 20. This indicates that these eight SSM features play an important part in the identification of piRNAs. Since five of the above eight SSM features are associated with nucleotide A and G ('A*G', 'A**G', 'A***G', 'A****G', 'A*****G'). So, we guess there may exist some biological significance. We also use a neural network algorithm called voting-based extreme learning machine (V-ELM) to identify real piRNAs. The Specificity (Sp) and Sensitivity (Sn) of our method are 95.48% and 94.61%, respectively in human species. This result shows that our method is more effective compared with those of the piRPred, piRNApredictor, Asym-Pibomd, Piano and McRUMs. The web service of V-ELMpiRNAPred is available for free at http://mm20132014.wicp.net:38601/velmprepiRNA/Main.jsp .\",\n",
              "  'authors': ['Pian, Cong',\n",
              "   'Chen, Yuan-Yuan',\n",
              "   'Zhang, Jin',\n",
              "   'Chen, Zhi',\n",
              "   'Zhang, Guang-Le',\n",
              "   'Li, Qiang',\n",
              "   'Yang, Tao',\n",
              "   'Zhang, Liang-Yun'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Databases, Genetic',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Nucleotide Motifs',\n",
              "   'RNA, Small Interfering'],\n",
              "  'pmid': '28178889',\n",
              "  'title': 'V-ELMpiRNAPred: Identification of human piRNAs by the voting-based extreme learning machine (V-ELM) with a new hybrid feature.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': 'The authors evaluated the effect of optimal neural network architecture selection on the performance of a computer-aided diagnostic system designed to detect microcalcification clusters on digitized mammograms.|The authors developed a computer program to detect microcalcification clusters automatically on digitized mammograms. Previously, they found that a properly selected and trained convolution neural network (CNN) could reduce false-positive (FP) findings and therefore improve the accuracy of microcalcification detection. In the current study, they evaluated the effectiveness of the CNN optimized with an automated optimization technique in improving the accuracy of the microcalcification detection program, comparing it with the manually selected CNN. An independent test data set was used, which included 472 mammograms selected from the University of South Florida public database and contained 253 biopsy-proved malignant clusters.|At an FP rate of 0.7 cluster per image, the film-based sensitivity was 84.6% for the optimized CNN, compared with 77.2% for the manually selected CNN. For clusters imaged on both craniocaudal and mediolateral oblique views, a cluster could be considered detected when it was detected on one or both views. For this case-based approach, at an FP rate of 0.7 per image, the sensitivity was 93.3% for the optimized and 87.0% for the manually selected CNN.|The classification of true and false signals is an important step in the microcalcification detection program. An optimized CNN can effectively reduce FP findings and improve the accuracy of the computer-aided detection system.',\n",
              "  'authors': ['Gurcan, Metin N',\n",
              "   'Chan, Heang-Ping',\n",
              "   'Sahiner, Berkman',\n",
              "   'Hadjiiski, Lubomir',\n",
              "   'Petrick, Nicholas',\n",
              "   'Helvie, Mark A'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Breast Neoplasms',\n",
              "   'Calcinosis',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Mammography',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '11942656',\n",
              "  'title': 'Optimal neural network architecture selection: improvement in computerized detection of microcalcifications.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1752-0509',\n",
              "  'IF_2013': '2.853',\n",
              "  'IF_2014': '2.435',\n",
              "  'IF_2015': '2.213',\n",
              "  'IF_2016': '2.303',\n",
              "  'IF_2017': '2.05',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1752-0509',\n",
              "  'IsoAbbr': 'BMC Syst Biol',\n",
              "  'JrId': 33336,\n",
              "  'MedAbbr': 'BMC Syst Biol',\n",
              "  'NLMid': '101301827',\n",
              "  'Titre': 'BMC Syst Biol',\n",
              "  'abstract': 'The fungal pathogen Fusarium graminearum (telomorph Gibberella zeae) is the causal agent of several destructive crop diseases, where a set of genes usually work in concert to cause diseases to crops. To function appropriately, the F. graminearum proteins inside one cell should be assigned to different compartments, i.e. subcellular localizations. Therefore, the subcellular localizations of F. graminearum proteins can provide insights into protein functions and pathogenic mechanisms of this destructive pathogen fungus. Unfortunately, there are no subcellular localization information for F. graminearum proteins available now. Computational approaches provide an alternative way to predicting F. graminearum protein subcellular localizations due to the expensive and time-consuming biological experiments in lab.|In this paper, we developed a novel predictor, namely FGsub, to predict F. graminearum protein subcellular localizations from the primary structures. First, a non-redundant fungi data set with subcellular localization annotation is collected from UniProtKB database and used as training set, where the subcellular locations are classified into 10 groups. Subsequently, Support Vector Machine (SVM) is trained on the training set and used to predict F. graminearum protein subcellular localizations for those proteins that do not have significant sequence similarity to those in training set. The performance of SVMs on training set with 10-fold cross-validation demonstrates the efficiency and effectiveness of the proposed method. In addition, for F. graminearum proteins that have significant sequence similarity to those in training set, BLAST is utilized to transfer annotations of homologous proteins to uncharacterized F. graminearum proteins so that the F. graminearum proteins are annotated more comprehensively.|In this work, we present FGsub to predict F. graminearum protein subcellular localizations in a comprehensive manner. We make four fold contributions to this filed. First, we present a new algorithm to cope with imbalance problem that arises in protein subcellular localization prediction, which can solve imbalance problem and avoid false positive results. Second, we design an ensemble classifier which employs feature selection to further improve prediction accuracy. Third, we use BLAST to complement machine learning based methods, which enlarges our prediction coverage. Last and most important, we predict the subcellular localizations of 12786 F. graminearum proteins, which provide insights into protein functions and pathogenic mechanisms of this destructive pathogen fungus.',\n",
              "  'authors': ['Sun, Chenglei',\n",
              "   'Zhao, Xing-Ming',\n",
              "   'Tang, Weihua',\n",
              "   'Chen, Luonan'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'BMC systems biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'Fungal Proteins',\n",
              "   'Fusarium',\n",
              "   'Protein Folding',\n",
              "   'Protein Interaction Domains and Motifs',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Software Validation'],\n",
              "  'pmid': '20840726',\n",
              "  'title': 'FGsub: Fusarium graminearum protein subcellular localizations predicted from primary structures.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'Searching for a target object in a cluttered scene constitutes a fundamental challenge in daily vision. Visual search must be selective enough to discriminate the target from distractors, invariant to changes in the appearance of the target, efficient to avoid exhaustive exploration of the image, and must generalize to locate novel target objects with zero-shot training. Previous work on visual search has focused on searching for perfect matches of a target after extensive category-specific training. Here, we show for the first time that humans can efficiently and invariantly search for natural objects in complex scenes. To gain insight into the mechanisms that guide visual search, we propose a biologically inspired computational model that can locate targets without exhaustive sampling and which can generalize to novel objects. The model provides an approximation to the mechanisms integrating bottom-up and top-down signals during search in natural scenes.',\n",
              "  'authors': ['Zhang, Mengmi',\n",
              "   'Feng, Jiashi',\n",
              "   'Ma, Keng Teck',\n",
              "   'Lim, Joo Hwee',\n",
              "   'Zhao, Qi',\n",
              "   'Kreiman, Gabriel'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': ['Adult',\n",
              "   'Attention',\n",
              "   'Computer Simulation',\n",
              "   'Cues',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Visual',\n",
              "   'Psychophysics',\n",
              "   'Reaction Time',\n",
              "   'Time Factors',\n",
              "   'Vision, Ocular',\n",
              "   'Visual Perception',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30213937',\n",
              "  'title': 'Finding any Waldo with zero-shot invariant and efficient visual search.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['SA'],\n",
              "  'ESSN': '1878-3570',\n",
              "  'IF_2013': '3.922',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0002-8223',\n",
              "  'ISSN_online': '1878-3570',\n",
              "  'ISSN_print': '0002-8223',\n",
              "  'IsoAbbr': 'J Am Diet Assoc',\n",
              "  'JrId': 4442,\n",
              "  'MedAbbr': 'J Am Diet Assoc',\n",
              "  'NLMid': '7503061',\n",
              "  'Titre': 'J Am Diet Assoc',\n",
              "  'abstract': \"Planning nutritious and appetizing menus is a complex task that researchers have tried to computerize since the early 1960s. We have attempted to facilitate computer-assisted menu planning by modeling the reasoning an expert dietitian uses to plan menus. Two independent expert systems were built, each designed to plan a daily menu meeting the nutrition needs and personal preferences of an individual client. One system modeled rule-based, or logical, reasoning, whereas the other modeled case-based, or experiential, reasoning. The 2 systems were evaluated and their strengths and weaknesses identified. A hybrid system was built, combining the best of both systems. The hybrid system represents an important step forward because it plans daily menus in accordance with a person's needs and preferences; the Reference Daily Intakes; the Dietary Guidelines for Americans; and accepted aesthetic standards for color, texture, temperature, taste, and variety. Additional work to expand the system's scope and to enhance the user interface will be needed to make it a practical tool. Our system framework could be applied to special-purpose menu planning for patients in medical settings or adapted for institutional use. We conclude that an artificial intelligence approach has practical use for computer-assisted menu planning.\",\n",
              "  'authors': ['Petot, G J', 'Marling, C', 'Sterling, L'],\n",
              "  'categories': ['NUTRITION & DIETETICS'],\n",
              "  'journal': 'Journal of the American Dietetic Association',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Decision Making, Computer-Assisted',\n",
              "   'Evaluation Studies as Topic',\n",
              "   'Humans',\n",
              "   'Menu Planning'],\n",
              "  'pmid': '9739801',\n",
              "  'title': 'An artificial intelligence system for computer-assisted menu planning.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IA'],\n",
              "  'ESSN': '1095-6840',\n",
              "  'IF_2013': '2.674',\n",
              "  'IF_2014': '2.47',\n",
              "  'IF_2015': '2.667',\n",
              "  'IF_2016': '2.585',\n",
              "  'IF_2017': '2.564',\n",
              "  'ISSN': '0016-6480',\n",
              "  'ISSN_online': '1095-6840',\n",
              "  'ISSN_print': '0016-6480',\n",
              "  'IsoAbbr': 'Gen. Comp. Endocrinol.',\n",
              "  'JrId': 3857,\n",
              "  'MedAbbr': 'Gen Comp Endocrinol',\n",
              "  'NLMid': '0370735',\n",
              "  'Titre': 'Gen. Comp. Endocrinol.',\n",
              "  'abstract': \"The integration of various fields of investigation is of key importance to fully comprehending endocrine function. Here, I enact the theoretical framework of Nikolaas Tinbergen's four questions for understanding behavior to help bridge the wide gap that exists between our relatively reductionist molecular knowledge of a particular neurohormone, gonadotropin-inhibitory hormone (GnIH), and its place in animal behavior. Hypothalamic GnIH, upon its discovery in 2000, was so named because of its inhibitory effect on the release of the gonadotropins, luteinizing hormone (LH) and follicle stimulating hormone (FSH), from the pituitary. Because gonadotropins are necessary for reproduction, this finding stimulated questions about the functional significance of GnIH in reproduction and sexual behavior. After over a decade of research, invaluable knowledge has been gained regarding the mechanistic attributes of GnIH (mammalian homolog, RFamide-related peptide (RFRP)) in a variety of vertebrate species. However, many questions remain regarding the effect of the environment on GnIH and the subsequent effects of GnIH on behavior. I review the role of GnIH in shaping behavior using the framework of Tinbergen's four questions of mechanism, ontogeny, function and phylogeny. The studies I review were conducted in various species of mammals, birds, and in one species of fish. Because GnIH can play a role in mediating behaviors such as those important for reproduction, sociality, feeding, and the stress response in a variety of species, an integrative approach to the study of GnIH will help provide a multipronged schema for answering questions of GnIH function. By using the framework highlighted by Tinbergen's four questions, we will deepen and enhance our knowledge of the role of hormones in behavior from the point of view of the mechanisms involved. \",\n",
              "  'authors': ['Calisi, Rebecca M'],\n",
              "  'categories': ['ENDOCRINOLOGY & METABOLISM'],\n",
              "  'journal': 'General and comparative endocrinology',\n",
              "  'keywords': ['Animals',\n",
              "   'Behavior, Animal',\n",
              "   'Biological Ontologies',\n",
              "   'Gonadotropins',\n",
              "   'Hypothalamic Hormones',\n",
              "   'Neurosecretory Systems',\n",
              "   'Phylogeny',\n",
              "   'Reproduction',\n",
              "   'Vertebrates'],\n",
              "  'pmid': '24704003',\n",
              "  'title': \"An integrative overview of the role of gonadotropin-inhibitory hormone in behavior: applying Tinbergen's four questions.\",\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Decoding continuous hind limb joint angles from sensory recordings of neural system provides a feedback for closed-loop control of hind limb movement using functional electrical stimulation. So far, many attempts have been done to extract sensory information from dorsal root ganglia and sensory nerves. In this work, we examine decoding joint angles trajectories from the single-electrode extracellular recording of dorsal horn gray matter of the spinal cord during passive limb movement in anesthetized cats. In this study, a processing framework based on ensemble learning approach is propose to combine firing rate (FR) and interspike interval (ISI) information of the neuronal activity. For this purpose, a stacked generalization approach based on recurrent neural network is proposed to enhance decoding accuracy of the movement kinematics. The results show that the high precision neural decoding of limb movement can be achieved even with a single electrode implanted in the spinal cord gray matter.',\n",
              "  'authors': ['Yeganegi, Hamed', 'Fathi, Yaser', 'Erfanian, Abbas'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Cats',\n",
              "   'Electric Stimulation',\n",
              "   'Gray Matter',\n",
              "   'Hindlimb',\n",
              "   'Movement',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Posterior Horn Cells'],\n",
              "  'pmid': '29330489',\n",
              "  'title': 'Decoding hind limb kinematics from neuronal activity of the dorsal horn neurons using multiple level learning algorithm.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['GA'],\n",
              "  'ESSN': '1600-0846',\n",
              "  'IF_2013': '1.536',\n",
              "  'IF_2014': '1.309',\n",
              "  'IF_2015': '1.776',\n",
              "  'IF_2016': '1.662',\n",
              "  'IF_2017': '1.489',\n",
              "  'ISSN': '0909-752X',\n",
              "  'ISSN_online': '1600-0846',\n",
              "  'ISSN_print': '0909-752X',\n",
              "  'IsoAbbr': 'Skin Res Technol',\n",
              "  'JrId': 22357,\n",
              "  'MedAbbr': 'Skin Res Technol',\n",
              "  'NLMid': '9504453',\n",
              "  'Titre': 'Skin Res Technol',\n",
              "  'abstract': 'Computer-aided pattern classification of melanoma and other pigmented skin lesions is one of the most important tasks for clinical diagnosis. To differentiate between benign and malignant lesions, the extraction of color, architectural order, symmetry of pattern and homogeneity (CASH) is a challenging task.|In this article, a novel pattern classification system (PCS) based on the clinical CASH rule is presented to classify among six classes of patterns. The PCS system consists of the following five steps: transformation to the CIE L*a*b* color space, pre-processing to enhance the tumor region and removal of hairs, tumor-area segmentation, color and texture feature extraction, and finally, classification based on a multiclass support vector machine.|The PCS system is tested on a total of 180 dermoscopic images. To test the performance of the PCS diagnostic classifier, the area under the receiver operating characteristics curve (AUC) is utilized. The proposed classifier achieved a sensitivity of 91.64%, specificity of 94.14%, and AUC of 0.948.|The experimental results demonstrate that the proposed pattern classifier is highly accurate and classify between benign and malignant lesions into some extend. The PCS method is fully automatic and can accurately detect different patterns from dermoscopy images using color and texture properties. Additional pattern features can be included to investigate the impact of pattern classification based on the CASH rule.',\n",
              "  'authors': ['Abbas, Qaisar', 'Celebi, M Emre', 'Fondón, Irene'],\n",
              "  'categories': ['DERMATOLOGY'],\n",
              "  'journal': 'Skin research and technology : official journal of International Society for Bioengineering and the Skin (ISBS) [and] International Society for Digital Imaging of Skin (ISDIS) [and] International Society for Skin Imaging (ISSI)',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged, 80 and over',\n",
              "   'Artificial Intelligence',\n",
              "   'Dermoscopy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Skin Neoplasms'],\n",
              "  'pmid': '22093020',\n",
              "  'title': 'Computer-aided pattern classification system for dermoscopy images.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Von Hippel-Lindau disease is a heritable syndrome that confers an increased risk of developing various benign and malignant tumors to those with a germline mutation of the tumor suppressor gene. We present a case of a male patient who initially presented at age 9 with headaches, fevers, and fatigue. He was found to have a left pheochromocytoma which was successfully managed with open total adrenalectomy. He presented again at age 18 with a second pheochromocytoma in the right adrenal gland. DNA analysis revealed a de novo Val84Leu mutation in the Von Hippel-Lindau gene, not seen in either parent. The challenge presented was that of balancing the obvious benefits of cortical- sparing adrenalectomy with the risk of tumor recurrence in spared tissue. Ultimately, management consisted of a robot-assisted laparoscopic partial right adrenalectomy with successful preservation of adrenocortical function.',\n",
              "  'authors': ['St Julien, Jamii', 'Ball, Douglas', 'Schulick, Richard'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adrenal Cortex',\n",
              "   'Adrenal Gland Neoplasms',\n",
              "   'Adrenalectomy',\n",
              "   'Child',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Neoplasms, Second Primary',\n",
              "   'Pheochromocytoma',\n",
              "   'Robotics',\n",
              "   'Time Factors',\n",
              "   'von Hippel-Lindau Disease'],\n",
              "  'pmid': '17004871',\n",
              "  'title': 'Robot-assisted cortical-sparing adrenalectomy in a patient with Von Hippel-Lindau disease and bilateral pheochromocytomas separated by 9 years.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Based on the analysis and comparison of several annealing strategies, we present a flexible annealing chaotic neural network which has flexible controlling ability and quick convergence rate to optimization problem. The proposed network has rich and adjustable chaotic dynamics at the beginning, and then can converge quickly to stable states. We test the network on the maximum clique problem by some graphs of the DIMACS clique instances, p-random and k random graphs. The simulations show that the flexible annealing chaotic neural network can get satisfactory solutions at very little time and few steps. The comparison between our proposed network and other chaotic neural networks denotes that the proposed network has superior executive efficiency and better ability to get optimal or near-optimal solution.',\n",
              "  'authors': ['Yang, Gang', 'Tang, Zheng', 'Zhang, Zhiqiang', 'Zhu, Yunyi'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Mathematics',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '17640099',\n",
              "  'title': 'A flexible annealing chaotic neural network to maximum clique problem.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2323',\n",
              "  'IF_2013': '2.348',\n",
              "  'IF_2014': '2.642',\n",
              "  'IF_2015': '2.523',\n",
              "  'IF_2016': '2.673',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '0364-2313',\n",
              "  'ISSN_online': '1432-2323',\n",
              "  'ISSN_print': '0364-2313',\n",
              "  'IsoAbbr': 'World J Surg',\n",
              "  'JrId': 8075,\n",
              "  'MedAbbr': 'World J Surg',\n",
              "  'NLMid': '7704052',\n",
              "  'Titre': 'World J Surg',\n",
              "  'abstract': '[corrected] The purpose of the study was to assess the role of artificial neural networks (ANNs) in the diagnosis of appendicitis in patients presenting with acute right iliac fossa (RIF) pain and comparing its performance with the assessment made by experienced clinicians and the Alvarado score.|After training and testing an ANN, data from 60 patients presenting with suspected appendicitis over a 6-month period to a teaching hospital was collected prospectively. Accuracy of diagnosing appendicitis by the clinician, the Alvarado score, and the ANN was compared.|The sensitivity, specificity, and positive and negative predictive values of the ANN were 100%, 97.2%, 96.0%, and 100% respectively. The ability of the ANN to exclude accurately the diagnosis of appendicitis in patients without true appendicitis was statistically significant compared to the clinical performance (p=0.031) and Alvarado score of >or=6 (p=0.004) and nearly significant compared to the Alvarado score of >or=7 (p=0.063).|ANNs can be an effective tool for accurately diagnosing appendicitis and may reduce unnecessary appendectomies.',\n",
              "  'authors': ['Prabhudesai, S G',\n",
              "   'Gould, S',\n",
              "   'Rekhraj, S',\n",
              "   'Tekkis, P P',\n",
              "   'Glazer, G',\n",
              "   'Ziprin, P'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'World journal of surgery',\n",
              "  'keywords': ['Adult',\n",
              "   'Appendectomy',\n",
              "   'Appendicitis',\n",
              "   'Cohort Studies',\n",
              "   'Female',\n",
              "   'Flank Pain',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Sex Factors'],\n",
              "  'pmid': '18043966',\n",
              "  'title': 'Artificial neural networks: useful aid in diagnosing acute appendicitis.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '1873-3557',\n",
              "  'IF_2013': '2.129',\n",
              "  'IF_2014': '2.353',\n",
              "  'IF_2015': '2.653',\n",
              "  'IF_2016': '2.536',\n",
              "  'IF_2017': '2.88',\n",
              "  'ISSN': '1386-1425',\n",
              "  'ISSN_online': '1873-3557',\n",
              "  'ISSN_print': '1386-1425',\n",
              "  'IsoAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'JrId': 20555,\n",
              "  'MedAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'NLMid': '9602533',\n",
              "  'Titre': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'abstract': 'Spectroscopy is a powerful non-destructive quantization tool. In this paper, the technology is used to predict the nitrogen content of Masson pine seedling leaves. Masson pine is widely planted in China, and its nitrogen content is an important index for evaluating the vigour of seedings. To establish a better prediction model, an improved 1D convolutional neural network architecture, named the variable weighted convolutional neural network (VWCNN), is proposed in this research. The new model can automatically force the network attention onto the important spectrum wavelengths and is able to improve the generalization ability of the basic 1D-CNN model. For 219 fresh Masson pine seedling leaves, it shows better results in the prediction accuracy and robustness compared to those derived from the traditional shallow prediction model and other CNN-based models. VWCNN can achieve a 0.984 R2 and 0.038 RMSE value in training dataset and 0.925 R2 value and 0.075 RMSE value in the test dataset. The proposed model was also tested on a public corn kernel dataset. For the dataset output, moisture, oil, protein, and starch, the new model also achieves state-of-the-art prediction results.',\n",
              "  'authors': ['Ni, Chao', 'Wang, Dongyi', 'Tao, Yang'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy',\n",
              "  'keywords': ['China',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nitrogen',\n",
              "   'Pinus',\n",
              "   'Plant Leaves',\n",
              "   'Seedlings',\n",
              "   'Spectroscopy, Near-Infrared'],\n",
              "  'pmid': '30343107',\n",
              "  'title': 'Variable weighted convolutional neural network for the nitrogen content quantization of Masson pine seedling leaves with near-infrared spectroscopy.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Gentrification is multidimensional and complex, but there is general agreement that visible changes to neighbourhoods are a clear manifestation of the process. Recent advances in computer vision and deep learning provide a unique opportunity to support automated mapping or 'deep mapping' of perceptual environmental attributes. We present a Siamese convolutional neural network (SCNN) that automatically detects gentrification-like visual changes in temporal sequences of Google Street View (GSV) images. Our SCNN achieves 95.6% test accuracy and is subsequently applied to GSV sequences at 86110 individual properties over a 9-year period in Ottawa, Canada. We use Kernel Density Estimation (KDE) to produce maps that illustrate where the spatial concentration of visual property improvements was highest within the study area at different times from 2007-2016. We find strong concordance between the mapped SCNN results and the spatial distribution of building permits in the City of Ottawa from 2011 to 2016. Our mapped results confirm those urban areas that are known to be undergoing gentrification as well as revealing areas undergoing gentrification that were previously unknown. Our approach differs from previous works because we examine the atomic unit of gentrification, namely, the individual property, for visual property improvements over time and we rely on KDE to describe regions of high spatial intensity that are indicative of gentrification processes.\",\n",
              "  'authors': ['Ilic, Lazar', 'Sawada, M', 'Zarzelli, Amaury'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30865701',\n",
              "  'title': 'Deep mapping gentrification in a large Canadian city using deep learning and Google Street View.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1875-7855',\n",
              "  'IF_2013': '5.103',\n",
              "  'IF_2014': '2.833',\n",
              "  'IF_2015': '1.655',\n",
              "  'IF_2016': '2.68',\n",
              "  'IF_2017': '3.174',\n",
              "  'ISSN': '0079-6123',\n",
              "  'ISSN_online': '1875-7855',\n",
              "  'ISSN_print': '0079-6123',\n",
              "  'IsoAbbr': 'Prog. Brain Res.',\n",
              "  'JrId': 6691,\n",
              "  'MedAbbr': 'Prog Brain Res',\n",
              "  'NLMid': '0376441',\n",
              "  'Titre': 'Prog. Brain Res.',\n",
              "  'abstract': 'How the brain contends with naturalistic viewing conditions when it must cope with concurrent streams of diverse sensory inputs and internally generated thoughts is still largely an open question. In this study, we used fMRI to record brain activity while a group of 18 participants watched an edited dance duet accompanied by a soundtrack. After scanning, participants performed a short behavioral task to identify neural correlates of dance segments that could later be recalled. Intersubject correlation (ISC) analysis was used to identify the brain regions correlated among observers, and the results of this ISC map were used to define a set of regions for subsequent analysis of functional connectivity. The resulting network was found to be composed of eight subnetworks and the significance of these subnetworks is discussed. While most subnetworks could be explained by sensory and motor processes, two subnetworks appeared related more to complex cognition. These results inform our understanding of the neural basis of common experience in watching dance and open new directions for the study of complex cognition.',\n",
              "  'authors': ['Pollick, Frank E',\n",
              "   'Vicary, Staci',\n",
              "   'Noble, Katie',\n",
              "   'Kim, Naree',\n",
              "   'Jang, Seonhee',\n",
              "   'Stevens, Catherine J'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Progress in brain research',\n",
              "  'keywords': ['Adult',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Dancing',\n",
              "   'Female',\n",
              "   'Functional Laterality',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Linear Models',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Neuropsychological Tests',\n",
              "   'Oxygen',\n",
              "   'Photic Stimulation',\n",
              "   'Visual Perception',\n",
              "   'Young Adult'],\n",
              "  'pmid': '29779744',\n",
              "  'title': 'Exploring collective experience in watching dance through intersubject correlation and functional connectivity of fMRI brain activity.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Some pediatric surgeons may be reluctant to use robotic surgery for small patients because the only available surgical robot might seem too large for smaller patients. However, we have found this concern invalid. We have been successful in a wide variety of minimally invasive surgery procedures using robotics for general surgery applications in small children. However, several technical issues must be considered in order to optimize this technology for these children. In this article, we present a retrospective review of 45 patients of less than 10 kg who underwent robotic surgery and discuss the adjustments we made in order to adapt this technology to our small patients in a wide variety of general surgical procedures.',\n",
              "  'authors': ['Meehan, John J'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Body Weight',\n",
              "   'Child, Preschool',\n",
              "   'Digestive System Surgical Procedures',\n",
              "   'Fundoplication',\n",
              "   'Humans',\n",
              "   'Infant',\n",
              "   'Infant, Newborn',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Operating Rooms',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Thoracic Surgical Procedures'],\n",
              "  'pmid': '19694560',\n",
              "  'title': 'Robotic surgery in small children: is there room for this?',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': \"Nonsynonymous single nucleotide polymorphisms (nsSNPs) are prevalent in genomes and are closely associated with inherited diseases. To facilitate identifying disease-associated nsSNPs from a large number of neutral nsSNPs, it is important to develop computational tools to predict the nsSNP's phenotypic effect (disease-associated versus neutral). nsSNPAnalyzer, a web-based software developed for this purpose, extracts structural and evolutionary information from a query nsSNP and uses a machine learning method called Random Forest to predict the nsSNP's phenotypic effect. nsSNPAnalyzer server is available at http://snpanalyzer.utmem.edu/.\",\n",
              "  'authors': ['Bao, Lei', 'Zhou, Mi', 'Cui, Yan'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Phenotype',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Protein Conformation',\n",
              "   'Sequence Alignment',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Software',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '15980516',\n",
              "  'title': 'nsSNPAnalyzer: identifying disease-associated nonsynonymous single nucleotide polymorphisms.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1752-0509',\n",
              "  'IF_2013': '2.853',\n",
              "  'IF_2014': '2.435',\n",
              "  'IF_2015': '2.213',\n",
              "  'IF_2016': '2.303',\n",
              "  'IF_2017': '2.05',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1752-0509',\n",
              "  'IsoAbbr': 'BMC Syst Biol',\n",
              "  'JrId': 33336,\n",
              "  'MedAbbr': 'BMC Syst Biol',\n",
              "  'NLMid': '101301827',\n",
              "  'Titre': 'BMC Syst Biol',\n",
              "  'abstract': 'Translating in vitro results to clinical tests is a major challenge in systems biology. Here we present a new Multi-Task learning framework which integrates thousands of cell line expression experiments to reconstruct drug specific response networks in cancer.|The reconstructed networks correctly identify several shared key proteins and pathways while simultaneously highlighting many cell type specific proteins. We used top proteins from each drug network to predict survival for patients prescribed the drug.|Predictions based on proteins from the in-vitro derived networks significantly outperformed predictions based on known cancer genes indicating that Multi-Task learning can indeed identify accurate drug response networks.',\n",
              "  'authors': ['Ruffalo, Matthew',\n",
              "   'Stojanov, Petar',\n",
              "   'Pillutla, Venkata Krishna',\n",
              "   'Varma, Rohan',\n",
              "   'Bar-Joseph, Ziv'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'BMC systems biology',\n",
              "  'keywords': ['Antineoplastic Agents',\n",
              "   'Computational Biology',\n",
              "   'Machine Learning',\n",
              "   'Neoplasms',\n",
              "   'Survival Analysis',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '29017547',\n",
              "  'title': 'Reconstructing cancer drug response networks using multitask learning.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'This study aimed to compare the short- and long-term results of minimally invasive surgery (MIS) and open surgery for primary early gastric cancer (EGC) at a single high-volume institution.|The clinicopathologic and survival data of primary gastric cancer patients who underwent a minimally invasive radical gastrectomy at Seoul National University Hospital from December 2003 to January 2012 were retrospectively analyzed. For comparison of short-term outcomes, the data for 1,112 patients who underwent a radical open gastrectomy from 2007 to 2011 were collected. For long-term outcome analysis, the data for 962 patients who underwent a radical open gastrectomy from 2004 to 2006 were collected. Because the application of MIS was limited to suspected EGC, the control groups were similarly limited to patients deemed to have EGC as shown by preoperative endoscopy, endoscopic ultrasound, or both.|The review of our database identified 1,013 patients who had undergone MIS for gastric cancer. In the short-term outcome analysis, the MIS group showed statistically better results than the open surgery group in terms of postoperative hospital stay (8.7 vs. 11.3 days; p < 0.001), estimated blood loss (75.4 vs. 142.3 ml; p < 0.001), and overall complication rate (17.5 vs. 24.4 %; p < 0.001). In the subset analysis of total gastrectomy, the local complication rate was much higher in the MIS group than in the open surgery group. Both uni- and multivariate analyses showed that not only the surgical approach but also age, chronic liver disease, chronic renal disease, and additional organ resection had significant effects on complications. In the long-term outcome analysis, the two groups showed comparable disease-free survival rates.|The use of MIS for EGC showed a shorter operation time, a shorter postoperative hospital stay, and a lower overall complication rate than open surgery but a comparable disease-free survival rate. Total gastrectomy in the MIS group was associated with a higher complication rate than in the open group. Therefore, a new stable surgical technique needs to be established.',\n",
              "  'authors': ['Oh, Seung-Young',\n",
              "   'Kwon, Sebastianus',\n",
              "   'Lee, Kyung-Goo',\n",
              "   'Suh, Yun-Suhk',\n",
              "   'Choe, Hwi-Nyeong',\n",
              "   'Kong, Seong-Ho',\n",
              "   'Lee, Hyuk-Joon',\n",
              "   'Kim, Woo Ho',\n",
              "   'Yang, Han-Kwang'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Adult',\n",
              "   'Disease-Free Survival',\n",
              "   'Early Diagnosis',\n",
              "   'Endoscopy, Gastrointestinal',\n",
              "   'Endosonography',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Gastrectomy',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Neoplasm Staging',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Stomach Neoplasms',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24263458',\n",
              "  'title': 'Outcomes of minimally invasive surgery for early gastric cancer are comparable with those for open surgery: analysis of 1,013 minimally invasive surgeries at a single institution.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1755-8794',\n",
              "  'IF_2013': '3.914',\n",
              "  'IF_2014': '2.873',\n",
              "  'IF_2015': '2.726',\n",
              "  'IF_2016': '2.848',\n",
              "  'IF_2017': '3.317',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1755-8794',\n",
              "  'IsoAbbr': 'BMC Med Genomics',\n",
              "  'JrId': 34003,\n",
              "  'MedAbbr': 'BMC Med Genomics',\n",
              "  'NLMid': '101319628',\n",
              "  'Titre': 'BMC Med Genomics',\n",
              "  'abstract': 'Neurodegenerative diseases (NDs) are characterized by the progressive loss of neurons in the human brain. Although the majority of NDs are sporadic, evidence is accumulating that they have a strong genetic component. Therefore, significant efforts have been made in recent years to not only identify disease-causing genes but also genes that modify the severity of NDs, so-called genetic modifiers. To date there exists no compendium that lists and cross-links genetic modifiers of different NDs.|In order to address this need, we present NeuroGeM, the first comprehensive knowledgebase providing integrated information on genetic modifiers of nine different NDs in the model organisms D. melanogaster, C. elegans, and S. cerevisiae. NeuroGeM cross-links curated genetic modifier information from the different NDs and provides details on experimental conditions used for modifier identification, functional annotations, links to homologous proteins and color-coded protein-protein interaction networks to visualize modifier interactions. We demonstrate how this database can be used to generate new understanding through meta-analysis. For instance, we reveal that the Drosophila genes DnaJ-1, thread, Atx2, and mub are generic modifiers that affect multiple if not all NDs.|As the first compendium of genetic modifiers, NeuroGeM will assist experimental and computational scientists in their search for the pathophysiological mechanisms underlying NDs. http://chibi.ubc.ca/neurogem.',\n",
              "  'authors': ['Na, Dokyun',\n",
              "   'Rouf, Mushfiqur',\n",
              "   \"O'Kane, Cahir J\",\n",
              "   'Rubinsztein, David C',\n",
              "   'Gsponer, Jörg'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC medical genomics',\n",
              "  'keywords': ['Animals',\n",
              "   'Computational Biology',\n",
              "   'Data Mining',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Neurodegenerative Diseases'],\n",
              "  'pmid': '24229347',\n",
              "  'title': 'NeuroGeM, a knowledgebase of genetic modifiers in neurodegenerative diseases.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.197',\n",
              "  'IF_2014': '1.56',\n",
              "  'IF_2015': '1.76',\n",
              "  'IF_2016': '1.751',\n",
              "  'IF_2017': '1.666',\n",
              "  'ISSN': '0959-3330',\n",
              "  'ISSN_online': '1479-487X',\n",
              "  'ISSN_print': '0959-3330',\n",
              "  'IsoAbbr': 'Environ Technol',\n",
              "  'JrId': 22034,\n",
              "  'MedAbbr': 'Environ Technol',\n",
              "  'NLMid': '9884939',\n",
              "  'Titre': 'Environ Technol',\n",
              "  'abstract': \"Poly(acrylamide-co-diallyldimethylammonium chloride) (PDA), which is usually prepared by free radical polymerization of acrylamide monomer (AM) onto the cationic monomer dimethyl diallyl ammonium chloride (DMDAAC), has been widely applied to wastewater treatment; however, the free-radical polymerization is always incomplete with residual AM remaining in the PDA. The residual AM affects the PDA's performance while also posing as a potential threat to human health; therefore, during preparation of the PDA, the rapid detection of the residual AM plays an important role in controlling the residual AM while improving the PDA's performance. The objective of this study was to explore the possibilities for applying near-infrared (NIR) spectroscopy as a potential tool for detecting the residual AM in combination with a statistical tool. In this study, the radial basis function (RBF) network model as the statistical tool was combined with NIR spectroscopy for detection of the residual AM. The experimental results showed that five wavelengths in the NIR spectroscopy were the most important characteristic adsorption peaks, particular at 971.95 and 1077 nm. The simulation of the RBF model presented higher performance with R2-value greater than 0.98, RMSEC and RMSEP less than 7.22 x 10(-5) and coefficient of variation (CV) of the predicted residual AM less than 10%, which demonstrated the feasibility of the NIR spectroscopy being a rapid detection tool for prediction of the residual AM using the RBF model. Wavelet de-nosing was used for removing the interference/noise in the NIR spectroscopy and improved the generalization ability of the RBF model.\",\n",
              "  'authors': ['Zhu, Guocheng',\n",
              "   'Zheng, Huaili',\n",
              "   'Zhang, Peng',\n",
              "   'Jiao, Zhaojie',\n",
              "   'Yin, Jun'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental technology',\n",
              "  'keywords': ['Acrylamide',\n",
              "   'Acrylic Resins',\n",
              "   'Allyl Compounds',\n",
              "   'Feasibility Studies',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Quaternary Ammonium Compounds',\n",
              "   'Spectroscopy, Near-Infrared'],\n",
              "  'pmid': '23530319',\n",
              "  'title': 'Near-infrared spectroscopy as a potential tool with radial basis function for measurement of residual acrylamide in organic polymer.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Sleep quality is an important health indicator, and the current measurements of sleep rely on questionnaires, polysomnography, etc., which are intrusive, expensive or time consuming. Therefore, a more nonintrusive, inexpensive and convenient method needs to be developed. Use of the Kinect sensor to capture one's gait pattern can reveal whether his/her sleep quality meets the requirements. Fifty-nine healthy students without disabilities were recruited as participants. The Pittsburgh Sleep Quality Index (PSQI) and Kinect sensors were used to acquire the sleep quality scores and gait data. After data preprocessing, gait features were extracted for training machine learning models that predicted sleep quality scores based on the data. The t-test indicated that the following joints had stronger weightings in the prediction: the Head, Spine Shoulder, Wrist Left, Hand Right, Thumb Left, Thumb Right, Hand Tip Left, Hip Left, and Foot Left. For sleep quality prediction, the best result was achieved by Gaussian processes, with a correlation of 0.78 (p < 0.001). For the subscales, the best result was 0.51 for daytime dysfunction (p < 0.001) by linear regression. Gait can reveal sleep quality quite well. This method is a good supplement to the existing methods in identifying sleep quality more ecologically and less intrusively.\",\n",
              "  'authors': ['Liu, Xingyun',\n",
              "   'Sun, Bingli',\n",
              "   'Zhang, Zhan',\n",
              "   'Wang, Yameng',\n",
              "   'Tang, Haina',\n",
              "   'Zhu, Tingshao'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31553783',\n",
              "  'title': 'Gait can reveal sleep quality with machine learning models.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1938-1352',\n",
              "  'IF_2013': '1.688',\n",
              "  'IF_2014': '1.43',\n",
              "  'IF_2015': '1.043',\n",
              "  'IF_2016': '1.277',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0748-7711',\n",
              "  'ISSN_online': '1938-1352',\n",
              "  'ISSN_print': '0748-7711',\n",
              "  'IsoAbbr': 'J Rehabil Res Dev',\n",
              "  'JrId': 5189,\n",
              "  'MedAbbr': 'J Rehabil Res Dev',\n",
              "  'NLMid': '8410047',\n",
              "  'Titre': 'J Rehabil Res Dev',\n",
              "  'abstract': 'A walker that could help navigate and avoid collisions with obstacles could help reduce health costs and increase the quality of care and independence of thousands of people. This study evaluated the safety and performance of the Veterans Affairs Personal Adaptive Mobility Aid (VA-PAMAID). We performed engineering tests on the VA-PAMAID to determine safety factors, including stability, energy consumption, fatigue life, and sensor and control malfunctions. The VA-PAMAID traveled 10.9 km on a full charge and avoided obstacles while traveling at a speed of up to 1.2 m/s. No failures occurred during static stability, climatic, or fatigue testing. Some problems were encountered during obstacle climbing and sensor and control testing. The VA-PAMAID has good range, has adequate reaction time, and is structurally sound. Clinical trials are planned to compare the device to other low-technical adaptive mobility devices.',\n",
              "  'authors': ['Rentschler, Andrew J',\n",
              "   'Cooper, Rory A',\n",
              "   'Blasch, Bruce',\n",
              "   'Boninger, Michael L'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'Journal of rehabilitation research and development',\n",
              "  'keywords': ['Equipment Design',\n",
              "   'Equipment Safety',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Walkers'],\n",
              "  'pmid': '15080227',\n",
              "  'title': 'Intelligent walkers for the elderly: performance and safety testing of VA-PAMAID robotic walker.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1873-3476',\n",
              "  'IF_2013': '3.785',\n",
              "  'IF_2014': '3.65',\n",
              "  'IF_2015': '3.994',\n",
              "  'IF_2016': '3.649',\n",
              "  'IF_2017': '3.862',\n",
              "  'ISSN': '0378-5173',\n",
              "  'ISSN_online': '1873-3476',\n",
              "  'ISSN_print': '0378-5173',\n",
              "  'IsoAbbr': 'Int J Pharm',\n",
              "  'JrId': 21201,\n",
              "  'MedAbbr': 'Int J Pharm',\n",
              "  'NLMid': '7804127',\n",
              "  'Titre': 'Int J Pharm',\n",
              "  'abstract': \"Establishing a method to evaluate the reliability of an optimal solution is an exciting challenge for the nonlinear response surface method. We reported previously that the bootstrap (BS) resampling technique and Kohonen's self-organizing map are promising tools for meeting this challenge. To understand the usefulness of these techniques further, we employed a formulation optimization study of photocrosslinked polyacrylic acid (PAA) hydrogel as a case study. In a series of experiments, a large number of optimal solutions were generated with BS resampling and they were classified into three distinct clusters with SOM clustering. Using analysis of Bayesian estimation, we clarified the mode of generating clusters; e.g., cluster 2 was distinguished by the difference in features between the BS optimal solutions and the original optimal solution, whereas cluster 3 was distinguished by the substantial change in the shape of the response surfaces. We concluded that cluster 1 represents the global optimal solution, and then estimated 95% confidence intervals of the optimal solutions using the BS optimal solutions. These findings prove that our method is a valid approach for evaluating nonlinear optimal solutions. This method has applications for establishing a science-based rationale for, and a design space in, pharmaceutical formulation development.\",\n",
              "  'authors': ['Onuki, Yoshinori',\n",
              "   'Kikuchi, Shingo',\n",
              "   'Yasuda, Akihito',\n",
              "   'Takayama, Kozo'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'International journal of pharmaceutics',\n",
              "  'keywords': ['Acrylic Resins',\n",
              "   'Bayes Theorem',\n",
              "   'Chemistry, Pharmaceutical',\n",
              "   'Cluster Analysis',\n",
              "   'Dosage Forms',\n",
              "   'Drug Carriers',\n",
              "   'Gels',\n",
              "   'Methacrylates',\n",
              "   'Molecular Structure',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Photochemical Processes',\n",
              "   'Reproducibility of Results',\n",
              "   'Technology, Pharmaceutical',\n",
              "   'Ultraviolet Rays'],\n",
              "  'pmid': '20558260',\n",
              "  'title': 'Role of individual test samples in optimal solutions in pharmaceuticals predicted using a nonlinear response surface method.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Independent component analysis (ICA) has found great promise in magnetic resonance (MR) image analysis. Unfortunately, two key issues have been overlooked and not investigated. One is the lack of MR images to be used to unmix signal sources of interest. Another is the use of random initial projection vectors by ICA, which causes inconsistent results. In order to address the first issue, this paper introduces a band-expansion process (BEP) to generate an additional new set of images from the original MR images via nonlinear functions. These newly generated images are then combined with the original MR images to provide sufficient MR images for ICA analysis. In order to resolve the second issue, a prioritized ICA (PICA) is designed to rank the ICA-generated independent components (ICs) so that MR brain tissue substances can be unmixed and separated by different ICs in a prioritized order. Finally, BEP and PICA are combined to further develop a new ICA-based approach, referred to as PICA-BEP to perform MR image analysis.',\n",
              "  'authors': ['Ouyang, Yen-Chieh',\n",
              "   'Chen, Hsian-Min',\n",
              "   'Chai, Jyh-Wen',\n",
              "   'Chen, Clayton Chi-chang',\n",
              "   'Poon, Sek-Kwong',\n",
              "   'Yang, Ching-Wen',\n",
              "   'Lee, San-Kan',\n",
              "   'Chang, Chein-I'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Phantoms, Imaging',\n",
              "   'Principal Component Analysis',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '18714830',\n",
              "  'title': 'Band expansion-based over-complete independent component analysis for multispectral processing of magnetic resonance images.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1546-3141',\n",
              "  'IF_2013': '2.744',\n",
              "  'IF_2014': '2.731',\n",
              "  'IF_2015': '2.66',\n",
              "  'IF_2016': '2.778',\n",
              "  'IF_2017': '3.125',\n",
              "  'ISSN': '0361-803X',\n",
              "  'ISSN_online': '1546-3141',\n",
              "  'ISSN_print': '0361-803X',\n",
              "  'IsoAbbr': 'AJR Am J Roentgenol',\n",
              "  'JrId': 377,\n",
              "  'MedAbbr': 'AJR Am J Roentgenol',\n",
              "  'NLMid': '7708173',\n",
              "  'Titre': 'AJR Am J Roentgenol',\n",
              "  'abstract': 'A massive-training artificial neural network is a nonlinear pattern recognition tool used to suppress rib opacity on chest radiographs while soft-tissue contrast is maintained. We investigated the effect of rib suppression with a massive-training artificial neural network on the performance of radiologists in the detection of pulmonary nodules on chest radiographs.|We used 60 chest radiographs; 30 depicted solitary pulmonary nodules, and 30 showed no nodules. A stratified random-sampling scheme was used to select the images from the standard digital image database developed by the Japanese Society of Radiologic Technology. The mean diameter of the 30 pulmonary nodules was 14.7 +/- 4.1 (SD) mm. Receiver operating characteristic analysis was used to evaluate observer performance in the detection of pulmonary nodules first on the chest radiographs without and then on the radiographs with rib suppression. Seven board-certified radiologists and five radiology residents participated in this observer study.|For all 12 observers, the mean values of the area under the best-fit receiver operating characteristic curve for images without and with rib suppression were 0.816 +/- 0.077 and 0.843 +/- 0.074; the difference was statistically significant (p = 0.019). The mean areas under the curve for images without and with rib suppression were 0.848 +/- 0.059 and 0.883 +/- 0.050 for the seven board-certified radiologists (p = 0.011) and 0.770 +/- 0.081 and 0.788 +/- 0.074 for the five radiology residents (p = 0.310).|In the detection of pulmonary nodules, evaluation of a combination of rib-suppressed and original chest radiographs significantly improved the diagnostic performance of radiologists over the use of chest radiographs alone.',\n",
              "  'authors': ['Oda, Seitaro',\n",
              "   'Awai, Kazuo',\n",
              "   'Suzuki, Kenji',\n",
              "   'Yanaga, Yumi',\n",
              "   'Funama, Yoshinori',\n",
              "   'MacMahon, Heber',\n",
              "   'Yamashita, Yasuyuki'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'AJR. American journal of roentgenology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Clinical Competence',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Inservice Training',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'ROC Curve',\n",
              "   'Radiography, Thoracic',\n",
              "   'Radiology',\n",
              "   'Solitary Pulmonary Nodule'],\n",
              "  'pmid': '19843717',\n",
              "  'title': 'Performance of radiologists in detection of small pulmonary nodules on chest radiographs: effect of rib suppression with a massive-training artificial neural network.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': \"Telestration is an important teaching tool in minimally invasive surgery (MIS). While robotic surgery offers the added benefit of three-dimensional (3-D) visualization, telestration technology does not currently exist for this modality. This project aimed to develop a video algorithm to accurately translate a mentor's two-dimensional (2-D) telestration into a 3-D telestration in the da Vinci visual field.|A prototype 3-D telestration system was constructed to translate 2-D telestration from a mentor station into 3-D graphics for the trainee at the robotic console. This system uses fast image correlation algorithms to allow 2-D images to be placed over the same anatomic location in the two separate video channels of the stereoscopic robotic visualization system. Three subjects of varying surgical backgrounds, blinded to the mode of telestration (2-D vs. 3-D), were tested in the laboratory, using a simulated robotic task.|There were few technologic errors (2), only one of which resulted in a task error, in 99 total trials. Only the experienced MIS staff surgeon had a significantly faster task time in 2-D than in 3-D (P < 0.05). The MIS fellow recorded the fastest task times in 2-D and 3-D (P < 0.05). There were nine task errors, six of which were committed by the MIS fellow. The nonsurgeon trainee had the least number of errors but also had the slowest times.|Robotic telestration in 3-D is feasible and does not negatively impact performance in laboratory tasks. We plan to refine the prototype and investigate its use in vivo.\",\n",
              "  'authors': ['Ali, Mohamed R',\n",
              "   'Loggins, Jamie P',\n",
              "   'Fuller, William D',\n",
              "   'Miller, Brian E',\n",
              "   'Hasser, Christopher J',\n",
              "   'Yellowlees, Peter',\n",
              "   'Vidovszky, Tamas J',\n",
              "   'Rasmussen, Jason J',\n",
              "   'Pierce, Jonathan'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Feasibility Studies',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Teaching Materials'],\n",
              "  'pmid': '18266586',\n",
              "  'title': '3-D telestration: a teaching tool for robotic surgery.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'There is a need for a standardized curriculum for training and assessment of robotic surgeons to proficiency, followed by high-stakes testing (HST) for certification.|To standardize the curriculum and certification of robotic surgeons, a series of consensus conferences attended by 14 leading international surgical societies have been used to compile the outcomes measures and curriculum that should form the basis for a Fundamentals of Robotic Surgery (FRS) programme.|A set of 25 outcomes measures and a curriculum for teaching the skills needed to safely use current generation surgical robotic systems has been developed and accepted by a committee of experienced robotic surgeons across 14 specialties.|A standardized process for certifying the skills of a robotic surgeon has begun to emerge. The work described here documents both the processes used for developing educational material and the educational content of a robotic curriculum.',\n",
              "  'authors': ['Smith, Roger', 'Patel, Vipul', 'Satava, Richard'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Artifacts',\n",
              "   'Certification',\n",
              "   'Clinical Competence',\n",
              "   'Curriculum',\n",
              "   'Humans',\n",
              "   'International Cooperation',\n",
              "   'Robotic Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Societies, Medical',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24277315',\n",
              "  'title': 'Fundamentals of robotic surgery: a course of basic robotic surgery skills based upon a 14-society consensus template of outcomes measures and curriculum development.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1530-9932',\n",
              "  'IF_2013': '1.776',\n",
              "  'IF_2014': '1.641',\n",
              "  'IF_2015': '1.954',\n",
              "  'IF_2016': '2.451',\n",
              "  'IF_2017': '2.666',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1530-9932',\n",
              "  'IsoAbbr': 'AAPS PharmSciTech',\n",
              "  'JrId': 30088,\n",
              "  'MedAbbr': 'AAPS PharmSciTech',\n",
              "  'NLMid': '100960111',\n",
              "  'Titre': 'AAPS PharmSciTech',\n",
              "  'abstract': 'The aim of this project was to expand a previously developed prototype expert network for use in the analysis of multiple biopharmaceutics classification system (BCS) class II drugs. The model drugs used were carbamazepine, chlorpropamide, diazepam, ibuprofen, ketoprofen, naproxen, and piroxicam. Recommended formulations were manufactured and tested for dissolution performance. A comprehensive training data set for the model drugs was developed and used to retrain the artificial neural network. The training and the system were validated based on the comparison of predicted and observed performance of the recommended formulations. The initial test of the system resulted in high error values, indicating poor prediction capabilities for drugs other than piroxicam. A new data set, containing 182 batches, was used for retraining. Ten percent of the test batches were used for cross-validation, resulting in models with R2 > or = 70%. The comparison of observed performance to the predicted performance found that the system predicted successfully. The hybrid network was generally able to predict the amount of drug dissolved within 5% for the model drugs. Through validation, the system was proven to be capable of designing formulations that met specific drug performance criteria. By including parameters to address wettability and the intrinsic dissolution characteristics of the drugs, the hybrid system was shown to be suitable for analysis of multiple BCS class II drugs.',\n",
              "  'authors': ['Wilson, Wendy I', 'Peng, Yun', 'Augsburger, Larry L'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'AAPS PharmSciTech',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Capsules',\n",
              "   'Chemistry, Pharmaceutical',\n",
              "   'Gelatin',\n",
              "   'Reproducibility of Results',\n",
              "   'Technology, Pharmaceutical'],\n",
              "  'pmid': '16354004',\n",
              "  'title': 'Generalization of a prototype intelligent hybrid system for hard gelatin capsule formulation development.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['EE'],\n",
              "  'ESSN': '1420-3049',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '2.416',\n",
              "  'IF_2015': '2.465',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '3.098',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1420-3049',\n",
              "  'IsoAbbr': 'Molecules',\n",
              "  'JrId': 22275,\n",
              "  'MedAbbr': 'Molecules',\n",
              "  'NLMid': '100964009',\n",
              "  'Titre': 'Molecules',\n",
              "  'abstract': 'Identifying possible drug-target interactions (DTIs) has become an important task in drug research and development. Although high-throughput screening is becoming available, experimental methods narrow down the validation space because of extremely high cost, low success rate, and time consumption. Therefore, various computational models have been exploited to infer DTI candidates.|We introduced relevant databases and packages, mainly provided a comprehensive review of computational models for DTI identification, including network-based algorithms and machine learning-based methods. Specially, machine learning-based methods mainly include bipartite local model, matrix factorization, regularized least squares, and deep learning.|Although computational methods have obtained significant improvement in the process of DTI prediction, these models have their limitations. We discussed potential avenues for boosting DTI prediction accuracy as well as further directions.',\n",
              "  'authors': ['Zhou, Liqian',\n",
              "   'Li, Zejun',\n",
              "   'Yang, Jialiang',\n",
              "   'Tian, Geng',\n",
              "   'Liu, Fuxing',\n",
              "   'Wen, Hong',\n",
              "   'Peng, Li',\n",
              "   'Chen, Min',\n",
              "   'Xiang, Ju',\n",
              "   'Peng, Lihong'],\n",
              "  'categories': ['CHEMISTRY, ORGANIC'],\n",
              "  'journal': 'Molecules (Basel, Switzerland)',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Databases, Factual',\n",
              "   'Drug Discovery',\n",
              "   'Machine Learning',\n",
              "   'Molecular Docking Simulation',\n",
              "   'Molecular Dynamics Simulation',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Software'],\n",
              "  'pmid': '31052598',\n",
              "  'title': 'Revealing Drug-Target Interactions with Computational Models and Algorithms.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': '1) To use data-driven method to examine clinical codes (risk factors) of a medical condition in primary care electronic health records (EHRs) that can accurately predict a diagnosis of the condition in secondary care EHRs. 2) To develop and validate a disease phenotyping algorithm for rheumatoid arthritis using primary care EHRs.|This study linked routine primary and secondary care EHRs in Wales, UK. A machine learning based scheme was used to identify patients with rheumatoid arthritis from primary care EHRs via the following steps: i) selection of variables by comparing relative frequencies of Read codes in the primary care dataset associated with disease case compared to non-disease control (disease/non-disease based on the secondary care diagnosis); ii) reduction of predictors/associated variables using a Random Forest method, iii) induction of decision rules from decision tree model. The proposed method was then extensively validated on an independent dataset, and compared for performance with two existing deterministic algorithms for RA which had been developed using expert clinical knowledge.|Primary care EHRs were available for 2,238,360 patients over the age of 16 and of these 20,667 were also linked in the secondary care rheumatology clinical system. In the linked dataset, 900 predictors (out of a total of 43,100 variables) in the primary care record were discovered more frequently in those with versus those without RA. These variables were reduced to 37 groups of related clinical codes, which were used to develop a decision tree model. The final algorithm identified 8 predictors related to diagnostic codes for RA, medication codes, such as those for disease modifying anti-rheumatic drugs, and absence of alternative diagnoses such as psoriatic arthritis. The proposed data-driven method performed as well as the expert clinical knowledge based methods.|Data-driven scheme, such as ensemble machine learning methods, has the potential of identifying the most informative predictors in a cost-effective and rapid way to accurately and reliably classify rheumatoid arthritis or other complex medical conditions in primary care EHRs.',\n",
              "  'authors': ['Zhou, Shang-Ming',\n",
              "   'Fernandez-Gutierrez, Fabiola',\n",
              "   'Kennedy, Jonathan',\n",
              "   'Cooksey, Roxanne',\n",
              "   'Atkinson, Mark',\n",
              "   'Denaxas, Spiros',\n",
              "   'Siebert, Stefan',\n",
              "   'Dixon, William G',\n",
              "   \"O'Neill, Terence W\",\n",
              "   'Choy, Ernest',\n",
              "   'Sudlow, Cathie',\n",
              "   'Brophy, Sinead'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Antirheumatic Agents',\n",
              "   'Arthritis, Rheumatoid',\n",
              "   'Electronic Health Records',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Primary Health Care'],\n",
              "  'pmid': '27135409',\n",
              "  'title': 'Defining Disease Phenotypes in Primary Care Electronic Health Records by a Machine Learning Approach: A Case Study in Identifying Rheumatoid Arthritis.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1469-8137',\n",
              "  'IF_2013': '6.373',\n",
              "  'IF_2014': '7.672',\n",
              "  'IF_2015': '7.21',\n",
              "  'IF_2016': '7.33',\n",
              "  'IF_2017': '7.433',\n",
              "  'ISSN': '0028-646X',\n",
              "  'ISSN_online': '1469-8137',\n",
              "  'ISSN_print': '0028-646X',\n",
              "  'IsoAbbr': 'New Phytol.',\n",
              "  'JrId': 8596,\n",
              "  'MedAbbr': 'New Phytol',\n",
              "  'NLMid': '9882884',\n",
              "  'Titre': 'New Phytol.',\n",
              "  'abstract': 'Recent advances in gene function prediction rely on ensemble approaches that integrate results from multiple inference methods to produce superior predictions. Yet, these developments remain largely unexplored in plants. We have explored and compared two methods to integrate 10 gene co-function networks for Arabidopsis thaliana and demonstrate how the integration of these networks produces more accurate gene function predictions for a larger fraction of genes with unknown function. These predictions were used to identify genes involved in mitochondrial complex I formation, and for five of them, we confirmed the predictions experimentally. The ensemble predictions are provided as a user-friendly online database, EnsembleNet. The methods presented here demonstrate that ensemble gene function prediction is a powerful method to boost prediction performance, whereas the EnsembleNet database provides a cutting-edge community tool to guide experimentalists.',\n",
              "  'authors': ['Hansen, Bjoern Oest',\n",
              "   'Meyer, Etienne H',\n",
              "   'Ferrari, Camilla',\n",
              "   'Vaid, Neha',\n",
              "   'Movahedi, Sara',\n",
              "   'Vandepoele, Klaas',\n",
              "   'Nikoloski, Zoran',\n",
              "   'Mutwil, Marek'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'The New phytologist',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Benchmarking',\n",
              "   'Databases, Genetic',\n",
              "   'Electron Transport Complex I',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Genes, Plant',\n",
              "   'Mutation',\n",
              "   'Software'],\n",
              "  'pmid': '29205376',\n",
              "  'title': 'Ensemble gene function prediction database reveals genes important for complex I formation in Arabidopsis thaliana.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1471-2962',\n",
              "  'IF_2013': '2.864',\n",
              "  'IF_2014': '2.147',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.97',\n",
              "  'IF_2017': '2.746',\n",
              "  'ISSN': '1364-503X',\n",
              "  'ISSN_online': '1471-2962',\n",
              "  'ISSN_print': '1364-503X',\n",
              "  'IsoAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'JrId': 30111,\n",
              "  'MedAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'NLMid': '101133385',\n",
              "  'Titre': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'abstract': \"This theme issue has the founding ambition of landscaping data ethics as a new branch of ethics that studies and evaluates moral problems related to data (including generation, recording, curation, processing, dissemination, sharing and use), algorithms (including artificial intelligence, artificial agents, machine learning and robots) and corresponding practices (including responsible innovation, programming, hacking and professional codes), in order to formulate and support morally good solutions (e.g. right conducts or right values). Data ethics builds on the foundation provided by computer and information ethics but, at the same time, it refines the approach endorsed so far in this research field, by shifting the level of abstraction of ethical enquiries, from being information-centric to being data-centric. This shift brings into focus the different moral dimensions of all kinds of data, even data that never translate directly into information but can be used to support actions or generate behaviours, for example. It highlights the need for ethical analyses to concentrate on the content and nature of computational operations-the interactions among hardware, software and data-rather than on the variety of digital technologies that enable them. And it emphasizes the complexity of the ethical challenges posed by data science. Because of such complexity, data ethics should be developed from the start as a macroethics, that is, as an overall framework that avoids narrow, ad hoc approaches and addresses the ethical impact and implications of data science and its applications within a consistent, holistic and inclusive framework. Only as a macroethics will data ethics provide solutions that can maximize the value of data science for our societies, for all of us and for our environments.This article is part of the themed issue 'The ethical impact of data science'.\",\n",
              "  'authors': ['Floridi, Luciano', 'Taddeo, Mariarosaria'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Philosophical transactions. Series A, Mathematical, physical, and engineering sciences',\n",
              "  'keywords': ['Information Science'],\n",
              "  'pmid': '28336805',\n",
              "  'title': 'What is data ethics?',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'When people are asked to palpate a novel soft object to discern its physical properties such as texture, elasticity, and even non-homogeneity, they not only regulate probing behaviors, but also the co-contraction level of antagonistic muscles to control the mechanical impedance of fingers. It is suspected that such behavior tries to enhance haptic perception by regulating the function of mechanoreceptors at different depths of the fingertips and proprioceptive sensors such as tendon and spindle sensors located in muscles. In this paper, we designed and fabricated a novel two-degree of freedom variable stiffness indentation probe to investigate whether the regulation of internal stiffness, indentation, and probe sweeping velocity (PSV) variables affect the accuracy of the depth estimation of stiff inclusions in an artificial silicon phantom using information gain metrics. Our experimental results provide new insights into not only the biological phenomena of haptic perception but also new opportunities to design and control soft robotic probes. ',\n",
              "  'authors': ['Sornkarn, Nantachai',\n",
              "   'Dasgupta, Prokar',\n",
              "   'Nanayakkara, Thrishantha'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Fingers',\n",
              "   'Humans',\n",
              "   'Mechanoreceptors',\n",
              "   'Models, Theoretical',\n",
              "   'Perception',\n",
              "   'Phantoms, Imaging',\n",
              "   'Robotics',\n",
              "   'Touch Perception'],\n",
              "  'pmid': '27257814',\n",
              "  'title': 'Morphological Computation of Haptic Perception of a Controllable Stiffness Probe.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': 'The electroencephalography (EEG) signals are known to involve the firings of neurons in the brain. The P300 wave is a high potential caused by an event-related stimulus. The detection of P300s included in the measured EEG signals is widely investigated. The difficulties in detecting them are that they are mixed with other signals generated over a large brain area and their amplitudes are very small due to the distance and resistivity differences in their transmittance.|A novel real-time feature extraction method for detecting P300 waves by combining an adaptive nonlinear principal component analysis (ANPCA) and a multilayer neural network is proposed. The measured EEG signals are first filtered using a sixth-order band-pass filter with cut-off frequencies of 1 Hz and 12 Hz. The proposed ANPCA scheme consists of four steps: pre-separation, whitening, separation, and estimation. In the experiment, four different inter-stimulus intervals (ISIs) are utilized: 325 ms, 350 ms, 375 ms, and 400 ms.|The developed multi-stage principal component analysis method applied at the pre-separation step has reduced the external noises and artifacts significantly. The introduced adaptive law in the whitening step has made the subsequent algorithm in the separation step to converge fast. The separation performance index has varied from -20 dB to -33 dB due to randomness of source signals. The robustness of the ANPCA against background noises has been evaluated by comparing the separation performance indices of the ANPCA with four algorithms (NPCA, NSS-JD, JADE, and SOBI), in which the ANPCA algorithm demonstrated the shortest iteration time with performance index about 0.03. Upon this, it is asserted that the ANPCA algorithm successfully separates mixed source signals.|The independent components produced from the observed data using the proposed method illustrated that the extracted signals were clearly the P300 components elicited by task-related stimuli. The experiment using 350 ms ISI showed the best performance. Since the proposed method does not use down-sampling and averaging, it can be used as a viable tool for real-time clinical applications.',\n",
              "  'authors': ['Turnip, Arjon', 'Hong, Keum-Shik', 'Jeong, Myung-Yung'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Artifacts',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Computer Simulation',\n",
              "   'Electroencephalography',\n",
              "   'Event-Related Potentials, P300',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Principal Component Analysis',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '21939560',\n",
              "  'title': 'Real-time feature extraction of P300 component using adaptive nonlinear principal component analysis.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1573-9686',\n",
              "  'IF_2013': '3.231',\n",
              "  'IF_2014': '3.195',\n",
              "  'IF_2015': '2.887',\n",
              "  'IF_2016': '3.221',\n",
              "  'IF_2017': '3.405',\n",
              "  'ISSN': '0090-6964',\n",
              "  'ISSN_online': '1573-9686',\n",
              "  'ISSN_print': '0090-6964',\n",
              "  'IsoAbbr': 'Ann Biomed Eng',\n",
              "  'JrId': 561,\n",
              "  'MedAbbr': 'Ann Biomed Eng',\n",
              "  'NLMid': '0361512',\n",
              "  'Titre': 'Ann Biomed Eng',\n",
              "  'abstract': \"In this study, to advance smart health applications which have increasing security/privacy requirements, we propose a novel highly wearable ECG-based user identification system, empowered by both non-standard convenient ECG lead configurations and deep learning techniques. Specifically, to achieve a super wearability, we suggest situating all the ECG electrodes on the left upper-arm, or behind the ears, and successfully obtain weak but distinguishable ECG waveforms. Afterwards, to identify individuals from weak ECG, we further present a two-stage framework, including ECG imaging and deep feature learning/identification. In the former stage, the ECG heartbeats are projected to a 2D state space, to reveal heartbeats' trajectory behaviors and produce 2D images by a split-then-hit method. In the second stage, a convolutional neural network is introduced to automatically learn the intricate patterns directly from the ECG image representations without heavy feature engineering, and then perform user identification. Experimental results on two acquired datasets using our wearable prototype, show a promising identification rate of 98.4% (single-arm-ECG) and 91.1% (ear-ECG), respectively. To the best of our knowledge, it is the first study on the feasibility of using single-arm-ECG/ear-ECG for user identification purpose, which is expected to contribute to pervasive ECG-based user identification in smart health applications.\",\n",
              "  'authors': ['Zhang, Qingxue', 'Zhou, Dian'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Annals of biomedical engineering',\n",
              "  'keywords': ['Arm',\n",
              "   'Biometric Identification',\n",
              "   'Ear',\n",
              "   'Electrocardiography',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Wearable Electronic Devices'],\n",
              "  'pmid': '29030801',\n",
              "  'title': 'Deep Arm/Ear-ECG Image Learning for Highly Wearable Biometric Human Identification.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1879-0038',\n",
              "  'IF_2013': '2.082',\n",
              "  'IF_2014': '2.138',\n",
              "  'IF_2015': '2.319',\n",
              "  'IF_2016': '2.415',\n",
              "  'IF_2017': '2.498',\n",
              "  'ISSN': '0378-1119',\n",
              "  'ISSN_online': '1879-0038',\n",
              "  'ISSN_print': '0378-1119',\n",
              "  'IsoAbbr': 'Gene',\n",
              "  'JrId': 3886,\n",
              "  'MedAbbr': 'Gene',\n",
              "  'NLMid': '7706761',\n",
              "  'Titre': 'Gene',\n",
              "  'abstract': 'Polycystic ovary syndrome (PCOS) is one of the most common endocrine disorders in reproductive-aged women. However, the exact pathophysiology of PCOS remains largely unclear. We performed deep sequencing to investigate the mRNA and long noncoding RNA (lncRNA) expression profiles in the ovarian tissues of letrozole-induced PCOS rat model and control rats. A total of 2147 mRNAs and 158 lncRNAs were differentially expressed between the PCOS models and control. Gene ontology analysis indicated that differentially expressed mRNAs were associated with biological adhesion, reproduction, and metabolic process. Pathway analysis results indicated that these aberrantly expressed mRNAs were related to several specific signaling pathways, including insulin resistance, steroid hormone biosynthesis, PPAR signaling pathway, cell adhesion molecules, autoimmune thyroid disease, and AMPK signaling pathway. The relative expression levels of mRNAs and lncRNAs were validated through qRT-PCR. LncRNA-miRNA-mRNA network was constructed to explore ceRNAs involved in the PCOS model and were also verified by qRTPCR experiment. These findings may provide insight into the pathogenesis of PCOS and clues to find key diagnostic and therapeutic roles of lncRNA in PCOS.',\n",
              "  'authors': ['Fu, Lu-Lu',\n",
              "   'Xu, Ying',\n",
              "   'Li, Dan-Dan',\n",
              "   'Dai, Xiao-Wei',\n",
              "   'Xu, Xin',\n",
              "   'Zhang, Jing-Shun',\n",
              "   'Ming, Hao',\n",
              "   'Zhang, Xue-Ying',\n",
              "   'Zhang, Guo-Qing',\n",
              "   'Ma, Ya-Lan',\n",
              "   'Zheng, Lian-Wen'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Gene',\n",
              "  'keywords': ['Animals',\n",
              "   'Disease Models, Animal',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Humans',\n",
              "   'Letrozole',\n",
              "   'Nitriles',\n",
              "   'Ovary',\n",
              "   'Polycystic Ovary Syndrome',\n",
              "   'RNA, Long Noncoding',\n",
              "   'RNA, Messenger',\n",
              "   'Rats',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Triazoles'],\n",
              "  'pmid': '29505837',\n",
              "  'title': 'Expression profiles of mRNA and long noncoding RNA in the ovaries of letrozole-induced polycystic ovary syndrome rat model through deep sequencing.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Neural learning algorithms generally involve a number of identical processing units, which are fully or partially connected, and involve an update function, such as a ramp, a sigmoid or a Gaussian function for instance. Some variations also exist, where units can be heterogeneous, or where an alternative update technique is employed, such as a pulse stream generator. Associated with connections are numerical values that must be adjusted using a learning rule, and and dictated by parameters that are learning rule specific, such as momentum, a learning rate, a temperature, amongst others. Usually, neural learning algorithms involve local updates, and a global interaction between units is often discouraged, except in instances where units are fully connected, or involve synchronous updates. In all of these instances, concurrency within a neural algorithm cannot be fully exploited without a suitable implementation strategy. A design scheme is described for translating a neural learning algorithm from inception to implementation on a parallel machine using PVM or MPI libraries, or onto programmable logic such as FPGAs. A designer must first describe the algorithm using a specialised Neural Language, from which a Petri net (PN) model is constructed automatically for verification, and building a performance model. The PN model can be used to study issues such as synchronisation points, resource sharing and concurrency within a learning rule. Specialised constructs are provided to enable a designer to express various aspects of a learning rule, such as the number and connectivity of neural nodes, the interconnection strategies, and information flows required by the learning algorithm. A scheduling and mapping strategy is then used to translate this PN model onto a multiprocessor template. We demonstrate our technique using a Kohonen and backpropagation learning rules, implemented on a loosely coupled workstation cluster, and a dedicated parallel machine, with PVM libraries.',\n",
              "  'authors': ['Rana, O F'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computers',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Programming Languages'],\n",
              "  'pmid': '11011794',\n",
              "  'title': 'Automating parallel implementation of neural learning algorithms.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1552-5783',\n",
              "  'IF_2013': '3.661',\n",
              "  'IF_2014': '3.404',\n",
              "  'IF_2015': '3.427',\n",
              "  'IF_2016': '3.303',\n",
              "  'IF_2017': '3.388',\n",
              "  'ISSN': '0146-0404',\n",
              "  'ISSN_online': '1552-5783',\n",
              "  'ISSN_print': '0146-0404',\n",
              "  'IsoAbbr': 'Invest. Ophthalmol. Vis. Sci.',\n",
              "  'JrId': 4371,\n",
              "  'MedAbbr': 'Invest Ophthalmol Vis Sci',\n",
              "  'NLMid': '7703701',\n",
              "  'Titre': 'Invest. Ophthalmol. Vis. Sci.',\n",
              "  'abstract': 'Four videokeratographic methods for keratoconus detection were compared with a neural network approach.|A classification neural network for keratoconus screening was designed to detect the presence of keratoconus (KC) or keratoconus suspects (KCS); a separate cone severity network graded the severity of conelike topography patterns consistent with KC or KCS. Three hundred TMS-1 examinations (Tomey) were randomly divided into training and test sets. Ten topographic indexes were network inputs. Nine categories were used: normal, astigmatism, KC, KCS, contact lens-induced warpage, pellucid marginal degeneration, photorefractive keratectomy, radial keratotomy, and penetrating keratoplasty. KC was subdivided into KC1 (mild), KC2 (moderate), and KC3 (advanced). There were three outputs for the classification network (KC, KCS, and OTHER); target output values of 0 = OTHER, 0.25 = KCS, 0.5 = KC1, 0.75 = KC2, and 1.0 = KC3 were used for the severity network.|The best-trained classification network had 100% accuracy, specificity, and sensitivity for the test set. The severity network had mean outputs (+/-standard deviation) of OTHER = 0.02+/-0.02, KCS = 0.21+/-0.05, KC1 = 0.52+/-0.17, KC2 = 0.74+/-0.12, and KC3 = 0.91+/-0.15. The severity network output for all categories was well correlated to the keratoconus prediction index (R = 0.892, P < 0.0001). The classification network had an overall accuracy and specificity significantly better (P < or = 0.005) than the Klyce/Maeda keratoconus index (KCI) test, the Rabinowitz test (K & I-S), and simulated keratometry (average Sim K). However, there were no significant differences in keratoconus sensitivity between the classification network, KCI, and K & I-S. The sensitivity and specificity of average Sim K were significantly worse than those of the other tests. The classification network had significantly better sensitivity (P < 0.001) and specificity (P = 0.025) for KCS detection than the K & I-S.|The neural networks completely distinguished KC from KCS and from topographies that resembled KC. The network approach equaled the sensitivity of currently used tests for keratoconus detection and outperformed them in terms of accuracy and specificity.',\n",
              "  'authors': ['Smolek, M K', 'Klyce, S D'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Investigative ophthalmology & visual science',\n",
              "  'keywords': ['Cornea',\n",
              "   'Corneal Topography',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Keratoconus',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '9344352',\n",
              "  'title': 'Current keratoconus detection methods compared with a neural network approach.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1879-0038',\n",
              "  'IF_2013': '2.082',\n",
              "  'IF_2014': '2.138',\n",
              "  'IF_2015': '2.319',\n",
              "  'IF_2016': '2.415',\n",
              "  'IF_2017': '2.498',\n",
              "  'ISSN': '0378-1119',\n",
              "  'ISSN_online': '1879-0038',\n",
              "  'ISSN_print': '0378-1119',\n",
              "  'IsoAbbr': 'Gene',\n",
              "  'JrId': 3886,\n",
              "  'MedAbbr': 'Gene',\n",
              "  'NLMid': '7706761',\n",
              "  'Titre': 'Gene',\n",
              "  'abstract': 'Understanding the mechanisms that enable growth and survival of an organism while driving it to the full range of its adaptation is fundamental to the issues of biodiversity and evolution, particularly regarding global climatic changes. Here we report the Illumina RNA-sequencing (RNA-seq) and de novo assembly of the blue mussel Mytilus edulis transcriptome during early development. This study is based on high-throughput data, which associates genome-wide differentially expressed transcript (DET) patterns with early activation of developmental processes. Approximately 50,383 high-quality contigs were assembled. Over 8000 transcripts were associated with functional proteins from public databases. Coding and non-coding genes served to design customized microarrays targeting every developmental stage, which encompass major transitions in tissue organization. Consequently, multi-processing pattern exploration protocols applied to 3633 DETs helped discover 12 unique coordinated eigengenes supposedly implicated in various physiological and morphological changes that larvae undergo during early development. Moreover, dynamic Bayesian networks (DBNs) provided key insights to understand stage-specific molecular mechanisms activated throughout ontogeny. In addition, delayed and contemporaneous interactions between DETs were coerced with 16 relevant regulators that interrelated in non-random genetic regulatory networks (GRNs). Genes associated with mechanisms of neural and muscular development have been characterized and further included in dynamic networks necessary in growth and functional morphology. This is the first large-scale study being dedicated to M. edulis throughout early ontogeny. Integration between RNA-seq and microarray data enabled a high-throughput exploration of hidden processes essential in growth and survival of microscopic mussel larvae. Our integrative approach will support a holistic understanding of systems biology and will help establish new links between environmental assessment and functional development of marine bivalves.',\n",
              "  'authors': ['Bassim, Sleiman',\n",
              "   'Tanguy, Arnaud',\n",
              "   'Genard, Bertrand',\n",
              "   'Moraga, Dario',\n",
              "   'Tremblay, Rejean'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Gene',\n",
              "  'keywords': ['Animals',\n",
              "   'Bayes Theorem',\n",
              "   'Chromatin Assembly and Disassembly',\n",
              "   'Fuzzy Logic',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Regulatory Networks',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Larva',\n",
              "   'Metamorphosis, Biological',\n",
              "   'Mytilus edulis',\n",
              "   'Principal Component Analysis',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '25158132',\n",
              "  'title': 'Identification of Mytilus edulis genetic regulators during early development.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1534-4908',\n",
              "  'IF_2013': '0.938',\n",
              "  'IF_2014': '1.14',\n",
              "  'IF_2015': '1.019',\n",
              "  'IF_2016': '1.02',\n",
              "  'IF_2017': '0.986',\n",
              "  'ISSN': '1530-4515',\n",
              "  'ISSN_online': '1534-4908',\n",
              "  'ISSN_print': '1530-4515',\n",
              "  'IsoAbbr': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'JrId': 21595,\n",
              "  'MedAbbr': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'NLMid': '100888751',\n",
              "  'Titre': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'abstract': 'We introduce a new technique using a snake retractor to allow for a complete central lymph node dissection (CND) under a more optimal view for robotic thyroid surgery using bilateral axillo-breast approaches.|Fifty-eight patients diagnosed with papillary thyroid cancer who underwent robotic thyroidectomy from April to December 2010 were analyzed. A total of 20 patients underwent a robotic surgery without use of snake retractor initially, and 38 patients underwent surgery using snake retractor. Ipsilateral CNDs were performed in all cases.|The number of harvested lymph nodes was 5.3 without use of the snake retractor and 7.9 with use (P = 0.04). Serum thyroglobulins off thyroid hormone in total cases were 0.6 and 0.2, respectively (P = 0.42). The rate of postoperative complications including transient hypocalcemia (n = 8) and transient recurrent laryngeal nerve palsy (n = 2) were low between the 2 groups without a significance.|Utilization of the snake retractor facilitated safely effective CND with using of 2 arms simultaneously, while providing a superior view to that obtained with robotic thyroid surgery using bilateral axillo-breast approach.',\n",
              "  'authors': ['Kim, Wan W', 'Jung, Jin H', 'Park, Ho Y'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical laparoscopy, endoscopy & percutaneous techniques',\n",
              "  'keywords': ['Adult',\n",
              "   'Carcinoma',\n",
              "   'Carcinoma, Papillary',\n",
              "   'Endoscopy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hypocalcemia',\n",
              "   'Lymph Node Excision',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Robotics',\n",
              "   'Surgical Instruments',\n",
              "   'Thyroid Cancer, Papillary',\n",
              "   'Thyroid Neoplasms',\n",
              "   'Thyroidectomy',\n",
              "   'Vocal Cord Paralysis'],\n",
              "  'pmid': '23386162',\n",
              "  'title': 'New technique using the snake retractor for complete lymph node dissection in robotic thyroid surgery:  initial experiences.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Respiratory syncytial virus (RSV) causes infections that range from common cold to severe lower respiratory tract infection requiring high-level medical care. Prediction of the course of disease in individual patients remains challenging at the first visit to the pediatric wards and RSV infections may rapidly progress to severe disease. In this study we investigate whether there exists a genomic signature that can accurately predict the course of RSV. We used early blood microarray transcriptome profiles from 39 hospitalized infants that were followed until recovery and of which the level of disease severity was determined retrospectively. Applying support vector machine learning on age by sex standardized transcriptomic data, an 84 gene signature was identified that discriminated hospitalized infants with eventually less severe RSV infection from infants that suffered from most severe RSV disease. This signature yielded an area under the receiver operating characteristic curve (AUC) of 0.966 using leave-one-out cross-validation on the experimental data and an AUC of 0.858 on an independent validation cohort consisting of 53 infants. A combination of the gene signature with age and sex yielded an AUC of 0.971. Thus, the presented signature may serve as the basis to develop a prognostic test to support clinical management of RSV patients.',\n",
              "  'authors': ['Jong, Victor L',\n",
              "   'Ahout, Inge M L',\n",
              "   'van den Ham, Henk-Jan',\n",
              "   'Jans, Jop',\n",
              "   'Zaaraoui-Boutahar, Fatiha',\n",
              "   'Zomer, Aldert',\n",
              "   'Simonetti, Elles',\n",
              "   'Bijl, Maarten A',\n",
              "   'Brand, H Kim',\n",
              "   'van IJcken, Wilfred F J',\n",
              "   'de Jonge, Marien I',\n",
              "   'Fraaij, Pieter L',\n",
              "   'de Groot, Ronald',\n",
              "   'Osterhaus, Albert D M E',\n",
              "   'Eijkemans, Marinus J',\n",
              "   'Ferwerda, Gerben',\n",
              "   'Andeweg, Arno C'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Bronchiolitis, Viral',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Humans',\n",
              "   'Infant',\n",
              "   'Infant, Newborn',\n",
              "   'Male',\n",
              "   'Prognosis',\n",
              "   'Respiratory Syncytial Virus Infections',\n",
              "   'Respiratory Syncytial Viruses',\n",
              "   'Severity of Illness Index',\n",
              "   'Support Vector Machine',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27833115',\n",
              "  'title': 'Transcriptome assists prognosis of disease severity in respiratory syncytial virus infected infants.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1520-6882',\n",
              "  'IF_2013': '5.825',\n",
              "  'IF_2014': '5.636',\n",
              "  'IF_2015': '5.886',\n",
              "  'IF_2016': '6.32',\n",
              "  'IF_2017': '6.042',\n",
              "  'ISSN': '0003-2700',\n",
              "  'ISSN_online': '1520-6882',\n",
              "  'ISSN_print': '0003-2700',\n",
              "  'IsoAbbr': 'Anal. Chem.',\n",
              "  'JrId': 519,\n",
              "  'MedAbbr': 'Anal Chem',\n",
              "  'NLMid': '0370536',\n",
              "  'Titre': 'Anal. Chem.',\n",
              "  'abstract': 'The most serious, and yet unsolved, problem of constructing molecular computing devices consists in connecting all of these molecular events into a usable device. This report demonstrates the use of Boolean logic tree for analyzing the chemical event network based on graphene, organic dye, thrombin aptamer, and Fenton reaction, organizing and connecting these basic chemical events. And this chemical event network can be utilized to implement fluorescent combinatorial logic (including basic logic gates and complex integrated logic circuits) and fuzzy logic computing. On the basis of the Boolean logic tree analysis and logic computing, these basic chemical events can be considered as programmable \"words\" and chemical interactions as \"syntax\" logic rules to construct molecular search engine for performing intelligent molecular search query. Our approach is helpful in developing the advanced logic program based on molecules for application in biosensing, nanotechnology, and drug delivery. ',\n",
              "  'authors': ['Huang, Wei Tao', 'Luo, Hong Qun', 'Li, Nian Bing'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytical chemistry',\n",
              "  'keywords': ['Fuzzy Logic', 'Graphite', 'Information Storage and Retrieval'],\n",
              "  'pmid': '24739071',\n",
              "  'title': 'Boolean logic tree of graphene-based chemical system for molecular computation and intelligent molecular search query.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'Recent case series have now been reported for robotic cystectomy. Despite the novelty of these approaches, few reports have been published on the oncologic and clinical outcomes, even in the short term. We report the oncologic and short-term clinical outcomes of 50 patients who underwent robotic-assisted radical cystectomy.|A total of 50 patients underwent robotic-assisted laparoscopic radical cystectomy and extracorporeal urinary diversion for bladder cancer. The outcome measures included the pathologic outcomes, complication rate, timing of adjuvant chemotherapy, disease recurrence, and overall and disease-specific survival.|Robotic cystectomy was performed in 40 men and 10 women at a mean age of 63.6 years. Of the 50 patients, 66% had Stage pT2 or less, 14% had pT3 disease, and 20% Stage N+ disease. No patient had positive surgical margins. The mean number of lymph nodes removed was 19 (range 8-37). The mean clinical follow-up was 13.2 months. Seven patients had evidence of recurrent disease. Three patients died of advanced urothelial carcinoma, and two died of other causes. Longer term complications included stomal hernia in 2 and partial ureteral obstruction in 1. Eleven patients underwent adjuvant chemotherapy for Stage pT3 disease and N+ disease at a mean of 7.2 weeks postoperatively.|The clinical and oncologic follow-up of patients undergoing robotic radical cystectomy appears to be favorable in the short term. As our follow-up increases, we should expect to continue to truly define the long-term clinical appropriateness and oncologic success of this procedure.',\n",
              "  'authors': ['Pruthi, Raj S', 'Wallen, Eric M'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Cohort Studies',\n",
              "   'Cystectomy',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Lymph Nodes',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Medical Oncology',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Bladder',\n",
              "   'Urinary Bladder Neoplasms'],\n",
              "  'pmid': '18586308',\n",
              "  'title': 'Is robotic radical cystectomy an appropriate treatment for bladder cancer? Short-term oncologic and clinical follow-up in 50 consecutive patients.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'How multiple sensory cues are integrated in neural circuitry remains a challenge. The common hypothesis is that information integration might be accomplished in a dedicated multisensory integration area receiving feedforward inputs from the modalities. However, recent experimental evidence suggests that it is not a single multisensory brain area, but rather many multisensory brain areas that are simultaneously involved in the integration of information. Why many mutually connected areas should be needed for information integration is puzzling. Here, we investigated theoretically how information integration could be achieved in a distributed fashion within a network of interconnected multisensory areas. Using biologically realistic neural network models, we developed a decentralized information integration system that comprises multiple interconnected integration areas. Studying an example of combining visual and vestibular cues to infer heading direction, we show that such a decentralized system is in good agreement with anatomical evidence and experimental observations. In particular, we show that this decentralized system can integrate information optimally. The decentralized system predicts that optimally integrated information should emerge locally from the dynamics of the communication between brain areas and sheds new light on the interpretation of the connectivity between multisensory brain areas.|To extract information reliably from ambiguous environments, the brain integrates multiple sensory cues, which provide different aspects of information about the same entity of interest. Here, we propose a decentralized architecture for multisensory integration. In such a system, no processor is in the center of the network topology and information integration is achieved in a distributed manner through reciprocally connected local processors. Through studying the inference of heading direction with visual and vestibular cues, we show that the decentralized system can integrate information optimally, with the reciprocal connections between processers determining the extent of cue integration. Our model reproduces known multisensory integration behaviors observed in experiments and sheds new light on our understanding of how information is integrated in the brain.',\n",
              "  'authors': ['Zhang, Wen-Hao', 'Chen, Aihua', 'Rasch, Malte J', 'Wu, Si'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Bayes Theorem',\n",
              "   'Computer Simulation',\n",
              "   'Electronic Data Processing',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Neurons',\n",
              "   'Sensation'],\n",
              "  'pmid': '26758843',\n",
              "  'title': 'Decentralized Multisensory Information Integration in Neural Systems.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Disease comorbidity is very common and has significant impact on disease treatment. Revealing the associations among diseases may help to understand the mechanisms of diseases, improve the prevention and treatment of diseases, and support the discovery of new drugs or new uses of existing drugs.|In this paper, we introduced a mathematical model to represent gene related diseases with a series of associated genes based on the overrepresentation of genes and diseases in PubMed literature. We also illustrated an efficient way to reveal the implicit connections between COPD and other diseases based on this model.|We applied this approach to analyze the relationships between Chronic Obstructive Pulmonary Disease (COPD) and other diseases under the Lung diseases branch in the Medical subject heading index system and detected 4 novel diseases relevant to COPD. As judged by domain experts, the F score of our approach is up to 77.6%.|The results demonstrate the effectiveness of the gene fingerprint model for diseases on the basis of medical literature.',\n",
              "  'authors': ['Chen, Guocai',\n",
              "   'Jia, Yuxi',\n",
              "   'Zhu, Lisha',\n",
              "   'Li, Ping',\n",
              "   'Zhang, Lin',\n",
              "   'Tao, Cui',\n",
              "   'Jim Zheng, W'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Comorbidity',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Literature Based Discovery',\n",
              "   'Models, Theoretical',\n",
              "   'PubMed',\n",
              "   'Pulmonary Disease, Chronic Obstructive'],\n",
              "  'pmid': '30700303',\n",
              "  'title': 'Gene fingerprint model for literature based detection of the associations among complex diseases: a case study of COPD.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'The lack of triangulation of standard endoscopic devices limits the degree of freedom for surgical maneuvers during natural orifice transluminal endoscopic surgery (NOTES). This study explored the feasibility of adapting an intuitively controlled master and slave transluminal endoscopic robot (MASTER) the authors developed to facilitate wedge hepatic resection in NOTES.|The MASTER consists of a master controller, a telesurgical workstation, and a slave manipulator that holds two end-effectors: a grasper, and a monopolar electrocautery hook. The master controller is attached to the wrist and fingers of the operator and connected to the manipulator by electrical and wire cables. Movements of the operator are detected and converted into control signals driving the slave manipulator via a tendon-sheath power transmission mechanism allowing nine degrees of freedom. Using this system, wedge hepatic resection was performed through the transgastric route on two female pigs under general anesthesia. Entry into the peritoneal cavity was via a 10-mm incision made on the anterior wall of the stomach by the electrocautery hook. Wedge hepatic resection was performed using the robotic grasper and hook. Hemostasis was achieved with the electrocautery hook. After the procedure, the resected liver tissue was retrieved through the mouth using the grasper.|Using the MASTER, transgastric wedge hepatic resection was successfully performed on two pigs with no laparoscopic assistance. The entire procedure took 9.4 min (range, 8.5-10.2 min), with 7.1 min (range, 6-8.2 min) spent on excision of the liver tissue. The robotics-controlled device was able to grasp, retract, and excise the liver specimen successfully in the desired plane.|This study demonstrated for the first time that the MASTER could effectively mitigate the technical constraints normally encountered in NOTES procedures. With it, the triangulation of surgical tools and the manipulation of tissue became easy, and wedge hepatic resection could be accomplished successfully without the need for assistance using laparoscopic instruments.',\n",
              "  'authors': ['Phee, S J',\n",
              "   'Ho, K Y',\n",
              "   'Lomanto, D',\n",
              "   'Low, S C',\n",
              "   'Huynh, V A',\n",
              "   'Kencana, A P',\n",
              "   'Yang, K',\n",
              "   'Sun, Z L',\n",
              "   'Chung, S C Sydney'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Animals',\n",
              "   'Disease Models, Animal',\n",
              "   'Equipment Design',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Hepatectomy',\n",
              "   'Natural Orifice Endoscopic Surgery',\n",
              "   'Pilot Projects',\n",
              "   'Robotics',\n",
              "   'Stomach',\n",
              "   'Swine',\n",
              "   'Video Recording'],\n",
              "  'pmid': '20177915',\n",
              "  'title': 'Natural orifice transgastric endoscopic wedge hepatic resection in an experimental model using an intuitively controlled master and slave transluminal endoscopic robot (MASTER).',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1557-9832',\n",
              "  'IF_2013': '1.352',\n",
              "  'IF_2014': '1.366',\n",
              "  'IF_2015': '1.69',\n",
              "  'IF_2016': '2.396',\n",
              "  'IF_2017': '1.897',\n",
              "  'ISSN': '0272-2712',\n",
              "  'ISSN_online': '1557-9832',\n",
              "  'ISSN_print': '0272-2712',\n",
              "  'IsoAbbr': 'Clin. Lab. Med.',\n",
              "  'JrId': 3087,\n",
              "  'MedAbbr': 'Clin Lab Med',\n",
              "  'NLMid': '8100174',\n",
              "  'Titre': 'Clin. Lab. Med.',\n",
              "  'abstract': 'Large-scale clinical databases provide a detailed perspective on patient phenotype in disease and the characteristics of health care processes. Important information is often contained in the relationships between the values and timestamps of sequences of clinical data. The analysis of clinical time sequence data across entire patient populations may reveal data patterns that enable a more precise understanding of disease presentation, progression, and response to therapy, and thus could be of great value for clinical and translational research. Recent work suggests that the combination of temporal data mining methods with techniques from artificial intelligence research on knowledge-based temporal abstraction may enable the mining of clinically relevant temporal features from these previously problematic general clinical data.',\n",
              "  'authors': ['Post, Andrew R', 'Harrison, James H'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinics in laboratory medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases as Topic',\n",
              "   'Humans',\n",
              "   'Medical Informatics',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Software',\n",
              "   'Time Factors'],\n",
              "  'pmid': '18194720',\n",
              "  'title': 'Temporal data mining.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1573-2959',\n",
              "  'IF_2013': '1.679',\n",
              "  'IF_2014': '1.679',\n",
              "  'IF_2015': '1.633',\n",
              "  'IF_2016': '1.687',\n",
              "  'IF_2017': '1.804',\n",
              "  'ISSN': '0167-6369',\n",
              "  'ISSN_online': '1573-2959',\n",
              "  'ISSN_print': '0167-6369',\n",
              "  'IsoAbbr': 'Environ Monit Assess',\n",
              "  'JrId': 22015,\n",
              "  'MedAbbr': 'Environ Monit Assess',\n",
              "  'NLMid': '8508350',\n",
              "  'Titre': 'Environ Monit Assess',\n",
              "  'abstract': 'Organophosphate (OP) and carbamate (CP) pesticides act by the inhibition of acetylcholinesterase (AChE). This enables the use of this enzyme for the detection of these pesticides in the environment. While many studies have looked at the effect of single pesticides on AChE, the effect of mixtures of pesticides still requires extensive investigation. This is important to evaluate the cumulative risk in the case of simultaneous exposure to multiple pesticides. Therefore we examined the effect of five different pesticides (carbaryl, carbofuran, parathion, demeton-S-methyl, and aldicarb) on AChE activity to determine whether combinations had an additive, synergistic, or antagonistic inhibitory effect. Results indicated that the mixtures had an additive inhibitory effect on AChE activity. The data from the assays of the mixtures were used to develop and train an artificial neural network (ANN) which was then utilised successfully for the identification of pesticides and their concentrations in mixtures. This study is significant because it evaluated mixtures of OPs and CPs where previous studies focused on either OPs or CPs. Previous studies have only examined up to three pesticides while this study evaluated mixtures of five pesticides simultaneously. This is also the first study where an ANN was able to utilise data from the inhibition of a single enzyme to differentiate five different pesticides and their concentrations from mixtures.',\n",
              "  'authors': ['Mwila, K', 'Burton, M H', 'Van Dyk, J S', 'Pletschke, B I'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental monitoring and assessment',\n",
              "  'keywords': ['Acetylcholinesterase',\n",
              "   'Carbamates',\n",
              "   'Cholinesterase Inhibitors',\n",
              "   'Drug Synergism',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Organophosphates',\n",
              "   'Pesticides',\n",
              "   'Risk Assessment'],\n",
              "  'pmid': '22684846',\n",
              "  'title': 'The effect of mixtures of organophosphate and carbamate pesticides on acetylcholinesterase and application of chemometrics to identify pesticides in mixtures.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Short-term traffic flow prediction is one of the most important issues in the field of intelligent transport system (ITS). Because of the uncertainty and nonlinearity, short-term traffic flow prediction is a challenging task. In order to improve the accuracy of short-time traffic flow prediction, a hybrid model (SSA-KELM) is proposed based on singular spectrum analysis (SSA) and kernel extreme learning machine (KELM). SSA is used to filter out the noise of traffic flow time series. Then, the filtered traffic flow data is used to train KELM model, the optimal input form of the proposed model is determined by phase space reconstruction, and parameters of the model are optimized by gravitational search algorithm (GSA). Finally, case validation is carried out using the measured data of an expressway in Xiamen, China. And the SSA-KELM model is compared with several well-known prediction models, including support vector machine, extreme learning machine, and single KLEM model. The experimental results demonstrate that performance of the proposed model is superior to that of the comparison models. Apart from accuracy improvement, the proposed model is more robust. ',\n",
              "  'authors': ['Shang, Qiang',\n",
              "   'Lin, Ciyun',\n",
              "   'Yang, Zhaosheng',\n",
              "   'Bing, Qichun',\n",
              "   'Zhou, Xiyang'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Automobiles',\n",
              "   'China',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Models, Theoretical',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '27551829',\n",
              "  'title': 'A Hybrid Short-Term Traffic Flow Prediction Model Based on Singular Spectrum Analysis and Kernel Extreme Learning Machine.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['XW'],\n",
              "  'ESSN': '1179-2035',\n",
              "  'IF_2013': '5.32',\n",
              "  'IF_2014': '5.038',\n",
              "  'IF_2015': '5.579',\n",
              "  'IF_2016': '6.832',\n",
              "  'IF_2017': '7.074',\n",
              "  'ISSN': '0112-1642',\n",
              "  'ISSN_online': '1179-2035',\n",
              "  'ISSN_print': '0112-1642',\n",
              "  'IsoAbbr': 'Sports Med',\n",
              "  'JrId': 7204,\n",
              "  'MedAbbr': 'Sports Med',\n",
              "  'NLMid': '8412297',\n",
              "  'Titre': 'Sports Med',\n",
              "  'abstract': 'Suitably processed electromyographic (EMG) signals can be combined with Hill-type musculoskeletal models to noninvasively achieve estimations of individual muscle forces. This method has particular advantages over other methods for the assessment of a given performance. The purpose of this review is to report on the current issues facing the human movement scientist who wishes to extend the kinetic information yielded by linked segment models to the kinetics of individual muscles. Such an extension is necessary when considering co-contraction of antagonistic muscles, the role of bi-articular muscles, co-ordination, movement efficiency or bone-on-bone forces. Currently, linked segment models have not been successfully extended to individual muscle forces for diagnostic purposes by using the EMG approach or any other approach. Most models have been designed for a specific purpose and have only been evaluated over a narrow range of movement conditions. More generalised models will require greater complexity and possibly more extensive calibration or an increased number of specific inputs or greater computational effort. This review shows the promise of the EMG approach and presents the challenges, as well as the strategies, that should enable more general, accurate and precise estimates of individual muscle forces.',\n",
              "  'authors': ['Dowling, J J'],\n",
              "  'categories': ['SPORT SCIENCES'],\n",
              "  'journal': 'Sports medicine (Auckland, N.Z.)',\n",
              "  'keywords': ['Animals',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Elbow Joint',\n",
              "   'Electromyography',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Muscle Contraction',\n",
              "   'Muscle Fatigue',\n",
              "   'Muscle, Skeletal',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '9291550',\n",
              "  'title': 'The use of electromyography for the noninvasive prediction of muscle forces. Current issues.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'In this paper, data mining is used to analyze the data on the differentiation of mammalian Mesenchymal Stem Cells (MSCs), aiming at discovering known and hidden rules governing MSC differentiation, following the establishment of a web-based public database containing experimental data on the MSC proliferation and differentiation. To this effect, a web-based public interactive database comprising the key parameters which influence the fate and destiny of mammalian MSCs has been constructed and analyzed using Classification Association Rule Mining (CARM) as a data-mining technique. The results show that the proposed approach is technically feasible and performs well with respect to the accuracy of (classification) prediction. Key rules mined from the constructed MSC database are consistent with experimental observations, indicating the validity of the method developed and the first step in the application of data mining to the study of MSCs.',\n",
              "  'authors': ['Wang, Weiqi',\n",
              "   'Wang, Yanbo Justin',\n",
              "   'Bañares-Alcántara, René',\n",
              "   'Coenen, Frans',\n",
              "   'Cui, Zhanfeng'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Cell Differentiation',\n",
              "   'Computer Simulation',\n",
              "   'Databases, Factual',\n",
              "   'Humans',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Mesenchymal Stem Cells',\n",
              "   'Models, Biological',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '20014470',\n",
              "  'title': 'Analysis of mesenchymal stem cell differentiation in vitro using classification association rule mining.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': 'A new framework for heart sound analysis is proposed. One of the most difficult processes in heart sound analysis is segmentation, due to interference form murmurs.|Equal number of cardiac cycles were extracted from heart sounds with different heart rates using information from envelopes of autocorrelation functions without the need to label individual fundamental heart sounds (FHS). The complete method consists of envelope detection, calculation of cardiac cycle lengths using auto-correlation of envelope signals, features extraction using discrete wavelet transform, principal component analysis, and classification using neural network bagging predictors.|The proposed method was tested on a set of heart sounds obtained from several on-line databases and recorded with an electronic stethoscope. Geometric mean was used as performance index. Average classification performance using ten-fold cross-validation was 0.92 for noise free case, 0.90 under white noise with 10 dB signal-to-noise ratio (SNR), and 0.90 under impulse noise up to 0.3 s duration.|The proposed method showed promising results and high noise robustness to a wide range of heart sounds. However, more tests are needed to address any bias that may have been introduced by different sources of heart sounds in the current training set, and to concretely validate the method. Further work include building a new training set recorded from actual patients, then further evaluate the method based on this new training set.',\n",
              "  'authors': ['Yuenyong, Sumeth',\n",
              "   'Nishihara, Akinori',\n",
              "   'Kongprawechnon, Waree',\n",
              "   'Tungpimolrut, Kanokvate'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Automation',\n",
              "   'Heart Sounds',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Principal Component Analysis',\n",
              "   'Reproducibility of Results',\n",
              "   'Statistics as Topic'],\n",
              "  'pmid': '21303558',\n",
              "  'title': 'A framework for automatic heart sound analysis without segmentation.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1573-2959',\n",
              "  'IF_2013': '1.679',\n",
              "  'IF_2014': '1.679',\n",
              "  'IF_2015': '1.633',\n",
              "  'IF_2016': '1.687',\n",
              "  'IF_2017': '1.804',\n",
              "  'ISSN': '0167-6369',\n",
              "  'ISSN_online': '1573-2959',\n",
              "  'ISSN_print': '0167-6369',\n",
              "  'IsoAbbr': 'Environ Monit Assess',\n",
              "  'JrId': 22015,\n",
              "  'MedAbbr': 'Environ Monit Assess',\n",
              "  'NLMid': '8508350',\n",
              "  'Titre': 'Environ Monit Assess',\n",
              "  'abstract': \"This article discusses a methodology for assessing and ranking a predefined universe of objects to assist in the selection of suitable areas for the construction of sanitary landfills. The methodology is, in principle, for universal application. Its development was initially based on the 'Value Analysis' methodology and then on the 'Fuzzy Eigenvector Method', or 'Fuzzy Logic' as it is sometimes referred to. The focus of the study herein is on the comparative appraisal of a group of elements which are part of the guidelines to be adopted when determining the choice of the most suitable site for a sanitary landfill. This set of guidelines includes technical, economical, environmental, social and political aspects. As an example of the methodology proposed, four different areas were chosen. They were studied to determine which is the most appropriate site for the development of a new solid waste fill site. The results for these four areas, together with the functions chosen and function weights, are presented here.\",\n",
              "  'authors': ['Mahler, Claudio F', \"De Lima, Gisele Sant'Anna\"],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental monitoring and assessment',\n",
              "  'keywords': ['Decision Making',\n",
              "   'Environmental Pollution',\n",
              "   'Forecasting',\n",
              "   'Fuzzy Logic',\n",
              "   'Guidelines as Topic',\n",
              "   'Refuse Disposal'],\n",
              "  'pmid': '12733814',\n",
              "  'title': 'Applying value analysis and fuzzy logic to select areas for installing waste fills.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1873-7560',\n",
              "  'IF_2013': '12.48',\n",
              "  'IF_2014': '13.938',\n",
              "  'IF_2015': '14.976',\n",
              "  'IF_2016': '16.265',\n",
              "  'IF_2017': '17.581',\n",
              "  'ISSN': '0302-2838',\n",
              "  'ISSN_online': '1873-7560',\n",
              "  'ISSN_print': '0302-2838',\n",
              "  'IsoAbbr': 'Eur. Urol.',\n",
              "  'JrId': 3630,\n",
              "  'MedAbbr': 'Eur Urol',\n",
              "  'NLMid': '7512719',\n",
              "  'Titre': 'Eur. Urol.',\n",
              "  'abstract': 'We report the first case of intra-abdominal combustion involving the plastic covering of monopolar scissors secondary to use of incorrect gas (oxygen [O(2)] instead of carbon dioxide [CO(2)]) during robot-assisted laparoscopic radical prostatectomy (RALP). The insufflating system was connected to a provisional O(2) gate into the operating theater. A patient underwent RALP and extended pelvic lymph node dissection for localized prostate cancer, according to standard technique. Approximately 1.5 h after the start of surgery, flames arose from the scissor tips during monopolar coagulation. After extinguishing the fire, we promptly withdrew and changed instruments before recognizing and resolving the cause of the incident. The procedure was carried out without patient injury, and the postoperative period was uneventful.',\n",
              "  'authors': ['Di Pierro, Giovanni B',\n",
              "   'Besmer, Ivo',\n",
              "   'Hefermehl, Lukas J',\n",
              "   'Beatrice, Josef',\n",
              "   'Danuser, Hansjörg',\n",
              "   'Beutler, Jonas',\n",
              "   'Mattei, Agostino'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'European urology',\n",
              "  'keywords': ['Abdomen',\n",
              "   'Carbon Dioxide',\n",
              "   'Fires',\n",
              "   'Humans',\n",
              "   'Insufflation',\n",
              "   'Intraoperative Complications',\n",
              "   'Male',\n",
              "   'Medical Errors',\n",
              "   'Oxygen',\n",
              "   'Prostatectomy',\n",
              "   'Robotics'],\n",
              "  'pmid': '20702029',\n",
              "  'title': 'Intra-abdominal fire due to insufflating oxygen instead of carbon dioxide during robot-assisted radical prostatectomy: case report and literature review.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1864-6433',\n",
              "  'IF_2013': '1.507',\n",
              "  'IF_2014': '1.677',\n",
              "  'IF_2015': '1.467',\n",
              "  'IF_2016': '1.396',\n",
              "  'IF_2017': '1.656',\n",
              "  'ISSN': '0914-7187',\n",
              "  'ISSN_online': '1864-6433',\n",
              "  'ISSN_print': '0914-7187',\n",
              "  'IsoAbbr': 'Ann Nucl Med',\n",
              "  'JrId': 1734,\n",
              "  'MedAbbr': 'Ann Nucl Med',\n",
              "  'NLMid': '8913398',\n",
              "  'Titre': 'Ann Nucl Med',\n",
              "  'abstract': 'Computer-aided diagnosis (CAD) software for bone scintigrams have recently been introduced as a clinical quality assurance tool. The purpose of this study was to compare the diagnostic accuracy of two CAD systems, one based on a European and one on a Japanese training database, in a group of bone scans from Japanese patients.|The two CAD software are trained to interpret bone scans using training databases consisting of bone scans with the desired interpretation, metastatic disease or not. One software was trained using 795 bone scans from European patients and the other with 904 bone scans from Japanese patients. The two CAD softwares were evaluated using the same group of 257 Japanese patients, who underwent bone scintigraphy because of suspected metastases of malignant tumors in 2009. The final diagnostic results made by clinicians were used as gold standard.|The Japanese CAD software showed a higher specificity and accuracy compared to the European CAD software [81 vs. 57 % (p < 0.05) and 82 vs. 61 % (p < 0.05), respectively]. The sensitivity was 90 % for the Japanese CAD software and 83 % for the European CAD software (n.s).|The CAD software trained with a Japanese database showed significantly higher performance than the corresponding CAD software trained with a European database for the analysis of bone scans from Japanese patients. These results could at least partly be caused by the physical differences between Japanese and European patients resulting in less influence of attenuation in Japanese patients and possible different judgement of count intensities of hot spots.',\n",
              "  'authors': ['Horikoshi, Hiroyuki',\n",
              "   'Kikuchi, Akihiro',\n",
              "   'Onoguchi, Masahisa',\n",
              "   'Sjöstrand, Karl',\n",
              "   'Edenbrandt, Lars'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Annals of nuclear medicine',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Bone and Bones',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Japan',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Radionuclide Imaging',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Software'],\n",
              "  'pmid': '22729550',\n",
              "  'title': 'Computer-aided diagnosis system for bone scintigrams from Japanese patients: importance of training database.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Modern psychometric theory provides many useful tools for ability testing, such as item response theory, computerised adaptive testing, and automatic item generation. However, these techniques have yet to be integrated into mainstream psychological practice. This is unfortunate, because modern psychometric techniques can bring many benefits, including sophisticated reliability measures, improved construct validity, avoidance of exposure effects, and improved efficiency. In the present research we therefore use these techniques to develop a new test of a well-studied psychological capacity: melodic discrimination, the ability to detect differences between melodies. We calibrate and validate this test in a series of studies. Studies 1 and 2 respectively calibrate and validate an initial test version, while Studies 3 and 4 calibrate and validate an updated test version incorporating additional easy items. The results support the new test's viability, with evidence for strong reliability and construct validity. We discuss how these modern psychometric techniques may also be profitably applied to other areas of music psychology and psychological science in general.\",\n",
              "  'authors': ['Harrison, Peter M C', 'Collins, Tom', 'Müllensiefen, Daniel'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Adult',\n",
              "   'Cognition',\n",
              "   'Discrimination (Psychology)',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Models, Psychological',\n",
              "   'Monte Carlo Method',\n",
              "   'Music',\n",
              "   'Psychometrics',\n",
              "   'Young Adult'],\n",
              "  'pmid': '28620165',\n",
              "  'title': 'Applying modern psychometric techniques to melodic discrimination testing: Item response theory, computerised adaptive testing, and automatic item generation.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1522-2586',\n",
              "  'IF_2013': '2.788',\n",
              "  'IF_2014': '3.21',\n",
              "  'IF_2015': '3.25',\n",
              "  'IF_2016': '3.083',\n",
              "  'IF_2017': '3.612',\n",
              "  'ISSN': '1053-1807',\n",
              "  'ISSN_online': '1522-2586',\n",
              "  'ISSN_print': '1053-1807',\n",
              "  'IsoAbbr': 'J Magn Reson Imaging',\n",
              "  'JrId': 2081,\n",
              "  'MedAbbr': 'J Magn Reson Imaging',\n",
              "  'NLMid': '9105850',\n",
              "  'Titre': 'J Magn Reson Imaging',\n",
              "  'abstract': 'To 1) describe textural features from diffusion-weighted images (DWI) and apparent diffusion coefficient (ADC) maps that can distinguish low-grade bladder cancer from high-grade, and 2) propose a radiomics-based strategy for cancer grading using texture features.|In all, 61 patients with bladder cancer (29 in high- and 32 in low-grade groups) were enrolled in this retrospective study. Histogram- and gray-level co-occurrence matrix (GLCM)-based radiomics features were extracted from cancerous volumes of interest (VOIs) on DWI and corresponding ADC maps of each patient acquired from 3.0T magnetic resonance imaging (MRI). A Mann-Whitney U-test was applied to select features with significant differences between low- and high-grade groups (P < 0.05). Then support vector machine with recursive feature elimination (SVM-RFE) and classification strategy was adopted to find an optimal feature subset and then to establish a classification model for grading.|A total 102 features were derived from each VOI and among them, 47 candidate features were selected, which showed significant intergroup differences (P < 0.05). By the SVM-RFE method, an optimal feature subset including 22 features was further selected from candidate features. The SVM classifier using the optimal feature subset achieved the best performance in bladder cancer grading, with an area under the receiver operating characteristic curve, accuracy, sensitivity, and specificity of 0.861, 82.9%, 78.4%, and 87.1%, respectively.|Textural features from DWI and ADC maps can reflect the difference between low- and high-grade bladder cancer, especially those GLCM features from ADC maps. The proposed radiomics strategy using these features, combined with the SVM classifier, may better facilitate image-based bladder cancer grading preoperatively.|3 Technical Efficacy: Stage 2 J. Magn. Reson. Imaging 2017;46:1281-1288.',\n",
              "  'authors': ['Zhang, Xi',\n",
              "   'Xu, Xiaopan',\n",
              "   'Tian, Qiang',\n",
              "   'Li, Baojuan',\n",
              "   'Wu, Yuxia',\n",
              "   'Yang, Zengyue',\n",
              "   'Liang, Zhengrong',\n",
              "   'Liu, Yang',\n",
              "   'Cui, Guangbin',\n",
              "   'Lu, Hongbing'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of magnetic resonance imaging : JMRI',\n",
              "  'keywords': ['Aged',\n",
              "   'Algorithms',\n",
              "   'Area Under Curve',\n",
              "   'Biomarkers',\n",
              "   'Diffusion Magnetic Resonance Imaging',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Period',\n",
              "   'ROC Curve',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Support Vector Machine',\n",
              "   'Urinary Bladder',\n",
              "   'Urinary Bladder Neoplasms'],\n",
              "  'pmid': '28199039',\n",
              "  'title': 'Radiomics assessment of bladder cancer grade using texture features from diffusion-weighted imaging.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1530-0285',\n",
              "  'IF_2013': '6.364',\n",
              "  'IF_2014': '6.187',\n",
              "  'IF_2015': '5.485',\n",
              "  'IF_2016': '5.728',\n",
              "  'IF_2017': '6.655',\n",
              "  'ISSN': '0893-3952',\n",
              "  'ISSN_online': '1530-0285',\n",
              "  'ISSN_print': '0893-3952',\n",
              "  'IsoAbbr': 'Mod. Pathol.',\n",
              "  'JrId': 6644,\n",
              "  'MedAbbr': 'Mod Pathol',\n",
              "  'NLMid': '8806605',\n",
              "  'Titre': 'Mod. Pathol.',\n",
              "  'abstract': 'Telepathology is the practice of digitizing histological or macroscopic images for transmission along telecommunication pathways for diagnosis, consultation, or continuing medical education. Previous studies have addressed static versus dynamic imaging in several specimen types with a wide variety of systems and communication pathways. The goal of this paper was to assess the validity of a Web-based telepathology system for frozen section consultation within the Army Medical Department. The system provides real-time, dynamic remote control of a robotic microscope over standard Internet connections. Oftentimes, a solo pathologist is called on to provide diagnostic services without the support of immediate second or expert consultation during an intraoperative consultation. The use of telepathology is attractive because it provides an opportunity for pathologists to obtain immediate consultation. For purposes of the study, 120 consecutive frozen section cases were diagnosed at a distance using the system. Intraobserver agreement between the telepathology diagnosis and glass slide diagnosis was observed. Diagnostic agreement was 100% for a wide variety of specimens. This study suggests that such a system will help support pathologists located at distant sites.',\n",
              "  'authors': ['Kaplan, Keith J',\n",
              "   'Burgess, Jeanette R',\n",
              "   'Sandberg, Glenn D',\n",
              "   'Myers, Cris P',\n",
              "   'Bigott, Thomas R',\n",
              "   'Greenspan, Renata B'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc',\n",
              "  'keywords': ['Cytodiagnosis',\n",
              "   'Frozen Sections',\n",
              "   'Humans',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Telepathology'],\n",
              "  'pmid': '12429799',\n",
              "  'title': 'Use of robotic telepathology for frozen-section diagnosis: a retrospective trial of a telepathology system for intraoperative consultation.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['OP'],\n",
              "  'ESSN': '1556-4029',\n",
              "  'IF_2013': '1.306',\n",
              "  'IF_2014': '1.16',\n",
              "  'IF_2015': '1.322',\n",
              "  'IF_2016': '1.127',\n",
              "  'IF_2017': '1.184',\n",
              "  'ISSN': '0022-1198',\n",
              "  'ISSN_online': '1556-4029',\n",
              "  'ISSN_print': '0022-1198',\n",
              "  'IsoAbbr': 'J. Forensic Sci.',\n",
              "  'JrId': 4738,\n",
              "  'MedAbbr': 'J Forensic Sci',\n",
              "  'NLMid': '0375370',\n",
              "  'Titre': 'J. Forensic Sci.',\n",
              "  'abstract': 'This article describes a practical procedure for rapidly searching a large database of cocaine signatures to identify database entries that closely resemble a given reference cocaine exhibit using a personal computer (PC). The procedure takes advantage of the pattern recognition capability of the multilayer perceptron neural network to identify similar cocaine signatures. A PC-based software implementation is now being used on a daily basis at the North Carolina State Bureau of Investigation (NCSBI) to aid forensic experts in identifying signatures that originate from the same batch. Intelligence reports generated from database searches have been useful to undercover agents in the field who are striving to build drug related conspiracy cases. This software was developed as a collaborative effort between the NCSBI and the Center for Systems and Engineering of the Research Triangle Institute.',\n",
              "  'authors': ['Casale, J F', 'Watterson, J W'],\n",
              "  'categories': ['MEDICINE, LEGAL'],\n",
              "  'journal': 'Journal of forensic sciences',\n",
              "  'keywords': ['Chromatography, Gas',\n",
              "   'Cocaine',\n",
              "   'Microcomputers',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Software',\n",
              "   'Street Drugs'],\n",
              "  'pmid': '8454990',\n",
              "  'title': 'A computerized neural network method for pattern recognition of cocaine signatures.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RA'],\n",
              "  'ESSN': '2050-5701',\n",
              "  'IF_2013': '1.632',\n",
              "  'IF_2014': '1.633',\n",
              "  'IF_2015': '1.285',\n",
              "  'IF_2016': '1.528',\n",
              "  'IF_2017': '1.625',\n",
              "  'ISSN': '2050-5698',\n",
              "  'ISSN_online': '2050-5701',\n",
              "  'ISSN_print': '2050-5698',\n",
              "  'IsoAbbr': 'Microscopy (Oxf)',\n",
              "  'JrId': 41052,\n",
              "  'MedAbbr': 'Microscopy (Oxf)',\n",
              "  'NLMid': '101595834',\n",
              "  'Titre': 'Microscopy (Oxf)',\n",
              "  'abstract': 'Segmentation and detection of mitotic nuclei is a challenging task. To address this problem, a Transfer Learning based fast and accurate system is proposed. To give the classifier a balanced dataset, this work exploits the concept of Transfer Learning by first using a pre-trained convolutional neural network (CNN) for segmentation, and then another Hybrid-CNN (with Weights Transfer and custom layers) for classification of mitoses. First, mitotic nuclei are automatically annotated, based on the ground truth centroids. The segmentation module then segments mitotic nuclei and also produces some false positives. Finally, the detection module is trained on the patches from the segmentation module and performs the final detection. Fine-tuning based Transfer Learning reduced training time, provided good initial weights, and improved the detection rate with F-measure of 0.713 and 76% area under the precision-recall curve for the challenging task of mitosis detection.',\n",
              "  'authors': ['Wahab, Noorul', 'Khan, Asifullah', 'Lee, Yeon Soo'],\n",
              "  'categories': ['MICROSCOPY'],\n",
              "  'journal': 'Microscopy (Oxford, England)',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Automation, Laboratory',\n",
              "   'Breast Neoplasms',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Mitosis',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '30722018',\n",
              "  'title': 'Transfer learning based deep CNN for segmentation and detection of mitoses in breast cancer histopathological images.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1471-2970',\n",
              "  'IF_2013': '6.314',\n",
              "  'IF_2014': '7.055',\n",
              "  'IF_2015': '5.847',\n",
              "  'IF_2016': '5.846',\n",
              "  'IF_2017': '5.666',\n",
              "  'ISSN': '0962-8436',\n",
              "  'ISSN_online': '1471-2970',\n",
              "  'ISSN_print': '0962-8436',\n",
              "  'IsoAbbr': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'JrId': 8277,\n",
              "  'MedAbbr': 'Philos Trans R Soc Lond B Biol Sci',\n",
              "  'NLMid': '7503623',\n",
              "  'Titre': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'abstract': \"There is fundamental doubt about whether the natural unit of measurement for temperament and personality corresponds to single traits or to multi-trait profiles that describe the functioning of a whole person. Biogenetic researchers of temperament usually assume they need to focus on individual traits that differ between individuals. Recent research indicates that a shift of emphasis to understand processes within the individual is crucial for identifying the natural building blocks of temperament. Evolution and development operate on adaptation of whole organisms or persons, not on individual traits or categories. Adaptive functioning generally depends on feedback among many variable processes in ways that are characteristic of complex adaptive systems, not machines with separate parts. Advanced methods of unsupervised machine learning can now be applied to genome-wide association studies and brain imaging in order to uncover the genotypic-phenotypic architecture of traits like temperament, which are strongly influenced by complex interactions, such as genetic epistasis, pleiotropy and gene-environment interactions. We have found that the heritability of temperament can be nearly fully explained by a large number of genetic variants that are unique for multi-trait profiles, not single traits. The implications of this finding for research design and precision medicine are discussed.This article is part of the theme issue 'Diverse perspectives on diversity: multi-disciplinary approaches to taxonomies of individual differences'.\",\n",
              "  'authors': ['Cloninger, C Robert', 'Zwir, Igor'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Philosophical transactions of the Royal Society of London. Series B, Biological sciences',\n",
              "  'keywords': ['Cluster Analysis',\n",
              "   'Epistasis, Genetic',\n",
              "   'Gene-Environment Interaction',\n",
              "   'Genetic Pleiotropy',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Genotype',\n",
              "   'Humans',\n",
              "   'Individuality',\n",
              "   'Phenotype',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Quantitative Trait, Heritable',\n",
              "   'Schizophrenia',\n",
              "   'Schizotypal Personality Disorder',\n",
              "   'Temperament',\n",
              "   'Unsupervised Machine Learning'],\n",
              "  'pmid': '29483348',\n",
              "  'title': 'What is the natural measurement unit of temperament: single traits or profiles?',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1471-2261',\n",
              "  'IF_2013': '1.5',\n",
              "  'IF_2014': '1.878',\n",
              "  'IF_2015': '1.916',\n",
              "  'IF_2016': '1.832',\n",
              "  'IF_2017': '1.812',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2261',\n",
              "  'IsoAbbr': 'BMC Cardiovasc Disord',\n",
              "  'JrId': 22329,\n",
              "  'MedAbbr': 'BMC Cardiovasc Disord',\n",
              "  'NLMid': '100968539',\n",
              "  'Titre': 'BMC Cardiovasc Disord',\n",
              "  'abstract': 'In recent years a number of algorithms for cardiovascular risk assessment has been proposed to the medical community. These algorithms consider a number of variables and express their results as the percentage risk of developing a major fatal or non-fatal cardiovascular event in the following 10 to 20 years|The author has identified three major pitfalls of these algorithms, linked to the limitation of the classical statistical approach in dealing with this kind of non linear and complex information. The pitfalls are the inability to capture the disease complexity, the inability to capture process dynamics, and the wide confidence interval of individual risk assessment. Artificial Intelligence tools can provide potential advantage in trying to overcome these limitations. The theoretical background and some application examples related to artificial neural networks and fuzzy logic have been reviewed and discussed.|The use of predictive algorithms to assess individual absolute risk of cardiovascular future events is currently hampered by methodological and mathematical flaws. The use of newer approaches, such as fuzzy logic and artificial neural networks, linked to artificial intelligence, seems to better address both the challenge of increasing complexity resulting from a correlation between predisposing factors, data on the occurrence of cardiovascular events, and the prediction of future events on an individual level.',\n",
              "  'authors': ['Grossi, Enzo'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'BMC cardiovascular disorders',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Cardiovascular Diseases',\n",
              "   'Confidence Intervals',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Logistic Models',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Risk Assessment'],\n",
              "  'pmid': '16672045',\n",
              "  'title': 'How artificial intelligence tools can be used to assess individual patient risk in cardiovascular disease: problems with the current methods.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1423-0399',\n",
              "  'IF_2013': '1.151',\n",
              "  'IF_2014': '1.426',\n",
              "  'IF_2015': '1.313',\n",
              "  'IF_2016': '1.611',\n",
              "  'IF_2017': '1.508',\n",
              "  'ISSN': '0042-1138',\n",
              "  'ISSN_online': '1423-0399',\n",
              "  'ISSN_print': '0042-1138',\n",
              "  'IsoAbbr': 'Urol. Int.',\n",
              "  'JrId': 7898,\n",
              "  'MedAbbr': 'Urol Int',\n",
              "  'NLMid': '0417373',\n",
              "  'Titre': 'Urol. Int.',\n",
              "  'abstract': \"To present robotic repair of vesicouterine fistulae (VUF) with and without hysterectomy in 3 cases and to discuss the technique with its outcome.|Three patients were diagnosed with VUF, of whom 2 had a prior history of multiple cesarean sections and 1 had obstructed labor. Preoperative diagnosis of VUF was based on classic history, cystoscopy and imaging studies. All patients underwent pure robotic repair of VUF with hysterectomy in 1 case. The steps of the technique are cystoscopy, bilateral ureteral catheterization, port placement, adhesiolysis, separation of bladder and uterus and excision of the VUF with freshening of margins, closure of the uterus and bladder, hysterectomy in 1 case, and omental interposition.|Robotic repair of VUF was successful in all cases with mean operative time of 127.5 min and average blood loss of 120 ml. One patient underwent simultaneous robotic hysterectomy. All patients were ambulatory on day 1 and were discharged on day 3 with indwelling Foley's catheter, which was removed on day 10. conclusion: Robotic repair of VUF is safe and effective with successful outcome in all cases and has all the advantages of open and laparoscopic surgery. If required, concomitant robotic hysterectomy can also be performed. This is the first case series in the world.\",\n",
              "  'authors': ['Hemal, Ashok K', 'Sharma, Nitin', 'Mukherjee, Satyadip'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urologia internationalis',\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Fistula',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Robotics',\n",
              "   'Urinary Bladder Fistula',\n",
              "   'Urologic Surgical Procedures',\n",
              "   'Uterine Diseases'],\n",
              "  'pmid': '19506407',\n",
              "  'title': 'Robotic repair of complex vesicouterine fistula with and without hysterectomy.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"In recent years, real-time face recognition has been a major topic of interest in developing intelligent human-machine interaction systems. Over the past several decades, researchers have proposed different algorithms for facial expression recognition, but there has been little focus on detection in real-time scenarios. The present work proposes a new algorithmic method of automated marker placement used to classify six facial expressions: happiness, sadness, anger, fear, disgust, and surprise. Emotional facial expressions were captured using a webcam, while the proposed algorithm placed a set of eight virtual markers on each subject's face. Facial feature extraction methods, including marker distance (distance between each marker to the center of the face) and change in marker distance (change in distance between the original and new marker positions), were used to extract three statistical features (mean, variance, and root mean square) from the real-time video sequence. The initial position of each marker was subjected to the optical flow algorithm for marker tracking with each emotional facial expression. Finally, the extracted statistical features were mapped into corresponding emotional facial expressions using two simple non-linear classifiers, K-nearest neighbor and probabilistic neural network. The results indicate that the proposed automated marker placement algorithm effectively placed eight virtual markers on each subject's face and gave a maximum mean emotion classification rate of 96.94% using the probabilistic neural network. \",\n",
              "  'authors': ['Maruthapillai, Vasanthan', 'Murugappan, Murugappan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biometric Identification',\n",
              "   'Emotions',\n",
              "   'Eye',\n",
              "   'Face',\n",
              "   'Facial Expression',\n",
              "   'Humans',\n",
              "   'Models, Statistical'],\n",
              "  'pmid': '26859884',\n",
              "  'title': 'Optimal Geometrical Set for Automated Marker Placement to Virtualized Real-Time Facial Emotions.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'Radiologists routinely compare multiple chest radiographs acquired from the same patient over time to more completely understand changes in anatomy and pathology. While such comparisons are achieved conventionally through a side-by-side display of images, image registration techniques have been developed to combine information from two separate radiographic images through construction of a \"temporal subtraction image.\" Although temporal subtraction images provide a powerful mechanism for the enhanced visualization of subtle change, errors in the clinical evaluation of these images may arise from misregistration artifacts that can mimic or obscure pathologic change. We have developed a computerized method for the automated assessment of registration accuracy as demonstrated in temporal subtraction images created from radiographic chest image pairs. The registration accuracy of 150 temporal subtraction images constructed from the computed radiography images of 72 patients was rated manually using a five-point scale ranging from \"5-excellent\" to \"1-poor;\" ratings of 3, 4, or 5 reflected clinically acceptable subtraction images, and ratings of 1 or 2 reflected clinically unacceptable images. Gray-level histogram-based features and texture measures are computed at multiple spatial scales within a \"lung mask\" region that encompasses both lungs in the temporal subtraction images. A subset of these features is merged through a linear discriminant classifier. With a leave-one-out-by-patient training/testing paradigm, the automated method attained an A(z) value of 0.92 in distinguishing between temporal subtraction images that demonstrated clinically acceptable and clinically unacceptable registration accuracy. A second linear discriminant classifier yielded an A(z) value of 0.82 based on a feature subset selected from an independent database of digitized film images. These methods are expected to advance the clinical utility of temporal subtraction images for chest radiography.',\n",
              "  'authors': ['Armato, Samuel G',\n",
              "   'Doshi, Devang J',\n",
              "   'Engelmann, Roger',\n",
              "   'Croteau, Charles L',\n",
              "   'MacMahon, Heber'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Radiography, Thoracic',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Subtraction Technique',\n",
              "   'Time Factors'],\n",
              "  'pmid': '16752558',\n",
              "  'title': 'Temporal subtraction in chest radiography: automated assessment of registration accuracy.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1943-2631',\n",
              "  'IF_2013': '4.866',\n",
              "  'IF_2014': '5.963',\n",
              "  'IF_2015': '4.644',\n",
              "  'IF_2016': '4.556',\n",
              "  'IF_2017': '4.075',\n",
              "  'ISSN': '0016-6731',\n",
              "  'ISSN_online': '1943-2631',\n",
              "  'ISSN_print': '0016-6731',\n",
              "  'IsoAbbr': 'Genetics',\n",
              "  'JrId': 3871,\n",
              "  'MedAbbr': 'Genetics',\n",
              "  'NLMid': '0374636',\n",
              "  'Titre': 'Genetics',\n",
              "  'abstract': 'Using a reduced subset of SNPs in a linear mixed model can improve power for genome-wide association studies, yet this can result in insufficient correction for population stratification. We propose a hybrid approach using principal components that does not inflate statistics in the presence of population stratification and improves power over standard linear mixed models. ',\n",
              "  'authors': ['Tucker, George', 'Price, Alkes L', 'Berger, Bonnie'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Genetics',\n",
              "  'keywords': ['Confounding Factors (Epidemiology)',\n",
              "   'Genetics, Population',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Models, Genetic',\n",
              "   'Multiple Sclerosis',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Principal Component Analysis'],\n",
              "  'pmid': '24788602',\n",
              "  'title': 'Improving the power of GWAS and avoiding confounding from population stratification with PC-Select.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1097-0150',\n",
              "  'IF_2013': '1.077',\n",
              "  'IF_2014': '0.69',\n",
              "  'IF_2015': '0.606',\n",
              "  'IF_2016': '1.667',\n",
              "  'IF_2017': '1.2',\n",
              "  'ISSN': '1092-9088',\n",
              "  'ISSN_online': '1097-0150',\n",
              "  'ISSN_print': '1092-9088',\n",
              "  'IsoAbbr': 'Comput. Aided Surg.',\n",
              "  'JrId': 20599,\n",
              "  'MedAbbr': 'Comput Aided Surg',\n",
              "  'NLMid': '9708375',\n",
              "  'Titre': 'Comput. Aided Surg.',\n",
              "  'abstract': 'To determine if image-guided robotic stereotactic radiosurgery (IGR-SRS) by CyberKnife achieves acceptable local control in resectable but medically inoperable patients with non-small cell lung cancer (NSCLC) or pulmonary metastasis, and to evaluate control rates and toxicity.|Treatment details and outcomes were reviewed for 95 patients (age range 33-96 years) with 136 histologically proven cancers treated by IGR-SRS at the CyberKnife Center of Miami between March 2004 and March 2007. Tumor volumes ranged from 1.2 cc to 338 cc. Targeting was accomplished using combined skeletal alignment and real-time tracking via fiducials placed within the tumor. Total doses ranged from 15 to 67.5 Gy delivered in 1 to 5 fractions.|Of the 95 patients treated, 78 (82%) are still alive at 1 to 36 months post-treatment. Nineteen patients have died, four from disease other than cancer progression. All patients but one achieved at least partial response to treatment and tolerated radiosurgery well. For the majority of our patients, fatigue had been the main side effect.|The delivery of precisely targeted high radiation doses with surgical precision to lung tumors in a hypo-fractionated fashion is feasible and safe. Image-guided robotic stereotactic radiosurgery (IGR-SRS) of lung tumors with the CyberKnife achieves excellent rates of local disease control with limited toxicity to surrounding tissues, and in many cases may be curative for patients for whom surgery is not an option.',\n",
              "  'authors': ['Brown, William T',\n",
              "   'Wu, Xiaodong',\n",
              "   'Wen, B-Chen',\n",
              "   'Fowler, John F',\n",
              "   'Fayad, Fahed',\n",
              "   'Amendola, Beatriz E',\n",
              "   'García, Silvio',\n",
              "   'De La Zerda, Alberto',\n",
              "   'Huang, Zhicong',\n",
              "   'Schwade, James G'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer aided surgery : official journal of the International Society for Computer Aided Surgery',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Carcinoma, Non-Small-Cell Lung',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Lung Neoplasms',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Staging',\n",
              "   'Radiosurgery',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '17957532',\n",
              "  'title': 'Early results of CyberKnife image-guided robotic stereotactic radiosurgery for treatment of lung tumors.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZD'],\n",
              "  'ESSN': '1940-1574',\n",
              "  'IF_2013': '2.37',\n",
              "  'IF_2014': '2.97',\n",
              "  'IF_2015': '2.931',\n",
              "  'IF_2016': '3.085',\n",
              "  'IF_2017': '3.022',\n",
              "  'ISSN': '0003-3197',\n",
              "  'ISSN_online': '1940-1574',\n",
              "  'ISSN_print': '0003-3197',\n",
              "  'IsoAbbr': 'Angiology',\n",
              "  'JrId': 541,\n",
              "  'MedAbbr': 'Angiology',\n",
              "  'NLMid': '0203706',\n",
              "  'Titre': 'Angiology',\n",
              "  'abstract': 'Currently, there is a growing interest in minimally invasive cardiac surgery, and despite early criticisms, it has become the preferred method of mitral valve repair and replacement in many institutions worldwide with excellent results. The interest in performing cardiac valve operations through minimal incisions was stimulated by Port Access technology and has evolved to include robotically assisted video-enhanced valve surgery. Robotic assistance has led to shorter operating times and represents an ideal tool to prepare for fully robotic-assisted cardiac procedures. This report will highlight minimally invasive mitral valve surgery with its evolution from Port Access techniques to fully robotic-assisted surgery. The nuances, strengths, and shortcomings, as well as the potential to enhance the valvular procedure, the promise to reduce hospital stay, earlier return to normal activity, less pain, better cosmesis, and the rethinking of surgical dogma that wide surgical exposure is essential for such complex intracardiac surgery are discussed.',\n",
              "  'authors': ['Donias, Harry W',\n",
              "   'Karamanoukian, Hratch L',\n",
              "   \"D'Ancona, Giuseppe\",\n",
              "   'Hoover, Eddie L'],\n",
              "  'categories': ['PERIPHERAL VASCULAR DISEASE'],\n",
              "  'journal': 'Angiology',\n",
              "  'keywords': ['Cardiac Catheterization',\n",
              "   'Heart Valve Diseases',\n",
              "   'Humans',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Mitral Valve',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '12593501',\n",
              "  'title': 'Minimally invasive mitral valve surgery: from Port Access to fully robotic-assisted surgery.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1433-8726',\n",
              "  'IF_2013': '3.423',\n",
              "  'IF_2014': '2.666',\n",
              "  'IF_2015': '2.397',\n",
              "  'IF_2016': '2.743',\n",
              "  'IF_2017': '2.981',\n",
              "  'ISSN': '0724-4983',\n",
              "  'ISSN_online': '1433-8726',\n",
              "  'ISSN_print': '0724-4983',\n",
              "  'IsoAbbr': 'World J Urol',\n",
              "  'JrId': 2429,\n",
              "  'MedAbbr': 'World J Urol',\n",
              "  'NLMid': '8307716',\n",
              "  'Titre': 'World J Urol',\n",
              "  'abstract': 'The robotic technique, which was first introduced in laparoscopic heart surgery, has revolutionized laparoscopic surgery over the last 5 years. In May 2000, our department accomplished the first robot assisted laparoscopic radical prostatectomy. Since that time we have performed more than 118 such procedures and several other laparoscopic operations using the robotic technique. We here summarize our experience in robot assisted laparoscopic radical prostatectomy as it has been developed over the past 3 years. Between May 2000 and May 2003, 118 patients with clinically localized prostate cancer were operated using the telerobotic da Vinci Surgical System. Operations were performed with a senior surgeon at the console, assisted by an assistant and a nurse at the operating table. Bilateral pelvic lymph node dissection was undertaken as a first step in all patients. In the initial 60 cases, we investigated different laparoscopic approaches. We used transperitoneal as well as extraperitoneal approaches. For dissection of the prostate we used ascending, descending as well as combined techniques. The combined ascending and descending technique via the transperitoneal route was chosen in 30 patients, and via the extraperitoneal route in seven patients. A modification of the descending Montsouris technique was performed in 81 patients. The robot assisted laparoscopic radical prostatectomy with the da Vinci system has been well standardized. After performing more than 100 radical prostatectomies with this system, we conclude that in our hands the Montsouris technique with only minor adoptions is the most appropriate technique for performing robot assisted radical prostatectomy.',\n",
              "  'authors': ['Wolfram, M',\n",
              "   'Bräutigam, R',\n",
              "   'Engl, T',\n",
              "   'Bentas, W',\n",
              "   'Heitkamp, S',\n",
              "   'Ostwald, M',\n",
              "   'Kramer, W',\n",
              "   'Binder, J',\n",
              "   'Blaheta, R',\n",
              "   'Jonas, D',\n",
              "   'Beecken, W-D'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'World journal of urology',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics'],\n",
              "  'pmid': '12851781',\n",
              "  'title': 'Robotic-assisted laparoscopic radical prostatectomy: the Frankfurt technique.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': \"The inverse problem in electrophysiology consists of the accurate estimation of the intracardiac electrical sources from a reduced set of electrodes at short distances and from outside the heart. This estimation can provide an image with relevant knowledge on arrhythmia mechanisms for the clinical practice. Methods based on truncated singular value decomposition (TSVD) and regularized least squares require a matrix inversion, which limits their resolution due to the unavoidable low-pass filter effect of the Tikhonov regularization techniques.|We propose to use, for the first time, a Mercer's kernel given by the Laplacian of the distance in the quasielectrostatic field equations, hence providing a Support Vector Regression (SVR) formulation by following the principles of the Dual Signal Model (DSM) principles for creating kernel algorithms.|Simulations in one- and two-dimensional models show the performance of our Laplacian distance kernel technique versus several conventional methods. Firstly, the one-dimensional model is adjusted for yielding recorded electrograms, similar to the ones that are usually observed in electrophysiological studies, and suitable strategy is designed for the free-parameter search. Secondly, simulations both in one- and two-dimensional models show larger noise sensitivity in the estimated transfer matrix than in the observation measurements, and DSM-SVR is shown to be more robust to noisy transfer matrix than TSVD.|These results suggest that our proposed DSM-SVR with Laplacian distance kernel can be an efficient alternative to improve the resolution in current and emerging intracardiac imaging systems.\",\n",
              "  'authors': ['Caulier-Cisterna, Raúl',\n",
              "   'Muñoz-Romero, Sergio',\n",
              "   'Sanromán-Junquera, Margarita',\n",
              "   'García-Alberola, Arcadi',\n",
              "   'Rojo-Álvarez, José Luis'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Electroencephalography',\n",
              "   'Electrophysiological Phenomena',\n",
              "   'Heart',\n",
              "   'Least-Squares Analysis',\n",
              "   'Models, Cardiovascular',\n",
              "   'Signal-To-Noise Ratio',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29925384',\n",
              "  'title': 'A new approach to the intracardiac inverse problem using Laplacian distance kernel.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5305',\n",
              "  'IF_2013': '1.735',\n",
              "  'IF_2014': '1.068',\n",
              "  'IF_2015': '1.069',\n",
              "  'IF_2016': '0.964',\n",
              "  'IF_2017': '1.039',\n",
              "  'ISSN': '0929-8665',\n",
              "  'ISSN_online': '1875-5305',\n",
              "  'ISSN_print': '0929-8665',\n",
              "  'IsoAbbr': 'Protein Pept. Lett.',\n",
              "  'JrId': 20398,\n",
              "  'MedAbbr': 'Protein Pept Lett',\n",
              "  'NLMid': '9441434',\n",
              "  'Titre': 'Protein Pept. Lett.',\n",
              "  'abstract': 'Hepatitis C virus (HCV) infection is a major cause of liver disease and a dangerous threat to public health. Hence, the problem of finding interactions between HCV and human proteins has received much attention. In this paper, we present an approach to predicting binding residues in HCV proteins using a support vector machine (SVM) classifier. Based on six biochemical properties of amino acids (sequence profile, accessible surface area, residue binding propensity, sequence entropy, hydrophobicity and conservation weight), the SVM classifier achieved an average accuracy of 93%. Contiguous residues in the sequence act together to determine a binding site, and a window of 11 residues (the target residue and 5 adjacent residues on each side) gave the best result in our study. Our approach has been implemented in a program called BSFinder (Binding Site Finder), which is available at http://wilab.inha.ac.kr/bsfinder. BSFinder will be of considerable help in predicting binding residues and potential interacting partners of a protein.',\n",
              "  'authors': ['Chen, Yu', 'Han, Kyungsook'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Protein and peptide letters',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Artificial Intelligence',\n",
              "   'Binding Sites',\n",
              "   'Computational Biology',\n",
              "   'Hepacivirus',\n",
              "   'Models, Molecular',\n",
              "   'Software',\n",
              "   'Viral Proteins'],\n",
              "  'pmid': '19356134',\n",
              "  'title': 'BSFINDER: finding binding sites of HCV proteins using a support vector machine.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Imaging of cancer with 18F-fluorodeoxyglucose positron emission tomography (18F-FDG PET) has become a standard component of diagnosis and staging in oncology, and is becoming more important as a quantitative monitor of individual response to therapy. In this article we investigate the challenging problem of predicting a patient\\'s response to neoadjuvant chemotherapy from a single 18F-FDG PET scan taken prior to treatment. We take a \"radiomics\" approach whereby a large amount of quantitative features is automatically extracted from pretherapy PET images in order to build a comprehensive quantification of the tumor phenotype. While the dominant methodology relies on hand-crafted texture features, we explore the potential of automatically learning low- to high-level features directly from PET scans. We report on a study that compares the performance of two competing radiomics strategies: an approach based on state-of-the-art statistical classifiers using over 100 quantitative imaging descriptors, including texture features as well as standardized uptake values, and a convolutional neural network, 3S-CNN, trained directly from PET scans by taking sets of adjacent intra-tumor slices. Our experimental results, based on a sample of 107 patients with esophageal cancer, provide initial evidence that convolutional neural networks have the potential to extract PET imaging representations that are highly predictive of response to therapy. On this dataset, 3S-CNN achieves an average 80.7% sensitivity and 81.6% specificity in predicting non-responders, and outperforms other competing predictive models. ',\n",
              "  'authors': ['Ypsilantis, Petros-Pavlos',\n",
              "   'Siddique, Musib',\n",
              "   'Sohn, Hyon-Mok',\n",
              "   'Davies, Andrew',\n",
              "   'Cook, Gary',\n",
              "   'Goh, Vicky',\n",
              "   'Montana, Giovanni'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Esophageal Neoplasms',\n",
              "   'Female',\n",
              "   'Fluorodeoxyglucose F18',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoadjuvant Therapy',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '26355298',\n",
              "  'title': 'Predicting Response to Neoadjuvant Chemotherapy with PET Imaging Using Convolutional Neural Networks.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Rice is a staple food in Asia and it contributes significantly to the Gross Domestic Product (GDP) of Malaysia and other developing countries. Brown Planthopper (BPH) causes high levels of economic loss in Malaysia. Identification of BPH presence and monitoring of its abundance has been conducted manually by experts and is time-consuming, fatiguing and tedious. Automated detection of BPH has been proposed by many studies to overcome human fallibility. However, all studies regarding automated recognition of BPH are investigated based on intact specimen although most of the specimens are imperfect, with missing parts have distorted shapes. The automated recognition of an imperfect insect image is more difficult than recognition of the intact specimen. This study proposes an automated, deep-learning-based detection pipeline, PENYEK, to identify BPH pest in images taken from a readily available sticky pad, constructed by clipping plastic sheets onto steel plates and spraying with glue. This study explores the effectiveness of a convolutional neural network (CNN) architecture, VGG16, in classifying insects as BPH or benign based on grayscale images constructed from Euclidean Distance Maps (EDM). The pipeline identified imperfect images of BPH with an accuracy of 95% using deep-learning's hyperparameters: softmax, a mini-batch of 30 and an initial learning rate of 0.0001.\",\n",
              "  'authors': ['Nazri, Azree', 'Mazlan, Norida', 'Muharam, Farrah'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Agriculture',\n",
              "   'Algorithms',\n",
              "   'Animals',\n",
              "   'Deep Learning',\n",
              "   'Electronic Data Processing',\n",
              "   'Environmental Monitoring',\n",
              "   'Humans',\n",
              "   'Insect Control',\n",
              "   'Insecta',\n",
              "   'Malaysia',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oryza',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Software'],\n",
              "  'pmid': '30571683',\n",
              "  'title': 'PENYEK: Automated brown planthopper detection from imperfect sticky pad images using deep convolutional neural network.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Accurate prediction of the pathogenicity of genomic variants, especially nonsynonymous single nucleotide variants (nsSNVs), is essential in biomedical research and clinical genetics. Most current prediction methods build a generic classifier for all genes. However, different genes and gene families have different features. We investigated whether gene-specific and family-specific customized classifiers could improve prediction accuracy. Customized gene-specific and family-specific attributes were selected with AIC, BIC, and LASSO, and Support Vector Machine classifiers were generated for 254 genes and 152 gene families, covering a total of 5,985 genes. Our results showed that the customized attributes reflected key features of the genes and gene families, and the customized classifiers achieved higher prediction accuracy than the generic classifier. The customized classifiers and the generic classifier for other genes and families were integrated into a new tool named iFish (integrated Functional inference of SNVs in human, http://ifish.cbi.pku.edu.cn). iFish outperformed other methods on benchmark datasets as well as on prioritization of candidate causal variants from whole exome sequencing. iFish provides a user-friendly web-based interface and supports other functionalities such as integration of genetic evidence. iFish would facilitate high-throughput evaluation and prioritization of nsSNVs in human genetics research.',\n",
              "  'authors': ['Wang, Meng', 'Wei, Liping'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Humans',\n",
              "   'Multigene Family',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '27527004',\n",
              "  'title': 'iFish: predicting the pathogenicity of human nonsynonymous variants using gene-specific/family-specific attributes and classifiers.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'To determine whether Cyberknife technology can be applied to renal tissue safely and effectively. The goal was to achieve the high efficacy of a surgical treatment, with the low morbidity of a noninvasive intervention.|The Cyberknife is a frameless, image-guided radiosurgical device. This innovative extracorporeal treatment combines a linear accelerator mounted on a highly maneuverable robotic arm. The Cyberknife is unique in that it divides the high-dose radiation necessary to ablate the lesion completely into up to 1200 beams. Each one of these beams of radiation has a significantly reduced dose. Therefore, the individual dose of each beam is essentially benign to the pathway and surrounding tissue. However, at the focal point of these beams, the dose is additive, and the desired ablative dose is attained. Predetermined \"lesions\" in 16 kidneys were treated in vivo in the porcine model. Complete treatment was accomplished in one session per animal, with no complications. Gross and histologic evaluations were completed at 4, 6, or 8 weeks.|The degree of radiation changes correlated with longer treatment intervals. After 8 weeks, the lesions showed complete fibrosis. The zones of complete fibrosis were characterized by dense, paucicellular connective tissue completely devoid of all normal kidney elements, including tubules and glomeruli.|This initial preclinical evaluation of the Cyberknife for extracorporeal renal tissue ablation appears to be very promising and demonstrated its ability to ablate a targeted area precisely and completely with relative sparing of the surrounding tissue. This innovative technology introduces an exciting approach as a potential treatment option of renal masses in the future.',\n",
              "  'authors': ['Ponsky, Lee E',\n",
              "   'Crownover, Richard L',\n",
              "   'Rosen, Michael J',\n",
              "   'Rodebaugh, Raymond F',\n",
              "   'Castilla, Elias A',\n",
              "   'Brainard, Jennifer',\n",
              "   'Cherullo, Edward E',\n",
              "   'Novick, Andrew C'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Animals',\n",
              "   'Carcinoma, Renal Cell',\n",
              "   'Evaluation Studies as Topic',\n",
              "   'Female',\n",
              "   'Kidney',\n",
              "   'Kidney Neoplasms',\n",
              "   'Models, Animal',\n",
              "   'Particle Accelerators',\n",
              "   'Radiosurgery',\n",
              "   'Radiotherapy, Computer-Assisted',\n",
              "   'Radiotherapy, Conformal',\n",
              "   'Robotics',\n",
              "   'Stereotaxic Techniques',\n",
              "   'Swine'],\n",
              "  'pmid': '12639631',\n",
              "  'title': 'Initial evaluation of Cyberknife technology for extracorporeal renal tissue ablation.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1526-9914',\n",
              "  'IF_2013': '1.11',\n",
              "  'IF_2014': '1.172',\n",
              "  'IF_2015': '1.444',\n",
              "  'IF_2016': '1.338',\n",
              "  'IF_2017': '1.301',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1526-9914',\n",
              "  'IsoAbbr': 'J Appl Clin Med Phys',\n",
              "  'JrId': 22471,\n",
              "  'MedAbbr': 'J Appl Clin Med Phys',\n",
              "  'NLMid': '101089176',\n",
              "  'Titre': 'J Appl Clin Med Phys',\n",
              "  'abstract': 'We have automated the determination of the placement location of the dosimetry ion chamber within intensity-modulated radiotherapy (IMRT) fields, as part of streamlining the entire IMRT quality assurance process. This paper describes the mathematical image-processing techniques to arrive at the appropriate measurement locations within the planar dose maps of the IMRT fields. A specific spot within the found region is identified based on its flatness, radiation magnitude, location, area, and the avoidance of the interleaf spaces. The techniques used include applying a Laplacian, dilation, erosion, region identification, and measurement point selection based on three parameters: the size of the erosion operator, the gradient, and the importance of the area of a region versus its magnitude. These three parameters are adjustable by the user. However, the first one requires tweaking in extremely rare occasions, the gradient requires rare adjustments, and the last parameter needs occasional fine-tuning. This algorithm has been tested in over 50 cases. In about 5% of cases, the algorithm does not find a measurement point due to the extremely steep and narrow regions within the fluence maps. In such cases, manual selection of a point is allowed by our code, which is also difficult to ascertain, since the fluence map does not yield itself to an appropriate measurement point selection.',\n",
              "  'authors': ['Zaini, Mehran R', 'Forest, Gary J', 'Loshek, David D'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of applied clinical medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Quality Assurance, Health Care',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Radiometry',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Radiotherapy, Conformal',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '16421497',\n",
              "  'title': 'Image processing for IMRT QA dosimetry.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'Investigators have reported that men undergoing radical retropubic prostatectomy (RRP) for prostate cancer (PCa) can experience significant changes in nocturia after surgery. We examined the effect of robot-assisted laparoscopic prostatectomy (RALP) on nocturia symptoms in men with PCa from baseline to 1-year follow-up.|Between August 2006 and August 2010, 116 patients undergoing RALP for clinically localized PCa had baseline and 1-year Expanded Prostate Cancer Index Composite (EPIC) questionnaire data obtained. Patients were divided into three groups with respect to nocturia: N1 (0 or 1 episode per night), N2 (2 episodes per night), and N3 (3 or more episodes per night).|N1 (63 patients) had stable or worsened nocturia with 78% of patients retaining their N1 status, 21% progressed to N2 status, and 2% progressed to N3. N2 (29 patients) were 52% N1, 34% N2, and 14% N3 after surgery. The N3 (24 patients) had 29% improved to N1, 38% improved to N2, and 33% remained N3. EPIC urinary function and incontinence subscale scores were 92.3 and 91.3 preintervention and 85.2 and 76.6, (P=<0.001 for both), respectively, at follow-up. The combination of improved obstructive symptomatology and continence after RALP resulted in no net change in urinary bother.|RALP is associated with improved symptomatology in patients with the greatest level of preinterventional nocturia. Patients with minimal preoperative nocturia may experience worsening of nocturia.',\n",
              "  'authors': ['Hutchinson, Ryan C',\n",
              "   'Thiel, David D',\n",
              "   'Tavlarides, Andrea M',\n",
              "   'Diehl, Nancy N',\n",
              "   'Parker, Alexander S'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nocturia',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Surveys and Questionnaires'],\n",
              "  'pmid': '22332794',\n",
              "  'title': 'The effect of robot-assisted laparoscopic prostatectomy on nocturia.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Each year more than 7 million people die from cardiac arrhythmias. Yet no robust solution exists today to detect such heart anomalies right at the moment they occur. The purpose of this study was to design a personalized health monitoring system that can detect early occurrences of arrhythmias from an individual's electrocardiogram (ECG) signal. We first modelled the common causes of arrhythmias in the signal domain as a degradation of normal ECG beats to abnormal beats. Using the degradation models, we performed abnormal beat synthesis which created potential abnormal beats from the average normal beat of the individual. Finally, a Convolutional Neural Network (CNN) was trained using real normal and synthesized abnormal beats. As a personalized classifier, the trained CNN can monitor ECG beats in real time for arrhythmia detection. Over 34 patients' ECG records with a total of 63,341 ECG beats from the MIT-BIH arrhythmia benchmark database, we have shown that the probability of detecting one or more abnormal ECG beats among the first three occurrences is higher than 99.4% with a very low false-alarm rate.\",\n",
              "  'authors': ['Kiranyaz, Serkan', 'Ince, Turker', 'Gabbouj, Moncef'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Arrhythmias, Cardiac',\n",
              "   'Databases, Factual',\n",
              "   'Electrocardiography',\n",
              "   'Humans',\n",
              "   'Monitoring, Physiologic',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Precision Medicine',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '28839215',\n",
              "  'title': 'Personalized Monitoring and Advance Warning System for Cardiac Arrhythmias.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'We consider the problem of how to recover the state and parameter values of typical model neurons, such as Hindmarsh-Rose, FitzHugh-Nagumo, Morris-Lecar, from in-vitro measurements of membrane potentials. In control theory, in terms of observer design, model neurons qualify as locally observable. However, unlike most models traditionally addressed in control theory, no parameter-independent diffeomorphism exists, such that the original model equations can be transformed into adaptive canonic observer form. For a large class of model neurons, however, state and parameter reconstruction is possible nevertheless. We propose a method which, subject to mild conditions on the richness of the measured signal, allows model parameters and state variables to be reconstructed up to an equivalence class.',\n",
              "  'authors': ['Tyukin, Ivan',\n",
              "   'Steur, Erik',\n",
              "   'Nijmeijer, Henk',\n",
              "   'Fairhurst, David',\n",
              "   'Song, Inseon',\n",
              "   'Semyanov, Alexey',\n",
              "   'Van Leeuwen, Cees'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Algorithms',\n",
              "   'Biological Clocks',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '20556847',\n",
              "  'title': 'State and parameter estimation for canonic models of neural oscillators.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Image-guided neurosurgery has become a standard practice in the last few years, with more than 2000 surgical navigation stations installed worldwide. In the same time several reports have also demonstrated the efficacy and accuracy of computed tomography angiography (CTA) in assessing cerebral vascular pathologies. Therefore, the CTA data have recently been implemented into the different navigation systems available on the market, making this new technique widely applied. The objective of this paper is to discuss and evaluate the clinical usefulness of navigated CTA in planning and performing surgery of neurovascular lesions.|Raw images acquired from an helical CTA are automatically post-processed on an independent workstation by using a three-dimensional (3D) volume-rendering images engine and/or using thresholding and drawing tools.|The data obtained provide useful information in the preoperative stage by reconstructing the vascular tree with regard to lesion volume, aneurysm neck, dome projection, perforating vessels and their relationship with the lesion and the surrounding anatomy. Furthermore, it can help in the identification of an arteriovenous malformation (AVM) nidus and recognition of its feeding and draining vessels.|This fascinating technique can give some invaluable advantages on the management of cerebral vascular lesions and provides excellent information not always available on traditional digital subtraction angiography investigation. It has also proved to be very accurate, particularly regarding the correlation between the 3D volume-rendered CT angiography and the intraoperative findings.',\n",
              "  'authors': ['Chibbaro, S', 'Tacconi, L'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Cerebral Angiography',\n",
              "   'Cerebrovascular Disorders',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Microsurgery',\n",
              "   'Neuronavigation',\n",
              "   'Robotics',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '17520627',\n",
              "  'title': 'Image-guided microneurosurgical management of vascular lesions using navigated computed tomography angiography. An advanced IGS technology application.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1546-1726',\n",
              "  'IF_2013': '14.976',\n",
              "  'IF_2014': '16.095',\n",
              "  'IF_2015': '16.724',\n",
              "  'IF_2016': '17.839',\n",
              "  'IF_2017': '19.912',\n",
              "  'ISSN': '1097-6256',\n",
              "  'ISSN_online': '1546-1726',\n",
              "  'ISSN_print': '1097-6256',\n",
              "  'IsoAbbr': 'Nat. Neurosci.',\n",
              "  'JrId': 21092,\n",
              "  'MedAbbr': 'Nat Neurosci',\n",
              "  'NLMid': '9809671',\n",
              "  'Titre': 'Nat. Neurosci.',\n",
              "  'abstract': 'Microcircuits composed of principal neuron and interneuron dendrites have an important role in shaping the representation of sensory information in the olfactory bulb. Here we establish the physiological features governing synaptic signaling in dendrodendritic microcircuits of olfactory bulb glomeruli. We show that dendritic gamma-aminobutyric acid (GABA) release from periglomerular neurons mediates inhibition of principal tufted cells, retrograde inhibition of sensory input and lateral signaling onto neighboring periglomerular cells. We find that L-type dendritic Ca(2+) spikes in periglomerular cells underlie dendrodendritic transmission by depolarizing periglomerular dendrites and activating P/Q type channels that trigger GABA release. Ca(2+) spikes in periglomerular cells are evoked by powerful excitatory inputs from a single principal cell, and glutamate release from the dendrites of single principal neurons activates a large ensemble of periglomerular cells.',\n",
              "  'authors': ['Murphy, Gabe J', 'Darcy, Daniel P', 'Isaacson, Jeffry S'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Nature neuroscience',\n",
              "  'keywords': ['3-Pyridinecarboxylic acid, 1,4-dihydro-2,6-dimethyl-5-nitro-4-(2-(trifluoromethyl)phenyl)-, Methyl ester',\n",
              "   'Animals',\n",
              "   'Animals, Newborn',\n",
              "   'Bicuculline',\n",
              "   'Cadmium',\n",
              "   'Calcium',\n",
              "   'Calcium Channel Agonists',\n",
              "   'Calcium Channel Blockers',\n",
              "   'Calcium Signaling',\n",
              "   'Chelating Agents',\n",
              "   'Dendrites',\n",
              "   'Diagnostic Imaging',\n",
              "   'Dihydropyridines',\n",
              "   'Dose-Response Relationship, Drug',\n",
              "   'Drug Interactions',\n",
              "   'Egtazic Acid',\n",
              "   'Electric Stimulation',\n",
              "   'GABA Antagonists',\n",
              "   'In Vitro Techniques',\n",
              "   'Membrane Potentials',\n",
              "   'Neural Conduction',\n",
              "   'Neural Inhibition',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nickel',\n",
              "   'Nimodipine',\n",
              "   'Olfactory Bulb',\n",
              "   'Patch-Clamp Techniques',\n",
              "   'Phosphinic Acids',\n",
              "   'Potassium',\n",
              "   'Propanolamines',\n",
              "   'Pyrimidines',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Signal Transduction',\n",
              "   'Synapses',\n",
              "   'Synaptic Transmission',\n",
              "   'Tetrodotoxin',\n",
              "   'Time Factors',\n",
              "   'Valine',\n",
              "   'gamma-Aminobutyric Acid'],\n",
              "  'pmid': '15696160',\n",
              "  'title': 'Intraglomerular inhibition: signaling mechanisms of an olfactory microcircuit.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1569-9285',\n",
              "  'IF_2013': '1.109',\n",
              "  'IF_2014': '1.155',\n",
              "  'IF_2015': '1.329',\n",
              "  'IF_2016': '1.857',\n",
              "  'IF_2017': '1.756',\n",
              "  'ISSN': '1569-9293',\n",
              "  'ISSN_online': '1569-9285',\n",
              "  'ISSN_print': '1569-9293',\n",
              "  'IsoAbbr': 'Interact Cardiovasc Thorac Surg',\n",
              "  'JrId': 33507,\n",
              "  'MedAbbr': 'Interact Cardiovasc Thorac Surg',\n",
              "  'NLMid': '101158399',\n",
              "  'Titre': 'Interact Cardiovasc Thorac Surg',\n",
              "  'abstract': 'The tumour-node metastasis (TNM) classification system is anatomically based. We investigated whether the addition of simple physiological variables, age and body mass index (BMI), would affect survival curves, i.e. a composite anatomical and physiological staging system. We retrospectively analysed a prospectively validated thoracic surgery database (n = 1981). Cox multivariate analysis was performed to determine possible significant factors. Kaplan-Meier survival curves were constructed with combined anatomical and physiological factors. Cox multivariate analysis revealed age (P < 0.001) and BMI (P = 0.01) as significant factors affecting survival. Receiver operating curve analysis determined cut-off levels for age of 67 and BMI of 27.6. A composite anatomical and physiological survival curve based on TNM for BMI > 27.6 and age < 67 was produced. Age and BMI criteria resulted in significantly different survival curves, for stage I (P < 0.0001) and stage II (P = 0.0032), but not for stage III (P = 0.06). Neural network analysis confirmed the importance of BMI and age above cancer stage with regard to long-term survival. Combining age < 67, BMI > 27.6 and TNM anatomical classification results in very different estimated survival curves from the usual TNM system. Patients from stages I, II and III may have survival equivalent to a stage higher or lower depending on their age and BMI.',\n",
              "  'authors': ['Poullis, Michael',\n",
              "   'McShane, James',\n",
              "   'Shaw, Mathew',\n",
              "   'Woolley, Steven',\n",
              "   'Shackcloth, Michael',\n",
              "   'Page, Richard',\n",
              "   'Mediratta, Neeraj'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Interactive cardiovascular and thoracic surgery',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Aged',\n",
              "   'Biopsy',\n",
              "   'Body Mass Index',\n",
              "   'England',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'Lung Neoplasms',\n",
              "   'Lymph Nodes',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Multivariate Analysis',\n",
              "   'Neoplasm Staging',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Predictive Value of Tests',\n",
              "   'Proportional Hazards Models',\n",
              "   'ROC Curve',\n",
              "   'Retrospective Studies',\n",
              "   'Risk Assessment',\n",
              "   'Risk Factors',\n",
              "   'Time Factors',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '22419795',\n",
              "  'title': 'Lung cancer staging: a physiological update.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1557-8259',\n",
              "  'IF_2013': '1.341',\n",
              "  'IF_2014': '1.49',\n",
              "  'IF_2015': '1.264',\n",
              "  'IF_2016': '1.264',\n",
              "  'IF_2017': '1.514',\n",
              "  'ISSN': '0030-6665',\n",
              "  'ISSN_online': '1557-8259',\n",
              "  'ISSN_print': '0030-6665',\n",
              "  'IsoAbbr': 'Otolaryngol. Clin. North Am.',\n",
              "  'JrId': 6320,\n",
              "  'MedAbbr': 'Otolaryngol Clin North Am',\n",
              "  'NLMid': '0144042',\n",
              "  'Titre': 'Otolaryngol. Clin. North Am.',\n",
              "  'abstract': 'Recent technologic advances have engendered alternative and innovative approaches to thyroid surgery aimed at reducing cosmetic sequelae. Minimally invasive techniques via small anterior cervical incisions hidden in natural skin creases and remote access approaches that eliminate anterior neck incisions entirely have emerged as viable options for patients who regard cosmesis as a priority. The safe application of these techniques to both benign and malignant thyroid disease has been evaluated. ',\n",
              "  'authors': ['Duke, William S', 'Chaung, Katrina', 'Terris, David J'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'Otolaryngologic clinics of North America',\n",
              "  'keywords': ['Cicatrix',\n",
              "   'Humans',\n",
              "   'Lymphatic Irradiation',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Patient Selection',\n",
              "   'Robotics',\n",
              "   'Thyroid Neoplasms',\n",
              "   'Thyroidectomy'],\n",
              "  'pmid': '25041956',\n",
              "  'title': 'Contemporary surgical techniques.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1423-0275',\n",
              "  'IF_2013': '0.667',\n",
              "  'IF_2014': '0.88',\n",
              "  'IF_2015': '1',\n",
              "  'IF_2016': '1.055',\n",
              "  'IF_2017': '1.012',\n",
              "  'ISSN': '0301-1569',\n",
              "  'ISSN_online': '1423-0275',\n",
              "  'ISSN_print': '0301-1569',\n",
              "  'IsoAbbr': 'ORL J. Otorhinolaryngol. Relat. Spec.',\n",
              "  'JrId': 6294,\n",
              "  'MedAbbr': 'ORL J Otorhinolaryngol Relat Spec',\n",
              "  'NLMid': '0334721',\n",
              "  'Titre': 'ORL J. Otorhinolaryngol. Relat. Spec.',\n",
              "  'abstract': 'The advancement of transoral surgical techniques for the management of oropharyngeal carcinomas has raised questions about how adjuvant therapy can best be integrated. Some of these questions have come from the application of established oncologic principles of adjuvant therapy, and some are unique to the evolving experience with transoral surgery and in particular with the recent advancement of robotic surgery. It is important for all members of the multidisciplinary treatment team to have a clear understanding of the foundation for adjuvant therapy and the issues unique to transoral robotic surgery to provide optimal patient care as this new treatment paradigm gains popularity for its safe application.',\n",
              "  'authors': ['Quon, Harry', \"O'Malley, Bert W\", 'Weinstein, Gregory S'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'ORL; journal for oto-rhino-laryngology and its related specialties',\n",
              "  'keywords': ['Chemotherapy, Adjuvant',\n",
              "   'Combined Modality Therapy',\n",
              "   'Humans',\n",
              "   'Oral Surgical Procedures',\n",
              "   'Oropharyngeal Neoplasms',\n",
              "   'Radiotherapy',\n",
              "   'Robotics'],\n",
              "  'pmid': '21389744',\n",
              "  'title': 'Postoperative adjuvant therapy after transoral robotic resection for oropharyngeal carcinomas: rationale and current treatment approach.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1464-410X',\n",
              "  'IF_2013': '3.13',\n",
              "  'IF_2014': '3.533',\n",
              "  'IF_2015': '4.387',\n",
              "  'IF_2016': '4.439',\n",
              "  'IF_2017': '4.688',\n",
              "  'ISSN': '1464-4096',\n",
              "  'ISSN_online': '1464-410X',\n",
              "  'ISSN_print': '1464-4096',\n",
              "  'IsoAbbr': 'BJU Int.',\n",
              "  'JrId': 21363,\n",
              "  'MedAbbr': 'BJU Int',\n",
              "  'NLMid': '100886721',\n",
              "  'Titre': 'BJU Int.',\n",
              "  'abstract': 'To evaluate the incidence of lower limb compartment syndrome (LLCS) in robot-assisted radical prostatectomy (RARP) and the prevalence of risk factors in patients with LLCS.|Data were collected from 17 UK robotic surgery institutions for a multicentre analysis. Data were used to calculate the incidence of LLCS and the prevalence of risk factors.|A total of 3110 RARPs were performed by 17 institutions between 2004 and 2011. There were nine cases of LLCS, giving an incidence of 0.29%. Seven of these required fasciotomy. The prevalence of risk factors was as follows: console time >4 h in 8/9 cases, early learning curve (<20 cases) in 3/9 cases; obesity (BMI >30 kg/m(2) ) in 5/9 cases; and peripheral vascular disease in 2/9 cases. One patient with LLCS was positioned incorrectly.|The serious complication LLCS occurs in RARP but has a low incidence. Long operating times, surgical inexperience, poor patient positioning, obesity and vascular disease appear to be risk factors.',\n",
              "  'authors': ['Pridgeon, Simon', 'Bishop, Conrad V', 'Adshead, Jim'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BJU international',\n",
              "  'keywords': ['Compartment Syndromes',\n",
              "   'Humans',\n",
              "   'Lower Extremity',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Robotics',\n",
              "   'United Kingdom'],\n",
              "  'pmid': '23879906',\n",
              "  'title': 'Lower limb compartment syndrome as a complication of robot-assisted radical prostatectomy: the UK experience.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1553-3514',\n",
              "  'IF_2013': '1.338',\n",
              "  'IF_2014': '1.458',\n",
              "  'IF_2015': '1.358',\n",
              "  'IF_2016': '1.909',\n",
              "  'IF_2017': '1.549',\n",
              "  'ISSN': '1553-3506',\n",
              "  'ISSN_online': '1553-3514',\n",
              "  'ISSN_print': '1553-3506',\n",
              "  'IsoAbbr': 'Surg Innov',\n",
              "  'JrId': 32322,\n",
              "  'MedAbbr': 'Surg Innov',\n",
              "  'NLMid': '101233809',\n",
              "  'Titre': 'Surg Innov',\n",
              "  'abstract': \"INTRODUCTION AND INDICATIONS: Robotic surgery has numerous advantages in rectal cancer surgery. Studies have reported the advantages associated with single-port approaches, such as eliminating the need for additional incisions, as well as the difficulties inherent in this technique. The authors present a hybrid technique that they performed using a robotic total mesorectal excision with the aid of a single port-device. Materials and methods. The authors performed the technique on 2 patients using a single-port device through an umbilical incision and 3 accessory ports for the robotic arms. There was no need to place ports for the assistant's equipment or for an assistant incision.|The operation time was 177.5 minutes, and there were no intraoperative or postoperative complications. Both patients were discharged 7 days after the operation.|This technical variation is an additional step forward for oncological surgery with minimal damage to the abdominal wall.\",\n",
              "  'authors': ['Jiménez-Rodríguez, Rosa M',\n",
              "   'Pavón, José M Díaz',\n",
              "   'de la Portilla, Fernando',\n",
              "   'Sillero, Emilio Prendes',\n",
              "   'Dussort, Jean M H Cadet',\n",
              "   'Padillo, Javier'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical innovation',\n",
              "  'keywords': ['Aged',\n",
              "   'Digestive System Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Rectal Neoplasms',\n",
              "   'Robotics',\n",
              "   'Umbilicus'],\n",
              "  'pmid': '22314274',\n",
              "  'title': 'Robotic-assisted total mesorectal excision with the aid of a single-port device.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['LI'],\n",
              "  'ESSN': '1538-9375',\n",
              "  'IF_2013': '4.781',\n",
              "  'IF_2014': '4.939',\n",
              "  'IF_2015': '6.616',\n",
              "  'IF_2016': '5.775',\n",
              "  'IF_2017': '5.325',\n",
              "  'ISSN': '1525-8610',\n",
              "  'ISSN_online': '1538-9375',\n",
              "  'ISSN_print': '1525-8610',\n",
              "  'IsoAbbr': 'J Am Med Dir Assoc',\n",
              "  'JrId': 30377,\n",
              "  'MedAbbr': 'J Am Med Dir Assoc',\n",
              "  'NLMid': '100893243',\n",
              "  'Titre': 'J Am Med Dir Assoc',\n",
              "  'abstract': \"To investigate the psychosocial effects of the companion robot, Paro, in a rest home/hospital setting in comparison to a control group.|Randomized controlled trial. Residents were randomized to the robot intervention group or a control group that attended normal activities instead of Paro sessions. Sessions took place twice a week for an hour over 12 weeks. Over the trial period, observations were conducted of residents' social behavior when interacting as a group with the robot. As a comparison, observations were also conducted of all the residents during general activities when the resident dog was or was not present.|A residential care facility in Auckland, New Zealand.|Forty residents in hospital and rest home care.|Residents completed a baseline measure assessing cognitive status, loneliness, depression, and quality of life. At follow-up, residents completed a questionnaire assessing loneliness, depression, and quality of life. During observations, behavior was noted and collated for instances of talking and stroking the dog/robot.|In comparison with the control group, residents who interacted with the robot had significant decreases in loneliness over the period of the trial. Both the resident dog and the seal robot made an impact on the social environment in comparison to when neither was present. Residents talked to and touched the robot significantly more than the resident dog. A greater number of residents were involved in discussion about the robot in comparison with the resident dog and conversation about the robot occurred more.|Paro is a positive addition to this environment and has benefits for older people in nursing home care. Paro may be able to address some of the unmet needs of older people that a resident animal may not, particularly relating to loneliness.\",\n",
              "  'authors': ['Robinson, Hayley',\n",
              "   'Macdonald, Bruce',\n",
              "   'Kerse, Ngaire',\n",
              "   'Broadbent, Elizabeth'],\n",
              "  'categories': ['GERIATRICS & GERONTOLOGY'],\n",
              "  'journal': 'Journal of the American Medical Directors Association',\n",
              "  'keywords': ['Aged',\n",
              "   'Animals',\n",
              "   'Depression',\n",
              "   'Dogs',\n",
              "   'Female',\n",
              "   'Geriatric Assessment',\n",
              "   'Humans',\n",
              "   'Loneliness',\n",
              "   'Male',\n",
              "   'New Zealand',\n",
              "   'Nursing Homes',\n",
              "   'Quality of Life',\n",
              "   'Robotics'],\n",
              "  'pmid': '23545466',\n",
              "  'title': 'The psychosocial effects of a companion robot: a randomized controlled trial.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1089-8638',\n",
              "  'IF_2013': '3.959',\n",
              "  'IF_2014': '4.333',\n",
              "  'IF_2015': '4.517',\n",
              "  'IF_2016': '4.632',\n",
              "  'IF_2017': '4.894',\n",
              "  'ISSN': '0022-2836',\n",
              "  'ISSN_online': '1089-8638',\n",
              "  'ISSN_print': '0022-2836',\n",
              "  'IsoAbbr': 'J. Mol. Biol.',\n",
              "  'JrId': 4967,\n",
              "  'MedAbbr': 'J Mol Biol',\n",
              "  'NLMid': '2985088R',\n",
              "  'Titre': 'J. Mol. Biol.',\n",
              "  'abstract': 'A neural network-based tool, TargetP, for large-scale subcellular location prediction of newly identified proteins has been developed. Using N-terminal sequence information only, it discriminates between proteins destined for the mitochondrion, the chloroplast, the secretory pathway, and \"other\" localizations with a success rate of 85% (plant) or 90% (non-plant) on redundancy-reduced test sets. From a TargetP analysis of the recently sequenced Arabidopsis thaliana chromosomes 2 and 4 and the Ensembl Homo sapiens protein set, we estimate that 10% of all plant proteins are mitochondrial and 14% chloroplastic, and that the abundance of secretory proteins, in both Arabidopsis and Homo, is around 10%. TargetP also predicts cleavage sites with levels of correctly predicted sites ranging from approximately 40% to 50% (chloroplastic and mitochondrial presequences) to above 70% (secretory signal peptides). TargetP is available as a web-server at http://www.cbs.dtu.dk/services/TargetP/.',\n",
              "  'authors': ['Emanuelsson, O', 'Nielsen, H', 'Brunak, S', 'von Heijne, G'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of molecular biology',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Arabidopsis',\n",
              "   'Biological Transport',\n",
              "   'Chloroplasts',\n",
              "   'Cytoplasm',\n",
              "   'Databases, Factual',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Mitochondria',\n",
              "   'Molecular Sequence Data',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nuclear Proteins',\n",
              "   'Plant Proteins',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'Protein Sorting Signals',\n",
              "   'Proteins',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Software'],\n",
              "  'pmid': '10891285',\n",
              "  'title': 'Predicting subcellular localization of proteins based on their N-terminal amino acid sequence.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['QA'],\n",
              "  'ESSN': '1643-3750',\n",
              "  'IF_2013': '1.216',\n",
              "  'IF_2014': '1.433',\n",
              "  'IF_2015': '1.405',\n",
              "  'IF_2016': '1.585',\n",
              "  'IF_2017': '1.894',\n",
              "  'ISSN': '1234-1010',\n",
              "  'ISSN_online': '1643-3750',\n",
              "  'ISSN_print': '1234-1010',\n",
              "  'IsoAbbr': 'Med. Sci. Monit.',\n",
              "  'JrId': 21912,\n",
              "  'MedAbbr': 'Med Sci Monit',\n",
              "  'NLMid': '9609063',\n",
              "  'Titre': 'Med. Sci. Monit.',\n",
              "  'abstract': 'BACKGROUND The aims of this study were to use RNA expression profile bioinformatics data from cases of thyroid cancer from the Cancer Genome Atlas (TCGA), the Kyoto Encyclopedia of Genes and Genomes (KEGG), and the Gene Ontology (GO) databases to construct a competing endogenous RNA (ceRNA) network of mRNAs, long noncoding RNAs (lncRNAs), and microRNAs (miRNAs). MATERIAL AND METHODS TCGA provided RNA profiles from 515 thyroid cancer tissues and 56 normal thyroid tissues. The DESeq R package analyzed high-throughput sequencing data on differentially expressed RNAs. GO and KEGG pathway analysis used the DAVID 6.8 and the ClusterProfile R package. Kaplan-Meier survival statistics and Cox regression analysis were performed. The thyroid cancer ceRNA network was constructed based on the miRDB, miRTarBase, and TargetScan databases. RESULTS There were 1,098 mRNAs associated with thyroid cancer; 101 mRNAs were associated with overall survival (OS). Multivariate analysis developed a risk scoring system that identified seven signature mRNAs, with a discriminative value of 0.88, determined by receiver operating characteristic (ROC) curve analysis. A ceRNA network included 13 mRNAs, 31 lncRNAs, and seven miRNAs. Four out of the 31 lncRNAs and all miRNAs were down-regulated, and the remaining RNAs were upregulated. Two lncRNAs (MIR1281A2HG and OPCML-IT1) and one miRNA (miR-184) were significantly associated with OS in patients with thyroid cancer. CONCLUSIONS Differential RNA expression profiling in thyroid cancer was used to construct a ceRNA network of mRNAs, lncRNAs, and miRNAs that showed potential in evaluating prognosis.',\n",
              "  'authors': ['Xu, Yuanxin', 'Chen, Jiuwei', 'Yang, Zhihui', 'Xu, Lihua'],\n",
              "  'categories': ['MEDICINE, RESEARCH & EXPERIMENTAL'],\n",
              "  'journal': 'Medical science monitor : international medical journal of experimental and clinical research',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'MicroRNAs',\n",
              "   'Prognosis',\n",
              "   'RNA',\n",
              "   'RNA, Long Noncoding',\n",
              "   'RNA, Messenger',\n",
              "   'Thyroid Neoplasms',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '30745559',\n",
              "  'title': 'Identification of RNA Expression Profiles in Thyroid Cancer to Construct a Competing Endogenous RNA (ceRNA) Network of mRNAs, Long Noncoding RNAs (lncRNAs), and microRNAs (miRNAs).',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'The recovery of potency following radical prostatectomy is complex and has a very wide range. In this study, we analyzed in detail the precise pattern of recovery of potency following robot-assisted radical prostatectomy (RARP).|Prospectively collected database of patients with a minimum follow-up of 1 year after RARP were evaluated retrospectively. Of 503 patients identified, 483 patients completed the sexual health inventory for men (SHIM) preoperatively and postoperatively every 3 months for the first 12 months. Overall potency, usage of phosphodiesterase type-5 (PDE-5) inhibitors, and return to baseline erectile function were evaluated. Potency was defined as having erection that is sufficient for sexual intercourse more than 50% of attempts, while quality potency was defined as being potent without the use of PDE-5 inhibitors.|Preoperatively, the overall potency and quality potency rate were 67.1% and 48.1%, respectively. Postoperatively, the overall potency rate was 61.4%, while the quality potency rate was 37.2%. In multivariate regression analysis, independent predictors of potency recovery were young age (<60), preoperative potency status, and bilateral preservation of neurovascular bundles (NVBs). In men with SHIM>21, the overall potency and quality potency rate were 79.7% and 41.2%, respectively. More importantly, only 21.4% of the men with normal erection preoperatively (SHIM>21) returned to baseline erectile function (SHIM>21) 12 months after surgery.|This study indicates that young age (<60), preoperative potency, and bilateral preservation of NVBs were positive predictors of potency recovery following RARP. However, an overwhelming majority of men experience a deterioration in the overall quality of erection after RARP.',\n",
              "  'authors': ['Woo, Seung Hyo',\n",
              "   'Kang, Dong Il',\n",
              "   'Ha, Yun-Sok',\n",
              "   'Salmasi, Amirali Hassanzadeh',\n",
              "   'Kim, Jeong Hyun',\n",
              "   'Lee, Dong-Hyeon',\n",
              "   'Kim, Wun-Jae',\n",
              "   'Kim, Isaac Yi'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Erectile Dysfunction',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Grading',\n",
              "   'Neoplasm Staging',\n",
              "   'Postoperative Complications',\n",
              "   'Prognosis',\n",
              "   'Prospective Studies',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Survival Rate'],\n",
              "  'pmid': '23987521',\n",
              "  'title': 'Comprehensive analysis of sexual function outcome in prostate cancer patients after robot-assisted radical prostatectomy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '2160-1836',\n",
              "  'IF_2013': '2.511',\n",
              "  'IF_2014': '3.198',\n",
              "  'IF_2015': '2.91',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '2.742',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2160-1836',\n",
              "  'IsoAbbr': 'G3 (Bethesda)',\n",
              "  'JrId': 39472,\n",
              "  'MedAbbr': 'G3 (Bethesda)',\n",
              "  'NLMid': '101566598',\n",
              "  'Titre': 'G3 (Bethesda)',\n",
              "  'abstract': 'Haematobia irritans, commonly known as the horn fly, is a globally distributed blood-feeding pest of cattle that is responsible for significant economic losses to cattle producers. Chemical insecticides are the primary means for controlling this pest but problems with insecticide resistance have become common in the horn fly. To provide a foundation for identification of genomic loci for insecticide resistance and for discovery of new control technology, we report the sequencing, assembly, and annotation of the horn fly genome. The assembled genome is 1.14 Gb, comprising 76,616 scaffolds with N50 scaffold length of 23 Kb. Using RNA-Seq data, we have predicted 34,413 gene models of which 19,185 have been assigned functional annotations. Comparative genomics analysis with the Dipteran flies Musca domestica L., Drosophila melanogaster, and Lucilia cuprina, show that the horn fly is most closely related to M. domestica, sharing 8,748 orthologous clusters followed by D. melanogaster and L. cuprina, sharing 7,582 and 7,490 orthologous clusters respectively. We also identified a gene locus for the sodium channel protein in which mutations have been previously reported that confers target site resistance to the most common class of pesticides used in fly control. Additionally, we identified 276 genomic loci encoding members of metabolic enzyme gene families such as cytochrome P450s, esterases and glutathione S-transferases, and several genes orthologous to sex determination pathway genes in other Dipteran species.',\n",
              "  'authors': ['Konganti, Kranti',\n",
              "   'Guerrero, Felix D',\n",
              "   'Schilkey, Faye',\n",
              "   'Ngam, Peter',\n",
              "   'Jacobi, Jennifer L',\n",
              "   'Umale, Pooja E',\n",
              "   'Perez de Leon, Adalberto A',\n",
              "   'Threadgill, David W'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'G3 (Bethesda, Md.)',\n",
              "  'keywords': ['Animals',\n",
              "   'Cluster Analysis',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Insect',\n",
              "   'Inactivation, Metabolic',\n",
              "   'Insecticide Resistance',\n",
              "   'Male',\n",
              "   'Models, Genetic',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Multigene Family',\n",
              "   'Muscidae',\n",
              "   'Phylogeny',\n",
              "   'RNA, Messenger',\n",
              "   'Sex Determination Processes',\n",
              "   'Sodium Channels',\n",
              "   'Whole Genome Sequencing'],\n",
              "  'pmid': '29602812',\n",
              "  'title': 'A Whole Genome Assembly of the Horn Fly, Haematobia irritans, and Prediction of Genes with Roles in Metabolism and Sex Determination.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DA'],\n",
              "  'ESSN': '1542-0086',\n",
              "  'IF_2013': '3.832',\n",
              "  'IF_2014': '3.972',\n",
              "  'IF_2015': '3.632',\n",
              "  'IF_2016': '3.656',\n",
              "  'IF_2017': '3.495',\n",
              "  'ISSN': '0006-3495',\n",
              "  'ISSN_online': '1542-0086',\n",
              "  'ISSN_print': '0006-3495',\n",
              "  'IsoAbbr': 'Biophys. J.',\n",
              "  'JrId': 1174,\n",
              "  'MedAbbr': 'Biophys J',\n",
              "  'NLMid': '0370626',\n",
              "  'Titre': 'Biophys. J.',\n",
              "  'abstract': 'Advances in biotechnology and computer science are providing the possibility to construct mathematical models for complex biological networks and systematically understand their properties. Traditional network identification approaches, however, cannot accurately recover the model parameters from the noisy laboratory measurements. This article introduces the concept of optimal identification (OI), which utilizes a global inversion algorithm to extract the full distribution of parameters consistent with the laboratory data. In addition, OI integrates suitable computational algorithms with experimental capabilities in a closed loop fashion to maximally reduce the breadth of the extracted parameter distribution. The closed loop OI procedure seeks out the optimal set of control chemical fluxes and data observations that actively filter out experimental noise and enhance the sensitivity to the desired parameters. In this fashion, the highest quality network parameters can be attained from inverting the tailored laboratory data. The operation of OI is illustrated by identifying a simulated tRNA proofreading mechanism, in which OI provides superior solutions for all the rate constants compared with suboptimal and nonoptimal methods.',\n",
              "  'authors': ['Feng, Xiao-jiang', 'Rabitz, Herschel'],\n",
              "  'categories': ['BIOPHYSICS'],\n",
              "  'journal': 'Biophysical journal',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Cell Physiological Phenomena',\n",
              "   'Computer Simulation',\n",
              "   'Kinetics',\n",
              "   'Models, Biological',\n",
              "   'Models, Chemical',\n",
              "   'Protein Biosynthesis',\n",
              "   'RNA, Transfer',\n",
              "   'Research Design',\n",
              "   'Signal Transduction'],\n",
              "  'pmid': '14990460',\n",
              "  'title': 'Optimal identification of biochemical reaction networks.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1873-3573',\n",
              "  'IF_2013': '3.511',\n",
              "  'IF_2014': '3.545',\n",
              "  'IF_2015': '4.035',\n",
              "  'IF_2016': '4.162',\n",
              "  'IF_2017': '4.244',\n",
              "  'ISSN': '0039-9140',\n",
              "  'ISSN_online': '1873-3573',\n",
              "  'ISSN_print': '0039-9140',\n",
              "  'IsoAbbr': 'Talanta',\n",
              "  'JrId': 22720,\n",
              "  'MedAbbr': 'Talanta',\n",
              "  'NLMid': '2984816R',\n",
              "  'Titre': 'Talanta',\n",
              "  'abstract': 'To deal with heterogeneous classification problem efficiently, each heterogeneous object was represented by a set of measurements obtained on different part of it, and the heterogeneous classification problem was reformulated in the framework of multi-instance learning (MIL). Based on a variant of count-based MIL assumption, a maximum count least squares support vector machine (maxc-LS-SVM) learning algorithm was developed. The algorithm was tested on a set of toy datasets. It was found that maxc-LS-SVM inherits all the sound characters of both LS-SVM and MIL framework. A comparison study between the proposed approach and the other two MIL approaches (i.e., mi-SVM and MI-SVM) was performed on a real wolfberry fruit spectral dataset. The results demonstrate that by formulating the heterogeneous classification problem as a MIL one, the heterogeneous classification problem can be solved by the proposed maxc-LS-SVM algorithm effectively. ',\n",
              "  'authors': ['Lin, Zhaozhou',\n",
              "   'Jia, Shuaiyun',\n",
              "   'Luo, Gan',\n",
              "   'Dai, Xingxing',\n",
              "   'Xu, Bing',\n",
              "   'Wu, Zhisheng',\n",
              "   'Shi, Xinyuan',\n",
              "   'Qiao, Yanjiang'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Talanta',\n",
              "  'keywords': ['Fruit',\n",
              "   'Least-Squares Analysis',\n",
              "   'Lycium',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '25476295',\n",
              "  'title': 'Dealing with heterogeneous classification problem in the framework of multi-instance learning.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The CRISPR/Cas9-sgRNA system has recently become a popular tool for genome editing and a very hot topic in the field of medical research. In this system, Cas9 protein is directed to a desired location for gene engineering and cleaves target DNA sequence which is complementary to a 20-nucleotide guide sequence found within the sgRNA. A lot of experimental efforts, ranging from in vivo selection to in silico modeling, have been made for efficient designing of sgRNAs in CRISPR/Cas9 system. In this article, we present a novel tool, called CRISPRpred, for efficient in silico prediction of sgRNAs on-target activity which is based on the applications of Support Vector Machine (SVM) model. To conduct experiments, we have used a benchmark dataset of 17 genes and 5310 guide sequences where there are only 20% true values. CRISPRpred achieves Area Under Receiver Operating Characteristics Curve (AUROC-Curve), Area Under Precision Recall Curve (AUPR-Curve) and maximum Matthews Correlation Coefficient (MCC) as 0.85, 0.56 and 0.48, respectively. Our tool shows approximately 5% improvement in AUPR-Curve and after analyzing all evaluation metrics, we find that CRISPRpred is better than the current state-of-the-art. CRISPRpred is enough flexible to extract relevant features and use them in a learning algorithm. The source code of our entire software with relevant dataset can be found in the following link: https://github.com/khaled-buet/CRISPRpred.',\n",
              "  'authors': ['Rahman, Md Khaledur', 'Rahman, M Sohel'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'CRISPR-Cas Systems',\n",
              "   'Computer Simulation',\n",
              "   'Gene Editing',\n",
              "   'RNA, Guide',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '28767689',\n",
              "  'title': 'CRISPRpred: A flexible and efficient tool for sgRNAs on-target activity prediction in CRISPR/Cas9 systems.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-9568',\n",
              "  'IF_2013': '3.669',\n",
              "  'IF_2014': '3.181',\n",
              "  'IF_2015': '2.975',\n",
              "  'IF_2016': '2.941',\n",
              "  'IF_2017': '2.832',\n",
              "  'ISSN': '0953-816X',\n",
              "  'ISSN_online': '1460-9568',\n",
              "  'ISSN_print': '0953-816X',\n",
              "  'IsoAbbr': 'Eur. J. Neurosci.',\n",
              "  'JrId': 2607,\n",
              "  'MedAbbr': 'Eur J Neurosci',\n",
              "  'NLMid': '8918110',\n",
              "  'Titre': 'Eur. J. Neurosci.',\n",
              "  'abstract': \"Functional neuro-imaging studies of Parkinson's disease (PD) patients and animal models show inconsistent cortical responses to sensory stimulation: some present increased sensorimotor cortex activation contradicting classical basal ganglia-cortex circuitry models, whereas others show decreased activation. As functional neuro-imaging activation is defined as the signal difference between stimulation ON and stimulation OFF, reduced 'activation' can point to either increased neuronal activity during stimulation ON or to decreased basal neuronal activity during stimulation OFF. A unique non-invasive method that uses the temporal and the spatial variances of functional magnetic resonance imaging signal is employed here to compare basal neuronal activity levels and 'functional homogeneity' between groups. Based on the assumption that the temporal variance reflects average neuronal activity, the variance of activity within a predefined region is defined as the region's 'functional homogeneity', which is assumed to estimate neuronal synchronization. Comparison of temporal and spatial variances of the sensorimotor cortex and the striatum in the 6-hydroxydopamine (6-OHDA) PD rat model and a control rat group show bilaterally decreased temporal and spatial variances in the 6-OHDA rat group, suggesting bilateral reduction of basal neuronal activity levels together with an increase in local neuronal synchronization in line with classical basal ganglia-cortex circuit models.\",\n",
              "  'authors': ['Pelled, Galit',\n",
              "   'Bergman, Hagai',\n",
              "   'Ben-Hur, Tamir',\n",
              "   'Goelman, Gadi'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The European journal of neuroscience',\n",
              "  'keywords': ['Adrenergic Agents',\n",
              "   'Animals',\n",
              "   'Corpus Striatum',\n",
              "   'Disease Models, Animal',\n",
              "   'Functional Laterality',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oxidopamine',\n",
              "   'Oxygen',\n",
              "   'Parkinson Disease',\n",
              "   'Physical Stimulation',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Somatosensory Cortex'],\n",
              "  'pmid': '15869519',\n",
              "  'title': \"Reduced basal activity and increased functional homogeneity in sensorimotor and striatum of a Parkinson's disease rat model: a functional MRI study.\",\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1534-6285',\n",
              "  'IF_2013': '1.513',\n",
              "  'IF_2014': '1.513',\n",
              "  'IF_2015': '1.597',\n",
              "  'IF_2016': '2.124',\n",
              "  'IF_2017': '1.983',\n",
              "  'ISSN': '1527-2737',\n",
              "  'ISSN_online': '1534-6285',\n",
              "  'ISSN_print': '1527-2737',\n",
              "  'IsoAbbr': 'Curr Urol Rep',\n",
              "  'JrId': 29748,\n",
              "  'MedAbbr': 'Curr Urol Rep',\n",
              "  'NLMid': '100900943',\n",
              "  'Titre': 'Curr Urol Rep',\n",
              "  'abstract': 'Laparoscopic surgery of the upper urinary tract has reduced the morbidity related to large abdominal incisions and has resulted in significant advantages over open surgery. Nevertheless, the pursuit for even more minimally invasive alternatives to laparoscopy has led to the concept of scarless surgery and the approach of laparoendoscopic single-site surgery (LESS). LESS is currently a feasible approach for the majority of kidney surgical procedures, and there is intense debate regarding its efficiency and advantages. In the present review of the literature, the current status of upper urinary LESS and its advantages and disadvantages, as well the technological and technical evolution, are presented. ',\n",
              "  'authors': ['Kallidonis, Panagiotis',\n",
              "   'Kontogiannis, Stavros',\n",
              "   'Kyriazis, Iason',\n",
              "   'Georgiopoulos, Ioannis',\n",
              "   'Al-Aown, Abdulrahman',\n",
              "   'Stolzenburg, Jens-Uwe',\n",
              "   'Liatsikos, Evangelos'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current urology reports',\n",
              "  'keywords': ['Equipment Design',\n",
              "   'Forecasting',\n",
              "   'Humans',\n",
              "   'Kidney Diseases',\n",
              "   'Laparoscopes',\n",
              "   'Laparoscopy',\n",
              "   'Nephrectomy',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics'],\n",
              "  'pmid': '23740382',\n",
              "  'title': 'Laparoendoscopic single-site surgery in kidney surgery: clinical experience and future perspectives.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1094-4087',\n",
              "  'IF_2013': '3.525',\n",
              "  'IF_2014': '3.488',\n",
              "  'IF_2015': '3.148',\n",
              "  'IF_2016': '3.307',\n",
              "  'IF_2017': '3.356',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1094-4087',\n",
              "  'IsoAbbr': 'Opt Express',\n",
              "  'JrId': 29989,\n",
              "  'MedAbbr': 'Opt Express',\n",
              "  'NLMid': '101137103',\n",
              "  'Titre': 'Opt Express',\n",
              "  'abstract': \"Holograms of colloidal particles can be analyzed with the Lorenz-Mie theory of light scattering to measure individual particles' three-dimensional positions with nanometer precision while simultaneously estimating their sizes and refractive indexes. Extracting this wealth of information begins by detecting and localizing features of interest within individual holograms. Conventionally approached with heuristic algorithms, this image analysis problem can be solved faster and more generally with machine-learning techniques. We demonstrate that two popular machine-learning algorithms, cascade classifiers and deep convolutional neural networks (CNN), can solve the feature-localization problem orders of magnitude faster than current state-of-the-art techniques. Our CNN implementation localizes holographic features precisely enough to bootstrap more detailed analyses based on the Lorenz-Mie theory of light scattering. The wavelet-based Haar cascade proves to be less precise, but is so computationally efficient that it creates new opportunities for applications that emphasize speed and low cost. We demonstrate its use as a real-time targeting system for holographic optical trapping.\",\n",
              "  'authors': ['Hannel, Mark D',\n",
              "   'Abdulali, Aidan',\n",
              "   \"O'Brien, Michael\",\n",
              "   'Grier, David G'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics express',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30114772',\n",
              "  'title': 'Machine-learning techniques for fast and accurate feature localization in holograms of colloidal particles.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': \"For the functional control of prosthetic hand, it is insufficient to obtain only the motion pattern information. As far as practicality is concerned, the control of the prosthetic hand force is indispensable. The application value of prosthetic hand will be greatly improved if the stable grip of prosthetic hand can be achieved. To address this problem, in this study, a bio-signal control method for grasping control of a prosthetic hand is proposed to improve patient's sense of using prosthetic hand and the thus improving the quality of life.|A MYO gesture control armband is used to collect the surface electromyographic (sEMG) signals from the upper limb. The overlapping sliding window scheme are applied for data segmentation and the correlated features are extracted from each segmented data. Principal component analysis (PCA) methods are then deployed for dimension reduction. Deep neural network is used to generate sEMG-force regression model for force prediction at different levels. The predicted force values are input to a fuzzy controller for the grasping control of a prosthetic hand. A vibration feedback device is used to feed grasping force value back to patient's arm to improve patient's sense of using prosthetic hand and realize accurate grasping. To test the effectiveness of the scheme, 15 able-bodied subjects participated in the experiments.|The classification results indicated that 8-channel sEMG applying all four time-domain features, with PCA reduction from 32 to 8 dimensions results in the highest classification accuracy. Based on the experimental results from 15 participants, the average recognition rate is over 95%. On the other hand, from the statistical results of standard deviation, the between-subject variations ranges from 3.58 to 1.25%, proving that the robustness and stability of the proposed approach.|The method proposed hereto control grasping power through the patient's own sEMG signal, which achieves a high recognition rate to improve the success rate of grip and increases the sense of operation and also brings the gospel for upper extremity amputation patients.\",\n",
              "  'authors': ['Li, Chuanjiang',\n",
              "   'Ren, Jian',\n",
              "   'Huang, Huaiqi',\n",
              "   'Wang, Bin',\n",
              "   'Zhu, Yanfei',\n",
              "   'Hu, Huosheng'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Artificial Limbs',\n",
              "   'Electromyography',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Hand',\n",
              "   'Hand Strength',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Muscles',\n",
              "   'Principal Component Analysis'],\n",
              "  'pmid': '30081927',\n",
              "  'title': 'PCA and deep learning based myoelectric grasping control of a prosthetic hand.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZC'],\n",
              "  'ESSN': '1548-4475',\n",
              "  'IF_2013': '0.814',\n",
              "  'IF_2014': '0.742',\n",
              "  'IF_2015': '0.681',\n",
              "  'IF_2016': '0.767',\n",
              "  'IF_2017': '0.833',\n",
              "  'ISSN': '0093-7355',\n",
              "  'ISSN_online': '1548-4475',\n",
              "  'ISSN_print': '0093-7355',\n",
              "  'IsoAbbr': 'Lab Anim (NY)',\n",
              "  'JrId': 22393,\n",
              "  'MedAbbr': 'Lab Anim (NY)',\n",
              "  'NLMid': '0417737',\n",
              "  'Titre': 'Lab Anim (NY)',\n",
              "  'abstract': 'Automation technologies are improving efficiency in the vivarium and helping institutions keep up with the growing number of mutant rodents in their colonies.',\n",
              "  'authors': ['Weintraub, Arlene'],\n",
              "  'categories': ['VETERINARY SCIENCES'],\n",
              "  'journal': 'Lab animal',\n",
              "  'keywords': ['Animal Husbandry',\n",
              "   'Animals',\n",
              "   'Animals, Laboratory',\n",
              "   'Housing, Animal',\n",
              "   'Robotics'],\n",
              "  'pmid': '27991871',\n",
              "  'title': 'Automation: robots in the vivarium.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2154-2317',\n",
              "  'IF_2013': '0.663',\n",
              "  'IF_2014': '0.957',\n",
              "  'IF_2015': '0.558',\n",
              "  'IF_2016': '0.5',\n",
              "  'IF_2017': '0.587',\n",
              "  'ISSN': '2154-2287',\n",
              "  'ISSN_online': '2154-2317',\n",
              "  'ISSN_print': '2154-2287',\n",
              "  'IsoAbbr': 'IEEE Pulse',\n",
              "  'JrId': 38326,\n",
              "  'MedAbbr': 'IEEE Pulse',\n",
              "  'NLMid': '101541727',\n",
              "  'Titre': 'IEEE Pulse',\n",
              "  'abstract': 'Advances in machine learning as applied to functional magnetic resonance imaging (fMRI) data offer the possibility of pretesting and classifying marketing communications using unbiased pattern recognition algorithms. By using these algorithms to analyze brain responses to brands, products, or existing marketing communications that either failed or succeeded in the marketplace and identifying the patterns of brain activity that characterize success or failure, future planned campaigns or new products can now be pretested to determine how well the resulting brain responses match the desired (successful) pattern of brain activity without the need for verbal feedback. This major advance in signal processing is poised to revolutionize the application of these brain-imaging techniques in the marketing sector by offering greater accuracy of prediction in terms of consumer acceptance of new brands, products, and campaigns at a speed that makes them accessible as routine pretesting tools that will clearly demonstrate return on investment.',\n",
              "  'authors': ['Calvert, Gemma A', 'Brammer, Michael J'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE pulse',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Choice Behavior',\n",
              "   'Consumer Behavior',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Marketing',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '22678839',\n",
              "  'title': 'Predicting consumer behavior: using novel mind-reading approaches.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'The primary objective of this study is to determine if training on the Mimic dV-Trainer (MdVT) simulator results in improved ability on the da Vinci surgical system (dVSS) using exercises with inanimate objects.|Twelve trainees (MdVT group) and 10 residents and one fellow (dVSS group) were recruited for the study. Each participant in the MdVT group completed one session of five exercises on the dVSS that were scored for timing and accuracy, followed by four training sessions on the MdVT, and concluded with a final session on the dVSS in which the initial exercises were repeated. Improvement on the dVSS exercises was compared with dVSS group who completed four to six training sessions using the same exercises on the dVSS without any simulator training.|Both groups had similar significant improvements in the Letter Board and String Running exercises for both timing and accuracy. The MdVT group demonstrated significant improvement in the Pattern Cutting and Peg Board times. Only the dVSS group significantly improved in the Knot Tying time and the Peg Board accuracy.|Training with the MdVT provided similar improvement on five exercises performed on the dVSS when compared with training on the dVSS alone. The use of this simulator in resident and student training may help bridge the gap between the safe acquisition of surgical skills and effective performance during live robot-assisted surgery.',\n",
              "  'authors': ['Lerner, Michelle A',\n",
              "   'Ayalew, Mikias',\n",
              "   'Peine, William J',\n",
              "   'Sundaram, Chandru P'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '20334558',\n",
              "  'title': 'Does training on a virtual reality robotic simulator improve performance on the da Vinci surgical system?',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'The objective of this study was to compare the efficiency of manual and robotically assisted laparoscopic surgery.|To evaluate the surgical efficiency in a set of basic endoscopic movements, 20 medical students without any surgical experience were selected to perform at random a set of laparoscopic tasks either manually or robotic assisted (Zeus). This task consisted of dropping beads into receptacles, running a 25-cm rope, capping a hypodermic needle, suturing, and performing a laparoscopic cholecystectomy on a cadaver liver of a pig. A quantitative time-action analysis was performed to evaluate the efficacy and skill performance in terms of time and the number of actions.|The dropping beads exercise and the laparoscopic cholecystectomy required more time when performed with robotic assistance, as compared with manual performance (respectively, median, 78.5 s; range, 63 - 122 s vs median, 144.5 s; range, 100 - 169 s; p <0.01 and median, 34.0 min; range 11-44 min vs median, 46.5 s; range, 21 - 79 min; p = 0.05). A tendency toward fewer total actions in all the robotically assisted exercises was observed. However, significance was shown only in the rope-passing task (median, 71; range, 59 - 87 vs median, 62; range, 57-80; p = 0.05). Grasping the beads, the rope, and either the needle or the cap were tasks that required fewer actions to complete when performed with robotically assistance (respectively, median, 11; range, 10 - 14 vs median, 12.5; range, 11 - 15; p <0.01; median, 56; range, 55 - 60 vs median, 60.5 min; range, 55 - 65; p = 0.03, and median, 6; range, 4 - 21 vs median, 10.5; range, 6 - 38; p = 0.02). As compared with the robotically assisted rope-passing exercise, more failures were made in the manually performed procedure (p = 0.03), mainly caused by unintentional dropping of the rope (p = 0.02).|Robotically assisted laparoscopic surgery by participants without any surgical experience might require more time, but actions can be performed equally or more precisely as compared with manual laparoscopic surgery.',\n",
              "  'authors': ['Nio, D',\n",
              "   'Bemelman, W A',\n",
              "   'Boer, K T',\n",
              "   'Dunker, M S',\n",
              "   'Gouma, D J',\n",
              "   'Gulik, T M'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Animals',\n",
              "   'Laparoscopy',\n",
              "   'Man-Machine Systems',\n",
              "   'Robotics',\n",
              "   'Students, Medical',\n",
              "   'Swine',\n",
              "   'Task Performance and Analysis'],\n",
              "  'pmid': '11928018',\n",
              "  'title': 'Efficiency of manual versus robotical (Zeus) assisted laparoscopic surgery in the performance of standardized tasks.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': \"Advanced glycation end products (AGEs) are highly associated with hyperglycemia in human skin tissue, and they also have the autofluorescence characteristic. A self-developed optical noninvasive detection device was used to measure the autofluorescence in human skin tissue, and then a neural network pattern recognition model was used to assess the risk of diabetes mellitus of the subject under survey. After the fluorescence spectra were acquired and processed with principal component analysis, four of the leading principal components were chosen to represent a whole spectrum. The established neural network pattern recognition model has 4 input nodes, 6 hidden nodes and 1 output node. A dataset consisting of 487 cases collected in Anhui Provincial Hospital was used to train the model. Seventy percent cases were used as the training set, 15% as the validation set and 15% as the test set. The model can output subject's risk of diabetes mellitus, or a dichotomous judgment. Receiver operating characteristic curve can be drawn with the area under curve of 0. 81, with standard error of 0. 02. When using 0. 5 as the threshold between diabetes mellitus and non-diabetes mellitus, the sensitivity and specificity of this model is 72. 4% and 77. 6% respectively, and the overall accuracy is 74. 9%. The method using human skin autofluorescence spectrum combined with neural network pattern recognition model is proposed for the first time, and the results show that this method has a better screening effect compared with currently used fasting plasma glucose and HbAlc.\",\n",
              "  'authors': ['Li, Fei',\n",
              "   'Wang, Yi-Kun',\n",
              "   'Zhu, Ling',\n",
              "   'Zhang, Yuan-Zhi',\n",
              "   'Ji, Min',\n",
              "   'Zhang, Long',\n",
              "   'Liu, Yong',\n",
              "   'Wang, An'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Diabetes Mellitus',\n",
              "   'Fluorescence',\n",
              "   'Glycation End Products, Advanced',\n",
              "   'Humans',\n",
              "   'Hyperglycemia',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Risk Assessment',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Skin',\n",
              "   'Spectrometry, Fluorescence'],\n",
              "  'pmid': '25095432',\n",
              "  'title': '[Research on noninvasive risk evaluation of diabetes mellitus based on neural network pattern recognition].',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1527-1315',\n",
              "  'IF_2013': '6.214',\n",
              "  'IF_2014': '6.867',\n",
              "  'IF_2015': '6.798',\n",
              "  'IF_2016': '7.296',\n",
              "  'IF_2017': '7.469',\n",
              "  'ISSN': '0033-8419',\n",
              "  'ISSN_online': '1527-1315',\n",
              "  'ISSN_print': '0033-8419',\n",
              "  'IsoAbbr': 'Radiology',\n",
              "  'JrId': 6859,\n",
              "  'MedAbbr': 'Radiology',\n",
              "  'NLMid': '0401260',\n",
              "  'Titre': 'Radiology',\n",
              "  'abstract': 'The purpose of this study was to develop an automated scheme to facilitate detection of localized ground-glass opacity (GGO) in the lung at computed tomography (CT). Institutional review board approval and informed consent were not required. Two radiologists reviewed CT images from 14 patients (five men, nine women) who had lung cancer or metastasis and whose malignancy was classified as GGO. The lung region was sampled and completely covered with contiguous, 50% overlapping regions of interest (ROIs) measuring 30 x 30 pixels in size. The lung area within each ROI was analyzed to compute texture features and gaussian curve fitting features. Performance of the artificial neural networks (ANNs) measured by using the area under the receiver operating characteristic curve was 0.92. With a threshold of 0.9, the sensitivity of the ANN for detecting GGO ROIs was 94.3% (280 of 297 ROIs), and the positive predictive value was 29.1% (280 of 963 ROIs). A computerized scheme may hold promise in facilitating detection of localized GGO at CT.',\n",
              "  'authors': ['Kim, Kwang Gi',\n",
              "   'Goo, Jin Mo',\n",
              "   'Kim, Jong Hyo',\n",
              "   'Lee, Hyun Ju',\n",
              "   'Min, Byung Goo',\n",
              "   'Bae, Kyongtae T',\n",
              "   'Im, Jung-Gi'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Radiology',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Lung Neoplasms',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Predictive Value of Tests',\n",
              "   'ROC Curve',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '16192320',\n",
              "  'title': 'Computer-aided diagnosis of localized ground-glass opacity in the lung at CT: initial experience.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1879-1883',\n",
              "  'IF_2013': '2.406',\n",
              "  'IF_2014': '2.291',\n",
              "  'IF_2015': '2.403',\n",
              "  'IF_2016': '2.612',\n",
              "  'IF_2017': '2.141',\n",
              "  'ISSN': '0002-9610',\n",
              "  'ISSN_online': '1879-1883',\n",
              "  'ISSN_print': '0002-9610',\n",
              "  'IsoAbbr': 'Am. J. Surg.',\n",
              "  'JrId': 471,\n",
              "  'MedAbbr': 'Am J Surg',\n",
              "  'NLMid': '0370473',\n",
              "  'Titre': 'Am. J. Surg.',\n",
              "  'abstract': 'Surgical use of robotics, or computer-assisted surgical systems (CAS), has evolved over the last 10 years; for the treatment of chest diseases, however, the development has really occurred in the last 3 to 4 years. This brief history means that there have been few publications in the medical literature, and those that exist are mostly case reports. Moreover, because of this modest experience, robotic thoracic procedures currently take more time than nonrobotic cases and, thus, are more expensive. The surgical learning curve appears to be steep, especially for the more complex procedures. As surgeons gain greater experience and the complexity and cost of the equipment are reduced, we should expect to see greater utility of CAS in thoracic surgery.',\n",
              "  'authors': ['Kernstine, Kemp H'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'American journal of surgery',\n",
              "  'keywords': ['Esophagus',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Mediastinal Diseases',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Pneumonectomy',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Thoracic Surgical Procedures'],\n",
              "  'pmid': '15476658',\n",
              "  'title': 'Robotics in thoracic surgery.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['QU'],\n",
              "  'ESSN': '1521-4028',\n",
              "  'IF_2013': '1.822',\n",
              "  'IF_2014': '1.823',\n",
              "  'IF_2015': '1.585',\n",
              "  'IF_2016': '1.438',\n",
              "  'IF_2017': '1.58',\n",
              "  'ISSN': '0233-111X',\n",
              "  'ISSN_online': '1521-4028',\n",
              "  'ISSN_print': '0233-111X',\n",
              "  'IsoAbbr': 'J. Basic Microbiol.',\n",
              "  'JrId': 5170,\n",
              "  'MedAbbr': 'J Basic Microbiol',\n",
              "  'NLMid': '8503885',\n",
              "  'Titre': 'J. Basic Microbiol.',\n",
              "  'abstract': 'Detailed information on structural changes that occur during ontogenesis of Rhizophagus irregularis in axenically developed coculture is limited. Our study aims to investigate the series of events that occur during mycorrhizal ontogenesis under axenic condition through basic and advanced microscopic techniques followed by comparison among these to identify the suitable technique for rapid and detailed analysis of mycorrhizal structures. Three stages were identified in mycorrhizal ontogenesis from initiation (preinfection stage of hyphae; its branching, infection and appressoria formation; epidermal opening; and hyphal entry), progression (arbuscular development; hyphal coils and vesicles) to maturity (extraradical spores). Scanning electron microscopy was found to be an efficient tool for studying spatial three-dimensional progression. Adding to the advantages of advanced microscopy, potential of autofluorescence to explore the stages of symbiosis nondestructively was also established. We also report imaging of ultrathin sections by bright field microscopy to provide finer details at subcellular interface. Owing to the merits of nondestructive sampling, ease of sample preparation, autofluorescence (no dye required), no use of toxic chemicals, rapid analysis and in depth characterization confocal laser scanning microscopy was identified as the most preferred technique. The method thus developed can be used for detailed structural inquisition of mycorrhizal symbiosis both in in planta and in an in vitro system.',\n",
              "  'authors': ['Chaudhary, Shikha',\n",
              "   'Gupta, Priyanka',\n",
              "   'Srivastava, Shivani',\n",
              "   'Adholeya, Alok'],\n",
              "  'categories': ['MICROBIOLOGY'],\n",
              "  'journal': 'Journal of basic microbiology',\n",
              "  'keywords': ['Axenic Culture',\n",
              "   'Biological Ontologies',\n",
              "   'Glomeromycota',\n",
              "   'Hyphae',\n",
              "   'Microscopy',\n",
              "   'Mycorrhizae',\n",
              "   'Plant Roots'],\n",
              "  'pmid': '31074496',\n",
              "  'title': 'Understanding dynamics of Rhizophagus irregularis ontogenesis in axenically developed coculture through basic and advanced microscopic techniques.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"In past, numerous methods have been developed for predicting efficacy of short interfering RNA (siRNA). However these methods have been developed for predicting efficacy of fully complementary siRNA against a gene. Best of author's knowledge no method has been developed for predicting efficacy of mismatch siRNA against a gene. In this study, a systematic attempt has been made to identify highly effective complementary as well as mismatch siRNAs for silencing a gene.Support vector machine (SVM) based models have been developed for predicting efficacy of siRNAs using composition, binary and hybrid pattern siRNAs. We achieved maximum correlation 0.67 between predicted and actual efficacy of siRNAs using hybrid model. All models were trained and tested on a dataset of 2182 siRNAs and performance was evaluated using five-fold cross validation techniques. The performance of our method desiRm is comparable to other well-known methods. In this study, first time attempt has been made to design mutant siRNAs (mismatch siRNAs). In this approach we mutated a given siRNA on all possible sites/positions with all possible nucleotides. Efficacy of each mutated siRNA is predicted using our method desiRm. It is well known from literature that mismatches between siRNA and target affects the silencing efficacy. Thus we have incorporated the rules derived from base mismatches experimental data to find out over all efficacy of mutated or mismatch siRNAs. Finally we developed a webserver, desiRm (http://www.imtech.res.in/raghava/desirm/) for designing highly effective siRNA for silencing a gene. This tool will be helpful to design siRNA to degrade disease isoform of heterozygous single nucleotide polymorphism gene without depleting the wild type protein.\",\n",
              "  'authors': ['Ahmed, Firoz', 'Raghava, Gajendra P S'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Base Pair Mismatch',\n",
              "   'Base Sequence',\n",
              "   'Gene Silencing',\n",
              "   'Internet',\n",
              "   'Models, Genetic',\n",
              "   'Molecular Sequence Data',\n",
              "   'Nucleotides',\n",
              "   'Position-Specific Scoring Matrices',\n",
              "   'RNA, Complementary',\n",
              "   'RNA, Small Interfering',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '21853133',\n",
              "  'title': 'Designing of highly effective complementary and mismatch siRNAs for silencing a gene.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1619-7089',\n",
              "  'IF_2013': '5.217',\n",
              "  'IF_2014': '5.383',\n",
              "  'IF_2015': '5.537',\n",
              "  'IF_2016': '7.277',\n",
              "  'IF_2017': '7.704',\n",
              "  'ISSN': '1619-7070',\n",
              "  'ISSN_online': '1619-7089',\n",
              "  'ISSN_print': '1619-7070',\n",
              "  'IsoAbbr': 'Eur. J. Nucl. Med. Mol. Imaging',\n",
              "  'JrId': 27055,\n",
              "  'MedAbbr': 'Eur J Nucl Med Mol Imaging',\n",
              "  'NLMid': '101140988',\n",
              "  'Titre': 'Eur. J. Nucl. Med. Mol. Imaging',\n",
              "  'abstract': 'Techniques from the field of artificial intelligence, and more specifically machine (deep) learning methods, have been core components of most recent developments in the field of medical imaging. They are already being exploited or are being considered to tackle most tasks, including image reconstruction, processing (denoising, segmentation), analysis and predictive modelling. In this review we introduce and define these key concepts and discuss how the techniques from this field can be applied to nuclear medicine imaging applications with a particular focus on radio(geno)mics.',\n",
              "  'authors': ['Visvikis, Dimitris',\n",
              "   'Cheze Le Rest, Catherine',\n",
              "   'Jaouen, Vincent',\n",
              "   'Hatt, Mathieu'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European journal of nuclear medicine and molecular imaging',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31280350',\n",
              "  'title': 'Artificial intelligence, machine (deep) learning and radio(geno)mics: definitions and nuclear medicine imaging applications.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1097-0215',\n",
              "  'IF_2013': '5.007',\n",
              "  'IF_2014': '5.085',\n",
              "  'IF_2015': '5.531',\n",
              "  'IF_2016': '6.513',\n",
              "  'IF_2017': '7.36',\n",
              "  'ISSN': '0020-7136',\n",
              "  'ISSN_online': '1097-0215',\n",
              "  'ISSN_print': '0020-7136',\n",
              "  'IsoAbbr': 'Int. J. Cancer',\n",
              "  'JrId': 4284,\n",
              "  'MedAbbr': 'Int J Cancer',\n",
              "  'NLMid': '0042124',\n",
              "  'Titre': 'Int. J. Cancer',\n",
              "  'abstract': 'Fibroblasts and myofibroblasts from normal, fibrotic or tumoral breast tissues present multiple quantitative differences in gene expression even when grown in isolation. We were therefore prompted to investigate whether one could recognize various subtypes by their constitutive-gene expression profile. Quantitative autoradiographic data for 34 constitutively expressed transcripts were submitted to multivariate analysis of variance, followed by discriminant analysis and single linkage cluster analysis. Models assuming up to 8 putative fibroblast subtypes (among fibroblasts or myofibroblasts from breast skin, normal mammary stroma, tumor-adjacent \"normal\" stroma, post-radiation fibrosis lesions and benign or malignant tumors) and an epithelial-cell group used as an internal control resulted in 100% correct classification. Myofibroblasts from various origins clustered close to, although distinctly apart from, their corresponding alpha-smooth-muscle-actin-negative counterparts. Malignant tumor fibroblasts were phenotypically more distant from normal cells compared with other pathological types. Our results support the hypothesis of co-adaptive transformation of stromal and epithelial tissues during breast tumoral development and suggest that different types of fibroblasts give rise to different types of myofibroblasts. Discriminant analysis of quantitative molecular variation may be considered for the development of a powerful artificial-intelligence method for cell typing and should be particularly useful when no reliable discrete molecular markers are available.',\n",
              "  'authors': ['Spanakis, E', 'Brouty-Boyé, D'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'International journal of cancer',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Neoplasms',\n",
              "   'Discriminant Analysis',\n",
              "   'Enzymes',\n",
              "   'Female',\n",
              "   'Fibroblasts',\n",
              "   'Fibrosis',\n",
              "   'Gene Expression',\n",
              "   'Growth Substances',\n",
              "   'Humans',\n",
              "   'Interleukins',\n",
              "   'Multivariate Analysis',\n",
              "   'Nuclear Proteins',\n",
              "   'Phenotype',\n",
              "   'Protein Kinases',\n",
              "   'Skin'],\n",
              "  'pmid': '9139876',\n",
              "  'title': 'Discrimination of fibroblast subtypes by multivariate analysis of gene expression.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1827-1626',\n",
              "  'IF_2013': '0.707',\n",
              "  'IF_2014': '0.678',\n",
              "  'IF_2015': '0.877',\n",
              "  'IF_2016': '1.115',\n",
              "  'IF_2017': '0.554',\n",
              "  'ISSN': '0026-4733',\n",
              "  'ISSN_online': '1827-1626',\n",
              "  'ISSN_print': '0026-4733',\n",
              "  'IsoAbbr': 'Minerva Chir',\n",
              "  'JrId': 5828,\n",
              "  'MedAbbr': 'Minerva Chir',\n",
              "  'NLMid': '0400726',\n",
              "  'Titre': 'Minerva Chir',\n",
              "  'abstract': 'After nearly a century of performing a thyroid-ectomy essentially the way it was described by Theodore Kocher in the nineteenth century, the technique has suddenly and rapidly evolved. It can now be accomplished endoscopically in many patients who therefore benefit from the reduced dissection and smaller incisions associated with the approach. While many of the cosmetic, quality of life, and functional improvements have now been documented, an improved understanding of the procedure and the appropriate indications for its application will continue to develop even as the technique itself evolves, and as new approaches emerge.',\n",
              "  'authors': ['Terris, D J'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Minerva chirurgica',\n",
              "  'keywords': ['Endoscopy', 'Humans', 'Robotics', 'Thyroidectomy'],\n",
              "  'pmid': '19648854',\n",
              "  'title': 'Endoscopic and robotic thyroidectomy: past and future.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': \"This paper presents an application of adaptive noise cancellation with neural-network-based fuzzy inference system (NNFIS) for rapid estimation of visual evoked potentials (VEPs). Usually a recorded VEP is severely contaminated by background ongoing activities of the spontaneous EEG signal in the human brain. Many approaches have been adopted to enhance the signal-to-noise ratio (SNR) of the recorded signal. However, nonlinear dynamic methods are rarely investigated in view of their complexity, and the fact that the nonlinear characteristics of the signal are hard to determine in general. An adaptive noise cancellation method with NNFIS was carefully designed to estimate the VEP signal. NNFIS, based on Takagi and Sugeno's fuzzy model, has the advantage of being linear-in-parameter; thus the conventional adaptive methods can be efficiently utilized to estimate its parameters. Another advantage of NNFIS lies in that it can track the dynamic behavior of VEP in a real-time fashion because the VEP variation tracking is important for critical patient monitoring in the clinical situation. A series of computer experiments conducted on simulated and real-test responses have confirmed the superiority of the method developed in this paper.\",\n",
              "  'authors': ['Yin, Haie', 'Zeng, Yanjun', 'Zhang, Jianhua', 'Pan, Yinfu'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electroencephalography',\n",
              "   'Evoked Potentials, Visual',\n",
              "   'Feedback',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '14644601',\n",
              "  'title': 'Application of adaptive noise cancellation with neural-network-based fuzzy inference system for visual evoked potentials estimation.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'Renal biopsies are usually performed by a nephrologist and require ultrasound guidance from an expert. This procedure can be time-consuming and expensive.|We propose using a robotic arm, controlled from a remote location, to perform ultrasound guidance during renal biopsies in postrenal transplant patients. The procedure would be performed under ultrasound guidance from a radiologist, controlling a robotic arm from one institution, while the nephrologist would perform the biopsy at another. This would allow biopsies to be performed under the guidance of an expert in ultrasonography, without requiring him to leave his institution.|Four patients underwent teleoperated renal biopsy. None of the patients had any postbiopsy complication. Total time used to perform the renal biopsies was significantly increased when compared with conventional biopsy technique. Compared with conventional ultrasound guidance, the nephrologists performing the biopsies reported discomfort with the robotic arm carrying out the ultrasonography.|We demonstrated the feasibility of teleoperated ultrasound-guided renal biopsies. This technique can be used to compensate for the absence of an ultrasonography expert and save time. Additional invasive urologic procedures using the ESTELE robot arm, including renal punctures before percutaneous nephrolithotomy and prostate biopsies, are planned.',\n",
              "  'authors': ['Bruyère, Franck', 'Ayoub, Jean', 'Arbeille, Philippe'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Biopsy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Kidney',\n",
              "   'Kidney Transplantation',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Ultrasonics',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '21050028',\n",
              "  'title': 'Use of a telerobotic arm to perform ultrasound guidance during renal biopsy in transplant recipients: a preliminary study.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1097-4199',\n",
              "  'IF_2013': '15.982',\n",
              "  'IF_2014': '15.054',\n",
              "  'IF_2015': '13.974',\n",
              "  'IF_2016': '14.024',\n",
              "  'IF_2017': '14.318',\n",
              "  'ISSN': '0896-6273',\n",
              "  'ISSN_online': '1097-4199',\n",
              "  'ISSN_print': '0896-6273',\n",
              "  'IsoAbbr': 'Neuron',\n",
              "  'JrId': 1600,\n",
              "  'MedAbbr': 'Neuron',\n",
              "  'NLMid': '8809320',\n",
              "  'Titre': 'Neuron',\n",
              "  'abstract': 'Long-term potentiation and depression (LTP and LTD) in excitatory synapses can coexist, the former being triggered by stimuli that produce strong postsynaptic excitation and the latter by stimuli that produce weaker postsynaptic excitation. It has not been determined whether these properties also apply to LTP and LTD in the inhibitory synapses between Purkinje neurons and the neurons of the deep cerebellar nuclei (DCN), a site that has been implicated in certain types of motor learning. DCN cells exhibit a prominent rebound depolarization (RD) and associated spike burst upon release from hyperpolarization. In these cells, LTP can be elicited by short, high-frequency trains of inhibitory postsynaptic potentials (IPSPs), which reliably evoke an RD. LTD is induced if the same protocol is applied with conditions where the amount of postsynaptic excitation is reduced. The polarity of the change in synaptic strength is correlated with the amount of RD-evoked spike firing during the induction protocol. Thus, some important computational principles that govern the induction of use-dependent change in excitatory synaptic efficacy also apply to inhibitory synapses.',\n",
              "  'authors': ['Aizenman, C D', 'Manis, P B', 'Linden, D J'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Neuron',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Animals',\n",
              "   'Calcium',\n",
              "   'Cerebellar Nuclei',\n",
              "   'Electric Stimulation',\n",
              "   'Electrophysiology',\n",
              "   'In Vitro Techniques',\n",
              "   'Long-Term Potentiation',\n",
              "   'Neural Inhibition',\n",
              "   'Neuronal Plasticity',\n",
              "   'Rats',\n",
              "   'Synapses',\n",
              "   'Synaptic Transmission',\n",
              "   'Time Factors'],\n",
              "  'pmid': '9808468',\n",
              "  'title': 'Polarity of long-term synaptic gain change is related to postsynaptic spike firing at a cerebellar inhibitory synapse.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Controllability and observability of complex systems are vital concepts in many fields of science. The network structure of the system plays a crucial role in determining its controllability and observability. Because most naturally occurring complex systems show dynamic changes in their network connectivity, it is important to understand how perturbations in the connectivity affect the controllability of the system. To this end, we studied the control structure of different types of artificial, social and biological neuronal networks (BNN) as their connections were progressively pruned using four different pruning strategies. We show that the BNNs are more similar to scale-free networks than to small-world networks, when comparing the robustness of their control structure to structural perturbations. We introduce a new graph descriptor, 'the cardinality curve', to quantify the robustness of the control structure of a network to progressive edge pruning. Knowing the susceptibility of control structures to different pruning methods could help design strategies to destroy the control structures of dangerous networks such as epidemic networks. On the other hand, it could help make useful networks more resistant to edge attacks. \",\n",
              "  'authors': ['Mengiste, Simachew Abebe', 'Aertsen, Ad', 'Kumar, Arvind'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons'],\n",
              "  'pmid': '26674854',\n",
              "  'title': 'Effect of edge pruning on structural controllability and observability of complex networks.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1941-0506',\n",
              "  'IF_2013': '1.919',\n",
              "  'IF_2014': '2.168',\n",
              "  'IF_2015': '1.4',\n",
              "  'IF_2016': '2.84',\n",
              "  'IF_2017': '3.078',\n",
              "  'ISSN': '1077-2626',\n",
              "  'ISSN_online': '1941-0506',\n",
              "  'ISSN_print': '1077-2626',\n",
              "  'IsoAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'JrId': 31980,\n",
              "  'MedAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'NLMid': '9891704',\n",
              "  'Titre': 'IEEE Trans Vis Comput Graph',\n",
              "  'abstract': 'The stable local classification of discrete surfaces with respect to features such as edges and corners or concave and convex regions, respectively, is as quite difficult as well as indispensable for many surface processing applications. Usually, the feature detection is done via a local curvature analysis. If concerned with large triangular and irregular grids, e.g., generated via a marching cube algorithm, the detectors are tedious to treat and a robust classification is hard to achieve. Here, a local classification method on surfaces is presented which avoids the evaluation of discretized curvature quantities. Moreover, it provides an indicator for smoothness of a given discrete surface and comes together with a built-in multiscale. The proposed classification tool is based on local zero and first moments on the discrete surface. The corresponding integral quantities are stable to compute and they give less noisy results compared to discrete curvature quantities. The stencil width for the integration of the moments turns out to be the scale parameter. Prospective surface processing applications are the segmentation on surfaces, surface comparison, and matching and surface modeling. Here, a method for feature preserving fairing of surfaces is discussed to underline the applicability of the presented approach.',\n",
              "  'authors': ['Clarenz, Ulrich', 'Rumpf, Martin', 'Telea, Alexandru'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE transactions on visualization and computer graphics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Graphics',\n",
              "   'Computer Simulation',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '15794134',\n",
              "  'title': 'Robust feature detection and local classification for surfaces based on moment analysis.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Current research aims at identifying voluntary brain activation in patients who are behaviorally diagnosed as being unconscious, but are able to perform commands by modulating their brain activity patterns. This involves machine learning techniques and feature extraction methods such as applied in brain computer interfaces. In this study, we try to answer the question if features/classification methods which show advantages in healthy participants are also accurate when applied to data of patients with disorders of consciousness. A sample of healthy participants (N = 22), patients in a minimally conscious state (MCS; N = 5), and with unresponsive wakefulness syndrome (UWS; N = 9) was examined with a motor imagery task which involved imagery of moving both hands and an instruction to hold both hands firm. We extracted a set of 20 features from the electroencephalogram and used linear discriminant analysis, k-nearest neighbor classification, and support vector machines (SVM) as classification methods. In healthy participants, the best classification accuracies were seen with coherences (mean = .79; range = .53-.94) and power spectra (mean = .69; range = .40-.85). The coherence patterns in healthy participants did not match the expectation of central modulated [Formula: see text]-rhythm. Instead, coherence involved mainly frontal regions. In healthy participants, the best classification tool was SVM. Five patients had at least one feature-classifier outcome with p[Formula: see text]0.05 (none of which were coherence or power spectra), though none remained significant after false-discovery rate correction for multiple comparisons. The present work suggests the use of coherences in patients with disorders of consciousness because they show high reliability among healthy subjects and patient groups. However, feature extraction and classification is a challenging task in unresponsive patients because there is no ground truth to validate the results.',\n",
              "  'authors': ['Höller, Yvonne',\n",
              "   'Bergmann, Jürgen',\n",
              "   'Thomschewski, Aljoscha',\n",
              "   'Kronbichler, Martin',\n",
              "   'Höller, Peter',\n",
              "   'Crone, Julia S',\n",
              "   'Schmid, Elisabeth V',\n",
              "   'Butz, Kevin',\n",
              "   'Nardone, Raffaele',\n",
              "   'Trinka, Eugen'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Brain',\n",
              "   'Brain-Computer Interfaces',\n",
              "   'Consciousness Disorders',\n",
              "   'Discriminant Analysis',\n",
              "   'Electroencephalography',\n",
              "   'Humans',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24282545',\n",
              "  'title': 'Comparison of EEG-features and classification methods for motor imagery in patients with disorders of consciousness.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Electrical Impedance Tomography (EIT) is a powerful non-invasive technique for imaging applications. The goal is to estimate the electrical properties of living tissues by measuring the potential at the boundary of the domain. Being safe with respect to patient health, non-invasive, and having no known hazards, EIT is an attractive and promising technology. However, it suffers from a particular technical difficulty, which consists of solving a nonlinear inverse problem in real time. Several nonlinear approaches have been proposed as a replacement for the linear solver, but in practice very few are capable of stable, high-quality, and real-time EIT imaging because of their very low robustness to errors and inaccurate modeling, or because they require considerable computational effort.|In this paper, a post-processing technique based on an artificial neural network (ANN) is proposed to obtain a nonlinear solution to the inverse problem, starting from a linear solution. While common reconstruction methods based on ANNs estimate the solution directly from the measured data, the method proposed here enhances the solution obtained from a linear solver.|Applying a linear reconstruction algorithm before applying an ANN reduces the effects of noise and modeling errors. Hence, this approach significantly reduces the error associated with solving 2D inverse problems using machine-learning-based algorithms.|This work presents radical enhancements in the stability of nonlinear methods for biomedical EIT applications.',\n",
              "  'authors': ['Martin, Sébastien', 'Choi, Charles T M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Electric Impedance',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Phantoms, Imaging',\n",
              "   'Tomography'],\n",
              "  'pmid': '29206856',\n",
              "  'title': 'A novel post-processing scheme for two-dimensional electrical impedance tomography based on artificial neural networks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Synaptic plasticity, the cellular basis for learning and memory, is mediated by a complex biochemical network of signaling proteins. These proteins are compartmentalized in dendritic spines, the tiny, bulbous, post-synaptic structures found on neuronal dendrites. The ability to screen a high number of molecular targets for their effect on dendritic spine structural plasticity will require a high-throughput imaging system capable of stimulating and monitoring hundreds of dendritic spines in various conditions. For this purpose, we present a program capable of automatically identifying dendritic spines in live, fluorescent tissue. Our software relies on a machine learning approach to minimize any need for parameter tuning from the user. Custom thresholding and binarization functions serve to \"clean\" fluorescent images, and a neural network is trained using features based on the relative shape of the spine perimeter and its corresponding dendritic backbone. Our algorithm is rapid, flexible, has over 90% accuracy in spine detection, and bundled with our user-friendly, open-source, MATLAB-based software package for spine analysis.',\n",
              "  'authors': ['Smirnov, Michael S', 'Garrett, Tavita R', 'Yasuda, Ryohei'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Dendritic Spines',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Mice',\n",
              "   'Microscopy',\n",
              "   'Neuronal Plasticity',\n",
              "   'Neurons',\n",
              "   'Software'],\n",
              "  'pmid': '29975722',\n",
              "  'title': 'An open-source tool for analysis and automatic identification of dendritic spines using machine learning.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Aerial surveys of marine mammals are routinely conducted to assess and monitor species' habitat use and population status. In Australia, dugongs (Dugong dugon) are regularly surveyed and long-term datasets have formed the basis for defining habitat of high conservation value and risk assessments of human impacts. Unmanned aerial vehicles (UAVs) may facilitate more accurate, human-risk free, and cheaper aerial surveys. We undertook the first Australian UAV survey trial in Shark Bay, western Australia. We conducted seven flights of the ScanEagle UAV, mounted with a digital SLR camera payload. During each flight, ten transects covering a 1.3 km(2) area frequently used by dugongs, were flown at 500, 750 and 1000 ft. Image (photograph) capture was controlled via the Ground Control Station and the capture rate was scheduled to achieve a prescribed 10% overlap between images along transect lines. Images were manually reviewed post hoc for animals and scored according to sun glitter, Beaufort Sea state and turbidity. We captured 6243 images, 627 containing dugongs. We also identified whales, dolphins, turtles and a range of other fauna. Of all possible dugong sightings, 95% (CI = 90%, 98%) were subjectively classed as 'certain' (unmistakably dugongs). Neither our dugong sighting rate, nor our ability to identify dugongs with certainty, were affected by UAV altitude. Turbidity was the only environmental variable significantly affecting the dugong sighting rate. Our results suggest that UAV systems may not be limited by sea state conditions in the same manner as sightings from manned surveys. The overlap between images proved valuable for detecting animals that were masked by sun glitter in the corners of images, and identifying animals initially captured at awkward body angles. This initial trial of a basic camera system has successfully demonstrated that the ScanEagle UAV has great potential as a tool for marine mammal aerial surveys. \",\n",
              "  'authors': ['Hodgson, Amanda', 'Kelly, Natalie', 'Peel, David'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Aircraft',\n",
              "   'Animals',\n",
              "   'Aquatic Organisms',\n",
              "   'Dugong',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Robotics',\n",
              "   'Sunlight'],\n",
              "  'pmid': '24223967',\n",
              "  'title': 'Unmanned aerial vehicles (UAVs) for surveying marine fauna: a dugong case study.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['AD'],\n",
              "  'ESSN': '1525-3171',\n",
              "  'IF_2013': '1.544',\n",
              "  'IF_2014': '1.672',\n",
              "  'IF_2015': '1.685',\n",
              "  'IF_2016': '1.908',\n",
              "  'IF_2017': '2.216',\n",
              "  'ISSN': '0032-5791',\n",
              "  'ISSN_online': '1525-3171',\n",
              "  'ISSN_print': '0032-5791',\n",
              "  'IsoAbbr': 'Poult. Sci.',\n",
              "  'JrId': 6560,\n",
              "  'MedAbbr': 'Poult Sci',\n",
              "  'NLMid': '0401150',\n",
              "  'Titre': 'Poult. Sci.',\n",
              "  'abstract': 'Previously, evaluation of the first 2 wk of daily growth velocity with an artificial neural network (ANN) provided an effective noninvasive approach for predicting the susceptibility of broilers to pulmonary hypertension syndrome (PHS). This study was conducted to define the minimum number of days of growth data and the type of ANN required for the best prediction of PHS susceptibility. Four experiments were conducted in which broilers were weighed daily at 0800 h. In Experiment 1, Hubbard male broilers were reared to 50 d of age, with 13 developing PHS and 33 remaining normal (N), for a PHS:N ratio of 13:33. In Experiment 2, ANAK broilers were exposed to cool temperatures (16 to 17 C) from 17 to 42 d of age, resulting in a PHS:N ratio of 16:46 for males. In Experiments 3 and 4, Hubbard male and female chicks from a base population and a PHS-resistant line were exposed to cool temperatures from 17 to 42 d (Experiment 3) or 49 d of age (Experiment 4). The PHS:N ratios were 40:68 for males and 6:96 for females in Experiment 3 and 26:91 for males and 10:58 for females in Experiment 4. Four ANN, back propagation (BP3), Ward back propagation (WardBP), probabilistic (PNN), and general regression (GRNN), were evaluated for their ability to predict PHS in the shortest number of days based on daily growth velocities (BWd+1-BWd). A 100% prediction of PHS and N birds was considered the criterion of success. Starting with 14 d of data, each ANN was trained on daily growth velocity, and the number of predictive days was reduced with each run of the ANN. The best ANN was a GRNN, which correctly diagnosed PHS and N male broilers on 4 and 6 d of growth velocity data for Experiments 1 and 2, respectively. The results were poorer with the BP3, WardBP, and PNN. The diagnostic ability of the neural network was not consistent over all four experiments. In Experiment 2, a minimum of 6 d was required for 100% PHS detection for males. In Experiment 3, the best diagnostic value for males was 93% PHS detection and 100% N detection at 15 d. For females, the 100% PHS detection occurred at a minimum of 8 d. In Experiment 4, males had 100% PHS and N detection at a minimum of 11 d. Females had a 100% PHS and N detection at a minimum of 10 d. An attempt to build a single neural network that would detect PHS susceptibility in Hubbard (Experiment 1) and ANAK (Experiment 2) broilers was unsuccessful. The application (validation) of neural networks between experiments also was not successful (data not presented). However, these studies demonstrate that within a breed or line reared under similar selection pressures for ascites, a GRNN based on the first 14 d of growth velocity can detect, with at least 93% accuracy, broilers susceptible to PHS.',\n",
              "  'authors': ['Roush, W B',\n",
              "   'Wideman, R F',\n",
              "   'Cahaner, A',\n",
              "   'Deeb, N',\n",
              "   'Cravener, T L'],\n",
              "  'categories': ['AGRICULTURE, DAIRY & ANIMAL SCIENCE'],\n",
              "  'journal': 'Poultry science',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Animals',\n",
              "   'Chickens',\n",
              "   'Disease Susceptibility',\n",
              "   'Female',\n",
              "   'Hypertension, Pulmonary',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Poultry Diseases',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Syndrome',\n",
              "   'Temperature',\n",
              "   'Weight Gain'],\n",
              "  'pmid': '11261552',\n",
              "  'title': 'Minimal number of chicken daily growth velocities for artificial neural network detection of pulmonary hypertension syndrome (PHS).',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1758-0463',\n",
              "  'IF_2013': '4.457',\n",
              "  'IF_2014': '3.372',\n",
              "  'IF_2015': '2.627',\n",
              "  'IF_2016': '3.29',\n",
              "  'IF_2017': '3.978',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1758-0463',\n",
              "  'IsoAbbr': 'Database (Oxford)',\n",
              "  'JrId': 36699,\n",
              "  'MedAbbr': 'Database (Oxford)',\n",
              "  'NLMid': '101517697',\n",
              "  'Titre': 'Database (Oxford)',\n",
              "  'abstract': 'We investigate the application of deep learning to biocuration tasks that involve classification of text associated with biomedical evidence in primary research articles. We developed a large-scale corpus of molecular papers derived from PubMed and PubMed Central open access records and used it to train deep learning word embeddings under the GloVe, FastText and ELMo algorithms. We applied those models to a distant supervised method classification task based on text from figure captions or fragments surrounding references to figures in the main text using a variety or models and parameterizations. We then developed document classification (triage) methods for molecular interaction papers by using deep learning mechanisms of attention to aggregate classification-based decisions over selected paragraphs in the document. We were able to obtain triage performance with an accuracy of 0.82 using a combined convolutional neural network, bi-directional long short-term memory architecture augmented by attention to produce a single decision for triage. In this work, we hope to encourage biocuration systems developers to apply deep learning methods to their specialized tasks by repurposing large-scale word embedding to apply to their data.',\n",
              "  'authors': ['Burns, Gully A', 'Li, Xiangci', 'Peng, Nanyun'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Database : the journal of biological databases and curation',\n",
              "  'keywords': ['Deep Learning',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Publications',\n",
              "   'Semantics'],\n",
              "  'pmid': '30938776',\n",
              "  'title': 'Building deep learning models for evidence classification from the open access biomedical literature.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1755-8794',\n",
              "  'IF_2013': '3.914',\n",
              "  'IF_2014': '2.873',\n",
              "  'IF_2015': '2.726',\n",
              "  'IF_2016': '2.848',\n",
              "  'IF_2017': '3.317',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1755-8794',\n",
              "  'IsoAbbr': 'BMC Med Genomics',\n",
              "  'JrId': 34003,\n",
              "  'MedAbbr': 'BMC Med Genomics',\n",
              "  'NLMid': '101319628',\n",
              "  'Titre': 'BMC Med Genomics',\n",
              "  'abstract': 'Normal tissue samples are often employed as a control for understanding disease mechanisms, however, collecting matched normal tissues from patients is difficult in many instances. In cancer research, for example, the open cancer resources such as TCGA and TARGET do not provide matched tissue samples for every cancer or cancer subtype. The recent GTEx project has profiled samples from healthy individuals, providing an excellent resource for this field, yet the feasibility of using GTEx samples as the reference remains unanswered.|We analyze RNA-Seq data processed from the same computational pipeline and systematically evaluate GTEx as a potential reference resource. We use those cancers that have adjacent normal tissues in TCGA as a benchmark for the evaluation. To correlate tumor samples and normal samples, we explore top varying genes, reduced features from principal component analysis, and encoded features from an autoencoder neural network. We first evaluate whether these methods can identify the correct tissue of origin from GTEx for a given cancer and then seek to answer whether disease expression signatures are consistent between those derived from TCGA and from GTEx.|Among 32 TCGA cancers, 18 cancers have less than 10 matched adjacent normal tissue samples. Among three methods, autoencoder performed the best in predicting tissue of origin, with 12 of 14 cancers correctly predicted. The reason for misclassification of two cancers is that none of normal samples from GTEx correlate well with any tumor samples in these cancers. This suggests that GTEx has matched tissues for the majority cancers, but not all. While using autoencoder to select proper normal samples for disease signature creation, we found that disease signatures derived from normal samples selected via an autoencoder from GTEx are consistent with those derived from adjacent samples from TCGA in many cases. Interestingly, choosing top 50 mostly correlated samples regardless of tissue type performed reasonably well or even better in some cancers.|Our findings demonstrate that samples from GTEx can serve as reference normal samples for cancers, especially those do not have available adjacent tissue samples. A deep-learning based approach holds promise to select proper normal samples.',\n",
              "  'authors': ['Zeng, William Z D',\n",
              "   'Glicksberg, Benjamin S',\n",
              "   'Li, Yangyan',\n",
              "   'Chen, Bin'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC medical genomics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30704474',\n",
              "  'title': 'Selecting precise reference normal tissue samples for cancer research using a deep learning approach.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1839-3535',\n",
              "  'IF_2013': '0.651',\n",
              "  'IF_2014': '0.65',\n",
              "  'IF_2015': '0.886',\n",
              "  'IF_2016': '0.836',\n",
              "  'IF_2017': '0.673',\n",
              "  'ISSN': '1037-6178',\n",
              "  'ISSN_online': '1839-3535',\n",
              "  'ISSN_print': '1037-6178',\n",
              "  'IsoAbbr': 'Contemp Nurse',\n",
              "  'JrId': 2147,\n",
              "  'MedAbbr': 'Contemp Nurse',\n",
              "  'NLMid': '9211867',\n",
              "  'Titre': 'Contemp Nurse',\n",
              "  'abstract': 'Technology in the healthcare sector is undergoing rapid development. One of the most prominent areas of healthcare in which robots are implemented is nursing homes. However, nursing and technology are often considered as being contradictory, an attitude originating in the view of \"the natural\" versus \"the artificial\". Social media mirror this view, including in attitudes and societal debates regarding nursing and care robots. However, little is known about this topic in previous research.|To examine user behaviour in social media platforms on the topic of nursing and care robots.|A retrospective and cross-sectional observation study design was applied.|Data were collected via the Alchemy streaming application programming interface. Data from social media were collected from 1 January 2014 to 5 January 2016. The data set consisted of 12,311 mentions in total.|Nursing and care robots are a small-scale topic of discussion in social media. Twitter was found to be the largest channel in terms of volume, followed by Tumblr. News channels had the highest percentage of visibility, while forums and Tumblr had the least. It was found in the data that 67.9% of the mentions were positive, 24.4% were negative and 7.8% were neutral.|The volume and visibility of the data on nursing robots found in social media, as well as the attitudes to nursing robots found there, indicate that nursing care robots, which are seen as representing a next step in technological development in healthcare, are a topic on the rise in social media. These findings are likely to be related to the idea that nursing care robots are on the breakthrough of replacing human labour in healthcare worldwide.',\n",
              "  'authors': ['Salzmann-Erikson, Martin', 'Eriksson, Henrik'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Contemporary nurse',\n",
              "  'keywords': ['Adult',\n",
              "   'Attitude to Computers',\n",
              "   'Cross-Sectional Studies',\n",
              "   'Female',\n",
              "   'Health Personnel',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nursing Care',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Social Media'],\n",
              "  'pmid': '28980506',\n",
              "  'title': 'A descriptive statistical analysis of volume, visibility and attitudes regarding nursing and care robots in social media.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DA'],\n",
              "  'ESSN': '1542-0086',\n",
              "  'IF_2013': '3.832',\n",
              "  'IF_2014': '3.972',\n",
              "  'IF_2015': '3.632',\n",
              "  'IF_2016': '3.656',\n",
              "  'IF_2017': '3.495',\n",
              "  'ISSN': '0006-3495',\n",
              "  'ISSN_online': '1542-0086',\n",
              "  'ISSN_print': '0006-3495',\n",
              "  'IsoAbbr': 'Biophys. J.',\n",
              "  'JrId': 1174,\n",
              "  'MedAbbr': 'Biophys J',\n",
              "  'NLMid': '0370626',\n",
              "  'Titre': 'Biophys. J.',\n",
              "  'abstract': 'Identifying the fold class of a protein sequence of unknown structure is a fundamental problem in modern biology. We apply a supervised learning algorithm to the classification of protein sequences with low sequence identity from a library of 174 structural classes created with the Combinatorial Extension structural alignment methodology. A class of rules is considered that assigns test sequences to structural classes based on the closest match of an amino acid index profile of the test sequence to a profile centroid for each class. A mathematical optimization procedure is applied to determine an amino acid index of maximal structural discriminatory power by maximizing the ratio of between-class to within-class profile variation. The optimal index is computed as the solution to a generalized eigenvalue problem, and its performance for fold classification is compared to that of other published indices. The optimal index has significantly more structural discriminatory power than all currently known indices, including average surrounding hydrophobicity, which it most closely resembles. It demonstrates >70% classification accuracy over all folds and nearly 100% accuracy on several folds with distinctive conserved structural features. Finally, there is a compelling universality to the optimal index in that it does not appear to depend strongly on the specific structural classes used in its computation.',\n",
              "  'authors': ['Leary, R H', 'Rosen, J B', 'Jambeck, P'],\n",
              "  'categories': ['BIOPHYSICS'],\n",
              "  'journal': 'Biophysical journal',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Sequence',\n",
              "   'Amino Acids',\n",
              "   'Artificial Intelligence',\n",
              "   'Models, Molecular',\n",
              "   'Molecular Sequence Data',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Protein Folding',\n",
              "   'Proteins',\n",
              "   'Sequence Analysis, Protein'],\n",
              "  'pmid': '14695283',\n",
              "  'title': 'An optimal structure-discriminative amino acid index for protein fold recognition.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The automatic detection and recognition of sound events by computers is a requirement for a number of emerging sensing and human computer interaction technologies. Recent advances in this field have been achieved by machine learning classifiers working in conjunction with time-frequency feature representations. This combination has achieved excellent accuracy for classification of discrete sounds. The ability to recognise sounds under real-world noisy conditions, called robust sound event classification, is an especially challenging task that has attracted recent research attention. Another aspect of real-word conditions is the classification of continuous, occluded or overlapping sounds, rather than classification of short isolated sound recordings. This paper addresses the classification of noise-corrupted, occluded, overlapped, continuous sound recordings. It first proposes a standard evaluation task for such sounds based upon a common existing method for evaluating isolated sound classification. It then benchmarks several high performing isolated sound classifiers to operate with continuous sound data by incorporating an energy-based event detection front end. Results are reported for each tested system using the new task, to provide the first analysis of their performance for continuous sound event detection. In addition it proposes and evaluates a novel Bayesian-inspired front end for the segmentation and detection of continuous sound recordings prior to classification.',\n",
              "  'authors': ['McLoughlin, Ian',\n",
              "   'Zhang, Haomin',\n",
              "   'Xie, Zhipeng',\n",
              "   'Song, Yan',\n",
              "   'Xiao, Wei',\n",
              "   'Phan, Huy'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Models, Theoretical',\n",
              "   'Sound'],\n",
              "  'pmid': '28892478',\n",
              "  'title': 'Continuous robust sound event classification using time-frequency features and deep learning.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2005-8330',\n",
              "  'IF_2013': '1.807',\n",
              "  'IF_2014': '1.571',\n",
              "  'IF_2015': '1.592',\n",
              "  'IF_2016': '2.156',\n",
              "  'IF_2017': '3.072',\n",
              "  'ISSN': '1229-6929',\n",
              "  'ISSN_online': '2005-8330',\n",
              "  'ISSN_print': '1229-6929',\n",
              "  'IsoAbbr': 'Korean J Radiol',\n",
              "  'JrId': 26838,\n",
              "  'MedAbbr': 'Korean J Radiol',\n",
              "  'NLMid': '100956096',\n",
              "  'Titre': 'Korean J Radiol',\n",
              "  'abstract': \"To compare the observer preference of image quality and radiation dose between non-grid, grid-like, and grid images.|Each of the 38 patients underwent bedside chest radiography with and without a grid. A grid-like image was generated from a non-grid image using SimGrid software (Samsung Electronics Co. Ltd.) employing deep-learning-based scatter correction technology. Two readers recorded the preference for 10 anatomic landmarks and the overall appearance on a five-point scale for a pair of non-grid and grid-like images, and a pair of grid-like and grid images, respectively, which were randomly presented. The dose area product (DAP) was also recorded. Wilcoxon's rank sum test was used to assess the significance of preference.|Both readers preferred grid-like images to non-grid images significantly (p < 0.001); with a significant difference in terms of the preference for grid images to grid-like images (p = 0.317, 0.034, respectively). In terms of anatomic landmarks, both readers preferred grid-like images to non-grid images (p < 0.05). No significant differences existed between grid-like and grid images except for the preference for grid images in proximal airways by two readers, and in retrocardiac lung and thoracic spine by one reader. The median DAP were 1.48 (range, 1.37-2.17) dGy*cm2 in grid images and 1.22 (range, 1.11-1.78) dGy*cm2 in grid-like images with a significant difference (p < 0.001).|The SimGrid software significantly improved the image quality of non-grid images to a level comparable to that of grid images with a relatively lower level of radiation exposure.\",\n",
              "  'authors': ['Ahn, Su Yeon', 'Chae, Kum Ju', 'Goo, Jin Mo'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Korean journal of radiology',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Humans',\n",
              "   'Prospective Studies',\n",
              "   'Radiation Dosage',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiography, Thoracic',\n",
              "   'Software',\n",
              "   'Thorax'],\n",
              "  'pmid': '29713231',\n",
              "  'title': 'The Potential Role of Grid-Like Software in Bedside Chest Radiography in Improving Image Quality and Dose Reduction: An Observer Preference Study.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': \"Most current algorithms for automatic glaucoma assessment using fundus images rely on handcrafted features based on segmentation, which are affected by the performance of the chosen segmentation method and the extracted features. Among other characteristics, convolutional neural networks (CNNs) are known because of their ability to learn highly discriminative features from raw pixel intensities.|In this paper, we employed five different ImageNet-trained models (VGG16, VGG19, InceptionV3, ResNet50 and Xception) for automatic glaucoma assessment using fundus images. Results from an extensive validation using cross-validation and cross-testing strategies were compared with previous works in the literature.|Using five public databases (1707 images), an average AUC of 0.9605 with a 95% confidence interval of 95.92-97.07%, an average specificity of 0.8580 and an average sensitivity of 0.9346 were obtained after using the Xception architecture, significantly improving the performance of other state-of-the-art works. Moreover, a new clinical database, ACRIMA, has been made publicly available, containing 705 labelled images. It is composed of 396 glaucomatous images and 309 normal images, which means, the largest public database for glaucoma diagnosis. The high specificity and sensitivity obtained from the proposed approach are supported by an extensive validation using not only the cross-validation strategy but also the cross-testing validation on, to the best of the authors' knowledge, all publicly available glaucoma-labelled databases.|These results suggest that using ImageNet-trained models is a robust alternative for automatic glaucoma screening system. All images, CNN weights and software used to fine-tune and test the five CNNs are publicly available, which could be used as a testbed for further comparisons.\",\n",
              "  'authors': ['Diaz-Pinto, Andres',\n",
              "   'Morales, Sandra',\n",
              "   'Naranjo, Valery',\n",
              "   'Köhler, Thomas',\n",
              "   'Mossi, Jose M',\n",
              "   'Navea, Amparo'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Databases, Factual',\n",
              "   'Fundus Oculi',\n",
              "   'Glaucoma',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Time Factors'],\n",
              "  'pmid': '30894178',\n",
              "  'title': 'CNNs for automatic glaucoma assessment using fundus images: an extensive validation.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1744-8328',\n",
              "  'IF_2013': '2.279',\n",
              "  'IF_2014': '2.249',\n",
              "  'IF_2015': '2.094',\n",
              "  'IF_2016': '2.212',\n",
              "  'IF_2017': '2.347',\n",
              "  'ISSN': '1473-7140',\n",
              "  'ISSN_online': '1744-8328',\n",
              "  'ISSN_print': '1473-7140',\n",
              "  'IsoAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'JrId': 29775,\n",
              "  'MedAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'NLMid': '101123358',\n",
              "  'Titre': 'Expert Rev Anticancer Ther',\n",
              "  'abstract': 'The standard localized therapies for prostate cancer include external-beam radiation therapy, brachytherapy and radical prostatectomy. There are several novel approaches in development aimed at improving local disease control and survival, and reducing post-treatment complications. In low-to-intermediate-risk patients, new radiation approaches are being explored to include hypofractionated robotic radiation therapy. For high-risk patients, the focus is on multimodality approaches, especially the addition of chemotherapy. Recent developments in radiation therapy and adjuvant chemotherapy are the focus of this review.',\n",
              "  'authors': ['Dawson, Nancy A', 'Collins, Sean P'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Expert review of anticancer therapy',\n",
              "  'keywords': ['Brachytherapy',\n",
              "   'Chemotherapy, Adjuvant',\n",
              "   'Clinical Trials as Topic',\n",
              "   'Combined Modality Therapy',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Survival Rate'],\n",
              "  'pmid': '19589034',\n",
              "  'title': 'Novel treatment methods for localized prostate cancer: hypofractionated robotic radiation therapy and adjuvant chemotherapy.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': \"Macaque monkeys were tested on a delayed-match-to-multiple-sample task, with either a limited set of well trained images (in randomized sequence) or with never-before-seen images. They performed much better with novel images. False positives were mostly limited to catch-trial image repetitions from the preceding trial. This result implies extremely effective one-shot learning, resembling Standing's finding that people detect familiarity for 10,000 once-seen pictures (with 80% accuracy) (Standing, 1973). Familiarity memory may differ essentially from identification, which embeds and generates contextual information. When encountering another person, we can say immediately whether his or her face is familiar. However, it may be difficult for us to identify the same person. To accompany the psychophysical findings, we present a generic neural network model reproducing these behaviors, based on the same conservative Hebbian synaptic plasticity that generates delay activity identification memory. Familiarity becomes the first step toward establishing identification. Adding an inter-trial reset mechanism limits false positives for previous-trial images. The model, unlike previous proposals, relates repetition-recognition with enhanced neural activity, as recently observed experimentally in 92% of differential cells in prefrontal cortex, an area directly involved in familiarity recognition. There may be an essential functional difference between enhanced responses to novel versus to familiar images: The maximal signal from temporal cortex is for novel stimuli, facilitating additional sensory processing of newly acquired stimuli. The maximal signal for familiar stimuli arising in prefrontal cortex facilitates the formation of selective delay activity, as well as additional consolidation of the memory of the image in an upstream cortical module.\",\n",
              "  'authors': ['Yakovlev, Volodya',\n",
              "   'Amit, Daniel J',\n",
              "   'Romani, Sandro',\n",
              "   'Hochstein, Shaul'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Behavior, Animal',\n",
              "   'Computer Simulation',\n",
              "   'Discrimination (Psychology)',\n",
              "   'Haplorhini',\n",
              "   'Identification (Psychology)',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Photic Stimulation',\n",
              "   'ROC Curve',\n",
              "   'Recognition (Psychology)',\n",
              "   'Task Performance and Analysis'],\n",
              "  'pmid': '18171941',\n",
              "  'title': 'Universal memory mechanism for familiarity recognition and identification.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1437-1588',\n",
              "  'IF_2013': '1.006',\n",
              "  'IF_2014': '1.422',\n",
              "  'IF_2015': '1.499',\n",
              "  'IF_2016': '1.147',\n",
              "  'IF_2017': '0.901',\n",
              "  'ISSN': '1436-9990',\n",
              "  'ISSN_online': '1437-1588',\n",
              "  'ISSN_print': '1436-9990',\n",
              "  'IsoAbbr': 'Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz',\n",
              "  'JrId': 31961,\n",
              "  'MedAbbr': 'Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz',\n",
              "  'NLMid': '101181368',\n",
              "  'Titre': 'Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz',\n",
              "  'abstract': 'The term \"Service robotics\" describes semi- or fully autonomous technical systems able to perform services useful to the well-being of humans. Service robots have the potential to support and disburden both persons in need of care as well as nursing care staff. In addition, they can be used in prevention and rehabilitation in order to reduce or avoid the need for help. Products currently available to support people in domestic environments are mainly cleaning or remote-controlled communication robots. Examples of current research activities are the (further) development of mobile robots as advanced communication assistants or the development of (semi) autonomous manipulation aids and multifunctional household assistants. Transport robots are commonly used in many hospitals. In nursing care facilities, the first evaluations have already been made. So-called emotional robots are now sold as products and can be used for therapeutic, occupational, or entertainment activities. ',\n",
              "  'authors': ['Graf, B', 'Heyer, T', 'Klein, B', 'Wallhoff, F'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'Bundesgesundheitsblatt, Gesundheitsforschung, Gesundheitsschutz',\n",
              "  'keywords': ['Forecasting',\n",
              "   'Geriatric Nursing',\n",
              "   'Germany',\n",
              "   'Robotics',\n",
              "   'Technology Assessment, Biomedical',\n",
              "   'Therapy, Computer-Assisted'],\n",
              "  'pmid': '23884531',\n",
              "  'title': '[Service robots in elderly care. Possible application areas and current state of developments].',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1873-7560',\n",
              "  'IF_2013': '12.48',\n",
              "  'IF_2014': '13.938',\n",
              "  'IF_2015': '14.976',\n",
              "  'IF_2016': '16.265',\n",
              "  'IF_2017': '17.581',\n",
              "  'ISSN': '0302-2838',\n",
              "  'ISSN_online': '1873-7560',\n",
              "  'ISSN_print': '0302-2838',\n",
              "  'IsoAbbr': 'Eur. Urol.',\n",
              "  'JrId': 3630,\n",
              "  'MedAbbr': 'Eur Urol',\n",
              "  'NLMid': '7512719',\n",
              "  'Titre': 'Eur. Urol.',\n",
              "  'abstract': 'Intraoperative frozen-section analysis allows real-time histologic assessment of surgical margins (SMs) and identification of candidates for nerve-sparing (NS) procedures.|To examine the efficacy and oncologic safety of a systematic neurovascular structure-adjacent frozen-section examination (NeuroSAFE) during NS radical prostatectomy (RP).|From January 2002 to June 2011, 11 069 consecutive RPs were performed at the University Medical Center Hamburg-Eppendorf. Of these, 5392 (49%) were conducted with NeuroSAFE.|Our NeuroSAFE approach included the whole laterorectal circumference of the prostate to determine the SM status of the complete neurovascular tissue-corresponding prostatic surface.|The impact of NeuroSAFE on NS frequency, SM status, and biochemical recurrence (BCR) was analyzed by chi-square test, and by Kaplan-Meier analyses in propensity score-based matched cohorts.|Positive SMs (PSMs) were detected in 1368 (25%) NeuroSAFE RPs, leading to a secondary resection of the ipsilateral neurovascular tissue. Secondary wide resection resulted in conversion to a definitive negative SM (NSM) status in 1180 (86%) patients. In NeuroSAFE RPs, frequency of NS was significantly higher (all stages: 97% vs 81%; pT2: 99% vs 92%; pT3a: 94% vs 72%; pT3b: 88% vs 40%; p<0.0001) and PSM rates were significantly lower (all stages: 15% vs 22%; pT2: 7% vs 12%; pT3a: 21% vs 32%; p<0.0001) than in the matched non-NeuroSAFE RPs. In propensity score-based comparisons, NeuroSAFE had no negative impact on BCR (pT2, p=0.06; pT3a, p=0.17, pT3b, p=0.99), and BCR-free survival of patients with conversion to NSM did not differ significantly from patients with primarily NSM (pT2, p=0.16; pT3, p=0.26). The accuracy of our NeuroSAFE approach was 97% with a false-negative rate of 2.5%. The major limitations of this study are its retrospective nature and relatively short follow-up.|NeuroSAFE enables real-time histologic monitoring of the oncologic safety of a NS procedure. Systematic NeuroSAFE significantly increases NS frequencies and reduces PSMs. Patients with a NeuroSAFE-detected PSM could be converted to a prognostically more favorable NSM status by secondary wide resection.',\n",
              "  'authors': ['Schlomm, Thorsten',\n",
              "   'Tennstedt, Pierre',\n",
              "   'Huxhold, Caroline',\n",
              "   'Steuber, Thomas',\n",
              "   'Salomon, Georg',\n",
              "   'Michl, Uwe',\n",
              "   'Heinzer, Hans',\n",
              "   'Hansen, Jens',\n",
              "   'Budäus, Lars',\n",
              "   'Steurer, Stefan',\n",
              "   'Wittmer, Corinna',\n",
              "   'Minner, Sarah',\n",
              "   'Haese, Alexander',\n",
              "   'Sauter, Guido',\n",
              "   'Graefen, Markus',\n",
              "   'Huland, Hartwig'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'European urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Cryoultramicrotomy',\n",
              "   'Disease-Free Survival',\n",
              "   'Frozen Sections',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Grading',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Prostate',\n",
              "   'Prostate-Specific Antigen',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '22591631',\n",
              "  'title': 'Neurovascular structure-adjacent frozen-section examination (NeuroSAFE) increases nerve-sparing frequency and reduces positive surgical margins in open and robot-assisted laparoscopic radical prostatectomy: experience after 11,069 consecutive patients.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'The da Vinci robot-assisted laparoscopic radical prostatectomy is a relatively new approach that is revolutionizing the surgical treatment of localized prostate cancer. Since its introduction, several improvements have been made in the robot design model, as well as in the surgical technique for prostatectomy. One of the more recent advances in this technology has been the introduction of a four-arm robot model. This modified system allows the operating surgeon to use the fourth arm for key steps and maneuvers during the operation, thereby decreasing the reliance on advanced assistant laparoscopic skills. In this report, we describe our modifications for the extraperitoneal approach for laparoscopic removal of the prostate using the four-arm da Vinci surgical system.|During a 24-month period, 154 consecutive patients with clinically localized prostate cancer underwent extraperitoneal robot-assisted laparoscopic radical prostatectomy using the four-arm da Vinci robot system. All cases were videotaped and subsequently reviewed. Important factors regarding extraperitoneal access, patient positioning, port placement, and assistant role with or without the fourth arm were defined.|Our experience has revealed that the extraperitoneal approach allows for a more natural patient position during the operation and avoids intraperitoneal organ injury. The addition of the fourth arm to the da Vinci robot provides the operating surgeon with a great deal of independence, which facilitates all aspects of robot-assisted laparoscopic prostatectomy. It allows the operating surgeon to retract tissue during critical steps in this challenging operation and reduces the reliance on highly trained laparoscopic assistants.',\n",
              "  'authors': ['Esposito, Michael P',\n",
              "   'Ilbeigi, Pedram',\n",
              "   'Ahmed, Mutahar',\n",
              "   'Lanteri, Vincent'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Equipment Design',\n",
              "   'Humans',\n",
              "   'Laparoscopes',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Robotics'],\n",
              "  'pmid': '16140096',\n",
              "  'title': 'Use of fourth arm in da Vinci robot-assisted extraperitoneal laparoscopic prostatectomy: novel technique.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '2001-0370',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '4.148',\n",
              "  'IF_2015': '4.148',\n",
              "  'IF_2016': '4.148',\n",
              "  'IF_2017': '4.148',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2001-0370',\n",
              "  'IsoAbbr': 'Comput Struct Biotechnol J',\n",
              "  'JrId': 40350,\n",
              "  'MedAbbr': 'Comput Struct Biotechnol J',\n",
              "  'NLMid': '101585369',\n",
              "  'Titre': 'Comput Struct Biotechnol J',\n",
              "  'abstract': 'Unlabelled Image.',\n",
              "  'authors': ['Forghani, Reza',\n",
              "   'Savadjiev, Peter',\n",
              "   'Chatterjee, Avishek',\n",
              "   'Muthukrishnan, Nikesh',\n",
              "   'Reinhold, Caroline',\n",
              "   'Forghani, Behzad'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Computational and structural biotechnology journal',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31388413',\n",
              "  'title': 'Radiomics and Artificial Intelligence for Biomarker and Prediction Model Development in Oncology.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'Unraveling functional and ancestral relationships between proteins as well as structure-prediction procedures require powerful protein-alignment methods. A structure-alignment method is presented where the problem is mapped onto a cost function containing both fuzzy (Potts) assignment variables and atomic coordinates. The cost function is minimized by using an iterative scheme, where at each step mean field theory methods at finite \"temperatures\" are used for determining fuzzy assignment variables followed by exact translation and rotation of atomic coordinates weighted by their corresponding fuzzy assignment variables. The approach performs very well when compared with other methods, requires modest central processing unit consumption, and is robust with respect to choice of iteration parameters for a wide range of proteins.',\n",
              "  'authors': ['Blankenbecler, Richard',\n",
              "   'Ohlsson, Mattias',\n",
              "   'Peterson, Carsten',\n",
              "   'Ringner, Markus'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Fuzzy Logic',\n",
              "   'Models, Molecular',\n",
              "   'Molecular Structure',\n",
              "   'Protein Conformation',\n",
              "   'Proteins',\n",
              "   'Sequence Alignment'],\n",
              "  'pmid': '14526099',\n",
              "  'title': 'Matching protein structures with fuzzy alignments.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1768-3238',\n",
              "  'IF_2013': '1.681',\n",
              "  'IF_2014': '0.981',\n",
              "  'IF_2015': '1.064',\n",
              "  'IF_2016': '1.1',\n",
              "  'IF_2017': '1.313',\n",
              "  'ISSN': '1631-0691',\n",
              "  'ISSN_online': '1768-3238',\n",
              "  'ISSN_print': '1631-0691',\n",
              "  'IsoAbbr': 'C. R. Biol.',\n",
              "  'JrId': 27041,\n",
              "  'MedAbbr': 'C R Biol',\n",
              "  'NLMid': '101140040',\n",
              "  'Titre': 'C. R. Biol.',\n",
              "  'abstract': \"Living systems are capable to have appropriate responses to unpredictable environment. This kind of self-organization seems to operate as a self-programming machine, i.e. an organization able to modify itself. Until now the models of self-organization of living beings proposed are functions solutions of differential systems or transition functions of automata. These functions are fixed and these models are therefore unable to modify their organization. On the other hand, computer science propose a lot of models having the properties of adaptive systems of living beings, but all these models depend on the comparison between a goal and the results and ingenious choices of parameters by programmers, whereas there are no programmer's intention nor choice in the living systems. From two best known examples of adaptive systems of living beings, nervous system and immune system that have in common that the external signals modify the rewriting of their organization and therefore work as self-organizing machines, we devised machines with a finite set of inputs, based upon a recurrence, are able to rewrite their organization (Self-programming machines or m(sp)) whenever external conditions vary and have striking properties of adaptation. M(sp) have similar properties whatever the operation defining the recurrence maybe. These results bring us to make the following statement: adaptive properties of living systems can be explained by their ability to rewrite their organization whenever external conditions vary under the only assumption that the rewriting mechanism be a deterministic constant recurrence in a finite state set.\",\n",
              "  'authors': ['Moulin, Jean-Paul'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Comptes rendus biologies',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Automation',\n",
              "   'Computer Communication Networks',\n",
              "   'Computer Simulation',\n",
              "   'Computer Systems',\n",
              "   'Humans'],\n",
              "  'pmid': '16545754',\n",
              "  'title': 'Adaptive properties of living beings: proposal for a generic mechanism. (Self-programming machines III).',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'The objective of this study was to assess the surgical, oncologic, and short-term functional outcomes of patients with a history of transurethral resection of the prostate (TURP) who underwent robot-assisted laparoscopic prostatectomy (RALP).|The records of 2000 men who underwent RALP from February 2006 to April 2010 were retrospectively reviewed. A total of 80 men had undergone TURP before RALP. A match-paired analysis was performed using our database to identify 80 additional men without a history of TURP with equivalent clinicopathologic characteristics to serve as a control group (non-TURP group). The parameters compared included patient preoperative clinicopathologic characteristics, intraopeoperative characteristics, postoperative oncologic characteristics, minor and major postoperative complications, continence, and potency.|The mean time between TURP and RALP was 3.6 months (3-6 months). Regarding preoperative characteristics, a statistical difference was only observed regarding preoperative patient potency in the TURP vs non-TURP group. Regarding intraopeoperative characteristics, a statistical difference was observed regarding the need for bladder neck reconstruction and skin-to-skin operative time. Regarding postoperative pathologic characteristics, the positive surgical margin rate was not significant when the two groups were compared. The continence and potency rates in 12 months were similar (87.5%/91.25%) and (70.3%/86.5%) for both patient cohorts.|Although the procedure is technically more demanding, exhibits a prolonged operative time and time interval before continence and potency returns, it can be safely performed without compromising functional results as well as the radical nature of cancer surgery.',\n",
              "  'authors': ['Zugor, Vahudin',\n",
              "   'Labanaris, Apostolos P',\n",
              "   'Porres, Daniel',\n",
              "   'Witt, Jorn H'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Aged',\n",
              "   'Humans',\n",
              "   'Intraoperative Care',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Preoperative Care',\n",
              "   'Prostate',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Time Factors',\n",
              "   'Transurethral Resection of Prostate',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '21913854',\n",
              "  'title': 'Surgical, oncologic, and short-term functional outcomes in patients undergoing robot-assisted prostatectomy after previous transurethral resection of the prostate.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-7544',\n",
              "  'IF_2013': '3.327',\n",
              "  'IF_2014': '3.357',\n",
              "  'IF_2015': '3.231',\n",
              "  'IF_2016': '3.277',\n",
              "  'IF_2017': '3.382',\n",
              "  'ISSN': '0306-4522',\n",
              "  'ISSN_online': '1873-7544',\n",
              "  'ISSN_print': '0306-4522',\n",
              "  'IsoAbbr': 'Neuroscience',\n",
              "  'JrId': 6087,\n",
              "  'MedAbbr': 'Neuroscience',\n",
              "  'NLMid': '7605074',\n",
              "  'Titre': 'Neuroscience',\n",
              "  'abstract': 'Combined physiological and connectional studies show significant non-topographic extrinsic projections to frequency-specific domains in the cat auditory cortex. These frequency-mismatched loci in the thalamus, ipsilateral cortex, and commissural system complement the predicted topographic and tonotopic projections. Two tonotopic areas, the primary auditory cortex (AI) and the anterior auditory field (AAF), were electrophysiologically characterized by their frequency organization. Next, either cholera toxin beta subunit or cholera toxin beta subunit gold conjugate was injected into frequency-matched locations in each area to reveal the projection pattern from the thalamus and cortex. Most retrograde labeling was found at tonotopically appropriate locations within a 1 mm-wide strip in the thalamus and a 2-3 mm-wide expanse of cortex (approximately 85%). However, approximately 13-30% of the neurons originated from frequency-mismatched locations far from their predicted positions in thalamic nuclei and cortical areas, respectively. We propose that these heterotopic projections satisfy at least three criteria that may be necessary to support the magnitude and character of plastic changes in physiological studies. First, they are found in the thalamus, ipsilateral and commissural cortex; since this reorganization could arise from any of these routes and may involve each, such projections ought to occur in them. Second, they originate from nuclei and areas with or without tonotopy; it is likely that plasticity is not exclusively shaped by spectral influences and not limited to cochleotopic regions. Finally, the projections are appropriate in magnitude and sign to plausibly support such rearrangements; given the rapidity of some aspects of plastic changes, they should be mediated by substantial existing connections. Alternative roles for these heterotopic projections are also considered.',\n",
              "  'authors': ['Lee, C C', 'Schreiner, C E', 'Imaizumi, K', 'Winer, J A'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Auditory Cortex',\n",
              "   'Auditory Pathways',\n",
              "   'Brain Mapping',\n",
              "   'Cats',\n",
              "   'Cholera Toxin',\n",
              "   'Dose-Response Relationship, Radiation',\n",
              "   'Electric Stimulation',\n",
              "   'Female',\n",
              "   'Functional Laterality',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Thalamus'],\n",
              "  'pmid': '15464293',\n",
              "  'title': 'Tonotopic and heterotopic projection systems in physiologically defined auditory cortex.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1744-8328',\n",
              "  'IF_2013': '2.279',\n",
              "  'IF_2014': '2.249',\n",
              "  'IF_2015': '2.094',\n",
              "  'IF_2016': '2.212',\n",
              "  'IF_2017': '2.347',\n",
              "  'ISSN': '1473-7140',\n",
              "  'ISSN_online': '1744-8328',\n",
              "  'ISSN_print': '1473-7140',\n",
              "  'IsoAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'JrId': 29775,\n",
              "  'MedAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'NLMid': '101123358',\n",
              "  'Titre': 'Expert Rev Anticancer Ther',\n",
              "  'abstract': 'Since the 1990s, minimal access surgery has been utilized in urology. In the past 15 years, robotic surgery has evolved and become a natural part of minimal access surgery. The dissemination has been fast and the opportunity of prospective trials has been missed. Nevertheless, robotic surgery has obvious benefits for the surgeon and patient. Even though the scientific evidence is not strong, robotic surgery is here to stay. However, there are lessons to learn from the implementation of the da Vinci system with regards to patient safety and prospective evaluation of the new technology. The future of surgery will include technologies derived from robotic surgery. ',\n",
              "  'authors': ['Müller, Stig',\n",
              "   'Grønning, Leif Erik',\n",
              "   'Nilsen, Frode S',\n",
              "   'Mygland, Vegard',\n",
              "   'Patel, Hiten R H'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Expert review of anticancer therapy',\n",
              "  'keywords': ['Humans',\n",
              "   'Male',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '25266367',\n",
              "  'title': 'Robotic and minimal access surgery: technology and surgical outcomes of radical prostatectomy for prostate cancer.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1534-7362',\n",
              "  'IF_2013': '2.727',\n",
              "  'IF_2014': '2.393',\n",
              "  'IF_2015': '2.341',\n",
              "  'IF_2016': '2.671',\n",
              "  'IF_2017': '2.266',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1534-7362',\n",
              "  'IsoAbbr': 'J Vis',\n",
              "  'JrId': 30247,\n",
              "  'MedAbbr': 'J Vis',\n",
              "  'NLMid': '101147197',\n",
              "  'Titre': 'J Vis',\n",
              "  'abstract': 'Bottom-up and top-down as well as low-level and high-level factors influence where we fixate when viewing natural scenes. However, the importance of each of these factors and how they interact remains a matter of debate. Here, we disentangle these factors by analyzing their influence over time. For this purpose, we develop a saliency model that is based on the internal representation of a recent early spatial vision model to measure the low-level, bottom-up factor. To measure the influence of high-level, bottom-up features, we use a recent deep neural network-based saliency model. To account for top-down influences, we evaluate the models on two large data sets with different tasks: first, a memorization task and, second, a search task. Our results lend support to a separation of visual scene exploration into three phases: the first saccade, an initial guided exploration characterized by a gradual broadening of the fixation density, and a steady state that is reached after roughly 10 fixations. Saccade-target selection during the initial exploration and in the steady state is related to similar areas of interest, which are better predicted when including high-level features. In the search data set, fixation locations are determined predominantly by top-down processes. In contrast, the first fixation follows a different fixation density and contains a strong central fixation bias. Nonetheless, first fixations are guided strongly by image properties, and as early as 200 ms after image onset, fixations are better predicted by high-level information. We conclude that any low-level, bottom-up factors are mainly limited to the generation of the first saccade. All saccades are better explained when high-level features are considered, and later, this high-level, bottom-up control can be overruled by top-down influences.',\n",
              "  'authors': ['Schütt, Heiko H',\n",
              "   'Rothkegel, Lars O M',\n",
              "   'Trukenbrod, Hans A',\n",
              "   'Engbert, Ralf',\n",
              "   'Wichmann, Felix A'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Journal of vision',\n",
              "  'keywords': ['Eye Movement Measurements',\n",
              "   'Eye Movements',\n",
              "   'Female',\n",
              "   'Fixation, Ocular',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Memory',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Photic Stimulation',\n",
              "   'Saccades',\n",
              "   'Vision, Ocular',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30821809',\n",
              "  'title': 'Disentangling bottom-up versus top-down and low-level versus high-level influences on eye movements over time.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'High-grade gliomas are the most aggressive malignant brain tumors. Accurate pre-operative prognosis for this cohort can lead to better treatment planning. Conventional survival prediction based on clinical information is subjective and could be inaccurate. Recent radiomics studies have shown better prognosis by using carefully-engineered image features from magnetic resonance images (MRI). However, feature engineering is usually time consuming, laborious and subjective. Most importantly, the engineered features cannot effectively encode other predictive but implicit information provided by multi-modal neuroimages. We propose a two-stage learning-based method to predict the overall survival (OS) time of high-grade gliomas patient. At the first stage, we adopt deep learning, a recently dominant technique of artificial intelligence, to automatically extract implicit and high-level features from multi-modal, multi-channel preoperative MRI such that the features are competent of predicting survival time. Specifically, we utilize not only contrast-enhanced T1 MRI, but also diffusion tensor imaging (DTI) and resting-state functional MRI (rs-fMRI), for computing multiple metric maps (including various diffusivity metric maps derived from DTI, and also the frequency-specific brain fluctuation amplitude maps and local functional connectivity anisotropy-related metric maps derived from rs-fMRI) from 68 high-grade glioma patients with different survival time. We propose a multi-channel architecture of 3D convolutional neural networks (CNNs) for deep learning upon those metric maps, from which high-level predictive features are extracted for each individual patch of these maps. At the second stage, those deeply learned features along with the pivotal limited demographic and tumor-related features (such as age, tumor size and histological type) are fed into a support vector machine (SVM) to generate the final prediction result (i.e., long or short overall survival time). The experimental results demonstrate that this multi-model, multi-channel deep survival prediction framework achieves an accuracy of 90.66%, outperforming all the competing methods. This study indicates highly demanded effectiveness on prognosis of deep learning technique in neuro-oncological applications for better individualized treatment planning towards precision medicine.',\n",
              "  'authors': ['Nie, Dong',\n",
              "   'Lu, Junfeng',\n",
              "   'Zhang, Han',\n",
              "   'Adeli, Ehsan',\n",
              "   'Wang, Jun',\n",
              "   'Yu, Zhengda',\n",
              "   'Liu, LuYan',\n",
              "   'Wang, Qian',\n",
              "   'Wu, Jinsong',\n",
              "   'Shen, Dinggang'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30705340',\n",
              "  'title': 'Multi-Channel 3D Deep Feature Learning for Survival Time Prediction of Brain Tumor Patients Using Multi-Modal Neuroimages.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1553-4669',\n",
              "  'IF_2013': '1.575',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.39',\n",
              "  'IF_2016': '3.061',\n",
              "  'IF_2017': '3.061',\n",
              "  'ISSN': '1553-4650',\n",
              "  'ISSN_online': '1553-4669',\n",
              "  'ISSN_print': '1553-4650',\n",
              "  'IsoAbbr': 'J Minim Invasive Gynecol',\n",
              "  'JrId': 32424,\n",
              "  'MedAbbr': 'J Minim Invasive Gynecol',\n",
              "  'NLMid': '101235322',\n",
              "  'Titre': 'J Minim Invasive Gynecol',\n",
              "  'abstract': 'The objectives of this article were to review the published scientific literature about robotics and its application to gynecologic oncology to date and to summarize findings of this advanced computerenhanced laparoscopic technique. Relevant sources were identified by a search of PUBMED from January 1950 to January 2009 using the key words Robot or Robotics and Cervical cancer, Endometrial cancer, Gynecologic oncology, and Ovarian cancer. Appropriate case reports, case series, retrospective studies, prospective trials, and review articles were selected. A total of 38 articles were identified on the subject, and 27 were included in the study. The data for gynecologic cancer show comparable results between robotic and laparoscopic surgery for estimated blood loss, operative time, length of hospital stay, and complications. Overall, there were more wound complications with the laparotomy approach compared with laparoscopy and robotic-assisted laparoscopy. There were more lymphocysts, lymphoceles, and lymphedema in the robotic-assisted laparoscopic group compared with the laparoscopy and laparotomy groups in patients with cervical cancer. Infectious and lung-related morbidity, postoperative ileus, and bleeding or clot formation were more commonly reported in the laparotomy group compared with the other 2 cohorts in patients with endometrial cancer. Computer-enhanced technology may enable more surgeons to convert laparotomies to laparoscopic surgery with its associated benefits. It seems that in the hands of experienced laparoscopic surgeons, final outcomes are the same with or without use of the robot. There is good evidence that robotic surgery facilitates laparoscopic surgery, with equivalent if not better operative time and comparable surgical outcomes, shorter hospital stay, and fewer major complications than with surgeries using the laparotomy approach.',\n",
              "  'authors': ['Cho, Jennifer E', 'Nezhat, Farr R'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Journal of minimally invasive gynecology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Female',\n",
              "   'Genital Neoplasms, Female',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Laparotomy',\n",
              "   'Robotics'],\n",
              "  'pmid': '19896593',\n",
              "  'title': 'Robotics and gynecologic oncology: review of the literature.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1613-9860',\n",
              "  'IF_2013': '2.189',\n",
              "  'IF_2014': '2.189',\n",
              "  'IF_2015': '2.189',\n",
              "  'IF_2016': '2.989',\n",
              "  'IF_2017': '2.521',\n",
              "  'ISSN': '1869-6716',\n",
              "  'ISSN_online': '1613-9860',\n",
              "  'ISSN_print': '1869-6716',\n",
              "  'IsoAbbr': 'Transl Behav Med',\n",
              "  'JrId': 39025,\n",
              "  'MedAbbr': 'Transl Behav Med',\n",
              "  'NLMid': '101554668',\n",
              "  'Titre': 'Transl Behav Med',\n",
              "  'abstract': \"Behavioral intervention technologies (BITs) are unique ways to incorporate the benefits of technology and psychology to address differing health needs through various media, including Internet interventions, mobile apps, and video games. BITs present several possible benefits, including increased dissemination and accessibility, cost-effectiveness, increased engagement, and decreased stigma, especially among youth. A behavioral coaching chatbot, Tess, addresses different facets of behavioral health, such as depression and anxiety. Available 24/7, Tess delivers customized integrative support, psychoeducation, and interventions through brief conversations via existing communication channels (i.e., SMS text messaging and Facebook Messenger). This study assessed the feasibility of integrating Tess in behavioral counseling of adolescent patients (n = 23; Mage = 15.20 years; Rangeage = 9.78-18.54 years; 57% female) coping with weight management and prediabetes symptoms. Tess engaged patients via a preferred method of communication (SMS text messaging) in individualized conversations to promote treatment adherence, behavior change, and overall wellness. Adolescent patients reported experiencing positive progress toward their goals 81% of the time. The 4,123 messages exchanged and patients' reported usefulness ratings (96% of the time) illustrate that adolescents engaged with and viewed this chatbot as helpful. These results highlight the feasibility and benefit of support through artificial intelligence, specifically in a pediatric setting, which could be scaled to serve larger groups of patients. As a partner to clinicians, Tess can continue the therapeutic interaction outside office hours while maintaining patient satisfaction. Due to Tess's capacity for continuous learning, future iterations may have additional features to increase the user experience.\",\n",
              "  'authors': ['Stephens, Taylor N',\n",
              "   'Joerin, Angela',\n",
              "   'Rauws, Michiel',\n",
              "   'Werk, Lloyd N'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'Translational behavioral medicine',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31094445',\n",
              "  'title': 'Feasibility of pediatric obesity and prediabetes treatment support through Tess, the AI behavioral coaching chatbot.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1497-0015',\n",
              "  'IF_2013': '2.406',\n",
              "  'IF_2014': '2.551',\n",
              "  'IF_2015': '2.952',\n",
              "  'IF_2016': '3.42',\n",
              "  'IF_2017': '3.612',\n",
              "  'ISSN': '0706-7437',\n",
              "  'ISSN_online': '1497-0015',\n",
              "  'ISSN_print': '0706-7437',\n",
              "  'IsoAbbr': 'Can J Psychiatry',\n",
              "  'JrId': 2768,\n",
              "  'MedAbbr': 'Can J Psychiatry',\n",
              "  'NLMid': '7904187',\n",
              "  'Titre': 'Can J Psychiatry',\n",
              "  'abstract': 'This paper considers the relevance of hippocampal dysfunction to symptom production in schizophrenia with the help of an integrative computational model of cortical processing. First, evidence for involvement of the hippocampus was systematically reviewed. Then, a neural network model of hippocampal-cortical interaction was proposed. The effects of a dysfunction in hippocampal-cortical interaction were explored through simulation experiments. Neural network simulation demonstrated that when the hippocampal function of classification and reduction of overlap in input is sub-optimal, increasing correlation among input patterns compromised storage capacity of the cortical network. Hippocampal dysfunction predisposes to the emergence of spurious retrieval states in cortical information processing. This provides a model for psychotic symptom formation in schizophrenia.',\n",
              "  'authors': ['Chen, E Y'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Canadian journal of psychiatry. Revue canadienne de psychiatrie',\n",
              "  'keywords': ['Cerebral Cortex',\n",
              "   'Hippocampus',\n",
              "   'Humans',\n",
              "   'Mental Processes',\n",
              "   'Mental Recall',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Schizophrenia',\n",
              "   'Schizophrenic Psychology'],\n",
              "  'pmid': '7874671',\n",
              "  'title': 'A neural network model of cortical information processing in schizophrenia. II--Role of hippocampal-cortical interaction: a review and a model.',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1971-3274',\n",
              "  'IF_2013': '1.855',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0393-5264',\n",
              "  'ISSN_online': '1971-3274',\n",
              "  'ISSN_print': '0393-5264',\n",
              "  'IsoAbbr': 'Funct. Neurol.',\n",
              "  'JrId': 3919,\n",
              "  'MedAbbr': 'Funct Neurol',\n",
              "  'NLMid': '8707746',\n",
              "  'Titre': 'Funct. Neurol.',\n",
              "  'abstract': \"Restoration of motor function is a priority of post-stroke rehabilitation, the aim being to facilitate the patient's reintegration into society. Innovative technologies for neurological rehabilitation must be easy to use and offer patients real benefits, and the treatments they provide must be efficacious and efficient. All these aspects must be carefully evaluated in their development. To achieve restoration of motor function after stroke, task-specific repetitive robot-assisted training of the upper and the lower extremity is currently the most promising approach. The results of clinical trials of robotic devices for upper limb (MIT-Manus, MIME, NeReBot, BiManuTrack, ARMin, ARMOR) and lower limb (LokoHelp, GangTrainer GT1, Haptic Walker, G-EO-Systems, Lokomat) training are here presented with the aim of highlighting the possible gains in motor function due to robotic therapy. Patients who receive robot-assisted training in combination with physiotherapy after stroke are more likely to achieve better motor function than patients trained without these devices, or only with these devices.\",\n",
              "  'authors': ['Waldner, Andreas', 'Tomelleri, Cristopher', 'Hesse, Stefan'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Functional neurology',\n",
              "  'keywords': ['Clinical Trials as Topic',\n",
              "   'Extremities',\n",
              "   'Humans',\n",
              "   'Recovery of Function',\n",
              "   'Rehabilitation',\n",
              "   'Robotics',\n",
              "   'Stroke Rehabilitation',\n",
              "   'Transfer (Psychology)'],\n",
              "  'pmid': '20412721',\n",
              "  'title': 'Transfer of scientific concepts to clinical practice: recent robot-assisted training studies.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1534-7362',\n",
              "  'IF_2013': '2.727',\n",
              "  'IF_2014': '2.393',\n",
              "  'IF_2015': '2.341',\n",
              "  'IF_2016': '2.671',\n",
              "  'IF_2017': '2.266',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1534-7362',\n",
              "  'IsoAbbr': 'J Vis',\n",
              "  'JrId': 30247,\n",
              "  'MedAbbr': 'J Vis',\n",
              "  'NLMid': '101147197',\n",
              "  'Titre': 'J Vis',\n",
              "  'abstract': 'Existing neural explanations of spontaneous percept switching under steady viewing of an ambiguous stimulus do not fit the fact that stimulus interruptions cause the same percept to reappear across many ON/OFF cycles. We present a simple neural model that explains the observed behavior and predicts several more complicated percept sequences, without invoking any \"high-level\" decision making or memory. Percept choice at stimulus onset, which differs fundamentally from standard percept switching, depends crucially on a hitherto neglected interaction between local \"shunting\" adaptation and a near-threshold neural baseline. Stimulus ON/OFF timing then controls the generation of repeating, alternating, or more complex choice sequences. Our model also explains \"priming\" versus \"habituation\" effects on percept choice, reinterprets recent neurophysiological data, and predicts the emergence of hysteresis at the level of percept sequences, with occasional noise-induced sequence \"hopping.\"',\n",
              "  'authors': ['Noest, A J', 'van Ee, R', 'Nijs, M M', 'van Wezel, R J A'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Journal of vision',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Habituation, Psychophysiologic',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Psychophysics',\n",
              "   'Visual Pathways',\n",
              "   'Visual Perception'],\n",
              "  'pmid': '17685817',\n",
              "  'title': 'Percept-choice sequences driven by interrupted ambiguous stimuli: a low-level neural model.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The geometrical confinement of small cell colonies gives differential cues to cells sitting at the periphery versus the core. To utilize this effect, for example to create spatially graded differentiation patterns of human mesenchymal stem cells (hMSCs) in vitro or to investigate underpinning mechanisms, the confinement needs to be robust for extended time periods. To create highly repeatable micro-fabricated structures for cellular patterning and high-throughput data mining, we employed here a simple casting method to fabricate more than 800 adhesive patches confined by agarose micro-walls. In addition, a machine learning based image processing software was developed (open code) to detect the differentiation patterns of the population of hMSCs automatically. Utilizing the agarose walls, the circular patterns of hMSCs were successfully maintained throughout 15 days of cell culture. After staining lipid droplets and alkaline phosphatase as the markers of adipogenic and osteogenic differentiation, respectively, the mega-pixels of RGB color images of hMSCs were processed by the software on a laptop PC within several minutes. The image analysis successfully showed that hMSCs sitting on the more central versus peripheral sections of the adhesive circles showed adipogenic versus osteogenic differentiation as reported previously, indicating the compatibility of patterned agarose walls to conventional microcontact printing. In addition, we found a considerable fraction of undifferentiated cells which are preferentially located at the peripheral part of the adhesive circles, even in differentiation-inducing culture media. In this study, we thus successfully demonstrated a simple framework for analyzing the patterned differentiation of hMSCs in confined microenvironments, which has a range of applications in biology, including stem cell biology.',\n",
              "  'authors': ['Tanaka, Nobuyuki',\n",
              "   'Yamashita, Tadahiro',\n",
              "   'Sato, Asako',\n",
              "   'Vogel, Viola',\n",
              "   'Tanaka, Yo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adipogenesis',\n",
              "   'Alkaline Phosphatase',\n",
              "   'Cell Culture Techniques',\n",
              "   'Cell Differentiation',\n",
              "   'Culture Media',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Mesenchymal Stem Cells',\n",
              "   'Osteogenesis',\n",
              "   'Sepharose',\n",
              "   'Software'],\n",
              "  'pmid': '28380036',\n",
              "  'title': 'Simple agarose micro-confinement array and machine-learning-based classification for analyzing the patterned differentiation of mesenchymal stem cells.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'Contouring a normal anatomical structure during radiation treatment planning requires significant time and effort. The authors present a fast and accurate semiautomatic contour delineation method to reduce the time and effort required of expert users.|Following an initial segmentation on one CT slice, the user marks the target organ and nontarget pixels with a few simple brush strokes. The algorithm calculates statistics from this information that, in turn, determines the parameters of an energy function containing both boundary and regional components. The method uses a conditional random field graphical model to define the energy function to be minimized for obtaining an estimated optimal segmentation, and a graph partition algorithm to efficiently solve the energy function minimization. Organ boundary statistics are estimated from the segmentation and propagated to subsequent images; regional statistics are estimated from the simple brush strokes that are either propagated or redrawn as needed on subsequent images. This greatly reduces the user input needed and speeds up segmentations. The proposed method can be further accelerated with graph-based interpolation of alternating slices in place of user-guided segmentation. CT images from phantom and patients were used to evaluate this method. The authors determined the sensitivity and specificity of organ segmentations using physician-drawn contours as ground truth, as well as the predicted-to-ground truth surface distances. Finally, three physicians evaluated the contours for subjective acceptability. Interobserver and intraobserver analysis was also performed and Bland-Altman plots were used to evaluate agreement.|Liver and kidney segmentations in patient volumetric CT images show that boundary samples provided on a single CT slice can be reused through the entire 3D stack of images to obtain accurate segmentation. In liver, our method has better sensitivity and specificity (0.925 and 0.995) than region growing (0.897 and 0.995) and level set methods (0.912 and 0.985) as well as shorter mean predicted-to-ground truth distance (2.13 mm) compared to regional growing (4.58 mm) and level set methods (8.55 mm and 4.74 mm). Similar results are observed in kidney segmentation. Physician evaluation of ten liver cases showed that 83% of contours did not need any modification, while 6% of contours needed modifications as assessed by two or more evaluators. In interobserver and intraobserver analysis, Bland-Altman plots showed our method to have better repeatability than the manual method while the delineation time was 15% faster on average.|Our method achieves high accuracy in liver and kidney segmentation and considerably reduces the time and labor required for contour delineation. Since it extracts purely statistical information from the samples interactively specified by expert users, the method avoids heuristic assumptions commonly used by other methods. In addition, the method can be expanded to 3D directly without modification because the underlying graphical framework and graph partition optimization method fit naturally with the image grid structure.',\n",
              "  'authors': ['Hu, Yu-Chi',\n",
              "   'Grossberg, Michael D',\n",
              "   'Wu, Abraham',\n",
              "   'Riaz, Nadeem',\n",
              "   'Perez, Carmen',\n",
              "   'Mageras, Gig S'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Humans',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Phantoms, Imaging',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '22830786',\n",
              "  'title': 'Interactive semiautomatic contour delineation using statistical conditional random fields framework.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'In this review paper, we first give a short overview of radiological image guidance in photon radiotherapy, placing emphasis on the fact that linac based radiotherapy has outpaced particle therapy in the adoption of volumetric image guidance. While cone beam computed tomography (CBCT) has been an established technique in linac treatment rooms for almost two decades, the widespread adoption of volumetric image guidance in particle therapy, whether by means of CBCT or in-room CT imaging, is recent. This lag may be attributable to the bespoke nature and lower number of particle therapy installations, as well as the differences in geometry between those installations and linac treatment rooms. In addition, for particle therapy the so called shift invariance of the dose distribution rarely applies. An overview of the different volumetric image guidance solutions found at modern particle therapy facilities is provided, covering gantry, nozzle, C-arm, and couch-mounted CBCT as well different in-room CT configurations. A summary of the use of in-room volumetric imaging data beyond anatomy-based positioning is also presented as well as the necessary corrections to CBCT images for accurate water equivalent thickness calculation. Finally, the use of non-ionizing imaging modalities is discussed.',\n",
              "  'authors': ['Landry, Guillaume', 'Hua, Chia-Ho'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Cone-Beam Computed Tomography',\n",
              "   'Humans',\n",
              "   'Radiotherapy, Image-Guided',\n",
              "   'Robotics'],\n",
              "  'pmid': '30421805',\n",
              "  'title': 'Current state and future applications of radiological image guidance for particle therapy.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['NI'],\n",
              "  'ESSN': '1664-3224',\n",
              "  'IF_2013': '5.695',\n",
              "  'IF_2014': '5.695',\n",
              "  'IF_2015': '5.695',\n",
              "  'IF_2016': '6.429',\n",
              "  'IF_2017': '5.511',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1664-3224',\n",
              "  'IsoAbbr': 'Front Immunol',\n",
              "  'JrId': 39309,\n",
              "  'MedAbbr': 'Front Immunol',\n",
              "  'NLMid': '101560960',\n",
              "  'Titre': 'Front Immunol',\n",
              "  'abstract': \"The adaptation of high-throughput sequencing to the B cell receptor and T cell receptor has made it possible to characterize the adaptive immune receptor repertoire (AIRR) at unprecedented depth. These AIRR sequencing (AIRR-seq) studies offer tremendous potential to increase the understanding of adaptive immune responses in vaccinology, infectious disease, autoimmunity, and cancer. The increasingly wide application of AIRR-seq is leading to a critical mass of studies being deposited in the public domain, offering the possibility of novel scientific insights through secondary analyses and meta-analyses. However, effective sharing of these large-scale data remains a challenge. The AIRR community has proposed minimal information about adaptive immune receptor repertoire (MiAIRR), a standard for reporting AIRR-seq studies. The MiAIRR standard has been operationalized using the National Center for Biotechnology Information (NCBI) repositories. Submissions of AIRR-seq data to the NCBI repositories typically use a combination of web-based and flat-file templates and include only a minimal amount of terminology validation. As a result, AIRR-seq studies at the NCBI are often described using inconsistent terminologies, limiting scientists' ability to access, find, interoperate, and reuse the data sets. In order to improve metadata quality and ease submission of AIRR-seq studies to the NCBI, we have leveraged the software framework developed by the Center for Expanded Data Annotation and Retrieval (CEDAR), which develops technologies involving the use of data standards and ontologies to improve metadata quality. The resulting CEDAR-AIRR (CAIRR) pipeline enables data submitters to: (i) create web-based templates whose entries are controlled by ontology terms, (ii) generate and validate metadata, and (iii) submit the ontology-linked metadata and sequence files (FASTQ) to the NCBI BioProject, BioSample, and Sequence Read Archive databases. Overall, CAIRR provides a web-based metadata submission interface that supports compliance with the MiAIRR standard. This pipeline is available at http://cairr.miairr.org, and will facilitate the NCBI submission process and improve the metadata quality of AIRR-seq studies.\",\n",
              "  'authors': ['Bukhari, Syed Ahmad Chan',\n",
              "   \"O'Connor, Martin J\",\n",
              "   'Martínez-Romero, Marcos',\n",
              "   'Egyedi, Attila L',\n",
              "   'Willrett, Debra',\n",
              "   'Graybeal, John',\n",
              "   'Musen, Mark A',\n",
              "   'Rubelt, Florian',\n",
              "   'Cheung, Kei-Hoi',\n",
              "   'Kleinstein, Steven H'],\n",
              "  'categories': ['IMMUNOLOGY'],\n",
              "  'journal': 'Frontiers in immunology',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Data Mining',\n",
              "   'Databases, Nucleic Acid',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Metadata',\n",
              "   'Receptors, Antigen, B-Cell',\n",
              "   'Receptors, Antigen, T-Cell',\n",
              "   'Reproducibility of Results',\n",
              "   'Software',\n",
              "   'User-Computer Interface',\n",
              "   'Workflow'],\n",
              "  'pmid': '30166985',\n",
              "  'title': 'The CAIRR Pipeline for Submitting Standards-Compliant B and T Cell Receptor Repertoire Sequencing Studies to the National Center for Biotechnology Information Repositories.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1759-5053',\n",
              "  'IF_2013': '10.807',\n",
              "  'IF_2014': '12.61',\n",
              "  'IF_2015': '14.435',\n",
              "  'IF_2016': '13.678',\n",
              "  'IF_2017': '16.99',\n",
              "  'ISSN': '1759-5045',\n",
              "  'ISSN_online': '1759-5053',\n",
              "  'ISSN_print': '1759-5045',\n",
              "  'IsoAbbr': 'Nat Rev Gastroenterol Hepatol',\n",
              "  'JrId': 35771,\n",
              "  'MedAbbr': 'Nat Rev Gastroenterol Hepatol',\n",
              "  'NLMid': '101500079',\n",
              "  'Titre': 'Nat Rev Gastroenterol Hepatol',\n",
              "  'abstract': 'Minimally invasive or minimal access surgery (MAS) for colon and rectal cancer was introduced in the early 1990s. Although laparoscopic colon surgery is now practiced worldwide, technical barriers, including a steep learning curve, preclude the widespread adoption of MAS techniques for rectal cancer. In addition, although randomized controlled trials have demonstrated that MAS techniques for colon cancer are oncologically equivalent to open surgery, similar confirmatory studies for rectal cancer have yet to be reported. In this Review, current evidence in support of laparoscopic and robotic total mesorectal excision for rectal cancer resection is presented. Other MAS approaches, such as transanal endoscopic microsurgery and natural orifice transluminal endoscopic surgery, are also discussed. ',\n",
              "  'authors': ['Hui, Vanessa W', 'Guillem, José G'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'Nature reviews. Gastroenterology & hepatology',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Natural Orifice Endoscopic Surgery',\n",
              "   'Rectal Neoplasms',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24192606',\n",
              "  'title': 'Minimal access surgery for rectal cancer: an update.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['SA'],\n",
              "  'ESSN': '1881-7742',\n",
              "  'IF_2013': '0.868',\n",
              "  'IF_2014': '0.827',\n",
              "  'IF_2015': '0.89',\n",
              "  'IF_2016': '0.674',\n",
              "  'IF_2017': '0.952',\n",
              "  'ISSN': '0301-4800',\n",
              "  'ISSN_online': '1881-7742',\n",
              "  'ISSN_print': '0301-4800',\n",
              "  'IsoAbbr': 'J. Nutr. Sci. Vitaminol.',\n",
              "  'JrId': 5059,\n",
              "  'MedAbbr': 'J Nutr Sci Vitaminol (Tokyo)',\n",
              "  'NLMid': '0402640',\n",
              "  'Titre': 'J. Nutr. Sci. Vitaminol.',\n",
              "  'abstract': 'In Japan, one of the most common causes of death in elderly people is aspiration pneumonia. Maintenance of oral hygiene and feeding functions are important elements, especially in patients with dysphagia caused by stroke, neurological diseases, and after operations on the head and neck cancer, as well as in the elderly to prevent aspiration pneumonia. It should also be noted that not only oral health care and physical therapy related to feeding functions but also dental treatment is included in the clinical management during interventions whenever needed. On the other hand, for the patients and/or elderly in need of assistance in maintaining a safe diet, it is recommended that a specialized team comprising physicians, dentists, and speech therapists in functional rehabilitation observes meal conditions of the subjects and evaluates such factors as meal contents, posture during meals, usage of dishes and cutlery, meal times, status of consciousness, perception, and motivation. First, I will present the clinical interventions to those patients, which include oral health care, dental treatment, physical therapy and meal assistance, as well as team approaches in those circumstances. In addition, clinical and basic research results will be introduced, which are expected to foster the understanding of physiology in chewing and swallowing. These results are also expected to develop the clinical technology to maintain or recover the feeding functions. ',\n",
              "  'authors': ['Inoue, Makoto'],\n",
              "  'categories': ['NUTRITION & DIETETICS'],\n",
              "  'journal': 'Journal of nutritional science and vitaminology',\n",
              "  'keywords': ['Aged',\n",
              "   'Deglutition',\n",
              "   'Deglutition Disorders',\n",
              "   'Eating',\n",
              "   'Humans',\n",
              "   'Japan',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity'],\n",
              "  'pmid': '26598894',\n",
              "  'title': 'Dysphagia Rehabilitation in Japan.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'With the development of robotic assistance, laparoscopic surgery has evolved over the past few years. Smaller instruments, better visibility, and growing surgical experience have allowed more complex reconstructive procedures to be performed laparoscopically. Herein, we describe our technique and experience in robot-assisted laparoscopic ureteroureterostomy (RALU) for the correction of ureteral obstruction in children.|RALU was performed in three children through a transperitoneal approach utilizing three ports. We describe our preference for patient positioning, robotic set-up, port placement, the use of a \"hitch stitch,\" and the option of placing a fourth port.|RALU is a reliable and efficient technique for the correction of ureteral obstruction in children and can be applied to any pathology that requires ureteral reconstruction, regardless if it is a duplicated or a single system.',\n",
              "  'authors': ['Passerotti, Carlo C',\n",
              "   'Diamond, David A',\n",
              "   'Borer, Joseph G',\n",
              "   'Eisner, Brian H',\n",
              "   'Barrisford, Glen',\n",
              "   'Nguyen, Hiep T'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'CD-ROM',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Robotics',\n",
              "   'Ureteral Obstruction',\n",
              "   'Ureterostomy'],\n",
              "  'pmid': '18307382',\n",
              "  'title': 'Robot-assisted laparoscopic ureteroureterostomy: description of technique.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'Minimally invasive approaches for large, symptomatic benign prostatic hyperplasia are replacing the gold standard open surgical approach, duplicating its results with lower morbidity. We describe our initial experience with robotic simple prostatectomy.|Since January 2007, robotic simple prostatectomy was performed via a transperitoneal approach in 7 patients with symptomatic significant prostatomegaly on transrectal ultrasound (mean 77.66 gm). Demographic, perioperative and outcome data were recorded and all procedures were performed by the same surgeon.|Average patient age was 63.2 years (range 56 to 72) and estimated blood loss was 298 ml (range 60 to 800). Average operative time was 205 minutes (range 120 to 300). Average hospital stay was 1.4 days (range 1 to 2), average Foley catheter duration was 7 days (range 6 to 9) and drains were removed after an average of 3.75 days (range 3 to 4). Mean specimen weight on pathological examination was 50.48 gm (range 40 to 64.5). Transfusion was necessary in 1 patient. No complications were documented. Considerable improvement from baseline was noted in International Prostate Symptom Score (preoperative vs postoperative 22 vs 7.25) and maximum urine flow (preoperative vs postoperative 17.75 vs 55.5 ml per minute). Four patients were in acute urinary retention preoperatively.|Robotic simple prostatectomy is a feasible, reproducible procedure. Further publications are expected with larger series and larger prostatic adenomas.',\n",
              "  'authors': ['Sotelo, Rene',\n",
              "   'Clavijo, Rafael',\n",
              "   'Carmona, Oswaldo',\n",
              "   'Garcia, Alejandro',\n",
              "   'Banda, Eduardo',\n",
              "   'Miranda, Marcelo',\n",
              "   'Fagin, Randy'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Cohort Studies',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Hyperplasia',\n",
              "   'Quality of Life',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '18076926',\n",
              "  'title': 'Robotic simple prostatectomy.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1573-9686',\n",
              "  'IF_2013': '3.231',\n",
              "  'IF_2014': '3.195',\n",
              "  'IF_2015': '2.887',\n",
              "  'IF_2016': '3.221',\n",
              "  'IF_2017': '3.405',\n",
              "  'ISSN': '0090-6964',\n",
              "  'ISSN_online': '1573-9686',\n",
              "  'ISSN_print': '0090-6964',\n",
              "  'IsoAbbr': 'Ann Biomed Eng',\n",
              "  'JrId': 561,\n",
              "  'MedAbbr': 'Ann Biomed Eng',\n",
              "  'NLMid': '0361512',\n",
              "  'Titre': 'Ann Biomed Eng',\n",
              "  'abstract': 'Diffusion tensor imaging (DTI) was performed on regions rostral to the injury site in four human subjects with chronic spinal cord injury (SCI) and equivalent regions in four neurologically intact subjects. Apparent diffusion coefficients were measured and compared between subjects. A fuzzy logic tissue classification algorithm was used to segment gray and white matter regions for morphometric analysis, including comparisons of cross-sectional areas of gray and white matter along with frontal and sagittal diameters. Results indicated a general decrease in both longitudinal and transverse diffusivity in the upper cervical segments of subjects with chronic SCI. Further, a decrease in the cross-sectional area of the entire spinal cord was observed in subjects with SCI, consistent with severe atrophy of the spinal cord. These observations have implications in tracking the progression of SCI from the acute to the chronic stages. We conclude that DTI with fuzzy logic tissue classification has potential for monitoring morphological changes in the spinal cord in people with SCI.',\n",
              "  'authors': ['Ellingson, Benjamin M', 'Ulmer, John L', 'Schmit, Brian D'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Annals of biomedical engineering',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Anatomy, Cross-Sectional',\n",
              "   'Diffusion Magnetic Resonance Imaging',\n",
              "   'Female',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nerve Fibers, Myelinated',\n",
              "   'Neural Pathways',\n",
              "   'Neurons',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Spinal Cord',\n",
              "   'Spinal Cord Injuries'],\n",
              "  'pmid': '18066663',\n",
              "  'title': 'Morphology and morphometry of human chronic spinal cord injury using diffusion tensor imaging and fuzzy logic.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1976-2437',\n",
              "  'IF_2013': '1.263',\n",
              "  'IF_2014': '1.287',\n",
              "  'IF_2015': '1.154',\n",
              "  'IF_2016': '1.537',\n",
              "  'IF_2017': '1.564',\n",
              "  'ISSN': '0513-5796',\n",
              "  'ISSN_online': '1976-2437',\n",
              "  'ISSN_print': '0513-5796',\n",
              "  'IsoAbbr': 'Yonsei Med. J.',\n",
              "  'JrId': 8109,\n",
              "  'MedAbbr': 'Yonsei Med J',\n",
              "  'NLMid': '0414003',\n",
              "  'Titre': 'Yonsei Med. J.',\n",
              "  'abstract': 'Robotic colorectal surgery has gradually been performed more with the help of the technological advantages of the da Vinci system. Advanced technological advantages of the da Vinci system compared with standard laparoscopic colorectal surgery have been reported. These are a stable camera platform, three-dimensional imaging, excellent ergonomics, tremor elimination, ambidextrous capability, motion scaling, and instruments with multiple degrees of freedom. However, despite these technological advantages, most studies did not report the clinical advantages of robotic colorectal surgery compared to standard laparoscopic colorectal surgery. Only one study recently implies the real benefits of robotic rectal cancer surgery. The purpose of this review article is to outline the early concerns of robotic colorectal surgery using the da Vinci system, to present early clinical outcomes from the most current series, and to discuss not only the safety and the feasibility but also the real benefits of robotic colorectal surgery. Moreover, this article will comment on the possible future clinical advantages and limitations of the da Vinci system in robotic colorectal surgery.',\n",
              "  'authors': ['Baik, Seung Hyuk'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Yonsei medical journal',\n",
              "  'keywords': ['Colorectal Neoplasms',\n",
              "   'Digestive System Surgical Procedures',\n",
              "   'History, 20th Century',\n",
              "   'History, 21st Century',\n",
              "   'Humans',\n",
              "   'Robotics'],\n",
              "  'pmid': '19108010',\n",
              "  'title': 'Robotic colorectal surgery.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '2050-084X',\n",
              "  'IF_2013': '8.519',\n",
              "  'IF_2014': '9.322',\n",
              "  'IF_2015': '8.303',\n",
              "  'IF_2016': '7.725',\n",
              "  'IF_2017': '7.616',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2050-084X',\n",
              "  'IsoAbbr': 'Elife',\n",
              "  'JrId': 40588,\n",
              "  'MedAbbr': 'Elife',\n",
              "  'NLMid': '101579614',\n",
              "  'Titre': 'Elife',\n",
              "  'abstract': 'In vivo calcium imaging through microendoscopic lenses enables imaging of previously inaccessible neuronal populations deep within the brains of freely moving animals. However, it is computationally challenging to extract single-neuronal activity from microendoscopic data, because of the very large background fluctuations and high spatial overlaps intrinsic to this recording modality. Here, we describe a new constrained matrix factorization approach to accurately separate the background and then demix and denoise the neuronal signals of interest. We compared the proposed method against previous independent components analysis and constrained nonnegative matrix factorization approaches. On both simulated and experimental data recorded from mice, our method substantially improved the quality of extracted cellular signals and detected more well-isolated neural signals, especially in noisy data regimes. These advances can in turn significantly enhance the statistical power of downstream analyses, and ultimately improve scientific conclusions derived from microendoscopic data.',\n",
              "  'authors': ['Zhou, Pengcheng',\n",
              "   'Resendez, Shanna L',\n",
              "   'Rodriguez-Romaguera, Jose',\n",
              "   'Jimenez, Jessica C',\n",
              "   'Neufeld, Shay Q',\n",
              "   'Giovannucci, Andrea',\n",
              "   'Friedrich, Johannes',\n",
              "   'Pnevmatikakis, Eftychios A',\n",
              "   'Stuber, Garret D',\n",
              "   'Hen, Rene',\n",
              "   'Kheirbek, Mazen A',\n",
              "   'Sabatini, Bernardo L',\n",
              "   'Kass, Robert E',\n",
              "   'Paninski, Liam'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'eLife',\n",
              "  'keywords': ['Animals',\n",
              "   'Brain',\n",
              "   'Calcium Signaling',\n",
              "   'Endoscopy',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Mice',\n",
              "   'Neurons',\n",
              "   'Video Recording'],\n",
              "  'pmid': '29469809',\n",
              "  'title': 'Efficient and accurate extraction of in vivo calcium signals from microendoscopic video data.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2041-3033',\n",
              "  'IF_2013': '1.144',\n",
              "  'IF_2014': '1.329',\n",
              "  'IF_2015': '0.996',\n",
              "  'IF_2016': '1.005',\n",
              "  'IF_2017': '1.124',\n",
              "  'ISSN': '0954-4119',\n",
              "  'ISSN_online': '2041-3033',\n",
              "  'ISSN_print': '0954-4119',\n",
              "  'IsoAbbr': 'Proc Inst Mech Eng H',\n",
              "  'JrId': 1315,\n",
              "  'MedAbbr': 'Proc Inst Mech Eng H',\n",
              "  'NLMid': '8908934',\n",
              "  'Titre': 'Proc Inst Mech Eng H',\n",
              "  'abstract': \"Bone cutting in total joint reconstructions requires a high accuracy to obtain a well-functioning and long-lasting prosthesis. Hence robot assistance can be useful to increase the precision of the surgical actions. A drawback of current robot systems is that they autonomously machine the bone, in that way ignoring the surgeon's experience and introducing a safety risk. This paper presents a semi-active milling procedure to overcome that drawback. In this procedure the surgeon controls robot motion by exerting forces on a force-controlled lever that is attached to the robot end effector. Meanwhile the robot constrains tool motion to the planned motion and generates a tool feed determined by the feed force that the surgeon executes. As a case study the presented milling procedure has been implemented on a laboratory set-up for robot-assisted preparation of the acetabulum in total hip arthroplasty. Two machining methods have been considered. In the first method the surgeon determines both milling trajectory and feed by the forces that he/she executes on the force-controlled lever. In the second method the cavity is machined contour by contour, and the surgeon only provides the feed. Machining experiments have shown that the first method results in large surface irregularities and is not useful. The second method, however, results in accurate cavity preparation and has therefore potential to be implemented in future robot systems.\",\n",
              "  'authors': ['Van Ham, G',\n",
              "   'Denis, K',\n",
              "   'Vander Sloten, J',\n",
              "   'Van Audekercke, R',\n",
              "   'Van der Perre, G',\n",
              "   'De Schutter, J',\n",
              "   'Simon, J P',\n",
              "   'Fabry, G'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Proceedings of the Institution of Mechanical Engineers. Part H, Journal of engineering in medicine',\n",
              "  'keywords': ['Acetabulum',\n",
              "   'Arthroplasty, Replacement, Hip',\n",
              "   'Computer Systems',\n",
              "   'Computer-Aided Design',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Hip Prosthesis',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Orthopedic Procedures',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '15934392',\n",
              "  'title': 'A semi-active milling procedure in view of preparing implantation beds in robot-assisted orthopaedic surgery.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Soil salinization is becoming a serious threat to crop yield all over the world. Nowadays, acquainting the specific molecular mechanisms underlying various abiotic stresses especially to salt stress should be of great importance. While the development of the high-throughout sequencing technology promoted the progress powerfully. The intricate perception, transduction and regulation mechanisms underlying salt stress are being illustrated more and more clearly. As a perennial halophytic plant, Kosteletzkya virginica is able to help us to understand the mechanisms more directly and effectively. We carried out the whole transcriptome analysis on young seedlings with or without salt treatment through high-throughout sequencing technology. The results revealed that the numbers of different expressed transcripts between control and different treatments are 4145 and 9134, respectively. The ORF prediction suggested that there were 94308 ORF out of the 103489 (91.10%) total transcripts. We also carried out further differential expression analysis through gene ontology (GO) classification, cluster of orthologous groups (COG) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis. In a word, our transcriptome study on Kosteletzkya virginica would provide direct and effective references for researches on molecular mechanisms of salt-tolerance, extending our view of salt tolerance in plant further. Above all, the related report in this paper is the first about Kosteletzkya virginica. ',\n",
              "  'authors': ['Tang, Xiaoli',\n",
              "   'Wang, Hongyan',\n",
              "   'Shao, Chuyang',\n",
              "   'Shao, Hongbo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Cluster Analysis',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Malvaceae',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Open Reading Frames',\n",
              "   'RNA, Messenger',\n",
              "   'Reproducibility of Results',\n",
              "   'Seedlings',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Sodium Chloride',\n",
              "   'Stress, Physiological',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '25901608',\n",
              "  'title': 'Global Gene Expression of Kosteletzkya virginica Seedlings Responding to Salt Stress.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1872-7727',\n",
              "  'IF_2013': '2.16',\n",
              "  'IF_2014': '2.369',\n",
              "  'IF_2015': '2.593',\n",
              "  'IF_2016': '2.462',\n",
              "  'IF_2017': '2.843',\n",
              "  'ISSN': '0720-048X',\n",
              "  'ISSN_online': '1872-7727',\n",
              "  'ISSN_print': '0720-048X',\n",
              "  'IsoAbbr': 'Eur J Radiol',\n",
              "  'JrId': 3584,\n",
              "  'MedAbbr': 'Eur J Radiol',\n",
              "  'NLMid': '8106411',\n",
              "  'Titre': 'Eur J Radiol',\n",
              "  'abstract': 'Given the fact that despite high expectations, expert systems have still an almost negligible impact on the practice of medicine, the present trends and methods for medical expert systems are discussed. An analysis of the diagnostic process in radiology leads to a discussion about possible computer support and on the conditions of use and acceptance of such systems.',\n",
              "  'authors': ['Gell, G'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European journal of radiology',\n",
              "  'keywords': ['Cost-Benefit Analysis',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Radiology Information Systems',\n",
              "   'Software'],\n",
              "  'pmid': '8348914',\n",
              "  'title': 'Expert systems as a support for radiological diagnosis.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'One of the limitations of the current computer-aided detection (CAD) of polyps in CT colonography (CTC) is a relatively large number of false-positive (FP) detections. Rectal tubes (RTs) are one of the typical sources of FPs because a portion of a RT, especially a portion of a bulbous tip, often exhibits a cap-like shape that closely mimics the appearance of a small polyp. Radiologists can easily recognize and dismiss RT-induced FPs; thus, they may lose their confidence in CAD as an effective tool if the CAD scheme generates such \"obvious\" FPs due to RTs consistently. In addition, RT-induced FPs may distract radiologists from less common true positives in the rectum. Therefore, removal RT-induced FPs as well as other types of FPs is desirable while maintaining a high sensitivity in the detection of polyps. We developed a three-dimensional (3D) massive-training artificial neural network (MTANN) for distinction between polyps and RTs in 3D CTC volumetric data. The 3D MTANN is a supervised volume-processing technique which is trained with input CTC volumes and the corresponding \"teaching\" volumes. The teaching volume for a polyp contains a 3D Gaussian distribution, and that for a RT contains zeros for enhancement of polyps and suppression of RTs, respectively. For distinction between polyps and nonpolyps including RTs, a 3D scoring method based on a 3D Gaussian weighting function is applied to the output of the trained 3D MTANN. Our database consisted of CTC examinations of 73 patients, scanned in both supine and prone positions (146 CTC data sets in total), with optical colonoscopy as a reference standard for the presence of polyps. Fifteen patients had 28 polyps, 15 of which were 5-9 mm and 13 were 10-25 mm in size. These CTC cases were subjected to our previously reported CAD scheme that included centerline-based segmentation of the colon, shape-based detection of polyps, and reduction of FPs by use of a Bayesian neural network based on geometric and texture features. Application of this CAD scheme yielded 96.4% (27/28) by-polyp sensitivity with 3.1 (224/73) FPs per patient, among which 20 FPs were caused by RTs. To eliminate the FPs due to RTs and possibly other normal structures, we trained a 3D MTANN with ten representative polyps and ten RTs, and applied the trained 3D MTANN to the above CAD true- and false-positive detections. In the output volumes of the 3D MTANN, polyps were represented by distributions of bright voxels, whereas RTs and other normal structures partly similar to RTs appeared as darker voxels, indicating the ability of the 3D MTANN to suppress RTs as well as other normal structures effectively. Application of the 3D MTANN to the CAD detections showed that the 3D MTANN eliminated all RT-induced 20 FPs, as well as 53 FPs due to other causes, without removal of any true positives. Overall, the 3D MTANN was able to reduce the FP rate of the CAD scheme from 3.1 to 2.1 FPs per patient (33% reduction), while the original by-polyp sensitivity of 96.4% was maintained.',\n",
              "  'authors': ['Suzuki, Kenji',\n",
              "   'Yoshida, Hiroyuki',\n",
              "   'Näppi, Janne',\n",
              "   'Dachman, Abraham H'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Automation',\n",
              "   'Colon',\n",
              "   'Colonic Neoplasms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'False Positive Reactions',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Observer Variation',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Polyps',\n",
              "   'ROC Curve',\n",
              "   'Rectum',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '17089846',\n",
              "  'title': 'Massive-training artificial neural network (MTANN) for reduction of false positives in computer-aided detection of polyps: Suppression of rectal tubes.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1095-9203',\n",
              "  'IF_2013': '31.477',\n",
              "  'IF_2014': '33.611',\n",
              "  'IF_2015': '34.661',\n",
              "  'IF_2016': '37.205',\n",
              "  'IF_2017': '41.058',\n",
              "  'ISSN': '0036-8075',\n",
              "  'ISSN_online': '1095-9203',\n",
              "  'ISSN_print': '0036-8075',\n",
              "  'IsoAbbr': 'Science',\n",
              "  'JrId': 7473,\n",
              "  'MedAbbr': 'Science',\n",
              "  'NLMid': '0404511',\n",
              "  'Titre': 'Science',\n",
              "  'abstract': 'Artificial intelligence, long a topic of basic computer science research, is now being applied to problems of scientific, technical, and commercial interest. Some consultation programs, although limited in versatility, have achieved levels of performance rivaling those of human experts. A collateral benefit of this work is the systematization of previously unformalized knowledge in areas such as medical diagnosis and geology.',\n",
              "  'authors': ['Duda, R O', 'Shortliffe, E H'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Science (New York, N.Y.)',\n",
              "  'keywords': ['Anti-Bacterial Agents',\n",
              "   'Bacterial Infections',\n",
              "   'Computers',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Humans',\n",
              "   'Research',\n",
              "   'Systems Analysis'],\n",
              "  'pmid': '6340198',\n",
              "  'title': 'Expert Systems Research.',\n",
              "  'year': '1983'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1563-5279',\n",
              "  'IF_2013': '1.528',\n",
              "  'IF_2014': '1.521',\n",
              "  'IF_2015': '1.717',\n",
              "  'IF_2016': '1.75',\n",
              "  'IF_2017': '1.848',\n",
              "  'ISSN': '0020-7454',\n",
              "  'ISSN_online': '1563-5279',\n",
              "  'ISSN_print': '0020-7454',\n",
              "  'IsoAbbr': 'Int. J. Neurosci.',\n",
              "  'JrId': 4312,\n",
              "  'MedAbbr': 'Int J Neurosci',\n",
              "  'NLMid': '0270707',\n",
              "  'Titre': 'Int. J. Neurosci.',\n",
              "  'abstract': 'The present study shows with a sophisticated quantitative method a change in blood flow velocity in a cerebral artery in response to acupuncture. Monitoring of transcranial Doppler sonography (TCD) was examined in 12 healthy volunteers (mean age: 35.2+/-4.4 years; 26-41 years; 4 female, 8 male) before, during and after acupuncture. In all persons the same unspecific scheme of acupuncture with a \"general increase of Qi-energy\" was used. The measurements were performed with new robotic TCD-probes allowing three-dimensional imaging by using multi-scan methods. The results showed a significant (p < 0.01; ANOVA; Tukey test) increase in mean blood flow velocity, which was measured at different depths of the right middle cerebral artery. Quantitative determination of the physiological effects during acupuncture can lead to a better acceptance of this unconventional complementary medical method.',\n",
              "  'authors': ['Litscher, G', 'Schwarz, G', 'Sandner-Kiesling, A', 'Hadolt, I'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The International journal of neuroscience',\n",
              "  'keywords': ['Acupuncture Therapy',\n",
              "   'Adult',\n",
              "   'Cerebral Cortex',\n",
              "   'Cerebrovascular Circulation',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Qi',\n",
              "   'Robotics',\n",
              "   'Ultrasonography, Doppler, Transcranial'],\n",
              "  'pmid': '9845012',\n",
              "  'title': 'Robotic transcranial Doppler sonography probes and acupuncture.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'An operator independent technique has been developed to quantitate the volume of white matter (WM), grey matter (GM), and cerebrospinal fluid (CSF) using spin-echo magnetic resonance images. Using skull stripped spin-echo images, CSF was removed using an automated thresholding technique. The bimodal histogram of the remaining images was used to train a perceptron and a single hidden layer neural network to output the percentage of GM and WM in the image. The output values were compared with those of a semiautomated technique employing a least square fitting technique [graduated nonconvexity algorithm (GNC)] applied to the bimodal histogram. This semiautomated technique allowed for intervention by the radiologist. Fourteen normal volunteers with eight contiguous slices each were analyzed. The individual percentages of WM, GM, and CSF of 40 slices from five subjects not used in the training set as well as the total percentages of GM, WM, and CSF in each individual were predicted using the two artificial network architectures. GM, WM, and CSF percentages were predicted within 7% for individual slices while total percentages of WM, GM, and CSF were computed accurately with an absolute error of less than 5% when compared to the semiautomated technique involving a trained neuroradiologist.',\n",
              "  'authors': ['Raff, U', 'Scherzinger, A L', 'Vargas, P F', 'Simon, J H'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Adult',\n",
              "   'Biophysical Phenomena',\n",
              "   'Biophysics',\n",
              "   'Brain',\n",
              "   'Cerebrospinal Fluid',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pilot Projects'],\n",
              "  'pmid': '7700201',\n",
              "  'title': 'Quantitation of grey matter, white matter, and cerebrospinal fluid from spin-echo magnetic resonance images using an artificial neural network technique.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'Dose distributions registered in water equivalent, polymer gel dosimeters were used to measure the output factors and off-axis profiles of the radiosurgical photon beams employed for CyberKnife radiosurgery. Corresponding measurements were also performed using a shielded silicon diode commonly employed for CyberKnife commissioning, the PinPoint ion chamber, and Gafchromic EBT films, for reasons of comparison. Polymer gel results of this work for the output factors of the 5, 7.5, and 10 mm diameter beams are (0.702 +/- 0.029), (0.872 +/- 0.039), and (0.929 +/- 0.041), respectively. Comparison of polymer gel and diode measurements shows that the latter overestimate output factors of the two small beams (5% for the 5 mm beam and 3% for the 7.5 mm beams). This is attributed to the nonwater equivalence of the high atomic number silicon material of the diode detector. On the other hand, the PinPoint chamber is found to underestimate output factors up to 10% for the 5 mm beam due to volume averaging effects. Polymer gel and EBT film output factor results are found in close agreement for all beam sizes, emphasizing the importance of water equivalence and fine detector sensitive volume for small field dosimetry. Relative off-axis profile results are in good agreement for all dosimeters used in this work, with noticeable differences observed only in the PinPoint estimate of the 80%-20% penumbra width, which is relatively overestimated.',\n",
              "  'authors': ['Pantelis, E',\n",
              "   'Antypas, C',\n",
              "   'Petrokokkinos, L',\n",
              "   'Karaiskos, P',\n",
              "   'Papagiannis, P',\n",
              "   'Kozicki, M',\n",
              "   'Georgiou, E',\n",
              "   'Sakelliou, L',\n",
              "   'Seimenis, I'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Calibration',\n",
              "   'Electrodes',\n",
              "   'Gels',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Photons',\n",
              "   'Polymers',\n",
              "   'Radiometry',\n",
              "   'Radiosurgery',\n",
              "   'Robotics',\n",
              "   'Silicon'],\n",
              "  'pmid': '18649464',\n",
              "  'title': 'Dosimetric characterization of CyberKnife radiosurgical photon beams using polymer gels.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DA'],\n",
              "  'ESSN': '1472-6807',\n",
              "  'IF_2013': '2.222',\n",
              "  'IF_2014': '1.178',\n",
              "  'IF_2015': '1.926',\n",
              "  'IF_2016': '1.771',\n",
              "  'IF_2017': '1.308',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6807',\n",
              "  'IsoAbbr': 'BMC Struct. Biol.',\n",
              "  'JrId': 22352,\n",
              "  'MedAbbr': 'BMC Struct Biol',\n",
              "  'NLMid': '101088689',\n",
              "  'Titre': 'BMC Struct. Biol.',\n",
              "  'abstract': 'Obtaining atomic-scale information about large-amplitude conformational transitions in proteins is a challenging problem for both experimental and computational methods. Such information is, however, important for understanding the mechanisms of interaction of many proteins.|This paper presents a computationally efficient approach, combining methods originating from robotics and computational biophysics, to model protein conformational transitions. The ability of normal mode analysis to predict directions of collective, large-amplitude motions is applied to bias the conformational exploration performed by a motion planning algorithm. To reduce the dimension of the problem, normal modes are computed for a coarse-grained elastic network model built on short fragments of three residues. Nevertheless, the validity of intermediate conformations is checked using the all-atom model, which is accurately reconstructed from the coarse-grained one using closed-form inverse kinematics.|Tests on a set of ten proteins demonstrate the ability of the method to model conformational transitions of proteins within a few hours of computing time on a single processor. These results also show that the computing time scales linearly with the protein size, independently of the protein topology. Further experiments on adenylate kinase show that main features of the transition between the open and closed conformations of this protein are well captured in the computed path.|The proposed method enables the simulation of large-amplitude conformational transitions in proteins using very few computational resources. The resulting paths are a first approximation that can directly provide important information on the molecular mechanisms involved in the conformational transition. This approximation can be subsequently refined and analyzed using state-of-the-art energy models and molecular modeling methods.',\n",
              "  'authors': ['Al-Bluwi, Ibrahim',\n",
              "   'Vaisset, Marc',\n",
              "   'Siméon, Thierry',\n",
              "   'Cortés, Juan'],\n",
              "  'categories': ['BIOPHYSICS'],\n",
              "  'journal': 'BMC structural biology',\n",
              "  'keywords': ['Adenylate Kinase',\n",
              "   'Computer Simulation',\n",
              "   'Models, Molecular',\n",
              "   'Protein Conformation',\n",
              "   'Protein Structure, Secondary',\n",
              "   'Proteins',\n",
              "   'Robotics'],\n",
              "  'pmid': '24564964',\n",
              "  'title': 'Modeling protein conformational transitions by a combination of coarse-grained normal mode analysis and robotics-inspired methods.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1873-5223',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '0.964',\n",
              "  'IF_2016': '1.314',\n",
              "  'IF_2017': '1.313',\n",
              "  'ISSN': '1471-5953',\n",
              "  'ISSN_online': '1873-5223',\n",
              "  'ISSN_print': '1471-5953',\n",
              "  'IsoAbbr': 'Nurse Educ Pract',\n",
              "  'JrId': 33515,\n",
              "  'MedAbbr': 'Nurse Educ Pract',\n",
              "  'NLMid': '101090848',\n",
              "  'Titre': 'Nurse Educ Pract',\n",
              "  'abstract': \"This paper reports on a unique practice based learning model to prepare undergraduate nursing students for clinical placement. The learning and teaching model described in this paper outlines the establishment of an entire on-campus simulated hospital and health service (SHHS) at the University of South Australia, School of Nursing and Midwifery. The model is pedagogically structured to immerse students in an authentic clinical environment to achieve deep learning in preparation for safe practice. A quality improvement cycle was used to evaluate the outcomes of the model in two phases: Phase 1: Purposive sampling of first and second year Bachelor of Nursing students from 2012 to 2015 who were surveyed about their satisfaction with the model of learning. Bachelor of Nursing students were invited to complete a survey about their experience with the teaching and learning model employed in the SHHS in response to the question, 'What aspects of the SHHS are the most important to your success?' Phase 2: External clinical stakeholders working with nursing students in clinical placements were asked to respond to questions about the preparedness of students educated in this model to transition to employment. The evaluation showed that the SHHS model positively influenced students' satisfaction and confidence and increased the perception of clinicians of the work readiness of students.\",\n",
              "  'authors': ['Parker, Barbara A', 'Grech, Carol'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Nurse education in practice',\n",
              "  'keywords': ['Australia',\n",
              "   'Clinical Competence',\n",
              "   'Education, Nursing, Baccalaureate',\n",
              "   'Female',\n",
              "   'High Fidelity Simulation Training',\n",
              "   'Hospitals',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Midwifery',\n",
              "   'Pregnancy',\n",
              "   'Students, Nursing',\n",
              "   'Surveys and Questionnaires'],\n",
              "  'pmid': '30241029',\n",
              "  'title': \"Authentic practice environments to support undergraduate nursing students' readiness for hospital placements. A new model of practice in an on campus simulated hospital and health service.\",\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2052-4463',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '4.836',\n",
              "  'IF_2015': '4.836',\n",
              "  'IF_2016': '4.836',\n",
              "  'IF_2017': '5.305',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2052-4463',\n",
              "  'IsoAbbr': 'Sci Data',\n",
              "  'JrId': 43056,\n",
              "  'MedAbbr': 'Sci Data',\n",
              "  'NLMid': '101640192',\n",
              "  'Titre': 'Sci Data',\n",
              "  'abstract': \"Advances in high-throughput sequencing are reshaping how we perceive microbial communities inhabiting the human body, with implications for therapeutic interventions. Several large-scale datasets derived from hundreds of human microbiome samples sourced from multiple studies are now publicly available. However, idiosyncratic data processing methods between studies introduce systematic differences that confound comparative analyses. To overcome these challenges, we developed GutCyc, a compendium of environmental pathway genome databases (ePGDBs) constructed from 418 assembled human microbiome datasets using MetaPathways, enabling reproducible functional metagenomic annotation. We also generated metabolic network reconstructions for each metagenome using the Pathway Tools software, empowering researchers and clinicians interested in visualizing and interpreting metabolic pathways encoded by the human gut microbiome. For the first time, GutCyc provides consistent annotations and metabolic pathway predictions, making possible comparative community analyses between health and disease states in inflammatory bowel disease, Crohn's disease, and type 2 diabetes. GutCyc data products are searchable online, or may be downloaded and explored locally using MetaPathways and Pathway Tools.\",\n",
              "  'authors': ['Hahn, Aria S',\n",
              "   'Altman, Tomer',\n",
              "   'Konwar, Kishori M',\n",
              "   'Hanson, Niels W',\n",
              "   'Kim, Dongjae',\n",
              "   'Relman, David A',\n",
              "   'Dill, David L',\n",
              "   'Hallam, Steven J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific data',\n",
              "  'keywords': ['Crohn Disease',\n",
              "   'Databases, Genetic',\n",
              "   'Diabetes Mellitus, Type 2',\n",
              "   'Gastrointestinal Microbiome',\n",
              "   'Geography, Medical',\n",
              "   'Humans',\n",
              "   'Inflammatory Bowel Diseases',\n",
              "   'Metabolic Networks and Pathways',\n",
              "   'Metagenome',\n",
              "   'Metagenomics'],\n",
              "  'pmid': '28398290',\n",
              "  'title': 'A geographically-diverse collection of 418 human gut microbiome pathway genome databases.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['QU'],\n",
              "  'ESSN': '1574-6968',\n",
              "  'IF_2013': '2.723',\n",
              "  'IF_2014': '2.121',\n",
              "  'IF_2015': '1.858',\n",
              "  'IF_2016': '1.765',\n",
              "  'IF_2017': '1.735',\n",
              "  'ISSN': '0378-1097',\n",
              "  'ISSN_online': '1574-6968',\n",
              "  'ISSN_print': '0378-1097',\n",
              "  'IsoAbbr': 'FEMS Microbiol. Lett.',\n",
              "  'JrId': 3862,\n",
              "  'MedAbbr': 'FEMS Microbiol Lett',\n",
              "  'NLMid': '7705721',\n",
              "  'Titre': 'FEMS Microbiol. Lett.',\n",
              "  'abstract': \"Sixteen representatives of three morphologically distinct groups of streptomycetes were recovered from soil using selective isolation procedures. Duplicated batches of the test strains were examined by Curie-point pyrolysis mass spectrometry and the first data set used for conventional multivariate statistical analyses and as a training set for an artificial neural network. The second set of data was used for 'operational fingerprinting' and for testing the artificial neural network. All of the test strains were correctly identified using the artificial neural network whereas only fifteen of the sixteen strains were assigned to the correct group using the conventional operational fingerprinting procedure. Artificial neural network analysis of pyrolysis mass spectrometric data provides a rapid, cost-effective and reproducible way of identifying and typing large numbers of microorganisms.\",\n",
              "  'authors': ['Chun, J', 'Atalan, E', 'Ward, A C', 'Goodfellow, M'],\n",
              "  'categories': ['MICROBIOLOGY'],\n",
              "  'journal': 'FEMS microbiology letters',\n",
              "  'keywords': ['Bacterial Typing Techniques',\n",
              "   'Evaluation Studies as Topic',\n",
              "   'Hot Temperature',\n",
              "   'Mass Spectrometry',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Streptomyces'],\n",
              "  'pmid': '8472913',\n",
              "  'title': 'Artificial neural network analysis of pyrolysis mass spectrometric data in the identification of Streptomyces strains.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Intrathoracic supernumary fifth-gland parathyroid disease is an unusual presentation for primary hyperparathyroidism. Patients generally present with enlargement of one of the normal four cervical parathyroid glands. This paper is a case report of a young woman who presented with typical symptoms of hyperparathyroidism. She, however, had an atypical anatomic finding, and this case offers a unique means of resection.',\n",
              "  'authors': ['Timmerman, Gary L',\n",
              "   'Allard, Brandon',\n",
              "   'Lovrien, Fred',\n",
              "   'Hickey, David'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hyperparathyroidism',\n",
              "   'Parathyroid Neoplasms',\n",
              "   'Robotics',\n",
              "   'Thoracoscopy',\n",
              "   'Tomography, Emission-Computed, Single-Photon',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '18266579',\n",
              "  'title': 'Hyperparathyroidism: robotic-assisted thoracoscopic resection of a supernumary anterior mediastinal parathyroid tumor.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '2047-7538',\n",
              "  'IF_2013': '8.476',\n",
              "  'IF_2014': '14.603',\n",
              "  'IF_2015': '13.6',\n",
              "  'IF_2016': '14.098',\n",
              "  'IF_2017': '13.625',\n",
              "  'ISSN': '2095-5545',\n",
              "  'ISSN_online': '2047-7538',\n",
              "  'ISSN_print': '2095-5545',\n",
              "  'IsoAbbr': 'Light Sci Appl',\n",
              "  'JrId': 41525,\n",
              "  'MedAbbr': 'Light Sci Appl',\n",
              "  'NLMid': '101610753',\n",
              "  'Titre': 'Light Sci Appl',\n",
              "  'abstract': 'The growing demands of brain science and artificial intelligence create an urgent need for the development of artificial neural networks (ANNs) that can mimic the structural, functional and biological features of human neural networks. Nanophotonics, which is the study of the behaviour of light and the light-matter interaction at the nanometre scale, has unveiled new phenomena and led to new applications beyond the diffraction limit of light. These emerging nanophotonic devices have enabled scientists to develop paradigm shifts of research into ANNs. In the present review, we summarise the recent progress in nanophotonics for emulating the structural, functional and biological features of ANNs, directly or indirectly.',\n",
              "  'authors': ['Zhang, Qiming',\n",
              "   'Yu, Haoyi',\n",
              "   'Barbiero, Martina',\n",
              "   'Wang, Baokai',\n",
              "   'Gu, Min'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Light, science & applications',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31098012',\n",
              "  'title': 'Artificial neural networks enabled by nanophotonics.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1474-547X',\n",
              "  'IF_2013': '39.207',\n",
              "  'IF_2014': '45.217',\n",
              "  'IF_2015': '44.002',\n",
              "  'IF_2016': '47.831',\n",
              "  'IF_2017': '53.254',\n",
              "  'ISSN': '0140-6736',\n",
              "  'ISSN_online': '1474-547X',\n",
              "  'ISSN_print': '0140-6736',\n",
              "  'IsoAbbr': 'Lancet',\n",
              "  'JrId': 5470,\n",
              "  'MedAbbr': 'Lancet',\n",
              "  'NLMid': '2985213R',\n",
              "  'Titre': 'Lancet',\n",
              "  'abstract': 'Long-chain polyunsaturated fatty acids (LCPUFA) are important for normal visual and brain development. Although present in human milk, LCPUFA have until recently been absent from artificial formulas, and infants may have limited ability to synthesise LCPUFA. To determine the clinical significance of this relative deficiency of LCPUFA, we undertook a randomised trial of the relation between LCPUFA supplementation and infant cognitive behaviour.|44 term infants had been randomised to a formula supplemented with LCPUFA (21) or not supplemented with LCPUFA (23), which they had taken from birth to age 4 months. Infant cognitive behaviour was assessed at 10 months of age by a means-end problem-solving test--the intentional execution of a sequence of steps to achieve a goal. The problem required three intermediate steps to achieve the final goal, uncovering and retrieving a hidden toy.|Infants who received LCPUFA-supplemented formula had significantly more intentional solutions than infants who received the no-LCPUFA formula (median 2.0 vs 0, p=0.021). Intention scores (median 14.0 vs 11.5 [maximum 18]) were also increased in this group (p=0.035).|These findings suggest that term infants may benefit from LCPUFA supplementation, and that the effects persist beyond the period of supplementation. Since higher problem-solving scores in infancy are related to higher childhood IQ scores, supplementation with LCPUFA may be important for the development of childhood intelligence.',\n",
              "  'authors': ['Willatts, P',\n",
              "   'Forsyth, J S',\n",
              "   'DiModugno, M K',\n",
              "   'Varma, S',\n",
              "   'Colvin, M'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Lancet (London, England)',\n",
              "  'keywords': ['Cognition',\n",
              "   'Fatty Acids, Unsaturated',\n",
              "   'Food, Fortified',\n",
              "   'Humans',\n",
              "   'Infant Food',\n",
              "   'Infant, Newborn'],\n",
              "  'pmid': '9728984',\n",
              "  'title': 'Effect of long-chain polyunsaturated fatty acids in infant formula on problem solving at 10 months of age.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1941-0506',\n",
              "  'IF_2013': '1.919',\n",
              "  'IF_2014': '2.168',\n",
              "  'IF_2015': '1.4',\n",
              "  'IF_2016': '2.84',\n",
              "  'IF_2017': '3.078',\n",
              "  'ISSN': '1077-2626',\n",
              "  'ISSN_online': '1941-0506',\n",
              "  'ISSN_print': '1077-2626',\n",
              "  'IsoAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'JrId': 31980,\n",
              "  'MedAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'NLMid': '9891704',\n",
              "  'Titre': 'IEEE Trans Vis Comput Graph',\n",
              "  'abstract': 'This paper describes Show Me, an integrated set of user interface commands and defaults that incorporate automatic presentation into a commercial visual analysis system called Tableau. A key aspect of Tableau is VizQL, a language for specifying views, which is used by Show Me to extend automatic presentation to the generation of tables of views (commonly called small multiple displays). A key research issue for the commercial application of automatic presentation is the user experience, which must support the flow of visual analysis. User experience has not been the focus of previous research on automatic presentation. The Show Me user experience includes the automatic selection of mark types, a command to add a single field to a view, and a pair of commands to build views for multiple fields. Although the use of these defaults and commands is optional, user interface logs indicate that Show Me is used by commercial users.',\n",
              "  'authors': ['Mackinlay, Jock', 'Hanrahan, Pat', 'Stolte, Chris'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE transactions on visualization and computer graphics',\n",
              "  'keywords': ['Computer Graphics',\n",
              "   'Expert Systems',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Programming Languages',\n",
              "   'Software',\n",
              "   'Software Design',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '17968057',\n",
              "  'title': 'Show me: automatic presentation for visual analysis.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1535-3907',\n",
              "  'IF_2013': '5.001',\n",
              "  'IF_2014': '4.245',\n",
              "  'IF_2015': '4.173',\n",
              "  'IF_2016': '4.268',\n",
              "  'IF_2017': '3.95',\n",
              "  'ISSN': '1535-3893',\n",
              "  'ISSN_online': '1535-3907',\n",
              "  'ISSN_print': '1535-3893',\n",
              "  'IsoAbbr': 'J. Proteome Res.',\n",
              "  'JrId': 30137,\n",
              "  'MedAbbr': 'J Proteome Res',\n",
              "  'NLMid': '101128775',\n",
              "  'Titre': 'J. Proteome Res.',\n",
              "  'abstract': 'Adult rattlesnakes within genus Crotalus express one of two distinct venom phenotypes, type I (hemorrhagic) and type II (neurotoxic). In Costa Rican Central American rattlesnake, ontogenetic changes in the concentration of miRNAs modulate venom type II to type I transition. Venomics and venom gland transcriptome analyses showed that adult C. simus and C. tzabcan expressed intermediate patterns between type II and type I venoms, whereas C. culminatus had a canonical type I venom. Neonate/juvenile and adult Mexican rattlesnakes showed notable inter- and intraspecific variability in the number, type, abundance and ontogenetic shifts of the transcriptional and translational venom gland activities. These results support a role for miRNAs in the ontogenetic venom compositional changes in the three congeneric Mexican rattlesnakes. It is worth noting the finding of dual-action miRNAs, which silence the translation of neurotoxic heterodimeric PLA2 crotoxin and acidic PLA2 mRNAs while simultaneously up-regulating SVMP-targeting mRNAs. Dual transcriptional regulation potentially explains the existence of mutually exclusive crotoxin-rich (type-II) and SVMP-rich (type-I) venom phenotypic dichotomy among rattlesnakes. Our results support the hypothesis that alterations of the distribution of miRNAs, modulating the translational activity of venom gland toxin-encoding mRNAs in response to an external cue, may contribute to the mechanism generating adaptive venom variability.',\n",
              "  'authors': ['Durban, Jordi',\n",
              "   'Sanz, Libia',\n",
              "   'Trevisan-Silva, Dilza',\n",
              "   'Neri-Castro, Edgar',\n",
              "   'Alagón, Alejandro',\n",
              "   'Calvete, Juan J'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Journal of proteome research',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Animals',\n",
              "   'Base Sequence',\n",
              "   'Chromatography, Reverse-Phase',\n",
              "   'Crotalid Venoms',\n",
              "   'Crotalus',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Genetic Variation',\n",
              "   'MicroRNAs',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Protein Biosynthesis',\n",
              "   'Proteogenomics',\n",
              "   'Proteome',\n",
              "   'Species Specificity',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '28731347',\n",
              "  'title': 'Integrated Venomics and Venom Gland Transcriptome Analysis of Juvenile and Adult Mexican Rattlesnakes Crotalus simus, C. tzabcan, and C. culminatus Revealed miRNA-modulated Ontogenetic Shifts.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Laparoscopic splenectomy is accepted as a safe approach in the surgical treatment of blood disorders worldwide. Compared with the laparotomic technique, it is associated with a lower risk of intraoperative bleeding, less postoperative pain, and faster discharge times. The advent of robotic surgery (RS) has changed the concept of minimally invasive surgery because, in addition to allowing a three-dimensional view, it permits greater freedom of movement and higher levels of accuracy than laparoscopic surgery (LS). The aim of this study was to comparatively evaluate whether RS presents advantages over LS in spleen surgery.|In two Surgical Units with experience in laparoscopic splenectomy, over a 7-year period, two groups of 45 patients underwent LS and RS. The two groups were well matched for demographic characteristics, indications, and spleen size.|No statistically significant differences were found regarding intraoperative blood loss, conversion rate to laparotomy, food intake, drain removal, postoperative complications, and median time to discharge. On the contrary, statistically increased differences were observed in median operative time and costs. In both groups, the transfusion and mortality rate was 0%. At the 6-month follow-up no surgical complications were observed.|Although RS offers a three-dimensional view, greater freedom of movement, and higher levels of accuracy, it is associated with longer operative times and higher costs. It can consequently be concluded that with the intrinsic limits of the study design used, at the current time, RS does not have any significant advantage over LS in splenectomy.',\n",
              "  'authors': ['Gelmini, Roberta',\n",
              "   'Franzoni, Chiara',\n",
              "   'Spaziani, Alessandro',\n",
              "   'Patriti, Alberto',\n",
              "   'Casciola, Luciano',\n",
              "   'Saviano, Massimo'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Splenectomy'],\n",
              "  'pmid': '21561335',\n",
              "  'title': 'Laparoscopic splenectomy: conventional versus robotic approach--a comparative study.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1873-3573',\n",
              "  'IF_2013': '3.511',\n",
              "  'IF_2014': '3.545',\n",
              "  'IF_2015': '4.035',\n",
              "  'IF_2016': '4.162',\n",
              "  'IF_2017': '4.244',\n",
              "  'ISSN': '0039-9140',\n",
              "  'ISSN_online': '1873-3573',\n",
              "  'ISSN_print': '0039-9140',\n",
              "  'IsoAbbr': 'Talanta',\n",
              "  'JrId': 22720,\n",
              "  'MedAbbr': 'Talanta',\n",
              "  'NLMid': '2984816R',\n",
              "  'Titre': 'Talanta',\n",
              "  'abstract': \"The present study addresses the prediction of the time of ripening and type of mixtures of milk (cow's, ewe's and goat's) in cheeses of varying composition using artificial neural networks (ANN). To accomplish this aim, neural networks were designed using as input data the content of 19 fatty acids obtained with GC-FID of the cheese fat and scores obtained from principal component analysis (PCA) of NIR spectra. The best model of neuronal networks for the identification of the type of mixtures of milk was obtained using the information concerning the fatty acid concentration (80% of correct results in the training phase and 75% in the validation phase). Regarding the information of the near-infrared (NIR) spectra a neural network was designed. The aforesaid neural network predicted the ripening of cheeses with 100% accuracy in both training and in validation. \",\n",
              "  'authors': ['Soto-Barajas, Milton Carlos',\n",
              "   'González-Martín, Ma Inmaculada',\n",
              "   'Salvador-Esteban, Javier',\n",
              "   'Hernández-Hierro, José Miguel',\n",
              "   'Moreno-Rodilla, Vidal',\n",
              "   'Vivar-Quintana, Ana Ma',\n",
              "   'Revilla, Isabel',\n",
              "   'Ortega, Iris Lobos',\n",
              "   'Morón-Sancho, Raúl',\n",
              "   'Curto-Diego, Belén'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Talanta',\n",
              "  'keywords': ['Animals',\n",
              "   'Cheese',\n",
              "   'Fatty Acids',\n",
              "   'Female',\n",
              "   'Fermentation',\n",
              "   'Goats',\n",
              "   'Milk',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Principal Component Analysis',\n",
              "   'Sheep',\n",
              "   'Spectroscopy, Near-Infrared'],\n",
              "  'pmid': '24148372',\n",
              "  'title': 'Prediction of the type of milk and degree of ripening in cheeses by means of artificial neural networks with data concerning fatty acids and near infrared spectroscopy.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1464-3685',\n",
              "  'IF_2013': '9.197',\n",
              "  'IF_2014': '9.176',\n",
              "  'IF_2015': '7.522',\n",
              "  'IF_2016': '7.738',\n",
              "  'IF_2017': '8.36',\n",
              "  'ISSN': '0300-5771',\n",
              "  'ISSN_online': '1464-3685',\n",
              "  'ISSN_print': '0300-5771',\n",
              "  'IsoAbbr': 'Int J Epidemiol',\n",
              "  'JrId': 4294,\n",
              "  'MedAbbr': 'Int J Epidemiol',\n",
              "  'NLMid': '7802871',\n",
              "  'Titre': 'Int J Epidemiol',\n",
              "  'abstract': 'Artificial neural networks (ANN) are gaining prominence as a method of classification in a wide range of disciplines. In this study ANN is applied to data from a verbal autopsy study as a means of classifying cause of death.|A simulated ANN was trained on a subset of verbal autopsy data, and the performance was tested on the remaining data. The performance of the ANN models were compared to two other classification methods (physician review and logistic regression) which have been tested on the same verbal autopsy data.|Artificial neural network models were as accurate as or better than the other techniques in estimating the cause-specific mortality fraction (CSMF). They estimated the CSMF within 10% of true value in 8 out of 16 causes of death. Their sensitivity and specificity compared favourably with that of data-derived algorithms based on logistic regression models.|Cross-validation is crucial in preventing the over-fitting of the ANN models to the training data. Artificial neural network models are a potentially useful technique for classifying causes of death from verbal autopsies. Large training data sets are needed to improve the performance of data-derived algorithms, in particular ANN models.',\n",
              "  'authors': ['Boulle, A', 'Chandramohan, D', 'Weller, P'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'International journal of epidemiology',\n",
              "  'keywords': ['Autopsy',\n",
              "   'Cause of Death',\n",
              "   'Classification',\n",
              "   'Data Collection',\n",
              "   'Ethiopia',\n",
              "   'Ghana',\n",
              "   'Humans',\n",
              "   'Logistic Models',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tanzania'],\n",
              "  'pmid': '11416074',\n",
              "  'title': 'A case study of using artificial neural networks for classifying cause of death from verbal autopsy.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1535-3907',\n",
              "  'IF_2013': '5.001',\n",
              "  'IF_2014': '4.245',\n",
              "  'IF_2015': '4.173',\n",
              "  'IF_2016': '4.268',\n",
              "  'IF_2017': '3.95',\n",
              "  'ISSN': '1535-3893',\n",
              "  'ISSN_online': '1535-3907',\n",
              "  'ISSN_print': '1535-3893',\n",
              "  'IsoAbbr': 'J. Proteome Res.',\n",
              "  'JrId': 30137,\n",
              "  'MedAbbr': 'J Proteome Res',\n",
              "  'NLMid': '101128775',\n",
              "  'Titre': 'J. Proteome Res.',\n",
              "  'abstract': 'Sound scoring methods for sequence database search algorithms such as Mascot and Sequest are essential for sensitive and accurate peptide and protein identifications from proteomic tandem mass spectrometry data. In this paper, we present a software package that interfaces Mascot with Percolator, a well performing machine learning method for rescoring database search results, and demonstrate it to be amenable for both low and high accuracy mass spectrometry data, outperforming all available Mascot scoring schemes as well as providing reliable significance measures. Mascot Percolator can be readily used as a stand alone tool or integrated into existing data analysis pipelines.',\n",
              "  'authors': ['Brosch, Markus', 'Yu, Lu', 'Hubbard, Tim', 'Choudhary, Jyoti'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Journal of proteome research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Chromatography, Liquid',\n",
              "   'Databases, Protein',\n",
              "   'Peptide Fragments',\n",
              "   'Proteomics',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Software',\n",
              "   'Tandem Mass Spectrometry'],\n",
              "  'pmid': '19338334',\n",
              "  'title': 'Accurate and sensitive peptide identification with Mascot Percolator.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1755-8794',\n",
              "  'IF_2013': '3.914',\n",
              "  'IF_2014': '2.873',\n",
              "  'IF_2015': '2.726',\n",
              "  'IF_2016': '2.848',\n",
              "  'IF_2017': '3.317',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1755-8794',\n",
              "  'IsoAbbr': 'BMC Med Genomics',\n",
              "  'JrId': 34003,\n",
              "  'MedAbbr': 'BMC Med Genomics',\n",
              "  'NLMid': '101319628',\n",
              "  'Titre': 'BMC Med Genomics',\n",
              "  'abstract': 'A substantial proportion of Autism Spectrum Disorder (ASD) risk resides in de novo germline and rare inherited genetic variation. In particular, rare copy number variation (CNV) contributes to ASD risk in up to 10% of ASD subjects. Despite the striking degree of genetic heterogeneity, case-control studies have detected specific burden of rare disruptive CNV for neuronal and neurodevelopmental pathways. Here, we used machine learning methods to classify ASD subjects and controls, based on rare CNV data and comprehensive gene annotations. We investigated performance of different methods and estimated the percentage of ASD subjects that could be reliably classified based on presumed etiologic CNV they carry.|We analyzed 1,892 Caucasian ASD subjects and 2,342 matched controls. Rare CNVs (frequency 1% or less) were detected using Illumina 1M and 1M-Duo BeadChips. Conditional Inference Forest (CF) typically performed as well as or better than other classification methods. We found a maximum AUC (area under the ROC curve) of 0.533 when considering all ASD subjects with rare genic CNVs, corresponding to 7.9% correctly classified ASD subjects and less than 3% incorrectly classified controls; performance was significantly higher when considering only subjects harboring de novo or pathogenic CNVs. We also found rare losses to be more predictive than gains and that curated neurally-relevant annotations (brain expression, synaptic components and neurodevelopmental phenotypes) outperform Gene Ontology and pathway-based annotations.|CF is an optimal classification approach for case-control rare CNV data and it can be used to prioritize subjects with variants potentially contributing to ASD risk not yet recognized. The neurally-relevant annotations used in this study could be successfully applied to rare CNV case-control data-sets for other neuropsychiatric disorders.',\n",
              "  'authors': ['Engchuan, Worrawat',\n",
              "   'Dhindsa, Kiret',\n",
              "   'Lionel, Anath C',\n",
              "   'Scherer, Stephen W',\n",
              "   'Chan, Jonathan H',\n",
              "   'Merico, Daniele'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC medical genomics',\n",
              "  'keywords': ['Autistic Disorder',\n",
              "   'Case-Control Studies',\n",
              "   'Computational Biology',\n",
              "   'DNA Copy Number Variations',\n",
              "   'Female',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Molecular Sequence Annotation'],\n",
              "  'pmid': '25783485',\n",
              "  'title': 'Performance of case-control rare copy number variation annotation in classification of autism.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1473-6586',\n",
              "  'IF_2013': '2.115',\n",
              "  'IF_2014': '2.333',\n",
              "  'IF_2015': '2.006',\n",
              "  'IF_2016': '1.796',\n",
              "  'IF_2017': '1.83',\n",
              "  'ISSN': '0963-0643',\n",
              "  'ISSN_online': '1473-6586',\n",
              "  'ISSN_print': '0963-0643',\n",
              "  'IsoAbbr': 'Curr Opin Urol',\n",
              "  'JrId': 20614,\n",
              "  'MedAbbr': 'Curr Opin Urol',\n",
              "  'NLMid': '9200621',\n",
              "  'Titre': 'Curr Opin Urol',\n",
              "  'abstract': 'In recent years, robotic partial nephrectomy has emerged as a technique for treatment of small renal masses alongside laparoscopic and open partial nephrectomy. Since this technique is still in a phase of adoption, many technical improvements, alterations and early experiences are published. We aim to review the recent literature, focus on recent advances in techniques and give an overview of published series.|Recent series confirm the feasibility of robotic partial nephrectomy and demonstrate perioperative data and short-term oncological outcomes that are at least comparable to laparoscopic series. The development of better renorrhaphy techniques and optimal use of the robotic features to gain console surgeon independence seem to be the main focus. Also alternative hilar control, early unclamping and off-clamp techniques are being developed to lower the ischaemic effect on the kidney. The learning curve seems to be less steep than laparoscopic techniques.|Robotic partial nephrectomy proves to be a well tolerated and efficacious minimally invasive option in the treatment of renal lesions. Main areas of interest are decreasing warm ischaemia time and modified renal closure techniques.',\n",
              "  'authors': ['Van Haute, Wim', 'Gavazzi, Andrea', 'Dasgupta, Prokar'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current opinion in urology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Nephrectomy',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '20616740',\n",
              "  'title': 'Current status of robotic partial nephrectomy.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1976-2437',\n",
              "  'IF_2013': '1.263',\n",
              "  'IF_2014': '1.287',\n",
              "  'IF_2015': '1.154',\n",
              "  'IF_2016': '1.537',\n",
              "  'IF_2017': '1.564',\n",
              "  'ISSN': '0513-5796',\n",
              "  'ISSN_online': '1976-2437',\n",
              "  'ISSN_print': '0513-5796',\n",
              "  'IsoAbbr': 'Yonsei Med. J.',\n",
              "  'JrId': 8109,\n",
              "  'MedAbbr': 'Yonsei Med J',\n",
              "  'NLMid': '0414003',\n",
              "  'Titre': 'Yonsei Med. J.',\n",
              "  'abstract': 'We prospectively assessed the early radiographic and clinical outcomes (minimum follow-up of 2 years) of robot-assisted pedicle screw fixation (Robot-PSF) and conventional freehand pedicle screw fixation (Conv-PSF).|Patients were randomly assigned to Robot-PSF (37 patients) or Conv-PSF (41 patients) for posterior interbody fusion surgery. The Robot-PSF group underwent minimally invasive pedicle screw fixation using a pre-planned robot-guided screw trajectory. The Conv-PSF underwent screw fixation using the freehand technique. Radiographic adjacent segment degeneration (ASD) was measured on plain radiographs, and clinical outcomes were measured using visual analogue scale (VAS) and Oswestry disability index (ODI) scores regularly after surgery.|The two groups had similar values for radiographic ASD, including University California at Los Angeles grade, vertebral translation, angular motion, and loss of disc height (p=0.320). At final follow-up, both groups had experienced significant improvements in back VAS, leg VAS, and ODI scores after surgery (p<0.001), although inter-group differences were not significant for back VAS (p=0.876), leg VAS (p=0.429), and ODI scores (p=0.952). In the Conv-PSF group, revision surgery was required for two of the 25 patients (8%), compared to no patients in the Robot-PSF group.|There were no significant differences in radiographic ASD and clinical outcomes between Robot-PSF and Conv-PSF. Thus, the advantages of robot-assisted surgery (accurate pedicle screw insertion and minimal facet joint violation) do not appear to be clinically significant.',\n",
              "  'authors': ['Park, Sang Min',\n",
              "   'Kim, Ho Joong',\n",
              "   'Lee, Se Yeon',\n",
              "   'Chang, Bong Soon',\n",
              "   'Lee, Choon Ki',\n",
              "   'Yeom, Jin S'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Yonsei medical journal',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Lumbar Vertebrae',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pain Measurement',\n",
              "   'Pedicle Screws',\n",
              "   'Prospective Studies',\n",
              "   'Radiography',\n",
              "   'Range of Motion, Articular',\n",
              "   'Robotics',\n",
              "   'Spinal Fusion',\n",
              "   'Treatment Outcome',\n",
              "   'Visual Analog Scale',\n",
              "   'Zygapophyseal Joint'],\n",
              "  'pmid': '29611407',\n",
              "  'title': 'Radiographic and Clinical Outcomes of Robot-Assisted Posterior Pedicle Screw Fixation: Two-Year Results from a Randomized Controlled Trial.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Proper placement of ports during robot-assisted endoscopic surgery is critical to the success of the procedure. In current practice, port placement methods do not consider the ability of the robot to manoeuvre the tools.|This paper proposes to choose the best port location such that the performance of the robot is maximized. The Global Conditioning Index (GCI) is used to optimize port placement when using the da Vinci((R)) surgical system during cardiac surgery.|The results show that, due to a singularity at the remote centre of motion, higher performance is obtained the further away the port is from the workspace. When compared to the ports selected by an expert surgeon, our results show that it is possible to increase robot performance by at least 29% for the left arm of the robot.|Selecting an adequate port location can improve robot performance and ensure that the instruments reach the surgical site.',\n",
              "  'authors': ['Trejos, A L', 'Patel, R V', 'Ross, I', 'Kiaii, B'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cardiovascular Surgical Procedures',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '18000946',\n",
              "  'title': 'Optimizing port placement for robot-assisted minimally invasive cardiac surgery.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1573-9686',\n",
              "  'IF_2013': '3.231',\n",
              "  'IF_2014': '3.195',\n",
              "  'IF_2015': '2.887',\n",
              "  'IF_2016': '3.221',\n",
              "  'IF_2017': '3.405',\n",
              "  'ISSN': '0090-6964',\n",
              "  'ISSN_online': '1573-9686',\n",
              "  'ISSN_print': '0090-6964',\n",
              "  'IsoAbbr': 'Ann Biomed Eng',\n",
              "  'JrId': 561,\n",
              "  'MedAbbr': 'Ann Biomed Eng',\n",
              "  'NLMid': '0361512',\n",
              "  'Titre': 'Ann Biomed Eng',\n",
              "  'abstract': \"This study was designed to analyze the magnetoencephalogram (MEG) background activity from 20 patients with probable Alzheimer's disease (AD) and 21 control subjects by using two nonlinear methods: sample entropy (SampEn), and Lempel-Ziv complexity (LZC). The former quantifies the signal regularity, and the latter is a complexity measure. The signals were acquired with a 148-channel whole-head magnetometer placed in a magnetically shielded room. Our results show that MEG recordings are less complex and more regular in patients with AD than in control subjects. Significant differences between both groups were found in 16 MEG channels with SampEn and in 134 with LZC (p < 0.01, Student's t test with Bonferroni's correction). Using receiver operating characteristic curves with a leave-one-out cross-validation procedure, accuracies of 70.73 and 78.05% were reached with SampEn and LZC, respectively. Additionally, we wanted to assess whether both nonlinear methods and an adaptive-network-based fuzzy interference system (ANFIS) could improve AD diagnosis. With this classifier, an accuracy of 85.37% was achieved. Our findings suggest the usefulness of our methodology to increase our insight into AD.\",\n",
              "  'authors': ['Gómez, Carlos',\n",
              "   'Hornero, Roberto',\n",
              "   'Abásolo, Daniel',\n",
              "   'Fernández, Alberto',\n",
              "   'Escudero, Javier'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Annals of biomedical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Alzheimer Disease',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Magnetoencephalography',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics'],\n",
              "  'pmid': '19130227',\n",
              "  'title': \"Analysis of MEG background activity in Alzheimer's disease using nonlinear methods and ANFIS.\",\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'A new intrinsically compliant robotic orthosis powered by pneumatic muscle actuators (PMA) was developed for treadmill training of neurologically impaired subjects. The robotic orthosis has hip and knee sagittal plane rotations actuated by antagonistic configuration of PMA. The orthosis has passive mechanisms to allow vertical and lateral translations of the trunk and a passive hip abduction/adduction joint. A foot lifter having a passive spring mechanism was used to ensure sufficient foot clearance during swing phase. A trajectory tracking controller was implemented to evaluate the performance of the robotic orthosis on a healthy subject. The results show that the robotic orthosis is able to perform the treadmill training task by providing sufficient torques to achieve physiological gait patterns and a realistic stepping experience. The orthosis is a new addition to the rapidly advancing field of robotic orthoses for treadmill training.',\n",
              "  'authors': ['Hussain, Shahid',\n",
              "   'Xie, Sheng Quan',\n",
              "   'Jamwal, Prashant K',\n",
              "   'Parsons, John'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Elasticity',\n",
              "   'Electrical Equipment and Supplies',\n",
              "   'Equipment Design',\n",
              "   'Feasibility Studies',\n",
              "   'Feedback',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Mechanical Phenomena',\n",
              "   'Muscle, Skeletal',\n",
              "   'Nervous System Diseases',\n",
              "   'Orthotic Devices',\n",
              "   'Rehabilitation',\n",
              "   'Robotics'],\n",
              "  'pmid': '22421099',\n",
              "  'title': 'An intrinsically compliant robotic orthosis for treadmill training.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1433-9285',\n",
              "  'IF_2013': '2.575',\n",
              "  'IF_2014': '2.537',\n",
              "  'IF_2015': '2.513',\n",
              "  'IF_2016': '2.922',\n",
              "  'IF_2017': '2.918',\n",
              "  'ISSN': '0933-7954',\n",
              "  'ISSN_online': '1433-9285',\n",
              "  'ISSN_print': '0933-7954',\n",
              "  'IsoAbbr': 'Soc Psychiatry Psychiatr Epidemiol',\n",
              "  'JrId': 7560,\n",
              "  'MedAbbr': 'Soc Psychiatry Psychiatr Epidemiol',\n",
              "  'NLMid': '8804358',\n",
              "  'Titre': 'Soc Psychiatry Psychiatr Epidemiol',\n",
              "  'abstract': 'This paper discusses a new method for locating errors in diagnostic computer scoring programs for structured clinical interviews. It was proposed as a test of the accuracy of the scoring program for the Composite International Diagnostic Interview, version 1.1. The proposal was to create an independent scoring program in a different computer language but serving the same criteria. Both programs were then applied to the same large set of valid (i.e., logically consistent) computer-generated test cases, and differences in diagnostic assignments reviewed. The method described can identify the program steps that account for the sources of the errors. Corrections can be made and the programs run again on new sets of test cases until discrepancy-free results are achieved. While this method cannot discover errors that are repeated in the two programs, it does discover more of the errors in a scoring program than we have previously been able to identify. This technique provides a systematic and rigorous approach to assuring the accuracy of scoring programs based on established algorithms.',\n",
              "  'authors': ['Marcus, S C', 'Robins, L N'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Social psychiatry and psychiatric epidemiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnostic Errors',\n",
              "   'Humans',\n",
              "   'Mental Disorders',\n",
              "   'Software Validation'],\n",
              "  'pmid': '9640093',\n",
              "  'title': 'Detecting errors in a scoring program: a method of double diagnosis using a computer-generated sample.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1523-1755',\n",
              "  'IF_2013': '8.52',\n",
              "  'IF_2014': '8.563',\n",
              "  'IF_2015': '7.683',\n",
              "  'IF_2016': '8.395',\n",
              "  'IF_2017': '8.429',\n",
              "  'ISSN': '0085-2538',\n",
              "  'ISSN_online': '1523-1755',\n",
              "  'ISSN_print': '0085-2538',\n",
              "  'IsoAbbr': 'Kidney Int.',\n",
              "  'JrId': 5428,\n",
              "  'MedAbbr': 'Kidney Int',\n",
              "  'NLMid': '0323470',\n",
              "  'Titre': 'Kidney Int.',\n",
              "  'abstract': 'Lupus nephritis is divided into six classes and scored according to activity and chronicity indices based on histologic findings. Treatment differs based on the pathologic findings. Renal biopsy is currently the only way to accurately predict class and activity and chronicity indices. We propose to use patterns of abundance of urine proteins to identify class and disease indices.|Urine was collected from 20 consecutive patients immediately prior to biopsy for evaluation of lupus nephritis. The International Society of Nephrology/Renal Pathology Society (ISN/RPS) class of lupus nephritis, activity, and chronicity indices were determined by a renal pathologist. Proteins were separated by two-dimensional gel electrophoresis. Artificial neural networks were trained on normalized spot abundance values.|Biopsy specimens were classified in the database according to ISN/RPS class, activity, and chronicity. Nine samples had characteristics of more than one class present. Receiver operating characteristic (ROC) curves of the trained networks demonstrated areas under the curve ranging from 0.85 to 0.95. The sensitivity and specificity for the ISN/RPS classes were class II 100%, 100%; III 86%, 100%; IV 100%, 92%; and V 92%, 50%. Activity and chronicity indices had r values of 0.77 and 0.87, respectively. A list of spots was obtained that provided diagnostic sensitivity to the analysis.|We have identified a list of protein spots that can be used to develop a clinical assay to predict ISN/RPS class and chronicity for patients with lupus nephritis. An assay based on antibodies against these spots could eliminate the need for renal biopsy, allow frequent evaluation of disease status, and begin specific therapy for patients with lupus nephritis.',\n",
              "  'authors': ['Oates, Jim C',\n",
              "   'Varghese, Sanju',\n",
              "   'Bland, Alison M',\n",
              "   'Taylor, Timothy P',\n",
              "   'Self, Sally E',\n",
              "   'Stanislaus, Romesh',\n",
              "   'Almeida, Jonas S',\n",
              "   'Arthur, John M'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Kidney international',\n",
              "  'keywords': ['Adult',\n",
              "   'Biomarkers',\n",
              "   'Diagnostic Techniques, Urological',\n",
              "   'Electrophoresis, Gel, Two-Dimensional',\n",
              "   'Humans',\n",
              "   'Lupus Nephritis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Proteinuria'],\n",
              "  'pmid': '16316334',\n",
              "  'title': 'Prediction of urinary protein markers in lupus nephritis.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'How useful can machine learning be in a quantum laboratory? Here we raise the question of the potential of intelligent machines in the context of scientific research. A major motivation for the present work is the unknown reachability of various entanglement classes in quantum experiments. We investigate this question by using the projective simulation model, a physics-oriented approach to artificial intelligence. In our approach, the projective simulation system is challenged to design complex photonic quantum experiments that produce high-dimensional entangled multiphoton states, which are of high interest in modern quantum experiments. The artificial intelligence system learns to create a variety of entangled states and improves the efficiency of their realization. In the process, the system autonomously (re)discovers experimental techniques which are only now becoming standard in modern quantum optical experiments-a trait which was not explicitly demanded from the system but emerged through the process of learning. Such features highlight the possibility that machines could have a significantly more creative role in future research.',\n",
              "  'authors': ['Melnikov, Alexey A',\n",
              "   'Poulsen Nautrup, Hendrik',\n",
              "   'Krenn, Mario',\n",
              "   'Dunjko, Vedran',\n",
              "   'Tiersch, Markus',\n",
              "   'Zeilinger, Anton',\n",
              "   'Briegel, Hans J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': [''],\n",
              "  'pmid': '29348200',\n",
              "  'title': 'Active learning machine learns to create new quantum experiments.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'To compare multidetector CT scan (MDCT) results with intraoperative findings in the detection of an inferior-pole pedicle crossing the ureteropelvic junction.|Over the 2-year study period, 35 patients receiving laparoscopic pyeloplasty underwent preoperative investigation with a novel MDCT protocol in order to detect crossing vessels. Postprocessing, including maximum intensity projection, volume-rendering technique, and multiplanar reconstruction, was used in addition to standard axial views.|All the arteries found during laparoscopic surgery were detected by MDCT, but one radiologic false-positive was noted at the beginning of the series. Seven veins were not detected with MDCT. In the only case featuring an isolated inferior-pole vein, the aberrant vessel was identified by MDCT.|Multidetector CT scanning is a highly accurate way of providing all the information necessary preoperatively concerning renal parenchymal anomalies, urinary stones, and collecting system and vessel anatomy. It helps physicians make appropriate therapeutic decisions and gives surgeons information about what they can expect during laparoscopic procedures.',\n",
              "  'authors': ['Sauer, Benoit',\n",
              "   'Flocquet, Muriel',\n",
              "   'Batch, Toufik',\n",
              "   'Blum, Alain',\n",
              "   'Hubert, Jacques'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Contrast Media',\n",
              "   'Humans',\n",
              "   'Kidney Pelvis',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Renal Artery',\n",
              "   'Renal Veins',\n",
              "   'Robotics',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Triiodobenzoic Acids',\n",
              "   'Ureter',\n",
              "   'Ureteral Obstruction'],\n",
              "  'pmid': '18315471',\n",
              "  'title': 'Vascular renal anatomy and the ureteropelvic junction: preoperative multidetector CT scanning with split-bolus injection as a predictor of laparoscopic findings.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1879-5447',\n",
              "  'IF_2013': '0.848',\n",
              "  'IF_2014': '0.882',\n",
              "  'IF_2015': '0.676',\n",
              "  'IF_2016': '1.171',\n",
              "  'IF_2017': '1.032',\n",
              "  'ISSN': '0158-9938',\n",
              "  'ISSN_online': '1879-5447',\n",
              "  'ISSN_print': '0158-9938',\n",
              "  'IsoAbbr': 'Australas Phys Eng Sci Med',\n",
              "  'JrId': 930,\n",
              "  'MedAbbr': 'Australas Phys Eng Sci Med',\n",
              "  'NLMid': '8208130',\n",
              "  'Titre': 'Australas Phys Eng Sci Med',\n",
              "  'abstract': 'The Electrocardiogram (ECG) is one of the most commonly known biological signals. Traditionally ECG recordings are analysed in the time-domain by skilled physicians. However, pathological conditions may not always be obvious in the original time-domain signal. Fourier analysis provides frequency information but has the disadvantage that time characteristics will be lost. Wavelet analysis, which provides both time and frequency information, can overcome this limitation. Here a new method, the combination of wavelet analysis and feature vectors, is applied with the intent to investigate its suitability as a diagnostic tool. ECG signals with normal and abnormal beats were examined. There were two stages in analysing ECG signals: feature extraction and feature classification. To extract features from ECG signals, wavelet decomposition was first applied and feature vectors of normalised energy and entropy were constructed. These feature vectors were used to classify signals. The results showed that normal beats and abnormal beats composed different clusters in most cases. In conclusion, the combination of wavelet transform and feature vectors has shown potential in detecting abnormalities in an ECG recording. It was also found that normalised energy and entropy are features, which are suitable for classification of ECG signals.',\n",
              "  'authors': ['Matsuyama, A', 'Jonkman, M'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Australasian physical & engineering sciences in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Arrhythmias, Cardiac',\n",
              "   'Artificial Intelligence',\n",
              "   'Cluster Analysis',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electrocardiography',\n",
              "   'Humans',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '16623217',\n",
              "  'title': 'The application of wavelet and feature vectors to ECG signals.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Large yellow croaker (Larimichthys crocea) is an important economic fish in China and Eastern Asia. Because of the exhaustive fishing and overdense aquaculture, the wild population and the mariculture of the species are facing serious challenges on germplasm degeneration and susceptibility to infectious disease agents. However, a comprehensive transcriptome from multi-tissues of the species has not been reported and functional molecular markers have not yet been detected and analyzed. In this work, we applied RNA-seq with the Illumina Hiseq2000 platform for a multi-tissue sample of large yellow croaker and assembled the transcriptome into 88,103 transcripts. Of them, 52,782 transcripts have been successfully annotated by nt/nr, InterPro, GO and KEGG database. Comparing with public fish proteins, we have found that 34,576 protein coding transcripts are shared in large yellow croaker with zebrafish, medaka, pufferfish, and stickleback. For functional markers, we have discovered 1,276 polymorphic SSRs and 261, 000 SNPs. The functional impact analysis of SNPs showed that the majority (~75%) of small variants cause synonymous mutations in proteins, followed by variations in 3' UTR region. The functional enrichment analysis illuminated that transcripts involved in DNA bindings, enzyme activities, and signal pathways prominently exhibit less single-nucleotide variants but genes for the constituent of the muscular tissue, the cytoskeleton, and the immunity system contain more frequent SNP mutations, which may reflect the structural and functional selections of the translated proteins. This is the first work for the high-throughput detection and analysis of functional polymorphic SSR and SNP markers in a comprehensive transcriptome of large yellow croaker. Our study provides valuable transcript sequence and functional marker resources for the quantitative trait locus (QTL) identification and molecular selection of the species in the research community. \",\n",
              "  'authors': ['Xiao, Shijun',\n",
              "   'Han, Zhaofang',\n",
              "   'Wang, Panpan',\n",
              "   'Han, Fang',\n",
              "   'Liu, Yang',\n",
              "   'Li, Jiongtang',\n",
              "   'Wang, Zhi Yong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Fish Proteins',\n",
              "   'Fishes',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Genetic Markers',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'INDEL Mutation',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Open Reading Frames',\n",
              "   'Perciformes',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Species Specificity',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '25909910',\n",
              "  'title': 'Functional marker detection and analysis on a comprehensive transcriptome of large yellow croaker by next generation sequencing.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZE'],\n",
              "  'ESSN': '1432-8798',\n",
              "  'IF_2013': '2.282',\n",
              "  'IF_2014': '2.39',\n",
              "  'IF_2015': '2.255',\n",
              "  'IF_2016': '2.058',\n",
              "  'IF_2017': '2.16',\n",
              "  'ISSN': '0304-8608',\n",
              "  'ISSN_online': '1432-8798',\n",
              "  'ISSN_print': '0304-8608',\n",
              "  'IsoAbbr': 'Arch. Virol.',\n",
              "  'JrId': 852,\n",
              "  'MedAbbr': 'Arch Virol',\n",
              "  'NLMid': '7506870',\n",
              "  'Titre': 'Arch. Virol.',\n",
              "  'abstract': \"Porcine reproductive and respiratory syndrome virus (PRRSV) is one of the most important pathogens in the swine industry, causing reproductive failure in sows and respiratory disorders in piglets. Glycosylated protein 5 (GP5) is a major envelope protein of the virus. It is essential for virus particle assembly and involved in viral pathogenesis. In the present study, we identified the host cellular proteins that interact with GP5 by performing immunoprecipitation in MARC-145 cells infected by a recombinant PRRSV containing a FLAG-tag insertion in GP5. In total, 122 cellular proteins were identified by LC-MS/MS. Gene Ontology and KEGG databases were used to map these proteins to different cellular processes, locations and functions. Interestingly, 10.24% of identified cellular proteins were involved in the process of translation. Follow up experiments demonstrated that expression of GP5 in transfected cells led to inhibition of translation of reporter genes. Interaction between GP5 and ATP synthase subunit alpha (ATP5A) was further confirmed by co-immunoprecipitation suggesting a possible role of GP5 in regulation of ATP production in cells. These data contribute to a better understanding of GP5's role in viral pathogenesis and virus-host interactions.\",\n",
              "  'authors': ['Zhang, Maodong', 'Zakhartchouk, Alexander'],\n",
              "  'categories': ['VIROLOGY'],\n",
              "  'journal': 'Archives of virology',\n",
              "  'keywords': ['Adenosine Triphosphate',\n",
              "   'Animals',\n",
              "   'Cell Line, Transformed',\n",
              "   'Epithelial Cells',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Host-Pathogen Interactions',\n",
              "   'Mitochondrial Proton-Translocating ATPases',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Porcine respiratory and reproductive syndrome virus',\n",
              "   'Protein Interaction Mapping',\n",
              "   'Signal Transduction',\n",
              "   'Swine',\n",
              "   'Viral Envelope Proteins'],\n",
              "  'pmid': '29497848',\n",
              "  'title': 'Characterization of the interactome of the porcine reproductive and respiratory syndrome virus glycoprotein-5.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '0219-1024',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '1225-8687',\n",
              "  'ISSN_online': '0219-1024',\n",
              "  'ISSN_print': '1225-8687',\n",
              "  'IsoAbbr': 'J. Biochem. Mol. Biol.',\n",
              "  'JrId': 21591,\n",
              "  'MedAbbr': 'J Biochem Mol Biol',\n",
              "  'NLMid': '9702084',\n",
              "  'Titre': 'J. Biochem. Mol. Biol.',\n",
              "  'abstract': \"Abuse of drugs can elicit compulsive drug seeking behaviors upon repeated administration, and ultimately leads to the phenomenon of addiction. We developed a procedure for the standardization of microarray gene expression data of rat brain in drug addiction and stored them in a single integrated database system, focusing on more effective data processing and interpretation. Another characteristic of the present database is that it has a systematic flexibility for statistical analysis and linking with other databases. Basically, we adopt an intelligent SQL querying system, as the foundation of our DB, in order to set up an interactive module which can automatically read the raw gene expression data in the standardized format. We maximize the usability of this DB, helping users study significant gene expression and identify biological function of the genes through integrated up-to-date gene information such as GO annotation and metabolic pathway. For collecting the latest information of selected gene from the database, we also set up the local BLAST search engine and nonredundant sequence database updated by NCBI server on a daily basis. We find that the present database is a useful query interface and data-mining tool, specifically for finding out the genes related to drug addiction. We apply this system to the identification and characterization of methamphetamine-induced genes' behavior in rat brain.\",\n",
              "  'authors': ['Suh, Young Ju',\n",
              "   'Yang, Moon Hee',\n",
              "   'Yoon, Suk Joon',\n",
              "   'Park, Jong Hoon'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of biochemistry and molecular biology',\n",
              "  'keywords': ['Animals',\n",
              "   'Brain',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Expression Profiling',\n",
              "   'Knowledge Bases',\n",
              "   'National Library of Medicine (U.S.)',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Rats',\n",
              "   'Substance-Related Disorders',\n",
              "   'United States'],\n",
              "  'pmid': '16889689',\n",
              "  'title': 'GEDA: new knowledge base of gene expression in drug addiction.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': \"Continuous blood pressure (BP) monitoring can provide invaluable information about individuals' health conditions. However, BP is conventionally measured using inconvenient cuff-based instruments, which prevents continuous BP monitoring. This paper presents an efficient algorithm, based on the pulse arrival time (PAT), for the continuous and cuffless estimation of the systolic BP, diastolic blood pressure (DBP), and mean arterial pressure (MAP) values.|The proposed framework estimates the BP values through processing vital signals and extracting two types of features, which are based on either physiological parameters or whole-based representation of vital signals. Finally, the regression algorithms are employed for the BP estimation. Although the proposed algorithm works reliably without any need for calibration, an optional calibration procedure is also suggested, which can improve the system's accuracy even further.|The proposed method is evaluated on about a thousand subjects using the Association for the Advancement of Medical Instrumentation (AAMI) and the British Hypertension Society (BHS) standards. The method complies with the AAMI standard in the estimation of DBP and MAP values. Regarding the BHS protocol, the results achieve grade A for the estimation of DBP and grade B for the estimation of MAP.|We conclude that by using the PAT in combination with informative features from the vital signals, the BP can be accurately and reliably estimated in a noninvasive fashion.|The results indicate that the proposed algorithm for the cuffless estimation of the BP can potentially enable mobile health-care gadgets to monitor the BP continuously.\",\n",
              "  'authors': ['Kachuee, Mohammad',\n",
              "   'Kiani, Mohammad Mahdi',\n",
              "   'Mohammadzade, Hoda',\n",
              "   'Shabany, Mahdi'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Blood Pressure',\n",
              "   'Blood Pressure Determination',\n",
              "   'Blood Pressure Monitors',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Monitoring, Ambulatory',\n",
              "   'Pulse Wave Analysis',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '27323356',\n",
              "  'title': 'Cuffless Blood Pressure Estimation Algorithms for Continuous Health-Care Monitoring.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1365-2931',\n",
              "  'IF_2013': '1.18',\n",
              "  'IF_2014': '1.271',\n",
              "  'IF_2015': '1.279',\n",
              "  'IF_2016': '1.418',\n",
              "  'IF_2017': '1.333',\n",
              "  'ISSN': '1364-5706',\n",
              "  'ISSN_online': '1365-2931',\n",
              "  'ISSN_print': '1364-5706',\n",
              "  'IsoAbbr': 'Minim Invasive Ther Allied Technol',\n",
              "  'JrId': 22836,\n",
              "  'MedAbbr': 'Minim Invasive Ther Allied Technol',\n",
              "  'NLMid': '9612996',\n",
              "  'Titre': 'Minim Invasive Ther Allied Technol',\n",
              "  'abstract': 'This paper presents a novel tactile sensing robot designed to detect breast lesions with minimum invasiveness to the tissue while providing exact documentation to make therapeutic and surgical decisions. The robot named \"Robotic Tactile Breast Mass Identifier (Robo-Tac-BMI) consists of an indentation probe controlled by a robotic system and a visualization interface to manipulate the end-effecter. Geometrical maps of the test points with related tension-relaxation curves are provided during clinical examinations. Utilizing the curves, three functional stiffness parameters are extracted locally for each test point. These parameters are employed to provide objective information to facilitate the surgeon\\'s task in the diagnostic procedure. Computational analysis is proposed for a real breast tissue model to study the capability of artificial tactile sensing in the mass detection. Indications of the mass existence are determined and employed as the basis of the Robo-Tac-BMI design and construction. Clinical trials are executed by the Robo-Tac-BMI on 161 cases. The results show that the robot has the potential to provide tissue mechanical properties unlike the conventional screening modalities carried out either by the surgeon or the imaging techniques which are not quantitative and lack documentation. Sonography with and without mammography is chosen as the \"gold standard\" tests.',\n",
              "  'authors': ['Mojra, Afsaneh',\n",
              "   'Najarian, Siamak',\n",
              "   'Towliat Kashani, Seyed Mohsen',\n",
              "   'Panahi, Farzad',\n",
              "   'Tehrani, Morteza Ali'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Minimally invasive therapy & allied technologies : MITAT : official journal of the Society for Minimally Invasive Therapy',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Neoplasms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Finite Element Analysis',\n",
              "   'Humans',\n",
              "   'Robotics'],\n",
              "  'pmid': '21919810',\n",
              "  'title': 'A novel robotic tactile mass detector with application in clinical breast examination.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Digital mammography is one of the most promising options to diagnose breast cancer which is the most common cancer in women. However, its effectiveness is enfeebled due to the difficulty in distinguishing actual cancer lesions from benign abnormalities, which results in unnecessary biopsy referrals. To overcome this issue, computer aided diagnosis (CADx) using machine learning techniques have been studied worldwide. Since this is a classification problem and the number of features obtainable from a mammogram image is infinite, a feature selection method that is tailored for use in the CADx systems is needed.|We propose a feature selection method based on multiple support vector machine recursive feature elimination (MSVM-RFE). We compared our method with four previously proposed feature selection methods which use support vector machine as the base classifier. Experiments were performed on lesions extracted from the Digital Database of Screening Mammography, the largest public digital mammography database available. We measured average accuracy over 5-fold cross validation on the 8 datasets we extracted.|Selecting from 8 features, conventional algorithms like SVM-RFE and multiple SVM-RFE showed slightly better performance than others. However, when selecting from 22 features, our proposed modified multiple SVM-RFE using boosting outperformed or was at least competitive to all others.|Our modified method may be a possible alternative to SVM-RFE or the original MSVM-RFE in many cases of interest. In the future, we need a specific method to effectively combine models trained during the feature selection process and a way to combine feature subsets generated from individual SVM-RFE instances.',\n",
              "  'authors': ['Yoon, Sejong', 'Kim, Saejoon'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Mammography',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Radiographic Image Enhancement'],\n",
              "  'pmid': '19891795',\n",
              "  'title': 'AdaBoost-based multiple SVM-RFE for classification of mammograms in DDSM.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'Higher organisms can establish complex associations between sensory events and motor responses. More remarkable than their complexity, however, is that the resulting sensory-motor maps can be selectively interchanged. For example, a person who speaks English and Spanish can read aloud \"con once, sin once,\" going effortlessly from one language to the other. What is the neural basis of this capacity? Here, a network model is presented in which multiple maps between sensory stimuli and motor actions are possible, but only one of them, depending on behavioral context, is implemented at any given time. The key is a nonlinear representation in which the gain of sensory responses is regulated by context information. Neuronal responses can indeed show variations in gain, as has been documented in the case of proprioceptive signals such as eye and head position, which can modulate visually triggered activity. However, in contrast to these, the contextual cues used here need not bear any relationship to the physical attributes of the stimuli; in particular, spatial location is irrelevant. The model thus postulates the existence of sensory neurons that are nonlinearly modulated by arbitrary context signals, a plausible and testable prediction. The proposed mechanism allows a network of neurons to effectively change the functional connectivity between its inputs and outputs and may partially explain how animals can quickly adapt their behavior to varying environmental conditions.',\n",
              "  'authors': ['Salinas, Emilio'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Cues',\n",
              "   'Eye Movements',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Motor Neurons',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons, Afferent',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Psychomotor Performance'],\n",
              "  'pmid': '14762129',\n",
              "  'title': 'Fast remapping of sensory stimuli onto motor actions on the basis of contextual modulation.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1873-5894',\n",
              "  'IF_2013': '2.022',\n",
              "  'IF_2014': '2.09',\n",
              "  'IF_2015': '1.98',\n",
              "  'IF_2016': '2.225',\n",
              "  'IF_2017': '2.564',\n",
              "  'ISSN': '0730-725X',\n",
              "  'ISSN_online': '1873-5894',\n",
              "  'ISSN_print': '0730-725X',\n",
              "  'IsoAbbr': 'Magn Reson Imaging',\n",
              "  'JrId': 5670,\n",
              "  'MedAbbr': 'Magn Reson Imaging',\n",
              "  'NLMid': '8214883',\n",
              "  'Titre': 'Magn Reson Imaging',\n",
              "  'abstract': \"Diffusion-weighted magnetic resonance imaging (DW-MRI) is of critical importance for characterizing in-vivo white matter. Models relating microarchitecture to observed DW-MRI signals as a function of diffusion sensitization are the lens through which DW-MRI data are interpreted. Numerous modern approaches offer opportunities to assess more complex intra-voxel structures. Nevertheless, there remains a substantial gap between intra-voxel estimated structures and ground truth captured by 3-D histology.|Herein, we propose a novel data-driven approach to model the non-linear mapping between observed DW-MRI signals and ground truth structures using a sequential deep neural network regression using residual block deep neural network (ResDNN). Training was performed on two 3-D histology datasets of squirrel monkey brains and validated on a third. A second validation was performed using scan-rescan datasets of 12 subjects from Human Connectome Project. The ResDNN was compared with multiple micro-structure reconstruction methods and super resolved-constrained spherical deconvolution (sCSD) in particular as baseline for both the validations.|Angular correlation coefficient (ACC) is a correlation/similarity measure and can be interpreted as accuracy when compared with a ground truth. The median ACC of ResDNN is 0.82 and median ACC's of different variants of CSD are 0.75, 0.77, 0.79. The mean, median and std. of ResDNN & sCSD ACC across 12 subjects from HCP are 0.74, 0.88, 0.31 and 0.61, 0.71, 0.31 respectively.|This work highlights the ability of deep learning to capture linkages between ex-vivo ground truth data with feasible MRI sequences. The data-driven approach is applicable to human in-vivo data and results in intriguingly high reproducibility of orientation structure.\",\n",
              "  'authors': ['Nath, Vishwesh',\n",
              "   'Schilling, Kurt G',\n",
              "   'Parvathaneni, Prasanna',\n",
              "   'Hansen, Colin B',\n",
              "   'Hainline, Allison E',\n",
              "   'Huo, Yuankai',\n",
              "   'Blaber, Justin A',\n",
              "   'Lyu, Ilwoo',\n",
              "   'Janve, Vaibhav',\n",
              "   'Gao, Yurui',\n",
              "   'Stepniewska, Iwona',\n",
              "   'Anderson, Adam W',\n",
              "   'Landman, Bennett A'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance imaging',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31323317',\n",
              "  'title': 'Deep learning reveals untapped information for local white-matter fiber reconstruction in diffusion-weighted MRI.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '2219-2840',\n",
              "  'IF_2013': '2.433',\n",
              "  'IF_2014': '2.369',\n",
              "  'IF_2015': '2.787',\n",
              "  'IF_2016': '3.365',\n",
              "  'IF_2017': '3.3',\n",
              "  'ISSN': '1007-9327',\n",
              "  'ISSN_online': '2219-2840',\n",
              "  'ISSN_print': '1007-9327',\n",
              "  'IsoAbbr': 'World J. Gastroenterol.',\n",
              "  'JrId': 22315,\n",
              "  'MedAbbr': 'World J Gastroenterol',\n",
              "  'NLMid': '100883448',\n",
              "  'Titre': 'World J. Gastroenterol.',\n",
              "  'abstract': 'The introduction of laparoscopy in the surgeon\\'s armamentarium was in fact a \"revolution in the history of surgery\". Since this technique involves insufflation of carbon dioxide it produces several pathophysiological changes which have to be understood by the anaesthesiologist who can modify the anaesthesia technique accordingly. Advantages of laparoscopy include reduced pain, small scars and early return to work. Certain complications specific to laparoscopic surgery are due to carboperitoneum and increased intra-abdominal pressure. Venous air embolism, although very rare, can be lethal if not managed promptly. Other complications include subcutaneous emphysema, haemodynamic compromise and arrhythmias. Although associated with minimal postoperative morbidity, postoperative pain, nausea and vomiting can be quite problematic. The limitations of laparoscopy have been overcome by the introduction of robotic surgery. There are important implications for the anaesthesiologist during robotic surgeries which have to be practiced accordingly. Robotic surgery has a learning curve for both the surgeon and the anaesthesiologist. The robot is bulky, and cannot be disengaged after docking. Therefore it is important that the anaesthetized patient remains immobile throughout surgery and anaesthesia is reversed only after the robot has been disengaged at the end of surgery. Advances in laparoscopy and robotic surgery have modified anaesthetic techniques too. ',\n",
              "  'authors': ['Sood, Jayashree'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'World journal of gastroenterology',\n",
              "  'keywords': ['Anesthesiology',\n",
              "   'Carbon Dioxide',\n",
              "   'Clinical Competence',\n",
              "   'Diffusion of Innovation',\n",
              "   'Humans',\n",
              "   'Insufflation',\n",
              "   'Laparoscopy',\n",
              "   'Learning Curve',\n",
              "   'Monitoring, Intraoperative',\n",
              "   'Patient Positioning',\n",
              "   'Patient Safety',\n",
              "   'Postoperative Complications',\n",
              "   'Preoperative Care',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '25339818',\n",
              "  'title': 'Advancing frontiers in anaesthesiology with laparoscopy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1662-453X',\n",
              "  'IF_2013': '3.656',\n",
              "  'IF_2014': '3.656',\n",
              "  'IF_2015': '3.398',\n",
              "  'IF_2016': '3.566',\n",
              "  'IF_2017': '3.877',\n",
              "  'ISSN': '1662-4548',\n",
              "  'ISSN_online': '1662-453X',\n",
              "  'ISSN_print': '1662-4548',\n",
              "  'IsoAbbr': 'Front Neurosci',\n",
              "  'JrId': 34771,\n",
              "  'MedAbbr': 'Front Neurosci',\n",
              "  'NLMid': '101478481',\n",
              "  'Titre': 'Front Neurosci',\n",
              "  'abstract': \"The whole-brain functional connectivity (FC) pattern obtained from resting-state functional magnetic resonance imaging data are commonly applied to study neuropsychiatric conditions such as autism spectrum disorder (ASD) by using different machine learning models. Recent studies indicate that both hyper- and hypo- aberrant ASD-associated FCs were widely distributed throughout the entire brain rather than only in some specific brain regions. Deep neural networks (DNN) with multiple hidden layers have shown the ability to systematically extract lower-to-higher level information from high dimensional data across a series of neural hidden layers, significantly improving classification accuracy for such data. In this study, a DNN with a novel feature selection method (DNN-FS) is developed for the high dimensional whole-brain resting-state FC pattern classification of ASD patients vs. typical development (TD) controls. The feature selection method is able to help the DNN generate low dimensional high-quality representations of the whole-brain FC patterns by selecting features with high discriminating power from multiple trained sparse auto-encoders. For the comparison, a DNN without the feature selection method (DNN-woFS) is developed, and both of them are tested with different architectures (i.e., with different numbers of hidden layers/nodes). Results show that the best classification accuracy of 86.36% is generated by the DNN-FS approach with 3 hidden layers and 150 hidden nodes (3/150). Remarkably, DNN-FS outperforms DNN-woFS for all architectures studied. The most significant accuracy improvement was 9.09% with the 3/150 architecture. The method also outperforms other feature selection methods, e.g., two sample t-test and elastic net. In addition to improving the classification accuracy, a Fisher's score-based biomarker identification method based on the DNN is also developed, and used to identify 32 FCs related to ASD. These FCs come from or cross different pre-defined brain networks including the default-mode, cingulo-opercular, frontal-parietal, and cerebellum. Thirteen of them are statically significant between ASD and TD groups (two sample t-test p < 0.05) while 19 of them are not. The relationship between the statically significant FCs and the corresponding ASD behavior symptoms is discussed based on the literature and clinician's expert knowledge. Meanwhile, the potential reason of obtaining 19 FCs which are not statistically significant is also provided.\",\n",
              "  'authors': ['Guo, Xinyu',\n",
              "   'Dominick, Kelli C',\n",
              "   'Minai, Ali A',\n",
              "   'Li, Hailong',\n",
              "   'Erickson, Craig A',\n",
              "   'Lu, Long J'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Frontiers in neuroscience',\n",
              "  'keywords': [''],\n",
              "  'pmid': '28871217',\n",
              "  'title': 'Diagnosing Autism Spectrum Disorder from Brain Resting-State Functional Connectivity Patterns Using a Deep Neural Network with a Novel Feature Selection Method.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '2073-4425',\n",
              "  'IF_2013': '1.151',\n",
              "  'IF_2014': '1.151',\n",
              "  'IF_2015': '3.242',\n",
              "  'IF_2016': '3.6',\n",
              "  'IF_2017': '3.191',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2073-4425',\n",
              "  'IsoAbbr': 'Genes (Basel)',\n",
              "  'JrId': 39257,\n",
              "  'MedAbbr': 'Genes (Basel)',\n",
              "  'NLMid': '101551097',\n",
              "  'Titre': 'Genes (Basel)',\n",
              "  'abstract': 'Finding non-coding RNA (ncRNA) genes has emerged over the past few years as a cutting-edge trend in bioinformatics. There are numerous computational intelligence (CI) challenges in the annotation and interpretation of ncRNAs because it requires a domain-related expert knowledge in CI techniques. Moreover, there are many classes predicted yet not experimentally verified by researchers. Recently, researchers have applied many CI methods to predict the classes of ncRNAs. However, the diverse CI approaches lack a definitive classification framework to take advantage of past studies. A few review papers have attempted to summarize CI approaches, but focused on the particular methodological viewpoints. Accordingly, in this article, we summarize in greater detail than previously available, the CI techniques for finding ncRNAs genes. We differentiate from the existing bodies of research and discuss concisely the technical merits of various techniques. Lastly, we review the limitations of ncRNA gene-finding CI methods with a point-of-view towards the development of new computational tools.',\n",
              "  'authors': ['Abbas, Qaisar',\n",
              "   'Raza, Syed Mansoor',\n",
              "   'Biyabani, Azizuddin Ahmed',\n",
              "   'Jaffar, Muhammad Arfan'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Genes',\n",
              "  'keywords': [''],\n",
              "  'pmid': '27918472',\n",
              "  'title': 'A Review of Computational Methods for Finding Non-Coding RNA Genes.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '2154-4212',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '1.093',\n",
              "  'IF_2015': '1.331',\n",
              "  'IF_2016': '1.325',\n",
              "  'IF_2017': '1.392',\n",
              "  'ISSN': '2151-8378',\n",
              "  'ISSN_online': '2154-4212',\n",
              "  'ISSN_print': '2151-8378',\n",
              "  'IsoAbbr': 'Female Pelvic Med Reconstr Surg',\n",
              "  'JrId': 38508,\n",
              "  'MedAbbr': 'Female Pelvic Med Reconstr Surg',\n",
              "  'NLMid': '101528690',\n",
              "  'Titre': 'Female Pelvic Med Reconstr Surg',\n",
              "  'abstract': 'We sought to characterize differences between African American women and white women in abdominal wall dimensions that could affect robotic port placement. By better understanding these differences, surgeons could assess and adjust port placement to accommodate varying abdominal wall anatomy.|A radiologist blinded to race-reviewed abdominal/pelvic computed tomographic scans of women aged 30 to 70 prescreened for demographic inclusion criteria. These consecutive scans were screened for radiologic exclusion criteria until 40 consecutive scans from each race were identified and included.|Eighty of 663 patients, 40 of each race, met demographic inclusion criteria. The most common radiologic feature disqualifying the scans included absence of the xiphoid process on the scan and anterior abdominal wall deformity. Demographic variables including age, weight, height, and body mass index were similar between groups. Symphysis pubis to umbilicus measurement was shorter in the African American group (15.7 [2.1] vs 17.1 [2.0]; P < 0.001) and intra-anterior superior iliac spine distance was narrower (21.4 [1.2] vs 23.8 [2.0]; P = 0.003), creating an overall smaller lower abdomen in African American women. Total abdominal length was the same between groups (36.6 [2.6] vs 36.7 [2.8]; P = 0.851). Using linear regression, height, weight, and body mass index did not affect lower abdominal dimensions, whereas age (P < 0.001) had a significant inverse relationship with the symphysis pubis to umbilicus measurement.|Lower abdominal dimensions between races vary, with the umbilicus serving as an inconsistent landmark. Variance exists that can be attributed to racial differences. Assessment of these dimensions at the time of robotic surgery could lead to improved port spacing and therefore fewer arm collisions, improving robotic efficiency.',\n",
              "  'authors': ['Parnell, Brent A',\n",
              "   'Midia, Esin C',\n",
              "   'Fielding, Julia R',\n",
              "   'Robinson, Barbara L',\n",
              "   'Matthews, Catherine A'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Female pelvic medicine & reconstructive surgery',\n",
              "  'keywords': ['Abdominal Wall',\n",
              "   'African Americans',\n",
              "   'European Continental Ancestry Group',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Surgical Procedures, Operative'],\n",
              "  'pmid': '23611935',\n",
              "  'title': 'Relationship between race and abdominal anatomy: effect on robotic port placement.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.197',\n",
              "  'IF_2014': '1.56',\n",
              "  'IF_2015': '1.76',\n",
              "  'IF_2016': '1.751',\n",
              "  'IF_2017': '1.666',\n",
              "  'ISSN': '0959-3330',\n",
              "  'ISSN_online': '1479-487X',\n",
              "  'ISSN_print': '0959-3330',\n",
              "  'IsoAbbr': 'Environ Technol',\n",
              "  'JrId': 22034,\n",
              "  'MedAbbr': 'Environ Technol',\n",
              "  'NLMid': '9884939',\n",
              "  'Titre': 'Environ Technol',\n",
              "  'abstract': \"The activated sludge process is commonly used for secondary wastewater treatment worldwide. This process is capable of achieving high quality effluent. However it has the reputation of being difficult to operate because of its poorly understood biological behaviour, variability of input flows and the need to incorporate qualitative data. To augment this incomplete knowledge with experience, knowledge-based systems were introduced in the 1980s however they didn't receive much popularity. This paper presents the Activated Sludge Expert system (ASExpert), which is a rule-based expert system plus a complete database tool proposed for use in activated sludge plants. The paper focuses on presenting the system's main features and capabilities to revive the interest in knowledge-based systems as a reliable means for monitoring plants. Then it presents the methodology adopted for ASExpert validation along with an assessment of testing results. Finally it concludes that expert systems technology has proved its importance for enhancing performance, especially if in the future it is integrated to a modern control system.\",\n",
              "  'authors': ['Sorour, M T',\n",
              "   'Bahgat, L M F',\n",
              "   'El, Iskandarani M A',\n",
              "   'Horan, N J'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental technology',\n",
              "  'keywords': ['Bioreactors', 'Expert Systems', 'Humans', 'Sewage'],\n",
              "  'pmid': '12211453',\n",
              "  'title': 'ASExpert: an integrated knowledge-based system for activated sludge plants.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Although reinforcement learning (RL) is suitable for highly uncertain systems, the applicability of this class of algorithms to medical treatment may be limited by the patient variability which dictates individualised tuning for their usually multiple algorithmic parameters. This study explores the feasibility of RL in the framework of artificial pancreas development for type 1 diabetes (T1D). In this approach, an Actor-Critic (AC) learning algorithm is designed and developed for the optimisation of insulin infusion for personalised glucose regulation. AC optimises the daily basal insulin rate and insulin:carbohydrate ratio for each patient, on the basis of his/her measured glucose profile. Automatic, personalised tuning of AC is based on the estimation of information transfer (IT) from insulin to glucose signals. Insulin-to-glucose IT is linked to patient-specific characteristics related to total daily insulin needs and insulin sensitivity (SI). The AC algorithm is evaluated using an FDA-accepted T1D simulator on a large patient database under a complex meal protocol, meal uncertainty and diurnal SI variation. The results showed that 95.66% of time was spent in normoglycaemia in the presence of meal uncertainty and 93.02% when meal uncertainty and SI variation were simultaneously considered. The time spent in hypoglycaemia was 0.27% in both cases. The novel tuning method reduced the risk of severe hypoglycaemia, especially in patients with low SI. ',\n",
              "  'authors': ['Daskalaki, Elena', 'Diem, Peter', 'Mougiakakou, Stavroula G'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Algorithms',\n",
              "   'Biomedical Research',\n",
              "   'Blood Glucose',\n",
              "   'Child',\n",
              "   'Cohort Studies',\n",
              "   'Computer Simulation',\n",
              "   'Diabetes Mellitus, Type 1',\n",
              "   'Feasibility Studies',\n",
              "   'Humans',\n",
              "   'Hyperglycemia',\n",
              "   'Insulin',\n",
              "   'Machine Learning',\n",
              "   'Time Factors'],\n",
              "  'pmid': '27441367',\n",
              "  'title': 'Model-Free Machine Learning in Biomedicine: Feasibility Study in Type 1 Diabetes.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1471-2962',\n",
              "  'IF_2013': '2.864',\n",
              "  'IF_2014': '2.147',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.97',\n",
              "  'IF_2017': '2.746',\n",
              "  'ISSN': '1364-503X',\n",
              "  'ISSN_online': '1471-2962',\n",
              "  'ISSN_print': '1364-503X',\n",
              "  'IsoAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'JrId': 30111,\n",
              "  'MedAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'NLMid': '101133385',\n",
              "  'Titre': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'abstract': 'Many species of animals and plants are supplied with diverse attachment devices, in which morphology depends on the species biology and the particular function in which the attachment device is involved. Many functional solutions have evolved independently in different lineages of animals and plants. Since the diversity of such biological structures is huge, there is a need for their classification. This paper, based on the original and literature data, proposes ordering of biological attachment systems according to several principles: (i) fundamental physical mechanism, according to which the system operates, (ii) biological function of the attachment device, and (iii) duration of the contact. Finally, we show a biomimetic potential of studies on biological attachment devices.',\n",
              "  'authors': ['Gorb, Stanislav N'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Philosophical transactions. Series A, Mathematical, physical, and engineering sciences',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Biology',\n",
              "   'Biomedical Engineering',\n",
              "   'Biomimetic Materials',\n",
              "   'Biomimetics',\n",
              "   'Equipment Design',\n",
              "   'Materials Testing',\n",
              "   'Nanotechnology',\n",
              "   'Parasites',\n",
              "   'Predatory Behavior',\n",
              "   'Robotics',\n",
              "   'Tissue Adhesions',\n",
              "   'Tissue Engineering',\n",
              "   'Weight-Bearing'],\n",
              "  'pmid': '18192171',\n",
              "  'title': \"Biological attachment devices: exploring nature's diversity for biomimetics.\",\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1879-1298',\n",
              "  'IF_2013': '3.499',\n",
              "  'IF_2014': '3.34',\n",
              "  'IF_2015': '3.698',\n",
              "  'IF_2016': '4.208',\n",
              "  'IF_2017': '4.427',\n",
              "  'ISSN': '0045-6535',\n",
              "  'ISSN_online': '1879-1298',\n",
              "  'ISSN_print': '0045-6535',\n",
              "  'IsoAbbr': 'Chemosphere',\n",
              "  'JrId': 8583,\n",
              "  'MedAbbr': 'Chemosphere',\n",
              "  'NLMid': '0320657',\n",
              "  'Titre': 'Chemosphere',\n",
              "  'abstract': 'We aim at predicting the effect of structure and isotopic substitutions on the equilibrium vapour pressure isotope effect of various organic compounds (alcohols, acids, alkanes, alkenes and aromatics) at intermediate temperatures. We attempt to explore quantitative structure property relationships by using artificial neural networks (ANN); the multi-layer perceptron (MLP) and compare the performances of it with multi-linear regression (MLR). These approaches are based on the relationship between the molecular structure (organic chain, polar functions, type of functions, type of isotope involved) of the organic compounds, and their equilibrium vapour pressure. A data set of 130 equilibrium vapour pressure isotope effects was used: 112 were used in the training set and the remaining 18 were used for the test/validation dataset. Two sets of descriptors were tested, a set with all the descriptors: number of(12)C, (13)C, (16)O, (18)O, (1)H, (2)H, OH functions, OD functions, CO functions, Connolly Solvent Accessible Surface Area (CSA) and temperature and a reduced set of descriptors. The dependent variable (the output) is the natural logarithm of the ratios of vapour pressures (ln R), expressed as light/heavy as in classical literature. Since the database is rather small, the leave-one-out procedure was used to validate both models. Considering higher determination coefficients and lower error values, it is concluded that the multi-layer perceptron provided better results compared to multi-linear regression. The stepwise regression procedure is a useful tool to reduce the number of descriptors. To our knowledge, a Quantitative Structure Property Relationship (QSPR) approach for isotopic studies is novel. ',\n",
              "  'authors': ['Parinet, Julien',\n",
              "   'Julien, Maxime',\n",
              "   'Nun, Pierrick',\n",
              "   'Robins, Richard J',\n",
              "   'Remaud, Gerald',\n",
              "   'Höhener, Patrick'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Chemosphere',\n",
              "  'keywords': ['Environmental Monitoring',\n",
              "   'Isotopes',\n",
              "   'Linear Models',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Vapor Pressure'],\n",
              "  'pmid': '25559176',\n",
              "  'title': 'Predicting equilibrium vapour pressure isotope effects by using artificial neural networks or multi-linear regression - A quantitative structure property relationship approach.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Compared to robotic injection of suspended cells (e.g., embryos and oocytes), fewer attempts were made to automate the injection of adherent cells (e.g., cancer cells and cardiomyocytes) due to their smaller size, highly irregular morphology, small thickness (a few micrometers thick), and large variations in thickness across cells. This paper presents a robotic system for automated microinjection of adherent cells. The system is embedded with several new capabilities: automatically locating micropipette tips; robustly detecting the contact of micropipette tip with cell culturing surface and directly with cell membrane; and precisely compensating for accumulative positioning errors. These new capabilities make it practical to perform adherent cell microinjection truly via computer mouse clicking in front of a computer monitor, on hundreds and thousands of cells per experiment (versus a few to tens of cells as state of the art). System operation speed, success rate, and cell viability rate were quantitatively evaluated based on robotic microinjection of over 4000 cells. This paper also reports the use of the new robotic system to perform cell-cell communication studies using large sample sizes. The gap junction function in a cardiac muscle cell line (HL-1 cells), for the first time, was quantified with the system. ',\n",
              "  'authors': ['Liu, Jun',\n",
              "   'Siragam, Vinayakumar',\n",
              "   'Gong, Zheng',\n",
              "   'Chen, Jun',\n",
              "   'Fridman, Michael D',\n",
              "   'Leung, Clement',\n",
              "   'Lu, Zhe',\n",
              "   'Ru, Changhai',\n",
              "   'Xie, Shaorong',\n",
              "   'Luo, Jun',\n",
              "   'Hamilton, Robert M',\n",
              "   'Sun, Yu'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Animals',\n",
              "   'Cell Adhesion',\n",
              "   'Cell Communication',\n",
              "   'Cell Line',\n",
              "   'Cell Separation',\n",
              "   'Cell Transplantation',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Man-Machine Systems',\n",
              "   'Mice',\n",
              "   'Microinjections',\n",
              "   'Micromanipulation',\n",
              "   'Myocytes, Cardiac',\n",
              "   'Robotics'],\n",
              "  'pmid': '25073160',\n",
              "  'title': 'Robotic adherent cell injection for characterizing cell-cell communication.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['WH'],\n",
              "  'ESSN': '1478-6362',\n",
              "  'IF_2013': '4.117',\n",
              "  'IF_2014': '3.753',\n",
              "  'IF_2015': '3.979',\n",
              "  'IF_2016': '4.121',\n",
              "  'IF_2017': '4.269',\n",
              "  'ISSN': '1478-6354',\n",
              "  'ISSN_online': '1478-6362',\n",
              "  'ISSN_print': '1478-6354',\n",
              "  'IsoAbbr': 'Arthritis Res. Ther.',\n",
              "  'JrId': 30289,\n",
              "  'MedAbbr': 'Arthritis Res Ther',\n",
              "  'NLMid': '101154438',\n",
              "  'Titre': 'Arthritis Res. Ther.',\n",
              "  'abstract': 'This study investigates the association between exposure to urate-lowering drugs (ULDs) and progression and recovery from chronic kidney disease (CKD).|We identified 5860 incident gout patients at Chang Gung Memorial Hospital from 2012 to 2015. Propensity score (PS)-weighted Cox proportional hazards model was used to estimate hazard ratios (HRs) for CKD progression and improvement. A separate analysis was conducted to assess the HR for CKD progression and CKD recovery among those with worsening CKD.|The incidence of CKD progression among allopurinol, febuxostat and uricosuric agent users were 1.98, 1.88 and 1.64 per 1000 person-days. Compared with allopurinol users, the PS-weighted HR (95% confidence intervals [CIs]) was 1.77 (0.85-1.76) for febuxostat users and 1.37 (0.71-1.37) for uricosuric agent users for CKD progression and 1.43 (1.26-1.62) for febuxostat users and 1.00 (0.88-1.14) for uricosuric agent users for CKD improvement. Compared to allopurinol users, the HRs for CKD progression were 1.14 (0.80-1.66) for febuxostat users and 0.92 (0.67-1.31) for uricosuric agent users. Among 741 patients who had CKD progression, the incidence of CKD recovery was 1.33, 6.21 and 3.53 per 1000 person-days for allopurinol, febuxostat and uricosuric agent users. The HRs (95% CIs) for recovery in febuxostat and uricosuric agent users were 2.17 (1.40-3.47) and 1.80 (1.20-2.83) compared to allopurinol users.|CKD progression and recovery are common in gout patients using ULDs. Febuxostat and benzbromarone were associated with a similar risk of CKD progression with allopurinol, which has a poorer recovery compared with other ULDs.',\n",
              "  'authors': ['Chung, Ting-Ting',\n",
              "   'Yu, Kuang-Hui',\n",
              "   'Kuo, Chang-Fu',\n",
              "   'Luo, Shue-Fen',\n",
              "   'Chiou, Meng-Jiun',\n",
              "   'Lan, Wen-Ching',\n",
              "   'Chen, Jung-Sheng',\n",
              "   'Tseng, Wen-Yi',\n",
              "   'Hsieh, Ao-Ho',\n",
              "   'Wang, Lian-Chin'],\n",
              "  'categories': ['RHEUMATOLOGY'],\n",
              "  'journal': 'Arthritis research & therapy',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31533805',\n",
              "  'title': 'Impact of urate-lowering drugs on the progression and recovery from chronic kidney disease among gout patients.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1868-7083',\n",
              "  'IF_2013': '6.219',\n",
              "  'IF_2014': '4.543',\n",
              "  'IF_2015': '4.327',\n",
              "  'IF_2016': '4.987',\n",
              "  'IF_2017': '6.091',\n",
              "  'ISSN': '1868-7075',\n",
              "  'ISSN_online': '1868-7083',\n",
              "  'ISSN_print': '1868-7075',\n",
              "  'IsoAbbr': 'Clin Epigenetics',\n",
              "  'JrId': 38673,\n",
              "  'MedAbbr': 'Clin Epigenetics',\n",
              "  'NLMid': '101516977',\n",
              "  'Titre': 'Clin Epigenetics',\n",
              "  'abstract': 'Bipolar disorder (BD) is a severe, familial psychiatric condition. Progress in understanding the aetiology of BD has been hampered by substantial phenotypic and genetic heterogeneity. We sought to mitigate these confounders by studying a multi-generational family multiply affected by BD and major depressive disorder (MDD), who carry an illness-linked haplotype on chromosome 4p. Within a family, aetiological heterogeneity is likely to be reduced, thus conferring greater power to detect illness-related changes. As accumulating evidence suggests that altered DNA methylation confers risk for BD and MDD, we compared genome-wide methylation between (i) affected carriers of the linked haplotype (ALH) and married-in controls (MIs), (ii) well unaffected haplotype carriers (ULH) and MI, (iii) ALH and ULH and (iv) all haplotype carriers (LH) and MI.|Nominally significant differences in DNA methylation were observed in all comparisons, with differences withstanding correction for multiple testing when the ALH or LH group was compared to the MIs. In both comparisons, we observed increased methylation at a locus in FANCI, which was accompanied by increased FANCI expression in the ALH group. FANCI is part of the Fanconi anaemia complementation (FANC) gene family, which are mutated in Fanconi anaemia and participate in DNA repair. Interestingly, several FANC genes have been implicated in psychiatric disorders. Regional analyses of methylation differences identified loci implicated in psychiatric illness by genome-wide association studies, including CACNB2 and the major histocompatibility complex. Gene ontology analysis revealed enrichment for methylation differences in neurologically relevant genes.|Our results highlight altered DNA methylation as a potential mechanism by which the linked haplotype might confer risk for mood disorders. Differences in the phenotypic outcome of haplotype carriers might, in part, arise from additional changes in DNA methylation that converge on neurologically important pathways. Further work is required to investigate the underlying mechanisms and functional consequences of the observed differences in methylation.',\n",
              "  'authors': ['Walker, Rosie May',\n",
              "   'Christoforou, Andrea Nikie',\n",
              "   'McCartney, Daniel L',\n",
              "   'Morris, Stewart W',\n",
              "   'Kennedy, Nicholas A',\n",
              "   'Morten, Peter',\n",
              "   'Anderson, Susan Maguire',\n",
              "   'Torrance, Helen Scott',\n",
              "   'Macdonald, Alix',\n",
              "   'Sussmann, Jessika Elizabeth',\n",
              "   'Whalley, Heather Clare',\n",
              "   'Blackwood, Douglas H R',\n",
              "   'McIntosh, Andrew Mark',\n",
              "   'Porteous, David John',\n",
              "   'Evans, Kathryn Louise'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Clinical epigenetics',\n",
              "  'keywords': ['Bipolar Disorder',\n",
              "   'DNA Methylation',\n",
              "   'Depressive Disorder, Major',\n",
              "   'Family',\n",
              "   'Fanconi Anemia Complementation Group Proteins',\n",
              "   'Female',\n",
              "   'Gene Ontology',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Haplotypes',\n",
              "   'Heterozygote',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Scotland'],\n",
              "  'pmid': '26798408',\n",
              "  'title': 'DNA methylation in a Scottish family multiply affected by bipolar disorder and major depressive disorder.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1536-5964',\n",
              "  'IF_2013': '4.867',\n",
              "  'IF_2014': '5.723',\n",
              "  'IF_2015': '2.133',\n",
              "  'IF_2016': '1.804',\n",
              "  'IF_2017': '2.028',\n",
              "  'ISSN': '0025-7974',\n",
              "  'ISSN_online': '1536-5964',\n",
              "  'ISSN_print': '0025-7974',\n",
              "  'IsoAbbr': 'Medicine (Baltimore)',\n",
              "  'JrId': 5748,\n",
              "  'MedAbbr': 'Medicine (Baltimore)',\n",
              "  'NLMid': '2985248R',\n",
              "  'Titre': 'Medicine (Baltimore)',\n",
              "  'abstract': 'The aim of this study was to investigate the clinical factors affecting the survival prognosis of lung adenocarcinoma, and to establish a predictive model of survival prognosis of lung adenocarcinoma by artificial neural network.Download the cancer genome atlas (TCGA) database for lung adenocarcinoma research data, perform cox regression analysis and descriptive statistics on the obtained clinical data, draw the survival curve by Kaplan-Meier method, select the independent variables that are statistically significant for constructing the artificial neural networks (ANN) model, and establish artificial neural network model.The number of valid cases included in the study was 524, including 280 men and 244 women, with an age range of 33 to 88 years, mean age 66.87 years, and median progression-free survival (PFS) was 37.7 months. The median overall survival time (OS) was 41.1 months. Cox multivariate analysis showed that smoking history, tumor stage, and surgical margin resection status were independently associated with PFS, and tumor stage and surgical margin resection status were independently associated with OS. The accuracy of the established ANN model itself was predicted to be 65.8%. The accuracy of correctly predicting the prognosis of the predicted samples was 75.0%, and the area under the receiver operating characteristic curve was 0.712.The clinical prognostic factors of lung adenocarcinoma include: smoking history, tumor stage, and surgical margin resection status. The established ANN model can be used to predict the prognosis of lung adenocarcinoma.',\n",
              "  'authors': ['Jiang, Na', 'Xu, Xianrong'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Medicine',\n",
              "  'keywords': ['Adenocarcinoma of Lung',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Cigarette Smoking',\n",
              "   'Female',\n",
              "   'Genomic Library',\n",
              "   'Humans',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'Logistic Models',\n",
              "   'Lung Neoplasms',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Staging',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Prognosis',\n",
              "   'ROC Curve',\n",
              "   'Regression Analysis',\n",
              "   'Risk Factors'],\n",
              "  'pmid': '31096483',\n",
              "  'title': 'Exploring the survival prognosis of lung adenocarcinoma based on the cancer genome atlas database using artificial neural network.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1096-7184',\n",
              "  'IF_2013': '8.258',\n",
              "  'IF_2014': '6.767',\n",
              "  'IF_2015': '8.201',\n",
              "  'IF_2016': '8.142',\n",
              "  'IF_2017': '7.674',\n",
              "  'ISSN': '1096-7176',\n",
              "  'ISSN_online': '1096-7184',\n",
              "  'ISSN_print': '1096-7176',\n",
              "  'IsoAbbr': 'Metab. Eng.',\n",
              "  'JrId': 21792,\n",
              "  'MedAbbr': 'Metab Eng',\n",
              "  'NLMid': '9815657',\n",
              "  'Titre': 'Metab. Eng.',\n",
              "  'abstract': 'Mechanistic biochemical network models describe the dynamics of intracellular metabolite pools in terms of substance concentrations, stoichiometry and reaction kinetics. Data from stimulus response experiments are currently the most informative source for in-vivo parameter estimation in such models. However, only a part of the parameters of classical enzyme kinetic models can usually be estimated from typical stimulus response data. For this reason, several alternative kinetic formats using different \"languages\" (e.g. linear, power laws, linlog, generic and convenience) have been proposed to reduce the model complexity. The present contribution takes a rigorous \"multi-lingual\" approach to data evaluation by translating biochemical network models from one kinetic format into another. For this purpose, a new high-performance algorithm has been developed and tested. Starting with a given model, it replaces as many kinetic terms as possible by alternative expressions while still reproducing the experimental data. Application of the algorithm to a published model for Escherichia coli\\'s sugar metabolism demonstrates the power of the new method. It is shown that model translation is a powerful tool to investigate the information content of stimulus response data and the predictive power of models. Moreover, the local and global approximation capabilities of the models are elucidated and some pitfalls of traditional single model approaches to data evaluation are revealed.',\n",
              "  'authors': ['Hadlich, Frieder', 'Noack, Stephan', 'Wiechert, Wolfgang'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Metabolic engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Kinetics',\n",
              "   'Metabolome',\n",
              "   'Models, Biological',\n",
              "   'Multienzyme Complexes',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Signal Transduction'],\n",
              "  'pmid': '19013536',\n",
              "  'title': 'Translating biochemical network models between different kinetic formats.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1874-1754',\n",
              "  'IF_2013': '6.175',\n",
              "  'IF_2014': '4.036',\n",
              "  'IF_2015': '4.638',\n",
              "  'IF_2016': '6.189',\n",
              "  'IF_2017': '4.034',\n",
              "  'ISSN': '0167-5273',\n",
              "  'ISSN_online': '1874-1754',\n",
              "  'ISSN_print': '0167-5273',\n",
              "  'IsoAbbr': 'Int. J. Cardiol.',\n",
              "  'JrId': 4286,\n",
              "  'MedAbbr': 'Int J Cardiol',\n",
              "  'NLMid': '8200291',\n",
              "  'Titre': 'Int. J. Cardiol.',\n",
              "  'abstract': 'Atrial fibrillation (AF) is the most common arrhythmia encountered in clinical practice while it has a significant impact on morbidity and mortality. The errors and pitfalls in the management of AF patients are not uncommon. These include errors in detection and management of the underlying conditions that promote and perpetuate the arrhythmia, in the selection and monitoring of antithrombotic treatment, in the selection of appropriate strategy for arrhythmia management (rate or rhythm control), in the cardioversion procedure, in the prevention of recurrence after cardioversion, in the acute or chronic control of heart rate, and in the monitoring of drug toxicities. The heterogeneity of the disease along with the diversity of current treatment options mainly account for these problems. Nevertheless, deep knowledge of the evidence-based therapeutic approaches, as well as the development of individualized therapeutic strategies, can substantially improve the effective management of such patients.',\n",
              "  'authors': ['Korantzopoulos, Panagiotis',\n",
              "   'Kolettis, Theofilos M',\n",
              "   'Goudevenos, John A',\n",
              "   'Siogas, Konstantinos'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'International journal of cardiology',\n",
              "  'keywords': ['Atrial Fibrillation',\n",
              "   'Evidence-Based Medicine',\n",
              "   'Heart Rate',\n",
              "   'Humans',\n",
              "   'Medical Errors',\n",
              "   'Problem-Based Learning',\n",
              "   'Risk Reduction Behavior',\n",
              "   'Secondary Prevention',\n",
              "   'Thromboembolism'],\n",
              "  'pmid': '16168803',\n",
              "  'title': 'Errors and pitfalls in the non-invasive management of atrial fibrillation.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2047-217X',\n",
              "  'IF_2013': '7.463',\n",
              "  'IF_2014': '7.463',\n",
              "  'IF_2015': '7.463',\n",
              "  'IF_2016': '6.871',\n",
              "  'IF_2017': '7.267',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2047-217X',\n",
              "  'IsoAbbr': 'Gigascience',\n",
              "  'JrId': 40809,\n",
              "  'MedAbbr': 'Gigascience',\n",
              "  'NLMid': '101596872',\n",
              "  'Titre': 'Gigascience',\n",
              "  'abstract': \"The plant bug Lygus hesperus Knight is a polyphagous pest of many economically important crops. Despite its pest status, little is known about the molecular mechanisms responsible for much of the biology of this species. Earlier Lygus transcriptome assemblies were limited by low read depth, or because they focused on specific conditions. To generate a more comprehensive transcriptome, we supplemented previous datasets with new reads corresponding to specific tissues (heads, antennae, and male reproductive tissues). This transcriptome augments current Lygus molecular resources and provides the foundational knowledge critical for future comparative studies.|An expanded, Trinity-based de novo transcriptome assembly for L. hesperus was generated using previously published whole body Illumina data, supplemented with 293 million bp of new raw sequencing data corresponding to five tissue-specific cDNA libraries and 11 Illumina sequencing runs. The updated transcriptome consists of 22,022 transcripts (average length of 2075 nt), 62 % of which contain complete open reading frames. Significant coverage of the BUSCO (benchmarking universal single-copy orthologs) dataset and robust metrics indicate that the transcriptome is a quality assembly with a high degree of completeness. Initial assessment of the new assembly's utility revealed that the length and abundance of transcripts predicted to regulate insect physiology and chemosensation have improved, compared with previous L. hesperus assemblies.|This transcriptome represents a significant expansion of Lygus transcriptome data, and improves foundational knowledge about the molecular mechanisms underlying L. hesperus biology. The dataset is publically available in NCBI and GigaDB as a resource for researchers.\",\n",
              "  'authors': ['Tassone, Erica E',\n",
              "   'Geib, Scott M',\n",
              "   'Hall, Brian',\n",
              "   'Fabrick, Jeffrey A',\n",
              "   'Brent, Colin S',\n",
              "   'Hull, J Joe'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'GigaScience',\n",
              "  'keywords': ['Animals',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Insect',\n",
              "   'Genitalia, Male',\n",
              "   'Heteroptera',\n",
              "   'Host-Parasite Interactions',\n",
              "   'Male',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Organ Specificity',\n",
              "   'Plant Diseases',\n",
              "   'Plants',\n",
              "   'Sequence Analysis, DNA',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '26823975',\n",
              "  'title': 'De novo construction of an expanded transcriptome assembly for the western tarnished plant bug, Lygus hesperus.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1557-9034',\n",
              "  'IF_2013': '1.187',\n",
              "  'IF_2014': '1.335',\n",
              "  'IF_2015': '1.297',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.257',\n",
              "  'ISSN': '1092-6429',\n",
              "  'ISSN_online': '1557-9034',\n",
              "  'ISSN_print': '1092-6429',\n",
              "  'IsoAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'JrId': 20817,\n",
              "  'MedAbbr': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'NLMid': '9706293',\n",
              "  'Titre': 'J Laparoendosc Adv Surg Tech A',\n",
              "  'abstract': 'Iatrogenic ureteral injuries are most commonly seen following pelvic and gynecologic surgery. If early conservative endoscopic management fails, surgical correction may be necessary. In this paper, we report 3 cases of the development of ureterovaginal fistula (UVF) after a total abdominal hysterectomy. All 3 cases underwent a robotic-assisted surgical correction.|Three female patients (37, 50, and 66 years) were diagnosed with a left-sided ureterovaginal fistula following an abdominal hysterectomy. All patients reported urinary leakage per vagina and were diagnosed on excretory urography. All patients underwent robotic repair of ureterovaginal fistula along with ureteral reimplantation.|In all patients, a standard six-port approach was used. Robotic endowrist movement and three-dimensional views aided significantly in the dissection of the viable ureter from periureteral fibrosis and scar. Prior to ureterovesical anastamosis, a double J pigtail stent was inserted intracorporeally through the left 5-mm assistant port. In all cases, the robotic ureteroneocystostomy was completed without complication. Total console surgical time ranged from 62 to 118 minutes. The Foley catheter was removed 24-48 hours after surgery. All patients were discharged completely dry with an indwelling stent in situ.|We report a case series of pure robotic ureteroneocystostomy for repair of complex UVF. The da Vinci robotic surgical system provides a tremendous advantage for the gross identification of viable structures within dense scar tissue as well as the identification of healthy ureter for reimplantation. We strongly recommend an early repair of these fistulae with robotic assistance to reduce the period of morbidity.',\n",
              "  'authors': ['Laungani, Rajesh',\n",
              "   'Patil, Nilesh',\n",
              "   'Krane, Louis S',\n",
              "   'Hemal, Ashok K',\n",
              "   'Raja, Sahabuddin',\n",
              "   'Bhandari, Mahendra',\n",
              "   'Menon, Mani'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of laparoendoscopic & advanced surgical techniques. Part A',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Iatrogenic Disease',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Ureteral Diseases',\n",
              "   'Vaginal Fistula'],\n",
              "  'pmid': '18699746',\n",
              "  'title': 'Robotic-assisted ureterovaginal fistula repair: report of efficacy and feasiblity.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-6882',\n",
              "  'IF_2013': '6.765',\n",
              "  'IF_2014': '6.628',\n",
              "  'IF_2015': '6.373',\n",
              "  'IF_2016': '6.133',\n",
              "  'IF_2017': '6.541',\n",
              "  'ISSN': '0959-4388',\n",
              "  'ISSN_online': '1873-6882',\n",
              "  'ISSN_print': '0959-4388',\n",
              "  'IsoAbbr': 'Curr. Opin. Neurobiol.',\n",
              "  'JrId': 2201,\n",
              "  'MedAbbr': 'Curr Opin Neurobiol',\n",
              "  'NLMid': '9111376',\n",
              "  'Titre': 'Curr. Opin. Neurobiol.',\n",
              "  'abstract': 'Substantial progress has been made in identifying the possible signals for initiating and terminating the appetitive aspects of feeding behavior in vertebrates. Strong evidence now implicates ATP (or an ATP-like molecule) and a fall in glucose in initiating feeding. In invertebrates, particular progress has been made in defining the nature and mechanisms of action of the neurotransmitters and peptide co-transmitters that regulate the consummatory aspects of feeding, and a number of new research tools for modelling the operation of simple feeding motor program networks have been developed.',\n",
              "  'authors': ['Kupfermann, I'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Current opinion in neurobiology',\n",
              "  'keywords': ['Biogenic Amines',\n",
              "   'Feeding Behavior',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Neuropeptide Y',\n",
              "   'Neuropeptides',\n",
              "   'Nitric Oxide',\n",
              "   'Signal Detection, Psychological'],\n",
              "  'pmid': '7888771',\n",
              "  'title': 'Neural control of feeding.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1098-1004',\n",
              "  'IF_2013': '5.05',\n",
              "  'IF_2014': '5.144',\n",
              "  'IF_2015': '5.089',\n",
              "  'IF_2016': '4.601',\n",
              "  'IF_2017': '5.359',\n",
              "  'ISSN': '1059-7794',\n",
              "  'ISSN_online': '1098-1004',\n",
              "  'ISSN_print': '1059-7794',\n",
              "  'IsoAbbr': 'Hum. Mutat.',\n",
              "  'JrId': 2408,\n",
              "  'MedAbbr': 'Hum Mutat',\n",
              "  'NLMid': '9215429',\n",
              "  'Titre': 'Hum. Mutat.',\n",
              "  'abstract': 'Variations in mismatch repair (MMR) system genes are causative of Lynch syndrome and other cancers. Thousands of variants have been identified in MMR genes, but the clinical relevance is known for only a small proportion. Recently, the InSiGHT group classified 2,360 MMR variants into five classes. One-third of variants, majority of which is nonsynonymous variants, remain to be of uncertain clinical relevance. Computational tools can be used to prioritize variants for disease relevance investigations. Previously, we classified 248 MMR variants as likely pathogenic and likely benign using PON-MMR. We have developed a novel tool, PON-MMR2, which is trained on a larger and more reliable dataset. In performance comparison, PON-MMR2 outperforms both generic tolerance prediction methods as well as methods optimized for MMR variants. It achieves accuracy and MCC of 0.89 and 0.78, respectively, in cross-validation and 0.86 and 0.69, respectively, on an independent test dataset. We classified 354 class 3 variants in InSiGHT database as well as all possible amino acid substitutions in four MMR proteins. Likely harmful variants mainly appear in the protein core, whereas likely benign variants are on the surface. PON-MMR2 is a highly reliable tool to prioritize variants for functional analysis. It is freely available at http://structure.bmc.lu.se/PON-MMR2/. ',\n",
              "  'authors': ['Niroula, Abhishek', 'Vihinen, Mauno'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Human mutation',\n",
              "  'keywords': ['Amino Acid Substitution',\n",
              "   'Colorectal Neoplasms, Hereditary Nonpolyposis',\n",
              "   'Computational Biology',\n",
              "   'DNA Mismatch Repair',\n",
              "   'DNA Repair Enzymes',\n",
              "   'Databases, Genetic',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Mutation',\n",
              "   'Nuclear Proteins',\n",
              "   'Reproducibility of Results',\n",
              "   'Software',\n",
              "   'Web Browser'],\n",
              "  'pmid': '26333163',\n",
              "  'title': 'Classification of Amino Acid Substitutions in Mismatch Repair Proteins Using PON-MMR2.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1572-8595',\n",
              "  'IF_2013': '1.552',\n",
              "  'IF_2014': '1.575',\n",
              "  'IF_2015': '1.676',\n",
              "  'IF_2016': '1.826',\n",
              "  'IF_2017': '1.522',\n",
              "  'ISSN': '1383-875X',\n",
              "  'ISSN_online': '1572-8595',\n",
              "  'ISSN_print': '1383-875X',\n",
              "  'IsoAbbr': 'J Interv Card Electrophysiol',\n",
              "  'JrId': 21155,\n",
              "  'MedAbbr': 'J Interv Card Electrophysiol',\n",
              "  'NLMid': '9708966',\n",
              "  'Titre': 'J Interv Card Electrophysiol',\n",
              "  'abstract': 'Recently, the use of robotic assisted surgery has been utilized in cardiac surgical procedures. The use of robotics may offer benefits in precision, stability and control of instruments remotely. We report early experience with a novel remote robotic catheter control system (CCS).|We used a computerized robotically controlled catheter system that enables the user to remotely manipulate the tip of a catheter precisely in three dimensions. We tested the robotic catheter control systems ability to navigate within the heart and to make precise, rapid and repeatable movements. We compared the CCS with the ability of a standard quadripolar steerable ablation catheter placed in a deflectable sheath to navigate and make precision movements. Twelve ex-vivo porcine hearts were utilized to permit accurate measurements of navigation and precision. Eight targets were selected for navigation and precision testing. Time was measured for the catheter to reach the predefined target from a specific starting point to test navigation. In addition, time was measured to contact a discrete 0.8 mm target in order to test precision.|The use of the CCS reduced the time needed for both navigation (8.5 +/- 13.9 sec vs. 22.7 +/- 26.7 sec, p = 0.002) and significantly decreased the time for precision targeting (10.1 +/- 6.9 sec vs. 29.6 +/- 26.4 sec, p < 0.001) in the specific RA and LA sites in the ex-vivo hearts.|The use of a computerized robotically assisted catheter control system is feasible and shows promise in rapid precision movement of the catheter. Further study is needed to elucidate the role of such a system in-vivo and in patient specific catheter ablation and mapping.',\n",
              "  'authors': ['Al-Ahmad, Amin', 'Grossman, Jessica D', 'Wang, Paul J'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Journal of interventional cardiac electrophysiology : an international journal of arrhythmias and pacing',\n",
              "  'keywords': ['Animals',\n",
              "   'Cardiac Catheterization',\n",
              "   'Catheter Ablation',\n",
              "   'Feasibility Studies',\n",
              "   'In Vitro Techniques',\n",
              "   'Robotics',\n",
              "   'Swine'],\n",
              "  'pmid': '15875110',\n",
              "  'title': 'Early experience with a computerized robotically controlled catheter system.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'The biological literature databases continue to grow rapidly with vital information that is important for conducting sound biomedical research and development. The current practices of manually searching for information and extracting pertinent knowledge are tedious, time-consuming tasks even for motivated biological researchers. Accurate and computationally efficient approaches in discovering relationships between biological objects from text documents are important for biologists to develop biological models. The term \"object\" refers to any biological entity such as a protein, gene, cell cycle, etc. and relationship refers to any dynamic action one object has on another, e.g. protein inhibiting another protein or one object belonging to another object such as, the cells composing an organ. This paper presents a novel approach to extract relationships between multiple biological objects that are present in a text document. The approach involves object identification, reference resolution, ontology and synonym discovery, and extracting object-object relationships. Hidden Markov Models (HMMs), dictionaries, and N-Gram models are used to set the framework to tackle the complex task of extracting object-object relationships. Experiments were carried out using a corpus of one thousand Medline abstracts. Intermediate results were obtained for the object identification process, synonym discovery, and finally the relationship extraction. For the thousand abstracts, 53 relationships were extracted of which 43 were correct, giving a specificity of 81 percent. These results are promising for multi-object identification and relationship finding from biological documents.',\n",
              "  'authors': ['Palakal, Mathew',\n",
              "   'Stephens, Matthew',\n",
              "   'Mukhopadhyay, Snehasis',\n",
              "   'Raje, Rajeev',\n",
              "   'Rhodes, Simon'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biology',\n",
              "   'Biopolymers',\n",
              "   'Cell Physiological Phenomena',\n",
              "   'Computational Biology',\n",
              "   'Documentation',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Models, Biological',\n",
              "   'Natural Language Processing',\n",
              "   'Periodicals as Topic',\n",
              "   'Terminology as Topic',\n",
              "   'Vocabulary, Controlled'],\n",
              "  'pmid': '15290775',\n",
              "  'title': 'Identification of biological relationships from text documents using efficient computational methods.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1347-4820',\n",
              "  'IF_2013': '3.685',\n",
              "  'IF_2014': '3.94',\n",
              "  'IF_2015': '4.124',\n",
              "  'IF_2016': '3.544',\n",
              "  'IF_2017': '2.895',\n",
              "  'ISSN': '1346-9843',\n",
              "  'ISSN_online': '1347-4820',\n",
              "  'ISSN_print': '1346-9843',\n",
              "  'IsoAbbr': 'Circ. J.',\n",
              "  'JrId': 27033,\n",
              "  'MedAbbr': 'Circ J',\n",
              "  'NLMid': '101137683',\n",
              "  'Titre': 'Circ. J.',\n",
              "  'abstract': 'Robotic totally endoscopic coronary artery bypass (TECAB) is a minimally invasive endoscopic surgical approach using the daVinci robotic telemanipulation system to perform coronary artery bypass grafting on the arrested or beating heart. It is a procedure that can be a useful alternative to the classic open procedure performed through sternotomy. After extensive modeling in cadavers, the first clinical case was performed in June 1998 placing a left internal thoracic artery graft (LITA) to the left anterior descending artery completely robotically on the arrested heart. During the early and late 2000s, international groups have adopted this evolving technology, which has included iterations such as beating-heart TECAB, use of bilateral ITA grafting and radial artery grafting, as well as 3- and 4-vessel TECAB. TECAB is combined with percutaneous coronary intervention in hybrid procedures. Despite increasing complexity of endoscopic coronary bypass surgery, conversion rates to open bypass surgery have dropped significantly and operative times have decreased. Published major morbidities and mortality rates in arrested-and beating-heart TECAB have been cumulatively in the 0-2% range and are considered well within the expected range for these highly complex surgical procedures. Long-term survival and freedom from major adverse events also meet the standards of open bypass surgery.',\n",
              "  'authors': ['Lee, Jeffrey D', 'Srivastava, Mukta', 'Bonatti, Johannes'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Circulation journal : official journal of the Japanese Circulation Society',\n",
              "  'keywords': ['Animals',\n",
              "   'Coronary Artery Bypass',\n",
              "   'Endoscopy',\n",
              "   'History, 20th Century',\n",
              "   'History, 21st Century',\n",
              "   'Humans',\n",
              "   'Percutaneous Coronary Intervention',\n",
              "   'Robotics'],\n",
              "  'pmid': '22893280',\n",
              "  'title': 'History and current status of robotic totally endoscopic coronary artery bypass.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1476-6256',\n",
              "  'IF_2013': '4.975',\n",
              "  'IF_2014': '5.23',\n",
              "  'IF_2015': '5.036',\n",
              "  'IF_2016': '4.825',\n",
              "  'IF_2017': '4.322',\n",
              "  'ISSN': '0002-9262',\n",
              "  'ISSN_online': '1476-6256',\n",
              "  'ISSN_print': '0002-9262',\n",
              "  'IsoAbbr': 'Am. J. Epidemiol.',\n",
              "  'JrId': 421,\n",
              "  'MedAbbr': 'Am J Epidemiol',\n",
              "  'NLMid': '7910653',\n",
              "  'Titre': 'Am. J. Epidemiol.',\n",
              "  'abstract': 'Historically, clinical epidemiologic research has been constrained by the costs and time associated with manually identifying cases and abstracting clinical data. In this issue, Carrell et al. (Am J Epidemiol. 2014;179(6);749-758) report on their impressive success using natural language processing techniques to correctly identify cases of cancer recurrence among women with previous breast cancer. They report a 10-fold decrease in the need for chart abstraction, though with an 8% loss in case detection. This commentary outlines some recent history associated with the development of \"high-throughput clinical phenotyping\" of electronic health records and speculates on the impact such computational capabilities may have for observational research and patient consent. ',\n",
              "  'authors': ['Chute, Christopher G'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'American journal of epidemiology',\n",
              "  'keywords': ['Breast Neoplasms',\n",
              "   'Electronic Health Records',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Natural Language Processing',\n",
              "   'Neoplasm Recurrence, Local'],\n",
              "  'pmid': '24488512',\n",
              "  'title': 'Invited commentary: Observational research in the age of the electronic health record.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'MicroRNAs (miRNAs) are a class of endogenous small non-coding RNAs that regulate various biological processes including sexual dimorphism. The oriental fruit fly Bactrocera dorsalis is one of the most destructive agricultural insect pests in many Asian countries. However, no miRNAs have been identified from the separate sex and gonads to elucidate sex gonad differentiation in B. dorsalis. In this study, we constructed four small RNA libraries from whole body of females, males (except ovaries and testes) and ovaries, testes of B. dorsalis for deep sequencing. The data analysis revealed 183 known and 120 novel miRNAs from these libraries. 18 female-biased and 16 male-biased miRNAs that may be involved in sexual differentiation were found by comparing the miRNA expression profiles in the four libraries. Using a bioinformatic approach, we predicted doublesex (dsx) as a target gene of the female-biased miR-989-3p which is considered as the key switch gene in the sex determination of tephritid insects. This study reveals the first miRNA profile related to the sex differentiation and gives a first insight into sex differences in miRNA expression of B. dorsalis which could facilitate studies of the reproductive organ specific roles of miRNAs. ',\n",
              "  'authors': ['Peng, Wei', 'Tariq, Kaleem', 'Xie, Junfei', 'Zhang, Hongyu'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Conserved Sequence',\n",
              "   'Female',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Genome, Insect',\n",
              "   'Male',\n",
              "   'MicroRNAs',\n",
              "   'Nucleic Acid Conformation',\n",
              "   'Nucleotides',\n",
              "   'Real-Time Polymerase Chain Reaction',\n",
              "   'Reproducibility of Results',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Sex Characteristics',\n",
              "   'Tephritidae'],\n",
              "  'pmid': '27441641',\n",
              "  'title': 'Identification and Characterization of Sex-Biased MicroRNAs in Bactrocera dorsalis (Hendel).',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '1757-9708',\n",
              "  'IF_2013': '3.996',\n",
              "  'IF_2014': '3.756',\n",
              "  'IF_2015': '3.371',\n",
              "  'IF_2016': '3.252',\n",
              "  'IF_2017': '3.294',\n",
              "  'ISSN': '1757-9694',\n",
              "  'ISSN_online': '1757-9708',\n",
              "  'ISSN_print': '1757-9694',\n",
              "  'IsoAbbr': 'Integr Biol (Camb)',\n",
              "  'JrId': 36347,\n",
              "  'MedAbbr': 'Integr Biol (Camb)',\n",
              "  'NLMid': '101478378',\n",
              "  'Titre': 'Integr Biol (Camb)',\n",
              "  'abstract': 'Three-dimensional (3D) tissue culture provides a physiologically relevant microenvironment for distinguishing malignant from non-malignant breast cell phenotypes. 3D culture assays can also be used to test novel cancer therapies and predict a differential response to radiation between normal and malignant cells in vivo. However, biological measurements in such complex models are difficult to quantify and current approaches do not allow for in-depth multifaceted assessment of individual colonies or unique sub-populations within the entire culture. This is in part due to the limitations of imaging at a range of depths in 3D culture resulting from optical aberrations and intensity attenuation. Here, we address these limitations by combining sample smearing techniques with high-throughput 2D imaging algorithms to accurately and rapidly quantify imaging features acquired from 3D cultures. Multiple high resolution imaging features especially designed to characterize 3D cultures show that non-malignant human breast cells surviving large doses of ionizing radiation acquire a \"swelled acinar\" phenotype with fewer and larger nuclei, loss of cell connectivity and diffused basement membrane. When integrating these imaging features into hierarchical clustering classification, we could also identify subpopulations of phenotypes from individual human tumor colonies treated with ionizing radiation or/and integrin inhibitors. Such tools have therefore the potential to further characterize cell culture populations after cancer treatment and identify novel phenotypes of resistance.',\n",
              "  'authors': ['Park, Catherine C',\n",
              "   'Georgescu, Walter',\n",
              "   'Polyzos, Aris',\n",
              "   'Pham, Christopher',\n",
              "   'Ahmed, Kazi M',\n",
              "   'Zhang, Hui',\n",
              "   'Costes, Sylvain V'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Integrative biology : quantitative biosciences from nano to macro',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Breast Neoplasms',\n",
              "   'Cell Line, Tumor',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Microscopy, Fluorescence',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '23407655',\n",
              "  'title': 'Rapid and automated multidimensional fluorescence microscopy profiling of 3D human breast cultures.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1531-8257',\n",
              "  'IF_2013': '5.634',\n",
              "  'IF_2014': '5.68',\n",
              "  'IF_2015': '6.01',\n",
              "  'IF_2016': '7.072',\n",
              "  'IF_2017': '8.324',\n",
              "  'ISSN': '0885-3185',\n",
              "  'ISSN_online': '1531-8257',\n",
              "  'ISSN_print': '0885-3185',\n",
              "  'IsoAbbr': 'Mov. Disord.',\n",
              "  'JrId': 5937,\n",
              "  'MedAbbr': 'Mov Disord',\n",
              "  'NLMid': '8610688',\n",
              "  'Titre': 'Mov. Disord.',\n",
              "  'abstract': \"Levodopa-induced dyskinesias (LID) in Parkinson's disease (PD) have remained a clinical challenge. We evaluated the feasibility of neural networks to detect LID and to quantify their severity in 16 patients with PD at rest and during various activities of daily living. The movements of the patients were measured using four pairs of accelerometers mounted on the wrist, upper arm, trunk, and leg on the most affected side. Using parameters obtained from the accelerometer signals, neural networks were trained to detect and to classify LID corresponding to the modified Abnormal Involuntary Movement Scale. Important parameters for classification appeared to be the mean segment velocity and the cross-correlation between accelerometers on the arm, trunk, and leg. Neural networks were able to distinguish voluntary movements from LID and to assess the severity of LID in various activities. Based on the results in this study, we conclude that neural networks are a valid and reliable method to detect and to assess the severity of LID corresponding to the modified Abnormal Involuntary Movement Scale.\",\n",
              "  'authors': ['Keijsers, N L',\n",
              "   'Horstink, M W',\n",
              "   'van Hilten, J J',\n",
              "   'Hoff, J I',\n",
              "   'Gielen, C C'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Movement disorders : official journal of the Movement Disorder Society',\n",
              "  'keywords': ['Activities of Daily Living',\n",
              "   'Adult',\n",
              "   'Antiparkinson Agents',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnosis, Differential',\n",
              "   'Dyskinesia, Drug-Induced',\n",
              "   'Feasibility Studies',\n",
              "   'Humans',\n",
              "   'Levodopa',\n",
              "   'Movement',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Parkinson Disease',\n",
              "   'Predictive Value of Tests',\n",
              "   'Severity of Illness Index'],\n",
              "  'pmid': '11104192',\n",
              "  'title': \"Detection and assessment of the severity of levodopa-induced dyskinesia in patients with Parkinson's disease by neural networks.\",\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['QU'],\n",
              "  'ESSN': '1471-2180',\n",
              "  'IF_2013': '2.976',\n",
              "  'IF_2014': '2.729',\n",
              "  'IF_2015': '2.581',\n",
              "  'IF_2016': '2.644',\n",
              "  'IF_2017': '2.829',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2180',\n",
              "  'IsoAbbr': 'BMC Microbiol.',\n",
              "  'JrId': 21992,\n",
              "  'MedAbbr': 'BMC Microbiol',\n",
              "  'NLMid': '100966981',\n",
              "  'Titre': 'BMC Microbiol.',\n",
              "  'abstract': 'The Gene Ontology project is a collaborative effort to provide descriptions of gene products in a consistent and computable language, and in a species-independent manner. The Gene Ontology is designed to be applicable to all organisms but up to now has been largely under-utilized for prokaryotes and viruses, in part because of a lack of appropriate ontology terms.|To address this issue, we have developed a set of Gene Ontology classes that are applicable to microbes and their hosts, improving both coverage and quality in this area of the Gene Ontology. Describing microbial and viral gene products brings with it the additional challenge of capturing both the host and the microbe. Recognising this, we have worked closely with annotation groups to test and optimize the GO classes, and we describe here a set of annotation guidelines that allow the controlled description of two interacting organisms.|Building on the microbial resources already in existence such as ViralZone, UniProtKB keywords and MeGO, this project provides an integrated ontology to describe interactions between microbial species and their hosts, with mappings to the external resources above. Housing this information within the freely-accessible Gene Ontology project allows the classes and annotation structure to be utilized by a large community of biologists and users.',\n",
              "  'authors': ['Foulger, R E',\n",
              "   'Osumi-Sutherland, D',\n",
              "   'McIntosh, B K',\n",
              "   'Hulo, C',\n",
              "   'Masson, P',\n",
              "   'Poux, S',\n",
              "   'Le Mercier, P',\n",
              "   'Lomax, J'],\n",
              "  'categories': ['MICROBIOLOGY'],\n",
              "  'journal': 'BMC microbiology',\n",
              "  'keywords': ['Gene Ontology',\n",
              "   'Host-Pathogen Interactions',\n",
              "   'Humans',\n",
              "   'Virus Physiological Phenomena',\n",
              "   'Viruses'],\n",
              "  'pmid': '26215368',\n",
              "  'title': 'Representing virus-host interactions and other multi-organism processes in the Gene Ontology.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'The brain represents visual objects with topographic cortical patterns. To address how distributed visual representations enable object categorization, we established predictive encoding models based on a deep residual network, and trained them to predict cortical responses to natural movies. Using this predictive model, we mapped human cortical representations to 64,000 visual objects from 80 categories with high throughput and accuracy. Such representations covered both the ventral and dorsal pathways, reflected multiple levels of object features, and preserved semantic relationships between categories. In the entire visual cortex, object representations were organized into three clusters of categories: biological objects, non-biological objects, and background scenes. In a finer scale specific to each cluster, object representations revealed sub-clusters for further categorization. Such hierarchical clustering of category representations was mostly contributed by cortical representations of object features from middle to high levels. In summary, this study demonstrates a useful computational strategy to characterize the cortical organization and representations of visual features for rapid categorization.',\n",
              "  'authors': ['Wen, Haiguang', 'Shi, Junxing', 'Chen, Wei', 'Liu, Zhongming'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Photic Stimulation',\n",
              "   'Time Factors',\n",
              "   'Visual Cortex'],\n",
              "  'pmid': '29491405',\n",
              "  'title': 'Deep Residual Network Predicts Cortical Representation and Organization of Visual Features for Rapid Categorization.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1522-2594',\n",
              "  'IF_2013': '3.398',\n",
              "  'IF_2014': '3.571',\n",
              "  'IF_2015': '3.782',\n",
              "  'IF_2016': '3.924',\n",
              "  'IF_2017': '4.082',\n",
              "  'ISSN': '0740-3194',\n",
              "  'ISSN_online': '1522-2594',\n",
              "  'ISSN_print': '0740-3194',\n",
              "  'IsoAbbr': 'Magn Reson Med',\n",
              "  'JrId': 5733,\n",
              "  'MedAbbr': 'Magn Reson Med',\n",
              "  'NLMid': '8505245',\n",
              "  'Titre': 'Magn Reson Med',\n",
              "  'abstract': \"To demonstrate accurate MR image reconstruction from undersampled k-space data using cross-domain convolutional neural networks (CNNs) METHODS: Cross-domain CNNs consist of 3 components: (1) a deep CNN operating on the k-space (KCNN), (2) a deep CNN operating on an image domain (ICNN), and (3) an interleaved data consistency operations. These components are alternately applied, and each CNN is trained to minimize the loss between the reconstructed and corresponding fully sampled k-spaces. The final reconstructed image is obtained by forward-propagating the undersampled k-space data through the entire network.|Performances of K-net (KCNN with inverse Fourier transform), I-net (ICNN with interleaved data consistency), and various combinations of the 2 different networks were tested. The test results indicated that K-net and I-net have different advantages/disadvantages in terms of tissue-structure restoration. Consequently, the combination of K-net and I-net is superior to single-domain CNNs. Three MR data sets, the T2 fluid-attenuated inversion recovery (T2 FLAIR) set from the Alzheimer's Disease Neuroimaging Initiative and 2 data sets acquired at our local institute (T2 FLAIR and T1 weighted), were used to evaluate the performance of 7 conventional reconstruction algorithms and the proposed cross-domain CNNs, which hereafter is referred to as KIKI-net. KIKI-net outperforms conventional algorithms with mean improvements of 2.29 dB in peak SNR and 0.031 in structure similarity.|KIKI-net exhibits superior performance over state-of-the-art conventional algorithms in terms of restoring tissue structures and removing aliasing artifacts. The results demonstrate that KIKI-net is applicable up to a reduction factor of 3 to 4 based on variable-density Cartesian undersampling.\",\n",
              "  'authors': ['Eo, Taejoon',\n",
              "   'Jun, Yohan',\n",
              "   'Kim, Taeseong',\n",
              "   'Jang, Jinseong',\n",
              "   'Lee, Ho-Joon',\n",
              "   'Hwang, Dosik'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain',\n",
              "   'Databases, Factual',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Signal-To-Noise Ratio'],\n",
              "  'pmid': '29624729',\n",
              "  'title': 'KIKI-net: cross-domain convolutional neural networks for reconstructing undersampled magnetic resonance images.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Phenotypic features associated with genes and diseases play an important role in disease-related studies and most of the available methods focus solely on the Online Mendelian Inheritance in Man (OMIM) database without considering the controlled vocabulary. The Human Phenotype Ontology (HPO) provides a standardized and controlled vocabulary covering phenotypic abnormalities in human diseases, and becomes a comprehensive resource for computational analysis of human disease phenotypes. Most of the existing HPO-based software tools cannot be used offline and provide only few similarity measures. Therefore, there is a critical need for developing a comprehensive and offline software for phenotypic features similarity based on HPO.|HPOSim is an R package for analyzing phenotypic similarity for genes and diseases based on HPO data. Seven commonly used semantic similarity measures are implemented in HPOSim. Enrichment analysis of gene sets and disease sets are also implemented, including hypergeometric enrichment analysis and network ontology analysis (NOA).|HPOSim can be used to predict disease genes and explore disease-related function of gene modules. HPOSim is open source and freely available at SourceForge (https://sourceforge.net/p/hposim/).',\n",
              "  'authors': ['Deng, Yue', 'Gao, Lin', 'Wang, Bingbo', 'Guo, Xingli'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Databases, Genetic',\n",
              "   'Disease',\n",
              "   'Gene Ontology',\n",
              "   'Genes',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Humans',\n",
              "   'Phenotype',\n",
              "   'Software'],\n",
              "  'pmid': '25664462',\n",
              "  'title': 'HPOSim: an R package for phenotypic similarity measure and enrichment analysis based on the human phenotype ontology.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'Although Particle Swarm Optimization (PSO) has demonstrated competitive performance in solving global optimization problems, it exhibits some limitations when dealing with optimization problems with high dimensionality and complex landscape. In this paper, we integrate some problem-oriented knowledge into the design of a certain PSO variant. The resulting novel PSO algorithm with an inner variable learning strategy (PSO-IVL) is particularly efficient for optimizing functions with symmetric variables. Symmetric variables of the optimized function have to satisfy a certain quantitative relation. Based on this knowledge, the inner variable learning (IVL) strategy helps the particle to inspect the relation among its inner variables, determine the exemplar variable for all other variables, and then make each variable learn from the exemplar variable in terms of their quantitative relations. In addition, we design a new trap detection and jumping out strategy to help particles escape from local optima. The trap detection operation is employed at the level of individual particles whereas the trap jumping out strategy is adaptive in its nature. Experimental simulations completed for some representative optimization functions demonstrate the excellent performance of PSO-IVL. The effectiveness of the PSO-IVL stresses a usefulness of augmenting evolutionary algorithms by problem-oriented domain knowledge. ',\n",
              "  'authors': ['Wu, Guohua',\n",
              "   'Pedrycz, Witold',\n",
              "   'Ma, Manhao',\n",
              "   'Qiu, Dishan',\n",
              "   'Li, Haifeng',\n",
              "   'Liu, Jin'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Models, Theoretical',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '24587746',\n",
              "  'title': 'A particle swarm optimization variant with an inner variable learning strategy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-9568',\n",
              "  'IF_2013': '3.669',\n",
              "  'IF_2014': '3.181',\n",
              "  'IF_2015': '2.975',\n",
              "  'IF_2016': '2.941',\n",
              "  'IF_2017': '2.832',\n",
              "  'ISSN': '0953-816X',\n",
              "  'ISSN_online': '1460-9568',\n",
              "  'ISSN_print': '0953-816X',\n",
              "  'IsoAbbr': 'Eur. J. Neurosci.',\n",
              "  'JrId': 2607,\n",
              "  'MedAbbr': 'Eur J Neurosci',\n",
              "  'NLMid': '8918110',\n",
              "  'Titre': 'Eur. J. Neurosci.',\n",
              "  'abstract': \"Cognitive behaviour requires complex context-dependent mapping between sensory stimuli and actions. The same stimulus can lead to different behaviours depending on the situation, or the same behaviour may be elicited by different cueing stimuli. Neurons in the primate prefrontal cortex show task-specific firing activity during working memory delay periods. These neurons provide a neural substrate for mapping stimulus and response in a flexible, context- or rule-dependent, fashion. We describe here an integrate-and-fire network model to explain and investigate the different types of working-memory-related neuronal activity observed. The model contains different populations (or pools) of neurons (as found neurophysiologically) in attractor networks which respond in the delay period to the stimulus object, the stimulus position ('sensory pools'), to combinations of the stimulus sensory properties (e.g. the object identity or object location) and the response ('intermediate pools'), and to the response required (left or right) ('premotor pools'). The pools are arranged hierarchically, are linked by associative synaptic connections, and have global inhibition through inhibitory interneurons to implement competition. It is shown that a biasing attentional input to define the current rule applied to the intermediate pools enables the system to select the correct response in what is a biased competition model of attention. The integrate-and-fire model not only produces realistic spiking dynamicals very similar to the neuronal data but also shows how dopamine could weaken and shorten the persistent neuronal activity in the delay period; and allows us to predict more response errors when dopamine is elevated because there is less different activity in the different pools of competing neurons, resulting in more conflict.\",\n",
              "  'authors': ['Deco, Gustavo', 'Rolls, Edmund T'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The European journal of neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Association Learning',\n",
              "   'Attention',\n",
              "   'Cues',\n",
              "   'Dopamine',\n",
              "   'Humans',\n",
              "   'Memory, Short-Term',\n",
              "   'Models, Neurological',\n",
              "   'N-Methylaspartate',\n",
              "   'Nerve Net',\n",
              "   'Neural Conduction',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Prefrontal Cortex',\n",
              "   'Random Allocation',\n",
              "   'Saccades',\n",
              "   'Synapses',\n",
              "   'Task Performance and Analysis',\n",
              "   'Time Factors',\n",
              "   'gamma-Aminobutyric Acid'],\n",
              "  'pmid': '14622200',\n",
              "  'title': 'Attention and working memory: a dynamical model of neuronal activity in the prefrontal cortex.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'Laparoscopic radical prostatectomy with or without a robot has been increasingly performed worldwide, primarily using a transperitoneal approach. We report our experience with daVinci(R) robot assisted extraperitoneal laparoscopic radical prostatectomy.|A total of 325 patients underwent robot assisted extraperitoneal laparoscopic radical prostatectomy for clinically localized prostate cancer at our center during a 2-year period. Perioperative data, and oncological and functional results were prospectively recorded.|Perioperative demographics included mean age, PSA and Gleason score, which were 60 years (range 42 to 76), 6.6 ng/ml (range 0.6 to 26) and 6 (range 5 to 9), respectively. Preoperative clinical stage was 81%, 16% and 3% for T1c, T2a and T2b, respectively. Average total operative time was 130 minutes (range 80 to 480). Intraoperative data included a mean blood loss of 196 cc with no open conversions. Bilateral, unilateral and nonnerve sparing prostatectomy was performed in 70%, 24% and 6% of patients, respectively. Of the patients 96% were discharged home within 8 to 23 hours of surgery. Pathological stage was pT2a, pT2b, pT3a and pT3b in 18%, 63%, 14% and 5% of all radical prostatectomy specimens, respectively, with an overall positive surgical margin rate of 13%. Two of 92 patients had positive nodal disease after lymph node dissection. Continence and erectile function were measured.|The extraperitoneal approach offers the advantages of improved dexterity and visualization of the robot, while avoiding the abdominal cavity and potential associated morbidity. As surgeons gain more experience with this new technology, the extraperitoneal approach simulating the standard open retropubic technique is likely to gain popularity.',\n",
              "  'authors': ['Joseph, J V',\n",
              "   'Rosenbaum, R',\n",
              "   'Madeb, R',\n",
              "   'Erturk, E',\n",
              "   'Patel, H R H'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Prospective Studies',\n",
              "   'Prostatectomy',\n",
              "   'Robotics'],\n",
              "  'pmid': '16469589',\n",
              "  'title': 'Robotic extraperitoneal radical prostatectomy: an alternative approach.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"In many scientific disciplines the use of a metaphor as an heuristic aid is not uncommon. A well known example in somatic medicine is the 'defense army metaphor' used to characterize the immune system. In fact, probably a large part of the everyday work of doctors consists of 'translating' scientific and clinical information (i.e. causes of disease, percentage of success versus risk of side-effects) into information tailored to the needs and capacities of the individual patient. The ability to do so in an effective way is at least partly what makes a clinician a good communicator. Schizophrenia is a severe psychiatric disorder which affects approximately 1% of the population. Over the last two decades a large amount of molecular-biological, imaging and genetic data have been accumulated regarding the biological underpinnings of schizophrenia. However, it remains difficult to understand how the characteristic symptoms of schizophrenia such as hallucinations and delusions are related to disturbances on the molecular-biological level. In general, psychiatry seems to lack a conceptual framework with sufficient explanatory power to link the mental- and molecular-biological domains.|Here, we present an essay-like study in which we propose to use visualized concepts stemming from the theory on dynamical complex systems as a 'visual metaphor' to bridge the mental- and molecular-biological domains in schizophrenia. We first describe a computer model of neural information processing; we show how the information processing in this model can be visualized, using concepts from the theory on complex systems. We then describe two computer models which have been used to investigate the primary theory on schizophrenia, the neurodevelopmental model, and show how disturbed information processing in these two computer models can be presented in terms of the visual metaphor previously described. Finally, we describe the effects of dopamine neuromodulation, of which disturbances have been frequently described in schizophrenia, in terms of the same visualized metaphor.|The conceptual framework and metaphor described offers a heuristic tool to understand the relationship between the mental- and molecular-biological domains in an intuitive way. The concepts we present may serve to facilitate communication between researchers, clinicians and patients.\",\n",
              "  'authors': ['van Beveren, Nico J M', 'de Haan, Lieuwe'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Dopamine',\n",
              "   'Humans',\n",
              "   'Immune System',\n",
              "   'Memory',\n",
              "   'Models, Biological',\n",
              "   'Models, Neurological',\n",
              "   'Models, Theoretical',\n",
              "   'Neurons',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Prefrontal Cortex',\n",
              "   'Schizophrenia',\n",
              "   'Schizophrenic Psychology',\n",
              "   'Software',\n",
              "   'Systems Theory'],\n",
              "  'pmid': '18648540',\n",
              "  'title': 'A visual metaphor describing neural dynamics in schizophrenia.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['MA'],\n",
              "  'ESSN': '1365-2257',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0141-9854',\n",
              "  'ISSN_online': '1365-2257',\n",
              "  'ISSN_print': '0141-9854',\n",
              "  'IsoAbbr': 'Clin Lab Haematol',\n",
              "  'JrId': 3079,\n",
              "  'MedAbbr': 'Clin Lab Haematol',\n",
              "  'NLMid': '7907061',\n",
              "  'Titre': 'Clin Lab Haematol',\n",
              "  'abstract': 'The morphological appearance of blood cells has an established association to clinical conditions. A novel system, DiffMaster Octavia for differential counting of blood leukocytes, has been evaluated. The system consisted of a microscope, 3-chip color charge coupled device (CCD) camera, automated motorized stage holder, electronic hardware for motor and light control and software for automatic cell location and image processing for preclassification of blood cells using artificial neural networks. The DiffMaster test method, was evaluated on 322 routine blood samples (400 cells per sample) using manual microscopy as reference method. The results showed a correlation of determination (r(2)) of 0.8-0.98 for the normal cell classes and blast cells. The DiffMaster correctly preclassified 89% of all leukocytes with a good reproducibility. After verification of the cell classes, the agreement between the test and reference method was 91% whether the sample was abnormal or normal. The clinical sensitivity was 98% and specificity 82%. The sensitivity to identify blast cells was slightly higher with the DiffMaster than manual microscopy. Similar levels of short-term imprecision for the two methods were found for all cell classes. In conclusion this study shows that the DiffMaster can provide a decision support system which, together with a qualified morphologist, can generate leukocyte differential count reports of high quality.',\n",
              "  'authors': ['Swolin, B',\n",
              "   'Simonsson, P',\n",
              "   'Backman, S',\n",
              "   'Löfqvist, I',\n",
              "   'Bredin, I',\n",
              "   'Johnsson, M'],\n",
              "  'categories': ['HEMATOLOGY'],\n",
              "  'journal': 'Clinical and laboratory haematology',\n",
              "  'keywords': ['Automation',\n",
              "   'Blast Crisis',\n",
              "   'Blood Cell Count',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Leukocyte Count',\n",
              "   'Microscopy',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '12755789',\n",
              "  'title': 'Differential counting of blood leukocytes using automated microscopy and a decision support system based on artificial neural networks--evaluation of DiffMaster Octavia.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '1365-2184',\n",
              "  'IF_2013': '3.28',\n",
              "  'IF_2014': '3.116',\n",
              "  'IF_2015': '3.084',\n",
              "  'IF_2016': '4.112',\n",
              "  'IF_2017': '4.936',\n",
              "  'ISSN': '0960-7722',\n",
              "  'ISSN_online': '1365-2184',\n",
              "  'ISSN_print': '0960-7722',\n",
              "  'IsoAbbr': 'Cell Prolif.',\n",
              "  'JrId': 1103,\n",
              "  'MedAbbr': 'Cell Prolif',\n",
              "  'NLMid': '9105195',\n",
              "  'Titre': 'Cell Prolif.',\n",
              "  'abstract': 'Epstein-Barr virus (EBV) transformation has been described as a routine method to establish human B lymphoblastoid cell lines. Each established lymphoblastoid cell line represents one unique genetic information carrier and can produce unlimited quantities of DNA materials available for downstream applications and research. Undoubtedly, it is of great value to human clinical and experimental genetic studies. However, the current process of EBV transformation requires much manpower in the routine renewal of medium, which is time-consuming. This situation can become a serious problem especially when establishing a human B lymphoblastoid cell bank. A modified and cost-effective protocol for EBV transformation should be considered.|In the present study, process in EBV transformation was modified to fit the requirements of robot handling.|1 mL of whole blood was demonstrated to be sufficient to perform EBV transformation. Additionally, EBV transformation can performed in 96-deep-well plates that are directly and widely used with automatic work platforms.|Based on these facts, a process of EBV transformation can be modified to fit the requirements of robot handling.',\n",
              "  'authors': ['Chang, I-C', 'Ko, H-W', 'Hwang, S-M'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Cell proliferation',\n",
              "  'keywords': ['B-Lymphocytes',\n",
              "   'Cell Line',\n",
              "   'Cell Transformation, Viral',\n",
              "   'Herpesvirus 4, Human',\n",
              "   'Humans',\n",
              "   'Robotics'],\n",
              "  'pmid': '19143759',\n",
              "  'title': 'Capacity of robot handling for Epstein-Barr virus transformation.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1873-233X',\n",
              "  'IF_2013': '4.368',\n",
              "  'IF_2014': '5.175',\n",
              "  'IF_2015': '5.656',\n",
              "  'IF_2016': '5.215',\n",
              "  'IF_2017': '4.982',\n",
              "  'ISSN': '0029-7844',\n",
              "  'ISSN_online': '1873-233X',\n",
              "  'ISSN_print': '0029-7844',\n",
              "  'IsoAbbr': 'Obstet Gynecol',\n",
              "  'JrId': 6204,\n",
              "  'MedAbbr': 'Obstet Gynecol',\n",
              "  'NLMid': '0401101',\n",
              "  'Titre': 'Obstet Gynecol',\n",
              "  'abstract': 'To compare conventional laparoscopic and robotic-assisted laparoscopic sacrocolpopexy for vaginal apex prolapse.|This single-center, blinded randomized trial included participants with stage 2-4 posthysterectomy vaginal prolapse. Participants were randomized to laparoscopic or robotic sacrocolpopexy. The primary outcome was total operative time from incision to closure. Secondary outcomes were postoperative pain, functional activity, bowel and bladder symptoms, quality of life, anatomic vaginal support, and cost from a health care system perspective.|A total of 78 patients enrolled and were randomized (laparoscopic n=38; robotic n=40). Total operative time was significantly longer in the robotic group compared with the laparoscopic group (+67-minute difference; 95% confidence interval [CI] 43-89; P<.001). Anesthesia time, total time in the operating room, total sacrocolpopexy time, and total suturing time were all significantly longer in the robotic group. Participants in the robotic group also had significantly higher pain at rest and with activity during weeks 3 through 5 after surgery and required longer use of nonsteroidal anti-inflammatory drugs (median, 20 compared with 11 days, P<.005). The robotic group incurred greater cost than the laparoscopic group (mean difference +$1,936; 95% CI $417-$3,454; P=.008). Both groups demonstrated significant improvement in vaginal support and functional outcomes 1 year after surgery with no differences between groups.|Robotic-assisted sacrocolpopexy results in longer operating time and increased pain and cost compared with the conventional laparoscopic approach.',\n",
              "  'authors': ['Paraiso, Marie Fidela R',\n",
              "   'Jelovsek, J Eric',\n",
              "   'Frick, Anna',\n",
              "   'Chen, Chi Chung Grace',\n",
              "   'Barber, Matthew D'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Obstetrics and gynecology',\n",
              "  'keywords': ['Aged',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Robotics',\n",
              "   'Single-Blind Method',\n",
              "   'Time Factors',\n",
              "   'Uterine Prolapse'],\n",
              "  'pmid': '21979458',\n",
              "  'title': 'Laparoscopic compared with robotic sacrocolpopexy for vaginal prolapse: a randomized controlled trial.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1522-2594',\n",
              "  'IF_2013': '3.398',\n",
              "  'IF_2014': '3.571',\n",
              "  'IF_2015': '3.782',\n",
              "  'IF_2016': '3.924',\n",
              "  'IF_2017': '4.082',\n",
              "  'ISSN': '0740-3194',\n",
              "  'ISSN_online': '1522-2594',\n",
              "  'ISSN_print': '0740-3194',\n",
              "  'IsoAbbr': 'Magn Reson Med',\n",
              "  'JrId': 5733,\n",
              "  'MedAbbr': 'Magn Reson Med',\n",
              "  'NLMid': '8505245',\n",
              "  'Titre': 'Magn Reson Med',\n",
              "  'abstract': 'To reduce noise and artifacts in (23)Na MRI with a Compressed Sensing reconstruction and a learned dictionary as sparsifying transform.|A three-dimensional dictionary-learning compressed sensing reconstruction algorithm (3D-DLCS) for the reconstruction of undersampled 3D radial (23)Na data is presented. The dictionary used as the sparsifying transform is learned with a K-singular-value-decomposition (K-SVD) algorithm. The reconstruction parameters are optimized on simulated data, and the quality of the reconstructions is assessed with peak signal-to-noise ratio (PSNR) and structural similarity (SSIM). The performance of the algorithm is evaluated in phantom and in vivo (23)Na MRI data of seven volunteers and compared with nonuniform fast Fourier transform (NUFFT) and other Compressed Sensing reconstructions.|The reconstructions of simulated data have maximal PSNR and SSIM for an undersampling factor (USF) of 10 with numbers of averages equal to the USF. For 10-fold undersampling, the PSNR is increased by 5.1 dB compared with the NUFFT reconstruction, and the SSIM by 24%. These results are confirmed by phantom and in vivo (23)Na measurements in the volunteers that show markedly reduced noise and undersampling artifacts in the case of 3D-DLCS reconstructions.|The 3D-DLCS algorithm enables precise reconstruction of undersampled (23)Na MRI data with markedly reduced noise and artifact levels compared with NUFFT reconstruction. Small structures are well preserved.',\n",
              "  'authors': ['Behl, Nicolas G R',\n",
              "   'Gnahm, Christine',\n",
              "   'Bachert, Peter',\n",
              "   'Ladd, Mark E',\n",
              "   'Nagel, Armin M'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance in medicine',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Brain',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Machine Learning',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Phantoms, Imaging',\n",
              "   'Sodium Isotopes',\n",
              "   'Young Adult'],\n",
              "  'pmid': '25989746',\n",
              "  'title': 'Three-dimensional dictionary-learning reconstruction of (23)Na MRI data.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1521-0111',\n",
              "  'IF_2013': '4.12',\n",
              "  'IF_2014': '4.128',\n",
              "  'IF_2015': '3.931',\n",
              "  'IF_2016': '3.922',\n",
              "  'IF_2017': '3.978',\n",
              "  'ISSN': '0026-895X',\n",
              "  'ISSN_online': '1521-0111',\n",
              "  'ISSN_print': '0026-895X',\n",
              "  'IsoAbbr': 'Mol. Pharmacol.',\n",
              "  'JrId': 5912,\n",
              "  'MedAbbr': 'Mol Pharmacol',\n",
              "  'NLMid': '0035623',\n",
              "  'Titre': 'Mol. Pharmacol.',\n",
              "  'abstract': 'The computer-automated structure evaluation program has been used to study 482 compounds relevant to the inhibition of the aldose reductase enzyme. Major activating/inactivating fragments were generated automatically. The significance of these molecular descriptors with respect to the activity of the compounds is discussed.',\n",
              "  'authors': ['Klopman, G', 'Buyukbingol, E'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Molecular pharmacology',\n",
              "  'keywords': ['Aldehyde Reductase',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Binding Sites',\n",
              "   'Drug Design',\n",
              "   'Regression Analysis',\n",
              "   'Software',\n",
              "   'Structure-Activity Relationship',\n",
              "   'Sugar Alcohol Dehydrogenases'],\n",
              "  'pmid': '3143909',\n",
              "  'title': 'An artificial intelligence approach to the study of the structural moieties relevant to drug-receptor interactions in aldose reductase inhibitors.',\n",
              "  'year': '1988'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1471-2962',\n",
              "  'IF_2013': '2.864',\n",
              "  'IF_2014': '2.147',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.97',\n",
              "  'IF_2017': '2.746',\n",
              "  'ISSN': '1364-503X',\n",
              "  'ISSN_online': '1471-2962',\n",
              "  'ISSN_print': '1364-503X',\n",
              "  'IsoAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'JrId': 30111,\n",
              "  'MedAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'NLMid': '101133385',\n",
              "  'Titre': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'abstract': \"For it is not the abundance of knowledge, but the interior feeling and taste of things, which is accustomed to satisfy the desire of the soul. (Saint Ignatius of Loyola). We argue that the boldest claims of big data (BD) are in need of revision and toning-down, in view of a few basic lessons learned from the science of complex systems. We point out that, once the most extravagant claims of BD are properly discarded, a synergistic merging of BD with big theory offers considerable potential to spawn a new scientific paradigm capable of overcoming some of the major barriers confronted by the modern scientific method originating with Galileo. These obstacles are due to the presence of nonlinearity, non-locality and hyperdimensions which one encounters frequently in multi-scale modelling of complex systems. This article is part of the theme issue 'Multiscale modelling, simulation and computing: from the desktop to the exascale'.\",\n",
              "  'authors': ['Succi, Sauro', 'Coveney, Peter V'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Philosophical transactions. Series A, Mathematical, physical, and engineering sciences',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30967041',\n",
              "  'title': 'Big data: the end of the scientific method?',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1530-0285',\n",
              "  'IF_2013': '6.364',\n",
              "  'IF_2014': '6.187',\n",
              "  'IF_2015': '5.485',\n",
              "  'IF_2016': '5.728',\n",
              "  'IF_2017': '6.655',\n",
              "  'ISSN': '0893-3952',\n",
              "  'ISSN_online': '1530-0285',\n",
              "  'ISSN_print': '0893-3952',\n",
              "  'IsoAbbr': 'Mod. Pathol.',\n",
              "  'JrId': 6644,\n",
              "  'MedAbbr': 'Mod Pathol',\n",
              "  'NLMid': '8806605',\n",
              "  'Titre': 'Mod. Pathol.',\n",
              "  'abstract': 'One hundred fourteen cervical smears obtained from 18 women developing biopsy-proven high-grade squamous intraepithelial lesions and two with invasive squamous carcinomas were analyzed by two pathologists using the PAPNET neural network-based automated screening system (PAPNET Analyses A and B). The smears were originally reported as negative and had been previously rescreened and reclassified according to The Bethesda System. Using the PAPNET video displays of 128 potentially abnormal cellular images per smear, each reviewer (PAPNET A and B) determined whether a smear required conventional rescreening. Results of the PAPNET triage were compared with the reclassification diagnoses of the smears by conventional microscopy. PAPNET Analysis A selected eight (14%) smears reclassified as negative, 25 (69%) as atypical squamous cells of undetermined significance, and 15 (71%) as squamous intraepithelial lesions (SIL) for rescreening. In PAPNET Analysis A, two (10%) SILs were not selected for rescreening, and four (19%) were considered unsatisfactory for analysis. PAPNET Analysis B selected 21 (37%) smears reclassified as negative, 25 (69%) as atypical squamous cells of undetermined significance, and 18 (86%) as SIL for review. In PAPNET Analysis B, two (10%) SILs were missed, and one (5%) smear was unsatisfactory for analysis. Each PAPNET analysis selected smears for rescreening in 19 (95%) of 20 patients and detected SILs in 10 patients that were missed in the original screening. Using PAPNET, SILs would have been detected a median of 56 months (PAPNET A) and 62 months (PAPNET B) before their actual discovery. These preliminary data suggest that PAPNET may help detect SILs missed in routine cytologic screening.(ABSTRACT TRUNCATED AT 250 WORDS)',\n",
              "  'authors': ['Sherman, M E',\n",
              "   'Mango, L J',\n",
              "   'Kelly, D',\n",
              "   'Paull, G',\n",
              "   'Ludin, V',\n",
              "   'Copeland, C',\n",
              "   'Solomon, D',\n",
              "   'Schiffman, M H'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc',\n",
              "  'keywords': ['Carcinoma, Squamous Cell',\n",
              "   'Cervical Intraepithelial Neoplasia',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Retrospective Studies',\n",
              "   'Uterine Cervical Neoplasms',\n",
              "   'Vaginal Smears'],\n",
              "  'pmid': '7937724',\n",
              "  'title': 'PAPNET analysis of reportedly negative smears preceding the diagnosis of a high-grade squamous intraepithelial lesion or carcinoma.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'After the social learning models were proposed, finding solutions to the games becomes a well-defined mathematical question. However, almost all papers on the games and their applications are based on solutions built either upon an ad-hoc argument or a twisted Bayesian analysis of the games. Here, we present logical gaps in those solutions and offer an exact solution of our own. We also introduce a minor extension to the original game so that not only logical differences but also differences in action outcomes among those solutions become visible. ',\n",
              "  'authors': ['Dai, Wenjie', 'Wang, Xin', 'Di, Zengru', 'Wu, Jinshan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Bayes Theorem',\n",
              "   'Fuzzy Logic',\n",
              "   'Game Theory',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Mathematical Concepts'],\n",
              "  'pmid': '25546736',\n",
              "  'title': 'Logical gaps in the approximate solutions of the social learning game and an exact solution.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1879-1360',\n",
              "  'IF_2013': '2.839',\n",
              "  'IF_2014': '2.736',\n",
              "  'IF_2015': '2.84',\n",
              "  'IF_2016': '2.809',\n",
              "  'IF_2017': '2.947',\n",
              "  'ISSN': '0022-3999',\n",
              "  'ISSN_online': '1879-1360',\n",
              "  'ISSN_print': '0022-3999',\n",
              "  'IsoAbbr': 'J Psychosom Res',\n",
              "  'JrId': 5218,\n",
              "  'MedAbbr': 'J Psychosom Res',\n",
              "  'NLMid': '0376333',\n",
              "  'Titre': 'J Psychosom Res',\n",
              "  'abstract': 'Periodic limb movement disorder (PLMD) and restless legs syndrome (RLS) are two sleep disorders characterized by abnormal leg movements and are responsible for deterioration in sleep quality. However, the prevalence of these disorders is not well known in the general population. This study aims to document the prevalence of RLS and PLMD in the general population and to identify factors associated with these conditions.|Cross-sectional studies were performed in the UK, Germany, Italy, Portugal and Spain. Overall, 18,980 subjects aged 15 to 100 years old representative of the general population of these five European countries underwent telephone interviews with the Sleep-EVAL system. A section of the questionnaire assessed leg symptoms during sleep. The diagnoses of PLMD and RLS were based on the minimal criteria provided by the International Classification of Sleep Disorders.|The prevalence of PLMD was 3.9% and RLS was 5.5%. RLS and PLMD were higher in women than in men. The prevalence of RLS significantly increased with age. In multivariate models, being a woman, the presence of musculoskeletal disease, heart disease, obstructive sleep apnea syndrome, cataplexy, doing physical activities close to bedtime and the presence of a mental disorder were significantly associated with both disorders. Factors specific to PLMD were: being a shift or night worker, snoring, daily coffee intake, use of hypnotics and stress. Factors solely associated with RLS were: advanced age, obesity, hypertension, loud snoring, drinking at least three alcoholic beverages per day, smoking more than 20 cigarettes per day and use of SSRI.|PLMD and RLS are prevalent in the general population. Both conditions are associated with several physical and mental disorders and may negatively impact sleep. Greater recognition of these sleep disorders is needed.',\n",
              "  'authors': ['Ohayon, Maurice M', 'Roth, Thomas'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Journal of psychosomatic research',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Expert Systems',\n",
              "   'Female',\n",
              "   'Health Status',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nocturnal Myoclonus Syndrome',\n",
              "   'Population Surveillance',\n",
              "   'Prevalence',\n",
              "   'Restless Legs Syndrome',\n",
              "   'Surveys and Questionnaires'],\n",
              "  'pmid': '12127170',\n",
              "  'title': 'Prevalence of restless legs syndrome and periodic limb movement disorder in the general population.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1758-0463',\n",
              "  'IF_2013': '4.457',\n",
              "  'IF_2014': '3.372',\n",
              "  'IF_2015': '2.627',\n",
              "  'IF_2016': '3.29',\n",
              "  'IF_2017': '3.978',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1758-0463',\n",
              "  'IsoAbbr': 'Database (Oxford)',\n",
              "  'JrId': 36699,\n",
              "  'MedAbbr': 'Database (Oxford)',\n",
              "  'NLMid': '101517697',\n",
              "  'Titre': 'Database (Oxford)',\n",
              "  'abstract': \"Bio-ontologies provide terminologies for the scientific community to describe biomedical entities in a standardized manner. There are multiple initiatives that are developing biomedical terminologies for the purpose of providing better annotation, data integration and mining capabilities. Terminology resources devised for multiple purposes inherently diverge in content and structure. A major issue of biomedical data integration is the development of overlapping terms, ambiguous classifications and inconsistencies represented across databases and publications. The disease ontology (DO) was developed over the past decade to address data integration, standardization and annotation issues for human disease data. We have established a DO cancer project to be a focused view of cancer terms within the DO. The DO cancer project mapped 386 cancer terms from the Catalogue of Somatic Mutations in Cancer (COSMIC), The Cancer Genome Atlas (TCGA), International Cancer Genome Consortium, Therapeutically Applicable Research to Generate Effective Treatments, Integrative Oncogenomics and the Early Detection Research Network into a cohesive set of 187 DO terms represented by 63 top-level DO cancer terms. For example, the COSMIC term 'kidney, NS, carcinoma, clear_cell_renal_cell_carcinoma' and TCGA term 'Kidney renal clear cell carcinoma' were both grouped to the term 'Disease Ontology Identification (DOID):4467 / renal clear cell carcinoma' which was mapped to the TopNodes_DOcancerslim term 'DOID:263 / kidney cancer'. Mapping of diverse cancer terms to DO and the use of top level terms (DO slims) will enable pan-cancer analysis across datasets generated from any of the cancer term sources where pan-cancer means including or relating to all or multiple types of cancer. The terms can be browsed from the DO web site (http://www.disease-ontology.org) and downloaded from the DO's Apache Subversion or GitHub repositories. Database URL: http://www.disease-ontology.org\",\n",
              "  'authors': ['Wu, Tsung-Jung',\n",
              "   'Schriml, Lynn M',\n",
              "   'Chen, Qing-Rong',\n",
              "   'Colbert, Maureen',\n",
              "   'Crichton, Daniel J',\n",
              "   'Finney, Richard',\n",
              "   'Hu, Ying',\n",
              "   'Kibbe, Warren A',\n",
              "   'Kincaid, Heather',\n",
              "   'Meerzaman, Daoud',\n",
              "   'Mitraka, Elvira',\n",
              "   'Pan, Yang',\n",
              "   'Smith, Krista M',\n",
              "   'Srivastava, Sudhir',\n",
              "   'Ward, Sari',\n",
              "   'Yan, Cheng',\n",
              "   'Mazumder, Raja'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Database : the journal of biological databases and curation',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Ontologies',\n",
              "   'Data Mining',\n",
              "   'Databases, Factual',\n",
              "   'Humans',\n",
              "   'Neoplasms'],\n",
              "  'pmid': '25841438',\n",
              "  'title': 'Generating a focused view of disease ontology cancer terms for pan-cancer data integration and analysis.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.58',\n",
              "  'IF_2014': '0.488',\n",
              "  'IF_2015': '0.918',\n",
              "  'IF_2016': '0.454',\n",
              "  'IF_2017': '0.398',\n",
              "  'ISSN': '0884-6812',\n",
              "  'IsoAbbr': 'Anal. Quant. Cytol. Histol.',\n",
              "  'JrId': 1348,\n",
              "  'MedAbbr': 'Anal Quant Cytol Histol',\n",
              "  'NLMid': '8506819',\n",
              "  'Titre': 'Anal. Quant. Cytol. Histol.',\n",
              "  'abstract': \"To develop expert systems for classification of follicular thyroid tumor at a preoperative stage.|Fine needle aspiration biopsy of the thyroid gland with a histologic conclusion of follicular cancer and follicular adenoma were the object of the morphometric study. General sample size was 4500 nuclei and 3000 aggregates.|Quantitative regularities of pathologic changes in thyrocyte nuclei and aggregates in follicular cancer and follicular adenoma were revealed. Threshold values and weighting coefficients of quantitative features of thyrocyte nuclei and aggregates characterizing cancer made the basis of the two expert systems. Expert systems included standard 2-D S-matrix containing threshold values of nuclei and aggregates in cancer and their weighting coefficients as well as 1-D scientific X-matrix designed for filling with quantitative features of the studied object. The diagnosis was verified by the value of a diagnostic index by means of comparing feature values in the corresponding elements of S- and X-matrices. After that, a diagnostic index was calculated taking into account the features' weighting coefficient.|The developed expert systems based on a set of quantitative features of thyrocyte nuclei and aggregates will allow assessing the malignant potential of a follicular thyroid tumor at a preoperative stage.\",\n",
              "  'authors': ['Kirillov, Vladimir', 'Emeliyanova, Olga'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Analytical and quantitative cytology and histology',\n",
              "  'keywords': ['Adenocarcinoma, Follicular',\n",
              "   'Biopsy',\n",
              "   'Cell Aggregation',\n",
              "   'Cell Nucleus',\n",
              "   'Diagnosis, Differential',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Karyometry',\n",
              "   'Models, Biological',\n",
              "   'Thyroid Gland',\n",
              "   'Thyroid Neoplasms'],\n",
              "  'pmid': '22611764',\n",
              "  'title': 'Differential diagnosis of follicular tumor by expert systems based on a set of quantitative features of thyrocyte nuclei and aggregates.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1468-2079',\n",
              "  'IF_2013': '2.809',\n",
              "  'IF_2014': '2.976',\n",
              "  'IF_2015': '3.036',\n",
              "  'IF_2016': '3.806',\n",
              "  'IF_2017': '3.384',\n",
              "  'ISSN': '0007-1161',\n",
              "  'ISSN_online': '1468-2079',\n",
              "  'ISSN_print': '0007-1161',\n",
              "  'IsoAbbr': 'Br J Ophthalmol',\n",
              "  'JrId': 1882,\n",
              "  'MedAbbr': 'Br J Ophthalmol',\n",
              "  'NLMid': '0421041',\n",
              "  'Titre': 'Br J Ophthalmol',\n",
              "  'abstract': \"We investigated whether it was useful to use machine learning algorithms to predict patients' vision related quality of life (VRQoL) from visual field (VF) and visual acuity (VA).|VRQoL was surveyed in 164 glaucomatous patients using the Sumi questionnaire. Their VRQoL score was predicted using machine learning algorithms (Random Forest, gradient boosting, support vector machine) based on total deviation (TD) values from integrated VF (IVF), VA, age and gender. For comparison, VRQoL score was predicted using standard linear regression with mean of IVF, TD values, and VA, and also the stepwise model selection by Akaike Information Criterion. Prediction error was calculated as root mean of the squared prediction error (RMSE) associated with the leave one out cross validation.|RMSEs associated with general VRQoL score were smaller for the machine learning algorithms (1.99 to 2.21) compared with the standard linear model and the stepwise model selection (2.35 to 3.20). A similar tendency was found in each individual VRQoL task score.|We found that it was advantageous to use machine learning methods to predict VRQoL accurately. These statistical methods could be used to help clinicians better understand patients' VRQoL without the need for extra tests other than standard VA and VF.\",\n",
              "  'authors': ['Hirasawa, Hiroyo',\n",
              "   'Murata, Hiroshi',\n",
              "   'Mayama, Chihiro',\n",
              "   'Araie, Makoto',\n",
              "   'Asaoka, Ryo'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'The British journal of ophthalmology',\n",
              "  'keywords': ['Activities of Daily Living',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Artificial Intelligence',\n",
              "   'Female',\n",
              "   'Glaucoma',\n",
              "   'Health Status Indicators',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Psychometrics',\n",
              "   'Quality of Life',\n",
              "   'Visual Acuity',\n",
              "   'Visual Field Tests',\n",
              "   'Visual Fields'],\n",
              "  'pmid': '24795333',\n",
              "  'title': 'Evaluation of various machine learning methods to predict vision-related quality of life from visual field data and visual acuity in patients with glaucoma.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1432-1106',\n",
              "  'IF_2013': '2.168',\n",
              "  'IF_2014': '2.036',\n",
              "  'IF_2015': '2.057',\n",
              "  'IF_2016': '1.917',\n",
              "  'IF_2017': '1.806',\n",
              "  'ISSN': '0014-4819',\n",
              "  'ISSN_online': '1432-1106',\n",
              "  'ISSN_print': '0014-4819',\n",
              "  'IsoAbbr': 'Exp Brain Res',\n",
              "  'JrId': 3641,\n",
              "  'MedAbbr': 'Exp Brain Res',\n",
              "  'NLMid': '0043312',\n",
              "  'Titre': 'Exp Brain Res',\n",
              "  'abstract': 'Robotic guidance is an engineered form of haptic-guidance training and intended to enhance motor learning in rehabilitation, surgery, and sports. However, its benefits (and pitfalls) are still debated. Here, we investigate the effects of different presentation modes on the reproduction of a spatiotemporal movement pattern. In three different groups of participants, the movement was demonstrated in three different modalities, namely visual, haptic, and visuo-haptic. After demonstration, participants had to reproduce the movement in two alternating recall conditions: haptic and visuo-haptic. Performance of the three groups during recall was compared with regard to spatial and dynamic movement characteristics. After haptic presentation, participants showed superior dynamic accuracy, whereas after visual presentation, participants performed better with regard to spatial accuracy. Added visual feedback during recall always led to enhanced performance, independent of the movement characteristic and the presentation modality. These findings substantiate the different benefits of different presentation modes for different movement characteristics. In particular, robotic guidance is beneficial for the learning of dynamic, but not of spatial movement characteristics.',\n",
              "  'authors': ['Lüttgen, Jenna', 'Heuer, Herbert'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Experimental brain research',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Male',\n",
              "   'Mental Recall',\n",
              "   'Movement',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Photic Stimulation',\n",
              "   'Psychomotor Performance',\n",
              "   'Robotics',\n",
              "   'Space Perception',\n",
              "   'Time Factors',\n",
              "   'Touch',\n",
              "   'Young Adult'],\n",
              "  'pmid': '22836521',\n",
              "  'title': 'Robotic guidance benefits the learning of dynamic, but not of spatial movement characteristics.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': 'Fingerprint recognition is one of the most widely used methods of biometrics. This method relies on the surface topography of a finger and, thus, is potentially vulnerable for spoofing by artificial dummies with embedded fingerprints. In this study, we applied the optical coherence tomography (OCT) technique to distinguish artificial materials commonly used for spoofing fingerprint scanning systems from the real skin. Several artificial fingerprint dummies made from household cement and liquid silicone rubber were prepared and tested using a commercial fingerprint reader and an OCT system. While the artificial fingerprints easily spoofed the commercial fingerprint reader, OCT images revealed the presence of them at all times. We also demonstrated that an autocorrelation analysis of the OCT images could be potentially used in automatic recognition systems.',\n",
              "  'authors': ['Cheng, Yezeng', 'Larin, Kirill V'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artifacts',\n",
              "   'Artificial Intelligence',\n",
              "   'Biometry',\n",
              "   'Computer Security',\n",
              "   'Dermatoglyphics',\n",
              "   'Fingers',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Skin',\n",
              "   'Statistics as Topic',\n",
              "   'Tomography, Optical Coherence'],\n",
              "  'pmid': '17151765',\n",
              "  'title': 'Artificial fingerprint recognition by using optical coherence tomography with autocorrelation analysis.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'To compare the outcomes and cost of open and robotic-assisted ileovesicostomy techniques for the adult neurogenic bladder patient.|Consecutive open and robotic-assisted ileovesicostomy procedures were retrospectively reviewed for demographic, operative, and postoperative recovery data. Surgical outcome was assessed by examining the incidence of postprocedure urinary incontinence, urinary tract infections (UTIs), and upper tract compromise. Total cost was calculated through summation of inpatient costs, including room/board, operating/recovery room, surgical supplies, professional fees, intensive care unit, and robotic maintenance.|Fifteen ileovesicostomy procedures (7 open, 8 robotic) were reviewed. Both groups had similar demographic and urodynamic data. Operative times (293 min open vs 330 min robotic, P = .24) were similar between techniques. There were trends toward lower operative blood loss (100 mL vs 257 mL, P = .09) and shorter hospital stays in the robotic group (8 days vs 11 days, P = .14). Ileovesicostomy was associated with improved urinary continence (P = .02) and trended toward a decreased incidence of postoperative chronic UTI (P = .13) for the entire group, and there was no difference between techniques regarding continence, chronic UTIs, and complications. No patients in either group developed postoperative hydronephrosis. Total inpatient cost for the open and robotic groups was $14,356 and $17,344 (P = .05), which differed primarily because of higher robotic operating room supply costs ($609 vs $3770, P <.001).|Robotic and open ileovesicostomy had similar surgical outcomes in this patient cohort, although total inpatient costs were significantly higher in the robotic group.',\n",
              "  'authors': ['Vanni, Alex J', 'Stoffel, John T'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Cohort Studies',\n",
              "   'Critical Care',\n",
              "   'Cystostomy',\n",
              "   'Female',\n",
              "   'Health Care Costs',\n",
              "   'Humans',\n",
              "   'Ileostomy',\n",
              "   'Intraoperative Complications',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Bladder, Neurogenic',\n",
              "   'Urodynamics',\n",
              "   'Urologic Surgical Procedures'],\n",
              "  'pmid': '21146864',\n",
              "  'title': 'Ileovesicostomy for the neurogenic bladder patient: outcome and cost comparison of open and robotic assisted techniques.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RB'],\n",
              "  'ESSN': '2169-5180',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '6.13',\n",
              "  'IF_2016': '8.649',\n",
              "  'IF_2017': '5.057',\n",
              "  'ISSN': '2169-5172',\n",
              "  'ISSN_online': '2169-5180',\n",
              "  'ISSN_print': '2169-5172',\n",
              "  'IsoAbbr': 'Soft Robot',\n",
              "  'JrId': 45563,\n",
              "  'MedAbbr': 'Soft Robot',\n",
              "  'NLMid': '101623819',\n",
              "  'Titre': 'Soft Robot',\n",
              "  'abstract': 'There is increasing interest in the use of soft materials in robotic applications ranging from wearable devices to soft grippers. While soft structures provide a number of favorable properties to robotic systems, sensing of large deformable soft structures is still a considerable challenge; sensors must not inhibit the mechanical properties of the soft body, and the potential infinite degree-of-freedom deformations mean that there is an intrinsically limited resolution of the sensing receptors. An approach to address these challenges using a conductive thermoplastic elastomer is proposed. This allows sensory strain information to be gained from deforming structures without disturbing the dynamics of the system enabling coverage of large soft surfaces. In this article, a theoretical framework is developed, which provides a set of design principles to optimize and characterize sensor implementation, allowing maximum information about location, posture, and shape of the object to be determined. The proposed approach has been tested experimentally for the case study of the universal gripper; investigating how a sensorized gripper can allow a robot to identify grasped objects to enable improved gripping and manipulation performance.',\n",
              "  'authors': ['Hughes, Josie', 'Iida, Fumiya'],\n",
              "  'categories': ['ROBOTICS'],\n",
              "  'journal': 'Soft robotics',\n",
              "  'keywords': ['Elastomers',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Touch',\n",
              "   'Wearable Electronic Devices'],\n",
              "  'pmid': '30256733',\n",
              "  'title': 'Tactile Sensing Applied to the Universal Gripper Using Conductive Thermoplastic Elastomer.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1522-726X',\n",
              "  'IF_2013': '2.396',\n",
              "  'IF_2014': '2.107',\n",
              "  'IF_2015': '2.181',\n",
              "  'IF_2016': '2.602',\n",
              "  'IF_2017': '2.602',\n",
              "  'ISSN': '1522-1946',\n",
              "  'ISSN_online': '1522-726X',\n",
              "  'ISSN_print': '1522-1946',\n",
              "  'IsoAbbr': 'Catheter Cardiovasc Interv',\n",
              "  'JrId': 21369,\n",
              "  'MedAbbr': 'Catheter Cardiovasc Interv',\n",
              "  'NLMid': '100884139',\n",
              "  'Titre': 'Catheter Cardiovasc Interv',\n",
              "  'abstract': 'Robotic percutaneous coronary interventions have recently been introduced in the cardiac catheterization laboratory. Robotics offers benefits of greater precision for stent placement and occupational hazard protection for operators and staff. First generation systems were able to advance and retract coronary wires, balloons, and stents, but did not have guide control functions. The second-generation robotic system (CorPath GRX) has an active guide management function offering the ability to move guide catheters. Expanding utilization of robotics to perform diagnostic coronary angiography would further reduce radiation scatter exposure and other occupational hazards to operators. This approach is particularly appealing in the setting of radial access, as universal radial diagnostic catheters can engage both the right and left coronary arteries without exchange. We describe here, the first two cases of such a procedure with the CorPath GRX robotic system.',\n",
              "  'authors': ['Swaminathan, Rajesh V', 'Rao, Sunil V'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Catheterization and cardiovascular interventions : official journal of the Society for Cardiac Angiography & Interventions',\n",
              "  'keywords': ['Aged',\n",
              "   'Cardiac Catheters',\n",
              "   'Catheterization, Peripheral',\n",
              "   'Coronary Angiography',\n",
              "   'Coronary Artery Disease',\n",
              "   'Coronary Vessels',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Predictive Value of Tests',\n",
              "   'Punctures',\n",
              "   'Radial Artery',\n",
              "   'Radiation Dosage',\n",
              "   'Radiation Exposure',\n",
              "   'Robotics',\n",
              "   'Time Factors'],\n",
              "  'pmid': '29314560',\n",
              "  'title': 'Robotic-assisted transradial diagnostic coronary angiography.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1873-6424',\n",
              "  'IF_2013': '3.902',\n",
              "  'IF_2014': '4.143',\n",
              "  'IF_2015': '4.839',\n",
              "  'IF_2016': '5.099',\n",
              "  'IF_2017': '4.358',\n",
              "  'ISSN': '0269-7491',\n",
              "  'ISSN_online': '1873-6424',\n",
              "  'ISSN_print': '0269-7491',\n",
              "  'IsoAbbr': 'Environ. Pollut.',\n",
              "  'JrId': 22002,\n",
              "  'MedAbbr': 'Environ Pollut',\n",
              "  'NLMid': '8804476',\n",
              "  'Titre': 'Environ. Pollut.',\n",
              "  'abstract': 'Soil heavy metal pollution due to long-term sewage irrigation is a serious environmental problem in many irrigation areas in northern China. Quickly identifying its pollution status is an important basis for remediation. Visible-near-infrared reflectance spectroscopy (VNIRS) provides a useful tool. In a case study, 76 soil samples were collected and their reflectance spectra were used to estimate cadmium (Cd) concentration by partial least squares regression (PLSR) and back propagation neural network (BPNN). To reduce noise, six pre-treatments were compared, in which orthogonal signal correction (OSC) was first used in soil Cd estimation. Spectral analysis and geostatistics were combined to identify Cd pollution hotspots. Results showed that Cd was accumulated in topsoil at the study area. OSC can effectively remove irrelevant information to improve prediction accuracy. More accurate estimation was achieved by applying a BPNN. Soil Cd pollution hotspots could be identified by interpolating the predicted values obtained from spectral estimates. ',\n",
              "  'authors': ['Chen, Tao',\n",
              "   'Chang, Qingrui',\n",
              "   'Clevers, J G P W',\n",
              "   'Kooistra, L'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental pollution (Barking, Essex : 1987)',\n",
              "  'keywords': ['Agricultural Irrigation',\n",
              "   'Cadmium',\n",
              "   'China',\n",
              "   'Environmental Monitoring',\n",
              "   'Environmental Pollution',\n",
              "   'Least-Squares Analysis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sewage',\n",
              "   'Soil',\n",
              "   'Soil Pollutants',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Time Factors'],\n",
              "  'pmid': '26188912',\n",
              "  'title': 'Rapid identification of soil cadmium pollution risk at regional scale based on visible and near-infrared spectroscopy.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Traditional robotic work cell design and programming are considered inefficient and outdated in current industrial and market demands. In this research, virtual reality (VR) technology is used to improve human-robot interface, whereby complicated commands or programming knowledge is not required. The proposed solution, known as VR-based Programming of a Robotic Work Cell (VR-Rocell), consists of two sub-programmes, which are VR-Robotic Work Cell Layout (VR-RoWL) and VR-based Robot Teaching System (VR-RoT). VR-RoWL is developed to assign the layout design for an industrial robotic work cell, whereby VR-RoT is developed to overcome safety issues and lack of trained personnel in robot programming. Simple and user-friendly interfaces are designed for inexperienced users to generate robot commands without damaging the robot or interrupting the production line. The user is able to attempt numerous times to attain an optimum solution. A case study is conducted in the Robotics Laboratory to assemble an electronics casing and it is found that the output models are compatible with commercial software without loss of information. Furthermore, the generated KUKA commands are workable when loaded into a commercial simulator. The operation of the actual robotic work cell shows that the errors may be due to the dynamics of the KUKA robot rather than the accuracy of the generated programme. Therefore, it is concluded that the virtual reality based solution approach can be implemented in an industrial robotic work cell. ',\n",
              "  'authors': ['Yap, Hwa Jen',\n",
              "   'Taha, Zahari',\n",
              "   'Dawal, Siti Zawiah Md',\n",
              "   'Chang, Siow-Wee'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Laboratories',\n",
              "   'Manufacturing Industry',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Software',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '25360663',\n",
              "  'title': 'Virtual reality based support system for layout planning and programming of an industrial robotic work cell.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1473-656X',\n",
              "  'IF_2013': '2.37',\n",
              "  'IF_2014': '2.067',\n",
              "  'IF_2015': '2.134',\n",
              "  'IF_2016': '2.416',\n",
              "  'IF_2017': '2.821',\n",
              "  'ISSN': '1040-872X',\n",
              "  'ISSN_online': '1473-656X',\n",
              "  'ISSN_print': '1040-872X',\n",
              "  'IsoAbbr': 'Curr. Opin. Obstet. Gynecol.',\n",
              "  'JrId': 1147,\n",
              "  'MedAbbr': 'Curr Opin Obstet Gynecol',\n",
              "  'NLMid': '9007264',\n",
              "  'Titre': 'Curr. Opin. Obstet. Gynecol.',\n",
              "  'abstract': \"To analyze the recent evidence on robotic hysterectomy while highlighting its benefits and challenges.|Increased rates of robotic hysterectomy have led to decreasing rates of abdominal hysterectomy, after rates of the latter approach have been stagnant for many years. Robotic surgery has also the possible advantage of a relatively short learning curve, even though the case number required to reach proficiency may be actually closer to 100 cases. Recent studies comparing robotic and laparoscopic hysterectomy for benign indications have not demonstrated a clear advantage for either approach in terms of complications, blood loss, and hospital stay. The higher cost of robotic hysterectomy remains a significant disadvantage of this surgical approach, although the total cost may decrease with increasing surgeon's experience (via shorter operative time) and may be offset in some circumstances by reduced hospital stay and cost of complications compared with abdominal hysterectomy.|The place of robotic hysterectomy in the gynecologic surgical armamentarium is still evolving. Although recent studies highlight the comparative outcomes of robotic and laparoscopic hysterectomy for benign cases, most surgeons are unlikely to be equally proficient in both techniques. Future studies will need to question whether subgroups of patients with complex benign disease such as endometriosis and pelvic adhesive disease may benefit from the robotic assistance.\",\n",
              "  'authors': ['Smorgick, Noam', 'As-Sanie, Sawsan'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Current opinion in obstetrics & gynecology',\n",
              "  'keywords': ['Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Laparoscopy',\n",
              "   'Learning Curve',\n",
              "   'Operative Time',\n",
              "   'Randomized Controlled Trials as Topic',\n",
              "   'Risk Assessment',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24914631',\n",
              "  'title': 'The benefits and challenges of robotic-assisted hysterectomy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1522-2586',\n",
              "  'IF_2013': '2.788',\n",
              "  'IF_2014': '3.21',\n",
              "  'IF_2015': '3.25',\n",
              "  'IF_2016': '3.083',\n",
              "  'IF_2017': '3.612',\n",
              "  'ISSN': '1053-1807',\n",
              "  'ISSN_online': '1522-2586',\n",
              "  'ISSN_print': '1053-1807',\n",
              "  'IsoAbbr': 'J Magn Reson Imaging',\n",
              "  'JrId': 2081,\n",
              "  'MedAbbr': 'J Magn Reson Imaging',\n",
              "  'NLMid': '9105850',\n",
              "  'Titre': 'J Magn Reson Imaging',\n",
              "  'abstract': \"Diffusion-weighted imaging (DWI) in MRI plays an increasingly important role in diagnostic applications and developing imaging biomarkers. Automated whole-breast segmentation is an important yet challenging step for quantitative breast imaging analysis. While methods have been developed on dynamic contrast-enhanced (DCE) MRI, automatic whole-breast segmentation in breast DWI MRI is still underdeveloped.|To develop a deep/transfer learning-based segmentation approach for DWI MRI scans and conduct an extensive study assessment on four imaging datasets from both internal and external sources.|Retrospective.|In all, 98 patients (144 MRI scans; 11,035 slices) of four different breast MRI datasets from two different institutions.|1.5T scanners with DCE sequence (Dataset 1 and Dataset 2) and DWI sequence. A 3.0T scanner with one external DWI sequence.|Deep learning models (UNet and SegNet) and transfer learning were used as segmentation approaches. The main DCE Dataset (4,251 2D slices from 39 patients) was used for pre-training and internal validation, and an unseen DCE Dataset (431 2D slices from 20 patients) was used as an independent test dataset for evaluating the pre-trained DCE models. The main DWI Dataset (6,343 2D slices from 75 MRI scans of 29 patients) was used for transfer learning and internal validation, and an unseen DWI Dataset (10 2D slices from 10 patients) was used for independent evaluation to the fine-tuned models for DWI segmentation. Manual segmentations by three radiologists (>10-year experience) were used to establish the ground truth for assessment. The segmentation performance was measured using the Dice Coefficient (DC) for the agreement between manual expert radiologist's segmentation and algorithm-generated segmentation.|The mean value and standard deviation of the DCs were calculated to compare segmentation results from different deep learning models.|For the segmentation on the DCE MRI, the average DC of the UNet was 0.92 (cross-validation on the main DCE dataset) and 0.87 (external evaluation on the unseen DCE dataset), both higher than the performance of the SegNet. When segmenting the DWI images by the fine-tuned models, the average DC of the UNet was 0.85 (cross-validation on the main DWI dataset) and 0.72 (external evaluation on the unseen DWI dataset), both outperforming the SegNet on the same datasets.|The internal and independent tests show that the deep/transfer learning models can achieve promising segmentation effects validated on DWI data from different institutions and scanner types. Our proposed approach may provide an automated toolkit to help computer-aided quantitative analyses of breast DWI images.|3 Technical Efficacy: Stage 2 J. Magn. Reson. Imaging 2019.\",\n",
              "  'authors': ['Zhang, Lei',\n",
              "   'Mohamed, Aly A',\n",
              "   'Chai, Ruimei',\n",
              "   'Guo, Yuan',\n",
              "   'Zheng, Bingjie',\n",
              "   'Wu, Shandong'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of magnetic resonance imaging : JMRI',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31301201',\n",
              "  'title': 'Automated deep learning method for whole-breast segmentation in diffusion-weighted breast MRI.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1433-0458',\n",
              "  'IF_2013': '0.538',\n",
              "  'IF_2014': '0.58',\n",
              "  'IF_2015': '0.852',\n",
              "  'IF_2016': '0.723',\n",
              "  'IF_2017': '0.893',\n",
              "  'ISSN': '0017-6192',\n",
              "  'ISSN_online': '1433-0458',\n",
              "  'ISSN_print': '0017-6192',\n",
              "  'IsoAbbr': 'HNO',\n",
              "  'JrId': 4047,\n",
              "  'MedAbbr': 'HNO',\n",
              "  'NLMid': '2985099R',\n",
              "  'Titre': 'HNO',\n",
              "  'abstract': 'Ever faster processor capacity is having an impact on computer-assisted or computer-aided surgery (CAS). The fusion of different imaging modalities enables functional data such as PET-CT, for example, to be available in image-guided surgery. Referencing of image data is the key to precise navigation. Intraoperative data acquisition is a new approach to improving accuracy. Thus, intraoperative CT conducted under navigational support enables automatic referencing of up-to-date image data. Alternatively, intraoperative magnetic resonance imaging or intraoperative sonography can be performed. Ultrasound systems have already been successfully integrated in existing navigational systems to compensate for intraoperative tissue shifting. Ultrasound systems may play a role in the future as a single modality in image-guided surgery in soft tissue of the neck and skull bone.',\n",
              "  'authors': ['Federspil, P A'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'HNO',\n",
              "  'keywords': ['Humans',\n",
              "   'Osteotomy',\n",
              "   'Otorhinolaryngologic Surgical Procedures',\n",
              "   'Radiography, Interventional',\n",
              "   'Robotics',\n",
              "   'Subtraction Technique',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '19711045',\n",
              "  'title': '[New developments in computer-assisted surgery (CAS). From intraoperative imaging to ultrasound-based navigation].',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Due to the complexity of the underlying biological processes, gene expression data obtained from DNA microarray technologies are typically noisy and have very high dimensionality and these make the mining of such data for gene function prediction very difficult. To tackle these difficulties, we propose to use an incremental fuzzy mining technique called incremental fuzzy mining (IFM). By transforming quantitative expression values into linguistic terms, such as highly or lowly expressed, IFM can effectively capture heterogeneity in expression data for pattern discovery. It does so using a fuzzy measure to determine if interesting association patterns exist between the linguistic gene expression levels. Based on these patterns, IFM can make accurate gene function predictions and these predictions can be made in such a way that each gene can be allowed to belong to more than one functional class with different degrees of membership. Gene function prediction problem can be formulated both as classification and clustering problems, and IFM can be used either as a classification technique or together with existing clustering algorithms to improve the cluster groupings discovered for greater prediction accuracies. IFM is characterized also by its being an incremental data mining technique so that the discovered patterns can be continually refined based only on newly collected data without the need for retraining using the whole dataset. For performance evaluation, IFM has been tested with real expression datasets for both classification and clustering tasks. Experimental results show that it can effectively uncover hidden patterns for accurate gene function predictions.',\n",
              "  'authors': ['Ma, Patrick C H', 'Chan, Keith C C'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cluster Analysis',\n",
              "   'Computational Biology',\n",
              "   'Data Mining',\n",
              "   'Databases, Genetic',\n",
              "   'Fibroblasts',\n",
              "   'Fuzzy Logic',\n",
              "   'Gene Expression Profiling',\n",
              "   'Genes',\n",
              "   'Humans',\n",
              "   'Models, Statistical',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '20403777',\n",
              "  'title': 'Incremental fuzzy mining of gene expression data for gene function prediction.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'This paper considers a decentralized fault tolerant control (DFTC) scheme for reconfigurable manipulators. With the appearance of norm-bounded failure, a dual closed-loop trajectory tracking control algorithm is proposed on the basis of the Lyapunov stability theory. Characterized by the modularization property, the actuator failure is estimated by the proposed decentralized sliding mode observer (DSMO). Moreover, the actuator failure can be treated in view of the local joint information, so its control performance degradation is independent of other normal joints. In addition, the presented DFTC scheme is significantly simplified in terms of the structure of the controller due to its dual closed-loop architecture, and its feasibility is highly reflected in the control of reconfigurable manipulators. Finally, the effectiveness of the proposed DFTC scheme is demonstrated using simulations. ',\n",
              "  'authors': ['Zhao, Bo', 'Li, Chenghao', 'Liu, Derong', 'Li, Yuanchun'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Humans',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Quality Control',\n",
              "   'Robotics'],\n",
              "  'pmid': '26181826',\n",
              "  'title': 'Decentralized Sliding Mode Observer Based Dual Closed-Loop Fault Tolerant Control for Reconfigurable Manipulator against Actuator Failure.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1873-3476',\n",
              "  'IF_2013': '3.785',\n",
              "  'IF_2014': '3.65',\n",
              "  'IF_2015': '3.994',\n",
              "  'IF_2016': '3.649',\n",
              "  'IF_2017': '3.862',\n",
              "  'ISSN': '0378-5173',\n",
              "  'ISSN_online': '1873-3476',\n",
              "  'ISSN_print': '0378-5173',\n",
              "  'IsoAbbr': 'Int J Pharm',\n",
              "  'JrId': 21201,\n",
              "  'MedAbbr': 'Int J Pharm',\n",
              "  'NLMid': '7804127',\n",
              "  'Titre': 'Int J Pharm',\n",
              "  'abstract': 'The aim of this work was to evaluate and optimize formulation of three-layer matrix tablets based on xanthan gum (XG) and sodium alginate for chronotherapeutic pH-independent release of verapamil HCl (VH). Artificial neural networks (ANN) were applied in the optimization and compared with multiple linear regression (MLR). A face-centered central composite experimental design was employed with three factors (mass fraction of VH in intermediate layer, X1, and of XG in matrix former of intermediate and outer layers, X2 and X3). The prepared tablets were tested for in vitro release in 0.1 N HCl and phosphate buffer (pH 7.5), tensile strength and friability. Furthermore, swelling observation and release modeling to Weibull function and power law equation of Peppas were employed to help further understanding of release behavior and mechanism. The releases (%) in phosphate buffer (pH 7.5) at 6, 12 and 24 h were selected as responses to depict the mode of release and similarity factor (f2), between release profiles in 0.1N HCl and pH 7.5 during the first 8 h, as response of pH-independence. A desirability function combining the four responses was constructed and overall desirability values were used for the ANN and MLR modeling. Five additional checkpoint formulations, within the experimental domain, were used to validate the external predictability of the models. The constructed ANN model fitted better to the overall desirability than the MLR model (R=0.838 vs. 0.670, for the additional checkpoint formulations) and therefore, was used for prediction of formulation with optimal in vitro drug release.',\n",
              "  'authors': ['Al-Zoubi, Nizar',\n",
              "   'Alkhatib, Hatim S',\n",
              "   'Alobaidi, Ghadah',\n",
              "   'Abdel-Rahim, Safwan',\n",
              "   'Obeidat, Wasfy',\n",
              "   'Malamataris, Stavros'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'International journal of pharmaceutics',\n",
              "  'keywords': ['Alginates',\n",
              "   'Chemistry, Pharmaceutical',\n",
              "   'Delayed-Action Preparations',\n",
              "   'Drug Chronotherapy',\n",
              "   'Drug Liberation',\n",
              "   'Glucuronic Acid',\n",
              "   'Hexuronic Acids',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Polysaccharides, Bacterial',\n",
              "   'Tablets',\n",
              "   'Verapamil'],\n",
              "  'pmid': '26276259',\n",
              "  'title': 'Optimization of pH-independent chronotherapeutic release of verapamil HCl from three-layer matrix tablets.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1098-1004',\n",
              "  'IF_2013': '5.05',\n",
              "  'IF_2014': '5.144',\n",
              "  'IF_2015': '5.089',\n",
              "  'IF_2016': '4.601',\n",
              "  'IF_2017': '5.359',\n",
              "  'ISSN': '1059-7794',\n",
              "  'ISSN_online': '1098-1004',\n",
              "  'ISSN_print': '1059-7794',\n",
              "  'IsoAbbr': 'Hum. Mutat.',\n",
              "  'JrId': 2408,\n",
              "  'MedAbbr': 'Hum Mutat',\n",
              "  'NLMid': '9215429',\n",
              "  'Titre': 'Hum. Mutat.',\n",
              "  'abstract': 'Most diseases, including those of genetic origin, express a continuum of severity. Clinical interventions for numerous diseases are based on the severity of the phenotype. Predicting severity due to genetic variants could facilitate diagnosis and choice of therapy. Although computational predictions have been used as evidence for classifying the disease relevance of genetic variants, special tools for predicting disease severity in large scale are missing. Here, we manually curated a dataset containing variants leading to severe and less severe phenotypes and studied the abilities of variation impact predictors to distinguish between them. We found that these tools cannot separate the two groups of variants. Then, we developed a novel machine-learning-based method, PON-PS (http://structure.bmc.lu.se/PON-PS), for the classification of amino acid substitutions associated with benign, severe, and less severe phenotypes. We tested the method using an independent test dataset and variants in four additional proteins. For distinguishing severe and nonsevere variants, PON-PS showed an accuracy of 61% in the test dataset, which is higher than for existing tolerance prediction methods. PON-PS is the first generic tool developed for this task. The tool can be used together with other evidence for improving diagnosis and prognosis and for prioritization of preventive interventions, clinical monitoring, and molecular tests.',\n",
              "  'authors': ['Niroula, Abhishek', 'Vihinen, Mauno'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Human mutation',\n",
              "  'keywords': ['Amino Acid Substitution',\n",
              "   'Computational Biology',\n",
              "   'Disease',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Genetic Variation',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Prognosis',\n",
              "   'Proteins',\n",
              "   'Reproducibility of Results',\n",
              "   'Severity of Illness Index'],\n",
              "  'pmid': '28070986',\n",
              "  'title': 'Predicting Severity of Disease-Causing Variants.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Using traditional approaches, a brain-computer interface (BCI) requires the collection of calibration data for new subjects prior to online use. Calibration time can be reduced or eliminated e.g., by subject-to-subject transfer of a pre-trained classifier or unsupervised adaptive classification methods which learn from scratch and adapt over time. While such heuristics work well in practice, none of them can provide theoretical guarantees. Our objective is to modify an event-related potential (ERP) paradigm to work in unison with the machine learning decoder, and thus to achieve a reliable unsupervised calibrationless decoding with a guarantee to recover the true class means.|We introduce learning from label proportions (LLP) to the BCI community as a new unsupervised, and easy-to-implement classification approach for ERP-based BCIs. The LLP estimates the mean target and non-target responses based on known proportions of these two classes in different groups of the data. We present a visual ERP speller to meet the requirements of LLP. For evaluation, we ran simulations on artificially created data sets and conducted an online BCI study with 13 subjects performing a copy-spelling task.|Theoretical considerations show that LLP is guaranteed to minimize the loss function similar to a corresponding supervised classifier. LLP performed well in simulations and in the online application, where 84.5% of characters were spelled correctly on average without prior calibration.|The continuously adapting LLP classifier is the first unsupervised decoder for ERP BCIs guaranteed to find the optimal decoder. This makes it an ideal solution to avoid tedious calibration sessions. Additionally, LLP works on complementary principles compared to existing unsupervised methods, opening the door for their further enhancement when combined with LLP.',\n",
              "  'authors': ['Hübner, David',\n",
              "   'Verhoeven, Thibault',\n",
              "   'Schmid, Konstantin',\n",
              "   'Müller, Klaus-Robert',\n",
              "   'Tangermann, Michael',\n",
              "   'Kindermans, Pieter-Jan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Brain-Computer Interfaces',\n",
              "   'Electroencephalography',\n",
              "   'Evoked Potentials',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Male',\n",
              "   'Unsupervised Machine Learning',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '28407016',\n",
              "  'title': 'Learning from label proportions in brain-computer interfaces: Online unsupervised learning with guarantees.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1879-5447',\n",
              "  'IF_2013': '0.848',\n",
              "  'IF_2014': '0.882',\n",
              "  'IF_2015': '0.676',\n",
              "  'IF_2016': '1.171',\n",
              "  'IF_2017': '1.032',\n",
              "  'ISSN': '0158-9938',\n",
              "  'ISSN_online': '1879-5447',\n",
              "  'ISSN_print': '0158-9938',\n",
              "  'IsoAbbr': 'Australas Phys Eng Sci Med',\n",
              "  'JrId': 930,\n",
              "  'MedAbbr': 'Australas Phys Eng Sci Med',\n",
              "  'NLMid': '8208130',\n",
              "  'Titre': 'Australas Phys Eng Sci Med',\n",
              "  'abstract': \"QRS complexes were classified using a Multi Layer Perceptron (MLP) using a modified version of the original Backpropagation (BP) algorithm. The input to the network were bitmaps of the QRS complexes represented in the form of a 20 x 20 matrix. A number of hidden layers (and neurons in each hidden layer) were experimented upon to observe the rate at which the network converged. Larger Networks were observed to find the minimum on the error curve with ease. Increasing the network size beyond a certain size did not improve the performance rate, rather it decreased the performance rate. It is evident that there exists an optimal neural network architecture for every given problem. The weights change rules in Backpropagation algorithm were modified to include a variation of the relationship between momentum and learning rate to observe any increase in network's performance rate. A learning rate adaptation factor was introduced into the learning algorithm to decrease the network's chances of missing a minimum on the error curve. The network was found to perform extremely well with the modified version of the algorithm. The network converged after only 9000 learning cycles when compared to 14,000 cycles with the original algorithm.\",\n",
              "  'authors': ['Mahalingam, N', 'Kumar, D'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Australasian physical & engineering sciences in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biomedical Engineering',\n",
              "   'Electrocardiography',\n",
              "   'Humans',\n",
              "   'Models, Cardiovascular',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '9409015',\n",
              "  'title': 'Neural networks for signal processing applications: ECG classification.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1897-4260',\n",
              "  'IF_2013': '0.537',\n",
              "  'IF_2014': '0.641',\n",
              "  'IF_2015': '0.747',\n",
              "  'IF_2016': '0.857',\n",
              "  'IF_2017': '0.817',\n",
              "  'ISSN': '0028-3843',\n",
              "  'ISSN_online': '1897-4260',\n",
              "  'ISSN_print': '0028-3843',\n",
              "  'IsoAbbr': 'Neurol. Neurochir. Pol.',\n",
              "  'JrId': 6061,\n",
              "  'MedAbbr': 'Neurol Neurochir Pol',\n",
              "  'NLMid': '0101265',\n",
              "  'Titre': 'Neurol. Neurochir. Pol.',\n",
              "  'abstract': \"Due to its high complexity, neurosurgery consists of a demanding learning curve that requires intense training and a deep knowledge of neuroanatomy. Microsurgical skill development can be achieved through various models of simulation, but as human cadaveric models are not always accessible, cadaveric animal models can provide a reliable environment in which to enhance the acquisition of surgical dexterity. The aim of this review was to analyse the current role of animal brains in laboratory training and to assess their correspondence to the procedures performed in humans.|A Pubmed literature search was performed to identify all the articles concerning training cranial and spinal techniques on large animal heads. The search terms were 'training model', and 'neurosurgery' in association with 'animal', 'sheep', 'cow', and 'swine'. The exclusion criteria were articles that were on human brains, experimental fundamental research, or on virtual simulators.|The search retrieved 119 articles, of which 25 were relevant to the purpose of this review. Owing to their similar neuroanatomy, bovine, porcine and ovine models prove to be reliable structures in simulating neurosurgical procedures. On bovine skulls, an interhemispheric transcalosal and retrosigmoid approach along with different approaches to the Circle of Willis can be recreated. Ovine model procedures have varied from lumbar discectomies on sheep spines to craniosynostosis surgery, whereas in ex vivo swine models, cadaveric dissections of lateral sulcus, median and posterior fossa have been achieved.|Laboratory training models enhance surgical advancements by familiarising trainee surgeons with certain neuroanatomical structures and promoting greater surgical dexterity. The accessibility of animal brains allows trainee surgeons to exercise techniques outside the operating theatre, thus optimising outcomes in human surgical procedures.\",\n",
              "  'authors': ['Morosanu, Cezar Octavian',\n",
              "   'Nicolae, Liviu',\n",
              "   'Moldovan, Remus',\n",
              "   'Farcasanu, Alexandru Stefan',\n",
              "   'Filip, Gabriela Adriana',\n",
              "   'Florian, Ioan Stefan'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Neurologia i neurochirurgia polska',\n",
              "  'keywords': ['Animals',\n",
              "   'Cadaver',\n",
              "   'Humans',\n",
              "   'Neurosurgery',\n",
              "   'Neurosurgical Procedures',\n",
              "   'Skull'],\n",
              "  'pmid': '30614516',\n",
              "  'title': 'Neurosurgical cadaveric and in vivo large animal training models for cranial and spinal approaches and techniques - a systematic review of the current literature.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Members of social network platforms often choose to reveal private information, and thus sacrifice some of their privacy, in exchange for the manifold opportunities and amenities offered by such platforms. In this article, we show that the seemingly innocuous combination of knowledge of confirmed contacts between members on the one hand and their email contacts to non-members on the other hand provides enough information to deduce a substantial proportion of relationships between non-members. Using machine learning we achieve an area under the (receiver operating characteristic) curve (AUC) of at least 0.85 for predicting whether two non-members known by the same member are connected or not, even for conservative estimates of the overall proportion of members, and the proportion of members disclosing their contacts.',\n",
              "  'authors': ['Horvát, Emőke-Ágnes',\n",
              "   'Horvát, Emöke-Ágnes',\n",
              "   'Hanselmann, Michael',\n",
              "   'Hamprecht, Fred A',\n",
              "   'Zweig, Katharina A'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'ROC Curve',\n",
              "   'Social Distance',\n",
              "   'Social Support'],\n",
              "  'pmid': '22493713',\n",
              "  'title': 'One plus one makes three (for social networks).',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1678-2690',\n",
              "  'IF_2013': '0.875',\n",
              "  'IF_2014': '0.734',\n",
              "  'IF_2015': '0.717',\n",
              "  'IF_2016': '0.861',\n",
              "  'IF_2017': '0.956',\n",
              "  'ISSN': '0001-3765',\n",
              "  'ISSN_online': '1678-2690',\n",
              "  'ISSN_print': '0001-3765',\n",
              "  'IsoAbbr': 'An. Acad. Bras. Cienc.',\n",
              "  'JrId': 484,\n",
              "  'MedAbbr': 'An Acad Bras Cienc',\n",
              "  'NLMid': '7503280',\n",
              "  'Titre': 'An. Acad. Bras. Cienc.',\n",
              "  'abstract': 'The study of radiation entrance and exit dynamics and energy consumption in a system is important for understanding the environmental processes that rule the biosphere-atmosphere interactions of all ecosystems. This study provides an analysis of the interaction of energy in the form of photosynthetically active radiation (PAR) in the Pantanal, a Brazilian wetland forest, by studying the variation of PAR reflectance and its interaction with local rainfall. The study site is located in Private Reserve of Natural Heritage, Mato Grosso State, Brazil, where the vegetation is a monodominant forest of Vochysia divergens Phol. The results showed a high correlation between the reflection of visible radiation and rainfall; however, the behavior was not the same at the three heights studied. An analysis of the hourly variation of the reflected waves also showed the seasonality of these phenomena in relation to the dry and rainy seasons. A predictive model for PAR was developed with a neural network that has a hidden layer, and it showed a determination coefficient of 0.938. This model showed that the Julian day and time of measurements had an inverse association with the wind profile and a direct association with the relative humidity profile. ',\n",
              "  'authors': ['Curado, Leone F A',\n",
              "   'Musis, Carlo R DE',\n",
              "   'Cunha, Cristiano R DA',\n",
              "   'Rodrigues, Thiago R',\n",
              "   'Pereira, Vinicius M R',\n",
              "   'Nogueira, José S',\n",
              "   'Sanches, Luciana'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Anais da Academia Brasileira de Ciencias',\n",
              "  'keywords': ['Brazil',\n",
              "   'Forests',\n",
              "   'Humidity',\n",
              "   'Multivariate Analysis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Photosynthesis',\n",
              "   'Radiation',\n",
              "   'Rain',\n",
              "   'Seasons',\n",
              "   'Time Factors',\n",
              "   'Wetlands',\n",
              "   'Wind'],\n",
              "  'pmid': '27556220',\n",
              "  'title': 'Modeling the reflection of Photosynthetically active radiation in a monodominant floodable forest in the Pantanal of Mato Grosso State using multivariate statistics and neural networks.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1618-727X',\n",
              "  'IF_2013': '1.2',\n",
              "  'IF_2014': '1.19',\n",
              "  'IF_2015': '1.406',\n",
              "  'IF_2016': '1.407',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0897-1889',\n",
              "  'ISSN_online': '1618-727X',\n",
              "  'ISSN_print': '0897-1889',\n",
              "  'IsoAbbr': 'J Digit Imaging',\n",
              "  'JrId': 1047,\n",
              "  'MedAbbr': 'J Digit Imaging',\n",
              "  'NLMid': '9100529',\n",
              "  'Titre': 'J Digit Imaging',\n",
              "  'abstract': 'Breast cancer is the most common invasive cancer among women and its incidence is increasing. Risk assessment is valuable and recent methods are incorporating novel biomarkers such as mammographic density. Artificial neural networks (ANN) are adaptive algorithms capable of performing pattern-to-pattern learning and are well suited for medical applications. They are potentially useful for calibrating full-field digital mammography (FFDM) for quantitative analysis. This study uses ANN modeling to estimate volumetric breast density (VBD) from FFDM on Japanese women with and without breast cancer. ANN calibration of VBD was performed using phantom data for one FFDM system. Mammograms of 46 Japanese women diagnosed with invasive carcinoma and 53 with negative findings were analyzed using ANN models learned. ANN-estimated VBD was validated against phantom data, compared intra-patient, with qualitative composition scoring, with MRI VBD, and inter-patient with classical risk factors of breast cancer as well as cancer status. Phantom validations reached an R 2 of 0.993. Intra-patient validations ranged from R 2 of 0.789 with VBD to 0.908 with breast volume. ANN VBD agreed well with BI-RADS scoring and MRI VBD with R 2 ranging from 0.665 with VBD to 0.852 with breast volume. VBD was significantly higher in women with cancer. Associations with age, BMI, menopause, and cancer status previously reported were also confirmed. ANN modeling appears to produce reasonable measures of mammographic density validated with phantoms, with existing measures of breast density, and with classical biomarkers of breast cancer. FFDM VBD is significantly higher in Japanese women with cancer.',\n",
              "  'authors': ['Wang, Jeff',\n",
              "   'Kato, Fumi',\n",
              "   'Yamashita, Hiroko',\n",
              "   'Baba, Motoi',\n",
              "   'Cui, Yi',\n",
              "   'Li, Ruijiang',\n",
              "   'Oyama-Manabe, Noriko',\n",
              "   'Shirato, Hiroki'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of digital imaging',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Density',\n",
              "   'Breast Neoplasms',\n",
              "   'Calibration',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Japan',\n",
              "   'Mammography',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Organ Size',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies'],\n",
              "  'pmid': '27832519',\n",
              "  'title': 'Automatic Estimation of Volumetric Breast Density Using Artificial Neural Network-Based Calibration of Full-Field Digital Mammography: Feasibility on Japanese Women With and Without Breast Cancer.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1365-2648',\n",
              "  'IF_2013': '1.685',\n",
              "  'IF_2014': '1.741',\n",
              "  'IF_2015': '1.917',\n",
              "  'IF_2016': '1.998',\n",
              "  'IF_2017': '2.267',\n",
              "  'ISSN': '0309-2402',\n",
              "  'ISSN_online': '1365-2648',\n",
              "  'ISSN_print': '0309-2402',\n",
              "  'IsoAbbr': 'J Adv Nurs',\n",
              "  'JrId': 4420,\n",
              "  'MedAbbr': 'J Adv Nurs',\n",
              "  'NLMid': '7609811',\n",
              "  'Titre': 'J Adv Nurs',\n",
              "  'abstract': 'To identify, evaluate and describe the psychometric properties of instruments that measure learning outcomes in healthcare students.|Meaningful learning is an active process that enables a wider and deeper understanding of concepts. It is the result of an interaction between new and prior knowledge and produces a long-standing change in knowledge and skills. In the field of education, validated and reliable instruments for assessing meaningful learning are needed.|A psychometric systematic review.|MEDLINE CINAHL, SCOPUS, ERIC, Cochrane Library, Psychology & Behavioural Sciences Collection Database from 1990-December 2013.|Using pre-determined inclusion criteria, three reviewers independently identified studies for full-text review. Then they extracted data for quality appraisal and graded instrument validity using the Consensus-based Standards for the selection of the health status Measurement INstruments checklist and the Psychometric Grading Framework.|Of the 57 studies identified for full-text review, 16 met the inclusion criteria and 13 different instruments were assessed. Following quality assessment, only one instrument was considered of good quality but it measured meaningful learning only in part; the others were either fair or poor. The Psychometric Grading Framework indicated that one instrument was weak, while the others were very weak. No instrument displayed adequate validity.|The systematic review produced a synthesis of the psychometric properties of tools that measure learning outcomes in students of healthcare disciplines. Measuring learning outcomes is very important when educating health professionals. The identified tools may constitute a starting point for the development of other assessment tools.',\n",
              "  'authors': ['Cadorin, Lucia',\n",
              "   'Bagnasco, Annamaria',\n",
              "   'Tolotti, Angela',\n",
              "   'Pagnucci, Nicola',\n",
              "   'Sasso, Loredana'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Journal of advanced nursing',\n",
              "  'keywords': ['Health Personnel',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Psychometrics',\n",
              "   'Students'],\n",
              "  'pmid': '26876619',\n",
              "  'title': 'Instruments for measuring meaningful learning in healthcare students: a systematic psychometric review.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1352-8661',\n",
              "  'IF_2013': '1.353',\n",
              "  'IF_2014': '2.869',\n",
              "  'IF_2015': '2.638',\n",
              "  'IF_2016': '1.718',\n",
              "  'IF_2017': '1.832',\n",
              "  'ISSN': '0968-5243',\n",
              "  'ISSN_online': '1352-8661',\n",
              "  'ISSN_print': '0968-5243',\n",
              "  'IsoAbbr': 'MAGMA',\n",
              "  'JrId': 20119,\n",
              "  'MedAbbr': 'MAGMA',\n",
              "  'NLMid': '9310752',\n",
              "  'Titre': 'MAGMA',\n",
              "  'abstract': 'We aimed to develop the first fully automated 3D gallbladder segmentation approach to perform volumetric analysis in volume data of magnetic resonance (MR) cholangiopancreatography (MRCP) sequences. Volumetric gallbladder analysis is performed for non-contrast-enhanced and secretin-enhanced MRCP sequences.|Native and secretin-enhanced MRCP volume data were produced with a 1.5-T MR system. Images of coronal maximum intensity projections (MIP) are used to automatically compute 2D characteristic shape features of the gallbladder in the MIP images. A gallbladder shape space is generated to derive 3D gallbladder shape features, which are then combined with 2D gallbladder shape features in a support vector machine approach to detect gallbladder regions in MRCP volume data. A region-based level set approach is used for fine segmentation. Volumetric analysis is performed for both sequences to calculate gallbladder volume differences between both sequences.|The approach presented achieves segmentation results with mean Dice coefficients of 0.917 in non-contrast-enhanced sequences and 0.904 in secretin-enhanced sequences.|This is the first approach developed to detect and segment gallbladders in MR-based volume data automatically in both sequences. It can be used to perform gallbladder volume determination in epidemiological studies and to detect abnormal gallbladder volumes or shapes. The positive volume differences between both sequences may indicate the quantity of the pancreatobiliary reflux.',\n",
              "  'authors': ['Gloger, Oliver',\n",
              "   'Bülow, Robin',\n",
              "   'Tönnies, Klaus',\n",
              "   'Völzke, Henry'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magma (New York, N.Y.)',\n",
              "  'keywords': ['Cholangiopancreatography, Magnetic Resonance',\n",
              "   'Cluster Analysis',\n",
              "   'Contrast Media',\n",
              "   'False Positive Reactions',\n",
              "   'Fourier Analysis',\n",
              "   'Fuzzy Logic',\n",
              "   'Gallbladder',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Statistical',\n",
              "   'Phantoms, Imaging',\n",
              "   'Principal Component Analysis',\n",
              "   'Reproducibility of Results',\n",
              "   'Secretin',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29177771',\n",
              "  'title': 'Automatic gallbladder segmentation using combined 2D and 3D shape features to perform volumetric analysis in native and secretin-enhanced MRCP sequences.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The popularity of online information generally experiences a rising and falling evolution. This paper considers the \"burst\", \"peak\", and \"fade\" key events together as a representative summary of popularity evolution. We propose a novel prediction task-predicting when popularity undergoes these key events. It is of great importance to know when these three key events occur, because doing so helps recommendation systems, online marketing, and containment of rumors. However, it is very challenging to solve this new prediction task due to two issues. First, popularity evolution has high variation and can follow various patterns, so how can we identify \"burst\", \"peak\", and \"fade\" in different patterns of popularity evolution? Second, these events usually occur in a very short time, so how can we accurately yet promptly predict them? In this paper we address these two issues. To handle the first one, we use a simple moving average to smooth variation, and then a universal method is presented for different patterns to identify the key events in popularity evolution. To deal with the second one, we extract different types of features that may have an impact on the key events, and then a correlation analysis is conducted in the feature selection step to remove irrelevant and redundant features. The remaining features are used to train a machine learning model. The feature selection step improves prediction accuracy, and in order to emphasize prediction promptness, we design a new evaluation metric which considers both accuracy and promptness to evaluate our prediction task. Experimental and comparative results show the superiority of our prediction solution.',\n",
              "  'authors': ['Hu, Ying',\n",
              "   'Hu, Changjun',\n",
              "   'Fu, Shushen',\n",
              "   'Fang, Mingzhe',\n",
              "   'Xu, Wenwen'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Access to Information',\n",
              "   'Bayes Theorem',\n",
              "   'Biological Evolution',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Linear Models',\n",
              "   'Models, Statistical',\n",
              "   'Social Media',\n",
              "   'Support Vector Machine',\n",
              "   'Time Factors'],\n",
              "  'pmid': '28046121',\n",
              "  'title': 'Predicting Key Events in the Popularity Evolution of Online Information.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': 'In full-parallax three-dimensional (3-D) imaging systems, the pixel cells often have the shape of a rhombus. Proper arrangement of pixels in these rhombic-shaped cells is important to maximize the quality of displayable 3-D images with a given display panel. The possible number of pixel arrangements in a rhombic cell with a definite dimension is found by considering the number of possible crossings between parallel line families forming the pixel cells, when the slopes of the lines are approximated by the ratio between the number of pixels in the vertical and horizontal directions. To make the rhombic cell have a uniquely defined pixel arrangement, its horizontal and vertical dimensions should be equal to the even multiple of the pixel dimension in their corresponding direction.',\n",
              "  'authors': ['Son, Jung-Young',\n",
              "   'Saveljev, Vladmir V',\n",
              "   'Kwack, Kae-Dal',\n",
              "   'Kim, Sung-Kyu',\n",
              "   'Park, Min-Chul'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Graphics',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Numerical Analysis, Computer-Assisted',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '16633418',\n",
              "  'title': 'Characteristics of pixel arrangements in various rhombuses for full-parallax three-dimensional image generation.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'This paper introduces a simple algorithm for tomographic reconstruction based on the use of a complexity regularization term. The regularization is formulated in the discrete cosine transform (DCT) domain by promoting a low-noise reconstruction having a high sparsity in the frequency domain. The resulting algorithm simply alternates between a maximum-likelihood (ML) expectation-maximization (EM) update and a decreasing sparsity constraint in the DCT domain. Applications to SPECT reconstruction and comparisons with a classical estimator using the best available regularization terms are given in order to illustrate the potential of our reconstruction technique.',\n",
              "  'authors': ['Mignotte, Max', 'Meunier, Jean', 'Soucy, Jean-Paul'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Bayes Theorem',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Likelihood Functions',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, Emission-Computed, Single-Photon'],\n",
              "  'pmid': '18270021',\n",
              "  'title': 'DCT-based complexity regularization for EM tomographic reconstruction.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['OP'],\n",
              "  'ESSN': '1872-6283',\n",
              "  'IF_2013': '2.115',\n",
              "  'IF_2014': '2.14',\n",
              "  'IF_2015': '1.95',\n",
              "  'IF_2016': '1.989',\n",
              "  'IF_2017': '1.974',\n",
              "  'ISSN': '0379-0738',\n",
              "  'ISSN_online': '1872-6283',\n",
              "  'ISSN_print': '0379-0738',\n",
              "  'IsoAbbr': 'Forensic Sci. Int.',\n",
              "  'JrId': 3760,\n",
              "  'MedAbbr': 'Forensic Sci Int',\n",
              "  'NLMid': '7902034',\n",
              "  'Titre': 'Forensic Sci. Int.',\n",
              "  'abstract': 'In this paper, a method for vehicle collision analysis based on qualitative mechanics is presented. In collisions where accurate quantitative information is difficult to gather, this analytic method can act as a useful complement. Based on qualitative dynamic theories, reasonings of various factors including qualitative vector, collision behavior, translational and rotational motion are suggested. Using the law of momentum conservation within the frame of rigid-body dynamics, vehicle collision analyses based on this qualitative reasoning were developed into an accident analysis tool. This newly developed qualitative collision analysis tool was verified using 45 cases of vehicle-to-vehicle collision test data, and its practicality was proved by performing analyses of various vehicle collision accidents.',\n",
              "  'authors': ['Han, Inhwan'],\n",
              "  'categories': ['MEDICINE, LEGAL'],\n",
              "  'journal': 'Forensic science international',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30145486',\n",
              "  'title': 'Analysis of vehicle collision accidents based on qualitative mechanics.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1537-6605',\n",
              "  'IF_2013': '10.987',\n",
              "  'IF_2014': '10.931',\n",
              "  'IF_2015': '10.794',\n",
              "  'IF_2016': '9.025',\n",
              "  'IF_2017': '8.855',\n",
              "  'ISSN': '0002-9297',\n",
              "  'ISSN_online': '1537-6605',\n",
              "  'ISSN_print': '0002-9297',\n",
              "  'IsoAbbr': 'Am. J. Hum. Genet.',\n",
              "  'JrId': 428,\n",
              "  'MedAbbr': 'Am J Hum Genet',\n",
              "  'NLMid': '0370475',\n",
              "  'Titre': 'Am. J. Hum. Genet.',\n",
              "  'abstract': 'Complex traits like type 1 diabetes mellitus (T1DM) are generally taken to be under the influence of multiple genes interacting with each other to confer disease susceptibility and/or protection. Although novel methods are being developed, analyses of whole-genome scans are most often performed with multipoint methods that work under the assumption that multiple trait loci are unrelated to each other; that is, most models specify the effect of only one locus at a time. We have applied a novel approach, which includes decision-tree construction and artificial neural networks, to the analysis of T1DM genome-scan data. We demonstrate that this approach (1) allows identification of all major susceptibility loci identified by nonparametric linkage analysis, (2) identifies a number of novel regions as well as combinations of markers with predictive value for T1DM, and (3) may be useful in characterizing markers in linkage disequilibrium with protective-gene variants. Furthermore, the approach outlined here permits combined analyses of genetic-marker data and information on environmental and clinical covariates.',\n",
              "  'authors': ['Pociot, Flemming',\n",
              "   'Karlsen, Allan E',\n",
              "   'Pedersen, Claus B',\n",
              "   'Aalund, Mogens',\n",
              "   'Nerup, Jørn'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'American journal of human genetics',\n",
              "  'keywords': ['Decision Trees',\n",
              "   'Diabetes Mellitus, Type 1',\n",
              "   'Genetic Markers',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Genome, Human',\n",
              "   'Humans',\n",
              "   'Linkage Disequilibrium',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '15024687',\n",
              "  'title': 'Novel analytical methods applied to type 1 diabetes genome-scan data.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Current preclinical screening methods do not adequately detect cardiotoxicity. Using human induced pluripotent stem cell-derived cardiomyocytes (iPS-CMs), more physiologically relevant preclinical or patient-specific screening to detect potential cardiotoxic effects of drug candidates may be possible. However, one of the persistent challenges for developing a high-throughput drug screening platform using iPS-CMs is the need to develop a simple and reliable method to measure key electrophysiological and contractile parameters. To address this need, we have developed a platform that combines machine learning paired with brightfield optical flow as a simple and robust tool that can automate the detection of cardiomyocyte drug effects. Using three cardioactive drugs of different mechanisms, including those with primarily electrophysiological effects, we demonstrate the general applicability of this screening method to detect subtle changes in cardiomyocyte contraction. Requiring only brightfield images of cardiomyocyte contractions, we detect changes in cardiomyocyte contraction comparable to - and even superior to - fluorescence readouts. This automated method serves as a widely applicable screening tool to characterize the effects of drugs on cardiomyocyte function. ',\n",
              "  'authors': ['Lee, Eugene K',\n",
              "   'Kurokawa, Yosuke K',\n",
              "   'Tu, Robin',\n",
              "   'George, Steven C',\n",
              "   'Khine, Michelle'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Cardiotonic Agents',\n",
              "   'Humans',\n",
              "   'Induced Pluripotent Stem Cells',\n",
              "   'Machine Learning',\n",
              "   'Myocardial Contraction',\n",
              "   'Myocytes, Cardiac',\n",
              "   'Optical Devices'],\n",
              "  'pmid': '26139150',\n",
              "  'title': 'Machine learning plus optical flow: a simple and sensitive method to detect cardioactive drugs.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '1872-7921',\n",
              "  'IF_2013': '2.709',\n",
              "  'IF_2014': '3.371',\n",
              "  'IF_2015': '3.333',\n",
              "  'IF_2016': '4.797',\n",
              "  'IF_2017': '3.034',\n",
              "  'ISSN': '0004-3702',\n",
              "  'ISSN_online': '1872-7921',\n",
              "  'ISSN_print': '0004-3702',\n",
              "  'IsoAbbr': 'Artif Intell',\n",
              "  'JrId': 36543,\n",
              "  'MedAbbr': 'Artif Intell',\n",
              "  'NLMid': '0262613',\n",
              "  'Titre': 'Artif Intell',\n",
              "  'abstract': 'In this paper we show that a corpus of a few thousand Wikipedia articles about concrete or visualizable concepts can be used to produce a low-dimensional semantic feature representation of those concepts. The purpose of such a representation is to serve as a model of the mental context of a subject during functional magnetic resonance imaging (fMRI) experiments. A recent study [19] showed that it was possible to predict fMRI data acquired while subjects thought about a concrete concept, given a representation of those concepts in terms of semantic features obtained with human supervision. We use topic models on our corpus to learn semantic features from text in an unsupervised manner, and show that those features can outperform those in [19] in demanding 12-way and 60-way classification tasks. We also show that these features can be used to uncover similarity relations in brain activation for different concepts which parallel those relations in behavioral data from human subjects.',\n",
              "  'authors': ['Pereira, Francisco', 'Botvinick, Matthew', 'Detre, Greg'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'Artificial intelligence',\n",
              "  'keywords': [''],\n",
              "  'pmid': '23243317',\n",
              "  'title': 'Using Wikipedia to learn semantic feature representations of concrete concepts in neuroimaging experiments.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1558-318X',\n",
              "  'IF_2013': '1.347',\n",
              "  'IF_2014': '1.204',\n",
              "  'IF_2015': '1.835',\n",
              "  'IF_2016': '2.22',\n",
              "  'IF_2017': '2.149',\n",
              "  'ISSN': '0094-0143',\n",
              "  'ISSN_online': '1558-318X',\n",
              "  'ISSN_print': '0094-0143',\n",
              "  'IsoAbbr': 'Urol. Clin. North Am.',\n",
              "  'JrId': 7899,\n",
              "  'MedAbbr': 'Urol Clin North Am',\n",
              "  'NLMid': '0423221',\n",
              "  'Titre': 'Urol. Clin. North Am.',\n",
              "  'abstract': \"Robotic-assisted laparoscopic radical prostatectomy (RALP) has been rapidly adopted in the last few years despite having a prolonged learning curve. This article describes the RALP learning curve, reviews in detail the challenging steps of the operation, describes the authors' RALP technique, and concludes with tips to overcome the learning curve.\",\n",
              "  'authors': ['Freire, Marcos P',\n",
              "   'Choi, Wesley W',\n",
              "   'Lei, Yin',\n",
              "   'Carvas, Fernando',\n",
              "   'Hu, Jim C'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Urologic clinics of North America',\n",
              "  'keywords': ['Humans', 'Laparoscopy', 'Male', 'Prostatectomy', 'Robotics'],\n",
              "  'pmid': '20152518',\n",
              "  'title': 'Overcoming the learning curve for robotic-assisted laparoscopic radical prostatectomy.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1476-6256',\n",
              "  'IF_2013': '4.975',\n",
              "  'IF_2014': '5.23',\n",
              "  'IF_2015': '5.036',\n",
              "  'IF_2016': '4.825',\n",
              "  'IF_2017': '4.322',\n",
              "  'ISSN': '0002-9262',\n",
              "  'ISSN_online': '1476-6256',\n",
              "  'ISSN_print': '0002-9262',\n",
              "  'IsoAbbr': 'Am. J. Epidemiol.',\n",
              "  'JrId': 421,\n",
              "  'MedAbbr': 'Am J Epidemiol',\n",
              "  'NLMid': '7910653',\n",
              "  'Titre': 'Am. J. Epidemiol.',\n",
              "  'abstract': 'Chronic noncommunicable diseases (NCDs) are now prevalent in many low- and middle-income countries and confer a heightened risk of disability. It is unclear how public health programs can identify the older adults at highest risk of disability related to NCDs within diverse developing country populations. We studied nationally representative survey data from 7,150 Indian adults older than 50 years of age who participated in the World Health Organization Study on Global Aging and Adult Health (2007-2010) to identify population subgroups who are highly disabled. Using machine-learning algorithms, we identified sociodemographic correlates of disability. Although having 2 or more symptomatic NCDs was a key correlate of disability, the prevalence of symptomatic, undiagnosed NCDs was highest among the lowest 2 wealth quintiles of Indian adults, contrary to prior hypotheses of increased NCDs with wealth. Women and persons from rural populations were also disproportionately affected by nondiagnosed NCDs, with high out-of-pocket health care expenditures increasing the probability of remaining symptomatic from NCDs. These findings also indicate that NCD prevalence surveillance studies in low- and middle-income countries should expand beyond self-reported diagnoses to include more extensive symptom- and examination-based surveys, given the likely high rate of surveillance bias due to barriers to diagnosis among vulnerable populations. ',\n",
              "  'authors': ['Basu, Sanjay', 'King, Abby C'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'American journal of epidemiology',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Chronic Disease',\n",
              "   'Cross-Sectional Studies',\n",
              "   'Developing Countries',\n",
              "   'Disability Evaluation',\n",
              "   'Female',\n",
              "   'Health Surveys',\n",
              "   'Humans',\n",
              "   'India',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Population Surveillance',\n",
              "   'Prevalence',\n",
              "   'Risk Factors',\n",
              "   'Self Report',\n",
              "   'Socioeconomic Factors',\n",
              "   'Vulnerable Populations',\n",
              "   'World Health Organization'],\n",
              "  'pmid': '24049156',\n",
              "  'title': 'Disability and chronic disease among older adults in India: detecting vulnerable populations through the WHO SAGE Study.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1097-0290',\n",
              "  'IF_2013': '4.164',\n",
              "  'IF_2014': '4.126',\n",
              "  'IF_2015': '4.243',\n",
              "  'IF_2016': '4.481',\n",
              "  'IF_2017': '3.952',\n",
              "  'ISSN': '0006-3592',\n",
              "  'ISSN_online': '1097-0290',\n",
              "  'ISSN_print': '0006-3592',\n",
              "  'IsoAbbr': 'Biotechnol. Bioeng.',\n",
              "  'JrId': 1200,\n",
              "  'MedAbbr': 'Biotechnol Bioeng',\n",
              "  'NLMid': '7502021',\n",
              "  'Titre': 'Biotechnol. Bioeng.',\n",
              "  'abstract': 'Market demand places great emphasis in industry on product quality. Consequently, process monitoring and control have become important aspects of systems engineering. In this article we detail the results of a 2-year study focusing on the development of a condition monitoring system for a fed-batch fermentation system operated by Biochemie Gmbh in Austria. We also demonstrate the suitability and limitations of current state of the art technologies in this field and suggest novel modifications and configurations to improve their suitability for application to a fed-batch fermentation system.',\n",
              "  'authors': ['Lennox, B',\n",
              "   'Montague, G A',\n",
              "   'Hiden, H G',\n",
              "   'Kornfeld, G',\n",
              "   'Goulding, P R'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Biotechnology and bioengineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Fermentation',\n",
              "   'Industrial Microbiology',\n",
              "   'Least-Squares Analysis',\n",
              "   'Multivariate Analysis',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '11370001',\n",
              "  'title': 'Process monitoring of an industrial fed-batch fermentation.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1878-0369',\n",
              "  'IF_2013': '0.753',\n",
              "  'IF_2014': '0.753',\n",
              "  'IF_2015': '0.753',\n",
              "  'IF_2016': '0.753',\n",
              "  'IF_2017': '0.794',\n",
              "  'ISSN': '0001-2092',\n",
              "  'ISSN_online': '1878-0369',\n",
              "  'ISSN_print': '0001-2092',\n",
              "  'IsoAbbr': 'AORN J',\n",
              "  'JrId': 691,\n",
              "  'MedAbbr': 'AORN J',\n",
              "  'NLMid': '0372403',\n",
              "  'Titre': 'AORN J',\n",
              "  'abstract': 'Robot-assisted surgery continues to grow in popularity worldwide. Competency and training of personnel for robot-assisted thoracic surgery (RATS) is less established compared with other robot-assisted specialties. Major differences between minimally invasive approaches to thoracic surgery (eg, video-assisted thoracoscopic surgery) and RATS are presented to address a paucity of literature on the subject. Although perioperative nursing considerations are universal to all robot-assisted procedures, there are nursing consideration specific to RATS. This article provides a RATS perioperative nursing development program for RN circulators and scrub personnel. Development of perioperative nursing knowledge and skills through implementation of targeted training programs enables nurses to provide a safe surgical experience for patients undergoing RATS. ',\n",
              "  'authors': ['Sarmanian, Julie D'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'AORN journal',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Perioperative Nursing',\n",
              "   'Program Development',\n",
              "   'Program Evaluation',\n",
              "   'Robotics',\n",
              "   'Staff Development',\n",
              "   'Thoracic Surgical Procedures'],\n",
              "  'pmid': '26323222',\n",
              "  'title': 'Robot-Assisted Thoracic Surgery (RATS): Perioperative Nursing Professional Development Program.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'The Planteome project (http://www.planteome.org) provides a suite of reference and species-specific ontologies for plants and annotations to genes and phenotypes. Ontologies serve as common standards for semantic integration of a large and growing corpus of plant genomics, phenomics and genetics data. The reference ontologies include the Plant Ontology, Plant Trait Ontology and the Plant Experimental Conditions Ontology developed by the Planteome project, along with the Gene Ontology, Chemical Entities of Biological Interest, Phenotype and Attribute Ontology, and others. The project also provides access to species-specific Crop Ontologies developed by various plant breeding and research communities from around the world. We provide integrated data on plant traits, phenotypes, and gene function and expression from 95 plant taxa, annotated with reference ontology terms. The Planteome project is developing a plant gene annotation platform; Planteome Noctua, to facilitate community engagement. All the Planteome ontologies are publicly available and are maintained at the Planteome GitHub site (https://github.com/Planteome) for sharing, tracking revisions and new requests. The annotated data are freely accessible from the ontology browser (http://browser.planteome.org/amigo) and our data repository.',\n",
              "  'authors': ['Cooper, Laurel',\n",
              "   'Meier, Austin',\n",
              "   'Laporte, Marie-Angélique',\n",
              "   'Elser, Justin L',\n",
              "   'Mungall, Chris',\n",
              "   'Sinn, Brandon T',\n",
              "   'Cavaliere, Dario',\n",
              "   'Carbon, Seth',\n",
              "   'Dunn, Nathan A',\n",
              "   'Smith, Barry',\n",
              "   'Qu, Botong',\n",
              "   'Preece, Justin',\n",
              "   'Zhang, Eugene',\n",
              "   'Todorovic, Sinisa',\n",
              "   'Gkoutos, Georgios',\n",
              "   'Doonan, John H',\n",
              "   'Stevenson, Dennis W',\n",
              "   'Arnaud, Elizabeth',\n",
              "   'Jaiswal, Pankaj'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Crops, Agricultural',\n",
              "   'Data Curation',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genome, Plant',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Phenotype',\n",
              "   'Plants',\n",
              "   'Software',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '29186578',\n",
              "  'title': 'The Planteome database: an integrated resource for reference ontologies, plant genomics and phenomics.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'A major challenge to developing functional electrical stimulation (FES) systems for paraplegic walking and widespread acceptance of these systems is the design of a robust control strategy that provides satisfactory tracking performance. The systems need to be robust against time-varying properties of neuromusculoskeletal dynamics, day-to-day variations, subject-to-subject variations, external disturbances, and must be easily applied without requiring offline identification during different experimental sessions. Another major problem related to walker-assisted FES-activated walking concerns the high metabolic rate and upper body effort that limit the clinical applications of FES systems. In this paper, we present a novel decentralized modular control framework for robust control of walker-assisted FES-activated walking. For each muscle-joint dynamics, an independent module control is designed, and the dynamics of the plant are identified online. This process requires no prior knowledge about the dynamics of the plant to be controlled and no offline learning phase. The module is based on adaptive fuzzy terminal sliding mode control and fuzzy logic control. The module control adjusts both pulse-amplitude and pulsewidth of the stimulation signal in such a way that upper body effort is minimized and the lower extremity walking pattern lies within a defined boundary of the reference trajectory. The proposed control strategy has been evaluated on three paraplegic subjects. The results showed that accurate tracking performance and smooth walking pattern were achieved. This favorable performance was obtained without requiring offline identification, manual adjustments, and predefined ON/OFF timing of the muscles.',\n",
              "  'authors': ['Nekoukar, Vahab', 'Erfanian, Abbas'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Adult',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Electric Stimulation Therapy',\n",
              "   'Fuzzy Logic',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Neural Prostheses',\n",
              "   'Paraplegia',\n",
              "   'Self-Help Devices',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Walkers',\n",
              "   'Walking'],\n",
              "  'pmid': '22868526',\n",
              "  'title': 'A decentralized modular control framework for robust control of FES-activated walker-assisted paraplegic walking using terminal sliding mode and fuzzy logic control.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1873-4588',\n",
              "  'IF_2013': '0.944',\n",
              "  'IF_2014': '1.242',\n",
              "  'IF_2015': '1.113',\n",
              "  'IF_2016': '1.381',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0892-1997',\n",
              "  'ISSN_online': '1873-4588',\n",
              "  'ISSN_print': '0892-1997',\n",
              "  'IsoAbbr': 'J Voice',\n",
              "  'JrId': 2487,\n",
              "  'MedAbbr': 'J Voice',\n",
              "  'NLMid': '8712262',\n",
              "  'Titre': 'J Voice',\n",
              "  'abstract': 'Scientific and clinical advances in perinatology and neonatology have enhanced the chances of survival of preterm and very low weight neonates. Infant cry analysis is a suitable noninvasive complementary tool to assess the neurologic state of infants particularly important in the case of preterm neonates. This article aims at exploiting differences between full-term and preterm infant cry with robust automatic acoustical analysis and data mining techniques.|Twenty-two acoustical parameters are estimated in more than 3000 cry units from cry recordings of 28 full-term and 10 preterm newborns.|Feature extraction is performed through the BioVoice dedicated software tool, developed at the Biomedical Engineering Lab, University of Firenze, Italy. Classification and pattern recognition is based on genetic algorithms for the selection of the best attributes. Training is performed comparing four classifiers: Logistic Curve, Multilayer Perceptron, Support Vector Machine, and Random Forest and three different testing options: full training set, 10-fold cross-validation, and 66% split.|Results show that the best feature set is made up by 10 parameters capable to assess differences between preterm and full-term newborns with about 87% of accuracy. Best results are obtained with the Random Forest method (receiver operating characteristic area, 0.94).|These 10 cry features might convey important additional information to assist the clinical specialist in the diagnosis and follow-up of possible delays or disorders in the neurologic development due to premature birth in this extremely vulnerable population of patients. The proposed approach is a first step toward an automatic infant cry recognition system for fast and proper identification of risk in preterm babies.',\n",
              "  'authors': ['Orlandi, Silvia',\n",
              "   'Reyes Garcia, Carlos Alberto',\n",
              "   'Bandini, Andrea',\n",
              "   'Donzelli, Gianpaolo',\n",
              "   'Manfredi, Claudia'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'Journal of voice : official journal of the Voice Foundation',\n",
              "  'keywords': ['Acoustics',\n",
              "   'Crying',\n",
              "   'Data Mining',\n",
              "   'Female',\n",
              "   'Gestational Age',\n",
              "   'Humans',\n",
              "   'Infant Behavior',\n",
              "   'Infant, Newborn',\n",
              "   'Infant, Premature',\n",
              "   'Logistic Models',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Software',\n",
              "   'Sound Spectrography',\n",
              "   'Support Vector Machine',\n",
              "   'Term Birth'],\n",
              "  'pmid': '26474712',\n",
              "  'title': 'Application of Pattern Recognition Techniques to the Classification of Full-Term and Preterm Infant Cry.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"We used a computational linguistic approach, exploiting machine learning techniques, to examine the letters written by King George III during mentally healthy and apparently mentally ill periods of his life. The aims of the study were: first, to establish the existence of alterations in the King's written language at the onset of his first manic episode; and secondly to identify salient sources of variation contributing to the changes. Effects on language were sought in two control conditions (politically stressful vs. politically tranquil periods and seasonal variation). We found clear differences in the letter corpus, across a range of different features, in association with the onset of mental derangement, which were driven by a combination of linguistic and information theory features that appeared to be specific to the contrast between acute mania and mental stability. The paucity of existing data relevant to changes in written language in the presence of acute mania suggests that lexical, syntactic and stylometric descriptions of written discourse produced by a cohort of patients with a diagnosis of acute mania will be necessary to support the diagnosis independently and to look for other periods of mental illness of the course of the King's life, and in other historically significant figures with similarly large archives of handwritten documents.\",\n",
              "  'authors': ['Rentoumi, Vassiliki',\n",
              "   'Peters, Timothy',\n",
              "   'Conlin, Jonathan',\n",
              "   'Garrard, Peter'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Bipolar Disorder',\n",
              "   'Famous Persons',\n",
              "   'Humans',\n",
              "   'Language',\n",
              "   'Linguistics',\n",
              "   'Machine Learning',\n",
              "   'Male'],\n",
              "  'pmid': '28328964',\n",
              "  'title': 'The acute mania of King George III: A computational linguistic analysis.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'In skeletal muscle the coordinated actions of two mechanically coupled Ca2+ channels-the 1,4-dihydropyridine receptor (Cav1.1) and the type 1 ryanodine receptor (RYR1)-underlie the molecular mechanism of rapid cytosolic [Ca2+] increase leading to contraction. While both [Ca2+]i and contractile activity have been implicated in the regulation of myogenesis, less is known about potential specific roles of Cav1.1 and RYR1 in skeletal muscle development. In this study, we analyzed the histology and the transcriptomic changes occurring at E14.5 -the end of primary myogenesis and around the onset of intrauterine limb movement, and at E18.5 -the end of secondary myogenesis, in WT, RYR1-/-, and Cav1.1-/- murine limb skeletal muscle. At E14.5 the muscle histology of both mutants exhibited initial alterations, which became much more severe at E18.5. Immunohistological analysis also revealed higher levels of activated caspase-3 in the Cav1.1-/- muscles at E14.5, indicating an increase in apoptosis. With WT littermates as controls, microarray analyses identified 61 and 97 differentially regulated genes (DEGs) at E14.5, and 493 and 1047 DEGs at E18.5, in RYR1-/- and Cav1.1-/- samples, respectively. Gene enrichment analysis detected no overlap in the affected biological processes and pathways in the two mutants at E14.5, whereas at E18.5 there was a significant overlap of DEGs in both mutants, affecting predominantly processes linked to muscle contraction. Moreover, the E18.5 vs. E14.5 comparison revealed multiple genotype-specific DEGs involved in contraction, cell cycle and miRNA-mediated signaling in WT, neuronal and bone development in RYR1-/-, and lipid metabolism in Cav1.1-/- samples. Taken together, our study reveals discrete changes in the global transcriptome occurring in limb skeletal muscle from E14.5 to E18.5 in WT, RYR1-/- and Cav1.1-/- mice. Our results suggest distinct functional roles for RYR1 and Cav1.1 in skeletal primary and secondary myogenesis.',\n",
              "  'authors': ['Filipova, Dilyana',\n",
              "   'Henry, Margit',\n",
              "   'Rotshteyn, Tamara',\n",
              "   'Brunn, Anna',\n",
              "   'Carstov, Mariana',\n",
              "   'Deckert, Martina',\n",
              "   'Hescheler, Jürgen',\n",
              "   'Sachinidis, Agapios',\n",
              "   'Pfitzer, Gabriele',\n",
              "   'Papadopoulos, Symeon'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Calcium Channels, L-Type',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Ontology',\n",
              "   'Hindlimb',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Mice, Knockout',\n",
              "   'Muscle Development',\n",
              "   'Muscle, Skeletal',\n",
              "   'Ryanodine Receptor Calcium Release Channel',\n",
              "   'Time Factors',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '29543863',\n",
              "  'title': 'Distinct transcriptomic changes in E14.5 mouse skeletal muscle lacking RYR1 or Cav1.1 converge at E18.5.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1097-0215',\n",
              "  'IF_2013': '5.007',\n",
              "  'IF_2014': '5.085',\n",
              "  'IF_2015': '5.531',\n",
              "  'IF_2016': '6.513',\n",
              "  'IF_2017': '7.36',\n",
              "  'ISSN': '0020-7136',\n",
              "  'ISSN_online': '1097-0215',\n",
              "  'ISSN_print': '0020-7136',\n",
              "  'IsoAbbr': 'Int. J. Cancer',\n",
              "  'JrId': 4284,\n",
              "  'MedAbbr': 'Int J Cancer',\n",
              "  'NLMid': '0042124',\n",
              "  'Titre': 'Int. J. Cancer',\n",
              "  'abstract': 'Our objective was to evaluate the feasibility and relative validity of interactive neural network assisted screening (Papnet) in primary mass screening for cervical cancer as a public health policy (routine screening). A randomized, ongoing trial involved 152,969 invitees and 108,686 attendees in the organized mass screening in Finland in 1999. Drawing invitations from the population registry, women were randomized 2:1 at an individual level to have their smear analyzed either conventionally or with Papnet. The distribution of smears to different cytological categories, detection rates of dysplasias, in situ carcinomas and cancers were estimated with smears analyzed either conventionally (72,461) or by Papnet (36,225). A total of 108,686 smears were screened and 449 were histologically confirmed as dysplasias and carcinomas. The detection rates for histologically verified carcinoma in situ/severe dysplasia, moderate and mild dysplasias were 0.14%, 0.14% and 0.13% with conventional and 0.14%, 0.14% and 0.11% with Papnet, respectively. The detection rate of invasive cancer was 0.06 per thousand (n = 4) with conventional method and 0.08 per thousand (n = 3) with Papnet. None of the differences were statistically significant (p > 0.05). Papnet was able to identify 92.5% of healthy women (normal cytology), and the specificity of conventional smear was 92.9%. The positive predictive value (Pap Classes III-V) of Papnet was slightly but not significantly better (55% vs. 51%). Papnet screening was feasible as a part of routine screening and performed equally well compared to conventional one methods used in Finland. Organized mass screening was practiced very successfully in the last 38 years. We are going to continue the trial to study the potential trends in cervical cancer incidence in both study arms.',\n",
              "  'authors': ['Nieminen, Pekka',\n",
              "   'Hakama, Matti',\n",
              "   'Viikki, Merja',\n",
              "   'Tarkkanen, Jussi',\n",
              "   'Anttila, Ahti'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'International journal of cancer',\n",
              "  'keywords': ['Adult',\n",
              "   'Carcinoma in Situ',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Finland',\n",
              "   'Humans',\n",
              "   'Mass Screening',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Invasiveness',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prospective Studies',\n",
              "   'Public Health',\n",
              "   'Quality Control',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Time Factors',\n",
              "   'Uterine Cervical Dysplasia',\n",
              "   'Uterine Cervical Neoplasms',\n",
              "   'Vaginal Smears'],\n",
              "  'pmid': '12471627',\n",
              "  'title': 'Prospective and randomised public-health trial on neural network-assisted screening for cervical cancer in Finland: results of the first year.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1941-0506',\n",
              "  'IF_2013': '1.919',\n",
              "  'IF_2014': '2.168',\n",
              "  'IF_2015': '1.4',\n",
              "  'IF_2016': '2.84',\n",
              "  'IF_2017': '3.078',\n",
              "  'ISSN': '1077-2626',\n",
              "  'ISSN_online': '1941-0506',\n",
              "  'ISSN_print': '1077-2626',\n",
              "  'IsoAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'JrId': 31980,\n",
              "  'MedAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'NLMid': '9891704',\n",
              "  'Titre': 'IEEE Trans Vis Comput Graph',\n",
              "  'abstract': \"As microscopes have a very shallow depth of field, Z-stacks (i.e. sets of images shot at different focal planes) are often acquired to fully capture a thick sample. Such stacks are viewed by users by navigating them through the mouse wheel. We propose a new technique of visualizing 3D point, line or area markers in such focus stacks, by displaying them with a depth-dependent defocus, simulating the microscope's optics; this leverages on the microscopists' ability to continuously twiddle focus, while implicitly performing a shape-from-focus reconstruction of the 3D structure of the sample. User studies confirm that the approach is effective, and can complement more traditional techniques such as color-based cues. We provide two implementations, one of which computes defocus in real time on the GPU, and examples of their application.\",\n",
              "  'authors': ['Giusti, Alessandro',\n",
              "   'Taddei, Pierluigi',\n",
              "   'Magli, Cristina',\n",
              "   'Corani, Giorgio',\n",
              "   'Gambardella, Luca',\n",
              "   'Gianaroli, Luca'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE transactions on visualization and computer graphics',\n",
              "  'keywords': ['Animals',\n",
              "   'Computer Graphics',\n",
              "   'Computer Systems',\n",
              "   'Depth Perception',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Mice',\n",
              "   'Microscopy',\n",
              "   'Siphonaptera',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '22034292',\n",
              "  'title': 'Artificial defocus for displaying markers in microscopy Z-stacks.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1097-6868',\n",
              "  'IF_2013': '3.973',\n",
              "  'IF_2014': '4.704',\n",
              "  'IF_2015': '4.681',\n",
              "  'IF_2016': '5.226',\n",
              "  'IF_2017': '5.732',\n",
              "  'ISSN': '0002-9378',\n",
              "  'ISSN_online': '1097-6868',\n",
              "  'ISSN_print': '0002-9378',\n",
              "  'IsoAbbr': 'Am. J. Obstet. Gynecol.',\n",
              "  'JrId': 439,\n",
              "  'MedAbbr': 'Am J Obstet Gynecol',\n",
              "  'NLMid': '0370476',\n",
              "  'Titre': 'Am. J. Obstet. Gynecol.',\n",
              "  'abstract': 'The aim of this study was to initiate neural net construction for the detection of cervical intraepithelial neoplasia by fluorescence imaging.|Thirty-three women with abnormal Papanicolaou smears underwent fluorescence imaging during colposcopy. With the use of >4000 training pixels and >1000 test pixels, intrapatient nets were constructed from the spectral data of 17 women. An interpatient net that discriminated between cervical intraepithelial neoplasia 1 and normal tissue classes among patients was constructed with the use of >2300 training pixels and >2000 test pixels from 12 women. Average correct classification rates were determined. Sensitivities, specificities, and positive and negative predictive values for cervical intraepithelial neoplasia grade 1 and normal tissue classes were calculated. Extrapolated false-color cervical images were generated.|Average correct classification rates were 96.5% for the intrapatient nets and 97.5% for the interpatient net. The sensitivity, specificity, and positive and negative predictive values for cervical intraepithelial neoplasia grade 1 were 98.2%, 98.9%, 71.4%, and 99.9%, respectively.|Initial results suggest that neural nets that are constructed from fluorescence imaging spectra may offer a potential method for the detection of cervical intraepithelial neoplasia.',\n",
              "  'authors': ['Parker, Mary F',\n",
              "   'Mooradian, Gregory C',\n",
              "   'Okimoto, Gordon S',\n",
              "   \"O'Connor, Dennis M\",\n",
              "   'Miyazawa, Kunio',\n",
              "   'Saggese, Steven J'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'American journal of obstetrics and gynecology',\n",
              "  'keywords': ['Cervical Intraepithelial Neoplasia',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Spectrometry, Fluorescence',\n",
              "   'Uterine Cervical Neoplasms'],\n",
              "  'pmid': '12193932',\n",
              "  'title': 'Initial neural net construction for the detection of cervical intraepithelial neoplasia by fluorescence imaging.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'Automated assignment of protein function has received considerable attention in recent years for genome-wide study. With the rapid accumulation of genome sequencing data produced by high-throughput experimental techniques, the process of manually predicting functional properties of proteins has become increasingly cumbersome. Such large genomics data sets can only be annotated computationally. However, automated assignment of functions to unknown protein is challenging due to its inherent difficulty and complexity. Previous studies have revealed that solving problems involving complicated objects with multiple semantic meanings using the multi-instance multi-label (MIML) framework is effective. For the protein function prediction problems, each protein object in nature may associate with distinct structural units (instances) and multiple functional properties (class labels) where each unit is described by an instance and each functional property is considered as a class label. Thus, it is convenient and natural to tackle the protein function prediction problem by using the MIML framework. In this paper, we propose a sparse Markov chain-based semi-supervised MIML method, called Sparse-Markov. A sparse transductive probability graph is constructed to encode the affinity information of the data based on ensemble of Hausdorff distance metrics. Our goal is to exploit the affinity between protein objects in the sparse transductive probability graph to seek a sparse steady state probability of the Markov chain model to do protein function prediction, such that two proteins are given similar functional labels if they are close to each other in terms of an ensemble Hausdorff distance in the graph. Experimental results on seven real-world organism data sets covering three biological domains show that our proposed Sparse-Markov method is able to achieve better performance than four state-of-the-art MIML learning algorithms. ',\n",
              "  'authors': ['Han, Chao',\n",
              "   'Chen, Jian',\n",
              "   'Wu, Qingyao',\n",
              "   'Mu, Shuai',\n",
              "   'Min, Huaqing'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Markov Chains',\n",
              "   'Proteins',\n",
              "   'Supervised Machine Learning'],\n",
              "  'pmid': '26493682',\n",
              "  'title': 'Sparse Markov chain-based semi-supervised multi-instance multi-label method for protein function prediction.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['MA'],\n",
              "  'ESSN': '1528-0020',\n",
              "  'IF_2013': '9.775',\n",
              "  'IF_2014': '10.452',\n",
              "  'IF_2015': '11.841',\n",
              "  'IF_2016': '13.164',\n",
              "  'IF_2017': '15.132',\n",
              "  'ISSN': '0006-4971',\n",
              "  'ISSN_online': '1528-0020',\n",
              "  'ISSN_print': '0006-4971',\n",
              "  'IsoAbbr': 'Blood',\n",
              "  'JrId': 1243,\n",
              "  'MedAbbr': 'Blood',\n",
              "  'NLMid': '7603509',\n",
              "  'Titre': 'Blood',\n",
              "  'abstract': 'We recently reported 3 risk factors (RFs) at diagnosis of chronic graft-versus-host disease (cGVHD) that were significantly associated with increased nonrelapse mortality. These included extensive skin involvement (ESI), thrombocytopenia (TP), and progressive type of onset (PTO). The hazard ratio (HR) for mortality of the patients with prognostic score (PS) between 0 and 2 (intermediate-risk; 1 RF) compared to those with PS 0 (favorable-risk; 0 RF) was 3.7 (95% CI, 1.4, 9.3); the HR for patients with PS equal to or more than 2 (high-risk; > 1 RF) compared with intermediate-risk group was 6.9 (3.8, 12.4). A rare presentation of TP and PTO without ESI yielded a PS of 1.8 (intermediate-risk). This paper reports the performance of the prognostic model and the individual RFs using data from an additional 1105 patients from University of Nebraska (n = 60), International Bone Marrow Transplantation Registry (n = 708), Fred Hutchinson Cancer Research Center (n = 188), and University of Minnesota (n = 149). The extent of skin involvement was quantified in 3 cohorts using the available data collected in different formats before the analysis. Although the HR for mortality of the patients in the intermediate-risk group versus those in the favorable-risk group ranged from 2.3 to 8.9 across the centers, it was between 1.6 to 6.9 for patients in the high-risk group versus those in the intermediate-risk group. Although TP itself was uniformly associated with increased risk of mortality across all test samples, ESI and PTO showed statistically significant associations with mortality in 1 and 2 cohorts, respectively. In conclusion, the model was predictive of cGVHD-specific survival, but the mortality hazard associated with ESI was lower in each of these test samples compared with the learning sample. Although the new clinical grading based on the model is promising because of its utility across multiple independent data sets, prospective validation is needed.',\n",
              "  'authors': ['Akpek, Gorgun',\n",
              "   'Lee, Stephanie J',\n",
              "   'Flowers, Mary E',\n",
              "   'Pavletic, Steven Z',\n",
              "   'Arora, Mukta',\n",
              "   'Lee, Shing',\n",
              "   'Piantadosi, Steven',\n",
              "   'Guthrie, Katherine A',\n",
              "   'Lynch, James C',\n",
              "   'Takatu, Alessandra',\n",
              "   'Horowitz, Mary M',\n",
              "   'Antin, Joseph H',\n",
              "   'Weisdorf, Daniel J',\n",
              "   'Martin, Paul J',\n",
              "   'Vogelsang, Georgia B'],\n",
              "  'categories': ['HEMATOLOGY'],\n",
              "  'journal': 'Blood',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Artificial Intelligence',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Chronic Disease',\n",
              "   'Databases, Factual',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Graft vs Host Disease',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Models, Statistical',\n",
              "   'Prognosis',\n",
              "   'Severity of Illness Index',\n",
              "   'Skin Diseases',\n",
              "   'Survival Analysis'],\n",
              "  'pmid': '12714524',\n",
              "  'title': 'Performance of a new clinical grading system for chronic graft-versus-host disease: a multicenter study.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1532-2335',\n",
              "  'IF_2013': '0.894',\n",
              "  'IF_2014': '1.018',\n",
              "  'IF_2015': '0.876',\n",
              "  'IF_2016': '0.868',\n",
              "  'IF_2017': '0.831',\n",
              "  'ISSN': '1525-7770',\n",
              "  'ISSN_online': '1532-2335',\n",
              "  'ISSN_print': '1525-7770',\n",
              "  'IsoAbbr': 'Nucleosides Nucleotides Nucleic Acids',\n",
              "  'JrId': 21699,\n",
              "  'MedAbbr': 'Nucleosides Nucleotides Nucleic Acids',\n",
              "  'NLMid': '100892832',\n",
              "  'Titre': 'Nucleosides Nucleotides Nucleic Acids',\n",
              "  'abstract': 'In past decades, prediction of genes in DNA sequences has attracted the attention of many researchers but due to its complex structure it is extremely intricate to correctly locate its position. A large number of regulatory regions are present in DNA that helps in transcription of a gene. Promoter is one such region and to find its location is a challenging problem. Various computational methods for promoter prediction have been developed over the past few years. This paper reviews these promoter prediction methods. Several difficulties and pitfalls encountered by these methods are also detailed, along with future research directions. ',\n",
              "  'authors': ['Singh, Shailendra', 'Kaur, Sukhbir', 'Goel, Neelam'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleosides, nucleotides & nucleic acids',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Eukaryota',\n",
              "   'Promoter Regions, Genetic'],\n",
              "  'pmid': '26158565',\n",
              "  'title': 'A Review of Computational Intelligence Methods for Eukaryotic Promoter Prediction.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1558-5034',\n",
              "  'IF_2013': '1.082',\n",
              "  'IF_2014': '1.198',\n",
              "  'IF_2015': '1.319',\n",
              "  'IF_2016': '1.13',\n",
              "  'IF_2017': '1.062',\n",
              "  'ISSN': '0887-2171',\n",
              "  'ISSN_online': '1558-5034',\n",
              "  'ISSN_print': '0887-2171',\n",
              "  'IsoAbbr': 'Semin. Ultrasound CT MR',\n",
              "  'JrId': 1659,\n",
              "  'MedAbbr': 'Semin Ultrasound CT MR',\n",
              "  'NLMid': '8504689',\n",
              "  'Titre': 'Semin. Ultrasound CT MR',\n",
              "  'abstract': 'The recent technological advancement of fast magnetic resonance imaging (MRI) sequences allowed the inclusion of diffusion tensor imaging, functional MRI, and proton MR spectroscopy in prenatal imaging protocols. These methods provide information beyond morphology and hold the key to improving several fields of human neuroscience and clinical diagnostics. Our review introduces the fundamental works that enabled these imaging techniques, and also highlights the most recent contributions to this emerging field of prenatal diagnostics, such as the structural and functional connectomic approach. We introduce the advanced image processing approaches that are extensively used to tackle fetal or maternal movement-related image artifacts, and which are necessary for the optimal interpretation of such imaging data. ',\n",
              "  'authors': ['Jakab, András',\n",
              "   'Pogledic, Ivana',\n",
              "   'Schwartz, Ernst',\n",
              "   'Gruber, Gerlinde',\n",
              "   'Mitter, Christian',\n",
              "   'Brugger, Peter C',\n",
              "   'Langs, Georg',\n",
              "   'Schöpf, Veronika',\n",
              "   'Kasprian, Gregor',\n",
              "   'Prayer, Daniela'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Seminars in ultrasound, CT, and MR',\n",
              "  'keywords': ['Biomarkers',\n",
              "   'Brain Diseases',\n",
              "   'Brain Mapping',\n",
              "   'Diffusion Tensor Imaging',\n",
              "   'Female',\n",
              "   'Fetal Diseases',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Male',\n",
              "   'Molecular Imaging',\n",
              "   'Prenatal Diagnosis',\n",
              "   'Proton Magnetic Resonance Spectroscopy'],\n",
              "  'pmid': '26614130',\n",
              "  'title': 'Fetal Cerebral Magnetic Resonance Imaging Beyond Morphology.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The study aimed to develop machine learning models that have strong prediction power and interpretability for diagnosis of glaucoma based on retinal nerve fiber layer (RNFL) thickness and visual field (VF). We collected various candidate features from the examination of retinal nerve fiber layer (RNFL) thickness and visual field (VF). We also developed synthesized features from original features. We then selected the best features proper for classification (diagnosis) through feature evaluation. We used 100 cases of data as a test dataset and 399 cases of data as a training and validation dataset. To develop the glaucoma prediction model, we considered four machine learning algorithms: C5.0, random forest (RF), support vector machine (SVM), and k-nearest neighbor (KNN). We repeatedly composed a learning model using the training dataset and evaluated it by using the validation dataset. Finally, we got the best learning model that produces the highest validation accuracy. We analyzed quality of the models using several measures. The random forest model shows best performance and C5.0, SVM, and KNN models show similar accuracy. In the random forest model, the classification accuracy is 0.98, sensitivity is 0.983, specificity is 0.975, and AUC is 0.979. The developed prediction models show high accuracy, sensitivity, specificity, and AUC in classifying among glaucoma and healthy eyes. It will be used for predicting glaucoma against unknown examination records. Clinicians may reference the prediction results and be able to make better decisions. We may combine multiple learning models to increase prediction accuracy. The C5.0 model includes decision rules for prediction. It can be used to explain the reasons for specific predictions.',\n",
              "  'authors': ['Kim, Seong Jae', 'Cho, Kyong Jin', 'Oh, Sejong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Area Under Curve',\n",
              "   'Datasets as Topic',\n",
              "   'Decision Trees',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Glaucoma',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'ROC Curve',\n",
              "   'Retina',\n",
              "   'Visual Field Tests'],\n",
              "  'pmid': '28542342',\n",
              "  'title': 'Development of machine learning models for diagnosis of glaucoma.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1534-6285',\n",
              "  'IF_2013': '1.513',\n",
              "  'IF_2014': '1.513',\n",
              "  'IF_2015': '1.597',\n",
              "  'IF_2016': '2.124',\n",
              "  'IF_2017': '1.983',\n",
              "  'ISSN': '1527-2737',\n",
              "  'ISSN_online': '1534-6285',\n",
              "  'ISSN_print': '1527-2737',\n",
              "  'IsoAbbr': 'Curr Urol Rep',\n",
              "  'JrId': 29748,\n",
              "  'MedAbbr': 'Curr Urol Rep',\n",
              "  'NLMid': '100900943',\n",
              "  'Titre': 'Curr Urol Rep',\n",
              "  'abstract': 'Ureteropelvic junction obstruction (UPJO) is a common congenital abnormality that often presents in adulthood. Open dismembered pyeloplasty was considered the gold standard for the management of this condition; however, recent advancements in laparoscopic and robotic surgery have dramatically shifted the landscape to more minimally invasive techniques. A literature search of ureteropelvic junction obstruction, pyeloplasty, endopyelotomy, laparoscopic pyeloplasty, robotic pyeloplasty, and microlaparoscopic pyeloplasty was performed. A focus was placed on literature published since 2013. Minimally invasive laparoscopic and robotic techniques have become the gold standard for the management of UPJO. With the rise of robotic pyeloplasty, open repairs are becoming less frequent, while endoscopic treatments have remained stable. Minimally invasive (robotic) techniques have become the gold standard for the management of UPJO. Newer, even less-invasive techniques are also showing promise, but technical challenges still exist. ',\n",
              "  'authors': ['Strother, Marshall C', 'Mucksavage, Phillip'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current urology reports',\n",
              "  'keywords': ['Adult',\n",
              "   'Humans',\n",
              "   'Kidney Pelvis',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Retroperitoneal Space',\n",
              "   'Robotic Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Ureteral Obstruction'],\n",
              "  'pmid': '26968419',\n",
              "  'title': 'Minimally Invasive Techniques for the Management of Adult UPJ Obstruction.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Optimal guidance is essential for the soft landing task. However, due to its high computational complexities, it is hardly applied to the autonomous guidance. In this paper, a computationally inexpensive optimal guidance algorithm based on the radial basis function neural network (RBFNN) is proposed. The optimization problem of the trajectory for soft landing on asteroids is formulated and transformed into a two-point boundary value problem (TPBVP). Combining the database of initial states with the relative initial co-states, an RBFNN is trained offline. The optimal trajectory of the soft landing is determined rapidly by applying the trained network in the online guidance. The Monte Carlo simulations of soft landing on the Eros433 are performed to demonstrate the effectiveness of the proposed guidance algorithm. ',\n",
              "  'authors': ['Zhang, Peng', 'Liu, Keping', 'Zhao, Bo', 'Li, Yuanchun'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Minor Planets',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Space Flight'],\n",
              "  'pmid': '26367382',\n",
              "  'title': 'A Computationally Inexpensive Optimal Guidance via Radial-Basis-Function Neural Network for Autonomous Soft Landing on Asteroids.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2193-1801',\n",
              "  'IF_2013': '0.982',\n",
              "  'IF_2014': '0.982',\n",
              "  'IF_2015': '0.982',\n",
              "  'IF_2016': '1.13',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2193-1801',\n",
              "  'IsoAbbr': 'Springerplus',\n",
              "  'JrId': 40876,\n",
              "  'MedAbbr': 'Springerplus',\n",
              "  'NLMid': '101597967',\n",
              "  'Titre': 'Springerplus',\n",
              "  'abstract': 'In this article, we propose the application of a modified Taylor series method (MTSM) for the approximation of nonlinear problems described on finite intervals. The issue of Taylor series method with mixed boundary conditions is circumvented using shooting constants and extra derivatives of the problem. In order to show the benefits of this proposal, three different kinds of problems are solved: three-point boundary valued problem (BVP) of third-order with a hyperbolic sine nonlinearity, two-point BVP for a second-order nonlinear differential equation with an exponential nonlinearity, and a two-point BVP for a third-order nonlinear differential equation with a radical nonlinearity. The result shows that the MTSM method is capable to generate easily computable and highly accurate approximations for nonlinear equations.|34L30.',\n",
              "  'authors': ['Vazquez-Leal, Hector',\n",
              "   'Benhammouda, Brahim',\n",
              "   'Filobello-Nino, Uriel Antonio',\n",
              "   'Sarmiento-Reyes, Arturo',\n",
              "   'Jimenez-Fernandez, Victor Manuel',\n",
              "   'Marin-Hernandez, Antonio',\n",
              "   'Herrera-May, Agustin Leobardo',\n",
              "   'Diaz-Sanchez, Alejandro',\n",
              "   'Huerta-Chua, Jesus'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'SpringerPlus',\n",
              "  'keywords': [''],\n",
              "  'pmid': '24790815',\n",
              "  'title': 'Modified Taylor series method for solving nonlinear differential equations with mixed boundary conditions defined on finite intervals.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Coreference resolution is one of the fundamental and challenging tasks in natural language processing. Resolving coreference successfully can have a significant positive effect on downstream natural language processing tasks, such as information extraction and question answering. The importance of coreference resolution for biomedical text analysis applications has increasingly been acknowledged. One of the difficulties in coreference resolution stems from the fact that distinct types of coreference (e.g., anaphora, appositive) are expressed with a variety of lexical and syntactic means (e.g., personal pronouns, definite noun phrases), and that resolution of each combination often requires a different approach. In the biomedical domain, it is common for coreference annotation and resolution efforts to focus on specific subcategories of coreference deemed important for the downstream task. In the current work, we aim to address some of these concerns regarding coreference resolution in biomedical text. We propose a general, modular framework underpinned by a smorgasbord architecture (Bio-SCoRes), which incorporates a variety of coreference types, their mentions and allows fine-grained specification of resolution strategies to resolve coreference of distinct coreference type-mention pairs. For development and evaluation, we used a corpus of structured drug labels annotated with fine-grained coreference information. In addition, we evaluated our approach on two other corpora (i2b2/VA discharge summaries and protein coreference dataset) to investigate its generality and ease of adaptation to other biomedical text types. Our results demonstrate the usefulness of our novel smorgasbord architecture. The specific pipelines based on the architecture perform successfully in linking coreferential mention pairs, while we find that recognition of full mention clusters is more challenging. The corpus of structured drug labels (SPL) as well as the components of Bio-SCoRes and some of the pipelines based on it are publicly available at https://github.com/kilicogluh/Bio-SCoRes. We believe that Bio-SCoRes can serve as a strong and extensible baseline system for coreference resolution of biomedical text. ',\n",
              "  'authors': ['Kilicoglu, Halil', 'Demner-Fushman, Dina'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Data Mining',\n",
              "   'Electronic Health Records',\n",
              "   'Natural Language Processing',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '26934708',\n",
              "  'title': 'Bio-SCoRes: A Smorgasbord Architecture for Coreference Resolution in Biomedical Text.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'To effectively and accurately detect and classify network intrusion data, this paper introduces a general regression neural network (GRNN) based on the artificial immune algorithm with elitist strategies (AIAE). The elitist archive and elitist crossover were combined with the artificial immune algorithm (AIA) to produce the AIAE-GRNN algorithm, with the aim of improving its adaptivity and accuracy. In this paper, the mean square errors (MSEs) were considered the affinity function. The AIAE was used to optimize the smooth factors of the GRNN; then, the optimal smooth factor was solved and substituted into the trained GRNN. Thus, the intrusive data were classified. The paper selected a GRNN that was separately optimized using a genetic algorithm (GA), particle swarm optimization (PSO), and fuzzy C-mean clustering (FCM) to enable a comparison of these approaches. As shown in the results, the AIAE-GRNN achieves a higher classification accuracy than PSO-GRNN, but the running time of AIAE-GRNN is long, which was proved first. FCM and GA-GRNN were eliminated because of their deficiencies in terms of accuracy and convergence. To improve the running speed, the paper adopted principal component analysis (PCA) to reduce the dimensions of the intrusive data. With the reduction in dimensionality, the PCA-AIAE-GRNN decreases in accuracy less and has better convergence than the PCA-PSO-GRNN, and the running speed of the PCA-AIAE-GRNN was relatively improved. The experimental results show that the AIAE-GRNN has a higher robustness and accuracy than the other algorithms considered and can thus be used to classify the intrusive data.',\n",
              "  'authors': ['Wu, Jianfa',\n",
              "   'Peng, Dahao',\n",
              "   'Li, Zhuping',\n",
              "   'Zhao, Li',\n",
              "   'Ling, Huanzhang'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cluster Analysis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Principal Component Analysis'],\n",
              "  'pmid': '25807466',\n",
              "  'title': 'Network intrusion detection based on a general regression neural network optimized by an improved artificial immune algorithm.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1552-4469',\n",
              "  'IF_2013': '13.217',\n",
              "  'IF_2014': '12.996',\n",
              "  'IF_2015': '12.709',\n",
              "  'IF_2016': '15.066',\n",
              "  'IF_2017': '13.843',\n",
              "  'ISSN': '1552-4450',\n",
              "  'ISSN_online': '1552-4469',\n",
              "  'ISSN_print': '1552-4450',\n",
              "  'IsoAbbr': 'Nat. Chem. Biol.',\n",
              "  'JrId': 32624,\n",
              "  'MedAbbr': 'Nat Chem Biol',\n",
              "  'NLMid': '101231976',\n",
              "  'Titre': 'Nat. Chem. Biol.',\n",
              "  'abstract': 'Our ability to directly relate results from test-tube biochemical experiments to the kinetics in living cells is very limited. Here we present experimental and analytical tools to directly study the kinetics of fast biochemical reactions in live cells. Dye-labeled molecules are electroporated into bacterial cells and tracked using super-resolved single-molecule microscopy. Trajectories are analyzed by machine-learning algorithms to directly monitor transitions between bound and free states. In particular, we measure the dwell time of tRNAs on ribosomes, and hence achieve direct measurements of translation rates inside living cells at codon resolution. We find elongation rates with tRNAPhe that are in perfect agreement with previous indirect estimates, and once fMet-tRNAfMet has bound to the 30S ribosomal subunit, initiation of translation is surprisingly fast and does not limit the overall rate of protein synthesis. The experimental and analytical tools for direct kinetics measurements in live cells have applications far beyond bacterial protein synthesis.',\n",
              "  'authors': ['Volkov, Ivan L',\n",
              "   'Lindén, Martin',\n",
              "   'Aguirre Rivera, Javier',\n",
              "   'Ieong, Ka-Weng',\n",
              "   'Metelev, Mikhail',\n",
              "   'Elf, Johan',\n",
              "   'Johansson, Magnus'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nature chemical biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Codon',\n",
              "   'Coloring Agents',\n",
              "   'Electroporation',\n",
              "   'Escherichia coli',\n",
              "   'Fluorescent Dyes',\n",
              "   'Kinetics',\n",
              "   'Machine Learning',\n",
              "   'Microscopy, Fluorescence',\n",
              "   'Microscopy, Video',\n",
              "   'Protein Biosynthesis',\n",
              "   'RNA, Messenger',\n",
              "   'RNA, Transfer',\n",
              "   'RNA, Transfer, Met',\n",
              "   'Ribosome Subunits, Small, Bacterial',\n",
              "   'Ribosomes',\n",
              "   'Single Molecule Imaging'],\n",
              "  'pmid': '29769736',\n",
              "  'title': 'tRNA tracking for direct measurements of protein synthesis kinetics in live cells.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-8355',\n",
              "  'IF_2013': '1.039',\n",
              "  'IF_2014': '0.912',\n",
              "  'IF_2015': '1.029',\n",
              "  'IF_2016': '1.139',\n",
              "  'IF_2017': '1.133',\n",
              "  'ISSN': '1572-3887',\n",
              "  'ISSN_online': '1875-8355',\n",
              "  'ISSN_print': '1572-3887',\n",
              "  'IsoAbbr': 'Protein J.',\n",
              "  'JrId': 31918,\n",
              "  'MedAbbr': 'Protein J',\n",
              "  'NLMid': '101212092',\n",
              "  'Titre': 'Protein J.',\n",
              "  'abstract': 'Ever since the signal hypothesis was proposed in 1971, the exact nature of signal peptides has been a focus point of research. The prediction of signal peptides and protein subcellular location from amino acid sequences has been an important problem in bioinformatics since the dawn of this research field, involving many statistical and machine learning technologies. In this review, we provide a historical account of how position-weight matrices, artificial neural networks, hidden Markov models, support vector machines and, lately, deep learning techniques have been used in the attempts to predict where proteins go. Because the secretory pathway was the first one to be studied both experimentally and through bioinformatics, our main focus is on the historical development of prediction methods for signal peptides that target proteins for secretion; prediction methods to identify targeting signals for other cellular compartments are treated in less detail.',\n",
              "  'authors': ['Nielsen, Henrik',\n",
              "   'Tsirigos, Konstantinos D',\n",
              "   'Brunak, Søren',\n",
              "   'von Heijne, Gunnar'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'The protein journal',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31119599',\n",
              "  'title': 'A Brief History of Protein Sorting Prediction.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Early recognition of ventricular fibrillation (VF) and electrical therapy are key for the survival of out-of-hospital cardiac arrest (OHCA) patients treated with automated external defibrillators (AED). AED algorithms for VF-detection are customarily assessed using Holter recordings from public electrocardiogram (ECG) databases, which may be different from the ECG seen during OHCA events. This study evaluates VF-detection using data from both OHCA patients and public Holter recordings. ECG-segments of 4-s and 8-s duration were analyzed. For each segment 30 features were computed and fed to state of the art machine learning (ML) algorithms. ML-algorithms with built-in feature selection capabilities were used to determine the optimal feature subsets for both databases. Patient-wise bootstrap techniques were used to evaluate algorithm performance in terms of sensitivity (Se), specificity (Sp) and balanced error rate (BER). Performance was significantly better for public data with a mean Se of 96.6%, Sp of 98.8% and BER 2.2% compared to a mean Se of 94.7%, Sp of 96.5% and BER 4.4% for OHCA data. OHCA data required two times more features than the data from public databases for an accurate detection (6 vs 3). No significant differences in performance were found for different segment lengths, the BER differences were below 0.5-points in all cases. Our results show that VF-detection is more challenging for OHCA data than for data from public databases, and that accurate VF-detection is possible with segments as short as 4-s. ',\n",
              "  'authors': ['Figuera, Carlos',\n",
              "   'Irusta, Unai',\n",
              "   'Morgado, Eduardo',\n",
              "   'Aramendi, Elisabete',\n",
              "   'Ayala, Unai',\n",
              "   'Wik, Lars',\n",
              "   'Kramer-Johansen, Jo',\n",
              "   'Eftestøl, Trygve',\n",
              "   'Alonso-Atienza, Felipe'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Automation',\n",
              "   'Databases as Topic',\n",
              "   'Defibrillators',\n",
              "   'Electrocardiography',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Out-of-Hospital Cardiac Arrest',\n",
              "   'Time Factors'],\n",
              "  'pmid': '27441719',\n",
              "  'title': 'Machine Learning Techniques for the Detection of Shockable Rhythms in Automated External Defibrillators.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'We propose a hybrid method for forecasting the wind speed. The wind speed data is first decomposed into intrinsic mode functions (IMFs) with empirical mode decomposition. Based on the partial autocorrelation factor of the individual IMFs, adaptive methods are then employed for the prediction of IMFs. Least squares-support vector machines are employed for IMFs with weak correlation factor, and autoregressive model with Kalman filter is employed for IMFs with high correlation factor. Multistep prediction with the proposed hybrid method resulted in improved forecasting. Results with wind speed data show that the proposed method provides better forecasting compared to the existing methods. ',\n",
              "  'authors': ['Tatinati, Sivanagaraja', 'Veluvolu, Kalyana C'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Forecasting',\n",
              "   'Least-Squares Analysis',\n",
              "   'Models, Statistical',\n",
              "   'Models, Theoretical',\n",
              "   'Regression Analysis',\n",
              "   'Support Vector Machine',\n",
              "   'Wind'],\n",
              "  'pmid': '24453872',\n",
              "  'title': 'A hybrid approach for short-term forecasting of wind speed.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1536-3686',\n",
              "  'IF_2013': '1.127',\n",
              "  'IF_2014': '1.129',\n",
              "  'IF_2015': '1.132',\n",
              "  'IF_2016': '1.588',\n",
              "  'IF_2017': '1',\n",
              "  'ISSN': '1075-2765',\n",
              "  'ISSN_online': '1536-3686',\n",
              "  'ISSN_print': '1075-2765',\n",
              "  'IsoAbbr': 'Am J Ther',\n",
              "  'JrId': 20856,\n",
              "  'MedAbbr': 'Am J Ther',\n",
              "  'NLMid': '9441347',\n",
              "  'Titre': 'Am J Ther',\n",
              "  'abstract': 'Fuzzy logic can handle questions to which the answers may be \"yes\" at one time and \"no\" at the other, or may be partially true and untrue. Pharmacodynamic data deal with questions such as \"Does a patient respond to a particular drug dose or not,\" or \"Does a drug cause the same effects at the same time in the same subject or not.\" Such questions are typically of a fuzzy nature and might, therefore, benefit from an analysis based on fuzzy logic.The objective was to assess whether fuzzy logic can improve the precision of predictive models for pharmacodynamic data.The methods and results were as follows: (1). The quantal pharmacodynamic effects of different induction dosages of thiopental on numbers of responding subjects were used as the first example. Regression analysis of the fuzzy-modeled outcome data on the input data provided a much better fit than did the unmodeled output values with r-square values of 0.852 (F-value = 40.34) and 0.555 (F-value = 8.74), respectively. (2). The time-response effect propranolol on peripheral arterial flow was used as a second example. Regression analysis of the fuzzy-modeled outcome data on the input data provided a better fit than did the unmodeled output values with r-square values of 0.990 (F-value = 416) and 0.977 (F-value = 168), respectively.Fuzzy modeling may better than conventional statistical method fit and predict pharmacodynamic data, such as, for example, quantal dose response and time response data. This may be relevant to future pharmacodynamic research. ',\n",
              "  'authors': ['Cleophas, Eugene P E M', 'Cleophas, Ton J'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'American journal of therapeutics',\n",
              "  'keywords': ['Adrenergic beta-Antagonists',\n",
              "   'Algorithms',\n",
              "   'Dose-Response Relationship, Drug',\n",
              "   'Forearm',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Models, Statistical',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Pharmacokinetics',\n",
              "   'Predictive Value of Tests',\n",
              "   'Propranolol',\n",
              "   'Regional Blood Flow',\n",
              "   'Regression Analysis'],\n",
              "  'pmid': '21317764',\n",
              "  'title': 'Fuzzy modeling, a novel approach to studying pharmacodynamics.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'N-glycolylneuraminic acid (Neu5Gc) is generated by hydroxylation of CMP-Neu5Ac to CMP-Neu5Gc, catalyzed by CMP-Neu5Ac hydroxylase (CMAH). However, humans lack this common mammalian cell surface molecule, Neu5Gc, due to inactivation of the CMAH gene during evolution. CMAH is one of several human-specific genes whose function has been lost by disruption or deletion of the coding frame. It has been suggested that CMAH inactivation has resulted in biochemical or physiological characteristics that have resulted in human-specific diseases.|To identify differential gene expression profiles associated with the loss of Neu5Gc expression, we performed microarray analysis using Illumina MouseRef-8 v2 Expression BeadChip, using the main tissues (lung, kidney, and heart) from control mice and CMP-Neu5Ac hydroxylase (Cmah) gene knock-out mice, respectively. Out of a total of 25,697 genes, 204, 162, and 147 genes were found to be significantly modulated in the lung, kidney, and heart tissues of the Cmah null mouse, respectively. In this study, we examined the gene expression profiles, using three commercial pathway analysis software packages: Ingenuity Pathways Analysis, Kyoto Encyclopedia of Genes and Genomes analysis, and Pathway Studio. The gene ontology analysis revealed that the top 6 biological processes of these genes included protein metabolism and modification, signal transduction, lipid, fatty acid, and steroid metabolism, nucleoside, nucleotide and nucleic acid metabolism, immunity and defense, and carbohydrate metabolism. Gene interaction network analysis showed a common network that was common to the different tissues of the Cmah null mouse. However, the expression of most sialytransferase mRNAs of Hanganutziu-Deicher antigen, sialy-Tn antigen, Forssman antigen, and Tn antigen was significantly down-regulated in the liver tissue of Cmah null mice.|Mice bearing a human-like deletion of the Cmah gene serve as an important model for the study of abnormal pathogenesis and/or metabolism caused by the evolutionary loss of Neu5Gc synthesis in humans.',\n",
              "  'authors': ['Kwon, Deug-Nam', 'Chang, Byung-Soo', 'Kim, Jin-Hoi'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Knockout Techniques',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Genotype',\n",
              "   'Immunohistochemistry',\n",
              "   'Kidney',\n",
              "   'Lung',\n",
              "   'Male',\n",
              "   'Mice',\n",
              "   'Mice, Knockout',\n",
              "   'Mixed Function Oxygenases',\n",
              "   'Myocardium',\n",
              "   'Protein Interaction Maps',\n",
              "   'Sialyltransferases',\n",
              "   'Signal Transduction',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '25229777',\n",
              "  'title': 'Gene expression and pathway analysis of effects of the CMAH deactivation on mouse lung, kidney and heart.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1558-2264',\n",
              "  'IF_2013': '1.064',\n",
              "  'IF_2014': '1.031',\n",
              "  'IF_2015': '1.184',\n",
              "  'IF_2016': '1.36',\n",
              "  'IF_2017': '1.42',\n",
              "  'ISSN': '0733-8651',\n",
              "  'ISSN_online': '1558-2264',\n",
              "  'ISSN_print': '0733-8651',\n",
              "  'IsoAbbr': 'Cardiol Clin',\n",
              "  'JrId': 2806,\n",
              "  'MedAbbr': 'Cardiol Clin',\n",
              "  'NLMid': '8300331',\n",
              "  'Titre': 'Cardiol Clin',\n",
              "  'abstract': 'With an understanding of the pathophysiology of ACS and an increasing number of early therapeutic options, there has been a shift in focus from ruling-out MI to identifying and stratifying risk in all patients with potential ACS. The presenting symptoms and ECG still remain the cornerstone of immediate diagnosis and triage. Through the application of new technologies, such as the cardiac troponins, and a reassessment of techniques, such as perfusion imaging and echocardiography, the clinician has an increasing selection of methods to rapidly assess chest pain of potential ischemic etiology. Coinciding with the evaluation of technology has been the development of the concept of the CPU and associated rapid diagnostic protocols. These protocols, whether they utilize the assistance of mathematic predictive instruments or represent simple triage schemes, form the backbone of a system to improve the care of patients with ACS in the current milieu of cost containment.',\n",
              "  'authors': ['deFilippi, C R', 'Runge, M S'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Cardiology clinics',\n",
              "  'keywords': ['Angina, Unstable',\n",
              "   'Biomarkers',\n",
              "   'Chest Pain',\n",
              "   'Electrocardiography',\n",
              "   'Emergency Service, Hospital',\n",
              "   'Exercise Test',\n",
              "   'Humans',\n",
              "   'Myocardial Infarction',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Point-of-Care Systems',\n",
              "   'Predictive Value of Tests',\n",
              "   'ROC Curve',\n",
              "   'Radionuclide Imaging',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '10384829',\n",
              "  'title': 'Evaluating the chest pain patient. Scope of the problem.',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Echo state networks (ESNs) with multi-clustered reservoir topology perform better in reservoir computing and robustness than those with random reservoir topology. However, these ESNs have a complex reservoir topology, which leads to difficulties in reservoir generation. This study focuses on the reservoir generation problem when ESN is used in environments with sufficient priori data available. Accordingly, a priori data-driven multi-cluster reservoir generation algorithm is proposed. The priori data in the proposed algorithm are used to evaluate reservoirs by calculating the precision and standard deviation of ESNs. The reservoirs are produced using the clustering method; only the reservoir with a better evaluation performance takes the place of a previous one. The final reservoir is obtained when its evaluation score reaches the preset requirement. The prediction experiment results obtained using the Mackey-Glass chaotic time series show that the proposed reservoir generation algorithm provides ESNs with extra prediction precision and increases the structure complexity of the network. Further experiments also reveal the appropriate values of the number of clusters and time window size to obtain optimal performance. The information entropy of the reservoir reaches the maximum when ESN gains the greatest precision. ',\n",
              "  'authors': ['Li, Xiumin', 'Zhong, Ling', 'Xue, Fangzheng', 'Zhang, Anguo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cluster Analysis',\n",
              "   'Computer Simulation',\n",
              "   'Database Management Systems',\n",
              "   'Databases, Factual',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '25875296',\n",
              "  'title': 'A priori data-driven multi-clustered reservoir generation algorithm for echo state network.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['UI'],\n",
              "  'ESSN': '1079-7114',\n",
              "  'IF_2013': '7.728',\n",
              "  'IF_2014': '7.512',\n",
              "  'IF_2015': '7.645',\n",
              "  'IF_2016': '8.462',\n",
              "  'IF_2017': '8.839',\n",
              "  'ISSN': '0031-9007',\n",
              "  'ISSN_online': '1079-7114',\n",
              "  'ISSN_print': '0031-9007',\n",
              "  'IsoAbbr': 'Phys. Rev. Lett.',\n",
              "  'JrId': 21217,\n",
              "  'MedAbbr': 'Phys Rev Lett',\n",
              "  'NLMid': '0401141',\n",
              "  'Titre': 'Phys. Rev. Lett.',\n",
              "  'abstract': 'We apply nonlinear dynamical system techniques to recurrent neural networks. In particular, we numerically analyze the dynamical system characteristics of the online learning process. By introducing the notion of inaccessibility, we show that the learning process is well characterized by strong nonhyperbolicity and inaccessibility, which is a greater uncertainty than chaotic unpredictability. These results are clearly contrasted with a gradient descent dynamics, or ordinary chaos.',\n",
              "  'authors': ['Saito, Asaki', 'Taiji, Makoto', 'Ikegami, Takashi'],\n",
              "  'categories': ['PHYSICS, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Physical review letters',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Learning',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '15525035',\n",
              "  'title': 'Inaccessibility in online learning of recurrent neural networks.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-6882',\n",
              "  'IF_2013': '6.765',\n",
              "  'IF_2014': '6.628',\n",
              "  'IF_2015': '6.373',\n",
              "  'IF_2016': '6.133',\n",
              "  'IF_2017': '6.541',\n",
              "  'ISSN': '0959-4388',\n",
              "  'ISSN_online': '1873-6882',\n",
              "  'ISSN_print': '0959-4388',\n",
              "  'IsoAbbr': 'Curr. Opin. Neurobiol.',\n",
              "  'JrId': 2201,\n",
              "  'MedAbbr': 'Curr Opin Neurobiol',\n",
              "  'NLMid': '9111376',\n",
              "  'Titre': 'Curr. Opin. Neurobiol.',\n",
              "  'abstract': 'Many recent studies of neurons recorded from cortex reveal complex temporal dynamics. How such dynamics embody the computations that ultimately lead to behavior remains a mystery. Approaching this issue requires developing plausible hypotheses couched in terms of neural dynamics. A tool ideally suited to aid in this question is the recurrent neural network (RNN). RNNs straddle the fields of nonlinear dynamical systems and machine learning and have recently seen great advances in both theory and application. I summarize recent theoretical and technological advances and highlight an example of how RNNs helped to explain perplexing high-dimensional neurophysiological data in the prefrontal cortex.',\n",
              "  'authors': ['Sussillo, David'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Current opinion in neurobiology',\n",
              "  'keywords': ['Animals',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prefrontal Cortex'],\n",
              "  'pmid': '24509098',\n",
              "  'title': 'Neural circuits as computational dynamical systems.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TQ'],\n",
              "  'ESSN': '1432-1076',\n",
              "  'IF_2013': '1.983',\n",
              "  'IF_2014': '1.89',\n",
              "  'IF_2015': '1.791',\n",
              "  'IF_2016': '1.921',\n",
              "  'IF_2017': '2.242',\n",
              "  'ISSN': '0340-6199',\n",
              "  'ISSN_online': '1432-1076',\n",
              "  'ISSN_print': '0340-6199',\n",
              "  'IsoAbbr': 'Eur. J. Pediatr.',\n",
              "  'JrId': 3621,\n",
              "  'MedAbbr': 'Eur J Pediatr',\n",
              "  'NLMid': '7603873',\n",
              "  'Titre': 'Eur. J. Pediatr.',\n",
              "  'abstract': 'In addition to conventional open surgery and endoscopic techniques, laparoscopic correction of vesicoureteric reflux, sometimes even robot-assisted, is becoming an alternative surgical treatment modality for this condition in a number of centres around the world. At least for a subgroup of patients laparoscopists are trying to develop new techniques in an effort to combine the best of both worlds: the minimal invasiveness of the STING and the same lasting effectiveness as in open surgery. The efficacy and potential advantages or disadvantages of these techniques are still under investigation. The different laparoscopic techniques and available data are presented.',\n",
              "  'authors': ['Callewaert, Piet R H'],\n",
              "  'categories': ['PEDIATRICS'],\n",
              "  'journal': 'European journal of pediatrics',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Cystoscopy',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Robotics',\n",
              "   'Surgical Procedures, Operative',\n",
              "   'Vesico-Ureteral Reflux'],\n",
              "  'pmid': '17372758',\n",
              "  'title': 'What is new in surgical treatment of vesicoureteric reflux?',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'Topographic organization is one of the main principles of organization in the human brain. Specifically, whole-brain topographic mapping using spectral analysis is responsible for one of the greatest advances in vision research. Thus, it is intriguing that although topography is a key feature also in the motor system, whole-body somatosensory-motor mapping using spectral analysis has not been conducted in humans outside M1/SMA. Here, using this method, we were able to map a homunculus in the globus pallidus, a key target area for deep brain stimulation, which has not been mapped noninvasively or in healthy subjects. The analysis clarifies contradictory and partial results regarding somatotopy in the caudal-cingulate zone and rostral-cingulate zone in the medial wall and in the putamen. Most of the results were confirmed at the single-subject level and were found to be compatible with results from animal studies. Using multivoxel pattern analysis, we could predict movements of individual body parts in these homunculi, thus confirming that they contain somatotopic information. Using functional connectivity, we demonstrate interhemispheric functional somatotopic connectivity of these homunculi, such that the somatotopy in one hemisphere could have been found given the connectivity pattern of the corresponding regions of interest in the other hemisphere. When inspecting the somatotopic and nonsomatotopic connectivity patterns, a similarity index indicated that the pattern of connected and nonconnected regions of interest across different homunculi is similar for different body parts and hemispheres. The results show that topographical gradients are even more widespread than previously assumed in the somatosensory-motor system. Spectral analysis can thus potentially serve as a gold standard for defining somatosensory-motor system areas for basic research and clinical applications. ',\n",
              "  'authors': ['Zeharia, Noa', 'Hertz, Uri', 'Flash, Tamar', 'Amedi, Amir'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Adult',\n",
              "   'Afferent Pathways',\n",
              "   'Brain Mapping',\n",
              "   'Female',\n",
              "   'Human Body',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Motor Cortex',\n",
              "   'Movement',\n",
              "   'Oxygen',\n",
              "   'Regression Analysis',\n",
              "   'Sensation',\n",
              "   'Sensory Deprivation',\n",
              "   'Spectrum Analysis',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '25698725',\n",
              "  'title': 'New whole-body sensory-motor gradients revealed using phase-locked analysis and verified using multivoxel pattern analysis and functional connectivity.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': \"Prediction of radiation pneumonitis (RP) has been shown to be challenging due to the involvement of a variety of factors including dose-volume metrics and radiosensitivity biomarkers. Some of these factors are highly correlated and might affect prediction results when combined. Bayesian network (BN) provides a probabilistic framework to represent variable dependencies in a directed acyclic graph. The aim of this study is to integrate the BN framework and a systems' biology approach to detect possible interactions among RP risk factors and exploit these relationships to enhance both the understanding and prediction of RP.|The authors studied 54 nonsmall-cell lung cancer patients who received curative 3D-conformal radiotherapy. Nineteen RP events were observed (common toxicity criteria for adverse events grade 2 or higher). Serum concentration of the following four candidate biomarkers were measured at baseline and midtreatment: alpha-2-macroglobulin, angiotensin converting enzyme (ACE), transforming growth factor, interleukin-6. Dose-volumetric and clinical parameters were also included as covariates. Feature selection was performed using a Markov blanket approach based on the Koller-Sahami filter. The Markov chain Monte Carlo technique estimated the posterior distribution of BN graphs built from the observed data of the selected variables and causality constraints. RP probability was estimated using a limited number of high posterior graphs (ensemble) and was averaged for the final RP estimate using Bayes' rule. A resampling method based on bootstrapping was applied to model training and validation in order to control under- and overfit pitfalls.|RP prediction power of the BN ensemble approach reached its optimum at a size of 200. The optimized performance of the BN model recorded an area under the receiver operating characteristic curve (AUC) of 0.83, which was significantly higher than multivariate logistic regression (0.77), mean heart dose (0.69), and a pre-to-midtreatment change in ACE (0.66). When RP prediction was made only with pretreatment information, the AUC ranged from 0.76 to 0.81 depending on the ensemble size. Bootstrap validation of graph features in the ensemble quantified confidence of association between variables in the graphs where ten interactions were statistically significant.|The presented BN methodology provides the flexibility to model hierarchical interactions between RP covariates, which is applied to probabilistic inference on RP. The authors' preliminary results demonstrate that such framework combined with an ensemble method can possibly improve prediction of RP under real-life clinical circumstances such as missing data or treatment plan adaptation.\",\n",
              "  'authors': ['Lee, Sangkyu',\n",
              "   'Ybarra, Norma',\n",
              "   'Jeyaseelan, Krishinima',\n",
              "   'Faria, Sergio',\n",
              "   'Kopek, Neil',\n",
              "   'Brisebois, Pascale',\n",
              "   'Bradley, Jeffrey D',\n",
              "   'Robinson, Clifford',\n",
              "   'Seuntjens, Jan',\n",
              "   'El Naqa, Issam'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Area Under Curve',\n",
              "   'Bayes Theorem',\n",
              "   'Biomarkers',\n",
              "   'Carcinoma, Non-Small-Cell Lung',\n",
              "   'Cohort Studies',\n",
              "   'Heart',\n",
              "   'Humans',\n",
              "   'Interleukin-6',\n",
              "   'Logistic Models',\n",
              "   'Machine Learning',\n",
              "   'Markov Chains',\n",
              "   'Monte Carlo Method',\n",
              "   'Multivariate Analysis',\n",
              "   'Peptidyl-Dipeptidase A',\n",
              "   'ROC Curve',\n",
              "   'Radiation Pneumonitis',\n",
              "   'Radiotherapy Dosage',\n",
              "   'Radiotherapy, Conformal',\n",
              "   'Transforming Growth Factors',\n",
              "   'alpha-Macroglobulins'],\n",
              "  'pmid': '25979036',\n",
              "  'title': 'Bayesian network ensemble as a multivariate strategy to predict radiation pneumonitis risk.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NI'],\n",
              "  'ESSN': '1529-2916',\n",
              "  'IF_2013': '24.973',\n",
              "  'IF_2014': '20.004',\n",
              "  'IF_2015': '19.381',\n",
              "  'IF_2016': '21.506',\n",
              "  'IF_2017': '21.809',\n",
              "  'ISSN': '1529-2908',\n",
              "  'ISSN_online': '1529-2916',\n",
              "  'ISSN_print': '1529-2908',\n",
              "  'IsoAbbr': 'Nat. Immunol.',\n",
              "  'JrId': 21750,\n",
              "  'MedAbbr': 'Nat Immunol',\n",
              "  'NLMid': '100941354',\n",
              "  'Titre': 'Nat. Immunol.',\n",
              "  'abstract': 'Lymphocytes are recruited from blood by high-endothelial venules (HEVs). We performed transcriptomic analyses and identified molecular signatures that distinguish HEVs from capillary endothelium and that define tissue-specific HEV specialization. Capillaries expressed gene programs for vascular development. HEV-expressed genes showed enrichment for genes encoding molecules involved in immunological defense and lymphocyte migration. We identify capillary and HEV markers and candidate mechanisms for regulated recruitment of lymphocytes, including a lymph node HEV-selective transmembrane mucin; transcriptional control of functionally specialized carbohydrate ligands for lymphocyte L-selectin; HEV expression of molecules for transendothelial migration; and metabolic programs for lipid mediators of lymphocyte motility and chemotaxis. We also elucidate a carbohydrate-recognition pathway that targets B cells to intestinal lymphoid tissues, defining CD22 as a lectin-homing receptor for mucosal HEVs. ',\n",
              "  'authors': ['Lee, Mike',\n",
              "   'Kiefel, Helena',\n",
              "   'LaJevic, Melissa D',\n",
              "   'Macauley, Matthew S',\n",
              "   'Kawashima, Hiroto',\n",
              "   \"O'Hara, Edward\",\n",
              "   'Pan, Junliang',\n",
              "   'Paulson, James C',\n",
              "   'Butcher, Eugene C'],\n",
              "  'categories': ['IMMUNOLOGY'],\n",
              "  'journal': 'Nature immunology',\n",
              "  'keywords': ['Animals',\n",
              "   'Capillaries',\n",
              "   'Cell Movement',\n",
              "   'Endothelial Cells',\n",
              "   'Endothelium',\n",
              "   'Female',\n",
              "   'Flow Cytometry',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Lymph Nodes',\n",
              "   'Lymphocytes',\n",
              "   'Lymphoid Tissue',\n",
              "   'Male',\n",
              "   'Mice, Inbred BALB C',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Mice, Knockout',\n",
              "   'Mice, Transgenic',\n",
              "   'Microscopy, Confocal',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Venules'],\n",
              "  'pmid': '25173345',\n",
              "  'title': 'Transcriptional programs of lymphoid tissue capillary and high endothelium reveal control mechanisms for lymphocyte homing.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Psychiatric illness is unlikely to arise from pathology occurring uniformly across all cell types in affected brain regions. Despite this, transcriptomic analyses of the human brain have typically been conducted using macro-dissected tissue due to the difficulty of performing single-cell type analyses with donated post-mortem brains. To address this issue statistically, we compiled a database of several thousand transcripts that were specifically-enriched in one of 10 primary cortical cell types in previous publications. Using this database, we predicted the relative cell type content for 833 human cortical samples using microarray or RNA-Seq data from the Pritzker Consortium (GSE92538) or publicly-available databases (GSE53987, GSE21935, GSE21138, CommonMind Consortium). These predictions were generated by averaging normalized expression levels across transcripts specific to each cell type using our R-package BrainInABlender (validated and publicly-released on github). Using this method, we found that the principal components of variation in the datasets strongly correlated with the predicted neuronal/glial content of the samples. This variability was not simply due to dissection-the relative balance of brain cell types appeared to be influenced by a variety of demographic, pre- and post-mortem variables. Prolonged hypoxia around the time of death predicted increased astrocytic and endothelial gene expression, illustrating vascular upregulation. Aging was associated with decreased neuronal gene expression. Red blood cell gene expression was reduced in individuals who died following systemic blood loss. Subjects with Major Depressive Disorder had decreased astrocytic gene expression, mirroring previous morphometric observations. Subjects with Schizophrenia had reduced red blood cell gene expression, resembling the hypofrontality detected in fMRI experiments. Finally, in datasets containing samples with especially variable cell content, we found that controlling for predicted sample cell content while evaluating differential expression improved the detection of previously-identified psychiatric effects. We conclude that accounting for cell type can greatly improve the interpretability of transcriptomic data.',\n",
              "  'authors': ['Hagenauer, Megan Hastings',\n",
              "   'Schulmann, Anton',\n",
              "   'Li, Jun Z',\n",
              "   'Vawter, Marquis P',\n",
              "   'Walsh, David M',\n",
              "   'Thompson, Robert C',\n",
              "   'Turner, Cortney A',\n",
              "   'Bunney, William E',\n",
              "   'Myers, Richard M',\n",
              "   'Barchas, Jack D',\n",
              "   'Schatzberg, Alan F',\n",
              "   'Watson, Stanley J',\n",
              "   'Akil, Huda'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Animals',\n",
              "   'Brain',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Mental Disorders',\n",
              "   'Mice',\n",
              "   'Oligonucleotide Array Sequence Analysis'],\n",
              "  'pmid': '30016334',\n",
              "  'title': 'Inference of cell type content from human brain transcriptomic datasets illuminates the effects of age, manner of death, dissection, and psychiatric diagnosis.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'The Academic Robotics Group prospectively studied 211 robotically assisted operations to assess the safety and utility of robotically assisted surgery.|All operations took place at one of four member institutions between June 2000 and June 2001 using the recently FDA-approved daVinci robotic system. A variety of procedures were undertaken, including antireflux surgery (69), cholecystectomy (36), Heller myotomy (26), bowel resection (17), donor nephrectomy (15), left internal mammery artery mobilization (14), gastric bypass (seven), splenectomy (seven), adrenalectomy (six), exploratory laparoscopy (three), pyloroplasty (four), gastrojejunostomy (two), distal pancreatectomy (one), duodenal polypectomy (one), esophagectomy (one), gastric mass resection (one), and lysis of adhesions (one).|Average operating room time was 188 min (range 45 to 387, SD = 83), surgical time 143 min (range 35 to 462, SD = 63), and robot time 90 min (range 12 to 235, SD = 47). Median length of stay was 1 day (range 0 to 37). There were 8 (4%) technical complications during procedures, five minor (four hook cautery dislodgement, one slipped robotic trocar) and three major (system malfunctions, two of which required conversion to standard laparoscopy). In all cases, technical problems caused only delay, without apparent altered outcome. There were medical/surgical complications in nine patients (4%). Six (3%) were considered major, including one death unrelated to the robotic procedure.|The results of robotic-assisted surgery compare favorably with those of conventional laparoscopy with respect to mortality, complications, and length of stay. Robotic-assisted surgery is safe and effective and is a new reality for American surgery. The role of these devices in surgery will expand as the technology evolves.',\n",
              "  'authors': ['Talamini, M A', 'Chapman, S', 'Horgan, S', 'Melvin, W S'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Digestive System Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Length of Stay',\n",
              "   'Nephrectomy',\n",
              "   'Postoperative Complications',\n",
              "   'Prospective Studies',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Treatment Outcome',\n",
              "   'United States'],\n",
              "  'pmid': '12915974',\n",
              "  'title': 'A prospective analysis of 211 robotic-assisted surgical procedures.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1095-8673',\n",
              "  'IF_2013': '2.121',\n",
              "  'IF_2014': '1.936',\n",
              "  'IF_2015': '2.198',\n",
              "  'IF_2016': '2.187',\n",
              "  'IF_2017': '2.051',\n",
              "  'ISSN': '0022-4804',\n",
              "  'ISSN_online': '1095-8673',\n",
              "  'ISSN_print': '0022-4804',\n",
              "  'IsoAbbr': 'J. Surg. Res.',\n",
              "  'JrId': 5287,\n",
              "  'MedAbbr': 'J Surg Res',\n",
              "  'NLMid': '0376340',\n",
              "  'Titre': 'J. Surg. Res.',\n",
              "  'abstract': 'Widespread diffusion of minimally-invasive surgery for gastric cancer treatment is limited by the complexity of performing an extended D2-lymphadenectomy. This surgical step can be facilitated by using robot-assisted surgery. The aim of this study is to describe our technique and short-term results of a consecutive series of full robotic gastrectomies with D2-lymphadenectomy for gastric cancer, using the da Vinci Surgical System.|Between May 2004 and December 2009, we performed 24 consecutive full robot-assisted total and subtotal gastrectomies with extended D2-lymphadenectomy for histologically-proven gastric adenocarcinoma. Data referring to 11 robot-assisted total gastrectomies and 13 subtotal gastrectomies were collected in a database and analyzed.|Median operative time was 267.50 min (255-305). Median intraoperative blood loss was 30 mL. Median number of harvested lymph nodes was 28 (23-34). Resection margins were negative in all cases. No conversions occurred. Surgery-related morbidity was 8%. Thirty-day mortality was 0%. Liquid diet started on postoperative d 5 (2-5). Median length of stay was 6 d (5-8).|Robot-assisted gastrectomy with D2-lymphadenectomy is a safe technique and allows achieving an adequate lymph node harvest and optimal R0-resection rates with low postoperative morbidity and the learning curve appears to be shorter than in laparoscopic surgery. Longer follow-up and randomized clinical trials are needed to define the role of robot-assistance in gastric cancer surgery.',\n",
              "  'authors': [\"D'Annibale, Annibale\",\n",
              "   'Pende, Vito',\n",
              "   'Pernazza, Graziano',\n",
              "   'Monsellato, Igor',\n",
              "   'Mazzocchi, Paolo',\n",
              "   'Lucandri, Giorgio',\n",
              "   'Morpurgo, Emilio',\n",
              "   'Contardo, Tania',\n",
              "   'Sovernigo, Gianna'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The Journal of surgical research',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Aged',\n",
              "   'Blood Loss, Surgical',\n",
              "   'Databases, Factual',\n",
              "   'Disease-Free Survival',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Gastrectomy',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Pilot Projects',\n",
              "   'Robotics',\n",
              "   'Stomach Neoplasms',\n",
              "   'Tissue and Organ Harvesting'],\n",
              "  'pmid': '21227455',\n",
              "  'title': 'Full robotic gastrectomy with extended (D2) lymphadenectomy for gastric cancer: surgical technique and preliminary results.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Incomprehension and denial of the theory of evolution among high school students has been observed to also occur when teachers are not equipped to deliver a compelling case also for human evolution based on fossil evidence. This paper assesses the outcomes of a novel inquiry-based paleoanthropology lab teaching human evolution to high-school students. The inquiry-based Be a Paleoanthropologist for a Day lab placed a dozen hominin skulls into the hands of high-school students. Upon measuring three variables of human evolution, students explain what they have observed and discuss findings. In the 2013/14 school year, 11 biology classes in 7 schools in the Greater New Orleans area participated in this lab. The interviewed teacher cohort unanimously agreed that the lab featuring hominin skull replicas and stimulating student inquiry was a pedagogically excellent method of delivering the subject of human evolution. First, the lab's learning path of transforming facts to data, information to knowledge, and knowledge to acceptance empowered students to themselves execute part of the science that underpins our understanding of deep time hominin evolution. Second, although challenging, the hands-on format of the lab was accessible to high-school students, most of whom were readily able to engage the lab's scientific process. Third, the lab's exciting and compelling pedagogy unlocked higher order thinking skills, effectively activating the cognitive, psychomotor and affected learning domains as defined in Bloom's taxonomy. Lastly, the lab afforded students a formative experience with a high degree of retention and epistemic depth. Further study is warranted to gauge the degree of these effects. \",\n",
              "  'authors': ['Bayer, Chris N', 'Luberda, Michael'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Comprehension',\n",
              "   'Educational Measurement',\n",
              "   'Hominidae',\n",
              "   'Humans',\n",
              "   'Laboratories',\n",
              "   'Paleontology',\n",
              "   'Skull',\n",
              "   'Statistics as Topic',\n",
              "   'Students',\n",
              "   'Teaching'],\n",
              "  'pmid': '27513927',\n",
              "  'title': 'Measure, Then Show: Grasping Human Evolution Through an Inquiry-Based, Data-driven Hominin Skulls Lab.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1756-6606',\n",
              "  'IF_2013': '4.345',\n",
              "  'IF_2014': '4.902',\n",
              "  'IF_2015': '3.745',\n",
              "  'IF_2016': '3.41',\n",
              "  'IF_2017': '3.449',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1756-6606',\n",
              "  'IsoAbbr': 'Mol Brain',\n",
              "  'JrId': 34230,\n",
              "  'MedAbbr': 'Mol Brain',\n",
              "  'NLMid': '101468876',\n",
              "  'Titre': 'Mol Brain',\n",
              "  'abstract': 'Hypomyelination in the central nerves system (CNS) is one of the most obviously pathological features in Niemann-Pick Type C disease (NPC), which is a rare neurodegenerative disorder caused by mutations in the NPC intracellular cholesterol transporter 1 or 2 (Npc1 or Npc2). Npc1 plays key roles in both neurons and oligodendrocytes during myelination, however, the linkage between the disturbed cholesterol transport and inhibited myelination is unrevealed. In this study, mass spectrometry (MS)-based differential quantitative proteomics was applied to compare protein composition in the corpus callosum between wild type (WT) and NPC mice. In total, 3009 proteins from both samples were identified, including myelin structural proteins, neuronal proteins, and astrocyte-specific proteins. In line to hypomyelination, our data revealed downregulation of myelin structural and indispensable proteins in Npc1 mutant mice. Notably, the reduced ceramide synthase 2 (Cers2), UDP glycosyltransferase 8 (Ugt8), and glycolipid transfer protein (Gltp) indicate the altered sphingolipid metabolism in the disease and the involvement of Gltp in myelination. The identification of most reported myelin structural proteins and proteins from other cell types advocates the use of the corpus callosum to investigate proteins in different cell types that regulate myelination.',\n",
              "  'authors': ['Yang, Fan',\n",
              "   'Guan, Yudong',\n",
              "   'Feng, Xiao',\n",
              "   'Rolfs, Arndt',\n",
              "   'Schlüter, Hartmut',\n",
              "   'Luo, Jiankai'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Molecular brain',\n",
              "  'keywords': ['Animals',\n",
              "   'Corpus Callosum',\n",
              "   'Gene Ontology',\n",
              "   'Mice, Inbred BALB C',\n",
              "   'Mice, Mutant Strains',\n",
              "   'Myelin Sheath',\n",
              "   'Nerve Tissue Proteins',\n",
              "   'Niemann-Pick Disease, Type C',\n",
              "   'Proteomics'],\n",
              "  'pmid': '30866987',\n",
              "  'title': 'Proteomics of the corpus callosum to identify novel factors involved in hypomyelinated Niemann-Pick Type C disease mice.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Difficulty regulating positive mood and energy is a feature that cuts across different pediatric psychiatric disorders. Yet, little is known regarding the neural mechanisms underlying different developmental trajectories of positive mood and energy regulation in youth. Recent studies indicate that machine learning techniques can help elucidate the role of neuroimaging measures in classifying individual subjects by specific symptom trajectory. Cortical thickness measures were extracted in sixty-eight anatomical regions covering the entire brain in 115 participants from the Longitudinal Assessment of Manic Symptoms (LAMS) study and 31 healthy comparison youth (12.5 y/o;-Male/Female = 15/16;-IQ = 104;-Right/Left handedness = 24/5). Using a combination of trajectories analyses, surface reconstruction, and machine learning techniques, the present study aims to identify the extent to which measures of cortical thickness can accurately distinguish youth with higher (n = 18) from those with lower (n = 34) trajectories of manic-like behaviors in a large sample of LAMS youth (n = 115; 13.6 y/o; M/F = 68/47, IQ = 100.1, R/L = 108/7). Machine learning analyses revealed that widespread cortical thickening in portions of the left dorsolateral prefrontal cortex, right inferior and middle temporal gyrus, bilateral precuneus, and bilateral paracentral gyri and cortical thinning in portions of the right dorsolateral prefrontal cortex, left ventrolateral prefrontal cortex, and right parahippocampal gyrus accurately differentiate (Area Under Curve = 0.89;p = 0.03) youth with different (higher vs lower) trajectories of positive mood and energy dysregulation over a period up to 5years, as measured by the Parent General Behavior Inventory-10 Item Mania Scale. Our findings suggest that specific patterns of cortical thickness may reflect transdiagnostic neural mechanisms associated with different temporal trajectories of positive mood and energy dysregulation in youth. This approach has potential to identify patterns of neural markers of future clinical course.',\n",
              "  'authors': ['Versace, Amelia',\n",
              "   'Sharma, Vinod',\n",
              "   'Bertocci, Michele A',\n",
              "   'Bebko, Genna',\n",
              "   'Iyengar, Satish',\n",
              "   'Dwojak, Amanda',\n",
              "   'Bonar, Lisa',\n",
              "   'Perlman, Susan B',\n",
              "   'Schirda, Claudiu',\n",
              "   'Travis, Michael',\n",
              "   'Gill, Mary Kay',\n",
              "   'Diwadkar, Vaibhav A',\n",
              "   'Sunshine, Jeffrey L',\n",
              "   'Holland, Scott K',\n",
              "   'Kowatch, Robert A',\n",
              "   'Birmaher, Boris',\n",
              "   'Axelson, David',\n",
              "   'Frazier, Thomas W',\n",
              "   'Arnold, L Eugene',\n",
              "   'Fristad, Mary A',\n",
              "   'Youngstrom, Eric A',\n",
              "   'Horwitz, Sarah M',\n",
              "   'Findling, Robert L',\n",
              "   'Phillips, Mary L'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Affect',\n",
              "   'Bipolar Disorder',\n",
              "   'Case-Control Studies',\n",
              "   'Cerebral Cortex',\n",
              "   'Child',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Longitudinal Studies',\n",
              "   'Machine Learning',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Neuroimaging',\n",
              "   'Parahippocampal Gyrus',\n",
              "   'Parietal Lobe',\n",
              "   'Prefrontal Cortex',\n",
              "   'Temporal Lobe'],\n",
              "  'pmid': '28683115',\n",
              "  'title': 'Using machine learning and surface reconstruction to accurately differentiate different trajectories of mood and energy dysregulation in youth.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1546-1726',\n",
              "  'IF_2013': '14.976',\n",
              "  'IF_2014': '16.095',\n",
              "  'IF_2015': '16.724',\n",
              "  'IF_2016': '17.839',\n",
              "  'IF_2017': '19.912',\n",
              "  'ISSN': '1097-6256',\n",
              "  'ISSN_online': '1546-1726',\n",
              "  'ISSN_print': '1097-6256',\n",
              "  'IsoAbbr': 'Nat. Neurosci.',\n",
              "  'JrId': 21092,\n",
              "  'MedAbbr': 'Nat Neurosci',\n",
              "  'NLMid': '9809671',\n",
              "  'Titre': 'Nat. Neurosci.',\n",
              "  'abstract': 'Speech production, like other sensorimotor behaviors, relies on multiple sensory inputs--audition, proprioceptive inputs from muscle spindles and cutaneous inputs from mechanoreceptors in the skin and soft tissues of the vocal tract. However, the capacity for intelligible speech by deaf speakers suggests that somatosensory input alone may contribute to speech motor control and perhaps even to speech learning. We assessed speech motor learning in cochlear implant recipients who were tested with their implants turned off. A robotic device was used to alter somatosensory feedback by displacing the jaw during speech. We found that implant subjects progressively adapted to the mechanical perturbation with training. Moreover, the corrections that we observed were for movement deviations that were exceedingly small, on the order of millimeters, indicating that speakers have precise somatosensory expectations. Speech motor learning is substantially dependent on somatosensory input.',\n",
              "  'authors': ['Nasir, Sazzad M', 'Ostry, David J'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Nature neuroscience',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Aged',\n",
              "   'Auditory Threshold',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Case-Control Studies',\n",
              "   'Cochlear Implants',\n",
              "   'Deafness',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Jaw',\n",
              "   'Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Movement',\n",
              "   'Robotics',\n",
              "   'Speech',\n",
              "   'Speech Acoustics'],\n",
              "  'pmid': '18794839',\n",
              "  'title': 'Speech motor learning in profoundly deaf adults.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Deficits in the visual processing of faces in autism spectrum disorder (ASD) individuals may be due to atypical brain organization and function. Studies assessing asymmetric brain function in ASD individuals have suggested that facial processing, which is known to be lateralized in neurotypical (NT) individuals, may be less lateralized in ASD. Here we used functional near-infrared spectroscopy (fNIRS) to first test this theory by comparing patterns of lateralized brain activity in homologous temporal-occipital facial processing regions during observation of faces in an ASD group and an NT group. As expected, the ASD participants showed reduced right hemisphere asymmetry for human faces, compared to the NT participants. Based on recent behavioral reports suggesting that robots can facilitate increased verbal interaction over human counterparts in ASD, we also measured responses to faces of robots to determine if these patterns of activation were lateralized in each group. In this exploratory test, both groups showed similar asymmetry patterns for the robot faces. Our findings confirm existing literature suggesting reduced asymmetry for human faces in ASD and provide a preliminary foundation for future testing of how the use of categorically different social stimuli in the clinical setting may be beneficial in this population. ',\n",
              "  'authors': ['Jung, Corinne E',\n",
              "   'Strother, Lars',\n",
              "   'Feil-Seifer, David J',\n",
              "   'Hutsler, Jeffrey J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Autism Spectrum Disorder',\n",
              "   'Behavior',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Child',\n",
              "   'Emotions',\n",
              "   'Facial Recognition',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Intelligence Tests',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Visual',\n",
              "   'Robotics',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Temporal Lobe',\n",
              "   'Young Adult'],\n",
              "  'pmid': '27389017',\n",
              "  'title': 'Atypical Asymmetry for Processing Human and Robot Faces in Autism Revealed by fNIRS.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1471-2970',\n",
              "  'IF_2013': '6.314',\n",
              "  'IF_2014': '7.055',\n",
              "  'IF_2015': '5.847',\n",
              "  'IF_2016': '5.846',\n",
              "  'IF_2017': '5.666',\n",
              "  'ISSN': '0962-8436',\n",
              "  'ISSN_online': '1471-2970',\n",
              "  'ISSN_print': '0962-8436',\n",
              "  'IsoAbbr': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'JrId': 8277,\n",
              "  'MedAbbr': 'Philos Trans R Soc Lond B Biol Sci',\n",
              "  'NLMid': '7503623',\n",
              "  'Titre': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'abstract': 'An information-processing model is outlined that predicts that performance on non-routine tasks can be impaired independently of performance on routine tasks. The model is related to views on frontal lobe functions, particularly those of Luria. Two methods of obtaining more rigorous tests of the model are discussed. One makes use of ideas from artificial intelligence to derive a task heavily loaded on planning abilities. A group of patients with left anterior lesions has a specific deficit on the task. Subsidiary investigations support the inference that this is a planning impairment.',\n",
              "  'authors': ['Shallice, T'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Philosophical transactions of the Royal Society of London. Series B, Biological sciences',\n",
              "  'keywords': ['Attention',\n",
              "   'Cognition Disorders',\n",
              "   'Discrimination Learning',\n",
              "   'Frontal Lobe',\n",
              "   'Humans',\n",
              "   'Mental Recall',\n",
              "   'Models, Neurological',\n",
              "   'Problem Solving',\n",
              "   'Psychological Tests',\n",
              "   'Visual Perception'],\n",
              "  'pmid': '6125971',\n",
              "  'title': 'Specific impairments of planning.',\n",
              "  'year': '1982'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1557-8259',\n",
              "  'IF_2013': '1.341',\n",
              "  'IF_2014': '1.49',\n",
              "  'IF_2015': '1.264',\n",
              "  'IF_2016': '1.264',\n",
              "  'IF_2017': '1.514',\n",
              "  'ISSN': '0030-6665',\n",
              "  'ISSN_online': '1557-8259',\n",
              "  'ISSN_print': '0030-6665',\n",
              "  'IsoAbbr': 'Otolaryngol. Clin. North Am.',\n",
              "  'JrId': 6320,\n",
              "  'MedAbbr': 'Otolaryngol Clin North Am',\n",
              "  'NLMid': '0144042',\n",
              "  'Titre': 'Otolaryngol. Clin. North Am.',\n",
              "  'abstract': 'In the last 20 years, there has been a significant increase in the diagnosis of benign and malignant thyroid tumors. With improved ultrasound technology and better access to sonographic imaging, many tumors are identified at earlier stages. Consequently, there has been an evolution in surgical technique, moving toward minimally invasive approaches. This article describes the technique of robotic thyroidectomy via transaxillary endoscopic approach without CO(2) insufflation.',\n",
              "  'authors': ['Holsinger, F Christopher',\n",
              "   'Terris, David J',\n",
              "   'Kuppersmith, Ronald B'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'Otolaryngologic clinics of North America',\n",
              "  'keywords': ['Axilla',\n",
              "   'Curriculum',\n",
              "   'Education, Medical, Continuing',\n",
              "   'Endoscopes',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Mentors',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Operating Rooms',\n",
              "   'Preoperative Care',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Thyroid Neoplasms',\n",
              "   'Thyroidectomy'],\n",
              "  'pmid': '20510721',\n",
              "  'title': 'Robotic thyroidectomy: operative technique using a transaxillary endoscopic approach without CO2 insufflation.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'This work presents an adaptive framework for building soft-sensors based on radial basis function (RBF) neural network models. The adaptive fuzzy means algorithm is utilized in order to evolve an RBF network, which approximates the unknown system based on input-output data from it. The methodology gradually builds the RBF network model, based on two separate levels of adaptation: On the first level, the structure of the hidden layer is modified by adding or deleting RBF centers, while on the second level, the synaptic weights are adjusted with the recursive least squares with exponential forgetting algorithm. The proposed approach is tested on two different systems, namely a simulated nonlinear DC Motor and a real industrial reactor. The results show that the produced soft-sensors can be successfully applied to model the two nonlinear systems. A comparison with two different adaptive modeling techniques, namely a dynamic evolving neural-fuzzy inference system (DENFIS) and neural networks trained with online backpropagation, highlights the advantages of the proposed methodology. ',\n",
              "  'authors': ['Alexandridis, Alex'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms', 'Fuzzy Logic', 'Neural Networks (Computer)'],\n",
              "  'pmid': '24156672',\n",
              "  'title': 'Evolving RBF neural networks for adaptive soft-sensor design.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Earthquake prediction has been a challenging research area, where a future occurrence of the devastating catastrophe is predicted. In this work, sixty seismic features are computed through employing seismological concepts, such as Gutenberg-Richter law, seismic rate changes, foreshock frequency, seismic energy release, total recurrence time. Further, Maximum Relevance and Minimum Redundancy (mRMR) criteria is applied to extract the relevant features. A Support Vector Regressor (SVR) and Hybrid Neural Network (HNN) based classification system is built to obtain the earthquake predictions. HNN is a step wise combination of three different Neural Networks, supported by Enhanced Particle Swarm Optimization (EPSO), to offer weight optimization at each layer. The newly computed seismic features in combination with SVR-HNN prediction system is applied on Hindukush, Chile and Southern California regions. The obtained numerical results show improved prediction performance for all the considered regions, compared to previous prediction studies.',\n",
              "  'authors': ['Asim, Khawaja M',\n",
              "   'Idris, Adnan',\n",
              "   'Iqbal, Talat',\n",
              "   'Martínez-Álvarez, Francisco'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'California',\n",
              "   'Chile',\n",
              "   'Computer Simulation',\n",
              "   'Earthquakes',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29975687',\n",
              "  'title': 'Earthquake prediction model using support vector regressor and hybrid neural networks.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1533-3450',\n",
              "  'IF_2013': '9.466',\n",
              "  'IF_2014': '9.343',\n",
              "  'IF_2015': '8.491',\n",
              "  'IF_2016': '8.966',\n",
              "  'IF_2017': '8.655',\n",
              "  'ISSN': '1046-6673',\n",
              "  'ISSN_online': '1533-3450',\n",
              "  'ISSN_print': '1046-6673',\n",
              "  'IsoAbbr': 'J. Am. Soc. Nephrol.',\n",
              "  'JrId': 1194,\n",
              "  'MedAbbr': 'J Am Soc Nephrol',\n",
              "  'NLMid': '9013836',\n",
              "  'Titre': 'J. Am. Soc. Nephrol.',\n",
              "  'abstract': 'The importance of the kidney distal convoluted tubule (DCT) and cortical collecting duct (CCD) is highlighted by various water and electrolyte disorders that arise when the unique transport properties of these segments are disturbed. Despite this critical role, little is known about which proteins have a regulatory role in these cells and how these cells can be regulated by individual physiologic stimuli. By combining proteomics, bioinformatics, and cell biology approaches, we found that the E3 ubiquitin ligase CHIP is highly expressed throughout the collecting duct; is modulated in abundance by vasopressin; interacts with aquaporin-2 (AQP2), Hsp70, and Hsc70; and can directly ubiquitylate the water channel AQP2 in vitro shRNA knockdown of CHIP in CCD cells increased AQP2 protein t1/2 and reduced AQP2 ubiquitylation, resulting in greater levels of AQP2 and phosphorylated AQP2. CHIP knockdown increased the plasma membrane abundance of AQP2 in these cells. Compared with wild-type controls, CHIP knockout mice or novel CRISPR/Cas9 mice without CHIP E3 ligase activity had greater AQP2 abundance and altered renal water handling, with decreased water intake and urine volume, alongside higher urine osmolality. We did not observe significant changes in other water- or sodium-transporting proteins in the gene-modified mice. In summary, these results suggest that CHIP regulates AQP2 and subsequently, renal water handling.',\n",
              "  'authors': ['Wu, Qi',\n",
              "   'Moeller, Hanne B',\n",
              "   'Stevens, Donté A',\n",
              "   'Sanchez-Hodge, Rebekah',\n",
              "   'Childers, Gabrielle',\n",
              "   'Kortenoeven, Marleen L A',\n",
              "   'Cheng, Lei',\n",
              "   'Rosenbaek, Lena L',\n",
              "   'Rubel, Carrie',\n",
              "   'Patterson, Cam',\n",
              "   'Pisitkun, Trairak',\n",
              "   'Schisler, Jonathan C',\n",
              "   'Fenton, Robert A'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of the American Society of Nephrology : JASN',\n",
              "  'keywords': ['Animals',\n",
              "   'Aquaporin 2',\n",
              "   'Cells, Cultured',\n",
              "   'Gene Ontology',\n",
              "   'Gene Silencing',\n",
              "   'HSP70 Heat-Shock Proteins',\n",
              "   'Homeostasis',\n",
              "   'Kidney Tubules, Collecting',\n",
              "   'Kidney Tubules, Distal',\n",
              "   'Mice',\n",
              "   'Proteomics',\n",
              "   'Ubiquitin-Protein Ligases',\n",
              "   'Ubiquitination',\n",
              "   'Water'],\n",
              "  'pmid': '29242247',\n",
              "  'title': 'CHIP Regulates Aquaporin-2 Quality Control and Body Water Homeostasis.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'RNA binding protein (RBP) plays an important role in cellular processes. Identifying RBPs by computation and experiment are both essential. Recently, an RBP predictor, RBPPred, is proposed in our group to predict RBPs. However, RBPPred is too slow for that it needs to generate PSSM matrix as its feature. Herein, based on the protein feature of RBPPred and Convolutional Neural Network (CNN), we develop a deep learning model called Deep-RBPPred. With the balance and imbalance training set, we obtain Deep-RBPPred-balance and Deep-RBPPred-imbalance models. Deep-RBPPred has three advantages comparing to previous methods. (1) Deep-RBPPred only needs few physicochemical properties based on protein sequences. (2) Deep-RBPPred runs much faster. (3) Deep-RBPPred has a good generalization ability. In the meantime, Deep-RBPPred is still as good as the state-of-the-art method. Testing in A. thaliana, S. cerevisiae and H. sapiens proteomes, MCC values are 0.82 (0.82), 0.65 (0.69) and 0.85 (0.80) for balance model (imbalance model) when the score cutoff is set to 0.5, respectively. In the same testing dataset, different machine learning algorithms (CNN and SVM) are also compared. The results show that CNN-based model can identify more RBPs than SVM-based. In comparing the balance and imbalance model, both CNN-base and SVM-based tend to favor the majority class in the imbalance set. Deep-RBPPred forecasts 280 (balance model) and 265 (imbalance model) of 299 new RBP. The sensitivity of balance model is about 7% higher than the state-of-the-art method. We also apply deep-RBPPred to 30 eukaryotes and 109 bacteria proteomes downloaded from Uniprot to estimate all possible RBPs. The estimating result shows that rates of RBPs in eukaryote proteomes are much higher than bacteria proteomes.',\n",
              "  'authors': ['Zheng, Jinfang',\n",
              "   'Zhang, Xiaoli',\n",
              "   'Zhao, Xunyi',\n",
              "   'Tong, Xiaoxue',\n",
              "   'Hong, Xu',\n",
              "   'Xie, Juan',\n",
              "   'Liu, Shiyong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30323214',\n",
              "  'title': 'Deep-RBPPred: Predicting RNA binding proteins in the proteome scale based on deep learning.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1527-1315',\n",
              "  'IF_2013': '6.214',\n",
              "  'IF_2014': '6.867',\n",
              "  'IF_2015': '6.798',\n",
              "  'IF_2016': '7.296',\n",
              "  'IF_2017': '7.469',\n",
              "  'ISSN': '0033-8419',\n",
              "  'ISSN_online': '1527-1315',\n",
              "  'ISSN_print': '0033-8419',\n",
              "  'IsoAbbr': 'Radiology',\n",
              "  'JrId': 6859,\n",
              "  'MedAbbr': 'Radiology',\n",
              "  'NLMid': '0401260',\n",
              "  'Titre': 'Radiology',\n",
              "  'abstract': 'To develop a generic support vector machine (SVM) model by using magnetic resonance (MR) imaging-based blood volume distribution data for preoperative glioma survival associations and to prospectively evaluate the diagnostic effectiveness of this model in autonomous patient data.|Institutional and regional medical ethics committees approved the study, and all patients signed a consent form. Two hundred thirty-five preoperative adult patients from two institutions with a subsequent histologically confirmed diagnosis of glioma after surgery were included retrospectively. An SVM learning technique was applied to MR imaging-based whole-tumor relative cerebral blood volume (rCBV) histograms. SVM models with the highest diagnostic accuracy for 6-month and 1-, 2-, and 3-year survival associations were trained on 101 patients from the first institution. With Cox survival analysis, the diagnostic effectiveness of the SVM models was tested on independent data from 134 patients at the second institution.|were adjusted for known survival predictors, including patient age, tumor size, neurologic status, and postsurgery treatment, and were compared with survival associations from an expert reader.|Compared with total qualitative assessment by an expert reader, the whole-tumor rCBV-based SVM model was the strongest parameter associated with 6-month and 1-, 2-, and 3-year survival in the independent patient data (area under the receiver operating characteristic curve, 0.794-0.851; hazard ratio, 5.4-21.2).|Machine learning by means of SVM in combination with whole-tumor rCBV histogram analysis can be used to identify early patient survival in aggressive gliomas. The SVM model returned higher diagnostic accuracy values than an expert reader, and the model appears to be insensitive to patient, observer, and institutional variations.',\n",
              "  'authors': ['Emblem, Kyrre E',\n",
              "   'Pinho, Marco C',\n",
              "   'Zöllner, Frank G',\n",
              "   'Due-Tonnessen, Paulina',\n",
              "   'Hald, John K',\n",
              "   'Schad, Lothar R',\n",
              "   'Meling, Torstein R',\n",
              "   'Rapalino, Otto',\n",
              "   'Bjornerud, Atle'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Radiology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Brain Neoplasms',\n",
              "   'Contrast Media',\n",
              "   'Female',\n",
              "   'Glioma',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Retrospective Studies',\n",
              "   'Support Vector Machine',\n",
              "   'Survival Analysis'],\n",
              "  'pmid': '25486589',\n",
              "  'title': 'A generic support vector machine model for preoperative glioma survival associations.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': \"To compare biochemical recurrence (BCR)-free survival and predictors of BCR in intermediate-risk (IR) and high-risk (HR) patients undergoing robotic-assisted laparoscopic prostatectomy (RALP) vs open radical prostatectomy (ORP).|We conducted a retrospective study on 1336 men with D'Amico IR or HR prostate cancer who underwent RALP or ORP between 2003 and 2009. Exclusion criteria were use of neoadjuvant therapy, <6 months of follow-up, and insufficient clinicopathologic data. We compared demographic, clinical, and pathologic variables between groups. Kaplan-Meier analysis was performed to compare the 5-year BCR-free survival between groups. Multivariate models were developed to determine whether surgical approach influences BCR.|A total of 979 IR and HR patients (237 ORP and 742 RALP patients) met inclusion criteria. Median follow-up was shorter for RALP (43 vs 63 months; P<.001). ORP patients had a higher median prostate-specific antigen level (7.9 vs 6.7 ng/mL; P<.002), significantly more Gleason sum 8-10 tumors, and more adverse pathologic features overall. There was no difference in positive surgical margins between groups. Pathologic features including extraprostatic extension, seminal vesicle involvement, lymph node involvement, pathologic Gleason sum, and positive surgical margin were significant independent predictors of BCR in multivariate analysis. Surgical approach (RALP vs ORP) did not predict BCR when controlling for other known predictors of BCR.|Among IR and HR prostate cancer patients, the oncologic outcomes are similar between RALP and ORP. Not surprisingly, adverse pathologic features are harbingers of BCR.\",\n",
              "  'authors': ['Ritch, Chad R',\n",
              "   'You, Chaochen',\n",
              "   'May, Alexandra T',\n",
              "   'Herrell, S Duke',\n",
              "   'Clark, Peter E',\n",
              "   'Penson, David F',\n",
              "   'Chang, Sam S',\n",
              "   'Cookson, Michael S',\n",
              "   'Smith, Joseph A',\n",
              "   'Barocas, Daniel A'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Cohort Studies',\n",
              "   'Disease-Free Survival',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Kaplan-Meier Estimate',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Multivariate Analysis',\n",
              "   'Neoplasm Invasiveness',\n",
              "   'Neoplasm Staging',\n",
              "   'Postoperative Complications',\n",
              "   'Predictive Value of Tests',\n",
              "   'Proportional Hazards Models',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Retrospective Studies',\n",
              "   'Risk Assessment',\n",
              "   'Robotics',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome',\n",
              "   'Urologic Surgical Procedures, Male'],\n",
              "  'pmid': '24746665',\n",
              "  'title': 'Biochemical recurrence-free survival after robotic-assisted laparoscopic vs open radical prostatectomy for intermediate- and high-risk prostate cancer.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.44',\n",
              "  'IF_2014': '0.408',\n",
              "  'IF_2015': '0.394',\n",
              "  'IF_2016': '0.494',\n",
              "  'IF_2017': '0.42',\n",
              "  'ISSN': '0001-5458',\n",
              "  'ISSN_print': '0001-5458',\n",
              "  'IsoAbbr': 'Acta Chir. Belg.',\n",
              "  'JrId': 43,\n",
              "  'MedAbbr': 'Acta Chir Belg',\n",
              "  'NLMid': '0370571',\n",
              "  'Titre': 'Acta Chir. Belg.',\n",
              "  'abstract': 'Beside conventional and endovascular techniques, laparoscopic surgery is becoming a third way to treat patients presenting with aortoiliac occlusive or aneurysmal diseases. Several different laparoscopic techniques have been described, but most authors are stressing on the need for development of a specific laparoscopic aortic instrumentation, in order to decrease the operative and clamping times and to reduce the learning curve. This article is presenting an overview of what is already available, as laparoscopic aortic clamps or laparoscopic intestinal retractor, and what is still experimental, like laparoscopic aortic staplers, anastomotic devices or robotic surgical systems. This important technologic challenge should lead to two major orientations: development of qualitative in vitro and in vivo experiments to test these new products and training courses to teach the manipulation of it. Minimally aggressive techniques are well adapted to a western population, which becomes older, and has access to constantly improving medical care; however, only a specific and ergonomic instrumentation will allow these new techniques to be widely embraced by the vascular surgical community.',\n",
              "  'authors': ['Alimi, Yves S'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Acta chirurgica Belgica',\n",
              "  'keywords': ['Anastomosis, Surgical',\n",
              "   'Aorta',\n",
              "   'Blood Vessel Prosthesis Implantation',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Robotics',\n",
              "   'Surgical Instruments',\n",
              "   'Suture Techniques',\n",
              "   'Vascular Surgical Procedures'],\n",
              "  'pmid': '15571015',\n",
              "  'title': 'Laparoscopic aortic surgery: recent development in instrumentation.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1573-2959',\n",
              "  'IF_2013': '1.679',\n",
              "  'IF_2014': '1.679',\n",
              "  'IF_2015': '1.633',\n",
              "  'IF_2016': '1.687',\n",
              "  'IF_2017': '1.804',\n",
              "  'ISSN': '0167-6369',\n",
              "  'ISSN_online': '1573-2959',\n",
              "  'ISSN_print': '0167-6369',\n",
              "  'IsoAbbr': 'Environ Monit Assess',\n",
              "  'JrId': 22015,\n",
              "  'MedAbbr': 'Environ Monit Assess',\n",
              "  'NLMid': '8508350',\n",
              "  'Titre': 'Environ Monit Assess',\n",
              "  'abstract': \"Heavy metals in the agricultural soils of reclaimed mining areas can contaminate food and endanger human health. The objective of this study is to effectively estimate the concentrations of heavy metals, such as zinc, chromium, arsenic, and lead, using hyperspectral sensor data and the random forest (RF) algorithm in the study area of Xuzhou, China. The RF's built-in feature selection ability and modeling expressive ability in heavy metal estimation of soil were explored. After the preprocessing of the spectrum obtained by an ASD (analytical spectral device) field spectrometer, the random forest algorithm was carried out to establish the estimation model based on the correlation-selected features and the full-spectrum features respectively. Results of all the different processes were compared with classical approaches, such as partial least squares (PLS) regression and support vector machine (SVM). In all the experimental results, from the perspective of models, the best estimation model for Zn (R2 = 0.9061; RMSE = 6.5008) is based on the full-spectrum data of continuum removal (CR) pretreatment, and the best models for Cr (R2 = 0.9110; RMSE = 4.5683), As (R2 = 0.9912; RMSE = 0.5327), and Pb (R2 = 0.9756; RMSE = 1.1694) are all derived from the correlation-selected features. And these best models of these heavy metals are all established by the RF method. The experiments in this paper show that random forests can make full use of the input spectral data in the estimation of four kinds of heavy metals, and the obtained models are superior to those established by traditional methods.\",\n",
              "  'authors': ['Tan, Kun', 'Ma, Weibo', 'Wu, Fuyu', 'Du, Qian'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental monitoring and assessment',\n",
              "  'keywords': ['Agriculture',\n",
              "   'Algorithms',\n",
              "   'Arsenic',\n",
              "   'China',\n",
              "   'Environmental Monitoring',\n",
              "   'Humans',\n",
              "   'Least-Squares Analysis',\n",
              "   'Metals, Heavy',\n",
              "   'Mining',\n",
              "   'Soil Pollutants',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '31214787',\n",
              "  'title': 'Random forest-based estimation of heavy metal concentration in agricultural soils with hyperspectral sensor data.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Traditional methods for monitoring influenza are haphazard and lack fine-grained details regarding the spatial and temporal dynamics of outbreaks. Twitter gives researchers and public health officials an opportunity to examine the spread of influenza in real-time and at multiple geographical scales. In this paper, we introduce an improved framework for monitoring influenza outbreaks using the social media platform Twitter. Relying upon techniques from geographic information science (GIS) and data mining, Twitter messages were collected, filtered, and analyzed for the thirty most populated cities in the United States during the 2013-2014 flu season. The results of this procedure are compared with national, regional, and local flu outbreak reports, revealing a statistically significant correlation between the two data sources. The main contribution of this paper is to introduce a comprehensive data mining process that enhances previous attempts to accurately identify tweets related to influenza. Additionally, geographical information systems allow us to target, filter, and normalize Twitter messages. ',\n",
              "  'authors': ['Allen, Chris',\n",
              "   'Tsou, Ming-Hsiang',\n",
              "   'Aslam, Anoshe',\n",
              "   'Nagel, Anna',\n",
              "   'Gawron, Jean-Mark'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Disease Outbreaks',\n",
              "   'Geographic Information Systems',\n",
              "   'Geography, Medical',\n",
              "   'Humans',\n",
              "   'Influenza, Human',\n",
              "   'Machine Learning',\n",
              "   'Public Health Surveillance',\n",
              "   'Social Media',\n",
              "   'United States'],\n",
              "  'pmid': '27455108',\n",
              "  'title': 'Applying GIS and Machine Learning Methods to Twitter Data for Multiscale Surveillance of Influenza.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.543',\n",
              "  'IF_2014': '0.549',\n",
              "  'IF_2015': '0.542',\n",
              "  'IF_2016': '0.51',\n",
              "  'IF_2017': '0.481',\n",
              "  'ISSN': '0036-9330',\n",
              "  'ISSN_print': '0036-9330',\n",
              "  'IsoAbbr': 'Scott Med J',\n",
              "  'JrId': 7474,\n",
              "  'MedAbbr': 'Scott Med J',\n",
              "  'NLMid': '2983335R',\n",
              "  'Titre': 'Scott Med J',\n",
              "  'abstract': \"Launched in 2012, Knowledge into Action is the national knowledge management strategy for the health and social care workforce in Scotland. It is transforming the role of the national digital knowledge service--NHS Education for Scotlands' Knowledge Network--and the NHSS librarian role to offer more active, tailored support for translating knowledge into frontline clinical practice. This includes the development of a national evidence search and summary service, help with converting knowledge into practical and usable formats for easy use at point of care and with using digital tools to share clinicians' learning, experience and expertise. Through this practical support, Knowledge into Action is contributing to quality and safety outcomes across NHS Scotland, building clinicians' capacity and capability in applying knowledge in frontline practice and service improvement.\",\n",
              "  'authors': ['Wales, Ann', 'Boyle, Derek'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Scottish medical journal',\n",
              "  'keywords': ['Evidence-Based Medicine',\n",
              "   'Humans',\n",
              "   'Information Dissemination',\n",
              "   'Information Services',\n",
              "   'Knowledge Bases',\n",
              "   'Quality Assurance, Health Care',\n",
              "   'Quality Improvement',\n",
              "   'Research Report',\n",
              "   'Scotland',\n",
              "   'Social Networking',\n",
              "   'Translational Medical Research'],\n",
              "  'pmid': '26449922',\n",
              "  'title': \"Scotland's knowledge network: a progress report on Knowledge into Action.\",\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['UI'],\n",
              "  'ESSN': '1079-7114',\n",
              "  'IF_2013': '7.728',\n",
              "  'IF_2014': '7.512',\n",
              "  'IF_2015': '7.645',\n",
              "  'IF_2016': '8.462',\n",
              "  'IF_2017': '8.839',\n",
              "  'ISSN': '0031-9007',\n",
              "  'ISSN_online': '1079-7114',\n",
              "  'ISSN_print': '0031-9007',\n",
              "  'IsoAbbr': 'Phys. Rev. Lett.',\n",
              "  'JrId': 21217,\n",
              "  'MedAbbr': 'Phys Rev Lett',\n",
              "  'NLMid': '0401141',\n",
              "  'Titre': 'Phys. Rev. Lett.',\n",
              "  'abstract': 'We study learning of probability distributions characterized by an unknown symmetry direction. Based on an entropic performance measure and the variational method of statistical mechanics we develop exact upper and lower bounds on the scaled critical number of examples below which learning of the direction is impossible. The asymptotic tightness of the bounds suggests an asymptotically optimal method for learning nonsmooth distributions.',\n",
              "  'authors': ['Herschkowitz, D', 'Opper, M'],\n",
              "  'categories': ['PHYSICS, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Physical review letters',\n",
              "  'keywords': ['Learning', 'Neural Networks (Computer)'],\n",
              "  'pmid': '11289883',\n",
              "  'title': 'Retarded learning: rigorous results from statistical mechanics.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['BA'],\n",
              "  'ESSN': '1573-2614',\n",
              "  'IF_2013': '1.448',\n",
              "  'IF_2014': '1.985',\n",
              "  'IF_2015': '1.819',\n",
              "  'IF_2016': '2.178',\n",
              "  'IF_2017': '2.45',\n",
              "  'ISSN': '1387-1307',\n",
              "  'ISSN_online': '1573-2614',\n",
              "  'ISSN_print': '1387-1307',\n",
              "  'IsoAbbr': 'J Clin Monit Comput',\n",
              "  'JrId': 21013,\n",
              "  'MedAbbr': 'J Clin Monit Comput',\n",
              "  'NLMid': '9806357',\n",
              "  'Titre': 'J Clin Monit Comput',\n",
              "  'abstract': 'The widespread implementation of computerized medical files in intensive care units (ICUs) over recent years has made available large databases of clinical data for the purpose of developing clinical prediction models. The typical intensive care unit has several information sources from which data is electronically collected as time series of varying time resolutions. We present an overview of research questions studied in the ICU setting that have been addressed through the automatic analysis of these large databases. We focus on automatic learning methods, specifically data mining approaches for predictive modeling based on these time series of clinical data. On the one hand we examine short and medium term predictions, which have as ultimate goal the development of early warning or decision support systems. On the other hand we examine long term outcome prediction models and evaluate their performance with respect to established scoring systems based on static admission and demographic data. ',\n",
              "  'authors': ['Güiza, Fabian', 'Van Eyck, Jelle', 'Meyfroidt, Geert'],\n",
              "  'categories': ['ANESTHESIOLOGY'],\n",
              "  'journal': 'Journal of clinical monitoring and computing',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Bayes Theorem',\n",
              "   'Critical Care',\n",
              "   'Data Mining',\n",
              "   'Decision Support Systems, Clinical',\n",
              "   'Humans',\n",
              "   'Intensive Care Units',\n",
              "   'Medical Records Systems, Computerized',\n",
              "   'Monitoring, Physiologic',\n",
              "   'Patient Admission',\n",
              "   'Prognosis',\n",
              "   'Risk',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '23179020',\n",
              "  'title': 'Predictive data mining on monitoring data from the intensive care unit.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'This paper proposes a new morphology-based approach for the interslice interpolation of current transformer (CT) and MRI datasets composed of parallel slices. Our approach is object based and accepts as input data binary slices belonging to the same anatomical structure. Such slices may contain one or more regions, since topological changes between two adjacent slices may occur. Our approach handles explicitly interslice topology changes by decomposing a many-to-many correspondence into three fundamental cases: one-to-one, one-to-many, and zero-to-one correspondences. The proposed interpolation process is iterative. One iteration of this process computes a transition sequence between a pair of corresponding input slices, and selects the element located at equal distance from the input slices. This algorithmic design yields a gradual, smooth change of shape between the input slices. Therefore, the main contribution of our approach is its ability to interpolate between two anatomic shapes by creating a smooth, gradual change of shape, and without generating over-smoothed interpolated shapes.',\n",
              "  'authors': ['Albu, Alexandra Branzan',\n",
              "   'Beugeling, Trevor',\n",
              "   'Laurendeau, Denis'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Anatomy, Cross-Sectional',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Models, Anatomic',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Subtraction Technique'],\n",
              "  'pmid': '18632365',\n",
              "  'title': 'A morphology-based approach for interslice interpolation of anatomical slices from volumetric images.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['AY'],\n",
              "  'ESSN': '1469-7580',\n",
              "  'IF_2013': '2.227',\n",
              "  'IF_2014': '2.097',\n",
              "  'IF_2015': '2.154',\n",
              "  'IF_2016': '2.182',\n",
              "  'IF_2017': '2.479',\n",
              "  'ISSN': '0021-8782',\n",
              "  'ISSN_online': '1469-7580',\n",
              "  'ISSN_print': '0021-8782',\n",
              "  'IsoAbbr': 'J. Anat.',\n",
              "  'JrId': 4486,\n",
              "  'MedAbbr': 'J Anat',\n",
              "  'NLMid': '0137162',\n",
              "  'Titre': 'J. Anat.',\n",
              "  'abstract': 'Living birds constitute the only vertebrate group whose brain volume relative to body size approaches the uniquely expanded values expressed by mammals. The broad suite of complex behaviors exhibited by crown-group birds, including sociality, vocal learning, parental care, and flying, suggests the origins of their encephalization was likely driven by a mosaic of selective pressures. If true, the historical pattern of brain expansion may be more complex than either a gradual expansion, as proposed by early studies of the avian brain, or a sudden expansion correlating with the appearance of flight. The origins of modern avian neuroanatomy are obscured by the more than 100 million years of evolution along their phylogenetic stem (from the origin of the modern radiation in the Middle Jurassic to the split from crocodile-line archosaurs). Here we use phylogenetic comparative approaches to explore which evolutionary scenarios best explain variation in measured volumes of digitally partitioned endocasts of modern birds and their non-avian ancestors. Our analyses suggest that variation in the relative volumes of the endocranium and cerebrum explain most of the structural variation in this lineage. Generalized multi-regime Ornstein-Uhlenbeck (OU) models suggest that powered flight does not appear to be a driver of observed variation, reinforcing the hypothesis that the deep history of the avian brain is complex, with nuances still to be discovered.',\n",
              "  'authors': ['Balanoff, Amy M', 'Smaers, Jeroen B', 'Turner, Alan H'],\n",
              "  'categories': ['ANATOMY & MORPHOLOGY'],\n",
              "  'journal': 'Journal of anatomy',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Birds',\n",
              "   'Brain',\n",
              "   'Dinosaurs',\n",
              "   'Fossils',\n",
              "   'Mammals',\n",
              "   'Neuroanatomy',\n",
              "   'Phylogeny',\n",
              "   'Skull'],\n",
              "  'pmid': '26538376',\n",
              "  'title': 'Brain modularity across the theropod-bird transition: testing the influence of flight on neuroanatomical variation.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1365-2648',\n",
              "  'IF_2013': '1.685',\n",
              "  'IF_2014': '1.741',\n",
              "  'IF_2015': '1.917',\n",
              "  'IF_2016': '1.998',\n",
              "  'IF_2017': '2.267',\n",
              "  'ISSN': '0309-2402',\n",
              "  'ISSN_online': '1365-2648',\n",
              "  'ISSN_print': '0309-2402',\n",
              "  'IsoAbbr': 'J Adv Nurs',\n",
              "  'JrId': 4420,\n",
              "  'MedAbbr': 'J Adv Nurs',\n",
              "  'NLMid': '7609811',\n",
              "  'Titre': 'J Adv Nurs',\n",
              "  'abstract': \"This paper reports on a study investigating the relationship of nursing students' approaches to learning and processing of information, science content knowledge, ability in interpreting and organizing clinical data (nursing assessment), and logical reasoning ability with the accuracy and quality of the nursing diagnosis made in a simulated diagnostic reasoning task. One hundred and sixty-nine second-year pre-service nursing students participated in the study. Results of path analyses indicated a set of pathways from surface approach to low-quality nursing diagnosis that reflected less competent diagnostic reasoning; and a second set from deep/achieving approach through content knowledge and logical reasoning to higher-quality nursing diagnosis that reflected more competent diagnostic reasoning. The implications of these findings for nurse education are discussed.\",\n",
              "  'authors': ['Cholowski, K M', 'Chan, L K'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Journal of advanced nursing',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Decision Support Techniques',\n",
              "   'Education, Nursing, Diploma Programs',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Logic',\n",
              "   'Male',\n",
              "   'Mental Processes',\n",
              "   'Middle Aged',\n",
              "   'Models, Nursing',\n",
              "   'New South Wales',\n",
              "   'Nursing Assessment',\n",
              "   'Nursing Diagnosis',\n",
              "   'Nursing Education Research',\n",
              "   'Problem Solving',\n",
              "   'Students, Nursing',\n",
              "   'Surveys and Questionnaires'],\n",
              "  'pmid': '1430619',\n",
              "  'title': 'Diagnostic reasoning among second-year nursing students.',\n",
              "  'year': '1992'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '2053-2644',\n",
              "  'IF_2013': '0.733',\n",
              "  'IF_2014': '0.752',\n",
              "  'IF_2015': '0.715',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0966-8519',\n",
              "  'ISSN_online': '2053-2644',\n",
              "  'ISSN_print': '0966-8519',\n",
              "  'IsoAbbr': 'J. Heart Valve Dis.',\n",
              "  'JrId': 2609,\n",
              "  'MedAbbr': 'J Heart Valve Dis',\n",
              "  'NLMid': '9312096',\n",
              "  'Titre': 'J. Heart Valve Dis.',\n",
              "  'abstract': 'Ring annuloplasty is a technically demanding and time-consuming thoracoscopic and/or robotic surgery. The initial experience of the authors with the intra-annular Bioring tricuspid ring using a minimally invasive access is reviewed, and some modifications are proposed to adapt to the particularities of these approaches.|Patients undergoing minimally invasive tricuspid annuloplasty were included prospectively between March and September 2008. The feasibility and ease of implantation were evaluated. A total of 10 patients (six females, four males) was included (eight with functional regurgitation, two with endocarditis). Six patients underwent surgery through a small anterolateral thoracotomy, and the da Vinci S robotic system was used in four cases.|The mean cardiopulmonary bypass and aortic cross-clamp times were 123 +/- 30 min and 86 +/- 28 min, respectively. Ring implantation was successful in all patients. There was one late death from multiple organ failure. None of the patients required reoperation. At discharge from hospital, seven patients had no or discrete tricuspid regurgitation (TR), and two had moderate TR with no tricuspid stenosis but remained stable during the follow up period.|The biodegradable Bioring offers a simple and quick implantation which is feasible and simplified in minimally invasive approaches.',\n",
              "  'authors': ['Panos, Aristotelis',\n",
              "   'Myers, Patrick O',\n",
              "   'Kalangos, Afksendiyos'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'The Journal of heart valve disease',\n",
              "  'keywords': ['Absorbable Implants',\n",
              "   'Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Endocarditis',\n",
              "   'Female',\n",
              "   'Heart Valve Prosthesis',\n",
              "   'Heart Valve Prosthesis Implantation',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Thoracoscopy',\n",
              "   'Tricuspid Valve',\n",
              "   'Tricuspid Valve Insufficiency',\n",
              "   'Young Adult'],\n",
              "  'pmid': '20369504',\n",
              "  'title': 'Thoracoscopic and robotic tricuspid valve annuloplasty with a biodegradable ring: an initial experience.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '1873-3557',\n",
              "  'IF_2013': '2.129',\n",
              "  'IF_2014': '2.353',\n",
              "  'IF_2015': '2.653',\n",
              "  'IF_2016': '2.536',\n",
              "  'IF_2017': '2.88',\n",
              "  'ISSN': '1386-1425',\n",
              "  'ISSN_online': '1873-3557',\n",
              "  'ISSN_print': '1386-1425',\n",
              "  'IsoAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'JrId': 20555,\n",
              "  'MedAbbr': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'NLMid': '9602533',\n",
              "  'Titre': 'Spectrochim Acta A Mol Biomol Spectrosc',\n",
              "  'abstract': 'Near-infrared (NIR) spectroscopy was used in simultaneous, non-destructive analysis of antipyriine and caffeine citrate tablets. Principal component artificial neural networks (PC-ANNs) were used to construct models for the analytes, using the testing set for external validation. Four pretreated spectra, namely, first-derivative, second-derivative, standard normal variate (SNV) and multiplicative scatter correction (MSC) spectra led to simplified and more robust models than conventional spectra. In PC-ANNs models, the spectra data were analyzed by principal component analysis (PCA) firstly. Then the scores of the principal compounds (PCs) were chosen as input nodes for input layer instead of the spectra data. The artificial neural networks (ANNs) models using the spectra data as input nodes were also established, which were compared with the PC-ANNs models. The result shows the SNV model of PC-ANNs multivariate calibration has the lowest training error and predicting error. The concept of the degree of approximation was introduced and performed as the selective criterion of the optimum network parameters.',\n",
              "  'authors': ['Dou, Ying',\n",
              "   'Zou, Tingting',\n",
              "   'Liu, Tong',\n",
              "   'Qu, Nan',\n",
              "   'Ren, Yulin'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Spectrochimica acta. Part A, Molecular and biomolecular spectroscopy',\n",
              "  'keywords': ['Antipyrine',\n",
              "   'Caffeine',\n",
              "   'Calibration',\n",
              "   'Citrates',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Principal Component Analysis',\n",
              "   'Reference Standards',\n",
              "   'Regression Analysis',\n",
              "   'Spectroscopy, Near-Infrared'],\n",
              "  'pmid': '17336581',\n",
              "  'title': 'Calibration in non-linear NIR spectroscopy using principal component artificial neural networks.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1943-619X',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0278-940X',\n",
              "  'ISSN_online': '1943-619X',\n",
              "  'ISSN_print': '0278-940X',\n",
              "  'IsoAbbr': 'Crit Rev Biomed Eng',\n",
              "  'JrId': 3193,\n",
              "  'MedAbbr': 'Crit Rev Biomed Eng',\n",
              "  'NLMid': '8208627',\n",
              "  'Titre': 'Crit Rev Biomed Eng',\n",
              "  'abstract': 'The patient with stable disabling angina must choose between bypass surgery, coronary angioplasty, or medical therapy. Estimation of comparative outcomes of these alternative therapies is difficult.|Utilizing Artificial Intelligence, an expert-system computerized decision aid, DATA (Decision Aid for Therapy of Angina), was developed to run on a personal computer and to calculate the probability of each possible clinical event based on individual patient characteristics. For each therapeutic option, relative clinical outcomes and anticipated charges are computed. Ten patients were evaluated retrospectively by DATA and by a group of cardiologists.|DATA agreed with the primary therapy given to all patients. The physician group underestimated the value of alternative therapies and underestimated charges for all therapies.|This decision aid allows comparison of alternative therapies in terms of relative patient outcome and anticipated costs can be better estimated.',\n",
              "  'authors': ['Hopkins, C B', 'Maysuria, H'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Critical reviews in biomedical engineering',\n",
              "  'keywords': ['Angina Pectoris',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Artery Bypass',\n",
              "   'Decision Support Techniques',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '10999363',\n",
              "  'title': 'DATA: a decision aid for management of the patient with stable angina pectoris.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Effective public health research and preparedness requires an accurate understanding of which virus species possess or are at risk of developing human transmissibility. Unfortunately, our ability to identify these viruses is limited by gaps in disease surveillance and an incomplete understanding of the process of viral adaptation. By fitting boosted regression trees to data on 224 human viruses and their associated traits, we developed a model that predicts the human transmission ability of zoonotic viruses with over 84% accuracy. This model identifies several viruses that may have an undocumented capacity for transmission between humans. Viral traits that predicted human transmissibility included infection of nonhuman primates, the absence of a lipid envelope, and detection in the human nervous system and respiratory tract. This predictive model can be used to prioritize high-risk viruses for future research and surveillance, and could inform an integrated early warning system for emerging infectious diseases.',\n",
              "  'authors': ['Walker, Joseph W',\n",
              "   'Han, Barbara A',\n",
              "   'Ott, Isabel M',\n",
              "   'Drake, John M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Communicable Diseases, Emerging',\n",
              "   'Epidemiological Monitoring',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Models, Biological',\n",
              "   'Public Health',\n",
              "   'Virus Diseases',\n",
              "   'Viruses',\n",
              "   'Zoonoses'],\n",
              "  'pmid': '30403733',\n",
              "  'title': 'Transmissibility of emerging viral zoonoses.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In this paper, we present a two-layer self-organizing neural network based method for handwritten digit recognition. The network consists of a base layer self-organizing map and a set of corresponding maps in the second layer. The input patterns are partitioned into subspace in the first layer. Patterns in a subspace are led to the second layer and a corresponding map is built according to the first layer performance. In the classification process, each pattern searches for several closest nodes from the base map and then it is classified into a specified class by determining the nearest model of the corresponding maps in the second layer. The new method yielded higher accuracy and faster performance than the ordinary self-organizing neural network.',\n",
              "  'authors': ['Wu, J', 'Yan, H', 'Chalmers, A'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Handwriting',\n",
              "   'Maps as Topic',\n",
              "   'Mathematics',\n",
              "   'Models, Neurological',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '7711966',\n",
              "  'title': 'Handwritten digit recognition using two-layer self-organizing maps.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1464-410X',\n",
              "  'IF_2013': '3.13',\n",
              "  'IF_2014': '3.533',\n",
              "  'IF_2015': '4.387',\n",
              "  'IF_2016': '4.439',\n",
              "  'IF_2017': '4.688',\n",
              "  'ISSN': '1464-4096',\n",
              "  'ISSN_online': '1464-410X',\n",
              "  'ISSN_print': '1464-4096',\n",
              "  'IsoAbbr': 'BJU Int.',\n",
              "  'JrId': 21363,\n",
              "  'MedAbbr': 'BJU Int',\n",
              "  'NLMid': '100886721',\n",
              "  'Titre': 'BJU Int.',\n",
              "  'abstract': \"What's known on the subject? and What does the study add? Robot assisted laparoscopic surgery (RALS) is slowly gaining acceptance in the field of paediatric urology. Accumulating data on safety and efficacy when performing paediatric robotic urologic procedures has led paediatric urologists to gradually embrace increasingly more complex reconstructive surgeries. Indeed, the unique and delicate movements generated by the robotic system make this technology ideal for children who often require reconstructive procedures. We critically review the current role of RALS in paediatric urology and to analyse the published data, with a special emphasis on the most common applications. We also propose a structured plan to expedite training and the surgical 'learning curve'.|To critically review the current role of robot-assisted laparoscopic surgery (RALS) in paediatric urology and to analyse the published data, with a special emphasis on the most common applications. One of the greatest benefits of RALS has been the ability to truly spread the application of minimally invasive surgery to paediatric surgical patients. The unique attributes of the robotic interface make this technology ideal for children with congenital anomalies, who often require reconstructive procedures. We also propose a structured plan to expedite training and the surgical 'learning curve'.|Currently, almost all urological surgical procedures in children have been performed with the assistance of the robotic interface. The most commonly performed procedures include pyeloplasty, nephrectomy/hemi-nephrectomy and surgery for vesico-ureteric reflux. Initial series of bladder augmentation and appendicovesicostomy are available.|Initial results with RALS are encouraging and have shown safety similar to open procedures, and outcomes at least equivalent to standard laparoscopy. Accumulating data have consistently shown that postoperative analgesia requirements and overall hospital stay are decreased. However, operative durations are significantly longer than their open counterparts, but this is decreasing as experience accumulates.|RALS is already part of paediatric urological surgery. Larger single-institution case series and comparative studies with the open approach and multi-institutional meta-analyses will help to identify the benefits of RALS in paediatric urology.\",\n",
              "  'authors': ['Orvieto, Marcelo A', 'Large, Michael', 'Gundeti, Mohan S'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BJU international',\n",
              "  'keywords': ['Animals',\n",
              "   'Child',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Reconstructive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Urologic Surgical Procedures',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '22429799',\n",
              "  'title': 'Robotic paediatric urology.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1464-410X',\n",
              "  'IF_2013': '3.13',\n",
              "  'IF_2014': '3.533',\n",
              "  'IF_2015': '4.387',\n",
              "  'IF_2016': '4.439',\n",
              "  'IF_2017': '4.688',\n",
              "  'ISSN': '1464-4096',\n",
              "  'ISSN_online': '1464-410X',\n",
              "  'ISSN_print': '1464-4096',\n",
              "  'IsoAbbr': 'BJU Int.',\n",
              "  'JrId': 21363,\n",
              "  'MedAbbr': 'BJU Int',\n",
              "  'NLMid': '100886721',\n",
              "  'Titre': 'BJU Int.',\n",
              "  'abstract': 'To identify differences between the ablative and extirpative minimally invasive techniques of laparoscopic cryoablation (LC) and robot-assisted laparoscopic partial nephrectomy (RPN), respectively, in treating small renal tumours in terms of safety, peri-operative morbidity and early oncological outcomes.|Between June 2008 and April 2012 56 patients underwent LC and from October 2010 to April 2012, 47 patients underwent RPN using the Da Vinci robotic platform (Intuitive Surgical, Sunnyvale, CA, USA). Data on intra-operative, postoperative and oncological outcomes were collected prospectively, and were analysed and compared for both groups.|The median patient ages were 69 and 60 for the LC and RPN groups, respectively (P < 0.05). There was no significant difference in disease stage, but there was a significant difference in tumour size, with patients in the RPN group having larger tumours. The mean operating times were 146 and 159 min for the LC and RPN groups, respectively (P = 0.094) and the mean blood loss was 47 and 94 mL for the LC and the RPN groups, respectively (P = 0.251). The median length of hospital stay (1 day) was the same for both groups and the mean warm ischaemia time was 23 min in the RPN group. The marginal change in preoperative and 6-week postoperative renal function was recorded: the mean postoperative increase in serum creatinine was 5.4 mmol/L in the LC group and 9.2 mmol/L in the RPN group. Of the 47 patients in the RPN group, two (4.3%) were converted to laparoscopic radical nephrectomy because of difficulty in controlling bleeding during hilar dissection. Only two patients (3.6%) had recurrence in the LC group, both of whom were treated with re-cryoablation. A total of 5.4% of patients in the LC and 4.3% in the RPN group had Clavien grade I postoperative complications, one patient in the LC group had a Clavien grade II complication, while 1.8 and 4.3% of patients had Clavien IIIb in the LC and RPN groups, respectively.|Our data confirm that LC is a successful, minimally invasive and safe treatment option for the management of small renal tumours, but the apparently similar characteristics of RPN suggest that an increasing proportion of patients, whatever their age or medical comorbidities, may be reasonably offered a robot-assisted extirpative procedure with the likely benefit of lower risk of local recurrence and need for retreatment.',\n",
              "  'authors': ['Emara, Amr M',\n",
              "   'Kommu, Sashi S',\n",
              "   'Hindley, Richard G',\n",
              "   'Barber, Neil J'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BJU international',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Carcinoma, Renal Cell',\n",
              "   'Creatine',\n",
              "   'Cryosurgery',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Neoplasm Staging',\n",
              "   'Nephrectomy',\n",
              "   'Patient Selection',\n",
              "   'Postoperative Complications',\n",
              "   'Prospective Studies',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24053473',\n",
              "  'title': \"Robot-assisted partial nephrectomy vs laparoscopic cryoablation for the small renal mass: redefining the minimally invasive 'gold standard'.\",\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1029-2330',\n",
              "  'IF_2013': '2.723',\n",
              "  'IF_2014': '2.741',\n",
              "  'IF_2015': '2.821',\n",
              "  'IF_2016': '3.068',\n",
              "  'IF_2017': '3.408',\n",
              "  'ISSN': '1061-186X',\n",
              "  'ISSN_online': '1029-2330',\n",
              "  'ISSN_print': '1061-186X',\n",
              "  'IsoAbbr': 'J Drug Target',\n",
              "  'JrId': 8530,\n",
              "  'MedAbbr': 'J Drug Target',\n",
              "  'NLMid': '9312476',\n",
              "  'Titre': 'J Drug Target',\n",
              "  'abstract': 'Miniaturizing in chip technology, optics, micro mechanics, medicine, gene and biotechnology requires highly precise positioning techniques. The motivation for the new manipulation technology is the desire to enter the micro- and nanoworld not only by viewing but also acting, altering micro- and nanosized objects. A new era on medicine are expected to happen in the coming years. Due to the advances in the field of nanotechnology, nanodevice manufacturing has been growing gradually. From such achievements in nanotechnology and recent results in biotechnology and genetics, the first operating biological nanorobots are expected to appear in the coming 5 years and more complex diamondoid based nanorobots will become available in about 10 years. In terms of time, it means a very near better future with significant improvements in medicine.',\n",
              "  'authors': ['Patel, Geeta M',\n",
              "   'Patel, Gayatri C',\n",
              "   'Patel, Ritesh B',\n",
              "   'Patel, Jayvadan K',\n",
              "   'Patel, Madhabhai'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Journal of drug targeting',\n",
              "  'keywords': ['Biocompatible Materials',\n",
              "   'Communication',\n",
              "   'Humans',\n",
              "   'Immune System',\n",
              "   'Nanostructures',\n",
              "   'Nanotechnology',\n",
              "   'Robotics'],\n",
              "  'pmid': '16608733',\n",
              "  'title': 'Nanorobot: a versatile tool in nanomedicine.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': \"Interpretation of the electroencephalographic (EEG) and magnetoencephalographic (MEG) signals requires off-line artifacts removal. Since artifacts share frequencies with brain activity, filtering is insufficient. Blind source separation, mainly through independent component analysis (ICA), is the gold-standard procedure for the identification of artifacts in multi-dimensional recordings. However, a classification of brain and artifactual independent components (ICs) is still required. Since ICs exhibit recognizable patterns, classification is usually performed by experts' visual inspection. This procedure is time consuming and prone to errors. Automatic ICs classification has been explored, often through complex ICs features extraction prior to classification. Relying on deep-learning ability of self-extracting the features of interest, we investigated the capabilities of convolutional neural networks (CNNs) for off-line, automatic artifact identification through ICs without feature selection.|A CNN was applied to spectrum and topography of a large dataset of few thousand samples of ICs obtained from multi-channel EEG and MEG recordings acquired during heterogeneous experimental settings and on different subjects. CNN performances, when applied to EEG, MEG, and combined EEG and MEG ICs, were explored and compared with state-of-the-art feature-based automatic classification.|Beyond state-of-the-art automatic classification accuracies were demonstrated through cross validation (92.4% EEG, 95.4% MEG, 95.6% EEG+MEG).|High CNN classification performances were achieved through heuristical selection of machinery hyperparameters and through the CNN self-selection of the features of interest.|Considering the large data availability of multi-channel EEG and MEG recordings, CNNs may be suited for classification of ICs of multi-channel brain electrophysiological recordings.\",\n",
              "  'authors': ['Croce, Pierpaolo',\n",
              "   'Zappasodi, Filippo',\n",
              "   'Marzetti, Laura',\n",
              "   'Merla, Arcangelo',\n",
              "   'Pizzella, Vittorio',\n",
              "   'Chiarelli, Antonio Maria'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30582523',\n",
              "  'title': 'Deep Convolutional Neural Networks for Feature-Less Automatic Classification of Independent Components in Multi-Channel Electrophysiological Brain Recordings.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['NI'],\n",
              "  'ESSN': '1365-2567',\n",
              "  'IF_2013': '3.735',\n",
              "  'IF_2014': '3.795',\n",
              "  'IF_2015': '4.078',\n",
              "  'IF_2016': '3.701',\n",
              "  'IF_2017': '3.358',\n",
              "  'ISSN': '0019-2805',\n",
              "  'ISSN_online': '1365-2567',\n",
              "  'ISSN_print': '0019-2805',\n",
              "  'IsoAbbr': 'Immunology',\n",
              "  'JrId': 4178,\n",
              "  'MedAbbr': 'Immunology',\n",
              "  'NLMid': '0374672',\n",
              "  'Titre': 'Immunology',\n",
              "  'abstract': 'Compared with HLA-DR molecules, the specificities of HLA-DP and HLA-DQ molecules have only been studied to a limited extent. The description of the binding motifs has been mostly anecdotal and does not provide a quantitative measure of the importance of each position in the binding core and the relative weight of different amino acids at a given position. The recent publication of larger data sets of peptide-binding to DP and DQ molecules opens the possibility of using data-driven bioinformatics methods to accurately define the binding motifs of these molecules. Using the neural network-based method NNAlign, we characterized the binding specificities of five HLA-DP and six HLA-DQ among the most frequent in the human population. The identified binding motifs showed an overall concurrence with earlier studies but revealed subtle differences. The DP molecules revealed a large overlap in the pattern of amino acid preferences at core positions, with conserved hydrophobic/aromatic anchors at P1 and P6, and an additional hydrophobic anchor at P9 in some variants. These results confirm the existence of a previously hypothesized supertype encompassing the most common DP alleles. Conversely, the binding motifs for DQ molecules appear more divergent, displaying unconventional anchor positions and in some cases rather unspecific amino acid preferences.',\n",
              "  'authors': ['Andreatta, Massimo', 'Nielsen, Morten'],\n",
              "  'categories': ['IMMUNOLOGY'],\n",
              "  'journal': 'Immunology',\n",
              "  'keywords': ['Alleles',\n",
              "   'Amino Acid Motifs',\n",
              "   'Binding Sites',\n",
              "   'Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'HLA-DP Antigens',\n",
              "   'HLA-DQ Antigens',\n",
              "   'Humans',\n",
              "   'Hydrophobic and Hydrophilic Interactions',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sequence Alignment'],\n",
              "  'pmid': '22352343',\n",
              "  'title': 'Characterizing the binding motifs of 11 common human HLA-DP and HLA-DQ molecules using NNAlign.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'ANISEED (www.aniseed.cnrs.fr) is the main model organism database for tunicates, the sister-group of vertebrates. This release gives access to annotated genomes, gene expression patterns, and anatomical descriptions for nine ascidian species. It provides increased integration with external molecular and taxonomy databases, better support for epigenomics datasets, in particular RNA-seq, ChIP-seq and SELEX-seq, and features novel interactive interfaces for existing and novel datatypes. In particular, the cross-species navigation and comparison is enhanced through a novel taxonomy section describing each represented species and through the implementation of interactive phylogenetic gene trees for 60% of tunicate genes. The gene expression section displays the results of RNA-seq experiments for the three major model species of solitary ascidians. Gene expression is controlled by the binding of transcription factors to cis-regulatory sequences. A high-resolution description of the DNA-binding specificity for 131 Ciona robusta (formerly C. intestinalis type A) transcription factors by SELEX-seq is provided and used to map candidate binding sites across the Ciona robusta and Phallusia mammillata genomes. Finally, use of a WashU Epigenome browser enhances genome navigation, while a Genomicus server was set up to explore microsynteny relationships within tunicates and with vertebrates, Amphioxus, echinoderms and hemichordates.',\n",
              "  'authors': ['Brozovic, Matija',\n",
              "   'Dantec, Christelle',\n",
              "   'Dardaillon, Justine',\n",
              "   'Dauga, Delphine',\n",
              "   'Faure, Emmanuel',\n",
              "   'Gineste, Mathieu',\n",
              "   'Louis, Alexandra',\n",
              "   'Naville, Magali',\n",
              "   'Nitta, Kazuhiro R',\n",
              "   'Piette, Jacques',\n",
              "   'Reeves, Wendy',\n",
              "   'Scornavacca, Céline',\n",
              "   'Simion, Paul',\n",
              "   'Vincentelli, Renaud',\n",
              "   'Bellec, Maelle',\n",
              "   'Aicha, Sameh Ben',\n",
              "   'Fagotto, Marie',\n",
              "   'Guéroult-Bellone, Marion',\n",
              "   'Haeussler, Maximilian',\n",
              "   'Jacox, Edwin',\n",
              "   'Lowe, Elijah K',\n",
              "   'Mendez, Mickael',\n",
              "   'Roberge, Alexis',\n",
              "   'Stolfi, Alberto',\n",
              "   'Yokomori, Rui',\n",
              "   'Brown, C Titus',\n",
              "   'Cambillau, Christian',\n",
              "   'Christiaen, Lionel',\n",
              "   'Delsuc, Frédéric',\n",
              "   'Douzery, Emmanuel',\n",
              "   'Dumollard, Rémi',\n",
              "   'Kusakabe, Takehiro',\n",
              "   'Nakai, Kenta',\n",
              "   'Nishida, Hiroki',\n",
              "   'Satou, Yutaka',\n",
              "   'Swalla, Billie',\n",
              "   'Veeman, Michael',\n",
              "   'Volff, Jean-Nicolas',\n",
              "   'Lemaire, Patrick'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Ciona intestinalis',\n",
              "   'DNA',\n",
              "   'Data Mining',\n",
              "   'Databases, Genetic',\n",
              "   'Datasets as Topic',\n",
              "   'Evolution, Molecular',\n",
              "   'Gene Expression',\n",
              "   'Gene Ontology',\n",
              "   'Genome',\n",
              "   'Internet',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Phylogeny',\n",
              "   'Protein Binding',\n",
              "   'Species Specificity',\n",
              "   'Transcription Factors',\n",
              "   'Transcription, Genetic',\n",
              "   'Urochordata',\n",
              "   'Vertebrates',\n",
              "   'Web Browser'],\n",
              "  'pmid': '29149270',\n",
              "  'title': 'ANISEED 2017: extending the integrated ascidian database to the exploration and evolutionary comparison of genome-scale datasets.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1532-8392',\n",
              "  'IF_2013': '2.806',\n",
              "  'IF_2014': '2.769',\n",
              "  'IF_2015': '2.791',\n",
              "  'IF_2016': '3.014',\n",
              "  'IF_2017': '3.125',\n",
              "  'ISSN': '0046-8177',\n",
              "  'ISSN_online': '1532-8392',\n",
              "  'ISSN_print': '0046-8177',\n",
              "  'IsoAbbr': 'Hum. Pathol.',\n",
              "  'JrId': 4136,\n",
              "  'MedAbbr': 'Hum Pathol',\n",
              "  'NLMid': '9421547',\n",
              "  'Titre': 'Hum. Pathol.',\n",
              "  'abstract': 'The critical role of expert judgment in clinical decision support has been accommodated in a class of approaches termed knowledge based systems. These have arisen from work in artificial intelligence on expert performance. Some of the perspectives and insights that have been gained from these approaches are briefly discussed.',\n",
              "  'authors': ['Williams, B T'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Human pathology',\n",
              "  'keywords': ['Bayes Theorem',\n",
              "   'Clinical Laboratory Techniques',\n",
              "   'Decision Making',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electronic Data Processing',\n",
              "   'Humans',\n",
              "   'Mental Processes',\n",
              "   'Probability'],\n",
              "  'pmid': '7011935',\n",
              "  'title': 'Symposium on the role of the laboratory in clinical decision making--Part II. Perspectives on clinical decisions.',\n",
              "  'year': '1981'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In the mammalian visual system, magnocellular pathway and parvocellular pathway cooperatively process visual information in parallel. The magnocellular pathway is more global and less particular about the details while the parvocellular pathway recognizes objects based on the local features. In many aspects, Neocognitron may be regarded as the artificial analogue of the parvocellular pathway. It is interesting then to model the magnocellular pathway. In order to achieve \"rotation invariance\" for Neocognitron, we propose a neural network model after the magnocellular pathway and expand its roles to include surmising the orientation of the input pattern prior to recognition. With the incorporation of the magnocellular pathway, a basic shift in the original paradigm has taken place. A pattern is now said to be recognized when and only when one of the winners of the magnocellular pathway is validified by the parvocellular pathway. We have implemented the magnocellular pathway coupled with Neocognitron parallel on transputers; our simulation programme is now able to recognize numerals in arbitrary orientation.',\n",
              "  'authors': ['Ting, C H'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Computer Simulation',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Orientation',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Retinal Ganglion Cells',\n",
              "   'Rotation',\n",
              "   'Visual Pathways'],\n",
              "  'pmid': '8049789',\n",
              "  'title': 'Magnocellular pathway for rotation invariant Neocognitron.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DY'],\n",
              "  'ESSN': '1096-987X',\n",
              "  'IF_2013': '3.601',\n",
              "  'IF_2014': '3.589',\n",
              "  'IF_2015': '3.648',\n",
              "  'IF_2016': '3.229',\n",
              "  'IF_2017': '3.221',\n",
              "  'ISSN': '0192-8651',\n",
              "  'ISSN_online': '1096-987X',\n",
              "  'ISSN_print': '0192-8651',\n",
              "  'IsoAbbr': 'J Comput Chem',\n",
              "  'JrId': 22857,\n",
              "  'MedAbbr': 'J Comput Chem',\n",
              "  'NLMid': '9878362',\n",
              "  'Titre': 'J Comput Chem',\n",
              "  'abstract': 'The usefulness of the quantum chemical descriptors, calculated at the level of the RHF theory using 6-31G basis set for QSAR study of 1,4-dihydropyridine-based calcium channel antagonist was examined. A data set containing 45 dihydropyridine derivatives with known activity was used. Multiple linear regressions combined with genetic algorithm for variable selection and an artificial neural network model combined with principal component analysis for dimension reduction and genetic algorithm for factor selection (PC-GA-ANN) were employed. Some multiparametric MLR equations with good statistical quality were obtained for different classes of dihydropyridine derivatives. The resulting equations suggest that the electronic properties of the atoms belonging to the backbone of the molecules as well as the conformation of the molecules affect the binding of these molecules with their receptor. In the PC-GA-ANN, The principal components of the descriptors data matrix were used as the input of the neural network and then genetic algorithm was applied to select the most relevant set of principal components. Two ANN models with five selected principal components were obtained. These models, which have high statistical qualities, can predict the activity of the molecules with prediction errors lower than +/-5%.',\n",
              "  'authors': ['Hemmateenejad, Bahram',\n",
              "   'Safarpour, Mohammad A',\n",
              "   'Miri, Ramin',\n",
              "   'Taghavi, Fariba'],\n",
              "  'categories': ['CHEMISTRY, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Journal of computational chemistry',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Calcium Channel Blockers',\n",
              "   'Computational Biology',\n",
              "   'Dihydropyridines',\n",
              "   'Linear Models',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Quantitative Structure-Activity Relationship'],\n",
              "  'pmid': '15224393',\n",
              "  'title': 'Application of ab initio theory to QSAR study of 1,4-dihydropyridine-based calcium channel blockers using GA-MLR and PC-GA-ANN procedures.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1473-5660',\n",
              "  'IF_2013': '1.144',\n",
              "  'IF_2014': '1.284',\n",
              "  'IF_2015': '1.25',\n",
              "  'IF_2016': '1.259',\n",
              "  'IF_2017': '1.432',\n",
              "  'ISSN': '0342-5282',\n",
              "  'ISSN_online': '1473-5660',\n",
              "  'ISSN_print': '0342-5282',\n",
              "  'IsoAbbr': 'Int J Rehabil Res',\n",
              "  'JrId': 4305,\n",
              "  'MedAbbr': 'Int J Rehabil Res',\n",
              "  'NLMid': '7805421',\n",
              "  'Titre': 'Int J Rehabil Res',\n",
              "  'abstract': 'The aim of this study was to explore the application of the IROMEC (Interactive social RObotic Mediators as Companions) robot to support play for children with severe physical disabilities in rehabilitation and special education. Within a 2-month pilot study, physically disabled children with a developmental age between 2 and 8 years participated in sessions with the robot. A combination of qualitative and quantitative outcome measures was used to collect data on aspects of feasibility, usability, barriers for the child as well as the therapist and an indication of the effects on playfulness and the achievement of therapeutic and educational goals. A total of 11 children participated in this study (four boys, seven girls). Individually Prioritized Problem Assessment scores indicated a positive effect of the IROMEC robot on the achievement of goals for the children. Professionals did observe meaningful application possibilities, but the lacking adaptability, expandability and technical stability of the robot platform make application in daily care practice impossible. The application of a robot like IROMEC for children with severe physical disabilities seems to be positive and worthwhile, but usability and feasibility aspects are crucial for success.',\n",
              "  'authors': ['van den Heuvel, Renée J F',\n",
              "   'Lexis, Monique A S',\n",
              "   'de Witte, Luc P'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'International journal of rehabilitation research. Internationale Zeitschrift fur Rehabilitationsforschung. Revue internationale de recherches de readaptation',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Disabled Children',\n",
              "   'Education, Special',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Pilot Projects',\n",
              "   'Play and Playthings',\n",
              "   'Robotics',\n",
              "   'Self-Help Devices'],\n",
              "  'pmid': '27755230',\n",
              "  'title': 'Can the IROMEC robot support play in children with severe physical disabilities? A pilot study.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Single-incision laparoscopic and natural orifice translumenal endoscopic surgery (NOTES) are technically challenging methods. Robotics might have the potential to overcome such hurdles with computer technology.|The da Vinci Standard and S System (Intuitive, Sunnyvale, USA) were used in human cadavers and pigs to perform single-incision transabdominal and transvaginal surgery. Robotic arms were crossed and control-switched to achieve intuitive control.|It was possible to perform robotic single-incision laparoscopy in the typical, intuitive fashion. Transvaginal set-up, including docking of the system and introduction of instruments into the abdominal cavity, was possible but no useful manipulation could be performed.|While robotic NOTES with the da Vinci surgical system was not successful, robotic single-incision surgery is feasible using the above set-up. This new approach seems to offer the advantages of single-incision surgery while maintaining the intuitive control of robotic surgery. Clinical application appears justified.',\n",
              "  'authors': ['Hagen, Monika E',\n",
              "   'Wagner, Oliver J',\n",
              "   'Inan, Ihsan',\n",
              "   'Morel, Philippe',\n",
              "   'Fasel, Jean',\n",
              "   'Jacobsen, Garth',\n",
              "   'Spivack, Adam',\n",
              "   'Thompson, Kari',\n",
              "   'Wong, Brian',\n",
              "   'Fischer, Lauren',\n",
              "   'Talamini, Mark',\n",
              "   'Horgan, Santiago'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Abdomen',\n",
              "   'Cadaver',\n",
              "   'Equipment Design',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Vagina'],\n",
              "  'pmid': '20812266',\n",
              "  'title': 'Robotic single-incision transabdominal and transvaginal surgery: initial experience with intersecting robotic arms.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'In the recent years, the use of motion tracking systems for acquisition of functional biomechanical gait data, has received increasing interest due to the richness and accuracy of the measured kinematic information. However, costs frequently restrict the number of subjects employed, and this makes the dimensionality of the collected data far higher than the available samples. This paper applies discriminant analysis algorithms to the classification of patients with different types of foot lesions, in order to establish an association between foot motion and lesion formation. With primary attention to small sample size situations, we compare different types of Bayesian classifiers and evaluate their performance with various dimensionality reduction techniques for feature extraction, as well as search methods for selection of raw kinematic variables. Finally, we propose a novel integrated method which fine-tunes the classifier parameters and selects the most relevant kinematic variables simultaneously. Performance comparisons are using robust resampling techniques such as Bootstrap 632+ and k-fold cross-validation. Results from experimentations with lesion subjects suffering from pathological plantar hyperkeratosis, show that the proposed method can lead to approximately 96% correct classification rates with less than 10% of the original features.',\n",
              "  'authors': ['Goulermas, John Yannis',\n",
              "   'Findlow, Andrew H',\n",
              "   'Nester, Christopher J',\n",
              "   'Howard, David',\n",
              "   'Bowker, Peter'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Computer Simulation',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Discriminant Analysis',\n",
              "   'Female',\n",
              "   'Foot',\n",
              "   'Foot Dermatoses',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Keratoderma, Palmoplantar',\n",
              "   'Leg',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Models, Biological',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Pressure Ulcer',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '16189968',\n",
              "  'title': 'Automated design of robust discriminant analysis classifier for foot pressure lesions using kinematic data.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1473-6586',\n",
              "  'IF_2013': '2.115',\n",
              "  'IF_2014': '2.333',\n",
              "  'IF_2015': '2.006',\n",
              "  'IF_2016': '1.796',\n",
              "  'IF_2017': '1.83',\n",
              "  'ISSN': '0963-0643',\n",
              "  'ISSN_online': '1473-6586',\n",
              "  'ISSN_print': '0963-0643',\n",
              "  'IsoAbbr': 'Curr Opin Urol',\n",
              "  'JrId': 20614,\n",
              "  'MedAbbr': 'Curr Opin Urol',\n",
              "  'NLMid': '9200621',\n",
              "  'Titre': 'Curr Opin Urol',\n",
              "  'abstract': 'To summarize the fundamental principles for technique of robot-assisted radical cystectomy (RARC) based on current peer reviewed literature. Also provide most recent evidence for the efficacy of RARC and Intracorporeal Ileal Conduit (ICIC).|Technical tricks have increased the efficiency of RARC and ICIC diversion. Perioperative and short-term outcomes have demonstrated that RARC is an acceptable alternative to open radical cystectomy. Acceptable positive surgical margin rates, thorough extended lymph node dissection based on tenets of oncological principles and acceptable short-term oncologic outcomes have been reported. Learning curve towards safe incorporation of intracorporeal urinary diversion and its evolution are presented.|The technical tips and tricks have led to evolution of technique translating into improved surgical outcomes. RARC is a well tolerated and effective alternative to open cystectomy and urinary diversion. Intracorporeal urinary diversion is the next challenge on the horizon with an acceptable learning curve and outcomes; this evolution will lead to improvement in quality of life after this morbid surgical procedure.',\n",
              "  'authors': ['Poch, Michael A',\n",
              "   'Raza, Johar',\n",
              "   'Nyquist, John',\n",
              "   'Guru, Khurshid A'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current opinion in urology',\n",
              "  'keywords': ['Cystectomy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Ileum',\n",
              "   'Male',\n",
              "   'Robotics',\n",
              "   'Urinary Bladder Neoplasms',\n",
              "   'Urinary Diversion',\n",
              "   'Urologic Neoplasms'],\n",
              "  'pmid': '23202288',\n",
              "  'title': 'Tips and tricks to robot-assisted radical cystectomy and intracorporeal diversion.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1933-690X',\n",
              "  'IF_2013': '1.967',\n",
              "  'IF_2014': '2.241',\n",
              "  'IF_2015': '2.444',\n",
              "  'IF_2016': '2.343',\n",
              "  'IF_2017': '2.011',\n",
              "  'ISSN': '1933-6896',\n",
              "  'ISSN_online': '1933-690X',\n",
              "  'ISSN_print': '1933-6896',\n",
              "  'IsoAbbr': 'Prion',\n",
              "  'JrId': 35519,\n",
              "  'MedAbbr': 'Prion',\n",
              "  'NLMid': '101472305',\n",
              "  'Titre': 'Prion',\n",
              "  'abstract': 'A number of fungal proteins are capable of adopting multiple alternative, self-perpetuating prion conformations. These prion variants are associated with functional alterations of the prion-forming protein and thus the generation of new, heritable traits that can be detrimental or beneficial. Here we sought to determine the extent to which the previously-reported ZnCl2-sensitivity trait of yeast harboring the [PSI+] prion is modulated by genetic background and prion variant, and whether this trait is accompanied by prion-dependent proteomic changes that could illuminate its physiological basis. We also examined the degree to which prion variant and genetic background influence other prion-dependent phenotypes. We found that ZnCl2 exposure not only reduces colony growth but also limits chronological lifespan of [PSI+] relative to [psi-] cells. This reduction in viability was observed for multiple prion variants in both the S288C and W303 genetic backgrounds. Quantitative proteomic analysis revealed that under exposure to ZnCl2 the expression of stress response proteins was elevated and the expression of proteins involved in energy metabolism was reduced in [PSI+] relative to [psi-] cells. These results suggest that cellular stress and slowed growth underlie the phenotypes we observed. More broadly, we found that prion variant and genetic background modulate prion-dependent changes in protein abundance and can profoundly impact viability in diverse environments. Thus, access to a constellation of prion variants combined with the accumulation of genetic variation together have the potential to substantially increase phenotypic diversity within a yeast population, and therefore to enhance its adaptation potential in changing environmental conditions.',\n",
              "  'authors': ['Allwein, Ben',\n",
              "   'Kelly, Christina',\n",
              "   'Kammoonah, Shaima',\n",
              "   'Mayor, Thibault',\n",
              "   'Cameron, Dale M'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Prion',\n",
              "  'keywords': ['Chlorides',\n",
              "   'Energy Metabolism',\n",
              "   'Gene Ontology',\n",
              "   'Genetic Background',\n",
              "   'Genetic Variation',\n",
              "   'Prions',\n",
              "   'Proteome',\n",
              "   'Saccharomyces cerevisiae',\n",
              "   'Saccharomyces cerevisiae Proteins',\n",
              "   'Stress, Physiological',\n",
              "   'Zinc Compounds'],\n",
              "  'pmid': '30773982',\n",
              "  'title': 'Prion-dependent proteome remodeling in response to environmental stress is modulated by prion variant and genetic background.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1873-5894',\n",
              "  'IF_2013': '2.022',\n",
              "  'IF_2014': '2.09',\n",
              "  'IF_2015': '1.98',\n",
              "  'IF_2016': '2.225',\n",
              "  'IF_2017': '2.564',\n",
              "  'ISSN': '0730-725X',\n",
              "  'ISSN_online': '1873-5894',\n",
              "  'ISSN_print': '0730-725X',\n",
              "  'IsoAbbr': 'Magn Reson Imaging',\n",
              "  'JrId': 5670,\n",
              "  'MedAbbr': 'Magn Reson Imaging',\n",
              "  'NLMid': '8214883',\n",
              "  'Titre': 'Magn Reson Imaging',\n",
              "  'abstract': 'Automatic extraction of the varying regions of magnetic resonance images is required as a prior step in a diagnostic intelligent system. The sparsest representation and high-dimensional feature are provided based on learned dictionary. The classification is done by employing the technique that computes the reconstruction error locally and non-locally of each pixel. The acquired results from the real and simulated images are superior to the best MRI segmentation method with regard to the stability advantages. In addition, it is segmented exactly through a formula taken from the distance and sparse factors. Also, it is done automatically taking sparse factor in unsupervised clustering methods whose results have been improved.',\n",
              "  'authors': ['Khalilzadeh, Mohammad Mahdi',\n",
              "   'Fatemizadeh, Emad',\n",
              "   'Behnam, Hamid'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance imaging',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '23260393',\n",
              "  'title': 'Automatic segmentation of brain MRI in high-dimensional local and non-local feature space based on sparse representation.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'Groundwater contamination is a serious threat to water supply. Risk assessment of groundwater contamination is an effective way to protect the safety of groundwater resource. Groundwater is a complex and fuzzy system with many uncertainties, which is impacted by different geological and hydrological factors. In order to deal with the uncertainty in the risk assessment of groundwater contamination, we propose an approach with analysis hierarchy process and fuzzy comprehensive evaluation integrated together. Firstly, the risk factors of groundwater contamination are identified by the sources-pathway-receptor-consequence method, and a corresponding index system of risk assessment based on DRASTIC model is established. Due to the complexity in the process of transitions between the possible pollution risks and the uncertainties of factors, the method of analysis hierarchy process is applied to determine the weights of each factor, and the fuzzy sets theory is adopted to calculate the membership degrees of each factor. Finally, a case study is presented to illustrate and test this methodology. It is concluded that the proposed approach integrates the advantages of both analysis hierarchy process and fuzzy comprehensive evaluation, which provides a more flexible and reliable way to deal with the linguistic uncertainty and mechanism uncertainty in groundwater contamination without losing important information. ',\n",
              "  'authors': ['Zhang, Qiuwen', 'Yang, Xiaohong', 'Zhang, Yan', 'Zhong, Ming'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Fuzzy Logic',\n",
              "   'Groundwater',\n",
              "   'Models, Theoretical',\n",
              "   'Risk Assessment',\n",
              "   'Water Pollution',\n",
              "   'Water Supply'],\n",
              "  'pmid': '24453883',\n",
              "  'title': 'Risk assessment of groundwater contamination: a multilevel fuzzy comprehensive evaluation approach based on DRASTIC model.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1547-5069',\n",
              "  'IF_2013': '1.772',\n",
              "  'IF_2014': '1.636',\n",
              "  'IF_2015': '1.521',\n",
              "  'IF_2016': '2.396',\n",
              "  'IF_2017': '2.662',\n",
              "  'ISSN': '1527-6546',\n",
              "  'ISSN_online': '1547-5069',\n",
              "  'ISSN_print': '1527-6546',\n",
              "  'IsoAbbr': 'J Nurs Scholarsh',\n",
              "  'JrId': 21727,\n",
              "  'MedAbbr': 'J Nurs Scholarsh',\n",
              "  'NLMid': '100911591',\n",
              "  'Titre': 'J Nurs Scholarsh',\n",
              "  'abstract': 'The purpose of this study was to describe the current evidence found in the nursing literature about robotics used to assist or augment nursing care.|A bibliometric analysis of published research focused on robotics in nursing care was conducted to analyze the trends of publications. A search of the Cumulative Index to Nursing and Allied Health Literature database was conducted. This analysis was used to determine the types and extent of robotic research presented in nursing and allied health literature, journals that publish robotic research, and the origins of the study.|Twenty-one articles met inclusion criteria and spanned the years 2004 to 2016. The main disciplines represented by first authors in these 21 articles were medicine (n = 4, 9%), nursing (n = 4, 9%), and psychiatric medicine (n = 4, 9%). Nine countries were represented by the first author. The majority of the specific studies reported using qualitative research methods (n = 4, 19%) with reports of other research designs being used. Further analysis of subsequent citations found that 248 subsequent citations were generated from these articles.|The application of robots has been used beyond typical physical day-to-day processes as many definitions of robotics suggest. Eleven (52%) of the 21 articles described the use of robots with aged patients. In some cases, robots were used as companions for older adults, as opposed to replacing mechanical and repetitive motions.|Robotics are being used globally in nursing care areas. While a limited amount of research on this topic in nursing exists, this study of the literature offers reports of applications of robots within nursing care areas.',\n",
              "  'authors': ['Carter-Templeton, Heather',\n",
              "   'Frazier, Rachel M',\n",
              "   'Wu, Lin',\n",
              "   'H Wyatt, Tami'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Journal of nursing scholarship : an official publication of Sigma Theta Tau International Honor Society of Nursing',\n",
              "  'keywords': ['Bibliometrics', 'Humans', 'Nursing', 'Robotics'],\n",
              "  'pmid': '29920944',\n",
              "  'title': 'Robotics in Nursing: A Bibliometric Analysis.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TI'],\n",
              "  'ESSN': '1471-5007',\n",
              "  'IF_2013': '6.217',\n",
              "  'IF_2014': '6.204',\n",
              "  'IF_2015': '7.295',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '7.929',\n",
              "  'ISSN': '1471-4922',\n",
              "  'ISSN_online': '1471-5007',\n",
              "  'ISSN_print': '1471-4922',\n",
              "  'IsoAbbr': 'Trends Parasitol.',\n",
              "  'JrId': 21990,\n",
              "  'MedAbbr': 'Trends Parasitol',\n",
              "  'NLMid': '100966034',\n",
              "  'Titre': 'Trends Parasitol.',\n",
              "  'abstract': 'Social interactions are important in everyday life for primates and many other group-living animals; however, these essential exchanges also provide opportunities for parasites to spread through social groups. Network analysis is a unique toolkit for studying pathogen transmission in a social context, and recent primate-parasite network studies shed light on linkages between behavior and infectious disease dynamics, providing insights for conservation and public health. We review existing literature on primate-parasite networks, examining determinants of infection risk, issues of network scale and temporal dynamics, and applications for disease control. We also discuss analytical and conceptual gaps that should be addressed to improve our understanding of how individual and group-level factors affect infection risk, while highlighting interesting areas for future research.',\n",
              "  'authors': ['Rushmore, Julie', 'Bisanzio, Donal', 'Gillespie, Thomas R'],\n",
              "  'categories': ['PARASITOLOGY'],\n",
              "  'journal': 'Trends in parasitology',\n",
              "  'keywords': ['Animals',\n",
              "   'Host-Parasite Interactions',\n",
              "   'Humans',\n",
              "   'Interpersonal Relations',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Parasitic Diseases',\n",
              "   'Primates',\n",
              "   'Research',\n",
              "   'Risk Factors'],\n",
              "  'pmid': '28279627',\n",
              "  'title': 'Making New Connections: Insights from Primate-Parasite Networks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'A number of recent advances have been achieved in the study of midbrain dopaminergic neurons. Understanding these advances and how they relate to one another requires a deep understanding of the computational models that serve as an explanatory framework and guide ongoing experimental inquiry. This intertwining of theory and experiment now suggests very clearly that the phasic activity of the midbrain dopamine neurons provides a global mechanism for synaptic modification. These synaptic modifications, in turn, provide the mechanistic underpinning for a specific class of reinforcement learning mechanisms that now seem to underlie much of human and animal behavior. This review describes both the critical empirical findings that are at the root of this conclusion and the fantastic theoretical advances from which this conclusion is drawn.',\n",
              "  'authors': ['Glimcher, Paul W'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Animals',\n",
              "   'Cells',\n",
              "   'Dopamine',\n",
              "   'Humans',\n",
              "   'Mesencephalon',\n",
              "   'Models, Neurological',\n",
              "   'Reinforcement (Psychology)',\n",
              "   'Reward'],\n",
              "  'pmid': '21389268',\n",
              "  'title': 'Understanding dopamine and reinforcement learning: the dopamine reward prediction error hypothesis.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5305',\n",
              "  'IF_2013': '1.735',\n",
              "  'IF_2014': '1.068',\n",
              "  'IF_2015': '1.069',\n",
              "  'IF_2016': '0.964',\n",
              "  'IF_2017': '1.039',\n",
              "  'ISSN': '0929-8665',\n",
              "  'ISSN_online': '1875-5305',\n",
              "  'ISSN_print': '0929-8665',\n",
              "  'IsoAbbr': 'Protein Pept. Lett.',\n",
              "  'JrId': 20398,\n",
              "  'MedAbbr': 'Protein Pept Lett',\n",
              "  'NLMid': '9441434',\n",
              "  'Titre': 'Protein Pept. Lett.',\n",
              "  'abstract': 'Prediction of thermophilic and mesophilic protein plays a crucial role in both biochemistry and bioengineering. In this study, a different mode of pseudo amino acid composition (PseAAC) was proposed to formulate the protein samples by integrating the amino acid composition, the physic chemical features, as well as the composition transition and distribution features, where each of the protein samples was represented by a numerical vector through the sequence-based approach. Using the support vector machine algorithm, an accurate and reliable classifier was constructed to predict the thermophilic and mesophilic proteins. Moreover, three feature reduction algorithms were obtained for locating the most vital features and reducing the size of feature space. Among the three feature reduction algorithms, the genetic algorithm performed best. Finally, with the reduced features extracted from the genetic algorithm, it was observed that for the selected dataset the new classifier achieved a high accuracy of 95.93% with the Matthews correlation coefficient of 0.9187.',\n",
              "  'authors': ['Wang, De', 'Yang, Liang', 'Fu, Zhengqi', 'Xia, Jingbo'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Protein and peptide letters',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acids',\n",
              "   'Proteins',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '21413920',\n",
              "  'title': 'Prediction of thermophilic protein with pseudo amino Acid composition: an approach from combined feature selection and reduction.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1469-8137',\n",
              "  'IF_2013': '6.373',\n",
              "  'IF_2014': '7.672',\n",
              "  'IF_2015': '7.21',\n",
              "  'IF_2016': '7.33',\n",
              "  'IF_2017': '7.433',\n",
              "  'ISSN': '0028-646X',\n",
              "  'ISSN_online': '1469-8137',\n",
              "  'ISSN_print': '0028-646X',\n",
              "  'IsoAbbr': 'New Phytol.',\n",
              "  'JrId': 8596,\n",
              "  'MedAbbr': 'New Phytol',\n",
              "  'NLMid': '9882884',\n",
              "  'Titre': 'New Phytol.',\n",
              "  'abstract': \"Several studies have investigated soil microbial biodiversity, but understanding of the mechanisms underlying plant responses to soil microbiota remains in its infancy. Here, we focused on tomato (Solanum lycopersicum), testing the hypothesis that plants grown on native soils display different responses to soil microbiotas. Using transcriptomics, proteomics, and biochemistry, we describe the responses of two tomato genotypes (susceptible or resistant to Fusarium oxysporum f. sp. lycopersici) grown on an artificial growth substrate and two native soils (conducive and suppressive to Fusarium). Native soils affected tomato responses by modulating pathways involved in responses to oxidative stress, phenol biosynthesis, lignin deposition, and innate immunity, particularly in the suppressive soil. In tomato plants grown on steam-disinfected soils, total phenols and lignin decreased significantly. The inoculation of a mycorrhizal fungus partly rescued this response locally and systemically. Plants inoculated with the fungal pathogen showed reduced disease symptoms in the resistant genotype in both soils, but the susceptible genotype was partially protected from the pathogen only when grown on the suppressive soil. The 'state of alert' detected in tomatoes reveals novel mechanisms operating in plants in native soils and the soil microbiota appears to be one of the drivers of these plant responses.\",\n",
              "  'authors': ['Chialva, Matteo',\n",
              "   'Salvioli di Fossalunga, Alessandra',\n",
              "   'Daghino, Stefania',\n",
              "   'Ghignone, Stefano',\n",
              "   'Bagnaresi, Paolo',\n",
              "   'Chiapello, Marco',\n",
              "   'Novero, Mara',\n",
              "   'Spadaro, Davide',\n",
              "   'Perotto, Silvia',\n",
              "   'Bonfante, Paola'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'The New phytologist',\n",
              "  'keywords': ['Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Lignin',\n",
              "   'Lycopersicon esculentum',\n",
              "   'Microbiota',\n",
              "   'Models, Biological',\n",
              "   'Plant Immunity',\n",
              "   'Plant Roots',\n",
              "   'Propanols',\n",
              "   'Proteome',\n",
              "   'Soil',\n",
              "   'Soil Microbiology',\n",
              "   'Stress, Physiological',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '29424928',\n",
              "  'title': 'Native soils with their microbiotas elicit a state of alert in tomato plants.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'This paper describes a new method for pruning artificial neural networks, using a measure of the neural complexity of the neural network. This measure is used to determine the connections that should be pruned. The measure computes the information-theoretic complexity of a neural network, which is similar to, yet different from previous research on pruning. The method proposed here shows how overly large and complex networks can be reduced in size, whilst retaining learnt behaviour and fitness. The technique proposed here helps to discover a network topology that matches the complexity of the problem it is meant to solve. This novel pruning technique is tested in a robot control domain, simulating a racecar. It is shown, that the proposed pruning method is a significant improvement over the most commonly used pruning method Magnitude Based Pruning. Furthermore, some of the pruned networks prove to be faster learners than the benchmark network that they originate from. This means that this pruning method can also help to unleash hidden potential in a network, because the learning time decreases substantially for a pruned a network, due to the reduction of dimensionality of the network.',\n",
              "  'authors': ['Jorgensen, Thomas D',\n",
              "   'Haynes, Barry P',\n",
              "   'Norlund, Charlotte C F'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Robotics',\n",
              "   'Software',\n",
              "   'Time Factors'],\n",
              "  'pmid': '18991362',\n",
              "  'title': 'Pruning artificial neural networks using neural complexity measures.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'This work describes a system for identifying event mentions in bio-molecular research abstracts that are either speculative (e.g. analysis of IkappaBalpha phosphorylation, where it is not specified whether phosphorylation did or did not occur) or negated (e.g. inhibition of IkappaBalpha phosphorylation, where phosphorylation did not occur). The data comes from a standard dataset created for the BioNLP 2009 Shared Task. The system uses a machine-learning approach, where the features used for classification are a combination of shallow features derived from the words of the sentences and more complex features based on the semantic outputs produced by a deep parser.|To detect event modification, we use a Maximum Entropy learner with features extracted from the data relative to the trigger words of the events. The shallow features are bag-of-words features based on a small sliding context window of 3-4 tokens on either side of the trigger word. The deep parser features are derived from parses produced by the English Resource Grammar and the RASP parser. The outputs of these parsers are converted into the Minimal Recursion Semantics formalism, and from this, we extract features motivated by linguistics and the data itself. All of these features are combined to create training or test data for the machine learning algorithm.|Over the test data, our methods produce approximately a 4% absolute increase in F-score for detection of event modification compared to a baseline based only on the shallow bag-of-words features.|Our results indicate that grammar-based techniques can enhance the accuracy of methods for detecting event modification.',\n",
              "  'authors': ['Mackinlay, Andrew', 'Martinez, David', 'Baldwin, Timothy'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Abstracting and Indexing as Topic',\n",
              "   'Algorithms',\n",
              "   'Biomedical Research',\n",
              "   'Humans',\n",
              "   'I-kappa B Proteins',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Linear Models',\n",
              "   'Natural Language Processing',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Phosphorylation',\n",
              "   'Principal Component Analysis',\n",
              "   'Semantics'],\n",
              "  'pmid': '22595089',\n",
              "  'title': 'Detecting modification of biomedical events using a deep parsing approach.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1536-0210',\n",
              "  'IF_2013': '4.453',\n",
              "  'IF_2014': '4.437',\n",
              "  'IF_2015': '4.887',\n",
              "  'IF_2016': '5.195',\n",
              "  'IF_2017': '6.224',\n",
              "  'ISSN': '0020-9996',\n",
              "  'ISSN_online': '1536-0210',\n",
              "  'ISSN_print': '0020-9996',\n",
              "  'IsoAbbr': 'Invest Radiol',\n",
              "  'JrId': 4373,\n",
              "  'MedAbbr': 'Invest Radiol',\n",
              "  'NLMid': '0045377',\n",
              "  'Titre': 'Invest Radiol',\n",
              "  'abstract': 'To assess the accuracy of logistic regression (LR) and artificial neural networks (NN) in the diagnosis of calvarial lesions using computed tomography (CT) and to establish the importance of the different features needed for the diagnosis.|One hundred sixty-seven patients with calvarial lesions as the only known disease were enrolled. The clinical and CT data were used for LR and NN models. Both models were tested with the leave-one-out method. The final results of each model were compared using the area under receiver operating characteristic curves (Az).|Of the lesions, 73.1%, were benign and 26.9% were malignant. There was no statistically significant difference between LR and NN in differentiating malignancy. In characterizing the histologic diagnoses, NN was statistically superior to LR. Important NN features needed for malignancy classification were age and edge definition, and for the histologic diagnoses matrix, marginal sclerosis, and age.|NNs offer wide possibilities over statistics for the study of calvarial lesions other than their superior diagnostic performance.',\n",
              "  'authors': ['Arana, E', 'Martí-Bonmatí, L', 'Paredes, R', 'Bautista, D'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Investigative radiology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Infant',\n",
              "   'Logistic Models',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'ROC Curve',\n",
              "   'Skull Neoplasms',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '9788136',\n",
              "  'title': 'Focal calvarial bone lesions. Comparison of logistic regression and neural network models.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'We report the first case of robot-assisted thoracoscopic resection of a mediastinal parathyroid adenoma in the aorto-pulmonary window. Intervention planning was based on preoperative CT-MIBI image fusion, a new imaging modality that enabled reliable and precise localization of the parathyroid. The technique consists of taking MIBI-SPECT and CT separately, using a fixation unit that provides reproducible positioning of the patients head and neck. The data sets are then superimposed upon each other using special software. After the localization process, a minimally invasive operation was performed using the DaVinci operating robot. The procedure proved not only to be feasible but also safe and not time-consuming. The postoperative course was uneventful, and the patient was discharged 4 days postoperatively. Compared to conventional thoracoscopic surgery, the robotic operating system provides better visualization of the operating field and facilitates the movement of the instruments. Precise preoperative imaging enables the careful planning of robot-assisted surgery for ectopic parathyroids located at relatively inaccessible regions such as the anterior mediastinum.',\n",
              "  'authors': ['Profanter, C',\n",
              "   'Schmid, T',\n",
              "   'Prommegger, R',\n",
              "   'Bale, R',\n",
              "   'Sauper, T',\n",
              "   'Bodner, J'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adenoma',\n",
              "   'Humans',\n",
              "   'Mediastinum',\n",
              "   'Parathyroid Neoplasms',\n",
              "   'Parathyroidectomy',\n",
              "   'Robotics',\n",
              "   'Thoracoscopy',\n",
              "   'Tomography, Emission-Computed, Single-Photon',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '14973683',\n",
              "  'title': 'Robot-assisted mediastinal parathyroidectomy.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1520-6882',\n",
              "  'IF_2013': '5.825',\n",
              "  'IF_2014': '5.636',\n",
              "  'IF_2015': '5.886',\n",
              "  'IF_2016': '6.32',\n",
              "  'IF_2017': '6.042',\n",
              "  'ISSN': '0003-2700',\n",
              "  'ISSN_online': '1520-6882',\n",
              "  'ISSN_print': '0003-2700',\n",
              "  'IsoAbbr': 'Anal. Chem.',\n",
              "  'JrId': 519,\n",
              "  'MedAbbr': 'Anal Chem',\n",
              "  'NLMid': '0370536',\n",
              "  'Titre': 'Anal. Chem.',\n",
              "  'abstract': 'The verification and recognition of peak-shaped signals in analytical data are ubiquitous scientific problems. Experimental data contain overlapping signals and noise, which make sensitive and reliable peak recognition difficult. A peak detection system based on a class of neural networks known as \"multilayered perceptrons\" has been created. The network was trained and evaluated with use of vapor-phase infrared spectral data. The results of varying the network architecture on system training and prediction performance along with refinement of the form of the input pattern are presented.',\n",
              "  'authors': ['Wythoff, B J', 'Levine, S P', 'Tomellini, S A'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytical chemistry',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Chemistry Techniques, Analytical',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Models, Theoretical',\n",
              "   'Neurons'],\n",
              "  'pmid': '2096735',\n",
              "  'title': 'Spectral peak verification and recognition using a multilayered neural network.',\n",
              "  'year': '1990'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1879-0038',\n",
              "  'IF_2013': '2.082',\n",
              "  'IF_2014': '2.138',\n",
              "  'IF_2015': '2.319',\n",
              "  'IF_2016': '2.415',\n",
              "  'IF_2017': '2.498',\n",
              "  'ISSN': '0378-1119',\n",
              "  'ISSN_online': '1879-0038',\n",
              "  'ISSN_print': '0378-1119',\n",
              "  'IsoAbbr': 'Gene',\n",
              "  'JrId': 3886,\n",
              "  'MedAbbr': 'Gene',\n",
              "  'NLMid': '7706761',\n",
              "  'Titre': 'Gene',\n",
              "  'abstract': 'The present study aimed to investigate the molecular mechanism underlying congenital obstructive nephropathy (CON).|The microarray dataset GSE70879 was downloaded from the Gene Expression Omnibus, including 3 kidney samples of megabladder mice and 4 control kidneys. Using this dataset, differentially expressed miRNAs (DEMs) were identified between the kidney samples from megabladder mice and controls, followed by identification of the target genes for these DEMs and construction of a DEM and target gene interaction network. Additionally, the target genes were subjected to Gene Ontology and pathway enrichment analyses, and were used for construction of a protein-protein interaction (PPI) network. Finally, regulatory networks were constructed to analyze transcription factors for the key miRNAs.|From 17 DEMs identified between kidney samples of megabladder mice and controls, 3 key miRNAs were screened, including mmu-miR-150-5p, mmu-miR-374b-5p and mmu-miR-126a-5p. The regulatory networks identified vascular endothelial growth factor A (Vegfa) as the common target gene of mmu-miR-150-5p and five transcription factors, including nuclear receptor subfamily 4, group A, member 2 (Nr4a2), Jun dimerisation protein 2 (Jdp2), Kruppel-like factor 6 (Klf6), Neurexophilin-3 (Nxph3) and RNA binding motif protein 17 (Rbm17). The gene encoding phosphatase and tensin homolog (Pten) was found to be co-regulated by mmu-miR-374b-5p and high mobility group protein A1 (Hmga1), whereas the kirsten rat sarcoma viral oncogene (Kras) was identified as a common target gene of mmu-miR-126a-5p and paired box 6 (Pax6).|In summary, the above-listed key miRNAs, transcription factors and key genes may be involved in the development of CON.',\n",
              "  'authors': ['Xin, Guangda', 'Chen, Rui', 'Zhang, Xiaofei'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Gene',\n",
              "  'keywords': ['Animals',\n",
              "   'Disease Models, Animal',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Mice',\n",
              "   'MicroRNAs',\n",
              "   'Microarray Analysis',\n",
              "   'Protein Interaction Maps',\n",
              "   'Signal Transduction',\n",
              "   'Transcription Factors',\n",
              "   'Ureteral Obstruction',\n",
              "   'Urinary Bladder Diseases',\n",
              "   'Urogenital Abnormalities'],\n",
              "  'pmid': '29410288',\n",
              "  'title': 'Identification of key microRNAs, transcription factors and genes associated with congenital obstructive nephropathy in a mouse model of megabladder.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1097-6795',\n",
              "  'IF_2013': '3.987',\n",
              "  'IF_2014': '4.056',\n",
              "  'IF_2015': '4.254',\n",
              "  'IF_2016': '6.852',\n",
              "  'IF_2017': '6.827',\n",
              "  'ISSN': '0894-7317',\n",
              "  'ISSN_online': '1097-6795',\n",
              "  'ISSN_print': '0894-7317',\n",
              "  'IsoAbbr': 'J Am Soc Echocardiogr',\n",
              "  'JrId': 1629,\n",
              "  'MedAbbr': 'J Am Soc Echocardiogr',\n",
              "  'NLMid': '8801388',\n",
              "  'Titre': 'J Am Soc Echocardiogr',\n",
              "  'abstract': \"Automated function imaging (AFI) is a novel algorithm based on speckle-tracking imaging that can be used for assessment of global longitudinal strain of the left ventricle. The purpose of this study was to evaluate the relation between global longitudinal peak systolic strain average (GLPSS Avg) assessed by AFI and left ventricular ejection fraction (LVEF).|The study population consisted of 222 consecutive patients with coronary artery disease (99 patients with acute ST-segment elevation myocardial infarction [STEMI] and 123 patients with advanced ischemic heart failure) and 20 age-matched controls. LVEF was calculated by Simpson's rule. The GLPSS Avg was obtained by AFI.|In the overall study group (65 +/- 10 years, 77% were men), mean GLPSS Avg was 11.1% +/- 4.8% and mean LVEF was 37% +/- 14%. Linear regression analysis showed a good correlation between GLPSS Avg and biplane LVEF for the overall study population (r = 0.83; P < .001). However, in patients with STEMI or heart failure the correlations were less strong (r = 0.42 and r = 0.62, both P < .001).|Systolic global longitudinal strain assessed by AFI was linearly related to biplane LVEF. In patients with STEMI or heart failure, less strong correlations were observed, suggesting that these 2 parameters reflect different aspects of systolic left ventricular function.\",\n",
              "  'authors': ['Delgado, Victoria',\n",
              "   'Mollema, Sjoerd A',\n",
              "   'Ypenburg, Claudia',\n",
              "   'Tops, Laurens F',\n",
              "   'van der Wall, Ernst E',\n",
              "   'Schalij, Martin J',\n",
              "   'Bax, Jeroen J'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Journal of the American Society of Echocardiography : official publication of the American Society of Echocardiography',\n",
              "  'keywords': ['Aged',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Artery Disease',\n",
              "   'Echocardiography',\n",
              "   'Elastic Modulus',\n",
              "   'Elasticity Imaging Techniques',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Statistics as Topic',\n",
              "   'Stroke Volume',\n",
              "   'Ventricular Dysfunction, Left'],\n",
              "  'pmid': '18992675',\n",
              "  'title': 'Relation between global left ventricular longitudinal strain assessed with novel automated function imaging and biplane left ventricular ejection fraction in patients with coronary artery disease.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'The gut-associated lymphoid tissue, connected with liver via bile and blood, constructs a local immune environment of both defense and tolerance. The gut-liver immunity has been well-studied in mammals, yet in fish remains largely unknown, even though enteritis as well as liver and gallbladder syndrome emerged as a limitation in aquaculture. In this study, we performed integrative bioinformatic analysis for both transcriptomic (gut and liver) and proteomic (intestinal mucus and bile) data, in both healthy and infected tilapias. We found more categories of immune transcripts in gut than liver, as well as more adaptive immune in gut meanwhile more innate in liver. Interestingly reduced differential immune transcripts between gut and liver upon inflammation were also revealed. In addition, more immune proteins in bile than intestinal mucus were identified. And bile probably providing immune effectors to intestinal mucus upon inflammation was deduced. Specifically, many key immune transcripts in gut or liver as well as key immune proteins in mucus or bile were demonstrated. Accordingly, we proposed a hypothesized profile of fish gut-liver immunity, during either homeostasis or inflammation. Current data suggested that fish gut and liver may collaborate immunologically while keep homeostasis using own strategies, including potential unique mechanisms.',\n",
              "  'authors': ['Wu, Nan',\n",
              "   'Song, Yu-Long',\n",
              "   'Wang, Bei',\n",
              "   'Zhang, Xiang-Yang',\n",
              "   'Zhang, Xu-Jie',\n",
              "   'Wang, Ya-Li',\n",
              "   'Cheng, Ying-Yin',\n",
              "   'Chen, Dan-Dan',\n",
              "   'Xia, Xiao-Qin',\n",
              "   'Lu, Yi-Shan',\n",
              "   'Zhang, Yong-An'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Animals',\n",
              "   'Bile',\n",
              "   'Fishes',\n",
              "   'Gastrointestinal Tract',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Homeostasis',\n",
              "   'Immunity, Mucosal',\n",
              "   'Inflammation',\n",
              "   'Liver',\n",
              "   'Mucus',\n",
              "   'Protein Domains',\n",
              "   'Proteome',\n",
              "   'RNA, Messenger',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27808112',\n",
              "  'title': 'Fish gut-liver immunity during homeostasis or inflammation revealed by integrative transcriptome and proteome studies.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1878-7886',\n",
              "  'IF_2013': '1.319',\n",
              "  'IF_2014': '1.75',\n",
              "  'IF_2015': '1.568',\n",
              "  'IF_2016': '2.419',\n",
              "  'IF_2017': '2.29',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1878-7886',\n",
              "  'IsoAbbr': 'J Visc Surg',\n",
              "  'JrId': 37669,\n",
              "  'MedAbbr': 'J Visc Surg',\n",
              "  'NLMid': '101532664',\n",
              "  'Titre': 'J Visc Surg',\n",
              "  'abstract': 'In head and neck surgery, minimally invasive approaches have been typically avoided due to concerns about visualization, possible damage to vital structures, and limited availability of effective instrumentation. The incorporation of robotic technology in surgery is now an accepted fact, and because of the complexities of certain laparoscopic procedures, the extended capabilities offered by robotic technology have gained wide acceptance. We report the case of a patient who underwent a robotic total thyroidectomy using a gasless right transaxillary approach. This technique provides a high quality image leading to improved visualization of vital structures during thyroidectomy with the added advantage of avoidance of a neck incision. Several issues regarding this technique remain to be clarified and evaluated in multicenter studies: patient selection, surgeon training and learning curve, postoperative morbidity due to recurrent nerve and parathyroid injury, long term oncologic and cosmetic results. However, we believe that robotic thyroid surgery using a gasless transaxillary approach will advance the frontiers of minimally invasive endocrine surgery.',\n",
              "  'authors': ['Brunaud, L',\n",
              "   'Germain, A',\n",
              "   'Zarnegar, R',\n",
              "   'Klein, M',\n",
              "   'Ayav, A',\n",
              "   'Bresler, L'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Journal of visceral surgery',\n",
              "  'keywords': ['Axilla',\n",
              "   'Female',\n",
              "   'Goiter',\n",
              "   'Humans',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Thyroidectomy'],\n",
              "  'pmid': '21095176',\n",
              "  'title': 'Robotic thyroid surgery using a gasless transaxillary approach: cosmetic improvement or improved quality of surgical dissection?',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1095-9203',\n",
              "  'IF_2013': '31.477',\n",
              "  'IF_2014': '33.611',\n",
              "  'IF_2015': '34.661',\n",
              "  'IF_2016': '37.205',\n",
              "  'IF_2017': '41.058',\n",
              "  'ISSN': '0036-8075',\n",
              "  'ISSN_online': '1095-9203',\n",
              "  'ISSN_print': '0036-8075',\n",
              "  'IsoAbbr': 'Science',\n",
              "  'JrId': 7473,\n",
              "  'MedAbbr': 'Science',\n",
              "  'NLMid': '0404511',\n",
              "  'Titre': 'Science',\n",
              "  'abstract': 'The graceful and agile movements of animals are difficult to analyze and emulate because locomotion is the result of a complex interplay of many components: the central and peripheral nervous systems, the musculoskeletal system, and the environment. The goals of biorobotics are to take inspiration from biological principles to design robots that match the agility of animals, and to use robots as scientific tools to investigate animal adaptive behavior. Used as physical models, biorobots contribute to hypothesis testing in fields such as hydrodynamics, biomechanics, neuroscience, and prosthetics. Their use may contribute to the design of prosthetic devices that more closely take human locomotion principles into account. ',\n",
              "  'authors': ['Ijspeert, Auke J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Science (New York, N.Y.)',\n",
              "  'keywords': ['Animals',\n",
              "   'Behavior, Animal',\n",
              "   'Bionics',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Locomotion',\n",
              "   'Prostheses and Implants',\n",
              "   'Prosthesis Design',\n",
              "   'Robotics',\n",
              "   'Skeleton'],\n",
              "  'pmid': '25301621',\n",
              "  'title': 'Biorobotics: using robots to emulate and investigate agile locomotion.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1938-3797',\n",
              "  'IF_2013': '0.786',\n",
              "  'IF_2014': '0.913',\n",
              "  'IF_2015': '0.814',\n",
              "  'IF_2016': '1.256',\n",
              "  'IF_2017': '1.298',\n",
              "  'ISSN': '1086-8089',\n",
              "  'ISSN_online': '1938-3797',\n",
              "  'ISSN_print': '1086-8089',\n",
              "  'IsoAbbr': 'JSLS',\n",
              "  'JrId': 21218,\n",
              "  'MedAbbr': 'JSLS',\n",
              "  'NLMid': '100884618',\n",
              "  'Titre': 'JSLS',\n",
              "  'abstract': 'Robotic technology has been used in a variety of surgical procedures for its 3D magnification and precision. Minimally invasive techniques have already become common in neurosurgery; however, robotic-assisted procedures in neurosurgery are still a relatively new frontier. This report describes the first use of robotic technology to resect a left thoracolumbar neurofibroma.|A 19-year-old male with a family history of neurofibromatosis was diagnosed with a suspected 3-cm x 4-cm neurofibroma in the T12-L1 left paraspinal area. His only complaint was back pain requiring narcotic analgesics. He had no other findings on physical examination or laboratory/radiologic workup.|After consulting urologic robotic surgeons, it was agreed to use the da Vinci robot (Intuitive Surgical, Sunnyvale, CA) for the resection of this mass. Following retroperitoneal laparoscopic access, the urologic surgeons opened the diaphragm and began the initial mobilization of the mass laparoscopically. The robot was docked, and the neurosurgeon operated the robot at the console to resect the mass from its nerve origin. There were no complications, and the mass, a confirmed neurofibroma, was completely removed. The patient was discharged on postoperative day 2; his back pain resolved, requiring no analgesia by the end of the first postoperative week.|This case provides early evidence that robotic assistance can be successfully used for the resection of a paraspinal neurofibroma.',\n",
              "  'authors': ['Moskowitz, Ross M',\n",
              "   'Young, Jennifer L',\n",
              "   'Box, Geoffrey N',\n",
              "   'Paré, Laura S',\n",
              "   'Clayman, Ralph V'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'JSLS : Journal of the Society of Laparoendoscopic Surgeons',\n",
              "  'keywords': ['Adult',\n",
              "   'Diagnosis, Differential',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Lumbar Vertebrae',\n",
              "   'Male',\n",
              "   'Neurofibroma',\n",
              "   'Retroperitoneal Space',\n",
              "   'Robotics',\n",
              "   'Spinal Neoplasms',\n",
              "   'Thoracic Vertebrae'],\n",
              "  'pmid': '19366544',\n",
              "  'title': 'Retroperitoneal transdiaphragmatic robotic-assisted laparoscopic resection of a left thoracolumbar neurofibroma.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '1758-1109',\n",
              "  'IF_2013': '1.736',\n",
              "  'IF_2014': '1.542',\n",
              "  'IF_2015': '1.377',\n",
              "  'IF_2016': '2.008',\n",
              "  'IF_2017': '3.046',\n",
              "  'ISSN': '1357-633X',\n",
              "  'ISSN_online': '1758-1109',\n",
              "  'ISSN_print': '1357-633X',\n",
              "  'IsoAbbr': 'J Telemed Telecare',\n",
              "  'JrId': 20593,\n",
              "  'MedAbbr': 'J Telemed Telecare',\n",
              "  'NLMid': '9506702',\n",
              "  'Titre': 'J Telemed Telecare',\n",
              "  'abstract': 'We studied the feasibility of using a humanoid robot as an assistant in the monitoring of nursing home residents. The robot can receive alarms via its wireless Internet connection and navigate independently to the room where the alarm originated. Once it has entered the room, the robot can transmit near real time images to the staff and also open a voice connection between the resident and the remote caregivers. This way the remote caregiver is able to check the situation in the room, and take appropriate actions. We tested the prototype robot in three private nursing homes in the Finnish county of South Ostrobothnia. During the testing, 2-4 alarms were produced by each participant and there were 29 alarms in total. The robot was able to navigate correctly to the room from which the alarm was sent and open the speech connection, as well as transmit images via the wireless Internet connection. The experiments provided evidence of the feasibility of using autonomous robots as assistants to nursing home staff in remote monitoring. The response from the nursing home residents was uniformly positive.',\n",
              "  'authors': ['Bäck, Iivari', 'Kallio, Jouko', 'Perälä, Sami', 'Mäkelä, Kari'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Journal of telemedicine and telecare',\n",
              "  'keywords': ['Feasibility Studies',\n",
              "   'Finland',\n",
              "   'Humans',\n",
              "   'Noise',\n",
              "   'Nursing Homes',\n",
              "   'Remote Consultation',\n",
              "   'Robotics'],\n",
              "  'pmid': '22912489',\n",
              "  'title': 'Remote monitoring of nursing home residents using a humanoid robot.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '2050-084X',\n",
              "  'IF_2013': '8.519',\n",
              "  'IF_2014': '9.322',\n",
              "  'IF_2015': '8.303',\n",
              "  'IF_2016': '7.725',\n",
              "  'IF_2017': '7.616',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2050-084X',\n",
              "  'IsoAbbr': 'Elife',\n",
              "  'JrId': 40588,\n",
              "  'MedAbbr': 'Elife',\n",
              "  'NLMid': '101579614',\n",
              "  'Titre': 'Elife',\n",
              "  'abstract': 'Among various advantages, their small size makes model organisms preferred subjects of investigation. Yet, even in model systems detailed analysis of numerous developmental processes at cellular level is severely hampered by their scale. For instance, secondary growth of Arabidopsis hypocotyls creates a radial pattern of highly specialized tissues that comprises several thousand cells starting from a few dozen. This dynamic process is difficult to follow because of its scale and because it can only be investigated invasively, precluding comprehensive understanding of the cell proliferation, differentiation, and patterning events involved. To overcome such limitation, we established an automated quantitative histology approach. We acquired hypocotyl cross-sections from tiled high-resolution images and extracted their information content using custom high-throughput image processing and segmentation. Coupled with automated cell type recognition through machine learning, we could establish a cellular resolution atlas that reveals vascular morphodynamics during secondary growth, for example equidistant phloem pole formation. DOI: http://dx.doi.org/10.7554/eLife.01567.001. ',\n",
              "  'authors': ['Sankar, Martial',\n",
              "   'Nieminen, Kaisa',\n",
              "   'Ragni, Laura',\n",
              "   'Xenarios, Ioannis',\n",
              "   'Hardtke, Christian S'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'eLife',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Automation, Laboratory',\n",
              "   'Cell Proliferation',\n",
              "   'Cell Size',\n",
              "   'High-Throughput Screening Assays',\n",
              "   'Hypocotyl',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Phloem',\n",
              "   'Plant Development',\n",
              "   'Time Factors',\n",
              "   'Xylem'],\n",
              "  'pmid': '24520159',\n",
              "  'title': 'Automated quantitative histology reveals vascular morphodynamics during Arabidopsis hypocotyl secondary growth.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.925',\n",
              "  'IF_2014': '0.986',\n",
              "  'IF_2015': '1.015',\n",
              "  'IF_2016': '1.048',\n",
              "  'IF_2017': '1.008',\n",
              "  'ISSN': '1790-5427',\n",
              "  'ISSN_print': '1790-5427',\n",
              "  'IsoAbbr': 'Hell J Nucl Med',\n",
              "  'JrId': 32588,\n",
              "  'MedAbbr': 'Hell J Nucl Med',\n",
              "  'NLMid': '101257471',\n",
              "  'Titre': 'Hell J Nucl Med',\n",
              "  'abstract': 'Artificial Neural Networks (ANN) are computer programs that emulate the  operation of a large number of processing units that mimic the fundamental mechanisms  of the biological activity of nervous cells as well as their connections and interactions.  As a human brain, an ANN has the ability to learn from the experience of general relations  between variables and thus ANN are particularly suitable to capture the natural  complexity of medical data. Today ANN are widely used as a tool for computer aided diagnosis.  This editorial discusses to what extent ANN can support Nuclear Medicine.',\n",
              "  'authors': ['Palumbo, Barbara', 'Fravolini, Mario Luca'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Hellenic journal of nuclear medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nuclear Medicine',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, Emission-Computed'],\n",
              "  'pmid': '23106047',\n",
              "  'title': 'To what extent can artificial neural network support nuclear medicine?',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The development of large-scale functional brain networks is a complex, lifelong process that can be investigated using resting-state functional connectivity MRI (rs-fcMRI). In this study, we aimed to decode the developmental dynamics of the whole-brain functional network in seven decades (8-79 years) of the human lifespan. We first used parametric curve fitting to examine linear and nonlinear age effect on the resting human brain, and then combined manifold learning and support vector machine methods to predict individuals\\' \"brain ages\" from rs-fcMRI data. We found that age-related changes in interregional functional connectivity exhibited spatially and temporally specific patterns. During brain development from childhood to senescence, functional connections tended to linearly increase in the emotion system and decrease in the sensorimotor system; while quadratic trajectories were observed in functional connections related to higher-order cognitive functions. The complex patterns of age effect on the whole-brain functional network could be effectively represented by a low-dimensional, nonlinear manifold embedded in the functional connectivity space, which uncovered the inherent structure of brain maturation and aging. Regression of manifold coordinates with age further showed that the manifold representation extracted sufficient information from rs-fcMRI data to make prediction about individual brains\\' functional development levels. Our study not only gives insights into the neural substrates that underlie behavioral and cognitive changes over age, but also provides a possible way to quantitatively describe the typical and atypical developmental progression of human brain function using rs-fcMRI.',\n",
              "  'authors': ['Wang, Lubin', 'Su, Longfei', 'Shen, Hui', 'Hu, Dewen'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Algorithms',\n",
              "   'Brain',\n",
              "   'Child',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Longevity',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nerve Net',\n",
              "   'Regression Analysis',\n",
              "   'Rest',\n",
              "   'Support Vector Machine',\n",
              "   'Young Adult'],\n",
              "  'pmid': '22952990',\n",
              "  'title': 'Decoding lifespan changes of the human brain using resting-state functional connectivity MRI.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'The representation of orientation information in the adult visual cortex is plastic as exemplified by phenomena such as perceptual learning or attention. Although these phenomena operate on different time scales and give rise to different changes in the response properties of neurons, both lead to an improvement in visual discrimination or detection tasks. If, however, optimal performance is indeed the goal, the question arises as to why the changes in neuronal response properties are so different. Here, we hypothesize that these differences arise naturally if optimal performance is achieved by means of different mechanisms. To evaluate this hypothesis, we set up a recurrent network model of a visual cortical hypercolumn and asked how each of four different parameter sets (strength of afferent and recurrent synapses, neuronal gains, and additive background inputs) must be changed to optimally improve the encoding accuracy of a particular set of visual stimuli. We find that the predicted changes in the population responses and the tuning functions were different for each set of parameters, hence were strongly dependent on the plasticity mechanism that was operative. An optimal change in the strength of the recurrent connections, for example, led to changes in the response properties that are similar to the changes observed in perceptual learning experiments. An optimal change in the neuronal gains led to changes mimicking neural effects of attention. Assuming the validity of the optimal encoding hypothesis, these model predictions can be used to disentangle the mechanisms of perceptual learning, attention, and other adaptation phenomena.',\n",
              "  'authors': ['Schwabe, Lars', 'Obermayer, Klaus'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Feedback',\n",
              "   'Learning',\n",
              "   'Macaca mulatta',\n",
              "   'Neural Inhibition',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Orientation',\n",
              "   'Synapses',\n",
              "   'Visual Cortex',\n",
              "   'Visual Perception'],\n",
              "  'pmid': '15800187',\n",
              "  'title': 'Adaptivity of tuning functions in a generic recurrent network model of a cortical hypercolumn.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Brain-Computer Interfaces (BCIs) strive to decode brain signals into control commands for severely handicapped people with no means of muscular control. These potential users of noninvasive BCIs display a large range of physical and mental conditions. Prior studies have shown the general applicability of BCI with patients, with the conflict of either using many training sessions or studying only moderately restricted patients. We present a BCI system designed to establish external control for severely motor-impaired patients within a very short time. Within only six experimental sessions, three out of four patients were able to gain significant control over the BCI, which was based on motor imagery or attempted execution. For the most affected patient, we found evidence that the BCI could outperform the best assistive technology (AT) of the patient in terms of control accuracy, reaction time and information transfer rate. We credit this success to the applied user-centered design approach and to a highly flexible technical setup. State-of-the art machine learning methods allowed the exploitation and combination of multiple relevant features contained in the EEG, which rapidly enabled the patients to gain substantial BCI control. Thus, we could show the feasibility of a flexible and tailorable BCI application in severely disabled users. This can be considered a significant success for two reasons: Firstly, the results were obtained within a short period of time, matching the tight clinical requirements. Secondly, the participating patients showed, compared to most other studies, very severe communication deficits. They were dependent on everyday use of AT and two patients were in a locked-in state. For the most affected patient a reliable communication was rarely possible with existing AT. ',\n",
              "  'authors': ['Höhne, Johannes',\n",
              "   'Holz, Elisa',\n",
              "   'Staiger-Sälzer, Pit',\n",
              "   'Müller, Klaus-Robert',\n",
              "   'Kübler, Andrea',\n",
              "   'Tangermann, Michael'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Brain-Computer Interfaces',\n",
              "   'Disabled Persons',\n",
              "   'Electroencephalography',\n",
              "   'Humans',\n",
              "   'Imagination',\n",
              "   'Middle Aged',\n",
              "   'Reaction Time',\n",
              "   'Self-Help Devices',\n",
              "   'Time Factors'],\n",
              "  'pmid': '25162231',\n",
              "  'title': 'Motor imagery for severely motor-impaired patients: evidence for brain-computer interfacing as superior control solution.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'Microcalcifications (microCas) are often early signs of breast cancer. However, detecting them is a difficult visual task and recognizing malignant lesions is a complex diagnostic problem. In recent years, several research groups have been working to develop computer-aided diagnosis (CAD) systems for X-ray mammography. In this paper, we propose a method to detect and classify microcalcifications. In order to discover the presence of microCas clusters, particular attention is paid to the analysis of the spatial arrangement of detected lesions. A fractal model has been used to describe the mammographic image, thus, allowing the use of a matched filtering stage to enhance microcalcifications against the background. A region growing algorithm, coupled with a neural classifier, detects existing lesions. Subsequently, a second fractal model is used to analyze their spatial arrangement so that the presence of microcalcification clusters can be detected and classified. Reported results indicate that fractal models provide an adequate framework for medical image processing; consequently high correct classification rates are achieved.',\n",
              "  'authors': ['Bocchi, L', 'Coppini, G', 'Nori, J', 'Valli, G'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Breast Neoplasms',\n",
              "   'Calcinosis',\n",
              "   'Cluster Analysis',\n",
              "   'Computer Simulation',\n",
              "   'Expert Systems',\n",
              "   'Fractals',\n",
              "   'Humans',\n",
              "   'Mammography',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Precancerous Conditions',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '15121055',\n",
              "  'title': 'Detection of single and clustered microcalcifications in mammograms using fractals models and neural networks.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The identification of drug target proteins (IDTP) plays a critical role in biometrics. The aim of this study was to retrieve potential drug target proteins (DTPs) from a collected protein dataset, which represents an overwhelming task of great significance. Previously reported methodologies for this task generally employ protein-protein interactive networks but neglect informative biochemical attributes. We formulated a novel framework utilizing biochemical attributes to address this problem. In the framework, a biased support vector machine (BSVM) was combined with the deep embedded representation extracted using a deep learning model, stacked auto-encoders (SAEs). In cases of non-drug target proteins (NDTPs) contaminated by DTPs, the framework is beneficial due to the efficient representation of the SAE and relief of the imbalance effect by the BSVM. The experimental results demonstrated the effectiveness of our framework, and the generalization capability was confirmed via comparisons to other models. This study is the first to exploit a deep learning model for IDTP. In summary, nearly 23% of the NDTPs were predicted as likely DTPs, which are awaiting further verification based on biomedical experiments.',\n",
              "  'authors': ['Wang, Qi',\n",
              "   'Feng, YangHe',\n",
              "   'Huang, JinCai',\n",
              "   'Wang, TengJiao',\n",
              "   'Cheng, GuangQuan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Biometry',\n",
              "   'Molecular Targeted Therapy',\n",
              "   'Proteins',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '28453576',\n",
              "  'title': 'A novel framework for the identification of drug target proteins: Combining stacked auto-encoders with a biased support vector machine.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1742-4682',\n",
              "  'IF_2013': '1.265',\n",
              "  'IF_2014': '0.95',\n",
              "  'IF_2015': '1.033',\n",
              "  'IF_2016': '1.46',\n",
              "  'IF_2017': '2',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1742-4682',\n",
              "  'IsoAbbr': 'Theor Biol Med Model',\n",
              "  'JrId': 32005,\n",
              "  'MedAbbr': 'Theor Biol Med Model',\n",
              "  'NLMid': '101224383',\n",
              "  'Titre': 'Theor Biol Med Model',\n",
              "  'abstract': 'Genetic algorithms instruct sophisticated biological organization. Three qualitative kinds of sequence complexity exist: random (RSC), ordered (OSC), and functional (FSC). FSC alone provides algorithmic instruction. Random and Ordered Sequence Complexities lie at opposite ends of the same bi-directional sequence complexity vector. Randomness in sequence space is defined by a lack of Kolmogorov algorithmic compressibility. A sequence is compressible because it contains redundant order and patterns. Law-like cause-and-effect determinism produces highly compressible order. Such forced ordering precludes both information retention and freedom of selection so critical to algorithmic programming and control. Functional Sequence Complexity requires this added programming dimension of uncoerced selection at successive decision nodes in the string. Shannon information theory measures the relative degrees of RSC and OSC. Shannon information theory cannot measure FSC. FSC is invariably associated with all forms of complex biofunction, including biochemical pathways, cycles, positive and negative feedback regulation, and homeostatic metabolism. The algorithmic programming of FSC, not merely its aperiodicity, accounts for biological organization. No empirical evidence exists of either RSC of OSC ever having produced a single instance of sophisticated biological organization. Organization invariably manifests FSC rather than successive random events (RSC) or low-informational self-ordering phenomena (OSC).',\n",
              "  'authors': ['Abel, David L', 'Trevors, Jack T'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Theoretical biology & medical modelling',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Computers',\n",
              "   'Genetics',\n",
              "   'Information Theory',\n",
              "   'Models, Genetic',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Polymers',\n",
              "   'Sequence Alignment',\n",
              "   'Software',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '16095527',\n",
              "  'title': 'Three subsets of sequence complexity and their relevance to biopolymeric information.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1873-4286',\n",
              "  'IF_2013': '3.288',\n",
              "  'IF_2014': '3.452',\n",
              "  'IF_2015': '3.052',\n",
              "  'IF_2016': '2.611',\n",
              "  'IF_2017': '2.757',\n",
              "  'ISSN': '1381-6128',\n",
              "  'ISSN_online': '1873-4286',\n",
              "  'ISSN_print': '1381-6128',\n",
              "  'IsoAbbr': 'Curr. Pharm. Des.',\n",
              "  'JrId': 21197,\n",
              "  'MedAbbr': 'Curr Pharm Des',\n",
              "  'NLMid': '9602487',\n",
              "  'Titre': 'Curr. Pharm. Des.',\n",
              "  'abstract': 'In silico prediction of the new drug-target interactions from existing databases is of important value for the drug discovery process. Currently, the amount of protein targets that have been identified experimentally is still very small compared with the entire human proteins. In order to predict protein-ligand interactions in an accurate manner, we have developed a support vector machine (SVM) model based on the chemical-protein interactions from STITCH. New features from ligand chemical space and interaction networks have been selected and encoded as the feature vectors for SVM analysis. Both the 5-fold cross validation and independent test show high predictive accuracy that outperforms the state-of-the-art method based on ligand similarity. Moreover, 91 distinct pairs of features have been selected to rebuild a simplifier model, which still maintains the same performance as that based on all 332 features. Then, this refined model is used to search for the potential D-amino acid oxidase inhibitors from STITCH database and the predicted results are finally validated by our wet experiments. Out of 10 candidates obtained, seven D-amino acid oxidase inhibitors have been verified, in which four are newly found in the present study, and one may have a new application in therapy of psychiatric disorders other than being an antineoplastic agent. Clearly, our model is capable of predicting potential new drugs or targets on a large scale with high efficiency. ',\n",
              "  'authors': ['Zhao, Mingzhu',\n",
              "   'Chang, Hao-Teng',\n",
              "   'Zhou, Qiang',\n",
              "   'Zeng, Tao',\n",
              "   'Shih, Chung-Shiuan',\n",
              "   'Liu, Zhi-Ping',\n",
              "   'Chen, Luonan',\n",
              "   'Wei, Dong-Qing'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Current pharmaceutical design',\n",
              "  'keywords': ['Antineoplastic Agents',\n",
              "   'Computer Simulation',\n",
              "   'D-Amino-Acid Oxidase',\n",
              "   'Databases, Factual',\n",
              "   'Drug Design',\n",
              "   'Drug Discovery',\n",
              "   'Humans',\n",
              "   'Ligands',\n",
              "   'Mental Disorders',\n",
              "   'Models, Theoretical',\n",
              "   'Molecular Targeted Therapy',\n",
              "   'Proteins',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24410568',\n",
              "  'title': 'Predicting protein-ligand interactions based on chemical preference features with its application to new D-amino acid oxidase inhibitor discovery.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1750-192X',\n",
              "  'IF_2013': '5.215',\n",
              "  'IF_2014': '4.649',\n",
              "  'IF_2015': '4.044',\n",
              "  'IF_2016': '4.541',\n",
              "  'IF_2017': '4.979',\n",
              "  'ISSN': '1750-1911',\n",
              "  'ISSN_online': '1750-192X',\n",
              "  'ISSN_print': '1750-1911',\n",
              "  'IsoAbbr': 'Epigenomics',\n",
              "  'JrId': 37560,\n",
              "  'MedAbbr': 'Epigenomics',\n",
              "  'NLMid': '101519720',\n",
              "  'Titre': 'Epigenomics',\n",
              "  'abstract': 'To assess whether DNA methylation is associated with coronary artery disease (CAD).|An epigenome-wide analysis has been performed on leucocytes from familial hypercholesterolemic (FH) men with (n = 6) or without CAD (n = 6). The results were replicated in an extended sample of FH men (n = 61) and in non-FH men (n = 100) for two of the top differentially methylated loci.|FH men with CAD had significantly more hypomethylated and hypermethylated loci and showed less DNA methylation level variability compared with men without CAD (p < 0.001). Moreover, COL14A1 and MMP9 DNA methylation levels were associated with CAD, age of onset of CAD or CAD risk factors.|These results suggest that epigenome-wide changes are associated with CAD occurrence in men.',\n",
              "  'authors': ['Guay, Simon-Pierre',\n",
              "   'Brisson, Diane',\n",
              "   'Mathieu, Patrick',\n",
              "   'Bossé, Yohan',\n",
              "   'Gaudet, Daniel',\n",
              "   'Bouchard, Luigi'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Epigenomics',\n",
              "  'keywords': ['Collagen',\n",
              "   'Coronary Artery Disease',\n",
              "   'DNA Methylation',\n",
              "   'Epigenesis, Genetic',\n",
              "   'Epigenomics',\n",
              "   'Gene Ontology',\n",
              "   'Genome-Wide Association Study',\n",
              "   'Glycoproteins',\n",
              "   'Humans',\n",
              "   'Hyperlipoproteinemia Type II',\n",
              "   'Male',\n",
              "   'Matrix Metalloproteinase 9',\n",
              "   'Middle Aged'],\n",
              "  'pmid': '25687463',\n",
              "  'title': 'A study in familial hypercholesterolemia suggests reduced methylomic plasticity in men with coronary artery disease.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1745-2422',\n",
              "  'IF_2013': '1.784',\n",
              "  'IF_2014': '1.683',\n",
              "  'IF_2015': '1.762',\n",
              "  'IF_2016': '2.228',\n",
              "  'IF_2017': '2.094',\n",
              "  'ISSN': '1743-4440',\n",
              "  'ISSN_online': '1745-2422',\n",
              "  'ISSN_print': '1743-4440',\n",
              "  'IsoAbbr': 'Expert Rev Med Devices',\n",
              "  'JrId': 32622,\n",
              "  'MedAbbr': 'Expert Rev Med Devices',\n",
              "  'NLMid': '101230445',\n",
              "  'Titre': 'Expert Rev Med Devices',\n",
              "  'abstract': 'Considerable research efforts have recently been dedicated to the establishment of various drug delivery systems (DDS) that are mechanical/physical, chemical and biological/molecular DDS. In this paper, we report on the recent advances in site-specific drug delivery (site-specific, controlled, targeted or smart drug delivery are terms used interchangeably in the literature, to mean to transport a drug or a therapeutic agent to a desired location within the body and release it as desired with negligibly small toxicity and side effect compared to classical drug administration means such as peroral, parenteral, transmucosal, topical and inhalation) based on mechanical/physical systems consisting of implantable and robotic drug delivery systems. While we specifically focus on the robotic or autonomous DDS, which can be reprogrammable and provide multiple doses of a drug at a required time and rate, we briefly cover the implanted DDS, which are well-developed relative to the robotic DDS, to highlight the design and performance requirements, and investigate issues associated with the robotic DDS. Critical research issues associated with both DDSs are presented to describe the research challenges ahead of us in order to establish soft robotic devices for clinical and biomedical applications. ',\n",
              "  'authors': ['Alici, Gursel'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Expert review of medical devices',\n",
              "  'keywords': ['Drug Carriers',\n",
              "   'Drug Delivery Systems',\n",
              "   'Humans',\n",
              "   'Magnetic Phenomena',\n",
              "   'Prostheses and Implants',\n",
              "   'Robotics'],\n",
              "  'pmid': '26415110',\n",
              "  'title': 'Towards soft robotic devices for site-specific drug delivery.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1553-4669',\n",
              "  'IF_2013': '1.575',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.39',\n",
              "  'IF_2016': '3.061',\n",
              "  'IF_2017': '3.061',\n",
              "  'ISSN': '1553-4650',\n",
              "  'ISSN_online': '1553-4669',\n",
              "  'ISSN_print': '1553-4650',\n",
              "  'IsoAbbr': 'J Minim Invasive Gynecol',\n",
              "  'JrId': 32424,\n",
              "  'MedAbbr': 'J Minim Invasive Gynecol',\n",
              "  'NLMid': '101235322',\n",
              "  'Titre': 'J Minim Invasive Gynecol',\n",
              "  'abstract': 'A protocol was established to standardize surgical training using the da Vinci Surgical System. Third- and fourth-year residents and first-year fellows in obstetrics and gynecology participated. The protocol includes online instruction and 2 hands-on modules: platform set-up and surgical skills. Platform set-up provides orientation to the console, visual platform, surgical cart (\"robot\"), camera set-up, port placement, and instrument insertion and removal. Surgical skills includes specific drills using rubber models that simulate human tissue: manipulation, dissection, and simple and advanced suturing. Performance times were recorded for each trainee, as well as previous robotic experience. Times were compared with goals established by Intuitive Surgical, Inc., to assess feasibility of this protocol and baseline robotic surgical aptitude. All trainees (n = 17) completed the training protocol. Performance times met goals for docking and dissection. These times also varied according to level of training. Performance times for manipulation and simple and advanced suturing were prolonged across all groups. Overall pass rates were 100% for docking, 90% for dissection, 11.8% for manipulation, and 0% for simple and advanced suturing. Dissection pass rates varied according to level of training. Performance times and pass rates were not improved with higher level of training or previous robotic experience. Resident and fellow instruction in new surgical technology is an important part of training in obstetrics and gynecology. Herein is reported a method to accomplish robotic training that standardizes instruction and assessment of skills.',\n",
              "  'authors': ['Geller, Elizabeth J', 'Schuler, Kevin M', 'Boggess, John F'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Journal of minimally invasive gynecology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Female',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Gynecology',\n",
              "   'Humans',\n",
              "   'Internship and Residency',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Teaching'],\n",
              "  'pmid': '21262591',\n",
              "  'title': 'Robotic surgical training program in gynecology: how to train residents and fellows.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'This paper deals with the blood glucose level modeling for Type 1 Diabetes Mellitus (T1DM) patients. The model is developed using a recurrent neural network trained with an extended Kalman filter based algorithm in order to develop an affine model, which captures the nonlinear behavior of the blood glucose metabolism. The goal is to derive a dynamical mathematical model for the T1DM as the response of a patient to meal and subcutaneous insulin infusion. Experimental data given by continuous glucose monitoring system is utilized for identification and for testing the applicability of the proposed scheme to T1DM subjects.',\n",
              "  'authors': ['Alanis, Alma Y',\n",
              "   'Leon, Blanca S',\n",
              "   'Sanchez, Edgar N',\n",
              "   'Ruiz-Velazquez, Eduardo'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Blood Glucose',\n",
              "   'Diabetes Mellitus, Type 1',\n",
              "   'Humans',\n",
              "   'Insulin',\n",
              "   'Models, Biological',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '22131301',\n",
              "  'title': 'Blood glucose level neural model for type 1 diabetes mellitus patients.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Following publication of the original article.',\n",
              "  'authors': ['Bajwa, Muhammad Naseer',\n",
              "   'Malik, Muhammad Imran',\n",
              "   'Siddiqui, Shoaib Ahmed',\n",
              "   'Dengel, Andreas',\n",
              "   'Shafait, Faisal',\n",
              "   'Neumeier, Wolfgang',\n",
              "   'Ahmed, Sheraz'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31366347',\n",
              "  'title': 'Correction to: Two-stage framework for optic disc localization and glaucoma classification in retinal fundus images using deep learning.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1546-1726',\n",
              "  'IF_2013': '14.976',\n",
              "  'IF_2014': '16.095',\n",
              "  'IF_2015': '16.724',\n",
              "  'IF_2016': '17.839',\n",
              "  'IF_2017': '19.912',\n",
              "  'ISSN': '1097-6256',\n",
              "  'ISSN_online': '1546-1726',\n",
              "  'ISSN_print': '1097-6256',\n",
              "  'IsoAbbr': 'Nat. Neurosci.',\n",
              "  'JrId': 21092,\n",
              "  'MedAbbr': 'Nat Neurosci',\n",
              "  'NLMid': '9809671',\n",
              "  'Titre': 'Nat. Neurosci.',\n",
              "  'abstract': \"Numerosity estimation is phylogenetically ancient and foundational to human mathematical learning, but its computational bases remain controversial. Here we show that visual numerosity emerges as a statistical property of images in 'deep networks' that learn a hierarchical generative model of the sensory input. Emergent numerosity detectors had response profiles resembling those of monkey parietal neurons and supported numerosity estimation with the same behavioral signature shown by humans and animals.\",\n",
              "  'authors': ['Stoianov, Ivilin', 'Zorzi, Marco'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Nature neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Computer Simulation',\n",
              "   'Haplorhini',\n",
              "   'Humans',\n",
              "   'Mathematics',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Parietal Lobe',\n",
              "   'Visual Pathways',\n",
              "   'Visual Perception'],\n",
              "  'pmid': '22231428',\n",
              "  'title': \"Emergence of a 'visual number sense' in hierarchical generative models.\",\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Esophageal adenocarcinoma (EAC) has become a major concern in Western countries due to rapid rises in incidence coupled with very poor survival rates. One of the key risk factors for the development of this cancer is the presence of Barrett's esophagus (BE), which is believed to form in response to repeated gastro-esophageal reflux. In this study we performed comparative, genome-wide expression profiling (using Illumina whole-genome Beadarrays) on total RNA extracted from esophageal biopsy tissues from individuals with EAC, BE (in the absence of EAC) and those with normal squamous epithelium. We combined these data with publically accessible raw data from three similar studies to investigate key gene and ontology differences between these three tissue states. The results support the deduction that BE is a tissue with enhanced glycoprotein synthesis machinery (DPP4, ATP2A3, AGR2) designed to provide strong mucosal defenses aimed at resisting gastro-esophageal reflux. EAC exhibits the enhanced extracellular matrix remodeling (collagens, IGFBP7, PLAU) effects expected in an aggressive form of cancer, as well as evidence of reduced expression of genes associated with mucosal (MUC6, CA2, TFF1) and xenobiotic (AKR1C2, AKR1B10) defenses. When our results are compared to previous whole-genome expression profiling studies keratin, mucin, annexin and trefoil factor gene groups are the most frequently represented differentially expressed gene families. Eleven genes identified here are also represented in at least 3 other profiling studies. We used these genes to discriminate between squamous epithelium, BE and EAC within the two largest cohorts using a support vector machine leave one out cross validation (LOOCV) analysis. While this method was satisfactory for discriminating squamous epithelium and BE, it demonstrates the need for more detailed investigations into profiling changes between BE and EAC.\",\n",
              "  'authors': ['Nancarrow, Derek J',\n",
              "   'Clouston, Andrew D',\n",
              "   'Smithers, B Mark',\n",
              "   'Gotley, David C',\n",
              "   'Drew, Paul A',\n",
              "   'Watson, David I',\n",
              "   'Tyagi, Sonika',\n",
              "   'Hayward, Nicholas K',\n",
              "   'Whiteman, David C'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Barrett Esophagus',\n",
              "   'Biomarkers, Tumor',\n",
              "   'Carcinoma, Squamous Cell',\n",
              "   'Case-Control Studies',\n",
              "   'Cell Proliferation',\n",
              "   'Esophageal Neoplasms',\n",
              "   'Esophagus',\n",
              "   'Female',\n",
              "   'Gene Expression Profiling',\n",
              "   'Genome, Human',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Mucous Membrane',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'RNA, Messenger',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
              "   'Support Vector Machine',\n",
              "   'Young Adult'],\n",
              "  'pmid': '21829465',\n",
              "  'title': \"Whole genome expression array profiling highlights differences in mucosal defense genes in Barrett's esophagus and esophageal adenocarcinoma.\",\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2193-1801',\n",
              "  'IF_2013': '0.982',\n",
              "  'IF_2014': '0.982',\n",
              "  'IF_2015': '0.982',\n",
              "  'IF_2016': '1.13',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2193-1801',\n",
              "  'IsoAbbr': 'Springerplus',\n",
              "  'JrId': 40876,\n",
              "  'MedAbbr': 'Springerplus',\n",
              "  'NLMid': '101597967',\n",
              "  'Titre': 'Springerplus',\n",
              "  'abstract': \"Prior research studies have shown that the peak signal to noise ratio (PSNR) is the most frequent watermarked image quality metric that is used for determining the levels of strength and weakness of watermarking algorithms. Conversely, normalised cross correlation (NCC) is the most common metric used after attacks were applied to a watermarked image to verify the strength of the algorithm used. Many researchers have used these approaches to evaluate their algorithms. These strategies have been used for a long time, however, which unfortunately limits the value of PSNR and NCC in reflecting the strength and weakness of the watermarking algorithms. This paper considers this issue to determine the threshold values of these two parameters in reflecting the amount of strength and weakness of the watermarking algorithms. We used our novel watermarking technique for embedding four watermarks in intermediate significant bits (ISB) of six image files one-by-one through replacing the image pixels with new pixels and, at the same time, keeping the new pixels very close to the original pixels. This approach gains an improved robustness based on the PSNR and NCC values that were gathered. A neural network model was built that uses the image quality metrics (PSNR and NCC) values obtained from the watermarking of six grey-scale images that use ISB as the desired output and that are trained for each watermarked image's PSNR and NCC. The neural network predicts the watermarked image's PSNR together with NCC after the attacks when a portion of the output of the same or different types of image quality metrics (PSNR and NCC) are obtained. The results indicate that the NCC metric fluctuates before the PSNR values deteriorate. \",\n",
              "  'authors': ['Zeki, Akram', 'Abubakar, Adamu', 'Chiroma, Haruna'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'SpringerPlus',\n",
              "  'keywords': [''],\n",
              "  'pmid': '27386317',\n",
              "  'title': 'An intermediate significant bit (ISB) watermarking technique using neural networks.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1089-8638',\n",
              "  'IF_2013': '3.959',\n",
              "  'IF_2014': '4.333',\n",
              "  'IF_2015': '4.517',\n",
              "  'IF_2016': '4.632',\n",
              "  'IF_2017': '4.894',\n",
              "  'ISSN': '0022-2836',\n",
              "  'ISSN_online': '1089-8638',\n",
              "  'ISSN_print': '0022-2836',\n",
              "  'IsoAbbr': 'J. Mol. Biol.',\n",
              "  'JrId': 4967,\n",
              "  'MedAbbr': 'J Mol Biol',\n",
              "  'NLMid': '2985088R',\n",
              "  'Titre': 'J. Mol. Biol.',\n",
              "  'abstract': 'A two-stage neural network has been used to predict protein secondary structure based on the position specific scoring matrices generated by PSI-BLAST. Despite the simplicity and convenience of the approach used, the results are found to be superior to those produced by other methods, including the popular PHD method according to our own benchmarking results and the results from the recent Critical Assessment of Techniques for Protein Structure Prediction experiment (CASP3), where the method was evaluated by stringent blind testing. Using a new testing set based on a set of 187 unique folds, and three-way cross-validation based on structural similarity criteria rather than sequence similarity criteria used previously (no similar folds were present in both the testing and training sets) the method presented here (PSIPRED) achieved an average Q3 score of between 76.5% to 78.3% depending on the precise definition of observed secondary structure used, which is the highest published score for any method to date. Given the success of the method in CASP3, it is reasonable to be confident that the evaluation presented here gives a fair indication of the performance of the method in general.',\n",
              "  'authors': ['Jones, D T'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of molecular biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Folding',\n",
              "   'Protein Structure, Secondary',\n",
              "   'Sequence Analysis'],\n",
              "  'pmid': '10493868',\n",
              "  'title': 'Protein secondary structure prediction based on position-specific scoring matrices.',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IA'],\n",
              "  'ESSN': '1557-8593',\n",
              "  'IF_2013': '2.293',\n",
              "  'IF_2014': '2.106',\n",
              "  'IF_2015': '2.198',\n",
              "  'IF_2016': '2.698',\n",
              "  'IF_2017': '2.921',\n",
              "  'ISSN': '1520-9156',\n",
              "  'ISSN_online': '1557-8593',\n",
              "  'ISSN_print': '1520-9156',\n",
              "  'IsoAbbr': 'Diabetes Technol. Ther.',\n",
              "  'JrId': 21552,\n",
              "  'MedAbbr': 'Diabetes Technol Ther',\n",
              "  'NLMid': '100889084',\n",
              "  'Titre': 'Diabetes Technol. Ther.',\n",
              "  'abstract': 'Easy-to-use insulin devices are an important tool in the treatment of diabetes. In this study, injection force requirements and dosage scale displays were evaluated for NovoPen 3 and NovoPen 4 (both from Novo Nordisk A/S, Copenhagen, Denmark).|To simulate 5 years of expected lifetime usage, 5,475 injections were performed automatically. Before and after lifetime testing, the force required to expel 60 U of insulin from NovoPen 3 and NovoPen 4 with 30-gauge or 31-gauge needles was measured. To compare dosage scale displays, digital images of multiple settings were made, and the total inked areas of digits were converted to mm(2).|At baseline, the mean +/- SD injection force of NovoPen 4 was 9.14 +/- 0.87 N and 16.55 +/- 1.17 N, which was significantly lower (P < 0.001) than for NovoPen 3, at 18.36 +/- 1.06 N and 29.81 +/- 1.26 N, with the 30-gauge and 31-gauge needle, respectively. After simulated lifetime testing, mean +/- SD injection force was 10.93 +/- 0.77 N and 17.77 +/- 1.14 N for NovoPen 4, which was significantly lower than the injection force of 18.54 +/- 0.94 N and 31.69 +/- 1.98 N for NovoPen 3 (P < 0.001). The mean dosage scale digit size was 1.63 mm(2) for NovoPen 3 and 7.82 mm(2) for NovoPen 4, with a mean difference of 6.19 mm(2). The display for NovoPen 4 was 4.74 times larger (P < 0.001).|The mean injection force required to operate NovoPen 4 was reduced up to 50% compared with NovoPen 3 (P < 0.001), and the mean dosage display for NovoPen 4 was over four times larger than for NovoPen 3 (P < 0.001). Based on these findings, patients with diabetes who have manual or visual impairment should find it easier to dose insulin with NovoPen 4.',\n",
              "  'authors': ['Donsmark, Morten',\n",
              "   'Herold, Lars',\n",
              "   'Kristensen, Carsten Mølgård'],\n",
              "  'categories': ['ENDOCRINOLOGY & METABOLISM'],\n",
              "  'journal': 'Diabetes technology & therapeutics',\n",
              "  'keywords': ['Confidence Intervals',\n",
              "   'Diabetes Mellitus',\n",
              "   'Drug Delivery Systems',\n",
              "   'Humans',\n",
              "   'Hypoglycemic Agents',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Injections',\n",
              "   'Insulin',\n",
              "   'Insulin Detemir',\n",
              "   'Insulin, Long-Acting',\n",
              "   'Insulin, Regular, Pork',\n",
              "   'Regression Analysis',\n",
              "   'Robotics'],\n",
              "  'pmid': '19764837',\n",
              "  'title': 'A comparison of injection force and dosage scale size between NovoPen 3 and NovoPen 4.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1878-7533',\n",
              "  'IF_2013': '4.942',\n",
              "  'IF_2014': '4.066',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '4.496',\n",
              "  'IF_2017': '3.9',\n",
              "  'ISSN': '1550-7289',\n",
              "  'ISSN_online': '1878-7533',\n",
              "  'ISSN_print': '1550-7289',\n",
              "  'IsoAbbr': 'Surg Obes Relat Dis',\n",
              "  'JrId': 33051,\n",
              "  'MedAbbr': 'Surg Obes Relat Dis',\n",
              "  'NLMid': '101233161',\n",
              "  'Titre': 'Surg Obes Relat Dis',\n",
              "  'abstract': 'Obesity is a nationwide epidemic, and the only evidence-based, durable treatment of this disease is bariatric surgery. This field has evolved drastically during the past decade. One of the latest advances has been the increased use of robotics within this field. The goal of our study was to perform a systematic review of the recent data to determine the safety and efficacy of robotic bariatric surgery. The setting was the University Hospitals Case Medical Center (Cleveland, OH).|A PubMed search was performed for robotic bariatric surgery from 2005 to 2011. The inclusion criteria were English language, original research, human, and bariatric surgical procedures. Perioperative data were then collected from each study and recorded.|A total of 18 studies were included in our review. The results of our systematic review showed that bariatric surgery, when performed with the use of robotics, had similar or lower complication rates compared with traditional laparoscopy. Two studies showed shorter operative times using the robot for Roux-en-Y gastric bypass, but 4 studies showed longer operative times in the robotic arm. In addition, the learning curve appears to be shorter when robotic gastric bypass is compared with the traditional laparoscopic approach. Most investigators agreed that robotic laparoscopic surgery provides superior imaging and freedom of movement compared with traditional laparoscopy.|The application of robotics appears to be a safe option within the realm of bariatric surgery. Prospective randomized trials comparing robotic and laparoscopic outcomes are needed to further define the role of robotics within the field of bariatric surgery. Longer follow-up times would also help elucidate any long-term outcomes differences with the use of robotics versus traditional laparoscopy.',\n",
              "  'authors': ['Fourman, Matthew M', 'Saber, Alan A'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgery for obesity and related diseases : official journal of the American Society for Bariatric Surgery',\n",
              "  'keywords': ['Adult',\n",
              "   'Bariatric Surgery',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Obesity, Morbid',\n",
              "   'Operative Time',\n",
              "   'Postoperative Complications',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Weight Loss'],\n",
              "  'pmid': '22579735',\n",
              "  'title': 'Robotic bariatric surgery: a systematic review.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1097-0150',\n",
              "  'IF_2013': '1.077',\n",
              "  'IF_2014': '0.69',\n",
              "  'IF_2015': '0.606',\n",
              "  'IF_2016': '1.667',\n",
              "  'IF_2017': '1.2',\n",
              "  'ISSN': '1092-9088',\n",
              "  'ISSN_online': '1097-0150',\n",
              "  'ISSN_print': '1092-9088',\n",
              "  'IsoAbbr': 'Comput. Aided Surg.',\n",
              "  'JrId': 20599,\n",
              "  'MedAbbr': 'Comput Aided Surg',\n",
              "  'NLMid': '9708375',\n",
              "  'Titre': 'Comput. Aided Surg.',\n",
              "  'abstract': \"In this article we present a new concept for computer-assisted colonoscopy based on a miniature robot capable of propelling itself semiautonomously along the colon. The miniature robot is designed to perform the same functions as current colonoscopy systems-i.e., visualization and tissue sampling for biopsy-and exploits an innovative inchworm-like locomotion principle based on adhering to the colon wall by vacuum suction. The miniature robot is connected by a thin and flexible umbilical cable to an external control unit; this unit provides pneumatic actuation signals in the appropriate sequence to the miniature robot, and information on the robot's functioning to the endoscopist, who can either teleoperate or directly supervise its operation. A prototype colonoscopy system using this robot has been fabricated and tested in vitro, with promising results. The proposed concept has strong potential for further development, since miniaturization and functional integration of instrumentation and tools, together with computer assistance, not only make colonoscopy more acceptable, but can also open up a wide range of new applications in endoluminal diagnosis, therapy, and surgery.\",\n",
              "  'authors': ['Dario, P', 'Carrozza, M C', 'Pietrabissa, A'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer aided surgery : official journal of the International Society for Computer Aided Surgery',\n",
              "  'keywords': ['Animals',\n",
              "   'Colonoscopes',\n",
              "   'Colonoscopy',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Miniaturization',\n",
              "   'Robotics',\n",
              "   'Swine'],\n",
              "  'pmid': '10417826',\n",
              "  'title': 'Development and in vitro testing of a miniature robotic system for computer-assisted colonoscopy.',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1095-8630',\n",
              "  'IF_2013': '3.188',\n",
              "  'IF_2014': '2.723',\n",
              "  'IF_2015': '3.131',\n",
              "  'IF_2016': '4.01',\n",
              "  'IF_2017': '4.005',\n",
              "  'ISSN': '0301-4797',\n",
              "  'ISSN_online': '1095-8630',\n",
              "  'ISSN_print': '0301-4797',\n",
              "  'IsoAbbr': 'J. Environ. Manage.',\n",
              "  'JrId': 22117,\n",
              "  'MedAbbr': 'J Environ Manage',\n",
              "  'NLMid': '0401664',\n",
              "  'Titre': 'J. Environ. Manage.',\n",
              "  'abstract': 'Fuzzy comprehensive assessment was applied to assess the quality of air, water and soil in Zhuzhou City, Hunan Province, China based on the monitoring data of 1997 and National Environmental Quality Standards of China. Results show that Zhuzhou air quality belonged to Class II with industry and traffic as the main control areas and SO2 as the major pollutant. Water quality within the Zhuzhou section of the Xiang River belonged to Class IV, with the Xiawan section as the main control area and NH3-N as the major pollutant. Soil environmental quality with respect to heavy metal pollution belonged to Class III, with Cd as the major pollutant. For those pollutants showing significant differences between the years of 1990-1997, predictions were made for the environmental quality for the period 1998-2003. According to the assessment results, cleaner production in the key pollution sources and the establishment of a municipal sewage treatment plant are proposed to improve the environmental quality of Zhuzhou City.',\n",
              "  'authors': ['Haiyan, Wang'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Journal of environmental management',\n",
              "  'keywords': ['China',\n",
              "   'Cities',\n",
              "   'Environmental Pollutants',\n",
              "   'Forecasting',\n",
              "   'Fuzzy Logic',\n",
              "   'Metals, Heavy',\n",
              "   'Models, Theoretical',\n",
              "   'Quality Control',\n",
              "   'Sewage'],\n",
              "  'pmid': '12448410',\n",
              "  'title': 'Assessment and prediction of overall environmental quality of Zhuzhou City, Hunan Province, China.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'With recent advancements in Semantic Web technologies, a new trend in MCQ item generation has emerged through the use of ontologies. Ontologies are knowledge representation structures that formally describe entities in a domain and their relationships, thus enabling automated inference and reasoning. Ontology-based MCQ item generation is still in its infancy, but substantial research efforts are being made in the field. However, the applicability of these models for use in an educational setting has not been thoroughly evaluated. In this paper, we present an experimental evaluation of an ontology-based MCQ item generation system known as OntoQue. The evaluation was conducted using two different domain ontologies. The findings of this study show that ontology-based MCQ generation systems produce satisfactory MCQ items to a certain extent. However, the evaluation also revealed a number of shortcomings with current ontology-based MCQ item generation systems with regard to the educational significance of an automatically constructed MCQ item, the knowledge level it addresses, and its language structure. Furthermore, for the task to be successful in producing high-quality MCQ items for learning assessments, this study suggests a novel, holistic view that incorporates learning content, learning objectives, lexical knowledge, and scenarios into a single cohesive framework. ',\n",
              "  'authors': ['Al-Yahya, Maha'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Choice Behavior',\n",
              "   'Internet',\n",
              "   'Language',\n",
              "   'Semantics'],\n",
              "  'pmid': '24982937',\n",
              "  'title': 'Ontology-based multiple choice question generation.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': \"It is widely believed that end-to-end training with the backpropagation algorithm is essential for learning good feature detectors in early layers of artificial neural networks, so that these detectors are useful for the task performed by the higher layers of that neural network. At the same time, the traditional form of backpropagation is biologically implausible. In the present paper we propose an unusual learning rule, which has a degree of biological plausibility and which is motivated by Hebb's idea that change of the synapse strength should be local-i.e., should depend only on the activities of the pre- and postsynaptic neurons. We design a learning algorithm that utilizes global inhibition in the hidden layer and is capable of learning early feature detectors in a completely unsupervised way. These learned lower-layer feature detectors can be used to train higher-layer weights in a usual supervised way so that the performance of the full network is comparable to the performance of standard feedforward networks trained end-to-end with a backpropagation algorithm on simple tasks.\",\n",
              "  'authors': ['Krotov, Dmitry', 'Hopfield, John J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30926658',\n",
              "  'title': 'Unsupervised learning by competing hidden units.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1523-6838',\n",
              "  'IF_2013': '5.756',\n",
              "  'IF_2014': '5.9',\n",
              "  'IF_2015': '6.269',\n",
              "  'IF_2016': '7.623',\n",
              "  'IF_2017': '7.129',\n",
              "  'ISSN': '0272-6386',\n",
              "  'ISSN_online': '1523-6838',\n",
              "  'ISSN_print': '0272-6386',\n",
              "  'IsoAbbr': 'Am. J. Kidney Dis.',\n",
              "  'JrId': 423,\n",
              "  'MedAbbr': 'Am J Kidney Dis',\n",
              "  'NLMid': '8110075',\n",
              "  'Titre': 'Am. J. Kidney Dis.',\n",
              "  'abstract': 'Artificial intelligence is playing an increasingly important role in many fields of medicine, assisting physicians in most steps of patient management. In nephrology, artificial intelligence can already be used to improve clinical care, hemodialysis prescriptions, and follow-up of transplant recipients. However, many nephrologists are still unfamiliar with the basic principles of medical artificial intelligence. This review seeks to provide an overview of medical artificial intelligence relevant to the practicing nephrologist, in all fields of nephrology. We define the core concepts of artificial intelligence and machine learning and cover the basics of the functioning of neural networks and deep learning. We also discuss the most recent clinical applications of artificial intelligence in nephrology and medicine; as an example, we describe how artificial intelligence can predict the occurrence of progressive immunoglobulin A nephropathy. Finally, we consider the future of artificial intelligence in clinical nephrology and its impact on medical practice, and conclude with a discussion of the ethical issues that the use of artificial intelligence raises in terms of clinical decision making, physician-patient relationship, patient privacy, and data collection.',\n",
              "  'authors': ['Niel, Olivier', 'Bastard, Paul'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'American journal of kidney diseases : the official journal of the National Kidney Foundation',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31451330',\n",
              "  'title': 'Artificial Intelligence in Nephrology: Core Concepts, Clinical Applications, and Perspectives.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'In recent years the combined use of functional electrical stimulation (FES) and robotic devices, called hybrid robotic rehabilitation systems, has emerged as a promising approach for rehabilitation of lower and upper limb motor functions. This paper presents a review of the state of the art of current hybrid robotic solutions for upper limb rehabilitation after stroke. For this aim, studies have been selected through a search using web databases: IEEE-Xplore, Scopus and PubMed. A total of 10 different hybrid robotic systems were identified, and they are presented in this paper. Selected systems are critically compared considering their technological components and aspects that form part of the hybrid robotic solution, the proposed control strategies that have been implemented, as well as the current technological challenges in this topic. Additionally, we will present and discuss the corresponding evidences on the effectiveness of these hybrid robotic therapies. The review also discusses the future trends in this field.',\n",
              "  'authors': ['Resquín, Francisco',\n",
              "   'Cuesta Gómez, Alicia',\n",
              "   'Gonzalez-Vargas, Jose',\n",
              "   'Brunetti, Fernando',\n",
              "   'Torricelli, Diego',\n",
              "   'Molina Rueda, Francisco',\n",
              "   'Cano de la Cuerda, Roberto',\n",
              "   'Miangolarra, Juan Carlos',\n",
              "   'Pons, José Luis'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Electric Stimulation Therapy',\n",
              "   'Hand Strength',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Stroke Rehabilitation',\n",
              "   'Systems Integration',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '27692878',\n",
              "  'title': 'Hybrid robotic systems for upper limb rehabilitation after stroke: A review.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1879-016X',\n",
              "  'IF_2013': '7.745',\n",
              "  'IF_2014': '9.723',\n",
              "  'IF_2015': '11',\n",
              "  'IF_2016': '11.127',\n",
              "  'IF_2017': '10.376',\n",
              "  'ISSN': '0163-7258',\n",
              "  'ISSN_online': '1879-016X',\n",
              "  'ISSN_print': '0163-7258',\n",
              "  'IsoAbbr': 'Pharmacol. Ther.',\n",
              "  'JrId': 6445,\n",
              "  'MedAbbr': 'Pharmacol Ther',\n",
              "  'NLMid': '7905840',\n",
              "  'Titre': 'Pharmacol. Ther.',\n",
              "  'abstract': 'Dysregulation of cell metabolism and redox balance is implicated in the pathogenesis and progression of cancer and autoimmune diseases. Because the cell proliferation and apoptotic regulatory pathways are interconnected with metabolic and redox signalling pathways, the current mono-target treatment is ineffective, and multi-drug resistance remains common. Complex diseases are often implicated in a network-based context of pathology; therefore, a new holistic intervention approach is required to block multi-crosstalk in such complicated circumstances. The use of therapeutic agents isolated from herbs to holistically modulate metabolism and redox state has been shown to relieve carcinoma growth and the inflammatory response in autoimmune disorders. Multiple clinically applied or novel herbal chemicals with metabolic and redox modulatory capacity as well as low toxicity have recently been identified. Moreover, new metabolic targets and mechanisms of drug action have been discovered, leading to the exploration of new pathways for drug repositioning, clinical biomarker spectra, clinical treatment strategies and drug development. Taken together with multiple supporting examples, the modulation of cell metabolism and the redox capacity using herbal chemicals is emerging as a new, alternative strategy for the holistic treatment of cancer and autoimmune disorders. In the future, the development of new diagnostic tools based on the detection of metabolic and redox biomarkers, reformulation of optimized herbal compositions using artificial intelligence, and the combination of herbs with mono-targeting drugs will reveal new potential for clinical application.',\n",
              "  'authors': ['Fan, Xing-Xing',\n",
              "   'Pan, Hu-Dan',\n",
              "   'Li, Ying',\n",
              "   'Guo, Rui-Jin',\n",
              "   'Leung, Elaine Lai-Han',\n",
              "   'Liu, Liang'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Pharmacology & therapeutics',\n",
              "  'keywords': ['Animals',\n",
              "   'Antineoplastic Agents',\n",
              "   'Apoptosis',\n",
              "   'Autoimmune Diseases',\n",
              "   'Cell Proliferation',\n",
              "   'Disease Progression',\n",
              "   'Drug Development',\n",
              "   'Drug Repositioning',\n",
              "   'Humans',\n",
              "   'Molecular Targeted Therapy',\n",
              "   'Neoplasms',\n",
              "   'Oxidation-Reduction',\n",
              "   'Plant Preparations'],\n",
              "  'pmid': '29953901',\n",
              "  'title': 'Novel therapeutic strategy for cancer and autoimmune conditions: Modulating cell metabolism and redox capacity.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'In this work, gene expression in autism spectrum disorder (ASD) is analyzed with the goal of selecting the most attributed genes and performing classification. The objective was achieved by utilizing a combination of various statistical filters and a wrapper-based geometric binary particle swarm optimization-support vector machine (GBPSO-SVM) algorithm. The utilization of different filters was accentuated by incorporating a mean and median ratio criterion to remove very similar genes. The results showed that the most discriminative genes that were identified in the first and last selection steps included the presence of a repetitive gene (CAPS2), which was assigned as the gene most highly related to ASD risk. The merged gene subset that was selected by the GBPSO-SVM algorithm was able to enhance the classification accuracy.',\n",
              "  'authors': ['Hameed, Shilan S', 'Hassan, Rohayanti', 'Muhammad, Fahmi F'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Autistic Disorder',\n",
              "   'Gene Expression',\n",
              "   'Humans',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29095904',\n",
              "  'title': 'Selection and classification of gene expression in autism disorder: Use of a combination of statistical filters and a GBPSO-SVM algorithm.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-2199',\n",
              "  'IF_2013': '8.305',\n",
              "  'IF_2014': '8.665',\n",
              "  'IF_2015': '8.285',\n",
              "  'IF_2016': '6.559',\n",
              "  'IF_2017': '6.308',\n",
              "  'ISSN': '1047-3211',\n",
              "  'ISSN_online': '1460-2199',\n",
              "  'ISSN_print': '1047-3211',\n",
              "  'IsoAbbr': 'Cereb. Cortex',\n",
              "  'JrId': 2171,\n",
              "  'MedAbbr': 'Cereb Cortex',\n",
              "  'NLMid': '9110718',\n",
              "  'Titre': 'Cereb. Cortex',\n",
              "  'abstract': 'Neuropsychologists commonly use the Wisconsin Card Sorting Test as a test of the integrity of frontal lobe functions. However, an account of its range of validity and of the neuronal mechanisms involved is lacking. We analyze the test at 3 different levels. First, the different versions of the test are described, and the results obtained with normal subjects and brain-lesioned patients are reviewed. Second, a computational analysis is used to reveal what algorithms may pass the test, and to predict their respective performances. At this stage, 3 cognitive components are isolated that may critically contribute to performance: the ability to change the current rule when negative reward occurs, the capacity to memorize previously tested rules in order to avoid testing them twice, and the possibility of rejecting some rules a priori by reasoning. Third, a model neuronal network embodying these 3 components is described. The coding units are clusters of neurons organized in layers, or assemblies. A sensorimotor loop enables the network to sort the input cards according to several criteria (color, form, etc.). A higher-level assembly of rule-coding clusters codes for the currently tested rule, which shifts when negative reward is received. Internal testing of the possible rules, analogous to a reasoning process, also occurs, by means of an endogenous auto-evaluation loop. When lesioned, the model reproduces the behavior of frontal lobe patients. Plausible biological or molecular implementations are presented for several of its components.',\n",
              "  'authors': ['Dehaene, S', 'Changeux, J P'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Cerebral cortex (New York, N.Y. : 1991)',\n",
              "  'keywords': ['Frontal Lobe',\n",
              "   'Humans',\n",
              "   'Mental Processes',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuropsychological Tests'],\n",
              "  'pmid': '1822726',\n",
              "  'title': 'The Wisconsin Card Sorting Test: theoretical analysis and modeling in a neuronal network.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1539-2031',\n",
              "  'IF_2013': '3.186',\n",
              "  'IF_2014': '3.498',\n",
              "  'IF_2015': '3.163',\n",
              "  'IF_2016': '3.328',\n",
              "  'IF_2017': '2.968',\n",
              "  'ISSN': '0192-0790',\n",
              "  'ISSN_online': '1539-2031',\n",
              "  'ISSN_print': '0192-0790',\n",
              "  'IsoAbbr': 'J. Clin. Gastroenterol.',\n",
              "  'JrId': 4779,\n",
              "  'MedAbbr': 'J Clin Gastroenterol',\n",
              "  'NLMid': '7910017',\n",
              "  'Titre': 'J. Clin. Gastroenterol.',\n",
              "  'abstract': 'To evaluate whether resumption of warfarin after hospitalization for lower gastrointestinal bleeding (LGIB) is associated with improved 90-day and 6-month survival.|LGIB is a common complication for patients on warfarin. There is limited data to guide clinicians on the optimal management of warfarin following hospitalization for LGIB.|We identified patients hospitalized with LGIB while on warfarin using a validated, machine-learning algorithm. Patients were classified as those who had warfarin resumed at discharge and those who did not. Univariate and multivariate Cox proportional hazards were used to determine whether resuming warfarin was associated with improved 90-day and 6-month mortality.|In total, 607 patients were admitted with warfarin-associated LGIB. A total of 403 (66.4%) patients had warfarin held at discharge. Discontinuation of warfarin was associated with an increased 90-day and 6-month mortality on univariate analysis [hazard ratio (HR), 2.07, 95% confidence interval (CI), 1.04-4.58, P=0.04; HR, 1.78, 95% CI, 1.02-3.27, P=0.04]. On multivariate regression adjusting for age, comorbidities, and transfusion requirement, only a higher Charlson Index was associated with increased 90-day mortality (HR, 1.18, 95% CI, 1.07-1.29, P=<0.001). At 6 months, only older age was associated with an increased mortality on multivariate regression (HR, 1.02, 95% CI, 1.00-1.05, P=0.02), with no significantly increased mortality risk with holding warfarin (HR, 1.48, 95% CI, 0.84-2.78, P=0.18) CONCLUSIONS:: There was no association between resumption of warfarin at discharge following hospitalization for LGIB and either 90-day or 6-month mortality on multivariate analysis. Mortality in LGIB was largely driven by age and comorbidities.',\n",
              "  'authors': ['Patel, Parita', 'Nigam, Neha', 'Sengupta, Neil'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'Journal of clinical gastroenterology',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Aged',\n",
              "   'Anticoagulants',\n",
              "   'Comorbidity',\n",
              "   'Data Mining',\n",
              "   'Data Warehousing',\n",
              "   'Drug Administration Schedule',\n",
              "   'Female',\n",
              "   'Gastrointestinal Hemorrhage',\n",
              "   'Hospitalization',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Patient Discharge',\n",
              "   'Prognosis',\n",
              "   'Retrospective Studies',\n",
              "   'Risk Assessment',\n",
              "   'Risk Factors',\n",
              "   'Time Factors',\n",
              "   'Warfarin'],\n",
              "  'pmid': '28266939',\n",
              "  'title': 'Resumption of Warfarin After Hospitalization for Lower Gastrointestinal Bleeding and Mortality Benefits.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DY'],\n",
              "  'ESSN': '1520-6890',\n",
              "  'IF_2013': '45.661',\n",
              "  'IF_2014': '46.568',\n",
              "  'IF_2015': '37.369',\n",
              "  'IF_2016': '47.928',\n",
              "  'IF_2017': '52.613',\n",
              "  'ISSN': '0009-2665',\n",
              "  'ISSN_online': '1520-6890',\n",
              "  'ISSN_print': '0009-2665',\n",
              "  'IsoAbbr': 'Chem. Rev.',\n",
              "  'JrId': 2913,\n",
              "  'MedAbbr': 'Chem Rev',\n",
              "  'NLMid': '2985134R',\n",
              "  'Titre': 'Chem. Rev.',\n",
              "  'abstract': 'Artificial intelligence (AI), and, in particular, deep learning as a subcategory of AI, provides opportunities for the discovery and development of innovative drugs. Various machine learning approaches have recently (re)emerged, some of which may be considered instances of domain-specific AI which have been successfully employed for drug discovery and design. This review provides a comprehensive portrayal of these machine learning techniques and of their applications in medicinal chemistry. After introducing the basic principles, alongside some application notes, of the various machine learning algorithms, the current state-of-the art of AI-assisted pharmaceutical discovery is discussed, including applications in structure- and ligand-based virtual screening, de novo drug design, physicochemical and pharmacokinetic property prediction, drug repurposing, and related aspects. Finally, several challenges and limitations of the current methods are summarized, with a view to potential future directions for AI-assisted drug discovery and design.',\n",
              "  'authors': ['Yang, Xin',\n",
              "   'Wang, Yifei',\n",
              "   'Byrne, Ryan',\n",
              "   'Schneider, Gisbert',\n",
              "   'Yang, Shengyong'],\n",
              "  'categories': ['CHEMISTRY, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Chemical reviews',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31294972',\n",
              "  'title': 'Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1476-4954',\n",
              "  'IF_2013': '1.208',\n",
              "  'IF_2014': '1.367',\n",
              "  'IF_2015': '1.674',\n",
              "  'IF_2016': '1.826',\n",
              "  'IF_2017': '1.493',\n",
              "  'ISSN': '1476-7058',\n",
              "  'ISSN_online': '1476-4954',\n",
              "  'ISSN_print': '1476-7058',\n",
              "  'IsoAbbr': 'J. Matern. Fetal. Neonatal. Med.',\n",
              "  'JrId': 29848,\n",
              "  'MedAbbr': 'J Matern Fetal Neonatal Med',\n",
              "  'NLMid': '101136916',\n",
              "  'Titre': 'J. Matern. Fetal. Neonatal. Med.',\n",
              "  'abstract': 'A model construction for classification of women with normal, hypertensive and preeclamptic pregnancy in different gestational ages using maternal heart rate variability (HRV) indexes.|In the present work, we applied the artificial neural network for the classification problem, using the signal composed by the time intervals between consecutive RR peaks (RR) (n = 568) obtained from ECG records. Beside the HRV indexes, we also considered other factors like maternal history and blood pressure measurements.|The obtained result reveals sensitivity for preeclampsia around 80% that increases for hypertensive and normal pregnancy groups. On the other hand, specificity is around 85-90%. These results indicate that the combination of HRV indexes with artificial neural networks (ANN) could be helpful for pregnancy study and characterization.',\n",
              "  'authors': ['Tejera, Eduardo',\n",
              "   'Jose Areias, Maria',\n",
              "   'Rodrigues, Ana',\n",
              "   'Ramõa, Ana',\n",
              "   'Manuel Nieto-Villar, Jose',\n",
              "   'Rebelo, Irene'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'The journal of maternal-fetal & neonatal medicine : the official journal of the European Association of Perinatal Medicine, the Federation of Asia and Oceania Perinatal Societies, the International Society of Perinatal Obstetricians',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Artificial Intelligence',\n",
              "   'Classification',\n",
              "   'Computer Simulation',\n",
              "   'Electrocardiography',\n",
              "   'Female',\n",
              "   'Health Status Indicators',\n",
              "   'Heart Rate',\n",
              "   'Humans',\n",
              "   'Hypertension, Pregnancy-Induced',\n",
              "   'Individuality',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Observer Variation',\n",
              "   'Pre-Eclampsia',\n",
              "   'Pregnancy',\n",
              "   'Pregnancy Complications, Cardiovascular',\n",
              "   'Young Adult'],\n",
              "  'pmid': '21250912',\n",
              "  'title': 'Artificial neural network for normal, hypertensive, and preeclamptic pregnancy classification using maternal heart rate variability indexes.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1677-6119',\n",
              "  'IF_2013': '0.956',\n",
              "  'IF_2014': '0.875',\n",
              "  'IF_2015': '0.871',\n",
              "  'IF_2016': '0.815',\n",
              "  'IF_2017': '0.976',\n",
              "  'ISSN': '1677-5538',\n",
              "  'ISSN_online': '1677-6119',\n",
              "  'ISSN_print': '1677-5538',\n",
              "  'IsoAbbr': 'Int Braz J Urol',\n",
              "  'JrId': 32214,\n",
              "  'MedAbbr': 'Int Braz J Urol',\n",
              "  'NLMid': '101158091',\n",
              "  'Titre': 'Int Braz J Urol',\n",
              "  'abstract': \"Optical coherence tomography (OCT) is a unique technology, developed to provide high resolution, cross sectional images of human tissue. The objective of this study was to explore the feasibility of OCT for the evaluation of positive surgical margins and extra capsular extension in robotic prostatectomy specimens and compare it to histopathology.|Radical prostatectomy was performed in 100 patients. Twenty OCT images of each specimen were taken from the base of the seminal vesicles (SV), apical and vesicle margins, peripheral and posterolateral area and any palpable nodule. Predictions were made regarding positive surgical margin, SV involvement, capsular invasion and compared with the final histopathology.|A total of 2000 OCT images were taken and analyzed. Out of 100 specimens, 85 had T2 disease, 15 had T3 disease with a median Gleason's score of 7 (range 6 to 9) and 10 had positive surgical margins. We predicted 21 specimens to have positive margins based on OCT images out of which 7 were truly positive and 14 were falsely positive. Based on OCT images, 79 specimens were predicted to have negative margins out of which 76 were truly negative and 3 were falsely negative. We found the sensitivity, specificity, positive predictive value and negative predictive value to be 70%, 84%, 33% and 96% respectively.|Our initial feasibility study established the template for the visual OCT characteristics of the prostate, SV and cancerous tissue. The negative predictive value of evaluating surgical margins was high.\",\n",
              "  'authors': ['Dangle, Pankaj P',\n",
              "   'Shah, Ketul K',\n",
              "   'Kaffenberger, Benjamin',\n",
              "   'Patel, Vipul R'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'International braz j urol : official journal of the Brazilian Society of Urology',\n",
              "  'keywords': ['Feasibility Studies',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostate',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, Optical Coherence'],\n",
              "  'pmid': '19538770',\n",
              "  'title': 'The use of high resolution optical coherence tomography to evaluate robotic radical prostatectomy specimens.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': \"The abundance and unstructured nature of biomedical texts, be it clinical or research content, impose significant challenges for the effective and efficient use of information and knowledge stored in such texts. Annotation of biomedical documents with machine intelligible semantics facilitates advanced, semantics-based text management, curation, indexing, and search. This paper focuses on annotation of biomedical entity mentions with concepts from relevant biomedical knowledge bases such as UMLS. As a result, the meaning of those mentions is unambiguously and explicitly defined, and thus made readily available for automated processing. This process is widely known as semantic annotation, and the tools that perform it are known as semantic annotators.Over the last dozen years, the biomedical research community has invested significant efforts in the development of biomedical semantic annotation technology. Aiming to establish grounds for further developments in this area, we review a selected set of state of the art biomedical semantic annotators, focusing particularly on general purpose annotators, that is, semantic annotation tools that can be customized to work with texts from any area of biomedicine. We also examine potential directions for further improvements of today's annotators which could make them even more capable of meeting the needs of real-world applications. To motivate and encourage further developments in this area, along the suggested and/or related directions, we review existing and potential practical applications and benefits of semantic annotators.\",\n",
              "  'authors': ['Jovanović, Jelena', 'Bagheri, Ebrahim'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Biomedical Research',\n",
              "   'Electronic Health Records',\n",
              "   'Humans',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Semantics'],\n",
              "  'pmid': '28938912',\n",
              "  'title': 'Semantic annotation in biomedicine: the current landscape.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1827-1758',\n",
              "  'IF_2013': '0.7',\n",
              "  'IF_2014': '0.965',\n",
              "  'IF_2015': '0.536',\n",
              "  'IF_2016': '0.984',\n",
              "  'IF_2017': '1.449',\n",
              "  'ISSN': '0393-2249',\n",
              "  'ISSN_online': '1827-1758',\n",
              "  'ISSN_print': '0393-2249',\n",
              "  'IsoAbbr': 'Minerva Urol Nefrol',\n",
              "  'JrId': 7387,\n",
              "  'MedAbbr': 'Minerva Urol Nefrol',\n",
              "  'NLMid': '8503649',\n",
              "  'Titre': 'Minerva Urol Nefrol',\n",
              "  'abstract': 'This review deals with the preliminary advances in laparoendoscopic single-site surgery (LESS) as applied to prostate surgery including the simple and radical prostatectomy approaches both robot assisted and robot independent. It analyzed current publications based on animal models and human patients. The authors searched published reports in major urological meeting abstracts, Embase and Medline (1966 to 25 August 2008), with no language restrictions. Key word searches included: \"prostate\", \"prostatectomy\", \"radical\", \"surgery\", \"robot\", \"da Vinci\", \"scarless\", \"scar free\", \"single port/trocar/incision\", \"intraumbilical\", and \"transumbilical\", \"natural orifice transluminal endoscopic surgery\" (NOTES), \"SILS\", \"OPUS\" and \"LESS\". The role of LESS prostatectomy with or without robotic aid has been proven to be technically feasible; however, it is important to note that the approach has significant technical challenges. The da Vinci Surgical System allows some of these ergonomic challenges to be obviated with potentially reduced instrument clash, reduced surgeon and assistant fatigue and better precision with target tasking such as performing the vesicourethral anastomosis. Preliminary consensus regarding oncological control is not yet available on a large scale. Currently, no specific advantage of the LESS approach has been convincingly proven apart from the intuitive improvement in cosmesis. The development, and soon to be launched, flexible robotic platforms towards the end of 2010 will usher with it further refinements making the LESS approach to radical prostatectomy potentially more feasible ergonomically and could see the approach gain a more widespread acceptance.',\n",
              "  'authors': ['Kumar, P', 'Kommu, S S', 'Challacombe, B J', 'Dasgup-Ta, P'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Minerva urologica e nefrologica = The Italian journal of urology and nephrology',\n",
              "  'keywords': ['Animals',\n",
              "   'Evidence-Based Medicine',\n",
              "   'Feasibility Studies',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '20944542',\n",
              "  'title': 'Laparoendoscopic single-site surgery (LESS) prostatectomy--robotic and conventional approach.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Feature extraction for Acoustic Bird Species Classification (ABSC) tasks has traditionally been based on parametric representations that were specifically developed for speech signals, such as Mel Frequency Cepstral Coefficients (MFCC). However, the discrimination capabilities of these features for ABSC could be enhanced by accounting for the vocal production mechanisms of birds, and, in particular, the spectro-temporal structure of bird sounds. In this paper, a new front-end for ABSC is proposed that incorporates this specific information through the non-negative decomposition of bird sound spectrograms. It consists of the following two different stages: short-time feature extraction and temporal feature integration. In the first stage, which aims at providing a better spectral representation of bird sounds on a frame-by-frame basis, two methods are evaluated. In the first method, cepstral-like features (NMF_CC) are extracted by using a filter bank that is automatically learned by means of the application of Non-Negative Matrix Factorization (NMF) on bird audio spectrograms. In the second method, the features are directly derived from the activation coefficients of the spectrogram decomposition as performed through NMF (H_CC). The second stage summarizes the most relevant information contained in the short-time features by computing several statistical measures over long segments. The experiments show that the use of NMF_CC and H_CC in conjunction with temporal integration significantly improves the performance of a Support Vector Machine (SVM)-based ABSC system with respect to conventional MFCC.',\n",
              "  'authors': ['Ludeña-Choez, Jimmy',\n",
              "   'Quispe-Soncco, Raisa',\n",
              "   'Gallardo-Antolín, Ascensión'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Birds',\n",
              "   'Databases, Factual',\n",
              "   'Sound Spectrography',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '28628630',\n",
              "  'title': 'Bird sound spectrogram decomposition through Non-Negative Matrix Factorization for the acoustic classification of bird species.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': \"Multi-modal fMRI imaging has been used to study brain development such as the difference of functional connectivities (FCs) between different ages. Canonical correlation analysis (CCA) has been used to find correlations between multiple imaging modalities. However, it is unrelated to phenotypes. On the other hand, regression models can identify phenotype related imaging features but overlook the cross-modal data correlation. Collaborative regression (CR) is thus introduced to incorporate correlation as a penalization term into the regression model. Nevertheless, the complex relationship (e.g., nonlinear predictive relationship) between multiple data yet cannot be captured using linear CR models. To this end, we propose a novel method, deep collaborative learning (DCL), to address their limitations. DCL first uses a deep network to represent original data and then seeks their correlations, while also linking the data representation with phenotypical information. Therefore, DCL can better combine complex correlations between multiple data sets in addition to their fitting to phenotypes, with the potential to overcome the limitations of several current models. Based on DCL model, we study the difference of FCs between different age groups and also use FCs as a fingerprint to predict cognitive abilities. Our experiments demonstrated higher accuracy of using DCL over other conventional models when classifying populations of different ages and cognitive scores. Moreover, our experiments showed that different age or cognition groups may exhibit more significant differences of FCs in several networks than others. Furthermore, DCL revealed that brain connections became stronger at adolescence stage, demonstrating the importance of adolescence for brain development. In addition, DCL detected strong correlations between default mode network (DMN) and other networks which were overlooked by linear CCA, demonstrating DCL's ability of detecting nonlinear correlations.\",\n",
              "  'authors': ['Hu, Wenxing',\n",
              "   'Cai, Biao',\n",
              "   'Zhang, Aiying',\n",
              "   'Calhoun, Vince D',\n",
              "   'Wang, Yu-Ping'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30872216',\n",
              "  'title': 'Deep collaborative learning with application to multimodal brain development study.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['UI'],\n",
              "  'ESSN': '1079-7114',\n",
              "  'IF_2013': '7.728',\n",
              "  'IF_2014': '7.512',\n",
              "  'IF_2015': '7.645',\n",
              "  'IF_2016': '8.462',\n",
              "  'IF_2017': '8.839',\n",
              "  'ISSN': '0031-9007',\n",
              "  'ISSN_online': '1079-7114',\n",
              "  'ISSN_print': '0031-9007',\n",
              "  'IsoAbbr': 'Phys. Rev. Lett.',\n",
              "  'JrId': 21217,\n",
              "  'MedAbbr': 'Phys Rev Lett',\n",
              "  'NLMid': '0401141',\n",
              "  'Titre': 'Phys. Rev. Lett.',\n",
              "  'abstract': 'Efficient swimming at low Reynolds numbers is a major concern of microbots. To compare the efficiencies of different swimmers we introduce the notion of \"a swimming drag coefficient\" which allows for the ranking of swimmers. We find the optimal swimmer within a certain class of two-dimensional swimmers using conformal mapping techniques.',\n",
              "  'authors': ['Avron, J E', 'Gat, O', 'Kenneth, O'],\n",
              "  'categories': ['PHYSICS, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Physical review letters',\n",
              "  'keywords': ['Amoeba',\n",
              "   'Animals',\n",
              "   'Biomimetic Materials',\n",
              "   'Locomotion',\n",
              "   'Models, Theoretical',\n",
              "   'Robotics',\n",
              "   'Swimming'],\n",
              "  'pmid': '15525183',\n",
              "  'title': 'Optimal swimming at low Reynolds numbers.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1532-2548',\n",
              "  'IF_2013': '7.394',\n",
              "  'IF_2014': '6.841',\n",
              "  'IF_2015': '6.28',\n",
              "  'IF_2016': '6.456',\n",
              "  'IF_2017': '5.949',\n",
              "  'ISSN': '0032-0889',\n",
              "  'ISSN_online': '1532-2548',\n",
              "  'ISSN_print': '0032-0889',\n",
              "  'IsoAbbr': 'Plant Physiol.',\n",
              "  'JrId': 6478,\n",
              "  'MedAbbr': 'Plant Physiol',\n",
              "  'NLMid': '0401224',\n",
              "  'Titre': 'Plant Physiol.',\n",
              "  'abstract': 'Systemic acquired resistance (SAR) is a plant defense response that provides long-lasting, broad-spectrum pathogen resistance to uninfected systemic leaves following an initial localized infection. In Arabidopsis (Arabidopsis thaliana), local infection with virulent or avirulent strains of Pseudomonas syringae pv tomato generates long-distance SAR signals that travel from locally infected to distant leaves through the phloem to establish SAR In this study, a proteomics approach was used to identify proteins that accumulate in phloem exudates in response to the induction of SAR To accomplish this, phloem exudates collected from mock-inoculated or SAR-induced leaves of wild-type Columbia-0 plants were subjected to label-free quantitative liquid chromatography-tandem mass spectrometry proteomics. Comparing mock- and SAR-induced phloem exudate proteomes, 16 proteins were enriched in phloem exudates collected from SAR-induced plants, while 46 proteins were suppressed. SAR-related proteins THIOREDOXIN h3, ACYL-COENZYME A-BINDING PROTEIN6, and PATHOGENESIS-RELATED1 were enriched in phloem exudates of SAR-induced plants, demonstrating the strength of this approach and suggesting a role for these proteins in the phloem during SAR To identify novel components of SAR, transfer DNA mutants of differentially abundant phloem proteins were assayed for SAR competence. This analysis identified a number of new proteins (m-type thioredoxins, major latex protein-like protein, ULTRAVIOLET-B RESISTANCE8 photoreceptor) that contribute to the SAR response. The Arabidopsis SAR phloem proteome is a valuable resource for understanding SAR long-distance signaling and the dynamic nature of the phloem during plant-pathogen interactions.',\n",
              "  'authors': ['Carella, Philip',\n",
              "   'Merl-Pham, Juliane',\n",
              "   'Wilson, Daniel C',\n",
              "   'Dey, Sanjukta',\n",
              "   'Hauck, Stefanie M',\n",
              "   'Vlot, A Corina',\n",
              "   'Cameron, Robin K'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Plant physiology',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Arabidopsis Proteins',\n",
              "   'Gene Ontology',\n",
              "   'Immunity, Innate',\n",
              "   'Phloem',\n",
              "   'Plant Diseases',\n",
              "   'Plant Exudates',\n",
              "   'Proteome',\n",
              "   'Proteomics',\n",
              "   'Reproducibility of Results',\n",
              "   'Signal Transduction',\n",
              "   'Ultraviolet Rays'],\n",
              "  'pmid': '27208255',\n",
              "  'title': 'Comparative Proteomics Analysis of Phloem Exudates Collected during the Induction of Systemic Acquired Resistance.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'In this paper, the first real-time implementation and perceptual evaluation of a singular value decomposition (SVD)-based optimal filtering technique for noise reduction in a dual microphone behind-the-ear (BTE) hearing aid is presented. This evaluation was carried out for a speech weighted noise and multitalker babble, for single and multiple jammer sound source scenarios. Two basic microphone configurations in the hearing aid were used. The SVD-based optimal filtering technique was compared against an adaptive beamformer, which is known to give significant improvements in speech intelligibility in noisy environment. The optimal filtering technique works without assumptions about a speaker position, unlike the two-stage adaptive beamformer. However this strategy needs a robust voice activity detector (VAD). A method to improve the performance of the VAD was presented and evaluated physically. By connecting the VAD to the output of the noise reduction algorithms, a good discrimination between the speech-and-noise periods and the noise-only periods of the signals was obtained. The perceptual experiments demonstrated that the SVD-based optimal filtering technique could perform as well as the adaptive beamformer in a single noise source scenario, i.e., the ideal scenario for the latter technique, and could outperform the adaptive beamformer in multiple noise source scenarios.',\n",
              "  'authors': ['Maj, Jean-Baptiste',\n",
              "   'Royackers, Liesbeth',\n",
              "   'Moonen, Marc',\n",
              "   'Wouters, Jan'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Adult',\n",
              "   'Artificial Intelligence',\n",
              "   'Auditory Perception',\n",
              "   'Computer Systems',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Female',\n",
              "   'Hearing Aids',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Noise',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Sound Spectrography',\n",
              "   'Therapy, Computer-Assisted',\n",
              "   'Transducers'],\n",
              "  'pmid': '16189969',\n",
              "  'title': 'SVD-based optimal filtering for noise reduction in dual microphone hearing aids: a real time implementation and perceptual evaluation.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1471-2970',\n",
              "  'IF_2013': '6.314',\n",
              "  'IF_2014': '7.055',\n",
              "  'IF_2015': '5.847',\n",
              "  'IF_2016': '5.846',\n",
              "  'IF_2017': '5.666',\n",
              "  'ISSN': '0962-8436',\n",
              "  'ISSN_online': '1471-2970',\n",
              "  'ISSN_print': '0962-8436',\n",
              "  'IsoAbbr': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'JrId': 8277,\n",
              "  'MedAbbr': 'Philos Trans R Soc Lond B Biol Sci',\n",
              "  'NLMid': '7503623',\n",
              "  'Titre': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'abstract': 'Research on pattern perception and rule learning, grounded in formal language theory (FLT) and using artificial grammar learning paradigms, has exploded in the last decade. This approach marries empirical research conducted by neuroscientists, psychologists and ethologists with the theory of computation and FLT, developed by mathematicians, linguists and computer scientists over the last century. Of particular current interest are comparative extensions of this work to non-human animals, and neuroscientific investigations using brain imaging techniques. We provide a short introduction to the history of these fields, and to some of the dominant hypotheses, to help contextualize these ongoing research programmes, and finally briefly introduce the papers in the current issue.',\n",
              "  'authors': ['Fitch, W Tecumseh', 'Friederici, Angela D', 'Hagoort, Peter'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Philosophical transactions of the Royal Society of London. Series B, Biological sciences',\n",
              "  'keywords': ['Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Empirical Research',\n",
              "   'Language',\n",
              "   'Learning',\n",
              "   'Neuroimaging',\n",
              "   'Neurosciences',\n",
              "   'Pattern Recognition, Visual',\n",
              "   'Perception'],\n",
              "  'pmid': '22688630',\n",
              "  'title': 'Pattern perception and computational complexity: introduction to the special issue.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1098-1004',\n",
              "  'IF_2013': '5.05',\n",
              "  'IF_2014': '5.144',\n",
              "  'IF_2015': '5.089',\n",
              "  'IF_2016': '4.601',\n",
              "  'IF_2017': '5.359',\n",
              "  'ISSN': '1059-7794',\n",
              "  'ISSN_online': '1098-1004',\n",
              "  'ISSN_print': '1059-7794',\n",
              "  'IsoAbbr': 'Hum. Mutat.',\n",
              "  'JrId': 2408,\n",
              "  'MedAbbr': 'Hum Mutat',\n",
              "  'NLMid': '9215429',\n",
              "  'Titre': 'Hum. Mutat.',\n",
              "  'abstract': 'Explosive growth in the generation of genotype-to-phenotype (G2P) data necessitates a concerted effort to tackle the logistical and informatics challenges this presents. The GEN2PHEN Project represents one such effort, with a broad strategy of uniting disparate G2P resources into a hybrid centralized-federated network. This is achieved through a holistic strategy focussed on three overlapping areas: data input standards and pipelines through which to submit and collect data (data in); federated, independent, extendable, yet interoperable database platforms on which to store and curate widely diverse datasets (data storage); and data formats and mechanisms with which to exchange, combine, and extract data (data exchange and output). To fully leverage this data network, we have constructed the \"G2P Knowledge Centre\" (http://www.gen2phen.org). This central platform provides holistic searching of the G2P data domain allied with facilities for data annotation and user feedback, access to extensive G2P and informatics resources, and tools for constructing online working communities centered on the G2P domain. Through the efforts of GEN2PHEN, and through combining data with broader community-derived knowledge, the Knowledge Centre opens up exciting possibilities for organizing, integrating, sharing, and interpreting new waves of G2P data in a collaborative fashion.',\n",
              "  'authors': ['Webb, Adam J', 'Thorisson, Gudmundur A', 'Brookes, Anthony J'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Human mutation',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Genotype',\n",
              "   'Information Services',\n",
              "   'Knowledge Bases',\n",
              "   'Phenotype',\n",
              "   'Research'],\n",
              "  'pmid': '21438073',\n",
              "  'title': 'An informatics project and online \"Knowledge Centre\" supporting modern genotype-to-phenotype research.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In standard Independent Component Analysis (ICA), a linear data model is used for a global description of the data. Even though linear ICA yields meaningful results in many cases, it can provide a crude approximation only for general nonlinear data distributions. In this paper a new structure is proposed, where local ICA models are used in connection with a suitable grouping algorithm clustering the data. The clustering part is responsible for an overall coarse nonlinear representation of the data, while linear ICA models of each cluster are used for describing local features of the data. The goal is to represent the data better than in linear ICA while avoiding computational difficulties related with nonlinear ICA. Several data grouping methods are considered, including standard K-means clustering, self-organizing maps, and neural gas. Connections to existing methods are discussed, and experimental results are given for artificial data and natural images. Furthermore, a general theoretical framework encompassing a large number of methods for representing data is introduced. These range from global, dense representation methods to local, very sparse coding methods. The proposed local ICA methods lie between these two extremes.',\n",
              "  'authors': ['Karhunen, J', 'Mălăroiu, S', 'Ilmoniemi, M'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Cluster Analysis',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '11307858',\n",
              "  'title': 'Local linear independent component analysis based on clustering.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Flowering Chinese cabbage is a stalk vegetable whose quality and yield are directly related to stalk development. However, no comprehensive investigations on stalk development have been performed. To address this issue, the present study used RNA sequencing to investigate transcriptional regulation at three key stages (seedling, bolting, and flowering) of stalk development in flowering Chinese cabbage. Anatomical analysis revealed that cell division was the main mode of stalk thickening and elongation at all key stages. Among the 35,327 genes expressed in shoot apices, 34,448 were annotated and 879 were identified as novel transcripts. We identified 11,514 differentially expressed genes (DEGs) among the three stages of stalk development. Functional analysis revealed that these DEGs were significantly enriched in 'ribosome' and 'plant hormone signal transduction' pathways and were involved in hormone signal transduction, cell cycle progression, and the regulation of flowering time. The roles of these genes in stalk development were explored, and a putative gene-regulation network for the stalk flowering time was established. These findings provide insight into the molecular mechanisms of stalk development in flowering Chinese cabbage that provides a new theoretical basis for stalk vegetable breeding.\",\n",
              "  'authors': ['Huang, Xinmin',\n",
              "   'Lei, Yuling',\n",
              "   'Guan, Hongling',\n",
              "   'Hao, Yanwei',\n",
              "   'Liu, Houcheng',\n",
              "   'Sun, Guangwen',\n",
              "   'Chen, Riyuan',\n",
              "   'Song, Shiwei'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Brassica',\n",
              "   'Cell Division',\n",
              "   'China',\n",
              "   'Flowers',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Genes, Plant',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Plant Cells',\n",
              "   'Plant Growth Regulators',\n",
              "   'Plant Shoots',\n",
              "   'RNA, Messenger',\n",
              "   'Seedlings',\n",
              "   'Signal Transduction',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '29138433',\n",
              "  'title': 'Transcriptomic analysis of the regulation of stalk development in flowering Chinese cabbage (Brassica campestris) by RNA sequencing.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1097-0150',\n",
              "  'IF_2013': '1.077',\n",
              "  'IF_2014': '0.69',\n",
              "  'IF_2015': '0.606',\n",
              "  'IF_2016': '1.667',\n",
              "  'IF_2017': '1.2',\n",
              "  'ISSN': '1092-9088',\n",
              "  'ISSN_online': '1097-0150',\n",
              "  'ISSN_print': '1092-9088',\n",
              "  'IsoAbbr': 'Comput. Aided Surg.',\n",
              "  'JrId': 20599,\n",
              "  'MedAbbr': 'Comput Aided Surg',\n",
              "  'NLMid': '9708375',\n",
              "  'Titre': 'Comput. Aided Surg.',\n",
              "  'abstract': 'In radiosurgery, a moving beam of radiation acts as an ablative surgical instrument. Conventional systems for radiosurgery use a cylindrical radiation beam of fixed cross section. The radiation source can be moved only along simple, standardized paths. A new radiosurgical system based on a six-degree-of-freedom robotic arm has been developed to overcome limitations of conventional systems. We address the following question: Can dose distributions generated by robotic radiosurgery be improved by using noncylindrical radiation beams? This includes static noncylindrical collimators and collimators of adaptable cross section. Geometric methods for planning the shape of the beam in addition to planning beam motion are developed. Design criteria considered in this context are treatment time, radiation penumbra, and transparency of interactive treatment planning. An experimental evaluation compares distributions generated with our new radiosurgical system using cylindrical beams to distributions generated with beams of adaptable, noncylindrical shapes.',\n",
              "  'authors': ['Schweikard, A', 'Adler, J R'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer aided surgery : official journal of the International Society for Computer Aided Surgery',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Intracranial Arteriovenous Malformations',\n",
              "   'Online Systems',\n",
              "   'Particle Accelerators',\n",
              "   'Patient Care Planning',\n",
              "   'Radiation Dosage',\n",
              "   'Radiopharmaceuticals',\n",
              "   'Radiosurgery',\n",
              "   'Robotics',\n",
              "   'Therapy, Computer-Assisted',\n",
              "   'Time Factors'],\n",
              "  'pmid': '9292264',\n",
              "  'title': 'Robotic radiosurgery with noncylindrical collimators.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2375-2548',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '11.511',\n",
              "  'IF_2016': '11.511',\n",
              "  'IF_2017': '11.511',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2375-2548',\n",
              "  'IsoAbbr': 'Sci Adv',\n",
              "  'JrId': 43628,\n",
              "  'MedAbbr': 'Sci Adv',\n",
              "  'NLMid': '101653440',\n",
              "  'Titre': 'Sci Adv',\n",
              "  'abstract': 'Manley et al. (Science Advances, 16 September 2016, p. e1501814) report the splitting of a transverse acoustic phonon branch below TC in the relaxor ferroelectric Pb[(Mg1/3Nb2/3)1-x Ti x ]O3 with x = 0.30 using neutron scattering methods. Manley et al. argue that this splitting occurs because these phonons hybridize with local, harmonic lattice vibrations associated with polar nanoregions. We show that splitting is absent when the measurement is made using a different neutron wavelength, and we suggest an alternative interpretation.',\n",
              "  'authors': ['Gehring, P M',\n",
              "   'Xu, Zhijun',\n",
              "   'Stock, C',\n",
              "   'Xu, Guangyong',\n",
              "   'Parshall, D',\n",
              "   'Harriger, L',\n",
              "   'Gehring, C A',\n",
              "   'Li, Xiaobing',\n",
              "   'Luo, Haosu'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Science advances',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30915391',\n",
              "  'title': 'Comment on \"Giant electromechanical coupling of relaxor ferroelectrics controlled by polar nanoregion vibrations\".',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['WF'],\n",
              "  'ESSN': '1529-7268',\n",
              "  'IF_2013': '3.451',\n",
              "  'IF_2014': '3.318',\n",
              "  'IF_2015': '3.471',\n",
              "  'IF_2016': '3.432',\n",
              "  'IF_2017': '3.184',\n",
              "  'ISSN': '0006-3363',\n",
              "  'ISSN_online': '1529-7268',\n",
              "  'ISSN_print': '0006-3363',\n",
              "  'IsoAbbr': 'Biol. Reprod.',\n",
              "  'JrId': 1121,\n",
              "  'MedAbbr': 'Biol Reprod',\n",
              "  'NLMid': '0207224',\n",
              "  'Titre': 'Biol. Reprod.',\n",
              "  'abstract': 'Mammalian spermatozoa acquire fertilizing potential as they undergo a series of changes during epididymal transit. One major facet of such is the alterations in the sperm glycome. Modifications of the sialic acid content within glycan moieties are known to regulate epitope presentation and cellular adhesion and signaling, all of which may be critical for sperm to successfully reach and fertilize the egg. To date, there is paucity of information regarding the sialic acid changes that occur on spermatozoa during epididymal transit. Therefore, the aim of this study was to identify N-linked sialylated glycoproteins in rat epididymal sperm and investigate whether they are regulated during epididymal transit. Sialylated glycopeptides from caput, corpus, and cauda spermatozoa were enriched using titanium dioxide beads. Bound N-linked glycopeptides were released by enzymatic deglycosylation using PNGase F and then analyzed by liquid chromatography tandem-mass spectrometry. A total of 92 unique N-linked sialylated glycopeptides were identified from 65 different proteins. These included members of the disintegrin and metalloproteinase domain-containing protein family (ADAM), Basigin, and Testis-expressed protein 101 (TEX101). Remarkably, label-free quantification showed that more than half of these peptides (48/92) were regulated during epididymal transit. Of interest, the protein TEX101 exhibited PNGase F-resistant deglycosylation under the conditions used in this study. The results from this study showed that changes in the N-linked sialoglycoprotein profile is a major hallmark of sperm maturation in rats. ',\n",
              "  'authors': ['Villaverde, Ana Izabel Silva Balbin',\n",
              "   'Hetherington, Louise',\n",
              "   'Baker, Mark A'],\n",
              "  'categories': ['REPRODUCTIVE BIOLOGY'],\n",
              "  'journal': 'Biology of reproduction',\n",
              "  'keywords': ['ADAM Proteins',\n",
              "   'Animals',\n",
              "   'Chromatography, Affinity',\n",
              "   'Epididymis',\n",
              "   'Gene Ontology',\n",
              "   'Glycopeptides',\n",
              "   'Male',\n",
              "   'Membrane Glycoproteins',\n",
              "   'Membrane Proteins',\n",
              "   'Peptide-N4-(N-acetyl-beta-glucosaminyl) Asparagine Amidase',\n",
              "   'Proteome',\n",
              "   'Rats',\n",
              "   'Rats, Wistar',\n",
              "   'Spermatozoa'],\n",
              "  'pmid': '26962120',\n",
              "  'title': 'Quantitative Glycopeptide Changes in Rat Sperm During Epididymal Transit.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1873-5894',\n",
              "  'IF_2013': '2.022',\n",
              "  'IF_2014': '2.09',\n",
              "  'IF_2015': '1.98',\n",
              "  'IF_2016': '2.225',\n",
              "  'IF_2017': '2.564',\n",
              "  'ISSN': '0730-725X',\n",
              "  'ISSN_online': '1873-5894',\n",
              "  'ISSN_print': '0730-725X',\n",
              "  'IsoAbbr': 'Magn Reson Imaging',\n",
              "  'JrId': 5670,\n",
              "  'MedAbbr': 'Magn Reson Imaging',\n",
              "  'NLMid': '8214883',\n",
              "  'Titre': 'Magn Reson Imaging',\n",
              "  'abstract': 'Temporal clustering analysis (TCA) has been proposed as a method for detecting the brain responses of a functional magnetic resonance imaging (fMRI) time series when the time and location of activation are completely unknown. But TCA is not suitable for treating the time series of the whole brain due to the existence of many inactive pixels. In theory, active pixels are located only in gray matter (GM). In this study, SPM2 was used to segment functional images into GM, white matter and cerebrospinal fluid, and only the pixels in GM were considered. Thus, most of inactive pixels are deleted, so that the sensitivity of TCA is greatly improved in the analysis of the whole brain. The same set of acupuncture fMRI data was treated using both conventional TCA and modified TCA (MTCA) for comparing their analytical ability. The results clearly show a significant improvement in the sensitivity achieved by MTCA.',\n",
              "  'authors': ['Lu, Na',\n",
              "   'Shan, Bao-Ci',\n",
              "   'Xu, Jian-Yang',\n",
              "   'Wang, Wei',\n",
              "   'Li, Kun-Cheng'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Magnetic resonance imaging',\n",
              "  'keywords': ['Acupuncture Therapy',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Cluster Analysis',\n",
              "   'Evoked Potentials',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '17451902',\n",
              "  'title': 'Improved temporal clustering analysis method applied to whole-brain data in acupuncture fMRI study.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1745-2422',\n",
              "  'IF_2013': '1.784',\n",
              "  'IF_2014': '1.683',\n",
              "  'IF_2015': '1.762',\n",
              "  'IF_2016': '2.228',\n",
              "  'IF_2017': '2.094',\n",
              "  'ISSN': '1743-4440',\n",
              "  'ISSN_online': '1745-2422',\n",
              "  'ISSN_print': '1743-4440',\n",
              "  'IsoAbbr': 'Expert Rev Med Devices',\n",
              "  'JrId': 32622,\n",
              "  'MedAbbr': 'Expert Rev Med Devices',\n",
              "  'NLMid': '101230445',\n",
              "  'Titre': 'Expert Rev Med Devices',\n",
              "  'abstract': \"Positive surgical margins represents incomplete resection by the surgeon, and the elimination of positive margins represents the only clinical feature during radical prostatectomy that can lead directly to improved cancer outcomes. The introduction of new robot-assisted technology and technical refinements has led to declines of positive surgical margins. Although margins induced by incomplete cancer resection by the surgeon have been reduced for organ-confined disease, the 'Holy Grail' of zero margins is not yet attainable in prostatectomy, and is more problematic in cancer that has penetrated beyond the prostate. Intraoperative frozen biopsies are imprecise. The union of real-time optical coherence tomography technology of the da Vinci robotic platform for identification of positive margin sites, and technical advances with wider excisions during surgery may provide promise for further reduction of surgical margins to zero.\",\n",
              "  'authors': ['Skarecky, Douglas W',\n",
              "   'Brenner, Mattew',\n",
              "   'Rajan, Sudhir',\n",
              "   'Rodriguez, Esequiel',\n",
              "   'Narula, Navneet',\n",
              "   'Melgoza, Frank',\n",
              "   'Ahlering, Thomas E'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Expert review of medical devices',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Neoplasm Invasiveness',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Neoplasm Staging',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Tomography, Optical Coherence',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '19093313',\n",
              "  'title': 'Zero positive surgical margins after radical prostatectomy: is the end in sight.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '2159-8290',\n",
              "  'IF_2013': '15.929',\n",
              "  'IF_2014': '19.453',\n",
              "  'IF_2015': '19.783',\n",
              "  'IF_2016': '20.011',\n",
              "  'IF_2017': '24.373',\n",
              "  'ISSN': '2159-8274',\n",
              "  'ISSN_online': '2159-8290',\n",
              "  'ISSN_print': '2159-8274',\n",
              "  'IsoAbbr': 'Cancer Discov',\n",
              "  'JrId': 39259,\n",
              "  'MedAbbr': 'Cancer Discov',\n",
              "  'NLMid': '101561693',\n",
              "  'Titre': 'Cancer Discov',\n",
              "  'abstract': 'A new machine-learning algorithm that recognizes patterns in genome-wide DNA-methylation data can help pathologists classify brain and spinal cord tumors. The diagnostic tool is freely available online, and could prove valuable for histologically ambiguous cases.',\n",
              "  'authors': [''],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Cancer discovery',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Central Nervous System Neoplasms',\n",
              "   'DNA Methylation',\n",
              "   'Humans',\n",
              "   'Machine Learning'],\n",
              "  'pmid': '29602827',\n",
              "  'title': 'Machine Learning Improves Diagnosis of CNS Cancers.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1432-0711',\n",
              "  'IF_2013': '1.279',\n",
              "  'IF_2014': '1.364',\n",
              "  'IF_2015': '1.68',\n",
              "  'IF_2016': '2.09',\n",
              "  'IF_2017': '2.236',\n",
              "  'ISSN': '0932-0067',\n",
              "  'ISSN_online': '1432-0711',\n",
              "  'ISSN_print': '0932-0067',\n",
              "  'IsoAbbr': 'Arch. Gynecol. Obstet.',\n",
              "  'JrId': 735,\n",
              "  'MedAbbr': 'Arch Gynecol Obstet',\n",
              "  'NLMid': '8710213',\n",
              "  'Titre': 'Arch. Gynecol. Obstet.',\n",
              "  'abstract': 'Total laparoscopic or robotic hysterectomy represents one of the most performed gynecological procedures nowadays. Minimally invasive procedures seem to increase the risk of vaginal cuff dehiscence (VCD). Barbed suture is a new class of suture introduced to aid surgeons during laparoscopic suturing, with the aim to reduce operative time, blood loss, and vaginal dehiscence.|We identified all articles that report a series of laparoscopic or robotic-assisted hysterectomy using barbed suture compared to conventional suture for vaginal cuff closure. The main outcome measures were vaginal cuff suturing time, vaginal bleeding, and vaginal dehiscence with or without small bowel evisceration. Suturing time was meta-analyzed as the standardized mean difference, which is the difference in means of two arms divided by pooled standard deviation with 95 % confidence intervals. For vaginal bleeding and dehiscence risk difference were calculated for each study and then meta-analyzed. Fixed models were considered if heterogeneity was low (I (2) < 50 %), otherwise random models were preferred.|We show that minor bleeding (RD = 0, 95 % CI 0.03-0.03; p value = 0.907) and VCD (RD = -0.01, 95 % CI 0.02-0.00; p value = 0.119) are comparable in minimally invasive hysterectomy with or without the use of barbed suture. The major bleeding (RD = -0.03; 95 % CI 0.05-0.00; p value = 0.047) appears borderline significant, but the difference between the two types of sutures is not high and the upper limit of 95 % IC is equal to 0 so they were considered comparable. Instead, meta-analysis on vaginal cuff suturing time demonstrates that suturing time is reduced with the use of barbed suture (SMD = -0.96, 95 % CI 1.26-0.70; p value <0.001).|Barbed suture is safe and well tolerated as traditional sutures and is associated with reduced operative time of laparoscopic vaginal vault closure.',\n",
              "  'authors': ['Bogliolo, Stefano',\n",
              "   'Musacchi, Valentina',\n",
              "   'Dominoni, Mattia',\n",
              "   'Cassani, Chiara',\n",
              "   'Gaggero, Chiara Roberta',\n",
              "   'De Silvestri, Annalisa',\n",
              "   'Gardella, Barbara',\n",
              "   'Spinillo, Arsenio'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Archives of gynecology and obstetrics',\n",
              "  'keywords': ['Blood Loss, Surgical',\n",
              "   'Colpotomy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Operative Time',\n",
              "   'Pregnancy',\n",
              "   'Robotics',\n",
              "   'Suture Techniques',\n",
              "   'Sutures',\n",
              "   'Treatment Outcome',\n",
              "   'Uterine Hemorrhage',\n",
              "   'Vagina'],\n",
              "  'pmid': '25700658',\n",
              "  'title': 'Barbed suture in minimally invasive hysterectomy: a systematic review and meta-analysis.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': 'Biology is experiencing a gradual but fast transformation from a laboratory-centred science towards a data-centred one. As such, it requires robust data engineering and the use of quantitative data analysis methods as part of database curation. This paper focuses on G protein-coupled receptors, a large and heterogeneous super-family of cell membrane proteins of interest to biology in general. One of its families, Class C, is of particular interest to pharmacology and drug design. This family is quite heterogeneous on its own, and the discrimination of its several sub-families is a challenging problem. In the absence of known crystal structure, such discrimination must rely on their primary amino acid sequences.|We are interested not as much in achieving maximum sub-family discrimination accuracy using quantitative methods, but in exploring sequence misclassification behavior. Specifically, we are interested in isolating those sequences showing consistent misclassification, that is, sequences that are very often misclassified and almost always to the same wrong sub-family. Random forests are used for this analysis due to their ensemble nature, which makes them naturally suited to gauge the consistency of misclassification. This consistency is here defined through the voting scheme of their base tree classifiers.|Detailed consistency results for the random forest ensemble classification were obtained for all receptors and for all data transformations of their unaligned primary sequences. Shortlists of the most consistently misclassified receptors for each subfamily and transformation, as well as an overall shortlist including those cases that were consistently misclassified across transformations, were obtained. The latter should be referred to experts for further investigation as a data curation task.|The automatic discrimination of the Class C sub-families of G protein-coupled receptors from their unaligned primary sequences shows clear limits. This study has investigated in some detail the consistency of their misclassification using random forest ensemble classifiers. Different sub-families have been shown to display very different discrimination consistency behaviors. The individual identification of consistently misclassified sequences should provide a tool for quality control to GPCR database curators.',\n",
              "  'authors': ['Shkurin, Aleksei', 'Vellido, Alfredo'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'Machine Learning',\n",
              "   'Receptors, G-Protein-Coupled'],\n",
              "  'pmid': '28830426',\n",
              "  'title': 'Using random forests for assistance in the curation of G-protein coupled receptor databases.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1440-1800',\n",
              "  'IF_2013': '1.046',\n",
              "  'IF_2014': '1.439',\n",
              "  'IF_2015': '1.141',\n",
              "  'IF_2016': '1.143',\n",
              "  'IF_2017': '1.159',\n",
              "  'ISSN': '1320-7881',\n",
              "  'ISSN_online': '1440-1800',\n",
              "  'ISSN_print': '1320-7881',\n",
              "  'IsoAbbr': 'Nurs Inq',\n",
              "  'JrId': 20129,\n",
              "  'MedAbbr': 'Nurs Inq',\n",
              "  'NLMid': '9505881',\n",
              "  'Titre': 'Nurs Inq',\n",
              "  'abstract': \"The Smart Home designed to extend older adults independence is emerging as a clinical solution to the growing ageing population. Nurses will and should play a key role in the development and application of Smart Home technology. Accordingly, conceptual frameworks are needed for nurse scientists who are collaborating with multidisciplinary research teams in developing an intelligent Smart Home that assists with managing older adults' health. We present a conceptual framework that is grounded in critical realism and pragmatism, informing a unique mixed methodological approach to generating, analyzing, and contextualizing sensor data for clinician-based machine learning. This framework can guide nurse scientists in knowledge construction as they participate in multidisciplinary health-assistive Smart Home and artificial intelligence research. In this paper, we review philosophical underpinnings and explicate how this framework can guide nurse scientists collaborating with engineers to develop intelligent health-assistive Smart Homes. It is critical that clinical nursing knowledge is integrated into Smart Home and artificial intelligence features. A conceptual framework and practical method will provide needed structure for knowledge construction by nurse scientists.\",\n",
              "  'authors': ['Dermody, Gordana', 'Fritz, Roschelle'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Nursing inquiry',\n",
              "  'keywords': ['Artificial Intelligence', 'Home Care Services', 'Humans'],\n",
              "  'pmid': '30417510',\n",
              "  'title': 'A conceptual framework for clinicians working with artificial intelligence and health-assistive Smart Homes.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In this paper, we review recent advances in blind source separation (BSS) and independent component analysis (ICA) for nonlinear mixing models. After a general introduction to BSS and ICA, we discuss in more detail uniqueness and separability issues, presenting some new results. A fundamental difficulty in the nonlinear BSS problem and even more so in the nonlinear ICA problem is that they provide non-unique solutions without extra constraints, which are often implemented by using a suitable regularization. In this paper, we explore two possible approaches. The first one is based on structural constraints. Especially, post-nonlinear mixtures are an important special case, where a nonlinearity is applied to linear mixtures. For such mixtures, the ambiguities are essentially the same as for the linear ICA or BSS problems. The second approach uses Bayesian inference methods for estimating the best statistical parameters, under almost unconstrained models in which priors can be easily added. In the later part of this paper, various separation techniques proposed for post-nonlinear mixtures and general nonlinear mixtures are reviewed.',\n",
              "  'authors': ['Jutten, Christian', 'Karhunen, Juha'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Bayes Theorem',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Principal Component Analysis',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '15593377',\n",
              "  'title': 'Advances in blind source separation (BSS) and independent component analysis (ICA) for nonlinear mixtures.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'We propose a novel method for movement assistance that is based on adaptive oscillators, i.e., mathematical tools that are capable of extracting the high-level features (amplitude, frequency, and offset) of a periodic signal. Such an oscillator acts like a filter on these features, but keeps its output in phase with respect to the input signal. Using a simple inverse model, we predicted the torque produced by human participants during rhythmic flexion-extension of the elbow. Feeding back a fraction of this estimated torque to the participant through an elbow exoskeleton, we were able to prove the assistance efficiency through a marked decrease of the biceps and triceps electromyography. Importantly, since the oscillator adapted to the movement imposed by the user, the method flexibly allowed us to change the movement pattern and was still efficient during the nonstationary epochs. This method holds promise for the development of new robot-assisted rehabilitation protocols because it does not require prespecifying a reference trajectory and does not require complex signal sensing or single-user calibration: the only signal that is measured is the position of the augmented joint. In this paper, we further demonstrate that this assistance was very intuitive for the participants who adapted almost instantaneously.',\n",
              "  'authors': ['Ronsse, Renaud',\n",
              "   'Vitiello, Nicola',\n",
              "   'Lenzi, Tommaso',\n",
              "   'van den Kieboom, Jesse',\n",
              "   'Carrozza, Maria Chiara',\n",
              "   'Ijspeert, Auke Jan'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Arm',\n",
              "   'Biological Clocks',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Man-Machine Systems',\n",
              "   'Models, Biological',\n",
              "   'Robotics',\n",
              "   'Therapy, Computer-Assisted'],\n",
              "  'pmid': '20977981',\n",
              "  'title': 'Human-robot synchrony: flexible assistance using adaptive oscillators.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'HeartLander is a miniature mobile robot designed to navigate over the epicardium of the beating heart for minimally invasive therapy. This paper presents a technique to decrease slippage and improve locomotion efficiency by synchronizing the locomotion with the intrapericardial pressure variations of the respiration and heartbeat cycles.|Respiratory and heartbeat phases were detected in real time using a chest-mounted accelerometer during locomotion in a porcine model in vivo. Trials were conducted over the lateral aspect of the heart surface to test synchronized locomotion against an unsynchronized control.|Offline evaluation showed that the respiration and heartbeat algorithms had accuracies of 100% and 88%, respectively. Synchronized trials exhibited significantly lower friction, higher efficiency, and greater total distance traveled than control trials.|Synchronization of the locomotion of HeartLander with respiration and heartbeat is feasible and results in safer and more efficient travel on the beating heart.',\n",
              "  'authors': ['Patronik, Nicholas A',\n",
              "   'Ota, Takeyoshi',\n",
              "   'Zenati, Marco A',\n",
              "   'Riviere, Cameron N'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Acceleration',\n",
              "   'Algorithms',\n",
              "   'Animals',\n",
              "   'Equipment Design',\n",
              "   'Heart',\n",
              "   'Heart Rate',\n",
              "   'Humans',\n",
              "   'Locomotion',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Pericardium',\n",
              "   'Reproducibility of Results',\n",
              "   'Respiration',\n",
              "   'Robotics',\n",
              "   'Swine'],\n",
              "  'pmid': '22009934',\n",
              "  'title': 'Synchronization of epicardial crawling robot with heartbeat and respiration for improved safety and efficiency of locomotion.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IA'],\n",
              "  'ESSN': '1557-8593',\n",
              "  'IF_2013': '2.293',\n",
              "  'IF_2014': '2.106',\n",
              "  'IF_2015': '2.198',\n",
              "  'IF_2016': '2.698',\n",
              "  'IF_2017': '2.921',\n",
              "  'ISSN': '1520-9156',\n",
              "  'ISSN_online': '1557-8593',\n",
              "  'ISSN_print': '1520-9156',\n",
              "  'IsoAbbr': 'Diabetes Technol. Ther.',\n",
              "  'JrId': 21552,\n",
              "  'MedAbbr': 'Diabetes Technol Ther',\n",
              "  'NLMid': '100889084',\n",
              "  'Titre': 'Diabetes Technol. Ther.',\n",
              "  'abstract': 'The purpose of this two-part review is to overview research use of the AIDA diabetes software simulator. AIDA is a diabetes computer program that permits the interactive simulation of plasma insulin and blood glucose profiles for teaching, demonstration, and self-learning purposes. It has been made freely available, without charge, on the Internet as a noncommercial contribution to continuing diabetes education. Since its launch in 1996 over 300,000 visits have been logged at the main AIDA Website-www.2aida.org-and over 60,000 copies of the AIDA program have been downloaded free-of-charge. This review describes research projects and ventures, undertaken for the most part by other research workers in the diabetes computing field, that have made use of the freeware AIDA software. Relevant research work was identified in three main ways: (i) by personal (e-mail/written) communications from researchers, (ii) via the ISI Web of Science citation database to identify published articles that referred to AIDA-related papers, and (iii) via searches on the Internet. In a number of cases research students who had sought advice about AIDA, and diabetes computing in general, provided copies of their research dissertations/theses upon the completion of their projects. The two reviews highlight some of the many and varied research projects that have made use of the AIDA diabetes simulation software to date. A wide variety of diabetes computing topics have been addressed. In Part 1 of the review, these range from testing decision support prototypes to training artificial neural networks. In Part 2 of the review, issues surrounding dietary assessments, developing new diabetes models, and performance monitoring of closed-loop insulin delivery devices are considered. Overall, research projects making use of AIDA have been identified in Australia, Italy, South Korea, the United Kingdom, and the United States. These reviews confirm an unexpected but useful benefit of distributing medical software, like AIDA, for free via the Internet-demonstrating how it is possible to have a synergistic benefit with other researchers-facilitating their own research projects in related medical fields. The reviews highlight a variety of these projects that have benefited from the free availability of the AIDA diabetes software simulator. In a number of cases these other research projects simply would not have been possible without unrestricted access to the AIDA software and/or technical descriptions of its workings. In addition, some specific common themes begin to emerge from the research ventures that have been reviewed. These include the use of simulated blood glucose data from the AIDA program for preliminary computerlab-based testing of other decision support prototypes. Issues surrounding such use of simulated data for separate prototype testing are discussed further in Part 2 of the review.',\n",
              "  'authors': ['Lehmann, Eldon D'],\n",
              "  'categories': ['ENDOCRINOLOGY & METABOLISM'],\n",
              "  'journal': 'Diabetes technology & therapeutics',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Diabetes Mellitus',\n",
              "   'Environmental Monitoring',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Materials Testing',\n",
              "   'Nerve Net',\n",
              "   'Patient Education as Topic',\n",
              "   'Software'],\n",
              "  'pmid': '12828827',\n",
              "  'title': 'Research use of the AIDA www.2aida.org diabetes software simulation program: a review-part 1. decision support testing and neural network training.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1879-0038',\n",
              "  'IF_2013': '2.082',\n",
              "  'IF_2014': '2.138',\n",
              "  'IF_2015': '2.319',\n",
              "  'IF_2016': '2.415',\n",
              "  'IF_2017': '2.498',\n",
              "  'ISSN': '0378-1119',\n",
              "  'ISSN_online': '1879-0038',\n",
              "  'ISSN_print': '0378-1119',\n",
              "  'IsoAbbr': 'Gene',\n",
              "  'JrId': 3886,\n",
              "  'MedAbbr': 'Gene',\n",
              "  'NLMid': '7706761',\n",
              "  'Titre': 'Gene',\n",
              "  'abstract': 'The maternal genome directs almost all aspects of early animal development. As development proceeds, the elimination of maternal gene products and zygotic genome activation (ZGA) occur during the maternal to zygotic transition (MZT). To study the molecular mechanisms regulating this developmental event in Eriocheir sinensis, RNA-Seq technology was applied to generate comprehensive information on transcriptome dynamics during early embryonic stages. In total, 32,088 annotated unigenes were obtained from the transcriptomes of fertilized eggs and embryos at the cleavage (2-4 cell) and blastula stage. A total of 566 maternal genes and 1165 zygotic genes were isolated, among which 103 and 266 genes were predicted conserved maternal transcripts (COMATs) and conserved zygotic transcripts (COZYTs), respectively. The COMATs performed housekeeping gene functions and may be essential for initiating early embryogenesis of the Bilateria. Furthermore, 87, 76 and 117 differentially expressed genes associated with the MZT, morphogenesis and immunity were identified when compared the three transcriptomic datasets. We also unmask that the MZT takes place around the cleavage stage, when the genes involved in the clearance of maternal gene products and the ZGA were significantly up-regulated. Taken together, these datasets provide a valuable resource for understanding the mechanisms of early developmental events in E. sinensis, and facilitate further studies on molecular mechanisms of asynchronous development in crabs.',\n",
              "  'authors': ['Ning, Junhao',\n",
              "   'Song, Chengwen',\n",
              "   'Luo, Danli',\n",
              "   'Liu, Yuan',\n",
              "   'Liu, Hourong',\n",
              "   'Cui, Zhaoxia'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Gene',\n",
              "  'keywords': ['Animals',\n",
              "   'Brachyura',\n",
              "   'Computational Biology',\n",
              "   'Embryonic Development',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Ontology',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Immunity',\n",
              "   'Immunity, Innate',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Reproducibility of Results',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '30321661',\n",
              "  'title': 'Comparative transcriptome analysis explores maternal to zygotic transition during Eriocheir sinensis early embryogenesis.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': \"Multirobot task allocation is a hot issue in the field of robot research. A new emotional model is used with the self-interested robot, which gives a new way to measure self-interested robots' individual cooperative willingness in the problem of multirobot task allocation. Emotional cooperation factor is introduced into self-interested robot; it is updated based on emotional attenuation and external stimuli. Then a multirobot pursuit task allocation algorithm is proposed, which is based on emotional cooperation factor. Combined with the two-step auction algorithm recruiting team leaders and team collaborators, set up pursuit teams, and finally use certain strategies to complete the pursuit task. In order to verify the effectiveness of this algorithm, some comparing experiments have been done with the instantaneous greedy optimal auction algorithm; the results of experiments show that the total pursuit time and total team revenue can be optimized by using this algorithm. \",\n",
              "  'authors': ['Fang, Baofu',\n",
              "   'Chen, Lu',\n",
              "   'Wang, Hao',\n",
              "   'Dai, Shuanglu',\n",
              "   'Zhong, Qiubo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Algorithms', 'Models, Theoretical', 'Robotics'],\n",
              "  'pmid': '25152925',\n",
              "  'title': 'Research on multirobot pursuit task allocation algorithm based on emotional cooperation factor.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1873-4863',\n",
              "  'IF_2013': '2.884',\n",
              "  'IF_2014': '2.871',\n",
              "  'IF_2015': '2.667',\n",
              "  'IF_2016': '2.599',\n",
              "  'IF_2017': '2.533',\n",
              "  'ISSN': '0168-1656',\n",
              "  'ISSN_online': '1873-4863',\n",
              "  'ISSN_print': '0168-1656',\n",
              "  'IsoAbbr': 'J. Biotechnol.',\n",
              "  'JrId': 1552,\n",
              "  'MedAbbr': 'J Biotechnol',\n",
              "  'NLMid': '8411927',\n",
              "  'Titre': 'J. Biotechnol.',\n",
              "  'abstract': 'Experimental evidence suggests that the cell membrane is a highly organized structure that is compartmentalized by the underlying membrane cytoskeleton (MSK). The interaction between the cell membrane and the cytoskeleton led to the \"picket-fence\" model, which was proposed to explain certain aspects of membrane compartmentalization. This model assumes that the MSK hinders and confines the motion of receptors and lipids to compartments in the membrane. However, the impact of the MSK on receptor clustering, aggregation, and downstream signaling remains unclear. For example, some evidence suggests that the MSK enhances dimerization, while other evidence suggests decreased dimerization and signaling. Herein, we use computational Monte Carlo simulations to examine the effects of MSK density and receptor concentration on receptor dimerization and clustering. Preliminary results suggest that the MSK may have the potential to induce receptor clustering, which is a function of both picket-fence density and receptor concentration.',\n",
              "  'authors': ['Costa, Michelle N',\n",
              "   'Radhakrishnan, Krishnan',\n",
              "   'Edwards, Jeremy S'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Journal of biotechnology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cell Membrane',\n",
              "   'Cluster Analysis',\n",
              "   'Computational Biology',\n",
              "   'ErbB Receptors',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Membrane Microdomains',\n",
              "   'Models, Biological',\n",
              "   'Monte Carlo Method'],\n",
              "  'pmid': '21167222',\n",
              "  'title': 'Monte Carlo simulations of plasma membrane corral-induced EGFR clustering.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Deep learning algorithms have achieved human-equivalent performance in image recognition. However, the majority of clinical data within electronic health records is inherently in a non-image format. Therefore, creating visual representations of clinical data could facilitate using cutting-edge deep learning models for predicting outcomes such as in-hospital mortality, while enabling clinician interpretability. The objective of this study was to develop a framework that first transforms longitudinal patient data into visual timelines and then utilizes deep learning to predict in-hospital mortality.|All adult consecutive patient admissions from 2008-2016 at a tertiary care center were included in this retrospective study. Two-dimensional visual representations for each patient were created with clinical variables on one dimension and time on the other. Predictors included vital signs, laboratory results, medications, interventions, nurse examinations, and diagnostic tests collected over the first 48 hours of the hospital stay. These visual timelines were utilized by a convolutional neural network with a recurrent layer model to predict in-hospital mortality. Seventy percent of the cohort was used for model derivation and 30% for independent validation. Of 115,825 hospital admissions, 2,926 (2.5%) suffered in-hospital mortality. Our model predicted in-hospital mortality significantly better than the Modified Early Warning Score (area under the receiver operating characteristic curve [AUC]: 0.91 vs. 0.76, P < 0.001) and the Sequential Organ Failure Assessment score (AUC: 0.91 vs. 0.57, P < 0.001) in the independent validation set. Class-activation heatmaps were utilized to highlight areas of the picture that were most important for making the prediction, thereby providing clinicians with insight into each individual patient's prediction.|We converted longitudinal patient data into visual timelines and applied a deep neural network for predicting in-hospital mortality more accurately than current standard clinical models, while allowing for interpretation. Our framework holds promise for predicting several important outcomes in clinical medicine.\",\n",
              "  'authors': ['Mayampurath, Anoop',\n",
              "   'Sanchez-Pinto, L Nelson',\n",
              "   'Carey, Kyle A',\n",
              "   'Venable, Laura-Ruth',\n",
              "   'Churpek, Matthew'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31365580',\n",
              "  'title': 'Combining patient visual timelines with deep learning to predict mortality.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-6882',\n",
              "  'IF_2013': '6.765',\n",
              "  'IF_2014': '6.628',\n",
              "  'IF_2015': '6.373',\n",
              "  'IF_2016': '6.133',\n",
              "  'IF_2017': '6.541',\n",
              "  'ISSN': '0959-4388',\n",
              "  'ISSN_online': '1873-6882',\n",
              "  'ISSN_print': '0959-4388',\n",
              "  'IsoAbbr': 'Curr. Opin. Neurobiol.',\n",
              "  'JrId': 2201,\n",
              "  'MedAbbr': 'Curr Opin Neurobiol',\n",
              "  'NLMid': '9111376',\n",
              "  'Titre': 'Curr. Opin. Neurobiol.',\n",
              "  'abstract': \"'High-level' vision lacks a single, agreed upon definition, but it might usefully be defined as those stages of visual processing that transition from analyzing local image structure to analyzing structure of the external world that produced those images. Much work in the last several decades has focused on object recognition as a framing problem for the study of high-level visual cortex, and much progress has been made in this direction. This approach presumes that the operational goal of the visual system is to read-out the identity of an object (or objects) in a scene, in spite of variation in the position, size, lighting and the presence of other nearby objects. However, while object recognition as a operational framing of high-level is intuitive appealing, it is by no means the only task that visual cortex might do, and the study of object recognition is beset by challenges in building stimulus sets that adequately sample the infinite space of possible stimuli. Here I review the successes and limitations of this work, and ask whether we should reframe our approaches to understanding high-level vision.\",\n",
              "  'authors': ['Cox, David Daniel'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Current opinion in neurobiology',\n",
              "  'keywords': ['Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Pattern Recognition, Visual',\n",
              "   'Vision, Ocular'],\n",
              "  'pmid': '24552691',\n",
              "  'title': 'Do we understand high-level vision?',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'Grain number per panicle (GNP) is a major determinant of grain yield in cereals. However, the mechanisms that regulate GNP remain unclear. To address this issue, we isolate a series of sorghum [Sorghum bicolor (L.) Moench] multiseeded (msd) mutants that can double GNP by increasing panicle size and altering floral development so that all spikelets are fertile and set grain. Through bulk segregant analysis by next-generation sequencing, we identify MSD1 as a TCP (Teosinte branched/Cycloidea/PCF) transcription factor. Whole-genome expression profiling reveals that jasmonic acid (JA) biosynthetic enzymes are transiently activated in pedicellate spikelets. Young msd1 panicles have 50% less JA than wild-type (WT) panicles, and application of exogenous JA can rescue the msd1 phenotype. Our results reveal a new mechanism for increasing GNP, with the potential to boost grain yield, and provide insight into the regulation of plant inflorescence architecture and development.',\n",
              "  'authors': ['Jiao, Yinping',\n",
              "   'Lee, Young Koung',\n",
              "   'Gladman, Nicholas',\n",
              "   'Chopra, Ratan',\n",
              "   'Christensen, Shawn A',\n",
              "   'Regulski, Michael',\n",
              "   'Burow, Gloria',\n",
              "   'Hayes, Chad',\n",
              "   'Burke, John',\n",
              "   'Ware, Doreen',\n",
              "   'Xin, Zhanguo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': ['Cyclopentanes',\n",
              "   'Edible Grain',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Inflorescence',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Oxylipins',\n",
              "   'Plant Growth Regulators',\n",
              "   'Plant Proteins',\n",
              "   'Seeds',\n",
              "   'Signal Transduction',\n",
              "   'Sorghum',\n",
              "   'Transcription Factors'],\n",
              "  'pmid': '29483511',\n",
              "  'title': 'MSD1 regulates pedicellate spikelet fertility in sorghum through the jasmonic acid pathway.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"This paper presents the concept of Simultaneous Localization and Multi-Mapping (SLAMM). It is a system that ensures continuous mapping and information preservation despite failures in tracking due to corrupted frames or sensor's malfunction; making it suitable for real-world applications. It works with single or multiple robots. In a single robot scenario the algorithm generates a new map at the time of tracking failure, and later it merges maps at the event of loop closure. Similarly, maps generated from multiple robots are merged without prior knowledge of their relative poses; which makes this algorithm flexible. The system works in real time at frame-rate speed. The proposed approach was tested on the KITTI and TUM RGB-D public datasets and it showed superior results compared to the state-of-the-arts in calibrated visual monocular keyframe-based SLAM. The mean tracking time is around 22 milliseconds. The initialization is twice as fast as it is in ORB-SLAM, and the retrieved map can reach up to 90 percent more in terms of information preservation depending on tracking loss and loop closure events. For the benefit of the community, the source code along with a framework to be run with Bebop drone are made available at https://github.com/hdaoud/ORBSLAMM.\",\n",
              "  'authors': ['Daoud, Hayyan Afeef',\n",
              "   'Md Sabri, Aznul Qalid',\n",
              "   'Loo, Chu Kiong',\n",
              "   'Mansoor, Ali Mohammed'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Robotics'],\n",
              "  'pmid': '29702697',\n",
              "  'title': 'SLAMM: Visual monocular SLAM with continuous mapping using multiple maps.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['BA'],\n",
              "  'ESSN': '1528-1175',\n",
              "  'IF_2013': '6.168',\n",
              "  'IF_2014': '5.879',\n",
              "  'IF_2015': '5.264',\n",
              "  'IF_2016': '5.785',\n",
              "  'IF_2017': '6.523',\n",
              "  'ISSN': '0003-3022',\n",
              "  'ISSN_online': '1528-1175',\n",
              "  'ISSN_print': '0003-3022',\n",
              "  'IsoAbbr': 'Anesthesiology',\n",
              "  'JrId': 533,\n",
              "  'MedAbbr': 'Anesthesiology',\n",
              "  'NLMid': '1300217',\n",
              "  'Titre': 'Anesthesiology',\n",
              "  'abstract': 'Radical prostatectomy (RP) is most commonly performed laparoscopically with a robot (robotic-assisted laparoscopic radical prostatectomy, R/PROST). Hysterectomy, which may be open hysterectomy (O/HYST) or laparoscopic hysterectomy (L/HYST), has been increasingly frequently done via robot (R/HYST). Small case series suggest increased corneal abrasions (CAs) with less invasive techniques.|The authors identified RP (166,942), O/HYST (583,298), or L/HYST (216,890) discharges with CA in the Nationwide Inpatient Sample (2000-2011). For 2009-2011, they determined odds ratios (ORs) and 95% confidence intervals (CIs) for CA, in R/PROST, non-R/PROST, L/HYST, O/HYST, and R/HYST. Uni- and multivariate models studied CA risk depending on surgical procedure, age, race, year, chronic illness, and malignancy.|In 2000-2011, 0.18% RP, 0.13% L/HYST, and 0.03% O/HYST sustained CA. Compared with 17,554 non-R/PROSTs (34 abrasions, 0.19%) in 2009-2011, OR was not significantly higher in 28,521 R/PROSTs (99, 0.35%; OR 1.508; CI 0.987 to 2.302; P < 0.057). CA significantly increased in L/HYST (70/51,323; 0.136%) versus O/HYST (70/191,199; 0.037%; OR 3.821; CI 2.594 to 5.630; P < 0.0001), further increasing in R/HYST (63/21, 213; 0.297%; OR 6.505; CI 4.323 to 9.788; P < 0.0001). For hysterectomy, risk of CA increased with age (OR 1.020; CI 1.007 to 1.034; P < 0.003) and number of chronic conditions (OR 1.139; CI 1.065 to 1.219; P < 0.0001). CA risk was likewise elevated in R/HYST with number of chronic conditions. Being African American significantly decreased CA risk in R/PROST and in R/HYST or L/HYST.|L/HYST increased CA nearly four-fold, and R/HYST approximately 6.5-fold versus O/HYST. Identifiable preoperative factors are associated with either increased risk (age, chronic conditions) or decreased risk (race).',\n",
              "  'authors': ['Sampat, Ajay',\n",
              "   'Parakati, Isaac',\n",
              "   'Kunnavakkam, Rangesh',\n",
              "   'Glick, David B',\n",
              "   'Lee, Nita K',\n",
              "   'Tenney, Meaghan',\n",
              "   'Eggener, Scott',\n",
              "   'Roth, Steven'],\n",
              "  'categories': ['ANESTHESIOLOGY'],\n",
              "  'journal': 'Anesthesiology',\n",
              "  'keywords': ['African Americans',\n",
              "   'Aged',\n",
              "   'Corneal Injuries',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Incidence',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Risk Factors',\n",
              "   'Robotics'],\n",
              "  'pmid': '25734923',\n",
              "  'title': 'Corneal abrasion in hysterectomy and prostatectomy: role of laparoscopic and robotic assistance.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['WF'],\n",
              "  'ESSN': '1529-7268',\n",
              "  'IF_2013': '3.451',\n",
              "  'IF_2014': '3.318',\n",
              "  'IF_2015': '3.471',\n",
              "  'IF_2016': '3.432',\n",
              "  'IF_2017': '3.184',\n",
              "  'ISSN': '0006-3363',\n",
              "  'ISSN_online': '1529-7268',\n",
              "  'ISSN_print': '0006-3363',\n",
              "  'IsoAbbr': 'Biol. Reprod.',\n",
              "  'JrId': 1121,\n",
              "  'MedAbbr': 'Biol Reprod',\n",
              "  'NLMid': '0207224',\n",
              "  'Titre': 'Biol. Reprod.',\n",
              "  'abstract': 'In cattle, maternal recognition of pregnancy occurs on Day 16 via secretion of interferon tau (IFNT) by the conceptus. The endometrium can distinguish between embryos with different developmental competencies. In eutherian mammals, X-chromosome inactivation (XCI) is required to ensure an equal transcriptional level of most X-linked genes for both male and female embryos in adult tissues, but this process is markedly different in cattle than mice. We examined how sexual dimorphism affected conceptus transcript abundance and amino acid composition as well as the endometrial transcriptome during the peri-implantation period of pregnancy. Of the 5132 genes that were differentially expressed on Day 19 in male compared to female conceptuses, 2.7% were located on the X chromosome. Concentrations of specific amino acids were higher in the uterine luminal fluid of male compared to female conceptuses, while female conceptuses had higher transcript abundance of specific amino acid transporters (SLC6A19 and SLC1A35). Of note, the endometrial transcriptome was not different in cattle gestating a male or a female conceptus. These data support the hypothesis that, far from being a blastocyst-specific phenomenon, XCI is incomplete before and during implantation in cattle. Despite differences in transcript abundance and amino acid utilization in male versus female conceptuses, the sex of the conceptus itself does not elicit a different transcriptomic response in the endometrium.',\n",
              "  'authors': ['Forde, Niamh',\n",
              "   'Maillo, Veronica',\n",
              "   \"O'Gaora, Peadar\",\n",
              "   'Simintiras, Constantine A',\n",
              "   'Sturmey, Roger G',\n",
              "   'Ealy, Alan D',\n",
              "   'Spencer, Thomas E',\n",
              "   'Gutierrez-Adan, Alfonso',\n",
              "   'Rizos, Dimitrios',\n",
              "   'Lonergan, Patrick'],\n",
              "  'categories': ['REPRODUCTIVE BIOLOGY'],\n",
              "  'journal': 'Biology of reproduction',\n",
              "  'keywords': ['Amino Acid Transport Systems',\n",
              "   'Amino Acids',\n",
              "   'Animals',\n",
              "   'Blastocyst',\n",
              "   'Cattle',\n",
              "   'Embryo Implantation',\n",
              "   'Endometrium',\n",
              "   'Female',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Ontology',\n",
              "   'Male',\n",
              "   'Pregnancy',\n",
              "   'Pregnancy, Animal',\n",
              "   'Sex Characteristics',\n",
              "   'Transcriptome',\n",
              "   'X Chromosome',\n",
              "   'X Chromosome Inactivation'],\n",
              "  'pmid': '27488033',\n",
              "  'title': 'Sexually Dimorphic Gene Expression in Bovine Conceptuses at the Initiation of Implantation.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': \"To compare the incidence of positive surgical margins obtained with robotic-assisted laparoscopic prostatectomy (RALP), during the initiation of a robotics program, with that from a similarly matched cohort of open radical retropubic prostatectomy (RRP) cases as performed by a single surgeon.|From December 2005 to March 2008, 63 patients underwent RRP and another 50 underwent RALP by a single urologist. The records were retrospectively reviewed, and 50 RRP patients were selected from the RRP group whose records were similar to the records of the 50 patients who had undergone RALP. We compared the incidence of positive surgical margins and the location of positive margins among the 2 groups. Additional variables evaluated included the preoperative prostate-specific antigen level, preoperative Gleason score, clinical stage, postoperative Gleason score, tumor volume, and pathologic stage.|The positive margin rate for the RRP group was 36% compared with 22% for the RALP group (P = .007). The incidence of positive margins for pathologic Stage pT2c disease in the RALP group was 22.8% compared with 42.8% in the RRP group, a statistically significant difference (P = .006). Fewer positive margins were found in the RALP Gleason score 7 group than in the RRP group, 29% vs 60%, again a statistically significant difference (P = .003).|We present our series comparing a single urologist's positive margin rates during the learning curve of a robotics program with his experience of a similarly matched cohort of RRP patients. A statistically significant lower positive margin rate can be achieved in RALP patients even during the learning period.\",\n",
              "  'authors': ['White, Michael A',\n",
              "   'De Haan, Alexander P',\n",
              "   'Stephens, D Dawon',\n",
              "   'Maatman, Thomas K',\n",
              "   'Maatman, Thomas J'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics'],\n",
              "  'pmid': '19167036',\n",
              "  'title': 'Comparative analysis of surgical margins between radical retropubic prostatectomy and RALP: are patients sacrificed during initiation of robotics program?',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1938-1352',\n",
              "  'IF_2013': '1.688',\n",
              "  'IF_2014': '1.43',\n",
              "  'IF_2015': '1.043',\n",
              "  'IF_2016': '1.277',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0748-7711',\n",
              "  'ISSN_online': '1938-1352',\n",
              "  'ISSN_print': '0748-7711',\n",
              "  'IsoAbbr': 'J Rehabil Res Dev',\n",
              "  'JrId': 5189,\n",
              "  'MedAbbr': 'J Rehabil Res Dev',\n",
              "  'NLMid': '8410047',\n",
              "  'Titre': 'J Rehabil Res Dev',\n",
              "  'abstract': 'Significant potential exists for enhancing physical rehabilitation following neurologic injury through the use of robotic and mechatronic devices (or \"rehabilitators\"). We review the development of a rehabilitator (the \"ARM Guide\") to diagnose and treat arm movement impairment following stroke and other brain injuries. As a diagnostic tool, the ARM Guide provides a basis for evaluation of several key motor impairments, including abnormal tone, incoordination, and weakness. As a therapeutic tool, the device provides a means to implement and evaluate active assist therapy for the arm. Initial results with three stroke subjects demonstrate that such therapy can produce quantifiable benefits in the chronic hemiparetic arm. Directions for future research regarding the efficacy and practicality of rehabilitators are discussed.',\n",
              "  'authors': ['Reinkensmeyer, D J',\n",
              "   'Kahn, L E',\n",
              "   'Averbuch, M',\n",
              "   'McKenna-Cole, A',\n",
              "   'Schmit, B D',\n",
              "   'Rymer, W Z'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'Journal of rehabilitation research and development',\n",
              "  'keywords': ['Adult',\n",
              "   'Arm',\n",
              "   'Brain Injuries',\n",
              "   'Brain Injury, Chronic',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Movement Disorders',\n",
              "   'Robotics',\n",
              "   'Stroke',\n",
              "   'Stroke Rehabilitation'],\n",
              "  'pmid': '11321001',\n",
              "  'title': 'Understanding and treating arm movement impairment after chronic brain injury: progress with the ARM guide.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SD'],\n",
              "  'ESSN': '1553-4669',\n",
              "  'IF_2013': '1.575',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.39',\n",
              "  'IF_2016': '3.061',\n",
              "  'IF_2017': '3.061',\n",
              "  'ISSN': '1553-4650',\n",
              "  'ISSN_online': '1553-4669',\n",
              "  'ISSN_print': '1553-4650',\n",
              "  'IsoAbbr': 'J Minim Invasive Gynecol',\n",
              "  'JrId': 32424,\n",
              "  'MedAbbr': 'J Minim Invasive Gynecol',\n",
              "  'NLMid': '101235322',\n",
              "  'Titre': 'J Minim Invasive Gynecol',\n",
              "  'abstract': 'To determine essential specifications for an active endoscope holder, a survey of laparoscopic procedures was conducted. A review of the literature highlighted the advantages and limitations of existing scope-holding systems. From this analysis, basic requirements were listed for such devices. Pursuant to this, an ergonomic and user-friendly laparoscope manipulator was designed to assist the surgeon. A first in vivo procedure demonstrated feasibility of the device and its value in clinical practice, enabling surgeons to work more comfortably.',\n",
              "  'authors': ['Herman, Benoît',\n",
              "   'Duy, Khanh Tran',\n",
              "   'Dehez, Bruno',\n",
              "   'Polet, Roland',\n",
              "   'Raucent, Benoit',\n",
              "   'Dombre, Etienne',\n",
              "   'Donnez, Jacques'],\n",
              "  'categories': ['OBSTETRICS & GYNECOLOGY'],\n",
              "  'journal': 'Journal of minimally invasive gynecology',\n",
              "  'keywords': ['Adult',\n",
              "   'Equipment Design',\n",
              "   'Female',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Man-Machine Systems',\n",
              "   'Robotics'],\n",
              "  'pmid': '19423066',\n",
              "  'title': 'Development and first in vivo trial of EvoLap, an active laparoscope positioner.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1941-0506',\n",
              "  'IF_2013': '1.919',\n",
              "  'IF_2014': '2.168',\n",
              "  'IF_2015': '1.4',\n",
              "  'IF_2016': '2.84',\n",
              "  'IF_2017': '3.078',\n",
              "  'ISSN': '1077-2626',\n",
              "  'ISSN_online': '1941-0506',\n",
              "  'ISSN_print': '1077-2626',\n",
              "  'IsoAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'JrId': 31980,\n",
              "  'MedAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'NLMid': '9891704',\n",
              "  'Titre': 'IEEE Trans Vis Comput Graph',\n",
              "  'abstract': 'We present a learning-based approach to reconstructing high-resolution three-dimensional (3D) shapes with detailed geometry and high-fidelity textures. Albeit extensively studied, algorithms for 3D reconstruction from multi-view depth-and-color (RGB-D) scans are still prone to measurement noise and occlusions; limited scanning or capturing angles also often lead to incomplete reconstructions. Propelled by recent advances in 3D deep learning techniques, in this paper, we introduce a novel computation and memory efficient cascaded 3D convolutional network architecture, which learns to reconstruct implicit surface representations as well as the corresponding color information from noisy and imperfect RGB-D maps. The proposed 3D neural network performs reconstruction in a progressive and coarse-to-fine manner, achieving unprecedented output resolution and fidelity. Meanwhile, an algorithm for end-to-end training of the proposed cascaded structure is developed. We further introduce Human10, a newly created dataset containing both detailed and textured full body reconstructions as well as corresponding raw RGB-D scans of 10 subjects. Qualitative and quantitative experimental results on both synthetic and real-world datasets demonstrate that the presented approach outperforms existing state-of-the-art work in terms of visual quality and accuracy of reconstructed models.',\n",
              "  'authors': ['Liu, Zheng-Ning',\n",
              "   'Cao, Yan-Pei',\n",
              "   'Kuang, Zheng-Fei',\n",
              "   'Kobbelt, Leif',\n",
              "   'Hu, Shi-Min'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE transactions on visualization and computer graphics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31449026',\n",
              "  'title': 'High-quality Textured 3D Shape Reconstruction with Cascaded Fully Convolutional Networks.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'We have performed >350 robotic radical prostatectomies in the last 2 years. A single surgeon (MM) performed 250 of these procedures using a technique developed at our institution, the Vattikuti Urology Institute. This article summarizes the technical highlights and interim results of the Vattikuti Institute Prostatectomy (VIP) technique. We prospectively collected baseline demographic data, such as age, race, body mass index (BMI), serum prostate-specific antigen values, prostate volume, Gleason score, percentage cancer, TNM clinical staging, and comorbidities. Urinary symptoms were measured with the International Prostate Symptom Score, and sexual health with the Sexual Health Inventory of Males. In addition, patients received the Expanded Prostate Inventory Composite at baseline and at 1, 3, 6, 12 and 18 months after the procedure via mail. Data collection is complete on 200 of the first 250 patient cases. Gleason score >or=7 was noted in 40% of patients. The average BMI was high (28), and 86% patients were classified as pathologic stage pT2a to pT2b. The mean operative time was 160 minutes and the mean blood loss was 153 mL. No patient required blood transfusion. At 6 months, 82% of the men who were <60 years of age and 75% of those >60 years of age had return of sexual function, and 64% and 38%, respectively, had sexual intercourse. At 6 months, 96% patients were continent.',\n",
              "  'authors': ['Menon, Mani', 'Tewari, Ashutosh'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Staging',\n",
              "   'Prospective Studies',\n",
              "   'Prostate',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Sexual Behavior',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '12657356',\n",
              "  'title': 'Robotic radical prostatectomy and the Vattikuti Urology Institute technique: an interim analysis of results and technical points.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1879-0720',\n",
              "  'IF_2013': '3.005',\n",
              "  'IF_2014': '3.35',\n",
              "  'IF_2015': '3.773',\n",
              "  'IF_2016': '3.756',\n",
              "  'IF_2017': '3.466',\n",
              "  'ISSN': '0928-0987',\n",
              "  'ISSN_online': '1879-0720',\n",
              "  'ISSN_print': '0928-0987',\n",
              "  'IsoAbbr': 'Eur J Pharm Sci',\n",
              "  'JrId': 21091,\n",
              "  'MedAbbr': 'Eur J Pharm Sci',\n",
              "  'NLMid': '9317982',\n",
              "  'Titre': 'Eur J Pharm Sci',\n",
              "  'abstract': 'In the present study a quantitative structure-activity relationship (QSAR) technique was developed to investigate the blood-to-brain barrier partitioning behavior (log BB) for various drugs and organic compounds. Important descriptors were selected by genetic algorithm-partial least square (GA-PLS) methods. Partial least squares (PLS) and support vector machine (SVM) methods were employed to construct linear and non-linear models, respectively. The results showed that, the log BB values calculated by SVM were in good agreement with the experimental data, and the performance of the SVM model was superior to the PLS model. The study provided a novel and effective method for predicting blood-to-brain barrier penetration of drugs, and disclosed that SVM can be used as a powerful chemometrics tool for QSAR studies.',\n",
              "  'authors': ['Golmohammadi, Hassan',\n",
              "   'Dashtbozorgi, Zahra',\n",
              "   'Acree, William E'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'European journal of pharmaceutical sciences : official journal of the European Federation for Pharmaceutical Sciences',\n",
              "  'keywords': ['Blood-Brain Barrier',\n",
              "   'Least-Squares Analysis',\n",
              "   'Molecular Structure',\n",
              "   'Pharmaceutical Preparations',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '22771548',\n",
              "  'title': 'Quantitative structure-activity relationship prediction of blood-to-brain partitioning behavior using support vector machine.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1943-2631',\n",
              "  'IF_2013': '4.866',\n",
              "  'IF_2014': '5.963',\n",
              "  'IF_2015': '4.644',\n",
              "  'IF_2016': '4.556',\n",
              "  'IF_2017': '4.075',\n",
              "  'ISSN': '0016-6731',\n",
              "  'ISSN_online': '1943-2631',\n",
              "  'ISSN_print': '0016-6731',\n",
              "  'IsoAbbr': 'Genetics',\n",
              "  'JrId': 3871,\n",
              "  'MedAbbr': 'Genetics',\n",
              "  'NLMid': '0374636',\n",
              "  'Titre': 'Genetics',\n",
              "  'abstract': 'In the yeast Saccharomyces cerevisiae, fermentation is the major pathway for energy production, even under aerobic conditions. However, when glucose becomes scarce, ethanol produced during fermentation is used as a carbon source, requiring a shift to respiration. This adaptation results in massive reprogramming of gene expression. Increased expression of genes for gluconeogenesis and the glyoxylate cycle is observed upon a shift to ethanol and, conversely, expression of some fermentation genes is reduced. The zinc cluster proteins Cat8, Sip4, and Rds2, as well as Adr1, have been shown to mediate this reprogramming of gene expression. In this study, we have characterized the gene YBR239C encoding a putative zinc cluster protein and it was named ERT1 (ethanol regulated transcription factor 1). ChIP-chip analysis showed that Ert1 binds to a limited number of targets in the presence of glucose. The strongest enrichment was observed at the promoter of PCK1 encoding an important gluconeogenic enzyme. With ethanol as the carbon source, enrichment was observed with many additional genes involved in gluconeogenesis and mitochondrial function. Use of lacZ reporters and quantitative RT-PCR analyses demonstrated that Ert1 regulates expression of its target genes in a manner that is highly redundant with other regulators of gluconeogenesis. Interestingly, in the presence of ethanol, Ert1 is a repressor of PDC1 encoding an important enzyme for fermentation. We also show that Ert1 binds directly to the PCK1 and PDC1 promoters. In summary, Ert1 is a novel factor involved in the regulation of gluconeogenesis as well as a key fermentation gene. ',\n",
              "  'authors': ['Gasmi, Najla',\n",
              "   'Jacques, Pierre-Etienne',\n",
              "   'Klimova, Natalia',\n",
              "   'Guo, Xiao',\n",
              "   'Ricciardi, Alessandra',\n",
              "   'Robert, François',\n",
              "   'Turcotte, Bernard'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Genetics',\n",
              "  'keywords': ['Base Sequence',\n",
              "   'Binding Sites',\n",
              "   'Fermentation',\n",
              "   'Fructose-Bisphosphatase',\n",
              "   'Gene Expression Regulation, Fungal',\n",
              "   'Gene Ontology',\n",
              "   'Gluconeogenesis',\n",
              "   'Mutation',\n",
              "   'Phosphoenolpyruvate Carboxykinase (GTP)',\n",
              "   'Promoter Regions, Genetic',\n",
              "   'Protein Binding',\n",
              "   'Saccharomyces cerevisiae',\n",
              "   'Saccharomyces cerevisiae Proteins',\n",
              "   'Transcription Factors'],\n",
              "  'pmid': '25123508',\n",
              "  'title': 'The switch from fermentation to respiration in Saccharomyces cerevisiae is regulated by the Ert1 transcriptional activator/repressor.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"To test the ability of machine learning classifiers (MLCs) using optical coherence tomography (OCT) and standard automated perimetry (SAP) parameters to discriminate between healthy and glaucomatous individuals, and to compare it to the diagnostic ability of the combined structure-function index (CSFI), general ophthalmologists and glaucoma specialists.|Cross-sectional prospective study.|Fifty eight eyes of 58 patients with early to moderate glaucoma (median value of the mean deviation = -3.44 dB; interquartile range, -6.0 to -2.4 dB) and 66 eyes of 66 healthy individuals underwent OCT and SAP tests. The diagnostic accuracy (area under the ROC curve-AUC) of 10 MLCs was compared to those obtained with the CSFI, 3 general ophthalmologists and 3 glaucoma specialists exposed to the same OCT and SAP data.|The AUCs obtained with MLCs ranged from 0.805 (Classification Tree) to 0.931 (Radial Basis Function Network, RBF). The sensitivity at 90% specificity ranged from 51.6% (Classification Tree) to 82.8% (Bagging, Multilayer Perceptron and Support Vector Machine Gaussian). The CSFI had a sensitivity of 79.3% at 90% specificity, and the highest AUC (0.948). General ophthalmologists and glaucoma specialists' grading had sensitivities of 66.2% and 83.8% at 90% specificity, and AUCs of 0.879 and 0.921, respectively. RBF (the best MLC), the CSFI, and glaucoma specialists showed significantly higher AUCs than that obtained by general ophthalmologists (P<0.05). However, there were no significant differences between the AUCs obtained by RBF, the CSFI, and glaucoma specialists (P>0.25).|Our findings suggest that both MLCs and the CSFI can be helpful in clinical practice and effectively improve glaucoma diagnosis in the primary eye care setting, when there is no glaucoma specialist available.\",\n",
              "  'authors': ['Shigueoka, Leonardo Seidi',\n",
              "   'Vasconcellos, José Paulo Cabral de',\n",
              "   'Schimiti, Rui Barroso',\n",
              "   'Reis, Alexandre Soares Castro',\n",
              "   'Oliveira, Gabriel Ozeas de',\n",
              "   'Gomi, Edson Satoshi',\n",
              "   'Vianna, Jayme Augusto Rocha',\n",
              "   'Lisboa, Renato Dichetti Dos Reis',\n",
              "   'Medeiros, Felipe Andrade',\n",
              "   'Costa, Vital Paulino'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cross-Sectional Studies',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Female',\n",
              "   'Glaucoma',\n",
              "   'Glaucoma, Open-Angle',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Ophthalmologists',\n",
              "   'Primary Health Care',\n",
              "   'Prospective Studies',\n",
              "   'ROC Curve',\n",
              "   'Supervised Machine Learning',\n",
              "   'Tomography, Optical Coherence',\n",
              "   'Visual Field Tests'],\n",
              "  'pmid': '30517157',\n",
              "  'title': 'Automated algorithms combining structure and function outperform general ophthalmologists in diagnosing glaucoma.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '2160-1836',\n",
              "  'IF_2013': '2.511',\n",
              "  'IF_2014': '3.198',\n",
              "  'IF_2015': '2.91',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '2.742',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2160-1836',\n",
              "  'IsoAbbr': 'G3 (Bethesda)',\n",
              "  'JrId': 39472,\n",
              "  'MedAbbr': 'G3 (Bethesda)',\n",
              "  'NLMid': '101566598',\n",
              "  'Titre': 'G3 (Bethesda)',\n",
              "  'abstract': 'Post-traumatic stress disorder is a concerning psychobehavioral disorder thought to emerge from the complex interaction between genetic and environmental factors. For soldiers exposed to combat, the risk of developing this disorder is twofold and diagnosis is often late, when much sequela has set in. To be able to identify and diagnose in advance those at \"risk\" of developing post-traumatic stress disorder, would greatly taper the gap between late sequelae and treatment. Therefore, this study sought to determine whether the transcriptome can be used to track the development of post-traumatic stress disorder in this unique and susceptible cohort of individuals. Gene expression levels in peripheral blood samples from 85 Canadian infantry soldiers (n = 58 participants negative for symptoms of post-traumatic stress disorder and n = 27 participants with symptoms of post-traumatic stress disorder) following return from deployment to Afghanistan were determined using RNA sequencing technology. Count-based gene expression quantification, normalization and differential analysis (with thorough correction for confounders) revealed genes associated to PTSD; LRP8 and GOLM1 These preliminary results provide a proof-of-principle for the diagnostic utility of blood-based gene expression profiles for tracking symptoms of post-traumatic stress disorder in soldiers returning from tour. It is also the first to report transcriptome-wide expression profiles alongside a post-traumatic symptom checklist.',\n",
              "  'authors': ['Boscarino, Cathy',\n",
              "   'Nalpathamkalam, Thomas',\n",
              "   'Pellecchia, Giovanna',\n",
              "   'Li, Weili',\n",
              "   'Thiruvahindrapuram, Bhooma',\n",
              "   'Merico, Daniele'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'G3 (Bethesda, Md.)',\n",
              "  'keywords': ['Adult',\n",
              "   'Biomarkers',\n",
              "   'Gene Expression Profiling',\n",
              "   'Humans',\n",
              "   'LDL-Receptor Related Proteins',\n",
              "   'Male',\n",
              "   'Membrane Proteins',\n",
              "   'Military Personnel',\n",
              "   'RNA, Messenger',\n",
              "   'Stress Disorders, Post-Traumatic',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '30622122',\n",
              "  'title': 'Using Next-Generation Sequencing Transcriptomics To Determine Markers of Post-traumatic Symptoms: Preliminary Findings from a Post-deployment Cohort of Soldiers.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'Some urologists who perform vasectomy reversals are not experienced with performing VE. A model to preoperatively identify patients who may require referral to an experienced VE surgeon was created (). We tested the model at multiple institutions.|The model had previously been designed in 483 patients who underwent vasectomy reversal at 1 institution (100% sensitive and 59% specific for predicting the need for VE). It was based on time since vasectomy and patient age. We tested it prospectively in 33 patients and retrospectively in a total of 312 at 6 other institutions. The predictive accuracy of the model was compared to using a simple duration from vasectomy cutoff alone, as is used in clinical practice.|The model had 84% sensitivity and 58% specificity for detecting the need for VE in a total of 345 patients at 7 institutions. If using only a duration from vasectomy cutoff of 10 years to predict the need for VE, sensitivity was only 69%. At a cutoff of 4 years sensitivity was 99% but specificity was only 23%. Thus, the model performed better than any specific duration cutoff alone.|The predictive model provides 84% sensitivity for detecting patients who may require VE during vasectomy reversal across 7 institutions (58% specificity). The model more accurately predicts the need for VE than using a specific duration from vasectomy cutoff alone.',\n",
              "  'authors': ['Parekattil, Sijo J',\n",
              "   'Kuang, Wayne',\n",
              "   'Kolettis, Peter N',\n",
              "   'Pasqualotto, Fabio F',\n",
              "   'Teloken, Patrick',\n",
              "   'Teloken, Claudio',\n",
              "   'Nangia, Ajay K',\n",
              "   'Daitch, James A',\n",
              "   'Niederberger, Craig',\n",
              "   'Thomas, Anthony J'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prospective Studies',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Vasovasostomy'],\n",
              "  'pmid': '16406922',\n",
              "  'title': 'Multi-institutional validation of vasectomy reversal predictor.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['QA'],\n",
              "  'ESSN': '1479-5876',\n",
              "  'IF_2013': '3.991',\n",
              "  'IF_2014': '3.93',\n",
              "  'IF_2015': '3.694',\n",
              "  'IF_2016': '3.786',\n",
              "  'IF_2017': '4.197',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1479-5876',\n",
              "  'IsoAbbr': 'J Transl Med',\n",
              "  'JrId': 31680,\n",
              "  'MedAbbr': 'J Transl Med',\n",
              "  'NLMid': '101190741',\n",
              "  'Titre': 'J Transl Med',\n",
              "  'abstract': 'Asthma is a complex, heterogeneous disorder with similar presenting symptoms but with varying underlying pathologies. Exhaled breath condensate (EBC) is a relatively unexplored matrix which reflects the signatures of respiratory epithelium, but is difficult to normalize for dilution.|Here we explored whether internally normalized global NMR spectrum patterns, combined with machine learning, could be useful for diagnostics or endotype discovery. Nuclear magnetic resonance (NMR) spectroscopy of EBC was performed in 89 asthmatic subjects from a prospective cohort and 20 healthy controls. A random forest classifier was built to differentiate between asthmatics and healthy controls. Clustering of the spectra was done using k-means to identify potential endotypes.|NMR spectra of the EBC could differentiate between asthmatics and healthy controls with 80% sensitivity and 75% specificity. Unsupervised clustering within the asthma group resulted in three clusters (n = 41,11, and 9). Cluster 1 patients had lower long-term exacerbation scores, when compared with other two clusters. Cluster 3 patients had lower blood eosinophils and higher neutrophils, when compared with other two clusters with a strong family history of asthma.|Asthma clusters derived from NMR spectra of EBC show important clinical and chemical differences, suggesting this as a useful tool in asthma endotype-discovery.',\n",
              "  'authors': ['Sinha, Anirban',\n",
              "   'Desiraju, Koundinya',\n",
              "   'Aggarwal, Kunal',\n",
              "   'Kutum, Rintu',\n",
              "   'Roy, Siddhartha',\n",
              "   'Lodha, Rakesh',\n",
              "   'Kabra, S K',\n",
              "   'Ghosh, Balaram',\n",
              "   'Sethi, Tavpritesh',\n",
              "   'Agrawal, Anurag'],\n",
              "  'categories': ['MEDICINE, RESEARCH & EXPERIMENTAL'],\n",
              "  'journal': 'Journal of translational medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Asthma',\n",
              "   'Breath Tests',\n",
              "   'Child',\n",
              "   'Cluster Analysis',\n",
              "   'Exhalation',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Metabolome',\n",
              "   'Metabolomics'],\n",
              "  'pmid': '29273025',\n",
              "  'title': 'Exhaled breath condensate metabolome clusters for endotype discovery in asthma.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Exposure to traumatic stressors and subsequent trauma-related mental changes may alter a person's risk-taking behavior. It is unclear whether this relationship depends on the specific types of traumatic experiences. Moreover, the association has never been tested in displaced individuals with substantial levels of traumatic experiences. The present study assessed risk-taking behavior in 56 displaced individuals by means of the balloon analogue risk task (BART). Exposure to traumatic events, symptoms of posttraumatic stress disorder and depression were assessed by means of semi-structured interviews. Using a novel statistical approach (stochastic gradient boosting machines), we analyzed predictors of risk-taking behavior. Exposure to organized violence was associated with less risk-taking, as indicated by fewer adjusted pumps in the BART, as was the reported experience of physical abuse and neglect, emotional abuse, and peer violence in childhood. However, civil traumatic stressors, as well as other events during childhood were associated with lower risk taking. This suggests that the association between global risk-taking behavior and exposure to traumatic stress depends on the particular type of the stressors that have been experienced.\",\n",
              "  'authors': ['Augsburger, Mareike', 'Elbert, Thomas'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Depression',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Life Change Events',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Refugees',\n",
              "   'Risk-Taking',\n",
              "   'Stress Disorders, Post-Traumatic',\n",
              "   'Young Adult'],\n",
              "  'pmid': '28498865',\n",
              "  'title': 'When do traumatic experiences alter risk-taking behavior? A machine learning analysis of reports from refugees.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'Rhythmic activity plays a central role in neural computations and brain functions ranging from homeostasis to attention, as well as in neurological and neuropsychiatric disorders. Despite this pervasiveness, little is known about the mechanisms whereby the frequency and power of oscillatory activity are modulated, and how they reflect the inputs received by neurons. Numerous studies have reported input-dependent fluctuations in peak frequency and power (as well as couplings across these features). However, it remains unresolved what mediates these spectral shifts among neural populations. Extending previous findings regarding stochastic nonlinear systems and experimental observations, we provide analytical insights regarding oscillatory responses of neural populations to stimulation from either endogenous or exogenous origins. Using a deceptively simple yet sparse and randomly connected network of neurons, we show how spiking inputs can reliably modulate the peak frequency and power expressed by synchronous neural populations without any changes in circuitry. Our results reveal that a generic, non-nonlinear and input-induced mechanism can robustly mediate these spectral fluctuations, and thus provide a framework in which inputs to the neurons bidirectionally regulate both the frequency and power expressed by synchronous populations. Theoretical and computational analysis of the ensuing spectral fluctuations was found to reflect the underlying dynamics of the input stimuli driving the neurons. Our results provide insights regarding a generic mechanism supporting spectral transitions observed across cortical networks and spanning multiple frequency bands. ',\n",
              "  'authors': ['Lefebvre, Jérémie',\n",
              "   'Hutt, Axel',\n",
              "   'Knebel, Jean-François',\n",
              "   'Whittingstall, Kevin',\n",
              "   'Murray, Micah M'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Periodicity'],\n",
              "  'pmid': '25698729',\n",
              "  'title': 'Stimulus statistics shape oscillations in nonlinear recurrent neural networks.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['OP'],\n",
              "  'ESSN': '1878-7487',\n",
              "  'IF_2013': '0.989',\n",
              "  'IF_2014': '0.76',\n",
              "  'IF_2015': '0.87',\n",
              "  'IF_2016': '1.135',\n",
              "  'IF_2017': '1.103',\n",
              "  'ISSN': '1752-928X',\n",
              "  'ISSN_online': '1878-7487',\n",
              "  'ISSN_print': '1752-928X',\n",
              "  'IsoAbbr': 'J Forensic Leg Med',\n",
              "  'JrId': 33405,\n",
              "  'MedAbbr': 'J Forensic Leg Med',\n",
              "  'NLMid': '101300022',\n",
              "  'Titre': 'J Forensic Leg Med',\n",
              "  'abstract': 'A deep learning artificial neural network was adapted to the task of sex determination of skeletal remains. The neural network was trained on images of 900 skulls virtually reconstructed from hospital CT scans. When tested on previously unseen images of skulls, the artificial neural network showed 95% accuracy at sex determination. Artificial intelligence methods require no significant expertise to implement once trained, are rapid to use, and have the potential to eliminate human bias from sex estimation of skeletal remains.',\n",
              "  'authors': ['Bewes, James',\n",
              "   'Low, Andrew',\n",
              "   'Morphett, Antony',\n",
              "   'Pate, F Donald',\n",
              "   'Henneberg, Maciej'],\n",
              "  'categories': ['MEDICINE, LEGAL'],\n",
              "  'journal': 'Journal of forensic and legal medicine',\n",
              "  'keywords': ['Adult',\n",
              "   'Australia',\n",
              "   'Deep Learning',\n",
              "   'Female',\n",
              "   'Forensic Anthropology',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sex Determination by Skeleton',\n",
              "   'Skull',\n",
              "   'Software',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '30639854',\n",
              "  'title': 'Artificial intelligence for sex determination of skeletal remains: Application of a deep learning artificial neural network to human skulls.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Medical and clinical question answering (QA) is highly concerned by researchers recently. Though there are remarkable advances in this field, the development in Chinese medical domain is relatively backward. It can be attributed to the difficulty of Chinese text processing and the lack of large-scale datasets. To bridge the gap, this paper introduces a Chinese medical QA dataset and proposes effective methods for the task.|We first construct a large scale Chinese medical QA dataset. Then we leverage deep matching neural networks to capture semantic interaction between words in questions and answers. Considering that Chinese Word Segmentation (CWS) tools may fail to identify clinical terms, we design a module to merge the word segments and produce a new representation. It learns the common compositions of words or segments by using convolutional kernels and selects the strongest signals by windowed pooling.|The best performer among popular CWS tools on our dataset is found. In our experiments, deep matching models substantially outperform existing methods. Results also show that our proposed semantic clustered representation module improves the performance of models by up to 5.5% Precision at 1 and 4.9% Mean Average Precision.|In this paper, we introduce a large scale Chinese medical QA dataset and cast the task into a semantic matching problem. We also compare different CWS tools and input units. Among the two state-of-the-art deep matching neural networks, MatchPyramid performs better. Results also show the effectiveness of the proposed semantic clustered representation module.',\n",
              "  'authors': ['He, Junqing', 'Fu, Mingming', 'Tu, Manshu'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30961607',\n",
              "  'title': 'Applying deep matching networks to Chinese medical question answering: a study and a dataset.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1520-8532',\n",
              "  'IF_2013': '1.448',\n",
              "  'IF_2014': '1.558',\n",
              "  'IF_2015': '1.457',\n",
              "  'IF_2016': '1.621',\n",
              "  'IF_2017': '1.566',\n",
              "  'ISSN': '1084-7529',\n",
              "  'ISSN_online': '1520-8532',\n",
              "  'ISSN_print': '1084-7529',\n",
              "  'IsoAbbr': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'JrId': 20813,\n",
              "  'MedAbbr': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'NLMid': '9800943',\n",
              "  'Titre': 'J Opt Soc Am A Opt Image Sci Vis',\n",
              "  'abstract': 'A method is proposed to reconstruct signals from incomplete data. The method, which can be interpreted both as a discrete implementation of the so-called prior discrete Fourier transform (PDFT) spectral estimation technique and as a variant of the algebraic reconstruction technique, allows one to incorporate prior information about the reconstructed signal to improve the resolution of the signal estimated. The context of diffraction tomography and image reconstruction from samples of the far-field scattering amplitude are used to explore the performance of the method. On the basis of numerical computations, the optimum choice of parameters is determined empirically by comparing image reconstructions of the noniterative PDFT algorithm and the proposed iterative scheme.',\n",
              "  'authors': ['Shieh, Hsin M',\n",
              "   'Byrne, Charles L',\n",
              "   'Testorf, Markus E',\n",
              "   'Fiddy, Michael A'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Journal of the Optical Society of America. A, Optics, image science, and vision',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Numerical Analysis, Computer-Assisted',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '16715147',\n",
              "  'title': 'Iterative image reconstruction using prior knowledge.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1095-8630',\n",
              "  'IF_2013': '3.188',\n",
              "  'IF_2014': '2.723',\n",
              "  'IF_2015': '3.131',\n",
              "  'IF_2016': '4.01',\n",
              "  'IF_2017': '4.005',\n",
              "  'ISSN': '0301-4797',\n",
              "  'ISSN_online': '1095-8630',\n",
              "  'ISSN_print': '0301-4797',\n",
              "  'IsoAbbr': 'J. Environ. Manage.',\n",
              "  'JrId': 22117,\n",
              "  'MedAbbr': 'J Environ Manage',\n",
              "  'NLMid': '0401664',\n",
              "  'Titre': 'J. Environ. Manage.',\n",
              "  'abstract': 'Flood disasters have been extremely severe in recent decades, and they account for about one third of all natural catastrophes throughout the world. In this study, a two-stage mixed-integer fuzzy programming with interval-valued membership functions (TMFP-IMF) approach is developed for flood-diversion planning under uncertainty. TMFP-IMF integrates the fuzzy flexible programming, two-stage stochastic programming, and integer programming within a general framework. A concept of interval-valued fuzzy membership function is introduced to address complexities of system uncertainties. TMFP-IMF can not only deal with uncertainties expressed as fuzzy sets and probability distributions, but also incorporate pre-regulated water-diversion policies directly into its optimization process. TMFP-IMF is applied to a hypothetical case study of flood-diversion planning for demonstrating its applicability. Results indicate that reasonable solutions can be generated for binary and continuous variables. A variety of flood-diversion and capacity-expansion schemes can be obtained under four scenarios, which enable decision makers (DMs) to identify the most desired one based on their perceptions and attitudes towards the objective-function value and constraints.',\n",
              "  'authors': ['Wang, S', 'Huang, G H'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Journal of environmental management',\n",
              "  'keywords': ['Decision Making',\n",
              "   'Disasters',\n",
              "   'Floods',\n",
              "   'Fuzzy Logic',\n",
              "   'Uncertainty'],\n",
              "  'pmid': '23376303',\n",
              "  'title': 'A two-stage mixed-integer fuzzy programming with interval-valued membership functions approach for flood-diversion planning.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Ontology Matching aims at identifying a set of semantic correspondences, called an alignment, between related ontologies. In recent years, there has been a growing interest in efficient and effective matching methods for large ontologies. However, alignments produced for large ontologies are often logically incoherent. It was only recently that the use of repair techniques to improve the coherence of ontology alignments began to be explored. This paper presents a novel modularization technique for ontology alignment repair which extracts fragments of the input ontologies that only contain the necessary classes and relations to resolve all detectable incoherences. The paper presents also an alignment repair algorithm that uses a global repair strategy to minimize both the degree of incoherence and the number of mappings removed from the alignment, while overcoming the scalability problem by employing the proposed modularization technique. Our evaluation shows that our modularization technique produces significantly small fragments of the ontologies and that our repair algorithm produces more complete alignments than other current alignment repair systems, while obtaining an equivalent degree of incoherence. Additionally, we also present a variant of our repair algorithm that makes use of the confidence values of the mappings to improve alignment repair. Our repair algorithm was implemented as part of AgreementMakerLight, a free and open-source ontology matching system. ',\n",
              "  'authors': ['Santos, Emanuel',\n",
              "   'Faria, Daniel',\n",
              "   'Pesquita, Catia',\n",
              "   'Couto, Francisco M'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biological Ontologies',\n",
              "   'Heuristics',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Semantics',\n",
              "   'Vocabulary, Controlled'],\n",
              "  'pmid': '26710335',\n",
              "  'title': 'Ontology Alignment Repair through Modularization and Confidence-Based Heuristics.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'The role of microRNAs (miRNAs) during cotton fiber development remains unclear. Here, a total of 54 miRNAs belonging to 39 families were selected to characterize miRNA regulatory mechanism in eight different fiber development stages in upland cotton cv BM-1. Among 54 miRNAs, 18 miRNAs were involved in cotton fiber initiation and eight miRNAs were related to fiber elongation and secondary wall biosynthesis. Additionally, 3,576 protein-coding genes were candidate target genes of these miRNAs, which are potentially involved in cotton fiber development. We also investigated the regulatory network of miRNAs and corresponding targets in fiber initiation and elongation, and secondary wall formation. Our Gene Ontology-based term classification and KEGG-based pathway enrichment analyses showed that the miRNA targets covered 220 biological processes, 67 molecular functions, 45 cellular components, and 10 KEGG pathways. Three of ten KEGG pathways were involved in lignan synthesis, cell elongation, and fatty acid biosynthesis, all of which have important roles in fiber development. Overall, our study shows the potential regulatory roles of miRNAs in cotton fiber development and the importance of miRNAs in regulating different cell types. This is helpful to design miRNA-based biotechnology for improving fiber quality and yield.',\n",
              "  'authors': ['Wang, Min',\n",
              "   'Sun, Runrun',\n",
              "   'Li, Chao',\n",
              "   'Wang, Qinglian',\n",
              "   'Zhang, Baohong'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Cotton Fiber',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Gossypium',\n",
              "   'MicroRNAs',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'RNA, Plant'],\n",
              "  'pmid': '28327647',\n",
              "  'title': 'MicroRNA expression profiles during cotton (Gossypium hirsutum L) fiber early development.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'The laser Doppler flowmetry (LDF) technique provides the monitoring of microvascular blood flow perfusion. However, LDF monitors based on fiber-optic transducers have the serious drawback of generating TRAnsient Signal High-values (TRASH) in signals. These TRASH correspond to artifacts for clinicians as they prevent interpretations of the signal when they are numerous. Moreover, TRASH exclude the possibility of direct signal processing and analyses. Therefore, in clinical routines, a human visual inspection of LDF signals is necessary to detect TRASH and to process the signals accordingly. This may be very time consuming. An algorithm able to localize TRASH automatically for their removal is therefore of interest. However, the development of such an algorithm is not an easy task as TRASH amplitude can be lower, higher, or in the same amplitude range as responses to stimuli such as post-occlusive hyperemia. The recently introduced empirical mode decomposition (EMD) has the advantage of splitting any kind of signal into fast and slow oscillations. Relying on these properties, the authors evaluate the possibility for EMD to localize TRASH automatically. For this purpose, LDF signals from 28 men of different ages are recorded at rest, during a vascular occlusion of 3 min, followed by a post-occlusive hyperemia. For each signal containing TRASH, the first intrinsic mode function obtained with the EMD is processed with a running window-based analysis in which a thresholding of the local maxima is carried out for the localization of TRASH. From the data, the use of a window width of 25 s is suggested. The results show effective and potential usefulness of this algorithm for an automatic localization of TRASH. Moreover, the method proposed has the advantage of being insensitive to the rapid increases of blood flow induced by post-occlusive hyperemia, which is of interest for clinicians. Because it is both local and fully data adaptive, EMD appears as an appealing processing technique for overcoming some of the limitations of the LDF.',\n",
              "  'authors': ['Humeau, Anne',\n",
              "   'Trzepizur, Wojciech',\n",
              "   'Rousseau, David',\n",
              "   'Chapeau-Blondeau, François',\n",
              "   'Abraham, Pierre'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Blood Flow Velocity',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Forearm',\n",
              "   'Humans',\n",
              "   'Laser-Doppler Flowmetry',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '19235369',\n",
              "  'title': 'Localization of transient signal high-values in laser Doppler flowmetry signals with an empirical mode decomposition.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'In an open surgery, identification of precise margins for curative tissue resection is performed by manual palpation. This is not the case for minimally invasive and robotic procedures, where tactile feedback is either distorted or not available. In this paper, we introduce the concept of intraoperative wireless tissue palpation. The wireless palpation probe (WPP) is a cylindrical device (15 mm in diameter, 60 mm in length) that can be deployed through a trocar incision and directly controlled by the surgeon to create a volumetric stiffness distribution map of the region of interest. This map can then be used to guide the tissue resection to minimize healthy tissue loss. The wireless operation prevents the need for a dedicated port and reduces the chance of instrument clashing in the operating field. The WPP is able to measure in real time the indentation pressure with a sensitivity of 34 Pa, the indentation depth with an accuracy of 0.68 mm, and the probe position with a maximum error of 11.3 mm in a tridimensional workspace. The WPP was assessed on the benchtop in detecting the local stiffness of two different silicone tissue simulators (elastic modulus ranging from 45 to 220 kPa), showing a maximum relative error below 5%. Then, in vivo trials were aimed to identify an agar-gel lump injected into a porcine liver and to assess the device usability within the frame of a laparoscopic procedure. The stiffness map created intraoperatively by the WPP was compared with a map generated ex vivo by a standard uniaxial material tester, showing less than 8% local stiffness error at the site of the lump.',\n",
              "  'authors': ['Beccani, Marco',\n",
              "   'Di Natali, Christian',\n",
              "   'Sliker, Levin J',\n",
              "   'Schoen, Jonathan A',\n",
              "   'Rentschler, Mark E',\n",
              "   'Valdastri, Pietro'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Animals',\n",
              "   'Feedback',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Liver',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Palpation',\n",
              "   'Robotics',\n",
              "   'Swine',\n",
              "   'Telemedicine',\n",
              "   'Wireless Technology'],\n",
              "  'pmid': '23974523',\n",
              "  'title': 'Wireless tissue palpation for intraoperative detection of lumps in the soft tissue.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['MQ'],\n",
              "  'ESSN': '1460-3659',\n",
              "  'IF_2013': '2.151',\n",
              "  'IF_2014': '2.351',\n",
              "  'IF_2015': '2.679',\n",
              "  'IF_2016': '2.256',\n",
              "  'IF_2017': '2.464',\n",
              "  'ISSN': '0306-3127',\n",
              "  'ISSN_online': '1460-3659',\n",
              "  'ISSN_print': '0306-3127',\n",
              "  'IsoAbbr': 'Soc Stud Sci',\n",
              "  'JrId': 23174,\n",
              "  'MedAbbr': 'Soc Stud Sci',\n",
              "  'NLMid': '7506743',\n",
              "  'Titre': 'Soc Stud Sci',\n",
              "  'abstract': \"This is a study of a well-known controversy in computer science, between artificial intelligence and neural networks researchers. It examines claims made by participants in the field that the controversy was triggered, shaped and finally resolved in connection to activities of state research agencies, particularly the US Department of Defense Advanced Research Projects Agency (ARPA). In contrast to these claims, this account concludes that, although ARPA's relations to the controversy among researchers were important, they were indirect. This study thus beings the analytical resources of studies of scientific controversy to bear on a domain which has so far gone largely unexamined in studies of 20th-century controversies - namely, state research policy and management. The account is based on contemporary documents, historical literatuare, and interviews with researchers and ARPA officials.\",\n",
              "  'authors': ['Guice, J'],\n",
              "  'categories': ['HISTORY & PHILOSOPHY OF SCIENCE'],\n",
              "  'journal': 'Social studies of science',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Computers',\n",
              "   'History, 20th Century',\n",
              "   'Nervous System',\n",
              "   'Neural Pathways',\n",
              "   'Research',\n",
              "   'State Medicine'],\n",
              "  'pmid': '11619937',\n",
              "  'title': 'Controversy and the state: lord ARPA and intelligent computing.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1911-9828',\n",
              "  'IF_2013': '0.742',\n",
              "  'IF_2014': '0.915',\n",
              "  'IF_2015': '1.179',\n",
              "  'IF_2016': '1.255',\n",
              "  'IF_2017': '1.327',\n",
              "  'ISSN': '0008-4174',\n",
              "  'ISSN_online': '1911-9828',\n",
              "  'ISSN_print': '0008-4174',\n",
              "  'IsoAbbr': 'Can J Occup Ther',\n",
              "  'JrId': 2736,\n",
              "  'MedAbbr': 'Can J Occup Ther',\n",
              "  'NLMid': '0406021',\n",
              "  'Titre': 'Can J Occup Ther',\n",
              "  'abstract': 'Currently, no evidence-based rehabilitation interventions are available for hyperkinetic movement disorders (HMD), including dyskinetic cerebral palsy (CP). Among these highly heterogeneous disorders, dystonia is the predominant disorder. The Cognitive Orientation to daily Occupational Performance (CO-OP) Approach-a task-oriented, performance-based intervention to enable participation-is currently being evaluated for its potential as an intervention option.|This paper reports the protocol for the second of two studies designed to evaluate the potential of CO-OP to improve functional outcomes for individuals with HMD following deep brain stimulation (DBS). This second study is a systematic replication across multiple treating therapists from multiple centres.|Systematic replications will be used across centres and treating therapists trained in the CO-OP, using a series of randomized multiple-baseline N-of-1 trials. Participants will be ages 6 to 21 years with HMD and DBS as indicated by the Manual Ability Classification System. Data collection will involve multiple data points collected at baseline, during intervention, and after intervention. The intervention will involve occupation-based goal setting followed by 10 individualized CO-OP sessions. The primary outcome measures are the Performance Quality Rating Scale and the Canadian Occupational Performance Measure. Outcome data will be plotted over time for each participant and supplemented with graph statistical analysis and estimate size effect for N-of-1 trials.|The results of this study will help to inform future training procedures and future clinical trials.',\n",
              "  'authors': ['Gimeno, Hortensia',\n",
              "   'Polatajko, Helene J',\n",
              "   'Cornelius, Victoria',\n",
              "   'Lin, Jean-Pierre',\n",
              "   'Brown, Richard G'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': \"Canadian journal of occupational therapy. Revue canadienne d'ergotherapie\",\n",
              "  'keywords': ['Activities of Daily Living',\n",
              "   'Adolescent',\n",
              "   'Cerebral Palsy',\n",
              "   'Child',\n",
              "   'Clinical Protocols',\n",
              "   'Deep Brain Stimulation',\n",
              "   'Dystonia',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Intelligence Tests',\n",
              "   'Male',\n",
              "   'Motor Skills',\n",
              "   'Occupational Therapy',\n",
              "   'Patient Compliance',\n",
              "   'Research Design',\n",
              "   'Self Care',\n",
              "   'Young Adult'],\n",
              "  'pmid': '29972046',\n",
              "  'title': \"Protocol for N-of-1 trials with replications across therapists for childhood-onset dystonia rehabilitation: Study 2: Protocole des essais à effectif unique avec répétitions par différents ergothérapeutes pour la réadaptation de la dystonie débutant dans l'enfance : Étude 2.\",\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'In 2004 we performed the first robot-assisted laparoscopic prostatectomy (RALP) at Aosta, Italy.|Data about our first 150 RALPs are presented.|Mean patient age was 65.2 (range 51-75) years; mean preoperative PSA level 7.9 (range 1.8-25.0) ng/ml. Nerve sparing was performed in 109 cases (72.7%) with lymph nodes dissection in 27 (18.0%). Operative time: patients 1-50, mean 213.3 (range 185-290) min; patients 51-100, 207.3 (range 185-335) min; patients 101-150, 171.7 (range 123-270) min. Two procedures were converted to open. Mean blood loss was 235 (range 20-1000) ml with two blood transfusions. One patient was reoperated for anastomotic leakage. Four patients were treated by colostomy and rectal suture for rectal injury (n = 3) or recto-urethral fistula (n = 1). The overall complication incidence was 13.3% (20 events), while mortality was nil.|The incidence of complications is slightly higher than in major pilot centres but it is comparable, hence RALP also appears feasible in our italian peripheral centre.',\n",
              "  'authors': ['Pierini, Paolo',\n",
              "   'Baldassarre, Emanuele',\n",
              "   'Lillaz, Beatrice',\n",
              "   'Viganò, Massimo',\n",
              "   'Merlo, Fabrizio'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Aged',\n",
              "   'Blood Loss, Surgical',\n",
              "   'Blood Volume',\n",
              "   'Colostomy',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Intraoperative Period',\n",
              "   'Italy',\n",
              "   'Laparoscopy',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Rectum',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '19035620',\n",
              "  'title': 'Robotic-assisted laparoscopic prostatectomy in Italy: perioperative data on the first 150 patients.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Automatic segmentation of brainstem gliomas and prediction of genotype (H3 K27M) mutation status based on magnetic resonance (MR) images are crucial but challenging tasks for computer-aided diagnosis in neurosurgery. In this paper, we present a novel cascaded deep convolutional neural network (CNN) to address these two challenging tasks simultaneously.|Our novel segmentation task contains two feature-fusion modules: the Gaussian-pyramid multiscale input features-fusion technique and the brainstem-region feature enhancement. The aim is to resolve very difficult problems in brainstem glioma segmentation. Our prediction model combines CNN features and support-vector-machine classifier to automatically predict genotypes without region-of-interest labeled-MR images and is learned jointly with the segmentation task. First, Gaussian-pyramid multiscale input feature fusion is added to our glioma-segmentation task to solve the problems of size variety and weak brainstem-gliomas boundaries. Second, the two feature-fusion modules provide both local and global contexts to retain higher frequency details for sharper tumor boundaries, handling the problem of the large variation of tumor shape, and volume resolution.|Experiments demonstrate that our cascaded CNN method achieves not only a good tumor segmentation result with a high Dice similarity coefficient of 77.03%, but also a competitive genotype prediction result with an average accuracy of 94.85% upon fivefold cross-validation.',\n",
              "  'authors': ['Liu, Jia',\n",
              "   'Chen, Fang',\n",
              "   'Pan, Changcun',\n",
              "   'Zhu, Mingyu',\n",
              "   'Zhang, Xinran',\n",
              "   'Zhang, Liwei',\n",
              "   'Liao, Hongen'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain Stem',\n",
              "   'Brain Stem Neoplasms',\n",
              "   'DNA Mutational Analysis',\n",
              "   'Genotyping Techniques',\n",
              "   'Glioma',\n",
              "   'Histones',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '29993462',\n",
              "  'title': 'A Cascaded Deep Convolutional Neural Network for Joint Segmentation and Genotype Prediction of Brainstem Gliomas.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Numerous therapeutic peptides do not enter the clinical trials just because of their high hemolytic activity. Recently, we developed a database, Hemolytik, for maintaining experimentally validated hemolytic and non-hemolytic peptides. The present study describes a web server and mobile app developed for predicting, and screening of peptides having hemolytic potency. Firstly, we generated a dataset HemoPI-1 that contains 552 hemolytic peptides extracted from Hemolytik database and 552 random non-hemolytic peptides (from Swiss-Prot). The sequence analysis of these peptides revealed that certain residues (e.g., L, K, F, W) and motifs (e.g., \"FKK\", \"LKL\", \"KKLL\", \"KWK\", \"VLK\", \"CYCR\", \"CRR\", \"RFC\", \"RRR\", \"LKKL\") are more abundant in hemolytic peptides. Therefore, we developed models for discriminating hemolytic and non-hemolytic peptides using various machine learning techniques and achieved more than 95% accuracy. We also developed models for discriminating peptides having high and low hemolytic potential on different datasets called HemoPI-2 and HemoPI-3. In order to serve the scientific community, we developed a web server, mobile app and JAVA-based standalone software (http://crdd.osdd.net/raghava/hemopi/). ',\n",
              "  'authors': ['Chaudhary, Kumardeep',\n",
              "   'Kumar, Ritesh',\n",
              "   'Singh, Sandeep',\n",
              "   'Tuknait, Abhishek',\n",
              "   'Gautam, Ankur',\n",
              "   'Mathur, Deepika',\n",
              "   'Anand, Priya',\n",
              "   'Varshney, Grish C',\n",
              "   'Raghava, Gajendra P S'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Computer Simulation',\n",
              "   'Databases, Protein',\n",
              "   'Hemolysis',\n",
              "   'Hemolytic Agents',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Mobile Applications',\n",
              "   'Peptides',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '26953092',\n",
              "  'title': 'A Web Server and Mobile App for Computing Hemolytic Potency of Peptides.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'Cockayne syndrome (CS) is a multisystem disorder with severe neurological symptoms. The majority of CS patients carry mutations in Cockayne syndrome group B (CSB), best known for its role in transcription-coupled nucleotide excision repair. Indeed, because various repair pathways are compromised in patient cells, CS is widely considered a genome instability syndrome. Here, we investigate the connection between the neuropathology of CS and dysregulation of gene expression. Transcriptome analysis of human fibroblasts revealed that even in the absence of DNA damage, CSB affects the expression of thousands of genes, many of which are neuronal genes. CSB is present in a significant subset of these genes, suggesting that regulation is direct, at the level of transcription. Importantly, reprogramming of CS fibroblasts to neuron-like cells is defective unless an exogenous CSB gene is introduced. Moreover, neuroblastoma cells from which CSB is depleted show defects in gene expression programs required for neuronal differentiation, and fail to differentiate and extend neurites. Likewise, neuron-like cells cannot be maintained without CSB. Finally, a number of disease symptoms may be explained by marked gene expression changes in the brain of patients with CS. Together, these data point to dysregulation of gene regulatory networks as a cause of the neurological symptoms in CS. ',\n",
              "  'authors': ['Wang, Yuming',\n",
              "   'Chakravarty, Probir',\n",
              "   'Ranes, Michael',\n",
              "   'Kelly, Gavin',\n",
              "   'Brooks, Philip J',\n",
              "   'Neilan, Edward',\n",
              "   'Stewart, Aengus',\n",
              "   'Schiavo, Giampietro',\n",
              "   'Svejstrup, Jesper Q'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Animals',\n",
              "   'Blotting, Western',\n",
              "   'Cell Line',\n",
              "   'Cell Line, Tumor',\n",
              "   'Cell Transdifferentiation',\n",
              "   'Cells, Cultured',\n",
              "   'Cockayne Syndrome',\n",
              "   'DNA Helicases',\n",
              "   'DNA Repair Enzymes',\n",
              "   'Fibroblasts',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'HEK293 Cells',\n",
              "   'Humans',\n",
              "   'Mice',\n",
              "   'Microscopy, Fluorescence',\n",
              "   'Neurons',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Poly-ADP-Ribose Binding Proteins',\n",
              "   'RNA Interference',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction'],\n",
              "  'pmid': '25249633',\n",
              "  'title': 'Dysregulation of gene expression as a cause of Cockayne syndrome neurological disease.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1873-4863',\n",
              "  'IF_2013': '2.884',\n",
              "  'IF_2014': '2.871',\n",
              "  'IF_2015': '2.667',\n",
              "  'IF_2016': '2.599',\n",
              "  'IF_2017': '2.533',\n",
              "  'ISSN': '0168-1656',\n",
              "  'ISSN_online': '1873-4863',\n",
              "  'ISSN_print': '0168-1656',\n",
              "  'IsoAbbr': 'J. Biotechnol.',\n",
              "  'JrId': 1552,\n",
              "  'MedAbbr': 'J Biotechnol',\n",
              "  'NLMid': '8411927',\n",
              "  'Titre': 'J. Biotechnol.',\n",
              "  'abstract': 'A new approach to nonlinear modeling and adaptive monitoring using fuzzy principal component regression (FPCR) is proposed and then applied to a real wastewater treatment plant (WWTP) data set. First, principal component analysis (PCA) is used to reduce the dimensionality of data and to remove collinearity. Second, the adaptive credibilistic fuzzy-c-means method is used to appropriately monitor diverse operating conditions based on the PCA score values. Then a new adaptive discrimination monitoring method is proposed to distinguish between a large process change and a simple fault. Third, a FPCR method is proposed, where the Takagi-Sugeno-Kang (TSK) fuzzy model is employed to model the relation between the PCA score values and the target output to avoid the over-fitting problem with original variables. Here, the rule bases, the centers and the widths of TSK fuzzy model are found by heuristic methods. The proposed FPCR method is applied to predict the output variable, the reduction of chemical oxygen demand in the full-scale WWTP. The result shows that it has the ability to model the nonlinear process and multiple operating conditions and is able to identify various operating regions and discriminate between a sustained fault and a simple fault (or abnormalities) occurring within the process data.',\n",
              "  'authors': ['Yoo, Chang Kyoo', 'Vanrolleghem, Peter A', 'Lee, In-Beum'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Journal of biotechnology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Environmental Monitoring',\n",
              "   'Fuzzy Logic',\n",
              "   'Models, Theoretical',\n",
              "   'Multivariate Analysis',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Principal Component Analysis',\n",
              "   'Water Purification'],\n",
              "  'pmid': '14511916',\n",
              "  'title': 'Nonlinear modeling and adaptive monitoring with fuzzy and multivariate statistical methods in biological wastewater treatment plants.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'Artificial activation of neural circuitry through electrical microstimulation and optogenetic techniques is important for both scientific discovery of circuit function and for engineered approaches to alleviate various disorders of the nervous system. However, evidence suggests that neural activity generated by artificial stimuli differs dramatically from normal circuit function, in terms of both the local neuronal population activity at the site of activation and the propagation to downstream brain structures. The precise nature of these differences and the implications for information processing remain unknown. Here, we used voltage-sensitive dye imaging of primary somatosensory cortex in the anesthetized rat in response to deflections of the facial vibrissae and electrical or optogenetic stimulation of thalamic neurons that project directly to the somatosensory cortex. Although the different inputs produced responses that were similar in terms of the average cortical activation, the variability of the cortical response was strikingly different for artificial versus sensory inputs. Furthermore, electrical microstimulation resulted in highly unnatural spatial activation of cortex, whereas optical input resulted in spatial cortical activation that was similar to that induced by sensory inputs. A thalamocortical network model suggested that observed differences could be explained by differences in the way in which artificial and natural inputs modulate the magnitude and synchrony of population activity. Finally, the variability structure in the response for each case strongly influenced the optimal inputs for driving the pathway from the perspective of an ideal observer of cortical activation when considered in the context of information transmission.|Artificial activation of neural circuitry through electrical microstimulation and optogenetic techniques is important for both scientific discovery and clinical translation. However, neural activity generated by these artificial means differs dramatically from normal circuit function, both locally and in the propagation to downstream brain structures. The precise nature of these differences and the implications for information processing remain unknown. The significance of this work is in quantifying the differences, elucidating likely mechanisms underlying the differences, and determining the implications for information processing.',\n",
              "  'authors': ['Millard, Daniel C',\n",
              "   'Whitmire, Clarissa J',\n",
              "   'Gollnick, Clare A',\n",
              "   'Rozell, Christopher J',\n",
              "   'Stanley, Garrett B'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Electric Stimulation',\n",
              "   'Female',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Optogenetics',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Somatosensory Cortex',\n",
              "   'Thalamus',\n",
              "   'Vibrissae'],\n",
              "  'pmid': '26609162',\n",
              "  'title': 'Electrical and Optical Activation of Mesoscale Neural Circuits with Implications for Coding.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1745-2422',\n",
              "  'IF_2013': '1.784',\n",
              "  'IF_2014': '1.683',\n",
              "  'IF_2015': '1.762',\n",
              "  'IF_2016': '2.228',\n",
              "  'IF_2017': '2.094',\n",
              "  'ISSN': '1743-4440',\n",
              "  'ISSN_online': '1745-2422',\n",
              "  'ISSN_print': '1743-4440',\n",
              "  'IsoAbbr': 'Expert Rev Med Devices',\n",
              "  'JrId': 32622,\n",
              "  'MedAbbr': 'Expert Rev Med Devices',\n",
              "  'NLMid': '101230445',\n",
              "  'Titre': 'Expert Rev Med Devices',\n",
              "  'abstract': 'Since the introduction of robotic technology, there have been significant changes to the field of gynecologic oncology. The number of minimally invasive procedures has drastically increased, with robotic procedures rising remarkably. With recent evidence suggesting that minimally invasive techniques should be the standard of care for early endometrial and cervical cancers, the push for new technology and advancements has continued. Several emerging robotic technologies have significant potential in the field of gynecologic oncology. The single-site robotic platform enables robotic surgery through a single incision; the Firefly camera detects the fluorescent dye indocyanine green, which may improve sensitivity in sentinel lymph node biopsy; and a robotic vessel-sealing device and stapler will continue to improve efficiency of the robotic surgeon. ',\n",
              "  'authors': ['Levinson, Kimberly L', 'Auer, Melinda', 'Escobar, Pedro F'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Expert review of medical devices',\n",
              "  'keywords': ['Female',\n",
              "   'Genital Neoplasms, Female',\n",
              "   'Humans',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics'],\n",
              "  'pmid': '24053253',\n",
              "  'title': 'Evolving technologies in robotic surgery for minimally invasive treatment of gynecologic cancers.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'The development of algorithms for designing artificial RNA sequences that fold into specific secondary structures has many potential biomedical and synthetic biology applications. To date, this problem remains computationally difficult, and current strategies to address it resort to heuristics and stochastic search techniques. The most popular methods consist of two steps: First a random seed sequence is generated; next, this seed is progressively modified (i.e. mutated) to adopt the desired folding properties. Although computationally inexpensive, this approach raises several questions such as (i) the influence of the seed; and (ii) the efficiency of single-path directed searches that may be affected by energy barriers in the mutational landscape. In this article, we present RNA-ensign, a novel paradigm for RNA design. Instead of taking a progressive adaptive walk driven by local search criteria, we use an efficient global sampling algorithm to examine large regions of the mutational landscape under structural and thermodynamical constraints until a solution is found. When considering the influence of the seeds and the target secondary structures, our results show that, compared to single-path directed searches, our approach is more robust, succeeds more often and generates more thermodynamically stable sequences. An ensemble approach to RNA design is thus well worth pursuing as a complement to existing approaches. RNA-ensign is available at http://csb.cs.mcgill.ca/RNAensign.',\n",
              "  'authors': ['Levin, Alex',\n",
              "   'Lis, Mieszko',\n",
              "   'Ponty, Yann',\n",
              "   \"O'Donnell, Charles W\",\n",
              "   'Devadas, Srinivas',\n",
              "   'Berger, Bonnie',\n",
              "   'Waldispühl, Jérôme'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Base Composition',\n",
              "   'Mutation',\n",
              "   'Nucleic Acid Conformation',\n",
              "   'RNA',\n",
              "   'Riboswitch',\n",
              "   'Software',\n",
              "   'Thermodynamics'],\n",
              "  'pmid': '22941632',\n",
              "  'title': 'A global sampling approach to designing and reengineering RNA secondary structures.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': 'The goal was to investigate the effects of choosing between different metrics in estimating the size of pulmonary nodules as a factor both of nodule characterization and of performance of computer aided detection systems, because the latter are always qualified with respect to a given size range of nodules.|This study used 265 whole-lung CT scans documented by the Lung Image Database Consortium (LIDC) using their protocol for nodule evaluation. Each inspected lesion was reviewed independently by four experienced radiologists who provided boundary markings for nodules larger than 3 mm. Four size metrics, based on the boundary markings, were considered: a unidimensional and two bidimensional measures on a single image slice and a volumetric measurement based on all the image slices. The radiologist boundaries were processed and those with four markings were analyzed to characterize the interradiologist variation, while those with at least one marking were used to examine the difference between the metrics.|The processing of the annotations found 127 nodules marked by all of the four radiologists and an extended set of 518 nodules each having at least one observation with three-dimensional sizes ranging from 2.03 to 29.4 mm (average 7.05 mm, median 5.71 mm). A very high interobserver variation was observed for all these metrics: 95% of estimated standard deviations were in the following ranges for the three-dimensional, unidimensional, and two bidimensional size metrics, respectively (in mm): 0.49-1.25, 0.67-2.55, 0.78-2.11, and 0.96-2.69. Also, a very large difference among the metrics was observed: 0.95 probability-coverage region widths for the volume estimation conditional on unidimensional, and the two bidimensional size measurements of 10 mm were 7.32, 7.72, and 6.29 mm, respectively.|The selection of data subsets for performance evaluation is highly impacted by the size metric choice. The LIDC plans to include a single size measure for each nodule in its database. This metric is not intended as a gold standard for nodule size; rather, it is intended to facilitate the selection of unique repeatable size limited nodule subsets.',\n",
              "  'authors': ['Reeves, Anthony P',\n",
              "   'Biancardi, Alberto M',\n",
              "   'Apanasovich, Tatiyana V',\n",
              "   'Meyer, Charles R',\n",
              "   'MacMahon, Heber',\n",
              "   'van Beek, Edwin J R',\n",
              "   'Kazerooni, Ella A',\n",
              "   'Yankelevitz, David',\n",
              "   'McNitt-Gray, Michael F',\n",
              "   'McLennan, Geoffrey',\n",
              "   'Armato, Samuel G',\n",
              "   'Henschke, Claudia I',\n",
              "   'Aberle, Denise R',\n",
              "   'Croft, Barbara Y',\n",
              "   'Clarke, Laurence P'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Calibration',\n",
              "   'Databases as Topic',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Knowledge Bases',\n",
              "   'Lung Neoplasms',\n",
              "   'Observer Variation',\n",
              "   'Radiology',\n",
              "   'Radiology Information Systems',\n",
              "   'Solitary Pulmonary Nodule',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '18035277',\n",
              "  'title': 'The Lung Image Database Consortium (LIDC): a comparison of different size metrics for pulmonary nodule measurements.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['QU'],\n",
              "  'ESSN': '2049-2618',\n",
              "  'IF_2013': '9',\n",
              "  'IF_2014': '9',\n",
              "  'IF_2015': '9',\n",
              "  'IF_2016': '8.496',\n",
              "  'IF_2017': '9.133',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2049-2618',\n",
              "  'IsoAbbr': 'Microbiome',\n",
              "  'JrId': 41826,\n",
              "  'MedAbbr': 'Microbiome',\n",
              "  'NLMid': '101615147',\n",
              "  'Titre': 'Microbiome',\n",
              "  'abstract': 'Microbiome studies often involve sequencing a marker gene to identify the microorganisms in samples of interest. Sequence classification is a critical component of this process, whereby sequences are assigned to a reference taxonomy containing known sequence representatives of many microbial groups. Previous studies have shown that existing classification programs often assign sequences to reference groups even if they belong to novel taxonomic groups that are absent from the reference taxonomy. This high rate of \"over classification\" is particularly detrimental in microbiome studies because reference taxonomies are far from comprehensive.|Here, we introduce IDTAXA, a novel approach to taxonomic classification that employs principles from machine learning to reduce over classification errors. Using multiple reference taxonomies, we demonstrate that IDTAXA has higher accuracy than popular classifiers such as BLAST, MAPSeq, QIIME, SINTAX, SPINGO, and the RDP Classifier. Similarly, IDTAXA yields far fewer over classifications on Illumina mock microbial community data when the expected taxa are absent from the training set. Furthermore, IDTAXA offers many practical advantages over other classifiers, such as maintaining low error rates across varying input sequence lengths and withholding classifications from input sequences composed of random nucleotides or repeats.|IDTAXA\\'s classifications may lead to different conclusions in microbiome studies because of the substantially reduced number of taxa that are incorrectly identified through over classification. Although misclassification error is relatively minor, we believe that many remaining misclassifications are likely caused by errors in the reference taxonomy. We describe how IDTAXA is able to identify many putative mislabeling errors in reference taxonomies, enabling training sets to be automatically corrected by eliminating spurious sequences. IDTAXA is part of the DECIPHER package for the R programming language, available through the Bioconductor repository or accessible online ( http://DECIPHER.codes ).',\n",
              "  'authors': ['Murali, Adithya', 'Bhargava, Aniruddha', 'Wright, Erik S'],\n",
              "  'categories': ['MICROBIOLOGY'],\n",
              "  'journal': 'Microbiome',\n",
              "  'keywords': ['Bacteria',\n",
              "   'DNA Barcoding, Taxonomic',\n",
              "   'Machine Learning',\n",
              "   'Metagenomics',\n",
              "   'Microbiota',\n",
              "   'Phylogeny',\n",
              "   'RNA, Ribosomal, 16S',\n",
              "   'Software'],\n",
              "  'pmid': '30092815',\n",
              "  'title': 'IDTAXA: a novel approach for accurate taxonomic classification of microbiome sequences.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['WC'],\n",
              "  'ESSN': '1938-1352',\n",
              "  'IF_2013': '1.688',\n",
              "  'IF_2014': '1.43',\n",
              "  'IF_2015': '1.043',\n",
              "  'IF_2016': '1.277',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0748-7711',\n",
              "  'ISSN_online': '1938-1352',\n",
              "  'ISSN_print': '0748-7711',\n",
              "  'IsoAbbr': 'J Rehabil Res Dev',\n",
              "  'JrId': 5189,\n",
              "  'MedAbbr': 'J Rehabil Res Dev',\n",
              "  'NLMid': '8410047',\n",
              "  'Titre': 'J Rehabil Res Dev',\n",
              "  'abstract': \"This article describes the design, validation, and application of a dynamic biomechanical model that assesses and monitors trajectory, position, orientation, force, and torque generated by upper-limb (UL) movement during robot-assisted therapy. The model consists of two links that represent the upper arm and forearm, with 5 degrees of freedom (DOF) for the shoulder and elbow joints. The model is a useful tool for enhancing the functionality of poststroke robot-assisted UL therapy. The individualized inertial segment parameters were based on anthropometric measurements. The model performed inverse dynamic analysis of UL movements to calculate reaction forces and moments acting about the 3-DOF shoulder and 2-DOF elbow joints. Real-time fused biofeedback of a 6-DOF force sensor and three-dimensional (3-D) pose sensors supported the model validation and application. The force sensor was mounted between the robot manipulator and the subject's wrist, while the 3-D pose sensors were fixed at specific positions on the subject's UL segments. The model input and output parameters were stored in the subject's database, which is part of the rehabilitation information system. We assigned 20 nondisabled subjects three different therapy exercises to test and validate the biomechanical model. We found that when the biomechanical model is taught an exercise, it can accurately predict a subject's actual UL joint angles and torques and confirm that the exercise is isolating the desired movement.\",\n",
              "  'authors': ['Abdullah, Hussein A',\n",
              "   'Tarry, Cole',\n",
              "   'Datta, Rahul',\n",
              "   'Mittal, Gauri S',\n",
              "   'Abderrahim, Mohamed'],\n",
              "  'categories': ['REHABILITATION'],\n",
              "  'journal': 'Journal of rehabilitation research and development',\n",
              "  'keywords': ['Adult',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Exercise Therapy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Monitoring, Physiologic',\n",
              "   'Range of Motion, Articular',\n",
              "   'Robotics',\n",
              "   'Stroke Rehabilitation',\n",
              "   'Upper Extremity',\n",
              "   'Young Adult'],\n",
              "  'pmid': '17551857',\n",
              "  'title': 'Dynamic biomechanical model for assessing and monitoring robot-assisted upper-limb therapy.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1750-2799',\n",
              "  'IF_2013': '7.782',\n",
              "  'IF_2014': '9.673',\n",
              "  'IF_2015': '9.646',\n",
              "  'IF_2016': '10.032',\n",
              "  'IF_2017': '12.423',\n",
              "  'ISSN': '1754-2189',\n",
              "  'ISSN_online': '1750-2799',\n",
              "  'ISSN_print': '1754-2189',\n",
              "  'IsoAbbr': 'Nat Protoc',\n",
              "  'JrId': 33047,\n",
              "  'MedAbbr': 'Nat Protoc',\n",
              "  'NLMid': '101284307',\n",
              "  'Titre': 'Nat Protoc',\n",
              "  'abstract': \"Electron cryotomography is the highest-resolution structural technique currently available that can be applied to unique objects such as flexible large protein complexes, irregular viruses, organelles and small cells. Specimens are preserved in a near-native, 'frozen-hydrated' state by vitrification. The thickness of the vitreous ice must be optimized for each specimen, and gold fiducials are typically added to facilitate image alignment. Here, we describe in detail our protocols for electron cryotomography sample preparation including (i) introduction of fiducial markers into the sample and (ii) sample vitrification. Because we almost exclusively use an automated, climate-controlled plunge-freezing device (the FEI Vitrobot) to vitrify our samples, we discuss its operation and parameters in detail. A session in which eight grids are prepared takes 1.5-2 h.\",\n",
              "  'authors': ['Iancu, Cristina V',\n",
              "   'Tivol, William F',\n",
              "   'Schooler, Jordan B',\n",
              "   'Dias, D Prabha',\n",
              "   'Henderson, Gregory P',\n",
              "   'Murphy, Gavin E',\n",
              "   'Wright, Elizabeth R',\n",
              "   'Li, Zhuo',\n",
              "   'Yu, Zhiheng',\n",
              "   'Briegel, Ariane',\n",
              "   'Gan, Lu',\n",
              "   'He, Yongning',\n",
              "   'Jensen, Grant J'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Nature protocols',\n",
              "  'keywords': ['Cryoelectron Microscopy',\n",
              "   'Cryopreservation',\n",
              "   'Robotics',\n",
              "   'Staining and Labeling',\n",
              "   'Tomography'],\n",
              "  'pmid': '17406539',\n",
              "  'title': 'Electron cryotomography sample preparation using the Vitrobot.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['BA'],\n",
              "  'ESSN': '1873-4529',\n",
              "  'IF_2013': '1.21',\n",
              "  'IF_2014': '1.194',\n",
              "  'IF_2015': '1.284',\n",
              "  'IF_2016': '1.677',\n",
              "  'IF_2017': '1.818',\n",
              "  'ISSN': '0952-8180',\n",
              "  'ISSN_online': '1873-4529',\n",
              "  'ISSN_print': '0952-8180',\n",
              "  'IsoAbbr': 'J Clin Anesth',\n",
              "  'JrId': 1601,\n",
              "  'MedAbbr': 'J Clin Anesth',\n",
              "  'NLMid': '8812166',\n",
              "  'Titre': 'J Clin Anesth',\n",
              "  'abstract': 'Insertion of drug-eluting stents is one of the strategies for treating patients with coronary artery disease. These patients can be a perioperative challenge in management as they need to be maintained on antiplatelet therapy to prevent stent thrombosis, which puts them at an increased risk for surgical bleeding. Recently revised guidelines on elective surgery following insertion of a drug-eluting stent recommend dual antiplatelet therapy for a period of twelve months. The management of a patient who presented for surgery more than two years after the insertion of a drug-eluting stent, and who developed in-stent thrombosis intraoperatively, is presented.',\n",
              "  'authors': ['Sharma, Aarti', 'Berkeley, Abiona'],\n",
              "  'categories': ['ANESTHESIOLOGY'],\n",
              "  'journal': 'Journal of clinical anesthesia',\n",
              "  'keywords': ['Aged',\n",
              "   'Coronary Artery Disease',\n",
              "   'Drug-Eluting Stents',\n",
              "   'Electrocardiography',\n",
              "   'Fatal Outcome',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Male',\n",
              "   'Platelet Aggregation Inhibitors',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Shock, Cardiogenic',\n",
              "   'Thrombosis',\n",
              "   'Ventricular Fibrillation'],\n",
              "  'pmid': '20006260',\n",
              "  'title': 'Intraoperative drug-eluting stent thrombosis in a patient undergoing robotic prostatectomy.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'The integration of speech recognition with natural language understanding raises issues of how to adapt natural language processing to the characteristics of spoken language; how to cope with errorful recognition output, including the use of natural language information to reduce recognition errors; and how to use information from the speech signal, beyond just the sequence of words, as an aid to understanding. This paper reviews current research addressing these questions in the Spoken Language Program sponsored by the Advanced Research Projects Agency (ARPA). I begin by reviewing some of the ways that spontaneous spoken language differs from standard written language and discuss methods of coping with the difficulties of spontaneous speech. I then look at how systems cope with errors in speech recognition and at attempts to use natural language information to reduce recognition errors. Finally, I discuss how prosodic information in the speech signal might be used to improve understanding.',\n",
              "  'authors': ['Moore, R C'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Cognition',\n",
              "   'Humans',\n",
              "   'Language',\n",
              "   'Linguistics',\n",
              "   'Speech',\n",
              "   'Speech Perception',\n",
              "   'User-Computer Interface',\n",
              "   'Vocabulary'],\n",
              "  'pmid': '7479813',\n",
              "  'title': 'Integration of speech with natural language understanding.',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1878-5506',\n",
              "  'IF_2013': '3.1',\n",
              "  'IF_2014': '3.154',\n",
              "  'IF_2015': '3.339',\n",
              "  'IF_2016': '3.391',\n",
              "  'IF_2017': '3.395',\n",
              "  'ISSN': '1389-9457',\n",
              "  'ISSN_online': '1878-5506',\n",
              "  'ISSN_print': '1389-9457',\n",
              "  'IsoAbbr': 'Sleep Med.',\n",
              "  'JrId': 21657,\n",
              "  'MedAbbr': 'Sleep Med',\n",
              "  'NLMid': '100898759',\n",
              "  'Titre': 'Sleep Med.',\n",
              "  'abstract': 'Atrial fibrillation (AF) is the most common type of sustained cardiac arrhythmia encountered in clinical practice, and its burden is expected to increase in most developed countries over the next few decades. Because AF can be silent, it is often not diagnosed until an AF-related complication occurs, such as stroke. AF is also associated with increased risk of heart failure, lower quality of life, and death. Anticoagulation has been shown to dramatically decrease embolic risk in the setting of atrial fibrillation, resulting in growing interest in early detection of previously undiagnosed AF. Newly developed monitoring devices have improved the detection of AF and have been recommended in guidelines for screening of AF in individuals aged 65 years and over. While screening is currently targeted to these older individuals, younger patients with obstructive sleep apnoea (OSA) are at higher risk of AF and stroke than the general population, indicating a need for targeted early detection of AF in this group. Compared to individuals without OSA, those with OSA are four times more likely to develop AF, and the risk of AF is strongly associated with OSA severity. The overall prevalence of AF among individuals with OSA remains unknown because of limitations related to study design and to the conventional methods previously used for AF detection. Recent and emerging technological advances may improve the detection of undiagnosed AF in high-risk population groups, such as those with OSA. In this clinical review, we discuss the methods of screening for AF and the applications of newer technologies for AF detection in patients with OSA. We conclude the review with a brief description of our research agenda in this area.',\n",
              "  'authors': ['Gahungu, Nestor',\n",
              "   'Judkins, Christopher',\n",
              "   'Gabbay, Eli',\n",
              "   'Playford, David'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Sleep medicine',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30954787',\n",
              "  'title': 'Advances in screening for undiagnosed atrial fibrillation for stroke prevention and implications for patients with obstructive sleep apnoea: a literature review and research agenda.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1576-8260',\n",
              "  'IF_2013': '0.309',\n",
              "  'IF_2014': '0.309',\n",
              "  'IF_2015': '0.307',\n",
              "  'IF_2016': '0.323',\n",
              "  'IF_2017': '0.465',\n",
              "  'ISSN': '0004-0614',\n",
              "  'ISSN_online': '1576-8260',\n",
              "  'ISSN_print': '0004-0614',\n",
              "  'IsoAbbr': 'Arch. Esp. Urol.',\n",
              "  'JrId': 738,\n",
              "  'MedAbbr': 'Arch Esp Urol',\n",
              "  'NLMid': '0064757',\n",
              "  'Titre': 'Arch. Esp. Urol.',\n",
              "  'abstract': 'Radical prostatectomy is currently the standard of care for localized prostate cancer. In the last decade, the minimally invasive surgery, especially the robotic surgery has  been growing and open techniques are less frequent performed. A non-systematic review of the literature is performed, highlighting the current situation of the perineal radical prostatectomy in the minimally invasive era, its  indications, and functional and oncological outcomes. Radical perineal prostatectomy, when compared with other surgical approaches, still experience favorable outcomes. Urologist might be abandoning an underused surgical  approach.',\n",
              "  'authors': ['Rioja, Jorge', 'Rincon Mayans, Anibal', 'Parra, Raul O'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Archivos espanoles de urologia',\n",
              "  'keywords': ['Humans',\n",
              "   'Length of Stay',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Obesity',\n",
              "   'Postoperative Complications',\n",
              "   'Prognosis',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Quality of Life',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '23117680',\n",
              "  'title': 'Perineal radical prostatectomy in the minimally invasive era.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1530-8561',\n",
              "  'IF_2013': '7.768',\n",
              "  'IF_2014': '7.911',\n",
              "  'IF_2015': '7.457',\n",
              "  'IF_2016': '8.008',\n",
              "  'IF_2017': '8.636',\n",
              "  'ISSN': '0009-9147',\n",
              "  'ISSN_online': '1530-8561',\n",
              "  'ISSN_print': '0009-9147',\n",
              "  'IsoAbbr': 'Clin. Chem.',\n",
              "  'JrId': 2992,\n",
              "  'MedAbbr': 'Clin Chem',\n",
              "  'NLMid': '9421549',\n",
              "  'Titre': 'Clin. Chem.',\n",
              "  'abstract': 'Robotic systems specifically designed for the automation of laboratory tasks are now available commercially. Equipped with computer, analytical hardware, and supporting software, these devices may soon revolutionize the concept of the clinical laboratory and usher in a new era in laboratory testing. We review the types of robots and motion-control software currently available and discuss examples of their applications that extend across many analytical areas. Several ongoing projects are concerned with the systematic integration of robotic devices with other laboratory automation. The integrated robotic laboratories emerging from this work portend a bright future for robotic automation. Many challenges remain, however, in training the individuals needed to develop and manage robotic laboratories, and in making this new technology cost-efficient.',\n",
              "  'authors': ['Felder, R A',\n",
              "   'Boyd, J C',\n",
              "   'Margrey, K',\n",
              "   'Holman, W',\n",
              "   'Savory, J'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinical chemistry',\n",
              "  'keywords': ['Laboratories, Hospital', 'Robotics', 'United States'],\n",
              "  'pmid': '2208683',\n",
              "  'title': 'Robotics in the medical laboratory.',\n",
              "  'year': '1990'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TQ'],\n",
              "  'ESSN': '1098-4275',\n",
              "  'IF_2013': '5.297',\n",
              "  'IF_2014': '5.473',\n",
              "  'IF_2015': '5.196',\n",
              "  'IF_2016': '5.705',\n",
              "  'IF_2017': '5.515',\n",
              "  'ISSN': '0031-4005',\n",
              "  'ISSN_online': '1098-4275',\n",
              "  'ISSN_print': '0031-4005',\n",
              "  'IsoAbbr': 'Pediatrics',\n",
              "  'JrId': 6405,\n",
              "  'MedAbbr': 'Pediatrics',\n",
              "  'NLMid': '0376422',\n",
              "  'Titre': 'Pediatrics',\n",
              "  'abstract': 'The outcome of prolonged, nontraumatic coma (greater than five days) in 16 children is reviewed to determine operational clinical parameters which may assist in both clinical decision making and counseling of parents. The children were evaluated by physical and neurologic examinations and school reports one to five years after coma. Six children are normal, six have minor handicaps (attention deficit disorder, minor motor disorders, mild retardation, personality disorders), and four have sustained major sequelae (severe retardation, uncontrolled seizures, blindness). Anoxia, as an etiology of coma and the need for assisted ventilation, were significant indicators of a less than normal outcome. Our results suggest that elevated intracranial pressure of greater than two days duration and deep coma of greater than two weeks were indicators of an abnormal outcome. In view of the improving technical capability to care for these children, but limited past experience, clearly defined and uniform criteria are needed both to assess children during coma and to evaluate them upon recovery.',\n",
              "  'authors': ['Margolis, L H', 'Shaywitz, B A'],\n",
              "  'categories': ['PEDIATRICS'],\n",
              "  'journal': 'Pediatrics',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Child',\n",
              "   'Child, Preschool',\n",
              "   'Coma',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Infant',\n",
              "   'Intelligence',\n",
              "   'Intracranial Pressure',\n",
              "   'Nervous System Diseases',\n",
              "   'Prognosis',\n",
              "   'Respiration, Artificial',\n",
              "   'Seizures',\n",
              "   'Time Factors'],\n",
              "  'pmid': '7360533',\n",
              "  'title': 'The outcome of prolonged coma in childhood.',\n",
              "  'year': '1980'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '2469-9322',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '0.489',\n",
              "  'IF_2017': '0.489',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2469-9322',\n",
              "  'IsoAbbr': 'Comput Assist Surg (Abingdon)',\n",
              "  'JrId': 45174,\n",
              "  'MedAbbr': 'Comput Assist Surg (Abingdon)',\n",
              "  'NLMid': '101681550',\n",
              "  'Titre': 'Comput Assist Surg (Abingdon)',\n",
              "  'abstract': 'The issue of an automated approach for detecting cervical cancer is proposed to improve the accuracy of recognition. Firstly, the cervical cancer histology source images are needed to use image preprocessing for reducing the impact brought by noise of images as well as the impact on subsequent precise feature extraction brought by irrelevant background. Secondly, the images are grouped into ten vertical images and the information of texture feature is extracted by Grey Level Co-occurrence Matrix (GLCM). GLCM is an effective tool to analyze the features of texture. The textures of different diseases in the source image of Cervical Cancer Histology (such as contrast, correlation, entropy, uniformity and energy, etc.) can all be obtained in this way. Thirdly, the image is segmented by using K-means clustering and Marker-controlled watershed Algorithm. And each vertical image is divided into three layers to calculate the areas of different layers. Based on GLCM and lesion area features, the tissues are investigated with segmentation by using Support Vector Machine (SVM) method. Finally, the results show that it is effective and feasible to recognize cervical cancer by automated approach and verified by experiment.',\n",
              "  'authors': ['Wei, Lisheng', 'Gan, Quan', 'Ji, Tao'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer assisted surgery (Abingdon, England)',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Algorithms',\n",
              "   'Carcinoma, Squamous Cell',\n",
              "   'Cluster Analysis',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Histological Techniques',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Immunohistochemistry',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Support Vector Machine',\n",
              "   'Uterine Cervical Neoplasms'],\n",
              "  'pmid': '29037083',\n",
              "  'title': 'Cervical cancer histology image identification method based on texture and lesion area features.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Recent studies have suggested that some neural computational mechanisms are based on the fine temporal structure of spiking activity. However, less effort has been devoted to investigating the evolutionary aspects of such mechanisms. In this paper we explore the issue of temporal neural computation from an evolutionary point of view, using a genetic simulation of the evolutionary development of neural systems. We evolve neural systems in an environment with selective pressure based on mate finding, and examine the temporal aspects of the evolved systems. In repeating evolutionary sessions, there was a significant increase during evolution in the mutual information between the evolved agent's temporal neural representation and the external environment. In ten different simulated evolutionary sessions, there was an increased effect of time-related neural ablations on the agents' fitness. These results suggest that in some fitness landscapes the emergence of temporal elements in neural computation is almost inevitable. Future research using similar evolutionary simulations may shed new light on various biological mechanisms.\",\n",
              "  'authors': ['Yerushalmi, Uri', 'Teicher, Mina'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biological Evolution',\n",
              "   'Cell Differentiation',\n",
              "   'Chromosome Mapping',\n",
              "   'Computer Simulation',\n",
              "   'Evolution, Molecular',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Models, Genetic',\n",
              "   'Models, Statistical',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Normal Distribution',\n",
              "   'Time Factors'],\n",
              "  'pmid': '18382654',\n",
              "  'title': 'Inevitable evolutionary temporal elements in neural processing: a study based on evolutionary simulations.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': 'The aim of this study was to investigate whether using a fractal dimension as an objective index (quantitative measure) to assess and control the \"visual\" or \"texture\" similarity of reference-image regions selected by a content-based image retrieval (CBIR) scheme would (or would not) affect the performance of the scheme in classification between image regions depicting suspicious breast masses.|An image data set depicting 1500 verified mass regions and 1500 false-positive mass regions was used. Fourteen morphologic and intensity distribution features and a fractal dimension were computed. A CBIR scheme using a k-nearest neighbor classifier was applied, and two experiments were conducted. In the first experiment, the CBIR scheme was evaluated using all 15 features. In the second experiment, the fractal dimension was used as a prescreening feature to guide the CBIR scheme to search for the most similar reference images that had similar measures in the fractal dimension.|The CBIR scheme achieved classification performance with areas under the receiver-operating characteristic curve of 0.857 (95% confidence interval [CI], 0.844-0.870) using 14 features and 0.866 (95% CI, 0.853-0.879) after adding the fractal dimension (P = .005 for both results). After using the fractal dimension as a prescreening feature, the CBIR scheme achieved an area under the receiver-operating characteristic curve of 0.851 (95% CI, 0.837-0.864), without a significant difference from the previous result using the original 14 features (P = .120). The difference of fractal dimension values between the selected similar reference images was reduced by 56.7%, indicating improvement in image texture similarity. In addition, more than half of references were discarded early, without similarity comparisons, indicating improvement in searching efficiency.|This study demonstrated the feasibility of applying a fractal dimension as an objective (quantitative) and efficient search index to assess and maintain the texture similarity of reference mass regions selected by a CBIR scheme without reducing the scheme\\'s performance in classifying suspicious breast masses.',\n",
              "  'authors': ['Park, Sang Cheol', 'Wang, Xiao-Hui', 'Zheng, Bin'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Breast Neoplasms',\n",
              "   'Database Management Systems',\n",
              "   'Female',\n",
              "   'Fractals',\n",
              "   'Humans',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Male',\n",
              "   'Mammography',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Radiology Information Systems',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '19524455',\n",
              "  'title': 'Assessment of performance improvement in content-based medical image retrieval schemes using fractal dimension.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'The combination of predictive data-driven models with frequent glucose measurements may provide for an early warning of impending glucose excursions and proactive regulatory interventions for diabetes patients. However, from a modeling perspective, before the benefits of such a strategy can be attained, we must first be able to quantitatively characterize the behavior of the model coefficients as well as the model predictions as a function of prediction horizon. We need to determine if the model coefficients reflect viable physiologic dependencies of the individual glycemic measurements and whether the model is stable with respect to small changes in noise levels, leading to accurate near-future predictions with negligible time lag. We assessed the behavior of linear autoregressive data-driven models developed under three possible modeling scenarios, using continuous glucose measurements of nine subjects collected on a minute-by-minute basis for approximately 5 days. Simulation results indicated that stable and accurate models for near-future glycemic predictions (< 60 min) with clinically acceptable time lags are attained only when the raw glucose measurements are smoothed and the model coefficients are regularized. This study provides a starting point for further needed investigations before real-time deployment can be considered.',\n",
              "  'authors': ['Gani, Adiwinata',\n",
              "   'Gribok, Andrei V',\n",
              "   'Rajaraman, Srinivasan',\n",
              "   'Ward, W Kenneth',\n",
              "   'Reifman, Jaques'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Blood Glucose',\n",
              "   'Computer Simulation',\n",
              "   'Diabetes Mellitus',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Models, Biological',\n",
              "   'Monitoring, Ambulatory',\n",
              "   'Predictive Value of Tests',\n",
              "   'Subcutaneous Tissue'],\n",
              "  'pmid': '19272928',\n",
              "  'title': 'Predicting subcutaneous glucose concentration in humans: data-driven glucose modeling.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1095-9203',\n",
              "  'IF_2013': '31.477',\n",
              "  'IF_2014': '33.611',\n",
              "  'IF_2015': '34.661',\n",
              "  'IF_2016': '37.205',\n",
              "  'IF_2017': '41.058',\n",
              "  'ISSN': '0036-8075',\n",
              "  'ISSN_online': '1095-9203',\n",
              "  'ISSN_print': '0036-8075',\n",
              "  'IsoAbbr': 'Science',\n",
              "  'JrId': 7473,\n",
              "  'MedAbbr': 'Science',\n",
              "  'NLMid': '0404511',\n",
              "  'Titre': 'Science',\n",
              "  'abstract': 'In coming to understand the world-in learning concepts, acquiring language, and grasping causal relations-our minds make inferences that appear to go far beyond the data available. How do we do it? This review describes recent approaches to reverse-engineering human learning and cognitive development and, in parallel, engineering more humanlike machine learning systems. Computational models that perform probabilistic inference over hierarchies of flexibly structured representations can address some of the deepest questions about the nature and origins of human thought: How does abstract knowledge guide learning and reasoning from sparse data? What forms does our knowledge take, across different domains and tasks? And how is that abstract knowledge itself acquired?',\n",
              "  'authors': ['Tenenbaum, Joshua B',\n",
              "   'Kemp, Charles',\n",
              "   'Griffiths, Thomas L',\n",
              "   'Goodman, Noah D'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Science (New York, N.Y.)',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Bayes Theorem',\n",
              "   'Cognition',\n",
              "   'Concept Formation',\n",
              "   'Humans',\n",
              "   'Knowledge',\n",
              "   'Learning',\n",
              "   'Models, Statistical',\n",
              "   'Theory of Mind',\n",
              "   'Thinking'],\n",
              "  'pmid': '21393536',\n",
              "  'title': 'How to grow a mind: statistics, structure, and abstraction.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DX'],\n",
              "  'ESSN': '1520-4804',\n",
              "  'IF_2013': '5.48',\n",
              "  'IF_2014': '5.447',\n",
              "  'IF_2015': '5.589',\n",
              "  'IF_2016': '6.259',\n",
              "  'IF_2017': '6.253',\n",
              "  'ISSN': '0022-2623',\n",
              "  'ISSN_online': '1520-4804',\n",
              "  'ISSN_print': '0022-2623',\n",
              "  'IsoAbbr': 'J. Med. Chem.',\n",
              "  'JrId': 4938,\n",
              "  'MedAbbr': 'J Med Chem',\n",
              "  'NLMid': '9716531',\n",
              "  'Titre': 'J. Med. Chem.',\n",
              "  'abstract': 'Sesquiterpene lactones are the active components of a variety of medicinal plants from the Asteraceae family. They possess biological activities such as the inhibition of NF-kappaB and the release inhibition of the vasoactive serotonin. On the basis of a data set of 54 SLs, we report the development of a quantitative model for the prediction of serotonin release inhibition. Comparing this model with a previous investigation of the target NF-kappaB, structural features necessary for specific compounds could be acquired. Atomic properties encoded by radial distribution function and molecular surface potentials encoded by autocorrelation were used as descriptors. Whereas some descriptors describe the structural requirements for both activities, other descriptors can be used to decide whether an SL is more active to NF-kappaB or to serotonin release. Again, counter propagation neural networks proved to be a valuable tool to establish structure-activity relationships that are necessary for the search for and optimization of lead structures.',\n",
              "  'authors': ['Wagner, Steffen',\n",
              "   'Arce, Raul',\n",
              "   'Murillo, Renato',\n",
              "   'Terfloth, Lothar',\n",
              "   'Gasteiger, Johann',\n",
              "   'Merfort, Irmgard'],\n",
              "  'categories': ['CHEMISTRY, MEDICINAL'],\n",
              "  'journal': 'Journal of medicinal chemistry',\n",
              "  'keywords': ['Animals',\n",
              "   'Blood Platelets',\n",
              "   'Cattle',\n",
              "   'Electricity',\n",
              "   'In Vitro Techniques',\n",
              "   'Lactones',\n",
              "   'Models, Molecular',\n",
              "   'NF-kappa B',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Serotonin',\n",
              "   'Serotonin Antagonists',\n",
              "   'Sesquiterpenes'],\n",
              "  'pmid': '18271521',\n",
              "  'title': \"Neural networks as valuable tools to differentiate between sesquiterpene lactones' inhibitory activity on serotonin release and on NF-kappaB.\",\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'Image jitters occur in the video of the autonomous robot moving on bricks road, which will reduce robot operation precision based on vision. In order to compensate the image jitters, the affine transformation kinematics were established for obtaining the six image motion parameters. The feature point pair detecting method was designed based on Eigen-value of the feature windows gradient matrix, and the motion parameters equation was solved using the least square method and the matching point pairs got based on the optical flow. The condition number of coefficient matrix was proposed to quantificationally analyse the effect of matching errors on parameters solving errors. Kalman filter was adopted to smooth image motion parameters. Computing cases show that more point pairs are beneficial for getting more precise motion parameters. The integrated jitters compensation software was developed with feature points detecting in subwindow. And practical experiments were conducted on two mobile robots. Results show that the compensation costing time is less than frame sample time and Kalman filter is valid for robot vision jitters compensation. ',\n",
              "  'authors': ['Wang, B R', 'Jin, Y L', 'Shao, D L', 'Xu, Y'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Models, Theoretical', 'Motion', 'Robotics', 'Vision, Ocular'],\n",
              "  'pmid': '24600320',\n",
              "  'title': 'Design of jitter compensation algorithm for robot vision based on optical flow and Kalman filter.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1528-1140',\n",
              "  'IF_2013': '7.188',\n",
              "  'IF_2014': '8.327',\n",
              "  'IF_2015': '8.569',\n",
              "  'IF_2016': '8.98',\n",
              "  'IF_2017': '9.203',\n",
              "  'ISSN': '0003-4932',\n",
              "  'ISSN_online': '1528-1140',\n",
              "  'ISSN_print': '0003-4932',\n",
              "  'IsoAbbr': 'Ann. Surg.',\n",
              "  'JrId': 646,\n",
              "  'MedAbbr': 'Ann Surg',\n",
              "  'NLMid': '0372354',\n",
              "  'Titre': 'Ann. Surg.',\n",
              "  'abstract': \"Most risk assessment tools assume that the impact of risk factors is linear and cumulative. Using novel machine-learning techniques, we sought to design an interactive, nonlinear risk calculator for Emergency Surgery (ES).|All ES patients in the American College of Surgeons National Surgical Quality Improvement Program (ACS-NSQIP) 2007 to 2013 database were included (derivation cohort). Optimal Classification Trees (OCT) were leveraged to train machine-learning algorithms to predict postoperative mortality, morbidity, and 18 specific complications (eg, sepsis, surgical site infection). Unlike classic heuristics (eg, logistic regression), OCT is adaptive and reboots itself with each variable, thus accounting for nonlinear interactions among variables. An application [Predictive OpTimal Trees in Emergency Surgery Risk (POTTER)] was then designed as the algorithms' interactive and user-friendly interface. POTTER performance was measured (c-statistic) using the 2014 ACS-NSQIP database (validation cohort) and compared with the American Society of Anesthesiologists (ASA), Emergency Surgery Score (ESS), and ACS-NSQIP calculators' performance.|Based on 382,960 ES patients, comprehensive decision-making algorithms were derived, and POTTER was created where the provider's answer to a question interactively dictates the subsequent question. For any specific patient, the number of questions needed to predict mortality ranged from 4 to 11. The mortality c-statistic was 0.9162, higher than ASA (0.8743), ESS (0.8910), and ACS (0.8975). The morbidity c-statistics was similarly the highest (0.8414).|POTTER is a highly accurate and user-friendly ES risk calculator with the potential to continuously improve accuracy with ongoing machine-learning. POTTER might prove useful as a tool for bedside preoperative counseling of ES patients and families.\",\n",
              "  'authors': ['Bertsimas, Dimitris',\n",
              "   'Dunn, Jack',\n",
              "   'Velmahos, George C',\n",
              "   'Kaafarani, Haytham M A'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Annals of surgery',\n",
              "  'keywords': ['Decision Support Techniques',\n",
              "   'Emergencies',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Postoperative Complications',\n",
              "   'Predictive Value of Tests',\n",
              "   'Risk Assessment',\n",
              "   'Surgical Procedures, Operative',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '30124479',\n",
              "  'title': 'Surgical Risk Is Not Linear: Derivation and Validation of a Novel, User-friendly, and Machine-learning-based Predictive OpTimal Trees in Emergency Surgery Risk (POTTER) Calculator.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YO'],\n",
              "  'ESSN': '1096-0929',\n",
              "  'IF_2013': '4.478',\n",
              "  'IF_2014': '3.854',\n",
              "  'IF_2015': '3.88',\n",
              "  'IF_2016': '4.081',\n",
              "  'IF_2017': '4.181',\n",
              "  'ISSN': '1096-6080',\n",
              "  'ISSN_online': '1096-0929',\n",
              "  'ISSN_print': '1096-6080',\n",
              "  'IsoAbbr': 'Toxicol. Sci.',\n",
              "  'JrId': 20941,\n",
              "  'MedAbbr': 'Toxicol Sci',\n",
              "  'NLMid': '9805461',\n",
              "  'Titre': 'Toxicol. Sci.',\n",
              "  'abstract': 'Chemical sensitization is an adverse immunologic response to chemical substances, inducing hypersensitivity in exposed individuals. Identifying chemical sensitizers is of great importance for chemical, pharmaceutical, and cosmetic industries, in order to prevent the use of sensitizers in consumer products. Historically, chemical sensitizers have been assessed mainly by in vivo methods, however, recently enforced European legislations urge and promote the development of animal-free test methods able to predict chemical sensitizers. Recently, we presented a predictive biomarker signature in the myeloid cell line MUTZ-3, for assessment of skin sensitizers. The identified genomic biomarkers were found to be involved in immunologically relevant pathways, induced by recognition of foreign substances and regulating dendritic cell maturation and cytoprotective mechanisms. We have developed the usage of this biomarker signature into a novel in vitro assay for assessment of chemical sensitizers, called Genomic Allergen Rapid Detection (GARD). The assay is based on chemical stimulation of MUTZ-3 cultures, using the compounds to be assayed as stimulatory agents. The readout of the assay is a transcriptional quantification of the genomic predictors, collectively termed the GARD Prediction Signature (GPS), using a complete genome expression array. Compounds are predicted as either sensitizers or nonsensitizers by a Support Vector Machine model. In this report, we provide a proof of concept for the functionality of the GARD assay by describing the classification of 26 blinded and 11 nonblinded chemicals as sensitizers or nonsensitizers. Based on these classifications, the accuracy, sensitivity, and specificity of the assay were estimated to 89, 89, and 88%, respectively. ',\n",
              "  'authors': ['Johansson, Henrik',\n",
              "   'Rydnert, Frida',\n",
              "   'Kühnl, Jochen',\n",
              "   'Schepky, Andreas',\n",
              "   'Borrebaeck, Carl',\n",
              "   'Lindstedt, Malin'],\n",
              "  'categories': ['TOXICOLOGY'],\n",
              "  'journal': 'Toxicological sciences : an official journal of the Society of Toxicology',\n",
              "  'keywords': ['Allergens',\n",
              "   'Animal Testing Alternatives',\n",
              "   'Biological Assay',\n",
              "   'Biomarkers',\n",
              "   'Cell Line, Tumor',\n",
              "   'Cell Survival',\n",
              "   'Dermatitis, Allergic Contact',\n",
              "   'Humans',\n",
              "   'Predictive Value of Tests',\n",
              "   'Principal Component Analysis',\n",
              "   'Reproducibility of Results',\n",
              "   'Support Vector Machine',\n",
              "   'Transcription, Genetic'],\n",
              "  'pmid': '24675087',\n",
              "  'title': 'Genomic allergen rapid detection in-house validation--a proof of concept.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1873-3573',\n",
              "  'IF_2013': '3.511',\n",
              "  'IF_2014': '3.545',\n",
              "  'IF_2015': '4.035',\n",
              "  'IF_2016': '4.162',\n",
              "  'IF_2017': '4.244',\n",
              "  'ISSN': '0039-9140',\n",
              "  'ISSN_online': '1873-3573',\n",
              "  'ISSN_print': '0039-9140',\n",
              "  'IsoAbbr': 'Talanta',\n",
              "  'JrId': 22720,\n",
              "  'MedAbbr': 'Talanta',\n",
              "  'NLMid': '2984816R',\n",
              "  'Titre': 'Talanta',\n",
              "  'abstract': 'The fingerprinting capacity of thin layer chromatography (TLC) and image analysis in the case of propolis samples collected in different area in Romania has been investigated. Fuzzy divisive hierarchical clustering approach was used as a powerful tool of samples discrimination and fingerprinting according to the geographical origin and local flora. The fuzzy partition and patterns obtained by membership degrees plot were in a very good agreement with floral origin and geographic location of Romanian propolis samples, and clearly illustrate the fuzziness concerning their similarities and difference. The results obtained strongly support that TLC via image analysis can be successfully employed in the fingerprinting methodologies if they are combined with appropriate fuzzy clustering method. The method developed in this paper might be also extended in the authenticity and origin control of fruits, herbs or derived products.',\n",
              "  'authors': ['Sârbu, Costel', 'Moţ, Augustin Cătălin'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Talanta',\n",
              "  'keywords': ['Chromatography, Thin Layer',\n",
              "   'Cluster Analysis',\n",
              "   'Ecosystem',\n",
              "   'Fuzzy Logic',\n",
              "   'Geography',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Propolis',\n",
              "   'Romania',\n",
              "   'Time Factors'],\n",
              "  'pmid': '21726746',\n",
              "  'title': 'Ecosystem discrimination and fingerprinting of Romanian propolis by hierarchical fuzzy clustering and image analysis of TLC patterns.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'We propose that catastrophic events are \"outliers\" with statistically different properties than the rest of the population and result from mechanisms involving amplifying critical cascades. We describe a unifying approach for modeling and predicting these catastrophic events or \"ruptures,\" that is, sudden transitions from a quiescent state to a crisis. Such ruptures involve interactions between structures at many different scales. Applications and the potential for prediction are discussed in relation to the rupture of composite materials, great earthquakes, turbulence, and abrupt changes of weather regimes, financial crashes, and human parturition (birth). Future improvements will involve combining ideas and tools from statistical physics and artificial/computational intelligence, to identify and classify possible universal structures that occur at different scales, and to develop application-specific methodologies to use these structures for prediction of the \"crises\" known to arise in each application of interest. We live on a planet and in a society with intermittent dynamics rather than a state of equilibrium, and so there is a growing and urgent need to sensitize students and citizens to the importance and impacts of ruptures in their multiple forms.',\n",
              "  'authors': ['Sornette, Didier'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Climate',\n",
              "   'Disasters',\n",
              "   'Economics',\n",
              "   'Female',\n",
              "   'Forecasting',\n",
              "   'Humans',\n",
              "   'Labor, Obstetric',\n",
              "   'Periodicity',\n",
              "   'Pregnancy'],\n",
              "  'pmid': '11875205',\n",
              "  'title': 'Predictability of catastrophic events: material rupture, earthquakes, turbulence, financial crashes, and human birth.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'The increasing cost of drug development together with a significant drop in the number of new drug approvals raises the need for innovative approaches for target identification and efficacy prediction. Here, we take advantage of our increasing understanding of the network-based origins of diseases to introduce a drug-disease proximity measure that quantifies the interplay between drugs targets and diseases. By correcting for the known biases of the interactome, proximity helps us uncover the therapeutic effect of drugs, as well as to distinguish palliative from effective treatments. Our analysis of 238 drugs used in 78 diseases indicates that the therapeutic effect of drugs is localized in a small network neighborhood of the disease genes and highlights efficacy issues for drugs used in Parkinson and several inflammatory disorders. Finally, network-based proximity allows us to predict novel drug-disease associations that offer unprecedented opportunities for drug repurposing and the detection of adverse effects. ',\n",
              "  'authors': ['Guney, Emre',\n",
              "   'Menche, Jörg',\n",
              "   'Vidal, Marc',\n",
              "   'Barábasi, Albert-László'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Drug Discovery',\n",
              "   'Models, Biological',\n",
              "   'Molecular Structure',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Structure-Activity Relationship'],\n",
              "  'pmid': '26831545',\n",
              "  'title': 'Network-based in silico drug efficacy screening.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1618-727X',\n",
              "  'IF_2013': '1.2',\n",
              "  'IF_2014': '1.19',\n",
              "  'IF_2015': '1.406',\n",
              "  'IF_2016': '1.407',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0897-1889',\n",
              "  'ISSN_online': '1618-727X',\n",
              "  'ISSN_print': '0897-1889',\n",
              "  'IsoAbbr': 'J Digit Imaging',\n",
              "  'JrId': 1047,\n",
              "  'MedAbbr': 'J Digit Imaging',\n",
              "  'NLMid': '9100529',\n",
              "  'Titre': 'J Digit Imaging',\n",
              "  'abstract': \"In this work, two different approaches are proposed for region of interest (ROI) segmentation using transrectal ultrasound (TRUS) images. The two methods aim to extract informative features that are able to characterize suspicious regions in the TRUS images. Both proposed methods are based on multi-resolution analysis that is characterized by its high localization in both the frequency and the spatial domains. Being highly localized in both domains, the proposed methods are expected to accurately identify the suspicious ROIs. On one hand, the first method depends on a Gabor filter that captures the high frequency changes in the image regions. On the other hand, the second method depends on classifying the wavelet coefficients of the image. It is shown in this paper that both methods reveal details in the ROIs which correlate with their pathological representations. It was found that there is a good match between the regions identified using the two methods, a result that supports the ability of each of the proposed methods to mimic the radiologist's decision in identifying suspicious regions. Studying two ROI segmentation methods is important since the only available dataset is the radiologist's suspicious regions, and there is a need to support the results obtained by either one of the proposed methods. This work is mainly a preliminary proof of concept study that will ultimately be expanded to a larger scale study whose aim will be introducing an assisting tool to help the radiologist identify the suspicious regions.\",\n",
              "  'authors': ['Mohamed, S S', 'Li, J', 'Salama, M M A', 'Freeman, G'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of digital imaging',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Prostate',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Rectum',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '18473140',\n",
              "  'title': 'Prostate tissue texture feature extraction for suspicious regions identification on TRUS images.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': \"This paper presents an efficient approach to achieve microparticles flocking with robotics and optical tweezers technologies. All particles trapped by optical tweezers can be automatically moved toward a predefined region without collision. The main contribution of this paper lies in the proposal of several solutions to the flocking manipulation of microparticles in microenvironments. First, a simple flocking controller is proposed to generate the desired positions and velocities for particles' movement. Second, a velocity saturation method is implemented to prevent the desired velocities from exceeding a safe limit. Third, a two-layer control architecture is proposed for the motion control of optical tweezers. This architecture can help make many robotic manipulations achievable under microenvironments. The proposed approach with these solutions can be applied to many bioapplications especially in cell engineering and biomedicine. Experiments on yeast cells with a robot-tweezers system are finally performed to verify the effectiveness of the proposed approach.\",\n",
              "  'authors': ['Chen, Haoyao', 'Wang, Can', 'Lou, Yunjiang'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Flocculation',\n",
              "   'Micromanipulation',\n",
              "   'Models, Theoretical',\n",
              "   'Optical Tweezers',\n",
              "   'Robotics',\n",
              "   'Yeasts'],\n",
              "  'pmid': '23380840',\n",
              "  'title': 'Flocking multiple microparticles with automatically controlled optical tweezers: solutions and experiments.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['FY'],\n",
              "  'ESSN': '2151-0903',\n",
              "  'IF_2013': '0.723',\n",
              "  'IF_2014': '0.682',\n",
              "  'IF_2015': '0.738',\n",
              "  'IF_2016': '0.877',\n",
              "  'IF_2017': '1.094',\n",
              "  'ISSN': '0886-9634',\n",
              "  'ISSN_online': '2151-0903',\n",
              "  'ISSN_print': '0886-9634',\n",
              "  'IsoAbbr': 'Cranio',\n",
              "  'JrId': 2858,\n",
              "  'MedAbbr': 'Cranio',\n",
              "  'NLMid': '8609491',\n",
              "  'Titre': 'Cranio',\n",
              "  'abstract': 'The purpose of this study was to assess the discriminative capacity of the McGill Pain Questionnaire (MPQ) in patients with temporomandibular joint disorders (TMD) or with facial pain disorder as somatoform disorder (referred to as \"atypical facial pain\") (FP). The MPQ was administered to 57 TMD and 34 FP patients. Weighted MPQ item scores, subscale Pain Rating Indexes (PRI), and total Pain Rating Index were tested for significant differences (Student\\'s t-test), and the frequency of descriptor choice was also analyzed. Furthermore, the data were processed through two systems based on a counter-propagation neural network: the Self-Organizing Map (SOM) system and a cluster-like analysis. In the FP group eleven MPQ item scores and five PRI scores were significantly higher than those of the TMJ group. There was a considerable difference in descriptor choice between the groups. SOM analysis and cluster-like analysis correctly discriminated 85% or more of the patients. In conclusion, the MPQ showed a consistent discriminative capacity between TMD and FP patients.',\n",
              "  'authors': ['Mongini, F', 'Italiano, M', 'Raviola, F', 'Mossolov, A'],\n",
              "  'categories': ['DENTISTRY, ORAL SURGERY & MEDICINE'],\n",
              "  'journal': 'Cranio : the journal of craniomandibular practice',\n",
              "  'keywords': ['Adult',\n",
              "   'Cluster Analysis',\n",
              "   'Diagnosis, Differential',\n",
              "   'Discriminant Analysis',\n",
              "   'Facial Pain',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Pain Measurement',\n",
              "   'Somatoform Disorders',\n",
              "   'Surveys and Questionnaires',\n",
              "   'Temporomandibular Joint Disorders'],\n",
              "  'pmid': '11202844',\n",
              "  'title': 'The McGill Pain Questionnaire in patients with TMJ pain and with facial pain as a somatoform disorder.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DS'],\n",
              "  'ESSN': '1525-1489',\n",
              "  'IF_2013': '3.262',\n",
              "  'IF_2014': '3.262',\n",
              "  'IF_2015': '3.262',\n",
              "  'IF_2016': '2.156',\n",
              "  'IF_2017': '2.514',\n",
              "  'ISSN': '0885-0666',\n",
              "  'ISSN_online': '1525-1489',\n",
              "  'ISSN_print': '0885-0666',\n",
              "  'IsoAbbr': 'J Intensive Care Med',\n",
              "  'JrId': 2161,\n",
              "  'MedAbbr': 'J Intensive Care Med',\n",
              "  'NLMid': '8610344',\n",
              "  'Titre': 'J Intensive Care Med',\n",
              "  'abstract': 'Adequate postoperative pain management is crucial in pediatric patients undergoing cardiac surgery because pain can lead to devastating short- and long-term consequences. This review discusses the limitations of current postoperative pain assessment and management in children after cardiac surgery, the obstacles to providing optimal treatment, and concepts to consider that may overcome these barriers.|MEDLINE and PubMed.|Effective pain management in infants and young children undergoing cardiac surgery continues to evolve with innovative methods of both assessment and therapy using newer drugs or novel routes of administration. Artificial intelligence- and machine learning-based pain assessment and patient-tailored management in both pain measurement and prevention are already being integrated into the routine of current practice.',\n",
              "  'authors': ['Pollak, Uri',\n",
              "   'Bronicki, Ronald A',\n",
              "   'Achuff, Barbara-Jo',\n",
              "   'Checchia, Paul A'],\n",
              "  'categories': ['CRITICAL CARE MEDICINE'],\n",
              "  'journal': 'Journal of intensive care medicine',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31446831',\n",
              "  'title': 'Postoperative Pain Management in Pediatric Patients Undergoing Cardiac Surgery: Where Are We Heading?',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1349-3299',\n",
              "  'IF_2013': '1.127',\n",
              "  'IF_2014': '1.073',\n",
              "  'IF_2015': '1.938',\n",
              "  'IF_2016': '2.017',\n",
              "  'IF_2017': '1.826',\n",
              "  'ISSN': '1349-2365',\n",
              "  'ISSN_online': '1349-3299',\n",
              "  'ISSN_print': '1349-2365',\n",
              "  'IsoAbbr': 'Int Heart J',\n",
              "  'JrId': 32373,\n",
              "  'MedAbbr': 'Int Heart J',\n",
              "  'NLMid': '101244240',\n",
              "  'Titre': 'Int Heart J',\n",
              "  'abstract': 'Ryanodine receptors (RyRs), the large homotetrameric protein complexes, regulate the release of calcium from intracellular stores into the cytosol and play vital roles in the excitation-contraction coupling of cells. However, the evolutionary relationship of RyRs in vertebrates has yet to be elucidated. We identified 22 RyRs from Homo sapiens, Mus musculus, Rattus norvegicus, Gallus gallus, Anolis carolinensis, Rana catesbeiana, and Danio rerio. The phylogenetic relationship, motifs analysis and reconstruction of ancestral RyRs showed that the members of RyR family in vertebrates were grouped into three clades: the RyR1 clade, the RyR2 clade, and the RyR3 clade. Positive selection existed in RyR gene evolution, which is consistent in three site models, and gene ontology (GO) analysis showed that the evolution of RyR family in vertebrates promotes RyRs function differentiation. At last, we predicted 140 mutation sites which may be involved in diseases and 57 phosphorylation sites among RyR1 sequence in human, as well as 61 mutation sites and 70 phosphorylation sites in human RyR2 sequences. Most of these potential sites are arranged in clusters. Our work provides insight into the origin and evolutionary process of RyRs in vertebrates, facilitating their functional investigations in the future.',\n",
              "  'authors': ['Ding, Zhiwen',\n",
              "   'Peng, Juan',\n",
              "   'Liang, Yanyan',\n",
              "   'Yang, Chunjie',\n",
              "   'Jiang, Guoliang',\n",
              "   'Ren, Jun',\n",
              "   'Zou, Yunzeng'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'International heart journal',\n",
              "  'keywords': ['Animals',\n",
              "   'Chickens',\n",
              "   'Evolution, Molecular',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Lizards',\n",
              "   'Mice',\n",
              "   'Mutation',\n",
              "   'Phylogeny',\n",
              "   'Rana catesbeiana',\n",
              "   'Rats',\n",
              "   'Ryanodine Receptor Calcium Release Channel',\n",
              "   'Selection, Genetic',\n",
              "   'Zebrafish'],\n",
              "  'pmid': '29151485',\n",
              "  'title': 'Evolution of Vertebrate Ryanodine Receptors Family in Relation to Functional Divergence and Conservation.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': 'Medical personnel in hospitals often works under great physical and mental strain. In medical decision-making, errors can never be completely ruled out. Several studies have shown that between 50 and 60% of adverse events could have been avoided through better organization, more attention or more effective security procedures. Critical situations especially arise during interdisciplinary collaboration and the use of complex medical technology, for example during surgical interventions and in perioperative settings (the period of time before, during and after surgical intervention).|In this paper, we present an ontology and an ontology-based software system, which can identify risks across medical processes and supports the avoidance of errors in particular in the perioperative setting. We developed a practicable definition of the risk notion, which is easily understandable by the medical staff and is usable for the software tools. Based on this definition, we developed a Risk Identification Ontology (RIO) and used it for the specification and the identification of perioperative risks.|An agent system was developed, which gathers risk-relevant data during the whole perioperative treatment process from various sources and provides it for risk identification and analysis in a centralized fashion. The results of such an analysis are provided to the medical personnel in form of context-sensitive hints and alerts. For the identification of the ontologically specified risks, we developed an ontology-based software module, called Ontology-based Risk Detector (OntoRiDe).|About 20 risks relating to cochlear implantation (CI) have already been implemented. Comprehensive testing has indicated the correctness of the data acquisition, risk identification and analysis components, as well as the web-based visualization of results.',\n",
              "  'authors': ['Uciteli, Alexandr',\n",
              "   'Neumann, Juliane',\n",
              "   'Tahar, Kais',\n",
              "   'Saleh, Kutaiba',\n",
              "   'Stucke, Stephan',\n",
              "   'Faulbrück-Röhr, Sebastian',\n",
              "   'Kaeding, André',\n",
              "   'Specht, Martin',\n",
              "   'Schmidt, Tobias',\n",
              "   'Neumuth, Thomas',\n",
              "   'Besting, Andreas',\n",
              "   'Stegemann, Dominik',\n",
              "   'Portheine, Frank',\n",
              "   'Herre, Heinrich'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Humans',\n",
              "   'Perioperative Period',\n",
              "   'Risk Assessment',\n",
              "   'Software'],\n",
              "  'pmid': '28877732',\n",
              "  'title': 'Ontology-based specification, identification and analysis of perioperative risks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'We aimed at facilitating percutaneous radiofrequency ablation (RFA) for large tumors with accurate overlapping ablation planning and robot-assisted needle insertion from a single incision port (SIP).|We developed a personalized and quantitative RFA planning method to obtain multiple needle overlapping ablation planning through a single incision. A robot with a remote center of motion mechanism was designed to perform needle insertions through a SIP according to the planning.|Numerical and visual evaluation showed that the planning could yield nearly full coverage over large tumors and greatly reduce both ablation times and the ablations of normal tissues. Ex vivo and in vivo experiments showed that our robot-assisted needle insertion system was capable of conducting the RFA procedure in large liver tumors from a SIP.|Robot-assisted RFA needle insertions from a SIP makes RFA treatment for large tumors more minimally invasive, predictable, and repeatable.',\n",
              "  'authors': ['Liu, Ping',\n",
              "   'Qin, Jing',\n",
              "   'Duan, Bin',\n",
              "   'Wang, Qiong',\n",
              "   'Tan, Xiaoyu',\n",
              "   'Zhao, Baoliang',\n",
              "   'Jonnathan, Peneyra Libao',\n",
              "   'Chui, Chee-Kong',\n",
              "   'Heng, Pheng-Ann'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Carcinoma, Hepatocellular',\n",
              "   'Catheter Ablation',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Liver',\n",
              "   'Liver Neoplasms',\n",
              "   'Motion',\n",
              "   'Needles',\n",
              "   'Phantoms, Imaging',\n",
              "   'Radiofrequency Ablation',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Robotics',\n",
              "   'Swine',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '30117266',\n",
              "  'title': 'Overlapping radiofrequency ablation planning and robot-assisted needle insertion for large liver tumors.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'Computer speech recognition of individuals with dysarthria, such as cerebral palsy patients requires a robust technique that can handle conditions of very high variability and limited training data. In this study, application of a 10 state ergodic hidden Markov model (HMM)/artificial neural network (ANN) hybrid structure for a dysarthric speech (isolated word) recognition system, intended to act as an assistive tool, was investigated. A small size vocabulary spoken by three cerebral palsy subjects was chosen. The effect of such a structure on the recognition rate of the system was investigated by comparing it with an ergodic hidden Markov model as a control tool. This was done in order to determine if this modified technique contributed to enhanced recognition of dysarthric speech. The speech was sampled at 11 kHz. Mel frequency cepstral coefficients were extracted from them using 15 ms frames and served as training input to the hybrid model setup. The subsequent results demonstrated that the hybrid model structure was quite robust in its ability to handle the large variability and non-conformity of dysarthric speech. The level of variability in input dysarthric speech patterns sometimes limits the reliability of the system. However, its application as a rehabilitation/control tool to assist dysarthric motor impaired individuals holds sufficient promise.',\n",
              "  'authors': ['Polur, Prasad D', 'Miller, Gerald E'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Dysarthria',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Markov Chains',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Sound Spectrography',\n",
              "   'Speech Production Measurement',\n",
              "   'Speech Recognition Software'],\n",
              "  'pmid': '16359906',\n",
              "  'title': 'Investigation of an HMM/ANN hybrid structure in pattern recognition application using cepstral analysis of dysarthric (distorted) speech signals.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1548-7105',\n",
              "  'IF_2013': '25.953',\n",
              "  'IF_2014': '32.072',\n",
              "  'IF_2015': '25.328',\n",
              "  'IF_2016': '25.062',\n",
              "  'IF_2017': '26.919',\n",
              "  'ISSN': '1548-7091',\n",
              "  'ISSN_online': '1548-7105',\n",
              "  'ISSN_print': '1548-7091',\n",
              "  'IsoAbbr': 'Nat. Methods',\n",
              "  'JrId': 32338,\n",
              "  'MedAbbr': 'Nat Methods',\n",
              "  'NLMid': '101215604',\n",
              "  'Titre': 'Nat. Methods',\n",
              "  'abstract': \"Spatiotemporal activity patterns in three-dimensionally organized cellular networks are fundamental to the function of the nervous system. Despite advances in functional imaging of cell populations, a method to resolve local network activity in three dimensions has been lacking. Here we introduce a three-dimensional (3D) line-scan technology for two-photon microscopy that permits fast fluorescence measurements from several hundred cells distributed in 3D space. We combined sinusoidal vibration of the microscope objective at 10 Hz with 'smart' movements of galvanometric x-y scanners to repeatedly scan the laser focus along a closed 3D trajectory. More than 90% of cell somata were sampled by the scan line within volumes of 250 microm side length. Using bulk-loading of calcium indicator, we applied this method to reveal spatiotemporal activity patterns in neuronal and astrocytic networks in the rat neocortex in vivo. Two-photon population imaging using 3D scanning opens the field for comprehensive studies of local network dynamics in intact tissue.\",\n",
              "  'authors': ['Göbel, Werner', 'Kampa, Björn M', 'Helmchen, Fritjof'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Nature methods',\n",
              "  'keywords': ['Animals',\n",
              "   'Astrocytes',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Lasers',\n",
              "   'Microscopy, Fluorescence, Multiphoton',\n",
              "   'Models, Neurological',\n",
              "   'Neocortex',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Rats',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '17143280',\n",
              "  'title': 'Imaging cellular network dynamics in three dimensions using fast 3D laser scanning.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'Monitoring of the interface temperature at skin level in lower-limb prosthesis is notoriously complicated. This is due to the flexible nature of the interface liners used impeding the required consistent positioning of the temperature sensors during donning and doffing. Predicting the in-socket residual limb temperature by monitoring the temperature between socket and liner rather than skin and liner could be an important step in alleviating complaints on increased temperature and perspiration in prosthetic sockets. In this work, we propose to implement an adaptive neuro fuzzy inference strategy (ANFIS) to predict the in-socket residual limb temperature. ANFIS belongs to the family of fused neuro fuzzy system in which the fuzzy system is incorporated in a framework which is adaptive in nature. The proposed method is compared to our earlier work using Gaussian processes for machine learning. By comparing the predicted and actual data, results indicate that both the modeling techniques have comparable performance metrics and can be efficiently used for non-invasive temperature monitoring.',\n",
              "  'authors': ['Mathur, Neha', 'Glesk, Ivan', 'Buis, Arjan'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Amputees',\n",
              "   'Artificial Limbs',\n",
              "   'Fuzzy Logic',\n",
              "   'Lower Extremity',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Normal Distribution',\n",
              "   'Skin Temperature'],\n",
              "  'pmid': '27452775',\n",
              "  'title': 'Comparison of adaptive neuro-fuzzy inference system (ANFIS) and Gaussian processes for machine learning (GPML) algorithms for the prediction of skin temperature in lower limb prostheses.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1097-0290',\n",
              "  'IF_2013': '4.164',\n",
              "  'IF_2014': '4.126',\n",
              "  'IF_2015': '4.243',\n",
              "  'IF_2016': '4.481',\n",
              "  'IF_2017': '3.952',\n",
              "  'ISSN': '0006-3592',\n",
              "  'ISSN_online': '1097-0290',\n",
              "  'ISSN_print': '0006-3592',\n",
              "  'IsoAbbr': 'Biotechnol. Bioeng.',\n",
              "  'JrId': 1200,\n",
              "  'MedAbbr': 'Biotechnol Bioeng',\n",
              "  'NLMid': '7502021',\n",
              "  'Titre': 'Biotechnol. Bioeng.',\n",
              "  'abstract': 'Although previous research has focused on phenol removal efficiencies using polyphenol oxidase in nonimmobilized and immobilized forms, there has been little consideration of the use of polyphenol oxidase in a biotransformation system for the production of catechols. In this study, polyphenol oxidase was successfully immobilized on various synthetic membranes and used to convert phenolic substrates to catechol products. A neural network model was developed and used to model the rates of substrate utilization and catechol production for both nonimmobilized and immobilized polyphenol oxidase. The results indicate that the biotransformation of the phenols to their corresponding catechols was strongly influenced by the immobilization support, resulting in differing yields of catechols. Hydrophilic membranes were found to be the most suitable immobilization supports for catechol production. The successful biocatalytic production of 3-methylcatechol, 4-methylcatechol, catechol, and 4-chlorocatechol is demonstrated.',\n",
              "  'authors': ['Boshoff, A', 'Burton, M H', 'Burton, S G'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Biotechnology and bioengineering',\n",
              "  'keywords': ['Catechol Oxidase',\n",
              "   'Catechols',\n",
              "   'Computer Simulation',\n",
              "   'Enzyme Activation',\n",
              "   'Enzymes, Immobilized',\n",
              "   'Membranes, Artificial',\n",
              "   'Models, Chemical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Phenols',\n",
              "   'Quality Control',\n",
              "   'Substrate Specificity'],\n",
              "  'pmid': '12740927',\n",
              "  'title': 'Optimization of catechol production by membrane-immobilized polyphenol oxidase: a modeling approach.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1618-727X',\n",
              "  'IF_2013': '1.2',\n",
              "  'IF_2014': '1.19',\n",
              "  'IF_2015': '1.406',\n",
              "  'IF_2016': '1.407',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0897-1889',\n",
              "  'ISSN_online': '1618-727X',\n",
              "  'ISSN_print': '0897-1889',\n",
              "  'IsoAbbr': 'J Digit Imaging',\n",
              "  'JrId': 1047,\n",
              "  'MedAbbr': 'J Digit Imaging',\n",
              "  'NLMid': '9100529',\n",
              "  'Titre': 'J Digit Imaging',\n",
              "  'abstract': 'A prototype expert system was developed to study the problems applying expert system technology to radiographic image interpretation. The Radiographic Image Interpretation System (RIIS) was developed on a microcomputer using Turbo Prolog, a low cost implementation of the prolog programming language. The present implementation of RIIS was developed to highlight potential problems in applying expert system technology in the evaluation of radiographic images. It was believed that the evaluation of this prototype expert system should include a large number of users unfamiliar with the program\\'s use as this would probably be the case in clinical use of an image interpretation expert system. At present, the expert system deals with a limited domain of focal bony lesions. Twenty cases of pathologically proven bony lesions of varying difficulty were used to evaluate potential problems in the use of this expert system technology. RIIS, with the 20 sample cases, was presented as an exhibit at the 1987 Radiological Society of North America (RSNA) meeting to evaluate the potential problems with inexperienced users. These results were compared with those of experienced users. When a musculoskeletal radiologist, familiar with the programs use, provided the \"proper description,\" the program averaged the correct diagnosis in the top five 80% of the time. During the program\\'s use at the RSNA meeting, the program selected the correct diagnosis in the top five 22% of the time.',\n",
              "  'authors': ['Piraino, D W',\n",
              "   'Richmond, B J',\n",
              "   'Uetani, M',\n",
              "   'Luetkehaus, T',\n",
              "   'Rockey, D',\n",
              "   'Belhobek, G',\n",
              "   'Armistead, J',\n",
              "   'Jones, F'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of digital imaging',\n",
              "  'keywords': ['Bone Neoplasms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '2488148',\n",
              "  'title': 'Problems in applying expert system technology to radiographic image interpretation.',\n",
              "  'year': '1989'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Research on specialized biological systems is often hampered by a lack of consistent terminology, especially across species. In bacterial Type IV secretion systems genes within one set of orthologs may have over a dozen different names. Classifying research publications based on biological processes, cellular components, molecular functions, and microorganism species should improve the precision and recall of literature searches allowing researchers to keep up with the exponentially growing literature, through resources such as the Pathosystems Resource Integration Center (PATRIC, patricbrc.org). We developed named entity recognition (NER) tools for four entities related to Type IV secretion systems: 1) bacteria names, 2) biological processes, 3) molecular functions, and 4) cellular components. These four entities are important to pathogenesis and virulence research but have received less attention than other entities, e.g., genes and proteins. Based on an annotated corpus, large domain terminological resources, and machine learning techniques, we developed recognizers for these entities. High accuracy rates (>80%) are achieved for bacteria, biological processes, and molecular function. Contrastive experiments highlighted the effectiveness of alternate recognition strategies; results of term extraction on contrasting document sets demonstrated the utility of these classes for identifying T4SS-related documents.',\n",
              "  'authors': ['Ananiadou, Sophia',\n",
              "   'Sullivan, Dan',\n",
              "   'Black, William',\n",
              "   'Levow, Gina-Anne',\n",
              "   'Gillespie, Joseph J',\n",
              "   'Mao, Chunhong',\n",
              "   'Pyysalo, Sampo',\n",
              "   'Kolluru, Balakrishna',\n",
              "   'Tsujii, Junichi',\n",
              "   'Sobral, Bruno'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Bacterial Secretion Systems',\n",
              "   'Natural Language Processing',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Terminology as Topic',\n",
              "   'Vocabulary, Controlled'],\n",
              "  'pmid': '21468321',\n",
              "  'title': 'Named entity recognition for bacterial Type IV secretion systems.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-6882',\n",
              "  'IF_2013': '6.765',\n",
              "  'IF_2014': '6.628',\n",
              "  'IF_2015': '6.373',\n",
              "  'IF_2016': '6.133',\n",
              "  'IF_2017': '6.541',\n",
              "  'ISSN': '0959-4388',\n",
              "  'ISSN_online': '1873-6882',\n",
              "  'ISSN_print': '0959-4388',\n",
              "  'IsoAbbr': 'Curr. Opin. Neurobiol.',\n",
              "  'JrId': 2201,\n",
              "  'MedAbbr': 'Curr Opin Neurobiol',\n",
              "  'NLMid': '9111376',\n",
              "  'Titre': 'Curr. Opin. Neurobiol.',\n",
              "  'abstract': 'Understanding how the brain performs computations requires understanding neuronal firing patterns at successive levels of processing-a daunting and seemingly intractable task. Two recent studies have made dramatic progress on this problem by showing how its dimensionality can be reduced. Using the retina as a model system, they demonstrated that multineuronal firing patterns can be predicted by pairwise interactions.',\n",
              "  'authors': ['Nirenberg, Sheila H', 'Victor, Jonathan D'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Current opinion in neurobiology',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Animals',\n",
              "   'Brain',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Retina',\n",
              "   'Sample Size'],\n",
              "  'pmid': '17709240',\n",
              "  'title': 'Analyzing the activity of large populations of neurons: how tractable is the problem?',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1873-3573',\n",
              "  'IF_2013': '3.511',\n",
              "  'IF_2014': '3.545',\n",
              "  'IF_2015': '4.035',\n",
              "  'IF_2016': '4.162',\n",
              "  'IF_2017': '4.244',\n",
              "  'ISSN': '0039-9140',\n",
              "  'ISSN_online': '1873-3573',\n",
              "  'ISSN_print': '0039-9140',\n",
              "  'IsoAbbr': 'Talanta',\n",
              "  'JrId': 22720,\n",
              "  'MedAbbr': 'Talanta',\n",
              "  'NLMid': '2984816R',\n",
              "  'Titre': 'Talanta',\n",
              "  'abstract': 'In this work, multivariate calibration based on partial least squares (PLS) and support vector regression (SVR) using the whole spectrum and variable selection by synergy interval (siPLS and siSVR) were applied to NIR spectra for the determination of animal fat biodiesel content in soybean biodiesel and B20 diesel blends. For all models, prediction errors, bias test for systematic errors and permutation test for trends in the residuals were calculated. The siSVR produced significantly lower prediction errors compared to the full spectrum methods and siPLS, with a root mean squares error (RMSEP) of 0.18%(w/w) (concentration range: 0.00%-69.00%(w/w)) in the soybean biodiesel blend and 0.10%(w/w) in the B20 diesel (concentration range: 0.00%-13.80%(w/w)). Additionally, in the models for the determination of animal fat biodiesel in blends with soybean diesel, PLS and SVR showed evidence of systematic errors, and PLS/siPLS presented trends in residuals based on the permutation test. For the B20 diesel, PLS presented evidence of systematic errors, and siPLS presented trends in the residuals.',\n",
              "  'authors': ['Filgueiras, Paulo Roberto',\n",
              "   'Alves, Júlio Cesar L',\n",
              "   'Poppi, Ronei Jesus'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Talanta',\n",
              "  'keywords': ['Animals',\n",
              "   'Biofuels',\n",
              "   'Calibration',\n",
              "   'Fats',\n",
              "   'Models, Theoretical',\n",
              "   'Soybeans',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24401458',\n",
              "  'title': 'Quantification of animal fat biodiesel in soybean biodiesel and B20 diesel blends using near infrared spectroscopy and synergy interval support vector regression.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RA'],\n",
              "  'ESSN': '1365-2818',\n",
              "  'IF_2013': '2.15',\n",
              "  'IF_2014': '2.331',\n",
              "  'IF_2015': '2.136',\n",
              "  'IF_2016': '1.692',\n",
              "  'IF_2017': '1.693',\n",
              "  'ISSN': '0022-2720',\n",
              "  'ISSN_online': '1365-2818',\n",
              "  'ISSN_print': '0022-2720',\n",
              "  'IsoAbbr': 'J Microsc',\n",
              "  'JrId': 4961,\n",
              "  'MedAbbr': 'J Microsc',\n",
              "  'NLMid': '0204522',\n",
              "  'Titre': 'J Microsc',\n",
              "  'abstract': 'A measure of texture, the nuclear diffuseness, was formulated for use in biological classification, and specifically to characterize quantitatively chromatin texture. Nuclear diffuseness corresponds to the amount of local intensity variation in the digitized image of a nuclear profile. As a setting in which to test the efficacy of nuclear diffuseness as a diagnostic tool, the identification of parathyroid adenoma and carcinoma was considered. Digitized images of sections of parathyroid chief cell nuclei were obtained from 16 biopsies, and the nuclear diffuseness, as well as other morphometric descriptors, were computed. With just the average nuclear diffuseness and average nuclear profile area, jackknife (leave-one-out) classification using an artificial neural network was able to diagnose correctly and unambiguously the condition (normal, parathyroid adenoma, or parathyroid carcinoma) in 15 of 16 cases. In one case, the neural network assigned a higher weight to the correct diagnosis, but was unable to distinguish between normal and adenoma conclusively.',\n",
              "  'authors': ['Einstein, A J', 'Barba, J', 'Unger, P D', 'Gil, J'],\n",
              "  'categories': ['MICROSCOPY'],\n",
              "  'journal': 'Journal of microscopy',\n",
              "  'keywords': ['Adenoma',\n",
              "   'Carcinoma',\n",
              "   'Cell Nucleus',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Parathyroid Neoplasms'],\n",
              "  'pmid': '7853389',\n",
              "  'title': 'Nuclear diffuseness as a measure of texture: definition and application to the computer-assisted diagnosis of parathyroid adenoma and carcinoma.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1897-4260',\n",
              "  'IF_2013': '0.537',\n",
              "  'IF_2014': '0.641',\n",
              "  'IF_2015': '0.747',\n",
              "  'IF_2016': '0.857',\n",
              "  'IF_2017': '0.817',\n",
              "  'ISSN': '0028-3843',\n",
              "  'ISSN_online': '1897-4260',\n",
              "  'ISSN_print': '0028-3843',\n",
              "  'IsoAbbr': 'Neurol. Neurochir. Pol.',\n",
              "  'JrId': 6061,\n",
              "  'MedAbbr': 'Neurol Neurochir Pol',\n",
              "  'NLMid': '0101265',\n",
              "  'Titre': 'Neurol. Neurochir. Pol.',\n",
              "  'abstract': 'Intracranial pressure (ICP) is influenced by an array of predictable and unpredictable factors. Statistical modelling of this signal has only limited applicability because of the significant load of stochastic components. We tested the efficiency of an alternative approach, based on the methodology of artificial neural networks (ANNs) in the on-line prediction of future values of ICP and in the classification of signal properties. Satisfactory accuracy of forecasting was achieved with the ANNs for a 3-minute prediction horizon, while the prediction quality with autoregressive models of statistical origin was proved unsatisfactory. The results obtained with the ANNs were further improved when signal pre-processing with wavelet transform was employed. Nevertheless, even with the ANN methodology, no sudden breakdowns in the ICP signal (which in this respect might be compared to a \"catastrophe\") can be forecast with any practical applicability. We therefore applied two ANN algorithms, oriented at classification and discrimination of the global properties of the ICP signal. The neural network was expected to discriminate those sets of signal properties, which were assumed to correspond to certain clinical conditions of the patient. In a \"dynamic pattern classification\" the network was presented with several sections of ICP records. This was combined with information about the assignment of a given record to one of four arbitrary classes of danger. In this mode no data pre-processing was carried out, in contrast to our second approach, in which the signal was pre-processed with statistical analyses and only these intermediate coefficients were fed to the ANN classifier. The results obtained with both classification methods at their present stage of training were similar and approximated to a 70% rate of judgements consistent with expert scoring. Nevertheless, the method based on the assessment of global parameters of the ICP record seems more promising, because it leaves the possibility of extending the set of training data by information from other diagnostic modalities. The study aims towards the development of a pseudo-intelligent computer expert system, which has would be taught salient links between data extracted from the ICP signal and higher- order data, which contributed to the expert score. Hence the system would be able to make decisions on the basis of a reduced set of input information, available from a standard monitoring modality.',\n",
              "  'authors': ['Mariak, Z', 'Swiercz, M', 'Krejza, J', 'Lewko, J', 'Lyson, T'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Neurologia i neurochirurgia polska',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Electrodiagnosis',\n",
              "   'Humans',\n",
              "   'Intracranial Hypertension',\n",
              "   'Intracranial Pressure',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '11317497',\n",
              "  'title': '[Analysis of intracranial pressure signals using artificial neural networks].',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1532-7361',\n",
              "  'IF_2013': '3.105',\n",
              "  'IF_2014': '3.38',\n",
              "  'IF_2015': '3.309',\n",
              "  'IF_2016': '3.904',\n",
              "  'IF_2017': '3.574',\n",
              "  'ISSN': '0039-6060',\n",
              "  'ISSN_online': '1532-7361',\n",
              "  'ISSN_print': '0039-6060',\n",
              "  'IsoAbbr': 'Surgery',\n",
              "  'JrId': 7662,\n",
              "  'MedAbbr': 'Surgery',\n",
              "  'NLMid': '0417347',\n",
              "  'Titre': 'Surgery',\n",
              "  'abstract': 'There is limited consensus regarding risk factors for postoperative bleeding. The objective of this work was to investigate the capability of machine learning techniques in combination with practice-based longitudinal electronic medical record data for identifying potential new risk factors for postoperative bleeding and predicting patients at high risk of postoperative bleeding.|A retrospective study was conducted for patients who underwent colorectal surgery 1998-2015 at a single tertiary referral center. Various predictors were extracted from electronic medical record. The outcome of interest was the occurrence of postoperative bleeding within 7 days of surgery. Logistic regression and gradient boosting machine models were trained. Area under the receiver operating curve and area under the precision recall curve were used to evaluate the performance to different models.|Of 13,399 cases undergoing colorectal resection, 1,680 (12.5%) experienced postoperative bleeding. A total of 299 variables were evaluated. Logistic regression and gradient boosting machine models returned an area under the receiver operating curve of 0.735 and 0.822 and area under the precision recall curve of 0.287 and 0.423, respectively. In addition to well-known risk factors for postoperative bleeding, nutrition (ranked third), weakness (ranked fifth), patient mobility (ranked sixth), and activity level (ranked eighth) were found to be novel predictors in the gradient boosting machine model based on permutation importance.|The study identified measures of functional capacity of patient as novel predictors of postoperative bleeding. The study found that risk of postoperative bleeding can be assessed, allowing for better use of human resources in addressing this important adverse event after surgery.',\n",
              "  'authors': ['Chen, David',\n",
              "   'Afzal, Naveed',\n",
              "   'Sohn, Sunghwan',\n",
              "   'Habermann, Elizabeth B',\n",
              "   'Naessens, James M',\n",
              "   'Larson, David W',\n",
              "   'Liu, Hongfang'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgery',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Colorectal Surgery',\n",
              "   'Electronic Health Records',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Hemorrhage',\n",
              "   'Retrospective Studies',\n",
              "   'Risk Assessment'],\n",
              "  'pmid': '30033185',\n",
              "  'title': 'Postoperative bleeding risk prediction for patients undergoing colorectal surgery.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'To standardize nomenclature and acronyms applied to robotic procedures in urology.|A review of the literature for robotic procedures in urology was performed. In addition, various acronyms were queried and frequency of use was noted. Additional literature reviewed included articles on standardization of scientific nomenclature used in other disciplines of health care. This topic was discussed within the newly formed working group of urologic robotic surgeons (WURS).|There is wide variation in the terminology and use of acronyms for robotic prostate and kidney procedures. The application of robotic, robot, robotic laparoscopic, robot laparoscopic, robotic-assisted, robot-assisted, robotic assisted, robot assisted, robotic-assisted laparoscopic, robot-assisted laparoscopic, robotic assisted laparoscopic, robot assisted laparoscopic, and robotic computer-assisted to procedures such as pyeloplasty and radical prostatectomy is far from uniform or standardized. Queries and searches using acronyms such as RALP, RARP, RRP, and RP yielded publications discussing both prostate and renal procedures applying the same acronyms. There are established databases of abbreviations and acronyms, including ARGH, SaRad, Acromed, and the Stanford Biomedical Abbreviation Server.|Standardization of the nomenclature applied to urologic robotic surgery is needed as the robotic literature continues to grow. The recommendation from the Scientific Committee of the Working Group of Urologic Robotic Surgeons is to apply \"robot assisted\" to the already established nomenclature currently used to describe equivalent open procedures. The use of standardized and uniform terminology for procedures performed with robots will facilitate literature search and research efforts in the future. The role of standardized nomenclature for billing purposes remains to be seen.',\n",
              "  'authors': ['Castle, Erik P', 'Lee, David'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Prostatectomy', 'Robotics', 'Terminology as Topic', 'Urology'],\n",
              "  'pmid': '18613788',\n",
              "  'title': 'Nomenclature of robotic procedures in urology.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['QA'],\n",
              "  'ESSN': '1545-326X',\n",
              "  'IF_2013': '15.478',\n",
              "  'IF_2014': '12.928',\n",
              "  'IF_2015': '10.954',\n",
              "  'IF_2016': '13.609',\n",
              "  'IF_2017': '14.97',\n",
              "  'ISSN': '0066-4219',\n",
              "  'ISSN_online': '1545-326X',\n",
              "  'ISSN_print': '0066-4219',\n",
              "  'IsoAbbr': 'Annu. Rev. Med.',\n",
              "  'JrId': 669,\n",
              "  'MedAbbr': 'Annu Rev Med',\n",
              "  'NLMid': '2985151R',\n",
              "  'Titre': 'Annu. Rev. Med.',\n",
              "  'abstract': 'Mass spectrometry-based diagnostics has the potential to revolutionize molecular medicine. Using modern mass-spectrometer technologies, clinical tests can be developed that are practical, robust, accurate, and inexpensive. Serum proteomic pattern profiling couples mass spectrometry with adaptive artificial-intelligence-based bioinformatics, which can now be employed to detect pathological states reflected in the serum proteome. With this approach, rapid and cost-effective tests with exquisite clinical sensitivity and specificity are emerging. These tools may dramatically change how disease is detected, monitored, and managed.',\n",
              "  'authors': ['Rosenblatt, Kevin P',\n",
              "   'Bryant-Greenwood, Peter',\n",
              "   'Killian, J Keith',\n",
              "   'Mehta, Arpita',\n",
              "   'Geho, David',\n",
              "   'Espina, Virginia',\n",
              "   'Petricoin, Emanuel F',\n",
              "   'Liotta, Lance A'],\n",
              "  'categories': ['MEDICINE, RESEARCH & EXPERIMENTAL'],\n",
              "  'journal': 'Annual review of medicine',\n",
              "  'keywords': ['Blood Proteins',\n",
              "   'Computational Biology',\n",
              "   'Humans',\n",
              "   'Mass Spectrometry',\n",
              "   'Neoplasms',\n",
              "   'Proteomics'],\n",
              "  'pmid': '14746511',\n",
              "  'title': 'Serum proteomics in cancer diagnosis and management.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'A novel method for detecting ventricular premature contraction (VPC) from the Holter system is proposed using wavelet transform (WT) and fuzzy neural network (FNN). The basic ideal and major advantage of this method is to reuse information that is used during QRS detection, a necessary step for most ECG classification algorithm, for VPC detection. To reduce the influence of different artifacts, the filter bank property of quadratic spline WT is explored. The QRS duration in scale three and the area under the QRS complex in scale four are selected as the characteristic features. It is found that the R wave amplitude has a marked influence on the computation of proposed characteristic features. Thus, it is necessary to normalize these features. This normalization process can reduce the effect of alternating R wave amplitude and achieve reliable VPC detection. After normalization and excluding the left bundle branch block beats, the accuracies for VPC classification using FNN is 99.79%. Features that are extracted using quadratic spline wavelet were used successfully by previous investigators for QRS detection. In this study, using the same wavelet, it is demonstrated that the proposed feature extraction method from different WT scales can effectively eliminate the influence of high and low-frequency noise and achieve reliable VPC classification. The two primary advantages of using same wavelet for QRS detection and VPC classification are less computation and less complexity during actual implementation.',\n",
              "  'authors': ['Shyu, Liang-Yu', 'Wu, Ying-Hsuan', 'Hu, Weichih'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electrocardiography, Ambulatory',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Ventricular Premature Complexes'],\n",
              "  'pmid': '15248543',\n",
              "  'title': 'Using wavelet transform and fuzzy neural network for VPC detection from the Holter ECG.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'The nervous system can adapt to external forces that perturb locomotion by correcting errors in limb movements. It is believed that supraspinal structures mediate these adaptations, whereas the spinal cord contributes only reflexive responses to perturbations. We examined whether the lumbar spinal cord in postnatal day 5 neonatal spinally transected (ST) rats corrected errors in hindlimb coordination through repetitive exposure to an external perturbation. A robotic device was used to deliver a viscous (velocity-dependent) force that opposed only the forward movement of the ankle in one hindlimb while the ST rats performed hindlimb stepping on a treadmill. We measured the interval between paw contact in the perturbed hindlimb and toe off in the unperturbed hindlimb. Before the force was activated, a normal pattern of coordination occurred: paw contact in the perturbed hindlimb occurred before toe off in the unperturbed hindlimb. This sequence was initially disrupted when the force was activated and the unperturbed hindlimb initiated swing during the swing phase of the perturbed hindlimb. Within five step cycles of exposure to the unilateral viscous force, however, the ST rats regained the preforce pattern of hindlimb coordination. These findings suggest that in the absence of supraspinal input, the lumbar spinal circuitry is capable of processing a complex ensemble of sensory information to maintain locomotor stability. Thus, the lumbar spinal circuitry may play a greater role in generating locomotor adaptations than previously thought.',\n",
              "  'authors': ['Heng, Chad', 'de Leon, Ray D'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Animals',\n",
              "   'Animals, Newborn',\n",
              "   'Female',\n",
              "   'Hindlimb',\n",
              "   'Learning',\n",
              "   'Locomotion',\n",
              "   'Lumbosacral Region',\n",
              "   'Psychomotor Performance',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Robotics',\n",
              "   'Spinal Cord',\n",
              "   'Viscosity'],\n",
              "  'pmid': '17687033',\n",
              "  'title': 'The rodent lumbar spinal cord learns to correct errors in hindlimb coordination caused by viscous force perturbations during stepping.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '2040-2309',\n",
              "  'IF_2013': '0.468',\n",
              "  'IF_2014': '0.754',\n",
              "  'IF_2015': '0.925',\n",
              "  'IF_2016': '0.965',\n",
              "  'IF_2017': '1.261',\n",
              "  'ISSN': '2040-2295',\n",
              "  'ISSN_online': '2040-2309',\n",
              "  'ISSN_print': '2040-2295',\n",
              "  'IsoAbbr': 'J Healthc Eng',\n",
              "  'JrId': 39708,\n",
              "  'MedAbbr': 'J Healthc Eng',\n",
              "  'NLMid': '101528166',\n",
              "  'Titre': 'J Healthc Eng',\n",
              "  'abstract': 'This work proposed a novel automatic three-dimensional (3D) magnetic resonance imaging (MRI) segmentation method which would be widely used in the clinical diagnosis of the most common and aggressive brain tumor, namely, glioma. The method combined a multipathway convolutional neural network (CNN) and fully connected conditional random field (CRF). Firstly, 3D information was introduced into the CNN which makes more accurate recognition of glioma with low contrast. Then, fully connected CRF was added as a postprocessing step which purposed more delicate delineation of glioma boundary. The method was applied to T2flair MRI images of 160 low-grade glioma patients. With 59 cases of data training and manual segmentation as the ground truth, the Dice similarity coefficient (DSC) of our method was 0.85 for the test set of 101 MRI images. The results of our method were better than those of another state-of-the-art CNN method, which gained the DSC of 0.76 for the same dataset. It proved that our method could produce better results for the segmentation of low-grade gliomas.',\n",
              "  'authors': ['Li, Zeju',\n",
              "   'Wang, Yuanyuan',\n",
              "   'Yu, Jinhua',\n",
              "   'Shi, Zhifeng',\n",
              "   'Guo, Yi',\n",
              "   'Chen, Liang',\n",
              "   'Mao, Ying'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Journal of healthcare engineering',\n",
              "  'keywords': ['Brain Neoplasms',\n",
              "   'Female',\n",
              "   'Glioma',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '29065666',\n",
              "  'title': 'Low-Grade Glioma Segmentation Based on CNN with Fully Connected CRF.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'We have earlier reported antileishmanial activity of hypericin by spermidine starvation. In the current report, we have used label free proteome quantitation approach to identify differentially modulated proteins after hypericin treatment. A total of 141 proteins were found to be differentially regulated with ANOVA P value less than 0.05 in hypericin treated Leishmania promastigotes. Differentially modulated proteins have been broadly classified under nine major categories. Increase in ribosomal protein S7 protein suggests the repression of translation. Inhibition of proteins related to ubiquitin proteasome system, RNA binding protein and translation initiation factor also suggests altered translation. We have also observed increased expression of Hsp 90, Hsp 83-1 and stress inducible protein 1. Significant decreased level of cyclophilin was observed. These stress related protein could be cellular response of the parasite towards hypericin induced cellular stress. Also, defective metabolism, biosynthesis and replication of nucleic acids, flagellar movement and signalling of the parasite were observed as indicated by altered expression of proteins involved in these pathways. The data was analyzed rigorously to get further insight into hypericin induced parasitic death.',\n",
              "  'authors': ['Singh, Shalini', 'Dubey, Vikash Kumar'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Antiprotozoal Agents',\n",
              "   'Cyclophilins',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Heat-Shock Proteins',\n",
              "   'Leishmania donovani',\n",
              "   'Life Cycle Stages',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Peptide Chain Initiation, Translational',\n",
              "   'Perylene',\n",
              "   'Proteome',\n",
              "   'Protozoan Proteins',\n",
              "   'RNA-Binding Proteins',\n",
              "   'Ribosomal Proteins',\n",
              "   'Spermidine',\n",
              "   'Ubiquitin'],\n",
              "  'pmid': '27123864',\n",
              "  'title': 'Quantitative Proteome Analysis of Leishmania donovani under Spermidine Starvation.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '1559-713X',\n",
              "  'IF_2013': '1.593',\n",
              "  'IF_2014': '1.477',\n",
              "  'IF_2015': '1.685',\n",
              "  'IF_2016': '1.615',\n",
              "  'IF_2017': '2.34',\n",
              "  'ISSN': '1559-2332',\n",
              "  'ISSN_online': '1559-713X',\n",
              "  'ISSN_print': '1559-2332',\n",
              "  'IsoAbbr': 'Simul Healthc',\n",
              "  'JrId': 35526,\n",
              "  'MedAbbr': 'Simul Healthc',\n",
              "  'NLMid': '101264408',\n",
              "  'Titre': 'Simul Healthc',\n",
              "  'abstract': 'Pelvic lymphadenectomy is a key component of the surgical treatment of several gynecologic cancers and involves mastery of complex anatomic relationships. Our aim was to demonstrate that the anatomy relevant to robotic pelvic lymphadenectomy can be modeled using low-cost techniques, thereby enabling simulation focused on surgical dissection, a task that integrates technical skills and anatomic knowledge.|A model of pelvic lymphadenectomy was constructed through experimentation with several different materials and a number of prototypes. In the final version, blood vessels were simulated by rubber tubing stented with wire and lymph nodes by cotton balls. Adipose and areolar tissue were simulated by a gelatin solution poured into the model and then allowed to cool and semisolidify. Three gynecologic oncologists and 2 gynecologic oncology fellows dissected the model using the surgical robot (da Vinci Surgical System) and completed a structured questionnaire. Five additional gynecologic oncologists assessed the model at a national conference.|The model received high ratings for face and content validity. Median ratings were almost all 4 of 5 or higher (range, 3-5). Participants who dissected the model (n = 5) unanimously rated it as \"useful for training throughout residency and fellowship.\"|A novel low-cost inanimate model of pelvic lymphadenectomy has been developed and rated highly for face and content validity. This model may permit more regular simulation sessions compared with alternatives such as cadaveric dissection and animal laboratories, thereby complementing them and facilitating distributed practice.',\n",
              "  'authors': ['Kiely, Daniel J',\n",
              "   'Gotlieb, Walter H',\n",
              "   'Jardon, Kris',\n",
              "   'Lau, Susie',\n",
              "   'Press, Joshua Z'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Simulation in healthcare : journal of the Society for Simulation in Healthcare',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Female',\n",
              "   'Genital Neoplasms, Female',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Humans',\n",
              "   'Lymph Node Excision',\n",
              "   'Manikins',\n",
              "   'Models, Biological',\n",
              "   'Robotics'],\n",
              "  'pmid': '25514589',\n",
              "  'title': 'Advancing surgical simulation in gynecologic oncology: robotic dissection of a novel pelvic lymphadenectomy model.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': \"This paper proposes a novel real-time electromyogram (EMG) pattern recognition for the control of a multifunction myoelectric hand from four channel EMG signals. To extract a feature vector from the EMG signal, we use a wavelet packet transform that is a generalized version of wavelet transform. For dimensionality reduction and nonlinear mapping of the features, we also propose a linear-nonlinear feature projection composed of principal components analysis (PCA) and a self-organizing feature map (SOFM). The dimensionality reduction by PCA simplifies the structure of the classifier and reduces processing time for the pattern recognition. The nonlinear mapping by SOFM transforms the PCA-reduced features into a new feature space with high class separability. Finally, a multilayer perceptron (MLP) is used as the classifier. Using an analysis of class separability by feature projections, we show that the recognition accuracy depends more on the class separability of the projected features than on the MLP's class separation ability. Consequently, the proposed linear-nonlinear projection method improves class separability and recognition accuracy. We implement a real-time control system for a multifunction virtual hand. Our experimental results show that all processes, including virtual hand control, are completed within 125 ms, and the proposed method is applicable to real-time myoelectric hand control without an operational time delay.\",\n",
              "  'authors': ['Chu, Jun-Uk', 'Moon, Inhyuk', 'Mun, Mu-Seong'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Artificial Intelligence',\n",
              "   'Artificial Limbs',\n",
              "   'Computer Simulation',\n",
              "   'Computer Systems',\n",
              "   'Electromyography',\n",
              "   'Hand',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Models, Biological',\n",
              "   'Muscle, Skeletal',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Principal Component Analysis',\n",
              "   'Prosthesis Design',\n",
              "   'Robotics',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '17073328',\n",
              "  'title': 'A real-time EMG pattern recognition system based on linear-nonlinear feature projection for a multifunction myoelectric hand.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['FY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.154',\n",
              "  'IF_2014': '1.154',\n",
              "  'IF_2015': '1.154',\n",
              "  'IF_2016': '1.436',\n",
              "  'IF_2017': '1.725',\n",
              "  'ISSN': '1463-4201',\n",
              "  'ISSN_print': '1463-4201',\n",
              "  'IsoAbbr': 'Int J Comput Dent',\n",
              "  'JrId': 22380,\n",
              "  'MedAbbr': 'Int J Comput Dent',\n",
              "  'NLMid': '100891504',\n",
              "  'Titre': 'Int J Comput Dent',\n",
              "  'abstract': 'The goal of this study is to present a completely new procedure for automatic occlusal surface reconstruction and to test it in several inlay situations. The starting point is the mathematical analysis of a large number of occlusal surfaces from a tooth library. Learning algorithms used in pattern recognition and image processing were adapted for the special needs of dental morphology. A biogeneric tooth model resulted which can mathematically describe a certain type of tooth only with a few parameters. This model was tested on 40 simulated inlay situations and as an example on two clinical inlay situations. The results show that fully automatic reconstruction was possible except for four cases. In a majority of the cases, no further interactive improvements would have been necessary. Because of the general, theoretically well-founded approach, fully automatic reconstruction of the occlusal surface can also be extended in the future to full crowns or reduced crown substructures (copings).',\n",
              "  'authors': ['Mehl, A', 'Blanz, V'],\n",
              "  'categories': ['DENTISTRY, ORAL SURGERY & MEDICINE'],\n",
              "  'journal': 'International journal of computerized dentistry',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Artificial Intelligence',\n",
              "   'Child',\n",
              "   'Computer-Aided Design',\n",
              "   'Dental Models',\n",
              "   'Dental Prosthesis Design',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Inlays',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Tooth Crown'],\n",
              "  'pmid': '15892521',\n",
              "  'title': 'New procedure for fully automatic occlusal surface reconstruction by means of a biogeneric tooth model.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'To help understand how semantic information is represented in the human brain, a number of previous studies have explored how a linear mapping from corpus derived semantic representations to corresponding patterns of fMRI brain activations can be learned. They have demonstrated that such a mapping for concrete nouns is able to predict brain activations with accuracy levels significantly above chance, but the more recent elaborations have achieved relatively little performance improvement over the original study. In fact, the absolute accuracies of all these models are still currently rather limited, and it is not clear which aspects of the approach need improving in order to achieve performance levels that might lead to better accounts of human capabilities. This paper presents a systematic series of computational experiments designed to identify the limiting factors of the approach. Two distinct series of artificial brain activation vectors with varying levels of noise are introduced to characterize how the brain activation data restricts performance, and improved corpus based semantic vectors are developed to determine how the word set and model inputs affect the results. These experiments lead to the conclusion that the current state-of-the-art input semantic representations are already operating nearly perfectly (at least for non-ambiguous concrete nouns), and that it is primarily the quality of the fMRI data that is limiting what can be achieved with this approach. The results allow the study to end with empirically informed suggestions about the best directions for future research in this area.',\n",
              "  'authors': ['Bullinaria, John A', 'Levy, Joseph P'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Neurological',\n",
              "   'Models, Psychological',\n",
              "   'Semantics'],\n",
              "  'pmid': '23526937',\n",
              "  'title': 'Limiting factors for mapping corpus-based semantic representations to brain activity.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Spike sorting is the mandatory first step in analyzing multiunit recording signals for studying information processing mechanisms within the nervous system. Extracellular recordings usually contain overlapped spikes produced by a number of neurons adjacent to the electrode, together with unknown background noise, which in turn induce some difficulties in neural signal identification. In this paper, we propose a robust method to deal with these problems, which employs an automatic overlap decomposition technique based on the relaxation algorithm that requires simple fast Fourier transforms. The performance of the presented system was tested at various signal-to-noise ratio levels based on synthetic data that were generated from real recordings.',\n",
              "  'authors': ['Wang, Guang-Li',\n",
              "   'Zhou, Yi',\n",
              "   'Chen, Ai-Hua',\n",
              "   'Zhang, Pu-Ming',\n",
              "   'Liang, Pei-Ji'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Cells, Cultured',\n",
              "   'Chickens',\n",
              "   'Nerve Net',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Retinal Ganglion Cells'],\n",
              "  'pmid': '16761848',\n",
              "  'title': 'A robust method for spike sorting with automatic overlap decomposition.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1471-2156',\n",
              "  'IF_2013': '2.356',\n",
              "  'IF_2014': '2.397',\n",
              "  'IF_2015': '2.152',\n",
              "  'IF_2016': '2.266',\n",
              "  'IF_2017': '2.469',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2156',\n",
              "  'IsoAbbr': 'BMC Genet.',\n",
              "  'JrId': 21880,\n",
              "  'MedAbbr': 'BMC Genet',\n",
              "  'NLMid': '100966978',\n",
              "  'Titre': 'BMC Genet.',\n",
              "  'abstract': 'Alcoholism is a serious public health problem. It has both genetic and environmental causes. In an effort to gain understanding of the underlying genetic susceptibility to alcoholism, a long-term study has been undertaken. The Collaborative Study on the Genetics of Alcoholism (COGA) provides a rich source of genetic and phenotypic data. One ongoing problem is the difficulty of reliably diagnosing alcoholism, despite many known risk factors and measurements. We have applied a well known pattern-matching method, neural network analysis, to phenotypic data provided to participants in Genetic Analysis Workshop 14 by COGA. The aim is to train the network to recognize complex phenotypic patterns that are characteristic of those with alcoholism as well as those who are free of symptoms. Our results indicate that this approach may be helpful in the diagnosis of alcoholism.|Training and testing of input/output pairs of risk factors by means of a \"feed-forward back-propagation\" neural network resulted in reliability of about 94% in predicting the presence or absence of alcoholism based on 36 input phenotypic risk factors. Pruning the neural network to remove relatively uninformative factors resulted in a reduced network of 14 input factors that was still 95% reliable. Some of the factors selected by the pruning steps have been identified as traits that show either linkage or association to potential candidate regions.|The complex, multivariate picture formed by known risk factors for alcoholism can be incorporated into a neural network analysis that reliably predicts the presence or absence of alcoholism about 94-95% of the time. Several characteristics that were identified by a pruned neural network have previously been shown to be important in this disease based on more traditional linkage and association studies. Neural networks therefore provide one less traditional approach to both identifying alcoholic individuals and determining the most informative risk factors.',\n",
              "  'authors': ['Falk, Catherine T'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC genetics',\n",
              "  'keywords': ['Alcohol Drinking',\n",
              "   'Alcoholism',\n",
              "   'Genetic Predisposition to Disease',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Phenotype',\n",
              "   'Risk Factors'],\n",
              "  'pmid': '16451590',\n",
              "  'title': 'Diagnosis of alcoholism based on neural network analysis of phenotypic risk factors.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1938-2421',\n",
              "  'IF_2013': '0.761',\n",
              "  'IF_2014': '0.91',\n",
              "  'IF_2015': '1.06',\n",
              "  'IF_2016': '0.957',\n",
              "  'IF_2017': '1.185',\n",
              "  'ISSN': '0148-4834',\n",
              "  'ISSN_online': '1938-2421',\n",
              "  'ISSN_print': '0148-4834',\n",
              "  'IsoAbbr': 'J Nurs Educ',\n",
              "  'JrId': 5052,\n",
              "  'MedAbbr': 'J Nurs Educ',\n",
              "  'NLMid': '7705432',\n",
              "  'Titre': 'J Nurs Educ',\n",
              "  'abstract': \"Nurse practitioners (NPs) have dual goals as primary care providers, combining the traditional goals of nursing with extended goals as diagnosticians. Diagnostic reasoning, therefore, is a critical component of NP education. Iliad, a computerized diagnostic reasoning expert system, has been used effectively to teach diagnostic skills to medical students. A pilot study was undertaken to determine the effects of Iliad training on NP students' diagnostic skill performance and to identify technical and instructional issues of implementation. The study found that the use of Iliad improved NP students' diagnostic reasoning, and that the training effects were modified by prior nursing experience. Successful use of Iliad required planning, faculty commitment, and technical support.\",\n",
              "  'authors': ['Lange, L L',\n",
              "   'Haak, S W',\n",
              "   'Lincoln, M J',\n",
              "   'Thompson, C B',\n",
              "   'Turner, C W',\n",
              "   'Weir, C',\n",
              "   'Foerster, V',\n",
              "   'Nilasena, D',\n",
              "   'Reeves, R'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'The Journal of nursing education',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Computer-Assisted Instruction',\n",
              "   'Curriculum',\n",
              "   'Decision Making',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnostic Errors',\n",
              "   'Education, Nursing, Graduate',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Logic',\n",
              "   'Nurse Practitioners',\n",
              "   'Pilot Projects'],\n",
              "  'pmid': '8986960',\n",
              "  'title': 'Use of Iliad to improve diagnostic performance of nurse practitioner students.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1534-3170',\n",
              "  'IF_2013': '1.925',\n",
              "  'IF_2014': '1.925',\n",
              "  'IF_2015': '1.83',\n",
              "  'IF_2016': '2.058',\n",
              "  'IF_2017': '1.958',\n",
              "  'ISSN': '1523-3782',\n",
              "  'ISSN_online': '1534-3170',\n",
              "  'ISSN_print': '1523-3782',\n",
              "  'IsoAbbr': 'Curr Cardiol Rep',\n",
              "  'JrId': 21832,\n",
              "  'MedAbbr': 'Curr Cardiol Rep',\n",
              "  'NLMid': '100888969',\n",
              "  'Titre': 'Curr Cardiol Rep',\n",
              "  'abstract': 'With the introduction of the da Vinci robotic surgical system (Intuitive Surgical, Mountain View, CA) into minimally invasive cardiac surgery, the outlook of performing coronary artery bypass operations \"closed chest\" became a reality. Between May 1999 and July 2000 this wrist-enhanced instrumentation was used in 143 patients (107 men, 36 women, median age 63 10.3 y). Thirteen patients suffering from coronary artery disease (CAD) were treated as totally endoscopic coronary artery bypass (TECAB), 79 patients underwent a minimally invasive direct coronary artery bypass procedure, and 35 patients were treated using the robotic-enhanced Dresden Technique. Preoperative survival was 100%. All patients in the TECAB group were operated upon via a three- or four-point stab incision using the da Vinci robot for internal mammary artery takedown and for performance of anastomoses. These new robotic-enhanced surgical techniques promote an optimistic way of thinking about the further development of these procedures and its application in patients suffering from CAD.',\n",
              "  'authors': ['Kappert, U',\n",
              "   'Schneider, J',\n",
              "   'Cichon, R',\n",
              "   'Gulielmos, V',\n",
              "   'Schade, I',\n",
              "   'Nicolai, J',\n",
              "   'Schueler, S'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Current cardiology reports',\n",
              "  'keywords': ['Aged',\n",
              "   'Coronary Angiography',\n",
              "   'Coronary Artery Bypass',\n",
              "   'Coronary Disease',\n",
              "   'Endoscopy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Prognosis',\n",
              "   'Robotics',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Survival Rate',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '11060584',\n",
              "  'title': 'Closed chest totally endoscopic coronary artery bypass surgery: fantasy or reality?',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'The IUPHAR/BPS Guide to PHARMACOLOGY (GtoPdb, http://www.guidetopharmacology.org) provides expert-curated molecular interactions between successful and potential drugs and their targets in the human genome. Developed by the International Union of Basic and Clinical Pharmacology (IUPHAR) and the British Pharmacological Society (BPS), this resource, and its earlier incarnation as IUPHAR-DB, is described in our 2014 publication. This update incorporates changes over the intervening seven database releases. The unique model of content capture is based on established and new target class subcommittees collaborating with in-house curators. Most information comes from journal articles, but we now also index kinase cross-screening panels. Targets are specified by UniProtKB IDs. Small molecules are defined by PubChem Compound Identifiers (CIDs); ligand capture also includes peptides and clinical antibodies. We have extended the capture of ligands and targets linked via published quantitative binding data (e.g. Ki, IC50 or Kd). The resulting pharmacological relationship network now defines a data-supported druggable genome encompassing 7% of human proteins. The database also provides an expanded substrate for the biennially published compendium, the Concise Guide to PHARMACOLOGY. This article covers content increase, entity analysis, revised curation strategies, new website features and expanded download options. ',\n",
              "  'authors': ['Southan, Christopher',\n",
              "   'Sharman, Joanna L',\n",
              "   'Benson, Helen E',\n",
              "   'Faccenda, Elena',\n",
              "   'Pawson, Adam J',\n",
              "   'Alexander, Stephen P H',\n",
              "   'Buneman, O Peter',\n",
              "   'Davenport, Anthony P',\n",
              "   'McGrath, John C',\n",
              "   'Peters, John A',\n",
              "   'Spedding, Michael',\n",
              "   'Catterall, William A',\n",
              "   'Fabbro, Doriano',\n",
              "   'Davies, Jamie A'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Databases, Pharmaceutical',\n",
              "   'Disease',\n",
              "   'Drug Discovery',\n",
              "   'Genome, Human',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Ligands',\n",
              "   'Patents as Topic',\n",
              "   'Phosphotransferases',\n",
              "   'Proteins'],\n",
              "  'pmid': '26464438',\n",
              "  'title': 'The IUPHAR/BPS Guide to PHARMACOLOGY in 2016: towards curated quantitative interactions between 1300 protein targets and 6000 ligands.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1530-8561',\n",
              "  'IF_2013': '7.768',\n",
              "  'IF_2014': '7.911',\n",
              "  'IF_2015': '7.457',\n",
              "  'IF_2016': '8.008',\n",
              "  'IF_2017': '8.636',\n",
              "  'ISSN': '0009-9147',\n",
              "  'ISSN_online': '1530-8561',\n",
              "  'ISSN_print': '0009-9147',\n",
              "  'IsoAbbr': 'Clin. Chem.',\n",
              "  'JrId': 2992,\n",
              "  'MedAbbr': 'Clin Chem',\n",
              "  'NLMid': '9421549',\n",
              "  'Titre': 'Clin. Chem.',\n",
              "  'abstract': 'Dideoxy-based chain termination sequencing developed by Sanger is the gold standard sequencing approach and allows clinical diagnostics of disorders with relatively low genetic heterogeneity. Recently, new next generation sequencing (NGS) technologies have found their way into diagnostic laboratories, enabling the sequencing of large targeted gene panels or exomes. The development of benchtop NGS instruments now allows the analysis of single genes or small gene panels, making these platforms increasingly competitive with Sanger sequencing.|We developed a generic automated ion semiconductor sequencing work flow that can be used in a clinical setting and can serve as a substitute for Sanger sequencing. Standard amplicon-based enrichment remained identical to PCR for Sanger sequencing. A novel postenrichment pooling strategy was developed, limiting the number of library preparations and reducing sequencing costs up to 70% compared to Sanger sequencing.|A total of 1224 known pathogenic variants were analyzed, yielding an analytical sensitivity of 99.92% and specificity of 99.99%. In a second experiment, a total of 100 patient-derived DNA samples were analyzed using a blind analysis. The results showed an analytical sensitivity of 99.60% and specificity of 99.98%, comparable to Sanger sequencing.|Ion semiconductor sequencing can be a first choice mutation scanning technique, independent of the genes analyzed.',\n",
              "  'authors': ['Diekstra, Adinda',\n",
              "   'Bosgoed, Ermanno',\n",
              "   'Rikken, Alwin',\n",
              "   'van Lier, Bart',\n",
              "   'Kamsteeg, Erik-Jan',\n",
              "   'Tychon, Marloes',\n",
              "   'Derks, Ronny C',\n",
              "   'van Soest, Ronald A',\n",
              "   'Mensenkamp, Arjen R',\n",
              "   'Scheffer, Hans',\n",
              "   'Neveling, Kornelia',\n",
              "   'Nelen, Marcel R'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinical chemistry',\n",
              "  'keywords': ['DNA',\n",
              "   'DNA Mutational Analysis',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Humans',\n",
              "   'Molecular Diagnostic Techniques',\n",
              "   'Polymerase Chain Reaction',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Semiconductors',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sequence Analysis, DNA'],\n",
              "  'pmid': '25274553',\n",
              "  'title': 'Translating sanger-based routine DNA diagnostics into generic massive parallel ion semiconductor sequencing.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1873-3476',\n",
              "  'IF_2013': '3.785',\n",
              "  'IF_2014': '3.65',\n",
              "  'IF_2015': '3.994',\n",
              "  'IF_2016': '3.649',\n",
              "  'IF_2017': '3.862',\n",
              "  'ISSN': '0378-5173',\n",
              "  'ISSN_online': '1873-3476',\n",
              "  'ISSN_print': '0378-5173',\n",
              "  'IsoAbbr': 'Int J Pharm',\n",
              "  'JrId': 21201,\n",
              "  'MedAbbr': 'Int J Pharm',\n",
              "  'NLMid': '7804127',\n",
              "  'Titre': 'Int J Pharm',\n",
              "  'abstract': 'Hydrogels-based systems (HBSs) for drug delivery are nowadays extensively used and the interest in modeling their behavior is dramatically increasing. In this review a critical overview on the modeling approaches is given, quantitatively and qualitatively analyzing the publications on the subject, the trend of the publications per year and the type of modeling approaches. It was found that, despite the drug release fitting models (i.e. Higuchi\\'s equation) are the most abundant, their use for HBSs is decreasing in the last years and luckily, considering the limiting assumption on which they were built, they will be confined to simple mathematical fitting equations. Within the mechanistic models the \"multi-component\" with the swelling approximation (mass transport only) and with the mechanics (fully coupled) are experiencing the highest growth rate, with much more interest toward the last one that, in the next years could be able to provide a first principles model. Statistical models, especially based on the response surface methodology, are rapidly spreading in the scientific community mainly thanks to their ability to be predictive, regardless of the phenomenology, in the analyzed design space with very low efforts. Neural Networks models for HBSs, in countertrend with their use in the pharmaceutical industry, have never take off preferring less data demanding statistical models.',\n",
              "  'authors': ['Caccavo, Diego'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'International journal of pharmaceutics',\n",
              "  'keywords': ['Drug Delivery Systems',\n",
              "   'Drug Liberation',\n",
              "   'Humans',\n",
              "   'Hydrogels',\n",
              "   'Models, Statistical',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '30763681',\n",
              "  'title': \"An overview on the mathematical modeling of hydrogels' behavior for drug delivery systems.\",\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': \"Long-distance ocean voyages may have substantial impacts on seamen's health, possibly causing malnutrition and other illness. Measures can possibly be taken to prevent such problems from happening through preparing special diet and making special precautions prior or during the sailing if a detailed understanding can be gained about what specific health effects such voyages may have on the seamen.|We present a computational study on 200 seamen using 41 chemistry indicators measured on their blood samples collected before and after the sailing. Our computational study is done using a data classification approach with a support vector machine-based classifier in conjunction with feature selections using a recursive feature elimination procedure.|Our analysis results suggest that among the 41 blood chemistry measures, nine are most likely to be affected during the sailing, which provide important clues about the specific effects of ocean voyage on seamen's health.|The identification of the nine blood chemistry measures provides important clues about the effects of long-distance voyage on seamen's health. These findings will prove to be useful to guide in improving the living and working environment, as well as food preparation on ships.\",\n",
              "  'authors': ['Lu, Yunmei',\n",
              "   'Gao, Yanhong',\n",
              "   'Cao, Zhongbo',\n",
              "   'Cui, Juan',\n",
              "   'Dong, Zhennan',\n",
              "   'Tian, Yaping',\n",
              "   'Xu, Ying'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Blood Chemical Analysis',\n",
              "   'Data Collection',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Naval Medicine',\n",
              "   'Ships',\n",
              "   'Travel'],\n",
              "  'pmid': '20219089',\n",
              "  'title': 'A study of health effects of long-distance ocean voyages on seamen using a data classification approach.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1477-9145',\n",
              "  'IF_2013': '3.002',\n",
              "  'IF_2014': '2.897',\n",
              "  'IF_2015': '2.914',\n",
              "  'IF_2016': '3.32',\n",
              "  'IF_2017': '3.179',\n",
              "  'ISSN': '0022-0949',\n",
              "  'ISSN_online': '1477-9145',\n",
              "  'ISSN_print': '0022-0949',\n",
              "  'IsoAbbr': 'J. Exp. Biol.',\n",
              "  'JrId': 4720,\n",
              "  'MedAbbr': 'J Exp Biol',\n",
              "  'NLMid': '0243705',\n",
              "  'Titre': 'J. Exp. Biol.',\n",
              "  'abstract': 'This paper reviews current mathematical models of sphincters and compares them with a new spatial neuromuscular control model based on known physiological properties. Almost all the sphincter models reviewed were constructed as a component of a more extensive model designed to mirror the overall behaviour of a larger system such as the lower urinary tract. This implied less detailed modelling of the sphincter component. It is concluded that current sphincter models are not suitable for mimicking detailed interactions between a neural controller and a sphincter. We therefore outline a new integrated model of the biomechanics and neural control of a sphincter. The muscle is represented as a lumped-mass model, providing the possibility of applying two- or three-dimensional modelling strategies. The neural network is a multi-compartment model that provides neural control signals at the level of action potentials. The integrated model was used to simulate a uniformly activated sphincter and a partially deficient innervation of the sphincter, resulting in a non-uniformly activated sphincter muscle. During the simulation, the pressure in the sphincter lumen was prescribed to increase sinusoidally to a value of 60 kPa. In the uniformly activated situation, the sphincter muscle remains closed, whereas the partially denervated sphincter is stretched open, although the muscle is intact.',\n",
              "  'authors': ['Heldoorn, M', 'Van Leeuwen, J L', 'Vanderschoot, J'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'The Journal of experimental biology',\n",
              "  'keywords': ['Animals',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Mathematics',\n",
              "   'Models, Biological',\n",
              "   'Muscle Denervation',\n",
              "   'Muscles',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Urinary Tract'],\n",
              "  'pmid': '11809776',\n",
              "  'title': 'Modelling the biomechanics and control of sphincters.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': \"Messenger RNA sequences possess specific nucleotide patterns distinguishing them from non-coding genomic sequences. In this study, we explore the utilization of modified Markov models to analyze sequences up to 44 bp, far beyond the 8-bp limit of conventional Markov models, for exon/intron discrimination. In order to analyze nucleotide sequences of this length, their information content is first reduced by conversion into shorter binary patterns via the application of numerous abstraction schemes. After the conversion of genomic sequences to binary strings, homogenous Markov models trained on the binary sequences are used to discriminate between exons and introns. We term this approach the Binary Abstraction Markov Model (BAMM). High-quality abstraction schemes for exon/intron discrimination are selected using optimization algorithms on supercomputers. The best MM classifiers are then combined using support vector machines into a single classifier. With this approach, over 95% classification accuracy is achieved without taking reading frame into account. With further development, the BAMM approach can be applied to sequences lacking the genetic code such as ncRNAs and 5'-untranslated regions.\",\n",
              "  'authors': ['Shepard, Samuel S',\n",
              "   'McSweeny, Andrew',\n",
              "   'Serpen, Gursel',\n",
              "   'Fedorov, Alexei'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Codon',\n",
              "   'Exons',\n",
              "   'Humans',\n",
              "   'Introns',\n",
              "   'Markov Chains',\n",
              "   'Sequence Analysis, DNA',\n",
              "   'Support Vector Machine',\n",
              "   'Untranslated Regions'],\n",
              "  'pmid': '22344692',\n",
              "  'title': 'Exploiting mid-range DNA patterns for sequence classification: binary abstraction Markov models.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1471-2962',\n",
              "  'IF_2013': '2.864',\n",
              "  'IF_2014': '2.147',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.97',\n",
              "  'IF_2017': '2.746',\n",
              "  'ISSN': '1364-503X',\n",
              "  'ISSN_online': '1471-2962',\n",
              "  'ISSN_print': '1364-503X',\n",
              "  'IsoAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'JrId': 30111,\n",
              "  'MedAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'NLMid': '101133385',\n",
              "  'Titre': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'abstract': \"Molecular biology is based on two great discoveries: the first is that genes carry hereditary information in the form of linear sequences of nucleotides; the second is that in protein synthesis a sequence of nucleotides is translated into a sequence of amino acids, a process that amounts to a transfer of information from genes to proteins. These discoveries have shown that the information of genes and proteins is the specific linear order of their sequences. This is a clear definition of information and there is no doubt that it reflects an experimental reality. What is not clear, however, is the ontological status of information, and the result is that today we have two conflicting paradigms in biology. One is the 'chemical paradigm', the idea that 'life is chemistry', or, more precisely, that 'life is an extremely complex form of chemistry'. The other is the 'information paradigm', the view that chemistry is not enough, that 'life is chemistry plus information'. This implies that there is an ontological difference between information and chemistry, a difference which is often expressed by saying that information-based processes like heredity and natural selection simply do not exist in the world of chemistry. Against this conclusion, the supporters of the chemical paradigm have argued that the concept of information is only a linguistic metaphor, a word that summarizes the result of countless underlying chemical reactions. The supporters of the information paradigm insist that information is a real and fundamental component of the living world, but have not been able to prove this point. As a result, the chemical view has not been abandoned and the two paradigms both coexist today. Here, it is shown that a solution to the ontological problem of information does exist. It comes from the idea that life is artefact-making, that genes and proteins are molecular artefacts manufactured by molecular machines and that artefacts necessarily require sequences and coding rules in addition to the quantities of physics and chemistry. More precisely, it is shown that the production of artefacts requires new observables that are referred to as nominable entities because they can be described only by naming their components in their natural order. From an ontological point of view, in conclusion, information is a nominable entity, a fundamental but not-computable observable.\",\n",
              "  'authors': ['Barbieri, Marcello'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Philosophical transactions. Series A, Mathematical, physical, and engineering sciences',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Life',\n",
              "   'Models, Biological',\n",
              "   'Molecular Biology',\n",
              "   'Proteins'],\n",
              "  'pmid': '26857676',\n",
              "  'title': 'What is information?',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'The aim of this study is to find a useful methodology to classify multiple distinct pulmonary conditions including the healthy condition and various pathological types, using pulmonary sounds data.|Fourteen-channel pulmonary sounds data of 40 subjects (healthy and pathological, where the pathologies are of obstructive and restrictive types) are modeled using a second order 250-point vector autoregressive model. The estimated model parameters are fed to support vector machine and Gaussian mixture model (GMM) classifiers which are used in various configurations, resulting in eight different methodologies in total.|Among the eight methodologies, the hierarchical GMM classifier yields the best performance with a total correct classification rate of 85%, where the term hierarchical refers here to first classifying the data into healthy and pathological classes, then the pathological class into obstructive and restrictive types. Both the sensitivity and specificity for the healthy versus pathological classification at the first stage of hierarchy are 90%.|The newly proposed methodologies provide improved results compared to the previous study. The hierarchical framework is suggested for diagnostic classification of pulmonary sounds, although the algorithms are still open for further improvements.|This study proposes new methodologies for diagnostic classification of pulmonary sounds, and suggests using a hierarchical framework for the first time.',\n",
              "  'authors': ['Sen, Ipek', 'Saraclar, Murat', 'Kahya, Yasemin P'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Humans',\n",
              "   'Lung Diseases',\n",
              "   'Normal Distribution',\n",
              "   'Respiratory Sounds',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Spirometry',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '25700439',\n",
              "  'title': 'A Comparison of SVM and GMM-Based Classifier Configurations for Diagnostic Classification of Pulmonary Sounds.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1364-3703',\n",
              "  'IF_2013': '4.485',\n",
              "  'IF_2014': '4.724',\n",
              "  'IF_2015': '4.335',\n",
              "  'IF_2016': '4.697',\n",
              "  'IF_2017': '4.188',\n",
              "  'ISSN': '1464-6722',\n",
              "  'ISSN_online': '1364-3703',\n",
              "  'ISSN_print': '1464-6722',\n",
              "  'IsoAbbr': 'Mol. Plant Pathol.',\n",
              "  'JrId': 21717,\n",
              "  'MedAbbr': 'Mol Plant Pathol',\n",
              "  'NLMid': '100954969',\n",
              "  'Titre': 'Mol. Plant Pathol.',\n",
              "  'abstract': 'Tobacco vein banding mosaic virus (TVBMV) is a potyvirus which mainly infects solanaceous crops. The helper component proteinase (HCpro) of a potyvirus is an RNA silencing suppressor protein and determines the severity of disease symptoms caused by different potyviruses, including TVBMV. It has been shown that substitution mutations introduced into the HCpro open reading frame (ORF) in a TVBMV infectious clone result in changes of Asp189 to Lys or Ile250 -Gln251 to Asp-Glu (Asp, aspartic acid; Gln, glutamine; Glu, glutamic acid; Ile, isoleucine). These amino acid changes eliminate the RNA silencing suppression activity of the mutant HCpro (HCm) and attenuate the disease symptoms caused by the mutant TVBMV (T-HCm) in Nicotiana benthamiana plants. Here, we used RNA-sequencing technology to compare gene expression in plants inoculated with the wild-type TVBMV (T-WT) with that in plants inoculated with T-HCm at 1, 2 and 10 days post-agroinfiltration (dpai). At 1 and 2 dpai, N. benthamiana genes related to the translation machinery were up-regulated, whereas genes related to lipid biosynthesis and metabolism or to responses to extracellular/external stimuli were down-regulated in leaves inoculated with T-WT or T-HCm. At 10 dpai, T-WT infection repressed photosynthesis-related genes. T-WT and T-HCm infections differentially perturbed the genes involved in the RNA silencing pathway. The salicylic acid and ethylene signalling pathways were induced, but the jasmonic acid signalling pathway was repressed after T-WT infection. Infections of T-WT and T-HCm also differentially regulated the genes involved in auxin signalling transduction, which is known to associate with the stunting phenotypes caused by TVBMV. These results illustrate the dynamic nature of TVBMV infection in N. benthamiana at the transcriptomic level.',\n",
              "  'authors': ['Geng, Chao',\n",
              "   'Wang, Hong-Yan',\n",
              "   'Liu, Jin',\n",
              "   'Yan, Zhi-Yong',\n",
              "   'Tian, Yan-Ping',\n",
              "   'Yuan, Xue-Feng',\n",
              "   'Gao, Rui',\n",
              "   'Li, Xiang-Dong'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Molecular plant pathology',\n",
              "  'keywords': ['Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Mutation',\n",
              "   'Photosynthesis',\n",
              "   'Plant Growth Regulators',\n",
              "   'Potyvirus',\n",
              "   'RNA Interference',\n",
              "   'Reproducibility of Results',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Signal Transduction',\n",
              "   'Tobacco',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27539720',\n",
              "  'title': 'Transcriptomic changes in Nicotiana benthamiana plants inoculated with the wild-type or an attenuated mutant of Tobacco vein banding mosaic virus.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Whole knee joint MR image datasets were used to compare the performance of geometric trabecular bone features and advanced machine learning techniques in predicting biomechanical strength properties measured on the corresponding ex vivo specimens. Changes of trabecular bone structure throughout the proximal tibia are indicative of several musculoskeletal disorders involving changes in the bone quality and the surrounding soft tissue. Recent studies have shown that MR imaging also allows non-invasive 3-D characterization of bone microstructure. Sophisticated features like the scaling index method (SIM) can estimate local structural and geometric properties of the trabecular bone and may improve the ability of MR imaging to determine local bone quality in vivo. A set of 67 bone cubes was extracted from knee specimens and their biomechanical strength estimated by the yield stress (YS) [in MPa] was determined through mechanical testing. The regional apparent bone volume fraction (BVF) and SIM derived features were calculated for each bone cube. A linear multiregression analysis (MultiReg) and a optimized support vector regression (SVR) algorithm were used to predict the YS from the image features. The prediction accuracy was measured by the root mean square error (RMSE) for each image feature on independent test sets. The best prediction result with the lowest prediction error of RMSE = 1.021 MPa was obtained with a combination of BVF and SIM features and by using SVR. The prediction accuracy with only SIM features and SVR (RMSE = 1.023 MPa) was still significantly better than BVF alone and MultiReg (RMSE = 1.073 MPa). The current study demonstrates that the combination of sophisticated bone structure features and supervised learning techniques can improve MR-based determination of trabecular bone quality.',\n",
              "  'authors': ['Huber, Markus B',\n",
              "   'Lancianese, Sarah L',\n",
              "   'Nagarajan, Mahesh B',\n",
              "   'Ikpot, Imoh Z',\n",
              "   'Lerner, Amy L',\n",
              "   'Wismuller, Axel'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Databases, Factual',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Knee Joint',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Models, Biological',\n",
              "   'Regression Analysis',\n",
              "   'Shear Strength',\n",
              "   'Tibia'],\n",
              "  'pmid': '21356612',\n",
              "  'title': 'Prediction of biomechanical properties of trabecular bone in MR images with geometric features and support vector regression.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'Near infrared (NIR) spectroscopy was applied for the fast and nondestructive determination of malondialdehyde (MDA) content in oilseed rape leaves. A total of 90 leaf samples were collected, the calibration set was composed of 60 samples, and the prediction set was composed of 30 samples. Different preprocessing methods were used before the calibration stage, including smoothing, standard normal variate, first and second derivative, and detrending. Then partial least squares (PLS) models were developed for the prediction of MDA content in oilseed rape leaves. The latent variables selected by PLS and effective wavelengths selected by successive projections algorithm (SPA) were used as the inputs of least square-support vector machine (LS-SVM) to develop LV-LS-SVM and SPA-LS-SVM models. The correlation coefficients (r) and root mean square error of prediction (RMSEP) were used as the model evaluation indices. Excellent results were achieved by LV-LS-SVM model, and the prediction results by LS-SVM model using detrending spectra were r = 0.999 9 and RMSEP = 0.530 2, and those by LS-SVM model using 2-Der spectra were r = 0.999 9 and RMSEP = 0.395 7. The results showed that NIR spectroscopy could be used for determination of MDA content in oilseed rape leaves, and an excellent prediction precision was achieved. This study supplied a new approach to the dynamic and continuous field monitoring of growing status of oilseed rape.',\n",
              "  'authors': ['Kong, Wen-Wen',\n",
              "   'Liu, Fei',\n",
              "   'Zou, Qiang',\n",
              "   'Fang, Hui',\n",
              "   'He, Yong'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brassica napus',\n",
              "   'Calibration',\n",
              "   'Least-Squares Analysis',\n",
              "   'Malondialdehyde',\n",
              "   'Models, Theoretical',\n",
              "   'Plant Leaves',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '21714244',\n",
              "  'title': '[Fast determination of malondialdehyde in oilseed rape leaves using near infrared spectroscopy].',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': \"Structural brain imaging is playing a vital role in identification of changes that occur in brain associated with Alzheimer's disease. This paper proposes an automated image processing based approach for the identification of AD from MRI of the brain. The proposed approach is novel in a sense that it has higher specificity/accuracy values despite the use of smaller feature set as compared to existing approaches. Moreover, the proposed approach is capable of identifying AD patients in early stages. The dataset selected consists of 85 age and gender matched individuals from OASIS database. The features selected are volume of GM, WM, and CSF and size of hippocampus. Three different classification models (SVM, MLP, and J48) are used for identification of patients and controls. In addition, an ensemble of classifiers, based on majority voting, is adopted to overcome the error caused by an independent base classifier. Ten-fold cross validation strategy is applied for the evaluation of our scheme. Moreover, to evaluate the performance of proposed approach, individual features and combination of features are fed to individual classifiers and ensemble based classifier. Using size of left hippocampus as feature, the accuracy achieved with ensemble of classifiers is 93.75%, with 100% specificity and 87.5% sensitivity. \",\n",
              "  'authors': ['Farhan, Saima', 'Fahiem, Muhammad Abuzar', 'Tauseef, Huma'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Alzheimer Disease',\n",
              "   'Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Case-Control Studies',\n",
              "   'Decision Trees',\n",
              "   'Female',\n",
              "   'Hippocampus',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '25276224',\n",
              "  'title': \"An ensemble-of-classifiers based approach for early diagnosis of Alzheimer's disease: classification using structural features of brain images.\",\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Multiple studies have provided evidence for distributed object representation in the brain, with several recent experiments leveraging basis function estimates for partial image reconstruction from fMRI data. Using a novel combination of statistical decomposition, generalized linear models, and stimulus averaging on previously examined image sets and Bayesian regression of recorded fMRI activity during presentation of these data sets, we identify a subset of relevant voxels that appear to code for covarying object features. Using a technique we term \"voxel-weighted averaging,\" we isolate image filters that these voxels appear to implement. The results, though very cursory, appear to have significant implications for hierarchical and deep-learning-type approaches toward the understanding of neural coding and representation. ',\n",
              "  'authors': ['Hart, Corey B', 'Rose, William J'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Bayes Theorem',\n",
              "   'Brain',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Photic Stimulation',\n",
              "   'Principal Component Analysis'],\n",
              "  'pmid': '23782790',\n",
              "  'title': 'Visual feature extraction from voxel-weighted averaging of stimulus images in 2 fMRI studies.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1095-9203',\n",
              "  'IF_2013': '31.477',\n",
              "  'IF_2014': '33.611',\n",
              "  'IF_2015': '34.661',\n",
              "  'IF_2016': '37.205',\n",
              "  'IF_2017': '41.058',\n",
              "  'ISSN': '0036-8075',\n",
              "  'ISSN_online': '1095-9203',\n",
              "  'ISSN_print': '0036-8075',\n",
              "  'IsoAbbr': 'Science',\n",
              "  'JrId': 7473,\n",
              "  'MedAbbr': 'Science',\n",
              "  'NLMid': '0404511',\n",
              "  'Titre': 'Science',\n",
              "  'abstract': \"As technical obstacles yield, en masse protein testing is poised to take one of biochemistry's most exciting techniques into the heart of cellular chemistry. On page 1760 of this issue, researchers report creating arrays of over 10,000 proteins on a piece of glass just half the size of a microscope slide. They then used their arrays to study a variety of protein functions, including identifying members of the array that bind to other free-floating proteins and to small, druglike molecules.\",\n",
              "  'authors': ['Service, R F'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Science (New York, N.Y.)',\n",
              "  'keywords': ['Biochemistry',\n",
              "   'Drug Evaluation, Preclinical',\n",
              "   'Molecular Probe Techniques',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Protein Binding',\n",
              "   'Protein Conformation',\n",
              "   'Proteins',\n",
              "   'Robotics',\n",
              "   'Serum Albumin, Bovine'],\n",
              "  'pmid': '11001728',\n",
              "  'title': \"Biochemistry. Protein arrays step out of DNA's shadow.\",\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'Robotic applications have achieved safe and precise thyroidectomy with notable cosmetic and functional benefits. This study was designed to document the influence of body habitus on robotic thyroidectomy in papillary thyroid carcinoma (PTC) patients.|From July 2009 to February 2010, 352 patients underwent robotic thyroidectomy using a gasless, transaxillary single-incision approach at Yonsei University Health System. Body habitus was described using body mass index category (normal weight, overweight, obese), neck length, shoulder width, and shoulder width to neck length ratios. The impact of body habitus on surgical outcomes was analyzed with respect to operation time, number of retrieved central nodes, bleeding amount, and postoperative complications.|Of the 352 patients, 217 underwent less than total thyroidectomy and 135 underwent total thyroidectomy. Operative variables (i.e. operation times, bleeding amounts, and numbers of retrieved central nodes) showed no significant differences between three BMI groups for less than total thyroidectomy. However, total operation and working space times were longer for obese patients during total thyroidectomy. In particular, shoulder width was positively correlated with total operation time, working space time, console time, and number of retrieved central nodes. On the other hand, postoperative complications were not significantly different in the three BMI groups and showed no significant correlation with the other indices of body habitus.|Standardized robotic thyroidectomy can be performed safely and feasibly in patients with a large body habitus despite longer operation times.',\n",
              "  'authors': ['Lee, Sohee',\n",
              "   'Park, Seulkee',\n",
              "   'Lee, Cho Rok',\n",
              "   'Son, Haiyoung',\n",
              "   'Kim, Jungwoo',\n",
              "   'Kang, Sang-Wook',\n",
              "   'Jeong, Jong Ju',\n",
              "   'Nam, Kee-Hyun',\n",
              "   'Chung, Woong Youn',\n",
              "   'Park, Cheong Soo'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Anthropometry',\n",
              "   'Blood Loss, Surgical',\n",
              "   'Body Mass Index',\n",
              "   'Carcinoma, Papillary',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neck',\n",
              "   'Operative Time',\n",
              "   'Patient Outcome Assessment',\n",
              "   'Postoperative Complications',\n",
              "   'Robotics',\n",
              "   'Shoulder',\n",
              "   'Thyroid Neoplasms',\n",
              "   'Thyroidectomy',\n",
              "   'Young Adult'],\n",
              "  'pmid': '23344507',\n",
              "  'title': 'The impact of body habitus on the surgical outcomes of transaxillary single-incision robotic thyroidectomy in papillary thyroid carcinoma patients.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['HY'],\n",
              "  'ESSN': '1542-9768',\n",
              "  'IF_2013': '3.865',\n",
              "  'IF_2014': '2.633',\n",
              "  'IF_2015': '3.12',\n",
              "  'IF_2016': '3.451',\n",
              "  'IF_2017': '3.531',\n",
              "  'ISSN': '1542-975X',\n",
              "  'ISSN_online': '1542-9768',\n",
              "  'ISSN_print': '1542-975X',\n",
              "  'IsoAbbr': 'Birth Defects Res. C Embryo Today',\n",
              "  'JrId': 30351,\n",
              "  'MedAbbr': 'Birth Defects Res C Embryo Today',\n",
              "  'NLMid': '101167665',\n",
              "  'Titre': 'Birth Defects Res. C Embryo Today',\n",
              "  'abstract': 'An emerging research area in computational biology and biotechnology is devoted to mathematical modeling and prediction of gene-expression patterns; it nowadays requests mathematics to deeply understand its foundations. This article surveys data mining and machine learning methods for an analysis of complex systems in computational biology. It mathematically deepens recent advances in modeling and prediction by rigorously introducing the environment and aspects of errors and uncertainty into the genetic context within the framework of matrix and interval arithmetics. Given the data from DNA microarray experiments and environmental measurements, we extract nonlinear ordinary differential equations which contain parameters that are to be determined. This is done by a generalized Chebychev approximation and generalized semi-infinite optimization. Then, time-discretized dynamical systems are studied. By a combinatorial algorithm which constructs and follows polyhedra sequences, the region of parametric stability is detected. In addition, we analyze the topological landscape of gene-environment networks in terms of structural stability. As a second strategy, we will review recent model selection and kernel learning methods for binary classification which can be used to classify microarray data for cancerous cells or for discrimination of other kind of diseases. This review is practically motivated and theoretically elaborated; it is devoted to a contribution to better health care, progress in medicine, a better education, and more healthy living conditions.',\n",
              "  'authors': ['Weber, Gerhard-Wilhelm',\n",
              "   'Ozöğür-Akyüz, Süreyya',\n",
              "   'Kropat, Erik'],\n",
              "  'categories': ['DEVELOPMENTAL BIOLOGY'],\n",
              "  'journal': 'Birth defects research. Part C, Embryo today : reviews',\n",
              "  'keywords': ['Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Computational Biology',\n",
              "   'Congenital Abnormalities',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Electronic Data Processing',\n",
              "   'Fungal Proteins',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Numerical Analysis, Computer-Assisted',\n",
              "   'Oligonucleotide Array Sequence Analysis'],\n",
              "  'pmid': '19530130',\n",
              "  'title': 'A review on data mining and continuous optimization applications in computational biology and medicine.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1687-5443',\n",
              "  'IF_2013': '3.608',\n",
              "  'IF_2014': '3.582',\n",
              "  'IF_2015': '3.568',\n",
              "  'IF_2016': '3.054',\n",
              "  'IF_2017': '3.161',\n",
              "  'ISSN': '2090-5904',\n",
              "  'ISSN_online': '1687-5443',\n",
              "  'ISSN_print': '2090-5904',\n",
              "  'IsoAbbr': 'Neural Plast.',\n",
              "  'JrId': 21183,\n",
              "  'MedAbbr': 'Neural Plast',\n",
              "  'NLMid': '100883417',\n",
              "  'Titre': 'Neural Plast.',\n",
              "  'abstract': 'Motor imagery electroencephalography (EEG) has been successfully used in locomotor rehabilitation programs. While the noise-assisted multivariate empirical mode decomposition (NA-MEMD) algorithm has been utilized to extract task-specific frequency bands from all channels in the same scale as the intrinsic mode functions (IMFs), identifying and extracting the specific IMFs that contain significant information remain difficult. In this paper, a novel method has been developed to identify the information-bearing components in a low-dimensional subspace without prior knowledge. Our method trains a Gaussian mixture model (GMM) of the composite data, which is comprised of the IMFs from both the original signal and noise, by employing kernel spectral regression to reduce the dimension of the composite data. The informative IMFs are then discriminated using a GMM clustering algorithm, the common spatial pattern (CSP) approach is exploited to extract the task-related features from the reconstructed signals, and a support vector machine (SVM) is applied to the extracted features to recognize the classes of EEG signals during different motor imagery tasks. The effectiveness of the proposed method has been verified by both computer simulations and motor imagery EEG datasets.',\n",
              "  'authors': ['She, Qingshan',\n",
              "   'Gan, Haitao',\n",
              "   'Ma, Yuliang',\n",
              "   'Luo, Zhizeng',\n",
              "   'Potter, Tom',\n",
              "   'Zhang, Yingchun'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Neural plasticity',\n",
              "  'keywords': ['Electroencephalography',\n",
              "   'Humans',\n",
              "   'Imagery (Psychotherapy)',\n",
              "   'Motor Skills',\n",
              "   'Psychomotor Performance',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '27891256',\n",
              "  'title': 'Scale-Dependent Signal Identification in Low-Dimensional Subspace: Motor Imagery Task Classification.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': 'There is a need for providing sensory feedback for myoelectric prosthesis users. Providing tactile feedback can improve object manipulation abilities, enhance the perceptual embodiment of myoelectric prostheses and help reduce phantom limb pain. Many amputees have referred sensation from their missing hand on their residual limbs (phantom maps). This skin area can serve as a target for providing amputees with non-invasive tactile sensory feedback. One of the challenges of providing sensory feedback on the phantom map is to define the accurate boundary of each phantom digit because the phantom map distribution varies from person to person.|In this paper, automatic phantom map detection methods based on four decomposition support vector machine algorithms and three sampling methods are proposed, complemented by fuzzy logic and active learning strategies. The algorithms and methods are tested on two databases: the first one includes 400 generated phantom maps, whereby the phantom map generation algorithm was based on our observation of the phantom maps to ensure smooth phantom digit edges, variety, and representativeness. The second database includes five reported phantom map images and transformations thereof. The accuracy and training/ classification time of each algorithm using a dense stimulation array (with 100 [Formula: see text] 100 actuators) and two coarse stimulation arrays (with 3 [Formula: see text] 5 and 4 [Formula: see text] 6 actuators) are presented and compared.|Both generated and reported phantom map images share the same trends. Majority-pooling sampling effectively increases the training size, albeit introducing some noise, and thus produces the smallest error rates among the three proposed sampling methods. For different decomposition architectures, one-vs-one reduces unclassified regions and in general has higher classification accuracy than the other architectures. By introducing fuzzy logic to bias the penalty parameter, the influence of pooling-induced noise is reduced. Moreover, active learning with different strategies was also tested and shown to improve the accuracy by introducing more representative training samples. Overall, dense arrays employing one-vs-one fuzzy support vector machines with majority-pooling sampling have the smallest average absolute error rate (8.78% for generated phantom maps and 11.5% for reported and transformed phantom map images). The detection accuracy of coarse arrays was found to be significantly lower than for dense array.|The results demonstrate the effectiveness of support vector machines using a dense array in detecting refined phantom map shapes, whereas coarse arrays are unsuitable for this task. We therefore propose a two-step approach, using first a non-wearable dense array to detect an accurate phantom map shape, then to apply a wearable coarse stimulation array customized according to the detection results. The proposed methodology can be used as a tool for helping haptic feedback designers and for tracking the evolvement of phantom maps.',\n",
              "  'authors': ['Huang, Huaiqi',\n",
              "   'Bruschini, Claudio',\n",
              "   'Antfolk, Christian',\n",
              "   'Enz, Christian',\n",
              "   'Li, Tao',\n",
              "   'Justiz, Jörn',\n",
              "   'Koch, Volker M'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Automation',\n",
              "   'Databases, Factual',\n",
              "   'Fuzzy Logic',\n",
              "   'Hand',\n",
              "   'Humans',\n",
              "   'Phantoms, Imaging',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29890988',\n",
              "  'title': 'Automatic hand phantom map generation and detection using decomposition support vector machines.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1618-727X',\n",
              "  'IF_2013': '1.2',\n",
              "  'IF_2014': '1.19',\n",
              "  'IF_2015': '1.406',\n",
              "  'IF_2016': '1.407',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0897-1889',\n",
              "  'ISSN_online': '1618-727X',\n",
              "  'ISSN_print': '0897-1889',\n",
              "  'IsoAbbr': 'J Digit Imaging',\n",
              "  'JrId': 1047,\n",
              "  'MedAbbr': 'J Digit Imaging',\n",
              "  'NLMid': '9100529',\n",
              "  'Titre': 'J Digit Imaging',\n",
              "  'abstract': 'Medical imaging technologies have allowed for in vivo exploration and evaluation of the human musculoskeletal system. Three-dimensional bone models generated using image-segmentation techniques provide a means to optimize individualized orthopedic surgical procedures using engineering analyses. However, many of the current segmentation techniques are not clinically practical due to the required time and human intervention. As a proof of concept, we demonstrate the use of an expectation maximization (EM) algorithm to segment the hand phalanx bones, and hypothesize that this semi-automated technique will improve the efficiency while providing similar definitions as compared to a manual rater. Our results show a relative overlap of the proximal, middle, and distal phalanx bones of 0.83, 0.79, and 0.72 for the EM technique when compared to validated manual segmentations. The EM segmentations were also compared to 3D surface scans of the cadaveric specimens, which resulted in distance maps showing an average distance for the proximal, middle, and distal phalanx bones of 0.45, 0.46, and 0.51 mm, respectively. The EM segmentation improved on the segmentation speed of the manual techniques by a factor of eight. Overall, the manual segmentations had greater relative overlap metric values, which suggests that the manual segmentations are a better fit to the actual surface of the bone. As shown by the comparison to the bone surface scans, the EM technique provides a similar representation of the anatomic structure and offers an increase in efficiency that could help to reduce the time needed for defining anatomical structures from CT scans.',\n",
              "  'authors': ['Ramme, Austin J',\n",
              "   'DeVries, Nicole',\n",
              "   'Kallemyn, Nicole A',\n",
              "   'Magnotta, Vincent A',\n",
              "   'Grosland, Nicole M'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of digital imaging',\n",
              "  'keywords': ['Aged',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Cadaver',\n",
              "   'Female',\n",
              "   'Finger Phalanges',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '18769967',\n",
              "  'title': 'Semi-automated phalanx bone segmentation using the expectation maximization algorithm.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Real-time simulation of organ deformation is one of the biggest challenges in virtual surgery, due to the conflicting requirements of real-time interactivity and simulation realism. In this paper we propose a method to overcome this challenge by introducing a multi-resolution modelling technique.|In our approach a reasonably coarse global model is locally enhanced, using a mesh subdivision and smoothing algorithm. The global model is based on a discretization of the boundary integral representation of three-dimensional deformable objects. Local refinements are provided at the tool-tissue interaction region by a local subdivision technique.|As an example, we have developed a deformable human kidney model generated from the Visible Human Dataset, with tissue properties determined from in vivo animal experiments. A mixed reality laparoscopic surgical training system has been developed, using an abdominal mannequin and force feedback devices.|The use of precomputation and structural re-analysis techniques results in a very rapid computation procedure. Validation of the simulator is in progress.',\n",
              "  'authors': ['Kim, Jung',\n",
              "   'Choi, Changmok',\n",
              "   'De, Suvranu',\n",
              "   'Srinivasan, Mandayam A'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Anatomy, Cross-Sectional',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Computer Graphics',\n",
              "   'Computer Simulation',\n",
              "   'Computer-Assisted Instruction',\n",
              "   'Data Display',\n",
              "   'Education, Medical',\n",
              "   'Elasticity',\n",
              "   'Feedback',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Kidney',\n",
              "   'Manikins',\n",
              "   'Models, Anatomic',\n",
              "   'Models, Biological',\n",
              "   'Phantoms, Imaging',\n",
              "   'Robotics',\n",
              "   'Software',\n",
              "   'Urologic Surgical Procedures',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '17619246',\n",
              "  'title': 'Virtual surgery simulation for medical training using multi-resolution organ models.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5305',\n",
              "  'IF_2013': '1.735',\n",
              "  'IF_2014': '1.068',\n",
              "  'IF_2015': '1.069',\n",
              "  'IF_2016': '0.964',\n",
              "  'IF_2017': '1.039',\n",
              "  'ISSN': '0929-8665',\n",
              "  'ISSN_online': '1875-5305',\n",
              "  'ISSN_print': '0929-8665',\n",
              "  'IsoAbbr': 'Protein Pept. Lett.',\n",
              "  'JrId': 20398,\n",
              "  'MedAbbr': 'Protein Pept Lett',\n",
              "  'NLMid': '9441434',\n",
              "  'Titre': 'Protein Pept. Lett.',\n",
              "  'abstract': 'Cysteine S-sulfenylation is a major type of dynamic post-translational modification of the protein that plays an important role in regulating many biological processes in both of prokaryotic and eukaryotic species. To understand the function of S-sulfenylated proteins, identification of S-sulfenylation sites is an essential step. Due to numerous restrictions of experimental methods, computational prediction of the potential S-sulfenylation sites becomes popular. In this review, we discuss the recent development and challenges in protein S-sulfenylation site prediction from the available datasets, algorithms and accessible services. We also demonstrate the encountered limitation and future perspective of the computational prediction tools.|The development of S-sulfenylation site prediction and their application is an emerging field of protein bioinformatics research. Accurate predictors are expected to identify general and species-specific S-sulfenylation sites when more experimental annotation data are available. Combining experimental and computational technologies will definitely accelerate an understanding of protein S-sulfenylation, discovering regulatory networks in living organisms.',\n",
              "  'authors': ['Hasan, Md Mehedi', 'Khatun, Mst Shamima', 'Kurata, Hiroyuki'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Protein and peptide letters',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computational Biology',\n",
              "   'Computer Simulation',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'Proteins',\n",
              "   'Sulfenic Acids',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '30182830',\n",
              "  'title': 'A Comprehensive Review of In silico Analysis for Protein S-sulfenylation Sites.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DX'],\n",
              "  'ESSN': '1873-4294',\n",
              "  'IF_2013': '3.453',\n",
              "  'IF_2014': '3.402',\n",
              "  'IF_2015': '2.9',\n",
              "  'IF_2016': '2.864',\n",
              "  'IF_2017': '3.374',\n",
              "  'ISSN': '1568-0266',\n",
              "  'ISSN_online': '1873-4294',\n",
              "  'ISSN_print': '1568-0266',\n",
              "  'IsoAbbr': 'Curr Top Med Chem',\n",
              "  'JrId': 27009,\n",
              "  'MedAbbr': 'Curr Top Med Chem',\n",
              "  'NLMid': '101119673',\n",
              "  'Titre': 'Curr Top Med Chem',\n",
              "  'abstract': 'Complex network approach allows the representation and analysis of complex systems of interacting agents in an ordered and effective manner, thus increasing the probability of discovering significant properties of them. In the present study, we defined and built for the first time a complex network based on data obtained from Immune Epitope Database for parasitic organisms. We then considered the general topology, the node degree distribution, and the local structure (triadic census) of this network. In addition, we calculated 9 node centrality measures for observed network and reported a comparative study of the real network with three theoretical models to detect similarities or deviations from these ideal networks.|The results obtained corroborate the utility of the complex network approach for handling information and data mining within the database under study.|They confirm that this type of approach can be considered a valuable tool for preliminary screening of the best experimental conditions to determine whether the amino acid sequences being studied are true epitopes or not.',\n",
              "  'authors': ['Vazquez-Prieto, Severo',\n",
              "   'Paniagua, Esperanza',\n",
              "   'Solana, Hugo',\n",
              "   'Ubeira, Florencio M'],\n",
              "  'categories': ['CHEMISTRY, MEDICINAL'],\n",
              "  'journal': 'Current topics in medicinal chemistry',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Animals',\n",
              "   'Data Mining',\n",
              "   'Databases, Factual',\n",
              "   'Epitopes',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Parasites'],\n",
              "  'pmid': '29231143',\n",
              "  'title': 'Complex Network Study of the Immune Epitope Database for Parasitic Organisms.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1744-7593',\n",
              "  'IF_2013': '4.116',\n",
              "  'IF_2014': '4.84',\n",
              "  'IF_2015': '5.434',\n",
              "  'IF_2016': '5.657',\n",
              "  'IF_2017': '5.553',\n",
              "  'ISSN': '1742-5247',\n",
              "  'ISSN_online': '1744-7593',\n",
              "  'ISSN_print': '1742-5247',\n",
              "  'IsoAbbr': 'Expert Opin Drug Deliv',\n",
              "  'JrId': 32621,\n",
              "  'MedAbbr': 'Expert Opin Drug Deliv',\n",
              "  'NLMid': '101228421',\n",
              "  'Titre': 'Expert Opin Drug Deliv',\n",
              "  'abstract': 'Ocular microrobots have the potential to change the way in which we treat a variety of diseases at the anterior and the posterior segments of the eye. Wireless manipulation and positioning of drug delivery magnetic millimeter and submillimeter platforms into the eye constitute a potential route for minimally invasive targeted therapy. However, the field is still in its infancy and faces challenges related to the fabrication, control an interaction with complex biological environments.|This review briefly introduces the complex anatomy and physiology of the eye, which renders limitations to the current treatments of ocular diseases. The topical administration of eye drops, intravitreal injections and drug delivery implants is briefly mentioned together with their drawbacks. The authors also analyze the minimally invasive microrobotic approach as an alternative method and report the recent advancements in the fabrication, control, manipulation and drug delivery.|Although microrobotics is a young field, a significant amount of work has been developed to face different challenges related to the minimally invasive manipulation of microdevices in the eye. Current research is already at the state of in vivo testing for systems and their biocompatibility. It is expected that the general concepts acquired will soon be applied for specific interventions, especially for posterior eye pathologies.',\n",
              "  'authors': ['Fusco, Stefano',\n",
              "   'Ullrich, Franziska',\n",
              "   'Pokki, Juho',\n",
              "   'Chatzipirpiridis, George',\n",
              "   'Özkale, Berna',\n",
              "   'Sivaraman, Kartik M',\n",
              "   'Ergeneman, Olgaç',\n",
              "   'Pané, Salvador',\n",
              "   'Nelson, Bradley J'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Expert opinion on drug delivery',\n",
              "  'keywords': ['Administration, Topical',\n",
              "   'Animals',\n",
              "   'Drug Administration Routes',\n",
              "   'Drug Delivery Systems',\n",
              "   'Drug Implants',\n",
              "   'Eye Diseases',\n",
              "   'Humans',\n",
              "   'Intravitreal Injections',\n",
              "   'Magnetics',\n",
              "   'Ophthalmic Solutions',\n",
              "   'Robotics'],\n",
              "  'pmid': '25001411',\n",
              "  'title': 'Microrobots: a new era in ocular drug delivery.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RA'],\n",
              "  'ESSN': '1878-4291',\n",
              "  'IF_2013': '2.062',\n",
              "  'IF_2014': '1.988',\n",
              "  'IF_2015': '1.838',\n",
              "  'IF_2016': '1.98',\n",
              "  'IF_2017': '1.728',\n",
              "  'ISSN': '0968-4328',\n",
              "  'ISSN_online': '1878-4291',\n",
              "  'ISSN_print': '0968-4328',\n",
              "  'IsoAbbr': 'Micron',\n",
              "  'JrId': 8531,\n",
              "  'MedAbbr': 'Micron',\n",
              "  'NLMid': '9312850',\n",
              "  'Titre': 'Micron',\n",
              "  'abstract': 'With the growing amount of high resolution microscopy images automatic nano-particle detection, shape analysis and size determination have gained importance for providing quantitative support that gives important information for the evaluation of the material. In this paper, we present a new method for detection of nano-particles and determination of their shapes and sizes simultaneously with deep learning. The proposed method employs multiple output convolutional neural networks (MO-CNN) and has two outputs: first is the detection output that gives the locations of the particles and the other one is the segmentation output for providing the boundaries of the nano-particles. The final sizes of particles are determined with the modified Hough algorithm that runs on the segmentation output. The proposed method is tested and evaluated on a dataset containing 17 TEM images of Fe3O4 and silica coated nano-particles. Also, we compared these results with U-net algorithm which is a popular deep learning method. The experiments showed that the proposed method has 98.23% accuracy for detection and 96.59% accuracy for segmentation of nano-particles.',\n",
              "  'authors': ['Oktay, Ayse Betul', 'Gurses, Anıl'],\n",
              "  'categories': ['MICROSCOPY'],\n",
              "  'journal': 'Micron (Oxford, England : 1993)',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30844638',\n",
              "  'title': 'Automatic detection, localization and segmentation of nano-particles with deep learning in microscopy images.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1522-2586',\n",
              "  'IF_2013': '2.788',\n",
              "  'IF_2014': '3.21',\n",
              "  'IF_2015': '3.25',\n",
              "  'IF_2016': '3.083',\n",
              "  'IF_2017': '3.612',\n",
              "  'ISSN': '1053-1807',\n",
              "  'ISSN_online': '1522-2586',\n",
              "  'ISSN_print': '1053-1807',\n",
              "  'IsoAbbr': 'J Magn Reson Imaging',\n",
              "  'JrId': 2081,\n",
              "  'MedAbbr': 'J Magn Reson Imaging',\n",
              "  'NLMid': '9105850',\n",
              "  'Titre': 'J Magn Reson Imaging',\n",
              "  'abstract': 'To retrospectively evaluate the performance of an automatic support vector machine (SVM) routine in combination with perfusion-based dynamic susceptibility contrast magnetic resonance imaging (DSC-MRI) for preoperative survival associations in patients with gliomas and compare our results to traditional MRI.|The study was approved by the Ethics Committee and informed consent was signed. Structural, diffusion- and perfusion-weighted MRI was performed at 1.5-T preoperatively in 94 adult patients (49 males, 45 females, 23-82 years; mean 51 years) later diagnosed with a primary glioma. Patients were randomly assigned in training and test datasets and the resulting DSC-based survival associations by SVM were compared to traditional MRI features including contrast-agent enhancement, perfusion- and diffusion-weighted imaging, tumor size, and location. The results were adjusted for age, neurological status, and postoperative factors associated with survival, including surgery and adjuvant therapy.|For 1- (26/33 alive, 11/14 deceased), 2- (15/21, 21/26), 3- (12/16, 27/31) and 4- (12/15, 28/32) year survival associations in the test dataset (47 patients), the SVM routine was the only biomarker to consistently associate with survival (Cox; P < 0.001).|The automatic machine learning routine presented in our study may provide the operator with a reliable instrument for assessing survival in patients with glioma.',\n",
              "  'authors': ['Emblem, Kyrre E',\n",
              "   'Due-Tonnessen, Paulina',\n",
              "   'Hald, John K',\n",
              "   'Bjornerud, Atle',\n",
              "   'Pinho, Marco C',\n",
              "   'Scheie, David',\n",
              "   'Schad, Lothar R',\n",
              "   'Meling, Torstein R',\n",
              "   'Zoellner, Frank G'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of magnetic resonance imaging : JMRI',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Brain Neoplasms',\n",
              "   'Female',\n",
              "   'Glioma',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Magnetic Resonance Angiography',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Preoperative Care',\n",
              "   'Prognosis',\n",
              "   'Reproducibility of Results',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Support Vector Machine',\n",
              "   'Survival Rate',\n",
              "   'Treatment Outcome',\n",
              "   'Young Adult'],\n",
              "  'pmid': '24753371',\n",
              "  'title': 'Machine learning in preoperative glioma MRI: survival associations by perfusion-based support vector machine outperforms traditional MRI.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1873-2933',\n",
              "  'IF_2013': '2.229',\n",
              "  'IF_2014': '2.275',\n",
              "  'IF_2015': '2.382',\n",
              "  'IF_2016': '2.434',\n",
              "  'IF_2017': '2.584',\n",
              "  'ISSN': '0009-9120',\n",
              "  'ISSN_online': '1873-2933',\n",
              "  'ISSN_print': '0009-9120',\n",
              "  'IsoAbbr': 'Clin. Biochem.',\n",
              "  'JrId': 2990,\n",
              "  'MedAbbr': 'Clin Biochem',\n",
              "  'NLMid': '0133660',\n",
              "  'Titre': 'Clin. Biochem.',\n",
              "  'abstract': 'Devices constructed for potential use as invasive bioprobes incorporate a selective receiving site for molecular or ionic recognition, and a transducer which is capable of translating a perturbation of physical chemistry of the determinant-site reaction (interaction) into a usable signal. Four types are envisioned--implants for general hospital use, transient-use probes to replace classical blood tests, short-term implantable probes and the long-term variety. Performance criteria are selectivity, sensitivity, fast response, site-reversible, small, rugged, inexpensive, biocompatible, calibratible, facile use by non-expert personnel and ease of telemetry. These demands, not surprisingly, create enormous challenges to the sensor specialist. With respect to biocompatibility the sensor must not be involved in infection, clot formation or antigenic response, and, furthermore, protein adsorption, etc., which can affect the sensor response should be avoided. Calibration remains a problem of monumental proportions. Many devices drift from calibrated levels even in in vitro experiments, let alone in the implanted milieu. One solution has been to carry out on-line switching between patient blood and standard solutions. However, this type of approach leaves a lot to be desired with respect to portability. Another method which is attracting increasing attention is the chemometric or artificial intelligence system involving compensation by multi-sensor array configurations. Sensitivity and limit-of-detection have attracted little research due to the overwhelming nature of other difficulties. In the present paper we evaluate a number of these technical problems and discuss the architecture of devices that are currently available. Finally, some thoughts as to priorities for re-directing sensor research in the bioprobe area are presented.',\n",
              "  'authors': ['Thompson, M', 'Vandenberg, E T'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinical biochemistry',\n",
              "  'keywords': ['Biocompatible Materials',\n",
              "   'Chemistry, Clinical',\n",
              "   'Humans',\n",
              "   'Monitoring, Physiologic',\n",
              "   'Quality Control',\n",
              "   'Transducers'],\n",
              "  'pmid': '3533320',\n",
              "  'title': 'In vivo probes: problems and perspectives.',\n",
              "  'year': '1986'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1433-8726',\n",
              "  'IF_2013': '3.423',\n",
              "  'IF_2014': '2.666',\n",
              "  'IF_2015': '2.397',\n",
              "  'IF_2016': '2.743',\n",
              "  'IF_2017': '2.981',\n",
              "  'ISSN': '0724-4983',\n",
              "  'ISSN_online': '1433-8726',\n",
              "  'ISSN_print': '0724-4983',\n",
              "  'IsoAbbr': 'World J Urol',\n",
              "  'JrId': 2429,\n",
              "  'MedAbbr': 'World J Urol',\n",
              "  'NLMid': '8307716',\n",
              "  'Titre': 'World J Urol',\n",
              "  'abstract': 'Salvage robotic assisted laparoscopic radical prostatectomy (SRALP) has now become a feasible alternative to open prostatectomy in experienced hands. Herein we review the urologic literature for the indications and outcomes of SRALP.|A comprehensive review of the peer reviewed literature was performed for reported cases of salvage robotic and open prostatectomy, with emphasis being on comparing oncologic outcomes.|Salvage prostatectomy, whether open or robotic, is a viable alternative for treatment of recurrent, localized, non-metastatic prostate cancer after failed primary radiotherapy. Although few reports of salvage robotic prostatectomy have been published with limited long-term follow-up, initial oncologic results seem at least comparable to the salvage open prostatectomy series. SRALP should be performed by centers with a dedicated and well experienced robotic urologic oncology program.|SRALP is a feasible surgical treatment option for patients with recurrence after primary radiotherapy with preliminary oncologic outcomes comparable to the open approach. Further long-term follow-up is needed to validate these results.',\n",
              "  'authors': ['Williams, Stephen B', 'Hu, Jim C'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'World journal of urology',\n",
              "  'keywords': ['Biopsy',\n",
              "   'Erectile Dysfunction',\n",
              "   'Humans',\n",
              "   'Incidence',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostate',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Salvage Therapy',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Incontinence'],\n",
              "  'pmid': '21104184',\n",
              "  'title': 'Salvage robotic assisted laparoscopic radical prostatectomy: indications and outcomes.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2154-2317',\n",
              "  'IF_2013': '0.663',\n",
              "  'IF_2014': '0.957',\n",
              "  'IF_2015': '0.558',\n",
              "  'IF_2016': '0.5',\n",
              "  'IF_2017': '0.587',\n",
              "  'ISSN': '2154-2287',\n",
              "  'ISSN_online': '2154-2317',\n",
              "  'ISSN_print': '2154-2287',\n",
              "  'IsoAbbr': 'IEEE Pulse',\n",
              "  'JrId': 38326,\n",
              "  'MedAbbr': 'IEEE Pulse',\n",
              "  'NLMid': '101541727',\n",
              "  'Titre': 'IEEE Pulse',\n",
              "  'abstract': 'Forty years ago, Les Baugh lost both of his arms in an electrical accident. With bilateral shoulder-level amputations, his options for prosthetic arms were limited. That changed two years ago, when Baugh underwent a surgical procedure at Johns Hopkins Hospital in Baltimore that allowed him to control state-of-the-art robotic arms using nerves that had been rerouted to his chest. Within ten days of training, he was able to control both arms simultaneously and move a cup from a lower shelf to a higher shelf-a task that previously had been impossible-just by thinking about how he wanted to move his arm. ',\n",
              "  'authors': ['Allen, Summer'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE pulse',\n",
              "  'keywords': ['Artificial Limbs',\n",
              "   'Biomedical Engineering',\n",
              "   'Humans',\n",
              "   'Rehabilitation',\n",
              "   'Robotics',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '27187533',\n",
              "  'title': 'New Prostheses and Orthoses Step Up their Game: Motorized Knees, Robotic Hands, and Exosuits Mark Advances in Rehabilitation Technology.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1534-7362',\n",
              "  'IF_2013': '2.727',\n",
              "  'IF_2014': '2.393',\n",
              "  'IF_2015': '2.341',\n",
              "  'IF_2016': '2.671',\n",
              "  'IF_2017': '2.266',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1534-7362',\n",
              "  'IsoAbbr': 'J Vis',\n",
              "  'JrId': 30247,\n",
              "  'MedAbbr': 'J Vis',\n",
              "  'NLMid': '101147197',\n",
              "  'Titre': 'J Vis',\n",
              "  'abstract': 'While we are scanning our environment, the retinal image changes with every saccade. Nevertheless, the visual system anticipates where an attended target will be next and attention is updated to the new location. Recently, two different types of perisaccadic attentional updates were discovered: predictive remapping of attention before saccade onset (Rolfs, Jonikaitis, Deubel, & Cavanagh, 2011) and lingering of attention after saccade (Golomb, Chun, & Mazer, 2008; Golomb, Pulido, Albrecht, Chun, & Mazer, 2010). We here propose a neuro-computational model located in lateral intraparietal cortex based on a previous model of perisaccadic space perception (Ziesche & Hamker, 2011, 2014). Our model can account for both types of updating of attention at a neural-systems level. The lingering effect originates from the late updating of the proprioceptive eye-position signal and the remapping from the early corollary-discharge signal. We put these results in relationship to predictive remapping of receptive fields and show that both phenomena arise from the same simple, recurrent neural circuit. Thus, together with the previously published results, the model provides a comprehensive framework for discussing multiple experimental observations that occur around saccades.',\n",
              "  'authors': ['Bergelt, Julia', 'Hamker, Fred H'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Journal of vision',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31323096',\n",
              "  'title': 'Spatial updating of attention across eye movements: A neuro-computational approach.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['HY'],\n",
              "  'ESSN': '1477-9129',\n",
              "  'IF_2013': '6.273',\n",
              "  'IF_2014': '6.462',\n",
              "  'IF_2015': '6.059',\n",
              "  'IF_2016': '5.843',\n",
              "  'IF_2017': '5.413',\n",
              "  'ISSN': '0950-1991',\n",
              "  'ISSN_online': '1477-9129',\n",
              "  'ISSN_print': '0950-1991',\n",
              "  'IsoAbbr': 'Development',\n",
              "  'JrId': 3468,\n",
              "  'MedAbbr': 'Development',\n",
              "  'NLMid': '8701744',\n",
              "  'Titre': 'Development',\n",
              "  'abstract': 'The Drosophila heart is composed of two distinct cell types, the contractile cardial cells (CCs) and the surrounding non-muscle pericardial cells (PCs), development of which is regulated by a network of conserved signaling molecules and transcription factors (TFs). Here, we used machine learning with array-based chromatin immunoprecipitation (ChIP) data and TF sequence motifs to computationally classify cell type-specific cardiac enhancers. Extensive testing of predicted enhancers at single-cell resolution revealed the added value of ChIP data for modeling cell type-specific activities. Furthermore, clustering the top-scoring classifier sequence features identified novel cardiac and cell type-specific regulatory motifs. For example, we found that the Myb motif learned by the classifier is crucial for CC activity, and the Myb TF acts in concert with two forkhead domain TFs and Polo kinase to regulate cardiac progenitor cell divisions. In addition, differential motif enrichment and cis-trans genetic studies revealed that the Notch signaling pathway TF Suppressor of Hairless [Su(H)] discriminates PC from CC enhancer activities. Collectively, these studies elucidate molecular pathways used in the regulatory decisions for proliferation and differentiation of cardiac progenitor cells, implicate Su(H) in regulating cell fate decisions of these progenitors, and document the utility of enhancer modeling in uncovering developmental regulatory subnetworks. ',\n",
              "  'authors': ['Ahmad, Shaad M',\n",
              "   'Busser, Brian W',\n",
              "   'Huang, Di',\n",
              "   'Cozart, Elizabeth J',\n",
              "   'Michaud, Sébastien',\n",
              "   'Zhu, Xianmin',\n",
              "   'Jeffries, Neal',\n",
              "   'Aboukhalil, Anton',\n",
              "   'Bulyk, Martha L',\n",
              "   'Ovcharenko, Ivan',\n",
              "   'Michelson, Alan M'],\n",
              "  'categories': ['DEVELOPMENTAL BIOLOGY'],\n",
              "  'journal': 'Development (Cambridge, England)',\n",
              "  'keywords': ['Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Cell Differentiation',\n",
              "   'Cell Division',\n",
              "   'Chromatin Immunoprecipitation',\n",
              "   'Classification',\n",
              "   'Drosophila',\n",
              "   'Enhancer Elements, Genetic',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Heart',\n",
              "   'Mutagenesis',\n",
              "   'Myoblasts, Cardiac',\n",
              "   'Stem Cells',\n",
              "   'Transcription Factors'],\n",
              "  'pmid': '24496624',\n",
              "  'title': 'Machine learning classification of cell-specific cardiac enhancers uncovers developmental subnetworks regulating progenitor cell division and cell fate specification.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1479-7364',\n",
              "  'IF_2013': '2.146',\n",
              "  'IF_2014': '2.146',\n",
              "  'IF_2015': '3.385',\n",
              "  'IF_2016': '3.327',\n",
              "  'IF_2017': '3.5',\n",
              "  'ISSN': '1473-9542',\n",
              "  'ISSN_online': '1479-7364',\n",
              "  'ISSN_print': '1473-9542',\n",
              "  'IsoAbbr': 'Hum. Genomics',\n",
              "  'JrId': 32174,\n",
              "  'MedAbbr': 'Hum Genomics',\n",
              "  'NLMid': '101202210',\n",
              "  'Titre': 'Hum. Genomics',\n",
              "  'abstract': \"Autosomal dominant polycystic kidney disease (ADPKD) causes progressive loss of renal function in adults as a consequence of the accumulation of cysts. ADPKD is the most common genetic cause of end-stage renal disease. Mutations in polycystin-1 occur in 87% of cases of ADPKD and mutations in polycystin-2 are found in 12% of ADPKD patients. The complexity of ADPKD has hampered efforts to identify the mechanisms underlying its pathogenesis. No current FDA (Federal Drug Administration)-approved therapies ameliorate ADPKD progression.|We used the de Almeida laboratory's sensitive new transcriptogram method for whole-genome gene expression data analysis to analyze microarray data from cell lines developed from cell isolates of normal kidney and of both non-cystic nephrons and cysts from the kidney of a patient with ADPKD. We compared results obtained using standard Ingenuity Volcano plot analysis, Gene Set Enrichment Analysis (GSEA) and transcriptogram analysis. Transcriptogram analysis confirmed the findings of Ingenuity, GSEA, and published analysis of ADPKD kidney data and also identified multiple new expression changes in KEGG (Kyoto Encyclopedia of Genes and Genomes) pathways related to cell growth, cell death, genetic information processing, nucleotide metabolism, signal transduction, immune response, response to stimulus, cellular processes, ion homeostasis and transport and cofactors, vitamins, amino acids, energy, carbohydrates, drugs, lipids, and glycans. Transcriptogram analysis also provides significance metrics which allow us to prioritize further study of these pathways.|Transcriptogram analysis identifies novel pathways altered in ADPKD, providing new avenues to identify both ADPKD's mechanisms of pathogenesis and pharmaceutical targets to ameliorate the progression of the disease.\",\n",
              "  'authors': ['de Almeida, Rita M C',\n",
              "   'Clendenon, Sherry G',\n",
              "   'Richards, William G',\n",
              "   'Boedigheimer, Michael',\n",
              "   'Damore, Michael',\n",
              "   'Rossetti, Sandro',\n",
              "   'Harris, Peter C',\n",
              "   'Herbert, Britney-Shea',\n",
              "   'Xu, Wei Min',\n",
              "   'Wandinger-Ness, Angela',\n",
              "   'Ward, Heather H',\n",
              "   'Glazier, James A',\n",
              "   'Bacallao, Robert L'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Human genomics',\n",
              "  'keywords': ['Adult',\n",
              "   'Case-Control Studies',\n",
              "   'Cell Line',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Metabolic Networks and Pathways',\n",
              "   'Middle Aged',\n",
              "   'Polycystic Kidney, Autosomal Dominant',\n",
              "   'TRPP Cation Channels',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27871310',\n",
              "  'title': 'Transcriptome analysis reveals manifold mechanisms of cyst development in ADPKD.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-2199',\n",
              "  'IF_2013': '8.305',\n",
              "  'IF_2014': '8.665',\n",
              "  'IF_2015': '8.285',\n",
              "  'IF_2016': '6.559',\n",
              "  'IF_2017': '6.308',\n",
              "  'ISSN': '1047-3211',\n",
              "  'ISSN_online': '1460-2199',\n",
              "  'ISSN_print': '1047-3211',\n",
              "  'IsoAbbr': 'Cereb. Cortex',\n",
              "  'JrId': 2171,\n",
              "  'MedAbbr': 'Cereb Cortex',\n",
              "  'NLMid': '9110718',\n",
              "  'Titre': 'Cereb. Cortex',\n",
              "  'abstract': 'A major challenge for computational neuroscience is to understand the computational function of lamina-specific synaptic connection patterns in stereotypical cortical microcircuits. Previous work on this problem had focused on hypothesized specific computational roles of individual layers and connections between layers and had tested these hypotheses through simulations of abstract neural network models. We approach this problem by studying instead the dynamical system defined by more realistic cortical microcircuit models as a whole and by investigating the influence that its laminar structure has on the transmission and fusion of information within this dynamical system. The circuit models that we examine consist of Hodgkin-Huxley neurons with dynamic synapses, based on detailed data from Thomson and others (2002), Markram and others (1998), and Gupta and others (2000). We investigate to what extent this cortical microcircuit template supports the accumulation and fusion of information contained in generic spike inputs into layer 4 and layers 2/3 and how well it makes this information accessible to projection neurons in layers 2/3 and layer 5. We exhibit specific computational advantages of such data-based lamina-specific cortical microcircuit model by comparing its performance with various types of control models that have the same components and the same global statistics of neurons and synaptic connections but are missing the lamina-specific structure of real cortical microcircuits. We conclude that computer simulations of detailed lamina-specific cortical microcircuit models provide new insight into computational consequences of anatomical and physiological data.',\n",
              "  'authors': ['Haeusler, Stefan', 'Maass, Wolfgang'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Cerebral cortex (New York, N.Y. : 1991)',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Cats',\n",
              "   'Cerebral Cortex',\n",
              "   'Computer Simulation',\n",
              "   'Feedback',\n",
              "   'Mental Processes',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Rats',\n",
              "   'Synapses'],\n",
              "  'pmid': '16481565',\n",
              "  'title': 'A statistical analysis of information-processing properties of lamina-specific cortical microcircuit models.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1549-5485',\n",
              "  'IF_2013': '4.375',\n",
              "  'IF_2014': '3.657',\n",
              "  'IF_2015': '2.906',\n",
              "  'IF_2016': '2.894',\n",
              "  'IF_2017': '2.671',\n",
              "  'ISSN': '1072-0502',\n",
              "  'ISSN_online': '1549-5485',\n",
              "  'ISSN_print': '1072-0502',\n",
              "  'IsoAbbr': 'Learn. Mem.',\n",
              "  'JrId': 20827,\n",
              "  'MedAbbr': 'Learn Mem',\n",
              "  'NLMid': '9435678',\n",
              "  'Titre': 'Learn. Mem.',\n",
              "  'abstract': 'Behavioral and neurobiological evidence shows that primacy and recency are subserved by memory systems for intermediate- and short-term memory, respectively. A widely accepted explanation of recency is that in short-term memory, new learning overwrites old learning. Primacy is not as well understood, but many hypotheses contend that initial items are better encoded into long-term memory because they have had more opportunity to be rehearsed. A simple, biologically motivated neural network model supports an alternative hypothesis of the distinct processing requirements for primacy and recency given single-trial learning without rehearsal. Simulations of the model exhibit either primacy or recency, but not both simultaneously. The incompatibility of primacy and recency clarifies possible reasons for two neurologically distinct systems. Inhibition, and its control of activity, determines those list items that are acquired and retained. Activity levels that are too low do not provide sufficient connections for learning to occur, while higher activity diminishes capacity. High recurrent inhibition, and progressively diminishing activity, allows acquisition and retention of early items, while later items are never acquired. Conversely, low recurrent inhibition, and the resulting high activity, allows continuous acquisition such that acquisition of later items eventually interferes with the retention of early items.',\n",
              "  'authors': ['Greene, A J', 'Prepscius, C', 'Levy, W B'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Learning & memory (Cold Spring Harbor, N.Y.)',\n",
              "  'keywords': ['Animals',\n",
              "   'Behavior',\n",
              "   'Cognition',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Memory',\n",
              "   'Memory, Short-Term',\n",
              "   'Models, Neurological',\n",
              "   'Neural Inhibition',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Retention (Psychology)',\n",
              "   'Synapses'],\n",
              "  'pmid': '10706602',\n",
              "  'title': 'Primacy versus recency in a quantitative model: activity is the critical distinction.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1432-1084',\n",
              "  'IF_2013': '4.338',\n",
              "  'IF_2014': '4.014',\n",
              "  'IF_2015': '3.64',\n",
              "  'IF_2016': '3.967',\n",
              "  'IF_2017': '4.027',\n",
              "  'ISSN': '0938-7994',\n",
              "  'ISSN_online': '1432-1084',\n",
              "  'ISSN_print': '0938-7994',\n",
              "  'IsoAbbr': 'Eur Radiol',\n",
              "  'JrId': 20371,\n",
              "  'MedAbbr': 'Eur Radiol',\n",
              "  'NLMid': '9114774',\n",
              "  'Titre': 'Eur Radiol',\n",
              "  'abstract': 'The purpose of this study was to develop a panoramic endoluminal display technique, the band view, which causes minimal image distortion, and to determine its feasibility as a time-efficient primary three-dimensional review method of CT colonography (CTC). Image distortion was compared between the band view and four other three-dimensional review modes using ten 10-mm and ten 20-mm electronically generated polyps. Diagnostic performance and interpretation time were compared between the band view and the conventional endoluminal view by two independent readers in 52 patients who underwent CTC and colonoscopy on the same day. Mean image distortion index values, in which 1 indicates no distortion and the larger value represents greater distortion, were significantly smaller with the band view (1.03 and 1.01 for 10-mm and 20-mm polyps, respectively) than with the filet view (1.65 and 1.55) or the virtual colon dissection (3.27 and 3.85) (P < or = 0.004). The sensitivity and specificity for detecting adenomatous polyps > or = 6 mm did not differ, but the mean interpretation time was significantly shorter with the band view than with the conventional endoluminal view by 1.8 and 4.5 minutes in readers 1 and 2, respectively (P < 0.0001). The band view can be a time-efficient alternative for primary three-dimensional review of CTC.',\n",
              "  'authors': ['Lee, Seung Soo',\n",
              "   'Park, Seong Ho',\n",
              "   'Kim, Jin Kook',\n",
              "   'Kim, Namkug',\n",
              "   'Lee, Jeongjin',\n",
              "   'Park, Beom Jin',\n",
              "   'Kim, Young Jun',\n",
              "   'Lee, Min Woo',\n",
              "   'Kim, Ah Young',\n",
              "   'Ha, Hyun Kwon'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artifacts',\n",
              "   'Artificial Intelligence',\n",
              "   'Colonography, Computed Tomographic',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '19290529',\n",
              "  'title': 'Panoramic endoluminal display with minimal image distortion using circumferential radial ray-casting for primary three-dimensional interpretation of CT colonography.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '2476-762X',\n",
              "  'IF_2013': '1.5',\n",
              "  'IF_2014': '2.514',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '1513-7368',\n",
              "  'ISSN_online': '2476-762X',\n",
              "  'ISSN_print': '1513-7368',\n",
              "  'IsoAbbr': 'Asian Pac. J. Cancer Prev.',\n",
              "  'JrId': 30329,\n",
              "  'MedAbbr': 'Asian Pac J Cancer Prev',\n",
              "  'NLMid': '101130625',\n",
              "  'Titre': 'Asian Pac. J. Cancer Prev.',\n",
              "  'abstract': 'Background: Esophageal adenocarcinoma (EAC) is one of the mostlethal cancers in the world with a very poor\\nprognosis. Identification of molecular diagnostic methods is an important goal. Since protein-protein interaction (PPI)\\nnetwork analysis is a suitable method for molecular assessment, in the present research a PPI network related to EAC\\nwas targeted. Material and Method: Cytoscape software and its applications including STRING DB, Cluster ONE\\nand ClueGO were applied to analyze the PPI network. Result: Among 182 EAC-related proteins which were identified,\\n129 were included in a main connected component. Proteins based on centrality analysis of characteristics such as\\ndegree, betweenness, closeness and stress were screened and key nodes were introduced. Two clusters were determined\\nof which only one was significant statistically. Gene ontology revealed 50 terms in three groups associated with EAC.\\nConclusion: The findings indicate nine crucial proteins could form a candidate biomarker panel for EAC. Furthermore,\\nan important cluster with 27 proteins related to the disease was identified. Gene ontology analysis of this cluster showed\\nmain related terms to closely correspond with those for colorectal cancer.',\n",
              "  'authors': ['Rezaei-Tavirani, Majid',\n",
              "   'Rezaei-Taviran, Sina',\n",
              "   'Mansouri, Mansouri',\n",
              "   'Rostami-Nejad, Mohammad',\n",
              "   'Rezaei-Tavirani, Mostafa'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Asian Pacific journal of cancer prevention : APJCP',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Biomarkers, Tumor',\n",
              "   'Cluster Analysis',\n",
              "   'Esophageal Neoplasms',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Prognosis',\n",
              "   'Protein Interaction Maps',\n",
              "   'Software'],\n",
              "  'pmid': '29286604',\n",
              "  'title': 'Protein-Protein Interaction Network Analysis for a Biomarker Panel Related to Human Esophageal Adenocarcinoma',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1663-9812',\n",
              "  'IF_2013': '3.802',\n",
              "  'IF_2014': '3.802',\n",
              "  'IF_2015': '4.418',\n",
              "  'IF_2016': '4.4',\n",
              "  'IF_2017': '3.831',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1663-9812',\n",
              "  'IsoAbbr': 'Front Pharmacol',\n",
              "  'JrId': 38627,\n",
              "  'MedAbbr': 'Front Pharmacol',\n",
              "  'NLMid': '101548923',\n",
              "  'Titre': 'Front Pharmacol',\n",
              "  'abstract': \"One of the most challenging puzzles in drug discovery is the identification and characterization of candidate drug of well-balanced profile between efficacy and safety. So far, extensive efforts have been made to evaluate this balance by estimating the quantitative structure-therapeutic relationship and exploring target profile of adverse drug reaction. Particularly, the therapeutic index (TI) has emerged as a key indicator illustrating this delicate balance, and a clinically successful agent requires a sufficient TI suitable for it corresponding indication. However, the TI information are largely unknown for most drugs, and the mechanism underlying the drugs with narrow TI (NTI drugs) is still elusive. In this study, the collective effects of human protein-protein interaction (PPI) network and biological system profile on the drugs' efficacy-safety balance were systematically evaluated. First, a comprehensive literature review of the FDA approved drugs confirmed their NTI status. Second, a popular feature selection algorithm based on artificial intelligence (AI) was adopted to identify key factors differencing the target mechanism between NTI and non-NTI drugs. Finally, this work revealed that the targets of NTI drugs were highly centralized and connected in human PPI network, and the number of similarity proteins and affiliated signaling pathways of the corresponding targets was much higher than those of non-NTI drugs. These findings together with the newly discovered features or feature groups clarified the key factors indicating drug's narrow TI, and could thus provide a novel direction for determining the delicate drug efficacy-safety balance.\",\n",
              "  'authors': ['Li, Xiao Xu',\n",
              "   'Yin, Jiayi',\n",
              "   'Tang, Jing',\n",
              "   'Li, Yinghong',\n",
              "   'Yang, Qingxia',\n",
              "   'Xiao, Ziyu',\n",
              "   'Zhang, Runyuan',\n",
              "   'Wang, Yunxia',\n",
              "   'Hong, Jiajun',\n",
              "   'Tao, Lin',\n",
              "   'Xue, Weiwei',\n",
              "   'Zhu, Feng'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Frontiers in pharmacology',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30429792',\n",
              "  'title': 'Determining the Balance Between Drug Efficacy and Safety by the Network and Biological System Profile of Its Therapeutic Target.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Identification of the innervation zone is widely used to optimize the accuracy and precision of noninvasive surface electromyography (EMG) signals because the EMG signal is strongly influenced by innervation zones. However, simply structured fusiform muscle, such as biceps brachii muscle, has been employed mainly due to the simplicity with which the propagation from raw EMG signals can be observed. In this study, the optimum electrode location (OEL), free from innervational influence, was investigated by the propagation pattern of action potentials for brachii muscles and more complicated deltoid muscle structures using an automatized signal analysis technique. The technique employed newly developed computer software with additional clinical uses and minimized subjective differences. EMG signals were recorded using surface array electrodes during voluntary isometric contractions obtained from 12 healthy male subjects. Peaks in EMG signals were detected and averaged for each muscle. The propagation patterns and OEL were examined from biceps brachii muscles for all subjects and from deltoid muscles for seven subjects. The estimated locations were partially confirmed by comparing the root mean squares of the EMG signals. These results show that propagation patterns and OEL could be estimated simply and automatically even from the surface EMG signals of deltoid muscles.',\n",
              "  'authors': ['Nishihara, Ken',\n",
              "   'Kawai, Hisashi',\n",
              "   'Gomi, Toshiaki',\n",
              "   'Terajima, Miho',\n",
              "   'Chiba, Yu'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electrodes',\n",
              "   'Electromyography',\n",
              "   'Humans',\n",
              "   'Muscle, Skeletal',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '18269999',\n",
              "  'title': 'Investigation of optimum electrode locations by using an automatized surface electromyography analysis technique.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1530-8561',\n",
              "  'IF_2013': '7.768',\n",
              "  'IF_2014': '7.911',\n",
              "  'IF_2015': '7.457',\n",
              "  'IF_2016': '8.008',\n",
              "  'IF_2017': '8.636',\n",
              "  'ISSN': '0009-9147',\n",
              "  'ISSN_online': '1530-8561',\n",
              "  'ISSN_print': '0009-9147',\n",
              "  'IsoAbbr': 'Clin. Chem.',\n",
              "  'JrId': 2992,\n",
              "  'MedAbbr': 'Clin Chem',\n",
              "  'NLMid': '9421549',\n",
              "  'Titre': 'Clin. Chem.',\n",
              "  'abstract': 'We demonstrate the possibility of automation of whole-cell functionality assays, e.g., mitogen-activated DNA synthesis, DNA repair synthesis, and assessment of drug-metabolizing enzymes, by use of magnetic separation technology. We have attached antibody-coupled magnetic microspheres to the surface of human T-lymphocytes before performing various assays. Evaluating the biological functions of T-cells estimated by the DNA-synthesis assays showed that the presence of antibody-coupled magnetic microspheres did not affect the results (P greater than 0.05). The concentration of adenosine diphosphate ribosyltransferase (EC 2.4.2.30) was shown to be influenced by the magnetic microspheres. However, the amount of enzyme activity induced by oxidative stress was not significantly altered. The results from assays of the phase II drug-metabolizing enzymes glutathione transferase (EC 2.5.1.18) and epoxide hydrolase (EC 3.3.2.3) as well as evaluation of the proliferative response of polyclonal activators (phytohemagglutinin, staphylococcal enterotoxin A, and pokeweed mitogen) support our conclusion that assays can be performed on viable magnetized cells. The use of magnetized cells holds promise for further applications in automated genotoxic and immunological cell assays of mononuclear leukocyte subsets. Laboratory robotics will be essential in bringing these assays into routine use.',\n",
              "  'authors': ['Schröder, U', 'Pero, R W', 'Sjögren, H O'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinical chemistry',\n",
              "  'keywords': ['Antibodies, Monoclonal',\n",
              "   'Antigens, Surface',\n",
              "   'Cell Separation',\n",
              "   'Electromagnetic Phenomena',\n",
              "   'Epoxide Hydrolases',\n",
              "   'Flow Cytometry',\n",
              "   'Glutathione Transferase',\n",
              "   'Humans',\n",
              "   'Leukocytes, Mononuclear',\n",
              "   'Microspheres',\n",
              "   'Poly(ADP-ribose) Polymerases',\n",
              "   'Robotics',\n",
              "   'T-Lymphocytes'],\n",
              "  'pmid': '2115412',\n",
              "  'title': 'Magnetically tagged subsets of human lymphocytes for assays with laboratory robotics.',\n",
              "  'year': '1990'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1755-8794',\n",
              "  'IF_2013': '3.914',\n",
              "  'IF_2014': '2.873',\n",
              "  'IF_2015': '2.726',\n",
              "  'IF_2016': '2.848',\n",
              "  'IF_2017': '3.317',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1755-8794',\n",
              "  'IsoAbbr': 'BMC Med Genomics',\n",
              "  'JrId': 34003,\n",
              "  'MedAbbr': 'BMC Med Genomics',\n",
              "  'NLMid': '101319628',\n",
              "  'Titre': 'BMC Med Genomics',\n",
              "  'abstract': \"Reliable detection of genome variations, especially insertions and deletions (indels), from single sample DNA sequencing data remains challenging, partially due to the inherent uncertainty involved in aligning sequencing reads to the reference genome. In practice a variety of ad hoc quality filtering methods are employed to produce more reliable lists of putative variants, but the resulting lists typically still include numerous false positives. Thus it would be desirable to be able to rigorously evaluate the degree to which each putative variant is supported by the data. Unfortunately, users who wish to do this, e.g. for the purpose of prioritizing validation experiments, have been faced with limited options.|Here we present EAGLE, a method for evaluating the degree to which sequencing data supports a given candidate genome variant. EAGLE incorporates candidate variants into explicit hypotheses about the individual's genome, and then computes the probability of the observed data (the sequencing reads) under each hypothesis. In comparison with methods which rely heavily on a particular alignment of the reads to the reference genome, EAGLE readily accounts for uncertainties that may arise from multi-mapping or local misalignment and uses the entire length of each read. We compared the scores assigned by several well-known variant callers to EAGLE for the task of ranking true putative variants on both simulated data and real genome sequencing based benchmarks. For indels, EAGLE obtained marked improvement on simulated data and a whole genome sequencing benchmark, and modest but statistically significant improvement on an exome sequencing benchmark.|EAGLE ranked true variants higher than the scores reported by the callers and can used to improve specificity in variant calling. EAGLE is freely available at https://github.com/tony-kuo/eagle .\",\n",
              "  'authors': ['Kuo, Tony', 'Frith, Martin C', 'Sese, Jun', 'Horton, Paul'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC medical genomics',\n",
              "  'keywords': ['Genomics',\n",
              "   'Haplotypes',\n",
              "   'Humans',\n",
              "   'Likelihood Functions',\n",
              "   'Probability',\n",
              "   'Software'],\n",
              "  'pmid': '29697369',\n",
              "  'title': 'EAGLE: Explicit Alternative Genome Likelihood Evaluator.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-6882',\n",
              "  'IF_2013': '6.765',\n",
              "  'IF_2014': '6.628',\n",
              "  'IF_2015': '6.373',\n",
              "  'IF_2016': '6.133',\n",
              "  'IF_2017': '6.541',\n",
              "  'ISSN': '0959-4388',\n",
              "  'ISSN_online': '1873-6882',\n",
              "  'ISSN_print': '0959-4388',\n",
              "  'IsoAbbr': 'Curr. Opin. Neurobiol.',\n",
              "  'JrId': 2201,\n",
              "  'MedAbbr': 'Curr Opin Neurobiol',\n",
              "  'NLMid': '9111376',\n",
              "  'Titre': 'Curr. Opin. Neurobiol.',\n",
              "  'abstract': 'Computational models can provide useful guidance in the design of behavioral and neurophysiological experiments and in the interpretation of complex, high dimensional biological data. Because many problems faced by the primate brain in the control of movement have parallels in robotic motor control, models and algorithms from robotics research provide useful inspiration, baseline performance, and sometimes direct analogs for neuroscience.',\n",
              "  'authors': ['Schaal, Stefan', 'Schweighofer, Nicolas'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Current opinion in neurobiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Bayes Theorem',\n",
              "   'Brain',\n",
              "   'Computers',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Models, Neurological',\n",
              "   'Movement',\n",
              "   'Robotics',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '16271466',\n",
              "  'title': 'Computational motor control in humans and robots.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1618-0631',\n",
              "  'IF_2013': '1.562',\n",
              "  'IF_2014': '1.397',\n",
              "  'IF_2015': '1.388',\n",
              "  'IF_2016': '1.543',\n",
              "  'IF_2017': '1.466',\n",
              "  'ISSN': '0344-0338',\n",
              "  'ISSN_online': '1618-0631',\n",
              "  'ISSN_print': '0344-0338',\n",
              "  'IsoAbbr': 'Pathol. Res. Pract.',\n",
              "  'JrId': 6536,\n",
              "  'MedAbbr': 'Pathol Res Pract',\n",
              "  'NLMid': '7806109',\n",
              "  'Titre': 'Pathol. Res. Pract.',\n",
              "  'abstract': 'This article describes the set up of a rule-based expert system for histologic typing and grading of invasive breast cancer, which is designed to be a user-friendly tool that may be helpful for teaching and to support diagnosis making. The system raises questions and offers fixed choices to the user (usually yes/no) until a histologic diagnosis can be made with reasonable probability or enough data are available to assign a grade. As to histologic typing, the expert system is able to make the following diagnoses: ductal carcinoma, lobular carcinoma, medullary carcinoma, colloidal carcinoma, tubular carcinoma, and invasive cribriform carcinoma. If the diagnosis \"ductal carcinoma\" is arrived, the system offers the option to assign a histologic grade to the lesion. A first evaluation of the system in 30 cases (five each of the different subtypes) with unequivocal diagnoses by two human experts showed that the system classified 29 of the tumours in the same way as the human experts. The discrepancy case was solved after adding one rule to the system. Ten cases where a discrepancy existed between the original diagnosis of a referring centre and a reviewing human expert were all classified by the expert system in the same way as the human expert. The expert system thus seems to perform well. Further plans for evaluating, modifying and expanding the system are disclosed.',\n",
              "  'authors': ['van Diest, P J', 'Belien, J A', 'Baak, J P'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Pathology, research and practice',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Adenocarcinoma, Mucinous',\n",
              "   'Breast Neoplasms',\n",
              "   'Carcinoma',\n",
              "   'Carcinoma, Intraductal, Noninfiltrating',\n",
              "   'Carcinoma, Transitional Cell',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Neoplasm Invasiveness',\n",
              "   'Neoplasm Staging',\n",
              "   'Software'],\n",
              "  'pmid': '1329049',\n",
              "  'title': 'An expert system for histological typing and grading of invasive breast cancer. First set up.',\n",
              "  'year': '1992'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Interactions with artificial agents often lack immediacy because agents respond slower than their users expect. Automatic speech recognisers introduce this delay by analysing a user's utterance only after it has been completed. Early, uncertain hypotheses of incremental speech recognisers can enable artificial agents to respond more timely. However, these hypotheses may change significantly with each update. Therefore, an already initiated action may turn into an error and invoke error cost. We investigated whether humans would use uncertain hypotheses for planning ahead and/or initiating their response. We designed a Ghost-in-the-Machine study in a bar scenario. A human participant controlled a bartending robot and perceived the scene only through its recognisers. The results showed that participants used uncertain hypotheses for selecting the best matching action. This is comparable to computing the utility of dialogue moves. Participants evaluated the available evidence and the error cost of their actions prior to initiating them. If the error cost was low, the participants initiated their response with only suggestive evidence. Otherwise, they waited for additional, more confident hypotheses if they still had time to do so. If there was time pressure but only little evidence, participants grounded their understanding with echo questions. These findings contribute to a psychologically plausible policy for human-robot interaction that enables artificial agents to respond more timely and socially appropriately under uncertainty.\",\n",
              "  'authors': ['Loth, Sebastian',\n",
              "   'Jettka, Katharina',\n",
              "   'Giuliani, Manuel',\n",
              "   'Kopp, Stefan',\n",
              "   'de Ruiter, Jan P'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Comprehension',\n",
              "   'Equipment Design',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Interpersonal Relations',\n",
              "   'Male',\n",
              "   'Robotics',\n",
              "   'Speech',\n",
              "   'Uncertainty',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30067853',\n",
              "  'title': 'Confidence in uncertainty: Error cost and commitment in early speech hypotheses.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1879-1700',\n",
              "  'IF_2013': '3.571',\n",
              "  'IF_2014': '2.483',\n",
              "  'IF_2015': '2.047',\n",
              "  'IF_2016': '2.31',\n",
              "  'IF_2017': '2.263',\n",
              "  'ISSN': '0265-931X',\n",
              "  'ISSN_online': '1879-1700',\n",
              "  'ISSN_print': '0265-931X',\n",
              "  'IsoAbbr': 'J Environ Radioact',\n",
              "  'JrId': 22004,\n",
              "  'MedAbbr': 'J Environ Radioact',\n",
              "  'NLMid': '8508119',\n",
              "  'Titre': 'J Environ Radioact',\n",
              "  'abstract': 'In this article, three computational intelligence (CI) models were developed to automatically detect anomalous behaviour in soil radon gas (222Rn) time series data. Data were obtained at a fault line and analysed using three machine learning techniques with the aim at identifying anomalies in temporal radon data prompted by seismic events. Radon concentrations were modelled with corresponding meteorological and statistical parameters. This leads to the estimation of soil radon gas without and with meteorological parameters. The comparison between computed radon concentration and actual radon concentrations was used in finding radon anomaly based upon automated system. The anomaly in radon time series data could be considered due to noise or seismic activity. Findings of study show that under meticulously characterized environments, on exclusion of noise contribution, seismic activity is responsible for anomalous behaviour seen in radon time series data.',\n",
              "  'authors': ['Tareen, Aleem Dad Khan',\n",
              "   'Asim, Khawaja M',\n",
              "   'Kearfott, Kimberlee Jane',\n",
              "   'Rafique, Muhammad',\n",
              "   'Nadeem, Malik Sajjad Ahmed',\n",
              "   'Iqbal, Talat',\n",
              "   'Rahman, Saeed Ur'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Journal of environmental radioactivity',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Earthquakes',\n",
              "   'Radiation Monitoring',\n",
              "   'Radon',\n",
              "   'Soil',\n",
              "   'Soil Pollutants, Radioactive'],\n",
              "  'pmid': '30861489',\n",
              "  'title': 'Automated anomalous behaviour detection in soil radon gas prior to earthquakes using computational intelligence techniques.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Extracting primary care information in terms of Patient/Problem, Intervention, Comparison and Outcome, known as PICO elements, is difficult as the volume of medical information expands and the health semantics is complex to capture it from unstructured information. The combination of the machine learning methods (MLMs) with rule based methods (RBMs) could facilitate and improve the PICO extraction. This paper studies the PICO elements extraction methods. The goal is to combine the MLMs with the RBMs to extract PICO elements in medical papers to facilitate answering clinical questions formulated with the PICO framework.|First, we analyze the aspects of the MLM model that influence the quality of the PICO elements extraction. Secondly, we combine the MLM approach with the RBMs in order to improve the PICO elements retrieval process. To conduct our experiments, we use a corpus of 1000 abstracts.|We obtain an F-score of 80% for P element, 64% for the I element and 92% for the O element. Given the nature of the used training corpus where P and I elements represent respectively only 6.5 and 5.8% of total sentences, the results are competitive with previously published ones.|Our study of the PICO element extraction shows that the task is very challenging. The MLMs tend to have an acceptable precision rate but they have a low recall rate when the corpus is not representative. The RBMs backed up the MLMs to increase the recall rate and consequently the combination of the two methods gave better results.',\n",
              "  'authors': ['Chabou, Samir', 'Iglewski, Michal'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Data Mining',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Medical Informatics Applications',\n",
              "   'Models, Statistical',\n",
              "   'Natural Language Processing'],\n",
              "  'pmid': '30509272',\n",
              "  'title': 'Combination of conditional random field with a rule based method in the extraction of PICO elements.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'We have investigated the use of an Artificial Neural Network (ANN) for the assessment of fall-risk (FR) in patients with different neural pathologies. The assessment integrates a clinical tool based on a wearable device (WD) with accelerometers (ACCs) and rate gyroscopes (GYROs) properly suited to identify trunk kinematic parameters that can be measured during a posturography test with different constraints. Our ANN--a Multi Layer Perceptron Neural Network with four layers and 272 neurones--shows to be able to classify patients in three well-known fall-risk levels. The training of the neural network was carried on three groups of 30 subjects with different Fall-Risk Tinetti scores. The validation of our neural network was carried out on three groups of 100 subjects with different Fall-Risk Tinetti scores and this validation demonstrated that the neural network had high specificity (> or =0.88); sensitivity (> or =0.87); area under Receiver-Operator Characteristic Curves (>0.854).',\n",
              "  'authors': ['Giansanti, Daniele',\n",
              "   'Maccioni, Giovanni',\n",
              "   'Cesinaro, Stefano',\n",
              "   'Benvenuti, Francesco',\n",
              "   'Macellari, Velio'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Acceleration',\n",
              "   'Accidental Falls',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Back',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Evaluation Studies as Topic',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Postural Balance',\n",
              "   'Posture',\n",
              "   'ROC Curve',\n",
              "   'Risk Assessment',\n",
              "   'Risk Factors',\n",
              "   'Rotation',\n",
              "   'Transducers'],\n",
              "  'pmid': '17560825',\n",
              "  'title': 'Assessment of fall-risk by means of a neural network based on parameters assessed by a wearable device during posturography.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1528-1140',\n",
              "  'IF_2013': '7.188',\n",
              "  'IF_2014': '8.327',\n",
              "  'IF_2015': '8.569',\n",
              "  'IF_2016': '8.98',\n",
              "  'IF_2017': '9.203',\n",
              "  'ISSN': '0003-4932',\n",
              "  'ISSN_online': '1528-1140',\n",
              "  'ISSN_print': '0003-4932',\n",
              "  'IsoAbbr': 'Ann. Surg.',\n",
              "  'JrId': 646,\n",
              "  'MedAbbr': 'Ann Surg',\n",
              "  'NLMid': '0372354',\n",
              "  'Titre': 'Ann. Surg.',\n",
              "  'abstract': 'To directly compare the oncologic outcomes of TLPD and OPD in the setting of pancreatic ductal adenocarcinoma.|Total laparoscopic pancreaticoduodenectomy (TLPD) has been demonstrated to be feasible and may have several potential advantages over open pancreaticoduodenectomy (OPD), including lower blood loss and shorter hospital stay. Whether potential advantages could allow patients to recover in a timelier manner and pursue adjuvant treatment options remains to be answered.|We reviewed data for all patients undergoing TLPD (N = 108) or OPD (N = 214) for pancreatic ductal adenocarcinoma at our institution between January 2008 and July 2013.|Neoadjuvant therapy, tumor size, node positivity, and margin-positive resection were not significantly different between the 2 groups. Median length of hospital stay was significantly longer in the OPD group (9 days; range, 5-73 days) than in the TLPD group (6 days; range, 4-118 days; P < 0.001). There was a significantly higher proportion of patients in the OPD group (12%) who had a delay of greater than 90 days or who did not receive adjuvant chemotherapy at all compared with that in the TLPD group (5%; P = 0.04). There was no significant difference in overall survival between the 2 groups (P = 0.22). A significantly longer progression-free survival was seen in the TLPD group than in the OPD group (P = 0.03).|TLPD is not only feasible in the setting of pancreatic ductal adenocarcinoma but also has advantages such as shorter hospital stay and faster recovery, allowing patients to recover in a timelier manner and pursue adjuvant treatment options. This study also demonstrated a longer progression-free survival in patients undergoing TLPD than those undergoing OPD.',\n",
              "  'authors': ['Croome, Kristopher P',\n",
              "   'Farnell, Michael B',\n",
              "   'Que, Florencia G',\n",
              "   'Reid-Lombardo, K Marie',\n",
              "   'Truty, Mark J',\n",
              "   'Nagorney, David M',\n",
              "   'Kendrick, Michael L'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Annals of surgery',\n",
              "  'keywords': ['Aged',\n",
              "   'Blood Loss, Surgical',\n",
              "   'Carcinoma, Pancreatic Ductal',\n",
              "   'Chemotherapy, Adjuvant',\n",
              "   'Disease-Free Survival',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoadjuvant Therapy',\n",
              "   'Pancreatic Neoplasms',\n",
              "   'Pancreaticoduodenectomy',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Time Factors'],\n",
              "  'pmid': '25203880',\n",
              "  'title': 'Total laparoscopic pancreaticoduodenectomy for pancreatic ductal adenocarcinoma: oncologic advantages over open approaches?',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['WH'],\n",
              "  'ESSN': '1529-0131',\n",
              "  'IF_2013': '7.871',\n",
              "  'IF_2014': '7.764',\n",
              "  'IF_2015': '8.955',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0004-3591',\n",
              "  'ISSN_online': '1529-0131',\n",
              "  'ISSN_print': '0004-3591',\n",
              "  'IsoAbbr': 'Arthritis Rheum.',\n",
              "  'JrId': 881,\n",
              "  'MedAbbr': 'Arthritis Rheum',\n",
              "  'NLMid': '0370605',\n",
              "  'Titre': 'Arthritis Rheum.',\n",
              "  'abstract': \"To examine the operating characteristics of the American College of Rheumatology (ACR) classification criteria for Churg-Strauss syndrome (CSS) and Wegener's granulomatosis (WG), and to develop and validate improved criteria for distinguishing CSS from WG.|The ACR classification criteria for WG and CSS were applied to 40 consecutive CSS patients age- and sex-matched with 40 patients with WG. Forty-three clinical, laboratory, and biopsy parameters were assessed. Artificial neural networks (ANNs) were trained and tested with all 43 parameters (set A) and with 15 solely clinical parameters documented at the initial manifestation of the disease (set B). The ANNs were trained with data from the first 27 CSS and 27 WG patients and validated with data from the next 13 consecutive CSS and 13 WG patients. To compare the ANNs with established methods, traditional format and classification tree criteria were generated using the same data sets.|Fourteen of 40 CSS patients fulfilled the ACR criteria for WG, while 4 WG patients met the ACR criteria for CSS. The ANN, in contrast, reliably distinguished all CSS cases from WG cases (parameter set A, accuracy 100%). For parameter set B, the ANN achieved an accuracy of 100% in the training phase and 96% for validation. The newly formulated traditional format and classification tree criteria reached an accuracy of 81% and 88%, respectively.|The ACR criteria for WG do not reliably differentiate between CSS and WG (specificity 65%). An ANN, however, could be trained to correctly allocate all but 1 patient on the basis of clinical data. Indeed, the ANN applied in this study proved superior to established methods of classification. We suggest that an ANN may be effectively applied in the classification of systemic vasculitides.\",\n",
              "  'authors': ['Schmitt, W H', 'Linder, R', 'Reinhold-Keller, E', 'Gross, W L'],\n",
              "  'categories': ['RHEUMATOLOGY'],\n",
              "  'journal': 'Arthritis and rheumatism',\n",
              "  'keywords': ['Churg-Strauss Syndrome',\n",
              "   'Cohort Studies',\n",
              "   'Diagnosis, Differential',\n",
              "   'Female',\n",
              "   'Granulomatosis with Polyangiitis',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '11508442',\n",
              "  'title': \"Improved differentiation between Churg-Strauss syndrome and Wegener's granulomatosis by an artificial neural network.\",\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Noise is an unavoidable factor in real sensor signals. We study how additive and convolutive noise can be reduced or even eliminated in the blind source separation (BSS) problem. Particular attention is paid to cases in which the number of sensors is larger than the number of sources. We propose various methods and associated adaptive learning algorithms for such an extended BSS problem. Performance and validity of the proposed approaches are demonstrated by extensive computer simulations.',\n",
              "  'authors': ['Karhunen, J', 'Cichocki, A', 'Kasprzak, W', 'Pajunen, P'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '9327277',\n",
              "  'title': 'On neural blind separation with noise suppression and redundancy reduction.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'In order to overcome the problems of poor understandability of the pattern recognition-based transient stability assessment (PRTSA) methods, a new rule extraction method based on extreme learning machine (ELM) and an improved Ant-miner (IAM) algorithm is presented in this paper. First, the basic principles of ELM and Ant-miner algorithm are respectively introduced. Then, based on the selected optimal feature subset, an example sample set is generated by the trained ELM-based PRTSA model. And finally, a set of classification rules are obtained by IAM algorithm to replace the original ELM network. The novelty of this proposal is that transient stability rules are extracted from an example sample set generated by the trained ELM-based transient stability assessment model by using IAM algorithm. The effectiveness of the proposed method is shown by the application results on the New England 39-bus power system and a practical power system--the southern power system of Hebei province.',\n",
              "  'authors': ['Li, Yang', 'Li, Guoqing', 'Wang, Zhenhao'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Area Under Curve',\n",
              "   'Machine Learning',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'ROC Curve'],\n",
              "  'pmid': '26091524',\n",
              "  'title': 'Rule Extraction Based on Extreme Learning Machine and an Improved Ant-Miner Algorithm for Transient Stability Assessment.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1432-0614',\n",
              "  'IF_2013': '3.811',\n",
              "  'IF_2014': '3.337',\n",
              "  'IF_2015': '3.376',\n",
              "  'IF_2016': '3.42',\n",
              "  'IF_2017': '3.34',\n",
              "  'ISSN': '0175-7598',\n",
              "  'ISSN_online': '1432-0614',\n",
              "  'ISSN_print': '0175-7598',\n",
              "  'IsoAbbr': 'Appl. Microbiol. Biotechnol.',\n",
              "  'JrId': 1580,\n",
              "  'MedAbbr': 'Appl Microbiol Biotechnol',\n",
              "  'NLMid': '8406612',\n",
              "  'Titre': 'Appl. Microbiol. Biotechnol.',\n",
              "  'abstract': 'FK506 (tacrolimus) is a 23-membered polyketide macrolide that possesses powerful immunosuppressant activity. In this study, feeding soybean oil into the fermentation culture of Streptomyces tsukubaensis improved FK506 production by 88.8%. To decipher the overproduction mechanism, comparative proteomic and metabolomic analysis was carried out. A total of 72 protein spots with differential expression in the two-dimensional gel electrophoresis (2-DE) were identified by matrix-assisted laser desorption/ionization time-of-flight/time-of-flight mass spectrometry (MALDI-TOF/TOF-MS), and 66 intracellular metabolites were measured by gas chromatography-mass spectrometer (GC-MS). The analysis of proteome and metabolome indicated that feeding soybean oil as a supplementary carbon source could not only strengthen the FK506 precursor metabolism and energy metabolism but also tune the pathways related to transcriptional regulation, translation, and stress response, suggesting a better intracellular metabolic environment for the synthesis of FK506. Based on these analyses, 20 key metabolites and precursors of FK506 were supplemented into the soybean oil medium. Among them, lysine, citric acid, shikimic acid, and malonic acid performed excellently for promoting the FK506 production and biomass. Especially, the addition of malonic acid achieved the highest FK506 production, which was 1.56-fold of that in soybean oil medium and 3.05-fold of that in initial medium. This report represented the first comprehensive study on the comparative proteomics and metabolomics applied in S. tsukubaensis, and it would be a rational guidance to further strengthen the FK506 production.',\n",
              "  'authors': ['Wang, Jun',\n",
              "   'Liu, Huanhuan',\n",
              "   'Huang, Di',\n",
              "   'Jin, Lina',\n",
              "   'Wang, Cheng',\n",
              "   'Wen, Jianping'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Applied microbiology and biotechnology',\n",
              "  'keywords': ['Bacterial Proteins',\n",
              "   'Bioreactors',\n",
              "   'Citric Acid',\n",
              "   'Electrophoresis, Gel, Two-Dimensional',\n",
              "   'Fermentation',\n",
              "   'Gas Chromatography-Mass Spectrometry',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Bacterial',\n",
              "   'Gene Ontology',\n",
              "   'Immunosuppressive Agents',\n",
              "   'Lysine',\n",
              "   'Malonates',\n",
              "   'Metabolic Networks and Pathways',\n",
              "   'Metabolomics',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Proteomics',\n",
              "   'Shikimic Acid',\n",
              "   'Soybean Oil',\n",
              "   'Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization',\n",
              "   'Streptomyces',\n",
              "   'Tacrolimus'],\n",
              "  'pmid': '28175948',\n",
              "  'title': 'Comparative proteomic and metabolomic analysis of Streptomyces tsukubaensis reveals the metabolic mechanism of FK506 overproduction by feeding soybean oil.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Although increasingly sophisticated algorithms have been proposed to decompose intramuscular electromyography signals into the concurrent activities of individual motor units (MUs), the human operator is still able to improve decomposition results by visual inspection. The rationale for this paper was to combine components from previous decomposition procedures in an expert systems approach utilizing fuzzy logic and attempting to replicate the thought process of an accomplished decomposer in order to minimize the user interaction subsequently needed to enhance decomposition results. The decomposition procedure is discussed and examples are given of the type of information it can yield. The method has been used to identify the discharge activities of up to 15 MUs with up to 95% accuracy.',\n",
              "  'authors': ['Erim, Zeynep', 'Lin, Winsean'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Action Potentials',\n",
              "   'Algorithms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Electromyography',\n",
              "   'Expert Systems',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Muscle, Skeletal',\n",
              "   'Neuromuscular Junction',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Synaptic Transmission'],\n",
              "  'pmid': '18713687',\n",
              "  'title': 'Decomposition of intramuscular EMG signals using a heuristic fuzzy expert system.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1573-9686',\n",
              "  'IF_2013': '3.231',\n",
              "  'IF_2014': '3.195',\n",
              "  'IF_2015': '2.887',\n",
              "  'IF_2016': '3.221',\n",
              "  'IF_2017': '3.405',\n",
              "  'ISSN': '0090-6964',\n",
              "  'ISSN_online': '1573-9686',\n",
              "  'ISSN_print': '0090-6964',\n",
              "  'IsoAbbr': 'Ann Biomed Eng',\n",
              "  'JrId': 561,\n",
              "  'MedAbbr': 'Ann Biomed Eng',\n",
              "  'NLMid': '0361512',\n",
              "  'Titre': 'Ann Biomed Eng',\n",
              "  'abstract': \"This paper compares the frontal plane hip function of subject's known to have had hip arthroplasty via either the lateral (LA) or posterior (PA) surgical approaches and a group of subjects associated with no pathology (NP). This is investigated through the Trendelenburg test using 3D motion analysis and classification. Here, a recent development on the Classification and Ranking Belief Simplex (CaRBS) technique, able to undertake n-state classification, so termed NCaRBS is employed. The relationship between post-operative hip function measured during a Trendelenburg Test using three patient characteristics (pelvic obliquity, frontal plane hip moment and frontal plane hip power) of LA, PA and NP subjects are modelled together. Using these characteristics, the classification accuracy was 93.75% for NP, 57.14% for LA, 38.46% for PA. There was a clear distinction between NP and post-surgical function. 3/6 LA subjects and 6/8 PA subjects were misclassified as having NP function, implying that greater function is restored following the PA to surgery. NCaRBS achieved a higher accuracy (65.116%) than through a linear discriminant analysis (48.837%). A Neural Network with two-nodes achieved the same accuracy (65.116%) and as expected was further improved with three-nodes (69.767%). A valuable benefit to the employment of the NCaRBS technique is the graphical exposition of the contribution of patient characteristics to the classification analysis. \",\n",
              "  'authors': ['Whatling, Gemma Marie', 'Holt, C A', 'Beynon, M J'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Annals of biomedical engineering',\n",
              "  'keywords': ['Arthroplasty, Replacement, Hip',\n",
              "   'Discriminant Analysis',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Posture'],\n",
              "  'pmid': '25613485',\n",
              "  'title': 'The application of NCaRBS to the Trendelenburg test and total hip arthroplasty outcome.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'Recently, increasing numbers of robotic-assisted laparoscopic radical prostatectomy (RALP) have been performed at many centers. During urethrovesical anastomosis in RALP, surgeons commonly use Large Needle Driver (Intuitive Surgical, Mountain View, Calif). In this study, we report a rare case with intraperitoneal breakage of the jaw of Large Needle Driver.',\n",
              "  'authors': ['Park, Sung Yul',\n",
              "   'Cho, Kang Su',\n",
              "   'Lee, Seung Wook',\n",
              "   'Soh, Byung Hyun',\n",
              "   'Rha, Koon Ho'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Anastomosis, Surgical',\n",
              "   'Equipment Failure',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Transurethral Resection of Prostate'],\n",
              "  'pmid': '18242390',\n",
              "  'title': 'Intraoperative breakage of needle driver jaw during robotic-assisted laparoscopic radical prostatectomy.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '2160-1836',\n",
              "  'IF_2013': '2.511',\n",
              "  'IF_2014': '3.198',\n",
              "  'IF_2015': '2.91',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '2.742',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2160-1836',\n",
              "  'IsoAbbr': 'G3 (Bethesda)',\n",
              "  'JrId': 39472,\n",
              "  'MedAbbr': 'G3 (Bethesda)',\n",
              "  'NLMid': '101566598',\n",
              "  'Titre': 'G3 (Bethesda)',\n",
              "  'abstract': 'High-throughput microscopy of many single cells generates high-dimensional data that are far from straightforward to analyze. One important problem is automatically detecting the cellular compartment where a fluorescently-tagged protein resides, a task relatively simple for an experienced human, but difficult to automate on a computer. Here, we train an 11-layer neural network on data from mapping thousands of yeast proteins, achieving per cell localization classification accuracy of 91%, and per protein accuracy of 99% on held-out images. We confirm that low-level network features correspond to basic image characteristics, while deeper layers separate localization classes. Using this network as a feature calculator, we train standard classifiers that assign proteins to previously unseen compartments after observing only a small number of training examples. Our results are the most accurate subcellular localization classifications to date, and demonstrate the usefulness of deep learning for high-throughput microscopy.',\n",
              "  'authors': ['Pärnamaa, Tanel', 'Parts, Leopold'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'G3 (Bethesda, Md.)',\n",
              "  'keywords': ['Fungal Proteins',\n",
              "   'High-Throughput Screening Assays',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Microscopy, Fluorescence',\n",
              "   'Protein Transport',\n",
              "   'Proteome',\n",
              "   'Yeasts'],\n",
              "  'pmid': '28391243',\n",
              "  'title': 'Accurate Classification of Protein Subcellular Localization from High-Throughput Microscopy Images Using Deep Learning.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1471-2970',\n",
              "  'IF_2013': '6.314',\n",
              "  'IF_2014': '7.055',\n",
              "  'IF_2015': '5.847',\n",
              "  'IF_2016': '5.846',\n",
              "  'IF_2017': '5.666',\n",
              "  'ISSN': '0962-8436',\n",
              "  'ISSN_online': '1471-2970',\n",
              "  'ISSN_print': '0962-8436',\n",
              "  'IsoAbbr': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'JrId': 8277,\n",
              "  'MedAbbr': 'Philos Trans R Soc Lond B Biol Sci',\n",
              "  'NLMid': '7503623',\n",
              "  'Titre': 'Philos. Trans. R. Soc. Lond., B, Biol. Sci.',\n",
              "  'abstract': \"Artificial neural networks are becoming increasingly popular as predictive statistical tools in ecosystem ecology and as models of signal processing in behavioural and evolutionary ecology. We demonstrate here that a commonly used network in ecology, the three-layer feed-forward network, trained with the backpropagation algorithm, can be extremely sensitive to the stochastic variation in training data that results from random sampling of the same underlying statistical distribution, with networks converging to several distinct predictive states. Using a random walk procedure to sample error-weight space, and Sammon dimensional reduction of weight arrays, we demonstrate that these different predictive states are not artefactual, due to local minima, but lie at the base of major error troughs in the error-weight surface. We further demonstrate that various gross weight compositions can produce the same predictive state, suggesting the analogy of weight space as a 'patchwork' of multiple predictive states. Our results argue for increased inclusion of stochastic training replication and analysis into ecological and behavioural applications of artificial neural networks.\",\n",
              "  'authors': ['Tosh, Colin R', 'Ruxton, Graeme D'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Philosophical transactions of the Royal Society of London. Series B, Biological sciences',\n",
              "  'keywords': ['Ecosystem',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '17255011',\n",
              "  'title': 'The need for stochastic replication of ecological neural networks.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1677-6119',\n",
              "  'IF_2013': '0.956',\n",
              "  'IF_2014': '0.875',\n",
              "  'IF_2015': '0.871',\n",
              "  'IF_2016': '0.815',\n",
              "  'IF_2017': '0.976',\n",
              "  'ISSN': '1677-5538',\n",
              "  'ISSN_online': '1677-6119',\n",
              "  'ISSN_print': '1677-5538',\n",
              "  'IsoAbbr': 'Int Braz J Urol',\n",
              "  'JrId': 32214,\n",
              "  'MedAbbr': 'Int Braz J Urol',\n",
              "  'NLMid': '101158091',\n",
              "  'Titre': 'Int Braz J Urol',\n",
              "  'abstract': 'To report the initial outcomes of robotic-assisted partial nephrectomy in a tertiary center in South America.|From 11/2008 to 12/2009, a total of 16 transperitoneal robotic-assisted partial nephrectomies were performed in 15 patients to treat 18 kidney tumors. One patient with bilateral tumor had two procedures, while two patients with two synchronous unilateral tumors had a single operation to remove them. Eleven (73%) patients were male and the right kidney was affected in 7 (46%) patients. The median patient age and tumor size were 57 years old and 30 mm, respectively. Five (28%) tumors were hilar and/or centrally located.|The median operative time, warm ischemia time and estimated blood loss was 140 min, 27 min and 120 mL, respectively. Blood transfusion was required in one patient with bilateral tumor, and one additional pyelolithotomy was performed due to a 15 mm stone located in the renal pelvis. The histopathology analysis showed 15 (83%) malignant tumors, which 10 (67%) were clear cell carcinoma. The median hospital stay was 72 hrs and no major complication was observed.|Robotic-assisted partial nephrectomy is safe and represents a valuable option to perform minimally invasive nephron-sparing surgery.',\n",
              "  'authors': ['Lemos, Gustavo C',\n",
              "   'Apezzato, Marcelo',\n",
              "   'Borges, Leonardo L',\n",
              "   'Colombo, José R'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'International braz j urol : official journal of the Brazilian Society of Urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Length of Stay',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Postoperative Complications',\n",
              "   'Robotics',\n",
              "   'South America',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '21888697',\n",
              "  'title': 'Robotic-assisted partial Nephrectomy: initial experience in South America.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'Cell protrusion is morphodynamically heterogeneous at the subcellular level. However, the mechanism of cell protrusion has been understood based on the ensemble average of actin regulator dynamics. Here, we establish a computational framework called HACKS (deconvolution of heterogeneous activity in coordination of cytoskeleton at the subcellular level) to deconvolve the subcellular heterogeneity of lamellipodial protrusion from live cell imaging. HACKS identifies distinct subcellular protrusion phenotypes based on machine-learning algorithms and reveals their underlying actin regulator dynamics at the leading edge. Using our method, we discover \"accelerating protrusion\", which is driven by the temporally ordered coordination of Arp2/3 and VASP activities. We validate our finding by pharmacological perturbations and further identify the fine regulation of Arp2/3 and VASP recruitment associated with accelerating protrusion. Our study suggests HACKS can identify specific subcellular protrusion phenotypes susceptible to pharmacological perturbation and reveal how actin regulator dynamics are changed by the perturbation.',\n",
              "  'authors': ['Wang, Chuangqi',\n",
              "   'Choi, Hee June',\n",
              "   'Kim, Sung-Jin',\n",
              "   'Desai, Aesha',\n",
              "   'Lee, Namgyu',\n",
              "   'Kim, Dohoon',\n",
              "   'Bae, Yongho',\n",
              "   'Lee, Kwonmoo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': ['Actin Cytoskeleton',\n",
              "   'Actin-Related Protein 2-3 Complex',\n",
              "   'Actins',\n",
              "   'Animals',\n",
              "   'Cell Adhesion Molecules',\n",
              "   'Cell Line',\n",
              "   'Cell Line, Tumor',\n",
              "   'Cell Movement',\n",
              "   'Cluster Analysis',\n",
              "   'Humans',\n",
              "   'Indoles',\n",
              "   'Intravital Microscopy',\n",
              "   'Machine Learning',\n",
              "   'Microfilament Proteins',\n",
              "   'Models, Biological',\n",
              "   'Phosphoproteins',\n",
              "   'Potoroidae',\n",
              "   'Pseudopodia',\n",
              "   'Software'],\n",
              "  'pmid': '29703977',\n",
              "  'title': 'Deconvolution of subcellular protrusion heterogeneity and the underlying actin regulator dynamics from live cell imaging.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'Swallow segmentation is a critical precursory step to the analysis of swallowing signal characteristics. In an effort to automatically segment swallows, we investigated artificial neural networks (ANN) with information from cervical dual-axis accelerometry, submental MMG, and nasal airflow. Our objectives were (1) to investigate the relationship between segmentation performance and the number of signal sources and (2) to identify the signals or signal combinations most useful for swallow segmentation. Signals were acquired from 17 healthy adults in both discrete and continuous swallowing tasks using five stimuli. Training and test feature vectors were constructed with variances from single or multiple signals, estimated within 200 ms moving windows with 50% overlap. Corresponding binary target labels (swallow or non-swallow) were derived by manual segmentation. A separate 3-layer ANN was trained for each participant-signal combination, and all possible signal combinations were investigated. As more signal sources were included, segmentation performance improved in terms of sensitivity, specificity, accuracy, and adjusted accuracy. The combination of all four signal sources achieved the highest mean accuracy and adjusted accuracy of 88.5% and 89.6%, respectively. A-P accelerometry proved to be the most discriminatory source, while the inclusion of MMG or nasal airflow resulted in the least performance improvement. These findings suggest that an ANN, multi-sensor fusion approach to segmentation is worthy of further investigation in swallowing studies.',\n",
              "  'authors': ['Lee, Joon', 'Steele, Catriona M', 'Chau, Tom'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Deglutition',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Movement',\n",
              "   'Neck',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Respiration',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '19646911',\n",
              "  'title': 'Swallow segmentation with artificial neural networks and multi-sensor fusion.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': 'Disease taxonomies have been designed for many applications, but they tend not to fully incorporate the growing amount of molecular-level knowledge of disease processes, inhibiting research efforts. Understanding the degree to which we can infer disease relationships from molecular data alone may yield insights into how to ultimately construct more modern taxonomies that integrate both physiological and molecular information.|We introduce a new technique we call Parent Promotion to infer hierarchical relationships between disease terms using disease-gene data. We compare this technique with both an established ontology inference method (CliXO) and a minimum weight spanning tree approach. Because there is no gold standard molecular disease taxonomy available, we compare our inferred hierarchies to both the Medical Subject Headings (MeSH) category C forest of diseases and to subnetworks of the Disease Ontology (DO). This comparison provides insights about the inference algorithms, choices of evaluation metrics, and the existing molecular content of various subnetworks of MeSH and the DO. Our results suggest that the Parent Promotion method performs well in most cases. Performance across MeSH trees is also correlated between inference methods. Specifically, inferred relationships are more consistent with those in smaller MeSH disease trees than larger ones, but there are some notable exceptions that may correlate with higher molecular content in MeSH.|Our experiments provide insights about learning relationships between diseases from disease genes alone. Future work should explore the prospect of disease term discovery from molecular data and how best to integrate molecular data with anatomical and clinical knowledge. This study nonetheless suggests that disease gene information has the potential to form an important part of the foundation for future representations of the disease landscape.',\n",
              "  'authors': ['Park, Jisoo', 'Hescott, Benjamin J', 'Slonim, Donna K'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Disease',\n",
              "   'Humans',\n",
              "   'Medical Subject Headings',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '28750648',\n",
              "  'title': 'Towards a more molecular taxonomy of disease.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The most informative probability distribution functions (PDFs) describing the Ramachandran phi-psi dihedral angle pair, a fundamental descriptor of backbone conformation of protein molecules, are derived from high-resolution X-ray crystal structures using an information-theoretic approach. The Information Maximization Device (IMD) is established, based on fundamental information-theoretic concepts, and then applied specifically to derive highly resolved phi-psi maps for all 20 single amino acid and all 8000 triplet sequences at an optimal resolution determined by the volume of current data. The paper shows that utilizing the latent information contained in all viable high-resolution crystal structures found in the Protein Data Bank (PDB), totaling more than 77,000 chains, permits the derivation of a large number of optimized sequence-dependent PDFs. This work demonstrates the effectiveness of the IMD and the superiority of the resulting PDFs by extensive fold recognition experiments and rigorous comparisons with previously published triplet PDFs. Because it automatically optimizes PDFs, IMD results in improved performance of knowledge-based potentials, which rely on such PDFs. Furthermore, it provides an easy computational recipe for empirically deriving other kinds of sequence-dependent structural PDFs with greater detail and precision. The high-resolution phi-psi maps derived in this work are available for download. ',\n",
              "  'authors': ['Solis, Armando D'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Crystallography, X-Ray',\n",
              "   'Databases, Protein',\n",
              "   'Knowledge Bases',\n",
              "   'Protein Conformation',\n",
              "   'Proteins'],\n",
              "  'pmid': '24896099',\n",
              "  'title': 'Deriving high-resolution protein backbone structure propensities from all crystal data using the information maximization device.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DY'],\n",
              "  'ESSN': '2296-2646',\n",
              "  'IF_2013': '3.994',\n",
              "  'IF_2014': '3.994',\n",
              "  'IF_2015': '3.994',\n",
              "  'IF_2016': '3.994',\n",
              "  'IF_2017': '4.155',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2296-2646',\n",
              "  'IsoAbbr': 'Front Chem',\n",
              "  'JrId': 42345,\n",
              "  'MedAbbr': 'Front Chem',\n",
              "  'NLMid': '101627988',\n",
              "  'Titre': 'Front Chem',\n",
              "  'abstract': 'Data mining approaches can uncover underlying patterns in chemical and pharmacological property space decisive for drug discovery and development. Two of the most common approaches are visualization and machine learning methods. Visualization methods use dimensionality reduction techniques in order to reduce multi-dimension data into 2D or 3D representations with a minimal loss of information. Machine learning attempts to find correlations between specific activities or classifications for a set of compounds and their features by means of recurring mathematical models. Both models take advantage of the different and deep relationships that can exist between features of compounds, and helpfully provide classification of compounds based on such features or in case of visualization methods uncover underlying patterns in the feature space. Drug-likeness has been studied from several viewpoints, but here we provide the first implementation in chemoinformatics of the t-Distributed Stochastic Neighbor Embedding (t-SNE) method for the visualization and the representation of chemical space, and the use of different machine learning methods separately and together to form a new ensemble learning method called AL Boost. The models obtained from AL Boost synergistically combine decision tree, random forests (RF), support vector machine (SVM), artificial neural network (ANN), k nearest neighbors (kNN), and logistic regression models. In this work, we show that together they form a predictive model that not only improves the predictive force but also decreases bias. This resulted in a corrected classification rate of over 0.81, as well as higher sensitivity and specificity rates for the models. In addition, separation and good models were also achieved for disease categories such as antineoplastic compounds and nervous system diseases, among others. Such models can be used to guide decision on the feature landscape of compounds and their likeness to either drugs or other characteristics, such as specific or multiple disease-category(ies) or organ(s) of action of a molecule.',\n",
              "  'authors': ['Yosipof, Abraham', 'Guedes, Rita C', 'García-Sosa, Alfonso T'],\n",
              "  'categories': ['CHEMISTRY, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Frontiers in chemistry',\n",
              "  'keywords': [''],\n",
              "  'pmid': '29868564',\n",
              "  'title': 'Data Mining and Machine Learning Models for Predicting Drug Likeness and Their Disease or Organ Category.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['LI'],\n",
              "  'ESSN': '1538-9375',\n",
              "  'IF_2013': '4.781',\n",
              "  'IF_2014': '4.939',\n",
              "  'IF_2015': '6.616',\n",
              "  'IF_2016': '5.775',\n",
              "  'IF_2017': '5.325',\n",
              "  'ISSN': '1525-8610',\n",
              "  'ISSN_online': '1538-9375',\n",
              "  'ISSN_print': '1525-8610',\n",
              "  'IsoAbbr': 'J Am Med Dir Assoc',\n",
              "  'JrId': 30377,\n",
              "  'MedAbbr': 'J Am Med Dir Assoc',\n",
              "  'NLMid': '100893243',\n",
              "  'Titre': 'J Am Med Dir Assoc',\n",
              "  'abstract': \"To investigate the suitability of a new eldercare robot (Guide) for people with dementia and their caregivers compared with one that has been successfully used before (Paro), and to generate suggestions for improved robot enhanced dementia care.|Cross-sectional study. A researcher demonstrated both robots in a random order to each staff member alone, or to each resident together with his/her relative(s). The researcher encouraged the participants to interact with each robot and asked staff and relatives a series of open ended questions about each robot.|A secure dementia residential facility in Auckland, New Zealand.|Ten people with dementia and 11 of their relatives, and five staff members.|Each robot interaction was video-taped and coded for the number of times the resident looked at, smiled, touched, and talked to and about each robot, as well as relative interactions with the resident. Qualitative analysis was used to code the open ended questions.|Residents smiled, touched and talked to Paro significantly more than Guide. Paro was found to be more acceptable to family members, staff, and residents, although many acknowledged that Guide had the potential to be useful if adapted for this population in terms of ergonomics and simplification.|Healthcare robots in dementia settings have to be simple and easy to use as well as stimulating and entertaining. This research highlights how eldercare robots may be adapted to have the best effects in dementia settings. It is concluded that Paro's sounds could be modified to be more acceptable to this population. The ergonomic design of Guide could be reviewed and the software application could be simplified and targeted to people with dementia.\",\n",
              "  'authors': ['Robinson, Hayley',\n",
              "   'MacDonald, Bruce A',\n",
              "   'Kerse, Ngaire',\n",
              "   'Broadbent, Elizabeth'],\n",
              "  'categories': ['GERIATRICS & GERONTOLOGY'],\n",
              "  'journal': 'Journal of the American Medical Directors Association',\n",
              "  'keywords': ['Animals',\n",
              "   'Cross-Sectional Studies',\n",
              "   'Dementia',\n",
              "   'Humans',\n",
              "   'New Zealand',\n",
              "   'Quality Improvement',\n",
              "   'Residential Facilities',\n",
              "   'Robotics',\n",
              "   'Social Support',\n",
              "   'Video Recording'],\n",
              "  'pmid': '23098418',\n",
              "  'title': 'Suitability of healthcare robots for a dementia unit and suggested improvements.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Cucurbit powdery mildew (PM) is one of the most severe fungal diseases, but the molecular mechanisms underlying PM resistance remain largely unknown, especially in pumpkin (Cucurbita moschata Duch.). The goal of this study was to identify gene expression differences in PM-treated plants (harvested at 24 h and 48 h after inoculation) and untreated (control) plants of inbred line \"112-2\" using RNA sequencing (RNA-Seq). The inbred line \"112-2\" has been purified over 8 consecutive generations of self-pollination and shows high resistance to PM. More than 7600 transcripts were examined in pumpkin leaves, and 3129 and 3080 differentially expressed genes (DEGs) were identified in inbred line \"112-2\" at 24 and 48 hours post inoculation (hpi), respectively. Based on the KEGG (Kyoto Encyclopedia of Genes and Genomes) pathway database and GO (Gene Ontology) database, a complex regulatory network for PM resistance that may involve hormone signal transduction pathways, transcription factors and defense responses was revealed at the transcription level. In addition, the expression profiles of 16 selected genes were analyzed using quantitative RT-PCR. Among these genes, the transcript levels of 6 DEGs, including bHLH87 (Basic Helix-loop-helix transcription factor), ERF014 (Ethylene response factor), WRKY21 (WRKY domain), HSF (heat stress transcription factor A), MLO3 (Mildew Locus O), and SGT1 (Suppressor of G-Two Allele of Skp1), in PM-resistant \"112-2\" were found to be significantly up- or down-regulated both before 9 hpi and at 24 hpi or 48 hpi; this behavior differed from that observed in the PM-susceptible material (cultivar \"Jiujiangjiaoding\"). The transcriptome data provide novel insights into the response of Cucurbita moschata to PM stress and are expected to be highly useful for dissecting PM defense mechanisms in this major vegetable and for improving pumpkin breeding with enhanced resistance to PM.',\n",
              "  'authors': ['Guo, Wei-Li',\n",
              "   'Chen, Bi-Hua',\n",
              "   'Chen, Xue-Jin',\n",
              "   'Guo, Yan-Yan',\n",
              "   'Yang, He-Lian',\n",
              "   'Li, Xin-Zheng',\n",
              "   'Wang, Guang-Yin'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Ascomycota',\n",
              "   'Cucurbita',\n",
              "   'Disease Resistance',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Metabolic Networks and Pathways',\n",
              "   'Photosynthesis',\n",
              "   'Plant Diseases',\n",
              "   'Plant Growth Regulators',\n",
              "   'Plant Leaves',\n",
              "   'RNA, Plant',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Signal Transduction',\n",
              "   'Transcription Factors'],\n",
              "  'pmid': '29320569',\n",
              "  'title': 'Transcriptome profiling of pumpkin (Cucurbita moschata Duch.) leaves infected with powdery mildew.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1742-1241',\n",
              "  'IF_2013': '2.538',\n",
              "  'IF_2014': '2.566',\n",
              "  'IF_2015': '2.226',\n",
              "  'IF_2016': '2.14',\n",
              "  'IF_2017': '2',\n",
              "  'ISSN': '1368-5031',\n",
              "  'ISSN_online': '1742-1241',\n",
              "  'ISSN_print': '1368-5031',\n",
              "  'IsoAbbr': 'Int. J. Clin. Pract.',\n",
              "  'JrId': 20688,\n",
              "  'MedAbbr': 'Int J Clin Pract',\n",
              "  'NLMid': '9712381',\n",
              "  'Titre': 'Int. J. Clin. Pract.',\n",
              "  'abstract': 'To report short- and medium-term oncological and functional outcomes of the first robotic-assisted laparoscopic radical cystectomy (RARC) series from the UK.|Thirty patients underwent RARC between 2004 and 2007 at our unit. We report oncological and functional outcomes of this procedure in 20 patients (17 ileal conduit and three Studer Pouches), who have completed at least 6 months of follow up.|There were 17 men and three women, median age 66 years (range 38-77 years). Median operating time was 330 min (range 295-510 min), and median blood loss 150 ml (range 100-1150 ml). There were two major complications (10%); a port site bleed and a rectal injury. The median follow up of this cohort is 23 months (range 7-44 months). One patient died of distant metastases at 8 months, and another developed a right ureteric tumour at 7 months. None of the patients had local pelvic or port site recurrence. The overall and disease-free survival are 95% and 90% respectively. Functional complications included a neovesico-urethral stricture at 3 months, a left upper ureteric stricture at 6 months and an incisional hernia at 12 months.|Robotic-assisted laparoscopic radical cystectomy is an emerging minimally invasive procedure which at short- to medium-term follow up, in our experience, is oncologically and functionally equivalent to open radical cystectomy.',\n",
              "  'authors': ['Dasgupta, P',\n",
              "   'Rimington, P',\n",
              "   'Murphy, D',\n",
              "   'Challacombe, B',\n",
              "   'Hemal, A',\n",
              "   'Elhage, O',\n",
              "   'Khan, M S'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'International journal of clinical practice',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Cystectomy',\n",
              "   'Disease-Free Survival',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Length of Stay',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Bladder Neoplasms'],\n",
              "  'pmid': '19143856',\n",
              "  'title': 'Robotic assisted radical cystectomy: short to medium-term oncologic and functional outcomes.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In this paper, a reduced order neural observer (RONO) with a time-varying learning rate is proposed. The proposed scheme is based on a discrete-time recurrent high order neural network (RHONN) trained with an extended Kalman filter (EKF)-based algorithm. A time-varying learning rate is designed in order to improve the learning of the neuronal network in presence of disturbances and parameter variations. This work includes the stability proof of the time-varying learning. The applicability of the developed observer is illustrated via simulations for a nonlinear anaerobic digestion process. ',\n",
              "  'authors': ['Gurubel, K J',\n",
              "   'Alanis, A Y',\n",
              "   'Sanchez, E N',\n",
              "   'Carlos-Hernandez, S'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Observation',\n",
              "   'Time Factors'],\n",
              "  'pmid': '24344696',\n",
              "  'title': 'A neural observer with time-varying learning rate: analysis and applications.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'Image segmentation plays an important role in medical image processing. Fuzzy c-means (FCM) is one of the popular clustering algorithms for medical image segmentation. But FCM is highly vulnerable to noise due to not considering the spatial information in image segmentation. This paper introduces medium mathematics system which is employed to process fuzzy information for image segmentation. It establishes the medium similarity measure based on the measure of medium truth degree (MMTD) and uses the correlation of the pixel and its neighbors to define the medium membership function. An improved FCM medical image segmentation algorithm based on MMTD which takes some spatial features into account is proposed in this paper. The experimental results show that the proposed algorithm is more antinoise than the standard FCM, with more certainty and less fuzziness. This will lead to its practicable and effective applications in medical image segmentation. ',\n",
              "  'authors': ['Zhou, Ningning', 'Yang, Tingting', 'Zhang, Shaobai'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Cluster Analysis',\n",
              "   'Computational Biology',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Models, Theoretical',\n",
              "   'Normal Distribution',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Signal-To-Noise Ratio',\n",
              "   'Software'],\n",
              "  'pmid': '24648852',\n",
              "  'title': 'An improved FCM medical image segmentation algorithm based on MMTD.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1549-5485',\n",
              "  'IF_2013': '4.375',\n",
              "  'IF_2014': '3.657',\n",
              "  'IF_2015': '2.906',\n",
              "  'IF_2016': '2.894',\n",
              "  'IF_2017': '2.671',\n",
              "  'ISSN': '1072-0502',\n",
              "  'ISSN_online': '1549-5485',\n",
              "  'ISSN_print': '1072-0502',\n",
              "  'IsoAbbr': 'Learn. Mem.',\n",
              "  'JrId': 20827,\n",
              "  'MedAbbr': 'Learn Mem',\n",
              "  'NLMid': '9435678',\n",
              "  'Titre': 'Learn. Mem.',\n",
              "  'abstract': 'The relative contributions of plasticity in the amygdala vs. its afferent pathways to conditioned fear remain controversial. Some believe that thalamic and cortical neurons transmitting information about the conditioned stimulus (CS) to the lateral amygdala (LA) serve a relay function. Others maintain that thalamic and/or cortical plasticity is critically involved in fear conditioning. To address this question, we developed a large-scale biophysical model of the LA that could reproduce earlier findings regarding the cellular correlates of fear conditioning in LA. We then conducted model experiments that examined whether fear memories depend on (1) training-induced increases in the responsiveness of thalamic and cortical neurons projecting to LA, (2) plasticity at the synapses they form in LA, and/or (3) plasticity at synapses between LA neurons. These tests revealed that training-induced increases in the responsiveness of afferent neurons are required for fear memory formation. However, once the memory has been formed, this factor is no longer required because the efficacy of the synapses that thalamic and cortical neurons form with LA cells has augmented enough to maintain the memory. In contrast, our model experiments suggest that plasticity at synapses between LA neurons plays a minor role in maintaining the fear memory. ',\n",
              "  'authors': ['Kim, Dongbeom', 'Paré, Denis', 'Nair, Satish S'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Learning & memory (Cold Spring Harbor, N.Y.)',\n",
              "  'keywords': ['Amygdala',\n",
              "   'Animals',\n",
              "   'Conditioning (Psychology)',\n",
              "   'Fear',\n",
              "   'Humans',\n",
              "   'Memory',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Neurons'],\n",
              "  'pmid': '23864645',\n",
              "  'title': 'Mechanisms contributing to the induction and storage of Pavlovian fear memories in the lateral amygdala.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '1945-4589',\n",
              "  'IF_2013': '4.886',\n",
              "  'IF_2014': '6.432',\n",
              "  'IF_2015': '3.979',\n",
              "  'IF_2016': '4.867',\n",
              "  'IF_2017': '5.179',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1945-4589',\n",
              "  'IsoAbbr': 'Aging (Albany NY)',\n",
              "  'JrId': 36571,\n",
              "  'MedAbbr': 'Aging (Albany NY)',\n",
              "  'NLMid': '101508617',\n",
              "  'Titre': 'Aging (Albany NY)',\n",
              "  'abstract': 'Multiple interventions in the aging process have been discovered to extend the healthspan of model organisms. Both industry and academia are therefore exploring possible transformative molecules that target aging and age-associated diseases. In this overview, we summarize the presented talks and discussion points of the 5th Annual Aging and Drug Discovery Forum 2018 in Basel, Switzerland. Here academia and industry came together, to discuss the latest progress and issues in aging research. The meeting covered talks about the mechanistic cause of aging, how longevity signatures may be highly conserved, emerging biomarkers of aging, possible interventions in the aging process and the use of artificial intelligence for aging research and drug discovery. Importantly, a consensus is emerging both in industry and academia, that molecules able to intervene in the aging process may contain the potential to transform both societies and healthcare.',\n",
              "  'authors': ['Bakula, Daniela',\n",
              "   'Aliper, Alexander M',\n",
              "   'Mamoshina, Polina',\n",
              "   'Petr, Michael A',\n",
              "   'Teklu, Amanuel',\n",
              "   'Baur, Joseph A',\n",
              "   'Campisi, Judith',\n",
              "   'Ewald, Collin Y',\n",
              "   'Georgievskaya, Anastasia',\n",
              "   'Gladyshev, Vadim N',\n",
              "   'Kovalchuk, Olga',\n",
              "   'Lamming, Dudley W',\n",
              "   'Luijsterburg, Martijn S',\n",
              "   'Martín-Montalvo, Alejandro',\n",
              "   'Maudsley, Stuart',\n",
              "   'Mkrtchyan, Garik V',\n",
              "   'Moskalev, Alexey',\n",
              "   'Olshansky, S Jay',\n",
              "   'Ozerov, Ivan V',\n",
              "   'Pickett, Alexander',\n",
              "   'Ristow, Michael',\n",
              "   'Zhavoronkov, Alex',\n",
              "   'Scheibye-Knudsen, Morten'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Aging',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30425188',\n",
              "  'title': 'Aging and drug discovery.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1873-4030',\n",
              "  'IF_2013': '1.839',\n",
              "  'IF_2014': '1.825',\n",
              "  'IF_2015': '1.619',\n",
              "  'IF_2016': '1.819',\n",
              "  'IF_2017': '1.923',\n",
              "  'ISSN': '1350-4533',\n",
              "  'ISSN_online': '1873-4030',\n",
              "  'ISSN_print': '1350-4533',\n",
              "  'IsoAbbr': 'Med Eng Phys',\n",
              "  'JrId': 20171,\n",
              "  'MedAbbr': 'Med Eng Phys',\n",
              "  'NLMid': '9422753',\n",
              "  'Titre': 'Med Eng Phys',\n",
              "  'abstract': 'Central venous catheterization involves venous puncture and catheter insertion for transfusions. Quantitative conditions that facilitate insertion of the needle, such as the angle and velocity of insertion, have not been clarified. We previously developed a robotic system for guiding the needle along a specified puncture path with high precision and are currently implementing a hardware design for a robotic system to assist in blood vessel puncture. In this study, we proposed the insertion angle and velocity for stopping the needle in a blood vessel, assuming that a robotic system such as ours is used during the procedure. We inserted a needle into a segment of porcine jugular vein and obtained the puncture reaction force. Evaluation indices were the magnitude of the sudden decrease in reaction force at the point at which the needle advances and the length of time that the needle is present within the vein. Results indicated that the conditions under which it was easiest to stop the needle inside the vein were an insertion angle range of 10-20 and an insertion velocity of 3mm/s.',\n",
              "  'authors': ['Kobayashi, Yo',\n",
              "   'Hamano, Ryutaro',\n",
              "   'Watanabe, Hiroki',\n",
              "   'Hong, Jaesung',\n",
              "   'Toyoda, Kazutaka',\n",
              "   'Hashizume, Makoto',\n",
              "   'Fujie, Masakatsu G'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Medical engineering & physics',\n",
              "  'keywords': ['Animals',\n",
              "   'Mechanical Phenomena',\n",
              "   'Needles',\n",
              "   'Punctures',\n",
              "   'Robotics',\n",
              "   'Swine',\n",
              "   'Vascular Surgical Procedures'],\n",
              "  'pmid': '23332178',\n",
              "  'title': 'Use of puncture force measurement to investigate the conditions of blood vessel needle insertion.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['FY'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.154',\n",
              "  'IF_2014': '1.154',\n",
              "  'IF_2015': '1.154',\n",
              "  'IF_2016': '1.436',\n",
              "  'IF_2017': '1.725',\n",
              "  'ISSN': '1463-4201',\n",
              "  'ISSN_print': '1463-4201',\n",
              "  'IsoAbbr': 'Int J Comput Dent',\n",
              "  'JrId': 22380,\n",
              "  'MedAbbr': 'Int J Comput Dent',\n",
              "  'NLMid': '100891504',\n",
              "  'Titre': 'Int J Comput Dent',\n",
              "  'abstract': 'John Naisbitt, in his 1982 book \"Megatrends,\" postulated an important characteristic of our information and knowledge society: \"We are drowning in information but starved for knowledge.\" Today, in the age of the Information and Knowledge society, we are faced with this problem every single day, because Web searches and information selection are highly time-consuming activities. Internet search engines attempt to employ intelligent search algorithms in order to optimize their search results. Nevertheless, the question remains how \"qualitative knowledge\" can be selected, ie, knowledge needed for supporting decisions in medicine and dentistry. Semantic search engines are one current approach to this problem. For this reason, a project entitled \"Mr. Q, your personal Web Assistant\" has been initiated and will be introduced in this paper.',\n",
              "  'authors': ['Ammann, A', 'Kiss, T', 'Hirsch, M', 'Matthies, H'],\n",
              "  'categories': ['DENTISTRY, ORAL SURGERY & MEDICINE'],\n",
              "  'journal': 'International journal of computerized dentistry',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Information Dissemination',\n",
              "   'Information Management',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Internet',\n",
              "   'Medical Informatics Applications',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Semantics',\n",
              "   'Systems Integration',\n",
              "   'Terminology as Topic'],\n",
              "  'pmid': '19216311',\n",
              "  'title': 'Where, what, why: Mr. Q on the Web.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Face recognition is challenging especially when the images from different persons are similar to each other due to variations in illumination, expression, and occlusion. If we have sufficient training images of each person which can span the facial variations of that person under testing conditions, sparse representation based classification (SRC) achieves very promising results. However, in many applications, face recognition often encounters the small sample size problem arising from the small number of available training images for each person. In this paper, we present a novel face recognition framework by utilizing low-rank and sparse error matrix decomposition, and sparse coding techniques (LRSE+SC). Firstly, the low-rank matrix recovery technique is applied to decompose the face images per class into a low-rank matrix and a sparse error matrix. The low-rank matrix of each individual is a class-specific dictionary and it captures the discriminative feature of this individual. The sparse error matrix represents the intra-class variations, such as illumination, expression changes. Secondly, we combine the low-rank part (representative basis) of each person into a supervised dictionary and integrate all the sparse error matrix of each individual into a within-individual variant dictionary which can be applied to represent the possible variations between the testing and training images. Then these two dictionaries are used to code the query image. The within-individual variant dictionary can be shared by all the subjects and only contribute to explain the lighting conditions, expressions, and occlusions of the query image rather than discrimination. At last, a reconstruction-based scheme is adopted for face recognition. Since the within-individual dictionary is introduced, LRSE+SC can handle the problem of the corrupted training data and the situation that not all subjects have enough samples for training. Experimental results show that our method achieves the state-of-the-art results on AR, FERET, FRGC and LFW databases. ',\n",
              "  'authors': ['Tang, Xin', 'Feng, Guo-Can', 'Li, Xiao-Xin', 'Cai, Jia-Xin'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Databases, Factual',\n",
              "   'Facial Recognition',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Least-Squares Analysis',\n",
              "   'Lighting',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Models, Statistical',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Sample Size',\n",
              "   'Software'],\n",
              "  'pmid': '26571112',\n",
              "  'title': 'Learning Low-Rank Class-Specific Dictionary and Sparse Intra-Class Variant Dictionary for Face Recognition.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Few studies of laparoscopic cholecystectomy (LC) outcome have used longitudinal data for more than two years. Moreover, no studies have considered group differences in factors other than outcome such as age and nonsurgical treatment. Additionally, almost all published articles agree that the essential issue of the internal validity (reproducibility) of the artificial neural network (ANN), support vector machine (SVM), Gaussian process regression (GPR) and multiple linear regression (MLR) models has not been adequately addressed. This study proposed to validate the use of these models for predicting quality of life (QOL) after LC and to compare the predictive capability of ANNs with that of SVM, GPR and MLR.|A total of 400 LC patients completed the SF-36 and the Gastrointestinal Quality of Life Index at baseline and at 2 years postoperatively. The criteria for evaluating the accuracy of the system models were mean square error (MSE) and mean absolute percentage error (MAPE). A global sensitivity analysis was also performed to assess the relative significance of input parameters in the system model and to rank the variables in order of importance. Compared to SVM, GPR and MLR models, the ANN model generally had smaller MSE and MAPE values in the training data set and test data set. Most ANN models had MAPE values ranging from 4.20% to 8.60%, and most had high prediction accuracy. The global sensitivity analysis also showed that preoperative functional status was the best parameter for predicting QOL after LC.|Compared with SVM, GPR and MLR models, the ANN model in this study was more accurate in predicting patient-reported QOL and had higher overall performance indices. Further studies of this model may consider the effect of a more detailed database that includes complications and clinical examination findings as well as more detailed outcome data.',\n",
              "  'authors': ['Shi, Hon-Yi',\n",
              "   'Lee, Hao-Hsien',\n",
              "   'Tsai, Jinn-Tsong',\n",
              "   'Ho, Wen-Hsien',\n",
              "   'Chen, Chieh-Fan',\n",
              "   'Lee, King-Teh',\n",
              "   'Chiu, Chong-Chi'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Cholecystectomy, Laparoscopic',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Linear Models',\n",
              "   'Longitudinal Studies',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Normal Distribution',\n",
              "   'Prospective Studies',\n",
              "   'Quality of Life',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '23284677',\n",
              "  'title': 'Comparisons of prediction models of quality of life after laparoscopic cholecystectomy: a longitudinal prospective study.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': \"A common feature that is typical of the patients with neurodegenerative (ND) disease is the impairment of motor function, which can interrupt the pathway from cerebrum to the muscle and thus cause movement disorders. For patients with amyotrophic lateral sclerosis disease (ALS), the impairment is caused by the loss of motor neurons. While for patients with Parkinson's disease (PD) and Huntington's disease (HD), it is related to the basal ganglia dysfunction. Previously studies have demonstrated the usage of gait analysis in characterizing the ND patients for the purpose of disease management. However, most studies focus on extracting characteristic features that can differentiate ND gait from normal gait. Few studies have demonstrated the feasibility of modelling the nonlinear gait dynamics in characterizing the ND gait. Therefore, in this study, a novel approach based on an adaptive neuro-fuzzy inference system (ANFIS) is presented for identification of the gait of patients with ND disease. The proposed ANFIS model combines neural network adaptive capabilities and the fuzzy logic qualitative approach. Gait dynamics such as stride intervals, stance intervals, and double support intervals were used as the input variables to the model. The particle swarm optimization (PSO) algorithm was utilized to learn the parameters of the ANFIS model. The performance of the system was evaluated in terms of sensitivity, specificity, and accuracy using the leave-one-out cross-validation method. The competitive classification results on a dataset of 13 ALS patients, 15 PD patients, 20 HD patients, and 16 healthy control subjects indicated the effectiveness of our approach in representing the gait characteristics of ND patients.\",\n",
              "  'authors': ['Ye, Qiang', 'Xia, Yi', 'Yao, Zhiming'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Algorithms',\n",
              "   'Amyotrophic Lateral Sclerosis',\n",
              "   'Fuzzy Logic',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Huntington Disease',\n",
              "   'Medical Informatics',\n",
              "   'Middle Aged',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurodegenerative Diseases',\n",
              "   'Parkinson Disease',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30363986',\n",
              "  'title': 'Classification of Gait Patterns in Patients with Neurodegenerative Disease Using Adaptive Neuro-Fuzzy Inference System.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Habitat classifications provide guidelines for mapping and comparing marine resources across geographic regions. Calcareous bio-concretions and their associated biota have not been exhaustively categorized. Furthermore, for management and conservation purposes, species and habitat mapping is critical. Recently, several developments have occurred in the field of predictive habitat modeling, and multiple methods are available. In this study, we defined the habitats constituting northern Adriatic biogenic reefs and created a predictive habitat distribution model. We used an updated dataset of the epibenthic assemblages to define the habitats, which we verified using the fuzzy k-means (FKM) clustering method. Redundancy analysis was employed to model the relationships between the environmental descriptors and the FKM membership grades. Predictive modelling was carried out to map habitats across the basin. Habitat A (opportunistic macroalgae, encrusting Porifera, bioeroders) characterizes reefs closest to the coastline, which are affected by coastal currents and river inputs. Habitat B is distinguished by massive Porifera, erect Tunicata, and non-calcareous encrusting algae (Peyssonnelia spp.). Habitat C (non-articulated coralline, Polycitor adriaticus) is predicted in deeper areas. The onshore-offshore gradient explains the variability of the assemblages because of the influence of coastal freshwater, which is the main driver of nutrient dynamics. This model supports the interpretation of Habitat A and C as the extremes of a gradient that characterizes the epibenthic assemblages, while Habitat B demonstrates intermediate characteristics. Areas of transition are a natural feature of the marine environment and may include a mixture of habitats and species. The habitats proposed are easy to identify in the field, are related to different environmental features, and may be suitable for application in studies focused on other geographic areas. The habitat model outputs provide insight into the environmental drivers that control the distribution of the habitat and can be used to guide future research efforts and cost-effective management and conservation plans. ',\n",
              "  'authors': ['Falace, Annalisa',\n",
              "   'Kaleb, Sara',\n",
              "   'Curiel, Daniele',\n",
              "   'Miotti, Chiara',\n",
              "   'Galli, Giovanni',\n",
              "   'Querin, Stefano',\n",
              "   'Ballesteros, Enric',\n",
              "   'Solidoro, Cosimo',\n",
              "   'Bandelj, Vinko'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Animals',\n",
              "   'Anthozoa',\n",
              "   'Biodiversity',\n",
              "   'Ecosystem',\n",
              "   'Fuzzy Logic',\n",
              "   'Models, Theoretical',\n",
              "   'Oceans and Seas'],\n",
              "  'pmid': '26560891',\n",
              "  'title': 'Calcareous Bio-Concretions in the Northern Adriatic Sea: Habitat Types, Environmental Factors that Influence Habitat Distributions, and Predictive Modeling.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': 'Pathogenesis of inflammatory diseases can be tracked by studying the causality relationships among the factors contributing to its development. We could, for instance, hypothesize on the connections of the pathogenesis outcomes to the observed conditions. And to prove such causal hypotheses we would need to have the full understanding of the causal relationships, and we would have to provide all the necessary evidences to support our claims. In practice, however, we might not possess all the background knowledge on the causality relationships, and we might be unable to collect all the evidence to prove our hypotheses.|In this work we propose a methodology for the translation of biological knowledge on causality relationships of biological processes and their effects on conditions to a computational framework for hypothesis testing. The methodology consists of two main points: hypothesis graph construction from the formalization of the background knowledge on causality relationships, and confidence measurement in a causality hypothesis as a normalized weighted path computation in the hypothesis graph. In this framework, we can simulate collection of evidences and assess confidence in a causality hypothesis by measuring it proportionally to the amount of available knowledge and collected evidences.|We evaluate our methodology on a hypothesis graph that represents both contributing factors which may cause cartilage degradation and the factors which might be caused by the cartilage degradation during osteoarthritis. Hypothesis graph construction has proven to be robust to the addition of potentially contradictory information on the simultaneously positive and negative effects. The obtained confidence measures for the specific causality hypotheses have been validated by our domain experts, and, correspond closely to their subjective assessments of confidences in investigated hypotheses. Overall, our methodology for a shared hypothesis testing framework exhibits important properties that researchers will find useful in literature review for their experimental studies, planning and prioritizing evidence collection acquisition procedures, and testing their hypotheses with different depths of knowledge on causal dependencies of biological processes and their effects on the observed conditions.',\n",
              "  'authors': ['Agibetov, Asan',\n",
              "   'Jiménez-Ruiz, Ernesto',\n",
              "   'Ondrésik, Marta',\n",
              "   'Solimando, Alessandro',\n",
              "   'Banerjee, Imon',\n",
              "   'Guerrini, Giovanna',\n",
              "   'Catalano, Chiara E',\n",
              "   'Oliveira, Joaquim M',\n",
              "   'Patanè, Giuseppe',\n",
              "   'Reis, Rui L',\n",
              "   'Spagnuolo, Michela'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Biological Ontologies', 'Computer Graphics', 'Inflammation'],\n",
              "  'pmid': '29422110',\n",
              "  'title': 'Supporting shared hypothesis testing in the biomedical domain.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['TD'],\n",
              "  'ESSN': '1879-1476',\n",
              "  'IF_2013': '1.004',\n",
              "  'IF_2014': '1.135',\n",
              "  'IF_2015': '1.038',\n",
              "  'IF_2016': '1.128',\n",
              "  'IF_2017': '1.387',\n",
              "  'ISSN': '0385-8146',\n",
              "  'ISSN_online': '1879-1476',\n",
              "  'ISSN_print': '0385-8146',\n",
              "  'IsoAbbr': 'Auris Nasus Larynx',\n",
              "  'JrId': 925,\n",
              "  'MedAbbr': 'Auris Nasus Larynx',\n",
              "  'NLMid': '7708170',\n",
              "  'Titre': 'Auris Nasus Larynx',\n",
              "  'abstract': 'Frequency modulation (FM) sounds were used as simultaneous maskers to estimate the psychophysical tuning curve (PTC). Maskers (800 ms duration) were FM sounds sweeping upward (1.2-->2.8 kHz) in Experiment I and downward (2.8-->1.2 kHz) in Experiment II. The signal was a pure tone of 2 kHz and the signal levels were 15-, 20-, 30-, and 35-dB SL. Masked PTCs were obtained as a function of masker frequency. The results were as follows: (i) When an upward-sweeping FM sound is used for high signal level, the slope of the PTC is shallower on the low frequency side and steeper on the high frequency side, and the 10-dB bandwidth of PTC coincides with that of the restricted PTC. (ii) When a downward-sweeping FM sound is used for the high signal level, the above relation for the PTC slope is reversed, and the 10-dB bandwidth for PTC coincides with that of unrestricted PTC. (iii) PTCs determined by masking in the presence of FM sounds apparently restrict the subject to listening to off-frequency, and it is shown that the bandwidth of PTC properly reflects the critical band. (iv) The properties of these PTCs can be easily explained by a neural network model of lateral inhibition, so we can accept a hypothesis that the hierarchical processing mechanism is incorporated within the inferior colliculus for parallel signal processing. (v) We conclude that the frequency resolution and the critical band filtering are, therefore, realized in a psychophysically relevant way in the auditory midbrain level.',\n",
              "  'authors': ['Isojima, G', 'Suzuki, T', 'Ishigami, H'],\n",
              "  'categories': ['OTORHINOLARYNGOLOGY'],\n",
              "  'journal': 'Auris, nasus, larynx',\n",
              "  'keywords': ['Adult',\n",
              "   'Attention',\n",
              "   'Auditory Pathways',\n",
              "   'Auditory Threshold',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Inferior Colliculi',\n",
              "   'Male',\n",
              "   'Neural Inhibition',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Perceptual Masking',\n",
              "   'Pitch Discrimination',\n",
              "   'Psychoacoustics',\n",
              "   'Sound Spectrography'],\n",
              "  'pmid': '7677631',\n",
              "  'title': 'Psychophysical tuning curve and critical band determined by masking in the presence of FM sounds.',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1536-0210',\n",
              "  'IF_2013': '4.453',\n",
              "  'IF_2014': '4.437',\n",
              "  'IF_2015': '4.887',\n",
              "  'IF_2016': '5.195',\n",
              "  'IF_2017': '6.224',\n",
              "  'ISSN': '0020-9996',\n",
              "  'ISSN_online': '1536-0210',\n",
              "  'ISSN_print': '0020-9996',\n",
              "  'IsoAbbr': 'Invest Radiol',\n",
              "  'JrId': 4373,\n",
              "  'MedAbbr': 'Invest Radiol',\n",
              "  'NLMid': '0045377',\n",
              "  'Titre': 'Invest Radiol',\n",
              "  'abstract': 'This study compares the performance of quantitative methods for the characterization of signal-time curves acquired by dynamic contrast-enhanced magnetic resonance mammography from 253 females.|Signal-time curves obtained from 105 parenchyma, 162 malignant, and 91 benign tissue regions were examined (243 lesions were histopathologically validated). A neural network, a nearest-neighbor, and a threshold classifier were applied to either the entire signal-time curve or pharmacokinetic and descriptive parameters calculated from the curves to differentiate between 2 (malignant or benign) or 3 tissue classes (malignant, benign, or parenchyma). The classifiers were tuned and evaluated according to their performance on 2 distinct subsets of the curves.|The accuracy determined for the neural network and the nearest-neighbor classifiers was nearly identical (approximately 80% in case of 3 tissue classes, and approximately 76% in case of the 2 classes). In contrast, the accuracy of the threshold classifier applied to the discrimination of 3 classes was low (65%).|Quantitative classifiers can support the radiologist in the diagnosis of breast lesions.',\n",
              "  'authors': ['Lucht, Robert E A',\n",
              "   'Delorme, Stefan',\n",
              "   'Hei, Jürgen',\n",
              "   'Knopp, Michael V',\n",
              "   'Weber, Marc-André',\n",
              "   'Griebel, Jürgen',\n",
              "   'Brix, Gunnar'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Investigative radiology',\n",
              "  'keywords': ['Breast',\n",
              "   'Breast Diseases',\n",
              "   'Breast Neoplasms',\n",
              "   'Contrast Media',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Mammography',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Time Factors'],\n",
              "  'pmid': '15973136',\n",
              "  'title': 'Classification of signal-time curves obtained by dynamic magnetic resonance mammography: statistical comparison of quantitative methods.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1545-4126',\n",
              "  'IF_2013': '22.66',\n",
              "  'IF_2014': '19.32',\n",
              "  'IF_2015': '14.265',\n",
              "  'IF_2016': '15.63',\n",
              "  'IF_2017': '14.675',\n",
              "  'ISSN': '0147-006X',\n",
              "  'ISSN_online': '1545-4126',\n",
              "  'ISSN_print': '0147-006X',\n",
              "  'IsoAbbr': 'Annu. Rev. Neurosci.',\n",
              "  'JrId': 635,\n",
              "  'MedAbbr': 'Annu Rev Neurosci',\n",
              "  'NLMid': '7804039',\n",
              "  'Titre': 'Annu. Rev. Neurosci.',\n",
              "  'abstract': 'System identification is a growing approach to sensory neurophysiology that facilitates the development of quantitative functional models of sensory processing. This approach provides a clear set of guidelines for combining experimental data with other knowledge about sensory function to obtain a description that optimally predicts the way that neurons process sensory information. This prediction paradigm provides an objective method for evaluating and comparing computational models. In this chapter we review many of the system identification algorithms that have been used in sensory neurophysiology, and we show how they can be viewed as variants of a single statistical inference problem. We then review many of the practical issues that arise when applying these methods to neurophysiological experiments: stimulus selection, behavioral control, model visualization, and validation. Finally we discuss several problems to which system identification has been applied recently, including one important long-term goal of sensory neuroscience: developing models of sensory systems that accurately predict neuronal responses under completely natural conditions.',\n",
              "  'authors': ['Wu, Michael C-K', 'David, Stephen V', 'Gallant, Jack L'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Annual review of neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Computer Simulation',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons, Afferent',\n",
              "   'Neurophysiology',\n",
              "   'Nonlinear Dynamics',\n",
              "   'Predictive Value of Tests'],\n",
              "  'pmid': '16776594',\n",
              "  'title': 'Complete functional characterization of sensory neurons by system identification.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5305',\n",
              "  'IF_2013': '1.735',\n",
              "  'IF_2014': '1.068',\n",
              "  'IF_2015': '1.069',\n",
              "  'IF_2016': '0.964',\n",
              "  'IF_2017': '1.039',\n",
              "  'ISSN': '0929-8665',\n",
              "  'ISSN_online': '1875-5305',\n",
              "  'ISSN_print': '0929-8665',\n",
              "  'IsoAbbr': 'Protein Pept. Lett.',\n",
              "  'JrId': 20398,\n",
              "  'MedAbbr': 'Protein Pept Lett',\n",
              "  'NLMid': '9441434',\n",
              "  'Titre': 'Protein Pept. Lett.',\n",
              "  'abstract': 'Protein palmitoylation is an important and common post-translational lipid modification of proteins and plays a critical role in various cellular processes. Identification of Palmitoylation sites is fundamental to decipher the mechanisms of these biological processes. However, experimental determination of palmitoylation residues without prior knowledge is laborious and costly. Thus computational approaches for prediction of palmitoylation sites in proteins have become highly desirable. Here, we propose PPWMs, a computational predictor using Position Weight Matrices (PWMs) encoding scheme and support vector machine (SVM) for identifying protein palmitoylation sites. Our PPWMs shows a nice predictive performance with the area under the ROC curve (AUC) of 0.9472 for the S-palmitoylation sites prediction and 0.9964 for the N-palmitoylation sites prediction on the newly proposed dataset. Comparison results show the superiority of PPWMs over two existing widely known palmitoylation site predictors CSS-Palm 2.0 and CKSAAP-Palm in many cases. Moreover, an attempt of incorporating structure information such as accessible surface area (ASA) and secondary structure (SS) into prediction is made and the structure characteristics are analyzed roughly. The corresponding software can be freely downloaded from http://math.cau.edu.cn/PPWMs.html.',\n",
              "  'authors': ['Li, Yu-Xin', 'Shao, Yuan-Hai', 'Deng, Nai-Yang'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Protein and peptide letters',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Lipoylation',\n",
              "   'Models, Biological',\n",
              "   'Position-Specific Scoring Matrices',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'Proteins'],\n",
              "  'pmid': '21054270',\n",
              "  'title': 'Improved prediction of palmitoylation sites using PWMs and SVM.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '1469-493X',\n",
              "  'IF_2013': '5.939',\n",
              "  'IF_2014': '6.032',\n",
              "  'IF_2015': '6.103',\n",
              "  'IF_2016': '6.264',\n",
              "  'IF_2017': '6.754',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1469-493X',\n",
              "  'IsoAbbr': 'Cochrane Database Syst Rev',\n",
              "  'JrId': 21711,\n",
              "  'MedAbbr': 'Cochrane Database Syst Rev',\n",
              "  'NLMid': '100909747',\n",
              "  'Titre': 'Cochrane Database Syst Rev',\n",
              "  'abstract': 'The choice of approach to the laparoscopic repair of inguinal hernia is controversial. There is a scarcity of data comparing the laparoscopic transabdominal preperitoneal (TAPP) approach with the laparoscopic totally extraperitoneal (TEP) approach and questions remain about their relative merits and risks.|To compare the clinical effectiveness and relative efficiency of laparoscopic TAPP and laparoscopic TEP for inguinal hernia repair.|We searched Medline Extra, Embase, Biosis, Science Citation Index, Cochrane Central Register of Controlled Trials (CENTRAL), Journals@ Ovid Full Text and the electronic version of the journal, Surgical Endoscopy. Recent conference proceedings by the following organisations were hand searched: Association of Endoscopic Surgeons of Great Britain & Ireland; International Congress of the European Association for Endoscopic Surgery; Scientific Session of the Society of American Gastrointestinal & Endoscopic Surgeons (SAGES); and the Italian Society of Endoscopic Surgery. In addition, specialists involved in research on the repair of inguinal hernia were contacted to ask for information about any further completed and ongoing trials, relevant websites were searched and reference lists of the all included studies were checked for additional reports.|All published and unpublished randomised controlled trials and quasi-randomised controlled trials comparing laparoscopic TAPP with laparoscopic TEP for inguinal hernia repair were eligible for inclusion. Non-randomised prospective studies were also eligible for inclusion to provide further comparative evidence of complications and adverse events.|Statistical analyses were performed using the fixed effects model and the results expressed as relative risk (RR) for dichotomous outcomes and weighted mean difference (WMD) for continuous outcomes with 95% confidence intervals (CI).|The search identified one RCT which reported no statistical difference between TAPP and TEP when considering duration of operation, haematoma, length of stay, time to return to usual activity and recurrence. The eight non-randomised studies suggest that TAPP is associated with higher rates of port-site hernias and visceral injuries whilst there appear to be more conversions with TEP. Vascular injuries and deep/mesh infections were rare and there was no obvious difference between the groups. No studies reporting economic evidence were identified. Very limited data were available on learning effects but these data suggest that operators become experienced at between 30 and 100 procedures.|There is insufficient data to allow conclusions to be drawn about the relative effectiveness of TEP compared with TAPP. Efforts should be made to start and complete adequately powered RCTs, which compare the different methods of laparoscopic repair.',\n",
              "  'authors': ['Wake, B L',\n",
              "   'McCormack, K',\n",
              "   'Fraser, C',\n",
              "   'Vale, L',\n",
              "   'Perez, J',\n",
              "   'Grant, A M'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'The Cochrane database of systematic reviews',\n",
              "  'keywords': ['Hernia, Inguinal',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Randomized Controlled Trials as Topic'],\n",
              "  'pmid': '15674961',\n",
              "  'title': 'Transabdominal pre-peritoneal (TAPP) vs totally extraperitoneal (TEP) laparoscopic techniques for inguinal hernia repair.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Botnet phenomenon in smartphones is evolving with the proliferation in mobile phone technologies after leaving imperative impact on personal computers. It refers to the network of computers, laptops, mobile devices or tablets which is remotely controlled by the cybercriminals to initiate various distributed coordinated attacks including spam emails, ad-click fraud, Bitcoin mining, Distributed Denial of Service (DDoS), disseminating other malwares and much more. Likewise traditional PC based botnet, Mobile botnets have the same operational impact except the target audience is particular to smartphone users. Therefore, it is import to uncover this security issue prior to its widespread adaptation. We propose SMARTbot, a novel dynamic analysis framework augmented with machine learning techniques to automatically detect botnet binaries from malicious corpus. SMARTbot is a component based off-device behavioral analysis framework which can generate mobile botnet learning model by inducing Artificial Neural Networks' back-propagation method. Moreover, this framework can detect mobile botnet binaries with remarkable accuracy even in case of obfuscated program code. The results conclude that, a classifier model based on simple logistic regression outperform other machine learning classifier for botnet apps' detection, i.e 99.49% accuracy is achieved. Further, from manual inspection of botnet dataset we have extracted interesting trends in those applications. As an outcome of this research, a mobile botnet dataset is devised which will become the benchmark for future studies. \",\n",
              "  'authors': ['Karim, Ahmad', 'Salleh, Rosli', 'Khan, Muhammad Khurram'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computer Security', 'Machine Learning'],\n",
              "  'pmid': '26978523',\n",
              "  'title': 'SMARTbot: A Behavioral Analysis Framework Augmented with Machine Learning to Identify Mobile Botnet Applications.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RX'],\n",
              "  'ESSN': '1931-7565',\n",
              "  'IF_2013': '3.385',\n",
              "  'IF_2014': '4.598',\n",
              "  'IF_2015': '3.667',\n",
              "  'IF_2016': '3.985',\n",
              "  'IF_2017': '3.719',\n",
              "  'ISSN': '1931-7557',\n",
              "  'ISSN_online': '1931-7565',\n",
              "  'ISSN_print': '1931-7557',\n",
              "  'IsoAbbr': 'Brain Imaging Behav',\n",
              "  'JrId': 34269,\n",
              "  'MedAbbr': 'Brain Imaging Behav',\n",
              "  'NLMid': '101300405',\n",
              "  'Titre': 'Brain Imaging Behav',\n",
              "  'abstract': \"Neuroimaging data has been widely used to predict clinical scores for automatic diagnosis of Alzheimer's disease (AD). For accurate clinical score prediction, one of the major challenges is high feature dimension of the imaging data. To address this issue, this paper presents an effective framework using a novel feature selection model via sparse learning. In contrast to previous approaches focusing on a single time point, this framework uses information at multiple time points. Specifically, a regularized correntropy with the spatial-temporal constraint is used to reduce the adverse effect of noise and outliers, and promote consistent and robust selection of features by exploring data characteristics. Furthermore, ensemble learning of support vector regression (SVR) is exploited to accurately predict AD scores based on the selected features. The proposed approach is extensively evaluated on the Alzheimer's disease neuroimaging initiative (ADNI) dataset. Our experiments demonstrate that the proposed approach not only achieves promising regression accuracy, but also successfully recognizes disease-related biomarkers.\",\n",
              "  'authors': ['Lei, Baiying',\n",
              "   'Hou, Wen',\n",
              "   'Zou, Wenbin',\n",
              "   'Li, Xia',\n",
              "   'Zhang, Cishen',\n",
              "   'Wang, Tianfu'],\n",
              "  'categories': ['NEUROIMAGING'],\n",
              "  'journal': 'Brain imaging and behavior',\n",
              "  'keywords': ['Alzheimer Disease',\n",
              "   'Brain',\n",
              "   'Cognitive Dysfunction',\n",
              "   'Cohort Studies',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Longitudinal Studies',\n",
              "   'Machine Learning',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Neuroimaging',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Spatio-Temporal Analysis'],\n",
              "  'pmid': '29582337',\n",
              "  'title': \"Longitudinal score prediction for Alzheimer's disease based on ensemble correntropy and spatial-temporal constraint.\",\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1471-2490',\n",
              "  'IF_2013': '1.937',\n",
              "  'IF_2014': '1.413',\n",
              "  'IF_2015': '1.606',\n",
              "  'IF_2016': '1.72',\n",
              "  'IF_2017': '1.792',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2490',\n",
              "  'IsoAbbr': 'BMC Urol',\n",
              "  'JrId': 22337,\n",
              "  'MedAbbr': 'BMC Urol',\n",
              "  'NLMid': '100968571',\n",
              "  'Titre': 'BMC Urol',\n",
              "  'abstract': 'Hem-o-lok clips are widely used during robot-assisted and laparoscopic radical prostatectomy to control the lateral pedicles. There are a few reports of hem-o-lok clip migration into the bladder or vesico-urethral anastomosis and only four cases of hem-o-lok clip migration resulting into bladder neck contracture. Herein, we describe the first case, to our knowledge, of hem-o-lok clip migration leading to severe bladder neck contracture and subsequent stress urinary incontinence.|A 62-year-old Caucasian man underwent robot-assisted laparoscopic radical prostatectomy for a T1c Gleason 8 prostate cancer. One month after surgery the patient was fully continent; however, three months later, he presented with acute urinary retention requiring suprapubic drainage. Urethroscopy showed a hem-o-lok clip strongly attached to the area between the vesico-urethral anastomosis and the urethral sphincter and a severe bladder neck contracture behind it. Following cold-knife urethral incision and clip removal, the bladder neck contracture was widely resected. At 3-month follow-up, the patient voided spontaneously with a peak flow rate of 9.5 ml/sec and absence of post-void residual urine, but leaked 240 ml urine at the 24-hour pad test. To date, at 1-year follow-up, his voiding situation remains unchanged.|The present report provides further evidence for the risk of hem-o-lok clip migration causing bladder neck contracture, and is the first to demonstrate the potential of such complication to result into stress urinary incontinence.',\n",
              "  'authors': ['Cormio, Luigi',\n",
              "   'Massenio, Paolo',\n",
              "   'Lucarelli, Giuseppe',\n",
              "   'Di Fino, Giuseppe',\n",
              "   'Selvaggio, Oscar',\n",
              "   'Micali, Salvatore',\n",
              "   'Carrieri, Giuseppe'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BMC urology',\n",
              "  'keywords': ['Contracture',\n",
              "   'Device Removal',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Surgical Instruments',\n",
              "   'Treatment Outcome',\n",
              "   'Urinary Bladder',\n",
              "   'Urinary Incontinence'],\n",
              "  'pmid': '24555468',\n",
              "  'title': 'Hem-o-lok clip: a neglected cause of severe bladder neck contracture and consequent urinary incontinence after robot-assisted laparoscopic radical prostatectomy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1938-3797',\n",
              "  'IF_2013': '0.786',\n",
              "  'IF_2014': '0.913',\n",
              "  'IF_2015': '0.814',\n",
              "  'IF_2016': '1.256',\n",
              "  'IF_2017': '1.298',\n",
              "  'ISSN': '1086-8089',\n",
              "  'ISSN_online': '1938-3797',\n",
              "  'ISSN_print': '1086-8089',\n",
              "  'IsoAbbr': 'JSLS',\n",
              "  'JrId': 21218,\n",
              "  'MedAbbr': 'JSLS',\n",
              "  'NLMid': '100884618',\n",
              "  'Titre': 'JSLS',\n",
              "  'abstract': 'Cervical insufficiency is a difficult condition to diagnose and can lead to preterm birth, miscarriage, or perinatal infant morbidity and mortality. We conducted this retrospective case study and literature review to evaluate the safety and efficacy of robot-assisted abdominal cerclage during pregnancy.|We conducted a case series and a systematic review that included patients who underwent robot-assisted abdominal cerclage during pregnancy from January 2010 through March 2016.|Six patients met the criteria for the case series. Median age was 34 years (range, 28-37) at the time of the procedure. In 5 cases, the indication for transabdominal cerclage was a failed vaginal cerclage in a previous pregnancy, whereas a scarred and shortened cervix caused by a previous dilatation and curettage-induced cervical laceration was the indication in the remaining case. Median operating time was 159.5 minutes (range, 124-204), and median estimated blood loss was 25 mL (range, 10-25). No surgeries were converted to laparotomies; all patients were discharged on postoperative day 1. The median gestational age at delivery was 37.5 weeks (range, 22-39). Five patients delivered between 36 and 39 weeks. No patients had chorioamnionitis or preterm premature rupture of membranes. One patient went into preterm labor at 22 weeks, and the cerclage was removed via minilaparotomy. Eight articles met the criteria for systematic review. Sixteen patients underwent robot-assisted abdominal cerclage during pregnancy. Median age was 31.5 years (range, 25-37). The major indication in most articles was previous failed transvaginal cerclage. The median gestational ages at time of procedure and delivery were 12 weeks (range, 10-15) and 37 weeks (range, 33-39), respectively.|Robot-assisted abdominal cerclage is safe and effective during pregnancy.',\n",
              "  'authors': ['Zeybek, Burak',\n",
              "   'Hill, Amanda',\n",
              "   'Menderes, Gulden',\n",
              "   'Borahay, Mostafa A',\n",
              "   'Azodi, Masoud',\n",
              "   'Kilic, Gokhan Sami'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'JSLS : Journal of the Society of Laparoendoscopic Surgeons',\n",
              "  'keywords': ['Adult',\n",
              "   'Cerclage, Cervical',\n",
              "   'Female',\n",
              "   'Gestational Age',\n",
              "   'Humans',\n",
              "   'Infant, Newborn',\n",
              "   'Pregnancy',\n",
              "   'Pregnancy Complications',\n",
              "   'Pregnancy Outcome',\n",
              "   'Premature Birth',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Uterine Cervical Incompetence'],\n",
              "  'pmid': '27904309',\n",
              "  'title': 'Robot-Assisted Abdominal Cerclage During Pregnancy.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1875-8908',\n",
              "  'IF_2013': '3.612',\n",
              "  'IF_2014': '4.151',\n",
              "  'IF_2015': '3.92',\n",
              "  'IF_2016': '3.731',\n",
              "  'IF_2017': '3.476',\n",
              "  'ISSN': '1387-2877',\n",
              "  'ISSN_online': '1875-8908',\n",
              "  'ISSN_print': '1387-2877',\n",
              "  'IsoAbbr': 'J. Alzheimers Dis.',\n",
              "  'JrId': 21942,\n",
              "  'MedAbbr': 'J Alzheimers Dis',\n",
              "  'NLMid': '9814863',\n",
              "  'Titre': 'J. Alzheimers Dis.',\n",
              "  'abstract': \"The hipppocampus is a brain structure located deep in the temporal lobe and is notable for its susceptibility to neurotoxic stimuli. It plays a vital role in governing learning and memory. It has been shown to be damaged by variety of factors, e.g., hypoxia, hypoperfusion, hypoglycemia, stress, and seizures, and the list of such factors keeps growing with time. Recently, the role of vascular factors in Alzheimer's disease in causing significant hippocampal damage has started emerging. Vascular factors are known to cause cerebral microlesions and contribute to many more brain pathologies. Though evidence supporting their effects on causing regional brain atrophy is mixed, several studies are indicating that medial temporal lobe may be particularly vulnerable to the damage caused by cardiovascular risk factors. Considering the association between neurodegeneration and vascular factors, a more rigorous scientific evaluation of the correlation between these two has been suggested.\",\n",
              "  'authors': ['Dhikav, Vikas', 'Anand, Kuljeet Singh'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': \"Journal of Alzheimer's disease : JAD\",\n",
              "  'keywords': ['Alzheimer Disease',\n",
              "   'Animals',\n",
              "   'Atrophy',\n",
              "   'Cardiovascular Diseases',\n",
              "   'Diabetes Mellitus',\n",
              "   'Hippocampus',\n",
              "   'Humans',\n",
              "   'Hypertension',\n",
              "   'Risk Factors',\n",
              "   'Vascular Diseases'],\n",
              "  'pmid': '22850312',\n",
              "  'title': \"Are vascular factors linked to the development of hippocampal atrophy in Alzheimer's disease?\",\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '1556-3669',\n",
              "  'IF_2013': '1.544',\n",
              "  'IF_2014': '1.668',\n",
              "  'IF_2015': '1.791',\n",
              "  'IF_2016': '2.031',\n",
              "  'IF_2017': '2.165',\n",
              "  'ISSN': '1530-5627',\n",
              "  'ISSN_online': '1556-3669',\n",
              "  'ISSN_print': '1530-5627',\n",
              "  'IsoAbbr': 'Telemed J E Health',\n",
              "  'JrId': 21916,\n",
              "  'MedAbbr': 'Telemed J E Health',\n",
              "  'NLMid': '100959949',\n",
              "  'Titre': 'Telemed J E Health',\n",
              "  'abstract': '\"Telepresence\" is an interesting field that includes virtual reality implementations with human-system interfaces, communication technologies, and robotics. This paper describes the development of a telepresence robot called Telepresence Robot for Interpersonal Communication (TRIC) for the purpose of interpersonal communication with the elderly in a home environment. The main aim behind TRIC\\'s development is to allow elderly populations to remain in their home environments, while loved ones and caregivers are able to maintain a higher level of communication and monitoring than via traditional methods. TRIC aims to be a low-cost, lightweight robot, which can be easily implemented in the home environment. Under this goal, decisions on the design elements included are discussed. In particular, the implementation of key autonomous behaviors in TRIC to increase the user\\'s capability of projection of self and operation of the telepresence robot, in addition to increasing the interactive capability of the participant as a dialogist are emphasized. The technical development and integration of the modules in TRIC, as well as human factors considerations are then described. Preliminary functional tests show that new users were able to effectively navigate TRIC and easily locate visual targets. Finally the future developments of TRIC, especially the possibility of using TRIC for home tele-health monitoring and tele-homecare visits are discussed.',\n",
              "  'authors': ['Tsai, Tzung-Cheng',\n",
              "   'Hsu, Yeh-Liang',\n",
              "   'Ma, An-I',\n",
              "   'King, Trevor',\n",
              "   'Wu, Chang-Huei'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Telemedicine journal and e-health : the official journal of the American Telemedicine Association',\n",
              "  'keywords': ['Aged',\n",
              "   'Communication',\n",
              "   'Equipment Design',\n",
              "   'Ergonomics',\n",
              "   'Health Services for the Aged',\n",
              "   'Home Care Services',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Telemedicine',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '17848109',\n",
              "  'title': 'Developing a telepresence robot for interpersonal communication with the elderly in a home environment.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1536-4844',\n",
              "  'IF_2013': '5.475',\n",
              "  'IF_2014': '4.464',\n",
              "  'IF_2015': '4.358',\n",
              "  'IF_2016': '4.525',\n",
              "  'IF_2017': '4.347',\n",
              "  'ISSN': '1078-0998',\n",
              "  'ISSN_online': '1536-4844',\n",
              "  'ISSN_print': '1078-0998',\n",
              "  'IsoAbbr': 'Inflamm. Bowel Dis.',\n",
              "  'JrId': 20933,\n",
              "  'MedAbbr': 'Inflamm Bowel Dis',\n",
              "  'NLMid': '9508162',\n",
              "  'Titre': 'Inflamm. Bowel Dis.',\n",
              "  'abstract': 'The gut-selective nature of vedolizumab has raised questions regarding increased joint pain or arthralgia with its use in inflammatory bowel disease (IBD) patients. As arthralgias are seldom coded and thus difficult to study, few studies have examined the comparative risk of arthralgia between vedolizumab and tumor necrosis factor inhibitor (TNFi). Our objectives were to evaluate the application of natural language processing (NLP) to identify arthralgia in the clinical notes and to compare the risk of arthralgia between vedolizumab and TNFi in IBD.|We performed a retrospective study using a validated electronic medical record (EMR)-based IBD cohort from 2 large tertiary care centers. The index date was the first date of vedolizumab or TNFi prescription. Baseline covariates were assessed 1 year before the index date; patients were followed 1 year after the index date. The primary outcome was arthralgia, defined using NLP. Using inverse probability of treatment weight to balance the cohorts, we then constructed Cox regression models to calculate the hazard ratio (HR) for arthralgia in the vedolizumab and TNFi groups.|We studied 367 IBD patients on vedolizumab and 1218 IBD patients on TNFi. Patients on vedolizumab were older (mean age, 41.2 vs 34.9 years) and had more prevalent use of immunomodulators (52.3% vs 31.9%) than TNFi users. Our data did not observe a significantly increased risk of arthralgia in the vedolizumab group compared with TNFi (HR, 1.20; 95% confidence interval, 0.97-1.49).|In this large observational study, we did not find a significantly increased risk of arthralgia associated with vedolizumab use compared with TNFi.',\n",
              "  'authors': ['Cai, Tianrun',\n",
              "   'Lin, Tzu-Chieh',\n",
              "   'Bond, Allison',\n",
              "   'Huang, Jie',\n",
              "   'Kane-Wanger, Gwendolyn',\n",
              "   'Cagan, Andrew',\n",
              "   'Murphy, Shawn N',\n",
              "   'Ananthakrishnan, Ashwin N',\n",
              "   'Liao, Katherine P'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'Inflammatory bowel diseases',\n",
              "  'keywords': ['Adult',\n",
              "   'Antibodies, Monoclonal, Humanized',\n",
              "   'Arthralgia',\n",
              "   'Electronic Health Records',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Gastrointestinal Agents',\n",
              "   'Humans',\n",
              "   'Immunosuppressive Agents',\n",
              "   'Inflammatory Bowel Diseases',\n",
              "   'Male',\n",
              "   'Natural Language Processing',\n",
              "   'Prognosis',\n",
              "   'Retrospective Studies',\n",
              "   'Tumor Necrosis Factor-alpha'],\n",
              "  'pmid': '29846617',\n",
              "  'title': 'The Association Between Arthralgia and Vedolizumab Using Natural Language Processing.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JY'],\n",
              "  'ESSN': '1750-3841',\n",
              "  'IF_2013': '1.791',\n",
              "  'IF_2014': '1.696',\n",
              "  'IF_2015': '1.649',\n",
              "  'IF_2016': '1.815',\n",
              "  'IF_2017': '2.018',\n",
              "  'ISSN': '0022-1147',\n",
              "  'ISSN_online': '1750-3841',\n",
              "  'ISSN_print': '0022-1147',\n",
              "  'IsoAbbr': 'J. Food Sci.',\n",
              "  'JrId': 20961,\n",
              "  'MedAbbr': 'J Food Sci',\n",
              "  'NLMid': '0014052',\n",
              "  'Titre': 'J. Food Sci.',\n",
              "  'abstract': 'Monitoring the level of tert-butylhydroquinone (TBHQ), a permissible antioxidant additive in edible vegetable oils in many countries, is important to ensure that oils and products that contain them comply with the relevant import regulations. Surface-enhanced Raman spectroscopy (SERS) technology coupled with chemometric methods including partial least squares (PLS) and support vector machine (SVM) regression was applied to determine levels of TBHQ in spiked corn oils (0 to 500 mg/kg, n = 40) and commercial vegetable oils (0 to 99.7 mg/kg, n = 25). The lowest detectable concentration was 5 mg/L for TBHQ in standard solutions and 10 mg/kg of TBHQ in vegetable oils from various plant sources. The TBHQ levels predicted by the PLS or SVM model had a high correlation with actual TBHQ levels in commercial oil samples (SVM: R(2) = 0.972, ratio of performance to deviation [RPD] = 5.55, root mean square error [RMSE] = 5.73 mg/kg; PLS: R(2) = 0.976, RPD = 6.43, RMSE = 4.94 mg/kg), indicating great potential of SERS methods for detection and quantification of TBHQ in oils from a variety of sources.|This study provides an alternative approach for analysis of trace amounts of chemicals in complex food matrices with surface-enhanced Raman spectroscopy.',\n",
              "  'authors': ['Pan, Yun',\n",
              "   'Lai, Keqiang',\n",
              "   'Fan, Yuxia',\n",
              "   'Li, Chunying',\n",
              "   'Pei, Lu',\n",
              "   'Rasco, Barbara A',\n",
              "   'Huang, Yiqun'],\n",
              "  'categories': ['FOOD SCIENCE & TECHNOLOGY'],\n",
              "  'journal': 'Journal of food science',\n",
              "  'keywords': ['Antioxidants',\n",
              "   'Humans',\n",
              "   'Hydroquinones',\n",
              "   'Least-Squares Analysis',\n",
              "   'Plant Oils',\n",
              "   'Spectrum Analysis, Raman',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24784825',\n",
              "  'title': 'Determination of tert-butylhydroquinone in vegetable oils using surface-enhanced Raman spectroscopy.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DQ'],\n",
              "  'ESSN': '1572-8595',\n",
              "  'IF_2013': '1.552',\n",
              "  'IF_2014': '1.575',\n",
              "  'IF_2015': '1.676',\n",
              "  'IF_2016': '1.826',\n",
              "  'IF_2017': '1.522',\n",
              "  'ISSN': '1383-875X',\n",
              "  'ISSN_online': '1572-8595',\n",
              "  'ISSN_print': '1383-875X',\n",
              "  'IsoAbbr': 'J Interv Card Electrophysiol',\n",
              "  'JrId': 21155,\n",
              "  'MedAbbr': 'J Interv Card Electrophysiol',\n",
              "  'NLMid': '9708966',\n",
              "  'Titre': 'J Interv Card Electrophysiol',\n",
              "  'abstract': 'Using conventional catheters, ablation of concealed parahisian accessory pathways may be difficult and high risk for heart block.|We describe the case of a concealed parahisian accessory pathway with three prior attempts to ablate using conventional methods (RF and cryotherapy). Using a remote magnetic navigation system, successful ablation occurred following a single RF lesion (total fluoroscopy time 17 min). In contrast to previous attempts, the patient remained asymptomatic during follow-up (12 months).|This is the first report of successful remote magnetic catheter ablation of a concealed parahisian AP. Magnetic catheter stability during RF application likely contributed to the success of this procedure and may have minimized the risk of AV block.',\n",
              "  'authors': ['Davis, Darryl R',\n",
              "   'Tang, Anthony S',\n",
              "   'Birnie, David H',\n",
              "   'Gollob, Michael H'],\n",
              "  'categories': ['CARDIAC & CARDIOVASCULAR SYSTEMS'],\n",
              "  'journal': 'Journal of interventional cardiac electrophysiology : an international journal of arrhythmias and pacing',\n",
              "  'keywords': ['Catheter Ablation',\n",
              "   'Cryotherapy',\n",
              "   'Female',\n",
              "   'Heart Conduction System',\n",
              "   'Humans',\n",
              "   'Magnetics',\n",
              "   'Middle Aged',\n",
              "   'Retreatment',\n",
              "   'Robotics',\n",
              "   'Treatment Failure'],\n",
              "  'pmid': '17109208',\n",
              "  'title': 'Successful ablation of a concealed parahisian accessory pathway using a remote magnetic navigation system following failure by conventional methods.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.905',\n",
              "  'IF_2014': '0.982',\n",
              "  'IF_2015': '1.066',\n",
              "  'IF_2016': '1.141',\n",
              "  'IF_2017': '1.352',\n",
              "  'ISSN': '1195-9479',\n",
              "  'ISSN_print': '1195-9479',\n",
              "  'IsoAbbr': 'Can J Urol',\n",
              "  'JrId': 21917,\n",
              "  'MedAbbr': 'Can J Urol',\n",
              "  'NLMid': '9515842',\n",
              "  'Titre': 'Can J Urol',\n",
              "  'abstract': 'Although malformations of the genitourinary tract are typically identified during childhood, they can remain silent until incidental detection in evaluation and treatment of other pathologies during adulthood. The advent of the minimally invasive era in urologic surgery has given rise to unique challenges in the surgical management of anomalies of the genitourinary tract.|This article reviews the embryology of anomalies of Wolffian duct (WD) derivatives with specific attention to the seminal vesicles, vas deferens, ureter, and kidneys. This is followed by a discussion of the history of the laparoscopic approach to WD derivative anomalies. Finally, we present two cases to describe technical considerations when managing these anomalies when encountered during robotic-assisted radical prostatectomy.|The University of Chicago Robotic Laparoscopic Radical Prostatectomy (RLRP) database was reviewed for cases where anomalies of WD derivatives were encountered. We describe how modifications in technique allowed for completion of the procedure without difficulty.|None|Of the 1230 RLRP procedures performed at our institution by three surgeons, only two cases (0.16%) have been noted to have a WD anomaly. These cases were able to be completed without difficulty by making simple modifications in technique.|Although uncommon, it is important for the urologist to be familiar with the origin and surgical management of WD anomalies, particularly when detected incidentally during surgery. Simple modifications in technique allow for completion of RLRP without difficulty.',\n",
              "  'authors': ['Acharya, Sujeet S',\n",
              "   'Gundeti, Mohan S',\n",
              "   'Zagaja, Gregory P',\n",
              "   'Shalhav, Arieh L',\n",
              "   'Zorn, Kevin C'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Canadian journal of urology',\n",
              "  'keywords': ['Adenocarcinoma',\n",
              "   'Aged',\n",
              "   'Humans',\n",
              "   'Kidney',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostate-Specific Antigen',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Seminal Vesicles',\n",
              "   'Ureter',\n",
              "   'Vas Deferens',\n",
              "   'Wolffian Ducts'],\n",
              "  'pmid': '19364437',\n",
              "  'title': 'Wolffian duct derivative anomalies: technical considerations when encountered during robot-assisted radical prostatectomy.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'High-utility sequential pattern mining (HUSPM) has become an important issue in the field of data mining. Several HUSPM algorithms have been designed to mine high-utility sequential patterns (HUPSPs). They have been applied in several real-life situations such as for consumer behavior analysis and event detection in sensor networks. Nonetheless, most studies on HUSPM have focused on mining HUPSPs in precise data. But in real-life, uncertainty is an important factor as data is collected using various types of sensors that are more or less accurate. Hence, data collected in a real-life database can be annotated with existing probabilities. This paper presents a novel pattern mining framework called high utility-probability sequential pattern mining (HUPSPM) for mining high utility-probability sequential patterns (HUPSPs) in uncertain sequence databases. A baseline algorithm with three optional pruning strategies is presented to mine HUPSPs. Moroever, to speed up the mining process, a projection mechanism is designed to create a database projection for each processed sequence, which is smaller than the original database. Thus, the number of unpromising candidates can be greatly reduced, as well as the execution time for mining HUPSPs. Substantial experiments both on real-life and synthetic datasets show that the designed algorithm performs well in terms of runtime, number of candidates, memory usage, and scalability for different minimum utility and minimum probability thresholds.',\n",
              "  'authors': ['Zhang, Binbin',\n",
              "   'Lin, Jerry Chun-Wei',\n",
              "   'Fournier-Viger, Philippe',\n",
              "   'Li, Ting'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Data Mining',\n",
              "   'Databases, Factual',\n",
              "   'Knowledge Bases',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Probability',\n",
              "   'Uncertainty'],\n",
              "  'pmid': '28742847',\n",
              "  'title': 'Mining of high utility-probability sequential patterns from uncertain databases.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'This paper has reviewed, with somewhat variable coverage, the nine MR image segmentation techniques itemized in Table II. A wide array of approaches have been discussed; each has its merits and drawbacks. We have also given pointers to other approaches not discussed in depth in this review. The methods reviewed fall roughly into four model groups: c-means, maximum likelihood, neural networks, and k-nearest neighbor rules. Both supervised and unsupervised schemes require human intervention to obtain clinically useful results in MR segmentation. Unsupervised techniques require somewhat less interaction on a per patient/image basis. Maximum likelihood techniques have had some success, but are very susceptible to the choice of training region, which may need to be chosen slice by slice for even one patient. Generally, techniques that must assume an underlying statistical distribution of the data (such as LML and UML) do not appear promising, since tissue regions of interest do not usually obey the distributional tendencies of probability density functions. The most promising supervised techniques reviewed seem to be FF/NN methods that allow hidden layers to be configured as examples are presented to the system. An example of a self-configuring network, FF/CC, was also discussed. The relatively simple k-nearest neighbor rule algorithms (hard and fuzzy) have also shown promise in the supervised category. Unsupervised techniques based upon fuzzy c-means clustering algorithms have also shown great promise in MR image segmentation. Several unsupervised connectionist techniques have recently been experimented with on MR images of the brain and have provided promising initial results. A pixel-intensity-based edge detection algorithm has recently been used to provide promising segmentations of the brain. This is also an unsupervised technique, older versions of which have been susceptible to oversegmenting the image because of the lack of clear boundaries between tissue types or finding uninteresting boundaries between slightly different types of the same tissue. To conclude, we offer some remarks about improving MR segmentation techniques. The better unsupervised techniques are too slow. Improving speed via parallelization and optimization will improve their competitiveness with, e.g., the k-nn rule, which is the fastest technique covered in this review. Another area for development is dynamic cluster validity. Unsupervised methods need better ways to specify and adjust c, the number of tissue classes found by the algorithm. Initialization is a third important area of research. Many of the schemes listed in Table II are sensitive to good initialization, both in terms of the parameters of the design, as well as operator selection of training data.(ABSTRACT TRUNCATED AT 400 WORDS)',\n",
              "  'authors': ['Bezdek, J C', 'Hall, L O', 'Clarke, L P'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Bayes Theorem',\n",
              "   'Biophysical Phenomena',\n",
              "   'Biophysics',\n",
              "   'Cluster Analysis',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated'],\n",
              "  'pmid': '8413011',\n",
              "  'title': 'Review of MR image segmentation techniques using pattern recognition.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1618-727X',\n",
              "  'IF_2013': '1.2',\n",
              "  'IF_2014': '1.19',\n",
              "  'IF_2015': '1.406',\n",
              "  'IF_2016': '1.407',\n",
              "  'IF_2017': '1.536',\n",
              "  'ISSN': '0897-1889',\n",
              "  'ISSN_online': '1618-727X',\n",
              "  'ISSN_print': '0897-1889',\n",
              "  'IsoAbbr': 'J Digit Imaging',\n",
              "  'JrId': 1047,\n",
              "  'MedAbbr': 'J Digit Imaging',\n",
              "  'NLMid': '9100529',\n",
              "  'Titre': 'J Digit Imaging',\n",
              "  'abstract': 'In this paper, a new neural network model inspired by the biological immune system functions is presented. The model, termed Artificial Immune-Activated Neural Network (AIANN), extracts classification knowledge from a training data set, which is then used to classify input patterns or vectors. The AIANN is based on a neuron activation function whose behavior is conceptually modeled after the chemical bonds between the receptors and epitopes in the biological immune system. The bonding is controlled through an energy measure to ensure accurate recognition. The AIANN model was applied to the segmentation of 3-dimensional magnetic resonance imaging (MRI) data of the brain and a contextual basis was developed for the segmentation problem. Evaluation of the segmentation results was performed using both real MRI data obtained from the Center for Morphometric Analysis at Massachusetts General Hospital and simulated MRI data generated using the McGill University BrainWeb MRI simulator. Experimental results demonstrated that the AIANN model attained higher average results than those obtained using published methods for real MRI data and simulated MRI data, especially at low levels of noise.',\n",
              "  'authors': ['Younis, Akmal',\n",
              "   'Ibrahim, Mohamed',\n",
              "   'Kabuka, Mansur',\n",
              "   'John, Nigel'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of digital imaging',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Immune System',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '18071820',\n",
              "  'title': 'An artificial immune-activated neural network applied to brain 3D MRI segmentation.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1558-1756',\n",
              "  'IF_2013': '1.116',\n",
              "  'IF_2014': '0.911',\n",
              "  'IF_2015': '1.203',\n",
              "  'IF_2016': '1.987',\n",
              "  'IF_2017': '1.64',\n",
              "  'ISSN': '0272-1716',\n",
              "  'ISSN_online': '1558-1756',\n",
              "  'ISSN_print': '0272-1716',\n",
              "  'IsoAbbr': 'IEEE Comput Graph Appl',\n",
              "  'JrId': 31993,\n",
              "  'MedAbbr': 'IEEE Comput Graph Appl',\n",
              "  'NLMid': '9881869',\n",
              "  'Titre': 'IEEE Comput Graph Appl',\n",
              "  'abstract': \"The Beaming project recreates, virtually, a real environment; using immersive VR, remote participants can visit the virtual model and interact with the people in the real environment. The real environment doesn't need extensive equipment and can be a space such as an office or meeting room, domestic environment, or social space. \",\n",
              "  'authors': ['Steed, Anthony',\n",
              "   'Steptoe, William',\n",
              "   'Oyekoya, Wole',\n",
              "   'Pece, Fabrizio',\n",
              "   'Weyrich, Tim',\n",
              "   'Kautz, Jan',\n",
              "   'Friedman, Doron',\n",
              "   'Peer, Angelika',\n",
              "   'Solazzi, Massimiliano',\n",
              "   'Tecchia, Franco',\n",
              "   'Bergamasco, Massimo',\n",
              "   'Slater, Mel'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE computer graphics and applications',\n",
              "  'keywords': ['Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Robotics',\n",
              "   'User-Computer Interface',\n",
              "   'Videoconferencing'],\n",
              "  'pmid': '24807305',\n",
              "  'title': 'Beaming: an asymmetric telepresence system.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '2160-1836',\n",
              "  'IF_2013': '2.511',\n",
              "  'IF_2014': '3.198',\n",
              "  'IF_2015': '2.91',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '2.742',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2160-1836',\n",
              "  'IsoAbbr': 'G3 (Bethesda)',\n",
              "  'JrId': 39472,\n",
              "  'MedAbbr': 'G3 (Bethesda)',\n",
              "  'NLMid': '101566598',\n",
              "  'Titre': 'G3 (Bethesda)',\n",
              "  'abstract': 'The veined rapa whelk (Rapana venosa) is widely consumed in China. Nevertheless, it preys on oceanic bivalves, thereby reducing this resource worldwide. Its larval metamorphosis comprises a transition from pelagic to benthic form, which involves considerable physiological and structural changes and has vital roles in its natural populations and commercial breeding. Thus, understanding the endogenous microRNAs (miRNAs) that drive metamorphosis is of great interest. This is the first study to use high-throughput sequencing to examine the alterations in miRNA expression that occur during metamorphosis in a marine gastropod. A total of 195 differentially expressed miRNAs were obtained. Sixty-five of these were expressed during the transition from precompetent to competent larvae. Thirty-three of these were upregulated and the others were downregulated. Another 123 miRNAs were expressed during the transition from competent to postlarvae. Ninety-six of these were upregulated and the remaining 27 were downregulated. The expression of miR-276-y, miR-100-x, miR-183-x, and miR-263-x showed a >100-fold change during development, while the miR-242-x and novel-m0052-3p expression levels changed over 3000-fold. Putative target gene coexpression, gene ontology, and pathway analyses suggest that these miRNAs play important parts in cell proliferation, migration, apoptosis, metabolic regulation, and energy absorption. Twenty miRNAs and their target genes involved in ingestion, digestion, cytoskeleton, cell adhesion, and apoptosis were identified. Nine of them were analyzed with real-time polymerase chain reaction (PCR), which showed an inverse correlation between the miRNAs and their relative expression levels. Our data elucidate the role of miRNAs in R. venosa metamorphic transition and serve as a solid basis for further investigations into regulatory mechanisms of gastropod metamorphosis.',\n",
              "  'authors': ['Song, Hao', 'Qi, Lu', 'Zhang, Tao', 'Wang, Hai-Yan'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'G3 (Bethesda, Md.)',\n",
              "  'keywords': ['Animals',\n",
              "   'China',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Larva',\n",
              "   'Metamorphosis, Biological',\n",
              "   'MicroRNAs',\n",
              "   'Primulaceae',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '29079680',\n",
              "  'title': 'Understanding microRNA Regulation Involved in the Metamorphosis of the Veined Rapa Whelk (Rapana venosa).',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.197',\n",
              "  'IF_2014': '1.56',\n",
              "  'IF_2015': '1.76',\n",
              "  'IF_2016': '1.751',\n",
              "  'IF_2017': '1.666',\n",
              "  'ISSN': '0959-3330',\n",
              "  'ISSN_online': '1479-487X',\n",
              "  'ISSN_print': '0959-3330',\n",
              "  'IsoAbbr': 'Environ Technol',\n",
              "  'JrId': 22034,\n",
              "  'MedAbbr': 'Environ Technol',\n",
              "  'NLMid': '9884939',\n",
              "  'Titre': 'Environ Technol',\n",
              "  'abstract': 'Wastewater containing high concentration of oxygen-demanding carbonaceous organics and nitrogenous materials (chemical oxygen demand (COD) and total Kjeldahl nitrogen (TKN)) as nutrients emanated from small- to large-scale slaughterhouse units cause depletion of dissolved oxygen in water bodies and attributes to the threat of eutrophication. Biological treatment of wastewater is a useful tool through ages for the treatment of wastewater owing to its cost-effectiveness, reliability along with its innocuous output features. This paper deals with the treatment of slaughter house wastewater by conducting a laboratory scale batch reactor with different input characterized samples, and the experimental results were explored for the formulation of feed-forward back-propagation artificial neural network (ANN) to predict the combined removal of COD and TKN. The ANN modelling was carried out using neural network tool box of MATLAB (version 7.0), with the Levenberg-Marquardt training algorithm. Various trials were examined for the training of the ANN model using the number of neurons in the hidden layer varying from 2 to 30. The mean square error function and regression analysis were also applied for performance analysis of the ANN model. All the input data were logged-in after carrying out detailed experiment in the laboratory with a view to examine the performance of the batch reactor for the treatment of slaughterhouse wastewater. The experimental results were used for testing and validating the ANN model.',\n",
              "  'authors': ['Kundu, Pradyut',\n",
              "   'Debsarkar, Anupam',\n",
              "   'Mukherjee, Somnath',\n",
              "   'Kumar, Sunil'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental technology',\n",
              "  'keywords': ['Abattoirs',\n",
              "   'Bioreactors',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Nitrification',\n",
              "   'Nitrogen',\n",
              "   'Organic Chemicals',\n",
              "   'Oxidation-Reduction',\n",
              "   'Waste Water',\n",
              "   'Water Purification'],\n",
              "  'pmid': '24701927',\n",
              "  'title': 'Artificial neural network modelling in biological removal of organic carbon and nitrogen for the treatment of slaughterhouse wastewater in a batch reactor.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1873-3476',\n",
              "  'IF_2013': '3.785',\n",
              "  'IF_2014': '3.65',\n",
              "  'IF_2015': '3.994',\n",
              "  'IF_2016': '3.649',\n",
              "  'IF_2017': '3.862',\n",
              "  'ISSN': '0378-5173',\n",
              "  'ISSN_online': '1873-3476',\n",
              "  'ISSN_print': '0378-5173',\n",
              "  'IsoAbbr': 'Int J Pharm',\n",
              "  'JrId': 21201,\n",
              "  'MedAbbr': 'Int J Pharm',\n",
              "  'NLMid': '7804127',\n",
              "  'Titre': 'Int J Pharm',\n",
              "  'abstract': 'Although skin permeability of an active ingredient can be severely affected by its ionization in a dose solution, most of the existing prediction models cannot predict such impacts. To provide reliable predictors, we curated a novel large dataset of in vitro human skin permeability coefficients for 322 entries comprising chemically diverse permeants whose ionization fractions can be calculated. Subsequently, we generated thousands of computational descriptors, including LogD (octanol-water distribution coefficient at a specific pH), and analyzed the dataset using nonlinear support vector regression (SVR) and Gaussian process regression (GPR) combined with greedy descriptor selection. The SVR model was slightly superior to the GPR model, with externally validated squared correlation coefficient, root mean square error, and mean absolute error values of 0.94, 0.29, and 0.21, respectively. These models indicate that Log D is effective for a comprehensive prediction of ionization effects on skin permeability. In addition, the proposed models satisfied the statistical criteria endorsed in recent model validation studies. These models can evaluate virtually generated compounds at any pH; therefore, they can be used for high-throughput evaluations of numerous active ingredients and optimization of their skin permeability with respect to permeant ionization.',\n",
              "  'authors': ['Baba, Hiromi',\n",
              "   'Ueno, Yusuke',\n",
              "   'Hashida, Mitsuru',\n",
              "   'Yamashita, Fumiyoshi'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'International journal of pharmaceutics',\n",
              "  'keywords': ['Databases, Factual',\n",
              "   'Humans',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Ions',\n",
              "   'Least-Squares Analysis',\n",
              "   'Linear Models',\n",
              "   'Normal Distribution',\n",
              "   'Permeability',\n",
              "   'Predictive Value of Tests',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Skin Absorption',\n",
              "   'Solubility',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '28279739',\n",
              "  'title': 'Quantitative prediction of ionization effect on human skin permeability.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1545-4126',\n",
              "  'IF_2013': '22.66',\n",
              "  'IF_2014': '19.32',\n",
              "  'IF_2015': '14.265',\n",
              "  'IF_2016': '15.63',\n",
              "  'IF_2017': '14.675',\n",
              "  'ISSN': '0147-006X',\n",
              "  'ISSN_online': '1545-4126',\n",
              "  'ISSN_print': '0147-006X',\n",
              "  'IsoAbbr': 'Annu. Rev. Neurosci.',\n",
              "  'JrId': 635,\n",
              "  'MedAbbr': 'Annu Rev Neurosci',\n",
              "  'NLMid': '7804039',\n",
              "  'Titre': 'Annu. Rev. Neurosci.',\n",
              "  'abstract': 'The curse of dimensionality poses severe challenges to both technical and conceptual progress in neuroscience. In particular, it plagues our ability to acquire, process, and model high-dimensional data sets. Moreover, neural systems must cope with the challenge of processing data in high dimensions to learn and operate successfully within a complex world. We review recent mathematical advances that provide ways to combat dimensionality in specific situations. These advances shed light on two dual questions in neuroscience. First, how can we as neuroscientists rapidly acquire high-dimensional data from the brain and subsequently extract meaningful models from limited amounts of these data? And second, how do brains themselves process information in their intrinsically high-dimensional patterns of neural activity as well as learn meaningful, generalizable models of the external world from limited experience?',\n",
              "  'authors': ['Ganguli, Surya', 'Sompolinsky, Haim'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Annual review of neuroscience',\n",
              "  'keywords': ['Brain',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Memory, Short-Term',\n",
              "   'Mental Processes',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Nonlinear Dynamics'],\n",
              "  'pmid': '22483042',\n",
              "  'title': 'Compressed sensing, sparsity, and dimensionality in neuronal information processing and data analysis.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'In the last century, engineering advances have transformed the practice of surgery. Keyhole surgical techniques offer a number of advantages over traditional open approaches including less postoperative pain, fewer wound complications, and reduced length of stay in hospital. However, they also present considerable technical challenges, particularly to surgeons performing new operative approaches, such as those through natural orifices. Advances in surgical robots have improved surgical visualization, dexterity, and manipulation consistency, thus greatly enhancing surgical performance and patient care. Clinically, however, robotic surgery is still in its infancy, and its use has remained limited to relatively few operations. In the paper, we will discuss the economic-, clinical-, and research-related factors that may act as barriers to the widespread utilization and development of surgical robots. In overcoming these barriers through a synergistic effort of both engineering and medicine, we highlight our future vision of robotic surgery, in both the short and long term.',\n",
              "  'authors': ['Marcus, Hani',\n",
              "   'Nandi, Dipankar',\n",
              "   'Darzi, Ara',\n",
              "   'Yang, Guang-Zhong'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Humans',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '23372075',\n",
              "  'title': 'Surgical robotics through a keyhole: from today\\'s translational barriers to tomorrow\\'s \"disappearing\" robots.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'Robotic laparoscopic surgery has been shown to decrease task completion time, reduce errors, and decrease training time, as compared with manual laparoscopic surgery. However, current literature has not addressed the physiologic effects, in particular muscle responses, to training with a robotic surgical system. The authors seek to determine the frequency response of electromyographic (EMG) signals of specific arm and hand muscles with training using the da Vinci Surgical System.|Seven right-handed medical students were trained in three tasks with the da Vinci Surgical System over 4 weeks. These subjects, along with eight control subjects, were tested before and after training. Electromyographic (EMG) signals were collected from four arm and hand muscles during the testing sessions, and the median EMG frequency and bandwidth were computed.|The median frequency and frequency bandwidth both were increased after training for two of the three tasks.|The results suggest that training reduces muscle fatigue as a result of faster and more deliberate movements. These changes occurred predominantly in muscles that were the dominant muscles for each task, whereas the more demanding task recruited more diverse motor units. An evaluation of the physiologic demands of robotic laparoscopic surgery using electromyography can provide us with a meaningful quantitative way to examine performance and skill acquisition.',\n",
              "  'authors': ['Judkins, T N', 'Oleynikov, D', 'Narazaki, K', 'Stergiou, N'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Adult',\n",
              "   'Arm',\n",
              "   'Education, Medical',\n",
              "   'Electromyography',\n",
              "   'Female',\n",
              "   'General Surgery',\n",
              "   'Hand',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Lifting',\n",
              "   'Male',\n",
              "   'Muscle, Skeletal',\n",
              "   'Robotics',\n",
              "   'Suture Techniques'],\n",
              "  'pmid': '16508816',\n",
              "  'title': 'Robotic surgery and training: electromyographic correlates of robotic laparoscopic training.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'We demonstrate an analog type logical device that combines metalloprotein and organic/inorganic materials and can make an interactive analog decision. Myoglobin is used as a functional biomolecule to generate electrochemical signals, and its original redox signal is controlled with various mercapto-acids by the distance effect between myoglobin and a metal surface in the process of electron transfer. Controlled signals are modulated with the introduction of inorganic materials including nanoparticles and metal ions. By forming a hybrid structure with various constituents of organic/inorganic materials, several functions for signal manipulation were achieved, including enhancement, suppression, and shift. Based on the manipulated signals of biomolecules, a novel logical system for interactive decision-making processes is proposed by selectively combining different signals. Through the arrangement of various output signals, we can define interactive logical results regulated by an inherent tendency (by metalloprotein), personal experience (by organic spacer sets), and environments (by inorganic materials). As a practical application, a group decision process is presented using the proposed logical device. The proposed flexible logic process could facilitate the realization of an artificial intelligence system by mimicking the sophisticated human logic process. ',\n",
              "  'authors': ['Chung, Yong-Ho',\n",
              "   'Lee, Taek',\n",
              "   'Yoo, Si-Youl',\n",
              "   'Min, Junhong',\n",
              "   'Choi, Jeong-Woo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Biosensing Techniques',\n",
              "   'Decision Making',\n",
              "   'Electrochemistry',\n",
              "   'Electronics',\n",
              "   'Humans',\n",
              "   'Metalloproteins',\n",
              "   'Models, Theoretical'],\n",
              "  'pmid': '26400018',\n",
              "  'title': 'Electrochemical Bioelectronic Device Consisting of Metalloprotein for Analog Decision Making.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The 2007-2008 financial crisis solidified the consensus among policymakers that a macro-prudential approach to regulation and supervision should be adopted. The currently preferred policy option is the regulation of capital requirements, with the main focus on combating procyclicality and on identifying the banks that have a high systemic importance, those that are \"too big to fail\". Here we argue that the concept of systemic risk should include the analysis of the system as a whole and we explore systematically the most important properties for policy purposes of networks topology on resistance to shocks. In a thorough study going from analytical models to empirical data, we show two sharp transitions from safe to risky regimes: 1) diversification becomes harmful with just a small fraction (~2%) of the shocks sampled from a fat tailed shock distributions and 2) when large shocks are present a critical link density exists where an effective giant cluster forms and most firms become vulnerable. This threshold depends on the network topology, especially on modularity. Firm size heterogeneity has important but diverse effects that are heavily dependent on shock characteristics. Similarly, degree heterogeneity increases vulnerability only when shocks are directed at the most connected firms. Furthermore, by studying the structure of the core of the transnational corporation network from real data, we show that its stability could be clearly increased by removing some of the links with highest centrality betweenness. Our results provide a novel insight and arguments for policy makers to focus surveillance on the connections between firms, in addition to capital requirements directed at the nodes.',\n",
              "  'authors': ['Loepfe, Lasse', 'Cabrales, Antonio', 'Sánchez, Angel'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Humans',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Risk'],\n",
              "  'pmid': '24147017',\n",
              "  'title': 'Towards a proper assignment of systemic risk: the combined roles of network topology and shock characteristics.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Previously we reported that LIM1863 colorectal cancer (CRC) cells secrete three distinct extracellular vesicle subtypes - two subpopulations of exosomes (apical EpCAM-Exos and basolateral A33-Exos) and shed microvesicles (sMVs) - with distinct protein and miRNA signatures. Here, we extend our omics approach to understand the fundamental role of LIM1863-derived EVs by performing a comprehensive analysis of their mRNAs and long non-coding RNAs (lncRNAs) using RNA-Seq. We show that 2,389 mRNAs, 317 pseudogene transcripts, 1,028 lncRNAs and 206 short non-coding RNAs selectively distributed to (i.e., are enriched in) LIM1863 EVs, relative to the parent cell. An Ensembl/UniProtKB analysis revealed 1,937 mRNAs encode canonical proteins, 348 isoforms (including splice-variant proteins), and 119 'missing proteins' (i.e., annotated in Ensembl but not UniProtKB). Further dissection of our protein/RNA data revealed that 6/151 observed RNA binding proteins have the potential to interact with ~75% of EV-enriched RNAs. Intriguingly, the co-existence of U1 and U2 ribonucleoproteins and their cognate snRNAs in LIM1863 EVs suggests a possible association of CRC EVs with recipient cell splicing events. Our data reveal several potential lncRNA CRC biomarkers and novel splicing/fusion genes that, collectively, will advance our understanding of EV biology in CRC and accelerate the development of EV-based diagnostics and therapeutics.\",\n",
              "  'authors': ['Chen, Maoshan',\n",
              "   'Xu, Rong',\n",
              "   'Ji, Hong',\n",
              "   'Greening, David W',\n",
              "   'Rai, Alin',\n",
              "   'Izumikawa, Keiichi',\n",
              "   'Ishikawa, Hideaki',\n",
              "   'Takahashi, Nobuhiro',\n",
              "   'Simpson, Richard J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Biomarkers, Tumor',\n",
              "   'Cell Line, Tumor',\n",
              "   'Cell-Derived Microparticles',\n",
              "   'Colonic Neoplasms',\n",
              "   'Databases, Factual',\n",
              "   'Exosomes',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'MicroRNAs',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'RNA, Long Noncoding',\n",
              "   'RNA, Neoplasm',\n",
              "   'Ribonucleoprotein, U1 Small Nuclear',\n",
              "   'Ribonucleoprotein, U2 Small Nuclear',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27917920',\n",
              "  'title': 'Transcriptome and long noncoding RNA sequencing of three extracellular vesicle subtypes released from the human colon cancer LIM1863 cell line.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1527-1315',\n",
              "  'IF_2013': '6.214',\n",
              "  'IF_2014': '6.867',\n",
              "  'IF_2015': '6.798',\n",
              "  'IF_2016': '7.296',\n",
              "  'IF_2017': '7.469',\n",
              "  'ISSN': '0033-8419',\n",
              "  'ISSN_online': '1527-1315',\n",
              "  'ISSN_print': '0033-8419',\n",
              "  'IsoAbbr': 'Radiology',\n",
              "  'JrId': 6859,\n",
              "  'MedAbbr': 'Radiology',\n",
              "  'NLMid': '0401260',\n",
              "  'Titre': 'Radiology',\n",
              "  'abstract': 'To use structural magnetic resonance (MR) images to identify a pattern of regional atrophy characteristic of mild Alzheimer disease (AD) and to investigate whether presence of this pattern prospectively can aid prediction of 1-year clinical decline and increased structural loss in mild cognitive impairment (MCI).|The study was conducted with institutional review board approval and compliance with HIPAA regulations. Written informed consent was obtained from each participant. High-throughput volumetric segmentation and cortical surface reconstruction methods were applied to MR images from 84 subjects with mild AD, 175 with MCI, and 139 healthy control (HC) subjects. Stepwise linear discriminant analysis was used to identify regions that best can aid discrimination of HC subjects from subjects with AD. A classifier trained on data from HC subjects and those with AD was applied to data from subjects with MCI to determine whether presence of phenotypic AD atrophy at baseline was predictive of clinical decline and structural loss.|Atrophy in mesial and lateral temporal, isthmus cingulate, and orbitofrontal areas aided discrimination of HC subjects from subjects with AD, with fully cross-validated sensitivity of 83% and specificity of 93%. Subjects with MCI who had phenotypic AD atrophy showed significantly greater 1-year clinical decline and structural loss than those who did not and were more likely to have progression to probable AD (annual progression rate of 29% for subjects with MCI who had AD atrophy vs 8% for those who did not).|Semiautomated, individually specific quantitative MR imaging methods can be used to identify a pattern of regional atrophy in MCI that is predictive of clinical decline. Such information may aid in prediction of patient prognosis and increase the efficiency of clinical trials.',\n",
              "  'authors': ['McEvoy, Linda K',\n",
              "   'Fennema-Notestine, Christine',\n",
              "   'Roddey, J Cooper',\n",
              "   'Hagler, Donald J',\n",
              "   'Holland, Dominic',\n",
              "   'Karow, David S',\n",
              "   'Pung, Christopher J',\n",
              "   'Brewer, James B',\n",
              "   'Dale, Anders M'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Radiology',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Alzheimer Disease',\n",
              "   'Artificial Intelligence',\n",
              "   'Atrophy',\n",
              "   'Brain',\n",
              "   'Cognition Disorders',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '19201945',\n",
              "  'title': 'Alzheimer disease: quantitative structural neuroimaging for detection and prediction of clinical and structural changes in mild cognitive impairment.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1471-2962',\n",
              "  'IF_2013': '2.864',\n",
              "  'IF_2014': '2.147',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.97',\n",
              "  'IF_2017': '2.746',\n",
              "  'ISSN': '1364-503X',\n",
              "  'ISSN_online': '1471-2962',\n",
              "  'ISSN_print': '1364-503X',\n",
              "  'IsoAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'JrId': 30111,\n",
              "  'MedAbbr': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'NLMid': '101133385',\n",
              "  'Titre': 'Philos Trans A Math Phys Eng Sci',\n",
              "  'abstract': 'Deep convolutional networks provide state-of-the-art classifications and regressions results over many high-dimensional problems. We review their architecture, which scatters data with a cascade of linear filter weights and nonlinearities. A mathematical framework is introduced to analyse their properties. Computations of invariants involve multiscale contractions with wavelets, the linearization of hierarchical symmetries and sparse separations. Applications are discussed. ',\n",
              "  'authors': ['Mallat, Stéphane'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Philosophical transactions. Series A, Mathematical, physical, and engineering sciences',\n",
              "  'keywords': [''],\n",
              "  'pmid': '26953183',\n",
              "  'title': 'Understanding deep convolutional networks.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1432-1009',\n",
              "  'IF_2013': '1.648',\n",
              "  'IF_2014': '1.724',\n",
              "  'IF_2015': '1.857',\n",
              "  'IF_2016': '1.878',\n",
              "  'IF_2017': '2.177',\n",
              "  'ISSN': '0364-152X',\n",
              "  'ISSN_online': '1432-1009',\n",
              "  'ISSN_print': '0364-152X',\n",
              "  'IsoAbbr': 'Environ Manage',\n",
              "  'JrId': 20301,\n",
              "  'MedAbbr': 'Environ Manage',\n",
              "  'NLMid': '7703893',\n",
              "  'Titre': 'Environ Manage',\n",
              "  'abstract': 'The Kohonen neural network was applied to hydrochemical data from the Detritic Aquifer of the Lower Andarax, situated in a semiarid zone in the southeast of Spain. An activation map was obtained for each of the sampling points, in which the spatial distribution of the activated neurons indicated different water qualities. To extract the information contained in the activation maps, they were divided into nine quadrats. Cartesian coordinates were assigned to each quadrant ( x, y), and for each sampling point, three derived variables were selected, which were assigned the values x and y of the corresponding quadrat. A classification was defined based on this simple matrix system which allows an easy and rapid means of evaluating the water quality. This assessment highlights the various processes that affect groundwater quality. The method generates output that is easier to interpret than from traditional statistical methods. The information is extracted from the activation maps without significant loss of information. The method is proposed for assessing water quality in hydrogeochemically complex areas, where large numbers of observations are made.',\n",
              "  'authors': ['Sanchez-Martos, Francisco',\n",
              "   'Aguilera, Pedro A',\n",
              "   'Garrido-Frenich, Antonia',\n",
              "   'Torres, Jose A',\n",
              "   'Pulido-Bosch, Antonio'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental management',\n",
              "  'keywords': ['Desert Climate',\n",
              "   'Geographic Information Systems',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Quality Control',\n",
              "   'Soil',\n",
              "   'Water Pollutants',\n",
              "   'Water Supply'],\n",
              "  'pmid': '12375091',\n",
              "  'title': 'Assessment of groundwater quality by means of self-organizing maps: application in a semiarid area.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"The aircraft environmental control system (ECS) is a critical aircraft system, which provides the appropriate environmental conditions to ensure the safe transport of air passengers and equipment. The functionality and reliability of ECS have received increasing attention in recent years. The heat exchanger is a particularly significant component of the ECS, because its failure decreases the system's efficiency, which can lead to catastrophic consequences. Fault diagnosis of the heat exchanger is necessary to prevent risks. However, two problems hinder the implementation of the heat exchanger fault diagnosis in practice. First, the actual measured parameter of the heat exchanger cannot effectively reflect the fault occurrence, whereas the heat exchanger faults are usually depicted by utilizing the corresponding fault-related state parameters that cannot be measured directly. Second, both the traditional Extended Kalman Filter (EKF) and the EKF-based Double Model Filter have certain disadvantages, such as sensitivity to modeling errors and difficulties in selection of initialization values. To solve the aforementioned problems, this paper presents a fault-related parameter adaptive estimation method based on strong tracking filter (STF) and Modified Bayes classification algorithm for fault detection and failure mode classification of the heat exchanger, respectively. Heat exchanger fault simulation is conducted to generate fault data, through which the proposed methods are validated. The results demonstrate that the proposed methods are capable of providing accurate, stable, and rapid fault diagnosis of the heat exchanger. \",\n",
              "  'authors': ['Ma, Jian', 'Lu, Chen', 'Liu, Hongmei'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Aircraft',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Bayes Theorem',\n",
              "   'Computer Simulation',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Hot Temperature',\n",
              "   'Models, Theoretical',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '25823010',\n",
              "  'title': 'Fault diagnosis for the heat exchanger of the aircraft environmental control system based on the strong tracking filter.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'MicroRNAs (miRNAs) play a key role in gene expression and regulation in various organisms. They control a wide range of biological processes and are involved in several types of cancers by causing mRNA degradation or translational inhibition. However, the functions of most miRNAs and their precise regulatory mechanisms remain elusive. With the accumulation of the expression data of miRNAs and mRNAs, many computational methods have been proposed to predict miRNA-mRNA regulatory relationship. However, most existing methods require the number of modules predefined that may be difficult to determine beforehand. Here, we propose a novel computational method to discover miRNA-mRNA regulatory modules by combining Phase-only correlation and improved rough-Fuzzy Clustering (MIMPFC). The proposed method is evaluated on three heterogeneous datasets, and the obtained results are further validated through relevant literatures, biological significance and functional enrichment analysis. The analysis results show that the identified modules are highly correlated with the biological conditions. A large part of the regulatory relationships found by MIMPFC has been confirmed in the experimentally verified databases. It demonstrates that the modules found by MIMPFC are biologically significant.',\n",
              "  'authors': ['Luo, Dan', 'Wang, Shu-Lin', 'Fang, Jianwen', 'Zhang, Wei'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Breast Neoplasms',\n",
              "   'Cluster Analysis',\n",
              "   'Computational Biology',\n",
              "   'Databases, Genetic',\n",
              "   'Fuzzy Logic',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'MicroRNAs',\n",
              "   'RNA, Messenger',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '29281954',\n",
              "  'title': 'MIMPFC: Identifying miRNA-mRNA regulatory modules by combining phase-only correlation and improved rough-fuzzy clustering.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1573-2959',\n",
              "  'IF_2013': '1.679',\n",
              "  'IF_2014': '1.679',\n",
              "  'IF_2015': '1.633',\n",
              "  'IF_2016': '1.687',\n",
              "  'IF_2017': '1.804',\n",
              "  'ISSN': '0167-6369',\n",
              "  'ISSN_online': '1573-2959',\n",
              "  'ISSN_print': '0167-6369',\n",
              "  'IsoAbbr': 'Environ Monit Assess',\n",
              "  'JrId': 22015,\n",
              "  'MedAbbr': 'Environ Monit Assess',\n",
              "  'NLMid': '8508350',\n",
              "  'Titre': 'Environ Monit Assess',\n",
              "  'abstract': 'In this paper, a new fuzzy methodology is developed to optimize water and waste load allocation (WWLA) in rivers under uncertainty. An interactive two-stage stochastic fuzzy programming (ITSFP) method is utilized to handle parameter uncertainties, which are expressed as fuzzy boundary intervals. An iterative linear programming (ILP) is also used for solving the nonlinear optimization model. To accurately consider the impacts of the water and waste load allocation strategies on the river water quality, a calibrated QUAL2Kw model is linked with the WWLA optimization model. The soil, water, atmosphere, and plant (SWAP) simulation model is utilized to determine the quantity and quality of each agricultural return flow. To control pollution loads of agricultural networks, it is assumed that a part of each agricultural return flow can be diverted to an evaporation pond and also another part of it can be stored in a detention pond. In detention ponds, contaminated water is exposed to solar radiation for disinfecting pathogens. Results of applying the proposed methodology to the Dez River system in the southwestern region of Iran illustrate its effectiveness and applicability for water and waste load allocation in rivers. In the planning phase, this methodology can be used for estimating the capacities of return flow diversion system and evaporation and detention ponds. ',\n",
              "  'authors': ['Tavakoli, Ali',\n",
              "   'Nikoo, Mohammad Reza',\n",
              "   'Kerachian, Reza',\n",
              "   'Soltani, Maryam'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental monitoring and assessment',\n",
              "  'keywords': ['Agriculture',\n",
              "   'Conservation of Natural Resources',\n",
              "   'Environmental Monitoring',\n",
              "   'Fuzzy Logic',\n",
              "   'Iran',\n",
              "   'Models, Theoretical',\n",
              "   'Rivers',\n",
              "   'Uncertainty',\n",
              "   'Waste Water',\n",
              "   'Water Pollution',\n",
              "   'Water Quality'],\n",
              "  'pmid': '25740683',\n",
              "  'title': 'River water quality management considering agricultural return flows: application of a nonlinear two-stage stochastic fuzzy programming.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1600-0447',\n",
              "  'IF_2013': '5.545',\n",
              "  'IF_2014': '5.605',\n",
              "  'IF_2015': '6.128',\n",
              "  'IF_2016': '6.79',\n",
              "  'IF_2017': '4.984',\n",
              "  'ISSN': '0001-690X',\n",
              "  'ISSN_online': '1600-0447',\n",
              "  'ISSN_print': '0001-690X',\n",
              "  'IsoAbbr': 'Acta Psychiatr Scand',\n",
              "  'JrId': 192,\n",
              "  'MedAbbr': 'Acta Psychiatr Scand',\n",
              "  'NLMid': '0370364',\n",
              "  'Titre': 'Acta Psychiatr Scand',\n",
              "  'abstract': \"Structural MRI (sMRI) increasingly offers insight into abnormalities inherent to schizophrenia. Previous machine learning applications suggest that individual classification is feasible and reliable and, however, is focused on the predictive performance of the clinical status in cross-sectional designs, which has limited biological perspectives. Moreover, most studies depend on relatively small cohorts or single recruiting site. Finally, no study controlled for disease stage or medication's effect. These elements cast doubt on previous findings' reproducibility.|We propose a machine learning algorithm that provides an interpretable brain signature. Using large datasets collected from 4 sites (276 schizophrenia patients, 330 controls), we assessed cross-site prediction reproducibility and associated predictive signature. For the first time, we evaluated the predictive signature regarding medication and illness duration using an independent dataset of first-episode patients.|Machine learning classifiers based on neuroanatomical features yield significant intersite prediction accuracies (72%) together with an excellent predictive signature stability. This signature provides a neural score significantly correlated with symptom severity and the extent of cognitive impairments. Moreover, this signature demonstrates its efficiency on first-episode psychosis patients (73% accuracy).|These results highlight the existence of a common neuroanatomical signature for schizophrenia, shared by a majority of patients even from an early stage of the disorder.\",\n",
              "  'authors': ['de Pierrefeu, A',\n",
              "   'Löfstedt, T',\n",
              "   'Laidi, C',\n",
              "   'Hadj-Selem, F',\n",
              "   'Bourgin, J',\n",
              "   'Hajek, T',\n",
              "   'Spaniel, F',\n",
              "   'Kolenic, M',\n",
              "   'Ciuciu, P',\n",
              "   'Hamdani, N',\n",
              "   'Leboyer, M',\n",
              "   'Fovet, T',\n",
              "   'Jardri, R',\n",
              "   'Houenou, J',\n",
              "   'Duchesnay, E'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Acta psychiatrica Scandinavica',\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Gray Matter',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Machine Learning',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Reproducibility of Results',\n",
              "   'Schizophrenia',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Severity of Illness Index'],\n",
              "  'pmid': '30242828',\n",
              "  'title': 'Identifying a neuroanatomical signature of schizophrenia, reproducible across sites and stages, using machine learning with structured sparsity.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Robotic-assisted surgical techniques are not yet well established among surgeon practice groups beyond a few surgical subspecialties. To help identify the facilitators and barriers to their adoption, this belief-elicitation study contextualized and supplemented constructs of the unified theory of acceptance and use of technology (UTAUT) in robotic-assisted surgery. Semi-structured individual interviews were conducted with 21 surgeons comprising two groups: users and nonusers. The main facilitators to adoption were Perceived Usefulness and Facilitating Conditions among both users and nonusers, followed by Attitude Toward Using Technology among users and Extrinsic Motivation among nonusers. The three main barriers to adoption for both users and nonusers were Perceived Ease of Use and Complexity, Perceived Usefulness, and Perceived Behavioral Control. This study's findings can assist surgeons, hospital and medical school administrators, and other policy makers on the proper adoption of robotic-assisted surgery and can guide future research on the development of theories and framing of hypotheses.\",\n",
              "  'authors': ['Benmessaoud, Christine', 'Kharrazi, Hadi', 'MacDorman, Karl F'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Data Collection',\n",
              "   'Practice Guidelines as Topic',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Surgical Equipment',\n",
              "   'Surgical Procedures, Operative',\n",
              "   'Technology'],\n",
              "  'pmid': '21283719',\n",
              "  'title': 'Facilitators and barriers to adopting robotic-assisted surgery: contextualizing the unified theory of acceptance and use of technology.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'In this paper, the singular LR fuzzy linear system is introduced. Such systems are divided into two parts: singular consistent LR fuzzy linear systems and singular inconsistent LR fuzzy linear systems. The capability of the generalized inverses such as Drazin inverse, pseudoinverse, and {1}-inverse in finding minimal solution of singular consistent LR fuzzy linear systems is investigated. ',\n",
              "  'authors': ['Nikuie, M', 'Ahmad, M Z'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Algorithms', 'Computer Simulation', 'Fuzzy Logic'],\n",
              "  'pmid': '24737977',\n",
              "  'title': 'Minimal solution of singular LR fuzzy linear systems.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2041-3033',\n",
              "  'IF_2013': '1.144',\n",
              "  'IF_2014': '1.329',\n",
              "  'IF_2015': '0.996',\n",
              "  'IF_2016': '1.005',\n",
              "  'IF_2017': '1.124',\n",
              "  'ISSN': '0954-4119',\n",
              "  'ISSN_online': '2041-3033',\n",
              "  'ISSN_print': '0954-4119',\n",
              "  'IsoAbbr': 'Proc Inst Mech Eng H',\n",
              "  'JrId': 1315,\n",
              "  'MedAbbr': 'Proc Inst Mech Eng H',\n",
              "  'NLMid': '8908934',\n",
              "  'Titre': 'Proc Inst Mech Eng H',\n",
              "  'abstract': 'This paper investigates new methods for measuring forces and tactile sense as a contribution towards relaying the sense of touch to the surgeon. The approach used is to determine a distribution of contact force using a small number of sensory outputs to detect the bending of a surface of known behaviour. Software algorithms have been produced to interpret the contacting force from sensory data, and have achieved a bandwidth of 30 Hz and an accuracy of 2 per cent. The sensor construction is of sufficiently low cost to produce a disposable unit and uses materials that are compatible with the invasive working environment.',\n",
              "  'authors': ['Brett, P N', 'Stone, R S'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Proceedings of the Institution of Mechanical Engineers. Part H, Journal of engineering in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Simulation',\n",
              "   'Equipment Design',\n",
              "   'Fiber Optic Technology',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Software',\n",
              "   'Stress, Mechanical',\n",
              "   'Therapy, Computer-Assisted',\n",
              "   'Touch',\n",
              "   'Transducers',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '9330542',\n",
              "  'title': 'A technique for measuring contact force distribution in minimally invasive surgical procedures.',\n",
              "  'year': '1997'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1095-8630',\n",
              "  'IF_2013': '3.188',\n",
              "  'IF_2014': '2.723',\n",
              "  'IF_2015': '3.131',\n",
              "  'IF_2016': '4.01',\n",
              "  'IF_2017': '4.005',\n",
              "  'ISSN': '0301-4797',\n",
              "  'ISSN_online': '1095-8630',\n",
              "  'ISSN_print': '0301-4797',\n",
              "  'IsoAbbr': 'J. Environ. Manage.',\n",
              "  'JrId': 22117,\n",
              "  'MedAbbr': 'J Environ Manage',\n",
              "  'NLMid': '0401664',\n",
              "  'Titre': 'J. Environ. Manage.',\n",
              "  'abstract': \"This paper introduces a violation analysis approach for the planning of regional solid waste management systems under uncertainty, based on an interval-parameter fuzzy integer programming (IPFIP) model. In this approach, several given levels of tolerable violation for system constraints are permitted. This is realized through a relaxation of the critical constraints using violation variables, such that the model's decision space can be expanded. Thus, solutions from the violation analysis will not necessarily satisfy all of the model's original constraints. Application of the developed methodology to the planning of a waste management system indicates that reasonable solutions can be generated through this approach. Considerable information regarding decisions of facility expansion and waste flow allocation within the waste management system were generated. The modeling results help to generate a number of decision alternatives under various system conditions, allowing for more in-depth analyses of tradeoffs between environmental and economic objectives as well as those between system optimality and reliability.\",\n",
              "  'authors': ['Huang, Y F', 'Baetz, B W', 'Huang, G H', 'Liu, L'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Journal of environmental management',\n",
              "  'keywords': ['Fuzzy Logic',\n",
              "   'Guideline Adherence',\n",
              "   'Models, Theoretical',\n",
              "   'Refuse Disposal',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '12369404',\n",
              "  'title': 'Violation analysis for solid waste management systems: an interval fuzzy programming approach.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1433-0563',\n",
              "  'IF_2013': '0.436',\n",
              "  'IF_2014': '0.444',\n",
              "  'IF_2015': '0.331',\n",
              "  'IF_2016': '0.304',\n",
              "  'IF_2017': '0.437',\n",
              "  'ISSN': '0340-2592',\n",
              "  'ISSN_online': '1433-0563',\n",
              "  'ISSN_print': '0340-2592',\n",
              "  'IsoAbbr': 'Urologe A',\n",
              "  'JrId': 7905,\n",
              "  'MedAbbr': 'Urologe A',\n",
              "  'NLMid': '1304110',\n",
              "  'Titre': 'Urologe A',\n",
              "  'abstract': \"Laparoscopic radical nephrectomy (LRN) is considered as a standard of care for T2 renal masses and T1 tumors not treatable by nephron-sparing surgery. It can be performed transperitoneally, retroperitoneoscopic or hand-assisted. However, the morbidity after laparoscopic nephrectomy has been shown to be lower than the open procedure and patients seem to benefit from early mobilization, less pain medication, shorter hospital stays and an earlier return to normal daily activities. Furthermore, the extent of perioperative activation of the systemic stress response appears to be less during laparoscopic procedures. This has been shown to have evidently beneficial clinical impact on patient's recovery; however, its importance for the oncologic prognosis is somewhat unclear. In addition, the progression-free and overall tumor-specific survival rates for laparoscopic nephrectomy are equivalent to those for open surgery. The experiences with robot-assistance for laparoscopic nephrectomy reported so far show no significant advantages over traditional laparoscopic nephrectomy. However, the problem of high costs of acquisition and operation of robots still remains unsolved. For the future, prospective studies are needed in order to compare the functional and oncological outcomes and cost-effectiveness of different methods of radical nephrectomy.\",\n",
              "  'authors': ['Hoda, M R', 'Fornara, P'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Der Urologe. Ausg. A',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Nephrectomy',\n",
              "   'Reconstructive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '22526177',\n",
              "  'title': '[Nephrectomy - pro laparoscopic].',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1558-349X',\n",
              "  'IF_2013': '2.283',\n",
              "  'IF_2014': '2.836',\n",
              "  'IF_2015': '2.929',\n",
              "  'IF_2016': '2.993',\n",
              "  'IF_2017': '3.383',\n",
              "  'ISSN': '1546-1440',\n",
              "  'ISSN_online': '1558-349X',\n",
              "  'ISSN_print': '1546-1440',\n",
              "  'IsoAbbr': 'J Am Coll Radiol',\n",
              "  'JrId': 33174,\n",
              "  'MedAbbr': 'J Am Coll Radiol',\n",
              "  'NLMid': '101190326',\n",
              "  'Titre': 'J Am Coll Radiol',\n",
              "  'abstract': 'The medical device industry is undergoing rapid change as innovation accelerates, new business models emerge, and artificial intelligence and the Internet of Things create disruptive possibilities in health care. On the innovation front, global annual patent applications related to medical devices have tripled in 10 years, and technology cycle times have halved in just 5 years. Connectivity has exploded-by 2021, the world will have more than three times as many smart connected devices as people-and more and more medical devices and processes contain integrated sensors. In this article, we report on recent McKinsey (McKinsey & Company, New York, New York) work to map start-ups and trends shaping the future of medical imaging. We identify technology clusters with prospects of future growth, look at some of their cutting-edge practices, and consider what the implications may be for our specialty.',\n",
              "  'authors': ['Alexander, Alan',\n",
              "   'McGill, Megan',\n",
              "   'Tarasova, Anna',\n",
              "   'Ferreira, Cara',\n",
              "   'Zurkiya, Delphine'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of the American College of Radiology : JACR',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30522844',\n",
              "  'title': 'Scanning the Future of Medical Imaging.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'Molecular events normally have significant meanings since they describe important biological interactions or alternations such as binding of a protein. As a crucial step of biological event extraction, event trigger identification has attracted much attention and many methods have been proposed. Traditionally those methods can be categorised into rule-based approach and machine learning approach and machine learning-based approaches have demonstrated its potential and outperformed rule-based approaches in many situations. However, machine learning-based approaches still face several challenges among which a notable one is how to model semantic and syntactic information of different words and incorporate it into the prediction model. There exist many ways to model semantic and syntactic information, among which word embedding is an effective one. Therefore, in order to address this challenge, in this study, a word embedding assisted neural network prediction model is proposed to conduct event trigger identification. The experimental study on commonly used dataset has shown its potential. It is believed that this study could offer researchers insights into semantic-aware solutions for event trigger identification. ',\n",
              "  'authors': ['Nie, Yifan',\n",
              "   'Rong, Wenge',\n",
              "   'Zhang, Yiyuan',\n",
              "   'Ouyang, Yuanxin',\n",
              "   'Xiong, Zhang'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Computational Biology',\n",
              "   'Data Mining',\n",
              "   'Databases, Factual',\n",
              "   'Natural Language Processing',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Semantics'],\n",
              "  'pmid': '25669328',\n",
              "  'title': 'Embedding assisted prediction architecture for event trigger identification.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': \"Choroidal perfusion measurements are complicated by the choroid's location posterior to the retina and its associated retinal blood vessels. Optical coherence tomography is a relatively new imaging technique with sufficient spatial resolution to isolate choroidal backscattering events from the posterior eye. We modified a speckle imaging algorithm to analyze sequential axial depth scans obtained from posterior rat eye to obtain an indicator of choroidal perfusion. This indicator is correlated with known changes in choroidal blood flow in response to increased intraocular pressure.\",\n",
              "  'authors': ['Wu, Frank I', 'Glucksberg, Matthew R'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Blood Flow Velocity',\n",
              "   'Choroid',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Ophthalmoscopy',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Tomography, Optical Coherence'],\n",
              "  'pmid': '15796241',\n",
              "  'title': 'Choroidal perfusion measurements made with optical coherence tomography.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1752-0509',\n",
              "  'IF_2013': '2.853',\n",
              "  'IF_2014': '2.435',\n",
              "  'IF_2015': '2.213',\n",
              "  'IF_2016': '2.303',\n",
              "  'IF_2017': '2.05',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1752-0509',\n",
              "  'IsoAbbr': 'BMC Syst Biol',\n",
              "  'JrId': 33336,\n",
              "  'MedAbbr': 'BMC Syst Biol',\n",
              "  'NLMid': '101301827',\n",
              "  'Titre': 'BMC Syst Biol',\n",
              "  'abstract': 'Signaling pathways are the key biological mechanisms that transduce extracellular signals to affect transcription factor mediated gene regulation within cells. A number of computational methods have been developed to identify the topological structure of a specific signaling pathway using protein-protein interaction data, but they are not designed for identifying active signaling pathways in an unbiased manner. On the other hand, there are statistical methods based on gene sets or pathway data that can prioritize likely active signaling pathways, but they do not make full use of active pathway structure that link receptor, kinases and downstream transcription factors.|Here, we present a method to simultaneously predict the set of active signaling pathways, together with their pathway structure, by integrating protein-protein interaction network and gene expression data. We evaluated the capacity for our method to predict active signaling pathways for dental epithelial cells, ocular lens epithelial cells, human pluripotent stem cell-derived lens epithelial cells, and lens fiber cells. This analysis showed our approach could identify all the known active pathways that are associated with tooth formation and lens development.|The results suggest that SPAGI can be a useful approach to identify the potential active signaling pathways given a gene expression profile. Our method is implemented as an open source R package, available via https://github.com/VCCRI/SPAGI/ .',\n",
              "  'authors': ['Kabir, Md Humayun',\n",
              "   'Patrick, Ralph',\n",
              "   'Ho, Joshua W K',\n",
              "   \"O'Connor, Michael D\"],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'BMC systems biology',\n",
              "  'keywords': ['Animals',\n",
              "   'Computational Biology',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Lens, Crystalline',\n",
              "   'Mice',\n",
              "   'Protein Interaction Maps',\n",
              "   'Signal Transduction',\n",
              "   'Tooth',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '30598083',\n",
              "  'title': 'Identification of active signaling pathways by integrating gene expression and protein interaction data.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1664-8021',\n",
              "  'IF_2013': '3.789',\n",
              "  'IF_2014': '3.789',\n",
              "  'IF_2015': '3.789',\n",
              "  'IF_2016': '3.789',\n",
              "  'IF_2017': '4.151',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1664-8021',\n",
              "  'IsoAbbr': 'Front Genet',\n",
              "  'JrId': 39184,\n",
              "  'MedAbbr': 'Front Genet',\n",
              "  'NLMid': '101560621',\n",
              "  'Titre': 'Front Genet',\n",
              "  'abstract': 'The promoter region is located near the transcription start sites and regulates transcription initiation of the gene by controlling the binding of RNA polymerase. Thus, promoter region recognition is an important area of interest in the field of bioinformatics. Numerous tools for promoter prediction were proposed. However, the reliability of these tools still needs to be improved. In this work, we propose a robust deep learning model, called DeePromoter, to analyze the characteristics of the short eukaryotic promoter sequences, and accurately recognize the human and mouse promoter sequences. DeePromoter combines a convolutional neural network (CNN) and a long short-term memory (LSTM). Additionally, instead of using non-promoter regions of the genome as a negative set, we derive a more challenging negative set from the promoter sequences. The proposed negative set reconstruction method improves the discrimination ability and significantly reduces the number of false positive predictions. Consequently, DeePromoter outperforms the previously proposed promoter prediction tools. In addition, a web-server for promoter prediction is developed based on the proposed methods and made available at https://home.jbnu.ac.kr/NSCL/deepromoter.htm.',\n",
              "  'authors': ['Oubounyt, Mhaned',\n",
              "   'Louadi, Zakaria',\n",
              "   'Tayara, Hilal',\n",
              "   'Chong, Kil To'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Frontiers in genetics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31024615',\n",
              "  'title': 'DeePromoter: Robust Promoter Predictor Using Deep Learning.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Many everyday tasks require the ability of two or more individuals to coordinate their actions with others to increase efficiency. Such an increase in efficiency can often be observed even after only very few trials. Previous work suggests that such behavioral adaptation can be explained within a probabilistic framework that integrates sensory input and prior experience. Even though higher cognitive abilities such as intention recognition have been described as probabilistic estimation depending on an internal model of the other agent, it is not clear whether much simpler daily interaction is consistent with a probabilistic framework. Here, we investigate whether the mechanisms underlying efficient coordination during manual interactions can be understood as probabilistic optimization. For this purpose we studied in several experiments a simple manual handover task concentrating on the action of the receiver. We found that the duration until the receiver reacts to the handover decreases over trials, but strongly depends on the position of the handover. We then replaced the human deliverer by different types of robots to further investigate the influence of the delivering movement on the reaction of the receiver. Durations were found to depend on movement kinematics and the robot's joint configuration. Modeling the task was based on the assumption that the receiver's decision to act is based on the accumulated evidence for a specific handover position. The evidence for this handover position is collected from observing the hand movement of the deliverer over time and, if appropriate, by integrating this sensory likelihood with prior expectation that is updated over trials. The close match of model simulations and experimental results shows that the efficiency of handover coordination can be explained by an adaptive probabilistic fusion of a-priori expectation and online estimation.\",\n",
              "  'authors': ['Huber, Markus',\n",
              "   'Kupferberg, Aleksandra',\n",
              "   'Lenz, Claus',\n",
              "   'Knoll, Alois',\n",
              "   'Brandt, Thomas',\n",
              "   'Glasauer, Stefan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Adult',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Interpersonal Relations',\n",
              "   'Models, Theoretical',\n",
              "   'Movement',\n",
              "   'Probability',\n",
              "   'Reaction Time',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Spatio-Temporal Analysis',\n",
              "   'Task Performance and Analysis',\n",
              "   'Young Adult'],\n",
              "  'pmid': '23724112',\n",
              "  'title': 'Spatiotemporal movement planning and rapid adaptation for manual interaction.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': \"Image-guided cryotherapy of renal cancer is an emerging alternative to surgical nephrectomy, particularly for those who cannot sustain the physical burden of surgery. It is well known that the outcome of this therapy depends on the accurate placement of the cryotherapy probe. Therefore, a robotic instrument guide may help physicians aim the cryotherapy probe precisely to maximize the efficacy of the treatment and avoid damage to critical surrounding structures. The objective of this paper was to propose a robotic instrument guide for orienting cryotherapy probes in image-guided cryotherapy of renal cancers. The authors propose a body-mounted robotic guide that is expected to be less susceptible to guidance errors caused by the patient's whole body motion.|Keeping the device's minimal footprint in mind, the authors developed and validated a body-mounted, robotic instrument guide that can maintain the geometrical relationship between the device and the patient's body, even in the presence of the patient's frequent body motions. The guide can orient the cryotherapy probe with the skin incision point as the remote-center-of-motion. The authors' validation studies included an evaluation of the mechanical accuracy and position repeatability of the robotic instrument guide. The authors also performed a mock MRI-guided cryotherapy procedure with a phantom to compare the advantage of robotically assisted probe replacements over a free-hand approach, by introducing organ motions to investigate their effects on the accurate placement of the cryotherapy probe. Measurements collected for performance analysis included accuracy and time taken for probe placements. Multivariate analysis was performed to assess if either or both organ motion and the robotic guide impacted these measurements.|The mechanical accuracy and position repeatability of the probe placement using the robotic instrument guide were 0.3 and 0.1 mm, respectively, at a depth of 80 mm. The phantom test indicated that the accuracy of probe placement was significantly better with the robotic instrument guide (4.1 mm) than without the guide (6.3 mm, p<0.001), even in the presence of body motion. When independent organ motion was artificially added, in addition to body motion, the advantage of accurate probe placement using the robotic instrument guide disappeared statistically [i.e., 6.0 mm with the robotic guide and 5.9 mm without the robotic guide (p = 0.906)]. When the robotic instrument guide was used, the total time required to complete the procedure was reduced from 19.6 to 12.7 min (p<0.001). Multivariable analysis indicated that the robotic instrument guide, not the organ motion, was the cause of statistical significance. The statistical power the authors obtained was 88% in accuracy assessment and 99% higher in duration measurement.|The body-mounted robotic instrument guide allows positioning of the probe during image-guided cryotherapy of renal cancer and was done in fewer attempts and in less time than the free-hand approach. The accuracy of the placement of the cryotherapy probe was better using the robotic instrument guide than without the guide when no organ motion was present. The accuracy between the robotic and free-hand approach becomes comparable when organ motion was present.\",\n",
              "  'authors': ['Hata, Nobuhiko',\n",
              "   'Song, Sang-Eun',\n",
              "   'Olubiyi, Olutayo',\n",
              "   'Arimitsu, Yasumichi',\n",
              "   'Fujimoto, Kosuke',\n",
              "   'Kato, Takahisa',\n",
              "   'Tuncali, Kemal',\n",
              "   'Tani, Soichiro',\n",
              "   'Tokuda, Junichi'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Cryosurgery',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Models, Biological',\n",
              "   'Movement',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '26843245',\n",
              "  'title': 'Body-mounted robotic instrument guide for image-guided cryotherapy of renal cancer.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'The cortical organization of language in bilinguals remains disputed. We studied 24 right-handed fluent bilinguals: 15 exposed to both Mandarin and English before the age of 6 years; and nine exposed to Mandarin in early childhood but English only after the age of 12 years. Blood oxygen level-dependent contrast functional magnetic resonance imaging was performed while subjects performed cued word generation in each language. Fixation was the control task. In both languages, activations were present in the prefrontal, temporal, and parietal regions, and the supplementary motor area. Activations in the prefrontal region were compared by (1) locating peak activations and (2) counting the number of voxels that exceeded a statistical threshold. Although there were differences in the magnitude of activation between the pair of languages, no subject showed significant differences in peak-location or hemispheric asymmetry of activations in the prefrontal language areas. Early and late bilinguals showed a similar pattern of overlapping activations. There are no significant differences in the cortical areas activated for both Mandarin and English at the single word level, irrespective of age of acquisition of either language.',\n",
              "  'authors': ['Chee, M W', 'Tan, E W', 'Thiel, T'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Analysis of Variance',\n",
              "   'Brain Mapping',\n",
              "   'Child',\n",
              "   'Child Language',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Multilingualism',\n",
              "   'Natural Language Processing',\n",
              "   'Oxygen'],\n",
              "  'pmid': '10191322',\n",
              "  'title': 'Mandarin and English single word processing studied with functional magnetic resonance imaging.',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1464-410X',\n",
              "  'IF_2013': '3.13',\n",
              "  'IF_2014': '3.533',\n",
              "  'IF_2015': '4.387',\n",
              "  'IF_2016': '4.439',\n",
              "  'IF_2017': '4.688',\n",
              "  'ISSN': '1464-4096',\n",
              "  'ISSN_online': '1464-410X',\n",
              "  'ISSN_print': '1464-4096',\n",
              "  'IsoAbbr': 'BJU Int.',\n",
              "  'JrId': 21363,\n",
              "  'MedAbbr': 'BJU Int',\n",
              "  'NLMid': '100886721',\n",
              "  'Titre': 'BJU Int.',\n",
              "  'abstract': 'To demonstrate the feasibility, and to report our single-centre perioperative outcomes of repeat robot-assisted partial nephrectomy (RAPN).|From June 2006 to June 2012, 490 patients underwent RAPN for a renal mass at our centre. Of these patients, nine who had undergone previous ipsilateral nephron-sparing surgery (NSS) were included in the analysis. Patient charts were reviewed to obtain demographic data, preoperative surgical history, operative details, and postoperative outcomes and follow-up data.|In all, 12 tumours were removed in nine patients (median age 69 years; six female). A third of the operations were performed on patients with a solitary kidney. The median (range) R.E.N.A.L. nephrometry score for the resected masses was 7 (4-8). The warm ischaemia time was 17.5 min and in three of the nine patients an unclamped procedure was performed. No intraoperative complications were registered, whereas only two minor complications occurred postoperatively. There were no renal unit losses. All surgical margins were negative. There was no significant difference between mean preoperative and latest postoperative mean estimated glomerular filtration rates (70.5 vs 63.5 mL/min/1.73 m(2) , P > 0.05). At a mean (sd) follow-up of 8.3 (13) months, eight of the nine patients with a pathology diagnosis of malignant neoplasm were alive and free from disease at the latest follow-up.|Although technically more demanding, repeat RAPN can be safely and effectively performed in patients presenting with local recurrence after primary NSS for kidney cancer.',\n",
              "  'authors': ['Autorino, Riccardo',\n",
              "   'Khalifeh, Ali',\n",
              "   'Laydner, Humberto',\n",
              "   'Samarasekera, Dinesh',\n",
              "   'Rizkala, Emad',\n",
              "   'Eyraud, Remi',\n",
              "   'Haber, Georges-Pascal',\n",
              "   'Stein, Robert J',\n",
              "   'Kaouk, Jihad H'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BJU international',\n",
              "  'keywords': ['Aged',\n",
              "   'Disease-Free Survival',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Glomerular Filtration Rate',\n",
              "   'Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Postoperative Period',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Time Factors',\n",
              "   'Treatment Outcome',\n",
              "   'Warm Ischemia'],\n",
              "  'pmid': '23578234',\n",
              "  'title': 'Repeat robot-assisted partial nephrectomy (RAPN): feasibility and early outcomes.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'The paper considers gradient training of fuzzy logic controller (FLC) presented in the form of neural network structure. The proposed neuro-fuzzy structure allows keeping linguistic meaning of fuzzy rule base. Its main adjustable parameters are shape determining parameters of the linguistic variables fuzzy values as well as that of the used as intersection operator parameterized T-norm. The backpropagation through time method was applied to train neuro-FLC for a highly non-linear plant (a biotechnological process). The obtained results are discussed with respect to adjustable parameters rationality. Conclusions are made with respect to the appropriate intersection operations too.',\n",
              "  'authors': ['Koprinkova-Hristova, Petia'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Fuzzy Logic',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '20945520',\n",
              "  'title': 'Backpropagation through time training of a neuro-fuzzy controller.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1573-2959',\n",
              "  'IF_2013': '1.679',\n",
              "  'IF_2014': '1.679',\n",
              "  'IF_2015': '1.633',\n",
              "  'IF_2016': '1.687',\n",
              "  'IF_2017': '1.804',\n",
              "  'ISSN': '0167-6369',\n",
              "  'ISSN_online': '1573-2959',\n",
              "  'ISSN_print': '0167-6369',\n",
              "  'IsoAbbr': 'Environ Monit Assess',\n",
              "  'JrId': 22015,\n",
              "  'MedAbbr': 'Environ Monit Assess',\n",
              "  'NLMid': '8508350',\n",
              "  'Titre': 'Environ Monit Assess',\n",
              "  'abstract': 'Determination of above ground biomass (AGB) of any forest is a longstanding scientific endeavor, which helps to estimate net primary productivity, carbon stock and other biophysical parameters of that forest. With advancement of geospatial technology in last few decades, AGB estimation now can be done using space-borne and airborne remotely sensed data. It is a well-established, time saving and cost effective technique with high precision and is frequently applied by the scientific community. It involves development of allometric equations based on correlations of ground-based forest biomass measurements with vegetation indices derived from remotely sensed data. However, selection of the best-fit and explanatory models of biomass estimation often becomes a difficult proposition with respect to the image data resolution (spatial and spectral) as well as the sensor platform position in space. Using Resourcesat-2 satellite data and Normalized Difference Vegetation Index (NDVI), this pilot scale study compared traditional linear and nonlinear models with an artificial intelligence-based non-parametric technique, i.e. artificial neural network (ANN) for formulation of the best-fit model to determine AGB of forest of the Bundelkhand region of India. The results confirmed the superiority of ANN over other models in terms of several statistical significance and reliability assessment measures. Accordingly, this study proposed the use of ANN instead of traditional models for determination of AGB and other bio-physical parameters of any dry deciduous forest of tropical sub-humid or semi-arid area. In addition, large numbers of sampling sites with different quadrant sizes for trees, shrubs, and herbs as well as application of LiDAR data as predictor variable were recommended for very high precision modelling in ANN for a large scale study.',\n",
              "  'authors': ['Deb, Dibyendu',\n",
              "   'Singh, J P',\n",
              "   'Deb, Shovik',\n",
              "   'Datta, Debajit',\n",
              "   'Ghosh, Arunava',\n",
              "   'Chaurasia, R S'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental monitoring and assessment',\n",
              "  'keywords': ['Biomass',\n",
              "   'Carbon',\n",
              "   'Environmental Monitoring',\n",
              "   'Forests',\n",
              "   'India',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pilot Projects',\n",
              "   'Reproducibility of Results',\n",
              "   'Satellite Imagery',\n",
              "   'Trees'],\n",
              "  'pmid': '29052047',\n",
              "  'title': 'An alternative approach for estimating above ground biomass using Resourcesat-2 satellite data and artificial neural network in Bundelkhand region of India.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1424-3911',\n",
              "  'IF_2013': '2.504',\n",
              "  'IF_2014': '2.837',\n",
              "  'IF_2015': '2.406',\n",
              "  'IF_2016': '2.58',\n",
              "  'IF_2017': '2.763',\n",
              "  'ISSN': '1424-3903',\n",
              "  'ISSN_online': '1424-3911',\n",
              "  'ISSN_print': '1424-3903',\n",
              "  'IsoAbbr': 'Pancreatology',\n",
              "  'JrId': 22282,\n",
              "  'MedAbbr': 'Pancreatology',\n",
              "  'NLMid': '100966936',\n",
              "  'Titre': 'Pancreatology',\n",
              "  'abstract': \"Pancreatic cancer is strongly associated with the development of hyperglycemia, peripheral insulin resistance and diabetes mellitus, especially when presented as new-onset diabetes mellitus. Peripheral insulin resistance and hyperinsulinemia have been suggested to promote growth of pancreatic cancer cells, and therefore a relation between long-standing diabetes mellitus type 2 and pancreatic cancer has been implied. Epidemiological studies, though, give incongruent results to this problem. There are data supporting a tumor-derived influence on glucose metabolism, insulin secretion and eventually the development of diabetes mellitus in early stages of pancreatic cancer. The only possibility for curative intent in pancreatic cancer is to diagnose the disease before symptoms occur. Patients with newly diagnosed diabetes mellitus type 2 or hyperglycemia as a risk group have been recommended for primary screening for pancreatic cancer. To date, there is no specific biomarker to identify patients with an asymptomatic pancreatic cancer. The review discuss the relationship between pancreatic cancer and diabetes mellitus and the possibility of secondary screening of patients with newly diagnosed diabetes mellitus type 2 or hyperglycemia in an artificial neural network. PubMed was searched for articles using the Mesh term 'pancreatic neoplasms' combined with 'insulin resistance' and 'glucose metabolism disorders'. Additional articles were retrieved through hyperlinks and by manually searching reference lists in original published articles. In total 36 articles were systematically reviewed. and IAP.\",\n",
              "  'authors': ['Bartosch-Härlid, A', 'Andersson, R'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'Pancreatology : official journal of the International Association of Pancreatology (IAP) ... [et al.]',\n",
              "  'keywords': ['Asymptomatic Diseases',\n",
              "   'Biomarkers, Tumor',\n",
              "   'Carcinoma, Pancreatic Ductal',\n",
              "   'Comorbidity',\n",
              "   'Diabetes Mellitus, Type 2',\n",
              "   'Humans',\n",
              "   'Hyperglycemia',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pancreatic Neoplasms'],\n",
              "  'pmid': '20720443',\n",
              "  'title': 'Diabetes mellitus in pancreatic cancer and the need for diagnosis of asymptomatic disease.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1537-744X',\n",
              "  'IF_2013': '1.219',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '2356-6140',\n",
              "  'ISSN_online': '1537-744X',\n",
              "  'ISSN_print': '2356-6140',\n",
              "  'IsoAbbr': 'ScientificWorldJournal',\n",
              "  'JrId': 27024,\n",
              "  'MedAbbr': 'ScientificWorldJournal',\n",
              "  'NLMid': '101131163',\n",
              "  'Titre': 'ScientificWorldJournal',\n",
              "  'abstract': 'Big data is a new trend at present, forcing the significant impacts on information technologies. In big data applications, one of the most concerned issues is dealing with large-scale data sets that often require computation resources provided by public cloud services. How to analyze big data efficiently becomes a big challenge. In this paper, we collaborate interval regression with the smooth support vector machine (SSVM) to analyze big data. Recently, the smooth support vector machine (SSVM) was proposed as an alternative of the standard SVM that has been proved more efficient than the traditional SVM in processing large-scale data. In addition the soft margin method is proposed to modify the excursion of separation margin and to be effective in the gray zone that the distribution of data becomes hard to be described and the separation margin between classes. ',\n",
              "  'authors': ['Huang, Chia-Hui', 'Yang, Keng-Chieh', 'Kao, Han-Ying'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'TheScientificWorldJournal',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Regression Analysis',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '25143968',\n",
              "  'title': 'Analyzing big data with the hybrid interval regression methods.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1876-7737',\n",
              "  'IF_2013': '3.929',\n",
              "  'IF_2014': '3.888',\n",
              "  'IF_2015': '3.867',\n",
              "  'IF_2016': '3.914',\n",
              "  'IF_2017': '3.722',\n",
              "  'ISSN': '1874-3919',\n",
              "  'ISSN_online': '1876-7737',\n",
              "  'ISSN_print': '1874-3919',\n",
              "  'IsoAbbr': 'J Proteomics',\n",
              "  'JrId': 34573,\n",
              "  'MedAbbr': 'J Proteomics',\n",
              "  'NLMid': '101475056',\n",
              "  'Titre': 'J Proteomics',\n",
              "  'abstract': 'The present research proposed general evaluation strategy named Null-Test for peptide identification algorithm in Shotgun proteomics. The Null-Test method based on random matching can be utilized to check whether the algorithm has a tendency to make a mistake or has potential bugs, faultiness, errors etc., and to validate the reliability of the identification algorithm. Unfortunately, none of the five famous identification software could pass the most stringent Null-Test. PatternLab had good performance in both Null-Test and routine search by making a good control on the overfitting with sound design. The fuzzy logics based method presented as another candidate strategy could pass the Null-Test and has competitive efficiency in peptide identification. Filtering the results by appropriate FDR would increase the number of discoveries in an experiment, at the cost of losing control of Type I errors. Thus, it is necessary to utilize some more stringent criteria when someone wants to design or analyze an algorithm/software. The more stringent criteria will facilitate the discovery of latent bugs, faultiness, errors etc. in the algorithm/software. It would be recommended to utilize independent search combining random database with statistics theorem to estimate the accurate FDR of the identified results.|In the past decades, considerable effort has been devoted to developing a sensitive algorithm for peptide identification in Shotgun proteomics. However, little attention has been paid to controlling the reliability of the identification algorithm at the design stage. The Null-Test based on random matching can be utilized to check whether the algorithm has a tendency to make a mistake or has potential bugs, faultiness, errors etc. However, it turns out that none of the five famous identification software could pass the most stringent Null-Test in the present study, which should be taken into account seriously. Accordingly, a candidate strategy based on fuzzy logics has been demonstrated the possibility that an identification algorithm can pass the Null-Test. PatternLab shows that earlier control on overfitting is valuable for designing an efficient algorithm.',\n",
              "  'authors': ['Zhang, Shu-Rong',\n",
              "   'Shan, Yi-Chu',\n",
              "   'Jiang, Hao',\n",
              "   'Liu, Jian-Hui',\n",
              "   'Zhou, Yuan',\n",
              "   'Zhang, Li-Hua',\n",
              "   'Zhang, Yu-Kui'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Journal of proteomics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Peptides',\n",
              "   'Proteomics',\n",
              "   'Software'],\n",
              "  'pmid': '28506863',\n",
              "  'title': 'The Null-Test for peptide identification algorithm in Shotgun proteomics.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2005-8330',\n",
              "  'IF_2013': '1.807',\n",
              "  'IF_2014': '1.571',\n",
              "  'IF_2015': '1.592',\n",
              "  'IF_2016': '2.156',\n",
              "  'IF_2017': '3.072',\n",
              "  'ISSN': '1229-6929',\n",
              "  'ISSN_online': '2005-8330',\n",
              "  'ISSN_print': '1229-6929',\n",
              "  'IsoAbbr': 'Korean J Radiol',\n",
              "  'JrId': 26838,\n",
              "  'MedAbbr': 'Korean J Radiol',\n",
              "  'NLMid': '100956096',\n",
              "  'Titre': 'Korean J Radiol',\n",
              "  'abstract': \"The purpose of this study was to develop a new method for automated lung nodule detection in serial section CT images with using the characteristics of the 3D appearance of the nodules that distinguish themselves from the vessels.|Lung nodules were detected in four steps. First, to reduce the number of region of interests (ROIs) and the computation time, the lung regions of the CTs were segmented using Genetic Cellular Neural Networks (G-CNN). Then, for each lung region, ROIs were specified with using the 8 directional search; +1 or -1 values were assigned to each voxel. The 3D ROI image was obtained by combining all the 2-Dimensional (2D) ROI images. A 3D template was created to find the nodule-like structures on the 3D ROI image. Convolution of the 3D ROI image with the proposed template strengthens the shapes that are similar to those of the template and it weakens the other ones. Finally, fuzzy rule based thresholding was applied and the ROI's were found. To test the system's efficiency, we used 16 cases with a total of 425 slices, which were taken from the Lung Image Database Consortium (LIDC) dataset.|The computer aided diagnosis (CAD) system achieved 100% sensitivity with 13.375 FPs per case when the nodule thickness was greater than or equal to 5.625 mm.|Our results indicate that the detection performance of our algorithm is satisfactory, and this may well improve the performance of computer-aided detection of lung nodules.\",\n",
              "  'authors': ['Ozekes, Serhat', 'Osman, Onur', 'Ucan, Osman N'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Korean journal of radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Automation',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'False Positive Reactions',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Lung Neoplasms',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '18253070',\n",
              "  'title': \"Nodule detection in a lung region that's segmented with using genetic cellular neural networks and 3D template matching with fuzzy rule based thresholding.\",\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1535-9484',\n",
              "  'IF_2013': '7.254',\n",
              "  'IF_2014': '6.564',\n",
              "  'IF_2015': '5.912',\n",
              "  'IF_2016': '6.54',\n",
              "  'IF_2017': '5.232',\n",
              "  'ISSN': '1535-9476',\n",
              "  'ISSN_online': '1535-9484',\n",
              "  'ISSN_print': '1535-9476',\n",
              "  'IsoAbbr': 'Mol. Cell Proteomics',\n",
              "  'JrId': 26755,\n",
              "  'MedAbbr': 'Mol Cell Proteomics',\n",
              "  'NLMid': '101125647',\n",
              "  'Titre': 'Mol. Cell Proteomics',\n",
              "  'abstract': 'Parenchymatous organs consist of multiple cell types, primarily defined as parenchymal cells (PCs) and nonparenchymal cells (NPCs). The cellular characteristics of these organs are not well understood. Proteomic studies facilitate the resolution of the molecular details of different cell types in organs. These studies have significantly extended our knowledge about organogenesis and organ cellular composition. Here, we present an atlas of the cell-type-resolved liver proteome. In-depth proteomics identified 6000 to 8000 gene products (GPs) for each cell type and a total of 10,075 GPs for four cell types. This data set revealed features of the cellular composition of the liver: (1) hepatocytes (PCs) express the least GPs, have a unique but highly homogenous proteome pattern, and execute fundamental liver functions; (2) the division of labor among PCs and NPCs follows a model in which PCs make the main components of pathways, but NPCs trigger the pathways; and (3) crosstalk among NPCs and PCs maintains the PC phenotype. This study presents the liver proteome at cell resolution, serving as a research model for dissecting the cell type constitution and organ features at the molecular level.',\n",
              "  'authors': ['Ding, Chen',\n",
              "   'Li, Yanyan',\n",
              "   'Guo, Feifei',\n",
              "   'Jiang, Ying',\n",
              "   'Ying, Wantao',\n",
              "   'Li, Dong',\n",
              "   'Yang, Dong',\n",
              "   'Xia, Xia',\n",
              "   'Liu, Wanlin',\n",
              "   'Zhao, Yan',\n",
              "   'He, Yangzhige',\n",
              "   'Li, Xianyu',\n",
              "   'Sun, Wei',\n",
              "   'Liu, Qiongming',\n",
              "   'Song, Lei',\n",
              "   'Zhen, Bei',\n",
              "   'Zhang, Pumin',\n",
              "   'Qian, Xiaohong',\n",
              "   'Qin, Jun',\n",
              "   'He, Fuchu'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Molecular & cellular proteomics : MCP',\n",
              "  'keywords': ['Animals',\n",
              "   'Gene Ontology',\n",
              "   'Liver',\n",
              "   'Mice',\n",
              "   'Proteome',\n",
              "   'Proteomics',\n",
              "   'Single-Cell Analysis'],\n",
              "  'pmid': '27562671',\n",
              "  'title': 'A Cell-type-resolved Liver Proteome.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'The inference of genetic regulatory networks (GRNs) provides insight into the cellular responses to signals. A class of recurrent neural networks (RNNs) capturing the dynamics of GRN has been used as a basis for inferring small-scale GRNs from gene expression time series. The Bayesian framework facilitates incorporating the hypothesis of GRN into the model estimation to improve the accuracy of GRN inference.|We present new methods for inferring small-scale GRNs based on RNNs. The weights of wires of RNN represent the strengths of gene-to-gene regulatory interactions. We use a class of automatic relevance determination (ARD) priors to enforce the sparsity in the maximum a posteriori (MAP) estimates of wire weights of RNN. A particle swarm optimization (PSO) is integrated as an optimization engine into the MAP estimation process. Likely networks of genes generated based on estimated wire weights are combined using the majority rule to determine a final estimated GRN. As an alternative, a class of <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:msub><mml:mrow><mml:mi>L</mml:mi></mml:mrow><mml:mrow><mml:mi>q</mml:mi></mml:mrow></mml:msub></mml:math> -norm ( <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:mi>q</mml:mi><mml:mo>=</mml:mo><mml:mn>1</mml:mn></mml:math> ) priors is used for attaining the sparse MAP estimates of wire weights of RNN. We also infer the GRN using the maximum likelihood (ML) estimates of wire weights of RNN. The RNN-based GRN inference algorithms, ARD-RNN, <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:msub><mml:mrow><mml:mi>L</mml:mi></mml:mrow><mml:mrow><mml:mi>q</mml:mi></mml:mrow></mml:msub></mml:math> -RNN, and ML-RNN are tested on simulated and experimental E. coli and yeast time series containing 6-11 genes and 7-19 data points. Published GRN inference algorithms based on regressions and mutual information networks are performed on the benchmark datasets to compare performances.|ARD and <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:msub><mml:mrow><mml:mi>L</mml:mi></mml:mrow><mml:mrow><mml:mi>q</mml:mi></mml:mrow></mml:msub></mml:math> -norm priors are used for the estimation of wire weights of RNN. Results of GRN inference experiments show that ARD-RNN, <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:msub><mml:mrow><mml:mi>L</mml:mi></mml:mrow><mml:mrow><mml:mi>q</mml:mi></mml:mrow></mml:msub></mml:math> -RNN have similar best accuracies on the simulated time series. The ARD-RNN is more accurate than <mml:math xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><mml:msub><mml:mrow><mml:mi>L</mml:mi></mml:mrow><mml:mrow><mml:mi>q</mml:mi></mml:mrow></mml:msub></mml:math> -RNN, ML-RNN, and mostly more accurate than the reference algorithms on the experimental time series. The effectiveness of ARD-RNN for inferring small-scale GRNs using gene expression time series of limited length is empirically verified.',\n",
              "  'authors': ['Chen, Chi-Kan'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Bayes Theorem',\n",
              "   'Cell Cycle',\n",
              "   'DNA Repair',\n",
              "   'Escherichia coli',\n",
              "   'Escherichia coli Proteins',\n",
              "   'Gene Expression',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Interrupted Time Series Analysis',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Yeasts'],\n",
              "  'pmid': '30051742',\n",
              "  'title': 'Inference of gene networks from gene expression time series using recurrent neural networks and sparse MAP estimation.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'Whole-body positron emission tomography/computed tomography (PET/CT) has become a standard method of imaging patients with various disease conditions, especially cancer. Body-wide accurate quantification of disease burden in PET/CT images is important for characterizing lesions, staging disease, prognosticating patient outcome, planning treatment, and evaluating disease response to therapeutic interventions. However, body-wide anatomy recognition in PET/CT is a critical first step for accurately and automatically quantifying disease body-wide, body-region-wise, and organwise. This latter process, however, has remained a challenge due to the lower quality of the anatomic information portrayed in the CT component of this imaging modality and the paucity of anatomic details in the PET component. In this paper, the authors demonstrate the adaptation of a recently developed automatic anatomy recognition (AAR) methodology [Udupa et al., \"Body-wide hierarchical fuzzy modeling, recognition, and delineation of anatomy in medical images,\" Med. Image Anal. 18, 752-771 (2014)] to PET/CT images. Their goal was to test what level of object localization accuracy can be achieved on PET/CT compared to that achieved on diagnostic CT images.|The authors advance the AAR approach in this work in three fronts: (i) from body-region-wise treatment in the work of Udupa et al. to whole body; (ii) from the use of image intensity in optimal object recognition in the work of Udupa et al. to intensity plus object-specific texture properties, and (iii) from the intramodality model-building-recognition strategy to the intermodality approach. The whole-body approach allows consideration of relationships among objects in different body regions, which was previously not possible. Consideration of object texture allows generalizing the previous optimal threshold-based fuzzy model recognition method from intensity images to any derived fuzzy membership image, and in the process, to bring performance to the level achieved on diagnostic CT and MR images in body-region-wise approaches. The intermodality approach fosters the use of already existing fuzzy models, previously created from diagnostic CT images, on PET/CT and other derived images, thus truly separating the modality-independent object assembly anatomy from modality-specific tissue property portrayal in the image.|Key ways of combining the above three basic ideas lead them to 15 different strategies for recognizing objects in PET/CT images. Utilizing 50 diagnostic CT image data sets from the thoracic and abdominal body regions and 16 whole-body PET/CT image data sets, the authors compare the recognition performance among these 15 strategies on 18 objects from the thorax, abdomen, and pelvis in object localization error and size estimation error. Particularly on texture membership images, object localization is within three voxels on whole-body low-dose CT images and 2 voxels on body-region-wise low-dose images of known true locations. Surprisingly, even on direct body-region-wise PET images, localization error within 3 voxels seems possible.|The previous body-region-wise approach can be extended to whole-body torso with similar object localization performance. Combined use of image texture and intensity property yields the best object localization accuracy. In both body-region-wise and whole-body approaches, recognition performance on low-dose CT images reaches levels previously achieved on diagnostic CT images. The best object recognition strategy varies among objects; the proposed framework however allows employing a strategy that is optimal for each object.',\n",
              "  'authors': ['Wang, Huiqian',\n",
              "   'Udupa, Jayaram K',\n",
              "   'Odhner, Dewey',\n",
              "   'Tong, Yubing',\n",
              "   'Zhao, Liming',\n",
              "   'Torigian, Drew A'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Abdomen',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Automation',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Radiography, Abdominal',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Torso',\n",
              "   'Whole Body Imaging'],\n",
              "  'pmid': '26745953',\n",
              "  'title': 'Automatic anatomy recognition in whole-body PET/CT images.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Long-lasting forms of synaptic plasticity that underlie learning and memory require new transcription and translation for their persistence. The remarkable polarity and compartmentalization of neurons raises questions about the spatial and temporal regulation of gene expression within neurons. Alternative cleavage and polyadenylation (APA) generates mRNA isoforms with different 3' untranslated regions (3'UTRs) and/or coding sequences. Changes in the 3'UTR composition of mRNAs can alter gene expression by regulating transcript localization, stability and/or translation, while changes in the coding sequences lead to mRNAs encoding distinct proteins. Using specialized 3' end deep sequencing methods, we undertook a comprehensive analysis of APA following induction of long-term potentiation (LTP) of mouse hippocampal CA3-CA1 synapses. We identified extensive LTP-induced APA changes, including a general trend of 3'UTR shortening and activation of intronic APA isoforms. Comparison with transcriptome profiling indicated that most APA regulatory events were uncoupled from changes in transcript abundance. We further show that specific APA regulatory events can impact expression of two molecules with known functions during LTP, including 3'UTR APA of Notch1 and intronic APA of Creb1. Together, our results reveal that activity-dependent APA provides an important layer of gene regulation during learning and memory.\",\n",
              "  'authors': ['Fontes, Mariana M',\n",
              "   'Guvenek, Aysegul',\n",
              "   'Kawaguchi, Riki',\n",
              "   'Zheng, Dinghai',\n",
              "   'Huang, Alden',\n",
              "   'Ho, Victoria M',\n",
              "   'Chen, Patrick B',\n",
              "   'Liu, Xiaochuan',\n",
              "   \"O'Dell, Thomas J\",\n",
              "   'Coppola, Giovanni',\n",
              "   'Tian, Bin',\n",
              "   'Martin, Kelsey C'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Animals',\n",
              "   'Cyclic AMP Response Element-Binding Protein',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Hippocampus',\n",
              "   'Long-Term Potentiation',\n",
              "   'Male',\n",
              "   'Mice',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Polyadenylation',\n",
              "   'RNA, Messenger',\n",
              "   'Receptor, Notch1'],\n",
              "  'pmid': '29234016',\n",
              "  'title': 'Activity-Dependent Regulation of Alternative Cleavage and Polyadenylation During Hippocampal Long-Term Potentiation.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1551-0018',\n",
              "  'IF_2013': '0.866',\n",
              "  'IF_2014': '0.84',\n",
              "  'IF_2015': '1.006',\n",
              "  'IF_2016': '1.035',\n",
              "  'IF_2017': '1.23',\n",
              "  'ISSN': '1547-1063',\n",
              "  'ISSN_online': '1551-0018',\n",
              "  'ISSN_print': '1547-1063',\n",
              "  'IsoAbbr': 'Math Biosci Eng',\n",
              "  'JrId': 33312,\n",
              "  'MedAbbr': 'Math Biosci Eng',\n",
              "  'NLMid': '101197794',\n",
              "  'Titre': 'Math Biosci Eng',\n",
              "  'abstract': 'Due to both the hidden nature and the irreversibility of Alzheimers disease (AD), it has become the killer of the elderly and is thus the focus of much attention in the medical field. Radiologists compare the predicted brain age with the ground truth in order to provide a preliminary analysis of AD, which helps doctors diagnose AD as early in its development as possible. In this paper, a transfer learning-based method of predicting brain age using MR images and dataset of a public brain was proposed. In order to get the best transfer results, we froze different layers and only fine-tuned the remaining layers. We used three planes of brain MR images together to predict age for the first time and experiment results showed that the proposed method performs better than the state-of-the-art method under mean absolute error metric by 0.6 years. In addition, to explore the relationship between brain MR images of different planes and predicted age accuracy, we used three different planes of brain MR images to predict age respectively for the first time and found that sagittal plane MR images outperformed two other planes in age estimation. Finally, our research identified, the effective regions that contribute to brain age estimation for cognitively normal individuals and for AD patients with deep learning. For AD patients, the effective region is mainly concentrated in the frontal lobe of the brain, verifying the relevant medical conclusions about AD.',\n",
              "  'authors': ['Jiang, Hai Tao',\n",
              "   'Guo, Jia Jia',\n",
              "   'Du, Hong Wei',\n",
              "   'Xu, Jin Zhang',\n",
              "   'Qiu, Ben Sheng'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Mathematical biosciences and engineering : MBE',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31499667',\n",
              "  'title': 'Transfer learning on T1-weighted images for brain age estimation.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1744-8042',\n",
              "  'IF_2013': '3.425',\n",
              "  'IF_2014': '3.218',\n",
              "  'IF_2015': '2.71',\n",
              "  'IF_2016': '2.35',\n",
              "  'IF_2017': '2.302',\n",
              "  'ISSN': '1462-2416',\n",
              "  'ISSN_online': '1744-8042',\n",
              "  'ISSN_print': '1462-2416',\n",
              "  'IsoAbbr': 'Pharmacogenomics',\n",
              "  'JrId': 22212,\n",
              "  'MedAbbr': 'Pharmacogenomics',\n",
              "  'NLMid': '100897350',\n",
              "  'Titre': 'Pharmacogenomics',\n",
              "  'abstract': 'The inference of genetic interactions from measured expression data is one of the most challenging tasks of modern functional genomics. When successful, the learned network of regulatory interactions yields a wealth of useful information. An inferred genetic network contains information about the pathway to which a gene belongs and which genes it interacts with. Furthermore, it explains the function of the gene in terms of how it influences other genes and indicates which genes are pathway initiators and therefore potential drug targets. Obviously, such wealth comes at a price and that of genetic network modeling is that it is an extremely complex task. Therefore, it is necessary to develop sophisticated computational tools that are able to extract relevant information from a limited set of microarray measurements and integrate this with different information sources, to come up with reliable hypotheses of a genetic regulatory network. Thus far, a multitude of modeling approaches have been proposed for discovering genetic networks. However, it is unclear what the advantages and disadvantages of each of the different approaches are and how their results can be compared. In this review, genetic network models are put in a historical perspective that explains why certain models were introduced. Various modeling assumptions and their consequences are also highlighted. In addition, an overview of the principal differences and similarities between the approaches is given by considering the qualitative properties of the chosen models and their learning strategies.',\n",
              "  'authors': ['van Someren, E P',\n",
              "   'Wessels, L F A',\n",
              "   'Backer, E',\n",
              "   'Reinders, M J T'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Pharmacogenomics',\n",
              "  'keywords': ['Gene Expression Regulation',\n",
              "   'Genetic Research',\n",
              "   'History, 20th Century',\n",
              "   'History, 21st Century',\n",
              "   'Humans',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '12164774',\n",
              "  'title': 'Genetic network modeling.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VE'],\n",
              "  'ESSN': '1745-1701',\n",
              "  'IF_2013': '8.607',\n",
              "  'IF_2014': '8.45',\n",
              "  'IF_2015': '7.757',\n",
              "  'IF_2016': '7.575',\n",
              "  'IF_2017': '6.944',\n",
              "  'ISSN': '0586-7614',\n",
              "  'ISSN_online': '1745-1701',\n",
              "  'ISSN_print': '0586-7614',\n",
              "  'IsoAbbr': 'Schizophr Bull',\n",
              "  'JrId': 7435,\n",
              "  'MedAbbr': 'Schizophr Bull',\n",
              "  'NLMid': '0236760',\n",
              "  'Titre': 'Schizophr Bull',\n",
              "  'abstract': 'Modern techniques have been applied to brain modeling, based on recent approaches in the artificial intelligence field that use brain-like \"connectionistic\" computational architectures. The model proposed by Cohen and Servan-Schreiber uses a gain parameter which they identify with dopamine function. They apply their model to neuroleptically treated schizophrenia patients who show improved task performance which they link to increased dopamine function and increased gain in the prefrontal cortex. However, evidence indicates that antipsychotic medications block dopamine (especially D2) receptors, decreasing mesolimbic and mesocortical dopamine function. If therapeutic dosages of neuroleptics diminish dopamine function, this would decrease gain in context modules needed for adequate task performance. Schizophrenia patients would perform more poorly by further reducing gain in their already compromised context modules. The current investigators suggest three possible ways to resolve this difficulty, to explain why normals perform more poorly when taking neuroleptics, although acute schizophrenia patients\\' performance may be enhanced in several areas. Evidence would suggest that multiple processes occur simultaneously in neuroleptically treated patients with some processes counterbalancing others.',\n",
              "  'authors': ['Jobe, T H',\n",
              "   'Harrow, M',\n",
              "   'Martin, E M',\n",
              "   'Whitfield, H J',\n",
              "   'Sands, J R'],\n",
              "  'categories': ['PSYCHIATRY'],\n",
              "  'journal': 'Schizophrenia bulletin',\n",
              "  'keywords': ['Antipsychotic Agents',\n",
              "   'Attention',\n",
              "   'Dopamine',\n",
              "   'Dopamine D2 Receptor Antagonists',\n",
              "   'Humans',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neural Pathways',\n",
              "   'Neuropsychological Tests',\n",
              "   'Prefrontal Cortex',\n",
              "   'Psychomotor Performance',\n",
              "   'Receptors, Dopamine D2',\n",
              "   'Reference Values',\n",
              "   'Schizophrenia',\n",
              "   'Schizophrenic Psychology'],\n",
              "  'pmid': '7973463',\n",
              "  'title': 'Schizophrenic deficits: neuroleptics and the prefrontal cortex.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1534-6285',\n",
              "  'IF_2013': '1.513',\n",
              "  'IF_2014': '1.513',\n",
              "  'IF_2015': '1.597',\n",
              "  'IF_2016': '2.124',\n",
              "  'IF_2017': '1.983',\n",
              "  'ISSN': '1527-2737',\n",
              "  'ISSN_online': '1534-6285',\n",
              "  'ISSN_print': '1527-2737',\n",
              "  'IsoAbbr': 'Curr Urol Rep',\n",
              "  'JrId': 29748,\n",
              "  'MedAbbr': 'Curr Urol Rep',\n",
              "  'NLMid': '100900943',\n",
              "  'Titre': 'Curr Urol Rep',\n",
              "  'abstract': 'The introduction of laparoscopy and robotic surgery revolutionized the surgical management of urologic patients. Nonetheless, we live in an era of rapid changes, and we are probably still in the infancy of technology applied to surgery. When considering currently available technologies, there are several unmet needs to be addressed. These include the application of augmented reality, haptic feedback, tissue recognition, distant remote control, miniaturization of surgical instruments, the learning curve typical of the introduction of novel techniques, and excessive costs. In the next few years, evolution in imaging modalities in pre- and intraoperative surgical planning, as well as the introduction of novel minimally invasive platforms, would in part address these issues, substantially improving surgical outcomes. In addition, validated training programs would allow for the safe implementation of novel techniques in the clinical practice. Finally, a reduction in costs would be necessary to make technology affordable and to optimize healthcare resources. ',\n",
              "  'authors': ['Gandaglia, Giorgio',\n",
              "   'Schatteman, Peter',\n",
              "   'De Naeyer, Geert',\n",
              "   \"D'Hondt, Frederiek\",\n",
              "   'Mottrie, Alexandre'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current urology reports',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Urologic Diseases',\n",
              "   'Urologic Surgical Procedures'],\n",
              "  'pmid': '26874531',\n",
              "  'title': 'Novel Technologies in Urologic Surgery: a Rapidly Changing Scenario.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1742-5662',\n",
              "  'IF_2013': '3.856',\n",
              "  'IF_2014': '3.917',\n",
              "  'IF_2015': '3.818',\n",
              "  'IF_2016': '3.579',\n",
              "  'IF_2017': '3.355',\n",
              "  'ISSN': '1742-5689',\n",
              "  'ISSN_online': '1742-5662',\n",
              "  'ISSN_print': '1742-5689',\n",
              "  'IsoAbbr': 'J R Soc Interface',\n",
              "  'JrId': 32845,\n",
              "  'MedAbbr': 'J R Soc Interface',\n",
              "  'NLMid': '101217269',\n",
              "  'Titre': 'J R Soc Interface',\n",
              "  'abstract': 'The interplay of mathematical modelling with experiments is one of the central elements in systems biology. The aim of reverse engineering is to infer, analyse and understand, through this interplay, the functional and regulatory mechanisms of biological systems. Reverse engineering is not exclusive of systems biology and has been studied in different areas, such as inverse problem theory, machine learning, nonlinear physics, (bio)chemical kinetics, control theory and optimization, among others. However, it seems that many of these areas have been relatively closed to outsiders. In this contribution, we aim to compare and highlight the different perspectives and contributions from these fields, with emphasis on two key questions: (i) why are reverse engineering problems so hard to solve, and (ii) what methods are available for the particular problems arising from systems biology? ',\n",
              "  'authors': ['Villaverde, Alejandro F', 'Banga, Julio R'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Journal of the Royal Society, Interface',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Automation',\n",
              "   'Bayes Theorem',\n",
              "   'Bioengineering',\n",
              "   'Biophysics',\n",
              "   'Markov Chains',\n",
              "   'Models, Biological',\n",
              "   'Models, Statistical',\n",
              "   'Monte Carlo Method',\n",
              "   'Systems Biology'],\n",
              "  'pmid': '24307566',\n",
              "  'title': 'Reverse engineering and identification in systems biology: strategies, perspectives and challenges.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1875-5550',\n",
              "  'IF_2013': '2.328',\n",
              "  'IF_2014': '3.154',\n",
              "  'IF_2015': '2.441',\n",
              "  'IF_2016': '2.576',\n",
              "  'IF_2017': '2.696',\n",
              "  'ISSN': '1389-2037',\n",
              "  'ISSN_online': '1875-5550',\n",
              "  'ISSN_print': '1389-2037',\n",
              "  'IsoAbbr': 'Curr. Protein Pept. Sci.',\n",
              "  'JrId': 29991,\n",
              "  'MedAbbr': 'Curr Protein Pept Sci',\n",
              "  'NLMid': '100960529',\n",
              "  'Titre': 'Curr. Protein Pept. Sci.',\n",
              "  'abstract': 'Efforts to use computers in predicting the secondary structure of proteins based only on primary structure information started over a quarter century ago [1-3]. Although the results were encouraging initially, the accuracy of the pioneering methods generally did not attain the level required for using predictions of secondary structures reliably in modelling the three-dimensional topology of proteins. During the last decade, however, the introduction of new computational techniques as well as the use of multiple sequence information has lead to a dramatic increase in the success rate of prediction methods, such that successful 3D modelling based on predicted secondary structure has become feasible [e.g., Ref 4]. This review is aimed at presenting an overview of the scale of the secondary structure prediction problem and associated pitfalls, as well as the history of the development of computational prediction methods. As recent successful strategies for secondary structure prediction all rely on multiple sequence information, some methods for accurate protein multiple sequence alignments will also be described. While the main focus is on prediction methods for globular proteins, also the prediction of trans-membrane segments within membrane proteins will be briefly summarised. Finally, an integrated iterative approach tying secondary structure prediction and multiple alignment will be introduced [5].',\n",
              "  'authors': ['Heringa, J'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Current protein & peptide science',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Sequence',\n",
              "   'Bacterial Proteins',\n",
              "   'Computational Biology',\n",
              "   'Consensus Sequence',\n",
              "   'Discriminant Analysis',\n",
              "   'Flavodoxin',\n",
              "   'Membrane Proteins',\n",
              "   'Methyl-Accepting Chemotaxis Proteins',\n",
              "   'Models, Molecular',\n",
              "   'Molecular Sequence Data',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Folding',\n",
              "   'Protein Structure, Secondary',\n",
              "   'Protein Structure, Tertiary',\n",
              "   'Sequence Alignment',\n",
              "   'Sequence Homology, Amino Acid'],\n",
              "  'pmid': '12369910',\n",
              "  'title': 'Computational methods for protein secondary structure prediction using multiple sequence alignments.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1534-4908',\n",
              "  'IF_2013': '0.938',\n",
              "  'IF_2014': '1.14',\n",
              "  'IF_2015': '1.019',\n",
              "  'IF_2016': '1.02',\n",
              "  'IF_2017': '0.986',\n",
              "  'ISSN': '1530-4515',\n",
              "  'ISSN_online': '1534-4908',\n",
              "  'ISSN_print': '1530-4515',\n",
              "  'IsoAbbr': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'JrId': 21595,\n",
              "  'MedAbbr': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'NLMid': '100888751',\n",
              "  'Titre': 'Surg Laparosc Endosc Percutan Tech',\n",
              "  'abstract': 'Previous comparisons between single-port laparoscopic appendectomy (SPLA) and multi-port laparoscopic appendectomy have been conflicting and limited. We compare our single-surgeon, SPLA experience with multi-port cases performed during the same time.|A retrospective chart review of 128 single-surgeon single-port and 941 multi-port laparoscopic appendectomy cases from April 2009 to December 2014 was conducted.|Patient demographics and preoperative laboratory values were comparable. SPLA was associated with shorter operative time (P=0.0001). There was no statistically significant difference in length of hospitalization, postoperative pain medication use, cost, postoperative complication rates (ileus, urinary retention, deep space infection), or readmission between the 2 groups. There were no postoperative incisional hernias in the single-port group. The single-port group had more postoperative oxycodone use (P=0.0110).|Our study supports recently published metaanalyses that fail to support older studies demonstrating longer operative times, and higher hernia rates with SPLA.',\n",
              "  'authors': ['Assali, Sarah',\n",
              "   'Eapen, Sarah',\n",
              "   'Carman, Terry',\n",
              "   'Horattas, Sophia',\n",
              "   'Daigle, Christopher R',\n",
              "   'Paranjape, Charudutt'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical laparoscopy, endoscopy & percutaneous techniques',\n",
              "  'keywords': ['Adult',\n",
              "   'Analgesics, Opioid',\n",
              "   'Appendectomy',\n",
              "   'Appendicitis',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Learning Curve',\n",
              "   'Male',\n",
              "   'Operative Time',\n",
              "   'Pain, Postoperative',\n",
              "   'Patient Readmission',\n",
              "   'Postoperative Complications',\n",
              "   'Retrospective Studies',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '29847482',\n",
              "  'title': 'Single-port Laparoscopic Appendectomy: Beyond the Learning Curve: A Retrospective Comparison With Multi-port Laparoscopic Appendectomy.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': 'To report the feasibility of a novel robotic technique for intracorporeal construction of a double Yang-Monti ileal conduit with bladder augmentation in an adult patient.|The Yang-Monti procedure can be considered as an alternative to Mitrofanoff appendicovesicostomy when the appendix is not available or in obese patients. To date, the robotic approach has not been described. We have described all the steps of our surgery, performed on a 35-year-old obese patient with post-traumatic paraplegia and a neurogenic bladder. We focused on the technical tips and highlighted all the differences with the standard open technique.|Our technique provided satisfactory results. No intraoperative complications occurred. The operative time was 250 minutes, and the intraoperative blood loss was 200 mL. The clinical course was uneventful.|We have shown that robotic intracorporeal realization of a double Yang-Monti ileal conduit with bladder augmentation is feasible, extending the potential indications of robot-assisted surgery. The evaluation of more cases is advised.',\n",
              "  'authors': ['Rey, Denis',\n",
              "   'Oderda, Marco',\n",
              "   'El Helou, Elie',\n",
              "   'Robbiani, Jacopo',\n",
              "   'Lopez, Laurent',\n",
              "   'Piechaud, Pierre-Thierry'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Obesity',\n",
              "   'Paraplegia',\n",
              "   'Robotics',\n",
              "   'Urinary Bladder',\n",
              "   'Urinary Bladder, Neurogenic',\n",
              "   'Urinary Diversion',\n",
              "   'Urinary Reservoirs, Continent'],\n",
              "  'pmid': '23706553',\n",
              "  'title': 'Feasibility of robotic double Yang-Monti ileal conduit with bladder augmentation: surgical technique.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['HY'],\n",
              "  'ESSN': '1477-9129',\n",
              "  'IF_2013': '6.273',\n",
              "  'IF_2014': '6.462',\n",
              "  'IF_2015': '6.059',\n",
              "  'IF_2016': '5.843',\n",
              "  'IF_2017': '5.413',\n",
              "  'ISSN': '0950-1991',\n",
              "  'ISSN_online': '1477-9129',\n",
              "  'ISSN_print': '0950-1991',\n",
              "  'IsoAbbr': 'Development',\n",
              "  'JrId': 3468,\n",
              "  'MedAbbr': 'Development',\n",
              "  'NLMid': '8701744',\n",
              "  'Titre': 'Development',\n",
              "  'abstract': \"In November 2016, developmental biologists, synthetic biologists and engineers gathered in Paris for a meeting called 'Engineering the embryo'. The participants shared an interest in exploring how synthetic systems can reveal new principles of embryonic development, and how the in vitro manipulation and modeling of development using stem cells can be used to integrate ideas and expertise from physics, developmental biology and tissue engineering. As we review here, the conference pinpointed some of the challenges arising at the intersection of these fields, along with great enthusiasm for finding new approaches and collaborations.\",\n",
              "  'authors': ['Kicheva, Anna', 'Rivron, Nicolas C'],\n",
              "  'categories': ['DEVELOPMENTAL BIOLOGY'],\n",
              "  'journal': 'Development (Cambridge, England)',\n",
              "  'keywords': ['Animals',\n",
              "   'Biophysics',\n",
              "   'Developmental Biology',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Humans',\n",
              "   'Mice',\n",
              "   'Models, Biological',\n",
              "   'Paris',\n",
              "   'Robotics',\n",
              "   'Stem Cells',\n",
              "   'Synthetic Biology',\n",
              "   'Systems Biology',\n",
              "   'Tissue Engineering'],\n",
              "  'pmid': '28246208',\n",
              "  'title': 'Creating to understand - developmental biology meets engineering in Paris.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'In recent years, an increasing number of people have become concerned about their health. Most chronic diseases are related to lifestyle, and daily activity records can be used as an important indicator of health. Specifically, using advanced technology to automatically monitor actual activities can effectively prevent and manage chronic diseases. The data used in this paper were obtained from acceleration sensors and gyroscopes integrated in smartphones. We designed an efficient Adaboost-Stump running on a smartphone to classify five common activities: cycling, running, sitting, standing, and walking and achieved a satisfactory classification accuracy of 98%. We designed an online learning method, and the classification model requires continuous training with actual data. The parameters in the model then become increasingly fitted to the specific user, which allows the classification accuracy to reach 95% under different use environments. In addition, this paper also utilized the OpenCL framework to design the program in parallel. This process can enhance the computing efficiency approximately ninefold.',\n",
              "  'authors': ['Li, Pengfei',\n",
              "   'Wang, Yu',\n",
              "   'Tian, Yu',\n",
              "   'Zhou, Tian-Shu',\n",
              "   'Li, Jing-Song'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Actigraphy',\n",
              "   'Algorithms',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Exercise',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Mobile Applications',\n",
              "   'Monitoring, Ambulatory',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Smartphone',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '27249822',\n",
              "  'title': 'An Automatic User-Adapted Physical Activity Classification Method Using Smartphones.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'Clinical under staging occurs in 40% to 60% of patients who undergo radical prostatectomy for prostate cancer. To decrease under staging several methods of predicting pathological stage preoperatively have been developed based on statistical logistic regression analysis and neural networks. To our knowledge none has been validated in our homogeneous regional patient population to date. We created logistic regression and neural network models, and implemented and adapted them into our practice. We also compared the 2 methods to determine their value and practicality in daily clinical practice. We present the results of our novel approach for predicting pathological staging of prostate adenocarcinoma.|Between 1986 and 1999, 600 white men from the Aragon region of Spain underwent surgery for prostate cancer; of whom 468 were selected for study. Predictive study variables included patient age, clinical stage, biopsy Gleason score and preoperative prostate specific antigen (PSA). The predicted result included in analysis was organ confined or nonorgan confined disease. Data were analyzed by multivariate logistic regression and a supervised neural network (multilayer perceptron and radial basis function). Results were compared by comparing the areas under the receiver operating characteristics curves.|We generated 5 logistic regression models. The model created with clinical staging, Gleason biopsy score and PSA distributed in 5 categories (p <0.001) with an area under the receiver operating characteristics curve of 0.840 proved to be most predictive of pathological stage. Similarly of the 6 neural network models evaluated the radial basis function model, which included age, clinical stage, Gleason biopsy score and preoperative PSA distributed in 5 categories with an area under the curve of 0.882, proved the most predictive but not superior to the logistic regression model. The difference in the area under the curves in the 2 chosen models was 0.042 (p = 0.1).|It is possible to generate useful predictive models of organ confined disease using logistic regression or neural networks with high indexes of clinical and statistical validity. However, using these variables neural networks did not prove to be better than logistic regression analysis. Therefore, better predictive variables must be identified, preferably nonlinear characteristics with respect to the probability of organ confined tumor, to generate better predictive models using neural networks.',\n",
              "  'authors': ['Borque, A',\n",
              "   'Sanz, G',\n",
              "   'Allepuz, C',\n",
              "   'Plaza, L',\n",
              "   'Gil, P',\n",
              "   'Rioja, L A'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Humans',\n",
              "   'Logistic Models',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neoplasm Staging',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '11586200',\n",
              "  'title': 'The use of neural networks and logistic regression analysis for predicting pathological stage in men undergoing radical prostatectomy: a population based study.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'We report a comparative analysis of a large series of laparoscopic and robotic partial nephrectomies performed by a high volume single surgeon at a tertiary care institution.|We retrospectively reviewed the medical charts of 500 patients treated with minimally invasive partial nephrectomy by a single surgeon between March 2002 and February 2012. Demographic and perioperative data were collected and statistically analyzed. R.E.N.A.L. (radius, exophytic/endophytic properties, nearness of tumor to the collecting system or sinus in mm, anterior/posterior and location relative to polar lines) nephrometry score was used to score tumors. Those scored as moderate and high complexity were designated as complex. Trifecta was defined as a combination of warm ischemia time less than 25 minutes, negative surgical margins and no perioperative complications.|Two groups were identified, including 261 patients with robotic and 231 with laparoscopic partial nephrectomy. Demographics were similar in the groups. The robotic group was significantly more morbid (Charlson comorbidity index 3.75 vs 1.26), included more complex tumors (R.E.N.A.L. score 5.98 vs 7.2), and had lower operative (169.9 vs 191.7 minutes) and warm ischemia (17.9 vs 25.2 minutes) time, intraoperative (2.6% vs 5.6%, each p <0.001) and postoperative (24.53% vs 32.03%, p = 0.004) complications, and positive margin rate (2.9% vs 5.6%, p <0.001). Thus, a higher overall trifecta rate was observed for robotic partial nephrectomy (58.7% vs 31.6%, p <0.001). The laparoscopic group had longer followup (3.43 vs 1.51 years, p <0.001) and no significant difference in postoperative changes in renal function. Main study limitations were the retrospective nature, arbitrary definition of trifecta and shorter followup in the RPN group.|Our large comparative analysis shows that robotic partial nephrectomy offers a wider range of indications, better operative outcomes and lower perioperative morbidity than laparoscopic partial nephrectomy. Overall, the quest for trifecta seems to be better accomplished by robotic partial nephrectomy, which is likely to become the new standard for minimally invasive partial nephrectomy.',\n",
              "  'authors': ['Khalifeh, Ali',\n",
              "   'Autorino, Riccardo',\n",
              "   'Hillyer, Shahab P',\n",
              "   'Laydner, Humberto',\n",
              "   'Eyraud, Remi',\n",
              "   'Panumatrassamee, Kamol',\n",
              "   'Long, Jean-Alexandre',\n",
              "   'Kaouk, Jihad H'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Humans',\n",
              "   'Kidney Neoplasms',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Outcome Assessment (Health Care)',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Warm Ischemia'],\n",
              "  'pmid': '23079376',\n",
              "  'title': 'Comparative outcomes and assessment of trifecta in 500 robotic and laparoscopic partial nephrectomy cases: a single surgeon experience.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1768-3378',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0003-4401',\n",
              "  'ISSN_online': '1768-3378',\n",
              "  'ISSN_print': '0003-4401',\n",
              "  'IsoAbbr': 'Ann Urol (Paris)',\n",
              "  'JrId': 655,\n",
              "  'MedAbbr': 'Ann Urol (Paris)',\n",
              "  'NLMid': '0212342',\n",
              "  'Titre': 'Ann Urol (Paris)',\n",
              "  'abstract': 'Transabdominal sacrocolpopexy offers an excellent definitive treatment option for patients with high grade vaginal vault prolapse with long-term success rates ranging from 93-99%. However, because it is a transabdominal procedure it is associated with increased morbidity compared with vaginal repairs. We describe a novel minimally invasive technique of vaginal vault prolapse repair and present out initial experience. The surgical technique involves placement of five laparoscopic ports: three for the Da Vinci robot and two for the assistant. A polypropylene mesh is then attached to the sacral promontory and to the vaginal apex using Gortex sutures. At the end of the case, the mesh material is the covered by the peritoneum. We also present our initial experience with this technique in 18 consecutive patients. The analysis focused on complications, urinary continence, patient satisfaction, and morbidity. Follow-up was conducted by provider-patient interview. Twenty-five patients underwent a robotic-assisted laparoscopic sacrocolpopexy at our institution in the past 24 months for severe symptomatic vaginal vault prolapse. 10/25 (40%) underwent a concomitant anti-incontinence procedure. Mean follow-up was 5. (1-12) months and mean age was 66 (47-82) years. Mean total operative time was 3.2 (2.25-4.75) hours. One patient had to be converted to an open procedure secondary to unfavorable anatomy. All but one patient were discharged from the hospital after an overnight stay; one patient left on postoperative day #2. Complications were limited to mild port site infections in two patients, which resolved with oral antibiotic therapy. One patient developed recurrent grade 3 rectocele, but had no evidence of cystocele or enterocele. We present a novel technique for vaginal vault prolapse repair that combines the advantages of open sacrocolpopexy with the decreased morbidity and improved cosmesis of laparoscopic surgery. It is associated with decreased hospital stay, low complication and conversion rates, and high patient satisfaction. While our early experience is encouraging, long-term data is needed to confirm these findings and establish longevity of the repair.',\n",
              "  'authors': ['Elliott, D S', 'Chow, G K'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': \"Annales d'urologie\",\n",
              "  'keywords': ['Colposcopy',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Middle Aged',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Suture Techniques',\n",
              "   'Uterine Prolapse'],\n",
              "  'pmid': '17338498',\n",
              "  'title': '[Management of vaginal vault prolapse repair with robotically-assisted laparoscopic sacrocolpopexy].',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'We describe a classification system for a novel imaging method for arthritic finger joints. The basis of this system is a laser imaging technique which is sensitive to the optical characteristics of finger joint tissue. From the laser images acquired at baseline and follow-up, finger joints can automatically be classified according to whether the inflammatory status has improved or worsened. To perform the classification task, various linear and kernel-based systems were implemented and their performances were compared. Based on the results presented in this paper, we conclude that the laser-based imaging permits a reliable classification of pathological finger joints, making it a sensitive method for detecting arthritic changes.',\n",
              "  'authors': ['Schwaighofer, Anton',\n",
              "   'Tresp, Volker',\n",
              "   'Mayer, Peter',\n",
              "   'Krause, Andreas',\n",
              "   'Beuthan, Jürgen',\n",
              "   'Rost, Helmut',\n",
              "   'Metzger, Georg',\n",
              "   'Müller, Gerhard A',\n",
              "   'Scheel, Alexander K'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Arthritis, Rheumatoid',\n",
              "   'Expert Systems',\n",
              "   'Finger Joint',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Lasers',\n",
              "   'Observer Variation',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '12669994',\n",
              "  'title': 'Classification of rheumatoid joint inflammation based on laser imaging.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'A new method for the discrimination of varieties of apple by means of near infrared spectroscopy (NIRS) was developed. First, principal component analysis (PCA) was used to compress thousands of spectral data into several variables and describe the body of spectra, the analysis suggested that the cumulate reliabilities of PC1 and PC2 (the first two principle components) were more than 98%, and the 2-dimentional plot was drawn with the scores of PC1 and PC2. It appeared to provide the best clustering of the varieties of apple. The loading plot was drawn with PC1 and PC2 through the whole wavelength region. The fingerprint spectra, which were sensitive to the variety of apple, were obtained from the loading plot. The fingerprint spectra were applied as ANN-BP inputs. Seventy five samples from three varieties were selected randomly, then they were used to build discrimination model. This model was used to predict the varieties of 15 unknown samples; the distinguishing rate of 100% was achieved. This model is reliable and practicable. So the present paper could offer a new approach to the fast discrimination of varieties of apple.',\n",
              "  'authors': ['He, Yong', 'Li, Xiao-Li', 'Shao, Yong-Ni'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Malus',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Principal Component Analysis',\n",
              "   'Quality Control',\n",
              "   'Spectroscopy, Near-Infrared'],\n",
              "  'pmid': '16883852',\n",
              "  'title': '[Discrimination of varieties of apple using near infrared spectra based on principal component analysis and artificial neural network model].',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Monoclonal gammopathy of undetermined significance (MGUS) is a plasma cell dyscrasia that can progress to malignant multiple myeloma (MM). Specific molecular biomarkers to classify the MGUS status and discriminate the initial asymptomatic phase of MM have not been identified. We examined the serum peptidome profile of MGUS patients and healthy volunteers using MALDI-TOF mass spectrometry and developed a predictive model for classifying serum samples. The predictive model was built using a support vector machine (SVM) supervised learning method tuned by applying a 20-fold cross-validation scheme. Predicting class labels in a blinded test set containing randomly selected MGUS and healthy control serum samples validated the model. The generalization performance of the predictive model was evaluated by a double cross-validation method that showed 88% average model accuracy, 89% average sensitivity and 86% average specificity. Our model, which classifies unknown serum samples as belonging to either MGUS patients or healthy individuals, can be applied to clinical diagnosis.',\n",
              "  'authors': ['Barceló, Francisca',\n",
              "   'Gomila, Rosa',\n",
              "   'de Paul, Ivan',\n",
              "   'Gili, Xavier',\n",
              "   'Segura, Jaume',\n",
              "   'Pérez-Montaña, Albert',\n",
              "   'Jimenez-Marco, Teresa',\n",
              "   'Sampol, Antonia',\n",
              "   'Portugal, José'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Biomarkers',\n",
              "   'Female',\n",
              "   'Hematologic Tests',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Monoclonal Gammopathy of Undetermined Significance',\n",
              "   'Proteome',\n",
              "   'Quality Control',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Serum',\n",
              "   'Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization',\n",
              "   'Support Vector Machine',\n",
              "   'Young Adult'],\n",
              "  'pmid': '30071092',\n",
              "  'title': 'MALDI-TOF analysis of blood serum proteome can predict the presence of monoclonal gammopathy of undetermined significance.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '1556-3669',\n",
              "  'IF_2013': '1.544',\n",
              "  'IF_2014': '1.668',\n",
              "  'IF_2015': '1.791',\n",
              "  'IF_2016': '2.031',\n",
              "  'IF_2017': '2.165',\n",
              "  'ISSN': '1530-5627',\n",
              "  'ISSN_online': '1556-3669',\n",
              "  'ISSN_print': '1530-5627',\n",
              "  'IsoAbbr': 'Telemed J E Health',\n",
              "  'JrId': 21916,\n",
              "  'MedAbbr': 'Telemed J E Health',\n",
              "  'NLMid': '100959949',\n",
              "  'Titre': 'Telemed J E Health',\n",
              "  'abstract': 'There exists a great clinical need for improving specialist consultation and utilization of echocardiography in areas remote from hospital-based care. This paper presents the development and first technical assessment of a concept of cardiovascular consultation utilizing long distance, real-time echocardiography as a diagnostic tool in rural areas. The development of CARdiological consultation at a DISTance (CARDISTA) was achieved in three stages, comprising tests of different broadband infrastructures, videoconference systems, microphones, cameras, monitors, and loudspeakers. The CARDISTA concept includes a cardiologist and a sonographer, a robotic arm (Medirob), a portable ultrasound machine, and presently available information technology using an advanced broadband backbone. The three stages provided, with some remaining doubts, echocardiographic examination at a distance comparable to hospital-based examinations. A continuous broadband capacity of 20 megabits per second (Mbps) seemed to be a vital component of CARDISTA for achieving the highest-quality imaging. With this broadband capacity, it was possible to achieve a transmission delay below 200 ms. The technical tests of the CARDISTA concept revealed promising results in enabling long distance real-time echocardiography for specialist consultation. CARDISTA is now ready for clinical testing and evaluation in rural areas for patients with heart diseases, especially heart failure.',\n",
              "  'authors': ['Boman, Kurt',\n",
              "   'Olofsson, Mona',\n",
              "   'Forsberg, Johan',\n",
              "   'Boström, Sven-Ake'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Telemedicine journal and e-health : the official journal of the American Telemedicine Association',\n",
              "  'keywords': ['Humans',\n",
              "   'Robotics',\n",
              "   'Rural Population',\n",
              "   'Teleradiology',\n",
              "   'Time Factors'],\n",
              "  'pmid': '19292622',\n",
              "  'title': 'Remote-controlled robotic arm for real-time echocardiography: the diagnostic future for patients in rural areas?',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1473-6586',\n",
              "  'IF_2013': '2.115',\n",
              "  'IF_2014': '2.333',\n",
              "  'IF_2015': '2.006',\n",
              "  'IF_2016': '1.796',\n",
              "  'IF_2017': '1.83',\n",
              "  'ISSN': '0963-0643',\n",
              "  'ISSN_online': '1473-6586',\n",
              "  'ISSN_print': '0963-0643',\n",
              "  'IsoAbbr': 'Curr Opin Urol',\n",
              "  'JrId': 20614,\n",
              "  'MedAbbr': 'Curr Opin Urol',\n",
              "  'NLMid': '9200621',\n",
              "  'Titre': 'Curr Opin Urol',\n",
              "  'abstract': 'Despite increasing laparoscopic expertise in reconstructive surgery, open procedures still represent the gold standard. Robot-assisted techniques increasingly replace laparoscopy. However, laparoscopy is also developing: by improvement of ergonomics, new instruments, and techniques further reducing access trauma. We evaluated the actual role of laparoscopy focusing on main indications of urologic reconstructive surgery.|We analysed the current literature (PubMed/Medline) concerning indications, perioperative results, complications, and long-term outcome of laparoscopy for pyeloplasty, ureteral reimplantation, stone surgery, management of vesico-vaginal fistula, sacrocolpopexy (including evidence level). For all indications, laparoscopy provides the advantages of less postoperative pain, blood loss, shorter convalescence, and minimal disfigurement. However, it requires expertise with endoscopic suturing. Most experience (N > 1000) exists with laparoscopic pyeloplasty and sacrocolpopexy which can be considered as valuable options (IIB). Concerning ureteral reimplantation and repair of vesico-vaginal fistula, only a limited number of cases were reported (N < 150) (III). Laparoscopic stone surgery may gain importance particularly in developing countries. Robot-assistance will definitively increase the application of laparoscopic techniques providing optimal ergonomics, whereas the role of single-port surgery will be limited.|Laparoscopy will increasingly be used for reconstructive urologic surgery. This trend will be supported by the widespread use of the DaVinci device.',\n",
              "  'authors': ['Rassweiler, Jens',\n",
              "   'Pini, Giovannalberto',\n",
              "   'Gözen, Ali S',\n",
              "   'Klein, Jan',\n",
              "   'Teber, Dogu'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current opinion in urology',\n",
              "  'keywords': ['Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Robotics',\n",
              "   'Ureter',\n",
              "   'Urinary Calculi',\n",
              "   'Urologic Surgical Procedures',\n",
              "   'Vaginal Fistula'],\n",
              "  'pmid': '20871405',\n",
              "  'title': 'Role of laparoscopy in reconstructive surgery.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1526-9914',\n",
              "  'IF_2013': '1.11',\n",
              "  'IF_2014': '1.172',\n",
              "  'IF_2015': '1.444',\n",
              "  'IF_2016': '1.338',\n",
              "  'IF_2017': '1.301',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1526-9914',\n",
              "  'IsoAbbr': 'J Appl Clin Med Phys',\n",
              "  'JrId': 22471,\n",
              "  'MedAbbr': 'J Appl Clin Med Phys',\n",
              "  'NLMid': '101089176',\n",
              "  'Titre': 'J Appl Clin Med Phys',\n",
              "  'abstract': \"The purpose of this paper is to evaluate the dosimetric capabilities and the radiation protection (RP) performance of a new automatic needle-loading system for permanent prostate implants, the Isoloader (Mentor Corp.). The unit has been used in more than 100 clinical cases at our institution. The Isoloader is a computerized workstation that allows automated seed testing by a solid-state CdZnTe radiation detector and loading in surgical needles. The seeds are received in a shielded and ready-to-use cartridge. Radiation protection measurements were done on a cartridge filled with 67 (125)I seeds and during dosimetric seed verification and needle loading. The reproducibility of the detector was tested and its accuracy was determined by comparison to specified activities of six calibration seeds and to their measurements in a calibrated well-chamber (WC). Finally, the times required to complete dosimetric verification and needle loading were evaluated. The cartridge was found to be adequately shielded, since no significant amount of radiation was detected around it. Radiation during seed assay was found to be worst at the cartridge's bottom, where it has a value of 15.2 microSv/h (1.4 microSv/h at 10 cm). For the needle-loading task, measurements were performed with a typical needle (three seeds) at the shielded needle holder surface yielding 307.2 microSv/h (8.3 microSv/h at 20 cm). Seed dosimetric verification takes an average of 15 s/seed, while it takes a mean time of 50 s/needle to complete the loading task. Measurements of the six seed activities were within 0.65% of the ordered activities and 1.9% higher on average than those from the WC (min = 0.7%; max = 3.5%). The reproducibility of the measurements of the CdZnTe detector was excellent, with an average of 0.01% of deviation from a reference measurement (N = 120; = 1.9%). We therefore conclude that the Isoloader is a safe, fast, and effective needle-loading system.\",\n",
              "  'authors': ['Morrier, Janelle',\n",
              "   'Varfalvy, Nicolas',\n",
              "   'Chrétien, Mario',\n",
              "   'Beaulieu, Luc'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of applied clinical medical physics',\n",
              "  'keywords': ['Brachytherapy',\n",
              "   'Equipment Design',\n",
              "   'Equipment Failure Analysis',\n",
              "   'Injections',\n",
              "   'Iodine Radioisotopes',\n",
              "   'Needles',\n",
              "   'Radiation Dosage',\n",
              "   'Radiation Monitoring',\n",
              "   'Radiation Protection',\n",
              "   'Risk Assessment',\n",
              "   'Risk Factors',\n",
              "   'Robotics',\n",
              "   'Specimen Handling'],\n",
              "  'pmid': '15738915',\n",
              "  'title': 'Evaluation of an automatic needle-loading system.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1089-8638',\n",
              "  'IF_2013': '3.959',\n",
              "  'IF_2014': '4.333',\n",
              "  'IF_2015': '4.517',\n",
              "  'IF_2016': '4.632',\n",
              "  'IF_2017': '4.894',\n",
              "  'ISSN': '0022-2836',\n",
              "  'ISSN_online': '1089-8638',\n",
              "  'ISSN_print': '0022-2836',\n",
              "  'IsoAbbr': 'J. Mol. Biol.',\n",
              "  'JrId': 4967,\n",
              "  'MedAbbr': 'J Mol Biol',\n",
              "  'NLMid': '2985088R',\n",
              "  'Titre': 'J. Mol. Biol.',\n",
              "  'abstract': 'Topology-representing neural networks are employed to generate pseudo-atomic structures of large-scale protein assemblies by combining high-resolution data with volumetric data at lower resolution. As an application example, actin monomers and structural subdomains are located in a three-dimensional (3D) image reconstruction from electron micrographs. To test the reliability of the method, the resolution of the atomic model of an actin polymer is lowered to a level typically encountered in electron microscopic reconstructions. The atomic model is restored with a precision nine times the nominal resolution of the corresponding low-resolution density. The presented self-organizing computing method may be used as an information-processing tool for the synthesis of structural data from a variety of biophysical sources.',\n",
              "  'authors': ['Wriggers, W', 'Milligan, R A', 'Schulten, K', 'McCammon, J A'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Journal of molecular biology',\n",
              "  'keywords': ['Actins',\n",
              "   'Algorithms',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Microscopy, Electron',\n",
              "   'Models, Molecular',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Protein Conformation',\n",
              "   'Proteins'],\n",
              "  'pmid': '9878345',\n",
              "  'title': 'Self-organizing neural networks bridge the biomolecular resolution gap.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1098-898X',\n",
              "  'IF_2013': '0.952',\n",
              "  'IF_2014': '1.094',\n",
              "  'IF_2015': '1.141',\n",
              "  'IF_2016': '1.374',\n",
              "  'IF_2017': '1.521',\n",
              "  'ISSN': '1089-7860',\n",
              "  'ISSN_online': '1098-898X',\n",
              "  'ISSN_print': '1089-7860',\n",
              "  'IsoAbbr': 'Semin Musculoskelet Radiol',\n",
              "  'JrId': 21891,\n",
              "  'MedAbbr': 'Semin Musculoskelet Radiol',\n",
              "  'NLMid': '9717520',\n",
              "  'Titre': 'Semin Musculoskelet Radiol',\n",
              "  'abstract': 'Although advancements in the last decade have automated much of the radiology workflow, there are several areas in the complex imaging process where standardization and innovation can be implemented. We discuss multiple tools and integrations that can help improve operational efficiency, quality, and safety.',\n",
              "  'authors': ['Desai, Vishal', 'Flanders, Adam', 'Zoga, Adam C'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Seminars in musculoskeletal radiology',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Biomedical Research',\n",
              "   'Computer Security',\n",
              "   'Decision Support Systems, Clinical',\n",
              "   'Humans',\n",
              "   'Quality Improvement',\n",
              "   'Radiology Department, Hospital',\n",
              "   'Radiology Information Systems',\n",
              "   'Workflow'],\n",
              "  'pmid': '30399617',\n",
              "  'title': 'Leveraging Technology to Improve Radiology Workflow.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': 'Biomedical text mining may target various kinds of valuable information embedded in the literature, but a critical obstacle to the extension of the mining targets is the cost of manual construction of labeled data, which are required for state-of-the-art supervised learning systems. Active learning is to choose the most informative documents for the supervised learning in order to reduce the amount of required manual annotations. Previous works of active learning, however, focused on the tasks of entity recognition and protein-protein interactions, but not on event extraction tasks for multiple event types. They also did not consider the evidence of event participants, which might be a clue for the presence of events in unlabeled documents. Moreover, the confidence scores of events produced by event extraction systems are not reliable for ranking documents in terms of informativity for supervised learning. We here propose a novel committee-based active learning method that supports multi-event extraction tasks and employs a new statistical method for informativity estimation instead of using the confidence scores from event extraction systems.|Our method is based on a committee of two systems as follows: We first employ an event extraction system to filter potential false negatives among unlabeled documents, from which the system does not extract any event. We then develop a statistical method to rank the potential false negatives of unlabeled documents 1) by using a language model that measures the probabilities of the expression of multiple events in documents and 2) by using a named entity recognition system that locates the named entities that can be event arguments (e.g. proteins). The proposed method further deals with unknown words in test data by using word similarity measures. We also apply our active learning method for the task of named entity recognition.|We evaluate the proposed method against the BioNLP Shared Tasks datasets, and show that our method can achieve better performance than such previous methods as entropy and Gibbs error based methods and a conventional committee-based method. We also show that the incorporation of named entity recognition into the active learning for event extraction and the unknown word handling further improve the active learning method. In addition, the adaptation of the active learning method into named entity recognition tasks also improves the document selection for manual annotation of named entities.',\n",
              "  'authors': ['Han, Xu', 'Kim, Jung-jae', 'Kwoh, Chee Keong'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Data Mining',\n",
              "   'Machine Learning',\n",
              "   'Natural Language Processing',\n",
              "   'Statistics as Topic'],\n",
              "  'pmid': '27127603',\n",
              "  'title': 'Active learning for ontological event extraction incorporating named entity recognition and unknown word handling.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.121',\n",
              "  'IF_2014': '0.935',\n",
              "  'IF_2015': '0.791',\n",
              "  'IF_2016': '0.647',\n",
              "  'IF_2017': '0.662',\n",
              "  'ISSN': '0219-6352',\n",
              "  'ISSN_print': '0219-6352',\n",
              "  'IsoAbbr': 'J. Integr. Neurosci.',\n",
              "  'JrId': 30080,\n",
              "  'MedAbbr': 'J Integr Neurosci',\n",
              "  'NLMid': '101156357',\n",
              "  'Titre': 'J. Integr. Neurosci.',\n",
              "  'abstract': 'We first present a method to mathematically build a learning rule for closed-loop neural networks. This rule is then applied to climbing fibers in the cerebellar cortex. Our analytical study is based on previous experimental non-analytical studies, which suggests that climbing fibers carry out an error signal to the brain. Thus, our goal is to find the class of functions for the activity propagated by climbing fibers, allowing the output of the Purkinje cell to converge towards a desired output. These functions must tend towards zero when the objective is reached. Our techniques are generalized to other network models.',\n",
              "  'authors': ['Daya, Bassam', 'Chauvet, Pierre', 'Rammal, Mohamad'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Journal of integrative neuroscience',\n",
              "  'keywords': ['Afferent Pathways',\n",
              "   'Animals',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Mathematics',\n",
              "   'Neural Inhibition',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Purkinje Cells'],\n",
              "  'pmid': '16783867',\n",
              "  'title': 'Mathematical constraints for building learning rules in the Purkinje cell system.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'This paper presents a review of state-of-the-art approaches to automatic extraction of biomolecular events from scientific texts. Events involving biomolecules such as genes, transcription factors, or enzymes, for example, have a central role in biological processes and functions and provide valuable information for describing physiological and pathogenesis mechanisms. Event extraction from biomedical literature has a broad range of applications, including support for information retrieval, knowledge summarization, and information extraction and discovery. However, automatic event extraction is a challenging task due to the ambiguity and diversity of natural language and higher-level linguistic phenomena, such as speculations and negations, which occur in biological texts and can lead to misunderstanding or incorrect interpretation. Many strategies have been proposed in the last decade, originating from different research areas such as natural language processing, machine learning, and statistics. This review summarizes the most representative approaches in biomolecular event extraction and presents an analysis of the current state of the art and of commonly used methods, features, and tools. Finally, current research trends and future perspectives are also discussed. ',\n",
              "  'authors': ['Vanegas, Jorge A',\n",
              "   'Matos, Sérgio',\n",
              "   'González, Fabio',\n",
              "   'Oliveira, José L'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Animals',\n",
              "   'Computational Biology',\n",
              "   'Data Mining',\n",
              "   'Databases, Factual',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Natural Language Processing',\n",
              "   'Systems Biology'],\n",
              "  'pmid': '26587051',\n",
              "  'title': 'An Overview of Biomolecular Event Extraction from Scientific Documents.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1873-4324',\n",
              "  'IF_2013': '4.517',\n",
              "  'IF_2014': '4.513',\n",
              "  'IF_2015': '4.712',\n",
              "  'IF_2016': '4.95',\n",
              "  'IF_2017': '5.123',\n",
              "  'ISSN': '0003-2670',\n",
              "  'ISSN_online': '1873-4324',\n",
              "  'ISSN_print': '0003-2670',\n",
              "  'IsoAbbr': 'Anal. Chim. Acta',\n",
              "  'JrId': 520,\n",
              "  'MedAbbr': 'Anal Chim Acta',\n",
              "  'NLMid': '0370534',\n",
              "  'Titre': 'Anal. Chim. Acta',\n",
              "  'abstract': 'An artificial neural network (ANN) was constructed and trained for the prediction of air-to-blood partition coefficients of volatile organic compounds. The inputs of this neural network are theoretically derived descriptors that were chosen by genetic algorithm (GA) and multiple linear regression (MLR) features selection techniques. These descriptors are: R maximal autocorrelation of lag 1 weighted by atomic Sanderson electronegativities (R1E+), electron density on the most negative atom in molecule (EDNA), maximum partial charge for C atom (MXPCC), surface weighted charge partial surface area (WNSA1), fractional charge partial surface area (FNSA2) and atomic charge weighted partial positive surface area (PPSA3). The standard errors of training, test and validation sets for the ANN model are 0.095, 0.148 and 0.120, respectively. Result obtained showed that nonlinear model can simulate the relationship between structural descriptors and the partition coefficients of the molecules in data set accurately.',\n",
              "  'authors': ['Konoz, Elahe', 'Golmohammadi, Hassan'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytica chimica acta',\n",
              "  'keywords': ['Air',\n",
              "   'Algorithms',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Organic Chemicals',\n",
              "   'Volatilization'],\n",
              "  'pmid': '18558108',\n",
              "  'title': 'Prediction of air-to-blood partition coefficients of volatile organic compounds using genetic algorithm and artificial neural network.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1541-0048',\n",
              "  'IF_2013': '4.229',\n",
              "  'IF_2014': '4.552',\n",
              "  'IF_2015': '4.138',\n",
              "  'IF_2016': '3.858',\n",
              "  'IF_2017': '4.38',\n",
              "  'ISSN': '0090-0036',\n",
              "  'ISSN_online': '1541-0048',\n",
              "  'ISSN_print': '0090-0036',\n",
              "  'IsoAbbr': 'Am J Public Health',\n",
              "  'JrId': 461,\n",
              "  'MedAbbr': 'Am J Public Health',\n",
              "  'NLMid': '1254074',\n",
              "  'Titre': 'Am J Public Health',\n",
              "  'abstract': 'Health disparities and solutions are heterogeneous within and among racial and ethnic groups, yet existing administrative databases lack the granularity to reflect important sociocultural distinctions. We measured the efficacy of a natural-language-processing algorithm to identify a specific immigrant group. The algorithm demonstrated accuracy and precision in identifying Somali patients from the electronic medical records at a single institution. This technology holds promise to identify and track immigrants and refugees in the United States in local health care settings.',\n",
              "  'authors': ['Wieland, Mark L',\n",
              "   'Wu, Stephen T',\n",
              "   'Kaggal, Vinod C',\n",
              "   'Yawn, Barbara P'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'American journal of public health',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Electronic Health Records',\n",
              "   'Health Status Disparities',\n",
              "   'Humans',\n",
              "   'Minnesota',\n",
              "   'Natural Language Processing',\n",
              "   'Refugees',\n",
              "   'Somalia'],\n",
              "  'pmid': '23327237',\n",
              "  'title': 'Tracking health disparities through natural-language processing.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Despite tremendous advances in artificial language synthesis, no machine has so far succeeded in deceiving a human. Most research focused on analyzing the behavior of \"good\" machine. We here choose an opposite strategy, by analyzing the behavior of \"bad\" humans, i.e., humans perceived as machine. The Loebner Prize in Artificial Intelligence features humans and artificial agents trying to convince judges on their humanness via computer-mediated communication. Using this setting as a model, we investigated here whether the linguistic behavior of human subjects perceived as non-human would enable us to identify some of the core parameters involved in the judgment of an agents\\' humanness. We analyzed descriptive and semantic aspects of dialogues in which subjects succeeded or failed to convince judges of their humanness. Using cognitive and emotional dimensions in a global behavioral characterization, we demonstrate important differences in the patterns of behavioral expressiveness of the judges whether they perceived their interlocutor as being human or machine. Furthermore, the indicators of interest displayed by the judges were predictive of the final judgment of humanness. Thus, we show that the judgment of an interlocutor\\'s humanness during a social interaction depends not only on his behavior, but also on the judge himself. Our results thus demonstrate that the judgment of humanness is in the eye of the beholder.',\n",
              "  'authors': ['Lortie, Catherine L', 'Guitton, Matthieu J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Cognition',\n",
              "   'Computer Simulation',\n",
              "   'Emotions',\n",
              "   'Humans',\n",
              "   'Judgment',\n",
              "   'Robotics'],\n",
              "  'pmid': '21966420',\n",
              "  'title': 'Judgment of the humanness of an interlocutor is in the eye of the beholder.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PY'],\n",
              "  'ESSN': '2077-0383',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '5.583',\n",
              "  'IF_2015': '5.583',\n",
              "  'IF_2016': '5.583',\n",
              "  'IF_2017': '5.583',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2077-0383',\n",
              "  'IsoAbbr': 'J Clin Med',\n",
              "  'JrId': 41233,\n",
              "  'MedAbbr': 'J Clin Med',\n",
              "  'NLMid': '101606588',\n",
              "  'Titre': 'J Clin Med',\n",
              "  'abstract': \"This is a narrative review of both the literature and Internet pertaining to telemedicine projects within the field of heart failure, with special attention placed on remote monitoring of second-generation projects and trials, particularly in France.|Since the beginning of the 2000's, several telemedicine projects and trials focused on chronic heart failure have been developed. The first telemedicine projects (e.g., TEN-HMS, BEAT-HF, Tele-HF, and TIM-HF) primarily investigated telemonitoring or for the older ones, telephone follow-up. Numerous second-generation telemedicine projects have emerged in Europe over the last ten years or are still under development for computer science heart failure, especially in Europe, such as SCAD, OSICAT, E-care, PRADO-INCADO, and TIM-HF2. The E-care telemonitoring project fits within the telemedicine 2.0 framework, based on connected objects, new information and communication technologies (ICT) and Web 2.0 technologies. E-care is the first telemedicine project including artificial intelligence (AI). TIM-HF2 is the first positive prospective randomized study with regards to EBM with positive significant clinical benefit, in terms of unplanned cardiovascular hospital admissions and all-cause deaths. The potential contribution of second-generation telemedicine projects in terms of mortality, morbidity, and number of hospitalizations avoided is currently under study. Their impact in terms of health economics is likewise being investigated, taking into account that the economic and social benefits brought up by telemedicine solutions were previously validated by the original telemedicine projects.\",\n",
              "  'authors': ['Andrès, Emmanuel',\n",
              "   'Talha, Samy',\n",
              "   'Zulfiqar, Abrar-Ahmad',\n",
              "   'Hajjam, Mohamed',\n",
              "   'Ervé, Sylvie',\n",
              "   'Hajjam, Jawad',\n",
              "   'Gény, Bernard',\n",
              "   'Hajjam El Hassani, Amir'],\n",
              "  'categories': ['MEDICINE, GENERAL & INTERNAL'],\n",
              "  'journal': 'Journal of clinical medicine',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30551588',\n",
              "  'title': 'Current Research and New Perspectives of Telemedicine in Chronic Heart Failure: Narrative Review and Points of Interest for the Clinician.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1432-1084',\n",
              "  'IF_2013': '4.338',\n",
              "  'IF_2014': '4.014',\n",
              "  'IF_2015': '3.64',\n",
              "  'IF_2016': '3.967',\n",
              "  'IF_2017': '4.027',\n",
              "  'ISSN': '0938-7994',\n",
              "  'ISSN_online': '1432-1084',\n",
              "  'ISSN_print': '0938-7994',\n",
              "  'IsoAbbr': 'Eur Radiol',\n",
              "  'JrId': 20371,\n",
              "  'MedAbbr': 'Eur Radiol',\n",
              "  'NLMid': '9114774',\n",
              "  'Titre': 'Eur Radiol',\n",
              "  'abstract': 'Retrospective reconstruction of ECG-gated images at different parts of the cardiac cycle allows the assessment of cardiac function by multi-detector row CT (MDCT) at the time of non-invasive coronary imaging. We compared the accuracy of such measurements by MDCT to cine magnetic resonance (MR). Forty patients underwent the assessment of global and regional cardiac function by 16-slice MDCT and cine MR. Left ventricular (LV) end-diastolic and end-systolic volumes estimated by MDCT (134+/-51 and 67+/-56 ml) were similar to those by MR (137+/-57 and 70+/-60 ml, respectively; both P=NS) and strongly correlated (r=0.92 and r=0.95, respectively; both P<0.001). Consequently, LV ejection fractions by MDCT and MR were also similar (55+/-21 vs. 56+/-21%; P=NS) and highly correlated (r=0.95; P<0.001). Regional end-diastolic and end-systolic wall thicknesses by MDCT were highly correlated (r=0.84 and r=0.92, respectively; both P<0.001), but significantly lower than by MR (8.3+/-1.8 vs. 8.8+/-1.9 mm and 12.7+/-3.4 vs. 13.3+/-3.5 mm, respectively; both P<0.001). Values of regional wall thickening by MDCT and MR were similar (54+/-30 vs. 51+/-31%; P=NS) and also correlated well (r=0.91; P<0.001). Retrospectively gated MDCT can accurately estimate LV volumes, EF and regional LV wall thickening compared to cine MR.',\n",
              "  'authors': ['Belge, Bénédicte',\n",
              "   'Coche, Emmanuel',\n",
              "   'Pasquet, Agnès',\n",
              "   'Vanoverschelde, Jean-Louis J',\n",
              "   'Gerber, Bernhard L'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Artery Disease',\n",
              "   'Electrocardiography',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Magnetic Resonance Imaging, Cine',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Stroke Volume',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Transducers',\n",
              "   'Ventricular Dysfunction, Left'],\n",
              "  'pmid': '16518654',\n",
              "  'title': 'Accurate estimation of global and regional cardiac function by retrospectively gated multidetector row computed tomography: comparison with cine magnetic resonance imaging.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'We present a method for the real time prediction of punctuate events in neural activity, based on the time-frequency spectrum of the signal, applicable both to continuous processes like local field potentials (LFPs) as well as to spike trains. We test it on recordings of LFP and spiking activity acquired previously from the lateral intraparietal area (LIP) of macaque monkeys performing a memory-saccade task. In contrast to earlier work, where trials with known start times were classified, our method detects and classifies trials directly from the data. It provides a means to quantitatively compare and contrast the content of LFP signals and spike trains: we find that the detector performance based on the LFP matches the performance based on spike rates. The method should find application in the development of neural prosthetics based on the LFP signal. Our approach uses a new feature vector, which we call the 2d cepstrum.',\n",
              "  'authors': ['Bokil, Hemant S',\n",
              "   'Pesaran, Bijan',\n",
              "   'Andersen, Richard A',\n",
              "   'Mitra, Partha P'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Cluster Analysis',\n",
              "   'Computer Simulation',\n",
              "   'Electroencephalography',\n",
              "   'Evoked Potentials, Visual',\n",
              "   'Macaca',\n",
              "   'Memory',\n",
              "   'Models, Neurological',\n",
              "   'Models, Statistical',\n",
              "   'Parietal Lobe',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Saccades'],\n",
              "  'pmid': '16916103',\n",
              "  'title': 'A method for detection and classification of events in neural activity.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'An initial unsynchronized ensemble of networking phase oscillators is further subjected to a growing process where a set of forcing oscillators, each one of them following the dynamics of a frequency pacemaker, are added to the pristine graph. Linking rules based on dynamical criteria are followed in the attachment process to force phase locking of the network with the external pacemaker. We show that the eventual locking occurs in correspondence to the arousal of a scale-free degree distribution in the original graph.',\n",
              "  'authors': ['Sendiña-Nadal, Irene',\n",
              "   'Buldú, Javier M',\n",
              "   'Leyva, Inmaculada',\n",
              "   'Boccaletti, Stefano'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oscillometry',\n",
              "   'Periodicity',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '18612423',\n",
              "  'title': 'Phase locking induces scale-free topologies in networks of coupled oscillators.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1941-0506',\n",
              "  'IF_2013': '1.919',\n",
              "  'IF_2014': '2.168',\n",
              "  'IF_2015': '1.4',\n",
              "  'IF_2016': '2.84',\n",
              "  'IF_2017': '3.078',\n",
              "  'ISSN': '1077-2626',\n",
              "  'ISSN_online': '1941-0506',\n",
              "  'ISSN_print': '1077-2626',\n",
              "  'IsoAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'JrId': 31980,\n",
              "  'MedAbbr': 'IEEE Trans Vis Comput Graph',\n",
              "  'NLMid': '9891704',\n",
              "  'Titre': 'IEEE Trans Vis Comput Graph',\n",
              "  'abstract': \"While deep learning models have achieved state-of-the-art accuracies for many prediction tasks, understanding these models remains a challenge. Despite the recent interest in developing visual tools to help users interpret deep learning models, the complexity and wide variety of models deployed in industry, and the large-scale datasets that they used, pose unique design challenges that are inadequately addressed by existing work. Through participatory design sessions with over 15 researchers and engineers at Facebook, we have developed, deployed, and iteratively improved ACTIVIS, an interactive visualization system for interpreting large-scale deep learning models and results. By tightly integrating multiple coordinated views, such as a computation graph overview of the model architecture, and a neuron activation view for pattern discovery and comparison, users can explore complex deep neural network models at both the instance- and subset-level. ACTIVIS has been deployed on Facebook's machine learning platform. We present case studies with Facebook researchers and engineers, and usage scenarios of how ACTIVIS may work with different models.\",\n",
              "  'authors': ['Kahng, Minsuk',\n",
              "   'Andrews, Pierre Y',\n",
              "   'Kalro, Aditya',\n",
              "   'Polo Chau, Duen Horng'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE transactions on visualization and computer graphics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '28866557',\n",
              "  'title': 'ACTIVIS: Visual Exploration of Industry-Scale Deep Neural Network Models.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1769-6917',\n",
              "  'IF_2013': '0.635',\n",
              "  'IF_2014': '0.604',\n",
              "  'IF_2015': '0.706',\n",
              "  'IF_2016': '0.853',\n",
              "  'IF_2017': '0.84',\n",
              "  'ISSN': '0007-4551',\n",
              "  'ISSN_online': '1769-6917',\n",
              "  'ISSN_print': '0007-4551',\n",
              "  'IsoAbbr': 'Bull Cancer',\n",
              "  'JrId': 2060,\n",
              "  'MedAbbr': 'Bull Cancer',\n",
              "  'NLMid': '0072416',\n",
              "  'Titre': 'Bull Cancer',\n",
              "  'abstract': 'To heal otherwise in oncology has become an imperative of Public Health and an economic imperative in France. Patients can therefore receive live most of their care outside of hospital with more ambulatory care. This ambulatory shift will benefit from the digital revolution and the development of digital health or e-health. Cancer research will also benefit with Big Data and artificial intelligence, which gather and analyze a huge amount of data. In this synthesis, we describe the different e-health tools and their potential impacts in oncology, at the levels of education and information of patients and caregivers, prevention, screening and diagnosis, treatment, follow-up, and research. A few randomized studies have already demonstrated clinical benefits. Large Big Data projects such as ConSoRe and Health Data Hub have been launched in France. We also discuss the issues and limitations of \"cancer outside the hospital walls and e-health\" from the point of view of patients, health care professionals, health facilities and government. This new organization will have to provide remote support \"outside the walls\" with care and follow-up of quality, continuous and prolonged in total safety and equity. Ongoing and future randomized clinical trials will need to definitively demonstrate areas of interest, advantages and drawbacks not only for patients, but also for caregivers, health facilities and governments.',\n",
              "  'authors': ['Bertucci, François',\n",
              "   'Le Corroller-Soriano, Anne-Gaëlle',\n",
              "   'Monneur, Audrey',\n",
              "   'Fluzin, Sylvain',\n",
              "   'Viens, Patrice',\n",
              "   'Maraninchi, Dominique',\n",
              "   'Goncalves, Anthony'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Bulletin du cancer',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31543271',\n",
              "  'title': '[E-health and \"Cancer outside the hospital walls\", Big Data and artificial intelligence].',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1095-8630',\n",
              "  'IF_2013': '3.188',\n",
              "  'IF_2014': '2.723',\n",
              "  'IF_2015': '3.131',\n",
              "  'IF_2016': '4.01',\n",
              "  'IF_2017': '4.005',\n",
              "  'ISSN': '0301-4797',\n",
              "  'ISSN_online': '1095-8630',\n",
              "  'ISSN_print': '0301-4797',\n",
              "  'IsoAbbr': 'J. Environ. Manage.',\n",
              "  'JrId': 22117,\n",
              "  'MedAbbr': 'J Environ Manage',\n",
              "  'NLMid': '0401664',\n",
              "  'Titre': 'J. Environ. Manage.',\n",
              "  'abstract': \"Flood management decision-making problems are often associated with multiple objectives and multiple stakeholders. The challenge remains on how to obtain the diversified opinions of a large number of stakeholders where uncertainty plays a major role. In response to this challenge, a methodology has been proposed to capture the views of multiple stakeholders using fuzzy set theory and fuzzy logic. Three possible forms of individual stakeholders' input, scale (crisp), linguistic (fuzzy), and conditional (fuzzy), are analyzed to obtain the aggregated input by using Fuzzy Expected Value. The methodology is applied to flood management in the Red River Basin, Manitoba, Canada. The outputs derived from the three different input types show good correlation. Thus, the proposed methodology shows alternative ways of soliciting the opinion of a large number of stakeholders and aggregating those opinions for use in multi-objective decision making.\",\n",
              "  'authors': ['Akter, Taslima', 'Simonovic, Slobodan P'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Journal of environmental management',\n",
              "  'keywords': ['Community Participation',\n",
              "   'Decision Making',\n",
              "   'Disasters',\n",
              "   'Fuzzy Logic'],\n",
              "  'pmid': '16046249',\n",
              "  'title': 'Aggregation of fuzzy views of a large number of stakeholders for multi-objective flood management decision-making.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1558-349X',\n",
              "  'IF_2013': '2.283',\n",
              "  'IF_2014': '2.836',\n",
              "  'IF_2015': '2.929',\n",
              "  'IF_2016': '2.993',\n",
              "  'IF_2017': '3.383',\n",
              "  'ISSN': '1546-1440',\n",
              "  'ISSN_online': '1558-349X',\n",
              "  'ISSN_print': '1546-1440',\n",
              "  'IsoAbbr': 'J Am Coll Radiol',\n",
              "  'JrId': 33174,\n",
              "  'MedAbbr': 'J Am Coll Radiol',\n",
              "  'NLMid': '101190326',\n",
              "  'Titre': 'J Am Coll Radiol',\n",
              "  'abstract': 'Correlation of pathology reports with radiology examinations has long been of interest to radiologists and helps to facilitate peer learning. Such correlation also helps meet regulatory requirements, ensures quality, and supports multidisciplinary conferences and patient care. Additional offshoots of such correlation include evaluating for and ensuring concordance of pathology results with radiology interpretation and procedures as well as ensuring specimen adequacy after biopsy. For much of the history of radiology, this correlation has been done manually, which is time consuming and cumbersome and provides coverage of only a fraction of radiology examinations performed. Electronic storage and indexing of radiology and pathology information laid the foundation for easier access and for the development of automated artificial intelligence methods to match pathology information with radiology reports. More recent techniques have resulted in near comprehensive coverage of radiology examinations with methods to present results and solicit feedback from end users. Newer deep learning language modeling techniques will advance these methods by providing more robust automated and comprehensive radiology-pathology correlation with the ability to rapidly, flexibly, and iteratively tune models to site and user preference.',\n",
              "  'authors': ['Filice, Ross W'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Journal of the American College of Radiology : JACR',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31492406',\n",
              "  'title': 'Radiology-Pathology Correlation to Facilitate Peer Learning: An Overview Including Recent Artificial Intelligence Methods.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"The RNA-binding protein Musashi-1 (MSI1) exerts essential roles in multiple cellular functions, such as maintenance of self-renewal and pluripotency of stem cells. MSI1 overexpression has been observed in several tumor tissues, including glioblastoma (GBM), and is considered as a well-established marker for tumor metastasis and recurrence. However, the molecular mechanisms by which MSI1 regulates cell migration are still undetermined. Here we reported that MSI1 alters cell morphology, promotes cell migration, and increases viscoelasticity of GBM cells. We also found that MSI1 directly binds to the 3'UTR of Tensin 3 (TNS3) mRNA, a negative regulator of cell migration, to inhibit its translation. Additionally, we identified that RhoA-GTP could be a potential regulator in MSI1/TNS3-mediated cell migration and morphological changes. In a xenograft animal model, high expression ratio of MSI1 to TNS3 enhanced GBM tumor migration. We also confirmed that MSI1 and TNS3 expressions are mutually exclusive in migratory tumor lesions, and GBM patients with MSI1high/TNS3low pattern tend to have poor clinical outcome. Taken together, our findings suggested a critical role of MSI1-TNS3 axis in regulating GBM migration and highlighted that the ratio of MSI1/TNS3 could predict metastatic and survival outcome of GBM patients.\",\n",
              "  'authors': ['Chen, Hsiao-Yun',\n",
              "   'Lin, Liang-Ting',\n",
              "   'Wang, Mong-Lien',\n",
              "   'Laurent, Benoit',\n",
              "   'Hsu, Chih-Hung',\n",
              "   'Pan, Chih-Ming',\n",
              "   'Jiang, Wan-Ru',\n",
              "   'Chen, Pau-Yuan',\n",
              "   'Ma, Hsin-I',\n",
              "   'Chen, Yi-Wei',\n",
              "   'Huang, Pin-I',\n",
              "   'Chiou, Arthur',\n",
              "   'Chiou, Shih-Hwa'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [\"3' Untranslated Regions\",\n",
              "   'Animals',\n",
              "   'Cell Adhesion',\n",
              "   'Cell Line, Tumor',\n",
              "   'Cell Movement',\n",
              "   'Cytoskeleton',\n",
              "   'Disease Progression',\n",
              "   'Elasticity',\n",
              "   'Gene Ontology',\n",
              "   'Glioblastoma',\n",
              "   'Guanosine Triphosphate',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Mice, Inbred BALB C',\n",
              "   'Mice, Nude',\n",
              "   'Nerve Tissue Proteins',\n",
              "   'Protein Binding',\n",
              "   'Protein Biosynthesis',\n",
              "   'RNA-Binding Proteins',\n",
              "   'Signal Transduction',\n",
              "   'Tensins',\n",
              "   'Viscosity',\n",
              "   'rhoA GTP-Binding Protein'],\n",
              "  'pmid': '28821879',\n",
              "  'title': 'Musashi-1 Enhances Glioblastoma Cell Migration and Cytoskeletal Dynamics through Translational Inhibition of Tensin3.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Many robotic needle-biopsy systems have been developed to enhance the accuracy of needle-biopsy intervention. These systems can reduce the intervention time and the radiation exposure of clinicians. However, respiratory-motion compensation is needed to ensure the accuracy and efficiency of needle biopsy intervention.|Human respiratory-motion data were acquired using three inertial measurement units (IMUs), and respiratory motion was simulated using the Stewart-Gough platform. Robotic needle intervention was performed using impedance and admittance control algorithms for respiratory-motion compensation using the Stewart-Gough platform and a gelatin phantom.|The impedance and admittance control algorithms can be used to compensate for respiratory motion during robotic needle insertion. The admittance control algorithm exhibits better performance than the impedance control algorithm.|The impedance and admittance control algorithms can be applied for respiratory-motion compensation during robotic needle insertion. However, further study is needed for them to become clinically feasible.',\n",
              "  'authors': ['Kim, Yeoun Jae',\n",
              "   'Seo, Jong Hyun',\n",
              "   'Kim, Hong Rae',\n",
              "   'Kim, Kwang Gi'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Biopsy, Needle',\n",
              "   'Electric Impedance',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Motion',\n",
              "   'Phantoms, Imaging',\n",
              "   'Respiration',\n",
              "   'Robotic Surgical Procedures',\n",
              "   'Robotics'],\n",
              "  'pmid': '27915466',\n",
              "  'title': 'Impedance and admittance control for respiratory-motion compensation during robotic needle insertion - a preliminary test.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'Minimally invasive surgical interventions performed using computer-assisted surgery (CAS) systems require reliable registration methods for pre-operatively acquired patient anatomy representations that are compatible with the minimally invasive paradigm. The use of brightness-mode ultrasound seems to be promising, if associated devices work in a computationally efficient and fully automatic manner.|This paper presents a rapid and fully automatic segmentation approach for ultrasound B-mode images capable of detecting echoes from bony structures. The algorithm focuses on the precise and rapid detection of bone contours usable for minimally invasive registration. The article introduces the image-processing scheme and a set-up enabling a direct comparison between manually digitized reference points and the segmented bone contours. The segmentation accuracy was assessed using cadaveric material.|The experimental evaluation revealed results in the same order of magnitude as a pointer-based surface digitization procedure.|The suggested segmentation approach provides a reliable means of detecting bony surface patches in ultrasound images.',\n",
              "  'authors': ['Kowal, Jens',\n",
              "   'Amstutz, Christoph',\n",
              "   'Langlotz, Frank',\n",
              "   'Talib, Haydar',\n",
              "   'Ballester, Miguel Gonzalez'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Osteotomy',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Pelvic Bones',\n",
              "   'Subtraction Technique',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '18008384',\n",
              "  'title': 'Automated bone contour detection in ultrasound B-mode images for minimally invasive registration in computer-assisted surgery-an in vitro evaluation.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': \"This paper explores feature reduction properties of independent component analysis (ICA) on breast cancer decision support system. Wisconsin diagnostic breast cancer (WDBC) dataset is reduced to one-dimensional feature vector computing an independent component (IC). The original data with 30 features and reduced one feature (IC) are used to evaluate diagnostic accuracy of the classifiers such as k-nearest neighbor (k-NN), artificial neural network (ANN), radial basis function neural network (RBFNN), and support vector machine (SVM). The comparison of the proposed classification using the IC with original feature set is also tested on different validation (5/10-fold cross-validations) and partitioning (20%-40%) methods. These classifiers are evaluated how to effectively categorize tumors as benign and malignant in terms of specificity, sensitivity, accuracy, F-score, Youden's index, discriminant power, and the receiver operating characteristic (ROC) curve with its criterion values including area under curve (AUC) and 95% confidential interval (CI). This represents an improvement in diagnostic decision support system, while reducing computational complexity. \",\n",
              "  'authors': ['Mert, Ahmet', 'Kılıç, Niyazi', 'Bilgili, Erdem', 'Akan, Aydin'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Biopsy, Fine-Needle',\n",
              "   'Breast Neoplasms',\n",
              "   'Computational Biology',\n",
              "   'Databases, Factual',\n",
              "   'Decision Support Systems, Clinical',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Principal Component Analysis',\n",
              "   'ROC Curve',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted'],\n",
              "  'pmid': '26078774',\n",
              "  'title': 'Breast Cancer Detection with Reduced Feature Set.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1338-4317',\n",
              "  'IF_2013': '1.642',\n",
              "  'IF_2014': '1.865',\n",
              "  'IF_2015': '1.961',\n",
              "  'IF_2016': '1.871',\n",
              "  'IF_2017': '1.696',\n",
              "  'ISSN': '0028-2685',\n",
              "  'ISSN_online': '1338-4317',\n",
              "  'ISSN_print': '0028-2685',\n",
              "  'IsoAbbr': 'Neoplasma',\n",
              "  'JrId': 6037,\n",
              "  'MedAbbr': 'Neoplasma',\n",
              "  'NLMid': '0377266',\n",
              "  'Titre': 'Neoplasma',\n",
              "  'abstract': 'This study sought to investigate genes related to recurrent risk and establish a support vector machine (SVM) classifier for prediction of recurrent risk in gastric cancer (GC).Based on the gene expression profiling dataset GSE26253, feature genes that were significantly associated with survival time and status were screened out. Subsequently, protein-protein interaction (PPI) network was constructed for these feature genes, and genes in this network was optimized using betweenness centrality algorithm in order to identify genes potentially correlated with GC (named as GCGs). In total, 1202 feature genes were identified to be significantly associated with survival time and status of GC, among of which, 65 genes were identified as a classifier that was able to recognize recurrence and nonrecurrence GC cases with a high sensitivity and specificity, predictive value (PPV), negative predictive value (NPV) and area under the receiver operating characteristic curve (AUC). Furthermore, the classifier was able to reasonably classify tumor samples in GSE15459 into high and low recurrent risk groups. Among those genes, a set of genes were predicted to have interactions (e.g. RHOA interacting with TGFBR1, PRKACA and PLCG1; TGFBR1 interacting with TGFBR2) and be involved in pathways like MAPK signaling (e.g. TGFBR1 and TGFBR2), adherens junction (e.g. RHOA) and apoptosis (e.g. PRKACA).The genes in the classifier model may be related to GC recurrence, and the classifier model may contribute to the prediction of recurrent risk in GC.',\n",
              "  'authors': ['Liu, B', 'Tan, J', 'Wang, X', 'Liu, X'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Neoplasma',\n",
              "  'keywords': ['Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Humans',\n",
              "   'Neoplasm Recurrence, Local',\n",
              "   'Stomach Neoplasms',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '29788739',\n",
              "  'title': 'Identification of recurrent risk-related genes and establishment of support vector machine prediction model for gastric cancer.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1097-0142',\n",
              "  'IF_2013': '4.901',\n",
              "  'IF_2014': '4.889',\n",
              "  'IF_2015': '5.649',\n",
              "  'IF_2016': '6.072',\n",
              "  'IF_2017': '6.537',\n",
              "  'ISSN': '0008-543X',\n",
              "  'ISSN_online': '1097-0142',\n",
              "  'ISSN_print': '0008-543X',\n",
              "  'IsoAbbr': 'Cancer',\n",
              "  'JrId': 2771,\n",
              "  'MedAbbr': 'Cancer',\n",
              "  'NLMid': '0374236',\n",
              "  'Titre': 'Cancer',\n",
              "  'abstract': 'Artificial neural networks now are used in many fields. They have become well established as viable, multipurpose, robust computational methodologies with solid theoretic support and with strong potential to be effective in any discipline, especially medicine. For example, neural networks can extract new medical information from raw data, build computer models that are useful for medical decision-making, and aid in the distribution of medical expertise. Because many important neural network applications currently are emerging, the authors have prepared this article to bring a clearer understanding of these biologically inspired computing paradigms to anyone interested in exploring their use in medicine. They discuss the historical development of neural networks and provide the basic operational mathematics for the popular multilayered perceptron. The authors also describe good training, validation, and testing techniques, and discuss measurements of performance and reliability, including the use of bootstrap methods to obtain confidence intervals. Because it is possible to predict outcomes for individual patients with a neural network, the authors discuss the paradigm shift that is taking place from previous \"bin-model\" approaches, in which patient outcome and management is assumed from the statistical groups in which the patient fits. The authors explain that with neural networks it is possible to mediate predictions for individual patients with prevalence and misclassification cost considerations using receiver operating characteristic methodology. The authors illustrate their findings with examples that include prostate carcinoma detection, coronary heart disease risk prediction, and medication dosing. The authors identify and discuss obstacles to success, including the need for expanded databases and the need to establish multidisciplinary teams. The authors believe that these obstacles can be overcome and that neural networks have a very important role in future medical decision support and the patient management systems employed in routine medical practice.',\n",
              "  'authors': ['Dayhoff, J E', 'DeLeo, J M'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Cancer',\n",
              "  'keywords': ['Decision Theory',\n",
              "   'Delivery of Health Care',\n",
              "   'Humans',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Outcome Assessment (Health Care)',\n",
              "   'Patient Care Planning',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '11309760',\n",
              "  'title': 'Artificial neural networks: opening the black box.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Neural trees are constructive algorithms which build decision trees whose nodes are binary neurons. We propose a new learning scheme, \"trio-learning,\" which leads to a significant reduction in the tree complexity. In this strategy, each node of the tree is optimized by taking into account the knowledge that it will be followed by two son nodes. Moreover, trio-learning can be used to build hybrid trees, with internal nodes and terminal nodes of different nature, for solving any standard tasks (e.g. classification, regression, density estimation). Significant results on a handwritten character classification are presented.',\n",
              "  'authors': [\"d'Alché-Buc, F\", 'Zwierski, D', 'Nadal, J P'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Mathematics',\n",
              "   'Models, Psychological',\n",
              "   'Models, Theoretical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '7711959',\n",
              "  'title': 'Trio learning: a new strategy for building hybrid neural trees.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1531-8257',\n",
              "  'IF_2013': '5.634',\n",
              "  'IF_2014': '5.68',\n",
              "  'IF_2015': '6.01',\n",
              "  'IF_2016': '7.072',\n",
              "  'IF_2017': '8.324',\n",
              "  'ISSN': '0885-3185',\n",
              "  'ISSN_online': '1531-8257',\n",
              "  'ISSN_print': '0885-3185',\n",
              "  'IsoAbbr': 'Mov. Disord.',\n",
              "  'JrId': 5937,\n",
              "  'MedAbbr': 'Mov Disord',\n",
              "  'NLMid': '8610688',\n",
              "  'Titre': 'Mov. Disord.',\n",
              "  'abstract': \"In Parkinson's disease (PD), levodopa and subthalamic nucleus (STN) stimulation lead to major improvement in motor symptoms. Effects of both treatments on cognition and affective status are less well understood. Motor, cognitive, and affective symptoms may relate to the dysfunctioning of parallel cortico-striatal loops. The aim of this study was to assess cognition, behavior, and mood, with and without both treatments in the same group of PD patients. A group of 22 nondemented PD patients was included in this study. Patients were tested twice before surgery (off and on levodopa) and twice 3 months after surgery (OFF and ON STN stimulation, off levodopa). Cognitive and affective effects of STN stimulation and levodopa had some common, but also different, effects. STN stimulation improved performance on the planning test, associated with the dorsolateral prefrontal cortex. However, the treatments had opposite effects on tests associated with the orbitofrontal cortex; specifically, levodopa impaired while STN stimulation improved performance on the extinction phase of a reversal/extinction task. Acutely, both treatments improved motivation and decreased fatigue and anxiety. On chronic treatment (3 months after surgery), depression improved, whereas apathy worsened 3 months after surgery. To conclude, there were significant but contrasting effects of levodopa and STN stimulation on cognition and affective functions.\",\n",
              "  'authors': ['Funkiewiez, Aurélie',\n",
              "   'Ardouin, Claire',\n",
              "   'Cools, Roshan',\n",
              "   'Krack, Paul',\n",
              "   'Fraix, Valérie',\n",
              "   'Batir, Alina',\n",
              "   'Chabardès, Stephan',\n",
              "   'Benabid, Alim-Louis',\n",
              "   'Robbins, Trevor W',\n",
              "   'Pollak, Pierre'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Movement disorders : official journal of the Movement Disorder Society',\n",
              "  'keywords': ['Affect',\n",
              "   'Antiparkinson Agents',\n",
              "   'Cerebral Cortex',\n",
              "   'Cognition',\n",
              "   'Combined Modality Therapy',\n",
              "   'Corpus Striatum',\n",
              "   'Deep Brain Stimulation',\n",
              "   'Depressive Disorder',\n",
              "   'Extinction, Psychological',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Levodopa',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Motivation',\n",
              "   'Neural Pathways',\n",
              "   'Neurologic Examination',\n",
              "   'Neuropsychological Tests',\n",
              "   'Problem Solving',\n",
              "   'Reversal Learning',\n",
              "   'Subthalamic Nucleus'],\n",
              "  'pmid': '16830317',\n",
              "  'title': \"Effects of levodopa and subthalamic nucleus stimulation on cognitive and affective functioning in Parkinson's disease.\",\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1533-0338',\n",
              "  'IF_2013': '1.886',\n",
              "  'IF_2014': '1.73',\n",
              "  'IF_2015': '2.185',\n",
              "  'IF_2016': '2.204',\n",
              "  'IF_2017': '1.646',\n",
              "  'ISSN': '1533-0346',\n",
              "  'ISSN_online': '1533-0338',\n",
              "  'ISSN_print': '1533-0346',\n",
              "  'IsoAbbr': 'Technol. Cancer Res. Treat.',\n",
              "  'JrId': 30245,\n",
              "  'MedAbbr': 'Technol Cancer Res Treat',\n",
              "  'NLMid': '101140941',\n",
              "  'Titre': 'Technol. Cancer Res. Treat.',\n",
              "  'abstract': 'Treatment planning is an essential step of the radiotherapy workflow. It has become more sophisticated over the past couple of decades with the help of computer science, enabling planners to design highly complex radiotherapy plans to minimize the normal tissue damage while persevering sufficient tumor control. As a result, treatment planning has become more labor intensive, requiring hours or even days of planner effort to optimize an individual patient case in a trial-and-error fashion. More recently, artificial intelligence has been utilized to automate and improve various aspects of medical science. For radiotherapy treatment planning, many algorithms have been developed to better support planners. These algorithms focus on automating the planning process and/or optimizing dosimetric trade-offs, and they have already made great impact on improving treatment planning efficiency and plan quality consistency. In this review, the smart planning tools in current clinical use are summarized in 3 main categories: automated rule implementation and reasoning, modeling of prior knowledge in clinical practice, and multicriteria optimization. Novel artificial intelligence-based treatment planning applications, such as deep learning-based algorithms and emerging research directions, are also reviewed. Finally, the challenges of artificial intelligence-based treatment planning are discussed for future works.',\n",
              "  'authors': ['Wang, Chunhao',\n",
              "   'Zhu, Xiaofeng',\n",
              "   'Hong, Julian C',\n",
              "   'Zheng, Dandan'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Technology in cancer research & treatment',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31495281',\n",
              "  'title': 'Artificial Intelligence in Radiotherapy Treatment Planning: Present and Future.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1542-6270',\n",
              "  'IF_2013': '2.923',\n",
              "  'IF_2014': '2.059',\n",
              "  'IF_2015': '2.119',\n",
              "  'IF_2016': '2.748',\n",
              "  'IF_2017': '2.765',\n",
              "  'ISSN': '1060-0280',\n",
              "  'ISSN_online': '1542-6270',\n",
              "  'ISSN_print': '1060-0280',\n",
              "  'IsoAbbr': 'Ann Pharmacother',\n",
              "  'JrId': 1999,\n",
              "  'MedAbbr': 'Ann Pharmacother',\n",
              "  'NLMid': '9203131',\n",
              "  'Titre': 'Ann Pharmacother',\n",
              "  'abstract': 'The outlook for pharmacy-related services foretells more involvement of both computers and information systems. Expert therapeutic systems and databases will enable pharmacists to expand their consultation potential through networks and improve the quality of healthcare that they provide. Therapeutic information management could be the largest pharmacy specialty of the future. As knowledge-based systems and networks become commonplace, there will be an increasing need for new components, system monitoring, and quality assurance. This is an opportunity for pharmacy to bring medical computing, as it relates to therapeutics, into the mainstream of the profession as a new discipline.',\n",
              "  'authors': ['Woodruff, A E', 'Hunt, C A'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'The Annals of pharmacotherapy',\n",
              "  'keywords': ['Clinical Pharmacy Information Systems',\n",
              "   'Databases, Bibliographic',\n",
              "   'Drug Therapy, Computer-Assisted',\n",
              "   'Expert Systems',\n",
              "   'Forecasting',\n",
              "   'Humans',\n",
              "   'Patient Education as Topic',\n",
              "   'Pharmacists',\n",
              "   'Quality of Health Care',\n",
              "   'Referral and Consultation'],\n",
              "  'pmid': '1606329',\n",
              "  'title': 'Involvement in medical informatics may enable pharmacists to expand their consultation potential and improve the quality of healthcare.',\n",
              "  'year': '1992'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The Asian longhorned beetle (ALB), Anoplophora glabripennis, is an invasive wood-borer in North America and Europe that threatens a variety of tree genera, including Acer and Populus. All invasive ALB populations occur in quarantine zones where they are under eradication, a process that is difficult and expensive, requiring extensive surveys and host tree removals. Although ALB has been described as an insect that is typically slow to disperse, some rare individuals that fly longer distances have the potential to start infestations outside of quarantine zones. Biological control using entomopathogenic fungi has been considered as another option for managing ALB infestations. The entomopathogenic fungus Metarhizium brunneum strain F52, registered for commercial use in the United States, is effective at killing ALB adults but information is lacking on how this entomopathogen affects ALB flight behavior before death. Using quarantine-reared ALB, flight mills were used to collect data on flight performance of beetles at multiple time points after infection. Healthy (uninfected) male ALB adults always flew significantly greater distances than females. The maximum observation for total flight distance was a healthy male that flew 10.9 km in 24 hours on a flight mill. ALB adults infected with M. brunneum F52 flew significantly shorter distances compared to healthy adults, starting one week after fungal exposure. Biological control of ALB with this fungal entomopathogen could help to reduce their dispersal in the environment and, thereby, decrease the risk of adults moving outside of quarantine zones.',\n",
              "  'authors': ['Clifton, Eric H',\n",
              "   'Cortell, Jason',\n",
              "   'Ye, Linqi',\n",
              "   'Rachman, Thomas',\n",
              "   'Hajek, Ann E'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31490991',\n",
              "  'title': 'Impacts of Metarhizium brunneum F52 infection on the flight performance of Asian longhorned beetles, Anoplophora glabripennis.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1742-5662',\n",
              "  'IF_2013': '3.856',\n",
              "  'IF_2014': '3.917',\n",
              "  'IF_2015': '3.818',\n",
              "  'IF_2016': '3.579',\n",
              "  'IF_2017': '3.355',\n",
              "  'ISSN': '1742-5689',\n",
              "  'ISSN_online': '1742-5662',\n",
              "  'ISSN_print': '1742-5689',\n",
              "  'IsoAbbr': 'J R Soc Interface',\n",
              "  'JrId': 32845,\n",
              "  'MedAbbr': 'J R Soc Interface',\n",
              "  'NLMid': '101217269',\n",
              "  'Titre': 'J R Soc Interface',\n",
              "  'abstract': \"Deep learning describes a class of machine learning algorithms that are capable of combining raw inputs into layers of intermediate features. These algorithms have recently shown impressive results across a variety of domains. Biology and medicine are data-rich disciplines, but the data are complex and often ill-understood. Hence, deep learning techniques may be particularly well suited to solve problems of these fields. We examine applications of deep learning to a variety of biomedical problems-patient classification, fundamental biological processes and treatment of patients-and discuss whether deep learning will be able to transform these tasks or if the biomedical sphere poses unique challenges. Following from an extensive literature review, we find that deep learning has yet to revolutionize biomedicine or definitively resolve any of the most pressing challenges in the field, but promising advances have been made on the prior state of the art. Even though improvements over previous baselines have been modest in general, the recent progress indicates that deep learning methods will provide valuable means for speeding up or aiding human investigation. Though progress has been made linking a specific neural network's prediction to input features, understanding how users should interpret these models to make testable hypotheses about the system under study remains an open challenge. Furthermore, the limited amount of labelled data for training presents problems in some domains, as do legal and privacy constraints on work with sensitive health records. Nonetheless, we foresee deep learning enabling changes at both bench and bedside with the potential to transform several areas of biology and medicine.\",\n",
              "  'authors': ['Ching, Travers',\n",
              "   'Himmelstein, Daniel S',\n",
              "   'Beaulieu-Jones, Brett K',\n",
              "   'Kalinin, Alexandr A',\n",
              "   'Do, Brian T',\n",
              "   'Way, Gregory P',\n",
              "   'Ferrero, Enrico',\n",
              "   'Agapow, Paul-Michael',\n",
              "   'Zietz, Michael',\n",
              "   'Hoffman, Michael M',\n",
              "   'Xie, Wei',\n",
              "   'Rosen, Gail L',\n",
              "   'Lengerich, Benjamin J',\n",
              "   'Israeli, Johnny',\n",
              "   'Lanchantin, Jack',\n",
              "   'Woloszynek, Stephen',\n",
              "   'Carpenter, Anne E',\n",
              "   'Shrikumar, Avanti',\n",
              "   'Xu, Jinbo',\n",
              "   'Cofer, Evan M',\n",
              "   'Lavender, Christopher A',\n",
              "   'Turaga, Srinivas C',\n",
              "   'Alexandari, Amr M',\n",
              "   'Lu, Zhiyong',\n",
              "   'Harris, David J',\n",
              "   'DeCaprio, Dave',\n",
              "   'Qi, Yanjun',\n",
              "   'Kundaje, Anshul',\n",
              "   'Peng, Yifan',\n",
              "   'Wiley, Laura K',\n",
              "   'Segler, Marwin H S',\n",
              "   'Boca, Simina M',\n",
              "   'Swamidass, S Joshua',\n",
              "   'Huang, Austin',\n",
              "   'Gitter, Anthony',\n",
              "   'Greene, Casey S'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Journal of the Royal Society, Interface',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Biomedical Research',\n",
              "   'Biomedical Technology',\n",
              "   'Decision Making',\n",
              "   'Deep Learning',\n",
              "   'Delivery of Health Care',\n",
              "   'Disease',\n",
              "   'Drug Design',\n",
              "   'Electronic Health Records',\n",
              "   'Humans',\n",
              "   'Terminology as Topic'],\n",
              "  'pmid': '29618526',\n",
              "  'title': 'Opportunities and obstacles for deep learning in biology and medicine.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': 'Fuzzy connectedness method has shown its effectiveness for fuzzy object extraction in recent years. However, two problems may occur when applying it to hepatic vessel segmentation task. One is the excessive computational cost, and the other is the difficulty of choosing a proper threshold value for final segmentation.|In this paper, an accelerated strategy based on a lookup table was presented first which can reduce the connectivity scene calculation time and achieve a speed-up factor of above 2. When the computing of the fuzzy connectedness relations is finished, a threshold is needed to generate the final result. Currently the threshold is preset by users. Since different thresholds may produce different outcomes, how to determine a proper threshold is crucial. According to our analysis of the hepatic vessel structure, a watershed-like method was used to find the optimal threshold. Meanwhile, by using Ostu algorithm to calculate the parameters for affinity relations and assigning the seed with the mean value, it is able to reduce the influence on the segmentation result caused by the location of the seed and enhance the robustness of fuzzy connectedness method.|Experiments based on four different datasets demonstrate the efficiency of the lookup table strategy. These experiments also show that an adaptive threshold found by watershed-like method can always generate correct segmentation results of hepatic vessels. Comparing to a refined region-growing algorithm that has been widely used for hepatic vessel segmentation, fuzzy connectedness method has advantages in detecting vascular edge and generating more than one vessel system through the weak connectivity of the vessel ends.|An improved algorithm based on fuzzy connectedness method is proposed. This algorithm has improved the performance of fuzzy connectedness method in hepatic vessel segmentation.',\n",
              "  'authors': ['Guo, Xiaoxi',\n",
              "   'Huang, Shaohui',\n",
              "   'Fu, Xiaozhu',\n",
              "   'Wang, Boliang',\n",
              "   'Huang, Xiaoyang'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Datasets as Topic',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Liver',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '26087652',\n",
              "  'title': 'Vascular segmentation in hepatic CT images using adaptive threshold fuzzy connectedness method.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1573-2568',\n",
              "  'IF_2013': '2.55',\n",
              "  'IF_2014': '2.613',\n",
              "  'IF_2015': '2.516',\n",
              "  'IF_2016': '2.875',\n",
              "  'IF_2017': '2.819',\n",
              "  'ISSN': '0163-2116',\n",
              "  'ISSN_online': '1573-2568',\n",
              "  'ISSN_print': '0163-2116',\n",
              "  'IsoAbbr': 'Dig. Dis. Sci.',\n",
              "  'JrId': 3413,\n",
              "  'MedAbbr': 'Dig Dis Sci',\n",
              "  'NLMid': '7902782',\n",
              "  'Titre': 'Dig. Dis. Sci.',\n",
              "  'abstract': 'Ambulatory long-term motility recording is used increasingly for evaluation of esophageal function. The enormous amount of motility data recorded by this method demands subsequent computer analysis. One of the most crucial steps of this analysis becomes the process of automatic selection of relevant pressure peaks at the various recording levels. Until now, this selection has been performed entirely by rule-based systems, requiring each pressure deflection to fit within predefined rigid numerical limits in order to be detected. However, due to great variations in the shapes of the pressure curves generated by muscular contractions, rule-based criteria do not always select the pressure events most relevant for further analysis. We have therefore been searching for a new concept for automatic event recognition. The present study describes a new system, based on the method of neurocomputing. A large sample of normal esophageal pressure deflections was used as a \"learning set,\" and the performance of the trained neural networks was subsequently verified on different sets of data from normal subjects. Our trained networks detected pressure deflections with sensitivities of 0.79-0.99 and accuracies of 0.89-0.98, depending on the recording level within the esophageal lumen. The neural networks often recognized peaks that clearly represented true contractions but that had been rejected by a rule-based system. We conclude that neural networks have potentials for automatic detections of esophageal, and possibly also other kinds of gastrointestinal, pressure variations.',\n",
              "  'authors': ['Kruse-Andersen, S',\n",
              "   'Rütz, K',\n",
              "   'Kolberg, J',\n",
              "   'Jakobsen, E',\n",
              "   'Madsen, T'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'Digestive diseases and sciences',\n",
              "  'keywords': ['Adult',\n",
              "   'Esophagus',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Muscle Contraction',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pressure'],\n",
              "  'pmid': '7648965',\n",
              "  'title': 'Automatic detection of esophageal pressure events. Is there an alternative to rule-based criteria?',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1618-1328',\n",
              "  'IF_2013': '2.77',\n",
              "  'IF_2014': '2.557',\n",
              "  'IF_2015': '2.971',\n",
              "  'IF_2016': '3.121',\n",
              "  'IF_2017': '2.833',\n",
              "  'ISSN': '0176-1617',\n",
              "  'ISSN_online': '1618-1328',\n",
              "  'ISSN_print': '0176-1617',\n",
              "  'IsoAbbr': 'J. Plant Physiol.',\n",
              "  'JrId': 8566,\n",
              "  'MedAbbr': 'J Plant Physiol',\n",
              "  'NLMid': '9882059',\n",
              "  'Titre': 'J. Plant Physiol.',\n",
              "  'abstract': 'To understand the biology of a plant in response to stress, insight into protein-protein interactions, which almost define cell behavior, is thought to be crucial. Here, we provide a comparative complexomics analysis of leaf whole cell lysate of two rice genotypes with contrasting responses to salt using two-dimensional blue native/SDS-PAGE (2D-BN/SDS-PAGE). We aimed to identify changes in subunit composition and stoichiometry of protein complexes elicited by salt. Using mild detergent for protein complex solubilization, we were able to identify 9 protein assemblies as hetero-oligomeric and 30 as homo-oligomeric complexes. A total of 20 proteins were identified as monomers in the 2D-BN/SDS-PAGE gels. In addition to identifying known protein complexes that confirm the technical validity of our analysis, we were also able to discover novel protein-protein interactions. Interestingly, an interaction was detected for glycolytic enzymes enolase (ENO1) and triosephosphate isomerase (TPI) and also for a chlorophyll a-b binding protein and RuBisCo small subunit. To show changes in subunit composition and stoichiometry of protein assemblies during salt stress, the differential abundance of interacting proteins was compared between salt-treated and control plants. A detailed exploration of some of the protein complexes provided novel insight into the function, composition, stoichiometry and dynamics of known and previously uncharacterized protein complexes in response to salt stress. ',\n",
              "  'authors': ['Hashemi, Amenehsadat',\n",
              "   'Gharechahi, Javad',\n",
              "   'Nematzadeh, Ghorbanali',\n",
              "   'Shekari, Faezeh',\n",
              "   'Hosseini, Seyed Abdollah',\n",
              "   'Salekdeh, Ghasem Hosseini'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Journal of plant physiology',\n",
              "  'keywords': ['Cell Extracts',\n",
              "   'Detergents',\n",
              "   'Electrophoresis, Polyacrylamide Gel',\n",
              "   'Gene Ontology',\n",
              "   'Multiprotein Complexes',\n",
              "   'Oryza',\n",
              "   'Phenotype',\n",
              "   'Plant Proteins',\n",
              "   'Protein Multimerization',\n",
              "   'Salinity',\n",
              "   'Sodium Chloride',\n",
              "   'Solubility',\n",
              "   'Stress, Physiological'],\n",
              "  'pmid': '27362847',\n",
              "  'title': 'Two-dimensional blue native/SDS-PAGE analysis of whole cell lysate protein complexes of rice in response to salt stress.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4794',\n",
              "  'IF_2013': '3.179',\n",
              "  'IF_2014': '3.292',\n",
              "  'IF_2015': '3.04',\n",
              "  'IF_2016': '3.416',\n",
              "  'IF_2017': '3.589',\n",
              "  'ISSN': '0146-9592',\n",
              "  'ISSN_online': '1539-4794',\n",
              "  'ISSN_print': '0146-9592',\n",
              "  'IsoAbbr': 'Opt Lett',\n",
              "  'JrId': 22734,\n",
              "  'MedAbbr': 'Opt Lett',\n",
              "  'NLMid': '7708433',\n",
              "  'Titre': 'Opt Lett',\n",
              "  'abstract': 'Subdiffusive reflectance captured at short source-detector separations provides increased sensitivity to the scattering phase function and hence allows superficial probing of the tissue ultrastructure. Consequently, estimation of subdiffusive optical parameters has been the subject of many recent studies focusing on lookup-table-based (LUT) inverse models. Since an adequate description of the subdiffusive reflectance requires additional scattering phase function related optical parameters, the LUT inverse models, which grow exponentially with the number of estimated parameters, become excessively large and computationally inefficient. Herein, we propose, to the best of our knowledge, the first artificial-neural-network-based inverse Monte Carlo model that overcomes the limitations of the LUT inverse models and thus allows efficient real-time estimation of optical parameters from subdiffusive spatially resolved reflectance. The proposed inverse model retains the accuracy, is about four orders of magnitude faster than the LUT inverse models, grows only linearly with the number of estimated optical parameters, and can be easily extended to estimate additional optical parameters.',\n",
              "  'authors': ['Ivančič, Matic',\n",
              "   'Naglič, Peter',\n",
              "   'Pernuš, Franjo',\n",
              "   'Likar, Boštjan',\n",
              "   'Bürmen, Miran'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics letters',\n",
              "  'keywords': ['Computer Simulation',\n",
              "   'Models, Biological',\n",
              "   'Models, Theoretical',\n",
              "   'Monte Carlo Method',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Optical Devices',\n",
              "   'Optical Phenomena',\n",
              "   'Scattering, Radiation'],\n",
              "  'pmid': '29905719',\n",
              "  'title': 'Efficient estimation of subdiffusive optical parameters in real time from spatially resolved reflectance by artificial neural networks.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['LI'],\n",
              "  'ESSN': '1468-2834',\n",
              "  'IF_2013': '3.107',\n",
              "  'IF_2014': '3.642',\n",
              "  'IF_2015': '4.201',\n",
              "  'IF_2016': '4.282',\n",
              "  'IF_2017': '4.013',\n",
              "  'ISSN': '0002-0729',\n",
              "  'ISSN_online': '1468-2834',\n",
              "  'ISSN_print': '0002-0729',\n",
              "  'IsoAbbr': 'Age Ageing',\n",
              "  'JrId': 338,\n",
              "  'MedAbbr': 'Age Ageing',\n",
              "  'NLMid': '0375655',\n",
              "  'Titre': 'Age Ageing',\n",
              "  'abstract': 'there is inadequate evidence to support currently formulated NHS strategies to achieve health promotion and preventative care in older people through broad-based screening and assessment in primary care. The most extensively evaluated delivery instrument for this purpose is Health Risk Appraisal (HRA). This article describes a trial using HRA to evaluate the effect on health behaviour and preventative-care uptake in older people in NHS primary care.|a randomised controlled trial was undertaken in three London primary care group practices. Functionally independent community-dwelling patients older than 65 years (n = 2,503) received a self-administered Health Risk Appraisal for Older Persons (HRA-O) questionnaire leading to computer-generated individualised written feedback to participants and general practitioners (GPs), integrated into practice information-technology (IT) systems. All primary care staff received training in preventative health in older people. The main outcome measures were self-reported health behaviour and preventative care uptake at 1-year follow-up.|of 2,503 individuals randomised, 2,006 respondents (80.1%) (intervention, n = 940, control n = 1,066) were available for analysis. Intervention group respondents reported slightly higher pneumococcal vaccination uptake and equivocal improvement in physical activity levels compared with controls. No significant differences were observed for any other categories of health behaviour or preventative care measures at 1-year follow-up.|HRA-O implemented in this way resulted in minimal improvement of health behaviour or uptake of preventative care measures in older people. Supplementary reinforcement involving contact by health professionals with patients over and above routine clinical encounters may be a prerequisite to the effectiveness of IT-based delivery systems for health promotion in older people.',\n",
              "  'authors': ['Harari, Danielle',\n",
              "   'Iliffe, Steve',\n",
              "   'Kharicha, Kalpa',\n",
              "   'Egger, Matthias',\n",
              "   'Gillmann, Gerhard',\n",
              "   'von Renteln-Kruse, W',\n",
              "   'Beck, John',\n",
              "   'Swift, Cameron',\n",
              "   'Stuck, Andreas'],\n",
              "  'categories': ['GERIATRICS & GERONTOLOGY'],\n",
              "  'journal': 'Age and ageing',\n",
              "  'keywords': ['Activities of Daily Living',\n",
              "   'Age Factors',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Aging',\n",
              "   'Evidence-Based Medicine',\n",
              "   'Expert Systems',\n",
              "   'Family Practice',\n",
              "   'Female',\n",
              "   'Geriatric Assessment',\n",
              "   'Health Behavior',\n",
              "   'Health Promotion',\n",
              "   'Health Services for the Aged',\n",
              "   'Health Status Indicators',\n",
              "   'Humans',\n",
              "   'London',\n",
              "   'Male',\n",
              "   'Medical Records Systems, Computerized',\n",
              "   'National Health Programs',\n",
              "   'Patient Acceptance of Health Care',\n",
              "   'Preventive Health Services',\n",
              "   'Surveys and Questionnaires',\n",
              "   'Time Factors'],\n",
              "  'pmid': '18755784',\n",
              "  'title': 'Promotion of health in older people: a randomised controlled trial of health risk appraisal in British general practice.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '2168-6173',\n",
              "  'IF_2013': '4.488',\n",
              "  'IF_2014': '3.318',\n",
              "  'IF_2015': '4.34',\n",
              "  'IF_2016': '5.625',\n",
              "  'IF_2017': '6.669',\n",
              "  'ISSN': '2168-6165',\n",
              "  'ISSN_online': '2168-6173',\n",
              "  'ISSN_print': '2168-6165',\n",
              "  'IsoAbbr': 'JAMA Ophthalmol',\n",
              "  'JrId': 40866,\n",
              "  'MedAbbr': 'JAMA Ophthalmol',\n",
              "  'NLMid': '101589539',\n",
              "  'Titre': 'JAMA Ophthalmol',\n",
              "  'abstract': 'Offline automated analysis of retinal images on a smartphone may be a cost-effective and scalable method of screening for diabetic retinopathy; however, to our knowledge, assessment of such an artificial intelligence (AI) system is lacking.|To evaluate the performance of Medios AI (Remidio), a proprietary, offline, smartphone-based, automated system of analysis of retinal images, to detect referable diabetic retinopathy (RDR) in images taken by a minimally trained health care worker with Remidio Non-Mydriatic Fundus on Phone, a smartphone-based, nonmydriatic retinal camera. Referable diabetic retinopathy is defined as any retinopathy more severe than mild diabetic retinopathy, with or without diabetic macular edema.|This prospective, cross-sectional, population-based study took place from August 2018 to September 2018. Patients with diabetes mellitus who visited various dispensaries administered by the Municipal Corporation of Greater Mumbai in Mumbai, India, on a particular day were included.|Three fields of the fundus (the posterior pole, nasal, and temporal fields) were photographed. The images were analyzed by an ophthalmologist and the AI system.|To evaluate the sensitivity and specificity of the offline automated analysis system in detecting referable diabetic retinopathy on images taken on the smartphone-based, nonmydriatic retinal imaging system by a health worker.|Of 255 patients seen in the dispensaries, 231 patients (90.6%) consented to diabetic retinopathy screening. The major reasons for not participating were unwillingness to wait for screening and the blurring of vision that would occur after dilation. Images from 18 patients were deemed ungradable by the ophthalmologist and hence were excluded. In the remaining participants (110 female patients [51.6%] and 103 male patients [48.4%]; mean [SD] age, 53.1 [10.3] years), the sensitivity and specificity of the offline AI system in diagnosing referable diabetic retinopathy were 100.0% (95% CI, 78.2%-100.0%) and 88.4% (95% CI, 83.2%-92.5%), respectively, and in diagnosing any diabetic retinopathy were 85.2% (95% CI, 66.3%-95.8%) and 92.0% (95% CI, 97.1%-95.4%), respectively, compared with ophthalmologist grading using the same images.|These pilot study results show promise in the use of an offline AI system in community screening for referable diabetic retinopathy with a smartphone-based fundus camera. The use of AI would enable screening for referable diabetic retinopathy in remote areas where services of an ophthalmologist are unavailable. This study was done on patients with diabetes who were visiting a dispensary that provides curative services to the population at the primary level. A study with a larger sample size may be needed to extend the results to general population screening, however.',\n",
              "  'authors': ['Natarajan, Sundaram',\n",
              "   'Jain, Astha',\n",
              "   'Krishnan, Radhika',\n",
              "   'Rogye, Ashwini',\n",
              "   'Sivaprasad, Sobha'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'JAMA ophthalmology',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31393538',\n",
              "  'title': 'Diagnostic Accuracy of Community-Based Diabetic Retinopathy Screening With an Offline Artificial Intelligence System on a Smartphone.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1572-8358',\n",
              "  'IF_2013': '1.231',\n",
              "  'IF_2014': '0.704',\n",
              "  'IF_2015': '0.825',\n",
              "  'IF_2016': '0.646',\n",
              "  'IF_2017': '0.857',\n",
              "  'ISSN': '0001-5342',\n",
              "  'ISSN_online': '1572-8358',\n",
              "  'ISSN_print': '0001-5342',\n",
              "  'IsoAbbr': 'Acta Biotheor.',\n",
              "  'JrId': 34,\n",
              "  'MedAbbr': 'Acta Biotheor',\n",
              "  'NLMid': '0421520',\n",
              "  'Titre': 'Acta Biotheor.',\n",
              "  'abstract': 'Is it possible to predict future life forms? In this paper it is argued that the answer to this question may well be positive. As a basis for predictions a rationale is used that is derived from historical data, e.g. from a hierarchical classification that ranks all building block systems, that have evolved so far. This classification is based on specific emergent properties that allow stepwise transitions, from low level building blocks to higher level ones. This paper shows how this hierarchy can be used for predicting future life forms. The extrapolations suggest several future neural network organisms. Major aspects of the structures of these organisms are predicted. The results can be considered of fundamental importance for several reasons. Firstly, assuming that the operator hierarchy is a proper basis for predictions, the result yields insight into the structure of future organisms. Secondly, the predictions are not extrapolations of presently observed trends, but are fully integrated with all historical system transitions in evolution. Thirdly, the extrapolations suggest the structures of intelligences that, one day, will possess more powerful brains than human beings. This study ends with a discussion of possibilities for falsification of the present theory, the implications of the present predictions in relation to recent developments in artificial intelligence and the philosophical implications of the role of humanity in evolution with regard to the creation of future neural network organisms.',\n",
              "  'authors': ['Jagers op Akkerhuis, G'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Acta biotheoretica',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Evolution',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Nerve Net'],\n",
              "  'pmid': '11558895',\n",
              "  'title': 'Extrapolating a hierarchy of building block systems towards future neural network organisms.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'To evaluate the geometric and dosimetric accuracies of the CyberKnife Synchrony respiratory tracking system (RTS) and to validate a method for pretreatment patient-specific delivery quality assurance (DQA).|An EasyCube phantom was mounted on the ExacTrac gating phantom, which can move along the superior-inferior (SI) axis of a patient to simulate a moving target. The authors compared dynamic and static measurements. For each case, a Gafchromic EBT3 film was positioned between two slabs of the EasyCube, while a PinPoint ionization chamber was placed in the appropriate space. There were three steps to their evaluation: (1) the field size, the penumbra, and the symmetry of six secondary collimators were measured along the two main orthogonal axes. Dynamic measurements with deliberately simulated errors were also taken. (2) The delivered dose distributions (from step 1) were compared with the planned ones, using the gamma analysis method. The local gamma passing rates were evaluated using three acceptance criteria: 3% local dose difference (LDD)/3 mm, 2%LDD/2 mm, and 3%LDD/1 mm. (3) The DQA plans for six clinical patients were irradiated in different dynamic conditions, to give a total of 19 cases. The measured and planned dose distributions were evaluated with the same gamma-index criteria used in step 2 and the measured chamber doses were compared with the planned mean doses in the sensitive volume of the chamber.|(1) A very slight enlargement of the field size and of the penumbra was observed in the SI direction (on average <1 mm), in line with the overall average CyberKnife system error for tracking treatments. (2) Comparison between the planned and the correctly delivered dose distributions confirmed the dosimetric accuracy of the RTS for simple plans. The multicriteria gamma analysis was able to detect the simulated errors, proving the robustness of their method of analysis. (3) All of the DQA clinical plans passed the tests, both in static and dynamic conditions. No statistically significant differences were found between static and dynamic cases, confirming the high degree of accuracy of the Synchrony RTS.|The presented methods and measurements verified the mechanical and dosimetric accuracy of the Synchrony RTS. Their method confirms the fact that the RTS, if used properly, is able to treat a moving target with great precision. By combining PinPoint ion chamber, EBT3 films, and gamma evaluation of dose distributions, their DQA method robustly validated the effectiveness of CyberKnife and Synchrony system.',\n",
              "  'authors': ['Mastella, E',\n",
              "   'Vigorito, S',\n",
              "   'Rondi, E',\n",
              "   'Piperno, G',\n",
              "   'Ferrari, A',\n",
              "   'Strata, E',\n",
              "   'Rozza, D',\n",
              "   'Jereczek-Fossa, B A',\n",
              "   'Cattani, F'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Film Dosimetry',\n",
              "   'Humans',\n",
              "   'Liver Neoplasms',\n",
              "   'Models, Anatomic',\n",
              "   'Motion',\n",
              "   'Phantoms, Imaging',\n",
              "   'Quality Assurance, Health Care',\n",
              "   'Radiosurgery',\n",
              "   'Radiotherapy Dosage',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Respiration',\n",
              "   'Robotics'],\n",
              "  'pmid': '27487873',\n",
              "  'title': 'Validation of a pretreatment delivery quality assurance method for the CyberKnife Synchrony system.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1742-2051',\n",
              "  'IF_2013': '3.183',\n",
              "  'IF_2014': '3.21',\n",
              "  'IF_2015': '2.829',\n",
              "  'IF_2016': '2.781',\n",
              "  'IF_2017': '2.759',\n",
              "  'ISSN': '1742-206X',\n",
              "  'ISSN_online': '1742-2051',\n",
              "  'ISSN_print': '1742-206X',\n",
              "  'IsoAbbr': 'Mol Biosyst',\n",
              "  'JrId': 33015,\n",
              "  'MedAbbr': 'Mol Biosyst',\n",
              "  'NLMid': '101251620',\n",
              "  'Titre': 'Mol Biosyst',\n",
              "  'abstract': \"Protein S-sulfenylation (SOH) is a type of post-translational modification through the oxidation of cysteine thiols to sulfenic acids. It acts as a redox switch to modulate versatile cellular processes and plays important roles in signal transduction, protein folding and enzymatic catalysis. Reversible SOH is also a key component for maintaining redox homeostasis and has been implicated in a variety of human diseases, such as cancer, diabetes, and atherosclerosis, due to redox imbalance. Despite its significance, the in situ trapping of the entire 'sulfenome' remains a major challenge. Yang et al. have recently experimentally identified about 1000 SOH sites, providing an enriched benchmark SOH dataset. In this work, we developed a new ensemble learning tool SOHPRED for identifying protein SOH sites based on the compositions of enriched amino acids and the physicochemical properties of residues surrounding SOH sites. SOHPRED was built based on four complementary predictors, i.e. a naive Bayesian predictor, a random forest predictor and two support vector machine predictors, whose training features are, respectively, amino acid occurrences, physicochemical properties, frequencies of k-spaced amino acid pairs and sequence profiles. Benchmarking experiments on the 5-fold cross validation and independent tests show that SOHPRED achieved AUC values of 0.784 and 0.799, respectively, which outperforms several previously developed tools. As a real application of SOHPRED, we predicted potential SOH sites for 193 S-sulfenylated substrates, which had been experimentally detected through a global sulfenome profiling in living cells, though the actual SOH sites were not determined. The web server of SOHPRED has been made publicly available at for the wider research community. The source codes and the benchmark datasets can be downloaded from the website.\",\n",
              "  'authors': ['Wang, Xiaofeng',\n",
              "   'Yan, Renxiang',\n",
              "   'Li, Jinyan',\n",
              "   'Song, Jiangning'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Molecular bioSystems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Amino Acid Sequence',\n",
              "   'Bayes Theorem',\n",
              "   'Catalysis',\n",
              "   'Computational Biology',\n",
              "   'Cysteine',\n",
              "   'Datasets as Topic',\n",
              "   'Humans',\n",
              "   'Oxidation-Reduction',\n",
              "   'Peptides',\n",
              "   'Position-Specific Scoring Matrices',\n",
              "   'Protein Folding',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'ROC Curve',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sulfenic Acids',\n",
              "   'Sulfhydryl Compounds',\n",
              "   'Support Vector Machine',\n",
              "   'Web Browser'],\n",
              "  'pmid': '27364688',\n",
              "  'title': 'SOHPRED: a new bioinformatics tool for the characterization and prediction of human S-sulfenylation sites.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZC'],\n",
              "  'ESSN': '1297-9716',\n",
              "  'IF_2013': '3.383',\n",
              "  'IF_2014': '2.815',\n",
              "  'IF_2015': '2.928',\n",
              "  'IF_2016': '2.798',\n",
              "  'IF_2017': '2.903',\n",
              "  'ISSN': '0928-4249',\n",
              "  'ISSN_online': '1297-9716',\n",
              "  'ISSN_print': '0928-4249',\n",
              "  'IsoAbbr': 'Vet. Res.',\n",
              "  'JrId': 2397,\n",
              "  'MedAbbr': 'Vet Res',\n",
              "  'NLMid': '9309551',\n",
              "  'Titre': 'Vet. Res.',\n",
              "  'abstract': 'Some aspects of automated clinical mastitis detection and diagnosis are discussed. Knowledge representation techniques for the different steps in the diagnostic process are presented. The main focus of this paper is on automated early detection, based on data that are automatically collected in the milking parlour. Principal component analysis, logistic regression and back-propagation neural networks were used in the analysis of the automatically collected data. The 3 techniques did not differ greatly in performance. All the techniques performed better when data from milking with observed clinical signs were used, compared with data from milking before clinical symptoms were noticed. Healthy quarters were mostly correctly classified by all techniques. It seems unlikely that all clinical mastitis cases can be detected at milking before visible clinical signs occur.',\n",
              "  'authors': ['Nielen, M', 'Schukken, Y H', 'Hogeveen, H'],\n",
              "  'categories': ['VETERINARY SCIENCES'],\n",
              "  'journal': 'Veterinary research',\n",
              "  'keywords': ['Animals',\n",
              "   'Cattle',\n",
              "   'Cell Count',\n",
              "   'Electric Conductivity',\n",
              "   'Electronic Data Processing',\n",
              "   'Female',\n",
              "   'Logistic Models',\n",
              "   'Mammary Glands, Animal',\n",
              "   'Mastitis, Bovine',\n",
              "   'Milk',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '8038802',\n",
              "  'title': 'Development of an on-line mastitis detection system within an integrated knowledge-based system for dairy farm management support.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['QA'],\n",
              "  'ESSN': '1643-3750',\n",
              "  'IF_2013': '1.216',\n",
              "  'IF_2014': '1.433',\n",
              "  'IF_2015': '1.405',\n",
              "  'IF_2016': '1.585',\n",
              "  'IF_2017': '1.894',\n",
              "  'ISSN': '1234-1010',\n",
              "  'ISSN_online': '1643-3750',\n",
              "  'ISSN_print': '1234-1010',\n",
              "  'IsoAbbr': 'Med. Sci. Monit.',\n",
              "  'JrId': 21912,\n",
              "  'MedAbbr': 'Med Sci Monit',\n",
              "  'NLMid': '9609063',\n",
              "  'Titre': 'Med. Sci. Monit.',\n",
              "  'abstract': 'Axillary lymph nodes (ALN) are the most commonly involved site of disease in breast cancer that has spread outside the primary lesion. Although sentinel node biopsy is a reliable way to manage ALN, there are still no good methods of predicting ALN status before surgery. Since morbidity in breast cancer surgery is predominantly related to ALN dissection, predictive models for lymph node involvement may provide a way to alert the surgeon in subgroups of patients.|A total of 1325 invasive breast cancer patients were analyzed using tumor biological parameters that included age, tumor size, grade, estrogen receptor, progesterone receptor, lymphovascular invasion, and HER2, to test their ability to predict ALN involvement. A support vector machine (SVM) was used as a classification model. The SVM is a machine-learning system developed using statistical learning theories to classify data points into 2 classes. Notably, SVM models have been applied in bioinformatics.|The SVM model correctly predicted ALN metastases in 74.7% of patients using tumor biological parameters. The predictive ability of luminal A, luminal B, triple negative, and HER2 subtypes using subgroup analysis showed no difference, and this predictive performance was inferior, with only 60% accuracy.|With an SVM model based on clinical pathologic parameters obtained in the primary tumor, it is possible to predict ALN status in order to alert the surgeon about breast cancer counseling and in decision-making for ALN management.',\n",
              "  'authors': ['Wu, Jia-Long',\n",
              "   'Tseng, Hsin-Shun',\n",
              "   'Yang, Li-Heng',\n",
              "   'Wu, Hwa-Koon',\n",
              "   'Kuo, Shou-Jen',\n",
              "   'Chen, Shou-Tung',\n",
              "   'Chen, Dar-Ren'],\n",
              "  'categories': ['MEDICINE, RESEARCH & EXPERIMENTAL'],\n",
              "  'journal': 'Medical science monitor : international medical journal of experimental and clinical research',\n",
              "  'keywords': ['Axilla',\n",
              "   'Breast Neoplasms',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Lymph Nodes',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Middle Aged',\n",
              "   'Prognosis',\n",
              "   'ROC Curve',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24714517',\n",
              "  'title': 'Prediction of axillary lymph node metastases in breast cancer patients based on pathologic information of the primary tumor.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'The accuracy of the surface extraction of magnetic resonance images of highly congruent joints with thin articular cartilage layers has a significant effect on the percentage errors and reproducibility of quantitative measurements (e.g., thickness and volume) of the articular cartilage. Traditional techniques such as gradient-based edge detection are not suitable for the extraction of these surfaces. This paper studies the extraction of articular cartilage surfaces using snakes, and a gradient vector flow (GVF)-based external force is proposed for this application. In order to make the GVF snake more stable and converge to the correct surfaces, directional gradient is used to produce the gradient vector flow. Experimental results show that the directional GVF snake is more robust than the traditional GVF snake for this application. Based on the newly developed snake model, an articular cartilage surface extraction algorithm is developed. Thickness is computed based on the surfaces extracted using the proposed algorithm. In order to make the thickness measurement more reproducible, a new thickness computation approach, which is called T-norm, is introduced. Experimental results show that the thickness measurement obtained by the new thickness computation approach has better reproducibility than that obtained by the existing thickness computation approaches.',\n",
              "  'authors': ['Tang, Jinshan',\n",
              "   'Millington, Steven',\n",
              "   'Acton, Scott T',\n",
              "   'Crandall, Jeff',\n",
              "   'Hurwitz, Shepard'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artifacts',\n",
              "   'Artificial Intelligence',\n",
              "   'Cadaver',\n",
              "   'Cartilage, Articular',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'In Vitro Techniques',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Surface Properties'],\n",
              "  'pmid': '16686412',\n",
              "  'title': 'Surface extraction and thickness measurement of the articular cartilage from MR images using directional gradient vector flow snakes.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'The technique of laparoscopic radical prostatectomy is difficult to master and is associated with a steep learning curve. We hypothesized that a structured approach to establishing a laparoscopic prostatectomy program would diminish complications during the learning process and that robotic technology would be useful in learning the operation.|A structured laparoscopic radical prostatectomy program was introduced at the Vattikuti Urology Institute on October 23, 2000. One of 2 surgeons with a combined experience of more than 500 laparoscopic radical prostatectomies performed or supervised the first prostatectomies, training a third surgeon with extensive \"open\" surgical skills but no laparoscopic experience. The \"trained\" surgeon then started performing the operation independently with robotic assistance. The results of this approach were analyzed at the end of 12 months.|We performed 48 laparoscopic radical prostatectomies and 50 robot assisted prostatectomies within the 12-month period. The preoperative and intraoperative demographical variables were comparable in both groups as were the operative times, changes in hemoglobin concentrations, durations of hospitalization, positive margin rates and overall complication rates. All measured parameters were comparable to the \"best-in-class\" values for laparoscopic radical prostatectomy reported in the literature.|A structured approach minimizes complications during the establishment of laparoscopic radical prostatectomy program. Robotic assistance helps skilled \"open\" surgeons learn the technique of laparoscopic radical prostatectomy.',\n",
              "  'authors': ['Menon, Mani',\n",
              "   'Shrivastava, Alok',\n",
              "   'Tewari, Ashutosh',\n",
              "   'Sarle, Richard',\n",
              "   'Hemal, Ashok',\n",
              "   'Peabody, James O',\n",
              "   'Vallancien, Guy'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics'],\n",
              "  'pmid': '12187196',\n",
              "  'title': 'Laparoscopic and robot assisted radical prostatectomy: establishment of a structured program and preliminary analysis of outcomes.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Protein sequence features are explored in relation to the production of over-expressed extracellular proteins by fungi. Knowledge on features influencing protein production and secretion could be employed to improve enzyme production levels in industrial bioprocesses via protein engineering. A large set, over 600 homologous and nearly 2,000 heterologous fungal genes, were overexpressed in Aspergillus niger using a standardized expression cassette and scored for high versus no production. Subsequently, sequence-based machine learning techniques were applied for identifying relevant DNA and protein sequence features. The amino-acid composition of the protein sequence was found to be most predictive and interpretation revealed that, for both homologous and heterologous gene expression, the same features are important: tyrosine and asparagine composition was found to have a positive correlation with high-level production, whereas for unsuccessful production, contributions were found for methionine and lysine composition. The predictor is available online at http://bioinformatics.tudelft.nl/hipsec. Subsequent work aims at validating these findings by protein engineering as a method for increasing expression levels per gene copy.',\n",
              "  'authors': ['van den Berg, Bastiaan A',\n",
              "   'Reinders, Marcel J T',\n",
              "   'Hulsman, Marc',\n",
              "   'Wu, Liang',\n",
              "   'Pel, Herman J',\n",
              "   'Roubos, Johannes A',\n",
              "   'de Ridder, Dick'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Artificial Intelligence',\n",
              "   'Aspergillus niger',\n",
              "   'Computational Biology',\n",
              "   'Electrophoresis, Polyacrylamide Gel',\n",
              "   'Enzymes',\n",
              "   'Fungal Proteins',\n",
              "   'Gene Expression Profiling',\n",
              "   'Genes, Fungal',\n",
              "   'Genetic Engineering',\n",
              "   'Industrial Microbiology',\n",
              "   'Molecular Sequence Data'],\n",
              "  'pmid': '23049690',\n",
              "  'title': 'Exploring sequence characteristics related to high-level production of secreted proteins in Aspergillus niger.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'The hexactinellids are a diverse group of predominantly deep sea sponges that synthesize elaborate fibrous skeletal systems of amorphous hydrated silica. As a representative example, members of the genus Euplectella have proved to be useful model systems for investigating structure-function relationships in these hierarchically ordered siliceous network-like composites. Despite recent advances in understanding the mechanistic origins of damage tolerance in these complex skeletal systems, the details of their synthesis have remained largely unexplored. Here, we describe a previously unidentified protein, named \"glassin,\" the main constituent in the water-soluble fraction of the demineralized skeletal elements of Euplectella. When combined with silicic acid solutions, glassin rapidly accelerates silica polycondensation over a pH range of 6-8. Glassin is characterized by high histidine content, and cDNA sequence analysis reveals that glassin shares no significant similarity with any other known proteins. The deduced amino acid sequence reveals that glassin consists of two similar histidine-rich domains and a connecting domain. Each of the histidine-rich domains is composed of three segments: an amino-terminal histidine and aspartic acid-rich sequence, a proline-rich sequence in the middle, and a histidine and threonine-rich sequence at the carboxyl terminus. Histidine always forms HX or HHX repeats, in which most of X positions are occupied by glycine, aspartic acid, or threonine. Recombinant glassin reproduces the silica precipitation activity observed in the native proteins. The highly modular composition of glassin, composed of imidazole, acidic, and hydroxyl residues, favors silica polycondensation and provides insights into the molecular mechanisms of skeletal formation in hexactinellid sponges. ',\n",
              "  'authors': ['Shimizu, Katsuhiko',\n",
              "   'Amano, Taro',\n",
              "   'Bari, Md Rezaul',\n",
              "   'Weaver, James C',\n",
              "   'Arima, Jiro',\n",
              "   'Mori, Nobuhiro'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Amino Acids',\n",
              "   'Animals',\n",
              "   'Aspartic Acid',\n",
              "   'Binding Sites',\n",
              "   'Cloning, Molecular',\n",
              "   'DNA, Complementary',\n",
              "   'Electrophoresis, Polyacrylamide Gel',\n",
              "   'Epitopes',\n",
              "   'Geography',\n",
              "   'Histidine',\n",
              "   'Hydrogen-Ion Concentration',\n",
              "   'Hydrolysis',\n",
              "   'Molecular Sequence Data',\n",
              "   'Peptides',\n",
              "   'Porifera',\n",
              "   'Proline',\n",
              "   'Protein Processing, Post-Translational',\n",
              "   'Proteins',\n",
              "   'Recombinant Proteins',\n",
              "   'Sequence Homology, Amino Acid',\n",
              "   'Silicon Dioxide',\n",
              "   'Solubility',\n",
              "   'Temperature',\n",
              "   'Threonine'],\n",
              "  'pmid': '26261346',\n",
              "  'title': 'Glassin, a histidine-rich protein from the siliceous skeletal system of the marine sponge Euplectella, directs silica polycondensation.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Characterization of the molecular attributes and spatial arrangements of cells and features within complex human tissues provides a critical basis for understanding processes involved in development and disease. Moreover, the ability to automate steps in the analysis and interpretation of histological images that currently require manual inspection by pathologists could revolutionize medical diagnostics. Toward this end, we developed a new imaging approach called multidimensional microscopic molecular profiling (MMMP) that can measure several independent molecular properties in situ at subcellular resolution for the same tissue specimen. MMMP involves repeated cycles of antibody or histochemical staining, imaging, and signal removal, which ultimately can generate information analogous to a multidimensional flow cytometry analysis on intact tissue sections. We performed a MMMP analysis on a tissue microarray containing a diverse set of 102 human tissues using a panel of 15 informative antibody and 5 histochemical stains plus DAPI. Large-scale unsupervised analysis of MMMP data, and visualization of the resulting classifications, identified molecular profiles that were associated with functional tissue features. We then directly annotated H&E images from this MMMP series such that canonical histological features of interest (e.g. blood vessels, epithelium, red blood cells) were individually labeled. By integrating image annotation data, we identified molecular signatures that were associated with specific histological annotations and we developed statistical models for automatically classifying these features. The classification accuracy for automated histology labeling was objectively evaluated using a cross-validation strategy, and significant accuracy (with a median per-pixel rate of 77% per feature from 15 annotated samples) for de novo feature prediction was obtained. These results suggest that high-dimensional profiling may advance the development of computer-based systems for automatically parsing relevant histological and cellular features from molecular imaging data of arbitrary human tissue samples, and can provide a framework and resource to spur the optimization of these technologies. ',\n",
              "  'authors': ['Riordan, Daniel P',\n",
              "   'Varma, Sushama',\n",
              "   'West, Robert B',\n",
              "   'Brown, Patrick O'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Automation',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Microscopy',\n",
              "   'Molecular Imaging',\n",
              "   'Software',\n",
              "   'Tissue Array Analysis',\n",
              "   'Unsupervised Machine Learning'],\n",
              "  'pmid': '26176839',\n",
              "  'title': 'Automated Analysis and Classification of Histological Tissue Features by Multi-Dimensional Microscopic Molecular Profiling.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1399-3054',\n",
              "  'IF_2013': '3.262',\n",
              "  'IF_2014': '3.138',\n",
              "  'IF_2015': '3.52',\n",
              "  'IF_2016': '3.33',\n",
              "  'IF_2017': '2.58',\n",
              "  'ISSN': '0031-9317',\n",
              "  'ISSN_online': '1399-3054',\n",
              "  'ISSN_print': '0031-9317',\n",
              "  'IsoAbbr': 'Physiol Plant',\n",
              "  'JrId': 8676,\n",
              "  'MedAbbr': 'Physiol Plant',\n",
              "  'NLMid': '1256322',\n",
              "  'Titre': 'Physiol Plant',\n",
              "  'abstract': 'Hexavalent chromium [Cr(VI)] is a non-essential metal for normal plants and is toxic to plants at high concentrations. However, signaling pathways and molecular mechanisms of its action on cell function and gene expression remain elusive. In this study, we found that Cr(VI) induced endogenous reactive oxygen species (ROS) generation and Ca(2+) accumulation and activated NADPH oxidase and calcium-dependent protein kinase. We investigated global transcriptional changes in rice roots by microarray analysis. Gene expression profiling indicated activation of abscisic acid-, ethylene- and jasmonic acid-mediated signaling and inactivation of gibberellic acid-related pathways in Cr(VI) stress-treated rice roots. Genes encoding signaling components such as the protein kinases domain of unknown function 26, receptor-like cytoplasmic kinase, LRK10-like kinase type 2 and protein phosphatase 2C, as well as transcription factors WRKY and apetala2/ethylene response factor were predominant during Cr(VI) stress. Genes involved in vesicle trafficking were subjected to functional characterization. Pretreating rice roots with a vesicle trafficking inhibitor, brefeldin A, effectively reduced Cr(VI)-induced ROS production. Suppression of the vesicle trafficking gene, Exo70, by virus-induced gene silencing strategies revealed that vesicle trafficking is required for mediation of Cr(VI)-induced ROS production. Taken together, these findings shed light on the molecular mechanisms in signaling pathways and transcriptional regulation in response to Cr stress in plants.',\n",
              "  'authors': ['Trinh, Ngoc-Nam',\n",
              "   'Huang, Tsai-Lien',\n",
              "   'Chi, Wen-Chang',\n",
              "   'Fu, Shih-Feng',\n",
              "   'Chen, Chi-Chien',\n",
              "   'Huang, Hao-Jen'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Physiologia plantarum',\n",
              "  'keywords': ['Biological Assay',\n",
              "   'Brefeldin A',\n",
              "   'Calcium',\n",
              "   'Chromium',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Gene Silencing',\n",
              "   'Genes, Plant',\n",
              "   'Models, Biological',\n",
              "   'NADPH Oxidases',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Oryza',\n",
              "   'Plant Proteins',\n",
              "   'Plant Roots',\n",
              "   'Protein Kinases',\n",
              "   'RNA, Messenger',\n",
              "   'Reactive Oxygen Species',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
              "   'Seedlings',\n",
              "   'Signal Transduction',\n",
              "   'Tobacco'],\n",
              "  'pmid': '24033343',\n",
              "  'title': 'Chromium stress response effect on signal transduction and expression of signaling genes in rice.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EA'],\n",
              "  'ESSN': '1520-6882',\n",
              "  'IF_2013': '5.825',\n",
              "  'IF_2014': '5.636',\n",
              "  'IF_2015': '5.886',\n",
              "  'IF_2016': '6.32',\n",
              "  'IF_2017': '6.042',\n",
              "  'ISSN': '0003-2700',\n",
              "  'ISSN_online': '1520-6882',\n",
              "  'ISSN_print': '0003-2700',\n",
              "  'IsoAbbr': 'Anal. Chem.',\n",
              "  'JrId': 519,\n",
              "  'MedAbbr': 'Anal Chem',\n",
              "  'NLMid': '0370536',\n",
              "  'Titre': 'Anal. Chem.',\n",
              "  'abstract': 'Neural network analysis of the response of an array of vapor-sensitive detectors has been used to identify six different types of aviation fuel. The data set included 96 samples of JP-4, JP-5, JP-7, JP-8, JetA, and aviation gasoline (AvGas). A sample of each neat fuel was injected into a continuous stream of breathing air through an injection port from a gas chromatograph. The aspirated sample was then swept from the injection port to the chamber without separation. In the chamber, the sample was exposed to an array of eight vapor-sensitive detectors. The analog output of each detector was digitized and stored while the sample was swept into and through the chamber. The response of each detector was then averaged and stored as the final response or pattern of each sample. It was clear from a visual inspection of each of the radar plots that there was a characteristic pattern in the response of the array to five of the six different fuel types. This was confirmed using neural network analysis to study the entire data set. A two-step procedure was developed to separate the patterns of all six fuel tyes into their respective classes. In the first step, fuels were separated into one of five groups: JP-4, JP-5, JP-7, AvGas, or a combined JP-8/JetA group. In the second step, the fuels in the combined group were separated into either JP-8 or JetA groups.',\n",
              "  'authors': ['McCarrick, C W',\n",
              "   'Ohmer, D T',\n",
              "   'Gilliland, L A',\n",
              "   'Edwards, P A',\n",
              "   'Mayfield, H T'],\n",
              "  'categories': ['CHEMISTRY, ANALYTICAL'],\n",
              "  'journal': 'Analytical chemistry',\n",
              "  'keywords': ['Chromatography, Gas',\n",
              "   'Gasoline',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '8946795',\n",
              "  'title': 'Fuel identification by neural network analysis of the response of vapor-sensitive sensor arrays.',\n",
              "  'year': '1996'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1433-8726',\n",
              "  'IF_2013': '3.423',\n",
              "  'IF_2014': '2.666',\n",
              "  'IF_2015': '2.397',\n",
              "  'IF_2016': '2.743',\n",
              "  'IF_2017': '2.981',\n",
              "  'ISSN': '0724-4983',\n",
              "  'ISSN_online': '1433-8726',\n",
              "  'ISSN_print': '0724-4983',\n",
              "  'IsoAbbr': 'World J Urol',\n",
              "  'JrId': 2429,\n",
              "  'MedAbbr': 'World J Urol',\n",
              "  'NLMid': '8307716',\n",
              "  'Titre': 'World J Urol',\n",
              "  'abstract': 'Our objective was to determine whether or not male reproductive microsurgery is adaptable to current robotic technology. We devised a model vas deferens system using rat vasa deferentia and completed vasal anastomosis with full-thickness and mucosal robotically placed, Sharpoint 10-0 bicurve nylon sutures. Experienced and inexperienced microsurgeons performed separate anastomoses. Both groups of microsurgeons completed anastomoses with accuracy and enhanced comfort. The robotic graspers had the dexterity to delicately handle the 10-0 sutures and needles. We conclude that through our model vas deferens system, we demonstrated the feasibility of applying robotic technology to male reproductive microsurgery. This pilot study reveals some advantages of the robot and suggests future use of this system in a new urological application domain.',\n",
              "  'authors': ['Schoor, Richard A', 'Ross, Lawrence', 'Niederberger, Craig'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'World journal of urology',\n",
              "  'keywords': ['Anastomosis, Surgical',\n",
              "   'Animals',\n",
              "   'Male',\n",
              "   'Microsurgery',\n",
              "   'Rats',\n",
              "   'Rats, Sprague-Dawley',\n",
              "   'Reconstructive Surgical Procedures',\n",
              "   'Robotics',\n",
              "   'Vas Deferens'],\n",
              "  'pmid': '12720036',\n",
              "  'title': 'Robotic assisted microsurgical vasal reconstruction in a model system.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1744-8328',\n",
              "  'IF_2013': '2.279',\n",
              "  'IF_2014': '2.249',\n",
              "  'IF_2015': '2.094',\n",
              "  'IF_2016': '2.212',\n",
              "  'IF_2017': '2.347',\n",
              "  'ISSN': '1473-7140',\n",
              "  'ISSN_online': '1744-8328',\n",
              "  'ISSN_print': '1473-7140',\n",
              "  'IsoAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'JrId': 29775,\n",
              "  'MedAbbr': 'Expert Rev Anticancer Ther',\n",
              "  'NLMid': '101123358',\n",
              "  'Titre': 'Expert Rev Anticancer Ther',\n",
              "  'abstract': 'Cancer of the prostate is the most common malignancy diagnosed in the male genitourinary tract. Although a number of treatment options are available for early prostate cancer, the gold standard of treatment remains retropubic radical prostatectomy. However, robotic-assisted laparoscopic radical prostatectomy (RALP) has become a forerunner in treatment options, yielding comparable medium-term perioperative and functional outcomes. For this our team utilized MEDLINE, searching for publications on perioperative and functional outcomes related to robotic prostatectomy. Robotic-assisted prostatectomy has allowed urologists to enter the realm of minimally invasive surgery by incorporating open surgery maneuvers into a laparoscopic environment. To date, RALP perioperative and functional outcomes are comparable to the gold standard; the collection of long-term data is needed in order to establish its true efficacy.',\n",
              "  'authors': ['Patel, Vipul R',\n",
              "   'Shah, Ketul',\n",
              "   'Palmer, Kenneth J',\n",
              "   'Thaly, Rahul',\n",
              "   'Coughlin, Geoff'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Expert review of anticancer therapy',\n",
              "  'keywords': ['Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics'],\n",
              "  'pmid': '17892427',\n",
              "  'title': 'Robotic-assisted laparoscopic radical prostatectomy: a report of the current state.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.286',\n",
              "  'IF_2014': '0.25',\n",
              "  'IF_2015': '0.3',\n",
              "  'IF_2016': '0.421',\n",
              "  'IF_2017': '0.167',\n",
              "  'ISSN': '0035-6050',\n",
              "  'ISSN_print': '0035-6050',\n",
              "  'IsoAbbr': 'Riv. Biol.',\n",
              "  'JrId': 7307,\n",
              "  'MedAbbr': 'Riv Biol',\n",
              "  'NLMid': '0416643',\n",
              "  'Titre': 'Riv. Biol.',\n",
              "  'abstract': 'We present a biomechanical model of morphogenesis highlighting the extensive formative capacities of stressed networks with a very simple initial geometry. They consist of a restricted number of kinematically independent elements exerting a pressure to each other and increasing thus the local curvatures. The pressure is applied as a series of periodic impulses and is opposed by a constant quasi-elastic resistance force. Single elements can be also regarded as the half wave-lengths of the undulations determined by the mechanical properties of a given body. All of the model parameters are assumed to be evenly spread throughout a body (no prepatterns are implied). On the other hand, the model parameters can be associated with genetic factors. Thus, our model relates to as yet unsolved problem of genetic regulation of shape formation. We classify the modeled shapes according to their symmetry orders and compare them with the ancient Echinodermata and with Arthropods. Possible evolutionary and developmental implications are discussed.',\n",
              "  'authors': ['Beloussov, Lev V', 'Grabovsky, Vassily I'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Rivista di biologia',\n",
              "  'keywords': ['Biological Evolution',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Stress, Mechanical'],\n",
              "  'pmid': '15055879',\n",
              "  'title': 'Formative capacities of mechanically stressed networks: developmental and evolutionary implications.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'By using confocal Raman spectroscopy, Raman spectra were measured in normal rat red blood cells, normal human red blood cells, STZ induced diabetetic rats red blood cells, Alloxan induced diabetetic rats red blood cells and human type 2 diabetes red blood cells. Then principal component analysis (PCA) with support vector machine (SVM) classifier was used for data analysis, and then the distance between classes was used to judge the degree of close to two kinds of rat model with type 2 diabetes. The results found significant differences in the Raman spectra of red blood cell in diabetic and normal red blood cells. To diabetic red blood cells, the peak in the amide VI C=O deformation vibration band is obvious, and amide V N-H deformation vibration band spectral lines appear deviation. Belong to phospholipid fatty acyl C-C skeleton, the 1 130 cm(-1) spectral line is enhanced and the 1 088 cm(-1) spectral line is abated, which show diabetes red cell membrane permeability increased. Raman spectra of PCA combined with SVM can well separate 5 types of red blood cells. Classifier test results show that the classification accuracy is up to 100%. Through the class distance between the two induced method and human type 2 diabetes, it is found that STZ induced model is more close to human type 2 diabetes. In conclusion, Raman spectroscopy can be used for diagnosis of diabetes and rats STZ induced diabetes method is closer to human type 2 diabetes.',\n",
              "  'authors': ['Wang, Lei',\n",
              "   'Liu, Gui-dong',\n",
              "   'Mu, Xin',\n",
              "   'Xiao, Hong-bin',\n",
              "   'Qi, Chao',\n",
              "   'Zhang, Si-qi',\n",
              "   'Niu Wen-ying, Guang-kun',\n",
              "   'Jiang, Yue-nan',\n",
              "   'Feng, Jing-qi',\n",
              "   'Bian, Lei'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Animals',\n",
              "   'Diabetes Mellitus, Type 2',\n",
              "   'Erythrocyte Count',\n",
              "   'Erythrocytes',\n",
              "   'Humans',\n",
              "   'Principal Component Analysis',\n",
              "   'Rats',\n",
              "   'Spectrum Analysis, Raman',\n",
              "   'Support Vector Machine',\n",
              "   'Vibration'],\n",
              "  'pmid': '26904817',\n",
              "  'title': '[Red Blood Cells Raman Spectroscopy Comparison of Type Two Diabetes Patients and Rats].',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1533-0338',\n",
              "  'IF_2013': '1.886',\n",
              "  'IF_2014': '1.73',\n",
              "  'IF_2015': '2.185',\n",
              "  'IF_2016': '2.204',\n",
              "  'IF_2017': '1.646',\n",
              "  'ISSN': '1533-0346',\n",
              "  'ISSN_online': '1533-0338',\n",
              "  'ISSN_print': '1533-0346',\n",
              "  'IsoAbbr': 'Technol. Cancer Res. Treat.',\n",
              "  'JrId': 30245,\n",
              "  'MedAbbr': 'Technol Cancer Res Treat',\n",
              "  'NLMid': '101140941',\n",
              "  'Titre': 'Technol. Cancer Res. Treat.',\n",
              "  'abstract': 'In this work, we have proposed an on-line computer-aided diagnostic system called \"UroImage\" that classifies a Transrectal Ultrasound (TRUS) image into cancerous or non-cancerous with the help of non-linear Higher Order Spectra (HOS) features and Discrete Wavelet Transform (DWT) coefficients. The UroImage system consists of an on-line system where five significant features (one DWT-based feature and four HOS-based features) are extracted from the test image. These on-line features are transformed by the classifier parameters obtained using the training dataset to determine the class. We trained and tested six classifiers. The dataset used for evaluation had 144 TRUS images which were split into training and testing sets. Three-fold and ten-fold cross-validation protocols were adopted for training and estimating the accuracy of the classifiers. The ground truth used for training was obtained using the biopsy results. Among the six classifiers, using 10-fold cross-validation technique, Support Vector Machine and Fuzzy Sugeno classifiers presented the best classification accuracy of 97.9% with equally high values for sensitivity, specificity and positive predictive value. Our proposed automated system, which achieved more than 95% values for all the performance measures, can be an adjunct tool to provide an initial diagnosis for the identification of patients with prostate cancer. The technique, however, is limited by the limitations of 2D ultrasound guided biopsy, and we intend to improve our technique by using 3D TRUS images in the future. ',\n",
              "  'authors': ['Pareek, Gyan',\n",
              "   'Acharya, U Rajendra',\n",
              "   'Sree, S Vinitha',\n",
              "   'Swapna, G',\n",
              "   'Yantri, Ratna',\n",
              "   'Martis, Roshan Joy',\n",
              "   'Saba, Luca',\n",
              "   'Krishnamurthi, Ganapathy',\n",
              "   'Mallarini, Giorgio',\n",
              "   'El-Baz, Ayman',\n",
              "   'Al Ekish, Shadi',\n",
              "   'Beland, Michael',\n",
              "   'Suri, Jasjit S'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Technology in cancer research & treatment',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prostate',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Rectum',\n",
              "   'Retrospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Support Vector Machine',\n",
              "   'Ultrasonography'],\n",
              "  'pmid': '23745787',\n",
              "  'title': 'Prostate tissue characterization/classification in 144 patient population using wavelet and higher order spectra features from transrectal ultrasound images.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1742-4658',\n",
              "  'IF_2013': '3.986',\n",
              "  'IF_2014': '4.001',\n",
              "  'IF_2015': '4.237',\n",
              "  'IF_2016': '3.902',\n",
              "  'IF_2017': '4.53',\n",
              "  'ISSN': '1742-464X',\n",
              "  'ISSN_online': '1742-4658',\n",
              "  'ISSN_print': '1742-464X',\n",
              "  'IsoAbbr': 'FEBS J.',\n",
              "  'JrId': 32231,\n",
              "  'MedAbbr': 'FEBS J',\n",
              "  'NLMid': '101229646',\n",
              "  'Titre': 'FEBS J.',\n",
              "  'abstract': \"The newly emerging field of systems biology involves a judicious interplay between high-throughput 'wet' experimentation, computational modelling and technology development, coupled to the world of ideas and theory. This interplay involves iterative cycles, such that systems biology is not at all confined to hypothesis-dependent studies, with intelligent, principled, hypothesis-generating studies being of high importance and consequently very far from aimless fishing expeditions. I seek to illustrate each of these facets. Novel technology development in metabolomics can increase substantially the dynamic range and number of metabolites that one can detect, and these can be exploited as disease markers and in the consequent and principled generation of hypotheses that are consistent with the data and achieve this in a value-free manner. Much of classical biochemistry and signalling pathway analysis has concentrated on the analyses of changes in the concentrations of intermediates, with 'local' equations - such as that of Michaelis and Menten v=(Vmax x S)/(S+K m) - that describe individual steps being based solely on the instantaneous values of these concentrations. Recent work using single cells (that are not subject to the intellectually unsupportable averaging of the variable displayed by heterogeneous cells possessing nonlinear kinetics) has led to the recognition that some protein signalling pathways may encode their signals not (just) as concentrations (AM or amplitude-modulated in a radio analogy) but via changes in the dynamics of those concentrations (the signals are FM or frequency-modulated). This contributes in principle to a straightforward solution of the crosstalk problem, leads to a profound reassessment of how to understand the downstream effects of dynamic changes in the concentrations of elements in these pathways, and stresses the role of signal processing (and not merely the intermediates) in biological signalling. It is this signal processing that lies at the heart of understanding the languages of cells. The resolution of many of the modern and postgenomic problems of biochemistry requires the development of a myriad of new technologies (and maybe a new culture), and thus regular input from the physical sciences, engineering, mathematics and computer science. One solution, that we are adopting in the Manchester Interdisciplinary Biocentre (http://www.mib.ac.uk/) and the Manchester Centre for Integrative Systems Biology (http://www.mcisb.org/), is thus to colocate individuals with the necessary combinations of skills. Novel disciplines that require such an integrative approach continue to emerge. These include fields such as chemical genomics, synthetic biology, distributed computational environments for biological data and modelling, single cell diagnostics/bionanotechnology, and computational linguistics/text mining.\",\n",
              "  'authors': ['Kell, Douglas B'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'The FEBS journal',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Hungary',\n",
              "   'Metabolism',\n",
              "   'Models, Biological',\n",
              "   'Systems Biology',\n",
              "   'United Kingdom'],\n",
              "  'pmid': '16478464',\n",
              "  'title': 'Theodor Bücher Lecture. Metabolomics, modelling and machine learning in systems biology - towards an understanding of the languages of cells. Delivered on 3 July 2005 at the 30th FEBS Congress and the 9th IUBMB conference in Budapest.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'The 3D convolutional neural network (CNN) is able to make full use of the spatial 3D context information of lung nodules, and the multi-view strategy has been shown to be useful for improving the performance of 2D CNN in classifying lung nodules. In this paper, we explore the classification of lung nodules using the 3D multi-view convolutional neural networks (MV-CNN) with both chain architecture and directed acyclic graph architecture, including 3D Inception and 3D Inception-ResNet. All networks employ the multi-view-one-network strategy. We conduct a binary classification (benign and malignant) and a ternary classification (benign, primary malignant and metastatic malignant) on Computed Tomography (CT) images from Lung Image Database Consortium and Image Database Resource Initiative database (LIDC-IDRI). All results are obtained via 10-fold cross validation. As regards the MV-CNN with chain architecture, results show that the performance of 3D MV-CNN surpasses that of 2D MV-CNN by a significant margin. Finally, a 3D Inception network achieved an error rate of 4.59% for the binary classification and 7.70% for the ternary classification, both of which represent superior results for the corresponding task. We compare the multi-view-one-network strategy with the one-view-one-network strategy. The results reveal that the multi-view-one-network strategy can achieve a lower error rate than the one-view-one-network strategy.',\n",
              "  'authors': ['Kang, Guixia', 'Liu, Kui', 'Hou, Beibei', 'Zhang, Ningbo'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Humans',\n",
              "   'Lung Neoplasms',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Solitary Pulmonary Nodule',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '29145492',\n",
              "  'title': '3D multi-view convolutional neural networks for lung nodule classification.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1538-3644',\n",
              "  'IF_2013': '4.297',\n",
              "  'IF_2014': '4.926',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0004-0010',\n",
              "  'ISSN_online': '1538-3644',\n",
              "  'ISSN_print': '0004-0010',\n",
              "  'IsoAbbr': 'Arch Surg',\n",
              "  'JrId': 845,\n",
              "  'MedAbbr': 'Arch Surg',\n",
              "  'NLMid': '9716528',\n",
              "  'Titre': 'Arch Surg',\n",
              "  'abstract': 'Total laparoscopic pancreaticoduodenectomy is a safe and effective therapeutic approach.|Single-institutional retrospective review.|Tertiary referral center.|All consecutive patients undergoing total laparoscopic pancreaticoduodenectomy from July 2007 through July 2009 at a single center (n = 62).|Blood loss, operative time, postoperative morbidity, length of hospital stay, and 30-day or in-hospital mortality.|Of 65 patients undergoing laparoscopic resection, 62 patients with a mean age of 66 years (SD, 12 years) underwent total laparoscopic pancreaticoduodenectomy. The pancreaticojejunostomy consisted of a duct-to-mucosa anastomosis with interrupted suture. Median operative time was 368 minutes (range, 258-608 minutes) and median blood loss was 240 mL (range, 30-1200 mL). Diagnosis was pancreatic adenocarcinoma (n = 31), intraductal papillary mucinous neoplasm (n = 12), periampullary adenocarcinoma (n = 8), neuroendocrine tumor (n = 4), chronic pancreatitis (n = 3), cholangiocarcinoma (n = 1), metastatic renal cell carcinoma (n = 1), cystadenoma (n = 1), and duodenal adenoma (n = 1). Median tumor size was 3 cm (range, 0.9-10.0 cm) and the median number of lymph nodes harvested was 15 (range, 6-31). Perioperative morbidity occurred in 26 patients and included pancreatic fistula (n = 11), delayed gastric emptying (n = 9), bleeding (n = 5), and deep vein thrombosis (n = 2). There was 1 postoperative mortality. Median length of hospital stay was 7 days (range, 4-69 days).|Laparoscopic pancreaticoduodenectomy is feasible, safe, and effective. Outcomes appear comparable with those via the open approach; however, controlled trials are needed. Despite this series representing experience within the learning curve, laparoscopic pancreaticoduodenectomy holds promise for providing advantages seen with minimally invasive approaches in other procedures.',\n",
              "  'authors': ['Kendrick, Michael L', 'Cusati, Daniel'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Archives of surgery (Chicago, Ill. : 1960)',\n",
              "  'keywords': ['Digestive System Diseases',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pancreatic Diseases',\n",
              "   'Pancreatic Neoplasms',\n",
              "   'Pancreaticoduodenectomy',\n",
              "   'Retrospective Studies',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '20083750',\n",
              "  'title': 'Total laparoscopic pancreaticoduodenectomy: feasibility and outcome in an early experience.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DB'],\n",
              "  'ESSN': '1475-2859',\n",
              "  'IF_2013': '4.25',\n",
              "  'IF_2014': '4.221',\n",
              "  'IF_2015': '3.744',\n",
              "  'IF_2016': '3.681',\n",
              "  'IF_2017': '3.831',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-2859',\n",
              "  'IsoAbbr': 'Microb. Cell Fact.',\n",
              "  'JrId': 29781,\n",
              "  'MedAbbr': 'Microb Cell Fact',\n",
              "  'NLMid': '101139812',\n",
              "  'Titre': 'Microb. Cell Fact.',\n",
              "  'abstract': 'For many microbial processes, the complexity of the metabolisms and the responses to transient and realistic conditions are difficult to capture in mechanistic models. The cells seem to have an innate intelligence that enables them to respond optimally to environmental changes. Some \"intelligent\" models have therefore been proposed and compared with a mechanistic model for fed-batch cultures of Ralstonia eutropha.|Two kinds of models have been proposed to describe such cellular behavior. Cybernetic models are derived through postulates of cellular intelligence and memory, and neural models use artificial intelligence through neural networks. Some competing models of both kinds have been compared for their ability to portray and optimize the synthesis of poly-beta-hydroxybutyrate by Ralstonia eutropha in fed-batch cultures with finite dispersion. Neural models enabled the formation of more of the polymer than cybernetic models, with lesser utilization of the carbon and nitrogen substrates. Both types of models were decidedly superior to a mechanistic model used as a reference, thus supporting the value of intelligent descriptions of microbial kinetics in incompletely dispersed bioreactors.|Neural and cybernetic models describe and optimize unsteady state fed-batch microbial reactors with finite dispersion more effectively than mechanistic models. However, these \"intelligent\" models too have weaknesses, and hence a hybrid approach combining such models with some mechanistic features is suggested.',\n",
              "  'authors': ['Patnaik, Pratap R'],\n",
              "  'categories': ['BIOTECHNOLOGY & APPLIED MICROBIOLOGY'],\n",
              "  'journal': 'Microbial cell factories',\n",
              "  'keywords': [''],\n",
              "  'pmid': '17686143',\n",
              "  'title': '\"Intelligent\" descriptions of microbial kinetics in finitely dispersed bioreactors: neural and cybernetic models for PHB biosynthesis by Ralstonia eutropha.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1534-6285',\n",
              "  'IF_2013': '1.513',\n",
              "  'IF_2014': '1.513',\n",
              "  'IF_2015': '1.597',\n",
              "  'IF_2016': '2.124',\n",
              "  'IF_2017': '1.983',\n",
              "  'ISSN': '1527-2737',\n",
              "  'ISSN_online': '1534-6285',\n",
              "  'ISSN_print': '1527-2737',\n",
              "  'IsoAbbr': 'Curr Urol Rep',\n",
              "  'JrId': 29748,\n",
              "  'MedAbbr': 'Curr Urol Rep',\n",
              "  'NLMid': '100900943',\n",
              "  'Titre': 'Curr Urol Rep',\n",
              "  'abstract': 'Inguinal lymph node dissection is a diagnostic and potentially curative treatment for penile carcinoma, which has historically been associated with high morbidity rates. This review summarizes the initial outcomes of robotic-assisted inguinal lymphadenectomy (RAIL) compared with the outcomes of the standard open and endoscopic approaches. The early experience suggests that RAIL may yield comparable oncologic outcomes, although future prospective studies of RAIL with greater numbers of participants and long-term follow-up are needed to evaluate the incidence and severity of perioperative and postoperative complications. ',\n",
              "  'authors': ['Kharadjian, Talar B', 'Matin, Surena F', 'Pettaway, Curtis A'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Current urology reports',\n",
              "  'keywords': ['Carcinoma, Squamous Cell',\n",
              "   'Humans',\n",
              "   'Inguinal Canal',\n",
              "   'Lymph Node Excision',\n",
              "   'Lymph Nodes',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Penile Neoplasms',\n",
              "   'Robotics',\n",
              "   'Sentinel Lymph Node Biopsy',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '24756453',\n",
              "  'title': 'Early experience of robotic-assisted inguinal lymphadenectomy: review of surgical outcomes relative to alternative approaches.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Grammatical agreement means that features associated with one linguistic unit (for example number or gender) become associated with another unit and then possibly overtly expressed, typically with morphological markers. It is one of the key mechanisms used in many languages to show that certain linguistic units within an utterance grammatically depend on each other. Agreement systems are puzzling because they can be highly complex in terms of what features they use and how they are expressed. Moreover, agreement systems have undergone considerable change in the historical evolution of languages. This article presents language game models with populations of agents in order to find out for what reasons and by what cultural processes and cognitive strategies agreement systems arise. It demonstrates that agreement systems are motivated by the need to minimize combinatorial search and semantic ambiguity, and it shows, for the first time, that once a population of agents adopts a strategy to invent, acquire and coordinate meaningful markers through social learning, linguistic self-organization leads to the spontaneous emergence and cultural transmission of an agreement system. The article also demonstrates how attested grammaticalization phenomena, such as phonetic reduction and conventionalized use of agreement markers, happens as a side effect of additional economizing principles, in particular minimization of articulatory effort and reduction of the marker inventory. More generally, the article illustrates a novel approach for studying how key features of human languages might emerge.',\n",
              "  'authors': ['Beuls, Katrien', 'Steels, Luc'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Coercion',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Language',\n",
              "   'Linguistics',\n",
              "   'Models, Theoretical',\n",
              "   'Phonetics',\n",
              "   'Semantics'],\n",
              "  'pmid': '23527055',\n",
              "  'title': 'Agent-based models of strategies for the emergence and evolution of grammatical agreement.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'Partial nephrectomy (PN) is a technically challenging procedure, making selection of appropriate patients paramount to a successful operation. To identify patients at increased risk of an adverse outcome after PN, there are a number of scoring systems available. The nephrometry score was initially described in a series of laparoscopic and open partial and radical nephrectomies. We compare the association of the nephrometry score with perioperative outcomes in a population of robot-assisted partial nephrectomies. A total of 119 patients were retrospectively reviewed. Correlation and regressional analysis was performed. We identified the separate variables R, E, N, and L to have limited correlation and no predictive value to patient outcomes. Nephrometry score and grade were found to have stronger correlation and predictive value than the individual components of the R.E.N.A.L. nephrometry score. Size of tumor measured on a continuous scale was found to have the strongest correlation and predictive value to outcomes. Outcomes predicted included operative time, length of stay, warm ischemia time, and entry into the collecting system.',\n",
              "  'authors': ['Sea, Jason C',\n",
              "   'Bahler, Clinton D',\n",
              "   'Mendonsa, Eric',\n",
              "   'Lucas, Steven M',\n",
              "   'Sundaram, Chandru P'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Kidney',\n",
              "   'Kidney Neoplasms',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nephrectomy',\n",
              "   'Prognosis',\n",
              "   'Retrospective Studies',\n",
              "   'Robotics',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Treatment Outcome',\n",
              "   'Young Adult'],\n",
              "  'pmid': '24205980',\n",
              "  'title': 'Comparison of measured renal tumor size versus R.E.N.A.L. nephrometry score in predicting patient outcomes after robot-assisted laparoscopic partial nephrectomy.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1768-3114',\n",
              "  'IF_2013': '1.074',\n",
              "  'IF_2014': '1.2',\n",
              "  'IF_2015': '1.009',\n",
              "  'IF_2016': '1.802',\n",
              "  'IF_2017': '1.9',\n",
              "  'ISSN': '0369-8114',\n",
              "  'ISSN_online': '1768-3114',\n",
              "  'ISSN_print': '0369-8114',\n",
              "  'IsoAbbr': 'Pathol. Biol.',\n",
              "  'JrId': 6376,\n",
              "  'MedAbbr': 'Pathol Biol (Paris)',\n",
              "  'NLMid': '0265365',\n",
              "  'Titre': 'Pathol. Biol.',\n",
              "  'abstract': 'An expert system (cadi-yac), written in Turbo-Prolog and working on IBM PC and Bacanal + (a management software of microbiology laboratory) was used to recognize and correct the phenotype of antibiotic sensibility. The results were there of API ATB system. The knowledge was adapted from two references works. A routine use of the expert system give a correct recognition of enzymatic profile in more than 80% cases for the beta-lactams and more than 98% cases for the aminosides. The mistakes detected by cadi-yac, were often interpreted as deficiency of API system by humans experts. The expert system mistakes (1.5%) were due composites phenotypes.',\n",
              "  'authors': ['Peyret, M', 'Flandrois, J P', 'Carret, G', 'Pichat, C'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Pathologie-biologie',\n",
              "  'keywords': ['Anti-Bacterial Agents',\n",
              "   'Enterobacteriaceae',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Microbial Sensitivity Tests',\n",
              "   'Quality Control'],\n",
              "  'pmid': '2797887',\n",
              "  'title': '[Interpretative reading and quality control of an antibiotic sensitivity test using an expert system. Application to the API ATB system and Enterobacteriaceae].',\n",
              "  'year': '1989'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Central Pattern Generators (CPGs) are a suitable paradigm to solve the problem of locomotion control in walking robots. CPGs are able to generate feed-forward signals to achieve a proper coordination among the robot legs. In literature they are often modelled as networks of coupled nonlinear systems. However the topic of feedback in these systems is rarely addressed. On the other hand feedback is essential for locomotion. In this paper the CPG for a hexapod robot is implemented through Cellular Neural Networks (CNNs). Feedback is included in the CPG controller by exploiting the dynamic properties of the CPG motor-neurons, such as synchronization issue and local bifurcations. These universal paradigms provide the essential issues to include sensory feedback in CPG architectures based on coupled nonlinear systems. Experiments on a dynamic model of a hexapod robot are presented to validate the approach introduced.',\n",
              "  'authors': ['Arena, Paolo',\n",
              "   'Fortuna, Luigi',\n",
              "   'Frasca, Mattia',\n",
              "   'Patane, Luca'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Feedback',\n",
              "   'Motor Activity',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Robotics'],\n",
              "  'pmid': '15031855',\n",
              "  'title': 'Sensory feedback in CNN-based central pattern generators.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1438-9010',\n",
              "  'IF_2013': '1.961',\n",
              "  'IF_2014': '1.402',\n",
              "  'IF_2015': '1.672',\n",
              "  'IF_2016': '1.554',\n",
              "  'IF_2017': '1.636',\n",
              "  'ISSN': '1438-9029',\n",
              "  'ISSN_online': '1438-9010',\n",
              "  'ISSN_print': '1438-9029',\n",
              "  'IsoAbbr': 'Rofo',\n",
              "  'JrId': 7048,\n",
              "  'MedAbbr': 'Rofo',\n",
              "  'NLMid': '7507497',\n",
              "  'Titre': 'Rofo',\n",
              "  'abstract': 'To assess renal artery stenosis (RAS) by 3D phase contrast (PC) MR angiography and dynamic perfusion imaging of the kidneys.|On a standard 1.0 T MR imaging system (Magnetom Expert, Siemens), 32 patients with angiographically proven unilateral RAS were examined using a 3D PC sequence (TR 40 ms/TE 9 ms/venc 30 cm/s). An ECG-gated Turbo-FLASH 2D sequence (TR 4.5 ms/TE 2.2 ms/TIeff. 400 ms) was applied to study the first pass of paramagnetic contrast agent (0.1 mmol Gd-DTPA/kg) through the kidneys. Signal intensity (SI) over time curves of the renal cortex were obtained and evaluated considering temporal relation and percentage of maximum SI compared to the aorta and normal kidneys. Analysis of the MRA was performed by two independent blinded readers. The gold-standard DSA was interpreted by consensus reading of two experienced radiologists.|RAS was detected by 3D PC MRA with a sensitivity of 93% and specificity of 81% (ppv 82%, npv 93%, accuracy 87%, kappa = 0.61). Maximum SI in RAS was significantly decreased (p < 0.001-0.0001). A temporally delayed enhancement of 1.5 +/- 1.3 s was found for RAS > 75% (p < 0.002) but not for RAS < 75% (p > 0.1).|3D PC MRA is capable of detecting RAS in a high percentage of patients. Dynamic perfusion imaging of the kidneys, applied additionally, can confirm the diagnosis and give valuable information about the hemodynamic relevance of RAS in suspected unilateral disease.',\n",
              "  'authors': ['Miller, S',\n",
              "   'Hahn, U',\n",
              "   'Schick, F',\n",
              "   'Nägele, T',\n",
              "   'Duda, S H',\n",
              "   'Eckstein, F S',\n",
              "   'Scheule, A M',\n",
              "   'Claussen, C D'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'RoFo : Fortschritte auf dem Gebiete der Rontgenstrahlen und der Nuklearmedizin',\n",
              "  'keywords': ['Contrast Media',\n",
              "   'Expert Systems',\n",
              "   'Gadolinium DTPA',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Angiography',\n",
              "   'Renal Artery Obstruction',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '10101356',\n",
              "  'title': '[Diagnosis of renal artery stenosis in 1.0 T using 3D phase contrast magnetic resonance angiography and dynamic contrast medium perfusion].',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'A unique capability of the CyberKnife system is dynamic target tracking. However, not all patients are eligible for this approach. Rather, their tumors are tracked statically using the vertebral column for alignment. When using static tracking, the internal target volume (ITV) is delineated on the four-dimensional (4D) CT scan and an additional margin is added to account for setup uncertainty [planning target volume (PTV)]. Treatment margins are difficult to estimate due to unpredictable variations in tumor motion and respiratory pattern during the course of treatment. The inability to track the target and detect changes in respiratory characteristics might result in geographic misses and local tumor recurrences. The purpose of this study is to develop a method to evaluate the adequacy of ITV-to-PTV margins for patients treated in this manner.|Data from 24 patients with lesions in the upper lobe (n = 12), middle lobe (n = 3), and lower lobe (n = 9) were included in this study. Each patient was treated with dynamic tracking and underwent 4DCT scanning at the time of simulation. Data including the 3D coordinates of the target over the course of treatment were extracted from the treatment log files and used to determine actual target motion in the superior-inferior (S-I), anterior-posterior (A-P), and left-right (L-R) directions. Different approaches were used to calculate anisotropic and isotropic margins, assuming that the tumor moves as a rigid body. Anisotropic margins were calculated by separating target motion in the three anatomical directions, and a uniform margin was calculated by shifting the gross tumor volume contours in the 3D space and by computing the percentage of overlap with the PTV. The analysis was validated by means of a theoretical formulation.|The three methods provided consistent results. A uniform margin of 4.5 mm around the ITV was necessary to assure 95% target coverage for 95% of the fractions included in the analysis. In the case of anisotropic margins, the expansion required in the S-I direction was larger (8.1 mm) than those in the L-R (4.9 mm) and A-P (4.5 mm) directions. This margin accounts for variations of target position within the same treatment fraction.|The use of bony alignment for CyberKnife lung stereotactic body radiation therapy requires careful considerations, in terms of the potential for increased toxicity or local miss. Our method could be used by other centers to determine the adequacy of ITV-to-PTV margins for their patients.',\n",
              "  'authors': ['Descovich, Martina',\n",
              "   'McGuinness, Christopher',\n",
              "   'Kannarunimit, Danita',\n",
              "   'Chen, Josephine',\n",
              "   'Pinnaduwage, Dilini',\n",
              "   'Pouliot, Jean',\n",
              "   'Kased, Norbert',\n",
              "   'Gottschalk, Alexander R',\n",
              "   'Yom, Sue S'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Aged',\n",
              "   'Four-Dimensional Computed Tomography',\n",
              "   'Humans',\n",
              "   'Lung Neoplasms',\n",
              "   'Movement',\n",
              "   'Radiosurgery',\n",
              "   'Radiotherapy Planning, Computer-Assisted',\n",
              "   'Robotics',\n",
              "   'Time Factors',\n",
              "   'Uncertainty'],\n",
              "  'pmid': '25735283',\n",
              "  'title': 'Comparison between target margins derived from 4DCT scans and real-time tumor motion tracking: insights from lung tumor patients treated with robotic radiosurgery.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['TC'],\n",
              "  'ESSN': '1433-0431',\n",
              "  'IF_2013': '0.665',\n",
              "  'IF_2014': '0.359',\n",
              "  'IF_2015': '0.373',\n",
              "  'IF_2016': '0.629',\n",
              "  'IF_2017': '0.632',\n",
              "  'ISSN': '0085-4530',\n",
              "  'ISSN_online': '1433-0431',\n",
              "  'ISSN_print': '0085-4530',\n",
              "  'IsoAbbr': 'Orthopade',\n",
              "  'JrId': 6314,\n",
              "  'MedAbbr': 'Orthopade',\n",
              "  'NLMid': '0331266',\n",
              "  'Titre': 'Orthopade',\n",
              "  'abstract': 'A new concept of THR surgery is presented: A combination of limited surgical approach, intraoperative digitizing, and use of a semi-active robot shows the important advantage of easy registration, reproducible positioning of surgical instruments, and less invasive surgery. Using this system originally designed for accurate socket implantation, it will be possible to perform total THR in the future. For preoperative planning, we are currently investigating the use of biplanar digital radiographics with transformation into a 3D-model in order to avoid CT scans in the pelvic area.',\n",
              "  'authors': ['Wahrburg, J', 'Kerschbaumer, F'],\n",
              "  'categories': ['ORTHOPEDICS'],\n",
              "  'journal': 'Der Orthopade',\n",
              "  'keywords': ['Arthroplasty, Replacement, Hip',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Robotics',\n",
              "   'Time Factors'],\n",
              "  'pmid': '10986711',\n",
              "  'title': '[Thoughts on the use of mechatronic implantation aids in minimal approaches in hip prostheses].',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Image registration of remotely sensed imagery is challenging, as complex deformations are common. Different deformations, such as affine and homogenous transformation, combined with multimodal data capturing can emerge in the data acquisition process. These effects, when combined, tend to compromise the performance of the currently available registration methods. A new image transform, known as geometric mean projection transform, is introduced in this work. As it is deformation invariant, it can be employed as a feature descriptor, whereby it analyzes the functions of all vertical and horizontal signals in local areas of the image. Moreover, an invariant feature correspondence method is proposed as a point matching algorithm, which incorporates new descriptor\\'s dissimilarity metric. Considering the image as a signal, the proposed approach utilizes a square Eigenvector correlation (SEC) based on the Eigenvector properties. In our experiments on standard test images sourced from \"Featurespace\" and \"IKONOS\" datasets, the proposed method achieved higher average accuracy relative to that obtained from other state of the art image registration techniques. The accuracy of the proposed method was assessed using six standard evaluation metrics. Furthermore, statistical analyses, including t-test and Friedman test, demonstrate that the method developed as a part of this study is superior to the existing methods.',\n",
              "  'authors': ['Kahaki, Seyed M M',\n",
              "   'Arshad, Haslina',\n",
              "   'Nordin, Md Jan',\n",
              "   'Ismail, Waidah'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Computer Graphics',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Satellite Imagery'],\n",
              "  'pmid': '30024921',\n",
              "  'title': 'Geometric feature descriptor and dissimilarity-based registration of remotely sensed imagery.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': \"To report and identify the predictors of early and long-term complications in a large series of extraperitoneal robot-assisted radical prostatectomy using the standardized Martin criteria.|A total of 1503 patients underwent extraperitoneal robot-assisted radical prostatectomy for the treatment of clinically localized prostate cancer from July 2003 to August 2010 at a tertiary referral center. The median follow-up was 28.9 months. Fisher's exact test was used to examine the association of multiple variables in a bivariate analysis with the incidence and types of complications. Independent predictors were also examined in a multivariate analysis using logistic regression models.|A total of 151 complications were recorded in 127 of 1503 patients (8.45%). Approximately one third (30.5%) were classified as major complications requiring intervention (Clavien grade III and IV). The most commonly encountered complications were lymphocele (1.46%), bladder neck contracture (1.33%), and anastomotic leak (1.20%). The operative time was a significant predictor of all complications and of major complications on multivariate analysis. Surgeon experience was also predictive of complications on multivariate analysis.|Extraperitoneal robot-assisted radical prostatectomy remains an underused alternative approach for the treatment of localized prostate cancer. Its safety profile is equivalent to that of other approaches in experienced hands.\",\n",
              "  'authors': ['Ghazi, Ahmed',\n",
              "   'Scosyrev, Emelian',\n",
              "   'Patel, Hitendra',\n",
              "   'Messing, Edward M',\n",
              "   'Joseph, Jean V'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Anastomotic Leak',\n",
              "   'Clinical Competence',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Logistic Models',\n",
              "   'Lymph Node Excision',\n",
              "   'Lymphocele',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Multivariate Analysis',\n",
              "   'Operative Time',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Robotics',\n",
              "   'Urinary Bladder Diseases'],\n",
              "  'pmid': '23374793',\n",
              "  'title': 'Complications associated with extraperitoneal robot-assisted radical prostatectomy using the standardized Martin classification.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1365-3040',\n",
              "  'IF_2013': '5.906',\n",
              "  'IF_2014': '6.96',\n",
              "  'IF_2015': '6.169',\n",
              "  'IF_2016': '6.173',\n",
              "  'IF_2017': '5.415',\n",
              "  'ISSN': '0140-7791',\n",
              "  'ISSN_online': '1365-3040',\n",
              "  'ISSN_print': '0140-7791',\n",
              "  'IsoAbbr': 'Plant Cell Environ.',\n",
              "  'JrId': 20442,\n",
              "  'MedAbbr': 'Plant Cell Environ',\n",
              "  'NLMid': '9309004',\n",
              "  'Titre': 'Plant Cell Environ.',\n",
              "  'abstract': 'The family of polyamine oxidases (PAO) in Arabidopsis (AtPAO1-5) mediates polyamine (PA) back-conversion, which reverses the PA biosynthetic pathway from spermine and its structural isomer thermospermine (tSpm) into spermidine and then putrescine. Here, we have studied the involvement of PA back-conversion in Arabidopsis salinity tolerance. AtPAO5 is the Arabidopsis PAO gene member most transcriptionally induced by salt stress. Two independent loss-of-function mutants (atpao5-2 and atpao5-3) were found to exhibit constitutively higher tSpm levels, with associated increased salt tolerance. Using global transcriptional and metabolomic analyses, the underlying mechanisms were studied. Stimulation of abscisic acid and jasmonate (JA) biosynthesis and accumulation of important compatible solutes, such as sugars, polyols and proline, as well as TCA cycle intermediates were observed in atpao5 mutants under salt stress. Expression analyses indicate that tSpm modulates the transcript levels of several target genes, including many involved in the biosynthesis and signalling of JA, some of which are already known to promote salinity tolerance. Transcriptional modulation by tSpm is isomer-dependent, thus demonstrating the specificity of this response. Overall, we conclude that tSpm triggers metabolic and transcriptional reprogramming that promotes salt stress tolerance in Arabidopsis.',\n",
              "  'authors': ['Zarza, Xavier',\n",
              "   'Atanasov, Kostadin E',\n",
              "   'Marco, Francisco',\n",
              "   'Arbona, Vicent',\n",
              "   'Carrasco, Pedro',\n",
              "   'Kopka, Joachim',\n",
              "   'Fotopoulos, Vasileios',\n",
              "   'Munnik, Teun',\n",
              "   'Gómez-Cadenas, Aurelio',\n",
              "   'Tiburcio, Antonio F',\n",
              "   'Alcázar, Rubén'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Plant, cell & environment',\n",
              "  'keywords': ['Abscisic Acid',\n",
              "   'Arabidopsis',\n",
              "   'Arabidopsis Proteins',\n",
              "   'Citric Acid Cycle',\n",
              "   'Cyclopentanes',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Hydrogen Peroxide',\n",
              "   'Ions',\n",
              "   'Loss of Function Mutation',\n",
              "   'Metabolome',\n",
              "   'Multigene Family',\n",
              "   'Oxidoreductases Acting on CH-NH2 Group Donors',\n",
              "   'Oxylipins',\n",
              "   'Phenotype',\n",
              "   'Principal Component Analysis',\n",
              "   'RNA, Messenger',\n",
              "   'Salt Tolerance',\n",
              "   'Sodium',\n",
              "   'Sodium Chloride',\n",
              "   'Spermine',\n",
              "   'Stress, Physiological',\n",
              "   'Transcription, Genetic',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '26791972',\n",
              "  'title': 'Polyamine oxidase 5 loss-of-function mutations in Arabidopsis thaliana trigger metabolic and transcriptional reprogramming and promote salt stress tolerance.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Scanning flow cytometry (SFCM) is characterized by the measurement of time-resolved pulses of fluorescence and scattering, enabling the high-throughput quantification of phytoplankton morphology and pigmentation. Quantifying variation at the single cell and colony level improves our ability to understand dynamics in natural communities. Automated high-frequency monitoring of these communities is presently limited by the absence of repeatable, rapid protocols to analyse SFCM datasets, where images of individual particles are not available. Here we demonstrate a repeatable, semi-automated method to (1) rapidly clean SFCM data from a phytoplankton community by removing signals that do not belong to live phytoplankton cells, (2) classify individual cells into trait clusters that correspond to functional groups, and (3) quantify the biovolumes of individual cells, the total biovolume of the whole community and the total biovolumes of the major functional groups. Our method involves the development of training datasets using lab cultures, the use of an unsupervised clustering algorithm to identify trait clusters, and machine learning tools (random forests) to (1) evaluate variable importance, (2) classify data points, and (3) estimate biovolumes of individual cells. We provide example datasets and R code for our analytical approach that can be adapted for analysis of datasets from other flow cytometers or scanning flow cytometers.',\n",
              "  'authors': ['Thomas, Mridul K',\n",
              "   'Fontana, Simone',\n",
              "   'Reyes, Marta',\n",
              "   'Pomati, Francesco'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cell Count',\n",
              "   'Cluster Analysis',\n",
              "   'Ecosystem',\n",
              "   'Flow Cytometry',\n",
              "   'Machine Learning',\n",
              "   'Phytoplankton'],\n",
              "  'pmid': '29746500',\n",
              "  'title': 'Quantifying cell densities and biovolumes of phytoplankton communities and functional groups using scanning flow cytometry, machine learning and unsupervised clustering.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Chronic low dose inorganic arsenic (iAs) exposure leads to changes in gene expression and epithelial-to-mesenchymal transformation. During this transformation, cells adopt a fibroblast-like phenotype accompanied by profound gene expression changes. While many mechanisms have been implicated in this transformation, studies that focus on the role of epigenetic alterations in this process are just emerging. DNA methylation controls gene expression in physiologic and pathologic states. Several studies show alterations in DNA methylation patterns in iAs-mediated pathogenesis, but these studies focused on single genes. We present a comprehensive genome-wide DNA methylation analysis using methyl-sequencing to measure changes between normal and iAs-transformed cells. Additionally, these differential methylation changes correlated positively with changes in gene expression and alternative splicing. Interestingly, most of these differentially methylated genes function in cell adhesion and communication pathways. To gain insight into how genomic DNA methylation patterns are regulated during iAs-mediated carcinogenesis, we show that iAs probably targets CTCF binding at the promoter of DNA methyltransferases, regulating their expression. These findings reveal how CTCF binding regulates DNA methyltransferase to reprogram the methylome in response to an environmental toxin.',\n",
              "  'authors': ['Rea, Matthew',\n",
              "   'Eckstein, Meredith',\n",
              "   'Eleazer, Rebekah',\n",
              "   'Smith, Caroline',\n",
              "   'Fondufe-Mittendorf, Yvonne N'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Alternative Splicing',\n",
              "   'Arsenic',\n",
              "   'CCCTC-Binding Factor',\n",
              "   'Cell Transformation, Neoplastic',\n",
              "   'Cellular Reprogramming',\n",
              "   'Chromosomes, Human',\n",
              "   'CpG Islands',\n",
              "   'DNA (Cytosine-5-)-Methyltransferases',\n",
              "   'DNA Methylation',\n",
              "   'Epithelial-Mesenchymal Transition',\n",
              "   'Gene Ontology',\n",
              "   'Genome, Human',\n",
              "   'Humans',\n",
              "   'Promoter Regions, Genetic',\n",
              "   'Protein Binding',\n",
              "   'Reproducibility of Results',\n",
              "   'Transcription, Genetic'],\n",
              "  'pmid': '28150704',\n",
              "  'title': 'Genome-wide DNA methylation reprogramming in response to inorganic arsenic links inhibition of CTCF binding, DNMT expression and cellular transformation.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Chronic Fatigue Syndrome (CFS), also known as myalgic encephalomyelitis, is a complex multifactorial disease that is characterized by the persistent presence of fatigue and other particular symptoms for a minimum of 6 months. Symptoms fail to dissipate after sufficient rest and have major effects on the daily functioning of CFS sufferers. CFS is a multi-system disease with a heterogeneous patient population showing a wide variety of functional disabilities and its biological basis remains poorly understood. Stable alterations in gene function in the immune system have been reported in several studies of CFS. Epigenetic modifications have been implicated in long-term effects on gene function, however, to our knowledge, genome-wide epigenetic modifications associated with CFS have not been explored. We examined the DNA methylome in peripheral blood mononuclear cells isolated from CFS patients and healthy controls using the Illumina HumanMethylation450 BeadChip array, controlling for invariant probes and probes overlapping polymorphic sequences. Gene ontology (GO) and network analysis of differentially methylated genes was performed to determine potential biological pathways showing changes in DNA methylation in CFS. We found an increased abundance of differentially methylated genes related to the immune response, cellular metabolism, and kinase activity. Genes associated with immune cell regulation, the largest coordinated enrichment of differentially methylated pathways, showed hypomethylation within promoters and other gene regulatory elements in CFS. These data are consistent with evidence of multisystem dysregulation in CFS and implicate the involvement of DNA modifications in CFS pathology. ',\n",
              "  'authors': ['de Vega, Wilfred C', 'Vernon, Suzanne D', 'McGowan, Patrick O'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'CpG Islands',\n",
              "   'DNA Methylation',\n",
              "   'Fatigue Syndrome, Chronic',\n",
              "   'Female',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Male'],\n",
              "  'pmid': '25111603',\n",
              "  'title': 'DNA methylation modifications associated with chronic fatigue syndrome.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RT'],\n",
              "  'ESSN': '1550-9397',\n",
              "  'IF_2013': '2.826',\n",
              "  'IF_2014': '3.053',\n",
              "  'IF_2015': '2.71',\n",
              "  'IF_2016': '3.429',\n",
              "  'IF_2017': '3.396',\n",
              "  'ISSN': '1550-9389',\n",
              "  'ISSN_online': '1550-9397',\n",
              "  'ISSN_print': '1550-9389',\n",
              "  'IsoAbbr': 'J Clin Sleep Med',\n",
              "  'JrId': 32554,\n",
              "  'MedAbbr': 'J Clin Sleep Med',\n",
              "  'NLMid': '101231977',\n",
              "  'Titre': 'J Clin Sleep Med',\n",
              "  'abstract': 'Portable devices for home sleep apnea testing are often limited by their inability to discriminate sleep/wake status, possibly resulting in underestimations. Tracheal sound (TS), which can be visualized as a spectrogram, carries information about apnea/hypopnea and sleep/wake status. We hypothesized that image analysis of all-night TS recordings by a deep neural network (DNN) would be capable of detecting breathing events and classifying sleep/wake status. The aim of this study is to develop a DNN-based system for sleep apnea testing and validate it using a large sampling of polysomnography (PSG) data.|PSG examinations for the evaluation of sleep-disordered breathing (SDB) were performed for 1,852 patients: 1,548 PSG records were used to develop the system, and the remaining 304 records were used for validation. TS spectrogram images were obtained every 60 seconds and labeled with the PSG scoring results (breathing event and sleep/wake status), then introduced to DNN learning. Two different DNNs were trained for breathing status and sleep/wake status, respectively.|A DNN with convolutional layers showed the best performance for discriminating breathing status. The same DNN structure was trained for sleep/wake discrimination. In the validation study, the DNN analysis was capable of discriminating the sleep/wake status with reasonable accuracy. The diagnostic sensitivity, specificity, and area under the receiver operating characteristic curves for diagnosis of SDB with apnea-hypopnea index of > 5, 15, and 30 were 0.98, 0.76, and 0.99; 0.97, 0.90, and 0.99; and 0.92, 0.94, and 0.98, respectively.|The developed system using the TS DNN analysis has a good performance for SDB testing.|Nakano H, Furukawa T, Tanigawa T. Tracheal sound analysis using a deep neural network to detect sleep apnea. J Clin Sleep Med. 2019;15(8): 1125-1133.',\n",
              "  'authors': ['Nakano, Hiroshi', 'Furukawa, Tomokazu', 'Tanigawa, Takeshi'],\n",
              "  'categories': ['CLINICAL NEUROLOGY'],\n",
              "  'journal': 'Journal of clinical sleep medicine : JCSM : official publication of the American Academy of Sleep Medicine',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31482834',\n",
              "  'title': 'Tracheal Sound Analysis Using a Deep Neural Network to Detect Sleep Apnea.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '2040-2309',\n",
              "  'IF_2013': '0.468',\n",
              "  'IF_2014': '0.754',\n",
              "  'IF_2015': '0.925',\n",
              "  'IF_2016': '0.965',\n",
              "  'IF_2017': '1.261',\n",
              "  'ISSN': '2040-2295',\n",
              "  'ISSN_online': '2040-2309',\n",
              "  'ISSN_print': '2040-2295',\n",
              "  'IsoAbbr': 'J Healthc Eng',\n",
              "  'JrId': 39708,\n",
              "  'MedAbbr': 'J Healthc Eng',\n",
              "  'NLMid': '101528166',\n",
              "  'Titre': 'J Healthc Eng',\n",
              "  'abstract': 'It is important to identify and prevent disease risk as early as possible through regular physical examinations. We formulate the disease risk prediction into a multilabel classification problem. A novel Ensemble Label Power-set Pruned datasets Joint Decomposition (ELPPJD) method is proposed in this work. First, we transform the multilabel classification into a multiclass classification. Then, we propose the pruned datasets and joint decomposition methods to deal with the imbalance learning problem. Two strategies size balanced (SB) and label similarity (LS) are designed to decompose the training dataset. In the experiments, the dataset is from the real physical examination records. We contrast the performance of the ELPPJD method with two different decomposition strategies. Moreover, the comparison between ELPPJD and the classic multilabel classification methods RAkEL and HOMER is carried out. The experimental results show that the ELPPJD method with label similarity strategy has outstanding performance.',\n",
              "  'authors': ['Li, Runzhi',\n",
              "   'Liu, Wei',\n",
              "   'Lin, Yusong',\n",
              "   'Zhao, Hongling',\n",
              "   'Zhang, Chaoyang'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Journal of healthcare engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Cholecystitis',\n",
              "   'Diabetes Mellitus',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Fatty Liver',\n",
              "   'Heart Diseases',\n",
              "   'Humans',\n",
              "   'Hypertension',\n",
              "   'Machine Learning',\n",
              "   'Medical Informatics',\n",
              "   'Obesity',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Risk Assessment',\n",
              "   'Software'],\n",
              "  'pmid': '29065647',\n",
              "  'title': 'An Ensemble Multilabel Classification for Disease Risk Prediction.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1873-3492',\n",
              "  'IF_2013': '2.764',\n",
              "  'IF_2014': '2.824',\n",
              "  'IF_2015': '2.799',\n",
              "  'IF_2016': '2.873',\n",
              "  'IF_2017': '2.926',\n",
              "  'ISSN': '0009-8981',\n",
              "  'ISSN_online': '1873-3492',\n",
              "  'ISSN_print': '0009-8981',\n",
              "  'IsoAbbr': 'Clin. Chim. Acta',\n",
              "  'JrId': 2993,\n",
              "  'MedAbbr': 'Clin Chim Acta',\n",
              "  'NLMid': '1302422',\n",
              "  'Titre': 'Clin. Chim. Acta',\n",
              "  'abstract': 'In this study, one hundred serum samples from healthy people and patients with rheumatoid arthritis (RA) were analyzed. Standard immunoassays for detection of 10 different RA markers and analysis of glycan markers on antibodies in 10 different assay formats with several lectins were applied for each serum sample. A dataset containing 2000 data points was data mined using artificial neural networks (ANN). We identified key RA markers, which can discriminate between healthy people and seropositive RA patients (serum containing autoantibodies) with accuracy of 83.3%. Combination of RA markers with glycan analysis provided much better discrimination accuracy of 92.5%. Immunoassays completely failed to identify seronegative RA patients (serum not containing autoantibodies), while glycan analysis correctly identified 43.8% of these patients. Further, we revealed other critical parameters for successful glycan analysis such as type of a sample, format of analysis and orientation of captured antibodies for glycan analysis.',\n",
              "  'authors': ['Chocholova, Erika',\n",
              "   'Bertok, Tomas',\n",
              "   'Jane, Eduard',\n",
              "   'Lorencova, Lenka',\n",
              "   'Holazova, Alena',\n",
              "   'Belicka, Ludmila',\n",
              "   'Belicky, Stefan',\n",
              "   'Mislovicova, Danica',\n",
              "   'Vikartovska, Alica',\n",
              "   'Imrich, Richard',\n",
              "   'Kasak, Peter',\n",
              "   'Tkac, Jan'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinica chimica acta; international journal of clinical chemistry',\n",
              "  'keywords': ['Arthritis, Rheumatoid',\n",
              "   'Artificial Intelligence',\n",
              "   'Autoantibodies',\n",
              "   'Biomarkers',\n",
              "   'Female',\n",
              "   'Glycomics',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Polysaccharides',\n",
              "   'Rheumatoid Factor'],\n",
              "  'pmid': '29486148',\n",
              "  'title': 'Glycomics meets artificial intelligence - Potential of glycan analysis for identification of seropositive and seronegative rheumatoid arthritis patients revealed.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1097-0142',\n",
              "  'IF_2013': '4.901',\n",
              "  'IF_2014': '4.889',\n",
              "  'IF_2015': '5.649',\n",
              "  'IF_2016': '6.072',\n",
              "  'IF_2017': '6.537',\n",
              "  'ISSN': '0008-543X',\n",
              "  'ISSN_online': '1097-0142',\n",
              "  'ISSN_print': '0008-543X',\n",
              "  'IsoAbbr': 'Cancer',\n",
              "  'JrId': 2771,\n",
              "  'MedAbbr': 'Cancer',\n",
              "  'NLMid': '0374236',\n",
              "  'Titre': 'Cancer',\n",
              "  'abstract': 'The current study assesses artificial intelligence methods to identify prostate carcinoma patients at low risk for lymph node spread. If patients can be assigned accurately to a low risk group, unnecessary lymph node dissections can be avoided, thereby reducing morbidity and costs.|A rule-derivation technology for simple decision-tree analysis was trained and validated using patient data from a large database (4,133 patients) to derive low risk cutoff values for Gleason sum and prostate specific antigen (PSA) level. An empiric analysis was used to derive a low risk cutoff value for clinical TNM stage. These cutoff values then were applied to 2 additional, smaller databases (227 and 330 patients, respectively) from separate institutions.|The decision-tree protocol derived cutoff values of < or = 6 for Gleason sum and < or = 10.6 ng/mL for PSA. The empiric analysis yielded a clinical TNM stage low risk cutoff value of < or = T2a. When these cutoff values were applied to the larger database, 44% of patients were classified as being at low risk for lymph node metastases (0.8% false-negative rate). When the same cutoff values were applied to the smaller databases, between 11 and 43% of patients were classified as low risk with a false-negative rate of between 0.0 and 0.7%.|The results of the current study indicate that a population of prostate carcinoma patients at low risk for lymph node metastases can be identified accurately using a simple decision algorithm that considers preoperative PSA, Gleason sum, and clinical TNM stage. The risk of lymph node metastases in these patients is < or = 1%; therefore, pelvic lymph node dissection may be avoided safely. The implications of these findings in surgical and nonsurgical treatment are significant.',\n",
              "  'authors': ['Crawford, E D',\n",
              "   'Batuello, J T',\n",
              "   'Snow, P',\n",
              "   'Gamito, E J',\n",
              "   'McLeod, D G',\n",
              "   'Partin, A W',\n",
              "   'Stone, N',\n",
              "   'Montie, J',\n",
              "   'Stock, R',\n",
              "   'Lynch, J',\n",
              "   'Brandt, J'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Cancer',\n",
              "  'keywords': ['Acid Phosphatase',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biopsy, Needle',\n",
              "   'Carcinoma',\n",
              "   'Cost Control',\n",
              "   'Databases as Topic',\n",
              "   'Decision Support Techniques',\n",
              "   'Decision Trees',\n",
              "   'False Negative Reactions',\n",
              "   'Humans',\n",
              "   'Lymph Node Excision',\n",
              "   'Lymphatic Metastasis',\n",
              "   'Male',\n",
              "   'Neoplasm Staging',\n",
              "   'Prostate-Specific Antigen',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Reproducibility of Results',\n",
              "   'Risk Factors'],\n",
              "  'pmid': '10813722',\n",
              "  'title': 'The use of artificial intelligence technology to predict lymph node spread in men with clinically localized prostate carcinoma.',\n",
              "  'year': '2000'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1558-3171',\n",
              "  'IF_2013': '1.932',\n",
              "  'IF_2014': '1.879',\n",
              "  'IF_2015': '2.068',\n",
              "  'IF_2016': '2.206',\n",
              "  'IF_2017': '1.947',\n",
              "  'ISSN': '0039-6109',\n",
              "  'ISSN_online': '1558-3171',\n",
              "  'ISSN_print': '0039-6109',\n",
              "  'IsoAbbr': 'Surg. Clin. North Am.',\n",
              "  'JrId': 7645,\n",
              "  'MedAbbr': 'Surg Clin North Am',\n",
              "  'NLMid': '0074243',\n",
              "  'Titre': 'Surg. Clin. North Am.',\n",
              "  'abstract': \"Ultimately, neurosurgery performed via a robotic interface will serve to improve the standard of a neurosurgeon's skills, thus making a good surgeon a better surgeon. In fact, computer and robotic instrumentation will become allies to the neurosurgeon through the use of these technologies in training, diagnostic, and surgical events. Nonetheless, these technologies are still in an early stage of development, and each device developed will entail its own set of challenges and limitations for use in clinical settings. The future operating room should be regarded as an integrated information system incorporating robotic surgical navigators and telecontrolled micromanipulators, with the capabilities of all principal neurosurgical concepts, sharing information, and under the control of a single person, the neurosurgeon. The eventual integration of robotic technology into mainstream clinical neurosurgery offers the promise of a future of safer, more accurate, and less invasive surgery that will result in improved patient outcome.\",\n",
              "  'authors': ['Nathoo, Narendra', 'Pesek, Todd', 'Barnett, Gene H'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The Surgical clinics of North America',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Humans',\n",
              "   'Nerve Net',\n",
              "   'Neurosurgical Procedures',\n",
              "   'Robotics',\n",
              "   'Telemedicine'],\n",
              "  'pmid': '14712870',\n",
              "  'title': 'Robotics and neurosurgery.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'Preprocess method of multiplicative scatter correction (MSC) was used to reject noises in the original spectra produced by the environmental physical factor effectively, then the principal components of near-infrared spectroscopy were calculated by nonlinear iterative partial least squares (NIPALS) before building the back propagation artificial neural networks method (BP-ANN), and the numbers of principal components were calculated by the method of cross validation. The calculated principal components were used as the inputs of the artificial neural networks model, and the artificial neural networks model was used to find the relation between chlorophyll in winter wheat and reflective spectrum, which can predict the content of chlorophyll in winter wheat. The correlation coefficient (r) of calibration set was 0.9604, while the standard deviation (SD) and relative standard deviation (RSD) was 0.187 and 5.18% respectively. The correlation coefficient (r) of predicted set was 0.9600, and the standard deviation (SD) and relative standard deviation (RSD) was 0.145 and 4.21% respectively. It means that the MSC-ANN algorithm can reject noises in the original spectra produced by the environmental physical factor effectively and set up an exact model to predict the contents of chlorophyll in living leaves veraciously to replace the classical method and meet the needs of fast analysis of agricultural products.',\n",
              "  'authors': ['Liang, Xue',\n",
              "   'Ji, Hai-yan',\n",
              "   'Wang, Peng-xin',\n",
              "   'Rao, Zhen-hong',\n",
              "   'Shen, Bing-hui'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Chlorophyll',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Triticum'],\n",
              "  'pmid': '20302111',\n",
              "  'title': '[Study of building quantitative analysis model for chlorophyll in winter wheat with reflective spectrum using MSC-ANN algorithm].',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"In this paper, we evaluate the control performance of SSVEP (steady-state visual evoked potential)- and P300-based models using Cerebot-a mind-controlled humanoid robot platform. Seven subjects with diverse experience participated in experiments concerning the open-loop and closed-loop control of a humanoid robot via brain signals. The visual stimuli of both the SSVEP- and P300- based models were implemented on a LCD computer monitor with a refresh frequency of 60 Hz. Considering the operation safety, we set the classification accuracy of a model over 90.0% as the most important mandatory for the telepresence control of the humanoid robot. The open-loop experiments demonstrated that the SSVEP model with at most four stimulus targets achieved the average accurate rate about 90%, whereas the P300 model with the six or more stimulus targets under five repetitions per trial was able to achieve the accurate rates over 90.0%. Therefore, the four SSVEP stimuli were used to control four types of robot behavior; while the six P300 stimuli were chosen to control six types of robot behavior. Both of the 4-class SSVEP and 6-class P300 models achieved the average success rates of 90.3% and 91.3%, the average response times of 3.65 s and 6.6 s, and the average information transfer rates (ITR) of 24.7 bits/min 18.8 bits/min, respectively. The closed-loop experiments addressed the telepresence control of the robot; the objective was to cause the robot to walk along a white lane marked in an office environment using live video feedback. Comparative studies reveal that the SSVEP model yielded faster response to the subject's mental activity with less reliance on channel selection, whereas the P300 model was found to be suitable for more classifiable targets and required less training. To conclude, we discuss the existing SSVEP and P300 models for the control of humanoid robots, including the models proposed in this paper. \",\n",
              "  'authors': ['Zhao, Jing', 'Li, Wei', 'Li, Mengfan'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Brain',\n",
              "   'Brain-Computer Interfaces',\n",
              "   'Electroencephalography',\n",
              "   'Event-Related Potentials, P300',\n",
              "   'Evoked Potentials, Visual',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Young Adult'],\n",
              "  'pmid': '26562524',\n",
              "  'title': 'Comparative Study of SSVEP- and P300-Based Models for the Telepresence Control of Humanoid Robots.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': \"An algorithm was developed to estimate noncircular orbit (NCO) single-photon emission computed tomography (SPECT) detector radius on a SPECT/CT imaging system using the CT images, for incorporation into collimator resolution modeling for iterative SPECT reconstruction. Simulated male abdominal (arms up), male head and neck (arms down) and female chest (arms down) anthropomorphic phantom, and ten patient, medium-energy SPECT/CT scans were acquired on a hybrid imaging system. The algorithm simulated inward SPECT detector radial motion and object contour detection at each projection angle, employing the calculated average CT image and a fixed Hounsfield unit (HU) threshold. Calculated radii were compared to the observed true radii, and optimal CT threshold values, corresponding to patient bed and clothing surfaces, were found to be between -970 and -950 HU. The algorithm was constrained by the 45 cm CT field-of-view (FOV), which limited the detected radii to < or = 22.5 cm and led to occasional radius underestimation in the case of object truncation by CT. Two methods incorporating the algorithm were implemented: physical model (PM) and best fit (BF). The PM method computed an offset that produced maximum overlap of calculated and true radii for the phantom scans, and applied that offset as a calculated-to-true radius transformation. For the BF method, the calculated-to-true radius transformation was based upon a linear regression between calculated and true radii. For the PM method, a fixed offset of +2.75 cm provided maximum calculated-to-true radius overlap for the phantom study, which accounted for the camera system's object contour detect sensor surface-to-detector face distance. For the BF method, a linear regression of true versus calculated radius from a reference patient scan was used as a calculated-to-true radius transform. Both methods were applied to ten patient scans. For -970 and -950 HU thresholds, the combined overall average root-mean-square (rms) error in radial position for eight patient scans without truncation were 3.37 cm (12.9%) for PM and 1.99 cm (8.6%) for BF, indicating BF is superior to PM in the absence of truncation. For two patient scans with truncation, the rms error was 3.24 cm (12.2%) for PM and 4.10 cm (18.2%) for BF. The slightly better performance of PM in the case of truncation is anomalous, due to FOV edge truncation artifacts in the CT reconstruction, and thus is suspect. The calculated NCO contour for a patient SPECT/CT scan was used with an iterative reconstruction algorithm that incorporated compensation for system resolution. The resulting image was qualitatively superior to the image obtained by reconstructing the data using the fixed radius stored by the scanner. The result was also superior to the image reconstructed using the iterative algorithm provided with the system, which does not incorporate resolution modeling. These results suggest that, under conditions of no or only mild lateral truncation of the CT scan, the algorithm is capable of providing radius estimates suitable for iterative SPECT reconstruction collimator geometric resolution modeling.\",\n",
              "  'authors': ['Liu, Ruijie Rachel', 'Erwin, William D'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Subtraction Technique',\n",
              "   'Tomography, Emission-Computed, Single-Photon',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '16964856',\n",
              "  'title': 'Automatic estimation of detector radial position for contoured SPECT acquisition using CT images on a SPECT/CT system.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'We develop a knowledge-based approach (called PROSP) for protein secondary structure prediction. The knowledge base contains small peptide fragments together with their secondary structural information. A quantitative measure M, called match rate, is defined to measure the amount of structural information that a target protein can extract from the knowledge base. Our experimental results show that proteins with a higher match rate will likely be predicted more accurately based on PROSP. That is, there is roughly a monotone correlation between the prediction accuracy and the amount of structure matching with the knowledge base. To fully utilize the strength of our knowledge base, a hybrid prediction method is proposed as follows: if the match rate of a target protein is at least 80%, we use the extracted information to make the prediction; otherwise, we adopt a popular machine-learning approach. This comprises our hybrid protein structure prediction (HYPROSP) approach. We use the DSSP and EVA data as our datasets and PSIPRED as our underlying machine-learning algorithm. For target proteins with match rate at least 80%, the average Q3 of PROSP is 3.96 and 7.2 better than that of PSIPRED on DSSP and EVA data, respectively.',\n",
              "  'authors': ['Wu, Kuen-Pin',\n",
              "   'Lin, Hsin-Nan',\n",
              "   'Chang, Jia-Ming',\n",
              "   'Sung, Ting-Yi',\n",
              "   'Hsu, Wen-Lian'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Internet',\n",
              "   'Peptide Fragments',\n",
              "   'Protein Structure, Secondary',\n",
              "   'Reproducibility of Results'],\n",
              "  'pmid': '15448186',\n",
              "  'title': 'HYPROSP: a hybrid protein secondary structure prediction algorithm--a knowledge-based approach.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': \"Previous transcriptome studies of the human endometrium have revealed hundreds of simultaneously up- and down-regulated genes that are involved in endometrial receptivity. However, the overlap between the studies is relatively small, and we are still searching for potential diagnostic biomarkers. Here we perform a meta-analysis of endometrial-receptivity associated genes on 164 endometrial samples (76 from 'pre-receptive' and 88 from mid-secretory, 'receptive' phase endometria) using a robust rank aggregation (RRA) method, followed by enrichment analysis, and regulatory microRNA prediction. We identify a meta-signature of endometrial receptivity involving 57 mRNA genes as putative receptivity markers, where 39 of these we confirm experimentally using RNA-sequencing method in two separate datasets. The meta-signature genes highlight the importance of immune responses, the complement cascade pathway and the involvement of exosomes in mid-secretory endometrial functions. Bioinformatic prediction identifies 348 microRNAs that could regulate 30 endometrial-receptivity associated genes, and we confirm experimentally the decreased expression of 19 microRNAs with 11 corresponding up-regulated meta-signature genes in our validation experiments. The 57 identified meta-signature genes and involved pathways, together with their regulatory microRNAs could serve as promising and sought-after biomarkers of endometrial receptivity, fertility and infertility.\",\n",
              "  'authors': ['Altmäe, Signe',\n",
              "   'Koel, Mariann',\n",
              "   'Võsa, Urmo',\n",
              "   'Adler, Priit',\n",
              "   'Suhorutšenko, Marina',\n",
              "   'Laisk-Podar, Triin',\n",
              "   'Kukushkina, Viktorija',\n",
              "   'Saare, Merli',\n",
              "   'Velthut-Meikas, Agne',\n",
              "   'Krjutškov, Kaarel',\n",
              "   'Aghajanova, Lusine',\n",
              "   'Lalitkumar, Parameswaran G',\n",
              "   'Gemzell-Danielsson, Kristina',\n",
              "   'Giudice, Linda',\n",
              "   'Simón, Carlos',\n",
              "   'Salumets, Andres'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Adult',\n",
              "   'Biomarkers',\n",
              "   'Computational Biology',\n",
              "   'Embryo Implantation',\n",
              "   'Endometrium',\n",
              "   'Exosomes',\n",
              "   'Female',\n",
              "   'Fertility',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Gene Regulatory Networks',\n",
              "   'Humans',\n",
              "   'Immunity, Innate',\n",
              "   'Menstrual Cycle',\n",
              "   'MicroRNAs',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'RNA, Messenger',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '28855728',\n",
              "  'title': 'Meta-signature of human endometrial receptivity: a meta-analysis and validation study of transcriptomic biomarkers.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'The displacement of the hyoid bone is one of the key components evaluated in the swallow study, as its motion during swallowing is related to overall swallowing integrity. In daily research settings, experts visually detect the hyoid bone in the video frames and manually plot hyoid bone position frame by frame. This study aims to develop an automatic method to localize the location of the hyoid bone in the video sequence. To automatically detect the location of the hyoid bone in a frame, we proposed a single shot multibox detector, a deep convolutional neural network, which is employed to detect and classify the location of the hyoid bone. We also evaluated the performance of two other state-of-art detection methods for comparison. The experimental results clearly showed that the single shot multibox detector can detect the hyoid bone with an average precision of 89.14% and outperform other auto-detection algorithms. We conclude that this automatic hyoid bone tracking system is accurate enough to be widely applied as a pre-processing step for image processing in dysphagia research, as well as a promising development that may be useful in the diagnosis of dysphagia.',\n",
              "  'authors': ['Zhang, Zhenwei', 'Coyle, James L', 'Sejdić, Ervin'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30120314',\n",
              "  'title': 'Automatic hyoid bone detection in fluoroscopic images using deep learning.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['CU'],\n",
              "  'ESSN': '1573-0875',\n",
              "  'IF_2013': '1.765',\n",
              "  'IF_2014': '1.11',\n",
              "  'IF_2015': '1.141',\n",
              "  'IF_2016': '1',\n",
              "  'IF_2017': '1.464',\n",
              "  'ISSN': '0169-6149',\n",
              "  'ISSN_online': '1573-0875',\n",
              "  'ISSN_print': '0169-6149',\n",
              "  'IsoAbbr': 'Orig Life Evol Biosph',\n",
              "  'JrId': 6291,\n",
              "  'MedAbbr': 'Orig Life Evol Biosph',\n",
              "  'NLMid': '8610391',\n",
              "  'Titre': 'Orig Life Evol Biosph',\n",
              "  'abstract': 'Many scientists, including one of the authors of the present paper, have devoted time to try to find a definition for life (Bersini and Reisse 2007). It is clear that a consensus will never be reached but, more importantly, it seems that the issue itself could be without major interest. It is indeed impossible to define a \"natural\" frontier between non-living and living systems and therefore also impossible to define dichotomic criteria which could be used in order to classify systems in one of these two classes (living or non-living). Fuzzy logic provides a natural way to deal with problems where class membership lacks sharply defined criteria. It also offers the possibility to avoid losing time with unnecessary controversies such as deciding whether a virus is, or is not, a living system.',\n",
              "  'authors': ['Bruylants, Gilles', 'Bartik, Kristin', 'Reisse, Jacques'],\n",
              "  'categories': ['BIOLOGY'],\n",
              "  'journal': 'Origins of life and evolution of the biosphere : the journal of the International Society for the Study of the Origin of Life',\n",
              "  'keywords': ['Biological Evolution',\n",
              "   'Fuzzy Logic',\n",
              "   'Life',\n",
              "   'Origin of Life'],\n",
              "  'pmid': '20204517',\n",
              "  'title': 'Is it useful to have a clear-cut definition of life? On the use of fuzzy logic in prebiotic chemistry.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['GA'],\n",
              "  'ESSN': '1097-6787',\n",
              "  'IF_2013': '5.004',\n",
              "  'IF_2014': '4.449',\n",
              "  'IF_2015': '5.621',\n",
              "  'IF_2016': '7.002',\n",
              "  'IF_2017': '6.898',\n",
              "  'ISSN': '0190-9622',\n",
              "  'ISSN_online': '1097-6787',\n",
              "  'ISSN_print': '0190-9622',\n",
              "  'IsoAbbr': 'J. Am. Acad. Dermatol.',\n",
              "  'JrId': 4671,\n",
              "  'MedAbbr': 'J Am Acad Dermatol',\n",
              "  'NLMid': '7907132',\n",
              "  'Titre': 'J. Am. Acad. Dermatol.',\n",
              "  'abstract': 'Despite good results in the treatment of most port-wine stains (PWS) with continuous-wave visible-light lasers, light PWS and those in certain locations respond less favorably and have a higher risk of scarring. Robotized scanning devices such as the Hexascan device have been developed for continuous-wave laser sources to produce greater target specificity, to increase reproducibility of results, and to decrease the incidence of adverse effects.|The purpose of this study was to compare the effects of the same wavelength of light (585 nm) on test sites within PWS with the flashlamp-pumped pulsed dye laser and a continuous-wave tunable dye laser scanned through a Hexascan robotized scanning device.|Two adjacent, noncontiguous sites within PWS were treated in 29 patients, one site with the flashlamp-pumped pulsed dye laser and the other with an argon-pumped continuous-wave tunable dye laser affixed to a Hexascan device.|Twenty-eight patients completed the study. The pulsed dye laser was found to be superior to the continuous-wave dye laser with the Hexascan device in 45% of patients, whereas the continuous-wave tunable dye laser with the Hexascan device was considered superior in 15%. There was no difference in the remaining 40%. Undesirable side effects were infrequent with both treatments. There was no significant difference in hypopigmentation or atrophic and hypertrophic scarring, but hyperpigmentation was more frequent with the continuous-wave dye laser with the Hexascan device.|Both the pulsed dye laser and continuous tunable-wave dye laser with the Hexascan device produce slight lightening after one treatment. The pulsed dye laser produces slightly greater lightening than the continuous-wave tunable dye laser with the Hexascan device 6 weeks after treatment of test areas within PWS in 40% of those treated. It also produced slightly less hypopigmentation and hyperpigmentation.',\n",
              "  'authors': ['Dover, J S',\n",
              "   'Geronemus, R',\n",
              "   'Stern, R S',\n",
              "   \"O'Hare, D\",\n",
              "   'Arndt, K A'],\n",
              "  'categories': ['DERMATOLOGY'],\n",
              "  'journal': 'Journal of the American Academy of Dermatology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Argon',\n",
              "   'Child',\n",
              "   'Cicatrix',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Hamartoma',\n",
              "   'Humans',\n",
              "   'Laser Coagulation',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pigmentation Disorders',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Skin',\n",
              "   'Skin Diseases',\n",
              "   'Skin Pigmentation',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '7829709',\n",
              "  'title': 'Dye laser treatment of port-wine stains: comparison of the continuous-wave dye laser with a robotized scanning device and the pulsed dye laser.',\n",
              "  'year': '1995'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TC'],\n",
              "  'ESSN': '1554-527X',\n",
              "  'IF_2013': '2.972',\n",
              "  'IF_2014': '2.986',\n",
              "  'IF_2015': '2.807',\n",
              "  'IF_2016': '2.692',\n",
              "  'IF_2017': '3.414',\n",
              "  'ISSN': '0736-0266',\n",
              "  'ISSN_online': '1554-527X',\n",
              "  'ISSN_print': '0736-0266',\n",
              "  'IsoAbbr': 'J. Orthop. Res.',\n",
              "  'JrId': 5097,\n",
              "  'MedAbbr': 'J Orthop Res',\n",
              "  'NLMid': '8404726',\n",
              "  'Titre': 'J. Orthop. Res.',\n",
              "  'abstract': 'We identified and compared the impingent-free range of motion (ROM) and subluxation potential for native hip, femoral head resurfacing (FHR), and total hip arthroplasty (THA). These constructs were also compared both with and without soft tissue to elucidate the role of the soft tissue. Five fresh-frozen bilateral hip specimens were mounted to a six-degree of freedom robotic manipulator. Under load-control parameters, in vivo mechanics were recreated to evaluate impingement free ROM, and the subluxation potential in two \"at risk\" positions for native hip, FHR, and THA. Impingement-free ROM of the skeletonized THA was greater than FHR for the anterior subluxation position. For skeletonized posterior subluxations, stability for THA and FHR constructs were similar, while a different pattern was observed for specimens with soft tissues intact. FHR constructs were more stable than THA constructs for both anterior and posterior subluxations. When the femoral neck is intact the joint has an earlier impingement profile placing the hip at risk for subluxation. However, FHR design was shown to be more stable than THA only when soft tissues were intact.',\n",
              "  'authors': ['Colbrunn, R W',\n",
              "   'Bottros, J J',\n",
              "   'Butler, R S',\n",
              "   'Klika, A K',\n",
              "   'Bonner, T F',\n",
              "   'Greeson, C',\n",
              "   'van den Bogert, A J',\n",
              "   'Barsoum, W K'],\n",
              "  'categories': ['ORTHOPEDICS'],\n",
              "  'journal': 'Journal of orthopaedic research : official publication of the Orthopaedic Research Society',\n",
              "  'keywords': ['Aged',\n",
              "   'Arthroplasty, Replacement, Hip',\n",
              "   'Cadaver',\n",
              "   'Femur Head',\n",
              "   'Hip Joint',\n",
              "   'Hip Prosthesis',\n",
              "   'Humans',\n",
              "   'Incidence',\n",
              "   'Joint Dislocations',\n",
              "   'Joint Instability',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Prosthesis Design',\n",
              "   'Range of Motion, Articular',\n",
              "   'Robotics'],\n",
              "  'pmid': '23494830',\n",
              "  'title': 'Impingement and stability of total hip arthroplasty versus femoral head resurfacing using a cadaveric robotics model.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '2473-4209',\n",
              "  'IF_2013': '3.012',\n",
              "  'IF_2014': '2.635',\n",
              "  'IF_2015': '2.496',\n",
              "  'IF_2016': '2.617',\n",
              "  'IF_2017': '2.884',\n",
              "  'ISSN': '0094-2405',\n",
              "  'ISSN_online': '2473-4209',\n",
              "  'ISSN_print': '0094-2405',\n",
              "  'IsoAbbr': 'Med Phys',\n",
              "  'JrId': 5648,\n",
              "  'MedAbbr': 'Med Phys',\n",
              "  'NLMid': '0425746',\n",
              "  'Titre': 'Med Phys',\n",
              "  'abstract': 'To automatically and efficiently segment the lesion area of the colonoscopy polyp image, a polyp segmentation method has been presented.|An ensemble model of pre-trained convolutional neural networks was proposed, using Unet-VGG, SegNet-VGG and PSPNet. Firstly, the Unet-VGG is obtained by using the first 10 layers of VGG16 as the contraction path of the left half of the Unet. Then, the SegNet-VGG is acquired by fine-tuned transfer learning VGG16, using the first 13 layers of VGG16 as the encoder of the SegNet and combined the original decoder of the SegNet. By adjusting the input size of the Unet-VGG, SegNet-VGG, and PSPNet, the pre-processed data can be correctly fed to the three network models. The three models are used as the basic trainer to train and segment the datasets. Based on the ensemble learning algorithm, the weight voting method is used to ensemble the segmentation results corresponding to single basic trainer.|Both IoU and DICE similarity score were used to evaluate the segmentation quality for cvc300 with 300 images, CVC-ClinicDB with 612 images and ETIS-LaribPolypDB with 196 images. From the experimental results, the IoU and DICE obtained by the proposed method for the cvc300 datasets can reach up to 96.16% and 98.04% respectively, the IoU and DICE for the CVC-ClinicDB datasets can reach up to 96.66% and 98.30% respectively, while the IoU and DICE for the ETIS-LaribPolypDB datasets can reach up to 96.95% and 98.45% respectively. Evaluation of the IoU and DICE in our methods shows higher accuracy than previous methods.|The experimental results show that the proposed method improved correspondingly in IoU and DICE compared to a single basic trainer. The range of improvement is 1.98%~6.38%. The proposed ensemble learning succeeds in automatic polyp segmentation, which potentially helps to establish more polyp datasets.',\n",
              "  'authors': ['Xudong, Guo',\n",
              "   'Na, Zhang',\n",
              "   'Jiefang, Guo',\n",
              "   'Huihe, Zhang',\n",
              "   'Youguo, Hao',\n",
              "   'Jingqing, Hang'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Medical physics',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31610020',\n",
              "  'title': 'Automated polyp segmentation for colonoscopy images：a method based on convolutional neural networks and ensemble learning.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': \"The advancement of natural orifice translumenal endoscopic surgery (NOTES) depends on the availability of a suitable platform. A 2008 Natural Orifice Surgery Consortium for Assessment and Research (NOSCAR) Working Group identified access, navigation, maneuverability, and stability to withstand instrument forces as the essential requirements for a successful NOTES platform [1]. No single NOTES platform can adequately achieve all four of these key capabilities. In this study, the authors tested a novel flexible endoscopic robotic platform in a cadaver to determine how it performs with respect to these specific requirements.|A highly maneuverable multichannel flexible robotic endoscopic platform developed for NOTES was used transgastrically in an adult cadaver to test the platform's ability to reach multiple intraabdominal targets. The device was under the control of the surgeon throughout the test. The surgeon was blinded to a control laparoscopic view. At each target organ, the platform was maneuvered via a joystick to provide different camera views and tool access orientation appropriate for the surgical tasks of each procedure. Standard endoscopic tools were used in the platform's two working channels to demonstrate the platform's ability to withstand the forces generated during tissue manipulation and clipping.|The platform reached each target organ site without difficulty, relying on the endoscopic view only. The platform's unique ability to maintain stability in two- and three-dimensional space resisted forces exerted by tools in the tool channels used for tissue manipulation and clipping. Endoscopic visualization allowed the surgeon to reposition the platform easily to gain a different camera view or tool access orientation about an organ (see video in Supplementary material).|The authors demonstrated that the four capabilities identified by NOSCAR can be provided by the flexible robotic endoscopic platform. These essential capabilities for the clinical implementation of NOTES were addressed specifically by this platform. The provision of these capabilities in a single device may further the advancement and adoption of NOTES.\",\n",
              "  'authors': ['Eisenberg, Dan', 'Storne, Eric', 'Belson, Amir'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Abdominal Cavity',\n",
              "   'Cadaver',\n",
              "   'Humans',\n",
              "   'Natural Orifice Endoscopic Surgery',\n",
              "   'Robotics',\n",
              "   'Stomach'],\n",
              "  'pmid': '20174943',\n",
              "  'title': 'Use of a flexible robotic transgastric natural orifice translumenal endoscopic surgery (NOTES) platform in a cadaver to test access, navigation, maneuverability, and stability.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2041-3033',\n",
              "  'IF_2013': '1.144',\n",
              "  'IF_2014': '1.329',\n",
              "  'IF_2015': '0.996',\n",
              "  'IF_2016': '1.005',\n",
              "  'IF_2017': '1.124',\n",
              "  'ISSN': '0954-4119',\n",
              "  'ISSN_online': '2041-3033',\n",
              "  'ISSN_print': '0954-4119',\n",
              "  'IsoAbbr': 'Proc Inst Mech Eng H',\n",
              "  'JrId': 1315,\n",
              "  'MedAbbr': 'Proc Inst Mech Eng H',\n",
              "  'NLMid': '8908934',\n",
              "  'Titre': 'Proc Inst Mech Eng H',\n",
              "  'abstract': 'Needle insertion into soft tissue is one of the most common medical interventions. This study provides macroscopic and microscopic observations of needle-gel interactions. A gelatin mixture is used as a soft-tissue simulant. For the macroscopic studies, system parameters, such as insertion velocity, needle diameter, gel elasticity, needle tip shape (including bevel angle) and insertion motion profile, are varied, while the maximum insertion force and maximum needle deflection are recorded. The needle tip and gel interactions are observed using confocal microscopic images. Observations indicate that increasing the insertion velocity and needle diameter results in larger insertion forces and smaller needle deflections. Varying the needle bevel angle from 8 degrees to 82 degrees results in the insertion force increasing monotonically, while the needle deflection does not. These variations are due to the coupling between gel rupture and tip compression interactions, which are observed during microscopic studies. Increasing the gel elasticity results in larger insertion forces and needle deflections. Varying the tip shapes demonstrates that bevel-tipped needles produce the largest deflection, but insertion force does not vary among the tested tip shapes. Insertion with different motion profiles are performed. Results show that adding I Hz rotational motion during linear insertion decreases the needle deflection. Increasing the rotational motion from I Hz to 5 Hz decreases the insertion force, while the needle deflection remains the same. A high-velocity (250 mm/s and 300 mm/s) tapping during insertion yields no significant decrease in needle deflection and a slight increase in insertion force.',\n",
              "  'authors': ['van Veen, Youri R J', 'Jahya, Alex', 'Misra, Sarthak'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Proceedings of the Institution of Mechanical Engineers. Part H, Journal of engineering in medicine',\n",
              "  'keywords': ['Biopsy, Needle',\n",
              "   'Brachytherapy',\n",
              "   'Elasticity',\n",
              "   'Equipment Design',\n",
              "   'Gelatin',\n",
              "   'Gels',\n",
              "   'Humans',\n",
              "   'Microscopy, Confocal',\n",
              "   'Models, Biological',\n",
              "   'Motion',\n",
              "   'Needles',\n",
              "   'Punctures',\n",
              "   'Robotics'],\n",
              "  'pmid': '22783760',\n",
              "  'title': 'Macroscopic and microscopic observations of needle insertion into gels.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-2747',\n",
              "  'IF_2013': '2.974',\n",
              "  'IF_2014': '2.718',\n",
              "  'IF_2015': '2.572',\n",
              "  'IF_2016': '3.033',\n",
              "  'IF_2017': '3.44',\n",
              "  'ISSN': '0361-9230',\n",
              "  'ISSN_online': '1873-2747',\n",
              "  'ISSN_print': '0361-9230',\n",
              "  'IsoAbbr': 'Brain Res. Bull.',\n",
              "  'JrId': 1921,\n",
              "  'MedAbbr': 'Brain Res Bull',\n",
              "  'NLMid': '7605818',\n",
              "  'Titre': 'Brain Res. Bull.',\n",
              "  'abstract': 'Theories of motor control often assume that the location of visual stimuli is expressed in non retinotopic frames of reference. The saccadic system is known in enough detail for us to examine the evidential basis of this assumption. The organization of the neural circuit that controls saccades is first summarized. It is shown to consist of at least two interconnected modules. The first one is the burst generator, which resides in the reticular formation, and is entrusted with the tasks of impedance matching, synergist coactivation and reciprocal inhibition between antagonists. The second is a metric computer, which resides in the superior colliculus and the cerebral cortex, and computes the size and direction of the desired movement. Alternative models of the burst generator are presented and their \"verisimilitude\" is assessed in the light of evidence concerning saccadic trajectories, neuronal discharge patterns, interneuronal connections, as well as the results of lesion and stimulation experiments. Several models of the \"metric computer\" in the superior colliculus are then examined; their performance is again evaluated in the light of psychophysical, anatomical, physiological, and clinical evidence. It is demonstrated that the location of visual stimuli need not be expressed in nonretinotopic frames of reference for either the burst generator or the metric computer to issue appropriate commands to move the eyes. Instead, using information concerning intervening movements of the eyes to update the location of visual stimuli in a retinotopic frame of reference suffices for the planning and execution of correct saccades. More generally, it is proposed that the location of sensory stimuli need not be expressed in higher order frames of reference (e.g., centered in the body or even in extrapersonal space) provided that their location in a sensorium specific map is updated on the basis of effector movements.',\n",
              "  'authors': ['Moschovakis, A K'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Brain research bulletin',\n",
              "  'keywords': ['Models, Theoretical',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oculomotor Nerve',\n",
              "   'Reference Values',\n",
              "   'Saccades'],\n",
              "  'pmid': '8886356',\n",
              "  'title': 'Neural network simulations of the primate oculomotor system. II. Frames of reference.',\n",
              "  'year': '1996'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'Clinical diagnosis has high requirements for the visual effect of medical images. To obtain rich detail features and clear edges for fusion medical images, an image fusion algorithm FFST-SR-PCNN based on fast finite shearlet transform (FFST) and sparse representation is proposed, aiming at the problem of poor clarity of edge details that is conducive to maintaining the details of source image in current algorithms. Firstly, the source image is decomposed into low-frequency coefficients and high-frequency coefficients by FFST. Secondly, the K-SVD method is used to train the low-frequency coefficients to obtain the overcomplete dictionary D, and then the OMP algorithm sparsely encodes the low-frequency coefficients to complete the fusion of the low-frequency coefficients. Then, a high-frequency coefficient is applied to excite a pulse-coupled neural network, and the fusion coefficient of the high-frequency coefficient is selected according to the number of ignitions. Finally, the fused low-frequency coefficient and high-frequency coefficient are reconstructed into the fused medical image by FFST inverse transform. The experimental results show that the image fusion result of the proposed algorithm is about 35% higher than the comparison algorithms for the edge information transfer factor QAB/F index and has achieved good results in both subjective visual effects and objective evaluation indicators.',\n",
              "  'authors': ['Tan, Ling', 'Yu, Xin'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Brain',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnostic Imaging',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Medical Informatics',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Software',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'Wavelet Analysis'],\n",
              "  'pmid': '30944576',\n",
              "  'title': 'Medical Image Fusion Based on Fast Finite Shearlet Transform and Sparse Representation.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1769-6917',\n",
              "  'IF_2013': '0.635',\n",
              "  'IF_2014': '0.604',\n",
              "  'IF_2015': '0.706',\n",
              "  'IF_2016': '0.853',\n",
              "  'IF_2017': '0.84',\n",
              "  'ISSN': '0007-4551',\n",
              "  'ISSN_online': '1769-6917',\n",
              "  'ISSN_print': '0007-4551',\n",
              "  'IsoAbbr': 'Bull Cancer',\n",
              "  'JrId': 2060,\n",
              "  'MedAbbr': 'Bull Cancer',\n",
              "  'NLMid': '0072416',\n",
              "  'Titre': 'Bull Cancer',\n",
              "  'abstract': 'Twenty years ago, laparoscopic surgery entirely changed the traditional style of surgical operations. However, it was difficult to perform some interventions, particularly in restricted space or if acts of reconstruction are necessary. It was necessary to develop new technologies such robotic surgery. We detail the different technologies and we precise the place of the robotic surgery in oncological surgery. In conclusion, we believe that, in the near future as robotic technology continues to develop, it could replace traditional surgery not only in the treatment of benign diseases but also in malignant illnesses.',\n",
              "  'authors': ['Marchal, Frédéric',\n",
              "   'Rauch, Philippe',\n",
              "   'Verhaeghe, Jean-Luc',\n",
              "   'Guillemin, François'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Bulletin du cancer',\n",
              "  'keywords': ['Colorectal Surgery',\n",
              "   'Equipment Design',\n",
              "   'Gynecologic Surgical Procedures',\n",
              "   'Laparoscopy',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Thoracic Surgical Procedures',\n",
              "   'Urologic Surgical Procedures'],\n",
              "  'pmid': '18156116',\n",
              "  'title': '[Laparoscopic surgery and robot].',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1520-4995',\n",
              "  'IF_2013': '3.194',\n",
              "  'IF_2014': '3.015',\n",
              "  'IF_2015': '2.876',\n",
              "  'IF_2016': '2.938',\n",
              "  'IF_2017': '2.997',\n",
              "  'ISSN': '0006-2960',\n",
              "  'ISSN_online': '1520-4995',\n",
              "  'ISSN_print': '0006-2960',\n",
              "  'IsoAbbr': 'Biochemistry',\n",
              "  'JrId': 1028,\n",
              "  'MedAbbr': 'Biochemistry',\n",
              "  'NLMid': '0370623',\n",
              "  'Titre': 'Biochemistry',\n",
              "  'abstract': 'Members of the Multiple antibiotic resistance Regulator (MarR) family of DNA binding proteins regulate transcription of a wide array of genes required for virulence and pathogenicity of bacteria. The present study reports the molecular characterization of HosA (Homologue of SlyA), a MarR protein, with respect to its target gene, DNA recognition motif, and nature of its ligand. Through a comparative genomics approach, we demonstrate that hosA is in synteny with nonoxidative hydroxyarylic acid decarboxylase (HAD) operon and is present exclusively within the mutS-rpoS polymorphic region in nine different genera of Enterobacteriaceae family. Using molecular biology and biochemical approach, we demonstrate that HosA binds to a palindromic sequence downstream to the transcription start site of divergently transcribed nonoxidative HAD operon and represses its expression. Furthermore, in silico analysis showed that the recognition motif for HosA is highly conserved in the upstream region of divergently transcribed operon in different genera of Enterobacteriaceae family. A systematic chemical search for the physiological ligand revealed that 4-hydroxybenzoic acid (4-HBA) interacts with HosA and derepresses HosA mediated repression of the nonoxidative HAD operon. Based on our study, we propose a model for molecular mechanism underlying the regulation of nonoxidative HAD operon by HosA in Enterobacteriaceae family. ',\n",
              "  'authors': ['Roy, Ajit', 'Ranjan, Akash'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Biochemistry',\n",
              "  'keywords': ['Carboxy-Lyases',\n",
              "   'Computational Biology',\n",
              "   'DNA Footprinting',\n",
              "   'Enzyme Repression',\n",
              "   'Escherichia coli Proteins',\n",
              "   'Expert Systems',\n",
              "   'Ligands',\n",
              "   'Models, Biological',\n",
              "   'Molecular Weight',\n",
              "   'Mutagenesis, Site-Directed',\n",
              "   'Mutation',\n",
              "   'Nucleotide Motifs',\n",
              "   'Operon',\n",
              "   'Parabens',\n",
              "   'Recombinant Proteins',\n",
              "   'Synteny',\n",
              "   'Transcription Factors',\n",
              "   'Uropathogenic Escherichia coli'],\n",
              "  'pmid': '26818787',\n",
              "  'title': 'HosA, a MarR Family Transcriptional Regulator, Represses Nonoxidative Hydroxyarylic Acid Decarboxylase Operon and Is Modulated by 4-Hydroxybenzoic Acid.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Currently, placental maturity is performed using subjective evaluation, which can be unreliable as it is highly dependent on the observations and experiences of clinicians. To address this problem, this paper proposes a method to automatically stage placenta maturity from B-mode ultrasound (US) images based on dense sampling and novel feature descriptors. Specifically, our proposed method first densely extracts features with a regular grid based on dense sampling instead of a few unreliable interest points. Followed by, these features are clustered using generative Gaussian mixture model (GMM) to obtain high order statistics of the features. The clustering representatives (i.e., cluster means) are encoded by Fisher vector (FV) for staging accuracy enhancement. Differing from the previous studies, a multi-layer FV is investigated to exploit the spatial information rather than the single layer FV. Experimental results show that the proposed method with the dense FV has achieved an area under the receiver of characteristics (AUC) of 96.77%, sensitivity and specificity of 98.04% and 93.75% for the placental maturity staging, respectively. Our experimental results also demonstrate that the dense feature outperforms the traditional sparse feature for placental maturity staging. ',\n",
              "  'authors': ['Lei, Baiying',\n",
              "   'Yao, Yuan',\n",
              "   'Chen, Siping',\n",
              "   'Li, Shengli',\n",
              "   'Li, Wanjun',\n",
              "   'Ni, Dong',\n",
              "   'Wang, Tianfu'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Cluster Analysis',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Placenta',\n",
              "   'Pregnancy',\n",
              "   'Support Vector Machine',\n",
              "   'Ultrasonography, Prenatal'],\n",
              "  'pmid': '26228175',\n",
              "  'title': 'Discriminative Learning for Automatic Staging of Placental Maturity via Multi-layer Fisher Vector.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-9568',\n",
              "  'IF_2013': '3.669',\n",
              "  'IF_2014': '3.181',\n",
              "  'IF_2015': '2.975',\n",
              "  'IF_2016': '2.941',\n",
              "  'IF_2017': '2.832',\n",
              "  'ISSN': '0953-816X',\n",
              "  'ISSN_online': '1460-9568',\n",
              "  'ISSN_print': '0953-816X',\n",
              "  'IsoAbbr': 'Eur. J. Neurosci.',\n",
              "  'JrId': 2607,\n",
              "  'MedAbbr': 'Eur J Neurosci',\n",
              "  'NLMid': '8918110',\n",
              "  'Titre': 'Eur. J. Neurosci.',\n",
              "  'abstract': 'Progression of progenitor cells towards neuronal differentiation is tightly linked with cell cycle control and the switch from proliferative to neuron-generating divisions. We have previously shown that the neuronal protein BM88 drives neuroblastoma cells towards exit from the cell cycle and differentiation into a neuronal phenotype in vitro. Here, we explored the role of BM88 during neuronal birth, cell cycle exit and the initiation of differentiation in vivo. By double- and triple-labelling with the S-phase marker BrdU or the late G2 and M-phase marker cyclin B1, antibodies to BM88 and markers of the neuronal or glial cell lineages, we demonstrate that in the rodent forebrain, BM88 is expressed in multipotential progenitor cells before terminal mitosis and in their neuronal progeny during the neurogenic interval, as well as in the adult. Further, we defined at E16 a cohort of proliferative progenitors that exit S phase in synchrony, and by following their fate for 24 h we show that BM88 is associated with the dynamics of neuron-generating divisions. Expression of BM88 was also evident in cycling cortical radial glial cells, which constitute the main neurogenic population in the cerebral cortex. In agreement, BM88 expression was markedly reduced and restricted to a smaller percentage of cells in the cerebral cortex of the Small eye mutant mice, which lack functional Pax6 and exhibit severe neurogenesis defects. Our data show an interesting correlation between BM88 expression and the progression of progenitor cells towards neuronal differentiation during the neurogenic interval.',\n",
              "  'authors': ['Koutmani, Yassemi',\n",
              "   'Hurel, Catherine',\n",
              "   'Patsavoudi, Evangelia',\n",
              "   'Hack, Michael',\n",
              "   'Gotz, Magdalena',\n",
              "   'Thomaidou, Dimitra',\n",
              "   'Matsas, Rebecca'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The European journal of neuroscience',\n",
              "  'keywords': ['Age Factors',\n",
              "   'Amino Acid Transport System X-AG',\n",
              "   'Animals',\n",
              "   'Animals, Newborn',\n",
              "   'Antigens',\n",
              "   'Biomarkers',\n",
              "   'Blotting, Western',\n",
              "   'Brain',\n",
              "   'Bromodeoxyuridine',\n",
              "   'Carbocyanines',\n",
              "   'Cell Count',\n",
              "   'Cell Cycle',\n",
              "   'Cell Differentiation',\n",
              "   'Cell Lineage',\n",
              "   'Cell Proliferation',\n",
              "   'Cyclin B',\n",
              "   'Cyclin B1',\n",
              "   'Embryo, Mammalian',\n",
              "   'Eye Proteins',\n",
              "   'Female',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Glial Fibrillary Acidic Protein',\n",
              "   'Homeodomain Proteins',\n",
              "   'Immunohistochemistry',\n",
              "   'Intermediate Filament Proteins',\n",
              "   'Male',\n",
              "   'Membrane Proteins',\n",
              "   'Mice',\n",
              "   'Mice, Inbred C57BL',\n",
              "   'Mice, Neurologic Mutants',\n",
              "   'Nerve Tissue Proteins',\n",
              "   'Nestin',\n",
              "   'Neural Cell Adhesion Molecule L1',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'PAX6 Transcription Factor',\n",
              "   'Paired Box Transcription Factors',\n",
              "   'Phosphopyruvate Hydratase',\n",
              "   'Pregnancy',\n",
              "   'Proteoglycans',\n",
              "   'RNA, Messenger',\n",
              "   'Rats',\n",
              "   'Rats, Wistar',\n",
              "   'Repressor Proteins',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
              "   'Sialic Acids',\n",
              "   'Stem Cells',\n",
              "   'Thymidine',\n",
              "   'Time Factors',\n",
              "   'Tritium'],\n",
              "  'pmid': '15548196',\n",
              "  'title': 'BM88 is an early marker of proliferating precursor cells that will differentiate into the neuronal lineage.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1662-5110',\n",
              "  'IF_2013': '2.95',\n",
              "  'IF_2014': '3.568',\n",
              "  'IF_2015': '3.879',\n",
              "  'IF_2016': '3.005',\n",
              "  'IF_2017': '3.131',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1662-5110',\n",
              "  'IsoAbbr': 'Front Neural Circuits',\n",
              "  'JrId': 34621,\n",
              "  'MedAbbr': 'Front Neural Circuits',\n",
              "  'NLMid': '101477940',\n",
              "  'Titre': 'Front Neural Circuits',\n",
              "  'abstract': 'Adaptable gain regulation is at the core of the forward controller operation performed by the cerebro-cerebellar loops and it allows the intensity of motor acts to be finely tuned in a predictive manner. In order to learn and store information about body-object dynamics and to generate an internal model of movement, the cerebellum is thought to employ long-term synaptic plasticity. LTD at the PF-PC synapse has classically been assumed to subserve this function (Marr, 1969). However, this plasticity alone cannot account for the broad dynamic ranges and time scales of cerebellar adaptation. We therefore tested the role of plasticity distributed over multiple synaptic sites (Hansel et al., 2001; Gao et al., 2012) by generating an analog cerebellar model embedded into a control loop connected to a robotic simulator. The robot used a three-joint arm and performed repetitive fast manipulations with different masses along an 8-shape trajectory. In accordance with biological evidence, the cerebellum model was endowed with both LTD and LTP at the PF-PC, MF-DCN and PC-DCN synapses. This resulted in a network scheme whose effectiveness was extended considerably compared to one including just PF-PC synaptic plasticity. Indeed, the system including distributed plasticity reliably self-adapted to manipulate different masses and to learn the arm-object dynamics over a time course that included fast learning and consolidation, along the lines of what has been observed in behavioral tests. In particular, PF-PC plasticity operated as a time correlator between the actual input state and the system error, while MF-DCN and PC-DCN plasticity played a key role in generating the gain controller. This model suggests that distributed synaptic plasticity allows generation of the complex learning properties of the cerebellum. The incorporation of further plasticity mechanisms and of spiking signal processing will allow this concept to be extended in a more realistic computational scenario. ',\n",
              "  'authors': ['Garrido, Jesús A',\n",
              "   'Luque, Niceto R',\n",
              "   \"D'Angelo, Egidio\",\n",
              "   'Ros, Eduardo'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Frontiers in neural circuits',\n",
              "  'keywords': ['Humans',\n",
              "   'Learning',\n",
              "   'Models, Neurological',\n",
              "   'Movement',\n",
              "   'Neuronal Plasticity',\n",
              "   'Neurons',\n",
              "   'Robotics',\n",
              "   'Synapses'],\n",
              "  'pmid': '24130518',\n",
              "  'title': 'Distributed cerebellar plasticity implements adaptable gain control in a manipulation task: a closed-loop robotic simulation.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['HB'],\n",
              "  'ESSN': '1935-9780',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '2.303',\n",
              "  'IF_2016': '3.198',\n",
              "  'IF_2017': '3.114',\n",
              "  'ISSN': '1935-9772',\n",
              "  'ISSN_online': '1935-9780',\n",
              "  'ISSN_print': '1935-9772',\n",
              "  'IsoAbbr': 'Anat Sci Educ',\n",
              "  'JrId': 35507,\n",
              "  'MedAbbr': 'Anat Sci Educ',\n",
              "  'NLMid': '101392205',\n",
              "  'Titre': 'Anat Sci Educ',\n",
              "  'abstract': 'One in six Americans is currently affected by neurologic disease. As the United States population ages, the number of neurologic complaints is expected to increase. Thus, there is a pressing need for more neurologists as well as more neurology training in other specialties. Often interest in neurology begins during medical school, so improving education in medical neural courses is a critical step toward producing more neurologists and better neurology training in other specialists. To this end, a novel applied neuroanatomy elective was designed at the University of Texas Health Science Center at San Antonio (UTHSCSA) to complement the traditional first-year medical neuroscience course and promote engagement and deep learning of the material with a focus on neurosensory pathways. The elective covered four neurosensory modalities (proprioception/balance, vision, auditory, and taste/olfaction) over four sessions, each with a short classroom component and a much longer activity component. At each session, students reviewed the neurosensory pathways through structured presentations and then applied them to preplanned interactive activities, many of which allowed students to utilize their artistic talents. Students were required to complete subjective pre-course and post-course surveys and reflections. The survey results and positive student comments suggest that the elective was a valuable tool when used in parallel with the traditional medical neuroscience course in promoting engagement and reinforcement of the neurosensory material.',\n",
              "  'authors': ['Dao, Vinh',\n",
              "   'Yeh, Pon-Hsiu',\n",
              "   'Vogel, Kristine S',\n",
              "   'Moore, Charleen M'],\n",
              "  'categories': ['EDUCATION, SCIENTIFIC DISCIPLINES'],\n",
              "  'journal': 'Anatomical sciences education',\n",
              "  'keywords': ['Anatomy, Artistic',\n",
              "   'Auditory Perception',\n",
              "   'Curriculum',\n",
              "   'Education, Medical, Undergraduate',\n",
              "   'Educational Measurement',\n",
              "   'Educational Status',\n",
              "   'Humans',\n",
              "   'Nervous System',\n",
              "   'Neuroanatomy',\n",
              "   'Postural Balance',\n",
              "   'Problem-Based Learning',\n",
              "   'Program Evaluation',\n",
              "   'Proprioception',\n",
              "   'Reinforcement (Psychology)',\n",
              "   'Schools, Medical',\n",
              "   'Sensation',\n",
              "   'Smell',\n",
              "   'Students, Medical',\n",
              "   'Surveys and Questionnaires',\n",
              "   'Taste',\n",
              "   'Teaching',\n",
              "   'Texas',\n",
              "   'Visual Perception'],\n",
              "  'pmid': '24920370',\n",
              "  'title': 'Applied neuroanatomy elective to reinforce and promote engagement with neurosensory pathways using interactive and artistic activities.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Robotic assistance is increasingly used in neurological rehabilitation for enhanced training. Furthermore, therapy robots have the potential for accurate assessment of motor function in order to diagnose the patient status, to measure therapy progress or to feedback the movement performance to the patient and therapist in real time. We investigated whether a set of robot-based assessments that encompasses kinematic, kinetic and timing metrics is applicable, safe, reliable and comparable to clinical metrics for measurement of arm motor function. Twenty-four healthy subjects and five patients after spinal cord injury underwent robot-based assessments using the exoskeleton robot ARMin. Five different tasks were performed with aid of a visual display. Ten kinematic, kinetic and timing assessment parameters were extracted on joint- and end-effector level (active and passive range of motion, cubic reaching volume, movement time, distance-path ratio, precision, smoothness, reaction time, joint torques and joint stiffness). For cubic volume, joint torques and the range of motion for most joints, good inter- and intra-rater reliability were found whereas precision, movement time, distance-path ratio and smoothness showed weak to moderate reliability. A comparison with clinical scores revealed good correlations between robot-based joint torques and the Manual Muscle Test. Reaction time and distance-path ratio showed good correlation with the \"Graded and Redefined Assessment of Strength, Sensibility and Prehension\" (GRASSP) and the Van Lieshout Test (VLT) for movements towards a predefined position in the center of the frontal plane. In conclusion, the therapy robot ARMin provides a comprehensive set of assessments that are applicable and safe. The first results with spinal cord injured patients and healthy subjects suggest that the measurements are widely reliable and comparable to clinical scales for arm motor function. The methods applied and results can serve as a basis for the future development of end-effector and exoskeleton-based robotic assessments. ',\n",
              "  'authors': ['Keller, Urs',\n",
              "   'Schölch, Sabine',\n",
              "   'Albisser, Urs',\n",
              "   'Rudhe, Claudia',\n",
              "   'Curt, Armin',\n",
              "   'Riener, Robert',\n",
              "   'Klamroth-Marganska, Verena'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Arm',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Movement',\n",
              "   'Range of Motion, Articular',\n",
              "   'Reproducibility of Results',\n",
              "   'Robotics',\n",
              "   'Spinal Cord Injuries',\n",
              "   'Treatment Outcome',\n",
              "   'Young Adult'],\n",
              "  'pmid': '25996374',\n",
              "  'title': 'Robot-assisted arm assessments in spinal cord injured patients: a consideration of concept study.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1614-7499',\n",
              "  'IF_2013': '2.757',\n",
              "  'IF_2014': '2.828',\n",
              "  'IF_2015': '2.76',\n",
              "  'IF_2016': '2.741',\n",
              "  'IF_2017': '2.8',\n",
              "  'ISSN': '0944-1344',\n",
              "  'ISSN_online': '1614-7499',\n",
              "  'ISSN_print': '0944-1344',\n",
              "  'IsoAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'JrId': 22074,\n",
              "  'MedAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'NLMid': '9441769',\n",
              "  'Titre': 'Environ Sci Pollut Res Int',\n",
              "  'abstract': \"This study investigates the impacts of different geological units on groundwater quality of an aquifer in southern Iran. The Kriging interpolation technique with a Gaussian semivariogram model was employed to prepare groundwater maps for different water quality constituents. In the next stage, two different models based on fuzzy analytic hierarchy process (AHP) and Dempster-Shafer theory (DST) were used to evaluate the overall water quality index based on the World Health Organization's drinking water standard in different parts of the aquifer. The DST model was able to generate water quality maps with 99.5%, 99%, and 95% confidence levels. The water quality maps were subsequently compared with the geology map of the area to determine the effects of different soil types on the water quality of the aquifer. Both methods showed poor water quality indices in the areas with an Asmari formation containing elevated levels of chloride and sodium ions. Comparison of water quality maps generated by the fuzzy-AHP and DST model revealed that the DST could more reliably handle the uncertainty in the water quality data, and thus was able to generate more accurate water quality maps. Increasing the confidence level in the DST model yielded water quality maps with a decreased overall water quality index. Results of this study could assist water management practices to generate water quality maps for their groundwater resources with confidence levels commensurate socio-economic importance of the region.\",\n",
              "  'authors': ['Mokarram, Marzieh', 'Hojati, Majid', 'Saber, Ali'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental science and pollution research international',\n",
              "  'keywords': ['Environmental Monitoring',\n",
              "   'Fuzzy Logic',\n",
              "   'Geologic Sediments',\n",
              "   'Groundwater',\n",
              "   'Iran',\n",
              "   'Models, Theoretical',\n",
              "   'Soil',\n",
              "   'Spatial Analysis',\n",
              "   'Water Quality'],\n",
              "  'pmid': '31073838',\n",
              "  'title': 'Application of Dempster-Shafer theory and fuzzy analytic hierarchy process for evaluating the effects of geological formation units on groundwater quality.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1471-2156',\n",
              "  'IF_2013': '2.356',\n",
              "  'IF_2014': '2.397',\n",
              "  'IF_2015': '2.152',\n",
              "  'IF_2016': '2.266',\n",
              "  'IF_2017': '2.469',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2156',\n",
              "  'IsoAbbr': 'BMC Genet.',\n",
              "  'JrId': 21880,\n",
              "  'MedAbbr': 'BMC Genet',\n",
              "  'NLMid': '100966978',\n",
              "  'Titre': 'BMC Genet.',\n",
              "  'abstract': 'Gene-environment interactions play an important role in the etiological pathway of complex diseases. An appropriate statistical method for handling a wide variety of complex situations involving interactions between variables is still lacking, especially when continuous variables are involved. The aim of this paper is to explore the ability of neural networks to model different structures of gene-environment interactions. A simulation study is set up to compare neural networks with standard logistic regression models. Eight different structures of gene-environment interactions are investigated. These structures are characterized by penetrance functions that are based on sigmoid functions or on combinations of linear and non-linear effects of a continuous environmental factor and a genetic factor with main effect or with a masking effect only.|In our simulation study, neural networks are more successful in modeling gene-environment interactions than logistic regression models. This outperfomance is especially pronounced when modeling sigmoid penetrance functions, when distinguishing between linear and nonlinear components, and when modeling masking effects of the genetic factor.|Our study shows that neural networks are a promising approach for analyzing gene-environment interactions. Especially, if no prior knowledge of the correct nature of the relationship between co-variables and response variable is present, neural networks provide a valuable alternative to regression methods that are limited to the analysis of linearly separable data.',\n",
              "  'authors': ['Günther, Frauke', 'Pigeot, Iris', 'Bammann, Karin'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'BMC genetics',\n",
              "  'keywords': ['Case-Control Studies',\n",
              "   'Gene-Environment Interaction',\n",
              "   'Logistic Models',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '22583704',\n",
              "  'title': 'Artificial neural networks modeling gene-environment interaction.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['EE'],\n",
              "  'ESSN': '1420-3049',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '2.416',\n",
              "  'IF_2015': '2.465',\n",
              "  'IF_2016': '2.861',\n",
              "  'IF_2017': '3.098',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1420-3049',\n",
              "  'IsoAbbr': 'Molecules',\n",
              "  'JrId': 22275,\n",
              "  'MedAbbr': 'Molecules',\n",
              "  'NLMid': '100964009',\n",
              "  'Titre': 'Molecules',\n",
              "  'abstract': 'Automation in combination with high throughput screening methods has revolutionised molecular biology in the last two decades. Today, many combinatorial libraries as well as several systems for automation are available. Depending on scope, budget and time, a different combination of library and experimental handling might be most effective. In this review we will discuss several concepts of combinatorial libraries and provide information as what to expect from these depending on the given context.',\n",
              "  'authors': ['Glökler, Jörn', 'Schütze, Tatjana', 'Konthur, Zoltán'],\n",
              "  'categories': ['CHEMISTRY, ORGANIC'],\n",
              "  'journal': 'Molecules (Basel, Switzerland)',\n",
              "  'keywords': ['Automation, Laboratory',\n",
              "   'Combinatorial Chemistry Techniques',\n",
              "   'High-Throughput Screening Assays',\n",
              "   'Peptide Library',\n",
              "   'Robotics',\n",
              "   'SELEX Aptamer Technique',\n",
              "   'Small Molecule Libraries'],\n",
              "  'pmid': '20428057',\n",
              "  'title': 'Automation in the high-throughput selection of random combinatorial libraries--different approaches for select applications.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'Screening mammography is the most effective means for early detection of breast cancer. Although general rules for discriminating malignant and benign lesions exist, radiologists are unable to perfectly detect and classify all lesions as malignant and benign, for many reasons which include, but are not limited to, overlap of features that distinguish malignancy, difficulty in estimating disease risk, and variability in recommended management. When predictive variables are numerous and interact, ad hoc decision making strategies based on experience and memory may lead to systematic errors and variability in practice. The integration of computer models to help radiologists increase the accuracy of mammography examinations in diagnostic decision making has gained increasing attention in the last two decades. In this study, we provide an overview of one of the most commonly used models, artificial neural networks (ANNs), in mammography interpretation and diagnostic decision making and discuss important features in mammography interpretation. We conclude by discussing several common limitations of existing research on ANN-based detection and diagnostic models and provide possible future research directions.',\n",
              "  'authors': ['Ayer, Turgay', 'Chen, Qiushi', 'Burnside, Elizabeth S'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Breast Neoplasms',\n",
              "   'Calcinosis',\n",
              "   'Computational Biology',\n",
              "   'Computer Simulation',\n",
              "   'Decision Making',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Mammography',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Radiology',\n",
              "   'Risk Factors'],\n",
              "  'pmid': '23781276',\n",
              "  'title': 'Artificial neural networks in mammography interpretation and diagnostic decision making.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'This paper proposes to characterize the continuum of colorectal cancer (CRC) using multiple texture features extracted from multispectral optical microscopy images. Three types of pathological tissues (PT) are considered: benign hyperplasia, intraepithelial neoplasia and carcinoma.|In the proposed approach, the region of interest containing PT is first extracted from multispectral images using active contour segmentation. This region is then encoded using texture features based on the Laplacian-of-Gaussian (LoG) filter, discrete wavelets (DW) and gray level co-occurrence matrices (GLCM). To assess the significance of textural differences between PT types, a statistical analysis based on the Kruskal-Wallis test is performed. The usefulness of texture features is then evaluated quantitatively in terms of their ability to predict PT types using various classifier models.|Preliminary results show significant texture differences between PT types, for all texture features (p-value < 0.01). Individually, GLCM texture features outperform LoG and DW features in terms of PT type prediction. However, a higher performance can be achieved by combining all texture features, resulting in a mean classification accuracy of 98.92%, sensitivity of 98.12%, and specificity of 99.67%.|These results demonstrate the efficiency and effectiveness of combining multiple texture features for characterizing the continuum of CRC and discriminating between pathological tissues in multispectral images.',\n",
              "  'authors': ['Chaddad, Ahmad',\n",
              "   'Desrosiers, Christian',\n",
              "   'Bouridane, Ahmed',\n",
              "   'Toews, Matthew',\n",
              "   'Hassan, Lama',\n",
              "   'Tanougast, Camel'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Colorectal Neoplasms',\n",
              "   'Diagnostic Imaging',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'In Vitro Techniques'],\n",
              "  'pmid': '26901134',\n",
              "  'title': 'Multi Texture Analysis of Colorectal Cancer Continuum Using Multispectral Imagery.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZD'],\n",
              "  'ESSN': '1827-1839',\n",
              "  'IF_2013': '1.014',\n",
              "  'IF_2014': '0.833',\n",
              "  'IF_2015': '0.899',\n",
              "  'IF_2016': '1.37',\n",
              "  'IF_2017': '1.156',\n",
              "  'ISSN': '0392-9590',\n",
              "  'ISSN_online': '1827-1839',\n",
              "  'ISSN_print': '0392-9590',\n",
              "  'IsoAbbr': 'Int Angiol',\n",
              "  'JrId': 4867,\n",
              "  'MedAbbr': 'Int Angiol',\n",
              "  'NLMid': '8402693',\n",
              "  'Titre': 'Int Angiol',\n",
              "  'abstract': 'With the rapid introduction of revolutionary technologies in surgical practice, such as computer-enhanced robotic surgery, the complexity in various aspects, including medical, legal and ethical, will increase exponentially. Our aim was to highlight important legal and ethical implications emerged from the application of robotic surgery.|Search of the pertinent medical and legal literature.|Robotic surgery may open new avenues in the near future in surgical practice. However, in robotic surgery, special training and experience along with high quality assessment are required in order to provide normal conscientious care and state-of-the-art treatment. While the legal basis for professional liability remains exactly the same, litigation with the use of robotic surgery may be complex. In case of an undesirable outcome, in addition to physician and hospital, the manufacturer of the robotic system may be sued. In respect to ethical issues in robotic surgery, equipment safety and reliability, provision of adequate information, and maintenance of confidentiality are all of paramount importance. Also, the cost of robotic surgery and the lack of such systems in most of the public hospitals may restrict the majority from the benefits offered by the new technology.|While surgical robotics will have a significant impact on surgical practice, it presents challenges so much in the realm of law and ethics as of medicine and health care.',\n",
              "  'authors': ['Mavroforou, A',\n",
              "   'Michalodimitrakis, E',\n",
              "   'Hatzitheo-Filou, C',\n",
              "   'Giannoukas, A'],\n",
              "  'categories': ['PERIPHERAL VASCULAR DISEASE'],\n",
              "  'journal': 'International angiology : a journal of the International Union of Angiology',\n",
              "  'keywords': ['Clinical Competence',\n",
              "   'Confidentiality',\n",
              "   'Consumer Product Safety',\n",
              "   'Equipment Safety',\n",
              "   'Government Regulation',\n",
              "   'Health Care Costs',\n",
              "   'Health Policy',\n",
              "   'Health Services Accessibility',\n",
              "   'Humans',\n",
              "   'Informed Consent',\n",
              "   'Liability, Legal',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted'],\n",
              "  'pmid': '20224537',\n",
              "  'title': 'Legal and ethical issues in robotic surgery.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1529-2401',\n",
              "  'IF_2013': '6.747',\n",
              "  'IF_2014': '6.344',\n",
              "  'IF_2015': '5.924',\n",
              "  'IF_2016': '5.988',\n",
              "  'IF_2017': '5.97',\n",
              "  'ISSN': '0270-6474',\n",
              "  'ISSN_online': '1529-2401',\n",
              "  'ISSN_print': '0270-6474',\n",
              "  'IsoAbbr': 'J. Neurosci.',\n",
              "  'JrId': 5035,\n",
              "  'MedAbbr': 'J Neurosci',\n",
              "  'NLMid': '8102140',\n",
              "  'Titre': 'J. Neurosci.',\n",
              "  'abstract': 'Hemodynamic research has recently clarified key nodes and links in brain networks implementing inhibitory control. Although fMRI methods are optimized for identifying the structure of brain networks, the relatively slow temporal course of fMRI limits the ability to characterize network operation. The latter is crucial for developing a mechanistic understanding of how brain networks shift dynamically to support inhibitory control. To address this critical gap, we applied spectrally resolved Granger causality (GC) and random forest machine learning tools to human EEG data in two large samples of adults (test sample n = 96, replication sample n = 237, total N = 333, both sexes) who performed a color-word Stroop task. Time-frequency analysis confirmed that recruitment of inhibitory control accompanied by slower behavioral responses was related to changes in theta and alpha/beta power. GC analyses revealed directionally asymmetric exchanges within frontal and between frontal and parietal brain areas: top-down influence of superior frontal gyrus (SFG) over both dorsal ACC (dACC) and inferior frontal gyrus (IFG), dACC control over middle frontal gyrus (MFG), and frontal-parietal exchanges (IFG, precuneus, MFG). Predictive analytics confirmed a combination of behavioral and brain-derived variables as the best set of predictors of inhibitory control demands, with SFG theta bearing higher classification importance than dACC theta and posterior beta tracking the onset of behavioral response. The present results provide mechanistic insight into the biological implementation of a psychological phenomenon: inhibitory control is implemented by dynamic routing processes during which the target response is upregulated via theta-mediated effective connectivity within key PFC nodes and via beta-mediated motor preparation.SIGNIFICANCE STATEMENT Hemodynamic neuroimaging research has recently clarified regional structures in brain networks supporting inhibitory control. However, due to inherent methodological constraints, much of this research has been unable to characterize the temporal dynamics of such networks (e.g., direction of information flow between nodes). Guided by fMRI research identifying the structure of brain networks supporting inhibitory control, results of EEG source analysis in a test sample (n = 96) and replication sample (n = 237) using effective connectivity and predictive analytics strategies advance a model of inhibitory control by characterizing the precise temporal dynamics by which this network operates and exemplify an approach by which mechanistic models can be developed for other key psychological processes.',\n",
              "  'authors': ['Popov, Tzvetan',\n",
              "   'Westner, Britta U',\n",
              "   'Silton, Rebecca L',\n",
              "   'Sass, Sarah M',\n",
              "   'Spielberg, Jeffrey M',\n",
              "   'Rockstroh, Brigitte',\n",
              "   'Heller, Wendy',\n",
              "   'Miller, Gregory A'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'The Journal of neuroscience : the official journal of the Society for Neuroscience',\n",
              "  'keywords': ['Adult',\n",
              "   'Alpha Rhythm',\n",
              "   'Beta Rhythm',\n",
              "   'Brain',\n",
              "   'Brain Mapping',\n",
              "   'Causality',\n",
              "   'Electroencephalography',\n",
              "   'Female',\n",
              "   'Frontal Lobe',\n",
              "   'Humans',\n",
              "   'Inhibition (Psychology)',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Nerve Net',\n",
              "   'Parietal Lobe',\n",
              "   'Reaction Time',\n",
              "   'Stroop Test',\n",
              "   'Theta Rhythm',\n",
              "   'Young Adult'],\n",
              "  'pmid': '29636394',\n",
              "  'title': 'Time Course of Brain Network Reconfiguration Supporting Inhibitory Control.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['GA'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '1.11',\n",
              "  'IF_2014': '1.069',\n",
              "  'IF_2015': '1.205',\n",
              "  'IF_2016': '1.446',\n",
              "  'IF_2017': '1.67',\n",
              "  'ISSN': '0969-0700',\n",
              "  'ISSN_print': '0969-0700',\n",
              "  'IsoAbbr': 'J Wound Care',\n",
              "  'JrId': 20146,\n",
              "  'MedAbbr': 'J Wound Care',\n",
              "  'NLMid': '9417080',\n",
              "  'Titre': 'J Wound Care',\n",
              "  'abstract': \"This study aimed to identify the risk factors that influence the healing process of venous leg ulcers treated with compression bandaging, with a view to predicting healing time.|A retrospective cohort study was performed on data collected prospectively on 325 consecutive patients presenting with 345 venous ulcers at the Salford Primary Care Trust leg ulcer clinic between January 1997 and December 1999. Use of an artificial neural network (ANN) technique accurately predicted the healing times for 68% of the patients.|The ANN demonstrated that healing was significantly related to a history of previous leg ulceration, 'quite wet' ulcer exudate, high body mass index, large initial total ulcer area, increasing age and male gender.|The ability to identify at presentation ulcers that might be resistant to standard therapy would allow early consideration of more radical treatments such as hospitalisation, wound debridement or venous surgery.\",\n",
              "  'authors': ['Taylor, R J', 'Taylor, A D', 'Smyth, J V'],\n",
              "  'categories': ['DERMATOLOGY'],\n",
              "  'journal': 'Journal of wound care',\n",
              "  'keywords': ['Adult',\n",
              "   'Age Factors',\n",
              "   'Aged',\n",
              "   'Bandages',\n",
              "   'Chronic Disease',\n",
              "   'Cohort Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Predictive Value of Tests',\n",
              "   'Prognosis',\n",
              "   'Proportional Hazards Models',\n",
              "   'Retrospective Studies',\n",
              "   'Risk Assessment',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Sex Factors',\n",
              "   'Time Factors',\n",
              "   'Varicose Ulcer',\n",
              "   'Wound Healing'],\n",
              "  'pmid': '11933726',\n",
              "  'title': 'Using an artificial neural network to predict healing times and risk factors for venous leg ulcers.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': \"This paper presents a system for classification of asthma and chronic obstructive pulmonary disease (COPD) based on fuzzy rules and the trained neural network.|Fuzzy rules and neural network parameters are defined according to Global Initiative for Asthma (GINA) and Global Initiative for chronic Obstructive Lung Disease (GOLD) guidelines. For neural network training more than one thousand medical reports obtained from database of the company CareFusion were used. Afterwards the system was validated on 455 patients by physicians from the Clinical Centre University of Sarajevo.|Out of 170 patients with asthma, 99.41% of patients were correctly classified. In addition, 99.19% of the 248 COPD patients were correctly classified. The system was 100% successful on 37 patients with normal lung function. Sensitivity of 99.28% and specificity of 100% in asthma and COPD classification were obtained.|Our neuro-fuzzy system for classification of asthma and COPD uses a combination of spirometry and Impulse Oscillometry System (IOS) test results, which in the very beginning enables more accurate classification. Additionally, using bronchodilatation and bronhoprovocation tests we get a complete patient's dynamic assessment, as opposed to the solution that provides a static assessment of the patient.\",\n",
              "  'authors': ['Badnjevic, Almir',\n",
              "   'Cifrek, Mario',\n",
              "   'Koruga, Dragan',\n",
              "   'Osmankovic, Dinko'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Asthma',\n",
              "   'Female',\n",
              "   'Fuzzy Logic',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oscillometry',\n",
              "   'Pulmonary Disease, Chronic Obstructive',\n",
              "   'Spirometry',\n",
              "   'Young Adult'],\n",
              "  'pmid': '26391218',\n",
              "  'title': 'Neuro-fuzzy classification of asthma and chronic obstructive pulmonary disease.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1471-2342',\n",
              "  'IF_2013': '0.983',\n",
              "  'IF_2014': '1.312',\n",
              "  'IF_2015': '1.663',\n",
              "  'IF_2016': '1.06',\n",
              "  'IF_2017': '1.635',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2342',\n",
              "  'IsoAbbr': 'BMC Med Imaging',\n",
              "  'JrId': 22333,\n",
              "  'MedAbbr': 'BMC Med Imaging',\n",
              "  'NLMid': '100968553',\n",
              "  'Titre': 'BMC Med Imaging',\n",
              "  'abstract': 'A bone scan is a common method for monitoring bone metastases in patients with advanced prostate cancer. The Bone Scan Index (BSI) measures the tumor burden on the skeleton, expressed as a percentage of the total skeletal mass. Previous studies have shown that BSI is associated with survival of prostate cancer patients. The objective in this study was to investigate to what extent regional BSI measurements, as obtained by an automated method, can improve the survival analysis for advanced prostate cancer.|The automated method for analyzing bone scan images computed BSI values for twelve skeletal regions, in a study population consisting of 1013 patients diagnosed with prostate cancer. In the survival analysis we used the standard Cox proportional hazards model and a more advanced non-linear method based on artificial neural networks. The concordance index (C-index) was used to measure the performance of the models.|A Cox model with age and total BSI obtained a C-index of 70.4%. The best Cox model with regional measurements from Costae, Pelvis, Scapula and the Spine, together with age, got a similar C-index (70.5%). The overall best single skeletal localisation, as measured by the C-index, was Costae. The non-linear model performed equally well as the Cox model, ruling out any significant non-linear interactions among the regional BSI measurements.|The present study showed that the localisation of bone metastases obtained from the bone scans in prostate cancer patients does not improve the performance of the survival models compared to models using the total BSI. However a ranking procedure indicated that some regions are more important than others.',\n",
              "  'authors': ['Kalderstam, Jonas',\n",
              "   'Sadik, May',\n",
              "   'Edenbrandt, Lars',\n",
              "   'Ohlsson, Mattias'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'BMC medical imaging',\n",
              "  'keywords': ['Aged',\n",
              "   'Bone Neoplasms',\n",
              "   'Bone and Bones',\n",
              "   'Disease Progression',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Medical Records Systems, Computerized',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Proportional Hazards Models',\n",
              "   'Prostatic Neoplasms'],\n",
              "  'pmid': '25012268',\n",
              "  'title': 'Analysis of regional bone scan index measurements for the survival of patients with prostate cancer.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1799-7267',\n",
              "  'IF_2013': '1.283',\n",
              "  'IF_2014': '1.258',\n",
              "  'IF_2015': '1.924',\n",
              "  'IF_2016': '2.197',\n",
              "  'IF_2017': '1.925',\n",
              "  'ISSN': '1457-4969',\n",
              "  'ISSN_online': '1799-7267',\n",
              "  'ISSN_print': '1457-4969',\n",
              "  'IsoAbbr': 'Scand J Surg',\n",
              "  'JrId': 29732,\n",
              "  'MedAbbr': 'Scand J Surg',\n",
              "  'NLMid': '101144297',\n",
              "  'Titre': 'Scand J Surg',\n",
              "  'abstract': 'Minimally invasive Heller myotomy has evolved the \"gold standard\" procedure for achalasia in the spectrum of current treatment options. The laparoscopic technique has proved superior to the thoracoscopic approach due to improved visualization of the esophagogastric junction. Operative controversies most recently include the length of the myotomy, especially of its fun-dic part, with respect to the balance between postoperative persistent dysphagia and development of gastroesophageal reflux, as well as the type of the added antireflux procedure. Peri-operative mortality should approach 0%, and favorable long-term results can be achieved in > 90%.',\n",
              "  'authors': ['Gockel, I', 'Sgourakis, G', 'Drescher, D G', 'Lang, H'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Scandinavian journal of surgery : SJS : official organ for the Finnish Surgical Society and the Scandinavian Surgical Society',\n",
              "  'keywords': ['Anti-Dyskinesia Agents',\n",
              "   'Botulinum Toxins',\n",
              "   'Catheterization',\n",
              "   'Esophageal Achalasia',\n",
              "   'Esophagoscopy',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Laparoscopy',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Postoperative Complications',\n",
              "   'Quality of Life',\n",
              "   'Robotics',\n",
              "   'Thoracoscopy'],\n",
              "  'pmid': '21737381',\n",
              "  'title': 'Impact of minimally invasive surgery in the spectrum of current achalasia treatment options.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-3792',\n",
              "  'IF_2013': '3.753',\n",
              "  'IF_2014': '4.471',\n",
              "  'IF_2015': '4.7',\n",
              "  'IF_2016': '5.157',\n",
              "  'IF_2017': '5.381',\n",
              "  'ISSN': '0022-5347',\n",
              "  'ISSN_online': '1527-3792',\n",
              "  'ISSN_print': '0022-5347',\n",
              "  'IsoAbbr': 'J. Urol.',\n",
              "  'JrId': 5331,\n",
              "  'MedAbbr': 'J Urol',\n",
              "  'NLMid': '0376374',\n",
              "  'Titre': 'J. Urol.',\n",
              "  'abstract': 'We developed an artificial neural network (ANN) for predicting spermatozoa prior to testicular biopsy in men with nonobstructive azoospermia. The performance of this ANN was compared to that of the standard logistic regression (LR) model.|Data were retrospectively collected from the physical examination and laboratory records of 303 patients who presented with infertility due to nonobstructive azoospermia. Input factors were patient age, duration of infertility, serum follicle-stimulating hormone, luteinizing hormone, total testosterone and prolactin, and left and right testicular volume. The ANN and LR models were constructed based on data on cases in which spermatozoa were and were not detected on testicular sperm extraction. The ANN was trained and validated with the data in the training (200 cases) and validation (30 cases) sets, and the model was then used to predict findings in the test set (73 cases). The LR model was constructed using the same data on the 230 training and validation cases. The same 73 patients served as the test set.|The sensitivity of the ANN model was significantly higher than that of the LR model (68% vs 28%, p < 0.0001). The neural network correctly predicted the outcome in 59 of the 73 test set patients (80.8%), whereas LR correctly predicted the outcome in 48 (65.7%, p = 0.07).|This ANN model, which is based on age, duration of infertility, serum hormone levels and testicular volumes, has clinically acceptable sensitivity. It may be of value for predicting spermatozoa in men with nonobstructive azoospermia.',\n",
              "  'authors': ['Samli, M Murat', 'Dogan, Ismet'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'The Journal of urology',\n",
              "  'keywords': ['Adult',\n",
              "   'Humans',\n",
              "   'Logistic Models',\n",
              "   'Male',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oligospermia',\n",
              "   'Retrospective Studies',\n",
              "   'Spermatozoa',\n",
              "   'Testis'],\n",
              "  'pmid': '15126820',\n",
              "  'title': 'An artificial neural network for predicting the presence of spermatozoa in the testes of men with nonobstructive azoospermia.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['UI'],\n",
              "  'ESSN': '1079-7114',\n",
              "  'IF_2013': '7.728',\n",
              "  'IF_2014': '7.512',\n",
              "  'IF_2015': '7.645',\n",
              "  'IF_2016': '8.462',\n",
              "  'IF_2017': '8.839',\n",
              "  'ISSN': '0031-9007',\n",
              "  'ISSN_online': '1079-7114',\n",
              "  'ISSN_print': '0031-9007',\n",
              "  'IsoAbbr': 'Phys. Rev. Lett.',\n",
              "  'JrId': 21217,\n",
              "  'MedAbbr': 'Phys Rev Lett',\n",
              "  'NLMid': '0401141',\n",
              "  'Titre': 'Phys. Rev. Lett.',\n",
              "  'abstract': 'Reconstructing network connectivity from the collective dynamics of a system typically requires access to its complete continuous-time evolution, although these are often experimentally inaccessible. Here we propose a theory for revealing physical connectivity of networked systems only from the event time series their intrinsic collective dynamics generate. Representing the patterns of event timings in an event space spanned by interevent and cross-event intervals, we reveal which other units directly influence the interevent times of any given unit. For illustration, we linearize an event-space mapping constructed from the spiking patterns in model neural circuits to reveal the presence or absence of synapses between any pair of neurons, as well as whether the coupling acts in an inhibiting or activating (excitatory) manner. The proposed model-independent reconstruction theory is scalable to larger networks and may thus play an important role in the reconstruction of networks from biology to social science and engineering.',\n",
              "  'authors': ['Casadiego, Jose', 'Maoutsa, Dimitra', 'Timme, Marc'],\n",
              "  'categories': ['PHYSICS, MULTIDISCIPLINARY'],\n",
              "  'journal': 'Physical review letters',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30118266',\n",
              "  'title': 'Inferring Network Connectivity from Event Timing Patterns.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1558-3171',\n",
              "  'IF_2013': '1.932',\n",
              "  'IF_2014': '1.879',\n",
              "  'IF_2015': '2.068',\n",
              "  'IF_2016': '2.206',\n",
              "  'IF_2017': '1.947',\n",
              "  'ISSN': '0039-6109',\n",
              "  'ISSN_online': '1558-3171',\n",
              "  'ISSN_print': '0039-6109',\n",
              "  'IsoAbbr': 'Surg. Clin. North Am.',\n",
              "  'JrId': 7645,\n",
              "  'MedAbbr': 'Surg Clin North Am',\n",
              "  'NLMid': '0074243',\n",
              "  'Titre': 'Surg. Clin. North Am.',\n",
              "  'abstract': \"Although minimally invasive hepatic resection surgery has shown decreased morbidity in select patients, conventional laparoscopic liver resection has inherent limitations with reduced freedom of movement within the abdominal cavity and 2-dimensional view of the operative field. Robotic liver surgery allows surgeons to perform advanced procedures with a potential for improved precision and ergonomics as well as a 3-dimensional view of the surgical site. However, use of the robot entails a steep learning curve and additional equipment. The purpose of this article is to summarize the emerging field of robotic liver surgery and include the authors' early experience with these operations.\",\n",
              "  'authors': ['Idrees, Kamran', 'Bartlett, David L'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The Surgical clinics of North America',\n",
              "  'keywords': ['Anastomosis, Surgical',\n",
              "   'Hepatectomy',\n",
              "   'Humans',\n",
              "   'Jejunum',\n",
              "   'Liver',\n",
              "   'Liver Diseases',\n",
              "   'Robotics'],\n",
              "  'pmid': '20637946',\n",
              "  'title': 'Robotic liver surgery.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1435-2451',\n",
              "  'IF_2013': '2.16',\n",
              "  'IF_2014': '2.191',\n",
              "  'IF_2015': '2.149',\n",
              "  'IF_2016': '2.203',\n",
              "  'IF_2017': '2.296',\n",
              "  'ISSN': '1435-2443',\n",
              "  'ISSN_online': '1435-2451',\n",
              "  'ISSN_print': '1435-2443',\n",
              "  'IsoAbbr': 'Langenbecks Arch Surg',\n",
              "  'JrId': 21014,\n",
              "  'MedAbbr': 'Langenbecks Arch Surg',\n",
              "  'NLMid': '9808285',\n",
              "  'Titre': 'Langenbecks Arch Surg',\n",
              "  'abstract': 'This pilot study evaluated the role of the DaVinci operation robot for laparoscopic antireflux surgery.|A robot-assisted laparoscopic Toupet-fundoplication was performed on nine consecutive patients with severe gastroesophageal reflux disease using the DaVinci robot system. The operative procedure was performed in the same way as for the conventional laparoscopic procedure. Clinical assessment and endoscopic and manometric follow-up investigations were performed 6 months after surgery in six of the patients.|The mean robotic operative time was 173 min (120-235). A mean of 25 min (12-45) was required to establish the pneumoperitoneum, to set the trocars, and to place the robot arms. There were no intraoperative complications. Six months after surgery none of the patients suffered from reflux symptoms and none of the patients had acute esophagitis. Postoperatively one patient complained of mild transient dysphagia. However, persistent dysphagia was not found in any of the patients. One further patient complained of mild bloating. No other side effects occurred. Manometrically there was a significant improvement in the function of the lower esophageal sphincter.|The robot-assisted partial posterior fundoplication is a safe procedure and provides a high-quality three-dimensional camera image that is superior to that with the conventional laparoscopic device. The handling of the instruments is precise, and intracorporeal suturing and knot tying is much easier than without the robotic technique. The procedure allows for an accurate approximation of the hiatal crura and for precise construction of the fundic wrap. However, robotic surgery is expensive and the setup of the system is time consuming at present.',\n",
              "  'authors': ['Wykypiel, H',\n",
              "   'Wetscher, G J',\n",
              "   'Klaus, A',\n",
              "   'Schmid, T',\n",
              "   'Gadenstaetter, M',\n",
              "   'Bodner, J',\n",
              "   'Bodner, E'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': \"Langenbeck's archives of surgery\",\n",
              "  'keywords': ['Adult',\n",
              "   'Female',\n",
              "   'Fundoplication',\n",
              "   'Gastroesophageal Reflux',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Pilot Projects',\n",
              "   'Robotics',\n",
              "   'Statistics, Nonparametric',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '12607121',\n",
              "  'title': 'Robot-assisted laparoscopic partial posterior fundoplication with the DaVinci system: initial experiences and technical aspects.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['FY'],\n",
              "  'ESSN': '1365-2842',\n",
              "  'IF_2013': '1.934',\n",
              "  'IF_2014': '1.682',\n",
              "  'IF_2015': '1.926',\n",
              "  'IF_2016': '2.098',\n",
              "  'IF_2017': '2.051',\n",
              "  'ISSN': '0305-182X',\n",
              "  'ISSN_online': '1365-2842',\n",
              "  'ISSN_print': '0305-182X',\n",
              "  'IsoAbbr': 'J Oral Rehabil',\n",
              "  'JrId': 5092,\n",
              "  'MedAbbr': 'J Oral Rehabil',\n",
              "  'NLMid': '0433604',\n",
              "  'Titre': 'J Oral Rehabil',\n",
              "  'abstract': 'The purpose of this study was to review the current status of biomarkers used in oro-facial pain conditions. Specifically, we critically appraise their relative strengths and weaknesses for assessing mechanisms associated with the oro-facial pain conditions and interpret that information in the light of their current value for use in diagnosis. In the third section, we explore biomarkers through the perspective of ontological realism. We discuss ontological problems of biomarkers as currently widely conceptualised and implemented. This leads to recommendations for research practice aimed to a better understanding of the potential contribution that biomarkers might make to oro-facial pain diagnosis and thereby fulfil our goal for an expanded multidimensional framework for oro-facial pain conditions that would include a third axis. ',\n",
              "  'authors': ['Ceusters, W',\n",
              "   'Nasri-Heir, C',\n",
              "   'Alnaas, D',\n",
              "   'Cairns, B E',\n",
              "   'Michelotti, A',\n",
              "   'Ohrbach, R'],\n",
              "  'categories': ['DENTISTRY, ORAL SURGERY & MEDICINE'],\n",
              "  'journal': 'Journal of oral rehabilitation',\n",
              "  'keywords': ['Biological Ontologies',\n",
              "   'Biomarkers',\n",
              "   'Congresses as Topic',\n",
              "   'Consensus',\n",
              "   'Dental Research',\n",
              "   'Facial Pain',\n",
              "   'Humans',\n",
              "   'Pain Measurement',\n",
              "   'Temporomandibular Joint Disorders',\n",
              "   'Terminology as Topic'],\n",
              "  'pmid': '26200973',\n",
              "  'title': 'Perspectives on next steps in classification of oro-facial pain - Part 3: biomarkers of chronic oro-facial pain - from research to clinic.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['RZ'],\n",
              "  'ESSN': '1558-3481',\n",
              "  'IF_2013': '0.429',\n",
              "  'IF_2014': '0.488',\n",
              "  'IF_2015': '0.284',\n",
              "  'IF_2016': '0.391',\n",
              "  'IF_2017': '0.488',\n",
              "  'ISSN': '0899-5885',\n",
              "  'ISSN_online': '1558-3481',\n",
              "  'ISSN_print': '0899-5885',\n",
              "  'IsoAbbr': 'Crit Care Nurs Clin North Am',\n",
              "  'JrId': 1529,\n",
              "  'MedAbbr': 'Crit Care Nurs Clin North Am',\n",
              "  'NLMid': '8912620',\n",
              "  'Titre': 'Crit Care Nurs Clin North Am',\n",
              "  'abstract': \"Today's successes would not be possible without the foundation of yesterday's practitioners and patients. This article assists in the exploration of cardiac surgery procedures, provides a brief historical review of the significant changes in cardiothoracic surgery, and provides an overview of current and future methods of treatment for coronary revascularization and heart failure. It is difficult for one article to encompass all aspects of cardiothoracic surgery. This article highlights many of the transforming moments that have led us to where we are today and explores the current trends of cardiac surgery and possibilities for tomorrow.\",\n",
              "  'authors': ['Miga, K Cheli'],\n",
              "  'categories': ['NURSING'],\n",
              "  'journal': 'Critical care nursing clinics of North America',\n",
              "  'keywords': ['Atrial Fibrillation',\n",
              "   'Cardiac Surgical Procedures',\n",
              "   'Cardiopulmonary Bypass',\n",
              "   'Coronary Artery Bypass',\n",
              "   'Heart Failure',\n",
              "   'Humans',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Robotics'],\n",
              "  'pmid': '18022520',\n",
              "  'title': 'Trends in cardiac surgery: exploring the past and looking into the future.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1744-8387',\n",
              "  'IF_2013': '3.542',\n",
              "  'IF_2014': '2.896',\n",
              "  'IF_2015': '3.465',\n",
              "  'IF_2016': '3.849',\n",
              "  'IF_2017': '3.489',\n",
              "  'ISSN': '1478-9450',\n",
              "  'ISSN_online': '1744-8387',\n",
              "  'ISSN_print': '1478-9450',\n",
              "  'IsoAbbr': 'Expert Rev Proteomics',\n",
              "  'JrId': 32340,\n",
              "  'MedAbbr': 'Expert Rev Proteomics',\n",
              "  'NLMid': '101223548',\n",
              "  'Titre': 'Expert Rev Proteomics',\n",
              "  'abstract': \"Elucidation of the networks of physical (functional) interactions present in cells and tissues is fundamental for understanding the molecular organization of biological systems, the mechanistic basis of essential and disease-related processes, and for functional annotation of previously uncharacterized proteins (via guilt-by-association or -correlation). After a decade in the field, we felt it timely to document our own experiences in the systematic analysis of protein interaction networks. Areas covered: Researchers worldwide have contributed innovative experimental and computational approaches that have driven the rapidly evolving field of 'functional proteomics'. These include mass spectrometry-based methods to characterize macromolecular complexes on a global-scale and sophisticated data analysis tools - most notably machine learning - that allow for the generation of high-quality protein association maps. Expert commentary: Here, we recount some key lessons learned, with an emphasis on successful workflows, and challenges, arising from our own and other groups' ongoing efforts to generate, interpret and report proteome-scale interaction networks in increasingly diverse biological contexts.\",\n",
              "  'authors': ['Havugimana, Pierre C', 'Hu, Pingzhao', 'Emili, Andrew'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Expert review of proteomics',\n",
              "  'keywords': ['Animals',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Protein Interaction Mapping',\n",
              "   'Proteomics',\n",
              "   'Systems Biology'],\n",
              "  'pmid': '28918672',\n",
              "  'title': 'Protein complexes, big data, machine learning and integrative proteomics: lessons learned over a decade of systematic analysis of protein interaction networks.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1873-2690',\n",
              "  'IF_2013': '2.352',\n",
              "  'IF_2014': '2.756',\n",
              "  'IF_2015': '2.928',\n",
              "  'IF_2016': '2.724',\n",
              "  'IF_2017': '2.718',\n",
              "  'ISSN': '0981-9428',\n",
              "  'ISSN_online': '1873-2690',\n",
              "  'ISSN_print': '0981-9428',\n",
              "  'IsoAbbr': 'Plant Physiol. Biochem.',\n",
              "  'JrId': 8707,\n",
              "  'MedAbbr': 'Plant Physiol Biochem',\n",
              "  'NLMid': '9882449',\n",
              "  'Titre': 'Plant Physiol. Biochem.',\n",
              "  'abstract': 'Polyphenols are important secondary metabolites and bioactive compounds in plants. Light is a vital abiotic factor that greatly impacts the content of polyphenols in plants. In spite of their importance the mechanism of polyphenol regulation still remains unknown in tobacco. A phytochrome B homolog, NtPHYB1K326, was isolated from Nicotiana tabacum cv. K326 to investigate the role of light receptors in the regulation of polyphenol metabolism in tobacco leaves. Furthermore, role of NtPHYB1K326 in polyphenol metabolism was analyzed by over-expression and RNAi-silencing approaches. Consistent and complemented results indicated involvement of NtPHYB1K326 in the regulation of polyphenol metabolism in tobacco leaves. Moreover, high levels of NtPHYB1K326 transcripts favor the accumulation of chlorogenic acid and its isomers, the key polyphenol component in tobacco leaves. Transcriptome analysis was also carried out for exploring the regulation mechanism of NtPHYB1K326 in the polyphenol metabolism. Compared with WT, 1665 and 1421 differentially-expressed genes were found in NtPHYB1K326-GFP and NtPHYB1K326-RNAi transgenic lines, respectively. Among these, about 30 genes were related to phenylpropanoid pathway, which is predominantly involved in synthesis of polyphenols. Further evidences from quantitative RT-PCR confirmed that NtPHYB1K326 may control phenylpropanoid pathway through regulating the transcription of PAL4 (phenylalanine ammonialyase 4), 4CL1 (4-coumarate:coenzyme A ligase 1) and COMT (caffeic acid 3-O-methyltransferase) genes.',\n",
              "  'authors': ['Zhao, Jiehong',\n",
              "   'Han, Jie',\n",
              "   'Zhang, Jie',\n",
              "   'Li, Zhenhua',\n",
              "   'Yu, Jing',\n",
              "   'Yu, Shizhou',\n",
              "   'Guo, Yushuang',\n",
              "   'Fu, Yongfu',\n",
              "   'Zhang, Xiaomei'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Plant physiology and biochemistry : PPB',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Arabidopsis Proteins',\n",
              "   'Chlorogenic Acid',\n",
              "   'Coenzyme A Ligases',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Immunoblotting',\n",
              "   'Methyltransferases',\n",
              "   'Phenols',\n",
              "   'Phenylalanine Ammonia-Lyase',\n",
              "   'Phylogeny',\n",
              "   'Phytochrome B',\n",
              "   'Plant Leaves',\n",
              "   'Plant Proteins',\n",
              "   'Plants, Genetically Modified',\n",
              "   'RNA Interference',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
              "   'Sequence Homology, Amino Acid',\n",
              "   'Tobacco'],\n",
              "  'pmid': '27636822',\n",
              "  'title': 'NtPHYB1K326, a homologous gene of Arabidopsis PHYB, positively regulates the content of phenolic compounds in tobacco.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Facial nerve segmentation is of considerable importance for preoperative planning of cochlear implantation. However, it is strongly influenced by the relatively low resolution of the cone-beam computed tomography (CBCT) images used in clinical practice. In this paper, we propose a super-resolution classification method, which refines a given initial segmentation of the facial nerve to a subvoxel classification level from CBCT/CT images. The super-resolution classification method learns the mapping from low-resolution CBCT/CT images to high-resolution facial nerve label images, obtained from manual segmentation on micro-CT images. We present preliminary results on dataset, 15 ex vivo samples scanned including pairs of CBCT/CT scans and high-resolution micro-CT scans, with a leave-one-out evaluation, and manual segmentations on micro-CT images as ground truth. Our experiments achieved a segmentation accuracy with a Dice coefficient of , surface-to-surface distance of , and Hausdorff distance of . We compared the proposed technique to two other semi-automated segmentation software tools, ITK-SNAP and GeoS, and show the ability of the proposed approach to yield subvoxel levels of accuracy in delineating the facial nerve.',\n",
              "  'authors': ['Lu, Ping',\n",
              "   'Barazzetti, Livia',\n",
              "   'Chandran, Vimal',\n",
              "   'Gavaghan, Kate',\n",
              "   'Weber, Stefan',\n",
              "   'Gerber, Nicolas',\n",
              "   'Reyes, Mauricio'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Cone-Beam Computed Tomography',\n",
              "   'Databases, Factual',\n",
              "   'Facial Nerve',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Supervised Machine Learning',\n",
              "   'X-Ray Microtomography'],\n",
              "  'pmid': '28459680',\n",
              "  'title': 'Highly Accurate Facial Nerve Segmentation Refinement From CBCT/CT Imaging Using a Super-Resolution Classification Approach.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1432-2218',\n",
              "  'IF_2013': '3.313',\n",
              "  'IF_2014': '3.256',\n",
              "  'IF_2015': '3.54',\n",
              "  'IF_2016': '3.747',\n",
              "  'IF_2017': '3.117',\n",
              "  'ISSN': '0930-2794',\n",
              "  'ISSN_online': '1432-2218',\n",
              "  'ISSN_print': '0930-2794',\n",
              "  'IsoAbbr': 'Surg Endosc',\n",
              "  'JrId': 7653,\n",
              "  'MedAbbr': 'Surg Endosc',\n",
              "  'NLMid': '8806653',\n",
              "  'Titre': 'Surg Endosc',\n",
              "  'abstract': 'The recent development of single-port robotic surgery in gynecologic surgery has added the benefits of minimally invasive surgery and greater aesthetic patient satisfaction to the other benefits that robotic surgery offers. However, single-port robotic hysterectomy has raised the question about a better approach to closure of the vaginal cuff. This report describes the technical feasibility of using robotic intracorporeal needles and extracorporeal knots to perform the vaginal cuff closure for selected patients. ',\n",
              "  'authors': ['Bogliolo, Stefano',\n",
              "   'Cassani, Chiara',\n",
              "   'Babilonti, Luciana',\n",
              "   'Spinillo, Arsenio'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Surgical endoscopy',\n",
              "  'keywords': ['Female',\n",
              "   'Humans',\n",
              "   'Hysterectomy',\n",
              "   'Laparoscopy',\n",
              "   'Learning Curve',\n",
              "   'Robotics'],\n",
              "  'pmid': '23943119',\n",
              "  'title': 'Vaginal cuff closure during robotic single-port hysterectomy: is the vaginal route always the best one?',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1527-9995',\n",
              "  'IF_2013': '2.132',\n",
              "  'IF_2014': '2.188',\n",
              "  'IF_2015': '2.187',\n",
              "  'IF_2016': '2.309',\n",
              "  'IF_2017': '2.3',\n",
              "  'ISSN': '0090-4295',\n",
              "  'ISSN_online': '1527-9995',\n",
              "  'ISSN_print': '0090-4295',\n",
              "  'IsoAbbr': 'Urology',\n",
              "  'JrId': 7907,\n",
              "  'MedAbbr': 'Urology',\n",
              "  'NLMid': '0366151',\n",
              "  'Titre': 'Urology',\n",
              "  'abstract': \"To investigate the renoprotective effect of nicardipine in patients undergoing robot-assisted laparoscopic radical prostatectomy (RALRP) in a prospective trial. Superior visualization of the increasingly performed RALRP requires pneumoperitoneum and extreme head-down tilt, both of which are associated with a decrease in postoperative renal function. Nicardipine causes preferential dilation of the renal arterioles and attenuates renal dysfunction after cardiac surgery.|After we obtained institutional review board approval, we randomly treated 100 patients undergoing RALRP with a continuous infusion of nicardipine at a rate of 0.5 microg/kg/min (nicardipine group, n = 50) or with normal saline (control group, n = 50) during surgery. We measured the serum creatinine (Cr) level and estimated glomerular filtration rate (eGFR) 1 day before surgery and the first and third postoperative days (POD 1 and 3, respectively).|Patients' characteristics and operative data were similar between groups. The serum Cr was significantly higher and the eGFR was significantly lower in the control group at POD 1. The number of patients having renal insufficiency (eGFR < 60 mL/min/1.73 m(2)) and abnormal serum creatinine level (>1.4 mg/dL) was significantly greater in the control group (9 vs 1, and 4 vs none, respectively) at POD 1.|Continuous infusion of low-dose nicardipine during RALRP seems to offset the deleterious effects of inevitable pneumoperitoneum and extreme head-down tilt on renal function in preserving the eGFR and attenuating the development of renal insufficiency in the immediate postoperative period.\",\n",
              "  'authors': ['Cho, Jang Eun',\n",
              "   'Shim, Jae Kwang',\n",
              "   'Chang, Jae Hyun',\n",
              "   'Oh, Young Jun',\n",
              "   'Kil, Hae Keum',\n",
              "   'Rha, Koon Ho',\n",
              "   'Kwak, Young Lan'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Urology',\n",
              "  'keywords': ['Aged',\n",
              "   'Creatinine',\n",
              "   'Follow-Up Studies',\n",
              "   'Glomerular Filtration Rate',\n",
              "   'Humans',\n",
              "   'Infusions, Intravenous',\n",
              "   'Kidney Function Tests',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Nicardipine',\n",
              "   'Perioperative Care',\n",
              "   'Pneumoperitoneum, Artificial',\n",
              "   'Postoperative Complications',\n",
              "   'Probability',\n",
              "   'Prospective Studies',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Reference Values',\n",
              "   'Renal Insufficiency',\n",
              "   'Risk Assessment',\n",
              "   'Robotics',\n",
              "   'Statistics, Nonparametric',\n",
              "   'Treatment Outcome',\n",
              "   'Vasodilator Agents'],\n",
              "  'pmid': '19394503',\n",
              "  'title': 'Effect of nicardipine on renal function after robot-assisted laparoscopic radical prostatectomy.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['KM'],\n",
              "  'ESSN': '1438-7948',\n",
              "  'IF_2013': '2.691',\n",
              "  'IF_2014': '2.479',\n",
              "  'IF_2015': '2.265',\n",
              "  'IF_2016': '3.496',\n",
              "  'IF_2017': '3.889',\n",
              "  'ISSN': '1438-793X',\n",
              "  'ISSN_online': '1438-7948',\n",
              "  'ISSN_print': '1438-793X',\n",
              "  'IsoAbbr': 'Funct. Integr. Genomics',\n",
              "  'JrId': 21734,\n",
              "  'MedAbbr': 'Funct Integr Genomics',\n",
              "  'NLMid': '100939343',\n",
              "  'Titre': 'Funct. Integr. Genomics',\n",
              "  'abstract': 'Microsporogenesis is an indispensable period for investigating microspore development and cytoplasmic male sterility (CMS) occurrence. Radish CMS line plays a critical role in elite F1 hybrid seed production and heterosis utilization. However, the molecular mechanisms of microspore development and CMS occurrence have not been thoroughly uncovered in radish. In this study, a comparative analysis of radish floral buds from a CMS line (NAU-WA) and its maintainer (NAU-WB) was conducted using next generation sequencing (NGS) technology. Digital gene expression (DGE) profiling revealed that 3504 genes were significantly differentially expressed between NAU-WA and NAU-WB library, among which 1910 were upregulated and 1594 were downregulated. Gene ontology (GO) analysis showed that these differentially expressed genes (DEGs) were mainly enriched in extracellular region, catalytic activity, and response to stimulus. KEGG enrichment analysis revealed that the DEGs were predominantly associated with flavonoid biosynthesis, glycolysis, and biosynthesis of secondary metabolites. Real-time quantitative PCR analysis showed that the expression profiles of 13 randomly selected DEGs were in high agreement with results from Illumina sequencing. Several candidate genes encoding ATP synthase, auxin response factor (ARF), transcription factors (TFs), chalcone synthase (CHS), and male sterility (MS) were responsible for microsporogenesis. Furthermore, a schematic diagram for functional interaction of DEGs from NAU-WA vs. NAU-WB library in radish plants was proposed. These results could provide new information on the dissection of the molecular mechanisms underlying microspore development and CMS occurrence in radish. ',\n",
              "  'authors': ['Xie, Yang',\n",
              "   'Zhang, Wei',\n",
              "   'Wang, Yan',\n",
              "   'Xu, Liang',\n",
              "   'Zhu, Xianwen',\n",
              "   'Muleke, Everlyne M',\n",
              "   'Liu, Liwang'],\n",
              "  'categories': ['GENETICS & HEREDITY'],\n",
              "  'journal': 'Functional & integrative genomics',\n",
              "  'keywords': ['Cytoplasm',\n",
              "   'Flowers',\n",
              "   'Gametogenesis, Plant',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Metabolic Networks and Pathways',\n",
              "   'Plant Infertility',\n",
              "   'Plant Proteins',\n",
              "   'Pollen',\n",
              "   'Raphanus',\n",
              "   'Seeds',\n",
              "   'Spores',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27465294',\n",
              "  'title': 'Comprehensive transcriptome-based characterization of differentially expressed genes involved in microsporogenesis of radish CMS line and its maintainer.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1638-6183',\n",
              "  'IF_2013': '3.123',\n",
              "  'IF_2014': '2.963',\n",
              "  'IF_2015': '2.474',\n",
              "  'IF_2016': '3.112',\n",
              "  'IF_2017': '3.188',\n",
              "  'ISSN': '0300-9084',\n",
              "  'ISSN_online': '1638-6183',\n",
              "  'ISSN_print': '0300-9084',\n",
              "  'IsoAbbr': 'Biochimie',\n",
              "  'JrId': 1043,\n",
              "  'MedAbbr': 'Biochimie',\n",
              "  'NLMid': '1264604',\n",
              "  'Titre': 'Biochimie',\n",
              "  'abstract': 'Information on the subcellular localization of bacterial proteins is essential for protein function prediction, genome annotation and drug design. Here we proposed a novel approach to predict the subcellular localization of bacterial proteins by fusing features from position-specific score matrix (PSSM), Gene Ontology (GO) and PROFEAT. A backward feature selection approach by linear kennel of SVM was then used to rank the integrated feature vectors and extract optimal features. Finally, SVM was applied for predicting protein subcellular locations based on these optimal features. To validate the performance of our method, we employed jackknife cross-validation tests on three low similarity datasets, i.e., M638, Gneg1456 and Gpos523. The overall accuracies of 94.98%, 93.21%, and 94.57% were achieved for these three datasets, which are higher (from 1.8% to 10.9%) than those by state-of-the-art tools. Comparison results suggest that our method could serve as a very useful vehicle for expediting the prediction of bacterial protein subcellular localization. ',\n",
              "  'authors': ['Li, Liqi',\n",
              "   'Yu, Sanjiu',\n",
              "   'Xiao, Weidong',\n",
              "   'Li, Yongsheng',\n",
              "   'Li, Maolin',\n",
              "   'Huang, Lan',\n",
              "   'Zheng, Xiaoqi',\n",
              "   'Zhou, Shiwen',\n",
              "   'Yang, Hua'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Biochimie',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Bacterial Proteins',\n",
              "   'Computational Biology',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Intestines',\n",
              "   'Intracellular Space',\n",
              "   'Protein Transport',\n",
              "   'Reproducibility of Results',\n",
              "   'Support Vector Machine',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '24929100',\n",
              "  'title': \"Prediction of bacterial protein subcellular localization by incorporating various features into Chou's PseAAC and a backward feature selection approach.\",\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1475-925X',\n",
              "  'IF_2013': '1.746',\n",
              "  'IF_2014': '1.427',\n",
              "  'IF_2015': '1.382',\n",
              "  'IF_2016': '1.683',\n",
              "  'IF_2017': '1.676',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1475-925X',\n",
              "  'IsoAbbr': 'Biomed Eng Online',\n",
              "  'JrId': 29824,\n",
              "  'MedAbbr': 'Biomed Eng Online',\n",
              "  'NLMid': '101147518',\n",
              "  'Titre': 'Biomed Eng Online',\n",
              "  'abstract': \"Androgen acts via androgen receptor (AR) and accurate measurement of the levels of AR protein expression is critical for prostate research. The expression of AR in paired specimens of benign prostate and prostate cancer from 20 African and 20 Caucasian Americans was compared to demonstrate an application of this system.|A set of 200 immunopositive and 200 immunonegative nuclei were collected from the images using a macro developed in Image Pro Plus. Linear Discriminant and Logistic Regression analyses were performed on the data to generate classification coefficients. Classification coefficients render the automated image analysis software independent of the type of immunostaining or image acquisition system used. The image analysis software performs local segmentation and uses nuclear shape and size to detect prostatic epithelial nuclei. AR expression is described by (a) percentage of immunopositive nuclei; (b) percentage of immunopositive nuclear area; and (c) intensity of AR expression among immunopositive nuclei or areas.|The percent positive nuclei and percent nuclear area were similar by race in both benign prostate hyperplasia and prostate cancer. In prostate cancer epithelial nuclei, African Americans exhibited 38% higher levels of AR immunostaining than Caucasian Americans (two sided Student's t-tests; P < 0.05). Intensity of AR immunostaining was similar between races in benign prostate.|The differences measured in the intensity of AR expression in prostate cancer were consistent with previous studies. Classification coefficients are required due to non-standardized immunostaining and image collection methods across medical institutions and research laboratories and helps customize the software for the specimen under study. The availability of a free, automated system creates new opportunities for testing, evaluation and use of this image analysis system by many research groups who study nuclear protein expression.\",\n",
              "  'authors': ['Singh, Swaroop S', 'Kim, Desok', 'Mohler, James L'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Biomedical engineering online',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biomarkers, Tumor',\n",
              "   'Cell Nucleus',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Programming Languages',\n",
              "   'Prostatic Hyperplasia',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Receptors, Androgen',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Software'],\n",
              "  'pmid': '15888205',\n",
              "  'title': 'Java Web Start based software for automated quantitative nuclear analysis of prostate cancer and benign prostate hyperplasia.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'FlyBase (http://flybase.org) is the leading website and database of Drosophila genes and genomes. Whether you are using the fruit fly Drosophila melanogaster as an experimental system or wish to understand Drosophila biological knowledge in relation to human disease or to other model systems, FlyBase can help you successfully find the information you are looking for. Here, we demonstrate some of our more advanced searching systems and highlight some of our new tools for searching the wealth of data on FlyBase. The first section explores gene function in FlyBase, using our TermLink tool to search with Controlled Vocabulary terms and our new RNA-Seq Search tool to search gene expression. The second section of this article describes a few ways to search genomic data in FlyBase, using our BLAST server and the new implementation of GBrowse 2, as well as our new FeatureMapper tool. Finally, we move on to discuss our most powerful search tool, QueryBuilder, before describing pre-computed cuts of the data and how to query the database programmatically. ',\n",
              "  'authors': ['St Pierre, Susan E',\n",
              "   'Ponting, Laura',\n",
              "   'Stefancsik, Raymund',\n",
              "   'McQuilton, Peter'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Animals',\n",
              "   'Databases, Genetic',\n",
              "   'Drosophila',\n",
              "   'Drosophila melanogaster',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Insect',\n",
              "   'Genome, Insect',\n",
              "   'Internet',\n",
              "   'Phenotype',\n",
              "   'Sequence Analysis, RNA'],\n",
              "  'pmid': '24234449',\n",
              "  'title': 'FlyBase 102--advanced approaches to interrogating FlyBase.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Adhesion constitutes one of the initial stages of infection in microbial diseases and is mediated by adhesins. Hence, identification and comprehensive knowledge of adhesins and adhesin-like proteins is essential to understand adhesin mediated pathogenesis and how to exploit its therapeutic potential. However, the knowledge about fungal adhesins is rudimentary compared to that of bacterial adhesins. In addition to host cell attachment and mating, the fungal adhesins play a significant role in homotypic and xenotypic aggregation, foraging and biofilm formation. Experimental identification of fungal adhesins is labor- as well as time-intensive. In this work, we present a Support Vector Machine (SVM) based method for the prediction of fungal adhesins and adhesin-like proteins. The SVM models were trained with different compositional features, namely, amino acid, dipeptide, multiplet fractions, charge and hydrophobic compositions, as well as PSI-BLAST derived PSSM matrices. The best classifiers are based on compositional properties as well as PSSM and yield an overall accuracy of 86%. The prediction method based on best classifiers is freely accessible as a world wide web based server at http://bioinfo.icgeb.res.in/faap. This work will aid rapid and rational identification of fungal adhesins, expedite the pace of experimental characterization of novel fungal adhesins and enhance our knowledge about role of adhesins in fungal infections.',\n",
              "  'authors': ['Ramana, Jayashree', 'Gupta, Dinesh'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biofilms',\n",
              "   'Cell Adhesion Molecules',\n",
              "   'Computational Biology',\n",
              "   'Computers',\n",
              "   'Databases, Protein',\n",
              "   'Fungal Proteins',\n",
              "   'Internet',\n",
              "   'Models, Statistical',\n",
              "   'Proteins',\n",
              "   'ROC Curve',\n",
              "   'Reproducibility of Results',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Software'],\n",
              "  'pmid': '20300572',\n",
              "  'title': 'FaaPred: a SVM-based prediction method for fungal adhesins and adhesin-like proteins.',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['HY'],\n",
              "  'ESSN': '1095-564X',\n",
              "  'IF_2013': '3.637',\n",
              "  'IF_2014': '3.547',\n",
              "  'IF_2015': '3.155',\n",
              "  'IF_2016': '2.944',\n",
              "  'IF_2017': '3.262',\n",
              "  'ISSN': '0012-1606',\n",
              "  'ISSN_online': '1095-564X',\n",
              "  'ISSN_print': '0012-1606',\n",
              "  'IsoAbbr': 'Dev. Biol.',\n",
              "  'JrId': 3389,\n",
              "  'MedAbbr': 'Dev Biol',\n",
              "  'NLMid': '0372762',\n",
              "  'Titre': 'Dev. Biol.',\n",
              "  'abstract': 'The axolotl (Ambystoma mexicanum) has long been the subject of biological research, primarily owing to its outstanding regenerative capabilities. However, the gene expression programs governing its embryonic development are particularly underexplored, especially when compared to other amphibian model species. Therefore, we performed whole transcriptome polyA+ RNA sequencing experiments on 17 stages of embryonic development. As the axolotl genome is unsequenced and its gene annotation is incomplete, we built de novo transcriptome assemblies for each stage and garnered functional annotation by comparing expressed contigs with known genes in other organisms. In evaluating the number of differentially expressed genes over time, we identify three waves of substantial transcriptome upheaval each followed by a period of relative transcriptome stability. The first wave of upheaval is between the one and two cell stage. We show that the number of differentially expressed genes per unit time is higher between the one and two cell stage than it is across the mid-blastula transition (MBT), the period of zygotic genome activation. We use total RNA sequencing to demonstrate that the vast majority of genes with increasing polyA+ signal between the one and two cell stage result from polyadenylation rather than de novo transcription. The first stable phase begins after the two cell stage and continues until the mid-blastula transition, corresponding with the pre-MBT phase of transcriptional quiescence in amphibian development. Following this is a peak of differential gene expression corresponding with the activation of the zygotic genome and a phase of transcriptomic stability from stages 9-11. We observe a third wave of transcriptomic change between stages 11 and 14, followed by a final stable period. The last two stable phases have not been documented in amphibians previously and correspond to times of major morphogenic change in the axolotl embryo: gastrulation and neurulation. These results yield new insights into global gene expression during early stages of amphibian embryogenesis and will help to further develop the axolotl as a model species for developmental and regenerative biology.',\n",
              "  'authors': ['Jiang, Peng',\n",
              "   'Nelson, Jeffrey D',\n",
              "   'Leng, Ning',\n",
              "   'Collins, Michael',\n",
              "   'Swanson, Scott',\n",
              "   'Dewey, Colin N',\n",
              "   'Thomson, James A',\n",
              "   'Stewart, Ron'],\n",
              "  'categories': ['DEVELOPMENTAL BIOLOGY'],\n",
              "  'journal': 'Developmental biology',\n",
              "  'keywords': ['Ambystoma mexicanum',\n",
              "   'Animals',\n",
              "   'Contig Mapping',\n",
              "   'Embryonic Development',\n",
              "   'Gene Expression Regulation, Developmental',\n",
              "   'Gene Ontology',\n",
              "   'Morphogenesis',\n",
              "   'Multigene Family',\n",
              "   'RNA, Messenger',\n",
              "   'Sequence Analysis, RNA',\n",
              "   'Transcription, Genetic',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '27475628',\n",
              "  'title': 'Analysis of embryonic development in the unsequenced axolotl: Waves of transcriptomic upheaval and stability.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1558-2531',\n",
              "  'IF_2013': '2.233',\n",
              "  'IF_2014': '2.347',\n",
              "  'IF_2015': '2.468',\n",
              "  'IF_2016': '3.577',\n",
              "  'IF_2017': '4.288',\n",
              "  'ISSN': '0018-9294',\n",
              "  'ISSN_online': '1558-2531',\n",
              "  'ISSN_print': '0018-9294',\n",
              "  'IsoAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'JrId': 4157,\n",
              "  'MedAbbr': 'IEEE Trans Biomed Eng',\n",
              "  'NLMid': '0012737',\n",
              "  'Titre': 'IEEE Trans Biomed Eng',\n",
              "  'abstract': 'Information processing in the nervous system is based on parallel computation, adaptation and learning. These features cannot be easily implemented on conventional silicon devices. In order to obtain a better insight of how neurons process information, we have explored the possibility of using biological neurons as parallel and adaptable computing elements for image processing and pattern recognition. Commercially available multielectrode arrays (MEAs) were used to record and stimulate the electrical activity from neuronal cultures. By mapping digital images, i.e., arrays of pixels, into the stimulation of neuronal cultures, a low and bandpass filtering of images could be quickly and easily obtained. Responses to specific spatial patterns of stimulation were potentiated by an appropriate training (tetanization). Learning allowed pattern recognition and extraction of spatial features in processed images. Therefore, neurocomputers, (i.e., hybrid devices containing man-made elements and natural neurons) seem feasible and may become a new generation of computing devices, to be developed by a synergy of Neuroscience and Material Science.',\n",
              "  'authors': ['Ruaro, Maria Elisabetta', 'Bonifazi, Paolo', 'Torre, Vincent'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'IEEE transactions on bio-medical engineering',\n",
              "  'keywords': ['Animals',\n",
              "   'Animals, Newborn',\n",
              "   'Artificial Intelligence',\n",
              "   'Bionics',\n",
              "   'Cell Culture Techniques',\n",
              "   'Cells, Cultured',\n",
              "   'Computing Methodologies',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Microelectrodes',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Rats',\n",
              "   'Rats, Wistar',\n",
              "   'Signal Processing, Computer-Assisted'],\n",
              "  'pmid': '15759567',\n",
              "  'title': 'Toward the neurocomputer: image processing and pattern recognition with neuronal cultures.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1433-0385',\n",
              "  'IF_2013': '0.516',\n",
              "  'IF_2014': '0.574',\n",
              "  'IF_2015': '0.63',\n",
              "  'IF_2016': '0.646',\n",
              "  'IF_2017': '0.669',\n",
              "  'ISSN': '0009-4722',\n",
              "  'ISSN_online': '1433-0385',\n",
              "  'ISSN_print': '0009-4722',\n",
              "  'IsoAbbr': 'Chirurg',\n",
              "  'JrId': 2936,\n",
              "  'MedAbbr': 'Chirurg',\n",
              "  'NLMid': '16140410R',\n",
              "  'Titre': 'Chirurg',\n",
              "  'abstract': 'The latest technical developments of minimally invasive thoracic surgery are characterized by robotic-assisted operative procedures. Robotic-assisted thymectomy is the most advanced method in this field.|A systematic literature search (PubMed, Medline) was carried out and the databank system of Intuitive Surgical (Sunnyvale, CA) was analysed. Target criteria were the analysis of the quantitative data over time, technical advantages and limiting factors of robotic-assisted thoracic surgery.|The da Vinci robotic system has been used in thoracic surgery since 2001, and up to 2012 a total of 10,895 robotic-assisted lobotomies have been carried out worldwide. A total of 12 ectopic parathyroid glands in the mediastinum were resected and published. Furthermore, more than 3,500 cases of robotic-assisted thymectomy were performed. A rapid increase in the number of operations has occurred particularly for thymectomy and lung resections.|Acceptance of robotic-assisted thymectomy for myasthenia and/or thymoma and mediastinal tumors is growing rapidly. For anatomic lung resection in lung cancer, robotic-assisted hilar and lymph node dissection due to this new quality are also comparable to open surgical techniques. The principles form the intrinsic technical advantages of the da Vinci robotic system.',\n",
              "  'authors': ['Ismail, M', 'Swierzy, M', 'Ulrich, M', 'Rückert, J C'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Der Chirurg; Zeitschrift fur alle Gebiete der operativen Medizen',\n",
              "  'keywords': ['Choristoma',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Mediastinal Diseases',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Parathyroid Glands',\n",
              "   'Parathyroidectomy',\n",
              "   'Pneumonectomy',\n",
              "   'Robotics',\n",
              "   'Thoracic Surgical Procedures',\n",
              "   'Thymectomy'],\n",
              "  'pmid': '23942960',\n",
              "  'title': '[Application of the da Vinci robotic system in thoracic surgery].',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['UM'],\n",
              "  'ESSN': '1365-2281',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0144-5979',\n",
              "  'ISSN_online': '1365-2281',\n",
              "  'ISSN_print': '0144-5979',\n",
              "  'IsoAbbr': 'Clin Physiol',\n",
              "  'JrId': 3080,\n",
              "  'MedAbbr': 'Clin Physiol',\n",
              "  'NLMid': '8309768',\n",
              "  'Titre': 'Clin Physiol',\n",
              "  'abstract': 'Artificial neural networks are computer programs that learn from examples. They have been successfully used to detect coronary artery disease from myocardial perfusion images. The purpose of the present study was to develop neural networks that could classify myocardial scintigrams regarding reversibility, localization, severity and extent of perfusion defects. Rest/exercise technetium-99m sestamibi scintigrams from 338 patients were studied. The classifications of two experts were employed as the gold standard. Artificial neural networks were trained to classify both reversible (ischaemia) and non-reversible (infarct) defects in three vascular territories, corresponding to the main coronary arteries. The extent (small or large) and severity (mild or severe) of the defects were described by the networks. After the training process, separate test sets were used to compare the neural networks with one of the experts who reclassified the scintigrams two months later. The neural networks made correct classifications in 71% of the test cases and the human expert in 70% (P=0.10). It was concluded that artificial neural networks can be trained to make clinical interpretations of myocardial perfusion scintigrams. The results indicate that networks can assist physicians in achieving correct interpretations and thereby improve the diagnostic accuracy of medical imaging.',\n",
              "  'authors': ['Lindahl, D', 'Palmer, J', 'Edenbrandt, L'],\n",
              "  'categories': ['PHYSIOLOGY'],\n",
              "  'journal': 'Clinical physiology (Oxford, England)',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Coronary Disease',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Diagnostic Errors',\n",
              "   'Female',\n",
              "   'Heart',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Tomography, Emission-Computed, Single-Photon'],\n",
              "  'pmid': '10583343',\n",
              "  'title': 'Myocardial SPET: artificial neural networks describe extent and severity of perfusion defects.',\n",
              "  'year': '1999'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1862-5274',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0288-2043',\n",
              "  'ISSN_online': '1862-5274',\n",
              "  'ISSN_print': '0288-2043',\n",
              "  'IsoAbbr': 'Radiat Med',\n",
              "  'JrId': 6943,\n",
              "  'MedAbbr': 'Radiat Med',\n",
              "  'NLMid': '8412264',\n",
              "  'Titre': 'Radiat Med',\n",
              "  'abstract': 'In a joint project between the Department of Computer and Information Sciences and the Department of Radiology, we are applying techniques of artificial intelligence to improve clinical performance in coronary arteries. Specifically, we are investigating how images from intravenous digital subtraction angiography (DSA) can be enhanced so that their efficacy for lesion detection and quantitation becomes comparable with that of the more dangerous procedure of selective coronary arteriography. The enhancement techniques (which include algorithms for 3-dimensional vessel detection, reconstruction and display, as well as for accurate lumen-size estimation) are based on models of (i) the 3-dimensional topological structure of the coronary arterial tree, (ii) myocardial dynamics, and (iii) the X-ray imaging process involved in producing digital subtraction angiograms. The evaluation of these model-driven visualization techniques is done by the standard psychophysical method of Receiver Operating Characteristic (ROC) analysis applied to observer performance tests on images from an animal coronary atherosclerosis model.',\n",
              "  'authors': ['Herman, G T',\n",
              "   'Axel, L',\n",
              "   'Bajcsy, R',\n",
              "   'Kundel, H',\n",
              "   'LeVeen, R',\n",
              "   'Udupa, J K',\n",
              "   'Wolf, G'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Radiation medicine',\n",
              "  'keywords': ['Analog-Digital Conversion',\n",
              "   'Angiography',\n",
              "   'Animals',\n",
              "   'Computers',\n",
              "   'Coronary Angiography',\n",
              "   'Coronary Disease',\n",
              "   'Data Display',\n",
              "   'Humans',\n",
              "   'Models, Biological',\n",
              "   'Psychophysics'],\n",
              "  'pmid': '6679903',\n",
              "  'title': 'Model-driven visualization of coronary arteries.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1460-2199',\n",
              "  'IF_2013': '8.305',\n",
              "  'IF_2014': '8.665',\n",
              "  'IF_2015': '8.285',\n",
              "  'IF_2016': '6.559',\n",
              "  'IF_2017': '6.308',\n",
              "  'ISSN': '1047-3211',\n",
              "  'ISSN_online': '1460-2199',\n",
              "  'ISSN_print': '1047-3211',\n",
              "  'IsoAbbr': 'Cereb. Cortex',\n",
              "  'JrId': 2171,\n",
              "  'MedAbbr': 'Cereb Cortex',\n",
              "  'NLMid': '9110718',\n",
              "  'Titre': 'Cereb. Cortex',\n",
              "  'abstract': 'Subfield analysis of the receptive fields (RFs) of parafoveal V4 complex cells demonstrates directly that most RFs are tiled by overlapping second-order excitatory inputs that for any given V4 cell are predominantly selective to the same preferred values of spatial frequency and orientation. These results extend hierarchical principles of RF organization in the spatial, orientation and spatial frequency domains, first recognized in V1, to an intermediate extrastriate cortex. Spatial interaction studies across subfields demonstrate that the responses of V4 neurons to paired stimuli may either decrease or increase as a function of inter-stimulus distance across the width axis. These intra-RF suppressions and facilitations vary independently in magnitude and spatial extent from cell to cell. These results taken together with the relatively large RF sizes of V4 neurons - as compared with RF sizes of their afferent inputs - lead us to hypothesize a novel property, namely that classes of stimulus configurations that enhance areal summation while reducing suppressive interactions between excitatory inputs will evoke especially robust responses. We tested, and found support for, this hypothesis by presenting stimuli consisting of optimally tuned sine-wave gratings visible only within an annular region and found that such stimuli vigorously activate V4 neurons at firing rates far higher than those evoked by comparable stimuli to either the full-field or central core. On the basis of these results we propose a framework for a new class of neural network models for the spatial RF organizations of prototypic V4 neurons.',\n",
              "  'authors': ['Pollen, Daniel A',\n",
              "   'Przybyszewski, Andrzej W',\n",
              "   'Rubin, Mark A',\n",
              "   'Foote, Warren'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Cerebral cortex (New York, N.Y. : 1991)',\n",
              "  'keywords': ['Animals',\n",
              "   'Brain Mapping',\n",
              "   'Electrophysiology',\n",
              "   'Macaca fascicularis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Orientation',\n",
              "   'Photic Stimulation',\n",
              "   'Space Perception',\n",
              "   'Visual Cortex',\n",
              "   'Visual Fields'],\n",
              "  'pmid': '12003860',\n",
              "  'title': 'Spatial receptive field organization of macaque V4 neurons.',\n",
              "  'year': '2002'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['KI'],\n",
              "  'ESSN': '1473-5687',\n",
              "  'IF_2013': '2.152',\n",
              "  'IF_2014': '2.253',\n",
              "  'IF_2015': '2.093',\n",
              "  'IF_2016': '1.968',\n",
              "  'IF_2017': '2.014',\n",
              "  'ISSN': '0954-691X',\n",
              "  'ISSN_online': '1473-5687',\n",
              "  'ISSN_print': '0954-691X',\n",
              "  'IsoAbbr': 'Eur J Gastroenterol Hepatol',\n",
              "  'JrId': 8589,\n",
              "  'MedAbbr': 'Eur J Gastroenterol Hepatol',\n",
              "  'NLMid': '9000874',\n",
              "  'Titre': 'Eur J Gastroenterol Hepatol',\n",
              "  'abstract': 'Acute lower gastrointestinal hemorrhage (LGIH) has traditionally been defined as bleeding that occurs distal to the ligament of Treitz. More recently, however, it has been subdivided into mid-intestinal (small bowel) hemorrhage and bleeding that originates from the colon. Acute LGIH has diverse etiologies, is a frequent cause of hospital admission, and is associated with significant patient morbidity and mortality, as well as substantial economic cost. In contrast to hemorrhage from the upper gastrointestinal tract (UGIH), the management of acute LGIH is less well defined; furthermore, there is a paucity of published studies that evaluate predictive models in this disorder. Nonetheless, extrapolating from what is known in UGIH, the development of reliable predictive models in LGIH may lead to improved patient care and outcome, by enhancing clinical triage, and by the more cost-effective use of limited healthcare resources. In this review, we discuss the technical development and potential use of artificial neural network in patients presenting with acute LGIH.',\n",
              "  'authors': ['Das, Ananya', 'Wong, Richard C K'],\n",
              "  'categories': ['GASTROENTEROLOGY & HEPATOLOGY'],\n",
              "  'journal': 'European journal of gastroenterology & hepatology',\n",
              "  'keywords': ['Acute Disease',\n",
              "   'Decision Making, Computer-Assisted',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Gastrointestinal Hemorrhage',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prognosis',\n",
              "   'Risk Assessment'],\n",
              "  'pmid': '17998830',\n",
              "  'title': 'Prediction of outcome in acute lower gastrointestinal hemorrhage: role of artificial neural network.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['PW'],\n",
              "  'ESSN': '1530-8561',\n",
              "  'IF_2013': '7.768',\n",
              "  'IF_2014': '7.911',\n",
              "  'IF_2015': '7.457',\n",
              "  'IF_2016': '8.008',\n",
              "  'IF_2017': '8.636',\n",
              "  'ISSN': '0009-9147',\n",
              "  'ISSN_online': '1530-8561',\n",
              "  'ISSN_print': '0009-9147',\n",
              "  'IsoAbbr': 'Clin. Chem.',\n",
              "  'JrId': 2992,\n",
              "  'MedAbbr': 'Clin Chem',\n",
              "  'NLMid': '9421549',\n",
              "  'Titre': 'Clin. Chem.',\n",
              "  'abstract': 'Setting up new immunoassays can be a laborious and expensive task. A relatively new form of multivariate analysis known as neural networks can be applied to this problem with potential savings in reagents and technician time. Neural network software programs for personal computers are now available. We applied one such software package (Brainmaker) to a model ELISA system for measuring human serum albumin. Random combinations of four variable ELISA conditions (antigen concentration, primary and secondary antibody titers, and time for chromagen development) were used to train a three-layered feed-forward network. The trained network was then used to predict measured absorbances as a function of the four input variables in separate cross-validation sets. The network adequately predicted the effect of the input variables on the absorbance produced. With use of such methods, optimal conditions for the linear dependence of absorbance on antigen concentration can be evaluated on the computer rather than in the laboratory, with subsequent savings of time and money.',\n",
              "  'authors': ['Vertosick, F T', 'Rehn, T'],\n",
              "  'categories': ['MEDICAL LABORATORY TECHNOLOGY'],\n",
              "  'journal': 'Clinical chemistry',\n",
              "  'keywords': ['Antibodies',\n",
              "   'Computer Simulation',\n",
              "   'Enzyme-Linked Immunosorbent Assay',\n",
              "   'Humans',\n",
              "   'Multivariate Analysis',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Serum Albumin',\n",
              "   'Software'],\n",
              "  'pmid': '8252719',\n",
              "  'title': 'Predicting behavior of an enzyme-linked immunoassay model by using commercially available neural network software.',\n",
              "  'year': '1993'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['QA'],\n",
              "  'ESSN': '1898-4002',\n",
              "  'IF_2013': '0.964',\n",
              "  'IF_2014': '1.105',\n",
              "  'IF_2015': '1.211',\n",
              "  'IF_2016': '1.364',\n",
              "  'IF_2017': '2.064',\n",
              "  'ISSN': '1896-1126',\n",
              "  'ISSN_online': '1898-4002',\n",
              "  'ISSN_print': '1896-1126',\n",
              "  'IsoAbbr': 'Adv Med Sci',\n",
              "  'JrId': 33296,\n",
              "  'MedAbbr': 'Adv Med Sci',\n",
              "  'NLMid': '101276222',\n",
              "  'Titre': 'Adv Med Sci',\n",
              "  'abstract': 'Autism spectrum disorder (ASD) is a neurodevelopmental disorder, of which Asperger syndrome and high-functioning autism are subtypes. Our goal is: 1) to determine whether a diagnostic model based on single-nucleotide polymorphisms (SNPs), brain regional thickness measurements, or brain regional volume measurements can distinguish Asperger syndrome from high-functioning autism; and 2) to compare the SNP, thickness, and volume-based diagnostic models.|Our study included 18 children with ASD: 13 subjects with high-functioning autism and 5 subjects with Asperger syndrome. For each child, we obtained 25 SNPs for 8 ASD-related genes; we also computed regional cortical thicknesses and volumes for 66 brain structures, based on structural magnetic resonance (MR) examination. To generate diagnostic models, we employed five machine-learning techniques: decision stump, alternating decision trees, multi-class alternating decision trees, logistic model trees, and support vector machines.|For SNP-based classification, three decision-tree-based models performed better than the other two machine-learning models. The performance metrics for three decision-tree-based models were similar: decision stump was modestly better than the other two methods, with accuracy = 90%, sensitivity = 0.95 and specificity = 0.75. All thickness and volume-based diagnostic models performed poorly. The SNP-based diagnostic models were superior to those based on thickness and volume. For SNP-based classification, rs878960 in GABRB3 (gamma-aminobutyric acid A receptor, beta 3) was selected by all tree-based models.|Our analysis demonstrated that SNP-based classification was more accurate than morphometry-based classification in ASD subtype classification. Also, we found that one SNP--rs878960 in GABRB3--distinguishes Asperger syndrome from high-functioning autism.',\n",
              "  'authors': ['Jiao, Y',\n",
              "   'Chen, R',\n",
              "   'Ke, X',\n",
              "   'Cheng, L',\n",
              "   'Chu, K',\n",
              "   'Lu, Z',\n",
              "   'Herskovits, E H'],\n",
              "  'categories': ['MEDICINE, RESEARCH & EXPERIMENTAL'],\n",
              "  'journal': 'Advances in medical sciences',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Asperger Syndrome',\n",
              "   'Autistic Disorder',\n",
              "   'Brain',\n",
              "   'Child',\n",
              "   'Child Development Disorders, Pervasive',\n",
              "   'Decision Support Techniques',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Polymorphism, Single Nucleotide',\n",
              "   'Predictive Value of Tests',\n",
              "   'Receptors, GABA-A',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity'],\n",
              "  'pmid': '22037176',\n",
              "  'title': 'Predictive models for subtypes of autism spectrum disorder based on single-nucleotide polymorphisms and magnetic resonance imaging.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['MQ'],\n",
              "  'ESSN': '1460-3659',\n",
              "  'IF_2013': '2.151',\n",
              "  'IF_2014': '2.351',\n",
              "  'IF_2015': '2.679',\n",
              "  'IF_2016': '2.256',\n",
              "  'IF_2017': '2.464',\n",
              "  'ISSN': '0306-3127',\n",
              "  'ISSN_online': '1460-3659',\n",
              "  'ISSN_print': '0306-3127',\n",
              "  'IsoAbbr': 'Soc Stud Sci',\n",
              "  'JrId': 23174,\n",
              "  'MedAbbr': 'Soc Stud Sci',\n",
              "  'NLMid': '7506743',\n",
              "  'Titre': 'Soc Stud Sci',\n",
              "  'abstract': \"The dominant practices of physical oceanography have recently shifted from being based on ship-based ocean sampling and sensing to being based on remote and robotic sensing using satellites, drifting floats and remotely operated and autonomous underwater vehicles. What are the implications of this change for the social relations of oceanographic science? This paper contributes to efforts to address this question, pursuing a situated view of ocean sensing technologies so as to contextualize and analyze new representations of the sea, and interactions between individual scientists, technologies and the ocean. By taking a broad view on oceanography through a 50-year shift from ship-based to remote and robotic sensing, I show the ways in which new technologies may provide an opportunity to fight what Oreskes has called 'ideologies of scientific heroism'. In particular, new sensing relations may emphasize the contributions of women and scientists from less well-funded institutions, as well as the ways in which oceanographic knowledge is always partial and dependent on interactions between nonhuman animals, technologies, and different humans. Thus, I argue that remote and robotic sensing technologies do not simply create more abstracted relations between scientists and the sea, but also may provide opportunities for more equitable scientific practice and refigured sensing relations.\",\n",
              "  'authors': ['Lehman, Jessica'],\n",
              "  'categories': ['HISTORY & PHILOSOPHY OF SCIENCE'],\n",
              "  'journal': 'Social studies of science',\n",
              "  'keywords': ['Humans',\n",
              "   'Interpersonal Relations',\n",
              "   'Oceanography',\n",
              "   'Oceans and Seas',\n",
              "   'Remote Sensing Technology',\n",
              "   'Research Personnel',\n",
              "   'Robotics',\n",
              "   'Ships',\n",
              "   'Technology'],\n",
              "  'pmid': '29199547',\n",
              "  'title': 'From ships to robots: The social relations of sensing the world ocean.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1872-6240',\n",
              "  'IF_2013': '2.828',\n",
              "  'IF_2014': '2.843',\n",
              "  'IF_2015': '2.561',\n",
              "  'IF_2016': '2.746',\n",
              "  'IF_2017': '3.125',\n",
              "  'ISSN': '0006-8993',\n",
              "  'ISSN_online': '1872-6240',\n",
              "  'ISSN_print': '0006-8993',\n",
              "  'IsoAbbr': 'Brain Res.',\n",
              "  'JrId': 1920,\n",
              "  'MedAbbr': 'Brain Res',\n",
              "  'NLMid': '0045503',\n",
              "  'Titre': 'Brain Res.',\n",
              "  'abstract': 'Healthy neurons have an optimal operating range, coded globally by the frequency of action potentials or locally by calcium. The maintenance of this range is governed by homeostatic plasticity. Here, we discuss how new approaches to treat depression alter synaptic activity. These approaches induce the neuron to recruit homeostatic mechanisms to relieve depression. Homeostasis generally implies that the direction of activity necessary to restore the neuron\\'s critical operating range is opposite in direction to its current activity pattern. Unconventional antidepressant therapies-deep brain stimulation and NMDAR antagonists-alter the neuron\\'s \"depressed\" state by pushing the neuron\\'s current activity in the same direction but to the extreme edge. These therapies rally the intrinsic drive of neurons in the opposite direction, thereby allowing the cell to return to baseline activity, form new synapses, and restore proper communication. In this review, we discuss seminal studies on protein synthesis dependent homeostatic plasticity and their contribution to our understanding of molecular mechanisms underlying the effectiveness of NMDAR antagonists as rapid antidepressants. Rapid antidepressant efficacy is likely to require a cascade of mRNA translational regulation. Emerging evidence suggests that changes in synaptic strength or intrinsic excitability converge on the same protein synthesis pathways, relieving depressive symptoms. Thus, we address the question: Are there multiple homeostatic mechanisms that induce the neuron and neuronal circuits to self-correct to regulate mood in vivo? Targeting alternative ways to induce homeostatic protein synthesis may provide, faster, safer, and longer lasting antidepressants. This article is part of a Special Issue entitled SI:RNA Metabolism in Disease.',\n",
              "  'authors': ['Raab-Graham, Kimberly F',\n",
              "   'Workman, Emily R',\n",
              "   'Namjoshi, Sanjeev',\n",
              "   'Niere, Farr'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Brain research',\n",
              "  'keywords': ['Animals',\n",
              "   'Antidepressive Agents',\n",
              "   'Autophagy',\n",
              "   'Brain',\n",
              "   'Depressive Disorder',\n",
              "   'Homeostasis',\n",
              "   'Humans',\n",
              "   'Neuronal Plasticity',\n",
              "   'Neurons',\n",
              "   'Protein Biosynthesis',\n",
              "   'Receptors, GABA-B',\n",
              "   'Receptors, N-Methyl-D-Aspartate',\n",
              "   'TOR Serine-Threonine Kinases'],\n",
              "  'pmid': '27125595',\n",
              "  'title': 'Pushing the threshold: How NMDAR antagonists induce homeostasis through protein synthesis to remedy depression.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'In this paper a new approach to the electroencephalogram (EEG) signal simulation based on the artificial neural networks (ANN) is proposed. The aim was to simulate the spontaneous human EEG background activity based solely on the experimentally acquired EEG data. Therefore, an EEG measurement campaign was conducted on a healthy awake adult in order to obtain an adequate ANN training data set. As demonstration of the performance of the ANN based approach, comparisons were made against autoregressive moving average (ARMA) filtering based method. Comprehensive quantitative and qualitative statistical analysis showed clearly that the EEG process obtained by the proposed method was in satisfactory agreement with the one obtained by measurements.',\n",
              "  'authors': ['Tomasevic, Nikola M',\n",
              "   'Neskovic, Aleksandar M',\n",
              "   'Neskovic, Natasa J'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Brain Waves',\n",
              "   'Computer Simulation',\n",
              "   'Electroencephalography',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Stochastic Processes'],\n",
              "  'pmid': '23627624',\n",
              "  'title': 'Artificial neural network based approach to EEG signal simulation.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1576-8260',\n",
              "  'IF_2013': '0.309',\n",
              "  'IF_2014': '0.309',\n",
              "  'IF_2015': '0.307',\n",
              "  'IF_2016': '0.323',\n",
              "  'IF_2017': '0.465',\n",
              "  'ISSN': '0004-0614',\n",
              "  'ISSN_online': '1576-8260',\n",
              "  'ISSN_print': '0004-0614',\n",
              "  'IsoAbbr': 'Arch. Esp. Urol.',\n",
              "  'JrId': 738,\n",
              "  'MedAbbr': 'Arch Esp Urol',\n",
              "  'NLMid': '0064757',\n",
              "  'Titre': 'Arch. Esp. Urol.',\n",
              "  'abstract': 'Radical prostatectomy is worldwide accepted as treatment for clinically localized prostate cancer. Its oncological results are excellent, so nowadays the functional outcomes: continence and potency, have become essential factors when evaluating the results. Open radical prostatectomy (ORP) is the gold standard against the new techniques, Laparoscopic (LRP) and robotic (RRP) must be compared. A systematic review of the literature is done to evaluate functional outcomes between the three approaches.|Systematic review in the databases: PubMed; EMBASE; Cochrane; SCOPUS; Science Citation Index for: \"radical retropubic prostatectomy\", \"open radical prostatectomy\"; \"laparoscopic prostatectomy\"; \"laparoscopic radical prostatectomy\"; \"robotic prostatectomy\"; \"robotic radical prostatectomy and functional assessment\"; \"continence\"; \"urinary function\"; \"incontinence\"; \"erectile function\"; \\'sexual function\"; \"quality of life\"; \"functional assessment\"; \"minimally invasive treatment was performed\".|The lack of randomized trials for this issue forces us to evaluate the functional results comparing the most important series of each approach, so the value of the results are very limited. Accepting 0 pads per day as continence definition and evaluated 12 months following surgery, the continence rates for each approach ORP, LRP and RRP are respectively: 8 1% (60-93%); 87% (82-95%) and 91% (84-98%). For erectile function admitting the capability for intercourse as potency definition, with or without the use of oral drugs and evaluated 12 months following surgery; the potency rates for each approach are: 68% (62-75%), 69% (52-78%) y 60% (20-97%).|Due the lack of randomized trials, we have to evaluate the most important contemporaneous series. The lack of homogeneity in evaluating and reporting results is evident in the urological community. With the limitations of the data available it seems to exist no differences between the three groups for functional outcomes. It is mandatory to develop randomized trials and achieve a consensus for the criteria at the time of evaluating the functional outcomes.',\n",
              "  'authors': ['Romero Otero, Javier', 'Martínez-Salamanca, Juan I'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Archivos espanoles de urologia',\n",
              "  'keywords': ['Erectile Dysfunction',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Robotics',\n",
              "   'Urinary Incontinence'],\n",
              "  'pmid': '17937336',\n",
              "  'title': '[Critical comparative analysis between open, laparoscopic and robotic radical prostatectomy: urinary continence and sexual function (part II)].',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1097-0150',\n",
              "  'IF_2013': '1.077',\n",
              "  'IF_2014': '0.69',\n",
              "  'IF_2015': '0.606',\n",
              "  'IF_2016': '1.667',\n",
              "  'IF_2017': '1.2',\n",
              "  'ISSN': '1092-9088',\n",
              "  'ISSN_online': '1097-0150',\n",
              "  'ISSN_print': '1092-9088',\n",
              "  'IsoAbbr': 'Comput. Aided Surg.',\n",
              "  'JrId': 20599,\n",
              "  'MedAbbr': 'Comput Aided Surg',\n",
              "  'NLMid': '9708375',\n",
              "  'Titre': 'Comput. Aided Surg.',\n",
              "  'abstract': 'The better understanding of the systems of the human body and their specific functions has led to minimally invasive and microsurgical procedures being carried out on an ever smaller scale. New therapies will require precision of 10 microm or better, which will only be achievable with robotic teleoperators. Experience with existing robot-assisted surgery systems shows that the human-machine interface is a critical component for acceptance of this technology, but no universally satisfactory interface has yet been found. Therefore, this article investigates a new concept based on kinesthetic motion feedback and presents details of the implementation of a first prototype.',\n",
              "  'authors': ['Urban, V',\n",
              "   'Wapler, M',\n",
              "   'Neugebauer, J',\n",
              "   'Hiller, A',\n",
              "   'Stallkamp, J',\n",
              "   'Weisener, T'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Computer aided surgery : official journal of the International Society for Computer Aided Surgery',\n",
              "  'keywords': ['Feedback',\n",
              "   'Humans',\n",
              "   'Kinesthesis',\n",
              "   'Microsurgery',\n",
              "   'Robotics',\n",
              "   'Surgical Equipment',\n",
              "   'Therapy, Computer-Assisted'],\n",
              "  'pmid': '10027496',\n",
              "  'title': 'Robot-assisted surgery system with kinesthetic feedback.',\n",
              "  'year': '1998'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '2227-4898',\n",
              "  'IF_2013': '0.5',\n",
              "  'IF_2014': '0.705',\n",
              "  'IF_2015': '0.939',\n",
              "  'IF_2016': '1.177',\n",
              "  'IF_2017': '1.166',\n",
              "  'ISSN': '2222-3959',\n",
              "  'ISSN_online': '2227-4898',\n",
              "  'ISSN_print': '2222-3959',\n",
              "  'IsoAbbr': 'Int J Ophthalmol',\n",
              "  'JrId': 39743,\n",
              "  'MedAbbr': 'Int J Ophthalmol',\n",
              "  'NLMid': '101553860',\n",
              "  'Titre': 'Int J Ophthalmol',\n",
              "  'abstract': 'To develop an automatic tool on screening diabetic retinopathy (DR) from diabetic patients.|We extracted textures from eye fundus images of each diabetes subject using grey level co-occurrence matrix method and trained a Bayesian model based on these textures. The receiver operating characteristic (ROC) curve was used to estimate the sensitivity and specificity of the Bayesian model.|A total of 1000 eyes fundus images from diabetic patients in which 298 eyes were diagnosed as DR by two ophthalmologists. The Bayesian model was trained using four extracted textures including contrast, entropy, angular second moment and correlation using a training dataset. The Bayesian model achieved a sensitivity of 0.949 and a specificity of 0.928 in the validation dataset. The area under the ROC curve was 0.938, and the 10-fold cross validation method showed that the average accuracy rate is 93.5%.|Textures extracted by grey level co-occurrence can be useful information for DR diagnosis, and a trained Bayesian model based on these textures can be an effective tool for DR screening among diabetic patients.',\n",
              "  'authors': ['Cao, Kai', 'Xu, Jie', 'Zhao, Wei-Qi'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'International journal of ophthalmology',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31341808',\n",
              "  'title': 'Artificial intelligence on diabetic retinopathy diagnosis: an automatic classification method based on grey level co-occurrence matrix and naive Bayesian model.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1094-4087',\n",
              "  'IF_2013': '3.525',\n",
              "  'IF_2014': '3.488',\n",
              "  'IF_2015': '3.148',\n",
              "  'IF_2016': '3.307',\n",
              "  'IF_2017': '3.356',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1094-4087',\n",
              "  'IsoAbbr': 'Opt Express',\n",
              "  'JrId': 29989,\n",
              "  'MedAbbr': 'Opt Express',\n",
              "  'NLMid': '101137103',\n",
              "  'Titre': 'Opt Express',\n",
              "  'abstract': 'This paper presents a comprehensive study on the contrast transfer function of de-noising algorithms. In order to cover a broad variety of methods, 45 de-noising algorithms are chosen considering their recognized efficiency in the different application domains of image processing. Advanced methods are targeted: wavelet transform-based algorithms with Daubechies, symlets, curvelets, contourlets, patch-based methods such as BM3D, NL-means algorithms and deep learning approaches; in addition, classical spatial filtering methods are considered, such as Wiener, median, Gauss filtering, and adaptive filtering approaches such as anisotropic diffusion and synthetic aperture radar filtering. The contrast transfer function is provided for each algorithm. Ranking of the set of de-noising algorithms is established according to proposed metrics. The paper provides practical methodology and novel results dedicated to the evaluation of the contrast transfer function of de-noising approaches from literature.',\n",
              "  'authors': ['Picart, Pascal', 'Montresor, Silvio'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics express',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31510613',\n",
              "  'title': 'Contrast transfer function of de-noising algorithms.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': 'Pelvic lymphadenectomy for prostate cancer is an important tool in the prognostication of the disease in selected patients who are at risk of occult lymph node metastases. This procedure, usually performed in conjunction with radical prostatectomy, had progressed successfully from an open approach to the current robot-assisted approach. The following article and accompanying video describe the surgical technique of robot-assisted pelvic lymphadenectomy for prostate cancer. We also discuss the indications, patient selection, preparation, complications, and tips to avoid the major pitfalls in the procedure.',\n",
              "  'authors': ['Png, Keng-Siang', 'Koch, Michael O'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Dissection',\n",
              "   'Humans',\n",
              "   'Intraoperative Complications',\n",
              "   'Laparoscopy',\n",
              "   'Lymph Node Excision',\n",
              "   'Male',\n",
              "   'Pelvis',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Surgical Instruments',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '23075403',\n",
              "  'title': 'Technique of pelvic lymphadenectomy after robot-assisted laparoscopic prostatectomy for prostate cancer.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['XQ'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.27',\n",
              "  'IF_2014': '0.292',\n",
              "  'IF_2015': '0.275',\n",
              "  'IF_2016': '0.344',\n",
              "  'IF_2017': '0.326',\n",
              "  'ISSN': '1000-0593',\n",
              "  'ISSN_print': '1000-0593',\n",
              "  'IsoAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'JrId': 22036,\n",
              "  'MedAbbr': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'NLMid': '9424805',\n",
              "  'Titre': 'Guang Pu Xue Yu Guang Pu Fen Xi',\n",
              "  'abstract': 'Near infrared (NIR) spectroscopy combined with pattern recognition was attempted to discriminate the freshness of eggs. The algorithm of one-class support vector machine (OC-SVM) was employed to solve the classification problem due to imbalanced number of training samples. In this work, 86 samples of eggs (71 samples of fresh eggs and 15 samples of unfresh eggs) were surveyed by Fourier transform NIR spectroscopy. Firstly, original spectra of eggs in the wave-number range of 10 000-4 000 cm(-1) were acquired. And then, principal component analysis (PCA) was employed to extract useful information from original spectral data, and the number of PCs was optimized. Finally, OC-SVM was performed to calibrate discrimination model, and the optimal PCs were used as the input eigenvectors of model. In order to obtain a good performance, the regularization parameter v and parameter sigma of the kernel function in OC-SVM model were optimized in building model. The optimal OC-SVM model was obtained with nu = 0.5 and sigma2 = 20.3. Experimental result shows that OC-SVM got better performance than conventional two-class SVM model under the same condition. The OC-SVM model was achieved with identification rates of 80 for both fresh eggs and unfresh eggs in the independent prediction set. The identification rates of fresh eggs were 100% in two-class SVM model. However, when the two-class SVM model was used to discriminate the unfresh eggs of, the identification rates were 0% in the independent prediction set. Compared with conventional two-class SVM model, the OC-SVM model showed its superior performance in discrimination of minority unfresh eggs samples. This work shows that it is feasible to identify egg freshness using NIR spectroscopy, and OC-SVM is an excellent choice in solving the problem of imbalanced number of samples in training set.',\n",
              "  'authors': ['Lin, Hao',\n",
              "   'Zhao, Jie-Wen',\n",
              "   'Chen, Quan-Sheng',\n",
              "   'Cai, Jian-Rong',\n",
              "   'Zhou, Ping'],\n",
              "  'categories': ['SPECTROSCOPY'],\n",
              "  'journal': 'Guang pu xue yu guang pu fen xi = Guang pu',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Eggs',\n",
              "   'Principal Component Analysis',\n",
              "   'Spectroscopy, Near-Infrared',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '20545133',\n",
              "  'title': '[Identification of egg freshness using near infrared spectroscopy and one class support vector machine algorithm].',\n",
              "  'year': '2010'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['TU'],\n",
              "  'ESSN': '1746-045X',\n",
              "  'IF_2013': '3.467',\n",
              "  'IF_2014': '3.539',\n",
              "  'IF_2015': '3.484',\n",
              "  'IF_2016': '3.876',\n",
              "  'IF_2017': '4.692',\n",
              "  'ISSN': '1746-0441',\n",
              "  'ISSN_online': '1746-045X',\n",
              "  'ISSN_print': '1746-0441',\n",
              "  'IsoAbbr': 'Expert Opin Drug Discov',\n",
              "  'JrId': 36030,\n",
              "  'MedAbbr': 'Expert Opin Drug Discov',\n",
              "  'NLMid': '101295755',\n",
              "  'Titre': 'Expert Opin Drug Discov',\n",
              "  'abstract': 'Introduction: Drug discovery is the process through which potential new compounds are identified by means of biology, chemistry, and pharmacology. Due to the high complexity of genomic data, AI techniques are increasingly needed to help reduce this and aid the adoption of optimal decisions. Phenotypic prediction is of particular use to drug discovery and precision medicine where sets of genes that predict a given phenotype are determined. Phenotypic prediction is an undetermined problem given that the number of monitored genetic probes markedly exceeds the number of collected samples (from patients). This imbalance creates ambiguity in the characterization of the biological pathways that are responsible for disease development. Areas covered: In this paper, the authors present AI methodologies that perform a robust deep sampling of altered genetic pathways to locate new therapeutic targets, assist in drug repurposing and speed up and optimize the drug selection process. Expert opinion: AI is a potential solution to a number of drug discovery problems, though one should, bear in mind that the quality of data predicts the overall quality of the prediction, as in any modeling task in data science. The use of transparent methodologies is crucial, particularly in drug repositioning/repurposing in rare diseases.',\n",
              "  'authors': ['Álvarez-Machancoses, Óscar', 'Fernández-Martínez, Juan Luis'],\n",
              "  'categories': ['PHARMACOLOGY & PHARMACY'],\n",
              "  'journal': 'Expert opinion on drug discovery',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31140873',\n",
              "  'title': 'Using artificial intelligence methods to speed up drug discovery.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SU'],\n",
              "  'ESSN': '1538-9235',\n",
              "  'IF_2013': '2.038',\n",
              "  'IF_2014': '1.603',\n",
              "  'IF_2015': '1.442',\n",
              "  'IF_2016': '1.409',\n",
              "  'IF_2017': '1.499',\n",
              "  'ISSN': '1040-5488',\n",
              "  'ISSN_online': '1538-9235',\n",
              "  'ISSN_print': '1040-5488',\n",
              "  'IsoAbbr': 'Optom Vis Sci',\n",
              "  'JrId': 6277,\n",
              "  'MedAbbr': 'Optom Vis Sci',\n",
              "  'NLMid': '8904931',\n",
              "  'Titre': 'Optom Vis Sci',\n",
              "  'abstract': 'Our main goal in this work was to develop an artificial neural network (NN) that could classify specific types of corneal shapes using Zernike coefficients as input. Other authors have implemented successful NN systems in the past and have demonstrated their efficiency using different parameters. Our claim is that, given the increasing popularity of Zernike polynomials among the eye care community, this may be an interesting choice to add complementing value and precision to existing methods. By using a simple and well-documented corneal surface representation scheme, which relies on corneal elevation information, one can generate simple NN input parameters that are independent of curvature definition and that are also efficient.|We have used the Matlab Neural Network Toolbox (MathWorks, Natick, MA) to implement a three-layer feed-forward NN with 15 inputs and 5 outputs. A database from an EyeSys System 2000 (EyeSys Vision, Houston, TX) videokeratograph installed at the Escola Paulista de Medicina-Sao Paulo was used. This database contained an unknown number of corneal types. From this database, two specialists selected 80 corneas that could be clearly classified into five distinct categories: (1) normal, (2) with-the-rule astigmatism, (3) against-the-rule astigmatism, (4) keratoconus, and (5) post-laser-assisted in situ keratomileusis. The corneal height (SAG) information of the 80 data files was fit with the first 15 Vision Science and it Applications (VSIA) standard Zernike coefficients, which were individually used to feed the 15 neurons of the input layer. The five output neurons were associated with the five typical corneal shapes. A group of 40 cases was randomly selected from the larger group of 80 corneas and used as the training set.|The NN responses were statistically analyzed in terms of sensitivity [true positive/(true positive + false negative)], specificity [true negative/(true negative + false positive)], and precision [(true positive + true negative)/total number of cases]. The mean values for these parameters were, respectively, 78.75, 97.81, and 94%.|Although we have used a relatively small training and testing set, results presented here should be considered promising. They are certainly an indication of the potential of Zernike polynomials as reliable parameters, at least in the cases presented here, as input data for artificial intelligence automation of the diagnosis process of videokeratography examinations. This technique should facilitate the implementation and add value to the classification methods already available. We also discuss briefly certain special properties of Zernike polynomials that are what we think make them suitable as NN inputs for this type of application.',\n",
              "  'authors': ['Carvalho, Luis Alberto'],\n",
              "  'categories': ['OPHTHALMOLOGY'],\n",
              "  'journal': 'Optometry and vision science : official publication of the American Academy of Optometry',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Corneal Topography',\n",
              "   'Diagnosis, Computer-Assisted',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Numerical Analysis, Computer-Assisted'],\n",
              "  'pmid': '15711463',\n",
              "  'title': 'Preliminary results of neural networks and zernike polynomials for classification of videokeratography maps.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '0.58',\n",
              "  'IF_2014': '0.488',\n",
              "  'IF_2015': '0.918',\n",
              "  'IF_2016': '0.454',\n",
              "  'IF_2017': '0.398',\n",
              "  'ISSN': '0884-6812',\n",
              "  'IsoAbbr': 'Anal. Quant. Cytol. Histol.',\n",
              "  'JrId': 1348,\n",
              "  'MedAbbr': 'Anal Quant Cytol Histol',\n",
              "  'NLMid': '8506819',\n",
              "  'Titre': 'Anal. Quant. Cytol. Histol.',\n",
              "  'abstract': 'To use a commercially available artificial neural network (ANN) software program to distinguish prognostically good and bad cases of chronic myeloid leukemia (CML).|A total of 40 patients with CML who developed blast crisis or proceeded in the accelerated phase were selected. They formed two groups, group 1 and group 2, of 20 patients each, who developed accelerated phase or blast crisis within 18 months or 30 months, respectively, after the initial diagnosis of the chronic phase of CML. The detailed clinical, hematologic, and morphometric data were collected in all these cases. A suitable ANN software program was used to analyze these data.|All cases were randomly distributed automatically by the program into three groups: training set (28), validation set (4), and test set (8). In the test set, the ANN program successfully classified all group I and group II patients.|We successfully used a commercially available ANN software program to develop a model able to classify prognostically good and bad cases of CML.',\n",
              "  'authors': ['Dey, Pranab',\n",
              "   'Lamba, Amit',\n",
              "   'Kumari, Savita',\n",
              "   'Marwaha, Neelam'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Analytical and quantitative cytology and histology',\n",
              "  'keywords': ['Humans',\n",
              "   'Leukemia, Myelogenous, Chronic, BCR-ABL Positive',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prognosis',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Software'],\n",
              "  'pmid': '22590811',\n",
              "  'title': 'Application of an artificial neural network in the prognosis of chronic myeloid leukemia.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1471-2229',\n",
              "  'IF_2013': '3.942',\n",
              "  'IF_2014': '3.813',\n",
              "  'IF_2015': '3.631',\n",
              "  'IF_2016': '3.964',\n",
              "  'IF_2017': '3.93',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2229',\n",
              "  'IsoAbbr': 'BMC Plant Biol.',\n",
              "  'JrId': 22326,\n",
              "  'MedAbbr': 'BMC Plant Biol',\n",
              "  'NLMid': '100967807',\n",
              "  'Titre': 'BMC Plant Biol.',\n",
              "  'abstract': 'Organelle genomes of Geraniaceae exhibit several unusual evolutionary phenomena compared to other angiosperm families including accelerated nucleotide substitution rates, widespread gene loss, reduced RNA editing, and extensive genomic rearrangements. Since most organelle-encoded proteins function in multi-subunit complexes that also contain nuclear-encoded proteins, it is likely that the atypical organellar phenomena affect the evolution of nuclear genes encoding organellar proteins. To begin to unravel the complex co-evolutionary interplay between organellar and nuclear genomes in this family, we sequenced nuclear transcriptomes of two species, Geranium maderense and Pelargonium x hortorum.|Normalized cDNA libraries of G. maderense and P. x hortorum were used for transcriptome sequencing. Five assemblers (MIRA, Newbler, SOAPdenovo, SOAPdenovo-trans [SOAPtrans], Trinity) and two next-generation technologies (454 and Illumina) were compared to determine the optimal transcriptome sequencing approach. Trinity provided the highest quality assembly of Illumina data with the deepest transcriptome coverage. An analysis to determine the amount of sequencing needed for de novo assembly revealed diminishing returns of coverage and quality with data sets larger than sixty million Illumina paired end reads for both species. The G. maderense and P. x hortorum transcriptomes contained fewer transcripts encoding the PLS subclass of PPR proteins relative to other angiosperms, consistent with reduced mitochondrial RNA editing activity in Geraniaceae. In addition, transcripts for all six plastid targeted sigma factors were identified in both transcriptomes, suggesting that one of the highly divergent rpoA-like ORFs in the P. x hortorum plastid genome is functional.|The findings support the use of the Illumina platform and assemblers optimized for transcriptome assembly, such as Trinity or SOAPtrans, to generate high-quality de novo transcriptomes with broad coverage. In addition, results indicated no major improvements in breadth of coverage with data sets larger than six billion nucleotides or when sampling RNA from four tissue types rather than from a single tissue. Finally, this work demonstrates the power of cross-compartmental genomic analyses to deepen our understanding of the correlated evolution of the nuclear, plastid, and mitochondrial genomes in plants.',\n",
              "  'authors': ['Zhang, Jin',\n",
              "   'Ruhlman, Tracey A',\n",
              "   'Mower, Jeffrey P',\n",
              "   'Jansen, Robert K'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'BMC plant biology',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Cell Nucleus',\n",
              "   'Crosses, Genetic',\n",
              "   'Databases, Genetic',\n",
              "   'Databases, Protein',\n",
              "   'Gene Library',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Geraniaceae',\n",
              "   'High-Throughput Nucleotide Sequencing',\n",
              "   'Molecular Sequence Annotation',\n",
              "   'Organelles',\n",
              "   'Plant Proteins',\n",
              "   'RNA, Ribosomal',\n",
              "   'Sequence Analysis, DNA',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '24373163',\n",
              "  'title': 'Comparative analyses of two Geraniaceae transcriptomes using next-generation sequencing.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Because of its richness and availability, micro-blogging has become an ideal platform for conducting psychological research. In this paper, we proposed to predict active users' personality traits through micro-blogging behaviors. 547 Chinese active users of micro-blogging participated in this study. Their personality traits were measured by the Big Five Inventory, and digital records of micro-blogging behaviors were collected via web crawlers. After extracting 839 micro-blogging behavioral features, we first trained classification models utilizing Support Vector Machine (SVM), differentiating participants with high and low scores on each dimension of the Big Five Inventory [corrected]. The classification accuracy ranged from 84% to 92%. We also built regression models utilizing PaceRegression methods, predicting participants' scores on each dimension of the Big Five Inventory. The Pearson correlation coefficients between predicted scores and actual scores ranged from 0.48 to 0.54. Results indicated that active users' personality traits could be predicted by micro-blogging behaviors.\",\n",
              "  'authors': ['Li, Lin',\n",
              "   'Li, Ang',\n",
              "   'Hao, Bibo',\n",
              "   'Guan, Zengda',\n",
              "   'Zhu, Tingshao'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Adult',\n",
              "   'Blogging',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Models, Psychological',\n",
              "   'Personality Assessment',\n",
              "   'Personality Inventory',\n",
              "   'Reproducibility of Results',\n",
              "   'Support Vector Machine',\n",
              "   'Surveys and Questionnaires',\n",
              "   'Young Adult'],\n",
              "  'pmid': '24465462',\n",
              "  'title': \"Predicting active users' personality based on micro-blogging behaviors.\",\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1529-4242',\n",
              "  'IF_2013': '3.328',\n",
              "  'IF_2014': '2.993',\n",
              "  'IF_2015': '3.087',\n",
              "  'IF_2016': '3.843',\n",
              "  'IF_2017': '3.475',\n",
              "  'ISSN': '0032-1052',\n",
              "  'ISSN_online': '1529-4242',\n",
              "  'ISSN_print': '0032-1052',\n",
              "  'IsoAbbr': 'Plast. Reconstr. Surg.',\n",
              "  'JrId': 6482,\n",
              "  'MedAbbr': 'Plast Reconstr Surg',\n",
              "  'NLMid': '1306050',\n",
              "  'Titre': 'Plast. Reconstr. Surg.',\n",
              "  'abstract': 'The latissimus dorsi muscle is a workhorse of reconstructive surgery. Traditional harvest technique requires a long, posterior donor-site incision. Endoscopic harvest is limited by technical challenges. Robotic technology permits a simpler, minimally invasive harvest technique.|Seven consecutive robotic latissimus dorsi muscle harvests were performed by a single surgeon. Two were used as free flaps for scalp reconstruction and the remaining five as pedicled flaps for breast reconstruction; three were for immediate, implant-based reconstruction with nipple-areola complex-sparing mastectomies, and two were for radiated breasts when the expander was exchanged for an implant. Harvest technique employed a short, axillary incision for pedicle dissection and two to three additional ports for robotic instrumentation.|All seven muscle flaps were harvested without converting to an open technique. Both free flaps were successfully transferred. All pedicled flaps resulted in successful breast reconstructions. Flap harvest complications included a single, temporary radial nerve palsy in the contralateral extremity, likely from positioning. There were no donor-site hematomas, seromas, or cutaneous thermal injuries. Robotic harvest time decreased from over 2 hours to about an hour over the study period.|Robotic harvest of the latissimus dorsi is a novel and effective method of muscle harvest. It offers technical advantages over endoscopic harvest and aesthetic advantages over the open technique.|Therapeutic, IV.',\n",
              "  'authors': ['Selber, Jesse C',\n",
              "   'Baumann, Donald P',\n",
              "   'Holsinger, F Chistopher'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'Plastic and reconstructive surgery',\n",
              "  'keywords': ['Cadaver',\n",
              "   'Endoscopy',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Follow-Up Studies',\n",
              "   'Humans',\n",
              "   'Mammaplasty',\n",
              "   'Mastectomy',\n",
              "   'Muscle, Skeletal',\n",
              "   'Robotics',\n",
              "   'Skin Transplantation',\n",
              "   'Thoracic Wall',\n",
              "   'Tissue and Organ Harvesting'],\n",
              "  'pmid': '22634647',\n",
              "  'title': 'Robotic latissimus dorsi muscle harvest:  a case series.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'The widespread use of electronic health records (EHRs) has generated massive clinical data storage. Association rules mining is a feasible technique to convert this large amount of data into usable knowledge for clinical decision making, research or billing. We present a data driven method to create a knowledge base linking medications to pathological conditions through their therapeutic indications from elements within the EHRs.|Association rules were created from the data of patients hospitalised between May 2012 and May 2013 in the department of Cardiology at the University Hospital of Strasbourg. Medications were extracted from the medication list, and the pathological conditions were extracted from the discharge summaries using a natural language processing tool. Association rules were generated along with different interestingness measures: chi square, lift, conviction, dependency, novelty and satisfaction. All medication-disease pairs were compared to the Summary of Product Characteristics, which is the gold standard. A score based on the other interestingness measures was created to filter the best rules, and the indices were calculated for the different interestingness measures.|After the evaluation against the gold standard, a list of accurate association rules was successfully retrieved. Dependency represents the best recall (0.76). Our score exhibited higher exactness (0.84) and precision (0.27) than all of the others interestingness measures. Further reductions in noise produced by this method must be performed to improve the classification precision.|Association rules mining using the unstructured elements of the EHR is a feasible technique to identify clinically accurate associations between medications and pathological conditions.',\n",
              "  'authors': ['Séverac, François',\n",
              "   'Sauleau, Erik A',\n",
              "   'Meyer, Nicolas',\n",
              "   'Lefèvre, Hassina',\n",
              "   'Nisand, Gabriel',\n",
              "   'Jay, Nicolas'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Cardiovascular Diseases',\n",
              "   'Data Mining',\n",
              "   'Electronic Health Records',\n",
              "   'Humans',\n",
              "   'Knowledge Bases',\n",
              "   'Natural Language Processing'],\n",
              "  'pmid': '25888890',\n",
              "  'title': 'Non-redundant association rules between diseases and medications: an automated method for knowledge base construction.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': \"Computerized management of medical information and 3D imaging has become the norm in everyday medical practice. Surgeons exploit these emerging technologies and bring information previously confined to the radiology rooms into the operating theatre. The paper reports the authors' experience with integrated stereoscopic 3D-rendered images in the da Vinci surgeon console.|Volume-rendered images were obtained from a standard computed tomography dataset using the OsiriX DICOM workstation. A custom OsiriX plugin was created that permitted the 3D-rendered images to be displayed in the da Vinci surgeon console and to appear stereoscopic. These rendered images were displayed in the robotic console using the TilePro multi-input display. The upper part of the screen shows the real endoscopic surgical field and the bottom shows the stereoscopic 3D-rendered images. These are controlled by a 3D joystick installed on the console, and are updated in real time.|Five patients underwent a robotic augmented reality-enhanced procedure. The surgeon was able to switch between the classical endoscopic view and a combined virtual view during the procedure. Subjectively, the addition of the rendered images was considered to be an undeniable help during the dissection phase.|With the rapid evolution of robotics, computer-aided surgery is receiving increasing interest. This paper details the authors' experience with 3D-rendered images projected inside the surgical console. The use of this intra-operative mixed reality technology is considered very useful by the surgeon. It has been shown that the usefulness of this technique is a step toward computer-aided surgery that will progress very quickly over the next few years.\",\n",
              "  'authors': ['Volonté, Francesco',\n",
              "   'Buchs, Nicolas C',\n",
              "   'Pugin, François',\n",
              "   'Spaltenstein, Joël',\n",
              "   'Schiltz, Boris',\n",
              "   'Jung, Minoa',\n",
              "   'Hagen, Monika',\n",
              "   'Ratib, Osman',\n",
              "   'Morel, Philippe'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Depth Perception',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Minimally Invasive Surgical Procedures',\n",
              "   'Models, Anatomic',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '23239589',\n",
              "  'title': 'Augmented reality to the rescue of the minimally invasive surgeon. The usefulness of the interposition of stereoscopic images in the Da Vinci™ robotic console.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DR'],\n",
              "  'ESSN': '1554-8635',\n",
              "  'IF_2013': '11.423',\n",
              "  'IF_2014': '11.753',\n",
              "  'IF_2015': '9.108',\n",
              "  'IF_2016': '8.593',\n",
              "  'IF_2017': '11.1',\n",
              "  'ISSN': '1554-8627',\n",
              "  'ISSN_online': '1554-8635',\n",
              "  'ISSN_print': '1554-8627',\n",
              "  'IsoAbbr': 'Autophagy',\n",
              "  'JrId': 33022,\n",
              "  'MedAbbr': 'Autophagy',\n",
              "  'NLMid': '101265188',\n",
              "  'Titre': 'Autophagy',\n",
              "  'abstract': \"Autophagy is an intracellular degradation mechanism in response to nutrient starvation. Via autophagy, some nonessential cellular constituents are degraded in a lysosome-dependent manner to generate biomolecules that can be utilized for maintaining the metabolic homeostasis. Although it is known that under starvation the global protein synthesis is significantly reduced mainly due to suppression of MTOR (mechanistic target of rapamycin serine/threonine kinase), emerging evidence demonstrates that de novo protein synthesis is involved in the autophagic process. However, characterizing these de novo proteins has been an issue with current techniques. Here, we developed a novel method to identify newly synthesized proteins during starvation-mediated autophagy by combining bio-orthogonal noncanonical amino acid tagging (BONCAT) and isobaric tags for relative and absolute quantitation (iTRAQTM). Using bio-orthogonal metabolic tagging, L-azidohomoalanine (AHA) was incorporated into newly synthesized proteins which were then enriched with avidin beads after a click reaction between alkyne-bearing biotin and AHA's bio-orthogonal azide moiety. The enriched proteins were subjected to iTRAQ labeling for protein identification and quantification using liquid chromatography-tandem mass spectrometry (LC-MS/MS). Via the above approach, we identified and quantified a total of 1176 proteins and among them 711 proteins were found to meet our defined criteria as de novo synthesized proteins during starvation-mediated autophagy. The characterized functional profiles of the 711 newly synthesized proteins by bioinformatics analysis suggest their roles in ensuring the prosurvival outcome of autophagy. Finally, we performed validation assays for some selected proteins and found that knockdown of some genes has a significant impact on starvation-induced autophagy. Thus, we think that the BONCAT-iTRAQ approach is effective in the identification of newly synthesized proteins and provides useful insights to the molecular mechanisms and biological functions of autophagy.\",\n",
              "  'authors': ['Wang, Jigang',\n",
              "   'Zhang, Jianbin',\n",
              "   'Lee, Yew-Mun',\n",
              "   'Koh, Pin-Lang',\n",
              "   'Ng, Shukie',\n",
              "   'Bao, Feichao',\n",
              "   'Lin, Qingsong',\n",
              "   'Shen, Han-Ming'],\n",
              "  'categories': ['CELL BIOLOGY'],\n",
              "  'journal': 'Autophagy',\n",
              "  'keywords': ['Alanine',\n",
              "   'Autophagy',\n",
              "   'Gene Ontology',\n",
              "   'HeLa Cells',\n",
              "   'Humans',\n",
              "   'Protein Biosynthesis',\n",
              "   'Proteomics',\n",
              "   'Reproducibility of Results',\n",
              "   'Signal Transduction',\n",
              "   'Staining and Labeling',\n",
              "   'Transcription, Genetic'],\n",
              "  'pmid': '27463841',\n",
              "  'title': 'Quantitative chemical proteomics profiling of de novo protein synthesis during starvation-mediated autophagy.',\n",
              "  'year': '2016'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'There are no widely used models in clinical care to predict outcome in acute lower gastro-intestinal bleeding (ALGIB). If available these could help triage patients at presentation to appropriate levels of care/intervention and improve medical resource utilisation. We aimed to apply a state-of-the-art machine learning classifier, gradient boosting (GB), to predict outcome in ALGIB using non-endoscopic measurements as predictors.|Non-endoscopic variables from patients with ALGIB attending the emergency departments of two teaching hospitals were analysed retrospectively for training/internal validation (n=170) and external validation (n=130) of the GB model. The performance of the GB algorithm in predicting recurrent bleeding, clinical intervention and severe bleeding was compared to a multiple logic regression (MLR) model and two published MLR-based prediction algorithms (BLEED and Strate prediction rule).|The GB algorithm had the best negative predictive values for the chosen outcomes (>88%). On internal validation the accuracy of the GB algorithm for predicting recurrent bleeding, therapeutic intervention and severe bleeding were (88%, 88% and 78% respectively) and superior to the BLEED classification (64%, 68% and 63%), Strate prediction rule (78%, 78%, 67%) and conventional MLR (74%, 74% 62%). On external validation the accuracy was similar to conventional MLR for recurrent bleeding (88% vs. 83%) and therapeutic intervention (91% vs. 87%) but superior for severe bleeding (83% vs. 71%).|The gradient boosting algorithm accurately predicts outcome in patients with acute lower gastrointestinal bleeding and outperforms multiple logistic regression based models. These may be useful for risk stratification of patients on presentation to the emergency department.',\n",
              "  'authors': ['Ayaru, Lakshmana',\n",
              "   'Ypsilantis, Petros-Pavlos',\n",
              "   'Nanapragasam, Abigail',\n",
              "   'Choi, Ryan Chang-Ho',\n",
              "   'Thillanathan, Anish',\n",
              "   'Min-Ho, Lee',\n",
              "   'Montana, Giovanni'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Acute Disease',\n",
              "   'Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Cohort Studies',\n",
              "   'Female',\n",
              "   'Gastrointestinal Hemorrhage',\n",
              "   'Humans',\n",
              "   'Logistic Models',\n",
              "   'Machine Learning',\n",
              "   'Male',\n",
              "   'Medical Informatics',\n",
              "   'Middle Aged',\n",
              "   'Prognosis',\n",
              "   'Young Adult'],\n",
              "  'pmid': '26172121',\n",
              "  'title': 'Prediction of Outcome in Acute Lower Gastrointestinal Bleeding Using Gradient Boosting.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1878-108X',\n",
              "  'IF_2013': '12.902',\n",
              "  'IF_2014': '13.555',\n",
              "  'IF_2015': '12.504',\n",
              "  'IF_2016': '11.124',\n",
              "  'IF_2017': '11.439',\n",
              "  'ISSN': '0166-2236',\n",
              "  'ISSN_online': '1878-108X',\n",
              "  'ISSN_print': '0166-2236',\n",
              "  'IsoAbbr': 'Trends Neurosci.',\n",
              "  'JrId': 7840,\n",
              "  'MedAbbr': 'Trends Neurosci',\n",
              "  'NLMid': '7808616',\n",
              "  'Titre': 'Trends Neurosci.',\n",
              "  'abstract': \"Recent decades have seen dramatic progress in brain research. These advances were often buttressed by probing single variables to make circumscribed discoveries, typically through null hypothesis significance testing. New ways for generating massive data fueled tension between the traditional methodology that is used to infer statistically relevant effects in carefully chosen variables, and pattern-learning algorithms that are used to identify predictive signatures by searching through abundant information. In this article we detail the antagonistic philosophies behind two quantitative approaches: certifying robust effects in understandable variables, and evaluating how accurately a built model can forecast future outcomes. We discourage choosing analytical tools via categories such as 'statistics' or 'machine learning'. Instead, to establish reproducible knowledge about the brain, we advocate prioritizing tools in view of the core motivation of each quantitative analysis: aiming towards mechanistic insight or optimizing predictive accuracy.\",\n",
              "  'authors': ['Bzdok, Danilo', 'Ioannidis, John P A'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Trends in neurosciences',\n",
              "  'keywords': ['Animals',\n",
              "   'Biomedical Research',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Data Science',\n",
              "   'Forecasting',\n",
              "   'Humans',\n",
              "   'Models, Theoretical',\n",
              "   'Neurosciences'],\n",
              "  'pmid': '30808574',\n",
              "  'title': 'Exploration, Inference, and Prediction in Neuroscience and Biomedicine.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1872-6240',\n",
              "  'IF_2013': '2.828',\n",
              "  'IF_2014': '2.843',\n",
              "  'IF_2015': '2.561',\n",
              "  'IF_2016': '2.746',\n",
              "  'IF_2017': '3.125',\n",
              "  'ISSN': '0006-8993',\n",
              "  'ISSN_online': '1872-6240',\n",
              "  'ISSN_print': '0006-8993',\n",
              "  'IsoAbbr': 'Brain Res.',\n",
              "  'JrId': 1920,\n",
              "  'MedAbbr': 'Brain Res',\n",
              "  'NLMid': '0045503',\n",
              "  'Titre': 'Brain Res.',\n",
              "  'abstract': 'Organisms often evolve as compromises, and many of these compromises can be expressed in terms of energy efficiency. Thus, many authors analyze energetic costs processes during information transmission in the brain. In this paper we study information transmission rate per energy used in a class of ring, brain inspired neural networks, which we assume to involve components like excitatory and inhibitory neurons or long-range connections. Choosing model of neuron we followed a probabilistic approach proposed by Levy and Baxter (2002), which contains all essential qualitative mechanisms participating in the transmission process and provides results consistent with physiologically observed values. Our research shows that all network components, in broad range of conditions, significantly improve the information-energetic efficiency. It turned out that inhibitory neurons can improve the information-energetic transmission efficiency by 50%, while long-range connections can improve the efficiency even by 70%. We also found that the most effective is the network with the smallest size: we observed that two times increase of the size can cause even three times decrease of the information-energetic efficiency. This article is part of a Special Issue entitled Neural Coding 2012. ',\n",
              "  'authors': ['Paprocki, Bartosz', 'Szczepanski, Janusz'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Brain research',\n",
              "  'keywords': ['Energy Metabolism',\n",
              "   'Information Theory',\n",
              "   'Nerve Net',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons'],\n",
              "  'pmid': '23891793',\n",
              "  'title': 'Transmission efficiency in ring, brain inspired neuronal networks. Information and energetic aspects.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1879-1026',\n",
              "  'IF_2013': '3.163',\n",
              "  'IF_2014': '4.099',\n",
              "  'IF_2015': '3.976',\n",
              "  'IF_2016': '4.9',\n",
              "  'IF_2017': '4.61',\n",
              "  'ISSN': '0048-9697',\n",
              "  'ISSN_online': '1879-1026',\n",
              "  'ISSN_print': '0048-9697',\n",
              "  'IsoAbbr': 'Sci. Total Environ.',\n",
              "  'JrId': 7472,\n",
              "  'MedAbbr': 'Sci Total Environ',\n",
              "  'NLMid': '0330500',\n",
              "  'Titre': 'Sci. Total Environ.',\n",
              "  'abstract': 'In this study, we developed Different Artificial Intelligence (AI) models namely Artificial Neural Network (ANN), Adaptive Network based Fuzzy Inference System (ANFIS) and Support Vector Machine (SVM) for the prediction of Compression Coefficient of soil (Cc) which is one of the most important geotechnical parameters. A Monte Carlo approach was used for the sensitivity analysis of the AI models and input parameters. For the construction and validation of the models, 189 soft clayey soil samples were analyzed. In the models study, 13 input parameters: depth of sample, bulk density, plasticity index, moisture content, clay content, specific gravity, void ratio, liquid limit, dry density, porosity, plastic limit, degree of saturation, and liquidity index were used to obtain one output parameter \"Cc\". Validation of the models was done using statistical methods such as Mean Absolute Error (MAE), Root Mean Square Error (RMSE), and Coefficient of determination (R2). Results of the model validation indicate that though performance of all the three models is good but SVM model is the best in the prediction of Cc. The Monte Carlo method based sensitivity analysis results show that out of the 13 input parameters considered for the models study, four parameters namely clay, degree of saturation, specific gravity and depth of sample are the most relevant in the prediction of Cc, and other parameters (bulk density, dry density, void ratio and porosity) are the most insignificant parameters for the prediction of Cc. Removal of these insignificant parameters helped to reduce the dimension of the input space and also model running time, and improved significantly the performance of the AI models. The results of this study might help in selecting the suitable AI models and input parameters for better and quick prediction of the Cc of soil.',\n",
              "  'authors': ['Pham, Binh Thai',\n",
              "   'Nguyen, Manh Duc',\n",
              "   'Dao, Dong Van',\n",
              "   'Prakash, Indra',\n",
              "   'Ly, Hai-Bang',\n",
              "   'Le, Tien-Thinh',\n",
              "   'Ho, Lanh Si',\n",
              "   'Nguyen, Kien Trung',\n",
              "   'Ngo, Trinh Quoc',\n",
              "   'Hoang, Vu',\n",
              "   'Son, Le Hoang',\n",
              "   'Ngo, Huong Thanh Thi',\n",
              "   'Tran, Hieu Trung',\n",
              "   'Do, Ngoc Minh',\n",
              "   'Van Le, Hiep',\n",
              "   'Ho, Huu Loc',\n",
              "   'Tien Bui, Dieu'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'The Science of the total environment',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31082591',\n",
              "  'title': 'Development of artificial intelligence models for the prediction of Compression Coefficient of soil: An application of Monte Carlo sensitivity analysis.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1757-6334',\n",
              "  'IF_2013': '0.931',\n",
              "  'IF_2014': '0.783',\n",
              "  'IF_2015': '0.785',\n",
              "  'IF_2016': '0.8',\n",
              "  'IF_2017': '0.991',\n",
              "  'ISSN': '0219-7200',\n",
              "  'ISSN_online': '1757-6334',\n",
              "  'ISSN_print': '0219-7200',\n",
              "  'IsoAbbr': 'J Bioinform Comput Biol',\n",
              "  'JrId': 31986,\n",
              "  'MedAbbr': 'J Bioinform Comput Biol',\n",
              "  'NLMid': '101187344',\n",
              "  'Titre': 'J Bioinform Comput Biol',\n",
              "  'abstract': 'The functional annotation of amino acid sequences is one of the most important problems in bioinformatics. Different programs have been successfully applied for recognition of some functional classes; nevertheless, many functional groups still cannot be predicted with the required accuracy. We developed a new method for protein function recognition using the original approach of sequence description. Each sequence of the training set is compared with the query sequence, and the local similarity scores are calculated for the query sequence positions and used as input data for the original classifier. The method was tested using leave-one-out cross-validation for three data sets covering 58 enzyme classes. Two tested sets including noncrossing functional classes were recognized with high accuracy at various levels of classification hierarchy. The majority of these classes were predicted with 100% accuracy, showing a prediction ability comparable with the HMMer method and an accuracy superior to the SVM-Prot program. When the tested set was composed of intersected classes of ligand specificity, the prediction accuracy was less; however, the accuracy increased as the size of the predicted class expanded. The proposed method can be used for both predicting protein functional class and selecting the functionally significant sites in a sequence.',\n",
              "  'authors': ['Alexandrov, Kirill',\n",
              "   'Sobolev, Boris',\n",
              "   'Filimonov, Dmitry',\n",
              "   'Poroikov, Vladimir'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of bioinformatics and computational biology',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Artificial Intelligence',\n",
              "   'Computer Simulation',\n",
              "   'Models, Chemical',\n",
              "   'Molecular Sequence Data',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Proteins',\n",
              "   'Sequence Alignment',\n",
              "   'Sequence Analysis, Protein',\n",
              "   'Sequence Homology, Amino Acid',\n",
              "   'Structure-Activity Relationship'],\n",
              "  'pmid': '18763738',\n",
              "  'title': 'Recognition of protein function using the local similarity.',\n",
              "  'year': '2008'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1471-2229',\n",
              "  'IF_2013': '3.942',\n",
              "  'IF_2014': '3.813',\n",
              "  'IF_2015': '3.631',\n",
              "  'IF_2016': '3.964',\n",
              "  'IF_2017': '3.93',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2229',\n",
              "  'IsoAbbr': 'BMC Plant Biol.',\n",
              "  'JrId': 22326,\n",
              "  'MedAbbr': 'BMC Plant Biol',\n",
              "  'NLMid': '100967807',\n",
              "  'Titre': 'BMC Plant Biol.',\n",
              "  'abstract': 'Drought is a major constraint that leads to extensive losses to agricultural yield worldwide. The potential yield is largely determined during inflorescence development. However, to date, most investigations on plant response to drought have focused on vegetative development. This study describes the morphological changes of reproductive development and the comparison of transcriptomes under various drought conditions.|The plants grown were studied under two drought conditions: minimum for successful reproduction (45-50% soil water content, moderate drought, MD) and for survival (30-35%, severe drought, SD). MD plants can produce similar number of siliques on the main stem and similar number of seeds per silique comparing with well-water plants. The situation of SD plants was much worse than MD plants. The transcriptomes of inflorescences were further investigated at molecular level using microarrays. Our results showed more than four thousands genes with differential expression under severe drought and less than two thousand changed under moderate drought condition (with 2-fold change and q-value < 0.01). We found a group of genes with increased expression as the drought became more severe, suggesting putative adaptation to the dehydration. Interestingly, we also identified genes with alteration only under the moderate but not the severe drought condition, indicating the existence of distinct sets of genes responsive to different levels of water availability. Further cis-element analyses of the putative regulatory sequences provided more information about the underlying mechanisms for reproductive responses to drought, suggesting possible novel candidate genes that protect those developing flowers under drought stress.|Different pathways may be activated in response to moderate and severe drought in reproductive tissues, potentially helping plant to maximize its yield and balance the resource consumption between vegetative and reproductive development under dehydration stresses.',\n",
              "  'authors': ['Ma, Xuan', 'Sukiran, Noor Liyana', 'Ma, Hong', 'Su, Zhao'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'BMC plant biology',\n",
              "  'keywords': ['Abscisic Acid',\n",
              "   'Arabidopsis',\n",
              "   'Cluster Analysis',\n",
              "   'Down-Regulation',\n",
              "   'Droughts',\n",
              "   'Flowers',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Phenotype',\n",
              "   'Promoter Regions, Genetic',\n",
              "   'Reproduction',\n",
              "   'Signal Transduction',\n",
              "   'Stress, Physiological',\n",
              "   'Transcription Factors',\n",
              "   'Transcriptome',\n",
              "   'Up-Regulation'],\n",
              "  'pmid': '24928551',\n",
              "  'title': 'Moderate drought causes dramatic floral transcriptomic reprogramming to ensure successful reproductive development in Arabidopsis.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['CO'],\n",
              "  'ESSN': '1535-9484',\n",
              "  'IF_2013': '7.254',\n",
              "  'IF_2014': '6.564',\n",
              "  'IF_2015': '5.912',\n",
              "  'IF_2016': '6.54',\n",
              "  'IF_2017': '5.232',\n",
              "  'ISSN': '1535-9476',\n",
              "  'ISSN_online': '1535-9484',\n",
              "  'ISSN_print': '1535-9476',\n",
              "  'IsoAbbr': 'Mol. Cell Proteomics',\n",
              "  'JrId': 26755,\n",
              "  'MedAbbr': 'Mol Cell Proteomics',\n",
              "  'NLMid': '101125647',\n",
              "  'Titre': 'Mol. Cell Proteomics',\n",
              "  'abstract': 'Quantitative mass-spectrometry-based spatial proteomics involves elaborate, expensive, and time-consuming experimental procedures, and considerable effort is invested in the generation of such data. Multiple research groups have described a variety of approaches for establishing high-quality proteome-wide datasets. However, data analysis is as critical as data production for reliable and insightful biological interpretation, and no consistent and robust solutions have been offered to the community so far. Here, we introduce the requirements for rigorous spatial proteomics data analysis, as well as the statistical machine learning methodologies needed to address them, including supervised and semi-supervised machine learning, clustering, and novelty detection. We present freely available software solutions that implement innovative state-of-the-art analysis pipelines and illustrate the use of these tools through several case studies involving multiple organisms, experimental designs, mass spectrometry platforms, and quantitation techniques. We also propose sound analysis strategies for identifying dynamic changes in subcellular localization by comparing and contrasting data describing different biological conditions. We conclude by discussing future needs and developments in spatial proteomics data analysis. ',\n",
              "  'authors': ['Gatto, Laurent',\n",
              "   'Breckels, Lisa M',\n",
              "   'Burger, Thomas',\n",
              "   'Nightingale, Daniel J H',\n",
              "   'Groen, Arnoud J',\n",
              "   'Campbell, Callum',\n",
              "   'Nikolovski, Nino',\n",
              "   'Mulvey, Claire M',\n",
              "   'Christoforou, Andy',\n",
              "   'Ferro, Myriam',\n",
              "   'Lilley, Kathryn S'],\n",
              "  'categories': ['BIOCHEMICAL RESEARCH METHODS'],\n",
              "  'journal': 'Molecular & cellular proteomics : MCP',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Mass Spectrometry',\n",
              "   'Proteomics',\n",
              "   'Software',\n",
              "   'Sound'],\n",
              "  'pmid': '24846987',\n",
              "  'title': 'A foundation for reliable spatial proteomics data analysis.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'In this paper, we study the collaboration of perception and action representations involved in cursive letter recognition and production. We propose a mathematical formulation for the whole perception-action loop, based on probabilistic modeling and bayesian inference, which we call the Bayesian Action-Perception (BAP) model. Being a model of both perception and action processes, the purpose of this model is to study the interaction of these processes. More precisely, the model includes a feedback loop from motor production, which implements an internal simulation of movement. Motor knowledge can therefore be involved during perception tasks. In this paper, we formally define the BAP model and show how it solves the following six varied cognitive tasks using bayesian inference: i) letter recognition (purely sensory), ii) writer recognition, iii) letter production (with different effectors), iv) copying of trajectories, v) copying of letters, and vi) letter recognition (with internal simulation of movements). We present computer simulations of each of these cognitive tasks, and discuss experimental predictions and theoretical developments.',\n",
              "  'authors': ['Gilet, Estelle', 'Diard, Julien', 'Bessière, Pierre'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Bayes Theorem',\n",
              "   'Computer Simulation',\n",
              "   'Handwriting',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Movement',\n",
              "   'Online Systems',\n",
              "   'Pattern Recognition, Visual',\n",
              "   'Reading',\n",
              "   'Robotics',\n",
              "   'Semantics'],\n",
              "  'pmid': '21674043',\n",
              "  'title': 'Bayesian action-perception computational model: interaction of production and recognition of cursive letters.',\n",
              "  'year': '2011'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['YA'],\n",
              "  'ESSN': '1478-596X',\n",
              "  'IF_2013': '1.532',\n",
              "  'IF_2014': '1.526',\n",
              "  'IF_2015': '1.511',\n",
              "  'IF_2016': '1.613',\n",
              "  'IF_2017': '1.472',\n",
              "  'ISSN': '1478-5951',\n",
              "  'ISSN_online': '1478-596X',\n",
              "  'ISSN_print': '1478-5951',\n",
              "  'IsoAbbr': 'Int J Med Robot',\n",
              "  'JrId': 33305,\n",
              "  'MedAbbr': 'Int J Med Robot',\n",
              "  'NLMid': '101250764',\n",
              "  'Titre': 'Int J Med Robot',\n",
              "  'abstract': 'A pneumatically controlled balloon actuator array has been developed to provide tactile feedback to the fingers during robotic surgery.|The actuator and pneumatics were integrated onto a robotic surgical system. Potential interference of the inactive system was evaluated using a timed robotic peg transfer task. System performance was evaluated by measuring human perception of the thumb and index finger.|No significant difference was found between performance with and without the inactive mounted actuator blocks. Subjects were able to determine inflation location with > 95% accuracy and five discrete inflation levels with both the index finger and thumb with accuracies of 94% and 92%. Temporal tests revealed that an 80 ms temporal separation was sufficient to detect balloon stimuli with high accuracy.|The mounted balloon actuators successfully transmitted tactile information to the index finger and thumb, while not hindering performance of robotic surgical movements.',\n",
              "  'authors': ['Franco, Miguel L',\n",
              "   'King, Chih-Hung',\n",
              "   'Culjat, Martin O',\n",
              "   'Lewis, Catherine E',\n",
              "   'Bisley, James W',\n",
              "   'Holmes, E Carmack',\n",
              "   'Grundfest, Warren S',\n",
              "   'Dutson, Erik P'],\n",
              "  'categories': ['SURGERY'],\n",
              "  'journal': 'The international journal of medical robotics + computer assisted surgery : MRCAS',\n",
              "  'keywords': ['Equipment Design',\n",
              "   'Feedback',\n",
              "   'Humans',\n",
              "   'Robotics',\n",
              "   'Surgery, Computer-Assisted',\n",
              "   'Touch'],\n",
              "  'pmid': '19086011',\n",
              "  'title': 'An integrated pneumatic tactile feedback actuator array for robotic surgery.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2041-1723',\n",
              "  'IF_2013': '10.742',\n",
              "  'IF_2014': '11.47',\n",
              "  'IF_2015': '11.329',\n",
              "  'IF_2016': '12.124',\n",
              "  'IF_2017': '12.353',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1723',\n",
              "  'IsoAbbr': 'Nat Commun',\n",
              "  'JrId': 37539,\n",
              "  'MedAbbr': 'Nat Commun',\n",
              "  'NLMid': '101528555',\n",
              "  'Titre': 'Nat Commun',\n",
              "  'abstract': 'If a three-dimensional physical electronic system emulating synapse networks could be built, that would be a significant step toward neuromorphic computing. However, the fabrication complexity of complementary metal-oxide-semiconductor architectures impedes the achievement of three-dimensional interconnectivity, high-device density, or flexibility. Here we report flexible three-dimensional artificial chemical synapse networks, in which two-terminal memristive devices, namely, electronic synapses (e-synapses), are connected by vertically stacking crossbar electrodes. The e-synapses resemble the key features of biological synapses: unilateral connection, long-term potentiation/depression, a spike-timing-dependent plasticity learning rule, paired-pulse facilitation, and ultralow-power consumption. The three-dimensional artificial synapse networks enable a direct emulation of correlated learning and trainable memory capability with strong tolerances to input faults and variations, which shows the feasibility of using them in futuristic electronic devices and can provide a physical platform for the realization of smart memories and machine learning and for operation of the complex algorithms involving hierarchical neural networks.High-density information storage calls for the development of modern electronics with multiple stacking architectures that increase the complexity of three-dimensional interconnectivity. Here, Wu et al. build a stacked yet flexible artificial synapse network using layer-by-layer solution processing.',\n",
              "  'authors': ['Wu, Chaoxing',\n",
              "   'Kim, Tae Whan',\n",
              "   'Choi, Hwan Young',\n",
              "   'Strukov, Dmitri B',\n",
              "   'Yang, J Joshua'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Nature communications',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Electronics',\n",
              "   'Humans',\n",
              "   'Learning',\n",
              "   'Long-Term Potentiation',\n",
              "   'Memory',\n",
              "   'Models, Neurological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neuronal Plasticity',\n",
              "   'Synapses'],\n",
              "  'pmid': '28963546',\n",
              "  'title': 'Flexible three-dimensional artificial synapse networks with correlated learning and trainable memory capability.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': 'Electronic medical records (EMRs) contain a variety of valuable medical concepts and relations. The ability to recognize relations between medical concepts described in EMRs enables the automatic processing of clinical texts, resulting in an improved quality of health-related data analysis. Driven by the 2010 i2b2/VA Challenge Evaluation, the relation recognition problem in EMRs has been studied by many researchers to address this important aspect of EMR information extraction.|This paper proposes an Attention-Based Deep Residual Network (ResNet) model to recognize medical concept relations in Chinese EMRs.|Our model achieves F1-score of 77.80% on the manually annotated Chinese EMRs corpus and outperforms the state-of-the-art approaches.|The residual network-based model can reduce the negative impact of corpus noise to parameter learning, and the combination of character position attention mechanism will enhance the identification features of different type of entities.',\n",
              "  'authors': ['Zhang, Zhichang', 'Zhou, Tong', 'Zhang, Yu', 'Pang, Yali'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30961580',\n",
              "  'title': 'Attention-based deep residual learning network for entity relation extraction in Chinese EMRs.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1569-8041',\n",
              "  'IF_2013': '6.578',\n",
              "  'IF_2014': '7.04',\n",
              "  'IF_2015': '9.269',\n",
              "  'IF_2016': '11.855',\n",
              "  'IF_2017': '13.926',\n",
              "  'ISSN': '0923-7534',\n",
              "  'ISSN_online': '1569-8041',\n",
              "  'ISSN_print': '0923-7534',\n",
              "  'IsoAbbr': 'Ann. Oncol.',\n",
              "  'JrId': 1846,\n",
              "  'MedAbbr': 'Ann Oncol',\n",
              "  'NLMid': '9007735',\n",
              "  'Titre': 'Ann. Oncol.',\n",
              "  'abstract': 'This study was aimed at investigating the clinical features and outcomes of follicular lymphoma (FL) patients younger than 40 years, which have not been extensively investigated yet.|One hundred and fifty-five patients younger than 40 years were retrospectively studied from a series of 1002 FL patients diagnosed in four different European Oncology Centres (Barcelona, Spain; Bellinzona, Switzerland; London, UK; Novara, Italy) from 1985 to 2010.|Patients younger than 40 had a lower incidence of elevated LDH, high beta2-microglobulin, and a high-risk Follicular Lymphoma International Prognostic Index (FLIPI) score, whereas bone marrow involvement and bulky and disseminated lymphadenopathy were more frequent. At a median follow-up of 10 years, younger patients, in comparison with those older than 40, had significantly better overall (OS), cause-specific survival (CSS), and progression-free survival (PFS), with 10-year OS rate of 81% versus 51% (P < 0.0001), 10-year CSS rate of 82% versus 60% (P < 0.0001), and 10-year PFS of 39% versus 24% (P = 0.0098). However, there were no significant CSS and PFS differences in comparison with the patients aged 40-60. In multivariate analysis, having the lymphoma diagnosed in the last two decades and a favourable FLIPI score were associated with a significantly longer PFS and CSS in younger patients, whereas only FLIPI retained statistical significance for OS.|In our series, FL patients younger than 40 have a median OS of 24 years and their outcome seems to be improving over time. However, they still have a significantly shorter life expectancy than that of an age-matched general healthy population.',\n",
              "  'authors': ['Conconi, A',\n",
              "   'Lobetti-Bodoni, C',\n",
              "   'Montoto, S',\n",
              "   'Lopez-Guillermo, A',\n",
              "   'Coutinho, R',\n",
              "   'Matthews, J',\n",
              "   'Franceschetti, S',\n",
              "   'Bertoni, F',\n",
              "   'Moccia, A',\n",
              "   'Rancoita, P M V',\n",
              "   'Gribben, J',\n",
              "   'Cavalli, F',\n",
              "   'Gaidano, G',\n",
              "   'Lister, T A',\n",
              "   'Montserrat, E',\n",
              "   'Ghielmini, M',\n",
              "   'Zucca, E'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Annals of oncology : official journal of the European Society for Medical Oncology',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Antineoplastic Combined Chemotherapy Protocols',\n",
              "   'Disease-Free Survival',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Italy',\n",
              "   'Life Expectancy',\n",
              "   'London',\n",
              "   'Lymphoma, Follicular',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Retrospective Studies',\n",
              "   'Rituximab',\n",
              "   'Spain',\n",
              "   'Switzerland',\n",
              "   'Young Adult'],\n",
              "  'pmid': '26362567',\n",
              "  'title': 'Life expectancy of young adults with follicular lymphoma.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['DX'],\n",
              "  'ESSN': '1873-4294',\n",
              "  'IF_2013': '3.453',\n",
              "  'IF_2014': '3.402',\n",
              "  'IF_2015': '2.9',\n",
              "  'IF_2016': '2.864',\n",
              "  'IF_2017': '3.374',\n",
              "  'ISSN': '1568-0266',\n",
              "  'ISSN_online': '1873-4294',\n",
              "  'ISSN_print': '1568-0266',\n",
              "  'IsoAbbr': 'Curr Top Med Chem',\n",
              "  'JrId': 27009,\n",
              "  'MedAbbr': 'Curr Top Med Chem',\n",
              "  'NLMid': '101119673',\n",
              "  'Titre': 'Curr Top Med Chem',\n",
              "  'abstract': 'Despite significant research in understanding of neoplastic diseases, the success rate for oncology drugs is relatively very low. A major challenge before the scientific community is to design new chemical entities that will be highly selective for cancer cells so as to minimize side effects. Classification models (CMs) models play a prominent role in prediction of the biological properties of newly designed compounds before their synthesis and prevent non-optimal use of resources. Though correlation models far outnumber classification models for development of various therapeutic agents but the significance of classification models for development of anti-cancer agents can not be underestimated. Various techniques employed for development of classification models for anti-cancer activity have been briefly reviewed. Moreover, successful use of some of these classification techniques for the development of models for anti-proliferative activity has been illustrated using a data set comprising of 53 analogues of N-Benzoylated phenoxazines and phenothiazines. Resulting classification models with high degree of accuracy can play a vital role in providing lead structures for the development of novel anti-proliferative agents for cancer chemotherapy.',\n",
              "  'authors': ['Dutt, R', 'Madan, A K'],\n",
              "  'categories': ['CHEMISTRY, MEDICINAL'],\n",
              "  'journal': 'Current topics in medicinal chemistry',\n",
              "  'keywords': ['Antineoplastic Agents',\n",
              "   'Computer-Aided Design',\n",
              "   'Decision Trees',\n",
              "   'Drug Design',\n",
              "   'Humans',\n",
              "   'Inhibitory Concentration 50',\n",
              "   'K562 Cells',\n",
              "   'Neoplasms',\n",
              "   'Oxazines',\n",
              "   'Phenothiazines',\n",
              "   'Principal Component Analysis',\n",
              "   'Structure-Activity Relationship',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '23368098',\n",
              "  'title': 'Classification models for anticancer activity.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': 'With our single-wavelength spectral-imaging-based Thai jasmine rice identification system, we emphasize here that a combination of an appropriate polynomial fitting function on the determined chain code and a well-trained neural network configuration is highly sufficient in achieving a low false acceptance rate (FAR) and a low false rejection rate (FRR). Experimental demonstration shows promising results in identifying our desired Thai jasmine rice from six unwanted rice varieties with FAR and FRR values of 6.2% and 7.1%, respectively. Additional key performances include a much faster identification time of 30.5 s, chemical-free analysis, robustness, and adaptive learning.',\n",
              "  'authors': ['Suwansukho, Kajpanya',\n",
              "   'Sumriddetchkajorn, Sarun',\n",
              "   'Buranasiri, Prathan'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Food Analysis',\n",
              "   'Food Technology',\n",
              "   'Microscopy, Fluorescence',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Numerical Analysis, Computer-Assisted',\n",
              "   'Oryza',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Spectrometry, Fluorescence'],\n",
              "  'pmid': '24787182',\n",
              "  'title': 'Fast fluorescent imaging-based Thai jasmine rice identification with polynomial fitting function and neural network analysis.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1827-1758',\n",
              "  'IF_2013': '0.7',\n",
              "  'IF_2014': '0.965',\n",
              "  'IF_2015': '0.536',\n",
              "  'IF_2016': '0.984',\n",
              "  'IF_2017': '1.449',\n",
              "  'ISSN': '0393-2249',\n",
              "  'ISSN_online': '1827-1758',\n",
              "  'ISSN_print': '0393-2249',\n",
              "  'IsoAbbr': 'Minerva Urol Nefrol',\n",
              "  'JrId': 7387,\n",
              "  'MedAbbr': 'Minerva Urol Nefrol',\n",
              "  'NLMid': '8503649',\n",
              "  'Titre': 'Minerva Urol Nefrol',\n",
              "  'abstract': 'Robot assisted laparoscopic prostatectomy (RALP) has become a widely accepted and applied surgical method of localized prostate cancer treatment. Initial studies suggest a shorter learning curve for RALP when compared to conventional laparoscopic prostatectomy. Although dexterity for the RALP procedure is increased by the da Vinci surgical robotic system compared to laparoscopic approaches, the anatomical and technical approaches to prostatectomy still require considerable experience in anatomical variations to become proficient and improve oncological and surgical approaches. Several aspects with respect to that can be recognized in the early phases of training. The following aspects helped the author in his initial 150 cases to improve surgical skills: use intraoperative transrectal ultrasound for bladder neck recognition, record and review surgical procedures, experiment with port positioning, training of table assistance. The more recent da Vinci surgical robot systems allow for new dimensions in surgical approaches with particular intraoperative imaging modalities not earlier so easy accessible during surgery. These properties render robot procedures appealing for the now-a-days more and more image-guided approach to surgery.',\n",
              "  'authors': ['Van Der Poel, H G', 'Van Muilekom, E', 'De Blok, W'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Minerva urologica e nefrologica = The Italian journal of urology and nephrology',\n",
              "  'keywords': ['Erectile Dysfunction',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Urinary Incontinence'],\n",
              "  'pmid': '19816388',\n",
              "  'title': 'Robot assisted laparoscopic prostatectomy: initial tips and tricks.',\n",
              "  'year': '2009'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1464-410X',\n",
              "  'IF_2013': '3.13',\n",
              "  'IF_2014': '3.533',\n",
              "  'IF_2015': '4.387',\n",
              "  'IF_2016': '4.439',\n",
              "  'IF_2017': '4.688',\n",
              "  'ISSN': '1464-4096',\n",
              "  'ISSN_online': '1464-410X',\n",
              "  'ISSN_print': '1464-4096',\n",
              "  'IsoAbbr': 'BJU Int.',\n",
              "  'JrId': 21363,\n",
              "  'MedAbbr': 'BJU Int',\n",
              "  'NLMid': '100886721',\n",
              "  'Titre': 'BJU Int.',\n",
              "  'abstract': 'To report the management of urachal anomalies using a robotically assisted approach.|Between January 2005 and February 2006, five patients (mean age 51 years, range 24-68) were diagnosed with urachal anomalies. Two basic robot-assisted surgical approaches were used for excising the urachal anomalies: excision of the urachal remnant via partial cystectomy, and radical cystectomy for excision of urachal adenocarcinoma.|All five cases were successful and the excised specimens were assessed histologically. The short-term oncological outcome in the three patients with histologically confirmed moderately differentiated adenocarcinoma showed no evidence of recurrent disease within a median interval of 8 months. Surveillance follow-up cystoscopy in the patients who had a partial cystectomy showed a well-healed bladder mucosa with no evidence of recurrence.|Radical excision of the urachal tract with partial cystectomy or radical cystectomy using the da Vinci robot is safe, effective and technically feasible.',\n",
              "  'authors': ['Madeb, Ralph',\n",
              "   'Knopf, Joy K',\n",
              "   'Nicholson, Craig',\n",
              "   'Donahue, Laurence A',\n",
              "   'Adcock, Brian',\n",
              "   'Dever, David',\n",
              "   'Tan, Beng Jit',\n",
              "   'Valvo, John R',\n",
              "   'Eichel, Louis'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'BJU international',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Cystectomy',\n",
              "   'Feasibility Studies',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Robotics',\n",
              "   'Treatment Outcome',\n",
              "   'Urachal Cyst',\n",
              "   'Urachus'],\n",
              "  'pmid': '16978280',\n",
              "  'title': 'The use of robotically assisted surgery for treating urachal anomalies.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['RU'],\n",
              "  'ESSN': '1873-7544',\n",
              "  'IF_2013': '3.327',\n",
              "  'IF_2014': '3.357',\n",
              "  'IF_2015': '3.231',\n",
              "  'IF_2016': '3.277',\n",
              "  'IF_2017': '3.382',\n",
              "  'ISSN': '0306-4522',\n",
              "  'ISSN_online': '1873-7544',\n",
              "  'ISSN_print': '0306-4522',\n",
              "  'IsoAbbr': 'Neuroscience',\n",
              "  'JrId': 6087,\n",
              "  'MedAbbr': 'Neuroscience',\n",
              "  'NLMid': '7605074',\n",
              "  'Titre': 'Neuroscience',\n",
              "  'abstract': 'It is currently believed that the mechanisms underlying spindle oscillations are related to those that generate spike and wave (SW) discharges. The mechanisms of transition between these two types of activity, however, are not well understood. In order to provide more insight into the dynamics of the neuronal networks leading to seizure generation in a rat experimental model of absence epilepsy we developed a computational model of thalamo-cortical circuits based on relevant (patho)physiological data. The model is constructed at the macroscopic level since this approach allows to investigate dynamical properties of the system and the role played by different mechanisms in the process of seizure generation, both at short and long time scales. The main results are the following: (i) SW discharges represent dynamical bifurcations that occur in a bistable neuronal network; (ii) the durations of paroxysmal and normal epochs have exponential distributions, indicating that transitions between these two stable states occur randomly over time with constant probabilities; (iii) the probabilistic nature of the onset of paroxysmal activity implies that it is not possible to predict its occurrence; (iv) the bistable nature of the dynamical system allows that an ictal state may be aborted by a single counter-stimulus.',\n",
              "  'authors': ['Suffczynski, P', 'Kalitzin, S', 'Lopes Da Silva, F H'],\n",
              "  'categories': ['NEUROSCIENCES'],\n",
              "  'journal': 'Neuroscience',\n",
              "  'keywords': ['Animals',\n",
              "   'Disease Models, Animal',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Rats',\n",
              "   'Rats, Wistar',\n",
              "   'Status Epilepticus',\n",
              "   'Thalamus'],\n",
              "  'pmid': '15207365',\n",
              "  'title': 'Dynamics of non-convulsive epileptic phenomena modeled by a bistable neuronal network.',\n",
              "  'year': '2004'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1471-2229',\n",
              "  'IF_2013': '3.942',\n",
              "  'IF_2014': '3.813',\n",
              "  'IF_2015': '3.631',\n",
              "  'IF_2016': '3.964',\n",
              "  'IF_2017': '3.93',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1471-2229',\n",
              "  'IsoAbbr': 'BMC Plant Biol.',\n",
              "  'JrId': 22326,\n",
              "  'MedAbbr': 'BMC Plant Biol',\n",
              "  'NLMid': '100967807',\n",
              "  'Titre': 'BMC Plant Biol.',\n",
              "  'abstract': 'The ginseng (Panax ginseng C.A. Meyer) is a perennial herbaceous plant that has been used in traditional oriental medicine for thousands of years. Ginsenosides, which have significant pharmacological effects on human health, are the foremost bioactive constituents in this plant. Having realized the importance of this plant to humans, an integrated omics resource becomes indispensable to facilitate genomic research, molecular breeding and pharmacological study of this herb.|The first draft genome sequences of P. ginseng cultivar \"Chunpoong\" were reported recently. Here, using the draft genome, transcriptome, and functional annotation datasets of P. ginseng, we have constructed the Ginseng Genome Database http://ginsengdb.snu.ac.kr /, the first open-access platform to provide comprehensive genomic resources of P. ginseng. The current version of this database provides the most up-to-date draft genome sequence (of approximately 3000 Mbp of scaffold sequences) along with the structural and functional annotations for 59,352 genes and digital expression of genes based on transcriptome data from different tissues, growth stages and treatments. In addition, tools for visualization and the genomic data from various analyses are provided. All data in the database were manually curated and integrated within a user-friendly query page.|This database provides valuable resources for a range of research fields related to P. ginseng and other species belonging to the Apiales order as well as for plant research communities in general. Ginseng genome database can be accessed at http://ginsengdb.snu.ac.kr /.',\n",
              "  'authors': ['Jayakodi, Murukarthick',\n",
              "   'Choi, Beom-Soon',\n",
              "   'Lee, Sang-Choon',\n",
              "   'Kim, Nam-Hoon',\n",
              "   'Park, Jee Young',\n",
              "   'Jang, Woojong',\n",
              "   'Lakshmanan, Meiyappan',\n",
              "   'Mohan, Shobhana V G',\n",
              "   'Lee, Dong-Yup',\n",
              "   'Yang, Tae-Jin'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'BMC plant biology',\n",
              "  'keywords': ['Databases, Genetic',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genome, Plant',\n",
              "   'Ginsenosides',\n",
              "   'Panax'],\n",
              "  'pmid': '29649979',\n",
              "  'title': 'Ginseng Genome Database: an open-access platform for genomics of Panax ginseng.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': 'Three-dimensional high-spatial-resolution angiograms of the coronary arteries were acquired with an electrocardiogram-gated three-dimensional rotational angiography technique on an interventional X-ray system.|During selective injection of contrast material in the left and right coronary artery, projection images were obtained in eight pigs during a continuous rotation of the X-ray gantry over an angular range of 180 degrees within 8 seconds.|Three-dimensional tomographic reconstruction depicted the proximal, medial, and distal sections of the main arteries as well as the main bifurcations in multiple cardiac phases in all animals.|For the first time, this feasibility study shows that a three-dimensional angiogram of the coronary arteries can be obtained intraprocedurally in a conventional interventional suite by means of tomographic reconstruction from projection images.',\n",
              "  'authors': ['Rasche, Volker',\n",
              "   'Movassaghi, Babak',\n",
              "   'Grass, Michael',\n",
              "   'Schäfer, Dirk',\n",
              "   'Kühl, Harald P',\n",
              "   'Günther, Rolf W',\n",
              "   'Bücker, Arno'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Animals',\n",
              "   'Artificial Intelligence',\n",
              "   'Coronary Angiography',\n",
              "   'Feasibility Studies',\n",
              "   'Imaging, Three-Dimensional',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Radiographic Image Enhancement',\n",
              "   'Radiographic Image Interpretation, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Swine'],\n",
              "  'pmid': '16627206',\n",
              "  'title': 'Three-dimensional X-ray coronary angiography in the porcine model: A feasibility study.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '2041-3033',\n",
              "  'IF_2013': '1.144',\n",
              "  'IF_2014': '1.329',\n",
              "  'IF_2015': '0.996',\n",
              "  'IF_2016': '1.005',\n",
              "  'IF_2017': '1.124',\n",
              "  'ISSN': '0954-4119',\n",
              "  'ISSN_online': '2041-3033',\n",
              "  'ISSN_print': '0954-4119',\n",
              "  'IsoAbbr': 'Proc Inst Mech Eng H',\n",
              "  'JrId': 1315,\n",
              "  'MedAbbr': 'Proc Inst Mech Eng H',\n",
              "  'NLMid': '8908934',\n",
              "  'Titre': 'Proc Inst Mech Eng H',\n",
              "  'abstract': 'This article presents a virtual prototyping study of a semi-active lower limb prosthesis to improve the functionality of an amputee during prosthesis-environment interaction for level ground walking. Articulated ankle-foot prosthesis and a single-axis semi-active prosthetic knee with active and passive operating modes were considered. Data for level ground walking were collected using a photogrammetric method in order to develop a base-line simulation model and with the hip kinematics input to verify the proposed design. The simulated results show that the semi-active lower limb prosthesis is able to move efficiently in passive mode, and the activation time of the knee actuator can be reduced by approximately 50%. Therefore, this semi-active system has the potential to reduce the energy consumption of the actuators required during level ground walking and requires less compensation from the amputee due to lower deviation of the vertical excursion of body centre of mass. ',\n",
              "  'authors': ['Lui, Zhen Wei',\n",
              "   'Awad, Mohammed I',\n",
              "   'Abouhossein, Alireza',\n",
              "   'Dehghani-Sanij, Abbas A',\n",
              "   'Messenger, Neil'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Proceedings of the Institution of Mechanical Engineers. Part H, Journal of engineering in medicine',\n",
              "  'keywords': ['Adult',\n",
              "   'Artificial Limbs',\n",
              "   'Biomechanical Phenomena',\n",
              "   'Computer-Aided Design',\n",
              "   'Gait',\n",
              "   'Humans',\n",
              "   'Leg',\n",
              "   'Male',\n",
              "   'Photogrammetry',\n",
              "   'Prosthesis Design',\n",
              "   'Robotics',\n",
              "   'Walking'],\n",
              "  'pmid': '25991714',\n",
              "  'title': 'Virtual prototyping of a semi-active transfemoral prosthetic leg.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'E',\n",
              "  'Disciplines': ['BA'],\n",
              "  'ESSN': '1432-055X',\n",
              "  'IF_2013': '0.743',\n",
              "  'IF_2014': '0.757',\n",
              "  'IF_2015': '0.964',\n",
              "  'IF_2016': '1.039',\n",
              "  'IF_2017': '0.995',\n",
              "  'ISSN': '0003-2417',\n",
              "  'ISSN_online': '1432-055X',\n",
              "  'ISSN_print': '0003-2417',\n",
              "  'IsoAbbr': 'Anaesthesist',\n",
              "  'JrId': 517,\n",
              "  'MedAbbr': 'Anaesthesist',\n",
              "  'NLMid': '0370525',\n",
              "  'Titre': 'Anaesthesist',\n",
              "  'abstract': 'The improved drug therapy leads to increasingly older patients with complex comorbidities in the discipline of operative urology. Today, improved technical equipment provides new operational capabilities in the field of urology. The prone and lithotomy position during surgery leads to physiological changes that affect anesthesia management. The surgical risk of procedures such as transurethral surgery of the prostate or bladder is being altered by laser surgery and other new technologies. Although the incidence of transurethral resection (TUR) syndrome has been reduced in recent years, the intrusion of irrigation fluid still has to be considered during anesthesia. Robot-assisted surgery has successfully completed the experimental stage and is widely used so that new targets have to be challenged. Ureterorenoscopy is performed with flexible, small caliber ureteroscopes which even allow treatment of renal calculi under analgosedation within short time periods. Percutaneous nephrostomy and litholapaxy are still frequently performed in the prone position. With respect to the risks arising from patient positioning, supine or lateral positioning should be considered in individual cases. A good communication between the surgeon and anesthetist allows deviation from daily routine procedures if special indications require a modified approach. In conclusion, a profound knowledge of the (patho-)physiology of general anesthesia and endourological diseases enables anesthetists to provide a prospective type anesthesia, which should prevent the occurrence of life-threatening incidents.',\n",
              "  'authors': ['Kiss, T', 'Bluth, T', 'Heller, A'],\n",
              "  'categories': ['ANESTHESIOLOGY'],\n",
              "  'journal': 'Der Anaesthesist',\n",
              "  'keywords': ['Aged',\n",
              "   'Anesthesia, General',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Male',\n",
              "   'Postoperative Complications',\n",
              "   'Posture',\n",
              "   'Robotics',\n",
              "   'Transurethral Resection of Prostate',\n",
              "   'Ureteral Obstruction',\n",
              "   'Ureteroscopy',\n",
              "   'Urinary Bladder Neoplasms',\n",
              "   'Urologic Surgical Procedures',\n",
              "   'Urologic Surgical Procedures, Male'],\n",
              "  'pmid': '22875058',\n",
              "  'title': '[Anesthesia in endourological and robot-assisted interventions].',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DM'],\n",
              "  'ESSN': '1538-7445',\n",
              "  'IF_2013': '9.284',\n",
              "  'IF_2014': '9.329',\n",
              "  'IF_2015': '8.556',\n",
              "  'IF_2016': '9.122',\n",
              "  'IF_2017': '9.13',\n",
              "  'ISSN': '0008-5472',\n",
              "  'ISSN_online': '1538-7445',\n",
              "  'ISSN_print': '0008-5472',\n",
              "  'IsoAbbr': 'Cancer Res.',\n",
              "  'JrId': 2786,\n",
              "  'MedAbbr': 'Cancer Res',\n",
              "  'NLMid': '2984705R',\n",
              "  'Titre': 'Cancer Res.',\n",
              "  'abstract': 'To investigate the phenotype associated with estrogen receptor alpha (ER) expression in breast carcinoma, gene expression profiles of 58 node-negative breast carcinomas discordant for ER status were determined using DNA microarray technology. Using artificial neural networks as well as standard hierarchical clustering techniques, the tumors could be classified according to ER status, and a list of genes which discriminate tumors according to ER status was generated. The artificial neural networks could accurately predict ER status even when excluding top discriminator genes, including ER itself. By reference to the serial analysis of gene expression database, we found that only a small proportion of the 100 most important ER discriminator genes were also regulated by estradiol in MCF-7 cells. The results provide evidence that ER+ and ER- tumors display remarkably different gene-expression phenotypes not solely explained by differences in estrogen responsiveness.',\n",
              "  'authors': ['Gruvberger, S',\n",
              "   'Ringnér, M',\n",
              "   'Chen, Y',\n",
              "   'Panavally, S',\n",
              "   'Saal, L H',\n",
              "   'Borg A, M',\n",
              "   'Fernö, C',\n",
              "   'Peterson, P S',\n",
              "   'Meltzer, S'],\n",
              "  'categories': ['ONCOLOGY'],\n",
              "  'journal': 'Cancer research',\n",
              "  'keywords': ['Breast Neoplasms',\n",
              "   'Calibration',\n",
              "   'Cell Lineage',\n",
              "   'Cluster Analysis',\n",
              "   'Estradiol',\n",
              "   'Estrogen Receptor alpha',\n",
              "   'Female',\n",
              "   'Gene Expression',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Neoplastic',\n",
              "   'Humans',\n",
              "   'Models, Genetic',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Oligonucleotide Array Sequence Analysis',\n",
              "   'Phenotype',\n",
              "   'Receptors, Estrogen',\n",
              "   'Reproducibility of Results',\n",
              "   'Tumor Cells, Cultured'],\n",
              "  'pmid': '11507038',\n",
              "  'title': 'Estrogen receptor status in breast cancer is associated with remarkably distinct gene expression patterns.',\n",
              "  'year': '2001'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DE'],\n",
              "  'ESSN': '1460-2431',\n",
              "  'IF_2013': '5.794',\n",
              "  'IF_2014': '5.526',\n",
              "  'IF_2015': '5.677',\n",
              "  'IF_2016': '5.83',\n",
              "  'IF_2017': '5.354',\n",
              "  'ISSN': '0022-0957',\n",
              "  'ISSN_online': '1460-2431',\n",
              "  'ISSN_print': '0022-0957',\n",
              "  'IsoAbbr': 'J. Exp. Bot.',\n",
              "  'JrId': 8651,\n",
              "  'MedAbbr': 'J Exp Bot',\n",
              "  'NLMid': '9882906',\n",
              "  'Titre': 'J. Exp. Bot.',\n",
              "  'abstract': 'Leaf senescence is a developmentally controlled process, which is additionally modulated by a number of adverse environmental conditions. Nitrogen shortage is a well-known trigger of precocious senescence in many plant species including crops, generally limiting biomass and seed yield. However, leaf senescence induced by nitrogen starvation may be reversed when nitrogen is resupplied at the onset of senescence. Here, the transcriptomic, hormonal, and global metabolic rearrangements occurring during nitrogen resupply-induced reversal of senescence in Arabidopsis thaliana were analysed. The changes induced by senescence were essentially in keeping with those previously described; however, these could, by and large, be reversed. The data thus indicate that plants undergoing senescence retain the capacity to sense and respond to the availability of nitrogen nutrition. The combined data are discussed in the context of the reversibility of the senescence programme and the evolutionary benefit afforded thereby. Future prospects for understanding and manipulating this process in both Arabidopsis and crop plants are postulated. ',\n",
              "  'authors': ['Balazadeh, Salma',\n",
              "   'Schildhauer, Jörg',\n",
              "   'Araújo, Wagner L',\n",
              "   'Munné-Bosch, Sergi',\n",
              "   'Fernie, Alisdair R',\n",
              "   'Proost, Sebastian',\n",
              "   'Humbeck, Klaus',\n",
              "   'Mueller-Roeber, Bernd'],\n",
              "  'categories': ['PLANT SCIENCES'],\n",
              "  'journal': 'Journal of experimental botany',\n",
              "  'keywords': ['Arabidopsis',\n",
              "   'Base Sequence',\n",
              "   'Botrytis',\n",
              "   'Cluster Analysis',\n",
              "   'Darkness',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Metabolome',\n",
              "   'Molecular Sequence Data',\n",
              "   'Nitrogen',\n",
              "   'Nucleotide Motifs',\n",
              "   'Plant Growth Regulators',\n",
              "   'Plant Leaves',\n",
              "   'RNA, Messenger',\n",
              "   'Transcriptome'],\n",
              "  'pmid': '24692653',\n",
              "  'title': 'Reversal of senescence by N resupply to N-starved Arabidopsis thaliana: transcriptomic and metabolomic consequences.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['FF'],\n",
              "  'ESSN': '1097-6760',\n",
              "  'IF_2013': '4.333',\n",
              "  'IF_2014': '4.676',\n",
              "  'IF_2015': '5.008',\n",
              "  'IF_2016': '5.352',\n",
              "  'IF_2017': '4.68',\n",
              "  'ISSN': '0196-0644',\n",
              "  'ISSN_online': '1097-6760',\n",
              "  'ISSN_print': '0196-0644',\n",
              "  'IsoAbbr': 'Ann Emerg Med',\n",
              "  'JrId': 557,\n",
              "  'MedAbbr': 'Ann Emerg Med',\n",
              "  'NLMid': '8002646',\n",
              "  'Titre': 'Ann Emerg Med',\n",
              "  'abstract': 'Clinical and ECG data from presentation are highly discriminatory for diagnosis of acute coronary syndromes, whereas definitive diagnosis from serial ECG and cardiac marker protein measurements is usually not available for several hours. Artificial neural networks are computer programs adept at pattern recognition tasks and have been used to analyze data from chest pain patients with a view to developing diagnostic algorithms that might improve triage practices in the emergency department. The aim of this study is to develop and optimize artificial neural network models for diagnosis of acute coronary syndrome, to test these models on data collected prospectively from different centers, and to establish whether the performance of these models was superior to that of models derived using a standard statistical technique, logistic regression.|The study used data from 3,147 patients presenting to 3 hospitals with acute chest pain. Data from hospital 1 were used to train the models, which were then tested on independent data from the other 2 hospitals. From 40 potential factors, variables were selected according to the logarithm of their likelihood ratios to produce models using 8, 13, 20, and 40 factors. Identical data were used for logistic regression and artificial neural network models. Calibration and performance were assessed, the latter using receiver operating characteristic (ROC) curve analysis.|Although the performance of artificial neural network models generally increased with increasing numbers of factors, this was insignificant. The 13-factor model was therefore used for the rest of the study owing to its marginally improved calibration over the smallest model. Area under the ROC curve (with standard error) was 0.97 (0.006). The overall sensitivity and specificity of this model for acute coronary syndrome diagnosis using the training data was 0.93. ROC curves for logistic regression and artificial neural network models applied to data from the 3 hospitals were identical. For the 13-factor artificial neural network model tested on data from hospitals 2 and 3, area under the ROC curves (standard error) were 0.93 (0.006) and 0.95 (0.009), respectively. Investigation of the performance of the artificial neural network models throughout the range of predicted probabilities showed that they were well calibrated.|This study confirms that artificial neural networks can offer a useful approach for developing diagnostic algorithms for chest pain patients; however, the exceptional performance and simplicity of the logistic model militates in favor of logistic regression for the present task. Our artificial neural network models were well calibrated and performed well on unseen data from different centers. These issues have not been addressed in previous studies. However, and unlike in previous studies, we did not find the performance of artificial neural network models to be significantly different from that of suitably optimized logistic regression models.',\n",
              "  'authors': ['Harrison, Robert F', 'Kennedy, R Lee'],\n",
              "  'categories': ['EMERGENCY MEDICINE'],\n",
              "  'journal': 'Annals of emergency medicine',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Chest Pain',\n",
              "   'Cohort Studies',\n",
              "   'Coronary Disease',\n",
              "   'Electrocardiography',\n",
              "   'Emergency Medicine',\n",
              "   'Emergency Service, Hospital',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Likelihood Functions',\n",
              "   'Logistic Models',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Process Assessment (Health Care)',\n",
              "   'Prospective Studies',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Syndrome',\n",
              "   'United Kingdom'],\n",
              "  'pmid': '16271675',\n",
              "  'title': 'Artificial neural network models for prediction of acute coronary syndromes using clinical data from the time of presentation.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['TM'],\n",
              "  'ESSN': '1472-4146',\n",
              "  'IF_2013': '2.551',\n",
              "  'IF_2014': '2.915',\n",
              "  'IF_2015': '2.912',\n",
              "  'IF_2016': '2.687',\n",
              "  'IF_2017': '2.894',\n",
              "  'ISSN': '0021-9746',\n",
              "  'ISSN_online': '1472-4146',\n",
              "  'ISSN_print': '0021-9746',\n",
              "  'IsoAbbr': 'J. Clin. Pathol.',\n",
              "  'JrId': 4642,\n",
              "  'MedAbbr': 'J Clin Pathol',\n",
              "  'NLMid': '0376601',\n",
              "  'Titre': 'J. Clin. Pathol.',\n",
              "  'abstract': 'Red blood cell (RBC) lysis resistance interferes with white blood cell (WBC) count and differential; still, its detection relies on the identification of an abnormal scattergram, and this is not clearly adverted by specific flags in the Beckman-Coulter DXH-800. The aims were to analyse precisely the effect of RBC lysis resistance interference in WBC counts, differentials and cell population data (CPD) and then to design, develop and implement a novel diagnostic machine learning (ML) model to optimise the detection of samples presenting this phenomenon.|WBC counts, differentials and CPD from 232 patients (anaemia or liver disease) were compared with 100 healthy controls (HC) using analysis of variance. The data were analysed after a corrective action, and the analyser differentials were also compared with the digital leucocyte differentials. The ML support vector machine (SVM) algorithm was trained with 70% of the samples (n=233) and the 30% remaining (n=99) were employed exclusively during the validation phase.|We identified that impedance WBC was not affected by the RBC lysis resistance interference while the DXH-800 differentials overestimated lymphoid subpopulations (17.6%), sometimes even yielding spurious lymphocytosis, and the latter were corrected when sample dilution was performed. The ML-SVM algorithm allowed the classification of the pathological groups when compared with HC with validation accuracies corresponding to 97.98%, 100% and 88.78% for the global, anaemia and liver disease groups, respectively.|The proposed algorithm has an impressive discriminatory potential and its application would be a valuable support system to detect spurious results due to RBC lysis resistance.',\n",
              "  'authors': ['Bigorra, Laura',\n",
              "   'Larriba, Iciar',\n",
              "   'Gutiérrez-Gallego, Ricardo'],\n",
              "  'categories': ['PATHOLOGY'],\n",
              "  'journal': 'Journal of clinical pathology',\n",
              "  'keywords': ['Anemia',\n",
              "   'Automation, Laboratory',\n",
              "   'Case-Control Studies',\n",
              "   'Erythrocytes',\n",
              "   'Hemolysis',\n",
              "   'Humans',\n",
              "   'Leukocyte Count',\n",
              "   'Leukocytes',\n",
              "   'Light',\n",
              "   'Liver Diseases',\n",
              "   'Machine Learning',\n",
              "   'Predictive Value of Tests',\n",
              "   'Reproducibility of Results',\n",
              "   'Scattering, Radiation'],\n",
              "  'pmid': '30992342',\n",
              "  'title': 'Machine learning algorithms for the detection of spurious white blood cell differentials due to erythrocyte lysis resistance.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1523-1755',\n",
              "  'IF_2013': '8.52',\n",
              "  'IF_2014': '8.563',\n",
              "  'IF_2015': '7.683',\n",
              "  'IF_2016': '8.395',\n",
              "  'IF_2017': '8.429',\n",
              "  'ISSN': '0085-2538',\n",
              "  'ISSN_online': '1523-1755',\n",
              "  'ISSN_print': '0085-2538',\n",
              "  'IsoAbbr': 'Kidney Int.',\n",
              "  'JrId': 5428,\n",
              "  'MedAbbr': 'Kidney Int',\n",
              "  'NLMid': '0323470',\n",
              "  'Titre': 'Kidney Int.',\n",
              "  'abstract': \"In clinical trials, equation 7 from the Modification of Diet in Renal Disease (MDRD) Study is the most accurate formula for the prediction of glomerular filtration rate (GFR) from serum creatinine. An alternative approach has been developed using evolving connectionist systems (ECOS), which are novel computing structures that can be trained to generate accurate output from a given set of input variables. This study aims to compare the prediction errors associated with each method, using data that reproduce routine clinical practice as opposed to the artificial setting of clinical trials.|The methods were compared using 441 radioisotope measurements of GFR in 178 chronic kidney disease patients from 12 centers in Australia and New Zealand. All clinical and laboratory measurements were obtained from the patients' center rather than central laboratories, as would be the case in routine clinical practice. Both the MDRD formula and ECOS used the same predictive variables, and both were optimized to the study cohort by stepwise regression and training, respectively.|Mean measured GFR in the cohort was 22.6 mL/min/1.73 m(2). The bias and precision of the MDRD formula were -3.5 mL/min/1.73 m(2) and 34.5%, respectively, improving to -1.2 mL/min/1.73 m(2) and 31.1% after maximal optimization of the formula to study data. The bias and precision of the ECOS were 0.7 mL/min/1.73 m(2) and 32.6%, respectively, improving to -0.1 mL/min/1.73 m(2) and 16.6% after maximal optimization of the system to study data. The prediction of GFR using ECOS was improved by accounting for the center from where clinical and laboratory measurements originated within the connectionist model.|Algebraic formulas will be associated with greater prediction error in routine clinical practice than in the original trials, and machine intelligence is more likely to predict GFR accurately in this setting.\",\n",
              "  'authors': ['Marshall, Mark Roger',\n",
              "   'Song, Qun',\n",
              "   'Ma, Tian Min',\n",
              "   'MacDonell, Stephen Gerard',\n",
              "   'Kasabov, Nikola Kirilov'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Kidney international',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Creatinine',\n",
              "   'Edetic Acid',\n",
              "   'Female',\n",
              "   'Glomerular Filtration Rate',\n",
              "   'Humans',\n",
              "   'Kidney Function Tests',\n",
              "   'Male',\n",
              "   'Mathematics',\n",
              "   'Middle Aged',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Prospective Studies'],\n",
              "  'pmid': '15840042',\n",
              "  'title': 'Evolving connectionist system versus algebraic formulas for prediction of renal function from serum creatinine.',\n",
              "  'year': '2005'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'Determination of the number of significant clusters in the sample represents a very important problem. It is expected that the outcome of clustering under a broad class of loss functions will be more stable if the correct number of clusters is used. In order to illustrate the model of universal clustering we consider 1) family of power loss functions in probabilistic space; 2) family of exponential loss functions in Euclidean space. The proposed model is general, and proved to be effective in application to the synthetic datasets.',\n",
              "  'authors': ['Nikulin, Vladimir'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Models, Statistical',\n",
              "   'Neural Networks (Computer)'],\n",
              "  'pmid': '17117494',\n",
              "  'title': 'On the universal clustering under a broad class of loss functions.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1094-4087',\n",
              "  'IF_2013': '3.525',\n",
              "  'IF_2014': '3.488',\n",
              "  'IF_2015': '3.148',\n",
              "  'IF_2016': '3.307',\n",
              "  'IF_2017': '3.356',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1094-4087',\n",
              "  'IsoAbbr': 'Opt Express',\n",
              "  'JrId': 29989,\n",
              "  'MedAbbr': 'Opt Express',\n",
              "  'NLMid': '101137103',\n",
              "  'Titre': 'Opt Express',\n",
              "  'abstract': 'Fourier ptychographic microscopy allows for the collection of images with a high space-bandwidth product at the cost of temporal resolution. In Fourier ptychographic microscopy, the light source of a conventional widefield microscope is replaced with a light-emitting diode (LED) matrix, and multiple images are collected with different LED illumination patterns. From these images, a higher-resolution image can be computationally reconstructed without sacrificing field-of-view. We use deep learning to achieve single-shot imaging without sacrificing the space-bandwidth product, reducing the acquisition time in Fourier ptychographic microscopy by a factor of 69. In our deep learning approach, a training dataset of high-resolution images is used to jointly optimize a single LED illumination pattern with the parameters of a reconstruction algorithm. Our work paves the way for high-throughput imaging in biological studies.',\n",
              "  'authors': ['Cheng, Yi Fei',\n",
              "   'Strachan, Megan',\n",
              "   'Weiss, Zachary',\n",
              "   'Deb, Moniher',\n",
              "   'Carone, Dawn',\n",
              "   'Ganapati, Vidya'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Optics express',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30696147',\n",
              "  'title': 'Illumination pattern design with deep learning for single-shot Fourier ptychographic microscopy.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': \"Integral to the mission of the National Institutes of Health-sponsored Lung Imaging Database Consortium is the accurate definition of the spatial location of pulmonary nodules. Because the majority of small lung nodules are not resected, a reference standard from histopathology is generally unavailable. Thus assessing the source of variability in defining the spatial location of lung nodules by expert radiologists using different software tools as an alternative form of truth is necessary.|The relative differences in performance of six radiologists each applying three annotation methods to the task of defining the spatial extent of 23 different lung nodules were evaluated. The variability of radiologists' spatial definitions for a nodule was measured using both volumes and probability maps (p-map). Results were analyzed using a linear mixed-effects model that included nested random effects.|Across the combination of all nodules, volume and p-map model parameters were found to be significant at P < .05 for all methods, all radiologists, and all second-order interactions except one. The radiologist and methods variables accounted for 15% and 3.5% of the total p-map variance, respectively, and 40.4% and 31.1% of the total volume variance, respectively.|Radiologists represent the major source of variance as compared with drawing tools independent of drawing metric used. Although the random noise component is larger for the p-map analysis than for volume estimation, the p-map analysis appears to have more power to detect differences in radiologist-method combinations. The standard deviation of the volume measurement task appears to be proportional to nodule volume.\",\n",
              "  'authors': ['Meyer, Charles R',\n",
              "   'Johnson, Timothy D',\n",
              "   'McLennan, Geoffrey',\n",
              "   'Aberle, Denise R',\n",
              "   'Kazerooni, Ella A',\n",
              "   'Macmahon, Heber',\n",
              "   'Mullan, Brian F',\n",
              "   'Yankelevitz, David F',\n",
              "   'van Beek, Edwin J R',\n",
              "   'Armato, Samuel G',\n",
              "   'McNitt-Gray, Michael F',\n",
              "   'Reeves, Anthony P',\n",
              "   'Gur, David',\n",
              "   'Henschke, Claudia I',\n",
              "   'Hoffman, Eric A',\n",
              "   'Bland, Peyton H',\n",
              "   'Laderach, Gary',\n",
              "   'Pais, Richie',\n",
              "   'Qing, David',\n",
              "   'Piker, Chris',\n",
              "   'Guo, Junfeng',\n",
              "   'Starkey, Adam',\n",
              "   'Max, Daniel',\n",
              "   'Croft, Barbara Y',\n",
              "   'Clarke, Laurence P'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Lung Neoplasms',\n",
              "   'Observer Variation',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Physicians',\n",
              "   'Professional Competence',\n",
              "   'Radiology',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Solitary Pulmonary Nodule',\n",
              "   'Task Performance and Analysis',\n",
              "   'Tomography, X-Ray Computed'],\n",
              "  'pmid': '16979075',\n",
              "  'title': 'Evaluation of lung MDCT nodule annotation across radiologists and methods.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['SY'],\n",
              "  'ESSN': '1539-4522',\n",
              "  'IF_2013': '1.649',\n",
              "  'IF_2014': '1.784',\n",
              "  'IF_2015': '1.598',\n",
              "  'IF_2016': '1.65',\n",
              "  'IF_2017': '1.791',\n",
              "  'ISSN': '0003-6935',\n",
              "  'ISSN_online': '1539-4522',\n",
              "  'ISSN_print': '1559-128X',\n",
              "  'IsoAbbr': 'Appl Opt',\n",
              "  'JrId': 22496,\n",
              "  'MedAbbr': 'Appl Opt',\n",
              "  'NLMid': '0247660',\n",
              "  'Titre': 'Appl Opt',\n",
              "  'abstract': 'The design of complex contact lenses involves numerous uncertain variables. How to help an optical designer to first design the optimal contact lens to reduce discomfort when wearing a pair of glasses is an essential design concern. This study examined the impact of aberrations on contact lenses to optimize a contact lens design for myopic and astigmatic eyes. In general, two aspherical surfaces can be assembled in an optical system to reduce the overall volume size. However, this design reduces the spherical aberration (SA) values at wide contact radii. The proposed optimization algorithm with optical design can be corrected to improve the SA value and, thus, reduce coma aberration (TCO) values and enhance the modulation transfer function (MTF). This means integrating a modified genetic algorithm (GA) with a neural network (NN) to optimize multiple-quality characteristics, namely the SA, TCO, and MTF, of contact lenses. When the proposed optional weight NN-GA is implemented, the weight values of the fitness function can be varied to adjust system performance. The method simplifies the selection of parameters in the optimization of optical systems. Compared with the traditional CODE V built-in optimal scheme, the proposed scheme is more flexible and intuitive to improve SA, TCO, and MTF values by 50.03%, 45.78%, and 24.7%, respectively. ',\n",
              "  'authors': ['Yen, Chih-Ta', 'Ye, Jhe-Wen'],\n",
              "  'categories': ['OPTICS'],\n",
              "  'journal': 'Applied optics',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Astigmatism',\n",
              "   'Computer Simulation',\n",
              "   'Contact Lenses',\n",
              "   'Equipment Design',\n",
              "   'Humans',\n",
              "   'Materials Testing',\n",
              "   'Myopia',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Refraction, Ocular',\n",
              "   'Refractive Errors',\n",
              "   'Refractometry',\n",
              "   'Software',\n",
              "   'Vision, Ocular'],\n",
              "  'pmid': '26479671',\n",
              "  'title': 'Aspherical lens design using hybrid neural-genetic algorithm of contact lenses.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['NE'],\n",
              "  'ESSN': '1531-5487',\n",
              "  'IF_2013': '6.178',\n",
              "  'IF_2014': '6.196',\n",
              "  'IF_2015': '6.075',\n",
              "  'IF_2016': '5.986',\n",
              "  'IF_2017': '4.991',\n",
              "  'ISSN': '1044-3983',\n",
              "  'ISSN_online': '1531-5487',\n",
              "  'ISSN_print': '1044-3983',\n",
              "  'IsoAbbr': 'Epidemiology',\n",
              "  'JrId': 1090,\n",
              "  'MedAbbr': 'Epidemiology',\n",
              "  'NLMid': '9009644',\n",
              "  'Titre': 'Epidemiology',\n",
              "  'abstract': 'Birthweight is often used as a proxy for fetal weight. Problems with this practice have recently been brought to light. We explore whether data available at birth can be used to predict estimated fetal weight using linear and quantile regression, random forests, Bayesian additive regression trees, and generalized boosted models. We train and validate each approach using 18,517 pregnancies (31,948 ultrasound visits) from the Magee-Womens Obstetric Maternal and Infant data and 240 pregnancies in a separate dataset of high-risk pregnancies. We also quantify the relation between smoking and small-for-gestational-age birth, defined as a birthweight in the lower 10th percentile of a population birthweight standard and estimated and predicted fetal weight standard. Using mean squared error and median absolute deviation criteria, quantile regression performed best among the regression-based approaches, but generalized boosted models performed best overall. Using the birthweight standard, smoking during pregnancy increased the risk of small-for-gestational-age 3.84-fold (95% CI: 2.70, 5.47). This ratio dropped to 1.65 (95% CI: 1.50, 1.81) when using the correct fetal weight standard, which was no different from the machine learning-based predicted standards, but higher than the regression-based predicted standards. Machine learning algorithms show promise in recovering missing fetal weight information. See video abstract at, http://links.lww.com/EDE/B314.',\n",
              "  'authors': ['Naimi, Ashley I', 'Platt, Robert W', 'Larkin, Jacob C'],\n",
              "  'categories': ['PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH'],\n",
              "  'journal': 'Epidemiology (Cambridge, Mass.)',\n",
              "  'keywords': ['Adult',\n",
              "   'Algorithms',\n",
              "   'Databases, Factual',\n",
              "   'Female',\n",
              "   'Fetal Development',\n",
              "   'Fetal Weight',\n",
              "   'Humans',\n",
              "   'Machine Learning',\n",
              "   'Pregnancy',\n",
              "   'Pregnancy Trimesters',\n",
              "   'Regression Analysis'],\n",
              "  'pmid': '29199998',\n",
              "  'title': 'Machine Learning for Fetal Growth Prediction.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['EW'],\n",
              "  'ESSN': '1558-1756',\n",
              "  'IF_2013': '1.116',\n",
              "  'IF_2014': '0.911',\n",
              "  'IF_2015': '1.203',\n",
              "  'IF_2016': '1.987',\n",
              "  'IF_2017': '1.64',\n",
              "  'ISSN': '0272-1716',\n",
              "  'ISSN_online': '1558-1756',\n",
              "  'ISSN_print': '0272-1716',\n",
              "  'IsoAbbr': 'IEEE Comput Graph Appl',\n",
              "  'JrId': 31993,\n",
              "  'MedAbbr': 'IEEE Comput Graph Appl',\n",
              "  'NLMid': '9881869',\n",
              "  'Titre': 'IEEE Comput Graph Appl',\n",
              "  'abstract': 'Information visualization has great potential to make sense of the increasing amount of data generated by complex machine-learning algorithms. We design a set of visualizations for a new deep-learning algorithm called FaceLift (goodcitylife.org/facelift). This algorithm is able to generate a beautified version of a given urban image (such as from Google Street View), and our visualizations compare pairs of original and beautified images. With those visualizations, we aim at helping practitioners understand what happened during the algorithmic beautification without requiring them to be machine-learning experts. We evaluate the effectiveness of our visualizations to do just that with a survey among practitioners. From the survey results, we derive general design guidelines on how information visualization makes complex machine-learning algorithms more understandable to a general audience.',\n",
              "  'authors': ['Kauer, Tobias',\n",
              "   'Joglekar, Sagar',\n",
              "   'Redi, Miriam',\n",
              "   'Aiello, Luca Maria',\n",
              "   'Quercia, Daniele'],\n",
              "  'categories': ['COMPUTER SCIENCE, SOFTWARE ENGINEERING'],\n",
              "  'journal': 'IEEE computer graphics and applications',\n",
              "  'keywords': [''],\n",
              "  'pmid': '30273128',\n",
              "  'title': 'Mapping and Visualizing Deep-Learning Urban Beautification.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': \"This study was designed to develop an automated method for estimating lung volume removed during lung volume reduction surgery (LVRS) using computed tomography (CT).|The CT examinations of six patients who underwent bilateral LVRS were analyzed in this study. The resected lung tissue (right and left) was weighed during pathologic examination. An automated computer scheme was developed to estimate the lung volume removed using the CT voxel values and lung specimen weight. The computed fraction of lung volume removed was evaluated across a range of simulated surgical planes (ie, other than parallel to the CT image plane) and CT reconstruction kernels, and it was compared with the surgeons' postsurgical estimates.|The computed fraction of the lung volume removed during LVRS was linearly correlated with the resected lung tissue weight (Pearson correlation = 0.697, P = .012). The computed fraction of lung volume removed ranged from 12.9% to 51.7% of the total lung volume. The surgeons' postsurgical estimates of lung volume removed ranged from 30% to 33%. The percent difference between the surgeons' estimates and the computed lung volume removed as a percentage of the surgeons' estimates ranged from -72.3% to 57.0% with mean absolute difference of 29.7% (+/-20.7).|The preliminary findings of this study suggest that the proposed quantitative model should provide an objective measure of lung volume removed during LVRS that may be used to investigate the relationship between lung volume removed and outcome.\",\n",
              "  'authors': ['Gilbert, Sebastien',\n",
              "   'Zheng, Bin',\n",
              "   'Leader, Joseph K',\n",
              "   'Luketich, James D',\n",
              "   'Fuhrman, Carl R',\n",
              "   'Landreneau, Rodney J',\n",
              "   'Gur, David',\n",
              "   'Sciurba, Frank C'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Absorptiometry, Photon',\n",
              "   'Aged',\n",
              "   'Artificial Intelligence',\n",
              "   'Carbon Monoxide',\n",
              "   'Female',\n",
              "   'Forced Expiratory Volume',\n",
              "   'Humans',\n",
              "   'Image Processing, Computer-Assisted',\n",
              "   'Linear Models',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Organ Size',\n",
              "   'Pneumonectomy',\n",
              "   'Pulmonary Emphysema',\n",
              "   'Research Design',\n",
              "   'Tomography, Spiral Computed',\n",
              "   'Total Lung Capacity',\n",
              "   'Treatment Outcome'],\n",
              "  'pmid': '17070456',\n",
              "  'title': 'Computerized estimation of the lung volume removed during lung volume reduction surgery.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': 'Membrane transport proteins (transporters) move hydrophilic substrates across hydrophobic membranes and play vital roles in most cellular functions. Transporters represent a diverse group of proteins that differ in topology, energy coupling mechanism, and substrate specificity as well as sequence similarity. Among the functional annotations of transporters, information about their transporting substrates is especially important. The experimental identification and characterization of transporters is currently costly and time-consuming. The development of robust bioinformatics-based methods for the prediction of membrane transport proteins and their substrate specificities is therefore an important and urgent task.|Support vector machine (SVM)-based computational models, which comprehensively utilize integrative protein sequence features such as amino acid composition, dipeptide composition, physico-chemical composition, biochemical composition, and position-specific scoring matrices (PSSM), were developed to predict the substrate specificity of seven transporter classes: amino acid, anion, cation, electron, protein/mRNA, sugar, and other transporters. An additional model to differentiate transporters from non-transporters was also developed. Among the developed models, the biochemical composition and PSSM hybrid model outperformed other models and achieved an overall average prediction accuracy of 76.69% with a Mathews correlation coefficient (MCC) of 0.49 and a receiver operating characteristic area under the curve (AUC) of 0.833 on our main dataset. This model also achieved an overall average prediction accuracy of 78.88% and MCC of 0.41 on an independent dataset.|Our analyses suggest that evolutionary information (i.e., the PSSM) and the AAIndex are key features for the substrate specificity prediction of transport proteins. In comparison, similarity-based methods such as BLAST, PSI-BLAST, and hidden Markov models do not provide accurate predictions for the substrate specificity of membrane transport proteins. TrSSP: The Transporter Substrate Specificity Prediction Server, a web server that implements the SVM models developed in this paper, is freely available at http://bioinfo.noble.org/TrSSP.',\n",
              "  'authors': ['Mishra, Nitish K', 'Chang, Junil', 'Zhao, Patrick X'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Amino Acid Sequence',\n",
              "   'Animals',\n",
              "   'Chemical Phenomena',\n",
              "   'Computational Biology',\n",
              "   'Databases, Protein',\n",
              "   'Evolution, Molecular',\n",
              "   'Humans',\n",
              "   'Markov Chains',\n",
              "   'Membrane Transport Proteins',\n",
              "   'Reproducibility of Results',\n",
              "   'Substrate Specificity',\n",
              "   'Support Vector Machine'],\n",
              "  'pmid': '24968309',\n",
              "  'title': 'Prediction of membrane transport proteins and their substrate specificities using primary sequence information.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'D',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['PT'],\n",
              "  'ESSN': '1472-6947',\n",
              "  'IF_2013': '1.496',\n",
              "  'IF_2014': '1.83',\n",
              "  'IF_2015': '2.042',\n",
              "  'IF_2016': '1.643',\n",
              "  'IF_2017': '2.134',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1472-6947',\n",
              "  'IsoAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'JrId': 22346,\n",
              "  'MedAbbr': 'BMC Med Inform Decis Mak',\n",
              "  'NLMid': '101088682',\n",
              "  'Titre': 'BMC Med Inform Decis Mak',\n",
              "  'abstract': \"Functional brain images such as Single-Photon Emission Computed Tomography (SPECT) and Positron Emission Tomography (PET) have been widely used to guide the clinicians in the Alzheimer's Disease (AD) diagnosis. However, the subjectivity involved in their evaluation has favoured the development of Computer Aided Diagnosis (CAD) Systems.|It is proposed a novel combination of feature extraction techniques to improve the diagnosis of AD. Firstly, Regions of Interest (ROIs) are selected by means of a t-test carried out on 3D Normalised Mean Square Error (NMSE) features restricted to be located within a predefined brain activation mask. In order to address the small sample-size problem, the dimension of the feature space was further reduced by: Large Margin Nearest Neighbours using a rectangular matrix (LMNN-RECT), Principal Component Analysis (PCA) or Partial Least Squares (PLS) (the two latter also analysed with a LMNN transformation). Regarding the classifiers, kernel Support Vector Machines (SVMs) and LMNN using Euclidean, Mahalanobis and Energy-based metrics were compared.|Several experiments were conducted in order to evaluate the proposed LMNN-based feature extraction algorithms and its benefits as: i) linear transformation of the PLS or PCA reduced data, ii) feature reduction technique, and iii) classifier (with Euclidean, Mahalanobis or Energy-based methodology). The system was evaluated by means of k-fold cross-validation yielding accuracy, sensitivity and specificity values of 92.78%, 91.07% and 95.12% (for SPECT) and 90.67%, 88% and 93.33% (for PET), respectively, when a NMSE-PLS-LMNN feature extraction method was used in combination with a SVM classifier, thus outperforming recently reported baseline methods.|All the proposed methods turned out to be a valid solution for the presented problem. One of the advances is the robustness of the LMNN algorithm that not only provides higher separation rate between the classes but it also makes (in combination with NMSE and PLS) this rate variation more stable. In addition, their generalization ability is another advance since several experiments were performed on two image modalities (SPECT and PET).\",\n",
              "  'authors': ['Chaves, Rosa',\n",
              "   'Ramírez, Javier',\n",
              "   'Górriz, Juan M',\n",
              "   'Illán, Ignacio A',\n",
              "   'Gómez-Río, Manuel',\n",
              "   'Carnero, Cristobal'],\n",
              "  'categories': ['MEDICAL INFORMATICS'],\n",
              "  'journal': 'BMC medical informatics and decision making',\n",
              "  'keywords': ['Adult',\n",
              "   'Aged',\n",
              "   'Aged, 80 and over',\n",
              "   'Algorithms',\n",
              "   'Alzheimer Disease',\n",
              "   'Artificial Intelligence',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Early Diagnosis',\n",
              "   'Female',\n",
              "   'Functional Neuroimaging',\n",
              "   'Humans',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Positron-Emission Tomography',\n",
              "   'Principal Component Analysis',\n",
              "   'Radiopharmaceuticals',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Spain',\n",
              "   'Tomography, Emission-Computed, Single-Photon'],\n",
              "  'pmid': '22849649',\n",
              "  'title': \"Effective diagnosis of Alzheimer's disease by means of large margin-based methodology.\",\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['JA'],\n",
              "  'ESSN': '1614-7499',\n",
              "  'IF_2013': '2.757',\n",
              "  'IF_2014': '2.828',\n",
              "  'IF_2015': '2.76',\n",
              "  'IF_2016': '2.741',\n",
              "  'IF_2017': '2.8',\n",
              "  'ISSN': '0944-1344',\n",
              "  'ISSN_online': '1614-7499',\n",
              "  'ISSN_print': '0944-1344',\n",
              "  'IsoAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'JrId': 22074,\n",
              "  'MedAbbr': 'Environ Sci Pollut Res Int',\n",
              "  'NLMid': '9441769',\n",
              "  'Titre': 'Environ Sci Pollut Res Int',\n",
              "  'abstract': 'Binding affinity of chemical to carbon is an important characteristic as it finds vast industrial applications. Experimental determination of the adsorption capacity of diverse chemicals onto carbon is both time and resource intensive, and development of computational approaches has widely been advocated. In this study, artificial intelligence (AI)-based ten different qualitative and quantitative structure-property relationship (QSPR) models (MLPN, RBFN, PNN/GRNN, CCN, SVM, GEP, GMDH, SDT, DTF, DTB) were established for the prediction of the adsorption capacity of structurally diverse chemicals to activated carbon following the OECD guidelines. Structural diversity of the chemicals and nonlinear dependence in the data were evaluated using the Tanimoto similarity index and Brock-Dechert-Scheinkman statistics. The generalization and prediction abilities of the constructed models were established through rigorous internal and external validation procedures performed employing a wide series of statistical checks. In complete dataset, the qualitative models rendered classification accuracies between 97.04 and 99.93%, while the quantitative models yielded correlation (R(2)) values of 0.877-0.977 between the measured and the predicted endpoint values. The quantitative prediction accuracies for the higher molecular weight (MW) compounds (class 4) were relatively better than those for the low MW compounds. Both in the qualitative and quantitative models, the Polarizability was the most influential descriptor. Structural alerts responsible for the extreme adsorption behavior of the compounds were identified. Higher number of carbon and presence of higher halogens in a molecule rendered higher binding affinity. Proposed QSPR models performed well and outperformed the previous reports. A relatively better performance of the ensemble learning models (DTF, DTB) may be attributed to the strengths of the bagging and boosting algorithms which enhance the predictive accuracies. The proposed AI models can be useful tools in screening the chemicals for their binding affinities toward carbon for their safe management.',\n",
              "  'authors': ['Gupta, Shikha',\n",
              "   'Basant, Nikita',\n",
              "   'Rai, Premanjali',\n",
              "   'Singh, Kunwar P'],\n",
              "  'categories': ['ENVIRONMENTAL SCIENCES'],\n",
              "  'journal': 'Environmental science and pollution research international',\n",
              "  'keywords': ['Adsorption',\n",
              "   'Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Carbon',\n",
              "   'Charcoal',\n",
              "   'Hazardous Substances',\n",
              "   'Industrial Waste',\n",
              "   'Models, Theoretical',\n",
              "   'Quantitative Structure-Activity Relationship'],\n",
              "  'pmid': '26160122',\n",
              "  'title': 'Modeling the binding affinity of structurally diverse industrial chemicals to carbon using the artificial intelligence approaches.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['CQ'],\n",
              "  'ESSN': '1362-4962',\n",
              "  'IF_2013': '8.808',\n",
              "  'IF_2014': '9.112',\n",
              "  'IF_2015': '9.202',\n",
              "  'IF_2016': '10.162',\n",
              "  'IF_2017': '11.561',\n",
              "  'ISSN': '0305-1048',\n",
              "  'ISSN_online': '1362-4962',\n",
              "  'ISSN_print': '0305-1048',\n",
              "  'IsoAbbr': 'Nucleic Acids Res.',\n",
              "  'JrId': 6133,\n",
              "  'MedAbbr': 'Nucleic Acids Res',\n",
              "  'NLMid': '0411011',\n",
              "  'Titre': 'Nucleic Acids Res.',\n",
              "  'abstract': 'The iBeetle-Base provides access to sequence and phenotype information for genes of the beetle Tribolium castaneum. It has been updated including more and updated data and new functions. RNAi phenotypes are now available for >50% of the genes, which represents an expansion of 60% compared to the previous version. Gene sequence information has been updated based on the new official gene set OGS3 and covers all genes. Interoperability with FlyBase has been enhanced: First, gene information pages of homologous genes are interlinked between both databases. Second, some steps of a new query pipeline allow transforming gene lists from either species into lists with related gene IDs, names or GO terms. This facilitates the comparative analysis of gene functions between fly and beetle. The backend of the pipeline is implemented as endpoints of a RESTful interface, such that it can be reused by other projects or tools. A novel online interface allows the community to propose GO terms for their gene of interest expanding the range of animals where GO terms are defined. iBeetle-Base is available at http://ibeetle-base.uni-goettingen.de/.',\n",
              "  'authors': ['Dönitz, Jürgen',\n",
              "   'Gerischer, Lizzy',\n",
              "   'Hahnke, Stefan',\n",
              "   'Pfeiffer, Stefan',\n",
              "   'Bucher, Gregor'],\n",
              "  'categories': ['BIOCHEMISTRY & MOLECULAR BIOLOGY'],\n",
              "  'journal': 'Nucleic acids research',\n",
              "  'keywords': ['Animals',\n",
              "   'Databases, Genetic',\n",
              "   'Gene Ontology',\n",
              "   'Phenotype',\n",
              "   'RNA Interference',\n",
              "   'Tribolium',\n",
              "   'User-Computer Interface'],\n",
              "  'pmid': '29069517',\n",
              "  'title': 'Expanded and updated data and a query pipeline for iBeetle-Base.',\n",
              "  'year': '2018'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['IG'],\n",
              "  'ESSN': '1573-9686',\n",
              "  'IF_2013': '3.231',\n",
              "  'IF_2014': '3.195',\n",
              "  'IF_2015': '2.887',\n",
              "  'IF_2016': '3.221',\n",
              "  'IF_2017': '3.405',\n",
              "  'ISSN': '0090-6964',\n",
              "  'ISSN_online': '1573-9686',\n",
              "  'ISSN_print': '0090-6964',\n",
              "  'IsoAbbr': 'Ann Biomed Eng',\n",
              "  'JrId': 561,\n",
              "  'MedAbbr': 'Ann Biomed Eng',\n",
              "  'NLMid': '0361512',\n",
              "  'Titre': 'Ann Biomed Eng',\n",
              "  'abstract': 'An adaptive neural network model that exhibits the optimality and homeostasis characteristics of the respiratory control system is described. Based upon the Hopfield network structure and a postulated Hebb-like respiratory synapse with correlational short-term potentiation, the model is capable of mimicking the normal ventilatory responses to exercise and CO2 inputs without the need for an explicit exercise stimulus. Results suggest the possibility of an adaptive neuronal mechanism that effects optimal homeostatic regulation of respiration in mammals.',\n",
              "  'authors': ['Poon, C S'],\n",
              "  'categories': ['ENGINEERING, BIOMEDICAL'],\n",
              "  'journal': 'Annals of biomedical engineering',\n",
              "  'keywords': ['Adaptation, Physiological',\n",
              "   'Animals',\n",
              "   'Computer Simulation',\n",
              "   'Exercise',\n",
              "   'Homeostasis',\n",
              "   'Mammals',\n",
              "   'Models, Biological',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Reference Values',\n",
              "   'Respiration'],\n",
              "  'pmid': '8239090',\n",
              "  'title': 'Adaptive neural network that subserves optimal homeostatic control of breathing.',\n",
              "  'year': ''},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['EP'],\n",
              "  'ESSN': '',\n",
              "  'IF_2013': '6.056',\n",
              "  'IF_2014': '6.507',\n",
              "  'IF_2015': '6.085',\n",
              "  'IF_2016': '6.333',\n",
              "  'IF_2017': '4.58',\n",
              "  'ISSN': '0129-0657',\n",
              "  'ISSN_online': '1793-6462',\n",
              "  'ISSN_print': '0129-0657',\n",
              "  'IsoAbbr': 'Int J Neural Syst',\n",
              "  'JrId': 2597,\n",
              "  'MedAbbr': 'Int J Neural Syst',\n",
              "  'NLMid': '9100527',\n",
              "  'Titre': 'Int J Neural Syst',\n",
              "  'abstract': 'We study the storage properties associated with generalized Hebbian learning rules which present four free parameters that allow for asymmetry. We also introduce two extra parameters in the post-synaptic potentials in order to further improve the critical capacity. Using signal-to-noise analysis, as well as computer simulations on an analog network, we discuss the performance of the rules for arbitrarily biased patterns and find that the critical storage capacity alpha c becomes maximal for a particular symmetric rule (alpha c diverges in the sparse coding limit). Departures from symmetry decrease alpha c but can increase the robustness of the model.',\n",
              "  'authors': ['Stariolo, D A', 'Tsallis, C'],\n",
              "  'categories': ['COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE'],\n",
              "  'journal': 'International journal of neural systems',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Membrane Potentials',\n",
              "   'Memory',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Neurons',\n",
              "   'Synapses'],\n",
              "  'pmid': '7812500',\n",
              "  'title': 'Optimal Hebbian learning rules and the role of asymmetry.',\n",
              "  'year': '1994'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '2045-2322',\n",
              "  'IF_2013': '5.078',\n",
              "  'IF_2014': '5.578',\n",
              "  'IF_2015': '5.228',\n",
              "  'IF_2016': '4.259',\n",
              "  'IF_2017': '4.122',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2045-2322',\n",
              "  'IsoAbbr': 'Sci Rep',\n",
              "  'JrId': 39356,\n",
              "  'MedAbbr': 'Sci Rep',\n",
              "  'NLMid': '101563288',\n",
              "  'Titre': 'Sci Rep',\n",
              "  'abstract': 'Grapevine is one of the most economically important and widely cultivated fruit crop worldwide. With the industrialization and the popular application of cupric fungicides in grape industry, copper stress and copper pollution are also the factors affecting grape production and berry and wine quality. Here, 3,843 transcripts were significantly differently expressed genes in response to Cu stress by RNA-seq, which included 1,892 up-regulated and 1,951 down-regulated transcripts. During this study we found many known and novel Cu-induced and -repressed genes. Biological analysis of grape samples were indicated that exogenous Cu can influence chlorophylls metabolism and photosynthetic activities of grapevine. Most ROS detoxification systems, including antioxidant enzyme, stress-related proteins and secondary metabolites were strongly induced. Concomitantly, abscisic acid functioned as a negative regulator in Cu stress, in opposite action to ethylene, auxin, jasmonic acid, and brassinolide. This study also identified a set of Cu stress specifically activated genes coding copper transporter, P1B-type ATPase, multidrug transporters. Overall, this work was carried out to gain insights into the copper-regulated and stress-responsive mechanisms in grapevine at transcriptome level. This research can also provide some genetic information that can help us in better vinery management and breeding Cu-resistant grape cultivars. ',\n",
              "  'authors': ['Leng, Xiangpeng',\n",
              "   'Jia, Haifeng',\n",
              "   'Sun, Xin',\n",
              "   'Shangguan, Lingfei',\n",
              "   'Mu, Qian',\n",
              "   'Wang, Baoju',\n",
              "   'Fang, Jinggui'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Scientific reports',\n",
              "  'keywords': ['Chlorophyll',\n",
              "   'Copper',\n",
              "   'Gene Expression Regulation, Plant',\n",
              "   'Gene Ontology',\n",
              "   'Genes, Plant',\n",
              "   'Photosynthesis',\n",
              "   'Plant Leaves',\n",
              "   'Plant Proteins',\n",
              "   'Reactive Oxygen Species',\n",
              "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
              "   'Signal Transduction',\n",
              "   'Transcriptome',\n",
              "   'Vitis'],\n",
              "  'pmid': '26673527',\n",
              "  'title': 'Comparative transcriptome analysis of grapevine in response to copper stress.',\n",
              "  'year': '2015'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1872-7727',\n",
              "  'IF_2013': '2.16',\n",
              "  'IF_2014': '2.369',\n",
              "  'IF_2015': '2.593',\n",
              "  'IF_2016': '2.462',\n",
              "  'IF_2017': '2.843',\n",
              "  'ISSN': '0720-048X',\n",
              "  'ISSN_online': '1872-7727',\n",
              "  'ISSN_print': '0720-048X',\n",
              "  'IsoAbbr': 'Eur J Radiol',\n",
              "  'JrId': 3584,\n",
              "  'MedAbbr': 'Eur J Radiol',\n",
              "  'NLMid': '8106411',\n",
              "  'Titre': 'Eur J Radiol',\n",
              "  'abstract': 'To analyze computed tomography (CT) features of primary non-teratomatous germ cell tumors of the mediastinum and to improve the diagnostic efficacy of CT for such tumor.|Fifteen patients with primary non-teratomatous germ cell tumors of the mediastinum, proven pathologically, were enrolled. All patients underwent non-enhanced and contrast-enhanced CT examinations. The CT features, including location, size, shape, edge, CT attenuation, involvement of adjacent structure and local or distant metastasis of each lesion were retrospectively analyzed.|Each case showed single mass, a total of 15 masses. 11 masses arose within the left anterosuperior mediastinum and 4 masses arose within the right anterosuperior mediastinum. 10 masses appeared lobulated and ill-circumscribed, and 5 masses appeared rounded or oval and well-circumscribed. The maximal diameter of all the masses ranged from 5 cm to 16 cm (mean, 11 cm). 12 masses revealed heterogeneous attenuation on non-enhanced CT images with patchy low-attenuation foci and stippled calcification, and showed moderately heterogeneous enhancement after contrast administration. 3 masses of the patients with seminoma revealed homogeneous attenuation on non-enhanced CT images and showed lightly homogeneous enhancement after contrast administration. Pericardial effusion, pleural effusion and involvement of adjacent vascular structures were observed in 6, 5 and 10 cases, respectively. Pulmonary metastasis were observed in 3 cases.|The characteristic CT findings of primary non-teratomatous germ cell tumors of the mediastinum include bulky, ill-circumscribed, lobulated masses, heterogeneous attenuation with low-attenuation foci and calcification on non-enhanced CT images and heterogeneous enhancement after contrast administration. Seminomas may show homogeneous attenuation and homogeneous enhancement after contrast administration. The tumor is apt to involve pericardium, pleura and adjacent vascular structure and develop distant metastasis. Primary non-teratomatous germ cell tumors should be considered when anterosuperior mediastinal masses are detected with these findings, especially in young males.',\n",
              "  'authors': ['Tian, Li',\n",
              "   'Liu, Li-Zhi',\n",
              "   'Cui, Chun-Yan',\n",
              "   'Zhang, Wei-Dong',\n",
              "   'Kuang, Ya-Ling'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European journal of radiology',\n",
              "  'keywords': ['Adolescent',\n",
              "   'Adult',\n",
              "   'Artificial Intelligence',\n",
              "   'Cluster Analysis',\n",
              "   'Computer Graphics',\n",
              "   'Computer Simulation',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Mediastinal Neoplasms',\n",
              "   'Models, Biological',\n",
              "   'Models, Statistical',\n",
              "   'Neoplasms, Germ Cell and Embryonal',\n",
              "   'Numerical Analysis, Computer-Assisted',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Subtraction Technique',\n",
              "   'Teratoma',\n",
              "   'Tomography, X-Ray Computed',\n",
              "   'User-Computer Interface',\n",
              "   'Young Adult'],\n",
              "  'pmid': '21353416',\n",
              "  'title': 'CT findings of primary non-teratomatous germ cell tumors of the mediastinum--a report of 15 cases.',\n",
              "  'year': '2012'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1619-7089',\n",
              "  'IF_2013': '5.217',\n",
              "  'IF_2014': '5.383',\n",
              "  'IF_2015': '5.537',\n",
              "  'IF_2016': '7.277',\n",
              "  'IF_2017': '7.704',\n",
              "  'ISSN': '1619-7070',\n",
              "  'ISSN_online': '1619-7089',\n",
              "  'ISSN_print': '1619-7070',\n",
              "  'IsoAbbr': 'Eur. J. Nucl. Med. Mol. Imaging',\n",
              "  'JrId': 27055,\n",
              "  'MedAbbr': 'Eur J Nucl Med Mol Imaging',\n",
              "  'NLMid': '101140988',\n",
              "  'Titre': 'Eur. J. Nucl. Med. Mol. Imaging',\n",
              "  'abstract': 'Artificial intelligence (AI) is currently regaining enormous interest due to the success of machine learning (ML), and in particular deep learning (DL). Image analysis, and thus radiomics, strongly benefits from this research. However, effectively and efficiently integrating diverse clinical, imaging, and molecular profile data is necessary to understand complex diseases, and to achieve accurate diagnosis in order to provide the best possible treatment. In addition to the need for sufficient computing resources, suitable algorithms, models, and data infrastructure, three important aspects are often neglected: (1) the need for multiple independent, sufficiently large and, above all, high-quality data sets; (2) the need for domain knowledge and ontologies; and (3) the requirement for multiple networks that provide relevant relationships among biological entities. While one will always get results out of high-dimensional data, all three aspects are essential to provide robust training and validation of ML models, to provide explainable hypotheses and results, and to achieve the necessary trust in AI and confidence for clinical applications.',\n",
              "  'authors': ['Holzinger, Andreas', 'Haibe-Kains, Benjamin', 'Jurisica, Igor'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'European journal of nuclear medicine and molecular imaging',\n",
              "  'keywords': [''],\n",
              "  'pmid': '31203421',\n",
              "  'title': 'Why imaging data alone is not enough: AI-based integration of imaging, omics, and clinical data.',\n",
              "  'year': '2019'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'C',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['VY'],\n",
              "  'ESSN': '1878-4046',\n",
              "  'IF_2013': '2.077',\n",
              "  'IF_2014': '1.751',\n",
              "  'IF_2015': '1.966',\n",
              "  'IF_2016': '2.128',\n",
              "  'IF_2017': '2.11',\n",
              "  'ISSN': '1076-6332',\n",
              "  'ISSN_online': '1878-4046',\n",
              "  'ISSN_print': '1076-6332',\n",
              "  'IsoAbbr': 'Acad Radiol',\n",
              "  'JrId': 20370,\n",
              "  'MedAbbr': 'Acad Radiol',\n",
              "  'NLMid': '9440159',\n",
              "  'Titre': 'Acad Radiol',\n",
              "  'abstract': 'Needle biopsy is currently the only way to confirm prostate cancer. To increase prostate cancer diagnostic rate, needles are expected to be deployed at suspicious cancer locations. High-contrast magnetic resonance (MR) imaging provides a powerful tool for detecting suspicious cancerous tissues. To do this, MR appearances of cancerous tissue should be characterized and learned from a sufficient number of prostate MR images with known cancer information. However, ground-truth cancer information is only available in histologic images. Therefore it is necessary to warp ground-truth cancerous regions in histological images to MR images by a registration procedure. The objective of this article is to develop a registration technique for aligning histological and MR images of the same prostate.|Five pairs of histological and T2-weighted MR images of radical prostatectomy specimens are collected. For each pair, registration is guided by two sets of correspondences that can be reliably established on prostate boundaries and internal salient bloblike structures of histologic and MR images.|Our developed registration method can accurately register histologic and MR images. It yields results comparable to manual registration, in terms of landmark distance and volume overlap. It also outperforms both affine registration and boundary-guided registration methods.|We have developed a novel method for deformable registration of histologic and MR images of the same prostate. Besides the collection of ground-truth cancer information in MR images, the method has other potential applications. An automatic, accurate registration of histologic and MR images actually builds a bridge between in vivo anatomical information and ex vivo pathologic information, which is valuable for various clinical studies.',\n",
              "  'authors': ['Zhan, Yiqiang',\n",
              "   'Ou, Yangming',\n",
              "   'Feldman, Michael',\n",
              "   'Tomaszeweski, John',\n",
              "   'Davatzikos, Christos',\n",
              "   'Shen, Dinggang'],\n",
              "  'categories': ['RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING'],\n",
              "  'journal': 'Academic radiology',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Biopsy',\n",
              "   'Humans',\n",
              "   'Image Enhancement',\n",
              "   'Image Interpretation, Computer-Assisted',\n",
              "   'Magnetic Resonance Imaging',\n",
              "   'Male',\n",
              "   'Pattern Recognition, Automated',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Reproducibility of Results',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Subtraction Technique'],\n",
              "  'pmid': '17964460',\n",
              "  'title': 'Registering histologic and MR images of prostate for image-based cancer detection.',\n",
              "  'year': '2007'},\n",
              " {'Cat_2013': 'D',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'D',\n",
              "  'Cat_2016': 'E',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '1748-6718',\n",
              "  'IF_2013': '1.018',\n",
              "  'IF_2014': '0.766',\n",
              "  'IF_2015': '0.887',\n",
              "  'IF_2016': '0.95',\n",
              "  'IF_2017': '1.545',\n",
              "  'ISSN': '1748-670X',\n",
              "  'ISSN_online': '1748-6718',\n",
              "  'ISSN_print': '1748-670X',\n",
              "  'IsoAbbr': 'Comput Math Methods Med',\n",
              "  'JrId': 34878,\n",
              "  'MedAbbr': 'Comput Math Methods Med',\n",
              "  'NLMid': '101277751',\n",
              "  'Titre': 'Comput Math Methods Med',\n",
              "  'abstract': 'Interference of power line (PLI) (fundamental frequency and its harmonics) is usually present in biopotential measurements. Despite all countermeasures, the PLI still corrupts physiological signals, for example, electromyograms (EMG), electroencephalograms (EEG), and electrocardiograms (ECG). When analyzing the fetal ECG (fECG) recorded on the maternal abdomen, the PLI represents a particular strong noise component, being sometimes 10 times greater than the fECG signal, and thus impairing the extraction of any useful information regarding the fetal health state. Many signal processing methods for cancelling the PLI from biopotentials are available in the literature. In this review study, six different principles are analyzed and discussed, and their performance is evaluated on simulated data (three different scenarios), based on five quantitative performance indices. ',\n",
              "  'authors': ['Ţarălungă, Dragoş-Daniel',\n",
              "   'Ungureanu, Georgeta-Mihaela',\n",
              "   'Gussi, Ilinca',\n",
              "   'Strungaru, Rodica',\n",
              "   'Wolf, Werner'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Computational and mathematical methods in medicine',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artifacts',\n",
              "   'Computational Biology',\n",
              "   'Computer Simulation',\n",
              "   'Electrocardiography',\n",
              "   'Electroencephalography',\n",
              "   'Female',\n",
              "   'Fetal Monitoring',\n",
              "   'Humans',\n",
              "   'Neural Networks (Computer)',\n",
              "   'Pregnancy',\n",
              "   'Signal Processing, Computer-Assisted',\n",
              "   'Wavelet Analysis'],\n",
              "  'pmid': '24660020',\n",
              "  'title': 'Fetal ECG extraction from abdominal signals: a review on suppression of fundamental power line interference component and its harmonics.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'NC',\n",
              "  'Cat_2014': 'NC',\n",
              "  'Cat_2015': 'NC',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['HL'],\n",
              "  'ESSN': '1475-3901',\n",
              "  'IF_2013': '',\n",
              "  'IF_2014': '',\n",
              "  'IF_2015': '',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '1475-3898',\n",
              "  'ISSN_online': '1475-3901',\n",
              "  'ISSN_print': '1475-3898',\n",
              "  'IsoAbbr': 'Qual Saf Health Care',\n",
              "  'JrId': 27051,\n",
              "  'MedAbbr': 'Qual Saf Health Care',\n",
              "  'NLMid': '101136980',\n",
              "  'Titre': 'Qual Saf Health Care',\n",
              "  'abstract': 'A novel methodological approach for identifying clusters of similar medical incidents by analyzing large databases of incident reports is described. The discovery of similar events allows the identification of patterns and trends, and makes possible the prediction of future events and the establishment of barriers and best practices. Two techniques from the fields of information science and artificial intelligence have been integrated--namely, case based reasoning and information retrieval--and very good clustering accuracies have been achieved on a test data set of incident reports from transfusion medicine. This work suggests that clustering should integrate the features of an incident captured in traditional form based records together with the detailed information found in the narrative included in event reports.',\n",
              "  'authors': ['Tsatsoulis, C', 'Amthauer, H A'],\n",
              "  'categories': ['HEALTH CARE SCIENCES & SERVICES'],\n",
              "  'journal': 'Quality & safety in health care',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Cluster Analysis',\n",
              "   'Humans',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Medical Errors',\n",
              "   'Sentinel Surveillance',\n",
              "   'Transfusion Reaction',\n",
              "   'United States'],\n",
              "  'pmid': '14645892',\n",
              "  'title': 'Finding clusters of similar events within clinical incident reports: a novel methodology combining case based reasoning and information retrieval.',\n",
              "  'year': '2003'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'D',\n",
              "  'Disciplines': ['MC'],\n",
              "  'ESSN': '2041-1480',\n",
              "  'IF_2013': '2.262',\n",
              "  'IF_2014': '2.262',\n",
              "  'IF_2015': '1.62',\n",
              "  'IF_2016': '1.845',\n",
              "  'IF_2017': '1.6',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '2041-1480',\n",
              "  'IsoAbbr': 'J Biomed Semantics',\n",
              "  'JrId': 37835,\n",
              "  'MedAbbr': 'J Biomed Semantics',\n",
              "  'NLMid': '101531992',\n",
              "  'Titre': 'J Biomed Semantics',\n",
              "  'abstract': 'An experimental protocol is a sequence of tasks and operations executed to perform experimental research in biological and biomedical areas, e.g. biology, genetics, immunology, neurosciences, virology. Protocols often include references to equipment, reagents, descriptions of critical steps, troubleshooting and tips, as well as any other information that researchers deem important for facilitating the reusability of the protocol. Although experimental protocols are central to reproducibility, the descriptions are often cursory. There is the need for a unified framework with respect to the syntactic structure and the semantics for representing experimental protocols.|In this paper we present \"SMART Protocols ontology\", an ontology for representing experimental protocols. Our ontology represents the protocol as a workflow with domain specific knowledge embedded within a document. We also present the S ample I nstrument R eagent O bjective (SIRO) model, which represents the minimal common information shared across experimental protocols. SIRO was conceived in the same realm as the Patient Intervention Comparison Outcome (PICO) model that supports search, retrieval and classification purposes in evidence based medicine. We evaluate our approach against a set of competency questions modeled as SPARQL queries and processed against a set of published and unpublished protocols modeled with the SP Ontology and the SIRO model. Our approach makes it possible to answer queries such as Which protocols use tumor tissue as a sample.|Improving reporting structures for experimental protocols requires collective efforts from authors, peer reviewers, editors and funding bodies. The SP Ontology is a contribution towards this goal. We build upon previous experiences and bringing together the view of researchers managing protocols in their laboratory work. Website: https://smartprotocols.github.io/ .',\n",
              "  'authors': ['Giraldo, Olga',\n",
              "   'García, Alexander',\n",
              "   'López, Federico',\n",
              "   'Corcho, Oscar'],\n",
              "  'categories': ['MATHEMATICAL & COMPUTATIONAL BIOLOGY'],\n",
              "  'journal': 'Journal of biomedical semantics',\n",
              "  'keywords': ['Animals',\n",
              "   'Biological Ontologies',\n",
              "   'Evidence-Based Medicine',\n",
              "   'Humans',\n",
              "   'Information Dissemination',\n",
              "   'Information Storage and Retrieval',\n",
              "   'Internet',\n",
              "   'Models, Theoretical',\n",
              "   'Semantic Web',\n",
              "   'Semantics',\n",
              "   'Terminology as Topic'],\n",
              "  'pmid': '29132408',\n",
              "  'title': 'Using semantics for representing experimental protocols.',\n",
              "  'year': '2017'},\n",
              " {'Cat_2013': 'C',\n",
              "  'Cat_2014': 'D',\n",
              "  'Cat_2015': 'C',\n",
              "  'Cat_2016': 'C',\n",
              "  'Cat_2017': 'C',\n",
              "  'Disciplines': ['ZA'],\n",
              "  'ESSN': '1557-900X',\n",
              "  'IF_2013': '2.095',\n",
              "  'IF_2014': '1.708',\n",
              "  'IF_2015': '2.107',\n",
              "  'IF_2016': '2.27',\n",
              "  'IF_2017': '2.038',\n",
              "  'ISSN': '0892-7790',\n",
              "  'ISSN_online': '1557-900X',\n",
              "  'ISSN_print': '0892-7790',\n",
              "  'IsoAbbr': 'J. Endourol.',\n",
              "  'JrId': 2414,\n",
              "  'MedAbbr': 'J Endourol',\n",
              "  'NLMid': '8807503',\n",
              "  'Titre': 'J. Endourol.',\n",
              "  'abstract': \"The aim of the present study was to investigate the learning curve and perioperative outcomes in 200 consecutive patients with prostate cancer who underwent robot-assisted radical prostatectomy (RARP).|Between August 2006 and August 2011, 200 patients with prostate cancer underwent RARP and were enrolled in this study. We prospectively collected the demographic data and analyzed the pathologic and functional outcomes. The operative outcomes analyzed were total operative time, estimated blood loss (EBL), positive surgical margin (PSM), incontinence, and perioperative complications. We also evaluated the relationship between the surgeon's experience and operative variables.|The sloping learning curve for this surgeon showed that total operative time was strongly correlated with the accumulation of experience for the initial 25 cases (|rs|=0.71, P<0.001). The average EBL was not strongly correlated with additional experience (|rs|<0.7). The PSM rate for the first 50 cases was significantly higher than that of the next 150 cases (34.8% vs 19.4%, P=0.035). The complication rate among the first 50 patients was significantly higher than that among the remaining 150 patients (32% vs 12.7%, P=0.002). The incontinence rate at 12 months was significantly higher for the first 100 cases compared with that for the next 100 cases (9.0% vs 1.0%, P=0.009). For the surgeon to optimize total operative time, PSM rate, complication rate, and incontinence rate, slope learning curves of 25, 50, 50, and 100 cases were needed.|The functional and pathologic results of this minimally invasive procedure seemed to be promising. Distinct learning curves were observed with respect to operative time, PSM, complication rate, and incontinence rate. Exposure to 100 surgeries would be necessary for a surgeon to adequately master the required skills.\",\n",
              "  'authors': ['Hashimoto, Takeshi',\n",
              "   'Yoshioka, Kunihiko',\n",
              "   'Gondo, Tatsuo',\n",
              "   'Kamoda, Naohiro',\n",
              "   'Satake, Naoya',\n",
              "   'Ozu, Choichiro',\n",
              "   'Horiguchi, Yutaka',\n",
              "   'Namiki, Kazunori',\n",
              "   'Nakashima, Jun',\n",
              "   'Tachibana, Masaaki'],\n",
              "  'categories': ['UROLOGY & NEPHROLOGY'],\n",
              "  'journal': 'Journal of endourology',\n",
              "  'keywords': ['Aged',\n",
              "   'Humans',\n",
              "   'Laparoscopy',\n",
              "   'Learning Curve',\n",
              "   'Male',\n",
              "   'Middle Aged',\n",
              "   'Operative Time',\n",
              "   'Postoperative Complications',\n",
              "   'Prostatectomy',\n",
              "   'Prostatic Neoplasms',\n",
              "   'Robotics',\n",
              "   'Time Factors'],\n",
              "  'pmid': '23834506',\n",
              "  'title': 'Learning curve and perioperative outcomes of robot-assisted radical prostatectomy in 200 initial Japanese cases by a single surgeon.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['DX'],\n",
              "  'ESSN': '1520-4804',\n",
              "  'IF_2013': '5.48',\n",
              "  'IF_2014': '5.447',\n",
              "  'IF_2015': '5.589',\n",
              "  'IF_2016': '6.259',\n",
              "  'IF_2017': '6.253',\n",
              "  'ISSN': '0022-2623',\n",
              "  'ISSN_online': '1520-4804',\n",
              "  'ISSN_print': '0022-2623',\n",
              "  'IsoAbbr': 'J. Med. Chem.',\n",
              "  'JrId': 4938,\n",
              "  'MedAbbr': 'J Med Chem',\n",
              "  'NLMid': '9716531',\n",
              "  'Titre': 'J. Med. Chem.',\n",
              "  'abstract': 'The purpose of this study was to explore the use of detailed biological data in combination with a statistical learning method for predicting the CYP1A2 and CYP2D6 inhibition. Data were extracted from the Aureus-Pharma highly structured databases which contain precise measures and detailed experimental protocol concerning the inhibition of the two cytochromes. The methodology used was Recursive Partitioning, an easy and quick method to implement. The building of models was preceded by the evaluation of the chemical space covered by the datasets. The descriptors used are available in the MOE software suite. The models reached at least 80% of Accuracy and often exceeded this percentage for the Sensitivity (Recall), Specificity, and Precision parameters. CYP2D6 datasets provided 11 models with Accuracy over 80%, while CYP1A2 datasets counted 5 high-accuracy models. Our models can be useful to predict the ADME properties during the drug discovery process and are indicated for high-throughput screening.',\n",
              "  'authors': ['Burton, Julien',\n",
              "   'Ijjaali, Ismail',\n",
              "   'Barberan, Olivier',\n",
              "   'Petitet, François',\n",
              "   'Vercauteren, Daniel P',\n",
              "   'Michel, André'],\n",
              "  'categories': ['CHEMISTRY, MEDICINAL'],\n",
              "  'journal': 'Journal of medicinal chemistry',\n",
              "  'keywords': ['Artificial Intelligence',\n",
              "   'Cytochrome P-450 CYP1A2',\n",
              "   'Cytochrome P-450 CYP1A2 Inhibitors',\n",
              "   'Cytochrome P-450 CYP2D6',\n",
              "   'Cytochrome P-450 CYP2D6 Inhibitors',\n",
              "   'Data Interpretation, Statistical',\n",
              "   'Databases, Factual',\n",
              "   'Drug Design',\n",
              "   'Enzyme Inhibitors',\n",
              "   'Models, Molecular',\n",
              "   'Quantitative Structure-Activity Relationship',\n",
              "   'Sensitivity and Specificity',\n",
              "   'Software'],\n",
              "  'pmid': '17034129',\n",
              "  'title': 'Recursive partitioning for the prediction of cytochromes P450 2D6 and 1A2 inhibition: importance of the quality of the dataset.',\n",
              "  'year': '2006'},\n",
              " {'Cat_2013': 'A',\n",
              "  'Cat_2014': 'A',\n",
              "  'Cat_2015': 'A',\n",
              "  'Cat_2016': 'A',\n",
              "  'Cat_2017': 'A',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1091-6490',\n",
              "  'IF_2013': '9.809',\n",
              "  'IF_2014': '9.674',\n",
              "  'IF_2015': '9.423',\n",
              "  'IF_2016': '9.661',\n",
              "  'IF_2017': '9.504',\n",
              "  'ISSN': '0027-8424',\n",
              "  'ISSN_online': '1091-6490',\n",
              "  'ISSN_print': '0027-8424',\n",
              "  'IsoAbbr': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'JrId': 6653,\n",
              "  'MedAbbr': 'Proc Natl Acad Sci U S A',\n",
              "  'NLMid': '7505876',\n",
              "  'Titre': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
              "  'abstract': 'Although biological effects of endocrine disrupting chemicals (EDCs) are often observed at unexpectedly low doses with occasional nonmonotonic dose-response characteristics, transcriptome-wide profiles of sensitivities or dose-dependent behaviors of the EDC responsive genes have remained unexplored. Here, we describe expressome analysis for the comprehensive examination of dose-dependent gene responses and its applications to characterize estrogen responsive genes in MCF-7 cells. Transcriptomes of MCF-7 cells exposed to varying concentrations of representative natural and xenobiotic estrogens for 48 h were determined by microarray and used for computational calculation of interpolated approximations of estimated transcriptomes for 300 doses uniformly distributed in log space for each chemical. The entire collection of these estimated transcriptomes, designated as the expressome, has provided unique opportunities to profile chemical-specific distributions of ligand sensitivities for large numbers of estrogen responsive genes, revealing that at low concentrations estrogens generally tended to suppress rather than to activate transcription. Gene ontology analysis demonstrated distinct functional enrichment between high- and low-sensitivity estrogen responsive genes, supporting the notion that a single EDC chemical can cause qualitatively distinct biological responses at different doses. Expressomal heatmap visualization of dose-dependent induction of Bisphenol A inducible genes showed a weak gene activation peak at a very low concentration range (ca. 0.1 nM) in addition to the main, strong gene activation peak at and above 100 nM. Thus, expressome analysis is a powerful approach to understanding the EDC dose-dependent dynamic changes in gene expression at the transcriptomal level, providing important information on the overall profiles of ligand sensitivities and nonmonotonic responses. ',\n",
              "  'authors': ['Shioda, Toshi',\n",
              "   'Rosenthal, Noël F',\n",
              "   'Coser, Kathryn R',\n",
              "   'Suto, Mizuki',\n",
              "   'Phatak, Mukta',\n",
              "   'Medvedovic, Mario',\n",
              "   'Carey, Vincent J',\n",
              "   'Isselbacher, Kurt J'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
              "  'keywords': ['Benzhydryl Compounds',\n",
              "   'Dose-Response Relationship, Drug',\n",
              "   'Endocrine Disruptors',\n",
              "   'Estrogens',\n",
              "   'Gene Expression Profiling',\n",
              "   'Gene Expression Regulation',\n",
              "   'Gene Ontology',\n",
              "   'Humans',\n",
              "   'MCF-7 Cells',\n",
              "   'Microarray Analysis',\n",
              "   'Phenols',\n",
              "   'RNA, Messenger'],\n",
              "  'pmid': '24062438',\n",
              "  'title': 'Expressomal approach for comprehensive analysis and visualization of ligand sensitivities of xenoestrogen responsive genes.',\n",
              "  'year': '2013'},\n",
              " {'Cat_2013': 'B',\n",
              "  'Cat_2014': 'B',\n",
              "  'Cat_2015': 'B',\n",
              "  'Cat_2016': 'B',\n",
              "  'Cat_2017': 'B',\n",
              "  'Disciplines': ['RO'],\n",
              "  'ESSN': '1932-6203',\n",
              "  'IF_2013': '3.534',\n",
              "  'IF_2014': '3.234',\n",
              "  'IF_2015': '3.057',\n",
              "  'IF_2016': '2.806',\n",
              "  'IF_2017': '2.766',\n",
              "  'ISSN': '',\n",
              "  'ISSN_online': '1932-6203',\n",
              "  'IsoAbbr': 'PLoS ONE',\n",
              "  'JrId': 33103,\n",
              "  'MedAbbr': 'PLoS One',\n",
              "  'NLMid': '101285081',\n",
              "  'Titre': 'PLoS ONE',\n",
              "  'abstract': \"Previous research has shown that interacting with natural environments vs. more urban or built environments can have salubrious psychological effects, such as improvements in attention and memory. Even viewing pictures of nature vs. pictures of built environments can produce similar effects. A major question is: What is it about natural environments that produces these benefits? Problematically, there are many differing qualities between natural and urban environments, making it difficult to narrow down the dimensions of nature that may lead to these benefits. In this study, we set out to uncover visual features that related to individuals' perceptions of naturalness in images. We quantified naturalness in two ways: first, implicitly using a multidimensional scaling analysis and second, explicitly with direct naturalness ratings. Features that seemed most related to perceptions of naturalness were related to the density of contrast changes in the scene, the density of straight lines in the scene, the average color saturation in the scene and the average hue diversity in the scene. We then trained a machine-learning algorithm to predict whether a scene was perceived as being natural or not based on these low-level visual features and we could do so with 81% accuracy. As such we were able to reliably predict subjective perceptions of naturalness with objective low-level visual features. Our results can be used in future studies to determine if these features, which are related to naturalness, may also lead to the benefits attained from interacting with nature. \",\n",
              "  'authors': ['Berman, Marc G',\n",
              "   'Hout, Michael C',\n",
              "   'Kardan, Omid',\n",
              "   'Hunter, MaryCarol R',\n",
              "   'Yourganov, Grigori',\n",
              "   'Henderson, John M',\n",
              "   'Hanayik, Taylor',\n",
              "   'Karimi, Hossein',\n",
              "   'Jonides, John'],\n",
              "  'categories': ['MULTIDISCIPLINARY SCIENCES'],\n",
              "  'journal': 'PloS one',\n",
              "  'keywords': ['Algorithms',\n",
              "   'Artificial Intelligence',\n",
              "   'Cities',\n",
              "   'Color Perception',\n",
              "   'Environment',\n",
              "   'Female',\n",
              "   'Humans',\n",
              "   'Male',\n",
              "   'Spatial Processing',\n",
              "   'Visual Perception',\n",
              "   'Young Adult'],\n",
              "  'pmid': '25531411',\n",
              "  'title': 'The perception of naturalness correlates with low-level visual features of environmental scenes.',\n",
              "  'year': '2014'},\n",
              " {'Cat_2013': 'E',\n",
              "  'Cat_2014': 'E',\n",
              "  'Cat_2015': 'E',\n",
              "  'Cat_2016': 'NC',\n",
              "  'Cat_2017': 'NC',\n",
              "  'Disciplines': ['OI'],\n",
              "  'ESSN': '2505-0044',\n",
              "  'IF_2013': '0.56',\n",
              "  'IF_2014': '0.5',\n",
              "  'IF_2015': '0.553',\n",
              "  'IF_2016': '',\n",
              "  'IF_2017': '',\n",
              "  'ISSN': '0189-6016',\n",
              "  'ISSN_online': '2505-0044',\n",
              "  'ISSN_print': '0189-6016',\n",
              "  'IsoAbbr': 'Afr J Tradit Complement Altern Med',\n",
              "  'JrId': 32711,\n",
              "  'MedAbbr': 'Afr J Tradit Complement Altern Med',\n",
              "  'NLMid': '101232990',\n",
              "  'Titre': 'Afr J Tradit Complement Altern Med',\n",
              "  'abstract': 'This paper aimed to demonstrate the research and development of a rule-based expert system for skin problem consulting in the areas of acne, melasma, freckle, wrinkle, and uneven skin tone, with recommended treatments from Thai traditional medicine knowledge.|The tool selected for developing the expert system is a software program written in the PHP language. MySQL database is used to work together with PHP for building database of the expert system. The system is web-based and can be reached from anywhere with Internet access.|The developed expert system gave recommendations on the skin problem treatment with Thai herbal recipes and Thai herbal cosmetics based on 416 rules derived from primary and secondary sources. The system had been tested by 50 users consisting of dermatologists, Thai traditional medicine doctors, and general users. The developed system was considered good for learning and consultation.|The present work showed how such a scattered body of traditional knowledge as Thai traditional medicine and herbal recipes could be collected, organised and made accessible to users and interested parties. The expert system developed herein should contribute in a meaningful way towards preserving the knowledge and helping promote the use of Thai traditional medicine as a practical alternative medicine for the treatment of illnesses.',\n",
              "  'authors': ['Nopparatkiat, Pornchai',\n",
              "   'na Nagara, Byaporn',\n",
              "   'Chansa-ngavej, Chuvej'],\n",
              "  'categories': ['INTEGRATIVE & COMPLEMENTARY MEDICINE'],\n",
              "  'journal': 'African journal of traditional, complementary, and alternative medicines : AJTCAM',\n",
              "  'keywords': ['Cosmetics',\n",
              "   'Databases, Factual',\n",
              "   'Dermatology',\n",
              "   'Expert Systems',\n",
              "   'Humans',\n",
              "   'Internet',\n",
              "   'Medicine, East Asian Traditional',\n",
              "   'Phytotherapy',\n",
              "   'Plant Preparations',\n",
              "   'Referral and Consultation',\n",
              "   'Skin Diseases',\n",
              "   'Software',\n",
              "   'Thailand'],\n",
              "  'pmid': '24653561',\n",
              "  'title': 'Expert system for skin problem consultation in Thai traditional medicine.',\n",
              "  'year': '2014'},\n",
              " ...]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 19
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "EJrtCMBYeFvS"
      },
      "source": [
        "## TODO: Extraire les titres et les catégories"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "0MOQZkZxfzIy"
      },
      "source": [
        "# mettre le titre en minuscule dans la variable X\n",
        "X = [ x['title'].lower() for x in data ]\n",
        "\n",
        " \n",
        "\n",
        "# mettre la catégorie (1e élément de la liste) dans la variable Y\n",
        "Y = [ x['categories'][0] for x in data ]"
      ],
      "execution_count": 20,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "CFN2trGeefrh"
      },
      "source": [
        "## TODO: Calculer la longueur maximale des titres et abstracts dans le dataset"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "voXtnTBEgIXU",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0fa71429-2666-41c9-994d-770b3b24a341"
      },
      "source": [
        "# longueur maximale des titres, variable max_len\n",
        "_len = [ len(title) for title in X ]\n",
        "max_len = max(_len)\n",
        "print(max_len)"
      ],
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "299\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Iv9MBmJsexz9"
      },
      "source": [
        "## TODO: Diviser le dataset en train (X_train, Y_train, Z_train) et test (X_test, Y_test, Z_test)"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rIcFAOvDgz0L"
      },
      "source": [
        "# X_train, Y_train\n",
        "X_train = X[:8000]\n",
        "Y_train = Y[:8000]\n",
        "\n",
        "\n",
        "# X_test, Y_test\n",
        "X_test = X[8000:]\n",
        "Y_test = Y[8000:]\n",
        "\n",
        "\n"
      ],
      "execution_count": 22,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "H-QnFIZt5AMc",
        "outputId": "1f9ccc9a-0229-4eb6-b0c4-6ee2ea9aac29"
      },
      "source": [
        "Y_train"
      ],
      "execution_count": 49,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ONCOLOGY',\n",
              " 'SPECTROSCOPY',\n",
              " 'SURGERY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'CHEMISTRY, MEDICINAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'TOXICOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'BIOLOGY',\n",
              " 'PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'OPTICS',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'PSYCHIATRY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PSYCHIATRY',\n",
              " 'REPRODUCTIVE BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENDOCRINOLOGY & METABOLISM',\n",
              " 'SURGERY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'ONCOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SPECTROSCOPY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SPECTROSCOPY',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'INFECTIOUS DISEASES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'SURGERY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'BIOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'SPECTROSCOPY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'EMERGENCY MEDICINE',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'PLANT SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'NEUROSCIENCES',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ONCOLOGY',\n",
              " 'EDUCATION, SCIENTIFIC DISCIPLINES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'FOOD SCIENCE & TECHNOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'SPECTROSCOPY',\n",
              " 'FOOD SCIENCE & TECHNOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PSYCHIATRY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ONCOLOGY',\n",
              " 'PSYCHIATRY',\n",
              " 'PLANT SCIENCES',\n",
              " 'CHEMISTRY, ORGANIC',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOPHYSICS',\n",
              " 'MEDICINE, LEGAL',\n",
              " 'COMPUTER SCIENCE, SOFTWARE ENGINEERING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CELL BIOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'DERMATOLOGY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'CHEMISTRY, MEDICINAL',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'AGRICULTURE, DAIRY & ANIMAL SCIENCE',\n",
              " 'BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'CHEMISTRY, MULTIDISCIPLINARY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'HISTORY & PHILOSOPHY OF SCIENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'PLANT SCIENCES',\n",
              " 'CELL BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENDOCRINOLOGY & METABOLISM',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ONCOLOGY',\n",
              " 'SPECTROSCOPY',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'RESPIRATORY SYSTEM',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'ORTHOPEDICS',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'SURGERY',\n",
              " 'PEDIATRICS',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'SURGERY',\n",
              " 'DERMATOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'REHABILITATION',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'FOOD SCIENCE & TECHNOLOGY',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'CHEMISTRY, ORGANIC',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RESPIRATORY SYSTEM',\n",
              " 'PATHOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'ORTHOPEDICS',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SPECTROSCOPY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'PATHOLOGY',\n",
              " 'CHEMISTRY, MEDICINAL',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'PHYSICS, MULTIDISCIPLINARY',\n",
              " 'DENTISTRY, ORAL SURGERY & MEDICINE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHYSIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'SURGERY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PSYCHIATRY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, CYBERNETICS',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'SURGERY',\n",
              " 'TOXICOLOGY',\n",
              " 'SURGERY',\n",
              " 'VETERINARY SCIENCES',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'PATHOLOGY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'BIOPHYSICS',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'HEMATOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICAL ETHICS',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'VIROLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MATERIALS SCIENCE, BIOMATERIALS',\n",
              " 'MEDICINE, LEGAL',\n",
              " 'SURGERY',\n",
              " 'PARASITOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NURSING',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'SURGERY',\n",
              " 'DENTISTRY, ORAL SURGERY & MEDICINE',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'PHYSIOLOGY',\n",
              " 'SURGERY',\n",
              " 'PATHOLOGY',\n",
              " 'SPECTROSCOPY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ORTHOPEDICS',\n",
              " 'SURGERY',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'PLANT SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PSYCHIATRY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'SURGERY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'SPECTROSCOPY',\n",
              " 'NEUROSCIENCES',\n",
              " 'GERIATRICS & GERONTOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PHYSIOLOGY',\n",
              " 'TRANSPLANTATION',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'PATHOLOGY',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'COMPUTER SCIENCE, SOFTWARE ENGINEERING',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'DENTISTRY, ORAL SURGERY & MEDICINE',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'PSYCHIATRY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CHEMISTRY, ORGANIC',\n",
              " 'NEUROSCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NURSING',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'DERMATOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'MATERIALS SCIENCE, MULTIDISCIPLINARY',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'DENTISTRY, ORAL SURGERY & MEDICINE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'ENDOCRINOLOGY & METABOLISM',\n",
              " 'MICROSCOPY',\n",
              " 'PATHOLOGY',\n",
              " 'OPTICS',\n",
              " 'PERIPHERAL VASCULAR DISEASE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'VIROLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'CELL BIOLOGY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'REHABILITATION',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ONCOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'CHEMISTRY, MEDICINAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'OPTICS',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ONCOLOGY',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'PSYCHIATRY',\n",
              " 'MARINE & FRESHWATER BIOLOGY',\n",
              " 'SURGERY',\n",
              " 'INTEGRATIVE & COMPLEMENTARY MEDICINE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHYSICS, MULTIDISCIPLINARY',\n",
              " 'SPORT SCIENCES',\n",
              " 'GERIATRICS & GERONTOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'OPTICS',\n",
              " 'CHEMISTRY, ORGANIC',\n",
              " 'BIOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'BIOPHYSICS',\n",
              " 'NEUROSCIENCES',\n",
              " 'CHEMISTRY, ORGANIC',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RHEUMATOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'PLANT SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NURSING',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PSYCHIATRY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'REHABILITATION',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'DERMATOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'VIROLOGY',\n",
              " 'TOXICOLOGY',\n",
              " 'OPTICS',\n",
              " 'NEUROSCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'NEUROIMAGING',\n",
              " 'NEUROSCIENCES',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'NURSING',\n",
              " 'SURGERY',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PSYCHIATRY',\n",
              " 'PATHOLOGY',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'CHEMISTRY, MEDICINAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'OTORHINOLARYNGOLOGY',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'PSYCHIATRY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'REHABILITATION',\n",
              " 'NEUROSCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'CELL BIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MICROBIOLOGY',\n",
              " 'PSYCHIATRY',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'BIOCHEMICAL RESEARCH METHODS',\n",
              " 'IMMUNOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'CRITICAL CARE MEDICINE',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'BIOLOGY',\n",
              " 'REHABILITATION',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'NUTRITION & DIETETICS',\n",
              " 'SURGERY',\n",
              " 'NEUROSCIENCES',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ECOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'NEUROSCIENCES',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'BIOTECHNOLOGY & APPLIED MICROBIOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'PHYSICS, MULTIDISCIPLINARY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'HEMATOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'OPTICS',\n",
              " 'MEDICINE, LEGAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'OPTICS',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'REHABILITATION',\n",
              " 'ENDOCRINOLOGY & METABOLISM',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'SURGERY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ROBOTICS',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ENDOCRINOLOGY & METABOLISM',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'SPECTROSCOPY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'SURGERY',\n",
              " 'TOXICOLOGY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MICROSCOPY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'CELL BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PERIPHERAL VASCULAR DISEASE',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'SURGERY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'DERMATOLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'NEUROSCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'CELL BIOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'ORTHOPEDICS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CLINICAL NEUROLOGY',\n",
              " 'SURGERY',\n",
              " 'NEUROIMAGING',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CARDIAC & CARDIOVASCULAR SYSTEMS',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPTICS',\n",
              " 'OPTICS',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'SURGERY',\n",
              " 'OBSTETRICS & GYNECOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'CHEMISTRY, MULTIDISCIPLINARY',\n",
              " 'REHABILITATION',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'SURGERY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'COMPUTER SCIENCE, SOFTWARE ENGINEERING',\n",
              " 'MEDICAL INFORMATICS',\n",
              " 'MEDICINE, RESEARCH & EXPERIMENTAL',\n",
              " 'OPTICS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'BIOLOGY',\n",
              " 'MATHEMATICAL & COMPUTATIONAL BIOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPHTHALMOLOGY',\n",
              " 'HEALTH CARE SCIENCES & SERVICES',\n",
              " 'SURGERY',\n",
              " 'PEDIATRICS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'ANESTHESIOLOGY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MEDICAL LABORATORY TECHNOLOGY',\n",
              " 'SURGERY',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MICROBIOLOGY',\n",
              " 'PLANT SCIENCES',\n",
              " 'BIOLOGY',\n",
              " 'ONCOLOGY',\n",
              " 'ORTHOPEDICS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SPECTROSCOPY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MEDICINE, LEGAL',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'SURGERY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'CELL BIOLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'ONCOLOGY',\n",
              " 'EMERGENCY MEDICINE',\n",
              " 'SURGERY',\n",
              " 'COMPUTER SCIENCE, SOFTWARE ENGINEERING',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'SURGERY',\n",
              " 'GASTROENTEROLOGY & HEPATOLOGY',\n",
              " 'ROBOTICS',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ONCOLOGY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'ENVIRONMENTAL SCIENCES',\n",
              " 'GENETICS & HEREDITY',\n",
              " 'MEDICINE, GENERAL & INTERNAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'COMPUTER SCIENCE, ARTIFICIAL INTELLIGENCE',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'SURGERY',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'ORTHOPEDICS',\n",
              " 'RADIOLOGY, NUCLEAR MEDICINE & MEDICAL IMAGING',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'ONCOLOGY',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'OPTICS',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'PHARMACOLOGY & PHARMACY',\n",
              " 'CHEMISTRY, ANALYTICAL',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " 'SURGERY',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'ENGINEERING, BIOMEDICAL',\n",
              " 'BIOCHEMISTRY & MOLECULAR BIOLOGY',\n",
              " 'BIOLOGY',\n",
              " 'NEUROSCIENCES',\n",
              " 'MULTIDISCIPLINARY SCIENCES',\n",
              " 'DEVELOPMENTAL BIOLOGY',\n",
              " 'UROLOGY & NEPHROLOGY',\n",
              " ...]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 49
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "kGbD4qX9610a"
      },
      "source": [
        "from sklearn.model_selection import train_test_split \n",
        "\n",
        "X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size = 0.2, random_state = 21)"
      ],
      "execution_count": 23,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "g3DcNJBxggZW"
      },
      "source": [
        "## Transformer la variable Y en vecteur numerique\n",
        "\n",
        "[\"Cat 1\", \"Cat 2\"] -> [0, 1]"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NczykEOT7avO"
      },
      "source": [
        "Y_train\n",
        "\n",
        "cat_to_id = {'<UNK>':0}\n",
        "\n",
        "for cat in Y_train: \n",
        "  if cat not in cat_to_id.keys(): \n",
        "    cat_to_id[cat] = len(cat_to_id)\n",
        "\n",
        "id_to_cat = { v: k for k,v in cat_to_id.items()}"
      ],
      "execution_count": 24,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "3e206jKzKUYt",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6c38e843-6cf4-43a0-fcad-6b0e7e68abf6"
      },
      "source": [
        "num_cat = len(cat_to_id)\n",
        "print(num_cat)"
      ],
      "execution_count": 25,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "96\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "f5EPsw9M_m6T"
      },
      "source": [
        "def preprocess_Y(Y, cat_to_id): \n",
        "  \"\"\"returns list of cat_ids for Y\n",
        "  \"\"\"\n",
        "  res = []\n",
        "  for ex in Y: \n",
        "    if ex not in cat_to_id.keys(): \n",
        "      res.append(cat_to_id['<UNK>'])\n",
        "    else:\n",
        "      res.append(cat_to_id[ex])\n",
        "  return np.array(res)"
      ],
      "execution_count": 26,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "5Znt2qz1CMCP"
      },
      "source": [
        "Y_train_id = preprocess_Y(Y_train, cat_to_id)\n",
        "Y_test_id = preprocess_Y(Y_test, cat_to_id)"
      ],
      "execution_count": 27,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "luxbxF0D8f4h",
        "outputId": "08b5e63d-33a5-436b-c1a7-cc326de90c52"
      },
      "source": [
        "Y_train_id"
      ],
      "execution_count": 28,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([ 1,  2,  2, ..., 24,  3, 12])"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 28
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "7I-bA1Dihtik"
      },
      "source": [
        "## Tokenizer les titres\n",
        "\n",
        "Pour cela vous pouvez utiliser la fonction `Tokenizer` de keras\n",
        "\n",
        "Le but est de transformer les textes en un vecteur numérique\n",
        "\n",
        "texte -> liste de tokens -> vecteur numérique\n",
        "\n",
        "\"Miaou le chat\" -> [\"Miaou\", \"le\", chat\"] -> [1, 2, 3]"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "o-8LPBeJjJl8"
      },
      "source": [
        "# Créer le tokenizer\n",
        "vocab_size = 10000\n",
        "\n",
        "tokenizer = tf.keras.preprocessing.text.Tokenizer(num_words = vocab_size)"
      ],
      "execution_count": 32,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "07eGO99-j5ZX"
      },
      "source": [
        "# Entrainer le tokenizer sur le train set \n",
        "tokenizer.fit_on_texts(X_train)"
      ],
      "execution_count": 33,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7nI6qhqplcKE"
      },
      "source": [
        "# Transformer les textes en vecteurs numeriques à l'aide du tokenizer\n",
        "X_train_seq = tokenizer.texts_to_sequences(X_train)\n",
        "X_test_seq = tokenizer.texts_to_sequences(X_test)\n",
        "\n"
      ],
      "execution_count": 34,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9gyAWoXvDrb6"
      },
      "source": [
        "max_len = max([len(x) for x in X_train_seq])\r\n"
      ],
      "execution_count": 35,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tMjkP-Sv-gIK",
        "outputId": "37e067e6-a169-478b-c40d-03e3c8bf6079"
      },
      "source": [
        "max_len"
      ],
      "execution_count": 36,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "39"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 36
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "Ao-Lj9nljVAH"
      },
      "source": [
        "## Faire un padding des sequences obtenues pour qu'elles aient toutes la même taille (cf la fonction `pad_sequences`)\n",
        "\n",
        "[1, 2, 3]       -> [0, 0, 1, 2 ,3]\n",
        "\n",
        "[4, 5, 6, 7, 8] -> [4, 5, 6, 7, 8]"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-aMIpV1clj8X"
      },
      "source": [
        "# Padding des sequences \n",
        "X_train_pad = tf.keras.preprocessing.sequence.pad_sequences(X_train_seq, maxlen = max_len, truncating= 'post')\n",
        "X_test_pad = tf.keras.preprocessing.sequence.pad_sequences(X_test_seq, maxlen = max_len, truncating= 'post')\n"
      ],
      "execution_count": 37,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "JZHz3Dljlk19",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6682009f-2733-4cab-ccd6-ec92efa54174"
      },
      "source": [
        "X_train_pad.shape\r\n"
      ],
      "execution_count": 70,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(8000, 39)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 70
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "tMrlhsz_l-Pz",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "ac381c5d-b7b6-4d49-e465-0780acf317fd"
      },
      "source": [
        "X_test_pad.shape\r\n"
      ],
      "execution_count": 71,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(2000, 39)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 71
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "HkstuTOO05sw",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7065ba03-d050-438b-9c62-213d741fb8f0"
      },
      "source": [
        "Y_test_id[0:100]"
      ],
      "execution_count": 76,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([ 6,  2,  8, 12,  0, 55, 21,  1, 57,  3, 12, 12, 11, 12, 11, 24, 24,\n",
              "       32,  7, 42, 12, 45,  5,  2,  9, 20, 27,  9, 12,  5, 38,  5, 12,  3,\n",
              "        5,  7,  2, 71,  3,  7,  7,  1, 34, 12,  2, 12,  5, 12, 12,  2,  5,\n",
              "       12,  5, 11, 39, 12,  9, 18, 12, 24, 32, 32, 24, 78,  5, 53,  3,  0,\n",
              "       24,  5,  5, 14, 12, 12, 12,  9,  7, 17, 10, 14,  8,  1, 43, 12,  5,\n",
              "       11,  6, 45,  9, 21,  7, 12, 20,  2,  2,  7, 24,  3,  7, 12])"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 76
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "2KCGGWdoj_I2"
      },
      "source": [
        "# Réseau de convolution pour la classification de texte\n",
        "\n",
        "Les réseaux convolutionnels peuvent également être utiliser pour le texte et notamment pour la classification de texte. Ici nous allons construire un CNN sur le même modèle que pour les images avec quelques petites spécificités. \n",
        "\n",
        "Comme le texte est une séquence de mots, il s'agit d'une séquence en 1 dimension. Nous appliquerons donc une convolution en 1D. \n",
        "\n",
        "Pour traiter du texte, la première couche de notre réseau va être constituée par une couche d'embedding. \n",
        "\n",
        "Pour rappel, le word embedding consiste à projeter les tokens dans un espace vectoriel qui va minimiser la distance entre les tokens qui sont utilisés dans des contextes similaires (et qui ont un sens proche ? )\n",
        "\n",
        "![Texte alternatif…](https://www.ibm.com/blogs/research/wp-content/uploads/2018/10/WMEFig1.png)\n",
        "\n",
        "Les embeddings peuvent être calculés de diverses façons. Par exemple word2vec, un des plus célèbres, se base sur 2 algorithmes frères Skip-gram et CBOW\n",
        "\n",
        "![Texte alternatif…](https://pathmind.com/images/wiki/word2vec_diagrams.png)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "UQ-GICwamQGd"
      },
      "source": [
        "Pour information, il existe aujourd'hui des algorithmes plus performants que word2vec comme [Fasttext](https://fasttext.cc) qui prend en compte des informations de sous-mots ou la famille des embeddings contextuels comme [ELMo](https://allennlp.org/elmo) ou [BERT](https://arxiv.org/abs/1810.04805) qui prennent en compte le contexte d'utilisation du mot pour calculer son vecteur. "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8L92Fw-TnUG4",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "outputId": "88124bc4-9fe8-4e75-922d-07571fcb4678"
      },
      "source": [
        "\n",
        "#Modèle 1\n",
        "\n",
        "\n",
        "embed_dim= 128\n",
        "dropout1 = 0.2\n",
        "conv_filters= 32\n",
        "conv_kernel = 2\n",
        "maxpool_size = 2\n",
        "dense_size = 128\n",
        "batch_size = 128\n",
        "epochs = 10\n",
        "\n",
        "# Créer le modèle avec au minimum\n",
        "\n",
        "model_cnn1 = tf.keras.models.Sequential()\n",
        "# Embedding \n",
        "model_cnn1.add(tf.keras.layers.Embedding(vocab_size, \n",
        "                                        embed_dim, \n",
        "                                        input_length= max_len))\n",
        "# Dropout\n",
        "model_cnn1.add(tf.keras.layers.Dropout(dropout1))\n",
        "# Convolution\n",
        "model_cnn1.add(tf.keras.layers.Conv1D(conv_filters, conv_kernel, \n",
        "                                     padding='valid', \n",
        "                                     strides= 1, \n",
        "                                     activation='relu'))\n",
        "model_cnn1.add(tf.keras.layers.MaxPooling1D(maxpool_size))\n",
        "model_cnn1.add(tf.keras.layers.Flatten())\n",
        "# Maxpooling\n",
        "# Dense\n",
        "model_cnn1.add(tf.keras.layers.Dense(dense_size,  activation='relu'))\n",
        "# Activation\n",
        "\n",
        "# Classifieur (Dense + activation softmax)\n",
        "model_cnn1.add(tf.keras.layers.Dense(num_cat))\n",
        "model_cnn1.add(tf.keras.layers.Activation('softmax'))\n",
        "\n",
        "# Compiler le modèle \n",
        "model_cnn1.compile(loss='sparse_categorical_crossentropy', \n",
        "                  optimizer='adam', \n",
        "                  metrics= ['accuracy'])\n",
        "\n",
        "# Afficher le summary du modèle\n",
        "print(model_cnn1.summary())"
      ],
      "execution_count": 9,
      "outputs": [
        {
          "output_type": "error",
          "ename": "NameError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-9-e59a7c5ae6e8>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     16\u001b[0m \u001b[0mmodel_cnn1\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtf\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkeras\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mmodels\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSequential\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     17\u001b[0m \u001b[0;31m# Embedding\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 18\u001b[0;31m model_cnn1.add(tf.keras.layers.Embedding(vocab_size, \n\u001b[0m\u001b[1;32m     19\u001b[0m                                         \u001b[0membed_dim\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     20\u001b[0m                                         input_length= max_len))\n",
            "\u001b[0;31mNameError\u001b[0m: name 'vocab_size' is not defined"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "P8TD5CkVnt-a",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9b3bf755-8b5d-4095-a058-4f61b3d88f98"
      },
      "source": [
        "# Fitter le modèle1 \n",
        "model_cnn1.fit(X_train_pad, Y_train_id, batch_size = batch_size , epochs = epochs)"
      ],
      "execution_count": 78,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/10\n",
            "63/63 [==============================] - 8s 15ms/step - loss: 3.9919 - accuracy: 0.1820\n",
            "Epoch 2/10\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 3.3984 - accuracy: 0.1990\n",
            "Epoch 3/10\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 3.0494 - accuracy: 0.2740\n",
            "Epoch 4/10\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 2.6485 - accuracy: 0.3580\n",
            "Epoch 5/10\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 2.2304 - accuracy: 0.4509\n",
            "Epoch 6/10\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.7687 - accuracy: 0.5518\n",
            "Epoch 7/10\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.4414 - accuracy: 0.6468\n",
            "Epoch 8/10\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.1447 - accuracy: 0.7340\n",
            "Epoch 9/10\n",
            "63/63 [==============================] - 1s 12ms/step - loss: 0.9274 - accuracy: 0.7803\n",
            "Epoch 10/10\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.7212 - accuracy: 0.8404\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb391ce7208>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 78
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "hlAKERnqy-Wu",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4251c3b7-ff3f-437a-bb75-667cd991d3c1"
      },
      "source": [
        "# Evaluer le modèle1\n",
        "model_cnn1.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 79,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 0s 2ms/step - loss: 3.9380 - accuracy: 0.2965\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[3.9380040168762207, 0.29649999737739563]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 79
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jfQVklBZB9Yh",
        "outputId": "9cedcfed-de0a-4c8a-e0dc-70a72615bc8d"
      },
      "source": [
        "\r\n",
        "#Modèle 2\r\n",
        "\r\n",
        "\r\n",
        "embed_dim2= 128\r\n",
        "dropout2 = 0.4\r\n",
        "conv_filters2= 32\r\n",
        "conv_kernel2 = 2\r\n",
        "maxpool_size2 = 4\r\n",
        "dense_size2 = 128\r\n",
        "batch_size2 = 128\r\n",
        "epochs2 = 40\r\n",
        "\r\n",
        "# Créer le modèle avec au minimum\r\n",
        "\r\n",
        "model_cnn2 = tf.keras.models.Sequential()\r\n",
        "# Embedding \r\n",
        "model_cnn2.add(tf.keras.layers.Embedding(vocab_size, \r\n",
        "                                        embed_dim2, \r\n",
        "                                        input_length= max_len))\r\n",
        "# Dropout\r\n",
        "model_cnn2.add(tf.keras.layers.Dropout(dropout2))\r\n",
        "# Convolution\r\n",
        "model_cnn2.add(tf.keras.layers.Conv1D(conv_filters2, conv_kernel2, \r\n",
        "                                     padding='valid', \r\n",
        "                                     strides= 1, \r\n",
        "                                     activation='elu'))\r\n",
        "model_cnn2.add(tf.keras.layers.MaxPooling1D(maxpool_size2))\r\n",
        "model_cnn2.add(tf.keras.layers.Flatten())\r\n",
        "# Maxpooling\r\n",
        "# Dense\r\n",
        "model_cnn2.add(tf.keras.layers.Dense(dense_size2,  activation='elu'))\r\n",
        "# Activation\r\n",
        "\r\n",
        "# Classifieur (Dense + activation softmax)\r\n",
        "model_cnn2.add(tf.keras.layers.Dense(num_cat))\r\n",
        "model_cnn2.add(tf.keras.layers.Activation('softmax'))\r\n",
        "\r\n",
        "# Compiler le modèle \r\n",
        "model_cnn2.compile(loss='sparse_categorical_crossentropy', \r\n",
        "                  optimizer='adam', \r\n",
        "                  metrics= ['accuracy'])\r\n",
        "\r\n",
        "# Afficher le summary du modèle\r\n",
        "print(model_cnn2.summary())"
      ],
      "execution_count": 84,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_3\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_3 (Embedding)      (None, 39, 128)           1280000   \n",
            "_________________________________________________________________\n",
            "dropout_3 (Dropout)          (None, 39, 128)           0         \n",
            "_________________________________________________________________\n",
            "conv1d_4 (Conv1D)            (None, 38, 32)            8224      \n",
            "_________________________________________________________________\n",
            "max_pooling1d_3 (MaxPooling1 (None, 9, 32)             0         \n",
            "_________________________________________________________________\n",
            "flatten_3 (Flatten)          (None, 288)               0         \n",
            "_________________________________________________________________\n",
            "dense_6 (Dense)              (None, 128)               36992     \n",
            "_________________________________________________________________\n",
            "dense_7 (Dense)              (None, 96)                12384     \n",
            "_________________________________________________________________\n",
            "activation_3 (Activation)    (None, 96)                0         \n",
            "=================================================================\n",
            "Total params: 1,337,600\n",
            "Trainable params: 1,337,600\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mh_7Eo-VCLDD",
        "outputId": "6da466f3-16c8-4f8e-ef15-f640f5372d7e"
      },
      "source": [
        "# Fitter le modèle 2\r\n",
        "model_cnn2.fit(X_train_pad, Y_train_id, batch_size = batch_size2 , epochs = epochs2)"
      ],
      "execution_count": 85,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 3.9374 - accuracy: 0.1669\n",
            "Epoch 2/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 3.4162 - accuracy: 0.1883\n",
            "Epoch 3/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 3.1374 - accuracy: 0.2615\n",
            "Epoch 4/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 2.8382 - accuracy: 0.3210\n",
            "Epoch 5/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 2.4691 - accuracy: 0.4023\n",
            "Epoch 6/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 2.1203 - accuracy: 0.4807\n",
            "Epoch 7/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.8201 - accuracy: 0.5425\n",
            "Epoch 8/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.5908 - accuracy: 0.5959\n",
            "Epoch 9/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.3356 - accuracy: 0.6583\n",
            "Epoch 10/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 1.1201 - accuracy: 0.7174\n",
            "Epoch 11/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.9302 - accuracy: 0.7632\n",
            "Epoch 12/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.8132 - accuracy: 0.7966\n",
            "Epoch 13/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.7125 - accuracy: 0.8270\n",
            "Epoch 14/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.6135 - accuracy: 0.8493\n",
            "Epoch 15/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.5260 - accuracy: 0.8690\n",
            "Epoch 16/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.4778 - accuracy: 0.8850\n",
            "Epoch 17/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.4037 - accuracy: 0.9030\n",
            "Epoch 18/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.3771 - accuracy: 0.9069\n",
            "Epoch 19/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.3383 - accuracy: 0.9189\n",
            "Epoch 20/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.2925 - accuracy: 0.9239\n",
            "Epoch 21/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.2547 - accuracy: 0.9394\n",
            "Epoch 22/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.2347 - accuracy: 0.9428\n",
            "Epoch 23/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.2200 - accuracy: 0.9477\n",
            "Epoch 24/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1932 - accuracy: 0.9544\n",
            "Epoch 25/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1941 - accuracy: 0.9502\n",
            "Epoch 26/40\n",
            "63/63 [==============================] - 1s 15ms/step - loss: 0.1764 - accuracy: 0.9570\n",
            "Epoch 27/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1536 - accuracy: 0.9622\n",
            "Epoch 28/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1555 - accuracy: 0.9634\n",
            "Epoch 29/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1459 - accuracy: 0.9631\n",
            "Epoch 30/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1252 - accuracy: 0.9689\n",
            "Epoch 31/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1277 - accuracy: 0.9672\n",
            "Epoch 32/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1256 - accuracy: 0.9649\n",
            "Epoch 33/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1045 - accuracy: 0.9752\n",
            "Epoch 34/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1161 - accuracy: 0.9714\n",
            "Epoch 35/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.1083 - accuracy: 0.9710\n",
            "Epoch 36/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.1142 - accuracy: 0.9685\n",
            "Epoch 37/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.0958 - accuracy: 0.9747\n",
            "Epoch 38/40\n",
            "63/63 [==============================] - 1s 14ms/step - loss: 0.0994 - accuracy: 0.9718\n",
            "Epoch 39/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.0874 - accuracy: 0.9763\n",
            "Epoch 40/40\n",
            "63/63 [==============================] - 1s 13ms/step - loss: 0.0818 - accuracy: 0.9782\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb332515588>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 85
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "NgoCUZIiCMRy",
        "outputId": "b4e20542-59c4-40d6-8a1e-87b26bfcae8b"
      },
      "source": [
        "# Evaluer le modèle2\r\n",
        "model_cnn2.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 86,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 0s 2ms/step - loss: 7.8279 - accuracy: 0.2425\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[7.827927112579346, 0.24250000715255737]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 86
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "nhndJOXtCVSr",
        "outputId": "3f5b641e-8ed4-427c-ac15-7e5e7e7476cb"
      },
      "source": [
        "\r\n",
        "#Modèle 3\r\n",
        "\r\n",
        "\r\n",
        "embed_dim3= 128\r\n",
        "dropout3 = 0.3\r\n",
        "conv_filters3= 64\r\n",
        "conv_kernel3 = 4\r\n",
        "maxpool_size3 = 4\r\n",
        "dense_size3 = 256\r\n",
        "batch_size3 = 256\r\n",
        "epochs3 = 100\r\n",
        "\r\n",
        "# Créer le modèle avec au minimum\r\n",
        "\r\n",
        "model_cnn3 = tf.keras.models.Sequential()\r\n",
        "# Embedding \r\n",
        "model_cnn3.add(tf.keras.layers.Embedding(vocab_size, \r\n",
        "                                        embed_dim3, \r\n",
        "                                        input_length= max_len))\r\n",
        "# Dropout\r\n",
        "model_cnn3.add(tf.keras.layers.Dropout(dropout3))\r\n",
        "# Convolution\r\n",
        "model_cnn3.add(tf.keras.layers.Conv1D(conv_filters3, conv_kernel3, \r\n",
        "                                     padding='valid', \r\n",
        "                                     strides= 1, \r\n",
        "                                     activation='relu'))\r\n",
        "model_cnn3.add(tf.keras.layers.MaxPooling1D(maxpool_size3))\r\n",
        "model_cnn3.add(tf.keras.layers.Flatten())\r\n",
        "# Maxpooling\r\n",
        "# Dense\r\n",
        "model_cnn3.add(tf.keras.layers.Dense(dense_size3,  activation='relu'))\r\n",
        "model_cnn3.add(tf.keras.layers.Dense(dense_size2,  activation='relu')) #ajout d'une nouvelle couche dense (fully-connected)\r\n",
        "# Activation\r\n",
        "\r\n",
        "# Classifieur (Dense + activation softmax)\r\n",
        "model_cnn3.add(tf.keras.layers.Dense(num_cat))\r\n",
        "model_cnn3.add(tf.keras.layers.Activation('softmax'))\r\n",
        "\r\n",
        "\r\n",
        "# Compiler le modèle \r\n",
        "model_cnn3.compile(loss='sparse_categorical_crossentropy', \r\n",
        "                  optimizer='adam', \r\n",
        "                  metrics= ['accuracy'])\r\n",
        "\r\n",
        "# Afficher le summary du modèle\r\n",
        "print(model_cnn3.summary())"
      ],
      "execution_count": 91,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_6\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_6 (Embedding)      (None, 39, 128)           1280000   \n",
            "_________________________________________________________________\n",
            "dropout_6 (Dropout)          (None, 39, 128)           0         \n",
            "_________________________________________________________________\n",
            "conv1d_7 (Conv1D)            (None, 36, 64)            32832     \n",
            "_________________________________________________________________\n",
            "max_pooling1d_6 (MaxPooling1 (None, 9, 64)             0         \n",
            "_________________________________________________________________\n",
            "flatten_6 (Flatten)          (None, 576)               0         \n",
            "_________________________________________________________________\n",
            "dense_13 (Dense)             (None, 256)               147712    \n",
            "_________________________________________________________________\n",
            "dense_14 (Dense)             (None, 128)               32896     \n",
            "_________________________________________________________________\n",
            "dense_15 (Dense)             (None, 96)                12384     \n",
            "_________________________________________________________________\n",
            "activation_5 (Activation)    (None, 96)                0         \n",
            "=================================================================\n",
            "Total params: 1,505,824\n",
            "Trainable params: 1,505,824\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dFHoPe2NCg8c",
        "outputId": "22869ac1-1b69-4bb9-ba97-71ef992b9a2e"
      },
      "source": [
        "# Fitter le modèle3 \r\n",
        "model_cnn3.fit(X_train_pad, Y_train_id, batch_size = batch_size3 , epochs = epochs3)"
      ],
      "execution_count": 88,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 4.2026 - accuracy: 0.1662\n",
            "Epoch 2/100\n",
            "32/32 [==============================] - 0s 15ms/step - loss: 3.4631 - accuracy: 0.1867\n",
            "Epoch 3/100\n",
            "32/32 [==============================] - 0s 15ms/step - loss: 3.3357 - accuracy: 0.2078\n",
            "Epoch 4/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 3.0283 - accuracy: 0.2778\n",
            "Epoch 5/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 2.6716 - accuracy: 0.3250\n",
            "Epoch 6/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 2.4076 - accuracy: 0.3707\n",
            "Epoch 7/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 2.0758 - accuracy: 0.4617\n",
            "Epoch 8/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 1.7637 - accuracy: 0.5440\n",
            "Epoch 9/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 1.5395 - accuracy: 0.6030\n",
            "Epoch 10/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 1.2842 - accuracy: 0.6635\n",
            "Epoch 11/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 1.0813 - accuracy: 0.7132\n",
            "Epoch 12/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.9597 - accuracy: 0.7488\n",
            "Epoch 13/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.7824 - accuracy: 0.7899\n",
            "Epoch 14/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.6657 - accuracy: 0.8204\n",
            "Epoch 15/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.5953 - accuracy: 0.8293\n",
            "Epoch 16/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.4909 - accuracy: 0.8680\n",
            "Epoch 17/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.4490 - accuracy: 0.8752\n",
            "Epoch 18/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.3522 - accuracy: 0.9026\n",
            "Epoch 19/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.3130 - accuracy: 0.9130\n",
            "Epoch 20/100\n",
            "32/32 [==============================] - 0s 15ms/step - loss: 0.2500 - accuracy: 0.9341\n",
            "Epoch 21/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.2255 - accuracy: 0.9398\n",
            "Epoch 22/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.2035 - accuracy: 0.9457\n",
            "Epoch 23/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1846 - accuracy: 0.9478\n",
            "Epoch 24/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1616 - accuracy: 0.9549\n",
            "Epoch 25/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1352 - accuracy: 0.9639\n",
            "Epoch 26/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1302 - accuracy: 0.9667\n",
            "Epoch 27/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1193 - accuracy: 0.9694\n",
            "Epoch 28/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.1106 - accuracy: 0.9699\n",
            "Epoch 29/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0810 - accuracy: 0.9792\n",
            "Epoch 30/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0836 - accuracy: 0.9768\n",
            "Epoch 31/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0727 - accuracy: 0.9818\n",
            "Epoch 32/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0729 - accuracy: 0.9823\n",
            "Epoch 33/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0706 - accuracy: 0.9810\n",
            "Epoch 34/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0645 - accuracy: 0.9825\n",
            "Epoch 35/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0633 - accuracy: 0.9829\n",
            "Epoch 36/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0537 - accuracy: 0.9871\n",
            "Epoch 37/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0503 - accuracy: 0.9866\n",
            "Epoch 38/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0540 - accuracy: 0.9860\n",
            "Epoch 39/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0624 - accuracy: 0.9819\n",
            "Epoch 40/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0531 - accuracy: 0.9855\n",
            "Epoch 41/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0448 - accuracy: 0.9881\n",
            "Epoch 42/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0445 - accuracy: 0.9883\n",
            "Epoch 43/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0340 - accuracy: 0.9904\n",
            "Epoch 44/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0427 - accuracy: 0.9871\n",
            "Epoch 45/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0391 - accuracy: 0.9895\n",
            "Epoch 46/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0416 - accuracy: 0.9885\n",
            "Epoch 47/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0496 - accuracy: 0.9840\n",
            "Epoch 48/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0579 - accuracy: 0.9843\n",
            "Epoch 49/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0452 - accuracy: 0.9870\n",
            "Epoch 50/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0414 - accuracy: 0.9879\n",
            "Epoch 51/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0258 - accuracy: 0.9933\n",
            "Epoch 52/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0389 - accuracy: 0.9887\n",
            "Epoch 53/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0371 - accuracy: 0.9879\n",
            "Epoch 54/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0360 - accuracy: 0.9891\n",
            "Epoch 55/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0260 - accuracy: 0.9920\n",
            "Epoch 56/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0354 - accuracy: 0.9906\n",
            "Epoch 57/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0319 - accuracy: 0.9909\n",
            "Epoch 58/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0297 - accuracy: 0.9907\n",
            "Epoch 59/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0284 - accuracy: 0.9910\n",
            "Epoch 60/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0311 - accuracy: 0.9917\n",
            "Epoch 61/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0293 - accuracy: 0.9914\n",
            "Epoch 62/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0331 - accuracy: 0.9896\n",
            "Epoch 63/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0263 - accuracy: 0.9923\n",
            "Epoch 64/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0284 - accuracy: 0.9915\n",
            "Epoch 65/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0257 - accuracy: 0.9917\n",
            "Epoch 66/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0270 - accuracy: 0.9927\n",
            "Epoch 67/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0249 - accuracy: 0.9934\n",
            "Epoch 68/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0269 - accuracy: 0.9921\n",
            "Epoch 69/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0201 - accuracy: 0.9943\n",
            "Epoch 70/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0317 - accuracy: 0.9904\n",
            "Epoch 71/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0191 - accuracy: 0.9950\n",
            "Epoch 72/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0161 - accuracy: 0.9957\n",
            "Epoch 73/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0224 - accuracy: 0.9940\n",
            "Epoch 74/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0185 - accuracy: 0.9944\n",
            "Epoch 75/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0264 - accuracy: 0.9916\n",
            "Epoch 76/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0271 - accuracy: 0.9916\n",
            "Epoch 77/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0252 - accuracy: 0.9917\n",
            "Epoch 78/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0214 - accuracy: 0.9932\n",
            "Epoch 79/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0180 - accuracy: 0.9947\n",
            "Epoch 80/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0141 - accuracy: 0.9954\n",
            "Epoch 81/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0148 - accuracy: 0.9964\n",
            "Epoch 82/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0194 - accuracy: 0.9933\n",
            "Epoch 83/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0213 - accuracy: 0.9936\n",
            "Epoch 84/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0203 - accuracy: 0.9938\n",
            "Epoch 85/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0223 - accuracy: 0.9947\n",
            "Epoch 86/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0239 - accuracy: 0.9933\n",
            "Epoch 87/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0201 - accuracy: 0.9947\n",
            "Epoch 88/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0225 - accuracy: 0.9933\n",
            "Epoch 89/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0253 - accuracy: 0.9910\n",
            "Epoch 90/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0304 - accuracy: 0.9927\n",
            "Epoch 91/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0201 - accuracy: 0.9931\n",
            "Epoch 92/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0179 - accuracy: 0.9950\n",
            "Epoch 93/100\n",
            "32/32 [==============================] - 1s 17ms/step - loss: 0.0211 - accuracy: 0.9934\n",
            "Epoch 94/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0205 - accuracy: 0.9940\n",
            "Epoch 95/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0208 - accuracy: 0.9942\n",
            "Epoch 96/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0206 - accuracy: 0.9933\n",
            "Epoch 97/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0246 - accuracy: 0.9924\n",
            "Epoch 98/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0194 - accuracy: 0.9942\n",
            "Epoch 99/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0233 - accuracy: 0.9935\n",
            "Epoch 100/100\n",
            "32/32 [==============================] - 1s 16ms/step - loss: 0.0210 - accuracy: 0.9936\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb3323ab208>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 88
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gghVXeoSCh7u",
        "outputId": "dc5fd0d6-584b-4427-a9f4-0aff65123982"
      },
      "source": [
        "# Evaluer le modèle3\r\n",
        "model_cnn3.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 89,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 0s 2ms/step - loss: 14.0731 - accuracy: 0.2470\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[14.073052406311035, 0.24699999392032623]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 89
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JWqvNb5Jp1Uo"
      },
      "source": [
        "# LSTM pour la classification de texte\n",
        "\n",
        "Il est également possible d'utiliser un autre type de réseau de neurones pour effectuer ce genre de tâches: les réseaux de neurones récurrents ou RNN.\n",
        "\n",
        "Les RNN sont conçus pour gérer les séquences. Le réseau prend les tokens un par un et calcule une représentation de la séquence à chaque pas qui tiens compte de tous les pas précédents \n",
        "\n",
        "![Texte alternatif…](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/Recurrent_neural_network_unfold.svg/450px-Recurrent_neural_network_unfold.svg.png)\n",
        "\n",
        "\n",
        "Il existe différents types de RNN. Ici nous utiliserons les Long Short-Term Memory (LSTM) qui permettent d'améliorer les performances sur des séquences longues avec une série de \"gates\". \n",
        "\n",
        "![Texte alternatif…](http://dprogrammer.org/wp-content/uploads/2019/04/RNN-vs-LSTM-vs-GRU-1200x361.png)"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "a6mO7VTjocwk",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "999f2a12-5c84-47a3-fb26-d24264fc8928"
      },
      "source": [
        "# Créer un réseau à base de LSTM avec au minimum:\n",
        "# Embedding\n",
        "# Dropout\n",
        "# LSTM\n",
        "# Dropout\n",
        "# Classifieur\n",
        "\n",
        "#Modèle lstm1\n",
        "\n",
        "lstm_size= 128\n",
        "dropout2= 0.2\n",
        "\n",
        "model_lstm1 = tf.keras.models.Sequential()\n",
        "# Embedding \n",
        "model_lstm1.add(tf.keras.layers.Embedding(vocab_size, \n",
        "                                        embed_dim, \n",
        "                                        input_length= max_len))\n",
        "# Dropout\n",
        "model_lstm1.add(tf.keras.layers.Dropout(dropout1))\n",
        "# LSTM\n",
        "model_lstm1.add(tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(lstm_size //2)))\n",
        "model_lstm1.add(tf.keras.layers.Dropout(dropout2))\n",
        "# Dense\n",
        "model_lstm1.add(tf.keras.layers.Dense(dense_size,  activation='relu'))\n",
        "\n",
        "# Classifieur (Dense + activation softmax)\n",
        "model_lstm1.add(tf.keras.layers.Dense(num_cat))\n",
        "model_lstm1.add(tf.keras.layers.Activation('softmax'))\n",
        "\n",
        "# Compiler le modèle \n",
        "model_lstm1.compile(loss='sparse_categorical_crossentropy', \n",
        "                  optimizer='adam', \n",
        "                  metrics= ['accuracy'])\n",
        "\n",
        "# Afficher le summary du modèle\n",
        "print(model_lstm1.summary())\n"
      ],
      "execution_count": 92,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_7\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_7 (Embedding)      (None, 39, 128)           1280000   \n",
            "_________________________________________________________________\n",
            "dropout_7 (Dropout)          (None, 39, 128)           0         \n",
            "_________________________________________________________________\n",
            "bidirectional (Bidirectional (None, 128)               98816     \n",
            "_________________________________________________________________\n",
            "dropout_8 (Dropout)          (None, 128)               0         \n",
            "_________________________________________________________________\n",
            "dense_16 (Dense)             (None, 128)               16512     \n",
            "_________________________________________________________________\n",
            "dense_17 (Dense)             (None, 96)                12384     \n",
            "_________________________________________________________________\n",
            "activation_6 (Activation)    (None, 96)                0         \n",
            "=================================================================\n",
            "Total params: 1,407,712\n",
            "Trainable params: 1,407,712\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VphlgAhjqSCD",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "74e32357-35a7-48bd-f4c2-3ca1572ceb40"
      },
      "source": [
        "# Fitter le modèle lstm 1\n",
        "model_lstm1.fit(X_train_pad, Y_train_id, batch_size = batch_size , epochs = epochs)"
      ],
      "execution_count": 93,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/10\n",
            "63/63 [==============================] - 4s 20ms/step - loss: 4.0701 - accuracy: 0.1649\n",
            "Epoch 2/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 3.4261 - accuracy: 0.2003\n",
            "Epoch 3/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 3.0346 - accuracy: 0.2763\n",
            "Epoch 4/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 2.4951 - accuracy: 0.3722\n",
            "Epoch 5/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 2.1139 - accuracy: 0.4694\n",
            "Epoch 6/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.8207 - accuracy: 0.5344\n",
            "Epoch 7/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.5686 - accuracy: 0.5805\n",
            "Epoch 8/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.3453 - accuracy: 0.6517\n",
            "Epoch 9/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.1523 - accuracy: 0.7002\n",
            "Epoch 10/10\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.0484 - accuracy: 0.7209\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb391e51320>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 93
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "jVEY_kx3qSnS",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "a40f4578-2876-4520-eaad-6d496536fdcb"
      },
      "source": [
        "# Evaluer le modèle lstm 1\n",
        "model_lstm1.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 94,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 1s 4ms/step - loss: 4.1149 - accuracy: 0.3115\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[4.1148600578308105, 0.31150001287460327]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 94
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0I6dQHArZYJ4",
        "outputId": "7b125be8-ff2f-4825-8956-6a30a6d4fea8"
      },
      "source": [
        "\r\n",
        "#Modèle lstm2\r\n",
        "\r\n",
        "lstm_size= 128\r\n",
        "\r\n",
        "\r\n",
        "model_lstm2 = tf.keras.models.Sequential()\r\n",
        "# Embedding \r\n",
        "model_lstm2.add(tf.keras.layers.Embedding(vocab_size, \r\n",
        "                                        embed_dim2, \r\n",
        "                                        input_length= max_len))\r\n",
        "# Dropout\r\n",
        "model_lstm2.add(tf.keras.layers.Dropout(dropout1))\r\n",
        "# LSTM\r\n",
        "model_lstm2.add(tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(lstm_size //2)))\r\n",
        "model_lstm2.add(tf.keras.layers.Dropout(dropout3))\r\n",
        "# Dense\r\n",
        "model_lstm2.add(tf.keras.layers.Dense(dense_size,  activation='relu'))\r\n",
        "\r\n",
        "# Classifieur (Dense + activation softmax)\r\n",
        "model_lstm2.add(tf.keras.layers.Dense(num_cat))\r\n",
        "model_lstm2.add(tf.keras.layers.Activation('softmax'))\r\n",
        "\r\n",
        "# Compiler le modèle \r\n",
        "model_lstm2.compile(loss='sparse_categorical_crossentropy', \r\n",
        "                  optimizer='adam', \r\n",
        "                  metrics= ['accuracy'])\r\n",
        "\r\n",
        "# Afficher le summary du modèle\r\n",
        "print(model_lstm2.summary())\r\n"
      ],
      "execution_count": 95,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_8\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_8 (Embedding)      (None, 39, 128)           1280000   \n",
            "_________________________________________________________________\n",
            "dropout_9 (Dropout)          (None, 39, 128)           0         \n",
            "_________________________________________________________________\n",
            "bidirectional_1 (Bidirection (None, 128)               98816     \n",
            "_________________________________________________________________\n",
            "dropout_10 (Dropout)         (None, 128)               0         \n",
            "_________________________________________________________________\n",
            "dense_18 (Dense)             (None, 128)               16512     \n",
            "_________________________________________________________________\n",
            "dense_19 (Dense)             (None, 96)                12384     \n",
            "_________________________________________________________________\n",
            "activation_7 (Activation)    (None, 96)                0         \n",
            "=================================================================\n",
            "Total params: 1,407,712\n",
            "Trainable params: 1,407,712\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "TTiwBIuBagHL",
        "outputId": "0322ea56-7f3c-4863-b644-420abc05c2ed"
      },
      "source": [
        "# Fitter le modèle lstm 2\r\n",
        "model_lstm2.fit(X_train_pad, Y_train_id, batch_size = batch_size2 , epochs = epochs2)"
      ],
      "execution_count": 96,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/40\n",
            "63/63 [==============================] - 4s 20ms/step - loss: 4.0629 - accuracy: 0.1594\n",
            "Epoch 2/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 3.4029 - accuracy: 0.2018\n",
            "Epoch 3/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 3.0564 - accuracy: 0.2588\n",
            "Epoch 4/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 2.6565 - accuracy: 0.3274\n",
            "Epoch 5/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 2.2613 - accuracy: 0.4225\n",
            "Epoch 6/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.9726 - accuracy: 0.4974\n",
            "Epoch 7/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.7463 - accuracy: 0.5529\n",
            "Epoch 8/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.5587 - accuracy: 0.5947\n",
            "Epoch 9/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.3928 - accuracy: 0.6367\n",
            "Epoch 10/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.1857 - accuracy: 0.6854\n",
            "Epoch 11/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 1.0393 - accuracy: 0.7203\n",
            "Epoch 12/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.9387 - accuracy: 0.7492\n",
            "Epoch 13/40\n",
            "63/63 [==============================] - 1s 19ms/step - loss: 0.8292 - accuracy: 0.7770\n",
            "Epoch 14/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.7539 - accuracy: 0.7950\n",
            "Epoch 15/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.6450 - accuracy: 0.8265\n",
            "Epoch 16/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.6113 - accuracy: 0.8337\n",
            "Epoch 17/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.5059 - accuracy: 0.8607\n",
            "Epoch 18/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.4694 - accuracy: 0.8759\n",
            "Epoch 19/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.3954 - accuracy: 0.8972\n",
            "Epoch 20/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.3795 - accuracy: 0.8991\n",
            "Epoch 21/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.3370 - accuracy: 0.9084\n",
            "Epoch 22/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2975 - accuracy: 0.9162\n",
            "Epoch 23/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2730 - accuracy: 0.9264\n",
            "Epoch 24/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2405 - accuracy: 0.9316\n",
            "Epoch 25/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2306 - accuracy: 0.9383\n",
            "Epoch 26/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2126 - accuracy: 0.9431\n",
            "Epoch 27/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1909 - accuracy: 0.9449\n",
            "Epoch 28/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.2117 - accuracy: 0.9411\n",
            "Epoch 29/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1721 - accuracy: 0.9522\n",
            "Epoch 30/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1735 - accuracy: 0.9538\n",
            "Epoch 31/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1593 - accuracy: 0.9556\n",
            "Epoch 32/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1513 - accuracy: 0.9556\n",
            "Epoch 33/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1407 - accuracy: 0.9653\n",
            "Epoch 34/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1249 - accuracy: 0.9647\n",
            "Epoch 35/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1048 - accuracy: 0.9743\n",
            "Epoch 36/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1220 - accuracy: 0.9664\n",
            "Epoch 37/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1125 - accuracy: 0.9692\n",
            "Epoch 38/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1044 - accuracy: 0.9703\n",
            "Epoch 39/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.0991 - accuracy: 0.9715\n",
            "Epoch 40/40\n",
            "63/63 [==============================] - 1s 20ms/step - loss: 0.1086 - accuracy: 0.9711\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb3a6375630>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 96
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VqXKj98WauxK",
        "outputId": "3941b03d-e415-42a9-aaa9-0dcd482a37f2"
      },
      "source": [
        "# Evaluer le modèle lstm 2\r\n",
        "model_lstm2.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 97,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 1s 4ms/step - loss: 9.5086 - accuracy: 0.2820\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[9.50861930847168, 0.28200000524520874]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 97
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2H_bLhp5aTOS",
        "outputId": "e65063fe-4405-478c-8b4b-376070f82872"
      },
      "source": [
        "\r\n",
        "#Modèle lstm3\r\n",
        "\r\n",
        "lstm_size= 128\r\n",
        "\r\n",
        "\r\n",
        "model_lstm3 = tf.keras.models.Sequential()\r\n",
        "# Embedding \r\n",
        "model_lstm3.add(tf.keras.layers.Embedding(vocab_size, \r\n",
        "                                        embed_dim3, \r\n",
        "                                        input_length= max_len))\r\n",
        "# Dropout\r\n",
        "model_lstm3.add(tf.keras.layers.Dropout(dropout2))\r\n",
        "# LSTM\r\n",
        "model_lstm3.add(tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(lstm_size //2)))\r\n",
        "model_lstm3.add(tf.keras.layers.Dropout(dropout3))\r\n",
        "# Dense\r\n",
        "model_lstm3.add(tf.keras.layers.Dense(dense_size3,  activation='elu'))\r\n",
        "\r\n",
        "# Classifieur (Dense + activation softmax)\r\n",
        "model_lstm3.add(tf.keras.layers.Dense(num_cat))\r\n",
        "model_lstm3.add(tf.keras.layers.Activation('softmax'))\r\n",
        "\r\n",
        "# Compiler le modèle \r\n",
        "model_lstm3.compile(loss='sparse_categorical_crossentropy', \r\n",
        "                  optimizer='adam', \r\n",
        "                  metrics= ['accuracy'])\r\n",
        "\r\n",
        "# Afficher le summary du modèle\r\n",
        "print(model_lstm3.summary())\r\n"
      ],
      "execution_count": 98,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_9\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_9 (Embedding)      (None, 39, 128)           1280000   \n",
            "_________________________________________________________________\n",
            "dropout_11 (Dropout)         (None, 39, 128)           0         \n",
            "_________________________________________________________________\n",
            "bidirectional_2 (Bidirection (None, 128)               98816     \n",
            "_________________________________________________________________\n",
            "dropout_12 (Dropout)         (None, 128)               0         \n",
            "_________________________________________________________________\n",
            "dense_20 (Dense)             (None, 256)               33024     \n",
            "_________________________________________________________________\n",
            "dense_21 (Dense)             (None, 96)                24672     \n",
            "_________________________________________________________________\n",
            "activation_8 (Activation)    (None, 96)                0         \n",
            "=================================================================\n",
            "Total params: 1,436,512\n",
            "Trainable params: 1,436,512\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9PQ7NYhLajFP",
        "outputId": "54b17473-fa5a-493a-df79-9e6769b3dca7"
      },
      "source": [
        "# Fitter le modèle lstm 3\r\n",
        "model_lstm3.fit(X_train_pad, Y_train_id, batch_size = batch_size3 , epochs = epochs3)"
      ],
      "execution_count": 99,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/100\n",
            "32/32 [==============================] - 3s 23ms/step - loss: 4.1598 - accuracy: 0.1679\n",
            "Epoch 2/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 3.4549 - accuracy: 0.1875\n",
            "Epoch 3/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 3.2635 - accuracy: 0.2402\n",
            "Epoch 4/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 2.9583 - accuracy: 0.2602\n",
            "Epoch 5/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 2.5994 - accuracy: 0.3313\n",
            "Epoch 6/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 2.2430 - accuracy: 0.4238\n",
            "Epoch 7/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.9756 - accuracy: 0.4826\n",
            "Epoch 8/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.7151 - accuracy: 0.5555\n",
            "Epoch 9/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.5464 - accuracy: 0.5872\n",
            "Epoch 10/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.3673 - accuracy: 0.6349\n",
            "Epoch 11/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.1780 - accuracy: 0.6766\n",
            "Epoch 12/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 1.0436 - accuracy: 0.7201\n",
            "Epoch 13/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.9202 - accuracy: 0.7508\n",
            "Epoch 14/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.8423 - accuracy: 0.7679\n",
            "Epoch 15/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.7229 - accuracy: 0.8091\n",
            "Epoch 16/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.6688 - accuracy: 0.8152\n",
            "Epoch 17/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.5863 - accuracy: 0.8419\n",
            "Epoch 18/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.4963 - accuracy: 0.8640\n",
            "Epoch 19/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.4563 - accuracy: 0.8754\n",
            "Epoch 20/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.4306 - accuracy: 0.8836\n",
            "Epoch 21/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.3741 - accuracy: 0.9005\n",
            "Epoch 22/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.3447 - accuracy: 0.9089\n",
            "Epoch 23/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.3144 - accuracy: 0.9119\n",
            "Epoch 24/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.2852 - accuracy: 0.9188\n",
            "Epoch 25/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.2488 - accuracy: 0.9342\n",
            "Epoch 26/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.2582 - accuracy: 0.9240\n",
            "Epoch 27/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.2244 - accuracy: 0.9418\n",
            "Epoch 28/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.2236 - accuracy: 0.9397\n",
            "Epoch 29/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.2050 - accuracy: 0.9408\n",
            "Epoch 30/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.2056 - accuracy: 0.9410\n",
            "Epoch 31/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1945 - accuracy: 0.9437\n",
            "Epoch 32/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.1761 - accuracy: 0.9482\n",
            "Epoch 33/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.1612 - accuracy: 0.9559\n",
            "Epoch 34/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1487 - accuracy: 0.9581\n",
            "Epoch 35/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1327 - accuracy: 0.9629\n",
            "Epoch 36/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1191 - accuracy: 0.9654\n",
            "Epoch 37/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1126 - accuracy: 0.9679\n",
            "Epoch 38/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.1081 - accuracy: 0.9678\n",
            "Epoch 39/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.1022 - accuracy: 0.9723\n",
            "Epoch 40/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0997 - accuracy: 0.9708\n",
            "Epoch 41/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1015 - accuracy: 0.9719\n",
            "Epoch 42/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0998 - accuracy: 0.9727\n",
            "Epoch 43/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0914 - accuracy: 0.9763\n",
            "Epoch 44/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0904 - accuracy: 0.9780\n",
            "Epoch 45/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0799 - accuracy: 0.9759\n",
            "Epoch 46/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.1188 - accuracy: 0.9641\n",
            "Epoch 47/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.1001 - accuracy: 0.9717\n",
            "Epoch 48/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0916 - accuracy: 0.9748\n",
            "Epoch 49/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.0681 - accuracy: 0.9829\n",
            "Epoch 50/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0742 - accuracy: 0.9804\n",
            "Epoch 51/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0617 - accuracy: 0.9813\n",
            "Epoch 52/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0702 - accuracy: 0.9800\n",
            "Epoch 53/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0646 - accuracy: 0.9822\n",
            "Epoch 54/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0641 - accuracy: 0.9810\n",
            "Epoch 55/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0594 - accuracy: 0.9829\n",
            "Epoch 56/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0728 - accuracy: 0.9795\n",
            "Epoch 57/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0699 - accuracy: 0.9796\n",
            "Epoch 58/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0699 - accuracy: 0.9802\n",
            "Epoch 59/100\n",
            "32/32 [==============================] - 1s 21ms/step - loss: 0.0625 - accuracy: 0.9812\n",
            "Epoch 60/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0547 - accuracy: 0.9851\n",
            "Epoch 61/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0425 - accuracy: 0.9893\n",
            "Epoch 62/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0455 - accuracy: 0.9877\n",
            "Epoch 63/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0424 - accuracy: 0.9884\n",
            "Epoch 64/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0477 - accuracy: 0.9870\n",
            "Epoch 65/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0440 - accuracy: 0.9868\n",
            "Epoch 66/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0419 - accuracy: 0.9877\n",
            "Epoch 67/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0436 - accuracy: 0.9867\n",
            "Epoch 68/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0440 - accuracy: 0.9883\n",
            "Epoch 69/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0418 - accuracy: 0.9872\n",
            "Epoch 70/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0606 - accuracy: 0.9838\n",
            "Epoch 71/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0595 - accuracy: 0.9812\n",
            "Epoch 72/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0398 - accuracy: 0.9891\n",
            "Epoch 73/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0377 - accuracy: 0.9894\n",
            "Epoch 74/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0347 - accuracy: 0.9903\n",
            "Epoch 75/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0388 - accuracy: 0.9870\n",
            "Epoch 76/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0468 - accuracy: 0.9872\n",
            "Epoch 77/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0442 - accuracy: 0.9883\n",
            "Epoch 78/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0698 - accuracy: 0.9800\n",
            "Epoch 79/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0483 - accuracy: 0.9859\n",
            "Epoch 80/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0541 - accuracy: 0.9853\n",
            "Epoch 81/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0365 - accuracy: 0.9870\n",
            "Epoch 82/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0399 - accuracy: 0.9875\n",
            "Epoch 83/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0367 - accuracy: 0.9898\n",
            "Epoch 84/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0403 - accuracy: 0.9886\n",
            "Epoch 85/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0352 - accuracy: 0.9888\n",
            "Epoch 86/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0351 - accuracy: 0.9898\n",
            "Epoch 87/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0340 - accuracy: 0.9891\n",
            "Epoch 88/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0287 - accuracy: 0.9926\n",
            "Epoch 89/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0251 - accuracy: 0.9912\n",
            "Epoch 90/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0297 - accuracy: 0.9927\n",
            "Epoch 91/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0281 - accuracy: 0.9907\n",
            "Epoch 92/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0349 - accuracy: 0.9883\n",
            "Epoch 93/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0381 - accuracy: 0.9905\n",
            "Epoch 94/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0371 - accuracy: 0.9889\n",
            "Epoch 95/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0621 - accuracy: 0.9841\n",
            "Epoch 96/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0502 - accuracy: 0.9843\n",
            "Epoch 97/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0422 - accuracy: 0.9874\n",
            "Epoch 98/100\n",
            "32/32 [==============================] - 1s 22ms/step - loss: 0.0378 - accuracy: 0.9892\n",
            "Epoch 99/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0217 - accuracy: 0.9950\n",
            "Epoch 100/100\n",
            "32/32 [==============================] - 1s 23ms/step - loss: 0.0247 - accuracy: 0.9928\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7fb3a59c1160>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 99
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1dRNfKikazOQ",
        "outputId": "19dee100-a913-4f1b-adb7-1e7c095e0567"
      },
      "source": [
        "# Evaluer le modèle lstm 3\r\n",
        "model_lstm3.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 100,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 1s 4ms/step - loss: 11.6791 - accuracy: 0.2710\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[11.67905044555664, 0.2709999978542328]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 100
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9x28y_mZsnbx"
      },
      "source": [
        "# Utiliser les embeddings pré-entrainés\n",
        "\n",
        "Pour améliorer la qualité de la représentation des mots, il est possible d'entrainer les embeddings sur de larges corpus de textes non annotés (typiquement Wikipedia). Ces modèles sont souvent disponibles en ligne et il est possible de les télécharger. Ici nous allons utiliser des embeddings [Glove](https://nlp.stanford.edu/projects/glove/) de taille 50d (pour des raisons techniques mais il vaut mieux utiliser des dimensions plus importantes entre 100 et 300) "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "TXZjI4xX1xn7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "0eb2f3d4-2b9c-4402-a76c-d88aa89f5389"
      },
      "source": [
        "# Fonction permettant de charger un embedding \n",
        "\n",
        "import numpy as np\n",
        "import re\n",
        "from nltk.tokenize import word_tokenize\n",
        "import nltk\n",
        "nltk.download('punkt')\n",
        "\n",
        "def load_glove_embeddings(fp, embedding_dim, include_empty_char=True):\n",
        "    \"\"\"\n",
        "    Loads pre-trained word embeddings (GloVe embeddings)\n",
        "        Inputs: - fp: filepath of pre-trained glove embeddings\n",
        "                - embedding_dim: dimension of each vector embedding\n",
        "                - generate_matrix: whether to generate an embedding matrix\n",
        "        Outputs:\n",
        "                - word2coefs: Dictionary. Word to its corresponding coefficients\n",
        "                - word2index: Dictionary. Word to word-index\n",
        "                - embedding_matrix: Embedding matrix for Keras Embedding layer\n",
        "    \"\"\"\n",
        "    # First, build the \"word2coefs\" and \"word2index\"\n",
        "    word2coefs = {} # word to its corresponding coefficients\n",
        "    word2index = {} # word to word-index\n",
        "    with open(fp) as f:\n",
        "        for idx, line in enumerate(f):\n",
        "            try:\n",
        "                data = [x.strip().lower() for x in line.split()]\n",
        "                word = data[0]\n",
        "                coefs = np.asarray(data[1:embedding_dim+1], dtype='float32')\n",
        "                word2coefs[word] = coefs\n",
        "                if word not in word2index:\n",
        "                    word2index[word] = len(word2index)\n",
        "            except Exception as e:\n",
        "                print('Exception occurred in `load_glove_embeddings`:', e)\n",
        "                continue\n",
        "        # End of for loop.\n",
        "    # End of with open\n",
        "    if include_empty_char:\n",
        "        word2index[''] = len(word2index)\n",
        "    # Second, build the \"embedding_matrix\"\n",
        "    # Words not found in embedding index will be all-zeros. Hence, the \"+1\".\n",
        "    vocab_size = len(word2coefs)+1 if include_empty_char else len(word2coefs)\n",
        "    embedding_matrix = np.zeros((vocab_size, embedding_dim))\n",
        "    for word, idx in word2index.items():\n",
        "        embedding_vec = word2coefs.get(word)\n",
        "        if embedding_vec is not None and embedding_vec.shape[0]==embedding_dim:\n",
        "            embedding_matrix[idx] = np.asarray(embedding_vec)\n",
        "    # return word2coefs, word2index, embedding_matrix\n",
        "    return word2index, np.asarray(embedding_matrix)"
      ],
      "execution_count": 101,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
            "[nltk_data]   Unzipping tokenizers/punkt.zip.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "uiWQhLQ81yaA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "efd2760a-f731-4f73-97d9-17d4fe52e6d6"
      },
      "source": [
        "# Télécharger les embeddings\n",
        "\n",
        "!wget https://github.com/kmr0877/IMDB-Sentiment-Classification-CBOW-Model/raw/master/glove.6B.50d.txt.gz\n",
        "!gunzip /content/glove.6B.50d.txt.gz"
      ],
      "execution_count": 102,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "--2021-01-03 13:42:45--  https://github.com/kmr0877/IMDB-Sentiment-Classification-CBOW-Model/raw/master/glove.6B.50d.txt.gz\n",
            "Resolving github.com (github.com)... 192.30.255.112\n",
            "Connecting to github.com (github.com)|192.30.255.112|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://raw.githubusercontent.com/kmr0877/IMDB-Sentiment-Classification-CBOW-Model/master/glove.6B.50d.txt.gz [following]\n",
            "--2021-01-03 13:42:45--  https://raw.githubusercontent.com/kmr0877/IMDB-Sentiment-Classification-CBOW-Model/master/glove.6B.50d.txt.gz\n",
            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 69182520 (66M) [application/octet-stream]\n",
            "Saving to: ‘glove.6B.50d.txt.gz’\n",
            "\n",
            "glove.6B.50d.txt.gz 100%[===================>]  65.98M   147MB/s    in 0.4s    \n",
            "\n",
            "2021-01-03 13:42:47 (147 MB/s) - ‘glove.6B.50d.txt.gz’ saved [69182520/69182520]\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 35
        },
        "id": "BCNUDglsopPP",
        "outputId": "180040ec-2298-4db3-fbd4-9edd9a16d5fc"
      },
      "source": [
        "pwd"
      ],
      "execution_count": 109,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "string"
            },
            "text/plain": [
              "'/content'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 109
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "a0CPaa_r3B3w",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9974e580-ab4a-4a9d-ba69-8e92580eb798"
      },
      "source": [
        "# Charger les embeddings à l'aide de la fonction load_glove_embeddings\n",
        "\n",
        "embeddings_index = dict()\n",
        "f = open('/content/glove.6B.50d.txt')\n",
        "\n",
        "for line in f:\n",
        "    values = line.split()\n",
        "    word = values[0]\n",
        "    coefs = np.asarray(values[1:], dtype='float32')\n",
        "    embeddings_index[word] = coefs\n",
        "\n",
        "f.close()\n",
        "print('Loaded %s word vectors.' % len(embeddings_index))"
      ],
      "execution_count": 110,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Loaded 400000 word vectors.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "x1T2Kdot3GKF"
      },
      "source": [
        "# ecrire une fonction de tokenization custom pour preprocesser les textes\n",
        "import numpy as np\n",
        "from keras.preprocessing.text import Tokenizer\n",
        "\n",
        "size_of_vocabulary = 400000\n",
        "\n",
        "\n",
        "embedding_matrix = np.zeros((size_of_vocabulary, 50))\n",
        "\n",
        "for word, i in tokenizer.word_index.items():\n",
        "    embedding_vector = embeddings_index.get(word)\n",
        "    if embedding_vector is not None:\n",
        "        embedding_matrix[i] = embedding_vector\n",
        "# Encoder les textes avec la fonction custom\n"
      ],
      "execution_count": 5,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "eF40dcAH4HEY"
      },
      "source": [
        "# Padding des sequences\n"
      ],
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "lqxsq-rY527k",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "9e1bc562-f03b-4703-8e03-0e1bd8746065"
      },
      "source": [
        "# Créer un modèle en chargeant les poids des embeddings dans le layer Embedding\n",
        "\n",
        "\n",
        "#Modèle lstm2\n",
        "\n",
        "lstm_size= 128\n",
        "embed_dim2= 128\n",
        "dropout2 = 0.4\n",
        "conv_filters2= 32\n",
        "conv_kernel2 = 2\n",
        "maxpool_size2 = 4\n",
        "dense_size2 = 128\n",
        "batch_size2 = 128\n",
        "epochs2 = 40\n",
        "\n",
        "model_lstmp = tf.keras.models.Sequential()\n",
        "# Embedding \n",
        "model_lstmp.add(tf.keras.layers.Embedding(size_of_vocabulary, \n",
        "                                        50, \n",
        "                                        weights=[embedding_matrix],\n",
        "                                        input_length=100,\n",
        "                                        trainable=False))\n",
        "# Dropout\n",
        "model_lstmp.add(tf.keras.layers.Dropout(dropout1))\n",
        "# LSTM\n",
        "model_lstmp.add(tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(lstm_size //2)))\n",
        "model_lstmp.add(tf.keras.layers.Dropout(dropout2))\n",
        "# Dense\n",
        "model_lstmp.add(tf.keras.layers.Dense(dense_size,  activation='relu'))\n",
        "\n",
        "# Classifieur (Dense + activation softmax)\n",
        "model_lstmp.add(tf.keras.layers.Dense(64))\n",
        "model_lstmp.add(tf.keras.layers.Activation('softmax'))\n",
        "\n",
        "# Compiler le modèle \n",
        "model_lstmp.compile(loss='sparse_categorical_crossentropy', \n",
        "                  optimizer='adam', \n",
        "                  metrics= ['accuracy'])\n",
        "\n",
        "# Afficher le summary du modèle\n",
        "print(model_lstmp.summary())\n"
      ],
      "execution_count": 12,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Model: \"sequential_4\"\n",
            "_________________________________________________________________\n",
            "Layer (type)                 Output Shape              Param #   \n",
            "=================================================================\n",
            "embedding_3 (Embedding)      (None, 100, 50)           20000000  \n",
            "_________________________________________________________________\n",
            "dropout_3 (Dropout)          (None, 100, 50)           0         \n",
            "_________________________________________________________________\n",
            "bidirectional_2 (Bidirection (None, 128)               58880     \n",
            "_________________________________________________________________\n",
            "dropout_4 (Dropout)          (None, 128)               0         \n",
            "_________________________________________________________________\n",
            "dense_1 (Dense)              (None, 128)               16512     \n",
            "_________________________________________________________________\n",
            "dense_2 (Dense)              (None, 64)                8256      \n",
            "_________________________________________________________________\n",
            "activation (Activation)      (None, 64)                0         \n",
            "=================================================================\n",
            "Total params: 20,083,648\n",
            "Trainable params: 83,648\n",
            "Non-trainable params: 20,000,000\n",
            "_________________________________________________________________\n",
            "None\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8VTS-j-v6RLV",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "6d59464e-a28d-44cb-c21e-b1c8a35e36fa"
      },
      "source": [
        "# Fitter le modèle\n",
        "\n",
        "model_lstmp.fit(X_train_pad, Y_train_id, batch_size = batch_size2 , epochs = 100)"
      ],
      "execution_count": 41,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Epoch 1/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 2/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 3/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 4/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 5/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 6/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 7/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 8/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 9/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 10/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 11/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 12/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 13/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 14/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 15/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 16/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 17/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 18/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 19/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 20/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 21/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 22/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 23/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 24/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 25/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 26/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 27/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 28/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 29/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 30/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 31/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 32/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 33/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 34/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 35/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 36/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 37/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 38/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 39/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 40/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 41/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 42/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 43/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 44/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 45/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 46/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 47/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 48/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 49/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 50/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 51/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 52/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 53/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 54/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 55/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 56/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 57/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 58/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 59/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 60/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 61/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 62/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 63/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 64/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 65/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 66/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 67/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 68/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 69/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 70/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 71/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 72/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 73/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 74/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 75/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 76/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 77/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 78/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 79/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 80/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 81/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 82/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 83/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 84/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 85/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 86/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 87/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 88/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 89/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 90/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 91/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 92/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 93/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 94/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 95/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 96/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 97/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 98/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 99/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n",
            "Epoch 100/100\n",
            "63/63 [==============================] - 0s 7ms/step - loss: nan - accuracy: 0.0000e+00\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<tensorflow.python.keras.callbacks.History at 0x7f0dc8031400>"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 41
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ikDrLZyC6b1c",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c49e4c79-94ba-4f3f-9c8f-5329f587c620"
      },
      "source": [
        "# evaluer le modèle\n",
        "model_lstmp.evaluate(X_test_pad, Y_test_id)"
      ],
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "63/63 [==============================] - 0s 4ms/step - loss: nan - accuracy: 0.0010\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[nan, 0.0010000000474974513]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 42
        }
      ]
    }
  ]
}